<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-03-26" />

<title>CorMotif_data</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">CorMotif_data</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-03-26</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-03-26
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtreedc26b1fd5099cd54f5245221a05c7e2e270820a0targetblankdc26b1fa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/dc26b1fd5099cd54f5245221a05c7e2e270820a0" target="_blank">dc26b1f</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtreedc26b1fd5099cd54f5245221a05c7e2e270820a0targetblankdc26b1fa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/dc26b1fd5099cd54f5245221a05c7e2e270820a0" target="_blank">dc26b1f</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/corMotifcustom.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/CorMotif_data.Rmd</code>) and
HTML (<code>docs/CorMotif_data.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/73d52a5b380b9b59331eb36476ceaa87663cd0d7/analysis/CorMotif_data.Rmd" target="_blank">73d52a5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-25
</td>
<td>
adding in new analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/5acc25545813111cee407a9ab042a3f3f7b13c57/docs/CorMotif_data.html" target="_blank">5acc255</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/f6beadb5477cdd0839924f565bd215a0512b453c/analysis/CorMotif_data.Rmd" target="_blank">f6beadb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-22
</td>
<td>
updates to DAR assessments
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/1650f780323d762f550b181742dba4cddf8877e7/docs/CorMotif_data.html" target="_blank">1650f78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/f017b87a070e6001b497481b1c7616d43604a542/analysis/CorMotif_data.Rmd" target="_blank">f017b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
<td>
adding Cormotif update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/32aa1a1d4d7d21ca39d1ea160ec98c9408d09bd9/docs/CorMotif_data.html" target="_blank">32aa1a1</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b64480b231a4bd357f8b785e3ccc9c6c7537df6e/analysis/CorMotif_data.Rmd" target="_blank">b64480b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
updates to cormotif data
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/25abe4aa5409f2364f0f91fc2b37b8e74e0b48fc/docs/CorMotif_data.html" target="_blank">25abe4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/721572e35a1f746b3351d2a8e7f7c854c4a15fde/analysis/CorMotif_data.Rmd" target="_blank">721572e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
adding in first cormotif
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cf2d4069cebb83f2cb80285db29220f628e246bb/analysis/CorMotif_data.Rmd" target="_blank">cf2d406</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
adding cormotif
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)

library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
library(bedr)</code></pre>
<pre class="r"><code>## Fit limma model using code as it is found in the original cormotif code. It has
## only been modified to add names to the matrix of t values, as well as the
## limma fits

limmafit.default &lt;- function(exprs,groupid,compid) {
  limmafits  &lt;- list()
  compnum    &lt;- nrow(compid)
  genenum    &lt;- nrow(exprs)
  limmat     &lt;- matrix(0,genenum,compnum)
  limmas2    &lt;- rep(0,compnum)
  limmadf    &lt;- rep(0,compnum)
  limmav0    &lt;- rep(0,compnum)
  limmag1num &lt;- rep(0,compnum)
  limmag2num &lt;- rep(0,compnum)

  rownames(limmat)  &lt;- rownames(exprs)
  colnames(limmat)  &lt;- rownames(compid)
  names(limmas2)    &lt;- rownames(compid)
  names(limmadf)    &lt;- rownames(compid)
  names(limmav0)    &lt;- rownames(compid)
  names(limmag1num) &lt;- rownames(compid)
  names(limmag2num) &lt;- rownames(compid)

  for(i in 1:compnum) {
    selid1 &lt;- which(groupid == compid[i,1])
    selid2 &lt;- which(groupid == compid[i,2])
    eset   &lt;- new(&quot;ExpressionSet&quot;, exprs=cbind(exprs[,selid1],exprs[,selid2]))
    g1num  &lt;- length(selid1)
    g2num  &lt;- length(selid2)
    designmat &lt;- cbind(base=rep(1,(g1num+g2num)), delta=c(rep(0,g1num),rep(1,g2num)))
    fit &lt;- lmFit(eset,designmat)
    fit &lt;- eBayes(fit)
    limmat[,i] &lt;- fit$t[,2]
    limmas2[i] &lt;- fit$s2.prior
    limmadf[i] &lt;- fit$df.prior
    limmav0[i] &lt;- fit$var.prior[2]
    limmag1num[i] &lt;- g1num
    limmag2num[i] &lt;- g2num
    limmafits[[i]] &lt;- fit

    # log odds
    # w&lt;-sqrt(1+fit$var.prior[2]/(1/g1num+1/g2num))
    # log(0.99)+dt(fit$t[1,2],g1num+g2num-2+fit$df.prior,log=TRUE)-log(0.01)-dt(fit$t[1,2]/w, g1num+g2num-2+fit$df.prior, log=TRUE)+log(w)
  }
  names(limmafits) &lt;- rownames(compid)
  limmacompnum&lt;-nrow(compid)
  result&lt;-list(t       = limmat,
               v0      = limmav0,
               df0     = limmadf,
               s20     = limmas2,
               g1num   = limmag1num,
               g2num   = limmag2num,
               compnum = limmacompnum,
               fits    = limmafits)
}

limmafit.counts &lt;-
  function (exprs, groupid, compid, norm.factor.method = &quot;TMM&quot;, voom.normalize.method = &quot;none&quot;)
  {
    limmafits  &lt;- list()
    compnum    &lt;- nrow(compid)
    genenum    &lt;- nrow(exprs)
    limmat     &lt;- matrix(NA,genenum,compnum)
    limmas2    &lt;- rep(0,compnum)
    limmadf    &lt;- rep(0,compnum)
    limmav0    &lt;- rep(0,compnum)
    limmag1num &lt;- rep(0,compnum)
    limmag2num &lt;- rep(0,compnum)

    rownames(limmat)  &lt;- rownames(exprs)
    colnames(limmat)  &lt;- rownames(compid)
    names(limmas2)    &lt;- rownames(compid)
    names(limmadf)    &lt;- rownames(compid)
    names(limmav0)    &lt;- rownames(compid)
    names(limmag1num) &lt;- rownames(compid)
    names(limmag2num) &lt;- rownames(compid)

    for (i in 1:compnum) {
      message(paste(&quot;Running limma for comparision&quot;,i,&quot;/&quot;,compnum))
      selid1 &lt;- which(groupid == compid[i, 1])
      selid2 &lt;- which(groupid == compid[i, 2])
      # make a new count data frame
      counts &lt;- cbind(exprs[, selid1], exprs[, selid2])

      # remove NAs
      not.nas &lt;- which(apply(counts, 1, function(x) !any(is.na(x))) == TRUE)

      # runn voom/limma
      d &lt;- DGEList(counts[not.nas,])
      d &lt;- calcNormFactors(d, method = norm.factor.method)
      g1num &lt;- length(selid1)
      g2num &lt;- length(selid2)
      designmat &lt;- cbind(base = rep(1, (g1num + g2num)), delta = c(rep(0,
                                                                       g1num), rep(1, g2num)))

      y &lt;- voom(d, designmat, normalize.method = voom.normalize.method)
      fit &lt;- lmFit(y, designmat)
      fit &lt;- eBayes(fit)

      limmafits[[i]] &lt;- fit
      limmat[not.nas, i] &lt;- fit$t[, 2]
      limmas2[i] &lt;- fit$s2.prior
      limmadf[i] &lt;- fit$df.prior
      limmav0[i] &lt;- fit$var.prior[2]
      limmag1num[i] &lt;- g1num
      limmag2num[i] &lt;- g2num
    }
    limmacompnum &lt;- nrow(compid)
    names(limmafits) &lt;- rownames(compid)
    result &lt;- list(t       = limmat,
                   v0      = limmav0,
                   df0     = limmadf,
                   s20     = limmas2,
                   g1num   = limmag1num,
                   g2num   = limmag2num,
                   compnum = limmacompnum,
                   fits    = limmafits)
  }

limmafit.list &lt;-
  function (fitlist, cmp.idx=2)
  {
    compnum    &lt;- length(fitlist)

    genes &lt;- c()
    for (i in 1:compnum) genes &lt;- unique(c(genes, rownames(fitlist[[i]])))

    genenum    &lt;- length(genes)
    limmat     &lt;- matrix(NA,genenum,compnum)
    limmas2    &lt;- rep(0,compnum)
    limmadf    &lt;- rep(0,compnum)
    limmav0    &lt;- rep(0,compnum)
    limmag1num &lt;- rep(0,compnum)
    limmag2num &lt;- rep(0,compnum)

    rownames(limmat)  &lt;- genes
    colnames(limmat)  &lt;- names(fitlist)
    names(limmas2)    &lt;- names(fitlist)
    names(limmadf)    &lt;- names(fitlist)
    names(limmav0)    &lt;- names(fitlist)
    names(limmag1num) &lt;- names(fitlist)
    names(limmag2num) &lt;- names(fitlist)

    for (i in 1:compnum) {
      this.t &lt;- fitlist[[i]]$t[,cmp.idx]
      limmat[names(this.t),i] &lt;- this.t

      limmas2[i]    &lt;- fitlist[[i]]$s2.prior
      limmadf[i]    &lt;- fitlist[[i]]$df.prior
      limmav0[i]    &lt;- fitlist[[i]]$var.prior[cmp.idx]
      limmag1num[i] &lt;- sum(fitlist[[i]]$design[,cmp.idx]==0)
      limmag2num[i] &lt;- sum(fitlist[[i]]$design[,cmp.idx]==1)
    }

    limmacompnum &lt;- compnum
    result &lt;- list(t       = limmat,
                   v0      = limmav0,
                   df0     = limmadf,
                   s20     = limmas2,
                   g1num   = limmag1num,
                   g2num   = limmag2num,
                   compnum = limmacompnum,
                   fits    = limmafits)

  }

## Rank genes based on statistics
generank&lt;-function(x) {
  xcol&lt;-ncol(x)
  xrow&lt;-nrow(x)
  result&lt;-matrix(0,xrow,xcol)
  z&lt;-(1:1:xrow)
  for(i in 1:xcol) {
    y&lt;-sort(x[,i],decreasing=TRUE,na.last=TRUE)
    result[,i]&lt;-match(x[,i],y)
    result[,i]&lt;-order(result[,i])
  }
  result
}

## Log-likelihood for moderated t under H0
modt.f0.loglike&lt;-function(x,df) {
  a&lt;-dt(x, df, log=TRUE)
  result&lt;-as.vector(a)
  flag&lt;-which(is.na(result)==TRUE)
  result[flag]&lt;-0
  result
}

## Log-likelihood for moderated t under H1
## param=c(df,g1num,g2num,v0)
modt.f1.loglike&lt;-function(x,param) {
  df&lt;-param[1]
  g1num&lt;-param[2]
  g2num&lt;-param[3]
  v0&lt;-param[4]
  w&lt;-sqrt(1+v0/(1/g1num+1/g2num))
  dt(x/w, df, log=TRUE)-log(w)
  a&lt;-dt(x/w, df, log=TRUE)-log(w)
  result&lt;-as.vector(a)
  flag&lt;-which(is.na(result)==TRUE)
  result[flag]&lt;-0
  result
}

## Correlation Motif Fit
cmfit.X&lt;-function(x, type, K=1, tol=1e-3, max.iter=100) {
  ## initialize
  xrow &lt;- nrow(x)
  xcol &lt;- ncol(x)
  loglike0 &lt;- list()
  loglike1 &lt;- list()
  p &lt;- rep(1, K)/K
  q &lt;- matrix(runif(K * xcol), K, xcol)
  q[1, ] &lt;- rep(0.01, xcol)
  for (i in 1:xcol) {
    f0 &lt;- type[[i]][[1]]
    f0param &lt;- type[[i]][[2]]
    f1 &lt;- type[[i]][[3]]
    f1param &lt;- type[[i]][[4]]
    loglike0[[i]] &lt;- f0(x[, i], f0param)
    loglike1[[i]] &lt;- f1(x[, i], f1param)
  }
  condlike &lt;- list()
  for (i in 1:xcol) {
    condlike[[i]] &lt;- matrix(0, xrow, K)
  }
  loglike.old &lt;- -1e+10
  for (i.iter in 1:max.iter) {
    if ((i.iter%%50) == 0) {
      print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations for K=&quot;,
                  K, sep = &quot;&quot;))
    }
    err &lt;- tol + 1
    clustlike &lt;- matrix(0, xrow, K)
    #templike &lt;- matrix(0, xrow, 2)
    templike1 &lt;- rep(0, xrow)
    templike2 &lt;- rep(0, xrow)
    for (j in 1:K) {
      for (i in 1:xcol) {
        templike1 &lt;- log(q[j, i]) + loglike1[[i]]
        templike2 &lt;- log(1 - q[j, i]) + loglike0[[i]]
        tempmax &lt;- Rfast::Pmax(templike1, templike2)

        templike1 &lt;- exp(templike1 - tempmax)
        templike2 &lt;- exp(templike2 - tempmax)

        tempsum &lt;- templike1 + templike2
        clustlike[, j] &lt;- clustlike[, j] + tempmax +
          log(tempsum)
        condlike[[i]][, j] &lt;- templike1/tempsum
      }
      clustlike[, j] &lt;- clustlike[, j] + log(p[j])
    }
    #tempmax &lt;- apply(clustlike, 1, max)
    tempmax &lt;- Rfast::rowMaxs(clustlike, value=TRUE)
    for (j in 1:K) {
      clustlike[, j] &lt;- exp(clustlike[, j] - tempmax)
    }
    #tempsum &lt;- apply(clustlike, 1, sum)
    tempsum &lt;- Rfast::rowsums(clustlike)
    for (j in 1:K) {
      clustlike[, j] &lt;- clustlike[, j]/tempsum
    }
    #p.new &lt;- (apply(clustlike, 2, sum) + 1)/(xrow + K)
    p.new &lt;- (Rfast::colsums(clustlike) + 1)/(xrow + K)
    q.new &lt;- matrix(0, K, xcol)
    for (j in 1:K) {
      clustpsum &lt;- sum(clustlike[, j])
      for (i in 1:xcol) {
        q.new[j, i] &lt;- (sum(clustlike[, j] * condlike[[i]][,
                                                           j]) + 1)/(clustpsum + 2)
      }
    }
    err.p &lt;- max(abs(p.new - p)/p)
    err.q &lt;- max(abs(q.new - q)/q)
    err &lt;- max(err.p, err.q)
    loglike.new &lt;- (sum(tempmax + log(tempsum)) + sum(log(p.new)) +
                      sum(log(q.new) + log(1 - q.new)))/xrow
    p &lt;- p.new
    q &lt;- q.new
    loglike.old &lt;- loglike.new
    if (err &lt; tol) {
      break
    }
  }
  clustlike &lt;- matrix(0, xrow, K)
  for (j in 1:K) {
    for (i in 1:xcol) {
      templike1 &lt;- log(q[j, i]) + loglike1[[i]]
      templike2 &lt;- log(1 - q[j, i]) + loglike0[[i]]
      tempmax &lt;- Rfast::Pmax(templike1, templike2)

      templike1 &lt;- exp(templike1 - tempmax)
      templike2 &lt;- exp(templike2 - tempmax)

      tempsum &lt;- templike1 + templike2
      clustlike[, j] &lt;- clustlike[, j] + tempmax + log(tempsum)
      condlike[[i]][, j] &lt;- templike1/tempsum
    }
    clustlike[, j] &lt;- clustlike[, j] + log(p[j])
  }
  #tempmax &lt;- apply(clustlike, 1, max)
  tempmax &lt;- Rfast::rowMaxs(clustlike, value=TRUE)
  for (j in 1:K) {
    clustlike[, j] &lt;- exp(clustlike[, j] - tempmax)
  }
  #tempsum &lt;- apply(clustlike, 1, sum)
  tempsum &lt;- Rfast::rowsums(clustlike)
  for (j in 1:K) {
    clustlike[, j] &lt;- clustlike[, j]/tempsum
  }
  p.post &lt;- matrix(0, xrow, xcol)
  for (j in 1:K) {
    for (i in 1:xcol) {
      p.post[, i] &lt;- p.post[, i] + clustlike[, j] * condlike[[i]][,
                                                                  j]
    }
  }
  loglike.old &lt;- loglike.old - (sum(log(p)) + sum(log(q) +
                                                    log(1 - q)))/xrow
  loglike.old &lt;- loglike.old * xrow
  result &lt;- list(p.post = p.post, motif.prior = p, motif.q = q,
                 loglike = loglike.old, clustlike=clustlike, condlike=condlike)
}

## Fit using (0,0,...,0) and (1,1,...,1)
cmfitall&lt;-function(x, type, tol=1e-3, max.iter=100) {
  ## initialize
  xrow&lt;-nrow(x)
  xcol&lt;-ncol(x)
  loglike0&lt;-list()
  loglike1&lt;-list()
  p&lt;-0.01

  ## compute loglikelihood
  L0&lt;-matrix(0,xrow,1)
  L1&lt;-matrix(0,xrow,1)
  for(i in 1:xcol) {
    f0&lt;-type[[i]][[1]]
    f0param&lt;-type[[i]][[2]]
    f1&lt;-type[[i]][[3]]
    f1param&lt;-type[[i]][[4]]
    loglike0[[i]]&lt;-f0(x[,i],f0param)
    loglike1[[i]]&lt;-f1(x[,i],f1param)
    L0&lt;-L0+loglike0[[i]]
    L1&lt;-L1+loglike1[[i]]
  }


  ## EM algorithm to get MLE of p and q
  loglike.old &lt;- -1e10
  for(i.iter in 1:max.iter) {
    if((i.iter%%50) == 0) {
      print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations&quot;,sep=&quot;&quot;))
    }
    err&lt;-tol+1

    ## compute posterior cluster membership
    clustlike&lt;-matrix(0,xrow,2)
    clustlike[,1]&lt;-log(1-p)+L0
    clustlike[,2]&lt;-log(p)+L1

    tempmax&lt;-apply(clustlike,1,max)
    for(j in 1:2) {
      clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
    }
    tempsum&lt;-apply(clustlike,1,sum)

    ## update motif occurrence rate
    for(j in 1:2) {
      clustlike[,j]&lt;-clustlike[,j]/tempsum
    }

    p.new&lt;-(sum(clustlike[,2])+1)/(xrow+2)

    ## evaluate convergence
    err&lt;-abs(p.new-p)/p

    ## evaluate whether the log.likelihood increases
    loglike.new&lt;-(sum(tempmax+log(tempsum))+log(p.new)+log(1-p.new))/xrow

    loglike.old&lt;-loglike.new
    p&lt;-p.new

    if(err&lt;tol) {
      break;
    }
  }

  ## compute posterior p
  clustlike&lt;-matrix(0,xrow,2)
  clustlike[,1]&lt;-log(1-p)+L0
  clustlike[,2]&lt;-log(p)+L1

  tempmax&lt;-apply(clustlike,1,max)
  for(j in 1:2) {
    clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
  }
  tempsum&lt;-apply(clustlike,1,sum)

  for(j in 1:2) {
    clustlike[,j]&lt;-clustlike[,j]/tempsum
  }

  p.post&lt;-matrix(0,xrow,xcol)
  for(i in 1:xcol) {
    p.post[,i]&lt;-clustlike[,2]
  }

  ## return

  #calculate back loglikelihood
  loglike.old&lt;-loglike.old-(log(p)+log(1-p))/xrow
  loglike.old&lt;-loglike.old*xrow
  result&lt;-list(p.post=p.post, motif.prior=p, loglike=loglike.old)
}

## Fit each dataset separately
cmfitsep&lt;-function(x, type, tol=1e-3, max.iter=100) {
  ## initialize
  xrow&lt;-nrow(x)
  xcol&lt;-ncol(x)
  loglike0&lt;-list()
  loglike1&lt;-list()
  p&lt;-0.01*rep(1,xcol)
  loglike.final&lt;-rep(0,xcol)

  ## compute loglikelihood
  for(i in 1:xcol) {
    f0&lt;-type[[i]][[1]]
    f0param&lt;-type[[i]][[2]]
    f1&lt;-type[[i]][[3]]
    f1param&lt;-type[[i]][[4]]
    loglike0[[i]]&lt;-f0(x[,i],f0param)
    loglike1[[i]]&lt;-f1(x[,i],f1param)
  }

  p.post&lt;-matrix(0,xrow,xcol)

  ## EM algorithm to get MLE of p
  for(coli in 1:xcol) {
    loglike.old &lt;- -1e10
    for(i.iter in 1:max.iter) {
      if((i.iter%%50) == 0) {
        print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations&quot;,sep=&quot;&quot;))
      }
      err&lt;-tol+1

      ## compute posterior cluster membership
      clustlike&lt;-matrix(0,xrow,2)
      clustlike[,1]&lt;-log(1-p[coli])+loglike0[[coli]]
      clustlike[,2]&lt;-log(p[coli])+loglike1[[coli]]

      tempmax&lt;-apply(clustlike,1,max)
      for(j in 1:2) {
        clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
      }
      tempsum&lt;-apply(clustlike,1,sum)

      ## evaluate whether the log.likelihood increases
      loglike.new&lt;-sum(tempmax+log(tempsum))/xrow

      ## update motif occurrence rate
      for(j in 1:2) {
        clustlike[,j]&lt;-clustlike[,j]/tempsum
      }

      p.new&lt;-(sum(clustlike[,2]))/(xrow)

      ## evaluate convergence
      err&lt;-abs(p.new-p[coli])/p[coli]
      loglike.old&lt;-loglike.new
      p[coli]&lt;-p.new

      if(err&lt;tol) {
        break;
      }
    }

    ## compute posterior p
    clustlike&lt;-matrix(0,xrow,2)
    clustlike[,1]&lt;-log(1-p[coli])+loglike0[[coli]]
    clustlike[,2]&lt;-log(p[coli])+loglike1[[coli]]

    tempmax&lt;-apply(clustlike,1,max)
    for(j in 1:2) {
      clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
    }
    tempsum&lt;-apply(clustlike,1,sum)

    for(j in 1:2) {
      clustlike[,j]&lt;-clustlike[,j]/tempsum
    }

    p.post[,coli]&lt;-clustlike[,2]
    loglike.final[coli]&lt;-loglike.old
  }


  ## return
  loglike.final&lt;-loglike.final*xrow
  result&lt;-list(p.post=p.post, motif.prior=p, loglike=loglike.final)
}

## Fit the full model
cmfitfull&lt;-function(x, type, tol=1e-3, max.iter=100) {
  ## initialize
  xrow&lt;-nrow(x)
  xcol&lt;-ncol(x)
  loglike0&lt;-list()
  loglike1&lt;-list()
  K&lt;-2^xcol
  p&lt;-rep(1,K)/K
  pattern&lt;-rep(0,xcol)
  patid&lt;-matrix(0,K,xcol)

  ## compute loglikelihood
  for(i in 1:xcol) {
    f0&lt;-type[[i]][[1]]
    f0param&lt;-type[[i]][[2]]
    f1&lt;-type[[i]][[3]]
    f1param&lt;-type[[i]][[4]]
    loglike0[[i]]&lt;-f0(x[,i],f0param)
    loglike1[[i]]&lt;-f1(x[,i],f1param)
  }
  L&lt;-matrix(0,xrow,K)
  for(i in 1:K)
  {
    patid[i,]&lt;-pattern
    for(j in 1:xcol) {
      if(pattern[j] &lt; 0.5) {
        L[,i]&lt;-L[,i]+loglike0[[j]]
      } else {
        L[,i]&lt;-L[,i]+loglike1[[j]]
      }
    }

    if(i &lt; K) {
      pattern[xcol]&lt;-pattern[xcol]+1
      j&lt;-xcol
      while(pattern[j] &gt; 1) {
        pattern[j]&lt;-0
        j&lt;-j-1
        pattern[j]&lt;-pattern[j]+1
      }
    }
  }

  ## EM algorithm to get MLE of p and q
  loglike.old &lt;- -1e10
  for(i.iter in 1:max.iter) {
    if((i.iter%%50) == 0) {
      print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations&quot;,sep=&quot;&quot;))
    }
    err&lt;-tol+1

    ## compute posterior cluster membership
    clustlike&lt;-matrix(0,xrow,K)
    for(j in 1:K) {
      clustlike[,j]&lt;-log(p[j])+L[,j]
    }

    tempmax&lt;-apply(clustlike,1,max)
    for(j in 1:K) {
      clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
    }
    tempsum&lt;-apply(clustlike,1,sum)

    ## update motif occurrence rate
    for(j in 1:K) {
      clustlike[,j]&lt;-clustlike[,j]/tempsum
    }

    p.new&lt;-(apply(clustlike,2,sum)+1)/(xrow+K)

    ## evaluate convergence
    err&lt;-max(abs(p.new-p)/p)

    ## evaluate whether the log.likelihood increases
    loglike.new&lt;-(sum(tempmax+log(tempsum))+sum(log(p.new)))/xrow

    loglike.old&lt;-loglike.new
    p&lt;-p.new

    if(err&lt;tol) {
      break;
    }
  }

  ## compute posterior p
  clustlike&lt;-matrix(0,xrow,K)
  for(j in 1:K) {
    clustlike[,j]&lt;-log(p[j])+L[,j]
  }

  tempmax&lt;-apply(clustlike,1,max)
  for(j in 1:K) {
    clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
  }
  tempsum&lt;-apply(clustlike,1,sum)

  for(j in 1:K) {
    clustlike[,j]&lt;-clustlike[,j]/tempsum
  }

  p.post&lt;-matrix(0,xrow,xcol)
  for(j in 1:K) {
    for(i in 1:xcol) {
      if(patid[j,i] &gt; 0.5) {
        p.post[,i]&lt;-p.post[,i]+clustlike[,j]
      }
    }
  }

  ## return
  #calculate back loglikelihood
  loglike.old&lt;-loglike.old-sum(log(p))/xrow
  loglike.old&lt;-loglike.old*xrow
  result&lt;-list(p.post=p.post, motif.prior=p, loglike=loglike.old)
}

generatetype&lt;-function(limfitted)
{
  jtype&lt;-list()
  df&lt;-limfitted$g1num+limfitted$g2num-2+limfitted$df0
  for(j in 1:limfitted$compnum)
  {
    jtype[[j]]&lt;-list(f0=modt.f0.loglike, f0.param=df[j], f1=modt.f1.loglike, f1.param=c(df[j],limfitted$g1num[j],limfitted$g2num[j],limfitted$v0[j]))
  }
  jtype
}

cormotiffit &lt;- function(exprs, groupid=NULL, compid=NULL, K=1, tol=1e-3,
                        max.iter=100, BIC=TRUE, norm.factor.method=&quot;TMM&quot;,
                        voom.normalize.method = &quot;none&quot;, runtype=c(&quot;logCPM&quot;,&quot;counts&quot;,&quot;limmafits&quot;), each=3)
{
  # first I want to do some typechecking. Input can be either a normalized
  # matrix, a count matrix, or a list of limma fits. Dispatch the correct
  # limmafit accordingly.
  # todo: add some typechecking here
  limfitted &lt;- list()
  if (runtype==&quot;counts&quot;) {
    limfitted &lt;- limmafit.counts(exprs,groupid,compid, norm.factor.method, voom.normalize.method)
  } else if (runtype==&quot;logCPM&quot;) {
    limfitted &lt;- limmafit.default(exprs,groupid,compid)
  } else if (runtype==&quot;limmafits&quot;) {
    limfitted &lt;- limmafit.list(exprs)
  } else {
    stop(&quot;runtype must be one of &#39;logCPM&#39;, &#39;counts&#39;, or &#39;limmafits&#39;&quot;)
  }


  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-list()
  ks &lt;- rep(K, each = each)
  fitresult &lt;- bplapply(1:length(ks), function(i, x, type, ks, tol, max.iter) {
    cmfit.X(x, type, K = ks[i], tol = tol, max.iter = max.iter)
  }, x=limfitted$t, type=jtype, ks=ks, tol=tol, max.iter=max.iter)

  best.fitresults &lt;- list()
  for (i in 1:length(K)) {
    w.k &lt;- which(ks==K[i])
    this.bic &lt;- c()
    for (j in w.k) this.bic[j] &lt;- -2 * fitresult[[j]]$loglike + (K[i] - 1 + K[i] * limfitted$compnum) * log(dim(limfitted$t)[1])
    w.min &lt;- which(this.bic == min(this.bic, na.rm = TRUE))[1]
    best.fitresults[[i]] &lt;- fitresult[[w.min]]
  }
  fitresult &lt;- best.fitresults

  bic &lt;- rep(0, length(K))
  aic &lt;- rep(0, length(K))
  loglike &lt;- rep(0, length(K))
  for (i in 1:length(K)) loglike[i] &lt;- fitresult[[i]]$loglike
  for (i in 1:length(K)) bic[i] &lt;- -2 * fitresult[[i]]$loglike + (K[i] - 1 + K[i] * limfitted$compnum) * log(dim(limfitted$t)[1])
  for (i in 1:length(K)) aic[i] &lt;- -2 * fitresult[[i]]$loglike + 2 * (K[i] - 1 + K[i] * limfitted$compnum)
  if(BIC==TRUE) {
    bestflag=which(bic==min(bic))
  }
  else {
    bestflag=which(aic==min(aic))
  }
  result&lt;-list(bestmotif=fitresult[[bestflag]],bic=cbind(K,bic),
               aic=cbind(K,aic),loglike=cbind(K,loglike), allmotifs=fitresult)

}

cormotiffitall&lt;-function(exprs,groupid,compid, tol=1e-3, max.iter=100)
{
  limfitted&lt;-limmafit(exprs,groupid,compid)
  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-cmfitall(limfitted$t,type=jtype,tol=1e-3,max.iter=max.iter)
}

cormotiffitsep&lt;-function(exprs,groupid,compid, tol=1e-3, max.iter=100)
{
  limfitted&lt;-limmafit(exprs,groupid,compid)
  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-cmfitsep(limfitted$t,type=jtype,tol=1e-3,max.iter=max.iter)
}

cormotiffitfull&lt;-function(exprs,groupid,compid, tol=1e-3, max.iter=100)
{
  limfitted&lt;-limmafit(exprs,groupid,compid)
  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-cmfitfull(limfitted$t,type=jtype,tol=1e-3,max.iter=max.iter)
}

plotIC&lt;-function(fitted_cormotif)
{
  oldpar&lt;-par(mfrow=c(1,2))
  plot(fitted_cormotif$bic[,1], fitted_cormotif$bic[,2], type=&quot;b&quot;,xlab=&quot;Motif Number&quot;, ylab=&quot;BIC&quot;, main=&quot;BIC&quot;)
  plot(fitted_cormotif$aic[,1], fitted_cormotif$aic[,2], type=&quot;b&quot;,xlab=&quot;Motif Number&quot;, ylab=&quot;AIC&quot;, main=&quot;AIC&quot;)
}

plotMotif&lt;-function(fitted_cormotif,title=&quot;&quot;)
{
  layout(matrix(1:2,ncol=2))
  u&lt;-1:dim(fitted_cormotif$bestmotif$motif.q)[2]
  v&lt;-1:dim(fitted_cormotif$bestmotif$motif.q)[1]
  image(u,v,t(fitted_cormotif$bestmotif$motif.q),
        col=gray(seq(from=1,to=0,by=-0.1)),xlab=&quot;Study&quot;,yaxt = &quot;n&quot;,
        ylab=&quot;Corr. Motifs&quot;,main=paste(title,&quot;pattern&quot;,sep=&quot; &quot;))
  axis(2,at=1:length(v))
  for(i in 1:(length(u)+1))
  {
    abline(v=(i-0.5))
  }
  for(i in 1:(length(v)+1))
  {
    abline(h=(i-0.5))
  }
  Ng=10000
  if(is.null(fitted_cormotif$bestmotif$p.post)!=TRUE)
    Ng=nrow(fitted_cormotif$bestmotif$p.post)
  genecount=floor(fitted_cormotif$bestmotif$motif.p*Ng)
  NK=nrow(fitted_cormotif$bestmotif$motif.q)
  plot(0,0.7,pch=&quot;.&quot;,xlim=c(0,1.2),ylim=c(0.75,NK+0.25),
       frame.plot=FALSE,axes=FALSE,xlab=&quot;No. of genes&quot;,ylab=&quot;&quot;, main=paste(title,&quot;frequency&quot;,sep=&quot; &quot;))
  segments(0,0.7,fitted_cormotif$bestmotif$motif.p[1],0.7)
  rect(0,1:NK-0.3,fitted_cormotif$bestmotif$motif.p,1:NK+0.3,
       col=&quot;dark grey&quot;)
  mtext(1:NK,at=1:NK,side=2,cex=0.8)
  text(fitted_cormotif$bestmotif$motif.p+0.15,1:NK,
       labels=floor(fitted_cormotif$bestmotif$motif.p*Ng))
}</code></pre>
<div id="data-import-and-set-up" class="section level2">
<h2>Data import and set up</h2>
<p>The first time I ran this data, I used all of the high confidence
peaks.</p>
<pre class="r"><code>high_conf_peak_counts &lt;- read.csv(&quot;data/high_conf_peak_counts.csv&quot;, row.names = 1)


high_conf_peak_counts &lt;- high_conf_peak_counts %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  dplyr::select(Ind1_75DA24h:Ind6_71V3h)
groupmat_names &lt;- data.frame(timeset = colnames(high_conf_peak_counts))

df_names &lt;-groupmat_names %&gt;% 
  mutate(timeset=gsub(&quot;75&quot;,&quot;1_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;87&quot;,&quot;2_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;77&quot;,&quot;3_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;79&quot;,&quot;4_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;78&quot;,&quot;5_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;71&quot;,&quot;6_&quot;,timeset)) %&gt;%
  mutate(timeset = gsub(&quot;24h&quot;,&quot;_24h&quot;,timeset),
       timeset = gsub(&quot;3h&quot;,&quot;_3h&quot;,timeset)) %&gt;%
  separate(timeset, into = c(NA,&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) 
  
 
group &lt;- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),6))
group &lt;- factor(group, levels =c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;,&quot;7&quot;,&quot;8&quot;,&quot;9&quot;,&quot;10&quot;,&quot;11&quot;,&quot;12&quot;))                        
indv &lt;- df_names$indv
# indv &lt;- factor(indv, levels = c(1,2,3,4,5,6))
time &lt;- df_names$time
# time &lt;- factor(time, levels =c(&quot;3h&quot;,&quot;24&quot;))
trt &lt;- df_names$trt
label &lt;- paste0(indv,&quot;_&quot;,trt,&quot;_&quot;,time)
group_fac &lt;- group
groupid &lt;- as.numeric(group_fac)

compid &lt;- data.frame(c1= c(2,4,6,8,10,1,3,5,7,9), c2 = c( 12,12,12,12,12,11,11,11,11,11))

y_TMM_cpm &lt;- cpm(high_conf_peak_counts, log = TRUE)
 

rownames(y_TMM_cpm) &lt;- rownames(high_conf_peak_counts)
colnames(y_TMM_cpm) &lt;- label
# y_TMM_cpm
# set.seed(31415)
# cormotif_initial &lt;- cormotiffit(exprs = y_TMM_cpm,
#                                 groupid = groupid,
#                                 compid = compid,
#                                 K=1:6, max.iter = 500, runtype=&quot;logCPM&quot;)
# saveRDS(cormotif_initial,&quot;data/cormotif_full_6_run.RDS&quot;)

cormotif_initial &lt;- readRDS(&quot;data/cormotif_full_6_run.RDS&quot;)
gene_prob_tran &lt;- cormotif_initial$bestmotif$p.post

rownames(gene_prob_tran) &lt;- rownames(y_TMM_cpm)
motif_prob &lt;- cormotif_initial$bestmotif$clustlike
rownames(motif_prob) &lt;- rownames(y_TMM_cpm)
# write.csv(motif_prob,&quot;data/cormotif_probability_all_6_list.csv&quot;)

Cormotif::plotIC(cormotif_initial)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/setting%20up%20data-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-setting-up-data-1">
Past versions of "setting up data-1.png"
</button>
</p>
<div id="fig-no-spaces-setting-up-data-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/25abe4aa5409f2364f0f91fc2b37b8e74e0b48fc/docs/figure/CorMotif_data.Rmd/setting up data-1.png" target="_blank">25abe4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Cormotif::plotMotif(cormotif_initial)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/setting%20up%20data-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-setting-up-data-2">
Past versions of "setting up data-2.png"
</button>
</p>
<div id="fig-no-spaces-setting-up-data-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/25abe4aa5409f2364f0f91fc2b37b8e74e0b48fc/docs/figure/CorMotif_data.Rmd/setting up data-2.png" target="_blank">25abe4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>myColors &lt;-  rev(c(&quot;#FFFFFF&quot;, &quot;#E6E6E6&quot; ,&quot;#CCCCCC&quot;, &quot;#B3B3B3&quot;, &quot;#999999&quot;, &quot;#808080&quot;, &quot;#666666&quot;,&quot;#4C4C4C&quot;, &quot;#333333&quot;, &quot;#191919&quot;,&quot;#000000&quot;))
plot.new()
legend(&#39;bottomleft&#39;,fill=myColors, legend =rev(c(&quot;0&quot;, &quot;0.1&quot;, &quot;0.2&quot;, &quot;0.3&quot;, &quot;0.4&quot;,  &quot;0.5&quot;, &quot;0.6&quot;, &quot;0.7&quot;, &quot;0.8&quot;,&quot;0.9&quot;, &quot;1&quot;)), box.col=&quot;white&quot;,title = &quot;Probability\nlegend&quot;, horiz=FALSE,title.cex=.8)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/setting%20up%20data-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-setting-up-data-3">
Past versions of "setting up data-3.png"
</button>
</p>
<div id="fig-no-spaces-setting-up-data-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/32aa1a1d4d7d21ca39d1ea160ec98c9408d09bd9/docs/figure/CorMotif_data.Rmd/setting up data-3.png" target="_blank">32aa1a1</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>total number of peaks = 162264.</p>
<p>Study breakdown: 1 = DNR_3, 2 = DOX_3, 3 = EPI_3, 4 = MTX_3, 5 =
TRZ_3<br />
6 = DNR_24, 7 = DOX_24, 8 = EPI_24, 9 = MTX_24, 10 = TRZ_24</p>
<p>I realized I may need to run the cormotif with JUST the highly
expressed set of highconf (hc) peaks and therefore reran cormotif on the
set with log2cpm rowmeans &gt; 0.</p>
<pre class="r"><code>high_conf_peak_counts &lt;- read.csv(&quot;data/high_conf_peak_counts.csv&quot;, row.names = 1)
my_hc_counts &lt;- high_conf_peak_counts %&gt;% 
  dplyr::select(Geneid,Ind1_75DA24h:Ind6_71V3h) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;)

lcpm &lt;- cpm(my_hc_counts, log=TRUE)  ### for determining the basic cutoffs

row_means &lt;- rowMeans(lcpm)
my_hc_filtered_counts &lt;- my_hc_counts[row_means &gt; 0,]



groupmat_names_he &lt;- data.frame(timeset = colnames(my_hc_filtered_counts))

df_names_he &lt;-groupmat_names_he %&gt;% 
  mutate(timeset=gsub(&quot;75&quot;,&quot;1_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;87&quot;,&quot;2_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;77&quot;,&quot;3_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;79&quot;,&quot;4_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;78&quot;,&quot;5_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;71&quot;,&quot;6_&quot;,timeset)) %&gt;%
  mutate(timeset = gsub(&quot;24h&quot;,&quot;_24h&quot;,timeset),
       timeset = gsub(&quot;3h&quot;,&quot;_3h&quot;,timeset)) %&gt;%
  separate(timeset, into = c(NA,&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) 
  
 
label &lt;- paste0(indv,&quot;_&quot;,trt,&quot;_&quot;,time)
group_fac &lt;- group
groupid &lt;- as.numeric(group_fac)

compid &lt;- data.frame(c1= c(2,4,6,8,10,1,3,5,7,9), c2 = c( 12,12,12,12,12,11,11,11,11,11))

y_TMM_cpm_he &lt;- cpm(my_hc_filtered_counts, log = TRUE)
 

rownames(y_TMM_cpm_he) &lt;- rownames(my_hc_filtered_counts)
colnames(y_TMM_cpm_he) &lt;- label
set.seed(31415)
# cormotif_initial_he &lt;- cormotiffit(exprs = y_TMM_cpm_he,
                                # groupid = groupid,
                                # compid = compid,
                                # K=1:6, max.iter = 500, runtype=&quot;logCPM&quot;)
# saveRDS(cormotif_initial_he,&quot;data/cormotif_full_6_run_he.RDS&quot;)

cormotif_initial_he&lt;- readRDS(&quot;data/cormotif_full_6_run_he.RDS&quot;)
gene_prob_tran_he &lt;- cormotif_initial_he$bestmotif$p.post

rownames(gene_prob_tran_he) &lt;- rownames(y_TMM_cpm_he)
motif_prob_he &lt;- cormotif_initial_he$bestmotif$clustlike
rownames(motif_prob_he) &lt;- rownames(y_TMM_cpm_he)
# write.csv(motif_prob_he,&quot;data/cormotif_probability_all_6_list_he.csv&quot;)

Cormotif::plotIC(cormotif_initial_he)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/hc%20high%20expressed%20motif-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-hc-high-expressed-motif-1">
Past versions of "hc high expressed motif-1.png"
</button>
</p>
<div id="fig-no-spaces-hc-high-expressed-motif-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1650f780323d762f550b181742dba4cddf8877e7/docs/figure/CorMotif_data.Rmd/hc high expressed motif-1.png" target="_blank">1650f78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Cormotif::plotMotif(cormotif_initial_he)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/hc%20high%20expressed%20motif-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-hc-high-expressed-motif-2">
Past versions of "hc high expressed motif-2.png"
</button>
</p>
<div id="fig-no-spaces-hc-high-expressed-motif-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1650f780323d762f550b181742dba4cddf8877e7/docs/figure/CorMotif_data.Rmd/hc high expressed motif-2.png" target="_blank">1650f78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>myColors &lt;-  rev(c(&quot;#FFFFFF&quot;, &quot;#E6E6E6&quot; ,&quot;#CCCCCC&quot;, &quot;#B3B3B3&quot;, &quot;#999999&quot;, &quot;#808080&quot;, &quot;#666666&quot;,&quot;#4C4C4C&quot;, &quot;#333333&quot;, &quot;#191919&quot;,&quot;#000000&quot;))
plot.new()
legend(&#39;bottomleft&#39;,fill=myColors, legend =rev(c(&quot;0&quot;, &quot;0.1&quot;, &quot;0.2&quot;, &quot;0.3&quot;, &quot;0.4&quot;,  &quot;0.5&quot;, &quot;0.6&quot;, &quot;0.7&quot;, &quot;0.8&quot;,&quot;0.9&quot;, &quot;1&quot;)), box.col=&quot;white&quot;,title = &quot;Probability\nlegend&quot;, horiz=FALSE,title.cex=.8)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/hc%20high%20expressed%20motif-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-hc-high-expressed-motif-3">
Past versions of "hc high expressed motif-3.png"
</button>
</p>
<div id="fig-no-spaces-hc-high-expressed-motif-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1650f780323d762f550b181742dba4cddf8877e7/docs/figure/CorMotif_data.Rmd/hc high expressed motif-3.png" target="_blank">1650f78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This filtering left me with 153355 peaks.</p>
<p>Doing the analysis on the filtered set</p>
<pre class="r"><code>motif_prob_he &lt;- read.csv(&quot;data/cormotif_probability_all_6_list_he.csv&quot;, row.names = 1)

clust1 &lt;- motif_prob_he %&gt;%
  as.data.frame() %&gt;%
  filter(V1&gt;0.5) %&gt;% 
  rownames
clust2 &lt;- motif_prob_he %&gt;%
  as.data.frame() %&gt;%
  filter(V2&gt;0.5) %&gt;% 
  rownames
clust3 &lt;- motif_prob_he %&gt;%
  as.data.frame() %&gt;%
  filter(V3&gt;0.5) %&gt;% 
  rownames
clust4 &lt;- motif_prob_he %&gt;%
  as.data.frame() %&gt;%
  filter(V4&gt;0.5) %&gt;% 
  rownames
backGL_full_he &lt;- row.names(motif_prob_he)
# saveRDS(backGL_full_he,&quot;data/full_he_peak_names.RDS&quot;)

unclassified_full &lt;- setdiff(backGL_full_he,union(clust1,union(clust2,union(clust4,clust3))))

motif_list_full &lt;- list(&quot;NR&quot;=clust1,&quot;ESR&quot;=clust2,&quot;EAR&quot;=clust3,&quot;LR&quot;=clust4)
# saveRDS(motif_list_full,&quot;data/motif_list_full&quot;)
# saveRDS(unclassified_full,&quot;data/unclassified_full_set_peaks.RDS&quot;)</code></pre>
<p>Seems to be grouping like RNA-seq. I am using the same nomenclature.
I filter each column of the likelihood of belonging to the cluster by
&gt; 0.5- This causes some number discrepancies in overall.</p>
<ul>
<li><ul>
<li>Clust1 has 85279 regions and is the <strong>No Response</strong>
set.</li>
</ul></li>
<li><ul>
<li>Clust2 has 15546 regions and is the <strong>Early-Sustained
Response</strong> set.</li>
</ul></li>
<li><p>- Clust3 has 10730 regions and is the <strong>Early-Acute
Response</strong> set.</p></li>
<li><ul>
<li>Clust4 has 38883 regions and is the <strong>Late-Response</strong>
set.</li>
</ul></li>
</ul>
<p>These contain 150438 total peaks out of 153355 for approximately
98.0978775 percent of all peaks accounted for.</p>
<pre class="r"><code>txdb &lt;- TxDb.Hsapiens.UCSC.hg38.knownGene

# loadFile_peakCall &lt;- function(){
#  file &lt;- choose.files()
#  file &lt;- readPeakFile(file, header = FALSE)
#  return(file)
# }
# 
# prepGRangeObj &lt;- function(seek_object){
#  seek_object$Peaks = seek_object$V4
#  seek_object$level = seek_object$V5
#  seek_object$V4 = seek_object$V5 = NULL
#  return(seek_object)
# }
list2env(motif_list_full, envir= .GlobalEnv)</code></pre>
<pre><code>&lt;environment: R_GlobalEnv&gt;</code></pre>
<pre class="r"><code>TSS = getBioRegion(TxDb=txdb, upstream=2000, downstream=2000, by = &quot;gene&quot;, 
                   type = &quot;start_site&quot;)
##converting lists to dataframes as bed files
# EAR %&gt;%
  
  # data.frame(&quot;EAR&quot; = EAR) %&gt;%
  #   mutate(name =EAR) %&gt;%
  # separate(EAR, into = c( &quot;chr&quot;,&quot;start&quot;,&quot;end&quot;)) %&gt;%
  #   # mutate(strand=&quot;.&quot;) %&gt;%
  #   mutate(start=as.integer(start), end=as.integer(end)) %&gt;%
  #   dplyr::select(chr, start,end,name) %&gt;%
  #   na.omit() %&gt;%
  #   # unite(.,col=&quot;start-end&quot;,start, end,sep = &quot;-&quot;) %&gt;%
  #   write.table(.,file = &quot;data/EAR_full.bed&quot;, row.names =FALSE,col.name=FALSE)
  # 
  # 
  # data.frame(&quot;unclassified_full&quot; = unclassified_full) %&gt;%
  #   mutate(name =unclassified_full) %&gt;%
  # separate(unclassified_full, into = c( &quot;chr&quot;,&quot;start&quot;,&quot;end&quot;)) %&gt;%
  #   # mutate(strand=&quot;.&quot;) %&gt;%
  #   mutate(start=as.integer(start), end=as.integer(end)) %&gt;%
  #   dplyr::select(chr, start,end,name) %&gt;%
  #   na.omit() %&gt;%
  #   # unite(.,col=&quot;start-end&quot;,start, end,sep = &quot;-&quot;) %&gt;%
  #   write.table(.,file = &quot;data/unclassified_full.bed&quot;, row.names =FALSE,col.name=FALSE)
  
  # 
  # data.frame(&quot;background_he&quot; = backGL_full_he) %&gt;%
  #   mutate(name =background_he) %&gt;%
  # separate(background_he, into = c( &quot;chr&quot;,&quot;start&quot;,&quot;end&quot;)) %&gt;%
  #   # mutate(strand=&quot;.&quot;) %&gt;%
  #   mutate(start=as.integer(start), end=as.integer(end)) %&gt;%
  #   dplyr::select(chr, start,end,name) %&gt;%
  #   na.omit() %&gt;%
  #   # unite(.,col=&quot;start-end&quot;,start, end,sep = &quot;-&quot;) %&gt;%
  #   write.table(.,file = &quot;data/background_he.bed&quot;, row.names =FALSE,col.name=FALSE)
  # # 
  # 
  # data.frame(&quot;ESR&quot; = ESR) %&gt;% 
  #   mutate(name =ESR) %&gt;% 
  # separate(ESR, into = c( &quot;chr&quot;,&quot;start&quot;,&quot;end&quot;)) %&gt;%
  #   # mutate(strand=&quot;.&quot;) %&gt;% 
  #   mutate(start=as.integer(start), end=as.integer(end)) %&gt;%
  #   dplyr::select(chr, start,end,name) %&gt;% 
  #   na.omit() %&gt;% 
  #   # unite(.,col=&quot;start-end&quot;,start, end,sep = &quot;-&quot;) %&gt;% 
  #   write.table(.,file = &quot;data/ESR_full.bed&quot;, row.names =FALSE,col.name=FALSE)
  # 
  # data.frame(&quot;LR&quot; = LR) %&gt;% 
  #   mutate(name =LR) %&gt;% 
  # separate(LR, into = c( &quot;chr&quot;,&quot;start&quot;,&quot;end&quot;)) %&gt;%
  #   # mutate(strand=&quot;.&quot;) %&gt;% 
  #   mutate(start=as.integer(start), end=as.integer(end)) %&gt;%
  #   dplyr::select(chr, start,end,name) %&gt;% 
  #   na.omit() %&gt;% 
  #   # unite(.,col=&quot;start-end&quot;,start, end,sep = &quot;-&quot;) %&gt;% 
  #   write.table(.,file = &quot;data/LR_full.bed&quot;, row.names =FALSE,col.name=FALSE)
  # 
  # data.frame(&quot;NR&quot; = NR) %&gt;% 
  #   mutate(name =NR) %&gt;% 
  # separate(NR, into = c( &quot;chr&quot;,&quot;start&quot;,&quot;end&quot;)) %&gt;%
  #   # mutate(strand=&quot;.&quot;) %&gt;% 
  #   mutate(start=as.integer(start), end=as.integer(end)) %&gt;%
  #   dplyr::select(chr, start,end,name) %&gt;% 
  #   na.omit() %&gt;% 
  #   # unite(.,col=&quot;start-end&quot;,start, end,sep = &quot;-&quot;) %&gt;% 
  #   write.table(.,file = &quot;data/NR_full.bed&quot;, row.names =FALSE,col.name=FALSE)

# 
# EAR_full_gr &lt;- bed_to_granges(&quot;data/n45_bedfiles/EAR_full.bed&quot;)
# ESR_full_gr &lt;- bed_to_granges(&quot;data/n45_bedfiles/ESR_full.bed&quot;)
# LR_full_gr &lt;- bed_to_granges(&quot;data/n45_bedfiles/LR_full.bed&quot;)
# NR_full_gr &lt;- bed_to_granges(&quot;data/n45_bedfiles/NR_full.bed&quot;)
# unclassified_gr &lt;- bed_to_granges(&quot;data/n45_bedfiles/unclassified_full.bed&quot;)
# background_gr &lt;- bed_to_granges(&quot;data/n45_bedfiles/background_he.bed&quot;)
# 
# mylist &lt;- list(EAR_full_gr,ESR_full_gr,LR_full_gr,NR_full_gr,unclassified_gr,background_gr)
# peakAnnoList&lt;- lapply(mylist, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# names(peakAnnoList) &lt;- c(&quot;EAR_full_gr&quot;,&quot;ESR_full_gr&quot;,&quot;LR_full_gr&quot;,&quot;NR_full_gr&quot;,&quot;unclassified_gr&quot;,&quot;background_gr&quot;)
# saveRDS(peakAnnoList, &quot;data/peakAnnoList_full_motif.RDS&quot;)

peakAnnoList_full_motif &lt;- readRDS(&quot;data/peakAnnoList_full_motif.RDS&quot;)
plotAnnoBar(peakAnnoList_full_motif, main = &quot;Genomic Feature Distribution, CorMotif&quot;)+ggtitle (&quot;Genomic Feature Distribution, CorMotif&quot;)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/assessment%20of%20each%20cluster-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-assessment-of-each-cluster-1">
Past versions of "assessment of each cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-assessment-of-each-cluster-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5acc25545813111cee407a9ab042a3f3f7b13c57/docs/figure/CorMotif_data.Rmd/assessment of each cluster-1.png" target="_blank">5acc255</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>##Epi_list_tagMatrix title was just because I was too lazy to change the name
# Epi_list_tagMatrix = lapply(Ind6_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, &quot;data/Ind6_TSS_peaks.RDS&quot;)</code></pre>
<pre class="r"><code>my_hc_filtered_counts &lt;-  readRDS(&quot;data/my_hc_filt_counts.RDS&quot;)
log_filt_hc_full &lt;- cpm(my_hc_filtered_counts, log = TRUE) %&gt;% as.data.frame()
drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

log_filt_hc_full %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(row.names(.) %in% motif_list_full$EAR[c(1,2,3)]) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm in EAR peaks&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/CorMotif_data.Rmd/fullset%20cormotif%20lfc%20EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>toplist_6 &lt;- readRDS(&quot;data/toplist_6.RDS&quot;)


toplist_6 %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(peak %in% motif_list_full$EAR) %&gt;% 
   mutate(logFC= abs(logFC)) %&gt;% 
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;logFC in EAR peaks&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/CorMotif_data.Rmd/mlfc%20EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>log_filt_hc_full %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(row.names(.) %in% motif_list_full$ESR[c(1,2,3)]) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm in ESR peaks&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<pre class="r"><code>toplist_6 %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(peak %in% motif_list_full$ESR) %&gt;% 
   mutate(logFC= abs(logFC)) %&gt;% 
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;logFC in ESR peaks&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/CorMotif_data.Rmd/mlfc%20ESR-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>log_filt_hc_full %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(row.names(.) %in% motif_list_full$LR[c(1,2,3)]) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm in LR peaks&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<pre class="r"><code>toplist_6 %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(peak %in% motif_list_full$LR) %&gt;% 
   mutate(logFC= abs(logFC)) %&gt;% 
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;logFC in LR peaks&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/CorMotif_data.Rmd/mlfc%20LR-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>log_filt_hc_full %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(row.names(.) %in% motif_list_full$NR[c(1,2,3)]) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm in NR peaks&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<pre class="r"><code>toplist_6 %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_full$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(peak %in% motif_list_full$NR) %&gt;% 
  mutate(logFC= abs(logFC)) %&gt;% 
  
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;logFC in NR peaks&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/CorMotif_data.Rmd/mlfc%20NR-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] bedr_0.0.0.9000                         
 [2] devtools_2.4.5                          
 [3] usethis_2.2.3                           
 [4] ggpubr_0.6.0                            
 [5] BiocParallel_1.36.0                     
 [6] Cormotif_1.48.0                         
 [7] affy_1.80.0                             
 [8] scales_1.3.0                            
 [9] VennDiagram_1.7.3                       
[10] futile.logger_1.4.3                     
[11] gridExtra_2.3                           
[12] ggfortify_0.4.16                        
[13] edgeR_4.0.16                            
[14] limma_3.58.1                            
[15] rtracklayer_1.62.0                      
[16] org.Hs.eg.db_3.18.0                     
[17] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[18] GenomicFeatures_1.54.3                  
[19] AnnotationDbi_1.64.1                    
[20] Biobase_2.62.0                          
[21] GenomicRanges_1.54.1                    
[22] GenomeInfoDb_1.38.7                     
[23] IRanges_2.36.0                          
[24] S4Vectors_0.40.2                        
[25] BiocGenerics_0.48.1                     
[26] ChIPseeker_1.38.0                       
[27] RColorBrewer_1.1-3                      
[28] kableExtra_1.4.0                        
[29] lubridate_1.9.3                         
[30] forcats_1.0.0                           
[31] stringr_1.5.1                           
[32] dplyr_1.1.4                             
[33] purrr_1.0.2                             
[34] readr_2.1.5                             
[35] tidyr_1.3.1                             
[36] tibble_3.2.1                            
[37] ggplot2_3.5.0                           
[38] tidyverse_2.0.0                         
[39] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.3                               
  [2] matrixStats_1.2.0                      
  [3] bitops_1.0-7                           
  [4] enrichplot_1.22.0                      
  [5] HDO.db_0.99.1                          
  [6] httr_1.4.7                             
  [7] profvis_0.3.8                          
  [8] backports_1.4.1                        
  [9] tools_4.3.1                            
 [10] utf8_1.2.4                             
 [11] R6_2.5.1                               
 [12] lazyeval_0.2.2                         
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.0                            
 [15] prettyunits_1.2.0                      
 [16] preprocessCore_1.64.0                  
 [17] cli_3.6.2                              
 [18] formatR_1.14                           
 [19] scatterpie_0.2.1                       
 [20] labeling_0.4.3                         
 [21] sass_0.4.9                             
 [22] Rsamtools_2.18.0                       
 [23] systemfonts_1.0.6                      
 [24] yulab.utils_0.1.4                      
 [25] DOSE_3.28.2                            
 [26] svglite_2.1.3                          
 [27] sessioninfo_1.2.2                      
 [28] plotrix_3.8-4                          
 [29] rstudioapi_0.15.0                      
 [30] RSQLite_2.3.5                          
 [31] generics_0.1.3                         
 [32] gridGraphics_0.5-1                     
 [33] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [34] BiocIO_1.12.0                          
 [35] gtools_3.9.5                           
 [36] car_3.1-2                              
 [37] GO.db_3.18.0                           
 [38] Matrix_1.6-5                           
 [39] fansi_1.0.6                            
 [40] abind_1.4-5                            
 [41] lifecycle_1.0.4                        
 [42] whisker_0.4.1                          
 [43] yaml_2.3.8                             
 [44] carData_3.0-5                          
 [45] SummarizedExperiment_1.32.0            
 [46] gplots_3.1.3.1                         
 [47] qvalue_2.34.0                          
 [48] SparseArray_1.2.4                      
 [49] BiocFileCache_2.10.1                   
 [50] blob_1.2.4                             
 [51] promises_1.2.1                         
 [52] crayon_1.5.2                           
 [53] miniUI_0.1.1.1                         
 [54] lattice_0.22-5                         
 [55] cowplot_1.1.3                          
 [56] KEGGREST_1.42.0                        
 [57] pillar_1.9.0                           
 [58] knitr_1.45                             
 [59] fgsea_1.28.0                           
 [60] rjson_0.2.21                           
 [61] boot_1.3-30                            
 [62] codetools_0.2-19                       
 [63] fastmatch_1.1-4                        
 [64] glue_1.7.0                             
 [65] getPass_0.2-4                          
 [66] ggfun_0.1.4                            
 [67] remotes_2.5.0                          
 [68] data.table_1.15.2                      
 [69] vctrs_0.6.5                            
 [70] png_0.1-8                              
 [71] treeio_1.26.0                          
 [72] gtable_0.3.4                           
 [73] cachem_1.0.8                           
 [74] xfun_0.42                              
 [75] S4Arrays_1.2.1                         
 [76] mime_0.12                              
 [77] tidygraph_1.3.1                        
 [78] statmod_1.5.0                          
 [79] interactiveDisplayBase_1.40.0          
 [80] ellipsis_0.3.2                         
 [81] nlme_3.1-164                           
 [82] ggtree_3.10.1                          
 [83] bit64_4.0.5                            
 [84] progress_1.2.3                         
 [85] filelock_1.0.3                         
 [86] rprojroot_2.0.4                        
 [87] bslib_0.6.1                            
 [88] affyio_1.72.0                          
 [89] KernSmooth_2.23-22                     
 [90] colorspace_2.1-0                       
 [91] DBI_1.2.2                              
 [92] tidyselect_1.2.1                       
 [93] processx_3.8.4                         
 [94] bit_4.0.5                              
 [95] compiler_4.3.1                         
 [96] curl_5.2.1                             
 [97] git2r_0.33.0                           
 [98] xml2_1.3.6                             
 [99] DelayedArray_0.28.0                    
[100] shadowtext_0.1.3                       
[101] caTools_1.18.2                         
[102] callr_3.7.5                            
[103] rappdirs_0.3.3                         
[104] digest_0.6.35                          
[105] rmarkdown_2.26                         
[106] XVector_0.42.0                         
[107] htmltools_0.5.7                        
[108] pkgconfig_2.0.3                        
[109] MatrixGenerics_1.14.0                  
[110] highr_0.10                             
[111] dbplyr_2.5.0                           
[112] fastmap_1.1.1                          
[113] htmlwidgets_1.6.4                      
[114] rlang_1.1.3                            
[115] shiny_1.8.0                            
[116] farver_2.1.1                           
[117] jquerylib_0.1.4                        
[118] jsonlite_1.8.8                         
[119] GOSemSim_2.28.1                        
[120] RCurl_1.98-1.14                        
[121] magrittr_2.0.3                         
[122] GenomeInfoDbData_1.2.11                
[123] ggplotify_0.1.2                        
[124] patchwork_1.2.0                        
[125] munsell_0.5.0                          
[126] Rcpp_1.0.12                            
[127] ape_5.7-1                              
[128] viridis_0.6.5                          
[129] stringi_1.8.3                          
[130] ggraph_2.2.1                           
[131] zlibbioc_1.48.0                        
[132] MASS_7.3-60.0.1                        
[133] pkgbuild_1.4.4                         
[134] AnnotationHub_3.10.0                   
[135] plyr_1.8.9                             
[136] parallel_4.3.1                         
[137] ggrepel_0.9.5                          
[138] Biostrings_2.70.2                      
[139] graphlayouts_1.1.1                     
[140] splines_4.3.1                          
[141] hms_1.1.3                              
[142] locfit_1.5-9.9                         
[143] ps_1.7.6                               
[144] igraph_2.0.3                           
[145] ggsignif_0.6.4                         
[146] pkgload_1.3.4                          
[147] reshape2_1.4.4                         
[148] biomaRt_2.58.2                         
[149] futile.options_1.0.1                   
[150] BiocVersion_3.18.1                     
[151] XML_3.99-0.16.1                        
[152] evaluate_0.23                          
[153] lambda.r_1.2.4                         
[154] BiocManager_1.30.22                    
[155] tzdb_0.4.0                             
[156] tweenr_2.0.3                           
[157] httpuv_1.6.14                          
[158] polyclip_1.10-6                        
[159] ggforce_0.4.2                          
[160] broom_1.0.5                            
[161] xtable_1.8-4                           
[162] restfulr_0.0.15                        
[163] tidytree_0.4.6                         
[164] rstatix_0.7.2                          
[165] later_1.3.2                            
[166] viridisLite_0.4.2                      
[167] aplot_0.2.2                            
[168] memoise_2.0.1                          
[169] GenomicAlignments_1.38.2               
[170] timechange_0.3.0                       </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
