<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2025-01-17" />

<title>Plot for 6c</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Plot for 6c</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2025-01-17</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-01-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree4a1d2bf877aaf7f52d0b091f4add59fdf97feffbtargetblank4a1d2bfa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/4a1d2bf877aaf7f52d0b091f4add59fdf97feffb" target="_blank">4a1d2bf</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree4a1d2bf877aaf7f52d0b091f4add59fdf97feffbtargetblank4a1d2bfa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/4a1d2bf877aaf7f52d0b091f4add59fdf97feffb" target="_blank">4a1d2bf</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  Correlationplot_scaled.pdf
    Untracked:  DOX_DAR_assess.Rmd
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/MRC_clusterlog2cpm.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  dataredo.RData
    Untracked:  datasave.RDS
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  output_1_Mecom.txt
    Untracked:  setup.RData
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/Enhancer_files_ff.Rmd
    Modified:   analysis/TE_analysis_ff.Rmd
    Modified:   analysis/final_four_analysis.Rmd
    Modified:   analysis/final_plot_attempt.Rmd
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/Correlation_of_GWASnPEAK.Rmd</code>) and HTML
(<code>docs/Correlation_of_GWASnPEAK.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4a1d2bf877aaf7f52d0b091f4add59fdf97feffb/analysis/Correlation_of_GWASnPEAK.Rmd" target="_blank">4a1d2bf</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
<td>
updates to plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/ae1542ccf79e3d6f74cfa1defac72e9355419585/docs/Correlation_of_GWASnPEAK.html" target="_blank">ae1542c</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/e179f611fcb6cc8a2ffea96328023383c0b10f42/analysis/Correlation_of_GWASnPEAK.Rmd" target="_blank">e179f61</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
<td>
updates with comments
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/d09c7dba1dd557be32e8b98f9d78292ddf16ea67/docs/Correlation_of_GWASnPEAK.html" target="_blank">d09c7db</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/20ed2fe1cf5a8ef38a6420b9dac02fba955dc25b/analysis/Correlation_of_GWASnPEAK.Rmd" target="_blank">20ed2fe</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
<td>
additional analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(ggfortify)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(edgeR)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(readxl)</code></pre>
<p>Notes to self(and anyone else who is reading this!):<br />
This is me applying the same code from my correlation_of_SNPnPeak.rmd
document.<br />
Summary of what I am doing: 1: create a list of peaks within +/-20 kb,
+/-10 kb, and +/- 5 kb of an RNA expressed gene TSS. (3 separate
lists)<br />
2: making a dataframe that has all ATAC 3 hour and 24 hr LFC by peak for
later ease of use. 3: creating lists of gwas SNPs (HF and ARR lists
only) that are either 1bp, 10kb, 20kb, or 50kb in length to determine
impact of the SNP on surrounding peaks.</p>
<pre class="r"><code>Collapsed_new_peaks &lt;- read_delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;, delim = &quot;\t&quot;, col_names = TRUE)
Collapsed_new_peaks_gr &lt;- Collapsed_new_peaks %&gt;% dplyr::select(chr:Peakid) %&gt;% GRanges()

peak_10kb_neargenes &lt;-
  Collapsed_new_peaks %&gt;% 
    dplyr::filter(dist_to_NG&lt;5000&amp;dist_to_NG&gt;-5000) %&gt;% 
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  dplyr::select(Peakid,NCBI_gene,SYMBOL)

peak_20kb_neargenes &lt;-
  Collapsed_new_peaks %&gt;% 
    dplyr::filter(dist_to_NG&lt;10000&amp;dist_to_NG&gt;-10000) %&gt;% 
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  dplyr::select(Peakid,NCBI_gene,SYMBOL)

peak_40kb_neargenes &lt;-
  Collapsed_new_peaks %&gt;% 
    dplyr::filter(dist_to_NG&lt;20000&amp;dist_to_NG&gt;-20000) %&gt;% 
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  dplyr::select(Peakid,NCBI_gene,SYMBOL)

RNA_median_3_lfc &lt;- readRDS(&quot;data/other_papers/RNA_median_3_lfc.RDS&quot;)
RNA_median_24_lfc &lt;- readRDS(&quot;data/other_papers/RNA_median_24_lfc.RDS&quot;)

ATAC_24_lfc &lt;- read_csv(&quot;data/Final_four_data/median_24_lfc.csv&quot;) 
ATAC_3_lfc &lt;- read_csv(&quot;data/Final_four_data/median_3_lfc.csv&quot;)



gwas_HF &lt;- readRDS(&quot;data/gwas_5_dataframe.RDS&quot;)
gwas_ARR &lt;- readRDS(&quot;data/gwas_2_dataframe.RDS&quot;)
Short_gwas_gr &lt;-
  gwas_ARR %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ARR&quot;) %&gt;% 
   rbind(gwas_HF %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HF&quot;)) %&gt;% 
  na.omit() %&gt;% 
 mutate(seqnames=paste0(&quot;chr&quot;,CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit() %&gt;%
   mutate(start=CHR_POS, end=CHR_POS, width=1) %&gt;% 
  GRanges()


Short_gwas_5k_gr &lt;- 
    gwas_ARR %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ARR&quot;) %&gt;% 
   rbind(gwas_HF %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HF&quot;)) %&gt;% 
  na.omit() %&gt;% 
 mutate(seqnames=paste0(&quot;chr&quot;,CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit() %&gt;%
   mutate(start=CHR_POS-5000, end=CHR_POS+4999) %&gt;% 
  GRanges()


Short_gwas_20k_gr &lt;- 
    gwas_ARR %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ARR&quot;) %&gt;% 
   rbind(gwas_HF %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HF&quot;)) %&gt;% 
  na.omit() %&gt;% 
 mutate(seqnames=paste0(&quot;chr&quot;,CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit() %&gt;%
  mutate(start=(CHR_POS-10000),end=(CHR_POS+9999), width=20000) %&gt;%
  distinct() %&gt;% 
  GRanges()


Short_gwas_50k_gr &lt;- 
    gwas_ARR %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ARR&quot;) %&gt;% 
   rbind(gwas_HF %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HF&quot;)) %&gt;% 
  na.omit() %&gt;% 
 mutate(seqnames=paste0(&quot;chr&quot;,CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit() %&gt;%
  mutate(start=(CHR_POS-25000),end=(CHR_POS+24999), width=50000) %&gt;%
  distinct() %&gt;% 
  GRanges()
 
gwas_peak_check &lt;- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_gr) %&gt;%
  as.data.frame()
# 
gwas_peak_check_10k &lt;- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_5k_gr) %&gt;%
  as.data.frame()
gwas_peak_check_20k &lt;- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_20k_gr) %&gt;% 
  as.data.frame()
 gwas_peak_check_50k &lt;- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_50k_gr) %&gt;% 
  as.data.frame()


ATAC_LFC &lt;- Collapsed_new_peaks %&gt;%
                 dplyr::select(Peakid) %&gt;% 
  left_join(.,(ATAC_3_lfc %&gt;% dplyr::select(peak, med_3h_lfc)), by=c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(ATAC_24_lfc %&gt;% dplyr::select(peak, med_24h_lfc)), by=c(&quot;Peakid&quot;=&quot;peak&quot;))</code></pre>
<p>Here I am doing the overlapping of the previous ranges of SNPs and
the full ATAC peak set. I also later create the data frames from the
reheat data, the reheat data using the p&lt;0.005 top genes, the cluster
names associated with each peak, and the list of TE/notTE associated
with each peak.</p>
<pre class="r"><code>ATAC_peaks_gr &lt;- Collapsed_new_peaks %&gt;% GRanges()

Peaks_cutoff &lt;- read_delim(&quot;data/Final_four_data/LCPM_matrix_ff.txt&quot;,delim = &quot;/&quot;) %&gt;% dplyr::select(Peakid)
  


gwas_short_list &lt;- gwas_peak_check %&gt;% as.data.frame %&gt;% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

gwas_10k_list &lt;- gwas_peak_check_10k %&gt;% distinct(SNPS,Peakid)%&gt;% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

gwas_20k_list &lt;- gwas_peak_check_20k %&gt;% distinct(SNPS,Peakid)%&gt;% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

gwas_50k_list &lt;- gwas_peak_check_50k %&gt;% distinct(SNPS,Peakid)%&gt;% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

Reheat_data &lt;- read_excel(&quot;data/other_papers/jah36123-sup-0002-tables2.xlsx&quot;)
top_reheat &lt;- Reheat_data %&gt;%
  dplyr::filter(fisher_pvalue&lt;0.005)
Nine_te_df &lt;- readRDS(&quot;data/Final_four_data/Nine_group_TE_df.RDS&quot;)
###needed to change TE status to at least 1 bp overlap
match &lt;- Nine_te_df %&gt;% 
   mutate(TEstatus=if_else(!is.na(per_ol),&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)) %&gt;% 
  distinct(Peakid,TEstatus,mrc,.keep_all = TRUE) </code></pre>
<div id="peaks-within-5kb---rna-tss-gwas-range---10-kb"
class="section level3">
<h3>Peaks within 5kb +/- RNA TSS, gwas range +/- 10 kb</h3>
<p>To break down what I am doing here: I start with the list of peaks
that overlap a gwas SNP that has been expanded by 20kb. I then only add
the RNA expressed genes associated with the peaks that are within +/- 5
kb of its TSS. I join the median LFC data frames for ATAC and RNA at 3
and 24 hours, the TEstatus, the reheat status and exclude any SNP-Peak
combinations that do not have RNA assigned. (This effectively is
filtering out peaks outside of the 10kb TSS range That would make the
list drop from 2019 to 298 rows)</p>
<pre class="r"><code>gwas_df &lt;-gwas_20k_list%&gt;% 
  as.data.frame() %&gt;%
  left_join(., peak_10kb_neargenes, by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;%
  left_join(., (ATAC_3_lfc %&gt;%
                  dplyr::select(peak,med_3h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., (ATAC_24_lfc %&gt;%
                  dplyr::select(peak,med_24h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;))%&gt;% 
  left_join(., RNA_median_3_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;%
  left_join(., RNA_median_24_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,&quot;reheat_gene&quot;,&quot;not_reheat_gene&quot;)) %&gt;% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %&gt;% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep =&quot;_&quot;,remove=FALSE) %&gt;% 
  left_join(.,(match %&gt;% 
                 group_by(Peakid) %&gt;%
                 filter(!(TEstatus==&quot;not_TE_peak&quot; &amp; any (TEstatus == &quot;TE_peak&quot;))) %&gt;% 
                 ungroup() %&gt;%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &amp;SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &amp;SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &amp;SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &amp;SNPS %in% gwas_50k_list$SNPS ~ 50)) %&gt;% 
  group_by(SNPS,Peakid) %&gt;% 
  summarize(name=unique(name),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=&quot;:&quot;),
           TEstatus=paste(unique(TEstatus),collapse=&quot;;&quot;),
          SYMBOL=paste(unique(SYMBOL),collapse=&quot;;&quot;),
           reheat=paste(unique(reheat),collapse=&quot;;&quot;),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP)) %&gt;% 
  na.omit(RNA_3h_lfc)

gwas_mat &lt;- gwas_df %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  as.matrix()
gwas_name_mat &lt;- gwas_df %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno &lt;- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat$TEstatus,reheat_status=gwas_name_mat$reheat,MRC=gwas_name_mat$mrc,direct_overlap=gwas_name_mat$dist_to_SNP,col= list(TE_status=c(&quot;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;TE_peak;not_TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak&quot;=&quot;lightblue&quot;),                                                     MRC = c(&quot;EAR_open&quot; = &quot;#F8766D&quot;,                                                                    &quot;EAR_close&quot; = &quot;#f6483c&quot;,&quot;ESR_open&quot; = &quot;#7CAE00&quot;,
                            &quot;ESR_close&quot; = &quot;#587b00&quot;,
                            &quot;ESR_opcl&quot;=&quot;grey40&quot;,
                            &quot;ESR_C&quot;=&quot;grey40&quot;,
                            &quot;ESR_clop&quot;=&quot;tan&quot;,
                             &quot;ESR_D&quot;=&quot;tan&quot;,
                               &quot;ESR_OC&quot; = &quot;#6a9500&quot;,
                                &quot;LR_open&quot; = &quot;#00BFC4&quot;,
                              &quot;LR_close&quot; = &quot;#008d91&quot;,
                              &quot;NR&quot; = &quot;#C77CFF&quot;,
                                &quot;not_mrc&quot;=&quot;black&quot;),
                    reheat_status=c(&quot;reheat_gene&quot;=&quot;green&quot;,&quot;not_reheat_gene&quot;=&quot;orange&quot;),
                    direct_overlap=c(&quot;0&quot;=&quot;red&quot;,&quot;10&quot;=&quot;pink&quot;,&quot;20&quot;=&quot;tan2&quot;,&quot;50&quot;=&quot;grey8&quot;)))
mat2 &lt;- gwas_mat  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = &quot;&quot;)
simply_map &lt;- ComplexHeatmap::Heatmap(gwas_mat,
                        left_annotation = row_anno,
                        show_row_names = TRUE,
                        # row_names_side = &quot;left&quot;,
                        row_names_max_width= max_text_width(rownames(gwas_mat),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = &quot;horizontal&quot;),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map, merge_legend = TRUE, heatmap_legend_side = &quot;bottom&quot;, 
    annotation_legend_side = &quot;bottom&quot;)</code></pre>
<p><img src="figure/Correlation_of_GWASnPEAK.Rmd/point-10k-20k-50k%20overlap-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-point-10k-20k-50k-overlap-1">
Past versions of "point-10k-20k-50k overlap-1.png"
</button>
</p>
<div id="fig-no-spaces-point-10k-20k-50k-overlap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d09c7dba1dd557be32e8b98f9d78292ddf16ea67/docs/figure/Correlation_of_GWASnPEAK.Rmd/point-10k-20k-50k overlap-1.png" target="_blank">d09c7db</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="peaks-within-5kb---rna-tss-gwas-range---25-kb"
class="section level3">
<h3>Peaks within 5kb +/- RNA TSS, gwas range +/- 25 kb</h3>
<p>For comparison, I went ahead and did the same this as above, but used
the +/- 25 kb expanded SNP range. This left me with 660 ATAC-SNP_RNA
sets.</p>
<pre class="r"><code>gwas_df &lt;-
gwas_50k_list%&gt;% 
  as.data.frame() %&gt;%
  left_join(., peak_10kb_neargenes, by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;%
  left_join(., (ATAC_3_lfc %&gt;%
                  dplyr::select(peak,med_3h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., (ATAC_24_lfc %&gt;%
                  dplyr::select(peak,med_24h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;))%&gt;% 
  left_join(., RNA_median_3_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;%
  left_join(., RNA_median_24_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,&quot;reheat_gene&quot;,&quot;not_reheat_gene&quot;)) %&gt;% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %&gt;% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep =&quot;_&quot;,remove=FALSE) %&gt;% 
  left_join(.,(match %&gt;% 
                 group_by(Peakid) %&gt;%
                 filter(!(TEstatus==&quot;not_TE_peak&quot; &amp; any (TEstatus == &quot;TE_peak&quot;))) %&gt;% 
                 ungroup() %&gt;%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &amp;SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &amp;SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &amp;SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &amp;SNPS %in% gwas_50k_list$SNPS ~ 50)) %&gt;% 
  group_by(SNPS,Peakid) %&gt;% 
  # mutate(Keep=case_when(SNPS))
  # group_by(Peakid) %&gt;% 
 summarize(name=unique(name),
           # SNPS=unique(SNPS),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=&quot;:&quot;),
           TEstatus=paste(unique(TEstatus),collapse=&quot;;&quot;),
          SYMBOL=paste(unique(SYMBOL),collapse=&quot;;&quot;),
           reheat=paste(unique(reheat),collapse=&quot;;&quot;),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %&gt;% 
  na.omit(RNA_3h_lfc)

gwas_mat &lt;- gwas_df %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  as.matrix()
gwas_name_mat &lt;- gwas_df %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno &lt;- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat$TEstatus,reheat_status=gwas_name_mat$reheat,MRC=gwas_name_mat$mrc,direct_overlap=gwas_name_mat$dist_to_SNP,col= list(TE_status=c(&quot;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;TE_peak;not_TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak&quot;=&quot;lightblue&quot;),                                                     MRC = c(&quot;EAR_open&quot; = &quot;#F8766D&quot;,                                                                    &quot;EAR_close&quot; = &quot;#f6483c&quot;,&quot;ESR_open&quot; = &quot;#7CAE00&quot;,
                            &quot;ESR_close&quot; = &quot;#587b00&quot;,
                            &quot;ESR_opcl&quot;=&quot;grey40&quot;,
                            &quot;ESR_C&quot;=&quot;grey40&quot;,
                            &quot;ESR_clop&quot;=&quot;tan&quot;,
                             &quot;ESR_D&quot;=&quot;tan&quot;,
                               &quot;ESR_OC&quot; = &quot;#6a9500&quot;,
                                &quot;LR_open&quot; = &quot;#00BFC4&quot;,
                              &quot;LR_close&quot; = &quot;#008d91&quot;,
                              &quot;NR&quot; = &quot;#C77CFF&quot;,
                                &quot;not_mrc&quot;=&quot;black&quot;),
                    reheat_status=c(&quot;reheat_gene&quot;=&quot;green&quot;,&quot;not_reheat_gene&quot;=&quot;orange&quot;),
                    direct_overlap=c(&quot;0&quot;=&quot;red&quot;,&quot;10&quot;=&quot;pink&quot;,&quot;20&quot;=&quot;tan2&quot;,&quot;50&quot;=&quot;grey8&quot;)))
mat2 &lt;- gwas_mat  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = &quot;&quot;)
simply_map &lt;- ComplexHeatmap::Heatmap(gwas_mat,
                        left_annotation = row_anno,
                        show_row_names = TRUE,
                        # row_names_side = &quot;left&quot;,
                        row_names_max_width= max_text_width(rownames(gwas_mat),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = &quot;horizontal&quot;),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map, merge_legend = TRUE, heatmap_legend_side = &quot;bottom&quot;, 
    annotation_legend_side = &quot;bottom&quot;)</code></pre>
<p><img src="figure/Correlation_of_GWASnPEAK.Rmd/point%20and%2010k%20only-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-point-and-10k-only-1">
Past versions of "point and 10k only-1.png"
</button>
</p>
<div id="fig-no-spaces-point-and-10k-only-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ae1542ccf79e3d6f74cfa1defac72e9355419585/docs/figure/Correlation_of_GWASnPEAK.Rmd/point and 10k only-1.png" target="_blank">ae1542c</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d09c7dba1dd557be32e8b98f9d78292ddf16ea67/docs/figure/Correlation_of_GWASnPEAK.Rmd/point and 10k only-1.png" target="_blank">d09c7db</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="peaks-within--20-kb-rna-tss-gwas-range---10-kb"
class="section level3">
<h3>Peaks within +/-20 kb RNA TSS ,gwas range +/- 10 kb</h3>
<p>warning, 702 rows below</p>
<pre class="r"><code>gwas_df &lt;-
gwas_20k_list%&gt;% 
  as.data.frame() %&gt;%
  left_join(., peak_40kb_neargenes, by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;%
  left_join(., (ATAC_3_lfc %&gt;%
                  dplyr::select(peak,med_3h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., (ATAC_24_lfc %&gt;%
                  dplyr::select(peak,med_24h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;))%&gt;% 
  left_join(., RNA_median_3_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;%
  left_join(., RNA_median_24_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,&quot;reheat_gene&quot;,&quot;not_reheat_gene&quot;)) %&gt;% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %&gt;% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep =&quot;_&quot;,remove=FALSE) %&gt;% 
  left_join(.,(match %&gt;% 
                 group_by(Peakid) %&gt;%
                 filter(!(TEstatus==&quot;not_TE_peak&quot; &amp; any (TEstatus == &quot;TE_peak&quot;))) %&gt;% 
                 ungroup() %&gt;%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &amp;SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &amp;SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &amp;SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &amp;SNPS %in% gwas_50k_list$SNPS ~ 50)) %&gt;% 
  group_by(SNPS,Peakid) %&gt;% 
  # mutate(Keep=case_when(SNPS))
  # group_by(Peakid) %&gt;% 
 summarize(name=unique(name),
           # SNPS=unique(SNPS),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=&quot;:&quot;),
           TEstatus=paste(unique(TEstatus),collapse=&quot;;&quot;),
          SYMBOL=paste(unique(SYMBOL),collapse=&quot;;&quot;),
           reheat=paste(unique(reheat),collapse=&quot;;&quot;),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %&gt;% 
  na.omit(RNA_3h_lfc)

gwas_mat &lt;- gwas_df %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  as.matrix()
gwas_name_mat &lt;- gwas_df %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno &lt;- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat$TEstatus,reheat_status=gwas_name_mat$reheat,MRC=gwas_name_mat$mrc,direct_overlap=gwas_name_mat$dist_to_SNP,col= list(TE_status=c(&quot;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;TE_peak;not_TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak&quot;=&quot;lightblue&quot;),                                                     MRC = c(&quot;EAR_open&quot; = &quot;#F8766D&quot;,                                                                    &quot;EAR_close&quot; = &quot;#f6483c&quot;,&quot;ESR_open&quot; = &quot;#7CAE00&quot;,
                            &quot;ESR_close&quot; = &quot;#587b00&quot;,
                            &quot;ESR_opcl&quot;=&quot;grey40&quot;,
                            &quot;ESR_C&quot;=&quot;grey40&quot;,
                            &quot;ESR_clop&quot;=&quot;tan&quot;,
                             &quot;ESR_D&quot;=&quot;tan&quot;,
                               &quot;ESR_OC&quot; = &quot;#6a9500&quot;,
                                &quot;LR_open&quot; = &quot;#00BFC4&quot;,
                              &quot;LR_close&quot; = &quot;#008d91&quot;,
                              &quot;NR&quot; = &quot;#C77CFF&quot;,
                                &quot;not_mrc&quot;=&quot;black&quot;),
                    reheat_status=c(&quot;reheat_gene&quot;=&quot;green&quot;,&quot;not_reheat_gene&quot;=&quot;orange&quot;),
                    direct_overlap=c(&quot;0&quot;=&quot;red&quot;,&quot;10&quot;=&quot;pink&quot;,&quot;20&quot;=&quot;tan2&quot;,&quot;50&quot;=&quot;grey8&quot;)))
mat2 &lt;- gwas_mat  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = &quot;&quot;)
simply_map &lt;- ComplexHeatmap::Heatmap(gwas_mat,
                        left_annotation = row_anno,
                        show_row_names = TRUE,
                        # row_names_side = &quot;left&quot;,
                        row_names_max_width= max_text_width(rownames(gwas_mat),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = &quot;horizontal&quot;),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map, merge_legend = TRUE, heatmap_legend_side = &quot;bottom&quot;, 
    annotation_legend_side = &quot;bottom&quot;)</code></pre>
<p><img src="figure/Correlation_of_GWASnPEAK.Rmd/20kb%20RNA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-20kb-RNA-1">
Past versions of "20kb RNA-1.png"
</button>
</p>
<div id="fig-no-spaces-20kb-RNA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ae1542ccf79e3d6f74cfa1defac72e9355419585/docs/figure/Correlation_of_GWASnPEAK.Rmd/20kb RNA-1.png" target="_blank">ae1542c</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d09c7dba1dd557be32e8b98f9d78292ddf16ea67/docs/figure/Correlation_of_GWASnPEAK.Rmd/20kb RNA-1.png" target="_blank">d09c7db</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="gwas-short-list-only" class="section level3">
<h3>gwas short list only</h3>
<p>To make life really easy, or the smallest set that was readable, I
used only the peaks that were directly overlapping a SNP, but filtered
out peaks that were more than +/- 5 kb from an expressed RNA TSS. This
gave me 33 ATAC-SNP-RNA rows.</p>
<pre class="r"><code>gwas_df_short &lt;-gwas_short_list%&gt;%
  as.data.frame() %&gt;%
  left_join(., peak_10kb_neargenes, by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;%
  left_join(., (ATAC_3_lfc %&gt;%
                  dplyr::select(peak,med_3h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., (ATAC_24_lfc %&gt;%
                  dplyr::select(peak,med_24h_lfc)),by=c(&quot;Peakid&quot;=&quot;peak&quot;))%&gt;% 
  left_join(., RNA_median_3_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;%
  left_join(., RNA_median_24_lfc,by =c(&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;, &quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;% 
  na.omit(RNA_median_24_lfc) %&gt;% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,&quot;reheat_gene&quot;,&quot;not_reheat_gene&quot;)) %&gt;% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %&gt;% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep =&quot;_&quot;,remove=FALSE) %&gt;% 
  left_join(.,(match %&gt;% 
                 group_by(Peakid) %&gt;%
                 filter(!(TEstatus==&quot;not_TE_peak&quot; &amp; any (TEstatus == &quot;TE_peak&quot;))) %&gt;% 
                 ungroup() %&gt;%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &amp;SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &amp;SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &amp;SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &amp;SNPS %in% gwas_50k_list$SNPS ~ 50)) %&gt;% 
  group_by(SNPS,Peakid) %&gt;% 
  # mutate(Keep=case_when(SNPS))
  # group_by(Peakid) %&gt;% 
 summarize(name=unique(name),
           # SNPS=unique(SNPS),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=&quot;:&quot;),
           TEstatus=paste(unique(TEstatus),collapse=&quot;;&quot;),
          SYMBOL=paste(unique(SYMBOL),collapse=&quot;;&quot;),
           reheat=paste(unique(reheat),collapse=&quot;;&quot;),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP)) #%&gt;% 
 
gwas_mat_short &lt;- gwas_df_short %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  as.matrix()
gwas_name_mat_short &lt;- gwas_df_short %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno_short &lt;- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat_short$TEstatus,reheat_status=gwas_name_mat_short$reheat,MRC=gwas_name_mat_short$mrc,direct_overlap=gwas_name_mat_short$dist_to_SNP,col= list(TE_status=c(&quot;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;TE_peak;not_TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak;TE_peak&quot;=&quot;goldenrod&quot;,
                                &quot;not_TE_peak&quot;=&quot;lightblue&quot;),                                                     MRC = c(&quot;EAR_open&quot; = &quot;#F8766D&quot;,                                                                    &quot;EAR_close&quot; = &quot;#f6483c&quot;,&quot;ESR_open&quot; = &quot;#7CAE00&quot;,
                            &quot;ESR_close&quot; = &quot;#587b00&quot;,
                            &quot;ESR_opcl&quot;=&quot;grey40&quot;,
                            &quot;ESR_C&quot;=&quot;grey40&quot;,
                            &quot;ESR_clop&quot;=&quot;tan&quot;,
                             &quot;ESR_D&quot;=&quot;tan&quot;,
                               &quot;ESR_OC&quot; = &quot;#6a9500&quot;,
                                &quot;LR_open&quot; = &quot;#00BFC4&quot;,
                              &quot;LR_close&quot; = &quot;#008d91&quot;,
                              &quot;NR&quot; = &quot;#C77CFF&quot;,
                                &quot;not_mrc&quot;=&quot;black&quot;),
                    reheat_status=c(&quot;reheat_gene&quot;=&quot;green&quot;,&quot;not_reheat_gene&quot;=&quot;orange&quot;),
                    direct_overlap=c(&quot;0&quot;=&quot;red&quot;,&quot;10&quot;=&quot;pink&quot;,&quot;20&quot;=&quot;tan2&quot;,&quot;50&quot;=&quot;grey8&quot;)))
mat2_short &lt;- gwas_mat_short  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = &quot;&quot;)
simply_map_short &lt;- ComplexHeatmap::Heatmap(gwas_mat_short,
                        left_annotation = row_anno_short,
                        show_row_names = TRUE,
                        # row_names_side = &quot;left&quot;,
                        row_names_max_width= max_text_width(rownames(gwas_mat_short),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = &quot;horizontal&quot;),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_short, merge_legend = TRUE, heatmap_legend_side = &quot;bottom&quot;, 
    annotation_legend_side = &quot;bottom&quot;)</code></pre>
<p><img src="figure/Correlation_of_GWASnPEAK.Rmd/noRNA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-noRNA-1">
Past versions of noRNA-1.png
</button>
</p>
<div id="fig-noRNA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ae1542ccf79e3d6f74cfa1defac72e9355419585/docs/figure/Correlation_of_GWASnPEAK.Rmd/noRNA-1.png" target="_blank">ae1542c</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d09c7dba1dd557be32e8b98f9d78292ddf16ea67/docs/figure/Correlation_of_GWASnPEAK.Rmd/noRNA-1.png" target="_blank">d09c7db</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="mapping-genes-of-interest" class="section level3">
<h3>mapping genes of interest:</h3>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
# K27_counts &lt;-  readRDS(&quot;data/Final_four_data/All_Raodahpeaks.RDS&quot;)
ATAC_counts &lt;- readRDS(&quot;data/Final_four_data/x4_filtered.RDS&quot;)
RNA_counts &lt;- readRDS(&quot;data/other_papers/Counts_RNA_ERMatthews.RDS&quot;)</code></pre>
<pre class="r"><code>df_gene &lt;- data.frame(SYMBOL=c(&quot;PSRC1&quot;,&quot;CDKN1A&quot;))
df_gene &lt;- df_gene %&gt;% 
left_join(., (RNA_median_24_lfc %&gt;% dplyr::select(ENTREZID,SYMBOL)), by = c (&quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) %&gt;% 
  left_join(., (gwas_df_short %&gt;% dplyr::select(SNPS,Peakid,mrc,SYMBOL)),by = c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;)) 
  
            
RNA_counts %&gt;% 
  column_to_rownames(&quot;ENTREZID&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(row.names(.) %in% df_gene$ENTREZID) %&gt;% 
  mutate(ENTREZID = row.names(.)) %&gt;%
  pivot_longer(cols = !ENTREZID, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  left_join(., df_gene, by =c(&quot;ENTREZID&quot;=&quot;ENTREZID&quot;)) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;trt&quot;,&quot;ind&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL+Peakid, scales=&quot;free_y&quot;)+
  ggtitle(&quot;RNA LFC of expressed gene&quot;)+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab(&quot;log2 cpm RNA&quot;)</code></pre>
<p><img src="figure/Correlation_of_GWASnPEAK.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ATAC_counts %&gt;% 
  cpm(., log = TRUE) %&gt;% 
   as.data.frame() %&gt;%
  rename_with(.,~gsub(pattern = &quot;Ind1_75&quot;, replacement = &quot;1_&quot;,.)) %&gt;%
  rename_with(.,~gsub(pattern = &quot;Ind2_87&quot;, replacement = &quot;2_&quot;,.)) %&gt;%
  rename_with(.,~gsub(pattern = &quot;Ind3_77&quot;, replacement = &quot;3_&quot;,.)) %&gt;%
  rename_with(.,~gsub(pattern = &quot;Ind6_71&quot;, replacement = &quot;6_&quot;,.)) %&gt;%
  rename_with(.,~gsub( &quot;DX&quot; ,&#39;DOX&#39;,.)) %&gt;%
  rename_with(.,~gsub( &quot;DA&quot; ,&#39;DNR&#39;,.)) %&gt;%
  rename_with(.,~gsub( &quot;E&quot; ,&#39;EPI&#39;,.)) %&gt;%
  rename_with(.,~gsub( &quot;T&quot; ,&#39;TRZ&#39;,.)) %&gt;%
  rename_with(.,~gsub( &quot;M&quot; ,&#39;MTX&#39;,.)) %&gt;%
  rename_with(.,~gsub( &quot;V&quot; ,&#39;VEH&#39;,.)) %&gt;%
  rename_with(.,~gsub(&quot;24h&quot;,&quot;_24h&quot;,.)) %&gt;%
  rename_with(.,~gsub(&quot;3h&quot;,&quot;_3h&quot;,.)) %&gt;% 
  dplyr::filter(row.names(.) %in% df_gene$Peakid) %&gt;% 
  mutate(Peakid = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peakid, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  left_join(., df_gene, by =c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;ind&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~Peakid+SYMBOL,scales=&quot;free_y&quot;)+
  ggtitle(&quot; ATAC accessibility&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab(&quot;log2 cpm ATAC&quot;)</code></pre>
<p><img src="figure/Correlation_of_GWASnPEAK.Rmd/unnamed-chunk-3-2.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] readxl_1.4.3                            
 [2] smplot2_0.2.4                           
 [3] cowplot_1.1.3                           
 [4] ComplexHeatmap_2.22.0                   
 [5] ggrepel_0.9.6                           
 [6] plyranges_1.26.0                        
 [7] ggsignif_0.6.4                          
 [8] genomation_1.38.0                       
 [9] edgeR_4.4.1                             
[10] limma_3.62.1                            
[11] ggpubr_0.6.0                            
[12] BiocParallel_1.40.0                     
[13] ggVennDiagram_1.5.2                     
[14] scales_1.3.0                            
[15] VennDiagram_1.7.3                       
[16] futile.logger_1.4.3                     
[17] gridExtra_2.3                           
[18] ggfortify_0.4.17                        
[19] rtracklayer_1.66.0                      
[20] org.Hs.eg.db_3.20.0                     
[21] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[22] GenomicFeatures_1.58.0                  
[23] AnnotationDbi_1.68.0                    
[24] Biobase_2.66.0                          
[25] GenomicRanges_1.58.0                    
[26] GenomeInfoDb_1.42.1                     
[27] IRanges_2.40.1                          
[28] S4Vectors_0.44.0                        
[29] BiocGenerics_0.52.0                     
[30] RColorBrewer_1.1-3                      
[31] broom_1.0.7                             
[32] kableExtra_1.4.0                        
[33] lubridate_1.9.4                         
[34] forcats_1.0.0                           
[35] stringr_1.5.1                           
[36] dplyr_1.1.4                             
[37] purrr_1.0.2                             
[38] readr_2.1.5                             
[39] tidyr_1.3.1                             
[40] tibble_3.2.1                            
[41] ggplot2_3.5.1                           
[42] tidyverse_2.0.0                         
[43] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] later_1.4.1                 BiocIO_1.16.0              
  [3] bitops_1.0-9                cellranger_1.1.0           
  [5] rpart_4.1.23                XML_3.99-0.17              
  [7] lifecycle_1.0.4             rstatix_0.7.2              
  [9] doParallel_1.0.17           rprojroot_2.0.4            
 [11] vroom_1.6.5                 processx_3.8.4             
 [13] lattice_0.22-6              backports_1.5.0            
 [15] magrittr_2.0.3              Hmisc_5.2-1                
 [17] sass_0.4.9                  rmarkdown_2.29             
 [19] jquerylib_0.1.4             yaml_2.3.10                
 [21] plotrix_3.8-4               httpuv_1.6.15              
 [23] DBI_1.2.3                   abind_1.4-8                
 [25] zlibbioc_1.52.0             RCurl_1.98-1.16            
 [27] nnet_7.3-19                 git2r_0.35.0               
 [29] circlize_0.4.16             GenomeInfoDbData_1.2.13    
 [31] svglite_2.1.3               codetools_0.2-20           
 [33] DelayedArray_0.32.0         xml2_1.3.6                 
 [35] tidyselect_1.2.1            shape_1.4.6.1              
 [37] farver_2.1.2                UCSC.utils_1.2.0           
 [39] base64enc_0.1-3             matrixStats_1.4.1          
 [41] GenomicAlignments_1.42.0    jsonlite_1.8.9             
 [43] GetoptLong_1.0.5            Formula_1.2-5              
 [45] iterators_1.0.14            systemfonts_1.1.0          
 [47] foreach_1.5.2               tools_4.4.2                
 [49] Rcpp_1.0.13-1               glue_1.8.0                 
 [51] SparseArray_1.6.0           xfun_0.49                  
 [53] MatrixGenerics_1.18.0       withr_3.0.2                
 [55] formatR_1.14                fastmap_1.2.0              
 [57] callr_3.7.6                 digest_0.6.37              
 [59] timechange_0.3.0            R6_2.5.1                   
 [61] seqPattern_1.38.0           colorspace_2.1-1           
 [63] RSQLite_2.3.9               generics_0.1.3             
 [65] data.table_1.16.4           htmlwidgets_1.6.4          
 [67] httr_1.4.7                  S4Arrays_1.6.0             
 [69] whisker_0.4.1               pkgconfig_2.0.3            
 [71] gtable_0.3.6                blob_1.2.4                 
 [73] impute_1.80.0               XVector_0.46.0             
 [75] htmltools_0.5.8.1           carData_3.0-5              
 [77] pwr_1.3-0                   clue_0.3-66                
 [79] png_0.1-8                   knitr_1.49                 
 [81] lambda.r_1.2.4              rstudioapi_0.17.1          
 [83] tzdb_0.4.0                  reshape2_1.4.4             
 [85] rjson_0.2.23                checkmate_2.3.2            
 [87] curl_6.0.1                  zoo_1.8-12                 
 [89] cachem_1.1.0                GlobalOptions_0.1.2        
 [91] KernSmooth_2.23-24          parallel_4.4.2             
 [93] foreign_0.8-87              restfulr_0.0.15            
 [95] pillar_1.10.0               vctrs_0.6.5                
 [97] promises_1.3.2              car_3.1-3                  
 [99] cluster_2.1.8               htmlTable_2.4.3            
[101] evaluate_1.0.1              magick_2.8.5               
[103] cli_3.6.3                   locfit_1.5-9.10            
[105] compiler_4.4.2              futile.options_1.0.1       
[107] Rsamtools_2.22.0            rlang_1.1.4                
[109] crayon_1.5.3                labeling_0.4.3             
[111] ps_1.8.1                    getPass_0.2-4              
[113] plyr_1.8.9                  fs_1.6.5                   
[115] stringi_1.8.4               viridisLite_0.4.2          
[117] gridBase_0.4-7              munsell_0.5.1              
[119] Biostrings_2.74.1           Matrix_1.7-1               
[121] BSgenome_1.74.0             patchwork_1.3.0            
[123] hms_1.1.3                   bit64_4.5.2                
[125] KEGGREST_1.46.0             statmod_1.5.0              
[127] SummarizedExperiment_1.36.0 memoise_2.0.1              
[129] bslib_0.8.0                 bit_4.5.0.1                </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
