<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-05-06" />

<title>DAR_analysis</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">DAR_analysis</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-05-06</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-06-13
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025tree352072cff4456db27a4929907609bc60f72d3b93targetblank352072ca">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/352072cff4456db27a4929907609bc60f72d3b93" target="_blank">352072c</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025tree352072cff4456db27a4929907609bc60f72d3b93targetblank352072ca"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/352072cff4456db27a4929907609bc60f72d3b93" target="_blank">352072c</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/H3K27ac_integration_noM.Rmd
    Ignored:    analysis/figure/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  RNA_seq_integration.Rmd
    Untracked:  Rplot.pdf
    Untracked:  Sig_meta
    Untracked:  analysis/.gitignore
    Untracked:  analysis/AC_shared_analysis.Rmd
    Untracked:  analysis/Cormotif_analysis_testing diff.Rmd
    Untracked:  analysis/Diagnosis-tmm.Rmd
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SNP_TAD_peaks.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/Top2B_analysis.Rmd
    Untracked:  analysis/making_master_peaks_list.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/Concatenations_for_export.R
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  my_plot.pdf
    Untracked:  my_plot.png
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/AF_HF_SNPs.Rmd
    Modified:   analysis/Cardiotox_SNPs.Rmd
    Modified:   analysis/Cormotif_analysis.Rmd
    Modified:   analysis/H3K27ac_initial_QC.Rmd
    Modified:   analysis/Jaspar_motif.Rmd
    Modified:   analysis/Jaspar_motif_ff.Rmd
    Modified:   analysis/TE_analysis_norm.Rmd
    Modified:   analysis/final_four_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/DEG_analysis.Rmd</code>) and HTML
(<code>docs/DEG_analysis.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/352072cff4456db27a4929907609bc60f72d3b93/analysis/DEG_analysis.Rmd" target="_blank">352072c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-13
</td>
<td>
updates to y and x axis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/DEG_analysis.html" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/601b63fa261330dd115b94ce15d41f86d5bbe4e6/analysis/DEG_analysis.Rmd" target="_blank">601b63f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
<td>
updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/eeaa3e6d4f6995208ca73f3731360b2573d86811/docs/DEG_analysis.html" target="_blank">eeaa3e6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7a30fff2fbf4f076ebc52ef6b7a07f81c06d5f26/analysis/DEG_analysis.Rmd" target="_blank">7a30fff</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-09
</td>
<td>
updateing enrichment test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/DEG_analysis.html" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/23a3ce13ad3a3a23f50845e5f2f03a76d0b39efd/analysis/DEG_analysis.Rmd" target="_blank">23a3ce1</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
<td>
adding in DAR analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/54e129a91ebbccd57aa37304477ba7555a7b225c/docs/DEG_analysis.html" target="_blank">54e129a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/cf05574ad343984d77e69cdddb51733b982ab92b/docs/DEG_analysis.html" target="_blank">cf05574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/19d6784f13ee74e7e6d723ba1f040cf96e1126e5/analysis/DEG_analysis.Rmd" target="_blank">19d6784</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
<td>
updates to volcano plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/DEG_analysis.html" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/9716d1aff93d8ebb21c9d2d18285524aeaca9c0f/analysis/DEG_analysis.Rmd" target="_blank">9716d1a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
<td>
updated with top3
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/cb0030eba148c79602eff81888c166c1269a43b5/docs/DEG_analysis.html" target="_blank">cb0030e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/df7eb88b3d80b927ad0d1e3206ab547af9d467f3/analysis/DEG_analysis.Rmd" target="_blank">df7eb88</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-08
</td>
<td>
spelling correction
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/5e6e4626ba4b860114d92a9fce8e3a6058534793/docs/DEG_analysis.html" target="_blank">5e6e462</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d969893e9d577cbba1f4c6a9ab4d3a6ad0bb1fb7/analysis/DEG_analysis.Rmd" target="_blank">d969893</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-07
</td>
<td>
updating new pages
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/DEG_analysis.html" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/01178cfceaab27a324812eb2f0cb7c03c91cdfa3/analysis/DEG_analysis.Rmd" target="_blank">01178cf</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
<td>
adding in segment
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(eulerr)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(data.table)
library(ggVennDiagram)</code></pre>
<div id="differential-analysis" class="section level3">
<h3>Differential analysis</h3>
<p>Loading counts matrix and making filtered matrix</p>
<pre class="r"><code>raw_counts &lt;- read_delim(&quot;data/Final_four_data/re_analysis/Raw_unfiltered_counts.tsv&quot;,delim=&quot;\t&quot;) %&gt;% 
  column_to_rownames(&quot;Peakid&quot;) %&gt;% 
  as.matrix()

lcpm &lt;- cpm(raw_counts, log= TRUE)
  ### for determining the basic cutoffs
filt_raw_counts &lt;- raw_counts[rowMeans(lcpm)&gt; 0,]

filt_raw_counts_noY &lt;- filt_raw_counts[!grepl(&quot;chrY&quot;,rownames(filt_raw_counts)),]
dim(filt_raw_counts_noY)</code></pre>
<pre><code>[1] 155557     48</code></pre>
<p>Number of filtered regions without the y chromosome = 155557
regions</p>
<div id="making-the-metadata-form" class="section level4">
<h4>making the metadata form</h4>
<pre class="r"><code>annotation_mat &lt;- data.frame(timeset=colnames(filt_raw_counts_noY)) %&gt;%
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;% 
  mutate(indv=factor(indv, levels = c(&quot;A&quot;,&quot;B&quot;,&quot;C&quot;,&quot;D&quot;))) %&gt;% 
  mutate(trt_time=paste0(trt,&quot;_&quot;,time))</code></pre>
<p>prepare DGE object</p>
<pre class="r"><code>group &lt;- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group &lt;- factor(group, levels =c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;,&quot;7&quot;,&quot;8&quot;,&quot;9&quot;,&quot;10&quot;,&quot;11&quot;,&quot;12&quot;))
dge &lt;-  DGEList.data.frame(counts = filt_raw_counts_noY, group = group, genes = row.names(filt_raw_counts_noY))
dge &lt;- calcNormFactors(dge)

dge$samples</code></pre>
<pre><code>          group lib.size norm.factors
D_DNR_24h     1 16022907    1.0239692
D_DNR_3h      2 12283494    0.9612342
D_DOX_24h     3 17860884    1.0367665
D_DOX_3h      4 13506791    1.0325656
D_EPI_24h     5 18628141    1.0327372
D_EPI_3h      6 11218019    1.0171289
D_MTX_24h     7 15070579    1.1107812
D_MTX_3h      8  8224116    1.0938773
D_TRZ_24h     9 13765197    0.9916489
D_TRZ_3h     10  9838944    1.0289011
D_VEH_24h    11 18137669    0.9855606
D_VEH_3h     12  5215243    1.1193711
A_DNR_24h     1 12446867    0.9913953
A_DNR_3h      2 13336679    0.9109168
A_DOX_24h     3 11024760    0.8994761
A_DOX_3h      4 11312301    0.9817107
A_EPI_24h     5 10054890    0.8306893
A_EPI_3h      6 13289458    0.8846067
A_MTX_24h     7 12051332    1.0488547
A_MTX_3h      8 19529308    0.9756453
A_TRZ_24h     9 11144980    0.8850322
A_TRZ_3h     10 10815793    0.9696953
A_VEH_24h    11 10644539    0.9044966
A_VEH_3h     12 10146179    1.0015305
B_DNR_24h     1  8695642    1.0170461
B_DNR_3h      2 11572135    0.8666718
B_DOX_24h     3  7780737    1.0039941
B_DOX_3h      4  6315637    0.8935147
B_EPI_24h     5  7912993    1.0275056
B_EPI_3h      6  7196001    0.9035920
B_MTX_24h     7  7434261    1.0947453
B_MTX_3h      8 10544429    0.8769442
B_TRZ_24h     9  6552039    0.9772581
B_TRZ_3h     10  6390372    0.9027404
B_VEH_24h    11  3521378    1.0063550
B_VEH_3h     12  4936492    1.0027569
C_DNR_24h     1 11796366    1.0773328
C_DNR_3h      2  6968392    1.0576684
C_DOX_24h     3  8352016    1.1219236
C_DOX_3h      4  5992702    1.0623451
C_EPI_24h     5  7970178    1.1143342
C_EPI_3h      6  5933236    1.0854547
C_MTX_24h     7  5584157    1.1803465
C_MTX_3h      8  9157251    1.0227009
C_TRZ_24h     9  5662913    1.0288892
C_TRZ_3h     10  4552166    1.0697477
C_VEH_24h    11  7597538    1.0237355
C_VEH_3h     12  6681133    1.0107246</code></pre>
<p>Making model matrix</p>
<pre class="r"><code>group_1 &lt;- c(rep(c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;),4))

 mm &lt;- model.matrix(~0 +group_1)
colnames(mm) &lt;-  c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;, &quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
mm</code></pre>
<pre><code>   DNR_24 DNR_3 DOX_24 DOX_3 EPI_24 EPI_3 MTX_24 MTX_3 TRZ_24 TRZ_3 VEH_24
1       1     0      0     0      0     0      0     0      0     0      0
2       0     1      0     0      0     0      0     0      0     0      0
3       0     0      1     0      0     0      0     0      0     0      0
4       0     0      0     1      0     0      0     0      0     0      0
5       0     0      0     0      1     0      0     0      0     0      0
6       0     0      0     0      0     1      0     0      0     0      0
7       0     0      0     0      0     0      1     0      0     0      0
8       0     0      0     0      0     0      0     1      0     0      0
9       0     0      0     0      0     0      0     0      1     0      0
10      0     0      0     0      0     0      0     0      0     1      0
11      0     0      0     0      0     0      0     0      0     0      1
12      0     0      0     0      0     0      0     0      0     0      0
13      1     0      0     0      0     0      0     0      0     0      0
14      0     1      0     0      0     0      0     0      0     0      0
15      0     0      1     0      0     0      0     0      0     0      0
16      0     0      0     1      0     0      0     0      0     0      0
17      0     0      0     0      1     0      0     0      0     0      0
18      0     0      0     0      0     1      0     0      0     0      0
19      0     0      0     0      0     0      1     0      0     0      0
20      0     0      0     0      0     0      0     1      0     0      0
21      0     0      0     0      0     0      0     0      1     0      0
22      0     0      0     0      0     0      0     0      0     1      0
23      0     0      0     0      0     0      0     0      0     0      1
24      0     0      0     0      0     0      0     0      0     0      0
25      1     0      0     0      0     0      0     0      0     0      0
26      0     1      0     0      0     0      0     0      0     0      0
27      0     0      1     0      0     0      0     0      0     0      0
28      0     0      0     1      0     0      0     0      0     0      0
29      0     0      0     0      1     0      0     0      0     0      0
30      0     0      0     0      0     1      0     0      0     0      0
31      0     0      0     0      0     0      1     0      0     0      0
32      0     0      0     0      0     0      0     1      0     0      0
33      0     0      0     0      0     0      0     0      1     0      0
34      0     0      0     0      0     0      0     0      0     1      0
35      0     0      0     0      0     0      0     0      0     0      1
36      0     0      0     0      0     0      0     0      0     0      0
37      1     0      0     0      0     0      0     0      0     0      0
38      0     1      0     0      0     0      0     0      0     0      0
39      0     0      1     0      0     0      0     0      0     0      0
40      0     0      0     1      0     0      0     0      0     0      0
41      0     0      0     0      1     0      0     0      0     0      0
42      0     0      0     0      0     1      0     0      0     0      0
43      0     0      0     0      0     0      1     0      0     0      0
44      0     0      0     0      0     0      0     1      0     0      0
45      0     0      0     0      0     0      0     0      1     0      0
46      0     0      0     0      0     0      0     0      0     1      0
47      0     0      0     0      0     0      0     0      0     0      1
48      0     0      0     0      0     0      0     0      0     0      0
   VEH_3
1      0
2      0
3      0
4      0
5      0
6      0
7      0
8      0
9      0
10     0
11     0
12     1
13     0
14     0
15     0
16     0
17     0
18     0
19     0
20     0
21     0
22     0
23     0
24     1
25     0
26     0
27     0
28     0
29     0
30     0
31     0
32     0
33     0
34     0
35     0
36     1
37     0
38     0
39     0
40     0
41     0
42     0
43     0
44     0
45     0
46     0
47     0
48     1
attr(,&quot;assign&quot;)
 [1] 1 1 1 1 1 1 1 1 1 1 1 1
attr(,&quot;contrasts&quot;)
attr(,&quot;contrasts&quot;)$group_1
[1] &quot;contr.treatment&quot;</code></pre>
<p>In this pipeline, I first run voom transformation, then estimate the
intra-individual correlation. Next I do voom again with correlation
info. I fit the linear model, define contrasts, then apply the contrasts
and perform eBayes to get statistics.</p>
<pre class="r"><code>y &lt;- voom(dge, mm,plot =FALSE)

corfit &lt;- duplicateCorrelation(y, mm, block = annotation_mat$indv)
 
v &lt;- voom(dge, mm, block = annotation_mat$indv, correlation = corfit$consensus)

fit &lt;- lmFit(v, mm, block = annotation_mat$indv, correlation = corfit$consensus)


cm &lt;- makeContrasts(
  DNR_3.VEH_3 = DNR_3-VEH_3,
  DOX_3.VEH_3 = DOX_3-VEH_3,
  EPI_3.VEH_3 = EPI_3-VEH_3,
  MTX_3.VEH_3 = MTX_3-VEH_3,
  TRZ_3.VEH_3 = TRZ_3-VEH_3,
  DNR_24.VEH_24 =DNR_24-VEH_24,
  DOX_24.VEH_24= DOX_24-VEH_24,
  EPI_24.VEH_24= EPI_24-VEH_24,
  MTX_24.VEH_24= MTX_24-VEH_24,
  TRZ_24.VEH_24= TRZ_24-VEH_24,
  levels = mm)


fit2&lt;- contrasts.fit(fit, contrasts=cm)

efit2 &lt;- eBayes(fit2)

results = decideTests(efit2)

summary(results)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 TRZ_3.VEH_3
Down         10868        2244        7162         444           1
NotSig      132819      152084      141323      154753      155556
Up           11870        1229        7072         360           0
       DNR_24.VEH_24 DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24 TRZ_24.VEH_24
Down           39400         32313         32932         14182             0
NotSig         75562         90737         89056        131307        155557
Up             40595         32507         33569         10068             0</code></pre>
<pre class="r"><code>plotSA(efit2, main=&quot;Mean-Variance trend for final model&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/figure/DEG_analysis.Rmd/unnamed-chunk-3-1.png" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>V.DNR_3.top= topTable(efit2, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_3.top= topTable(efit2, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_3.top= topTable(efit2, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_3.top= topTable(efit2, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_3.top= topTable(efit2, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DNR_24.top= topTable(efit2, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_24.top= topTable(efit2, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_24.top= topTable(efit2, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_24.top= topTable(efit2, coef=9, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_24.top= topTable(efit2, coef=10, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)


# plot_filenames &lt;- c(&quot;V.DNR_3.top&quot;,&quot;V.DOX_3.top&quot;,&quot;V.EPI_3.top&quot;,&quot;V.MTX_3.top&quot;,
#                     &quot;V.TRZ_.top&quot;,&quot;V.DNR_24.top&quot;,&quot;V.DOX_24.top&quot;,&quot;V.EPI_24.top&quot;,
#                     &quot;V.MTX_24.top&quot;,&quot;V.TRZ_24.top&quot;)
# plot_files &lt;- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
#                     V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
#                     V.MTX_24.top,V.TRZ_24.top)

save_list &lt;- list(&quot;DNR_3&quot;=V.DNR_3.top,&quot;DOX_3&quot;=V.DOX_3.top,&quot;EPI_3&quot;=V.EPI_3.top,&quot;MTX_3&quot;=V.MTX_3.top,&quot;TRZ_3&quot;=V.TRZ_3.top,&quot;DNR_24&quot;=V.DNR_24.top,&quot;DOX_24&quot;=V.DOX_24.top,&quot;EPI_24&quot;=V.EPI_24.top,&quot;MTX_24&quot;= V.MTX_24.top, &quot;TRZ_24&quot;=V.TRZ_24.top)

saveRDS(save_list,&quot;data/Final_four_data/re_analysis/Toptable_results.RDS&quot;)</code></pre>
</div>
</div>
<div id="volcano-plots" class="section level3">
<h3>Volcano Plots</h3>
<pre class="r"><code>volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
      # ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC &gt;= 0,&quot;B&quot;, &quot;C&quot;)))
    ##This is where I could add labels, but I have taken out
    # df &lt;- df %&gt;% mutate(genelabels = &quot;&quot;)
    # df$genelabels[1:topg] &lt;- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    ggrastr::geom_point_rast(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, &quot;npc&quot;)), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    ylim(0,25)+
    xlim(-6,6)+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}

v1 &lt;- volcanosig(V.DNR_3.top, 0.05)+ ggtitle(&quot;DNR 3 hour&quot;)
v2 &lt;- volcanosig(V.DNR_24.top, 0.05)+ ggtitle(&quot;DNR 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(V.DOX_3.top, 0.05)+ ggtitle(&quot;DOX 3 hour&quot;)
v4 &lt;- volcanosig(V.DOX_24.top, 0.05)+ ggtitle(&quot;DOX 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(V.EPI_3.top, 0.05)+ ggtitle(&quot;EPI 3 hour&quot;)
v6 &lt;- volcanosig(V.EPI_24.top, 0.05)+ ggtitle(&quot;EPI 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(V.MTX_3.top, 0.05)+ ggtitle(&quot;MTX 3 hour&quot;)
v8 &lt;- volcanosig(V.MTX_24.top, 0.05)+ ggtitle(&quot;MTX 24 hour&quot;)+ylab(&quot;&quot;)
v9 &lt;- volcanosig(V.TRZ_3.top, 0.05)+ ggtitle(&quot;TRZ 3 hour&quot;)
v10 &lt;- volcanosig(V.TRZ_24.top, 0.05)+ ggtitle(&quot;TRZ 24 hour&quot;)+ylab(&quot;&quot;)

plot_grid(v1,v2,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Volcano%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-1">
Past versions of "Volcano plots-1.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/cf05574ad343984d77e69cdddb51733b982ab92b/docs/figure/DEG_analysis.Rmd/Volcano plots-1.png" target="_blank">cf05574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/figure/DEG_analysis.Rmd/Volcano plots-1.png" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3,v4,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Volcano%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-2">
Past versions of "Volcano plots-2.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/cf05574ad343984d77e69cdddb51733b982ab92b/docs/figure/DEG_analysis.Rmd/Volcano plots-2.png" target="_blank">cf05574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/figure/DEG_analysis.Rmd/Volcano plots-2.png" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5,v6,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Volcano%20plots-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-3">
Past versions of "Volcano plots-3.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/cf05574ad343984d77e69cdddb51733b982ab92b/docs/figure/DEG_analysis.Rmd/Volcano plots-3.png" target="_blank">cf05574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/figure/DEG_analysis.Rmd/Volcano plots-3.png" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7,v8,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Volcano%20plots-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-4">
Past versions of "Volcano plots-4.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/cf05574ad343984d77e69cdddb51733b982ab92b/docs/figure/DEG_analysis.Rmd/Volcano plots-4.png" target="_blank">cf05574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/figure/DEG_analysis.Rmd/Volcano plots-4.png" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v9,v10,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Volcano%20plots-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-5">
Past versions of "Volcano plots-5.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/cf05574ad343984d77e69cdddb51733b982ab92b/docs/figure/DEG_analysis.Rmd/Volcano plots-5.png" target="_blank">cf05574</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fee38750b8bd0d686eca239965852f0d4ba838fe/docs/figure/DEG_analysis.Rmd/Volcano plots-5.png" target="_blank">fee3875</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Making the median dataframes by time. The files were saved as .csv
for future use.</p>
<pre class="r"><code>all_results &lt;- bind_rows(save_list, .id = &quot;group&quot;)

median_df &lt;- all_results %&gt;% 
  separate(group, into=c(&quot;trt&quot;,&quot;time&quot;),sep = &quot;_&quot;) %&gt;% 
  pivot_wider(., id_cols=c(time,genes), names_from = trt, values_from = logFC) %&gt;% 
  rowwise() %&gt;% 
  mutate(median_ATAC_lfc= median(c_across(DNR:TRZ)))

median_3_lfc &lt;-   median_df %&gt;%
    dplyr::filter(time == &quot;3&quot;) %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(time, genes,median_ATAC_lfc) %&gt;% 
  dplyr::rename(&quot;med_3h_lfc&quot;=median_ATAC_lfc, &quot;peak&quot;=genes)
  

median_24_lfc &lt;- median_df %&gt;%
    dplyr::filter(time == &quot;24&quot;) %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(time, genes,median_ATAC_lfc) %&gt;% 
  dplyr::rename(&quot;med_24h_lfc&quot;=median_ATAC_lfc,, &quot;peak&quot;=genes)
  
write_csv(median_3_lfc, &quot;data/Final_four_data/re_analysis/median_3_lfc_norm.csv&quot;)
write_csv(median_24_lfc, &quot;data/Final_four_data/re_analysis/median_24_lfc_norm.csv&quot;)</code></pre>
<p>Correlation of LFC between treatments</p>
<pre class="r"><code>FCmatrix_ff &lt;- subset(efit2$coefficients)

colnames(FCmatrix_ff) &lt;-
  c(&quot;DNR\n3h&quot;,
    &quot;DOX\n3h&quot;,
    &quot;EPI\n3h&quot;,
    &quot;MTX\n3h&quot;,
    &quot;TRZ\n3h&quot;,
    &quot;DNR\n24h&quot;,
    &quot;DOX\n24h&quot;,
    &quot;EPI\n24h&quot;,
    &quot;MTX\n24h&quot;,
    &quot;TRZ\n24h&quot;
     )


mat_col_ff &lt;-
  data.frame(
    time = c(rep(&quot;3 hours&quot;, 5), rep(&quot;24 hours&quot;, 5)),
    class = (c(
      &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;,&quot;nAC&quot;,  &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;,&quot;nAC&quot; 
    )))
rownames(mat_col_ff) &lt;- colnames(FCmatrix_ff)

mat_colors_ff &lt;-
  list(
    time = c(&quot;pink&quot;, &quot;chocolate4&quot;),
    class = c(&quot;yellow1&quot;, &quot;lightgreen&quot;))

names(mat_colors_ff$time) &lt;- unique(mat_col_ff$time)
names(mat_colors_ff$class) &lt;- unique(mat_col_ff$class)
# names(mat_colors_FC$TOP2i) &lt;- unique(mat_col_FC$TOP2i)
corrFC_ff &lt;- cor(FCmatrix_ff)

htanno_ff &lt;-  HeatmapAnnotation(df = mat_col_ff, col = mat_colors_ff)
Heatmap(corrFC_ff, top_annotation = htanno_ff)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/LFC%20correlation-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LFC-correlation-1">
Past versions of "LFC correlation-1.png"
</button>
</p>
<div id="fig-no-spaces-LFC-correlation-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/5e6e4626ba4b860114d92a9fce8e3a6058534793/docs/figure/DEG_analysis.Rmd/LFC correlation-1.png" target="_blank">5e6e462</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

# all_results &lt;- bind_rows(save_list, .id = &quot;group&quot;)
DNR_3_top3_ff &lt;- row.names(V.DNR_3.top[1:3,])

log_filt_ff &lt;-
  filt_raw_counts_noY %&gt;% 
   cpm(., log=TRUE)%&gt;%
  as.data.frame() 
  
row.names(log_filt_ff) &lt;- row.names(filt_raw_counts_noY)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%203hr%20accesibility-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-accesibility-1">
Past versions of "example 3hr accesibility-1.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-accesibility-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 3hr accesibility-1.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_3_top3_ff &lt;- row.names(V.DOX_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%203hr%20accesibility-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-accesibility-2">
Past versions of "example 3hr accesibility-2.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-accesibility-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 3hr accesibility-2.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_3_top3_ff &lt;- row.names(V.EPI_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%203hr%20accesibility-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-accesibility-3">
Past versions of "example 3hr accesibility-3.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-accesibility-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 3hr accesibility-3.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_3_top3_ff &lt;- row.names(V.MTX_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%203hr%20accesibility-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-accesibility-4">
Past versions of "example 3hr accesibility-4.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-accesibility-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 3hr accesibility-4.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_3_top3_ff &lt;- row.names(V.TRZ_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% TRZ_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour TRZ&quot;)+
 scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%203hr%20accesibility-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-accesibility-5">
Past versions of "example 3hr accesibility-5.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-accesibility-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 3hr accesibility-5.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DNR_24_top3_ff &lt;- row.names(V.DNR_24.top[1:3,])


log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%2024hr%20accesibility-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-accesibility-1">
Past versions of "example 24hr accesibility-1.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-accesibility-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 24hr accesibility-1.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_24_top3_ff &lt;- row.names(V.DOX_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%2024hr%20accesibility-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-accesibility-2">
Past versions of "example 24hr accesibility-2.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-accesibility-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 24hr accesibility-2.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_24_top3_ff &lt;- row.names(V.EPI_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%2024hr%20accesibility-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-accesibility-3">
Past versions of "example 24hr accesibility-3.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-accesibility-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 24hr accesibility-3.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_24_top3_ff &lt;- row.names(V.MTX_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%2024hr%20accesibility-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-accesibility-4">
Past versions of "example 24hr accesibility-4.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-accesibility-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 24hr accesibility-4.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_24_top3_ff &lt;- row.names(V.TRZ_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% TRZ_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/example%2024hr%20accesibility-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-accesibility-5">
Past versions of "example 24hr accesibility-5.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-accesibility-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/4163890db06ef0a2490e5fbb7ab30c8f53c3e201/docs/figure/DEG_analysis.Rmd/example 24hr accesibility-5.png" target="_blank">4163890</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="examining-around-the-adj.-p-value-cutoff"
class="section level4">
<h4>Examining around the adj. p value cutoff</h4>
<pre class="r"><code>DNR_closest &lt;-  V.DNR_3.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)


log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;Bottom DAR in 3 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%203%20hour-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-3-hour-1">
Past versions of "bottom slices 3 hour-1.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-3-hour-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 3 hour-1.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_closest &lt;-  V.DOX_3.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 3 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%203%20hour-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-3-hour-2">
Past versions of "bottom slices 3 hour-2.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-3-hour-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 3 hour-2.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_closest &lt;-  V.EPI_3.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 3 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%203%20hour-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-3-hour-3">
Past versions of "bottom slices 3 hour-3.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-3-hour-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 3 hour-3.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_closest &lt;-  V.MTX_3.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 3 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%203%20hour-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-3-hour-4">
Past versions of "bottom slices 3 hour-4.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-3-hour-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 3 hour-4.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_closest &lt;-  V.TRZ_3.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% TRZ_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 3 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%203%20hour-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-3-hour-5">
Past versions of "bottom slices 3 hour-5.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-3-hour-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 3 hour-5.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DNR_closest &lt;-  V.DNR_24.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)


log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;Bottom DAR in 24 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%2024%20hour-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-24-hour-1">
Past versions of "bottom slices 24 hour-1.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-24-hour-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 24 hour-1.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_closest &lt;-  V.DOX_24.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 24 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%2024%20hour-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-24-hour-2">
Past versions of "bottom slices 24 hour-2.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-24-hour-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 24 hour-2.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_closest &lt;-  V.EPI_24.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 24 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%2024%20hour-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-24-hour-3">
Past versions of "bottom slices 24 hour-3.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-24-hour-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 24 hour-3.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_closest &lt;-  V.MTX_24.top %&gt;%
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  slice_tail(n=5)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_closest$genes) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;bottom 5 DAR in 24 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/DEG_analysis.Rmd/bottom%20slices%2024%20hour-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bottom-slices-24-hour-4">
Past versions of "bottom slices 24 hour-4.png"
</button>
</p>
<div id="fig-no-spaces-bottom-slices-24-hour-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/bottom slices 24 hour-4.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># TRZ_closest &lt;-  V.TRZ_24.top %&gt;%
#   dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
#   slice_tail(n=5)
# 
# log_filt_ff %&gt;% 
#   dplyr::filter(row.names(.) %in% TRZ_closest$genes) %&gt;% 
#   mutate(Peak = row.names(.)) %&gt;% 
#   pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
#   separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
#   mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
#   mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
#   ggplot(., aes (x = time, y=counts))+
#   geom_boxplot(aes(fill=trt))+
#   facet_wrap(Peak~.)+
#   ggtitle(&quot;bottom 5 DAR in 24 hour TRZ&quot;)+
#   scale_fill_manual(values = drug_pal)+
#   theme_bw()</code></pre>
<pre class="r"><code>toptable_results &lt;- readRDS(&quot;data/Final_four_data/re_analysis/Toptable_results.RDS&quot;)
library(openxlsx)
output_dir &lt;- &quot;data/Final_four_data/re_analysis/ATAC_excel_outputs&quot;

# Create directory if it doesn&#39;t exist
if (!dir.exists(output_dir)) {
  dir.create(output_dir, recursive = TRUE)
}

# Export each data frame to a separate .xlsx file
for (name in names(toptable_results)) {
  # Create a new workbook
  wb &lt;- createWorkbook()
  
  # Add a worksheet (you can use the name as the sheet name too)
  addWorksheet(wb, name)
  
  # Write the data frame to the sheet
  writeData(wb, sheet = name, toptable_results[[name]])
  # Full file path using file.path()
  output_file &lt;- file.path(output_dir, paste0(name, &quot;.xlsx&quot;))
  saveWorkbook(wb, file = output_file, overwrite = TRUE)
}</code></pre>
<pre class="r"><code>toptable_results &lt;- readRDS(&quot;data/Final_four_data/re_analysis/Toptable_results.RDS&quot;)

all_results  &lt;- toptable_results %&gt;%
  imap(~ .x %&gt;% tibble::rownames_to_column(var = &quot;rowname&quot;) %&gt;%
         mutate(source = .y)) %&gt;%
  bind_rows()
all_results_list  &lt;- toptable_results %&gt;%
  imap(~ .x %&gt;% tibble::rownames_to_column(var = &quot;rowname&quot;) %&gt;%
         mutate(source = .y)) 

sig_meta_and_loc &lt;- all_results %&gt;% 
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;%  ## filter by pvalue
##Create parsed dataframe from &quot;rowname&quot; column, &quot;genes column will keep id&quot;
   separate(rowname, into = c(&quot;seqnames&quot;, &quot;start&quot;, &quot;end&quot;), sep = &quot;\\.&quot;, convert = TRUE)
###split into lists by DNR_3, etc..
sig_meta_and_loc_split &lt;- split(sig_meta_and_loc, sig_meta_and_loc$source)
### Convert to Granges for downstream
sig_meta_and_loc_split_gr &lt;- lapply(sig_meta_and_loc_split, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %&gt;% select(-seqnames, -start, -end)
  )
})


notsig_meta_and_loc &lt;- all_results %&gt;% 
  dplyr::filter(adj.P.Val&gt;0.05) %&gt;% 
  separate(rowname, into = c(&quot;seqnames&quot;,&quot;start&quot;,&quot;end&quot;), sep = &quot;\\.&quot;, convert=TRUE)
    
notsig_meta_and_loc_split &lt;- split(notsig_meta_and_loc, notsig_meta_and_loc$source)

notsig_meta_and_loc_split_gr &lt;- lapply(notsig_meta_and_loc_split, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %&gt;% select(-seqnames, -start, -end)
  )
})

all_DAR_regions &lt;- all_results %&gt;% 
   separate(rowname, into = c(&quot;seqnames&quot;, &quot;start&quot;, &quot;end&quot;), sep = &quot;\\.&quot;, convert = TRUE)
all_DAR_regions_list &lt;- split(all_DAR_regions, all_DAR_regions$source)

all_DAR_regions_gr &lt;- lapply(all_DAR_regions_list, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %&gt;% dplyr::select(-seqnames, -start, -end)
  )
})</code></pre>
<pre class="r"><code># Folder with input BED files

output_dir &lt;- &quot;data/Final_four_data/re_analysis/motif_beds_centered&quot;

# Create output folder if needed
dir.create(output_dir, showWarnings = FALSE)

# Loop through each BED file
for (name in names(sig_meta_and_loc_split_gr)) {
  gr &lt;- sig_meta_and_loc_split_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered &lt;- resize(gr, width = 200, fix = &quot;center&quot;)
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, &quot;sig_centered.bed&quot;)), format = &quot;BED&quot;)
}


### not significant DAR regions for xstreme

for (name in names(notsig_meta_and_loc_split_gr)) {
  gr &lt;- notsig_meta_and_loc_split_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered &lt;- resize(gr, width = 200, fix = &quot;center&quot;)
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, &quot;notsig_centered.bed&quot;)), format = &quot;BED&quot;)
}</code></pre>
</div>
</div>
<div id="examining-regions-between-sig-dars-and-trt_time"
class="section level3">
<h3>Examining regions between sig DARs and trt_time</h3>
<pre class="r"><code>sig_venn_list &lt;- sapply(sig_meta_and_loc_split, function(x) x$genes)
sig_venn_3hr &lt;- sig_venn_list[c(&quot;DOX_3&quot;,&quot;EPI_3&quot;, &quot;DNR_3&quot;,&quot;MTX_3&quot;)]
sig_venn_24hr &lt;- sig_venn_list[c(&quot;DOX_24&quot;,&quot;EPI_24&quot;, &quot;DNR_24&quot;,&quot;MTX_24&quot;)]

ggVennDiagram::ggVennDiagram(sig_venn_3hr)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/DAR%20venn-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DAR-venn-1">
Past versions of "DAR venn-1.png"
</button>
</p>
<div id="fig-no-spaces-DAR-venn-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/DAR venn-1.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(sig_venn_24hr)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/DAR%20venn-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DAR-venn-2">
Past versions of "DAR venn-2.png"
</button>
</p>
<div id="fig-no-spaces-DAR-venn-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/f0c5b69c8b71feca7f4b13ec3321c0f42f52174a/docs/figure/DEG_analysis.Rmd/DAR venn-2.png" target="_blank">f0c5b69</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>saveRDS(sig_meta_and_loc_split,&quot;Sig_meta&quot;)


sig_3hr_obj &lt;- ggVennDiagram::Venn(sig_venn_list[c(&quot;DOX_3&quot;,&quot;EPI_3&quot;, &quot;DNR_3&quot;,&quot;MTX_3&quot;)])
sig_24hr_obj &lt;- ggVennDiagram::Venn(sig_venn_list[c(&quot;DOX_24&quot;,&quot;EPI_24&quot;, &quot;DNR_24&quot;,&quot;MTX_24&quot;)])
sig_3hr_obj &lt;- ggVennDiagram::process_data(sig_3hr_obj)
sig_24hr_obj &lt;- ggVennDiagram::process_data(sig_24hr_obj)

sig_3hr_regions &lt;- ggVennDiagram::venn_region(sig_3hr_obj)
sig_24hr_regions&lt;- ggVennDiagram::venn_region(sig_24hr_obj)
sig_3hr_shared &lt;- sig_3hr_obj$regionLabel$item[[11]]
sig_24hr_shared &lt;-   sig_24hr_obj$regionLabel$item[[11]]

# saveRDS(sig_3hr_shared,&quot;data/Final_four_data/re_analysis/AC_shared_3hour_DARs.RDS&quot;)
# saveRDS(sig_24hr_shared,&quot;data/Final_four_data/re_analysis/AC_shared_24hour_DARs.RDS&quot;)</code></pre>
<div id="proportion-of-peaks-that-are-dars" class="section level4">
<h4>Proportion of peaks that are DARs</h4>
<pre class="r"><code>three_hour_df &lt;- all_results %&gt;% 
  dplyr::select(source, genes, logFC,adj.P.Val) %&gt;% 
  mutate(sig_val=if_else(adj.P.Val&lt;0.05,&quot;sig&quot;,&quot;not_sig&quot;)) %&gt;% 
  separate(source, into=c(&quot;trt&quot;,&quot;time&quot;),sep=&quot;_&quot;) %&gt;% 
  dplyr::filter(time==&quot;3&quot;) %&gt;% 
  mutate(trt=factor(trt, levels=c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;)))

twentyfour_hour_df &lt;- all_results %&gt;% 
  dplyr::select(source, genes, logFC,adj.P.Val) %&gt;% 
  mutate(sig_val=if_else(adj.P.Val&lt;0.05,&quot;sig&quot;,&quot;not_sig&quot;)) %&gt;% 
  separate(source, into=c(&quot;trt&quot;,&quot;time&quot;),sep=&quot;_&quot;) %&gt;% 
  dplyr::filter(time==&quot;24&quot;) %&gt;% 
  mutate(trt=factor(trt, levels=c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;)))</code></pre>
<pre class="r"><code>three_hour_df %&gt;% 
  mutate(sig_val=factor(sig_val,levels = c(&quot;not_sig&quot;,&quot;sig&quot;))) %&gt;% 
  ggplot(., aes(x=trt,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot;Proportion of significant regions by 3 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/3%20hour%20ggplot-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-3-hour-ggplot-1">
Past versions of "3 hour ggplot-1.png"
</button>
</p>
<div id="fig-no-spaces-3-hour-ggplot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/3 hour ggplot-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/eeaa3e6d4f6995208ca73f3731360b2573d86811/docs/figure/DEG_analysis.Rmd/3 hour ggplot-1.png" target="_blank">eeaa3e6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>twentyfour_hour_df %&gt;% 
  mutate(sig_val=factor(sig_val,levels = c(&quot;not_sig&quot;,&quot;sig&quot;))) %&gt;% 
  ggplot(., aes(x=trt,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot;Proportion of significant regions by 24 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/3%20hour%20ggplot-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-3-hour-ggplot-2">
Past versions of "3 hour ggplot-2.png"
</button>
</p>
<div id="fig-no-spaces-3-hour-ggplot-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/3 hour ggplot-2.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/eeaa3e6d4f6995208ca73f3731360b2573d86811/docs/figure/DEG_analysis.Rmd/3 hour ggplot-2.png" target="_blank">eeaa3e6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_sig &lt;- sig_meta_and_loc_split[c(&quot;DOX_3&quot;, &quot;DOX_24&quot;)]

DOXsig_up &lt;- lapply(DOX_sig, function(x) dplyr::filter(x, logFC &gt; 0))
names(DOXsig_up) &lt;- paste0(names(DOXsig_up), &quot;_up&quot;)

DOXsig_down &lt;- lapply(DOX_sig, function(x) dplyr::filter(x, logFC &lt; 0))
names(DOXsig_down) &lt;- paste0(names(DOXsig_down), &quot;_down&quot;)


DOXsig_up_gr &lt;- lapply(DOXsig_up, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %&gt;% select(-seqnames, -start, -end)
  )
})

DOXsig_down_gr &lt;- lapply(DOXsig_down, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %&gt;% select(-seqnames, -start, -end)
  )
})

output_dir &lt;- &quot;data/Final_four_data/re_analysis/motif_beds_centered&quot;

# Create output folder if needed
dir.create(output_dir, showWarnings = FALSE)

# Loop through each BED file
for (name in names(DOXsig_down_gr)) {
  gr &lt;- DOXsig_down_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered &lt;- resize(gr, width = 200, fix = &quot;center&quot;)
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, &quot;DOXsig_down_centered.bed&quot;)), format = &quot;BED&quot;)
}

# Loop through each BED file
for (name in names(DOXsig_up_gr)) {
  gr &lt;- DOXsig_up_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered &lt;- resize(gr, width = 200, fix = &quot;center&quot;)
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, &quot;DOXsig_up_centered.bed&quot;)), format = &quot;BED&quot;)
}</code></pre>
<pre class="r"><code> filt_DOX24_notup &lt;- all_results %&gt;% 
  dplyr::filter (source==&quot;DOX_24&quot;) %&gt;% 
  dplyr::filter(!genes %in% DOXsig_up$DOX_24_up$genes) %&gt;% 
  separate(rowname, into = c(&quot;seqnames&quot;, &quot;start&quot;, &quot;end&quot;), sep = &quot;\\.&quot;, convert = TRUE) 

filt_DOX24_notdown &lt;- all_results %&gt;% 
  dplyr::filter (source==&quot;DOX_24&quot;) %&gt;% 
  dplyr::filter(!genes %in% DOXsig_down$DOX_24_down$genes) %&gt;% 
  separate(rowname, into = c(&quot;seqnames&quot;, &quot;start&quot;, &quot;end&quot;), sep = &quot;\\.&quot;, convert = TRUE) 

# Now convert to GRanges
filt_DOX24_notup_gr &lt;- GRanges(
  seqnames = filt_DOX24_notup$seqnames,
  ranges = IRanges(start = filt_DOX24_notup$start, end = filt_DOX24_notup$end),
  mcols = filt_DOX24_notup %&gt;% select(-seqnames, -start, -end)
)
filt_DOX24_notdown_gr &lt;- GRanges(
  seqnames = filt_DOX24_notdown$seqnames,
  ranges = IRanges(start = filt_DOX24_notdown$start, end = filt_DOX24_notdown$end),
  mcols = filt_DOX24_notdown %&gt;% select(-seqnames, -start, -end)
)
DOX_not_list_gr &lt;- list(&quot;DOX24_notup&quot;=filt_DOX24_notup_gr,&quot;DOX24_notdown&quot;=filt_DOX24_notdown_gr)
# for (name in names(DOX_not_list)) {
#   gr &lt;- DOX_not_list[[name]]
#   
#   # Recenter each region to 200 bp around its midpoint
#   gr_centered &lt;- resize(gr, width = 200, fix = &quot;center&quot;)
#   
#   # Export to BED (auto converts to 0-based)
#   export(gr_centered, con = file.path(output_dir, paste0(name, &quot;DOX24not_centered.bed&quot;)), format = &quot;BED&quot;)
# }</code></pre>
<pre class="r"><code>txdb &lt;- TxDb.Hsapiens.UCSC.hg38.knownGene
### maybe use annotatePeakInBatch from ChIPpeakAnno
peakAnnoList_DOX_DAR &lt;- lapply(all_DAR_regions_gr, annotatePeak, tssRegion =c(-2000,2000), TxDb=txdb)</code></pre>
<pre><code>&gt;&gt; preparing features information...         2025-06-13 1:14:50 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:14:50 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:14:52 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:14:52 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:10 PM 
&gt;&gt; done...                   2025-06-13 1:15:10 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:10 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:10 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:12 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:12 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:16 PM 
&gt;&gt; done...                   2025-06-13 1:15:16 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:16 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:16 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:18 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:18 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:22 PM 
&gt;&gt; done...                   2025-06-13 1:15:22 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:23 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:23 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:24 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:24 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:28 PM 
&gt;&gt; done...                   2025-06-13 1:15:28 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:29 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:29 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:30 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:30 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:34 PM 
&gt;&gt; done...                   2025-06-13 1:15:34 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:34 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:34 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:36 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:36 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:40 PM 
&gt;&gt; done...                   2025-06-13 1:15:40 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:40 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:40 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:42 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:42 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:47 PM 
&gt;&gt; done...                   2025-06-13 1:15:47 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:47 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:47 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:48 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:48 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:53 PM 
&gt;&gt; done...                   2025-06-13 1:15:53 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:53 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:53 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:15:54 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:15:54 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:15:59 PM 
&gt;&gt; done...                   2025-06-13 1:15:59 PM 
&gt;&gt; preparing features information...         2025-06-13 1:15:59 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:15:59 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:01 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:01 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:07 PM 
&gt;&gt; done...                   2025-06-13 1:16:07 PM </code></pre>
<pre class="r"><code> # saveRDS(peakAnnoList_DOX_DAR, &quot;data/Final_four_data/re_analysis/DOX_DAR_annotated_peaks_chipannno.RDS&quot;)
filt_peakAnnoList_DOX_DAR &lt;- lapply(sig_meta_and_loc_split_gr,annotatePeak, tssRegion =c(-2000,2000), TxDb=txdb)</code></pre>
<pre><code>&gt;&gt; preparing features information...         2025-06-13 1:16:07 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:07 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:08 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:08 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:12 PM 
&gt;&gt; done...                   2025-06-13 1:16:12 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:12 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:12 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:13 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:13 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:17 PM 
&gt;&gt; done...                   2025-06-13 1:16:17 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:17 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:17 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:18 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:18 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:22 PM 
&gt;&gt; done...                   2025-06-13 1:16:22 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:23 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:23 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:23 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:23 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:27 PM 
&gt;&gt; done...                   2025-06-13 1:16:27 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:27 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:27 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:28 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:28 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:34 PM 
&gt;&gt; done...                   2025-06-13 1:16:34 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:34 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:34 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:35 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:35 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:40 PM 
&gt;&gt; done...                   2025-06-13 1:16:40 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:40 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:40 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:41 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:41 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:46 PM 
&gt;&gt; done...                   2025-06-13 1:16:46 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:47 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:47 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:47 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:47 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:52 PM 
&gt;&gt; done...                   2025-06-13 1:16:52 PM 
&gt;&gt; preparing features information...         2025-06-13 1:16:52 PM 
&gt;&gt; identifying nearest features...       2025-06-13 1:16:52 PM 
&gt;&gt; calculating distance from peak to TSS...  2025-06-13 1:16:52 PM 
&gt;&gt; assigning genomic annotation...       2025-06-13 1:16:52 PM 
&gt;&gt; assigning chromosome lengths          2025-06-13 1:16:57 PM 
&gt;&gt; done...                   2025-06-13 1:16:57 PM </code></pre>
<pre class="r"><code># saveRDS(filt_peakAnnoList_DOX_DAR, &quot;data/Final_four_data/re_analysis/filt_DOX_DAR_annotated_peaks_chipannno.RDS&quot;)
plotAnnoBar(peakAnnoList_DOX_DAR)+
  ggtitle (&quot;Genomic Feature Distribution, all DAR no filtering\n should look identical&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Genomic%20Features-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Genomic-Features-1">
Past versions of "Genomic Features-1.png"
</button>
</p>
<div id="fig-no-spaces-Genomic-Features-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/Genomic Features-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotAnnoBar(filt_peakAnnoList_DOX_DAR)+
  ggtitle (&quot;Genomic Feature Distribution, Significant regions \n using adj.P.Val &lt;0.05&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/Genomic%20Features-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Genomic-Features-2">
Past versions of "Genomic Features-2.png"
</button>
</p>
<div id="fig-no-spaces-Genomic-Features-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/Genomic Features-2.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># annotated_peak_TSS_chipanno &lt;-  peakAnnoList_DOX_DAR %&gt;%
#   imap(~ .x %&gt;% tibble::rownames_to_column(var = &quot;rowname&quot;) %&gt;%
#          mutate(source = .y)) </code></pre>
<pre class="r"><code>toplistall_RNA &lt;- readRDS(&quot;data/other_papers/toplistall_RNA.RDS&quot;) %&gt;% 
  mutate(logFC = logFC*(-1))
peakAnnoList_DOX_DAR &lt;- readRDS(&quot;data/Final_four_data/re_analysis/DOX_DAR_annotated_peaks_chipannno.RDS&quot;)
Assigned_genes_toPeak &lt;- peakAnnoList_DOX_DAR$DOX_24 %&gt;% as.data.frame() %&gt;% 
  dplyr::select(mcols.genes,annotation, geneId, distanceToTSS) %&gt;% 
  dplyr::rename(&quot;Peakid&quot;=mcols.genes)


RNA_results &lt;-
toplistall_RNA %&gt;% 
  dplyr::select(time:logFC) %&gt;% 
  tidyr::unite(&quot;sample&quot;,time, id) %&gt;% 
  pivot_wider(., id_cols = c(ENTREZID,SYMBOL),names_from = sample, values_from = logFC) %&gt;% 
  rename_with(~ str_replace(., &quot;hours&quot;, &quot;RNA&quot;))


DOX24_degs &lt;- toplistall_RNA %&gt;%
  dplyr::select(time:logFC,adj.P.Val) %&gt;% 
  dplyr::filter(id==&quot;DOX&quot;) %&gt;% 
  tidyr::unite(&quot;sample&quot;,time, id) %&gt;% 
  dplyr::select(sample:SYMBOL,adj.P.Val) %&gt;% 
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  dplyr::filter(sample==&quot;24_hours_DOX&quot;)

DOX3_degs &lt;- toplistall_RNA %&gt;%
  dplyr::select(time:logFC,adj.P.Val) %&gt;% 
  dplyr::filter(id==&quot;DOX&quot;) %&gt;% 
  tidyr::unite(&quot;sample&quot;,time, id) %&gt;% 
  dplyr::select(sample:SYMBOL,adj.P.Val) %&gt;% 
  dplyr::filter(adj.P.Val&lt;0.05) %&gt;% 
  dplyr::filter(sample==&quot;3_hours_DOX&quot;)

RNA_all_expressed &lt;-toplistall_RNA %&gt;% 
  dplyr::select(time:logFC,adj.P.Val) %&gt;% 
  dplyr::filter(id==&quot;DOX&quot;) %&gt;% 
  dplyr::filter(time==&quot;24_hours&quot;) %&gt;% 
  tidyr::unite(&quot;sample&quot;,time, id) %&gt;% 
  dplyr::select(ENTREZID, SYMBOL)  

Peak_gene_RNA_LFC &lt;- Assigned_genes_toPeak %&gt;% 
  left_join(., RNA_results, by =c(&quot;geneId&quot;=&quot;ENTREZID&quot;))


entrez_ids &lt;- Assigned_genes_toPeak$geneId  


gene_info &lt;- AnnotationDbi::select(
  org.Hs.eg.db,
  keys = entrez_ids,
  columns = c(&quot;SYMBOL&quot;),
  keytype = &quot;ENTREZID&quot;
)
gene_info_collapsed &lt;- gene_info %&gt;%
  group_by(ENTREZID) %&gt;%
  summarise(SYMBOL = paste(unique(SYMBOL), collapse = &quot;,&quot;), .groups = &quot;drop&quot;)</code></pre>
</div>
</div>
<div id="exploring-dars-and-expressed-genes" class="section level3">
<h3>Exploring DARs and expressed genes</h3>
<div id="hour-dox-dar-and-expressed-rna-genes-2kb"
class="section level4">
<h4>3 hour DOX DAR and expressed RNA genes 2kb</h4>
<pre class="r"><code>three_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-2000 &amp; distanceToTSS&lt;2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot;Expressed genes with and without DOX DARs within 2kb  at 3 hours &quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/3hr%20expressed%202kb-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>filtered_df_3hr_exp &lt;-
three_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-2000 &amp; distanceToTSS&lt;2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_3hr_exp &lt;- table(filtered_df_3hr_exp$EXP_RNA, filtered_df_3hr_exp$sig_val)
contingency_table_3hr_exp</code></pre>
<pre><code>         
            sig not_sig
  exp       722   24173
  not_exp   258   11441</code></pre>
<pre class="r"><code>fisher.test(contingency_table_3hr_exp)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_3hr_exp
p-value = 0.0001141
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.145271 1.535497
sample estimates:
odds ratio 
  1.324524 </code></pre>
</div>
<div id="hour-dox-dar-and-expressed-rna-genes-20kb"
class="section level4">
<h4>3 hour DOX DAR and expressed RNA genes 20kb</h4>
<pre class="r"><code>three_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot; DOX DARs with expressed genes at 3 hours within 20kb&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/3hr%20expressed%2020kb-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>filtered_df_3hr_exp20kb &lt;-
three_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_3hr_exp20kb &lt;- table(filtered_df_3hr_exp20kb$EXP_RNA, filtered_df_3hr_exp20kb$sig_val)
contingency_table_3hr_exp20kb</code></pre>
<pre><code>         
            sig not_sig
  exp      1595   59956
  not_exp   742   34095</code></pre>
<pre class="r"><code>fisher.test(contingency_table_3hr_exp20kb)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_3hr_exp20kb
p-value = 7.026e-06
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.118566 1.336834
sample estimates:
odds ratio 
  1.222358 </code></pre>
</div>
<div id="hour-dox-dar-and-expressed-rna-genes-2kb-1"
class="section level4">
<h4>24 hour DOX DAR and expressed RNA genes 2kb</h4>
<pre class="r"><code>twentyfour_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-2000 &amp; distanceToTSS&lt;2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot;Expressed genes with and without DOX DARs within 2kb  at 3 hours &quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/24hr%20expressed%202kb-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>filtered_df_24hr_exp &lt;-
twentyfour_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-2000 &amp; distanceToTSS&lt;2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_24hr_exp &lt;- table(filtered_df_24hr_exp$EXP_RNA, filtered_df_24hr_exp$sig_val)
contingency_table_24hr_exp</code></pre>
<pre><code>         
            sig not_sig
  exp      9482   15413
  not_exp  4301    7398</code></pre>
<pre class="r"><code>fisher.test(contingency_table_24hr_exp)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_24hr_exp
p-value = 0.01514
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.010881 1.107718
sample estimates:
odds ratio 
  1.058195 </code></pre>
</div>
<div id="hour-dox-dar-and-expressed-rna-genes-20kb-1"
class="section level4">
<h4>24 hour DOX DAR and expressed RNA genes 20kb</h4>
<pre class="r"><code>twentyfour_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot; DOX DARs with expressed genes at 3 hours within 20kb&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/24hr%20expressed%2020kb-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>filtered_df_24hr_exp20kb &lt;-
twentyfour_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,&quot;exp&quot;,&quot;not_exp&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_24hr_exp20kb &lt;- table(filtered_df_24hr_exp20kb$EXP_RNA, filtered_df_24hr_exp20kb$sig_val)
contingency_table_24hr_exp20kb</code></pre>
<pre><code>         
            sig not_sig
  exp     25562   35989
  not_exp 13965   20872</code></pre>
<pre class="r"><code>fisher.test(contingency_table_24hr_exp20kb)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_24hr_exp20kb
p-value = 1.21e-05
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.033450 1.090466
sample estimates:
odds ratio 
    1.0616 </code></pre>
</div>
<div id="hour-dar-and-degs" class="section level4">
<h4>3 hour DAR and DEGs</h4>
<p>Looking at DARs within 2kb of TSS</p>
<pre class="r"><code>three_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(DEG=if_else(geneId %in% DOX3_degs$ENTREZID,&quot;DEG&quot;,&quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-2000 &amp; distanceToTSS&lt;2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
  ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
   ggtitle(&quot; DOX DARs and not-DARs within 2kb of TSS of DEGs at 3 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/2kb%203hr%20DEGnDAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-2kb-3hr-DEGnDAR-1">
Past versions of "2kb 3hr DEGnDAR-1.png"
</button>
</p>
<div id="fig-no-spaces-2kb-3hr-DEGnDAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/2kb 3hr DEGnDAR-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>filtered_df_3hr &lt;-
three_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(Assigned_genes_toPeak, by = c(&quot;genes&quot; = &quot;Peakid&quot;)) %&gt;% 
  mutate(DEG = if_else(geneId %in% DOX3_degs$ENTREZID, &quot;DEG&quot;, &quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt; -2000 &amp; distanceToTSS &lt; 2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_3hr &lt;- table(filtered_df_3hr$DEG, filtered_df_3hr$sig_val)
contingency_table_3hr</code></pre>
<pre><code>         
            sig not_sig
  DEG         1      36
  not_DEG   979   35578</code></pre>
<pre class="r"><code>fisher.test(contingency_table_3hr)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_3hr
p-value = 1
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 0.02485041 6.00813519
sample estimates:
odds ratio 
  1.009478 </code></pre>
<p>Looking at DARs within 20kb of TSS</p>
<pre class="r"><code>three_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(DEG=if_else(geneId %in% DOX3_degs$ENTREZID,&quot;DEG&quot;,&quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot; DOX DARs and not-DARs within 20kb of TSS of DEGs at 3 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/20kb%203hr%20DEGnDAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-20kb-3hr-DEGnDAR-1">
Past versions of "20kb 3hr DEGnDAR-1.png"
</button>
</p>
<div id="fig-no-spaces-20kb-3hr-DEGnDAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/20kb 3hr DEGnDAR-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>filtered_df_3hr20k &lt;-
three_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(Assigned_genes_toPeak, by = c(&quot;genes&quot; = &quot;Peakid&quot;)) %&gt;% 
  mutate(DEG = if_else(geneId %in% DOX3_degs$ENTREZID, &quot;DEG&quot;, &quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt; -20000 &amp; distanceToTSS &lt; 20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_3hr20k &lt;- table(filtered_df_3hr20k$DEG, filtered_df_3hr20k$sig_val)
contingency_table_3hr20k</code></pre>
<pre><code>         
            sig not_sig
  DEG         4     115
  not_DEG  2333   93936</code></pre>
<pre class="r"><code>fisher.test(contingency_table_3hr20k)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_3hr20k
p-value = 0.5398
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 0.3749074 3.6886522
sample estimates:
odds ratio 
  1.400459 </code></pre>
<p>Looking at DARs with all distances to TSS</p>
<pre class="r"><code>three_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(DEG=if_else(geneId %in% DOX3_degs$ENTREZID,&quot;DEG&quot;,&quot;not_DEG&quot;)) %&gt;% 
  # dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 

 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
   ggtitle(&quot;DOX DARs and not-DARs within all distance of TSS of DEGs at 3 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/nodist%203hr%20DEGnDAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-nodist-3hr-DEGnDAR-1">
Past versions of "nodist 3hr DEGnDAR-1.png"
</button>
</p>
<div id="fig-no-spaces-nodist-3hr-DEGnDAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/nodist 3hr DEGnDAR-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>filtered_df_3hrnodist &lt;-
three_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(Assigned_genes_toPeak, by = c(&quot;genes&quot; = &quot;Peakid&quot;)) %&gt;% 
  mutate(DEG = if_else(geneId %in% DOX3_degs$ENTREZID, &quot;DEG&quot;, &quot;not_DEG&quot;)) %&gt;% 
  # dplyr::filter(distanceToTSS&gt; -20000 &amp; distanceToTSS &lt; 20000) %&gt;% 
  mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_3hrnodist &lt;- table(filtered_df_3hrnodist$DEG, filtered_df_3hrnodist$sig_val)
contingency_table_3hrnodist</code></pre>
<pre><code>         
             sig not_sig
  DEG          6     153
  not_DEG   3467  151931</code></pre>
<pre class="r"><code>fisher.test(contingency_table_3hrnodist)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_3hrnodist
p-value = 0.1743
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 0.6205303 3.8313098
sample estimates:
odds ratio 
  1.718498 </code></pre>
</div>
<div id="hour-dar-and-degs-1" class="section level4">
<h4>24 hour DAR and DEGs</h4>
<p>Looking at DARs within 2kb of TSS</p>
<pre class="r"><code>twentyfour_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(DEG=if_else(geneId %in% DOX24_degs$ENTREZID,&quot;DEG&quot;,&quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-2000 &amp; distanceToTSS&lt;2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot; DOX DARs and not-DARs within 2kb of TSS of DEGs at 24 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/2kb%2024hr%20DEGnDAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-2kb-24hr-DEGnDAR-1">
Past versions of "2kb 24hr DEGnDAR-1.png"
</button>
</p>
<div id="fig-no-spaces-2kb-24hr-DEGnDAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/2kb 24hr DEGnDAR-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>filtered_df_24hr &lt;-
twentyfour_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(Assigned_genes_toPeak, by = c(&quot;genes&quot; = &quot;Peakid&quot;)) %&gt;% 
  mutate(DEG = if_else(geneId %in% DOX24_degs$ENTREZID, &quot;DEG&quot;, &quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt; -2000 &amp; distanceToTSS &lt; 2000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_24hr &lt;- table(filtered_df_24hr$DEG, filtered_df_24hr$sig_val)
contingency_table_24hr</code></pre>
<pre><code>         
            sig not_sig
  DEG      4810    7211
  not_DEG  8973   15600</code></pre>
<pre class="r"><code>fisher.test(contingency_table_24hr)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_24hr
p-value = 9.982e-11
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.108569 1.213073
sample estimates:
odds ratio 
  1.159665 </code></pre>
<p>Looking at DARs within 20kb of TSS</p>
<pre class="r"><code>twentyfour_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(DEG=if_else(geneId %in% DOX24_degs$ENTREZID,&quot;DEG&quot;,&quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
  mutate(sig_val=factor(sig_val,levels = c(&quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
  ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
   ggtitle(&quot; DOX DARs and not-DARs within 20kb of TSS of DEGs at 24 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/20kb%2024hr%20DEGnDAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-20kb-24hr-DEGnDAR-1">
Past versions of "20kb 24hr DEGnDAR-1.png"
</button>
</p>
<div id="fig-no-spaces-20kb-24hr-DEGnDAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/20kb 24hr DEGnDAR-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>filtered_df_24hr20k &lt;-
twentyfour_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(Assigned_genes_toPeak, by = c(&quot;genes&quot; = &quot;Peakid&quot;)) %&gt;% 
  mutate(DEG = if_else(geneId %in% DOX24_degs$ENTREZID, &quot;DEG&quot;, &quot;not_DEG&quot;)) %&gt;% 
  dplyr::filter(distanceToTSS&gt; -20000 &amp; distanceToTSS &lt; 20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))



contingency_table_24hr20k &lt;- table(filtered_df_24hr20k$DEG, filtered_df_24hr20k$sig_val)
contingency_table_24hr20k</code></pre>
<pre><code>         
            sig not_sig
  DEG     12585   16899
  not_DEG 26942   39962</code></pre>
<pre class="r"><code>fisher.test(contingency_table_24hr20k)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_24hr20k
p-value = 2.314e-12
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.074263 1.135825
sample estimates:
odds ratio 
  1.104609 </code></pre>
<p>Looking at DARs with all distances to TSS</p>
<pre class="r"><code>twentyfour_hour_df %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot;) %&gt;% 
  left_join(., Assigned_genes_toPeak, by=c(&quot;genes&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(DEG=if_else(geneId %in% DOX24_degs$ENTREZID,&quot;DEG&quot;,&quot;not_DEG&quot;)) %&gt;% 
  # dplyr::filter(distanceToTSS&gt;-20000 &amp; distanceToTSS&lt;20000) %&gt;% 
   mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;))) %&gt;% 
 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position=&quot;fill&quot;)+
  theme_bw()+
  ggtitle(&quot; DOX DARs and not-DARs associated with TSS DEGs at 24 hours&quot;)+
  ylab(&quot;proportion&quot;)</code></pre>
<p><img src="figure/DEG_analysis.Rmd/nodist%2024hr%20DEGnDAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-nodist-24hr-DEGnDAR-1">
Past versions of "nodist 24hr DEGnDAR-1.png"
</button>
</p>
<div id="fig-no-spaces-nodist-24hr-DEGnDAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/04d68415322dfcce2a2ff0152f85fa568879f17e/docs/figure/DEG_analysis.Rmd/nodist 24hr DEGnDAR-1.png" target="_blank">04d6841</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-06-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>filtered_df_24hrnodist &lt;-
twentyfour_hour_df %&gt;% 
  dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  left_join(Assigned_genes_toPeak, by = c(&quot;genes&quot; = &quot;Peakid&quot;)) %&gt;% 
  mutate(DEG = if_else(geneId %in% DOX24_degs$ENTREZID, &quot;DEG&quot;, &quot;not_DEG&quot;)) %&gt;% 
  # dplyr::filter(distanceToTSS&gt; -20000 &amp; distanceToTSS &lt; 20000) %&gt;% 
  mutate(sig_val = factor(sig_val, levels = c( &quot;sig&quot;,&quot;not_sig&quot;)))


contingency_table_24hrnodist &lt;- table(filtered_df_24hrnodist$DEG, filtered_df_24hrnodist$sig_val)
contingency_table_24hrnodist</code></pre>
<pre><code>         
            sig not_sig
  DEG     18448   24529
  not_DEG 46372   66208</code></pre>
<pre class="r"><code>fisher.test(contingency_table_24hrnodist)</code></pre>
<pre><code>
    Fisher&#39;s Exact Test for Count Data

data:  contingency_table_24hrnodist
p-value = 5.671e-10
alternative hypothesis: true odds ratio is not equal to 1
95 percent confidence interval:
 1.049852 1.098289
sample estimates:
odds ratio 
  1.073801 </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggVennDiagram_1.5.2                     
 [2] data.table_1.17.0                       
 [3] smplot2_0.2.5                           
 [4] cowplot_1.1.3                           
 [5] ComplexHeatmap_2.22.0                   
 [6] ggrepel_0.9.6                           
 [7] plyranges_1.26.0                        
 [8] ggsignif_0.6.4                          
 [9] eulerr_7.0.2                            
[10] devtools_2.4.5                          
[11] usethis_3.1.0                           
[12] ggpubr_0.6.0                            
[13] BiocParallel_1.40.0                     
[14] scales_1.3.0                            
[15] VennDiagram_1.7.3                       
[16] futile.logger_1.4.3                     
[17] gridExtra_2.3                           
[18] ggfortify_0.4.17                        
[19] edgeR_4.4.2                             
[20] limma_3.62.2                            
[21] rtracklayer_1.66.0                      
[22] org.Hs.eg.db_3.20.0                     
[23] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[24] GenomicFeatures_1.58.0                  
[25] AnnotationDbi_1.68.0                    
[26] Biobase_2.66.0                          
[27] GenomicRanges_1.58.0                    
[28] GenomeInfoDb_1.42.3                     
[29] IRanges_2.40.1                          
[30] S4Vectors_0.44.0                        
[31] BiocGenerics_0.52.0                     
[32] ChIPseeker_1.42.1                       
[33] RColorBrewer_1.1-3                      
[34] broom_1.0.7                             
[35] kableExtra_1.4.0                        
[36] lubridate_1.9.4                         
[37] forcats_1.0.0                           
[38] stringr_1.5.1                           
[39] dplyr_1.1.4                             
[40] purrr_1.0.4                             
[41] readr_2.1.5                             
[42] tidyr_1.3.1                             
[43] tibble_3.2.1                            
[44] ggplot2_3.5.1                           
[45] tidyverse_2.0.0                         
[46] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.5                               
  [2] matrixStats_1.5.0                      
  [3] bitops_1.0-9                           
  [4] enrichplot_1.26.6                      
  [5] httr_1.4.7                             
  [6] doParallel_1.0.17                      
  [7] profvis_0.4.0                          
  [8] tools_4.4.2                            
  [9] backports_1.5.0                        
 [10] R6_2.6.1                               
 [11] lazyeval_0.2.2                         
 [12] GetoptLong_1.0.5                       
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.2                            
 [15] cli_3.6.4                              
 [16] formatR_1.14                           
 [17] Cairo_1.6-2                            
 [18] labeling_0.4.3                         
 [19] sass_0.4.9                             
 [20] Rsamtools_2.22.0                       
 [21] systemfonts_1.2.1                      
 [22] yulab.utils_0.2.0                      
 [23] foreign_0.8-88                         
 [24] DOSE_4.0.0                             
 [25] svglite_2.1.3                          
 [26] R.utils_2.13.0                         
 [27] sessioninfo_1.2.3                      
 [28] plotrix_3.8-4                          
 [29] pwr_1.3-0                              
 [30] rstudioapi_0.17.1                      
 [31] RSQLite_2.3.9                          
 [32] shape_1.4.6.1                          
 [33] generics_0.1.3                         
 [34] gridGraphics_0.5-1                     
 [35] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [36] BiocIO_1.16.0                          
 [37] vroom_1.6.5                            
 [38] gtools_3.9.5                           
 [39] car_3.1-3                              
 [40] GO.db_3.20.0                           
 [41] Matrix_1.7-3                           
 [42] ggbeeswarm_0.7.2                       
 [43] abind_1.4-8                            
 [44] R.methodsS3_1.8.2                      
 [45] lifecycle_1.0.4                        
 [46] whisker_0.4.1                          
 [47] yaml_2.3.10                            
 [48] carData_3.0-5                          
 [49] SummarizedExperiment_1.36.0            
 [50] gplots_3.2.0                           
 [51] qvalue_2.38.0                          
 [52] SparseArray_1.6.2                      
 [53] blob_1.2.4                             
 [54] promises_1.3.2                         
 [55] crayon_1.5.3                           
 [56] miniUI_0.1.1.1                         
 [57] ggtangle_0.0.6                         
 [58] lattice_0.22-6                         
 [59] KEGGREST_1.46.0                        
 [60] magick_2.8.5                           
 [61] pillar_1.10.1                          
 [62] knitr_1.49                             
 [63] fgsea_1.32.2                           
 [64] rjson_0.2.23                           
 [65] boot_1.3-31                            
 [66] codetools_0.2-20                       
 [67] fastmatch_1.1-6                        
 [68] glue_1.8.0                             
 [69] getPass_0.2-4                          
 [70] ggfun_0.1.8                            
 [71] remotes_2.5.0                          
 [72] vctrs_0.6.5                            
 [73] png_0.1-8                              
 [74] treeio_1.30.0                          
 [75] gtable_0.3.6                           
 [76] cachem_1.1.0                           
 [77] xfun_0.51                              
 [78] S4Arrays_1.6.0                         
 [79] mime_0.12                              
 [80] iterators_1.0.14                       
 [81] statmod_1.5.0                          
 [82] ellipsis_0.3.2                         
 [83] nlme_3.1-167                           
 [84] ggtree_3.14.0                          
 [85] bit64_4.6.0-1                          
 [86] rprojroot_2.0.4                        
 [87] bslib_0.9.0                            
 [88] vipor_0.4.7                            
 [89] rpart_4.1.24                           
 [90] KernSmooth_2.23-26                     
 [91] Hmisc_5.2-2                            
 [92] colorspace_2.1-1                       
 [93] DBI_1.2.3                              
 [94] nnet_7.3-20                            
 [95] ggrastr_1.0.2                          
 [96] tidyselect_1.2.1                       
 [97] processx_3.8.6                         
 [98] bit_4.6.0                              
 [99] compiler_4.4.2                         
[100] curl_6.2.1                             
[101] git2r_0.35.0                           
[102] htmlTable_2.4.3                        
[103] xml2_1.3.7                             
[104] DelayedArray_0.32.0                    
[105] checkmate_2.3.2                        
[106] caTools_1.18.3                         
[107] callr_3.7.6                            
[108] digest_0.6.37                          
[109] rmarkdown_2.29                         
[110] XVector_0.46.0                         
[111] base64enc_0.1-3                        
[112] htmltools_0.5.8.1                      
[113] pkgconfig_2.0.3                        
[114] MatrixGenerics_1.18.1                  
[115] fastmap_1.2.0                          
[116] GlobalOptions_0.1.2                    
[117] rlang_1.1.5                            
[118] htmlwidgets_1.6.4                      
[119] UCSC.utils_1.2.0                       
[120] shiny_1.10.0                           
[121] farver_2.1.2                           
[122] jquerylib_0.1.4                        
[123] zoo_1.8-13                             
[124] jsonlite_1.9.1                         
[125] GOSemSim_2.32.0                        
[126] R.oo_1.27.0                            
[127] RCurl_1.98-1.16                        
[128] magrittr_2.0.3                         
[129] Formula_1.2-5                          
[130] GenomeInfoDbData_1.2.13                
[131] ggplotify_0.1.2                        
[132] patchwork_1.3.0                        
[133] munsell_0.5.1                          
[134] Rcpp_1.0.14                            
[135] ape_5.8-1                              
[136] stringi_1.8.4                          
[137] zlibbioc_1.52.0                        
[138] plyr_1.8.9                             
[139] pkgbuild_1.4.6                         
[140] parallel_4.4.2                         
[141] Biostrings_2.74.1                      
[142] splines_4.4.2                          
[143] circlize_0.4.16                        
[144] hms_1.1.3                              
[145] locfit_1.5-9.12                        
[146] ps_1.9.0                               
[147] igraph_2.1.4                           
[148] reshape2_1.4.4                         
[149] pkgload_1.4.0                          
[150] futile.options_1.0.1                   
[151] XML_3.99-0.18                          
[152] evaluate_1.0.3                         
[153] lambda.r_1.2.4                         
[154] tzdb_0.4.0                             
[155] foreach_1.5.2                          
[156] httpuv_1.6.15                          
[157] clue_0.3-66                            
[158] xtable_1.8-4                           
[159] restfulr_0.0.15                        
[160] tidytree_0.4.6                         
[161] rstatix_0.7.2                          
[162] later_1.4.1                            
[163] viridisLite_0.4.2                      
[164] aplot_0.2.5                            
[165] beeswarm_0.4.0                         
[166] memoise_2.0.1                          
[167] GenomicAlignments_1.42.0               
[168] cluster_2.1.8.1                        
[169] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
