<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-05-07" />

<title>Enhancer_enrichment</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enhancer_enrichment</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-05-07</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-05-07
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treed969893e9d577cbba1f4c6a9ab4d3a6ad0bb1fb7targetblankd969893a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/d969893e9d577cbba1f4c6a9ab4d3a6ad0bb1fb7" target="_blank">d969893</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treed969893e9d577cbba1f4c6a9ab4d3a6ad0bb1fb7targetblankd969893a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/d969893e9d577cbba1f4c6a9ab4d3a6ad0bb1fb7" target="_blank">d969893</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  analysis/Diagnosis-tmm.Rmd
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/making_master_peaks_list.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/Jaspar_motif.Rmd
    Modified:   analysis/Jaspar_motif_ff.Rmd
    Modified:   analysis/final_four_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Enhancer_enrichment.Rmd</code>)
and HTML (<code>docs/Enhancer_enrichment.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/2db35c7902051323c92c3137ff9713afad84c268/analysis/Enhancer_enrichment.Rmd" target="_blank">2db35c7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-07
</td>
<td>
updates to analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(epitools)
library(circlize)</code></pre>
<pre class="r"><code>Collapsed_peaks &lt;- read_delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;,
                              delim = &quot;\t&quot;, 
                              escape_double = FALSE, 
                              trim_ws = TRUE)

Motif_list_gr &lt;- readRDS(&quot;data/Final_four_data/re_analysis/Motif_list_granges.RDS&quot;)
##order specific
df_list &lt;- plyr::llply(Motif_list_gr, as.data.frame)
### no change motif_list_gr names so they do not overwrite the dataframes
names(Motif_list_gr) &lt;- paste0(names(Motif_list_gr), &quot;_gr&quot;)
list2env(Motif_list_gr,envir= .GlobalEnv)</code></pre>
<pre><code>&lt;environment: R_GlobalEnv&gt;</code></pre>
<pre class="r"><code>list2env(df_list,envir= .GlobalEnv)</code></pre>
<pre><code>&lt;environment: R_GlobalEnv&gt;</code></pre>
<pre class="r"><code>mrc_lookup &lt;- bind_rows(
  (EAR_open  %&gt;% dplyr::select(Peakid) %&gt;% mutate(mrc = &quot;EAR_open&quot;)),  
  (EAR_close %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;EAR_close&quot;)),
  (ESR_open  %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;ESR_open&quot;)),
  (ESR_close %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;ESR_close&quot;)),
  (ESR_opcl   %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;ESR_opcl&quot;)),
  (ESR_clop   %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;ESR_clop&quot;)),
  (LR_open   %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;LR_open&quot;)),
  (LR_close  %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;LR_close&quot;)),
  (NR        %&gt;%  dplyr::select(Peakid) %&gt;%mutate(mrc = &quot;NR&quot;))
) %&gt;%
  distinct(Peakid, mrc) 

final_peaks &lt;- Collapsed_peaks %&gt;% 
  dplyr::filter(Peakid %in% mcols(all_regions_gr)$Peakid) %&gt;%
  left_join(.,(mrc_lookup %&gt;% 
              dplyr::select(Peakid, mrc)), by = &quot;Peakid&quot;) %&gt;%
  GRanges()</code></pre>
<div id="enhancers-and-response-clusters" class="section level3">
<h3>Enhancers and Response clusters</h3>
<p>First: obtained a list of cis Regulatory Elements from Encode Screen
[(<a href="https://screen.encodeproject.org/#"
class="uri">https://screen.encodeproject.org/#</a>)]</p>
<pre class="r"><code>cREs_HLV_46F &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed&quot;)

# cREs_HLV_53F &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF417JSF_ENCFF651XRK_ENCFF320IPT_ENCFF440RUS.7group.bed&quot;) 
 
NR_cREs &lt;- join_overlap_intersect(NR_gr,cREs_HLV_46F)
LR_open_cREs &lt;- join_overlap_intersect(LR_open_gr,cREs_HLV_46F)
LR_close_cREs &lt;- join_overlap_intersect(LR_close_gr,cREs_HLV_46F)
ESR_open_cREs &lt;- join_overlap_intersect(ESR_open_gr,cREs_HLV_46F)
ESR_close_cREs &lt;- join_overlap_intersect(ESR_close_gr,cREs_HLV_46F)

ESR_opcl_cREs &lt;- join_overlap_intersect(ESR_opcl_gr, cREs_HLV_46F)
ESR_clop_cREs &lt;- join_overlap_intersect(ESR_clop_gr, cREs_HLV_46F)
EAR_open_cREs &lt;- join_overlap_intersect(EAR_open_gr,cREs_HLV_46F)
EAR_close_cREs &lt;- join_overlap_intersect(EAR_close_gr,cREs_HLV_46F)
### These unique peaks are cREs that contain all types of cREs such as
# [1] &quot;Low-DNase&quot;                &quot;DNase-only&quot;               &quot;CTCF-only,CTCF-bound&quot;    
#  [4] &quot;PLS,CTCF-bound&quot;           &quot;PLS&quot;                      &quot;dELS&quot;                    
#  [7] &quot;pELS&quot;                     &quot;DNase-H3K4me3&quot;            &quot;DNase-H3K4me3,CTCF-bound&quot;
# [10] &quot;dELS,CTCF-bound&quot;          &quot;pELS,CTCF-bound&quot; #### NOT the exact ones I am interested in.      
 uni_EAR_open  &lt;-  EAR_open_cREs %&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_EAR_close &lt;- EAR_close_cREs %&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_open &lt;- ESR_open_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_close &lt;- ESR_close_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_opcl &lt;- ESR_opcl_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_clop &lt;- ESR_clop_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_LR_open &lt;- LR_open_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_LR_close &lt;- LR_close_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_NR &lt;- NR_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)


Whole_peaks &lt;- join_overlap_intersect(final_peaks, cREs_HLV_46F)


Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  left_join(.,mrc_lookup, by = c(&quot;Peakid&quot;,&quot;mrc&quot;))%&gt;% 
  group_by(blockCount, mrc) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = blockCount, values_from = n) %&gt;% 
  dplyr::select(mrc, PLS:&#39;pELS,CTCF-bound&#39;) %&gt;% 
  kable(., caption=&quot;Breakdown of peaks overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of peaks overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
PLS
</th>
<th style="text-align:right;">
PLS,CTCF-bound
</th>
<th style="text-align:right;">
dELS
</th>
<th style="text-align:right;">
dELS,CTCF-bound
</th>
<th style="text-align:right;">
pELS
</th>
<th style="text-align:right;">
pELS,CTCF-bound
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
264
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1185
</td>
<td style="text-align:right;">
219
</td>
<td style="text-align:right;">
215
</td>
<td style="text-align:right;">
60
</td>
<td style="text-align:right;">
1205
</td>
<td style="text-align:right;">
193
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
202
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:right;">
1276
</td>
<td style="text-align:right;">
174
</td>
<td style="text-align:right;">
399
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
120
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
197
</td>
<td style="text-align:right;">
39
</td>
<td style="text-align:right;">
255
</td>
<td style="text-align:right;">
47
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1251
</td>
<td style="text-align:right;">
315
</td>
<td style="text-align:right;">
2190
</td>
<td style="text-align:right;">
441
</td>
<td style="text-align:right;">
1443
</td>
<td style="text-align:right;">
312
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
125
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
877
</td>
<td style="text-align:right;">
78
</td>
<td style="text-align:right;">
280
</td>
<td style="text-align:right;">
47
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
10961
</td>
<td style="text-align:right;">
3057
</td>
<td style="text-align:right;">
6511
</td>
<td style="text-align:right;">
1685
</td>
<td style="text-align:right;">
12260
</td>
<td style="text-align:right;">
2852
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
128
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
246
</td>
<td style="text-align:right;">
52
</td>
<td style="text-align:right;">
151
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
NA
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>keep_cRE_names &lt;- c(&quot;CTCF-only,CTCF-bound&quot; ,&quot;PLS,CTCF-bound&quot;,&quot;PLS&quot;,&quot;dELS,CTCF-bound&quot;, &quot;pELS&quot;,&quot;pELS,CTCF-bound&quot;,&quot;dELS&quot;)
is_cRE &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount %in% keep_cRE_names) %&gt;% 
  distinct(Peakid,blockCount) 

is_CTCF &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;CTCF-only,CTCF-bound&quot;) %&gt;% 
  distinct(Peakid,blockCount) 

is_dELS &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;dELS,CTCF-bound&quot;|blockCount == &quot;dELS&quot;) %&gt;% 
  distinct(Peakid,blockCount) 
is_pELS &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;pELS,CTCF-bound&quot;|blockCount == &quot;pELS&quot;) %&gt;% 
  distinct(Peakid,blockCount) 
is_PLS &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;PLS,CTCF-bound&quot;|blockCount == &quot;PLS&quot;) %&gt;% 
  distinct(Peakid,blockCount) 


CRE_summary &lt;-final_peaks %&gt;% 
  as.data.frame() %&gt;% 
   mutate(cRE_status=if_else(Peakid %in% is_cRE$Peakid,&quot;cRE_peak&quot;,&quot;not_cRE_peak&quot;)) %&gt;% 
   mutate(CTCF_status=if_else(Peakid %in% is_CTCF$Peakid,&quot;CTCF_peak&quot;,&quot;not_CTCF_peak&quot;)) %&gt;% 
    mutate(dELS_status=if_else(Peakid %in% is_dELS$Peakid,&quot;dELS_peak&quot;,&quot;not_dELS_peak&quot;)) %&gt;% 
    mutate(pELS_status=if_else(Peakid %in% is_pELS$Peakid,&quot;pELS_peak&quot;,&quot;not_pELS_peak&quot;)) %&gt;% 
    mutate(PLS_status=if_else(Peakid %in% is_PLS$Peakid,&quot;PLS_peak&quot;,&quot;not_PLS_peak&quot;)) %&gt;% 
  dplyr::select(Peakid:PLS_status)</code></pre>
<pre class="r"><code>cRE_mat&lt;- CRE_summary %&gt;%
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(cRE_status, mrc) %&gt;% 
  tally %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;ESR_open&quot;,&quot;LR_open&quot;,&quot;ESR_opcl&quot;, &quot;EAR_close&quot;,&quot;ESR_close&quot;,&quot;LR_close&quot;,&quot;ESR_clop&quot;, &quot;NR&quot;))) %&gt;% 
    pivot_wider(id_cols = mrc, names_from = cRE_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)
cRE_mat</code></pre>
<pre><code>          cRE_peak not_cRE_peak
EAR_close      322         2753
EAR_open      1518         3381
ESR_clop        18          696
ESR_close     1846         6088
ESR_opcl        24          179
ESR_open       489         5788
LR_close      4411        14199
LR_open       1206        24411
NR           22087        63067</code></pre>
<pre class="r"><code>CTCF_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(CTCF_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;ESR_open&quot;,&quot;LR_open&quot;,&quot;ESR_opcl&quot;, &quot;EAR_close&quot;,&quot;ESR_close&quot;,&quot;LR_close&quot;,&quot;ESR_clop&quot;, &quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = CTCF_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)
CTCF_mat</code></pre>
<pre><code>          CTCF_peak not_CTCF_peak
EAR_close        53          3022
EAR_open        185          4714
ESR_close       289          7645
ESR_opcl          4           199
ESR_open         65          6212
LR_close        896         17714
LR_open         131         25486
NR             5280         79874</code></pre>
<pre class="r"><code>dELS_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(dELS_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;ESR_open&quot;,&quot;LR_open&quot;,&quot;ESR_opcl&quot;, &quot;EAR_close&quot;,&quot;ESR_close&quot;,&quot;LR_close&quot;,&quot;ESR_clop&quot;, &quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = dELS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)
dELS_mat</code></pre>
<pre><code>          dELS_peak not_dELS_peak
EAR_close       205          2870
EAR_open        208          4691
ESR_clop         14           700
ESR_close      1113          6821
ESR_opcl         14           189
ESR_open        186          6091
LR_close       1964         16646
LR_open         774         24843
NR             5686         79468</code></pre>
<pre class="r"><code>pELS_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(pELS_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;ESR_open&quot;,&quot;LR_open&quot;,&quot;ESR_opcl&quot;, &quot;EAR_close&quot;,&quot;ESR_close&quot;,&quot;LR_close&quot;,&quot;ESR_clop&quot;, &quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = pELS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)
pELS_mat</code></pre>
<pre><code>          not_pELS_peak pELS_peak
EAR_close          3020        55
EAR_open           4108       791
ESR_clop            710         4
ESR_close          7584       350
ESR_opcl            196         7
ESR_open           6090       187
LR_close          17499      1111
LR_open           25391       226
NR                77156      7998</code></pre>
<pre class="r"><code>PLS_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(PLS_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;ESR_open&quot;,&quot;LR_open&quot;,&quot;ESR_opcl&quot;, &quot;EAR_close&quot;,&quot;ESR_close&quot;,&quot;LR_close&quot;,&quot;ESR_clop&quot;, &quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = PLS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)
PLS_mat</code></pre>
<pre><code>          PLS_peak not_PLS_peak
EAR_close       15         3060
EAR_open       946         3953
ESR_close      194         7740
ESR_opcl         1          202
ESR_open       115         6162
LR_close      1174        17436
LR_open        124        25493
NR            9443        75711</code></pre>
</div>
<div id="odds-ratio-results" class="section level3">
<h3>odds ratio results</h3>
<pre class="r"><code>matrix_list &lt;- list(&quot;All cREs&quot;= cRE_mat,&quot;PLS&quot;=PLS_mat, &quot;dELS&quot;=dELS_mat, &quot;pELS&quot;=pELS_mat,&quot;CTCF&quot;= CTCF_mat)

results_or &lt;- data.frame(Matrix_Name = character(),
                      Row_Compared = character(),
                      Odds_Ratio = numeric(),
                      Lower_CI = numeric(),
                      Upper_CI = numeric(),
                      P_Value = numeric(),
                      stringsAsFactors = FALSE)

# Loop through each matrix in the list
for (matrix_name in names(matrix_list)) {
  current_matrix &lt;- matrix_list[[matrix_name]]
  n_rows &lt;- nrow(current_matrix)
  
  # Loop through each row of the current matrix (except the last row)
  for (i in 1:(n_rows - 1)) {
    # Perform odds ratio test between row i and the last row using epitools
    test_result &lt;- tryCatch({
      contingency_table &lt;- rbind(current_matrix[i, ], current_matrix[n_rows, ])
      
      # Check if any row in the contingency table contains only zeros
      if (any(rowSums(contingency_table) == 0)) {
        stop(&quot;Contingency table contains empty rows.&quot;)
      }
      
      oddsratio_result &lt;- oddsratio(contingency_table)
       # Ensure the oddsratio result has at least 2 rows
      if (nrow(oddsratio_result$measure) &lt; 2) {
        stop(&quot;oddsratio result does not have enough data.&quot;)
      }
      
     list(oddsratio = oddsratio_result, p.value = oddsratio_result$p.value[2,&quot;chi.square&quot;])
      
    }, error = function(e) {
      cat(&quot;Error in odds ratio test for row&quot;, i, &quot;in matrix&quot;, matrix_name, &quot;:&quot;, e$message, &quot;\n&quot;)
      return(NULL)
    })
    
    # Only store the result if test_result is valid (i.e., not NULL)
    if (!is.null(test_result)) {
      or_value &lt;- test_result$oddsratio$measure[2, &quot;estimate&quot;]
      lower_ci &lt;- test_result$oddsratio$measure[2, &quot;lower&quot;]
      upper_ci &lt;- test_result$oddsratio$measure[2, &quot;upper&quot;]
      p_value &lt;- test_result$oddsratio$p.value[2,&quot;chi.square&quot;]
      
      # Check if the values are numeric and valid (not NA)
      if (!is.na(or_value) &amp;&amp; !is.na(lower_ci) &amp;&amp; !is.na(upper_ci) &amp;&amp; !is.na(p_value)) {
        # Store the results in the dataframe
        results_or &lt;- rbind(results_or, data.frame(Matrix_Name = matrix_name,
                                             Row_Compared = rownames(current_matrix)[i],
                                             Odds_Ratio = or_value,
                                             Lower_CI = lower_ci,
                                             Upper_CI = upper_ci,
                                             P_Value = p_value))
      }
    }
  }
}

# Print the resulting dataframe
print(results_or) %&gt;% 
  kable(., caption = &quot;Odd ratio results and significance values of TSS and CpG enrichment compared to No response group&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<pre><code>           Matrix_Name Row_Compared  Odds_Ratio     Lower_CI    Upper_CI
estimate      All cREs    EAR_close  0.33413089  0.296852431  0.37476753
estimate1     All cREs     EAR_open  1.28208081  1.204176799  1.36444549
estimate2     All cREs     ESR_clop  0.07449141  0.044881181  0.11526243
estimate3     All cREs    ESR_close  0.86583822  0.819919213  0.91394794
estimate4     All cREs     ESR_opcl  0.38513592  0.244965612  0.57776218
estimate5     All cREs     ESR_open  0.24129445  0.219499033  0.26468097
estimate6     All cREs     LR_close  0.88706240  0.854672238  0.92051932
estimate7     All cREs      LR_open  0.14109037  0.132835268  0.14971335
estimate8          PLS    EAR_close  0.03975658  0.022797655  0.06349348
estimate9          PLS     EAR_open  1.91893859  1.781028384  2.06546507
estimate10         PLS    ESR_close  0.20112102  0.173599679  0.23158497
estimate11         PLS     ESR_opcl  0.04547902  0.001960244  0.19763155
estimate12         PLS     ESR_open  0.14985132  0.123743139  0.17948544
estimate13         PLS     LR_close  0.53990850  0.506787691  0.57468362
estimate14         PLS      LR_open  0.03905158  0.032501508  0.04641806
estimate15        dELS    EAR_close  0.99900769  0.862333112  1.15089561
estimate16        dELS     EAR_open  0.62013489  0.536764284  0.71242916
estimate17        dELS     ESR_clop  0.28276093  0.158262160  0.46103441
estimate18        dELS    ESR_close  2.28074889  2.128141997  2.44231013
estimate19        dELS     ESR_opcl  1.04649971  0.578818829  1.73536017
estimate20        dELS     ESR_open  0.42714993  0.367018246  0.49390476
estimate21        dELS     LR_close  1.64905922  1.562134953  1.74023431
estimate22        dELS      LR_open  0.43551277  0.403167198  0.46979014
estimate23        pELS    EAR_close  5.67510713  4.389460507  7.50639156
estimate24        pELS     EAR_open  0.53827209  0.497411548  0.58317755
estimate25        pELS     ESR_clop 17.69111986  7.596472261 58.02448266
estimate26        pELS    ESR_close  2.24519918  2.015399481  2.50961461
estimate27        pELS     ESR_opcl  2.83866273  1.442613956  6.71725577
estimate28        pELS     ESR_open  3.37301551  2.920119420  3.92119045
estimate29        pELS     LR_close  1.63251469  1.530689086  1.74282701
estimate30        pELS      LR_open 11.63779496 10.215900039 13.32964519
estimate31        CTCF    EAR_close  0.26612181  0.200005260  0.34577686
estimate32        CTCF     EAR_open  0.59417924  0.509991703  0.68785377
estimate33        CTCF    ESR_close  0.57214560  0.506046669  0.64430623
estimate34        CTCF     ESR_opcl  0.31605227  0.095743813  0.74376750
estimate35        CTCF     ESR_open  0.15868945  0.122843865  0.20107860
estimate36        CTCF     LR_close  0.76528751  0.711262119  0.82250029
estimate37        CTCF      LR_open  0.07785612  0.065111148  0.09222061
                 P_Value
estimate    1.802343e-83
estimate1   5.599162e-15
estimate2   4.381627e-46
estimate3   1.927568e-07
estimate4   4.541572e-06
estimate5  3.683956e-227
estimate6   2.371758e-10
estimate7   0.000000e+00
estimate8   8.886292e-78
estimate9   1.110003e-68
estimate10 4.276917e-129
estimate11  1.529496e-06
estimate12 2.101411e-118
estimate13  1.211921e-84
estimate14  0.000000e+00
estimate15  9.814613e-01
estimate16  2.206703e-11
estimate17  4.621167e-07
estimate18 5.432381e-128
estimate19  9.005404e-01
estimate20  4.973085e-31
estimate21  4.719946e-75
estimate22 3.078418e-106
estimate23  6.585935e-47
estimate24  4.096888e-54
estimate25  6.233974e-16
estimate26  6.780993e-50
estimate27  3.712335e-03
estimate28  4.040218e-66
estimate29  1.646500e-50
estimate30  0.000000e+00
estimate31  1.391090e-24
estimate32  4.816608e-12
estimate33  3.977173e-20
estimate34  1.248910e-02
estimate35  1.403504e-63
estimate36  4.517144e-13
estimate37 2.775286e-300</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Odd ratio results and significance values of TSS and CpG enrichment
compared to No response group
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Matrix_Name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Row_Compared
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Odds_Ratio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Lower_CI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Upper_CI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P_Value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
estimate
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.3341309
</td>
<td style="text-align:right;">
0.2968524
</td>
<td style="text-align:right;">
0.3747675
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate1
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1.2820808
</td>
<td style="text-align:right;">
1.2041768
</td>
<td style="text-align:right;">
1.3644455
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate2
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
0.0744914
</td>
<td style="text-align:right;">
0.0448812
</td>
<td style="text-align:right;">
0.1152624
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate3
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.8658382
</td>
<td style="text-align:right;">
0.8199192
</td>
<td style="text-align:right;">
0.9139479
</td>
<td style="text-align:right;">
0.0000002
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate4
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.3851359
</td>
<td style="text-align:right;">
0.2449656
</td>
<td style="text-align:right;">
0.5777622
</td>
<td style="text-align:right;">
0.0000045
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate5
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.2412944
</td>
<td style="text-align:right;">
0.2194990
</td>
<td style="text-align:right;">
0.2646810
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate6
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.8870624
</td>
<td style="text-align:right;">
0.8546722
</td>
<td style="text-align:right;">
0.9205193
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate7
</td>
<td style="text-align:left;">
All cREs
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.1410904
</td>
<td style="text-align:right;">
0.1328353
</td>
<td style="text-align:right;">
0.1497134
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate8
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.0397566
</td>
<td style="text-align:right;">
0.0227977
</td>
<td style="text-align:right;">
0.0634935
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate9
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1.9189386
</td>
<td style="text-align:right;">
1.7810284
</td>
<td style="text-align:right;">
2.0654651
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate10
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.2011210
</td>
<td style="text-align:right;">
0.1735997
</td>
<td style="text-align:right;">
0.2315850
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate11
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.0454790
</td>
<td style="text-align:right;">
0.0019602
</td>
<td style="text-align:right;">
0.1976316
</td>
<td style="text-align:right;">
0.0000015
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate12
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.1498513
</td>
<td style="text-align:right;">
0.1237431
</td>
<td style="text-align:right;">
0.1794854
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate13
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.5399085
</td>
<td style="text-align:right;">
0.5067877
</td>
<td style="text-align:right;">
0.5746836
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate14
</td>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.0390516
</td>
<td style="text-align:right;">
0.0325015
</td>
<td style="text-align:right;">
0.0464181
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate15
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.9990077
</td>
<td style="text-align:right;">
0.8623331
</td>
<td style="text-align:right;">
1.1508956
</td>
<td style="text-align:right;">
0.9814613
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate16
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.6201349
</td>
<td style="text-align:right;">
0.5367643
</td>
<td style="text-align:right;">
0.7124292
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate17
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
0.2827609
</td>
<td style="text-align:right;">
0.1582622
</td>
<td style="text-align:right;">
0.4610344
</td>
<td style="text-align:right;">
0.0000005
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate18
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
2.2807489
</td>
<td style="text-align:right;">
2.1281420
</td>
<td style="text-align:right;">
2.4423101
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate19
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
1.0464997
</td>
<td style="text-align:right;">
0.5788188
</td>
<td style="text-align:right;">
1.7353602
</td>
<td style="text-align:right;">
0.9005404
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate20
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.4271499
</td>
<td style="text-align:right;">
0.3670182
</td>
<td style="text-align:right;">
0.4939048
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate21
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1.6490592
</td>
<td style="text-align:right;">
1.5621350
</td>
<td style="text-align:right;">
1.7402343
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate22
</td>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.4355128
</td>
<td style="text-align:right;">
0.4031672
</td>
<td style="text-align:right;">
0.4697901
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate23
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
5.6751071
</td>
<td style="text-align:right;">
4.3894605
</td>
<td style="text-align:right;">
7.5063916
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate24
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.5382721
</td>
<td style="text-align:right;">
0.4974115
</td>
<td style="text-align:right;">
0.5831775
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate25
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
17.6911199
</td>
<td style="text-align:right;">
7.5964723
</td>
<td style="text-align:right;">
58.0244827
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate26
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
2.2451992
</td>
<td style="text-align:right;">
2.0153995
</td>
<td style="text-align:right;">
2.5096146
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate27
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
2.8386627
</td>
<td style="text-align:right;">
1.4426140
</td>
<td style="text-align:right;">
6.7172558
</td>
<td style="text-align:right;">
0.0037123
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate28
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
3.3730155
</td>
<td style="text-align:right;">
2.9201194
</td>
<td style="text-align:right;">
3.9211904
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate29
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1.6325147
</td>
<td style="text-align:right;">
1.5306891
</td>
<td style="text-align:right;">
1.7428270
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate30
</td>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
11.6377950
</td>
<td style="text-align:right;">
10.2159000
</td>
<td style="text-align:right;">
13.3296452
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate31
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.2661218
</td>
<td style="text-align:right;">
0.2000053
</td>
<td style="text-align:right;">
0.3457769
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate32
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.5941792
</td>
<td style="text-align:right;">
0.5099917
</td>
<td style="text-align:right;">
0.6878538
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate33
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.5721456
</td>
<td style="text-align:right;">
0.5060467
</td>
<td style="text-align:right;">
0.6443062
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate34
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.3160523
</td>
<td style="text-align:right;">
0.0957438
</td>
<td style="text-align:right;">
0.7437675
</td>
<td style="text-align:right;">
0.0124891
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate35
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.1586895
</td>
<td style="text-align:right;">
0.1228439
</td>
<td style="text-align:right;">
0.2010786
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate36
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.7652875
</td>
<td style="text-align:right;">
0.7112621
</td>
<td style="text-align:right;">
0.8225003
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate37
</td>
<td style="text-align:left;">
CTCF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.0778561
</td>
<td style="text-align:right;">
0.0651111
</td>
<td style="text-align:right;">
0.0922206
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>col_fun_OR = colorRamp2(c(0,1,1.5,5), c(&quot;blueviolet&quot;,&quot;white&quot;,&quot;lightgreen&quot;,&quot;green3&quot; ))
sig_mat_OR &lt;- results_or %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select( Matrix_Name,Row_Compared,P_Value) %&gt;%
  group_by(Row_Compared) %&gt;%
  mutate(rank_val=rank(P_Value, ties.method = &quot;first&quot;)) %&gt;%
  mutate(BH_correction= p.adjust(P_Value,method= &quot;BH&quot;)) %&gt;% 
  pivot_wider(., id_cols = Matrix_Name, names_from = Row_Compared, values_from = BH_correction) %&gt;% 
  dplyr::select(Matrix_Name,EAR_open,ESR_open,LR_open,ESR_opcl,EAR_close,ESR_close,LR_close,ESR_clop) %&gt;%
  column_to_rownames(&quot;Matrix_Name&quot;) %&gt;% 
  as.matrix() 

# saveRDS(results_or,&quot;data/Final_four_data/re_analysis/OR_results_cREs_df_1bp.RDS&quot;)</code></pre>
<pre class="r"><code>results_or %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select( Matrix_Name,Row_Compared,Odds_Ratio) %&gt;% 
  pivot_wider(., id_cols = Matrix_Name, names_from = Row_Compared, values_from = Odds_Ratio) %&gt;% 
  dplyr::select(Matrix_Name,EAR_open,ESR_open,LR_open,ESR_opcl,EAR_close,ESR_close,LR_close,ESR_clop) %&gt;%
  column_to_rownames(&quot;Matrix_Name&quot;) %&gt;% 
  as.matrix() %&gt;% 
  ComplexHeatmap::Heatmap(. ,col = col_fun_OR, 
                          cluster_rows=FALSE, 
                          cluster_columns=FALSE, 
                          column_names_side = &quot;top&quot;, 
                          column_names_rot = 45,
                          # na_col = &quot;black&quot;,
                          cell_fun = function(j, i, x, y, width, height, fill) {if (!is.na(sig_mat_OR[i, j]) &amp;&amp; sig_mat_OR[i, j] &lt; 0.05  &amp;&amp; .[i, j] &gt; 1) {
            grid.text(&quot;*&quot;, x, y, gp = gpar(fontsize = 20))}})</code></pre>
<p><img src="figure/Enhancer_enrichment.Rmd/heatmap%20of%20OR%20results-1.png" width="672" style="display: block; margin: auto;" />
note, this is not corrected for multiple testing</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] circlize_0.4.16                         
 [2] epitools_0.5-10.1                       
 [3] ggrepel_0.9.6                           
 [4] plyranges_1.26.0                        
 [5] ggsignif_0.6.4                          
 [6] genomation_1.38.0                       
 [7] smplot2_0.2.5                           
 [8] eulerr_7.0.2                            
 [9] biomaRt_2.62.1                          
[10] devtools_2.4.5                          
[11] usethis_3.1.0                           
[12] ggpubr_0.6.0                            
[13] BiocParallel_1.40.0                     
[14] scales_1.3.0                            
[15] VennDiagram_1.7.3                       
[16] futile.logger_1.4.3                     
[17] gridExtra_2.3                           
[18] ggfortify_0.4.17                        
[19] edgeR_4.4.2                             
[20] limma_3.62.2                            
[21] rtracklayer_1.66.0                      
[22] org.Hs.eg.db_3.20.0                     
[23] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[24] GenomicFeatures_1.58.0                  
[25] AnnotationDbi_1.68.0                    
[26] Biobase_2.66.0                          
[27] GenomicRanges_1.58.0                    
[28] GenomeInfoDb_1.42.3                     
[29] IRanges_2.40.1                          
[30] S4Vectors_0.44.0                        
[31] BiocGenerics_0.52.0                     
[32] ChIPseeker_1.42.1                       
[33] RColorBrewer_1.1-3                      
[34] broom_1.0.7                             
[35] kableExtra_1.4.0                        
[36] lubridate_1.9.4                         
[37] forcats_1.0.0                           
[38] stringr_1.5.1                           
[39] dplyr_1.1.4                             
[40] purrr_1.0.4                             
[41] readr_2.1.5                             
[42] tidyr_1.3.1                             
[43] tibble_3.2.1                            
[44] ggplot2_3.5.1                           
[45] tidyverse_2.0.0                         
[46] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.5                               
  [2] matrixStats_1.5.0                      
  [3] bitops_1.0-9                           
  [4] enrichplot_1.26.6                      
  [5] doParallel_1.0.17                      
  [6] httr_1.4.7                             
  [7] profvis_0.4.0                          
  [8] tools_4.4.2                            
  [9] backports_1.5.0                        
 [10] R6_2.6.1                               
 [11] lazyeval_0.2.2                         
 [12] GetoptLong_1.0.5                       
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.2                            
 [15] prettyunits_1.2.0                      
 [16] cli_3.6.4                              
 [17] formatR_1.14                           
 [18] sass_0.4.9                             
 [19] Rsamtools_2.22.0                       
 [20] systemfonts_1.2.1                      
 [21] yulab.utils_0.2.0                      
 [22] foreign_0.8-88                         
 [23] DOSE_4.0.0                             
 [24] svglite_2.1.3                          
 [25] R.utils_2.13.0                         
 [26] sessioninfo_1.2.3                      
 [27] plotrix_3.8-4                          
 [28] BSgenome_1.74.0                        
 [29] pwr_1.3-0                              
 [30] impute_1.80.0                          
 [31] rstudioapi_0.17.1                      
 [32] RSQLite_2.3.9                          
 [33] shape_1.4.6.1                          
 [34] generics_0.1.3                         
 [35] gridGraphics_0.5-1                     
 [36] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [37] BiocIO_1.16.0                          
 [38] vroom_1.6.5                            
 [39] gtools_3.9.5                           
 [40] car_3.1-3                              
 [41] GO.db_3.20.0                           
 [42] Matrix_1.7-3                           
 [43] abind_1.4-8                            
 [44] R.methodsS3_1.8.2                      
 [45] lifecycle_1.0.4                        
 [46] whisker_0.4.1                          
 [47] yaml_2.3.10                            
 [48] carData_3.0-5                          
 [49] SummarizedExperiment_1.36.0            
 [50] gplots_3.2.0                           
 [51] qvalue_2.38.0                          
 [52] SparseArray_1.6.2                      
 [53] BiocFileCache_2.14.0                   
 [54] blob_1.2.4                             
 [55] promises_1.3.2                         
 [56] crayon_1.5.3                           
 [57] miniUI_0.1.1.1                         
 [58] ggtangle_0.0.6                         
 [59] lattice_0.22-6                         
 [60] cowplot_1.1.3                          
 [61] KEGGREST_1.46.0                        
 [62] magick_2.8.5                           
 [63] ComplexHeatmap_2.22.0                  
 [64] pillar_1.10.1                          
 [65] knitr_1.49                             
 [66] fgsea_1.32.2                           
 [67] rjson_0.2.23                           
 [68] boot_1.3-31                            
 [69] codetools_0.2-20                       
 [70] fastmatch_1.1-6                        
 [71] glue_1.8.0                             
 [72] getPass_0.2-4                          
 [73] ggfun_0.1.8                            
 [74] data.table_1.17.0                      
 [75] remotes_2.5.0                          
 [76] vctrs_0.6.5                            
 [77] png_0.1-8                              
 [78] treeio_1.30.0                          
 [79] gtable_0.3.6                           
 [80] cachem_1.1.0                           
 [81] xfun_0.51                              
 [82] S4Arrays_1.6.0                         
 [83] mime_0.12                              
 [84] iterators_1.0.14                       
 [85] statmod_1.5.0                          
 [86] ellipsis_0.3.2                         
 [87] nlme_3.1-167                           
 [88] ggtree_3.14.0                          
 [89] bit64_4.6.0-1                          
 [90] filelock_1.0.3                         
 [91] progress_1.2.3                         
 [92] rprojroot_2.0.4                        
 [93] bslib_0.9.0                            
 [94] rpart_4.1.24                           
 [95] KernSmooth_2.23-26                     
 [96] Hmisc_5.2-2                            
 [97] colorspace_2.1-1                       
 [98] DBI_1.2.3                              
 [99] seqPattern_1.38.0                      
[100] nnet_7.3-20                            
[101] tidyselect_1.2.1                       
[102] processx_3.8.6                         
[103] bit_4.6.0                              
[104] compiler_4.4.2                         
[105] curl_6.2.1                             
[106] git2r_0.35.0                           
[107] httr2_1.1.1                            
[108] htmlTable_2.4.3                        
[109] xml2_1.3.7                             
[110] DelayedArray_0.32.0                    
[111] checkmate_2.3.2                        
[112] caTools_1.18.3                         
[113] callr_3.7.6                            
[114] rappdirs_0.3.3                         
[115] digest_0.6.37                          
[116] rmarkdown_2.29                         
[117] XVector_0.46.0                         
[118] base64enc_0.1-3                        
[119] htmltools_0.5.8.1                      
[120] pkgconfig_2.0.3                        
[121] MatrixGenerics_1.18.1                  
[122] dbplyr_2.5.0                           
[123] fastmap_1.2.0                          
[124] GlobalOptions_0.1.2                    
[125] rlang_1.1.5                            
[126] htmlwidgets_1.6.4                      
[127] UCSC.utils_1.2.0                       
[128] shiny_1.10.0                           
[129] farver_2.1.2                           
[130] jquerylib_0.1.4                        
[131] zoo_1.8-13                             
[132] jsonlite_1.9.1                         
[133] GOSemSim_2.32.0                        
[134] R.oo_1.27.0                            
[135] RCurl_1.98-1.16                        
[136] magrittr_2.0.3                         
[137] Formula_1.2-5                          
[138] GenomeInfoDbData_1.2.13                
[139] ggplotify_0.1.2                        
[140] patchwork_1.3.0                        
[141] munsell_0.5.1                          
[142] Rcpp_1.0.14                            
[143] ape_5.8-1                              
[144] stringi_1.8.4                          
[145] zlibbioc_1.52.0                        
[146] plyr_1.8.9                             
[147] pkgbuild_1.4.6                         
[148] parallel_4.4.2                         
[149] Biostrings_2.74.1                      
[150] splines_4.4.2                          
[151] hms_1.1.3                              
[152] locfit_1.5-9.12                        
[153] ps_1.9.0                               
[154] igraph_2.1.4                           
[155] reshape2_1.4.4                         
[156] pkgload_1.4.0                          
[157] futile.options_1.0.1                   
[158] XML_3.99-0.18                          
[159] evaluate_1.0.3                         
[160] lambda.r_1.2.4                         
[161] foreach_1.5.2                          
[162] tzdb_0.4.0                             
[163] httpuv_1.6.15                          
[164] clue_0.3-66                            
[165] gridBase_0.4-7                         
[166] xtable_1.8-4                           
[167] restfulr_0.0.15                        
[168] tidytree_0.4.6                         
[169] rstatix_0.7.2                          
[170] later_1.4.1                            
[171] viridisLite_0.4.2                      
[172] aplot_0.2.5                            
[173] memoise_2.0.1                          
[174] GenomicAlignments_1.42.0               
[175] cluster_2.1.8.1                        
[176] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
