<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-10-07" />

<title>Enhancer_files</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enhancer_files</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-10-07</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-10-07
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree728339c77271f45008abe731a28fe3854d8b924atargetblank728339ca">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/728339c77271f45008abe731a28fe3854d8b924a" target="_blank">728339c</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree728339c77271f45008abe731a28fe3854d8b924atargetblank728339ca"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/728339c77271f45008abe731a28fe3854d8b924a" target="_blank">728339c</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/output_100_MA0835.2.txt
    Ignored:    data/output_101_MA0018.4.txt
    Ignored:    data/output_102_MA1102.2.txt
    Ignored:    data/output_103_MA0162.4.txt
    Ignored:    data/output_104_MA0598.3.txt
    Ignored:    data/output_105_MA0473.3.txt
    Ignored:    data/output_106_MA0640.2.txt
    Ignored:    data/output_107_MA0761.2.txt
    Ignored:    data/output_108_MA0477.2.txt
    Ignored:    data/output_109_MA0062.3.txt
    Ignored:    data/output_10_MA0076.2.txt
    Ignored:    data/output_110_MA0035.4.txt
    Ignored:    data/output_111_MA0482.2.txt
    Ignored:    data/output_112_MA1104.2.txt
    Ignored:    data/output_113_MA0490.2.txt
    Ignored:    data/output_114_MA0491.2.txt
    Ignored:    data/output_115_MA1107.2.txt
    Ignored:    data/output_116_MA0496.3.txt
    Ignored:    data/output_117_MA0620.3.txt
    Ignored:    data/output_118_MA0500.2.txt
    Ignored:    data/output_119_MA0089.2.txt
    Ignored:    data/output_11_MA0258.2.txt
    Ignored:    data/output_120_MA0782.2.txt
    Ignored:    data/output_121_MA0090.3.txt
    Ignored:    data/output_122_MA0809.2.txt
    Ignored:    data/output_123_MA0093.3.txt
    Ignored:    data/output_124_MA0609.2.txt
    Ignored:    data/output_125_MA1684.1.txt
    Ignored:    data/output_126_MA1928.1.txt
    Ignored:    data/output_127_MA1929.1.txt
    Ignored:    data/output_128_MA1930.1.txt
    Ignored:    data/output_129_MA1934.1.txt
    Ignored:    data/output_12_MA0150.2.txt
    Ignored:    data/output_130_MA1940.1.txt
    Ignored:    data/output_131_MA1942.1.txt
    Ignored:    data/output_132_MA1944.1.txt
    Ignored:    data/output_133_MA1951.1.txt
    Ignored:    data/output_134_MA1959.1.txt
    Ignored:    data/output_135_MA1964.1.txt
    Ignored:    data/output_136_MA1970.1.txt
    Ignored:    data/output_137_MA1988.1.txt
    Ignored:    data/output_138_MA1992.1.txt
    Ignored:    data/output_139_MA1993.1.txt
    Ignored:    data/output_13_MA0140.2.txt
    Ignored:    data/output_140_MA1995.1.txt
    Ignored:    data/output_141_MA1997.1.txt
    Ignored:    data/output_142_MA2002.1.txt
    Ignored:    data/output_143_MA0037.4.txt
    Ignored:    data/output_144_MA0156.3.txt
    Ignored:    data/output_145_MA0474.3.txt
    Ignored:    data/output_146_MA0489.2.txt
    Ignored:    data/output_147_MA0493.2.txt
    Ignored:    data/output_148_MA0516.3.txt
    Ignored:    data/output_149_MA0521.2.txt
    Ignored:    data/output_14_MA0591.1.txt
    Ignored:    data/output_150_MA0633.2.txt
    Ignored:    data/output_151_MA0659.3.txt
    Ignored:    data/output_152_MA0685.2.txt
    Ignored:    data/output_153_MA0697.2.txt
    Ignored:    data/output_154_MA0740.2.txt
    Ignored:    data/output_155_MA0742.2.txt
    Ignored:    data/output_156_MA0764.3.txt
    Ignored:    data/output_157_MA0831.3.txt
    Ignored:    data/output_158_MA1472.2.txt
    Ignored:    data/output_159_MA1483.2.txt
    Ignored:    data/output_15_MA0595.1.txt
    Ignored:    data/output_160_MA1511.2.txt
    Ignored:    data/output_161_MA1633.2.txt
    Ignored:    data/output_16_MA0596.1.txt
    Ignored:    data/output_17_MA0599.1.txt
    Ignored:    data/output_18_MA0603.1.txt
    Ignored:    data/output_19_MA0645.1.txt
    Ignored:    data/output_1_MA0029.1.txt
    Ignored:    data/output_1_Mecom.txt
    Ignored:    data/output_20_MA0475.2.txt
    Ignored:    data/output_21_MA0655.1.txt
    Ignored:    data/output_22_MA0656.1.txt
    Ignored:    data/output_23_MA0660.1.txt
    Ignored:    data/output_24_MA0741.1.txt
    Ignored:    data/output_25_MA0747.1.txt
    Ignored:    data/output_26_MA0751.1.txt
    Ignored:    data/output_27_MA0760.1.txt
    Ignored:    data/output_28_MA0098.3.txt
    Ignored:    data/output_29_MA0762.1.txt
    Ignored:    data/output_2_MA0119.1.txt
    Ignored:    data/output_30_MA0498.2.txt
    Ignored:    data/output_31_MA0774.1.txt
    Ignored:    data/output_32_MA0775.1.txt
    Ignored:    data/output_33_MA0783.1.txt
    Ignored:    data/output_34_MA0797.1.txt
    Ignored:    data/output_35_MA0510.2.txt
    Ignored:    data/output_36_MA0808.1.txt
    Ignored:    data/output_37_MA0816.1.txt
    Ignored:    data/output_38_MA0840.1.txt
    Ignored:    data/output_39_MA0117.2.txt
    Ignored:    data/output_3_MA0139.1.txt
    Ignored:    data/output_40_MA0841.1.txt
    Ignored:    data/output_41_MA0036.3.txt
    Ignored:    data/output_42_MA1114.1.txt
    Ignored:    data/output_43_MA1121.1.txt
    Ignored:    data/output_44_MA0750.2.txt
    Ignored:    data/output_45_MA0099.3.txt
    Ignored:    data/output_46_MA1126.1.txt
    Ignored:    data/output_47_MA1128.1.txt
    Ignored:    data/output_48_MA1129.1.txt
    Ignored:    data/output_49_MA1130.1.txt
    Ignored:    data/output_4_MA0145.2.txt
    Ignored:    data/output_50_MA1131.1.txt
    Ignored:    data/output_51_MA1132.1.txt
    Ignored:    data/output_52_MA1133.1.txt
    Ignored:    data/output_53_MA1134.1.txt
    Ignored:    data/output_54_MA1135.1.txt
    Ignored:    data/output_55_MA1136.1.txt
    Ignored:    data/output_56_MA1137.1.txt
    Ignored:    data/output_57_MA1138.1.txt
    Ignored:    data/output_58_MA1139.1.txt
    Ignored:    data/output_59_MA1141.1.txt
    Ignored:    data/output_5_MA0476.1.txt
    Ignored:    data/output_60_MA1142.1.txt
    Ignored:    data/output_61_MA1143.1.txt
    Ignored:    data/output_62_MA1144.1.txt
    Ignored:    data/output_63_MA1145.1.txt
    Ignored:    data/output_64_MA1101.2.txt
    Ignored:    data/output_65_MA1474.1.txt
    Ignored:    data/output_66_MA1475.1.txt
    Ignored:    data/output_67_MA1484.1.txt
    Ignored:    data/output_68_MA1512.1.txt
    Ignored:    data/output_69_MA1516.1.txt
    Ignored:    data/output_6_MA0478.1.txt
    Ignored:    data/output_70_MA1517.1.txt
    Ignored:    data/output_71_MA1520.1.txt
    Ignored:    data/output_72_MA1521.1.txt
    Ignored:    data/output_73_MA1523.1.txt
    Ignored:    data/output_74_MA1527.1.txt
    Ignored:    data/output_75_MA1528.1.txt
    Ignored:    data/output_76_MA1564.1.txt
    Ignored:    data/output_77_MA1571.1.txt
    Ignored:    data/output_78_MA1572.1.txt
    Ignored:    data/output_79_MA1593.1.txt
    Ignored:    data/output_7_MA0488.1.txt
    Ignored:    data/output_80_MA1599.1.txt
    Ignored:    data/output_81_MA1628.1.txt
    Ignored:    data/output_82_MA1629.1.txt
    Ignored:    data/output_83_MA1100.2.txt
    Ignored:    data/output_84_MA0462.2.txt
    Ignored:    data/output_85_MA0766.2.txt
    Ignored:    data/output_86_MA1140.2.txt
    Ignored:    data/output_87_MA0842.2.txt
    Ignored:    data/output_88_MA0746.2.txt
    Ignored:    data/output_89_MA1713.1.txt
    Ignored:    data/output_8_MA0492.1.txt
    Ignored:    data/output_90_MA1721.1.txt
    Ignored:    data/output_91_MA1632.1.txt
    Ignored:    data/output_92_MA1634.1.txt
    Ignored:    data/output_93_MA1635.1.txt
    Ignored:    data/output_94_MA1641.1.txt
    Ignored:    data/output_95_MA1643.1.txt
    Ignored:    data/output_96_MA1645.1.txt
    Ignored:    data/output_97_MA1656.1.txt
    Ignored:    data/output_98_MA1726.1.txt
    Ignored:    data/output_99_MA1728.1.txt
    Ignored:    data/output_9_MA0501.1.txt
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  Correlationplot_scaled.pdf
    Untracked:  DOX_DAR_assess.Rmd
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/ReHeat_analysis.Rmd
    Untracked:  analysis/TE_analysis_old.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/MRC_clusterlog2cpm.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  output_1_Mecom.txt
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/CorMotif_data_n45.Rmd
    Modified:   analysis/Enrichment_motif.Rmd
    Modified:   analysis/Jaspar_motif_ff.Rmd
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Raodah.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd
    Modified:   analysis/TE_analysis.Rmd
    Modified:   analysis/final_four_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Enhancer_files_ff.Rmd</code>) and
HTML (<code>docs/Enhancer_files_ff.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/728339c77271f45008abe731a28fe3854d8b924a/analysis/Enhancer_files_ff.Rmd" target="_blank">728339c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
heat maps baby!
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/Enhancer_files_ff.html" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/Enhancer_files_ff.html" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/2132070dd5041e5130aac29a5e382ac0d8d4dc65/analysis/Enhancer_files_ff.Rmd" target="_blank">2132070</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
heatmmaps baby!
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/6e3a803363c1926c95753af42b3e5863c89d925f/docs/Enhancer_files_ff.html" target="_blank">6e3a803</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/74f647496f9f0f6706aa73245fbfc2e54dc01219/analysis/Enhancer_files_ff.Rmd" target="_blank">74f6474</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-30
</td>
<td>
updates to code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/e174c87731bfd131383d89f57da5b1d9bc0ac027/docs/Enhancer_files_ff.html" target="_blank">e174c87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/c473b77a9ea4230e8628181fed709cbb2acd0306/analysis/Enhancer_files_ff.Rmd" target="_blank">c473b77</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
adding the updated 8 groups
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/a6d16a72a16be6c9173e08d13f2171b5ecff5174/analysis/Enhancer_files_ff.Rmd" target="_blank">a6d16a7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-10
</td>
<td>
updates
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<p>####Loading</p>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)
library(plyranges)
library(genomation)
library(gprofiler2)
library(epitools)</code></pre>
<p>Data loading</p>
<pre class="r"><code># toplistall_RNA &lt;- readRDS(&quot;data/other_papers/toplistall_RNA.RDS&quot;) 
# S13Table &lt;- read.csv( &quot;data/other_papers/S13Table_Matthews2024.csv&quot;,row.names = 1)
# ##14021
# 
# EAR_RNA &lt;- S13Table %&gt;% 
#   dplyr::filter(MOTIF==&quot;EAR&quot;) %&gt;% 
#   dplyr::select(ENTREZID) %&gt;% 
#   mutate(ENTREZID= as.character(ENTREZID))
# ESR_RNA &lt;- S13Table %&gt;% 
#   dplyr::filter(MOTIF==&quot;ESR&quot;)%&gt;% 
#   dplyr::select(ENTREZID)%&gt;% 
#   mutate(ENTREZID= as.character(ENTREZID))
# LR_RNA &lt;- S13Table %&gt;% 
#   dplyr::filter(MOTIF==&quot;LR&quot;)%&gt;% 
#   dplyr::select(ENTREZID)%&gt;% 
#   mutate(ENTREZID=as.character(ENTREZID))
# NR_RNA &lt;- S13Table %&gt;% 
#   dplyr::filter(MOTIF==&quot;NR&quot;)%&gt;% 
#   dplyr::select(ENTREZID)%&gt;% 
#   mutate(ENTREZID= as.character(ENTREZID))

# exp_neargene_table &lt;- read_delim(&quot;data/n45_bedfiles/exp_neargene_table.tsv&quot;, 
#     delim = &quot;\t&quot;, escape_double = FALSE, 
#     trim_ws = TRUE)

TSS_NG_data &lt;- read_delim(&quot;data/Final_four_data/TSS_assigned_NG.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

col_ng_peak &lt;- read.delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;)

peakAnnoList_ff_8motif &lt;- readRDS(&quot;data/Final_four_data/peakAnnoList_ff_8motif.RDS&quot;)
# list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)

# peakAnnoList_3_n45 &lt;- readRDS(&quot;data/peakAnnoList_3_n45.RDS&quot;)
# peakAnnoList_24_n45&lt;- readRDS(&quot;data/peakAnnoList_24_n45.RDS&quot;)

all_peak_gr &lt;- as.GRanges(peakAnnoList_ff_8motif$background)

# Joined_tss_neargene &lt;- all_peak_gr %&gt;%
#   as.data.frame() %&gt;% 
#   dplyr::select(seqnames:id,geneId,distanceToTSS) %&gt;% 
#   dplyr::filter(!grepl(&quot;chrX&quot;,id)&amp; !grepl(&quot;chrY&quot;,id))%&gt;%
#   dplyr::rename(&quot;close_geneId&quot;=&quot;geneId&quot;) %&gt;% 
#   mutate(start=start+1) %&gt;% 
#   left_join(., TSS_NG_data, by=c(&quot;id&quot;=&quot;peakid&quot;, &quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;)) %&gt;%
#   dplyr::select(seqnames.x:close_geneId,distanceToTSS, entrezgene_id:dist_to_NG)
  


EAR_open &lt;- as.data.frame(peakAnnoList_ff_8motif$EAR_open)
EAR_open_gr &lt;-  as.GRanges(peakAnnoList_ff_8motif$EAR_open)
EAR_close &lt;- as.data.frame(peakAnnoList_ff_8motif$EAR_close)
EAR_close_gr &lt;-  as.GRanges(peakAnnoList_ff_8motif$EAR_close)


ESR_open &lt;- as.data.frame(peakAnnoList_ff_8motif$ESR_open)
ESR_open_gr &lt;- as.GRanges(peakAnnoList_ff_8motif$ESR_open)
ESR_close &lt;- as.data.frame(peakAnnoList_ff_8motif$ESR_close)
ESR_close_gr &lt;- as.GRanges(peakAnnoList_ff_8motif$ESR_close)
ESR_OC &lt;- as.data.frame(peakAnnoList_ff_8motif$ESR_OC)
ESR_OC_gr &lt;- as.GRanges(peakAnnoList_ff_8motif$ESR_OC)


LR_open &lt;- as.data.frame(peakAnnoList_ff_8motif$LR_open)
LR_open_gr &lt;-  as.GRanges(peakAnnoList_ff_8motif$LR_open)
LR_close &lt;- as.data.frame(peakAnnoList_ff_8motif$LR_close)
LR_close_gr &lt;-  as.GRanges(peakAnnoList_ff_8motif$LR_close)

NR &lt;- as.data.frame(peakAnnoList_ff_8motif$NR)
NR_gr &lt;-  as.GRanges(peakAnnoList_ff_8motif$NR)

EAR_open_list_all &lt;-  TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% EAR_open$Peakid) %&gt;% 
  mutate(MRC=&quot;EAR_open&quot;) %&gt;% 
  distinct(Peakid,.keep_all = TRUE)
EAR_close_list_all &lt;-  TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% EAR_close$Peakid) %&gt;% 
  mutate(MRC=&quot;EAR_close&quot;) %&gt;% 
  distinct(Peakid,.keep_all = TRUE)

ESR_open_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% ESR_open$Peakid) %&gt;% 
    mutate(MRC=&quot;ESR_open&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)
ESR_close_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% ESR_close$Peakid) %&gt;% 
    mutate(MRC=&quot;ESR_close&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)
ESR_OC_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% ESR_OC$Peakid) %&gt;% 
    mutate(MRC=&quot;ESR_OC&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)

LR_open_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% LR_open$Peakid) %&gt;% 
    mutate(MRC=&quot;LR_open&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)
LR_close_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% LR_close$Peakid) %&gt;% 
    mutate(MRC=&quot;LR_close&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)

NR_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% NR$Peakid) %&gt;% 
    mutate(MRC=&quot;NR&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)


ESR_C &lt;- readRDS(&quot;data/Final_four_data/ESR_C.RDS&quot;)
ESR_D &lt;- readRDS(&quot;data/Final_four_data/ESR_D.RDS&quot;)

ESRC_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in% NR$Peakid) %&gt;% 
    mutate(MRC=&quot;ESR_opcl&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)

ESRD_list_all &lt;- TSS_NG_data %&gt;%
    dplyr::filter(Peakid %in%ESR_D$Peakid) %&gt;% 
    mutate(MRC=&quot;ESR_clop&quot;)%&gt;% 
  distinct(Peakid,.keep_all = TRUE)

# ESR_opcl &lt;- 
  
  
# ESR_clop &lt;-   
  
peak_list_all_mrc &lt;- col_ng_peak %&gt;% 
  mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_OC$Peakid,&quot;ESR_OC&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR$Peakid,&quot;NR&quot;,&quot;not_mrc&quot;))))))))) %&gt;% 
  mutate(mrc9=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_C$Peakid,&quot;ESR_opcl&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR$Peakid,&quot;NR&quot;,if_else(Peakid %in% ESR_D$Peakid,&quot;ESR_clop&quot;,&quot;not_mrc&quot;))))))))) )
  
ESR_C &lt;- readRDS(&quot;data/Final_four_data/ESR_C.RDS&quot;)
ESR_opcl_gr &lt;- ESR_C %&gt;% GRanges()
ESR_D &lt;- readRDS(&quot;data/Final_four_data/ESR_D.RDS&quot;)
ESR_clop_gr &lt;- ESR_D %&gt;% GRanges()
EAR_open_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;EAR_open&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
EAR_close_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;EAR_close&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
  
ESR_open_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;ESR_open&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
ESR_close_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;ESR_close&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
ESR_OC_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;ESR_OC&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
ESR_opcl_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc9 ==&quot;ESR_opcl&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc9) %&gt;% 
  dplyr::rename(&quot;mrc&quot;=mrc9) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)


ESR_clop_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc9 ==&quot;ESR_clop&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc9) %&gt;% 
  dplyr::rename(&quot;mrc&quot;=mrc9) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)


LR_open_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;LR_open&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
LR_close_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;LR_close&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)

NR_NG_2k &lt;-   peak_list_all_mrc %&gt;%
      dplyr::filter(mrc ==&quot;NR&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene, delim= &quot;,&quot;) %&gt;% 
     separate_longer_delim(., cols=,SYMBOL, delim= &quot;,&quot;)  
 
  

median_24_lfc &lt;- read_csv(&quot;data/Final_four_data/median_24_lfc.csv&quot;) 
median_3_lfc &lt;- read_csv(&quot;data/Final_four_data/median_3_lfc.csv&quot;)


# saveRDS(peak_list_all_mrc, &quot;data/Final_four_data/Peak_list_all_mrc_NG.RDS&quot;)

Top2b_peaks_gr &lt;- readBed(&quot;data/n45_bedfiles/TOP2B_CM.bed&quot;)</code></pre>
<div id="enhancers-and-response-clusters" class="section level3">
<h3>Enhancers and Response clusters</h3>
<p>First: obtained a list of cis Regulatory Elements from Encode Screen
[(<a href="https://screen.encodeproject.org/#"
class="uri">https://screen.encodeproject.org/#</a>)]</p>
<pre class="r"><code># BiocManager::install(&quot;plyranges&quot;)
# enhancers_HLV_46F &lt;- genomation::readBed(&quot;C:/Users/renee/Downloads/Supplements folde manuscriptr/ENCODE/heart_left_ventricle_tissue_female_adult_46_years.enhancers.bed&quot;)
cREs_HLV_46F &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed&quot;)

# cREs_HLV_53F &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF417JSF_ENCFF651XRK_ENCFF320IPT_ENCFF440RUS.7group.bed&quot;) 
 
NR_cREs &lt;- join_overlap_intersect(NR_gr,cREs_HLV_46F)
LR_open_cREs &lt;- join_overlap_intersect(LR_open_gr,cREs_HLV_46F)
LR_close_cREs &lt;- join_overlap_intersect(LR_close_gr,cREs_HLV_46F)
ESR_open_cREs &lt;- join_overlap_intersect(ESR_open_gr,cREs_HLV_46F)
ESR_close_cREs &lt;- join_overlap_intersect(ESR_close_gr,cREs_HLV_46F)
ESR_OC_cREs &lt;- join_overlap_intersect(ESR_OC_gr,cREs_HLV_46F)
ESR_opcl_cREs &lt;- join_overlap_intersect(ESR_opcl_gr, cREs_HLV_46F)
ESR_clop_cREs &lt;- join_overlap_intersect(ESR_clop_gr, cREs_HLV_46F)
EAR_open_cREs &lt;- join_overlap_intersect(EAR_open_gr,cREs_HLV_46F)
EAR_close_cREs &lt;- join_overlap_intersect(EAR_close_gr,cREs_HLV_46F)
### These unique peaks are cREs that contain all types of cREs such as
# [1] &quot;Low-DNase&quot;                &quot;DNase-only&quot;               &quot;CTCF-only,CTCF-bound&quot;    
#  [4] &quot;PLS,CTCF-bound&quot;           &quot;PLS&quot;                      &quot;dELS&quot;                    
#  [7] &quot;pELS&quot;                     &quot;DNase-H3K4me3&quot;            &quot;DNase-H3K4me3,CTCF-bound&quot;
# [10] &quot;dELS,CTCF-bound&quot;          &quot;pELS,CTCF-bound&quot; #### NOT the exact ones I am interested in.      
 uni_EAR_open  &lt;-  EAR_open_cREs %&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_EAR_close &lt;- EAR_close_cREs %&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_open &lt;- ESR_open_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_close &lt;- ESR_close_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_OC &lt;- ESR_OC_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_opcl &lt;- ESR_opcl_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_ESR_clop &lt;- ESR_clop_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_LR_open &lt;- LR_open_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_LR_close &lt;- LR_close_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)
 uni_NR &lt;- NR_cREs%&gt;% as.data.frame() %&gt;% distinct(Peakid)


allpeak_gr &lt;- peak_list_all_mrc %&gt;% 
  GRanges(.)
Whole_peaks &lt;- join_overlap_intersect(allpeak_gr, cREs_HLV_46F)


Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  group_by(blockCount, mrc9) %&gt;%  tally %&gt;% 
  pivot_wider(., id_cols = mrc9, names_from = blockCount, values_from = n) %&gt;% 
  dplyr::select(mrc9, PLS:&#39;pELS,CTCF-bound&#39;) %&gt;% 
  kable(., caption=&quot;Breakdown of peaks overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of peaks overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc9
</th>
<th style="text-align:right;">
PLS
</th>
<th style="text-align:right;">
PLS,CTCF-bound
</th>
<th style="text-align:right;">
dELS
</th>
<th style="text-align:right;">
dELS,CTCF-bound
</th>
<th style="text-align:right;">
pELS
</th>
<th style="text-align:right;">
pELS,CTCF-bound
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
442
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
97
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
880
</td>
<td style="text-align:right;">
161
</td>
<td style="text-align:right;">
137
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:right;">
875
</td>
<td style="text-align:right;">
124
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
294
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
1588
</td>
<td style="text-align:right;">
248
</td>
<td style="text-align:right;">
543
</td>
<td style="text-align:right;">
106
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
173
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
171
</td>
<td style="text-align:right;">
35
</td>
<td style="text-align:right;">
302
</td>
<td style="text-align:right;">
66
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
833
</td>
<td style="text-align:right;">
218
</td>
<td style="text-align:right;">
1620
</td>
<td style="text-align:right;">
304
</td>
<td style="text-align:right;">
996
</td>
<td style="text-align:right;">
225
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
219
</td>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
1081
</td>
<td style="text-align:right;">
116
</td>
<td style="text-align:right;">
435
</td>
<td style="text-align:right;">
72
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
11375
</td>
<td style="text-align:right;">
3140
</td>
<td style="text-align:right;">
6464
</td>
<td style="text-align:right;">
1717
</td>
<td style="text-align:right;">
12589
</td>
<td style="text-align:right;">
2925
</td>
</tr>
<tr>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:right;">
176
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
339
</td>
<td style="text-align:right;">
70
</td>
<td style="text-align:right;">
235
</td>
<td style="text-align:right;">
47
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
NA
</td>
</tr>
</tbody>
</table>
</div>
<div id="visualization-of-categories" class="section level3">
<h3>Visualization of categories:</h3>
<p>First, filtering cRE set by type to include ‘CTCF-only,CTCF-bound’,
‘PLS’, ‘PLS,CTCF-bound’,‘dELS’, ‘dELS,CTCF-bound’, ‘pELS’,
‘pELS,CTCF-bound’.</p>
<p>Second, reclassify (new column) to group the CTCF-bound into their
respective groups so only 4 groups, (“CTCF-only”,“PLS”,“dELS”, “pELS”)
are created.</p>
<p>Third, left-join the median LFC for both 3 hour and 24 hour data to
data set and boxplot by group.</p>
<p>Fourth, rbind two more data frames to help with ggplot
visualization</p>
<p>PLS= promoter like sequences pELS= proximal enhancer like sequences
dELS=distal enhancer like sequences</p>
<div id="ear-open" class="section level4">
<h4>EAR open</h4>
<pre class="r"><code>all_EAR_open_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;EAR_open&quot;) %&gt;% 
  mutate(type=&quot;all_EAR_open&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_EAR_open_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;EAR_open&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_EAR_open$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

EAR_open_cRE_list &lt;- EAR_open_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;EAR_open&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


EAR_open_cRE_list %&gt;% 
  rbind(notCRE_EAR_open_list) %&gt;% 
  rbind(all_EAR_open_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste(&quot; 24 hour EAR_open\n n = &quot;,length(unique(EAR_open_cRE_list$Peakid)),&quot; out of &quot;,peakAnnoList_ff_8motif$EAR_open@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,(length(unique(EAR_open_cRE_list$Peakid))/peakAnnoList_ff_8motif$EAR_open@peakNum*100)),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()#+</code></pre>
<p><img src="figure/Enhancer_files_ff.Rmd/EAR%20cRE%20open3%20and%2024%20hours-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-EAR-cRE-open3-and-24-hours-1">
Past versions of "EAR cRE open3 and 24 hours-1.png"
</button>
</p>
<div id="fig-no-spaces-EAR-cRE-open3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/EAR cRE open3 and 24 hours-1.png" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6e3a803363c1926c95753af42b3e5863c89d925f/docs/figure/Enhancer_files_ff.Rmd/EAR cRE open3 and 24 hours-1.png" target="_blank">6e3a803</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/e174c87731bfd131383d89f57da5b1d9bc0ac027/docs/figure/Enhancer_files_ff.Rmd/EAR cRE open3 and 24 hours-1.png" target="_blank">e174c87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

EAR_open_complete_list &lt;- EAR_open_cRE_list %&gt;% 
  rbind(notCRE_EAR_open_list) %&gt;% 
  rbind(all_EAR_open_list)</code></pre>
</div>
<div id="ear-close" class="section level4">
<h4>EAR close</h4>
<pre class="r"><code>all_EAR_close_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;EAR_close&quot;) %&gt;% 
  mutate(type=&quot;all_EAR_close&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_EAR_close_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;EAR_close&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_EAR_close$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

EAR_close_cRE_list &lt;- EAR_close_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;EAR_close&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


EAR_close_cRE_list %&gt;% 
  rbind(notCRE_EAR_close_list) %&gt;% 
  rbind(all_EAR_close_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour EAR_close\n n = &quot;,length(unique(EAR_close_cRE_list$Peakid)),&quot; out of &quot;, peakAnnoList_ff_8motif$EAR_close@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(EAR_close_cRE_list$Peakid))/ peakAnnoList_ff_8motif$EAR_close@peakNum*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/EAR cRE close3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-EAR-cRE-close3-and-24-hours-1">
Past versions of "EAR cRE close3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-EAR-cRE-close3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/EAR cRE close3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/EAR cRE close3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

EAR_close_complete_list &lt;- EAR_close_cRE_list %&gt;% 
  rbind(notCRE_EAR_close_list) %&gt;% 
  rbind(all_EAR_close_list)</code></pre>
</div>
<div id="esr-close" class="section level4">
<h4>ESR close</h4>
<pre class="r"><code>all_ESR_close_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;ESR_close&quot;) %&gt;% 
  mutate(type=&quot;all_ESR_close&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_ESR_close_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;ESR_close&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_ESR_close$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

ESR_close_cRE_list &lt;- ESR_close_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;ESR_close&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


ESR_close_cRE_list %&gt;% 
  rbind(notCRE_ESR_close_list) %&gt;% 
  rbind(all_ESR_close_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour ESR_close\n n = &quot;,length(unique(ESR_close_cRE_list$Peakid)),&quot; out of &quot;,  peakAnnoList_ff_8motif$ESR_close@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(ESR_close_cRE_list$Peakid))/ peakAnnoList_ff_8motif$ESR_close@peakNum*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/ESR cRE close3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ESR-cRE-close3-and-24-hours-1">
Past versions of "ESR cRE close3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-ESR-cRE-close3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/ESR cRE close3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/ESR cRE close3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

ESR_close_complete_list &lt;- ESR_close_cRE_list %&gt;% 
  rbind(notCRE_ESR_close_list) %&gt;% 
  rbind(all_ESR_close_list)</code></pre>
</div>
<div id="esr-open" class="section level4">
<h4>ESR open</h4>
<pre class="r"><code>all_ESR_open_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;ESR_open&quot;) %&gt;% 
  mutate(type=&quot;all_ESR_open&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_ESR_open_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;ESR_open&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_ESR_open$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

ESR_open_cRE_list &lt;- ESR_open_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;ESR_open&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


ESR_open_cRE_list %&gt;% 
  rbind(notCRE_ESR_open_list) %&gt;% 
  rbind(all_ESR_open_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour ESR_open\n n = &quot;,length(unique(ESR_open_cRE_list$Peakid)),&quot; out of &quot;,  peakAnnoList_ff_8motif$ESR_open@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(ESR_open_cRE_list$Peakid))/ peakAnnoList_ff_8motif$ESR_open@peakNum*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/ESR cRE open 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ESR-cRE-open-3-and-24-hours-1">
Past versions of "ESR cRE open 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-ESR-cRE-open-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/ESR cRE open 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/ESR cRE open 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

ESR_open_complete_list &lt;- ESR_open_cRE_list %&gt;% 
  rbind(notCRE_ESR_open_list) %&gt;% 
  rbind(all_ESR_open_list)</code></pre>
</div>
<div id="esr-oc" class="section level4">
<h4>ESR OC</h4>
<pre class="r"><code>all_ESR_OC_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;ESR_OC&quot;) %&gt;% 
  # mutate(mrc=mrc9) %&gt;% 
  mutate(type=&quot;all_ESR_OC&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_ESR_OC_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;ESR_OC&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_ESR_OC$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

ESR_OC_cRE_list &lt;- ESR_OC_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;ESR_OC&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


ESR_OC_cRE_list %&gt;% 
  rbind(notCRE_ESR_OC_list) %&gt;% 
  rbind(all_ESR_OC_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour ESR_OC\n n = &quot;,length(unique(ESR_OC_cRE_list$Peakid)),&quot; out of &quot;,  peakAnnoList_ff_8motif$ESR_OC@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(ESR_OC_cRE_list$Peakid))/ peakAnnoList_ff_8motif$ESR_OC@peakNum*100),&quot;%)&quot;))#+</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/ESR cRE OC 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ESR-cRE-OC-3-and-24-hours-1">
Past versions of "ESR cRE OC 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-ESR-cRE-OC-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/ESR cRE OC 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/ESR cRE OC 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # facet_wrap(~direction)+
  # theme_bw()+
  # xlim(-4,4)

ESR_OC_complete_list &lt;- ESR_OC_cRE_list %&gt;% 
  rbind(notCRE_ESR_OC_list) %&gt;% 
  rbind(all_ESR_OC_list)</code></pre>
</div>
<div id="esr-openclose" class="section level4">
<h4>ESR open/close</h4>
<pre class="r"><code>all_ESR_opcl_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc9==&quot;ESR_opcl&quot;) %&gt;% 
  mutate(mrc=mrc9) %&gt;%
  mutate(type=&quot;all_ESR_opcl&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_ESR_opcl_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc9==&quot;ESR_opcl&quot;) %&gt;% 
  mutate(mrc=mrc9) %&gt;%
  dplyr::filter(!Peakid %in% uni_ESR_opcl$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

ESR_opcl_cRE_list &lt;- ESR_opcl_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;ESR_opcl&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


ESR_opcl_cRE_list %&gt;% 
  rbind(notCRE_ESR_opcl_list) %&gt;% 
  rbind(all_ESR_opcl_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour ESR_opcl\n n = &quot;,length(unique(ESR_opcl_cRE_list$Peakid)),&quot; out of &quot;,  length(ESR_C$Peakid), &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(ESR_opcl_cRE_list$Peakid))/  length(ESR_C$Peakid)*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/ESR cRE opcl 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ESR-cRE-opcl-3-and-24-hours-1">
Past versions of "ESR cRE opcl 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-ESR-cRE-opcl-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/ESR cRE opcl 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/ESR cRE opcl 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

ESR_opcl_complete_list &lt;- ESR_opcl_cRE_list %&gt;% 
  rbind(notCRE_ESR_opcl_list) %&gt;% 
  rbind(all_ESR_opcl_list)</code></pre>
</div>
<div id="esr-closeopen" class="section level4">
<h4>ESR close/open</h4>
<pre class="r"><code>all_ESR_clop_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc9==&quot;ESR_clop&quot;) %&gt;% 
  mutate(mrc=mrc9) %&gt;%
  mutate(type=&quot;all_ESR_clop&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_ESR_clop_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc9==&quot;ESR_clop&quot;) %&gt;% 
  mutate(mrc=mrc9) %&gt;%
  dplyr::filter(!Peakid %in% uni_ESR_clop$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

ESR_clop_cRE_list &lt;- ESR_clop_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;ESR_clop&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


ESR_clop_cRE_list %&gt;% 
  rbind(notCRE_ESR_clop_list) %&gt;% 
  rbind(all_ESR_clop_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour ESR_clop\n n = &quot;,length(unique(ESR_clop_cRE_list$Peakid)),&quot; out of &quot;,  length(ESR_C$Peakid), &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(ESR_clop_cRE_list$Peakid))/  length(ESR_C$Peakid)*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/ESR cRE clop 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ESR-cRE-clop-3-and-24-hours-1">
Past versions of "ESR cRE clop 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-ESR-cRE-clop-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/ESR cRE clop 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/ESR cRE clop 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

ESR_clop_complete_list &lt;- ESR_clop_cRE_list %&gt;% 
  rbind(notCRE_ESR_clop_list) %&gt;% 
  rbind(all_ESR_clop_list)</code></pre>
</div>
<div id="lr-open" class="section level4">
<h4>LR open</h4>
<pre class="r"><code>all_LR_open_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;LR_open&quot;) %&gt;% 
  mutate(type=&quot;all_LR_open&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_LR_open_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;LR_open&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_LR_open$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

LR_open_cRE_list &lt;- LR_open_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;LR_open&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


LR_open_cRE_list %&gt;% 
  rbind(notCRE_LR_open_list) %&gt;% 
  rbind(all_LR_open_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
 ggtitle(paste0(&quot; 24 hour LR_open\n n = &quot;,length(unique(LR_open_cRE_list$Peakid)),&quot; out of &quot;,  peakAnnoList_ff_8motif$LR_open@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(LR_open_cRE_list$Peakid))/ peakAnnoList_ff_8motif$LR_open@peakNum*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/LR cRE open 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LR-cRE-open-3-and-24-hours-1">
Past versions of "LR cRE open 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-LR-cRE-open-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/LR cRE open 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/LR cRE open 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

LR_open_complete_list &lt;- LR_open_cRE_list %&gt;% 
  rbind(notCRE_LR_open_list) %&gt;% 
  rbind(all_LR_open_list)
# LR_open</code></pre>
</div>
<div id="lr-close" class="section level4">
<h4>LR close</h4>
<pre class="r"><code>all_LR_close_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;LR_close&quot;) %&gt;% 
  mutate(type=&quot;all_LR_close&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_LR_close_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;LR_close&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_LR_close$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

LR_close_cRE_list &lt;- LR_close_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;LR_close&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


LR_close_cRE_list %&gt;% 
  rbind(notCRE_LR_close_list) %&gt;% 
  rbind(all_LR_close_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour LR_close\n n = &quot;,length(unique(LR_close_cRE_list$Peakid)),&quot; out of &quot;,  peakAnnoList_ff_8motif$LR_close@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(LR_close_cRE_list$Peakid))/ peakAnnoList_ff_8motif$LR_close@peakNum*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/LR cRE close 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LR-cRE-close-3-and-24-hours-1">
Past versions of "LR cRE close 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-LR-cRE-close-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/LR cRE close 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/LR cRE close 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)

LR_close_complete_list &lt;- LR_close_cRE_list %&gt;% 
  rbind(notCRE_LR_close_list) %&gt;% 
  rbind(all_LR_close_list)</code></pre>
</div>
<div id="no-response" class="section level4">
<h4>No response</h4>
<pre class="r"><code>all_NR_list &lt;- 
  peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;NR&quot;) %&gt;% 
  mutate(type=&quot;all_NR&quot;) %&gt;% 
   dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
    pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 
  
notCRE_NR_list &lt;- peak_list_all_mrc %&gt;% 
  dplyr::filter(mrc==&quot;NR&quot;) %&gt;% 
  dplyr::filter(!Peakid %in% uni_NR$Peakid) %&gt;% 
  mutate(type=&quot;not_cRE_peaks&quot;) %&gt;% 
  dplyr::select(Peakid,mrc,type) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 

NR_cRE_list &lt;- NR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|
                  blockCount ==&quot;PLS&quot;|
                  blockCount ==&quot;PLS,CTCF-bound&quot;|
                  blockCount ==&quot;dELS&quot;|
                  blockCount ==&quot;dELS,CTCF-bound&quot;|
                  blockCount ==&quot;pELS&quot;|
                  blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;, if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,                       if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;%
  dplyr::select(Peakid,type) %&gt;% 
    mutate(mrc=&quot;NR&quot;) %&gt;% 
  left_join(.,(median_24_lfc %&gt;% ungroup%&gt;% 
                 dplyr::select(peak,med_24h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,(median_3_lfc %&gt;%ungroup %&gt;% 
                 dplyr::select(peak,med_3h_lfc)),by = c(&quot;Peakid&quot;=&quot;peak&quot;)) %&gt;% 
  pivot_longer(cols = c(&quot;med_24h_lfc&quot;,&quot;med_3h_lfc&quot;), names_to = &quot;time&quot;, values_to = &quot;lfc&quot;) 


NR_cRE_list %&gt;% 
  rbind(notCRE_NR_list) %&gt;% 
  rbind(all_NR_list) %&gt;% 
  ggplot(., aes(y=type, x=lfc,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot; 24 hour NR\n n = &quot;,length(unique(NR_cRE_list$Peakid)),&quot; out of &quot;,  peakAnnoList_ff_8motif$NR@peakNum, &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(NR_cRE_list$Peakid))/ peakAnnoList_ff_8motif$NR@peakNum*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/NRcRE 3 and 24 hours-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-NRcRE-3-and-24-hours-1">
Past versions of "NRcRE 3 and 24 hours-1.pdf"
</button>
</p>
<div id="fig-no-spaces-NRcRE-3-and-24-hours-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/NRcRE 3 and 24 hours-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/NRcRE 3 and 24 hours-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # xlim(-4,4)


NR_complete_list &lt;- NR_cRE_list %&gt;% 
  rbind(notCRE_NR_list) %&gt;% 
  rbind(all_NR_list)</code></pre>
<div
id="go-analysis-of-neargenes-from-peaks-that-are-within-2kb-of-neargene-tss."
class="section level5">
<h5>GO analysis of neargenes from Peaks that are within 2kb of Neargene
TSS.</h5>
<pre class="r"><code>background_NGs &lt;- col_ng_peak %&gt;% 
  dplyr::select (Peakid,NCBI_gene:dist_to_NG) %&gt;% 
  # dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;% 
  distinct(SYMBOL)
EAR_openNG &lt;- EAR_open_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
EAR_closeNG &lt;- EAR_close_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
ESR_openNG &lt;- ESR_open_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
ESR_closeNG &lt;- ESR_close_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
ESR_OCNG &lt;- ESR_OC_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
LR_openNG &lt;- LR_open_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
LR_closeNG &lt;- LR_close_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)
NR_NG &lt;- NR_list_all %&gt;%
  dplyr::select (Peakid,SYMBOL,dist_to_NG) %&gt;% 
  dplyr::filter (dist_to_NG &gt;-2000&amp;dist_to_NG &lt;2000) %&gt;%
  distinct(SYMBOL)</code></pre>
<pre class="r"><code>scale_fill_mrc &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c(&quot;#F8766D&quot;,&quot;#f6483c&quot;,&quot;#7CAE00&quot;,&quot;#587b00&quot;,&quot;#6a9500&quot;,&quot;cornflowerblue&quot;,&quot;grey60&quot;,  &quot;#00BFC4&quot;,&quot;#008d91&quot;, &quot;#C77CFF&quot;), c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;,&quot;ESR_close&quot;, &quot;ESR_OC&quot;,&quot;ESR_opcl&quot;,&quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;)), 
    ...
  )
}</code></pre>
</div>
</div>
</div>
<div id="summarizing-plots" class="section level3">
<h3>summarizing plots:</h3>
<pre class="r"><code>ggplot_combo_df &lt;-
  rbind(EAR_open_complete_list,
                         EAR_close_complete_list,
                         ESR_open_complete_list,
                          ESR_close_complete_list,
                          ESR_OC_complete_list,
                          LR_open_complete_list,
                          LR_close_complete_list,
                          NR_complete_list) 
ggplot_combo_df %&gt;% 
  mutate(time=factor(time, levels= c(&quot;med_3h_lfc&quot;, &quot;med_24h_lfc&quot;), labels=c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;%
     mutate(type= if_else(type==&quot;Promoter-like&quot;,type,
                          if_else(type ==&quot;Proximal enhancer-like&quot;,type,if_else(type==&quot;Distal enhancer-like&quot;, type,if_else(type==&quot;CTCF-only&quot;,type,if_else(type==&quot;not_cRE_peaks&quot;, type, &quot;all&quot;)))))) %&gt;%
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;,&quot;not_cRE_peaks&quot;,&quot;all&quot;))) %&gt;% 
  
  ggplot(., aes(y=type, x=lfc, fill=mrc))+
    geom_boxplot()+
  # geom_pointrange(aes(xmin=min_val, xmax=max_val),position=position_dodge(width =.7),  width=.3,size=0.5,shape=15, fatten=8)+
  geom_vline(xintercept = 0, linetype=2)+
  facet_wrap(~time) + 
  theme_classic()+
  scale_fill_mrc()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/comboplots-1.pdf" width="1152" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-comboplots-1">
Past versions of comboplots-1.pdf
</button>
</p>
<div id="fig-comboplots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/comboplots-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/comboplots-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot_combo_df_9 &lt;-
  rbind(EAR_open_complete_list,
                         EAR_close_complete_list,
                         ESR_open_complete_list,
                          ESR_close_complete_list,
                          ESR_opcl_complete_list,
                          ESR_clop_complete_list,
                          LR_open_complete_list,
                          LR_close_complete_list,
                          NR_complete_list) 
ggplot_combo_df_9 %&gt;% 
  mutate(time=factor(time, levels= c(&quot;med_3h_lfc&quot;, &quot;med_24h_lfc&quot;), labels=c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;%
     mutate(type= if_else(type==&quot;Promoter-like&quot;,type,
                          if_else(type ==&quot;Proximal enhancer-like&quot;,type,if_else(type==&quot;Distal enhancer-like&quot;, type,if_else(type==&quot;CTCF-only&quot;,type,if_else(type==&quot;not_cRE_peaks&quot;, type, &quot;all&quot;)))))) %&gt;%
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;,&quot;not_cRE_peaks&quot;,&quot;all&quot;))) %&gt;% 
  
  ggplot(., aes(y=type, x=lfc, fill=mrc))+
    geom_boxplot()+
  # geom_pointrange(aes(xmin=min_val, xmax=max_val),position=position_dodge(width =.7),  width=.3,size=0.5,shape=15, fatten=8)+
  geom_vline(xintercept = 0, linetype=2)+
  facet_wrap(~time) + 
  theme_classic()+
  scale_fill_mrc()</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/comboplots2-1.pdf" width="1152" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-comboplots2-1">
Past versions of comboplots2-1.pdf
</button>
</p>
<div id="fig-comboplots2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/comboplots2-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/comboplots2-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>keep_cRE_names &lt;- c(&quot;CTCF-only,CTCF-bound&quot; ,&quot;PLS,CTCF-bound&quot;,&quot;PLS&quot;,&quot;dELS,CTCF-bound&quot;, &quot;pELS&quot;,&quot;pELS,CTCF-bound&quot;,&quot;dELS&quot;)
is_cRE &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount %in% keep_cRE_names) %&gt;% 
  distinct(Peakid,blockCount) 

is_CTCF &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;CTCF-only,CTCF-bound&quot;) %&gt;% 
  distinct(Peakid,blockCount) 

is_dELS &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;dELS,CTCF-bound&quot;|blockCount == &quot;dELS&quot;) %&gt;% 
  distinct(Peakid,blockCount) 
is_pELS &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;pELS,CTCF-bound&quot;|blockCount == &quot;pELS&quot;) %&gt;% 
  distinct(Peakid,blockCount) 
is_PLS &lt;- Whole_peaks %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(blockCount == &quot;PLS,CTCF-bound&quot;|blockCount == &quot;PLS&quot;) %&gt;% 
  distinct(Peakid,blockCount) 
  
 CRE_summary &lt;- col_ng_peak %&gt;% 
   mutate(cRE_status=if_else(Peakid %in% is_cRE$Peakid,&quot;cRE_peak&quot;,&quot;not_cRE_peak&quot;)) %&gt;% 
   mutate(CTCF_status=if_else(Peakid %in% is_CTCF$Peakid,&quot;CTCF_peak&quot;,&quot;not_CTCF_peak&quot;)) %&gt;% 
    mutate(dELS_status=if_else(Peakid %in% is_dELS$Peakid,&quot;dELS_peak&quot;,&quot;not_dELS_peak&quot;)) %&gt;% 
    mutate(pELS_status=if_else(Peakid %in% is_pELS$Peakid,&quot;pELS_peak&quot;,&quot;not_pELS_peak&quot;)) %&gt;% 
    mutate(PLS_status=if_else(Peakid %in% is_PLS$Peakid,&quot;PLS_peak&quot;,&quot;not_PLS_peak&quot;)) %&gt;% 
   mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_C$Peakid,&quot;ESR_opcl&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR$Peakid,&quot;NR&quot;,if_else(Peakid %in% ESR_D$Peakid,&quot;ESR_clop&quot;,&quot;not_mrc&quot;))))))))) )


cRE_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(cRE_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = cRE_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

CTCF_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(CTCF_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = CTCF_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

dELS_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(dELS_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = dELS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

pELS_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(pELS_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = pELS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

PLS_mat&lt;- CRE_summary %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(PLS_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = PLS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)</code></pre>
</div>
<div id="chi-square-results" class="section level3">
<h3>chi square results</h3>
<pre class="r"><code>matrix_list_cre &lt;- list(&quot;PLS&quot;=PLS_mat, &quot;dELS&quot;=dELS_mat, &quot;pELS&quot;=pELS_mat,&quot;CTCF&quot;= CTCF_mat,&quot;All cREs&quot;= cRE_mat)
results_cre &lt;- data.frame(Matrix_Name = character(),
                      Row_Compared = character(),
                      Chi_Square_Statistic = numeric(),
                      P_Value = numeric(),
                      stringsAsFactors = FALSE)
names_list_cre &lt;- names(matrix_list_cre)
# Loop through each matrix in the list
for (matrix_name in names_list_cre) {
  current_matrix &lt;- matrix_list_cre[[matrix_name]]
  n_rows &lt;- nrow(current_matrix)
  
  # Print matrix details for debugging
  cat(&quot;Processing Matrix:&quot;, matrix_name, &quot;\n&quot;)
  print(current_matrix)
  
  # Loop through each row of the current matrix (except the last row)
  for (i in 1:(n_rows - 1)) {
    # Print row details for debugging
    cat(&quot;Comparing row&quot;, i, &quot;with last row:\n&quot;)
    print(current_matrix[i, ])
    print(current_matrix[n_rows, ])
    
    # Perform chi-square test between row i and the last row
    test_result &lt;- tryCatch({
      chisq.test(rbind(current_matrix[i, ], current_matrix[n_rows, ]))
    }, error = function(e) {
      cat(&quot;Error in chi-square test:&quot;, e$message, &quot;\n&quot;)
      return(NULL)
    })
    
    # Only store the result if test_result is valid (i.e., not NULL)
    if (!is.null(test_result)) {
      results_cre &lt;- rbind(results_cre, data.frame(Matrix_Name = matrix_name,
                                           Row_Compared = rownames(current_matrix)[i],
                                           Chi_Square_Statistic = test_result$statistic,
                                           P_Value = test_result$p.value))
    }
  }
}</code></pre>
<pre><code>Processing Matrix: PLS 
          PLS_peak not_PLS_peak
EAR_close       48         4248
EAR_open       697         2461
ESR_close      288         9061
ESR_opcl         1           98
ESR_open       158         5114
LR_close       789        13283
LR_open        219        28318
NR            9778        76553
Comparing row 1 with last row:
    PLS_peak not_PLS_peak 
          48         4248 
    PLS_peak not_PLS_peak 
        9778        76553 
Comparing row 2 with last row:
    PLS_peak not_PLS_peak 
         697         2461 
    PLS_peak not_PLS_peak 
        9778        76553 
Comparing row 3 with last row:
    PLS_peak not_PLS_peak 
         288         9061 
    PLS_peak not_PLS_peak 
        9778        76553 
Comparing row 4 with last row:
    PLS_peak not_PLS_peak 
           1           98 
    PLS_peak not_PLS_peak 
        9778        76553 
Comparing row 5 with last row:
    PLS_peak not_PLS_peak 
         158         5114 
    PLS_peak not_PLS_peak 
        9778        76553 
Comparing row 6 with last row:
    PLS_peak not_PLS_peak 
         789        13283 
    PLS_peak not_PLS_peak 
        9778        76553 
Comparing row 7 with last row:
    PLS_peak not_PLS_peak 
         219        28318 
    PLS_peak not_PLS_peak 
        9778        76553 
Processing Matrix: dELS 
          dELS_peak not_dELS_peak
EAR_close       346          3950
EAR_open        125          3033
ESR_clop         31          1030
ESR_close      1354          7995
ESR_opcl          8            91
ESR_open        163          5109
LR_close       1478         12594
LR_open         955         27582
NR             5680         80651
Comparing row 1 with last row:
    dELS_peak not_dELS_peak 
          346          3950 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 2 with last row:
    dELS_peak not_dELS_peak 
          125          3033 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 3 with last row:
    dELS_peak not_dELS_peak 
           31          1030 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 4 with last row:
    dELS_peak not_dELS_peak 
         1354          7995 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 5 with last row:
    dELS_peak not_dELS_peak 
            8            91 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 6 with last row:
    dELS_peak not_dELS_peak 
          163          5109 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 7 with last row:
    dELS_peak not_dELS_peak 
         1478         12594 
    dELS_peak not_dELS_peak 
         5680         80651 
Comparing row 8 with last row:
    dELS_peak not_dELS_peak 
          955         27582 
    dELS_peak not_dELS_peak 
         5680         80651 
Processing Matrix: pELS 
          not_pELS_peak pELS_peak
EAR_close          4214        82
EAR_open           2590       568
ESR_clop           1056         5
ESR_close          8887       462
ESR_opcl             95         4
ESR_open           5058       214
LR_close          13295       777
LR_open           28189       348
NR                78113      8218
Comparing row 1 with last row:
not_pELS_peak     pELS_peak 
         4214            82 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 2 with last row:
not_pELS_peak     pELS_peak 
         2590           568 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 3 with last row:
not_pELS_peak     pELS_peak 
         1056             5 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 4 with last row:
not_pELS_peak     pELS_peak 
         8887           462 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 5 with last row:
not_pELS_peak     pELS_peak 
           95             4 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 6 with last row:
not_pELS_peak     pELS_peak 
         5058           214 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 7 with last row:
not_pELS_peak     pELS_peak 
        13295           777 
not_pELS_peak     pELS_peak 
        78113          8218 
Comparing row 8 with last row:
not_pELS_peak     pELS_peak 
        28189           348 
not_pELS_peak     pELS_peak 
        78113          8218 
Processing Matrix: CTCF 
          CTCF_peak not_CTCF_peak
EAR_close       101          4195
EAR_open         97          3061
ESR_close       377          8972
ESR_opcl          1            98
ESR_open         64          5208
LR_close        592         13480
LR_open         219         28318
NR             5403         80928
Comparing row 1 with last row:
    CTCF_peak not_CTCF_peak 
          101          4195 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Comparing row 2 with last row:
    CTCF_peak not_CTCF_peak 
           97          3061 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Comparing row 3 with last row:
    CTCF_peak not_CTCF_peak 
          377          8972 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Comparing row 4 with last row:
    CTCF_peak not_CTCF_peak 
            1            98 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Comparing row 5 with last row:
    CTCF_peak not_CTCF_peak 
           64          5208 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Comparing row 6 with last row:
    CTCF_peak not_CTCF_peak 
          592         13480 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Comparing row 7 with last row:
    CTCF_peak not_CTCF_peak 
          219         28318 
    CTCF_peak not_CTCF_peak 
         5403         80928 
Processing Matrix: All cREs 
          cRE_peak not_cRE_peak
EAR_close      564         3732
EAR_open      1032         2126
ESR_clop        36         1025
ESR_close     2324         7025
ESR_opcl        13           86
ESR_open       512         4760
LR_close      3128        10944
LR_open       1637        26900
NR           22534        63797
Comparing row 1 with last row:
    cRE_peak not_cRE_peak 
         564         3732 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 2 with last row:
    cRE_peak not_cRE_peak 
        1032         2126 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 3 with last row:
    cRE_peak not_cRE_peak 
          36         1025 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 4 with last row:
    cRE_peak not_cRE_peak 
        2324         7025 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 5 with last row:
    cRE_peak not_cRE_peak 
          13           86 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 6 with last row:
    cRE_peak not_cRE_peak 
         512         4760 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 7 with last row:
    cRE_peak not_cRE_peak 
        3128        10944 
    cRE_peak not_cRE_peak 
       22534        63797 
Comparing row 8 with last row:
    cRE_peak not_cRE_peak 
        1637        26900 
    cRE_peak not_cRE_peak 
       22534        63797 </code></pre>
<pre class="r"><code># Print the resulting dataframe
print(results_cre)</code></pre>
<pre><code>            Matrix_Name Row_Compared Chi_Square_Statistic       P_Value
X-squared           PLS    EAR_close          440.1586065  9.998327e-98
X-squared1          PLS     EAR_open          339.2781306  9.163922e-76
X-squared2          PLS    ESR_close          608.3795659 2.518823e-134
X-squared3          PLS     ESR_opcl            9.4848736  2.071729e-03
X-squared4          PLS     ESR_open          355.5913847  2.567814e-79
X-squared5          PLS     LR_close          419.6860256  2.856187e-93
X-squared6          PLS      LR_open         3008.0307322  0.000000e+00
X-squared7         dELS    EAR_close           14.1010187  1.732499e-04
X-squared8         dELS     EAR_open           34.0740951  5.305287e-09
X-squared9         dELS     ESR_clop           22.3614218  2.258657e-06
X-squared10        dELS    ESR_close          772.4658531 5.231320e-170
X-squared11        dELS     ESR_opcl            0.1595127  6.896056e-01
X-squared12        dELS     ESR_open          100.6132894  1.118150e-23
X-squared13        dELS     LR_close          280.7696601  5.103577e-63
X-squared14        dELS      LR_open          411.2476997  1.961260e-91
X-squared15        pELS    EAR_close          283.9817228  1.018430e-63
X-squared16        pELS     EAR_open          245.7142389  2.232575e-55
X-squared17        pELS     ESR_clop           99.6034670  1.861782e-23
X-squared18        pELS    ESR_close          213.7174103  2.122586e-48
X-squared19        pELS     ESR_opcl            2.8411858  9.187639e-02
X-squared20        pELS     ESR_open          176.5759605  2.710603e-40
X-squared21        pELS     LR_close          236.5801977  2.189762e-53
X-squared22        pELS      LR_open         2139.6174210  0.000000e+00
X-squared23        CTCF    EAR_close          108.8501702  1.750287e-25
X-squared24        CTCF     EAR_open           53.0905639  3.185189e-13
X-squared25        CTCF    ESR_close           73.2456334  1.144782e-17
X-squared26        CTCF     ESR_opcl            3.7947687  5.141298e-02
X-squared27        CTCF     ESR_open          224.3993326  9.927024e-51
X-squared28        CTCF     LR_close           90.3394922  2.006082e-21
X-squared29        CTCF      LR_open         1388.0944201 8.119789e-304
X-squared30    All cREs    EAR_close          362.0022835  1.031801e-80
X-squared31    All cREs     EAR_open           67.5954503  2.007302e-16
X-squared32    All cREs     ESR_clop          280.9720486  4.610746e-63
X-squared33    All cREs    ESR_close            6.7192904  9.537557e-03
X-squared34    All cREs     ESR_opcl            7.9684063  4.760083e-03
X-squared35    All cREs     ESR_open          708.0084840 5.422764e-156
X-squared36    All cREs     LR_close           95.2051750  1.716410e-22
X-squared37    All cREs      LR_open         5352.7289387  0.000000e+00</code></pre>
</div>
<div id="odds-ratio-results" class="section level3">
<h3>odds ratio results</h3>
<pre class="r"><code>results_or_cre &lt;- data.frame(Matrix_Name = character(),
                      Row_Compared = character(),
                      Odds_Ratio = numeric(),
                      Lower_CI = numeric(),
                      Upper_CI = numeric(),
                      P_Value = numeric(),
                      stringsAsFactors = FALSE)

# Loop through each matrix in the list
for (matrix_name in names(matrix_list_cre)) {
  current_matrix &lt;- matrix_list_cre[[matrix_name]]
  n_rows &lt;- nrow(current_matrix)
  
  # Loop through each row of the current matrix (except the last row)
  for (i in 1:(n_rows - 1)) {
    # Perform odds ratio test between row i and the last row using epitools
    test_result &lt;- tryCatch({
      contingency_table &lt;- rbind(current_matrix[i, ], current_matrix[n_rows, ])
      
      # Check if any row in the contingency table contains only zeros
      if (any(rowSums(contingency_table) == 0)) {
        stop(&quot;Contingency table contains empty rows.&quot;)
      }
      
      oddsratio_result &lt;- oddsratio(contingency_table)
       # Ensure the oddsratio result has at least 2 rows
      if (nrow(oddsratio_result$measure) &lt; 2) {
        stop(&quot;oddsratio result does not have enough data.&quot;)
      }
      
     
      
      list(oddsratio = oddsratio_result, p.value = oddsratio_result$p.value[2,&quot;chi.square&quot;])
      
    }, error = function(e) {
      cat(&quot;Error in odds ratio test for row&quot;, i, &quot;in matrix&quot;, matrix_name, &quot;:&quot;, e$message, &quot;\n&quot;)
      return(NULL)
    })
    
    # Only store the result if test_result is valid (i.e., not NULL)
    if (!is.null(test_result)) {
      or_value &lt;- test_result$oddsratio$measure[2, &quot;estimate&quot;]
      lower_ci &lt;- test_result$oddsratio$measure[2, &quot;lower&quot;]
      upper_ci &lt;- test_result$oddsratio$measure[2, &quot;upper&quot;]
      p_value &lt;- test_result$oddsratio$p.value[2,&quot;chi.square&quot;]
      
      # Check if the values are numeric and valid (not NA)
      if (!is.na(or_value) &amp;&amp; !is.na(lower_ci) &amp;&amp; !is.na(upper_ci) &amp;&amp; !is.na(p_value)) {
        # Store the results in the dataframe
        results_or_cre &lt;- rbind(results_or_cre, data.frame(Matrix_Name = matrix_name,
                                             Row_Compared = rownames(current_matrix)[i],
                                             Odds_Ratio = or_value,
                                             Lower_CI = lower_ci,
                                             Upper_CI = upper_ci,
                                             P_Value = p_value))
      }
    }
  }
}

# Print the resulting dataframe
print(results_or_cre)</code></pre>
<pre><code>           Matrix_Name Row_Compared  Odds_Ratio    Lower_CI    Upper_CI
estimate           PLS    EAR_close  0.08873621 0.065812502  0.11660880
estimate1          PLS     EAR_open  2.21770111 2.032219016  2.41712502
estimate2          PLS    ESR_close  0.24895997 0.220534742  0.27993125
estimate3          PLS     ESR_opcl  0.09139664 0.003984574  0.40173212
estimate4          PLS     ESR_open  0.24213064 0.205559544  0.28300079
estimate5          PLS     LR_close  0.46513004 0.431280151  0.50092660
estimate6          PLS      LR_open  0.06059274 0.052810332  0.06913244
estimate7         dELS    EAR_close  1.24427882 1.109370353  1.39116093
estimate8         dELS     EAR_open  0.58592627 0.486547473  0.69885736
estimate9         dELS     ESR_clop  0.42957113 0.293785618  0.60331312
estimate10        dELS    ESR_close  2.40486231 2.256021121  2.56194710
estimate11        dELS     ESR_opcl  1.27150395 0.564235628  2.46127114
estimate12        dELS     ESR_open  0.45345412 0.385612069  0.52930838
estimate13        dELS     LR_close  1.66651649 1.568633173  1.76946988
estimate14        dELS      LR_open  0.49167458 0.458256458  0.52702183
estimate15        pELS    EAR_close  5.39591596 4.363117475  6.77435238
estimate16        pELS     EAR_open  0.47959990 0.437140886  0.52714590
estimate17        pELS     ESR_clop 21.52462844 9.993402478 60.76085222
estimate18        pELS    ESR_close  2.02309077 1.839664132  2.23027407
estimate19        pELS     ESR_opcl  2.40604488 1.006916288  8.00941605
estimate20        pELS     ESR_open  2.48479009 2.168926163  2.86244577
estimate21        pELS     LR_close  1.79981942 1.669540172  1.94296644
estimate22        pELS      LR_open  8.51810105 7.658089103  9.50711264
estimate23        CTCF    EAR_close  0.36121744 0.294086600  0.43828484
estimate24        CTCF     EAR_open  0.47542566 0.385149006  0.57951433
estimate25        CTCF    ESR_close  0.62962941 0.565049971  0.69947166
estimate26        CTCF     ESR_opcl  0.17489552 0.007585741  0.76868647
estimate27        CTCF     ESR_open  0.18453548 0.142540114  0.23428761
estimate28        CTCF     LR_close  0.65796237 0.602678193  0.71696884
estimate29        CTCF      LR_open  0.11592111 0.100931600  0.13242166
estimate30    All cREs    EAR_close  0.42796559 0.390838761  0.46776367
estimate31    All cREs     EAR_open  1.37439768 1.273522528  1.48235079
estimate32    All cREs     ESR_clop  0.09988679 0.070295522  0.13708089
estimate33    All cREs    ESR_close  0.93663630 0.891429389  0.98380494
estimate34    All cREs     ESR_opcl  0.43256077 0.229614967  0.74782181
estimate35    All cREs     ESR_open  0.30461579 0.277414564  0.33375228
estimate36    All cREs     LR_close  0.80923740 0.775408638  0.84428393
estimate37    All cREs      LR_open  0.17230937 0.163521753  0.18147077
                 P_Value
estimate    5.891433e-98
estimate1   5.443040e-76
estimate2  1.624617e-134
estimate3   1.201674e-03
estimate4   1.667711e-79
estimate5   2.106876e-93
estimate6   0.000000e+00
estimate7   1.527816e-04
estimate8   4.251926e-09
estimate9   1.657225e-06
estimate10 2.926896e-170
estimate11  5.470573e-01
estimate12  8.329845e-24
estimate13  3.791406e-63
estimate14  1.456424e-91
estimate15  6.438159e-64
estimate16  1.382035e-55
estimate17  1.090912e-23
estimate18  1.606522e-48
estimate19  6.331709e-02
estimate20  1.952285e-40
estimate21  1.712292e-53
estimate22  0.000000e+00
estimate23  1.239545e-25
estimate24  2.406066e-13
estimate25  9.387493e-18
estimate26  3.110732e-02
estimate27  6.323587e-51
estimate28  1.668110e-21
estimate29 4.500441e-304
estimate30  7.326877e-81
estimate31  1.690552e-16
estimate32  2.545170e-63
estimate33  9.198866e-03
estimate34  3.310381e-03
estimate35 3.507452e-156
estimate36  1.548760e-22
estimate37  0.000000e+00</code></pre>
<pre class="r"><code>library(circlize)
col_fun_cre = colorRamp2(c(0,6), c(&quot;white&quot;, &quot;purple3&quot;))
sig_mat_cre &lt;- results_or_cre %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select( Matrix_Name,Row_Compared,P_Value) %&gt;% 
  pivot_wider(., id_cols = Matrix_Name, names_from = Row_Compared, values_from = P_Value) %&gt;% 
  column_to_rownames(&quot;Matrix_Name&quot;) %&gt;% 
  as.matrix() 


results_or_cre %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select( Matrix_Name,Row_Compared,Odds_Ratio) %&gt;% 
  pivot_wider(., id_cols = Matrix_Name, names_from = Row_Compared, values_from = Odds_Ratio) %&gt;% 
  column_to_rownames(&quot;Matrix_Name&quot;) %&gt;% 
  as.matrix() %&gt;% 
  ComplexHeatmap::Heatmap(. ,col = col_fun_cre, 
                          cluster_rows=FALSE, 
                          cluster_columns=FALSE, 
                          column_names_side = &quot;top&quot;, 
                          column_names_rot = 45,
                          cell_fun = function(j, i, x, y, width, height, fill) {if (!is.na(sig_mat_cre[i, j]) &amp;&amp; sig_mat_cre[i, j] &lt; 0.05)
            grid.text(&quot;*&quot;, x, y, gp = gpar(fontsize = 20))})</code></pre>
<embed src="figure/Enhancer_files_ff.Rmd/odds ratio test-1.pdf" width="672" style="display: block; margin: auto;" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-odds-ratio-test-1">
Past versions of "odds ratio test-1.pdf"
</button>
</p>
<div id="fig-no-spaces-odds-ratio-test-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0b1ae82df515222fd07a72a925941b9d4ff2ff1f/docs/figure/Enhancer_files_ff.Rmd/odds ratio test-1.pdf" target="_blank">0b1ae82</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d926a4a925590c9984eb97f3fdd0d0c30b8f98da/docs/figure/Enhancer_files_ff.Rmd/odds ratio test-1.pdf" target="_blank">d926a4a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="top2b" class="section level3">
<h3>Top2b</h3>
<p>to be determined…. ingnore for now</p>
</div>
<div id="counts-of-peaks-" class="section level3">
<h3>Counts of peaks-</h3>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] circlize_0.4.16                         
 [2] epitools_0.5-10.1                       
 [3] gprofiler2_0.2.3                        
 [4] genomation_1.36.0                       
 [5] plyranges_1.24.0                        
 [6] BSgenome.Hsapiens.UCSC.hg38_1.4.5       
 [7] BSgenome_1.72.0                         
 [8] BiocIO_1.14.0                           
 [9] MotifDb_1.46.0                          
[10] Biostrings_2.72.1                       
[11] XVector_0.44.0                          
[12] TFBSTools_1.42.0                        
[13] JASPAR2022_0.99.8                       
[14] BiocFileCache_2.12.0                    
[15] dbplyr_2.5.0                            
[16] devtools_2.4.5                          
[17] usethis_3.0.0                           
[18] ggpubr_0.6.0                            
[19] BiocParallel_1.38.0                     
[20] scales_1.3.0                            
[21] VennDiagram_1.7.3                       
[22] futile.logger_1.4.3                     
[23] gridExtra_2.3                           
[24] ggfortify_0.4.17                        
[25] edgeR_4.2.1                             
[26] limma_3.60.4                            
[27] rtracklayer_1.64.0                      
[28] org.Hs.eg.db_3.19.1                     
[29] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[30] GenomicFeatures_1.56.0                  
[31] AnnotationDbi_1.66.0                    
[32] Biobase_2.64.0                          
[33] GenomicRanges_1.56.1                    
[34] GenomeInfoDb_1.40.1                     
[35] IRanges_2.38.1                          
[36] S4Vectors_0.42.1                        
[37] BiocGenerics_0.50.0                     
[38] ChIPseeker_1.40.0                       
[39] RColorBrewer_1.1-3                      
[40] broom_1.0.7                             
[41] kableExtra_1.4.0                        
[42] lubridate_1.9.3                         
[43] forcats_1.0.0                           
[44] stringr_1.5.1                           
[45] dplyr_1.1.4                             
[46] purrr_1.0.2                             
[47] readr_2.1.5                             
[48] tidyr_1.3.1                             
[49] tibble_3.2.1                            
[50] ggplot2_3.5.1                           
[51] tidyverse_2.0.0                         
[52] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] R.methodsS3_1.8.2                      
  [2] vroom_1.6.5                            
  [3] urlchecker_1.0.1                       
  [4] poweRlaw_0.80.0                        
  [5] vctrs_0.6.5                            
  [6] digest_0.6.37                          
  [7] png_0.1-8                              
  [8] shape_1.4.6.1                          
  [9] git2r_0.33.0                           
 [10] ggrepel_0.9.6                          
 [11] magick_2.8.5                           
 [12] MASS_7.3-61                            
 [13] reshape2_1.4.4                         
 [14] httpuv_1.6.15                          
 [15] foreach_1.5.2                          
 [16] qvalue_2.36.0                          
 [17] withr_3.0.1                            
 [18] xfun_0.47                              
 [19] ggfun_0.1.6                            
 [20] ellipsis_0.3.2                         
 [21] memoise_2.0.1                          
 [22] profvis_0.4.0                          
 [23] systemfonts_1.1.0                      
 [24] tidytree_0.4.6                         
 [25] GlobalOptions_0.1.2                    
 [26] gtools_3.9.5                           
 [27] R.oo_1.26.0                            
 [28] Formula_1.2-5                          
 [29] KEGGREST_1.44.1                        
 [30] promises_1.3.0                         
 [31] httr_1.4.7                             
 [32] rstatix_0.7.2                          
 [33] restfulr_0.0.15                        
 [34] ps_1.8.0                               
 [35] rstudioapi_0.16.0                      
 [36] UCSC.utils_1.0.0                       
 [37] miniUI_0.1.1.1                         
 [38] generics_0.1.3                         
 [39] DOSE_3.30.5                            
 [40] processx_3.8.4                         
 [41] curl_5.2.3                             
 [42] zlibbioc_1.50.0                        
 [43] ggraph_2.2.1                           
 [44] polyclip_1.10-7                        
 [45] GenomeInfoDbData_1.2.12                
 [46] SparseArray_1.4.8                      
 [47] xtable_1.8-4                           
 [48] pracma_2.4.4                           
 [49] doParallel_1.0.17                      
 [50] evaluate_1.0.0                         
 [51] S4Arrays_1.4.1                         
 [52] hms_1.1.3                              
 [53] colorspace_2.1-1                       
 [54] filelock_1.0.3                         
 [55] magrittr_2.0.3                         
 [56] later_1.3.2                            
 [57] viridis_0.6.5                          
 [58] ggtree_3.12.0                          
 [59] lattice_0.22-6                         
 [60] getPass_0.2-4                          
 [61] XML_3.99-0.17                          
 [62] shadowtext_0.1.4                       
 [63] cowplot_1.1.3                          
 [64] matrixStats_1.4.1                      
 [65] pillar_1.9.0                           
 [66] nlme_3.1-166                           
 [67] iterators_1.0.14                       
 [68] pwalign_1.0.0                          
 [69] gridBase_0.4-7                         
 [70] caTools_1.18.3                         
 [71] compiler_4.4.1                         
 [72] stringi_1.8.4                          
 [73] SummarizedExperiment_1.34.0            
 [74] GenomicAlignments_1.40.0               
 [75] plyr_1.8.9                             
 [76] crayon_1.5.3                           
 [77] abind_1.4-8                            
 [78] gridGraphics_0.5-1                     
 [79] locfit_1.5-9.10                        
 [80] graphlayouts_1.2.0                     
 [81] bit_4.5.0                              
 [82] fastmatch_1.1-4                        
 [83] whisker_0.4.1                          
 [84] codetools_0.2-20                       
 [85] bslib_0.8.0                            
 [86] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [87] GetoptLong_1.0.5                       
 [88] plotly_4.10.4                          
 [89] mime_0.12                              
 [90] splines_4.4.1                          
 [91] Rcpp_1.0.13                            
 [92] knitr_1.48                             
 [93] blob_1.2.4                             
 [94] utf8_1.2.4                             
 [95] clue_0.3-65                            
 [96] seqLogo_1.70.0                         
 [97] fs_1.6.4                               
 [98] pkgbuild_1.4.4                         
 [99] ggsignif_0.6.4                         
[100] ggplotify_0.1.2                        
[101] Matrix_1.7-0                           
[102] callr_3.7.6                            
[103] statmod_1.5.0                          
[104] tzdb_0.4.0                             
[105] svglite_2.1.3                          
[106] tweenr_2.0.3                           
[107] pkgconfig_2.0.3                        
[108] tools_4.4.1                            
[109] cachem_1.1.0                           
[110] RSQLite_2.3.7                          
[111] viridisLite_0.4.2                      
[112] DBI_1.2.3                              
[113] splitstackshape_1.4.8                  
[114] impute_1.78.0                          
[115] fastmap_1.2.0                          
[116] rmarkdown_2.28                         
[117] Rsamtools_2.20.0                       
[118] sass_0.4.9                             
[119] patchwork_1.3.0                        
[120] carData_3.0-5                          
[121] farver_2.1.2                           
[122] tidygraph_1.3.1                        
[123] scatterpie_0.2.4                       
[124] yaml_2.3.10                            
[125] MatrixGenerics_1.16.0                  
[126] cli_3.6.3                              
[127] lifecycle_1.0.4                        
[128] lambda.r_1.2.4                         
[129] sessioninfo_1.2.2                      
[130] backports_1.5.0                        
[131] annotate_1.82.0                        
[132] timechange_0.3.0                       
[133] gtable_0.3.5                           
[134] rjson_0.2.23                           
[135] parallel_4.4.1                         
[136] ape_5.8                                
[137] jsonlite_1.8.9                         
[138] bitops_1.0-8                           
[139] bit64_4.5.2                            
[140] yulab.utils_0.1.7                      
[141] CNEr_1.40.0                            
[142] futile.options_1.0.1                   
[143] jquerylib_0.1.4                        
[144] highr_0.11                             
[145] GOSemSim_2.30.2                        
[146] R.utils_2.12.3                         
[147] lazyeval_0.2.2                         
[148] shiny_1.9.1                            
[149] htmltools_0.5.8.1                      
[150] enrichplot_1.24.4                      
[151] GO.db_3.19.1                           
[152] rappdirs_0.3.3                         
[153] formatR_1.14                           
[154] glue_1.8.0                             
[155] TFMPvalue_0.0.9                        
[156] httr2_1.0.5                            
[157] RCurl_1.98-1.16                        
[158] rprojroot_2.0.4                        
[159] treeio_1.28.0                          
[160] boot_1.3-31                            
[161] igraph_2.0.3                           
[162] R6_2.5.1                               
[163] gplots_3.1.3.1                         
[164] labeling_0.4.3                         
[165] cluster_2.1.6                          
[166] pkgload_1.4.0                          
[167] aplot_0.2.3                            
[168] DirichletMultinomial_1.46.0            
[169] DelayedArray_0.30.1                    
[170] tidyselect_1.2.1                       
[171] plotrix_3.8-4                          
[172] ggforce_0.4.2                          
[173] xml2_1.3.6                             
[174] car_3.1-3                              
[175] seqPattern_1.36.0                      
[176] munsell_0.5.1                          
[177] KernSmooth_2.23-24                     
[178] data.table_1.16.0                      
[179] htmlwidgets_1.6.4                      
[180] fgsea_1.30.0                           
[181] ComplexHeatmap_2.20.0                  
[182] rlang_1.1.4                            
[183] remotes_2.5.0                          
[184] fansi_1.0.6                            </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
