<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-06-13" />

<title>Enhancer_files</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enhancer_files</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-06-13</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-06-13
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtreeb68190d5f301921e58ca943385a51162ec250211targetblankb68190da">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/b68190d5f301921e58ca943385a51162ec250211" target="_blank">b68190d</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtreeb68190d5f301921e58ca943385a51162ec250211targetblankb68190da"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/b68190d5f301921e58ca943385a51162ec250211" target="_blank">b68190d</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/Fastqc_results.Rmd
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Enhancer_files.Rmd</code>) and
HTML (<code>docs/Enhancer_files.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b68190d5f301921e58ca943385a51162ec250211/analysis/Enhancer_files.Rmd" target="_blank">b68190d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-13
</td>
<td>
updates and new analysis
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5d5401708c50e87c3abf2b4c7e2a6b5acb142b07/analysis/Enhancer_files.Rmd" target="_blank">5d54017</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-12
</td>
<td>
adding commit
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ba8611bae735472bb1aba81fbca238aa9f8ecc23/analysis/Enhancer_files.Rmd" target="_blank">ba8611b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-02
</td>
<td>
updates of code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/553ad80601884b94f1f751dcec13baf5767877e0/docs/Enhancer_files.html" target="_blank">553ad80</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0f40db54c7d97bb4f6257e41acdcd6330057d426/analysis/Enhancer_files.Rmd" target="_blank">0f40db5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
<td>
adding in enhancer work
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/fe44e2c4b7ea586db982cc7b0d526dca7deb8771/docs/Enhancer_files.html" target="_blank">fe44e2c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/39404c91ae16f9d22762175f60bbe73898818aab/analysis/Enhancer_files.Rmd" target="_blank">39404c9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
<td>
adding in enhancer work
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
library(bedr)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)
library(plyranges)
library(genomation)</code></pre>
<p>Data loading</p>
<pre class="r"><code>toplistall_RNA &lt;- readRDS(&quot;data/other_papers/toplistall_RNA.RDS&quot;) 
S13Table &lt;- read.csv( &quot;data/other_papers/S13Table_Matthews2024.csv&quot;,row.names = 1)
##14021

EAR_RNA &lt;- S13Table %&gt;% 
  dplyr::filter(MOTIF==&quot;EAR&quot;) %&gt;% 
  dplyr::select(ENTREZID) %&gt;% 
  mutate(ENTREZID= as.character(ENTREZID))
ESR_RNA &lt;- S13Table %&gt;% 
  dplyr::filter(MOTIF==&quot;ESR&quot;)%&gt;% 
  dplyr::select(ENTREZID)%&gt;% 
  mutate(ENTREZID= as.character(ENTREZID))
LR_RNA &lt;- S13Table %&gt;% 
  dplyr::filter(MOTIF==&quot;LR&quot;)%&gt;% 
  dplyr::select(ENTREZID)%&gt;% 
  mutate(ENTREZID=as.character(ENTREZID))
NR_RNA &lt;- S13Table %&gt;% 
  dplyr::filter(MOTIF==&quot;NR&quot;)%&gt;% 
  dplyr::select(ENTREZID)%&gt;% 
  mutate(ENTREZID= as.character(ENTREZID))

exp_neargene_table &lt;- read_delim(&quot;data/n45_bedfiles/exp_neargene_table.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

# col_ng_peak &lt;- exp_neargene_table %&gt;%
#   group_by(chr, start, end, peakid) %&gt;%
#     summarise(NCBI_gene = paste(unique(ENTREZID),collapse=&quot;,&quot;),
#               ensembl_ID= paste(unique(ensembl_id),collapse = &quot;,&quot;),
#               SYMBOL= paste(unique(SYMBOL),collapse = &quot;,&quot;) , 
#               dist_to_NG =min(dist_to_NG)) %&gt;% 
#   ungroup()
col_ng_peak &lt;- read.csv(&quot;data/col_ng_peak.csv&quot;,row.names = 1)
peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)
# list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)

peakAnnoList_3_n45 &lt;- readRDS(&quot;data/peakAnnoList_3_n45.RDS&quot;)
peakAnnoList_24_n45&lt;- readRDS(&quot;data/peakAnnoList_24_n45.RDS&quot;)

all_peak_gr &lt;- as.GRanges(peakAnnoList_24_n45$background)

Joined_tss_neargene &lt;- all_peak_gr %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(seqnames:id,geneId,distanceToTSS) %&gt;% 
  dplyr::filter(!grepl(&quot;chrX&quot;,id)&amp; !grepl(&quot;chrY&quot;,id))%&gt;%
  dplyr::rename(&quot;close_geneId&quot;=&quot;geneId&quot;) %&gt;% 
  mutate(start=start+1) %&gt;% 
  left_join(., col_ng_peak, by=c(&quot;id&quot;=&quot;peakid&quot;, &quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;)) %&gt;% 
  dplyr::rename(&quot;NCBI_gene_NG&quot;=&quot;NCBI_gene&quot;) 

  
### THESE ARE DAR peaks (not total peaks)



DOX_3hr &lt;- as.data.frame(peakAnnoList_3_n45$DOX_3_n45)
DOX_3hr_gr &lt;- as.GRanges(peakAnnoList_3_n45$DOX_3_n45)
EPI_3hr &lt;- as.data.frame(peakAnnoList_3_n45$EPI_3_n45)
EPI_3hr_gr &lt;- as.GRanges(peakAnnoList_3_n45$EPI_3_n45)
DNR_3hr &lt;- as.data.frame(peakAnnoList_3_n45$DNR_3_n45)
DNR_3hr_gr &lt;- as.GRanges(peakAnnoList_3_n45$DNR_3_n45)
MTX_3hr &lt;- as.data.frame(peakAnnoList_3_n45$MTX_3_n45)
MTX_3hr_gr &lt;- as.GRanges(peakAnnoList_3_n45$MTX_3_n45)
# # TRZ_3hr &lt;- as.data.frame(peakAnnoList_3_n45$TRZ_3_n45)
# TRZ_3hr_gr &lt;- GRanges(TRZ_3hr)
DOX_3_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% DOX_3hr$id)
EPI_3_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% EPI_3hr$id)
DNR_3_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% DNR_3hr$id)
MTX_3_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% MTX_3hr$id)

### THESE ARE DAR peaks (not total peaks)

DOX_24hr &lt;- as.data.frame(peakAnnoList_24_n45$DOX_24_n45)
DOX_24hr_gr &lt;- as.GRanges(peakAnnoList_24_n45$DOX_24_n45)
EPI_24hr &lt;- as.data.frame(peakAnnoList_24_n45$EPI_24_n45)
EPI_24hr_gr &lt;-as.GRanges(peakAnnoList_24_n45$EPI_24_n45)
DNR_24hr &lt;- as.data.frame(peakAnnoList_24_n45$DNR_24_n45)
DNR_24hr_gr &lt;- as.GRanges(peakAnnoList_24_n45$DNR_24_n45)
MTX_24hr &lt;- as.data.frame(peakAnnoList_24_n45$MTX_24_n45)
MTX_24hr_gr &lt;- as.GRanges(peakAnnoList_24_n45$MTX_24_n45)
# TRZ_24hr &lt;- as.data.frame(peakAnnoList_24_n45$TRZ_24_n45)
# TRZ_24hr_gr &lt;- as.GRanges(peakAnnoList_24_n45$TRZ_24_n45)
## THESE ARE DAR peaks (not total peaks)
DOX_24_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% DOX_24hr$id)
EPI_24_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% EPI_24hr$id)
DNR_24_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% DNR_24hr$id)
MTX_24_peak_list &lt;-  col_ng_peak %&gt;%
  # dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% MTX_24hr$id)


EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)


ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr &lt;- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)


EAR_peak_list_20k &lt;-  col_ng_peak %&gt;%
  dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% EAR_df$id) %&gt;% 
  mutate(MRC=&quot;EAR&quot;) 

EAR_peak_list &lt;-  col_ng_peak %&gt;%
    dplyr::filter(peakid %in% EAR_df$id) %&gt;% 
  mutate(MRC=&quot;EAR&quot;) 

ESR_peak_list_20k &lt;- col_ng_peak %&gt;%
  dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% ESR_df$id) %&gt;% 
    mutate(MRC=&quot;ESR&quot;)

ESR_peak_list&lt;- col_ng_peak %&gt;%
    dplyr::filter(peakid %in% ESR_df$id) %&gt;% 
    mutate(MRC=&quot;ESR&quot;)

LR_peak_list_20k &lt;- col_ng_peak %&gt;%
  dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% LR_df$id) %&gt;% 
    mutate(MRC=&quot;LR&quot;)

LR_peak_list &lt;- col_ng_peak %&gt;%
    dplyr::filter(peakid %in% LR_df$id) %&gt;% 
    mutate(MRC=&quot;LR&quot;)

NR_peak_list_20k &lt;- col_ng_peak %&gt;%
  dplyr::filter(dist_to_NG&lt;20000) %&gt;%
    dplyr::filter(peakid %in% NR_df$id) %&gt;% 
    mutate(MRC=&quot;NR&quot;)
NR_peak_list &lt;- col_ng_peak %&gt;%
    dplyr::filter(peakid %in% NR_df$id) %&gt;% 
    mutate(MRC=&quot;NR&quot;)

peak_list_20k_MRC &lt;- EAR_peak_list_20k %&gt;% 
  rbind(ESR_peak_list_20k) %&gt;% 
  rbind(LR_peak_list_20k) %&gt;% 
  rbind(NR_peak_list_20k)

peak_list_MRC &lt;- EAR_peak_list %&gt;% 
  rbind(ESR_peak_list) %&gt;% 
  rbind(LR_peak_list) %&gt;% 
  rbind(NR_peak_list)

gr_EAR_peak_list&lt;-  col_ng_peak %&gt;%
    dplyr::filter(peakid %in% EAR_df$id) %&gt;% 
  mutate(MRC=&quot;EAR&quot;) %&gt;% 
  GRanges()

gr_ESR_peak_list &lt;- col_ng_peak %&gt;%
    dplyr::filter(peakid %in% ESR_df$id) %&gt;% 
    mutate(MRC=&quot;ESR&quot;)%&gt;% 
  GRanges()

gr_LR_peak_list &lt;- col_ng_peak %&gt;%
    dplyr::filter(peakid %in% LR_df$id) %&gt;% 
    mutate(MRC=&quot;LR&quot;)%&gt;% 
  GRanges()

gr_NR_peak_list &lt;- col_ng_peak %&gt;%
    dplyr::filter(peakid %in% NR_df$id) %&gt;% 
    mutate(MRC=&quot;NR&quot;)%&gt;% 

  GRanges()

n45_fullpeaks_gr &lt;- readBed(&quot;data/n45_bedfiles/DAR_DEG_background.bed&quot;)
###file for ATACseq replicability
ENCFF544RZR &lt;-readRDS (&quot;data/n45_bedfiles/ENCFF533RZR_gr.RDS&quot;)
# ENCFFFF544RZR_bed &lt;- read_delim(&quot;data/n45_bedfiles/ENCFF134HBK.bed&quot;, 
#      delim = &quot;\t&quot;, escape_double = FALSE, 
#      col_names = FALSE, trim_ws = TRUE)
#  ENCFF544RZR &lt;- ENCFF544RZR_bed %&gt;% 
#    rename(X1=&quot;chr&quot;,
#          X2=&quot;start&quot;,
#          X3=&quot;end&quot;,
#          X10=&quot;length&quot;) %&gt;%
#    GRanges()
 
Snyder_overlaps &lt;- join_overlap_intersect(n45_fullpeaks_gr,ENCFF544RZR)
# 84725
# length(find_overlaps(ENCFF544RZR,n45_fullpeaks_gr))
# test &lt;- (subsetByOverlaps(n45_fullpeaks_gr,ENCFF544RZR))
# test %&gt;% 
#   as.data.frame() %&gt;% 
#   distinct(name)
# head(Snyder_overlaps)
toplist_ATAC &lt;- readRDS(&quot;data/toplist_n45.RDS&quot;)

toplist_DOX24_ATAC &lt;- toplist_ATAC %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot; &amp; time==&quot;24 hours&quot;)</code></pre>
<div id="mrc-venns" class="section level3">
<h3>MRC venns</h3>
<p>These are Venns using the Neargenes between each set of peaks</p>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(EAR_peak_list$ensembl_ID), 
                                  unique(ESR_peak_list$ensembl_ID),  
                                  unique(LR_peak_list$ensembl_ID), 
                                  unique(NR_peak_list$ensembl_ID)),
                             category.names = c(&quot;EAR_NG&quot;,&quot;ESR_NG&quot;,&quot;LR_NG&quot;,&quot;NR_NG&quot;),
              show_intersect = FALSE,
              set_color = &quot;black&quot;,
              label = &quot;count&quot;,
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = 1,
              label_color = &quot;black&quot;,
              edge_lty = &quot;solid&quot;, set_size = 4)+
  ggtitle(&quot;Near gene distribution for all peaks&quot;)+
  scale_x_continuous(expand = expansion(mult = .2))</code></pre>
<p><img src="figure/Enhancer_files.Rmd/Venns%20of%20data-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Venns-of-data-1">
Past versions of "Venns of data-1.png"
</button>
</p>
<div id="fig-no-spaces-Venns-of-data-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fe44e2c4b7ea586db982cc7b0d526dca7deb8771/docs/figure/Enhancer_files.Rmd/Venns of data-1.png" target="_blank">fe44e2c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(EAR_peak_list$NCBI_gene),EAR_RNA$ENTREZID),
              category.names = c(&quot;EAR_ATAC&quot;,&quot;EAR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;EAR ATAC-neargenes and expressed response RNA&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))</code></pre>
<p><img src="figure/Enhancer_files.Rmd/Venns%20of%20data-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Venns-of-data-2">
Past versions of "Venns of data-2.png"
</button>
</p>
<div id="fig-no-spaces-Venns-of-data-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fe44e2c4b7ea586db982cc7b0d526dca7deb8771/docs/figure/Enhancer_files.Rmd/Venns of data-2.png" target="_blank">fe44e2c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(ESR_peak_list$NCBI_gene),ESR_RNA$ENTREZID),
              category.names = c(&quot;ESR_ATAC&quot;,&quot;ESR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;ESRATAC-neargenes and expressed response RNA&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))</code></pre>
<p><img src="figure/Enhancer_files.Rmd/Venns%20of%20data-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Venns-of-data-3">
Past versions of "Venns of data-3.png"
</button>
</p>
<div id="fig-no-spaces-Venns-of-data-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fe44e2c4b7ea586db982cc7b0d526dca7deb8771/docs/figure/Enhancer_files.Rmd/Venns of data-3.png" target="_blank">fe44e2c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(LR_peak_list$NCBI_gene),LR_RNA$ENTREZID),
              category.names = c(&quot;LR_ATAC&quot;,&quot;LR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;LR ATAC-neargenes and expressed response RNA&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))</code></pre>
<p><img src="figure/Enhancer_files.Rmd/Venns%20of%20data-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Venns-of-data-4">
Past versions of "Venns of data-4.png"
</button>
</p>
<div id="fig-no-spaces-Venns-of-data-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fe44e2c4b7ea586db982cc7b0d526dca7deb8771/docs/figure/Enhancer_files.Rmd/Venns of data-4.png" target="_blank">fe44e2c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(NR_peak_list$NCBI_gene),NR_RNA$ENTREZID),
              category.names = c(&quot;NR_ATAC&quot;,&quot;NR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;NR ATAC-neargenes and expressed response RNA&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+ 
  scale_x_continuous(expand = expansion(mult = .2))</code></pre>
<p><img src="figure/Enhancer_files.Rmd/Venns%20of%20data-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Venns-of-data-5">
Past versions of "Venns of data-5.png"
</button>
</p>
<div id="fig-no-spaces-Venns-of-data-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fe44e2c4b7ea586db982cc7b0d526dca7deb8771/docs/figure/Enhancer_files.Rmd/Venns of data-5.png" target="_blank">fe44e2c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="enhancers-and-response-clusters" class="section level3">
<h3>Enhancers and Response clusters</h3>
<p>First: obtained a list of cis Regulatory Elements from Encode Screen
[(<a href="https://screen.encodeproject.org/#"
class="uri">https://screen.encodeproject.org/#</a>)]</p>
<pre class="r"><code># BiocManager::install(&quot;plyranges&quot;)
# enhancers_HLV_46F &lt;- genomation::readBed(&quot;C:/Users/renee/Downloads/Supplements folde manuscriptr/ENCODE/heart_left_ventricle_tissue_female_adult_46_years.enhancers.bed&quot;)
 cREs_HLV_46F &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed&quot;)

cREs_HLV_53F &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF417JSF_ENCFF651XRK_ENCFF320IPT_ENCFF440RUS.7group.bed&quot;) 
 
NR_cREs &lt;- join_overlap_intersect(gr_NR_peak_list,cREs_HLV_46F)
LR_cREs &lt;- join_overlap_intersect(gr_LR_peak_list,cREs_HLV_46F)
ESR_cREs &lt;- join_overlap_intersect(gr_ESR_peak_list,cREs_HLV_46F)
EAR_cREs &lt;- join_overlap_intersect(gr_EAR_peak_list,cREs_HLV_46F)

# NR_cREs_all &lt;- join_overlap_intersect(NR_df_gr,cREs_HLV_46F)
# LR_cREs_all &lt;- join_overlap_intersect(LR_df_gr,cREs_HLV_46F)
# ESR_cREs_all &lt;- join_overlap_intersect(ESR_df_gr,cREs_HLV_46F)
# EAR_cREs_all &lt;- join_overlap_intersect(EAR_df_gr,cREs_HLV_46F)

# Whole_peaks &lt;- join_overlap_intersect(n45_fullpeaks_gr, cREs_HLV_46F)

NR_cREs %&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;NR&quot;, per=n/(length(unique(NR_cREs$peakid)))) %&gt;% 
  kable(., caption=paste0(&quot;No Response peaks overlapping cREs n= &quot;, length(unique(NR_cREs$peakid)))) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
No Response peaks overlapping cREs n= 69200
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
5458
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0788728
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
880
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0127168
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
647
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0093497
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
2580
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0372832
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
97871
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
1.4143208
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
11342
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.1639017
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
3133
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0452746
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
6361
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0919220
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
1698
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0245376
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
12562
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.1815318
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
2917
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0421532
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>LR_cREs%&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;LR&quot;, per=n/25610) %&gt;% 
kable(., caption=&quot;Late Response peaks overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Late Response peaks overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
841
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0328387
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0065599
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
77
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0030066
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
1422
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0555252
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
46840
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
1.8289731
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
1077
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0420539
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
261
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0101913
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
2675
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.1044514
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
414
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0161656
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
1446
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0564623
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
302
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0117923
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>ESR_cREs %&gt;%
  as.data.frame() %&gt;%
  # dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;) %&gt;%
  group_by(blockCount) %&gt;%
 count() %&gt;% 
  mutate(MRC=&quot;ESR&quot;, per=n/10122) %&gt;% 
  kable(., caption=&quot;Early-sustained response peaks overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Early-sustained response peaks overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
452
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0446552
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0127445
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
57
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0056313
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
970
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0958309
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
15493
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
1.5306264
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
479
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0473227
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0092867
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
1793
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.1771389
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
288
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0284529
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
853
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0842719
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
167
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0164987
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>EAR_cREs %&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;EAR&quot;, per=n/4813) %&gt;% 
  kable(., caption=&quot;Early acute response peaks overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Early acute response peaks overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
200
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0415541
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
38
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0078953
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0056098
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
298
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0619156
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
8301
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
1.7247039
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
931
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.1934344
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
171
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0355288
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
579
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.1202992
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0132973
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
976
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.2027841
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
136
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0282568
</td>
</tr>
</tbody>
</table>
</div>
<div id="visualization-of-categories" class="section level3">
<h3>Visualization of categories:</h3>
<p>First, filtering cRE set by type to include ‘CTCF-only,CTCF-bound’,
‘PLS’, ‘PLS,CTCF-bound’,‘dELS’, ‘dELS,CTCF-bound’, ‘pELS’,
‘pELS,CTCF-bound’.</p>
<p>Second, reclassify (new column) to group the CTCF-bound into their
respective groups so only 4 groups, (“CTCF-only”,“PLS”,“dELS”, “pELS”)
are created.</p>
<p>Third, left-join the DOX24hr ATAC LFC to data set and boxplot by
group.</p>
<p>PLS= promoter like sequences pELS= proximal enhancer like sequences
dELS=distal enhancer like sequences</p>
<pre class="r"><code>adj_toplist_DOX24_ATAC &lt;- toplist_DOX24_ATAC%&gt;% 
  dplyr::filter(adj.P.Val&lt;0.05,.keepall=TRUE)

EAR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot;EAR n = &quot;,length(unique(EAR_cREs$peakid)),&quot; out of &quot;, length(unique(EAR_df$id)), &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(EAR_cREs$peakid))/length(unique(EAR_df$id))*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/EARcRE%20maps-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ESR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
   ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot;ESR n = &quot;,length(unique(ESR_cREs$peakid)),&quot; out of &quot;, length(unique(ESR_df$id)), &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(ESR_cREs$peakid))/length(unique(ESR_df$id))*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/EARcRE%20maps-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>LR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(&quot;LR n = &quot;,length(unique(LR_cREs$peakid)),&quot; out of &quot;, length(unique(LR_df$id)), &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(LR_cREs$peakid))/length(unique(LR_df$id))*100),&quot;%)&quot;))+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/EARcRE%20maps-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>NR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
   ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  # facet_wrap(~direction)+
  ggtitle(paste0(&quot;NR n = &quot;,length(unique(NR_cREs$peakid)),&quot; out of &quot;, length(unique(NR_df$id)), &quot; total peaks (&quot;,sprintf(&quot;%.1f&quot;,length(unique(NR_cREs$peakid))/length(unique(NR_df$id))*100),&quot;%)&quot;))+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/EARcRE%20maps-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># cREs_HLV_46F %&gt;%
#   as.data.frame()# %&gt;%
#   dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
#   mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
#                       if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
#                               if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
#   right_join(., toplist_DOX24_ATAC, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
#     </code></pre>
</div>
<div id="ear-cres-with-all-peaks" class="section level3">
<h3>EAR cREs with all peaks</h3>
<pre class="r"><code>toplist_DOX24_ATAC_nop &lt;- toplist_ATAC %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot; &amp; time==&quot;24 hours&quot;)
EAR_cREs %&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;EAR&quot;, per=n/7321) %&gt;% 
  kable(., caption=&quot;Early acute response peaks (7321) overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Early acute response peaks (7321) overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
200
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0273187
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
38
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0051905
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0036880
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
298
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0407048
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
8301
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
1.1338615
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
931
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.1271684
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
171
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0233575
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
579
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0790876
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
64
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0087420
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
976
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.1333151
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
136
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
0.0185767
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>EAR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC_nop, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(&quot;EAR&quot;)+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/enhacerstuffforlater%20EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="esr-cres-with-all-peaks" class="section level3">
<h3>ESR cREs with all peaks</h3>
<pre class="r"><code>toplist_DOX24_ATAC_nop &lt;- toplist_ATAC %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot; &amp; time==&quot;24 hours&quot;)
ESR_cREs %&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;ESR&quot;, per=n/15665) %&gt;% 
  kable(., caption=&quot;Early sustained response peaks (15,665) overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Early sustained response peaks (15,665) overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
452
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0288541
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
129
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0082349
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
57
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0036387
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
970
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0619215
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
15493
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.9890201
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
479
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0305777
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0060006
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
1793
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.1144590
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
288
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0183849
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
853
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0544526
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
167
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
0.0106607
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>ESR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC_nop, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(&quot;ESR&quot;)+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/enhacerstuffforlater%20ESR-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="lr-cres-with-all-peaks" class="section level3">
<h3>LR cREs with all peaks</h3>
<pre class="r"><code>toplist_DOX24_ATAC_nop &lt;- toplist_ATAC %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot; &amp; time==&quot;24 hours&quot;)
LR_cREs %&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;LR&quot;, per=n/41759) %&gt;% 
  kable(., caption=&quot;Late response peaks (41,759) overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Late response peaks (41,759) overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
841
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0201394
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
168
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0040231
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
77
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0018439
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
1422
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0340525
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
46840
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
1.1216744
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
1077
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0257908
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
261
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0062501
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
2675
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0640580
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
414
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0099140
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
1446
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0346273
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
302
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
0.0072320
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>LR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC_nop, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(&quot;LR&quot;)+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/enhacerstuffforlater%20LR-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="nr-cres-with-all-peaks" class="section level3">
<h3>NR cREs with all peaks</h3>
<pre class="r"><code>toplist_DOX24_ATAC_nop &lt;- toplist_ATAC %&gt;% 
  dplyr::filter(trt==&quot;DOX&quot; &amp; time==&quot;24 hours&quot;)
NR_cREs %&gt;%
  as.data.frame() %&gt;%
  group_by(blockCount) %&gt;%
  count() %&gt;% 
  mutate(MRC=&quot;NR&quot;, per=n/83658) %&gt;% 
  kable(., caption=&quot;No response peaks (83,658) overlapping cREs&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
No response peaks (83,658) overlapping cREs
</caption>
<thead>
<tr>
<th style="text-align:left;">
blockCount
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:left;">
MRC
</th>
<th style="text-align:right;">
per
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CTCF-only,CTCF-bound
</td>
<td style="text-align:right;">
5458
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0652418
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3
</td>
<td style="text-align:right;">
880
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0105190
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-H3K4me3,CTCF-bound
</td>
<td style="text-align:right;">
647
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0077339
</td>
</tr>
<tr>
<td style="text-align:left;">
DNase-only
</td>
<td style="text-align:right;">
2580
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0308398
</td>
</tr>
<tr>
<td style="text-align:left;">
Low-DNase
</td>
<td style="text-align:right;">
97871
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
1.1698941
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS
</td>
<td style="text-align:right;">
11342
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.1355758
</td>
</tr>
<tr>
<td style="text-align:left;">
PLS,CTCF-bound
</td>
<td style="text-align:right;">
3133
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0374501
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS
</td>
<td style="text-align:right;">
6361
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0760358
</td>
</tr>
<tr>
<td style="text-align:left;">
dELS,CTCF-bound
</td>
<td style="text-align:right;">
1698
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0202969
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS
</td>
<td style="text-align:right;">
12562
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.1501590
</td>
</tr>
<tr>
<td style="text-align:left;">
pELS,CTCF-bound
</td>
<td style="text-align:right;">
2917
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
0.0348682
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>NR_cREs %&gt;%
  as.data.frame() %&gt;%
  dplyr::filter(blockCount==&quot;CTCF-only,CTCF-bound&quot;|blockCount ==&quot;PLS&quot;|blockCount ==&quot;PLS,CTCF-bound&quot;|blockCount ==&quot;dELS&quot;|blockCount ==&quot;dELS,CTCF-bound&quot;|blockCount ==&quot;pELS&quot;|blockCount ==&quot;pELS,CTCF-bound&quot;) %&gt;% 
  mutate(type=if_else(blockCount==&quot;CTCF-only,CTCF-bound&quot;,&quot;CTCF-only&quot;,
                      if_else(grepl(&quot;PLS&quot;,blockCount),&quot;Promoter-like&quot;,
                              if_else(grepl(&quot;pELS&quot;,blockCount),&quot;Proximal enhancer-like&quot;, &quot;Distal enhancer-like&quot;),blockCount))) %&gt;% 
  left_join(., toplist_DOX24_ATAC_nop, by =c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(logFC,peakid,type) %&gt;% 
  mutate(direction=if_else(logFC&gt;0,&quot;open&quot;,&quot;closed&quot;)) %&gt;% 
  mutate(type=factor(type, levels = c(&quot;Promoter-like&quot;,&quot;Proximal enhancer-like&quot;,&quot;Distal enhancer-like&quot;,&quot;CTCF-only&quot;))) %&gt;%
  mutate(direction=factor(direction, levels=c(&quot;open&quot;,&quot;closed&quot;))) %&gt;% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(&quot;NR&quot;)+
  # facet_wrap(~direction)+
  theme_bw()</code></pre>
<p><img src="figure/Enhancer_files.Rmd/enhacerstuffforlater%20NR-1.png" width="672" style="display: block; margin: auto;" /></p>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th>Cluster</th>
<th>eREs</th>
<th>not cREs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>NR</td>
<td>69200</td>
<td>14458</td>
</tr>
<tr class="even">
<td>LR</td>
<td>32847</td>
<td>8912</td>
</tr>
<tr class="odd">
<td>ESR</td>
<td>12020</td>
<td>3645</td>
</tr>
<tr class="even">
<td>EAR</td>
<td>5820</td>
<td>1501</td>
</tr>
</tbody>
</table>
<pre class="r"><code># EAR_20k_peaks_gr &lt;- GRanges(EAR_peak_list_20k)
# ESR_20k_peaks_gr &lt;- GRanges(ESR_peak_list_20k)
# LR_20k_peaks_gr &lt;- GRanges(LR_peak_list_20k)
# NR_20k_peaks_gr &lt;- GRanges(NR_peak_list_20k)
# 
# EAR_enh_peaks_20k &lt;- as.data.frame(subsetByOverlaps(EAR_20k_peaks_gr,enhancers_HLV_46F))
# ESR_enh_peaks_20k &lt;- as.data.frame(subsetByOverlaps(ESR_20k_peaks_gr,enhancers_HLV_46F))
# LR_enh_peaks_20k &lt;- as.data.frame(subsetByOverlaps(LR_20k_peaks_gr,enhancers_HLV_46F))
# NR_enh_peaks_20k &lt;- as.data.frame(subsetByOverlaps(NR_20k_peaks_gr,enhancers_HLV_46F))
# saveRDS(EAR_enh_peaks_20k,&quot;data/enhancerdata/EAR_enh_peaks_20k.RDS&quot;)
# saveRDS(ESR_enh_peaks_20k,&quot;data/enhancerdata/ESR_enh_peaks_20k.RDS&quot;)
# saveRDS(LR_enh_peaks_20k,&quot;data/enhancerdata/LR_enh_peaks_20k.RDS&quot;)
# saveRDS(NR_enh_peaks_20k,&quot;data/enhancerdata/NR_enh_peaks_20k.RDS&quot;)
### making a set that is with all peaks in motif
# EAR_enh_peaks_20k &lt;- subsetByOverlaps(NR_df_gr,enhancers_HLV_46F)</code></pre>
</div>
<div id="dar-cre-venns" class="section level3">
<h3>DAR cRE venns</h3>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(DOX_3_peak_list$peakid, 
                                  EPI_3_peak_list$peakid,  
                                  DNR_3_peak_list$peakid, 
                                  MTX_3_peak_list$peakid),
                             category.names = c(&quot;DOX peaks\n3 hour&quot;,&quot;EPI peaks\n3 hour&quot;,&quot;DNR peaks\n3 hour&quot;,&quot;MTX peaks\n3 hour&quot;),
              show_intersect = FALSE,
              set_color = &quot;black&quot;,
              label = &quot;count&quot;,
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = &quot;black&quot;,
              edge_lty = &quot;solid&quot;, set_size = 4)</code></pre>
<p><img src="figure/Enhancer_files.Rmd/DAR%20venns%203hr-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(DOX_3_peak_list$ensembl_ID), 
                                  unique(EPI_3_peak_list$ensembl_ID),  
                                  unique(DNR_3_peak_list$ensembl_ID), 
                                  unique(MTX_3_peak_list$ensembl_ID)),
                             category.names = c(&quot;DOX_NG\n3 hour&quot;,&quot;EPI_NG\n3 hour&quot;,&quot;DNR_NG\n3 hour&quot;,&quot;MTX_NG\n3 hour&quot;),
              show_intersect = FALSE,
              set_color = &quot;black&quot;,
              label = &quot;count&quot;,
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = &quot;black&quot;,
              edge_lty = &quot;solid&quot;, set_size = 4)</code></pre>
<p><img src="figure/Enhancer_files.Rmd/DAR%20venns%203hr-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># enhancerdf_DAR_3hr &lt;- data.frame(&quot;DAR&quot;=c(&quot;DOX_3&quot;, &quot;EPI_3&quot;,&quot;DNR_3&quot;,&quot;MTX_3&quot;), 
#               &quot;cRE_yes&quot;=c(length(findOverlaps(enhancers_HLV_46F,DOX_3hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,EPI_3hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,DNR_3hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,MTX_3hr_gr))),
#               &quot;cRE_no&quot;=c(length(DOX_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DOX_3hr_gr)),
#                        length(EPI_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,EPI_3hr_gr)), 
#                        length(DNR_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DNR_3hr_gr)),
#                        length(MTX_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,MTX_3hr_gr))))</code></pre>
<div id="dar-24-hour" class="section level4">
<h4>DAR 24 hour</h4>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(DOX_24_peak_list$peakid, 
                                  EPI_24_peak_list$peakid,  
                                  DNR_24_peak_list$peakid, 
                                  MTX_24_peak_list$peakid),
                             category.names = c(&quot;DOX peaks&quot;,&quot;EPI peaks&quot;,&quot;DNR peaks&quot;,&quot;MTX peaks&quot;),
              show_intersect = FALSE,
              set_color = &quot;black&quot;,
              label = &quot;count&quot;,
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = &quot;black&quot;,
              edge_lty = &quot;solid&quot;, set_size = 4)</code></pre>
<p><img src="figure/Enhancer_files.Rmd/DAR%20venns%2024hr-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(unique(DOX_24_peak_list$ensembl_ID), 
                                  unique(EPI_24_peak_list$ensembl_ID),  
                                  unique(DNR_24_peak_list$ensembl_ID), 
                                  unique(MTX_24_peak_list$ensembl_ID)),
                             category.names = c(&quot;DOX_NG&quot;,&quot;EPI_NG&quot;,&quot;DNR_NG&quot;,&quot;MTX_NG&quot;),
              show_intersect = FALSE,
              set_color = &quot;black&quot;,
              label = &quot;count&quot;,
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = &quot;black&quot;,
              edge_lty = &quot;solid&quot;, set_size = 4)</code></pre>
<p><img src="figure/Enhancer_files.Rmd/DAR%20venns%2024hr-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code># enhancerdf_DAR_24hr &lt;- data.frame(&quot;DAR&quot;=c(&quot;DOX_24&quot;, &quot;EPI_24&quot;,&quot;DNR_24&quot;,&quot;MTX_24&quot;), 
#               &quot;cRE_yes&quot;=c(length(findOverlaps(enhancers_HLV_46F,DOX_24hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,EPI_24hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,DNR_24hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,MTX_24hr_gr))),
#               &quot;cRE_no&quot;=c(length(DOX_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DOX_24hr_gr)),
#                        length(EPI_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,EPI_24hr_gr)), 
#                        length(DNR_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DNR_24hr_gr)),
#                        length(MTX_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,MTX_24hr_gr))))</code></pre>
</div>
</div>
<div id="enhancers-and-dar-24" class="section level3">
<h3>Enhancers and DAR 24</h3>
<pre class="r"><code># testdf &lt;- enhancerdf_DAR_24hr %&gt;% 
#   column_to_rownames(&quot;DAR&quot;) %&gt;% 
#   as.matrix() 
# paste(&quot;Late Response chi&quot;)
# chisq.test(testdf[c(1,2),])
# paste(&quot;Early sustained Response chi&quot;)
# chisq.test(testdf[c(1,3),])
# paste(&quot;Early-acute Response chi&quot;)
# chisq.test(testdf[c(1,4),])
# enhancerdf_DAR_3hr %&gt;% 
#       pivot_longer(., cols=c(cRE_yes,cRE_no), names_to = &quot;Enh_status&quot;, values_to = &quot;count&quot;) %&gt;% 
#   mutate(DAR=factor(DAR, levels = c(&quot;DOX_3&quot;,&quot;EPI_3&quot;, &quot;DNR_3&quot;,&quot;MTX_3&quot;))) %&gt;% 
#   ggplot(., aes(x = DAR, y= count ))+
#     geom_bar(position =&quot;fill&quot;, stat=&quot;identity&quot;, aes(fill=Enh_status))
# enhancerdf_DAR_24hr %&gt;% 
#       pivot_longer(., cols=c(cRE_yes,cRE_no), names_to = &quot;Enh_status&quot;, values_to = &quot;count&quot;) %&gt;% 
#   mutate(DAR=factor(DAR, levels = c(&quot;DOX_24&quot;,&quot;EPI_24&quot;, &quot;DNR_24&quot;,&quot;MTX_24&quot;))) %&gt;% 
#   ggplot(., aes(x = DAR, y= count ))+
#     geom_bar(position =&quot;fill&quot;, stat=&quot;identity&quot;, aes(fill=Enh_status))</code></pre>
<div id="noodling-around" class="section level6">
<h6>noodling around</h6>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] genomation_1.34.0                       
 [2] plyranges_1.22.0                        
 [3] BSgenome.Hsapiens.UCSC.hg38_1.4.5       
 [4] BSgenome_1.70.2                         
 [5] BiocIO_1.12.0                           
 [6] MotifDb_1.44.0                          
 [7] Biostrings_2.70.3                       
 [8] XVector_0.42.0                          
 [9] TFBSTools_1.40.0                        
[10] JASPAR2022_0.99.8                       
[11] BiocFileCache_2.10.2                    
[12] dbplyr_2.5.0                            
[13] bedr_1.0.7                              
[14] devtools_2.4.5                          
[15] usethis_2.2.3                           
[16] ggpubr_0.6.0                            
[17] BiocParallel_1.36.0                     
[18] Cormotif_1.48.0                         
[19] affy_1.80.0                             
[20] scales_1.3.0                            
[21] VennDiagram_1.7.3                       
[22] futile.logger_1.4.3                     
[23] gridExtra_2.3                           
[24] ggfortify_0.4.17                        
[25] edgeR_4.0.16                            
[26] limma_3.58.1                            
[27] rtracklayer_1.62.0                      
[28] org.Hs.eg.db_3.18.0                     
[29] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[30] GenomicFeatures_1.54.4                  
[31] AnnotationDbi_1.64.1                    
[32] Biobase_2.62.0                          
[33] GenomicRanges_1.54.1                    
[34] GenomeInfoDb_1.38.8                     
[35] IRanges_2.36.0                          
[36] S4Vectors_0.40.2                        
[37] BiocGenerics_0.48.1                     
[38] ChIPseeker_1.38.0                       
[39] RColorBrewer_1.1-3                      
[40] broom_1.0.6                             
[41] kableExtra_1.4.0                        
[42] lubridate_1.9.3                         
[43] forcats_1.0.0                           
[44] stringr_1.5.1                           
[45] dplyr_1.1.4                             
[46] purrr_1.0.2                             
[47] readr_2.1.5                             
[48] tidyr_1.3.1                             
[49] tibble_3.2.1                            
[50] ggplot2_3.5.1                           
[51] tidyverse_2.0.0                         
[52] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] R.methodsS3_1.8.2                      
  [2] vroom_1.6.5                            
  [3] progress_1.2.3                         
  [4] urlchecker_1.0.1                       
  [5] poweRlaw_0.80.0                        
  [6] vctrs_0.6.5                            
  [7] digest_0.6.35                          
  [8] png_0.1-8                              
  [9] git2r_0.33.0                           
 [10] ggrepel_0.9.5                          
 [11] MASS_7.3-60.0.1                        
 [12] reshape2_1.4.4                         
 [13] httpuv_1.6.15                          
 [14] qvalue_2.34.0                          
 [15] withr_3.0.0                            
 [16] xfun_0.44                              
 [17] ggfun_0.1.5                            
 [18] ellipsis_0.3.2                         
 [19] memoise_2.0.1                          
 [20] profvis_0.3.8                          
 [21] systemfonts_1.1.0                      
 [22] tidytree_0.4.6                         
 [23] gtools_3.9.5                           
 [24] R.oo_1.26.0                            
 [25] prettyunits_1.2.0                      
 [26] KEGGREST_1.42.0                        
 [27] promises_1.3.0                         
 [28] httr_1.4.7                             
 [29] rstatix_0.7.2                          
 [30] restfulr_0.0.15                        
 [31] ps_1.7.6                               
 [32] rstudioapi_0.16.0                      
 [33] ggVennDiagram_1.5.2                    
 [34] miniUI_0.1.1.1                         
 [35] generics_0.1.3                         
 [36] DOSE_3.28.2                            
 [37] processx_3.8.4                         
 [38] curl_5.2.1                             
 [39] zlibbioc_1.48.2                        
 [40] ggraph_2.2.1                           
 [41] polyclip_1.10-6                        
 [42] GenomeInfoDbData_1.2.11                
 [43] SparseArray_1.2.4                      
 [44] interactiveDisplayBase_1.40.0          
 [45] xtable_1.8-4                           
 [46] pracma_2.4.4                           
 [47] evaluate_0.23                          
 [48] S4Arrays_1.2.1                         
 [49] preprocessCore_1.64.0                  
 [50] hms_1.1.3                              
 [51] colorspace_2.1-0                       
 [52] filelock_1.0.3                         
 [53] magrittr_2.0.3                         
 [54] later_1.3.2                            
 [55] viridis_0.6.5                          
 [56] ggtree_3.10.1                          
 [57] lattice_0.22-6                         
 [58] getPass_0.2-4                          
 [59] XML_3.99-0.16.1                        
 [60] shadowtext_0.1.3                       
 [61] cowplot_1.1.3                          
 [62] matrixStats_1.3.0                      
 [63] pillar_1.9.0                           
 [64] nlme_3.1-164                           
 [65] gridBase_0.4-7                         
 [66] caTools_1.18.2                         
 [67] compiler_4.3.1                         
 [68] stringi_1.8.4                          
 [69] SummarizedExperiment_1.32.0            
 [70] GenomicAlignments_1.38.2               
 [71] plyr_1.8.9                             
 [72] crayon_1.5.2                           
 [73] abind_1.4-5                            
 [74] gridGraphics_0.5-1                     
 [75] locfit_1.5-9.9                         
 [76] graphlayouts_1.1.1                     
 [77] bit_4.0.5                              
 [78] fastmatch_1.1-4                        
 [79] whisker_0.4.1                          
 [80] codetools_0.2-20                       
 [81] bslib_0.7.0                            
 [82] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [83] mime_0.12                              
 [84] splines_4.3.1                          
 [85] Rcpp_1.0.12                            
 [86] HDO.db_0.99.1                          
 [87] knitr_1.47                             
 [88] blob_1.2.4                             
 [89] utf8_1.2.4                             
 [90] BiocVersion_3.18.1                     
 [91] seqLogo_1.68.0                         
 [92] fs_1.6.4                               
 [93] pkgbuild_1.4.4                         
 [94] ggsignif_0.6.4                         
 [95] ggplotify_0.1.2                        
 [96] Matrix_1.6-5                           
 [97] callr_3.7.6                            
 [98] statmod_1.5.0                          
 [99] tzdb_0.4.0                             
[100] svglite_2.1.3                          
[101] tweenr_2.0.3                           
[102] pkgconfig_2.0.3                        
[103] tools_4.3.1                            
[104] cachem_1.1.0                           
[105] RSQLite_2.3.7                          
[106] viridisLite_0.4.2                      
[107] DBI_1.2.3                              
[108] splitstackshape_1.4.8                  
[109] impute_1.76.0                          
[110] fastmap_1.2.0                          
[111] rmarkdown_2.27                         
[112] Rsamtools_2.18.0                       
[113] AnnotationHub_3.10.1                   
[114] sass_0.4.9                             
[115] patchwork_1.2.0                        
[116] BiocManager_1.30.23                    
[117] carData_3.0-5                          
[118] farver_2.1.2                           
[119] tidygraph_1.3.1                        
[120] scatterpie_0.2.2                       
[121] yaml_2.3.8                             
[122] MatrixGenerics_1.14.0                  
[123] cli_3.6.2                              
[124] lifecycle_1.0.4                        
[125] lambda.r_1.2.4                         
[126] sessioninfo_1.2.2                      
[127] backports_1.5.0                        
[128] annotate_1.80.0                        
[129] timechange_0.3.0                       
[130] gtable_0.3.5                           
[131] rjson_0.2.21                           
[132] parallel_4.3.1                         
[133] ape_5.8                                
[134] testthat_3.2.1.1                       
[135] jsonlite_1.8.8                         
[136] bitops_1.0-7                           
[137] bit64_4.0.5                            
[138] brio_1.1.5                             
[139] yulab.utils_0.1.4                      
[140] CNEr_1.38.0                            
[141] futile.options_1.0.1                   
[142] highr_0.11                             
[143] jquerylib_0.1.4                        
[144] GOSemSim_2.28.1                        
[145] R.utils_2.12.3                         
[146] lazyeval_0.2.2                         
[147] shiny_1.8.1.1                          
[148] htmltools_0.5.8.1                      
[149] enrichplot_1.22.0                      
[150] GO.db_3.18.0                           
[151] rappdirs_0.3.3                         
[152] formatR_1.14                           
[153] glue_1.7.0                             
[154] TFMPvalue_0.0.9                        
[155] RCurl_1.98-1.14                        
[156] rprojroot_2.0.4                        
[157] treeio_1.26.0                          
[158] boot_1.3-30                            
[159] igraph_2.0.3                           
[160] R6_2.5.1                               
[161] gplots_3.1.3.1                         
[162] labeling_0.4.3                         
[163] pkgload_1.3.4                          
[164] aplot_0.2.2                            
[165] DirichletMultinomial_1.44.0            
[166] DelayedArray_0.28.0                    
[167] tidyselect_1.2.1                       
[168] plotrix_3.8-4                          
[169] ggforce_0.4.2                          
[170] xml2_1.3.6                             
[171] car_3.1-2                              
[172] seqPattern_1.34.0                      
[173] munsell_0.5.1                          
[174] KernSmooth_2.23-24                     
[175] affyio_1.72.0                          
[176] data.table_1.15.4                      
[177] htmlwidgets_1.6.4                      
[178] fgsea_1.28.0                           
[179] biomaRt_2.58.2                         
[180] rlang_1.1.3                            
[181] remotes_2.5.0                          
[182] fansi_1.0.6                            </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
