<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-06-13" />

<title>SNP Enrichment Investigation</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">SNP Enrichment Investigation</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-06-13</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-06-13
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree224ae6273fb1959ff35a129134f526d2266df8adtargetblank224ae62a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/224ae6273fb1959ff35a129134f526d2266df8ad" target="_blank">224ae62</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree224ae6273fb1959ff35a129134f526d2266df8adtargetblank224ae62a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/224ae6273fb1959ff35a129134f526d2266df8ad" target="_blank">224ae62</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/Fastqc_results.Rmd
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Enrichment_motif.Rmd</code>) and
HTML (<code>docs/Enrichment_motif.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/224ae6273fb1959ff35a129134f526d2266df8ad/analysis/Enrichment_motif.Rmd" target="_blank">224ae62</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-13
</td>
<td>
more data added
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5d5401708c50e87c3abf2b4c7e2a6b5acb142b07/analysis/Enrichment_motif.Rmd" target="_blank">5d54017</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-12
</td>
<td>
adding commit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/67585d0efbafcd48329be5d34361fe54c83b6f41/docs/Enrichment_motif.html" target="_blank">67585d0</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/eab563a7535328a7b52babd407d8df8567886df5/analysis/Enrichment_motif.Rmd" target="_blank">eab563a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-06
</td>
<td>
removing old tables from view
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/784236c6af7e0cd3932fc4af52ecd957ffb4899f/docs/Enrichment_motif.html" target="_blank">784236c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/e2995acb5ec379d8483682d84a5ccff9f841b874/analysis/Enrichment_motif.Rmd" target="_blank">e2995ac</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
<td>
updates to gwas tables again
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/029478119a221db3b22f4c9d91e3636c0dbdea22/docs/Enrichment_motif.html" target="_blank">0294781</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ba9574a6831472cd29921f26e167d97894b4835d/analysis/Enrichment_motif.Rmd" target="_blank">ba9574a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
<td>
updates to gwas tables
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/c8acdab20dd506ee97064d9ebe04b75fdce58738/docs/Enrichment_motif.html" target="_blank">c8acdab</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/77e036cea0ad9abc5e1d7c78f6755e7d3ab0ed66/analysis/Enrichment_motif.Rmd" target="_blank">77e036c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-03
</td>
<td>
updates to SNP adding distinct
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/445c02d2621e069f2eea464ad301a82ee583a0b5/docs/Enrichment_motif.html" target="_blank">445c02d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/2d3d0db1c7e98efaaae2c5a3bddbb2d48ebd3f08/analysis/Enrichment_motif.Rmd" target="_blank">2d3d0db</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-02
</td>
<td>
update to page
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9d1572c1c14c856794dc01a8e9a87674e72346b7/analysis/Enrichment_motif.Rmd" target="_blank">9d1572c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-02
</td>
<td>
adding in SNP annotations with GWAS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/ee638023f02994f959e6d10d75bb1079ffae3558/docs/Enrichment_motif.html" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/e5fbfa699c958963850692c7d233002a223962b3/analysis/Enrichment_motif.Rmd" target="_blank">e5fbfa6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
<td>
updates for Sayan
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/f9232573eec6d3684ab9beaa8e0ff482996ed626/docs/Enrichment_motif.html" target="_blank">f923257</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1512fd56745c24bc47bd0d9f8851b18120c0c5c5/analysis/Enrichment_motif.Rmd" target="_blank">1512fd5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
<td>
updates to enrichment checks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/fe8e83141e561c4015a385b1b47d80040f3a6384/docs/Enrichment_motif.html" target="_blank">fe8e831</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5a78968277061633774c9b48d2a9291b045bf782/analysis/Enrichment_motif.Rmd" target="_blank">5a78968</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
<td>
updates to tables
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/Enrichment_motif.html" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/d2273946cd5af59bcdd3cc16a893f4a8b7440259/analysis/Enrichment_motif.Rmd" target="_blank">d227394</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
<td>
adding in DE and MRC name changes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/16adf5b48db844434089da978b910b181bd5fcba/docs/Enrichment_motif.html" target="_blank">16adf5b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8bfb48f4dc9db3a9095a1c814ed2ffbff9fa7dc6/analysis/Enrichment_motif.Rmd" target="_blank">8bfb48f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
<td>
adding other info
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/2ddd597cc8c6de11b96b1cd15e59f3a73490cacf/docs/Enrichment_motif.html" target="_blank">2ddd597</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/244cfc0ea215043ba095fa3a3f6af876da837f27/analysis/Enrichment_motif.Rmd" target="_blank">244cfc0</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-15
</td>
<td>
Adding in no extension
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/5115c784fbe50ada8ceba26aa953838cbeb79174/docs/Enrichment_motif.html" target="_blank">5115c78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1cc42d79947bff39826b90c3f33bf9d7f50a784d/analysis/Enrichment_motif.Rmd" target="_blank">1cc42d7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
updates to code to fix broken string
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/8d44ffbe4c257ea851be5795189adfaac98ee86a/docs/Enrichment_motif.html" target="_blank">8d44ffb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/caecf0befbdf2efe22332b08aad7bf561cf41c27/analysis/Enrichment_motif.Rmd" target="_blank">caecf0b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
adding new page
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
# library(bedr)
library(biomaRt)
library(plyranges)
library(genomation)</code></pre>
<div id="knowles-supp-data-check" class="section level3">
<h3>Knowles supp data check</h3>
<pre class="r"><code>Knowles_4&lt;- readRDS(&quot;data/Knowles_4.RDS&quot;)
Knowles_5 &lt;- readRDS(&quot;data/Knowles_5.RDS&quot;)
Knowles_6 &lt;- readRDS(&quot;data/Knowles_6.RDS&quot;)
SNP_supp_schneider &lt;- readRDS(&quot;data/SNP_supp_schneider.RDS&quot;)
peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)
list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)</code></pre>
<pre><code>&lt;environment: R_GlobalEnv&gt;</code></pre>
<pre class="r"><code>###How I made the granges files for overlaps
Knowles_4_noext &lt;- Knowles_4 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()


Knowles_4 &lt;- Knowles_4 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-500, end=pos+500) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()


# 
Knowles_5_noext &lt;- Knowles_5 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()

Knowles_5 &lt;- Knowles_5 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-500, end=pos+500) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()

Knowles_6_noext &lt;- Knowles_6 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, gene,pos) %&gt;%
  GRanges()
 
 Knowles_6 &lt;- Knowles_6 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-500, end=pos+500) %&gt;%
  dplyr::select(chr, start,end, gene,pos) %&gt;%
  GRanges()
 
 
SNP_supp_schneider &lt;-  SNP_supp_schneider %&gt;%
    na.omit() %&gt;%
     rename(X1=&quot;chr&quot;,X2= &quot;pos&quot;,X4=&quot;RSID&quot;) %&gt;%
    # rename(&quot;chr&quot;=X1, &quot;pos&quot; =X2,&quot;RSID&quot; =X4) %&gt;%
    mutate(start=pos-500, end=pos+500) %&gt;%
   dplyr::select(chr, start,end, RSID,pos) %&gt;%
    GRanges()

##split into granges objects
###make dataframe of each EAR,ESR,LR,NR promoters
# background_df &lt;- as.data.frame(background_gr)
# background_Gr &lt;- GRanges(background_df)
#  rtracklayer::export.bed(background_Gr, con=&quot;data/n45_bedfiles/background_n45.bed&quot;, format=&quot;bed&quot;)
EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_promotor &lt;- EAR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
EAR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)

# rtracklayer::export.bed(EAR_promo_gr, con=&quot;data/n45_bedfiles/EAR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_promotor &lt;- ESR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
ESR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)
 # rtracklayer::export.bed(ESR_promo_gr, con=&quot;data/n45_bedfiles/ESR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
 
LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_promotor &lt;- LR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
LR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)
 # rtracklayer::export.bed(LR_promo_gr, con=&quot;data/n45_bedfiles/LR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
 # LR_promotor %&gt;% 
 #   dplyr::filter(geneChr==6)

NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_promotor &lt;- NR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
 NR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)
 
 ENCFF134HBK &lt;- read_delim(&quot;data/n45_bedfiles/ENCFF134HBK.bed&quot;, 
     delim = &quot;\t&quot;, escape_double = FALSE, 
     col_names = FALSE, trim_ws = TRUE)
 ZNF384_chip &lt;- ENCFF134HBK %&gt;% 
   rename(X1=&quot;chr&quot;,
         X2=&quot;start&quot;,
         X3=&quot;end&quot;,
         X10=&quot;length&quot;) %&gt;%
   GRanges()</code></pre>
</div>
<div id="checking-for-enrichment-k4-k6-cardiotox"
class="section level3">
<h3>Checking for enrichment K4-K6 cardiotox:</h3>
<p>I have made 500 bp +/- files for each of the data sets using the eQTL
or SNP location. I also made a 1bp no extension file for each of the
data sets using the eQTL or SNP location as start and end as +1.</p>
<p>report:</p>
<table>
<caption>number of intersections between groups with 500 =/- bp around
SNP</caption>
<colgroup>
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>K_4(500)</th>
<th>K_5(359)</th>
<th>K_6(447)</th>
<th>schn_g(112)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>6</td>
<td>4</td>
<td>6</td>
<td>0</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>20</td>
<td>17</td>
<td>18</td>
<td>4</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>56</td>
<td>26</td>
<td>34</td>
<td>3</td>
</tr>
</tbody>
</table>
<p>report:</p>
<table>
<caption>number of intersections between groups without
extension</caption>
<thead>
<tr class="header">
<th>motif</th>
<th>K_4(500)</th>
<th>K_5(359)</th>
<th>K_6(447)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>1</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>3</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>13</td>
<td>9</td>
<td>8</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>30</td>
<td>15</td>
<td>19</td>
</tr>
</tbody>
</table>
<p>These numbers are different than the original lists from which they
derived. This is normally due to running <code>na.omit()</code> on the
data frame before GRanges transformation.<br />
</p>
</div>
<div id="gwas" class="section level3">
<h3>GWAS</h3>
<p>I downloaded 5 different trait associated lists from the GWAS
catalog</p>
<p>trait 1, 10 associations with trait <strong>cardiotoxicity</strong>
(from 10 rows) trait 2, 661 associations with trait <strong>cardiac
arrhythmia</strong> (from 1068 rows)</p>
<p>trait 3, 37 associations with trait <strong>response to
anthracycline-based chemotherapy</strong> (from 40 rows) trait 4, 4068
associations with trait <strong>heart disease</strong> (from 6139 rows)
trait 5, 398 associations with trait <strong>heart failure</strong>
(from (335 rows))</p>
<pre class="r"><code>##finding GWAS for Hearfailure: I used several traits from the GWAS database:
##1_cardiotoxicity  11 associations   gwas_1
gwas_1 &lt;- readRDS(&quot;data/gwas_1_dataframe.RDS&quot;)  

##2_ cardiac arrhythmia  1068 associations   gwas_2
gwas_2 &lt;- readRDS(&quot;data/gwas_2_dataframe.RDS&quot;)

##3_ Anthracycline_response   40 gwas_3
gwas_3 &lt;- readRDS(&quot;data/gwas_3_dataframe.RDS&quot;)

##4_ heart disease (biggest # of associations 6139) 
gwas_4 &lt;- readRDS(&quot;data/gwas_4_dataframe.RDS&quot;)

##5_ heart_failure gwas_5
gwas_5 &lt;- readRDS(&quot;data/gwas_5_dataframe.RDS&quot;)

##6 Coronary artery disease
CAD_gwas &lt;- readRDS( &quot;data/CAD_gwas_dataframe.RDS&quot;)

MI_gwas &lt;- readRDS(&quot;data/MI_gwas.RDS&quot;)

CAD_gwas_gr &lt;- CAD_gwas %&gt;%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  distinct() %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)
CAD_gwas_gr_ten &lt;- CAD_gwas %&gt;%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)

CAD_gwas_gr_noext &lt;- CAD_gwas %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;%
   distinct() %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)
 # rtracklayer::export.bed(CAD_gwas_gr_noext, con=&quot;data/n45_bedfiles/CAD_gwas_gr_noext.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)

MI_gwas_gr &lt;- MI_gwas %&gt;%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)
MI_gwas_gr_ten &lt;- MI_gwas %&gt;%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;%
   distinct() %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)

MI_gwas_gr_noext &lt;- MI_gwas %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)
 # rtracklayer::export.bed(MI_gwas_gr_noext , con=&quot;data/n45_bedfiles/MI_gwas_gr_noext.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)

gwas_1_gr &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_1_gr_ten &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
# rtracklayer::export.bed(gwas_1_gr , con=&quot;data/n45_bedfiles/cardiotox_gwas_gr_ext.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
gwas_1_gr_noext &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_2_gr &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
   mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()

gwas_2_gr_ten &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
   mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()

 
gwas_2_gr_noext &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()

# rtracklayer::export.bed(gwas_2_gr_noext , con=&quot;data/n45_bedfiles/ARR_gwas_gr_noext.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)

gwas_3_gr &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()


gwas_3_gr_ten &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_3_gr_noext &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_4_gr &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID))%&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
 
gwas_4_gr_ten &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID))%&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
gwas_4_gr_noext &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

# rtracklayer::export.bed(gwas_4_gr_noext , con=&quot;data/n45_bedfiles/HD_gwas_gr_noext.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)

gwas_5_gr &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
gwas_5_gr_ten &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()


gwas_5_gr_noext &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   distinct() %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
# saveRDS(list(&quot;gwas_2_gr_noext&quot;=gwas_2_gr_noext, &quot;gwas_4_gr_noext&quot;= gwas_4_gr_noext),&quot;data/test.list.RDS&quot;)
# gwas_2[(is.na(gwas_2$CHR_POS)),]

# rtracklayer::export.bed(gwas_5_gr_noext , con=&quot;data/n45_bedfiles/HF_LR_gwas_gr_noext.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)</code></pre>
<div id="gwas-overlap-findings" class="section level4">
<h4>GWAS overlap findings</h4>
<table style="width:100%;">
<caption>gwas trait enrichment with extensions of 5,000 bp +/- SNP
location</caption>
<colgroup>
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>gwas_1 (10)</th>
<th>ARR-gwas_2 (661)</th>
<th>gwas_3 (37)</th>
<th>gwas_4 (4068)</th>
<th>Heart Failure (398)</th>
<th>CAD_gwas (2276)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>40</td>
<td>2</td>
<td>200</td>
<td>16</td>
<td>106</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>1</td>
<td>104</td>
<td>4</td>
<td>410</td>
<td>46</td>
<td>202</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>4</td>
<td>204</td>
<td>11</td>
<td>1049</td>
<td>94</td>
<td>583</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>6</td>
<td>417</td>
<td>14</td>
<td>1932</td>
<td>177</td>
<td>1057</td>
</tr>
</tbody>
</table>
<table style="width:100%;">
<caption>gwas trait enrichment with extensions of 500 bp +/- SNP
location</caption>
<colgroup>
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
<col width="14%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>gwas_1 (10)</th>
<th>ARR-gwas_2 (661)</th>
<th>gwas_3 (37)</th>
<th>gwas_4 (4068)</th>
<th>Heart Failure (398)</th>
<th>CAD_gwas (2276)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>5</td>
<td>1</td>
<td>33</td>
<td>2</td>
<td>21</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>1</td>
<td>20</td>
<td>1</td>
<td>63</td>
<td>10</td>
<td>28</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>2</td>
<td>33</td>
<td>5</td>
<td>180</td>
<td>22</td>
<td>104</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>0</td>
<td>80</td>
<td>1</td>
<td>344</td>
<td>26</td>
<td>182</td>
</tr>
</tbody>
</table>
<table style="width:100%;">
<caption>gwas trait enrichment with no extension</caption>
<colgroup>
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>gwas_1 (10)</th>
<th>ARR-gwas_2 (661)</th>
<th>gwas_3 (37)</th>
<th>gwas_4 (4068)</th>
<th>Heart Failure (398)</th>
<th>CAD_gwas (2276)</th>
<th>MI_gwas (335)</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>2</td>
<td>0</td>
<td>12</td>
<td>0</td>
<td>21</td>
<td>1</td>
<td></td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>0</td>
<td>13</td>
<td>0</td>
<td>29</td>
<td>2</td>
<td>12</td>
<td>0</td>
<td></td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>2</td>
<td>18</td>
<td>4</td>
<td>99</td>
<td>15</td>
<td>59</td>
<td>10</td>
<td></td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>0</td>
<td>48</td>
<td>0</td>
<td>170</td>
<td>10</td>
<td>84</td>
<td>13</td>
<td></td>
</tr>
</tbody>
</table>
<p>These numbers are different than the original lists from which they
derived. This is normally due to running <code>na.omit()</code> for rows
missing name and location infromation and adding on the
<code>distinct</code> function to the data frame before GRanges
transformation. This reliably removes repeated SNPs. #</p>
</div>
<div id="chisquare-tests" class="section level4">
<h4>chisquare tests</h4>
<pre class="r"><code>chitest_gwas2_EAR &lt;- matrix(c(2,7319,48,83610), ncol=2, byrow=TRUE)
chitest_gwas2_ESR &lt;- matrix(c(13,15652,48,83610), ncol=2, byrow=TRUE)
chitest_gwas2_LR &lt;- matrix(c(18,41741,48,83610), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas2_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas2_EAR
X-squared = 0.62767, df = 1, p-value = 0.4282</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas2_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas2_ESR
X-squared = 1.0236, df = 1, p-value = 0.3117</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas2_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas2_LR
X-squared = 0.82444, df = 1, p-value = 0.3639</code></pre>
<pre class="r"><code>chitest_gwas4_EAR &lt;- matrix(c(12,7309,170,83488), ncol=2, byrow=TRUE)
chitest_gwas4_ESR &lt;- matrix(c(29,15636,170,83488), ncol=2, byrow=TRUE)
chitest_gwas4_LR &lt;- matrix(c(99,41660,170,83488), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas4_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas4_EAR
X-squared = 0.34246, df = 1, p-value = 0.5584</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas4_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas4_ESR
X-squared = 0.1348, df = 1, p-value = 0.7135</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas4_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas4_LR
X-squared = 1.3387, df = 1, p-value = 0.2473</code></pre>
<pre class="r"><code>chitest_gwas5_EAR &lt;- matrix(c(0,7321,10,83648), ncol=2, byrow=TRUE)
chitest_gwas5_ESR &lt;- matrix(c(2,15663,10,83648), ncol=2, byrow=TRUE)
chitest_gwas5_LR &lt;- matrix(c(15,41744,10,83648), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas5_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas5_EAR
X-squared = 0.12548, df = 1, p-value = 0.7232</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas5_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas5_ESR
X-squared = 2.5225e-23, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas5_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas5_LR
X-squared = 6.8709, df = 1, p-value = 0.008761</code></pre>
<pre class="r"><code>chitest_K4_EAR &lt;- matrix(c(0,7321,32,83626), ncol=2, byrow=TRUE)
chitest_K4_ESR &lt;- matrix(c(3,15662,32,83626), ncol=2, byrow=TRUE)
chitest_K4_LR &lt;- matrix(c(14,417545,32,83626), ncol=2, byrow=TRUE)
chisq.test(chitest_K4_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K4_EAR
X-squared = 1.8191, df = 1, p-value = 0.1774</code></pre>
<pre class="r"><code>chisq.test(chitest_K4_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K4_ESR
X-squared = 0.87801, df = 1, p-value = 0.3487</code></pre>
<pre class="r"><code>chisq.test(chitest_K4_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K4_LR
X-squared = 88.73, df = 1, p-value &lt; 2.2e-16</code></pre>
<pre class="r"><code>chitest_K5_EAR &lt;- matrix(c(1,7320,15,83643), ncol=2, byrow=TRUE)
chitest_K5_ESR &lt;- matrix(c(2,15663,15,83643), ncol=2, byrow=TRUE)
chitest_K5_LR &lt;- matrix(c(10,41749,15,83643), ncol=2, byrow=TRUE)
chisq.test(chitest_K5_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K5_EAR
X-squared = 1.5949e-23, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chisq.test(chitest_K5_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K5_ESR
X-squared = 0.014542, df = 1, p-value = 0.904</code></pre>
<pre class="r"><code>chisq.test(chitest_K5_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K5_LR
X-squared = 0.24911, df = 1, p-value = 0.6177</code></pre>
<pre class="r"><code>chitest_K6_EAR &lt;- matrix(c(0,7321,19,83639), ncol=2, byrow=TRUE)
chitest_K6_ESR &lt;- matrix(c(2,15663,19,83639), ncol=2, byrow=TRUE)
chitest_K6_LR &lt;- matrix(c(9,41750,19,83639), ncol=2, byrow=TRUE)
chisq.test(chitest_K6_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K6_EAR
X-squared = 0.75318, df = 1, p-value = 0.3855</code></pre>
<pre class="r"><code>chisq.test(chitest_K6_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K6_ESR
X-squared = 0.23644, df = 1, p-value = 0.6268</code></pre>
<pre class="r"><code>chisq.test(chitest_K6_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K6_LR
X-squared = 2.4408e-24, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chitest_CAD_EAR &lt;- matrix(c(21,7300,84,83574), ncol=2, byrow=TRUE)
chitest_CAD_ESR &lt;- matrix(c(12,15653,84,83574), ncol=2, byrow=TRUE)
chitest_CAD_LR &lt;- matrix(c(59,41700,84,83574), ncol=2, byrow=TRUE)
chisq.test(chitest_CAD_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_CAD_EAR
X-squared = 18.713, df = 1, p-value = 1.519e-05</code></pre>
<pre class="r"><code>chisq.test(chitest_CAD_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_CAD_ESR
X-squared = 0.54741, df = 1, p-value = 0.4594</code></pre>
<pre class="r"><code>chisq.test(chitest_CAD_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_CAD_LR
X-squared = 3.7359, df = 1, p-value = 0.05326</code></pre>
<pre class="r"><code>chitest_MI_EAR &lt;- matrix(c(1,7320,13,83645), ncol=2, byrow=TRUE)
chitest_MI_ESR &lt;- matrix(c(0,15665,13,83645), ncol=2, byrow=TRUE)
chitest_MI_LR &lt;- matrix(c(10,41749,13,83645), ncol=2, byrow=TRUE)
chisq.test(chitest_MI_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_MI_EAR
X-squared = 3.7957e-24, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chisq.test(chitest_MI_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_MI_ESR
X-squared = 1.392, df = 1, p-value = 0.2381</code></pre>
<pre class="r"><code>chisq.test(chitest_MI_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_MI_LR
X-squared = 0.66426, df = 1, p-value = 0.4151</code></pre>
<pre class="r"><code>chitest_HFresp &lt;- matrix(c(17,64728,10,83648),ncol=2, byrow = TRUE)
chisq.test(chitest_HFresp)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_HFresp
X-squared = 3.3563, df = 1, p-value = 0.06695</code></pre>
<pre class="r"><code>chitest_ARRresp &lt;- matrix(c(33,64712,48,83610),ncol=2, byrow = TRUE)
chisq.test(chitest_ARRresp)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_ARRresp
X-squared = 0.16977, df = 1, p-value = 0.6803</code></pre>
</div>
<div id="subsetting-overlaps" class="section level4">
<h4>subsetting overlaps</h4>
<pre class="r"><code>##sub-setting  peaks that overlap with at least 1 SNP in gwas2 noext

EAR_g2_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_2_gr_noext))
# subsetByOverlaps(gwas_2_gr_noext, ESR_df_gr)
ESR_g2_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_2_gr_noext))
LR_g2_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_2_gr_noext))
NR_g2_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_2_gr_noext))

EAR_g4_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_4_gr_noext))
ESR_g4_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_4_gr_noext))
LR_g4_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_4_gr_noext))
NR_g4_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_4_gr_noext))

EAR_g1_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_1_gr_noext))

ESR_g1_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_1_gr_noext))
LR_g1_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_1_gr_noext))
NR_g1_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_1_gr_noext))


EAR_g3_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_3_gr_noext))

ESR_g3_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_3_gr_noext))
LR_g3_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_3_gr_noext))
NR_g3_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_3_gr_noext))

EAR_g5_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_5_gr_noext))
ESR_g5_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_5_gr_noext))
LR_g5_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_5_gr_noext))
NR_g5_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_5_gr_noext))

EAR_MI_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, MI_gwas_gr))

ESR_MI_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, MI_gwas_gr))
LR_MI_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  MI_gwas_gr))
NR_MI_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, MI_gwas_gr))

EAR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, CAD_gwas_gr))

ESR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, CAD_gwas_gr))
LR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  CAD_gwas_gr))
NR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, CAD_gwas_gr))

###Now I can pull the geneID list and see if genes are DE from RNAseq data</code></pre>
</div>
</div>
<div id="rna-mrc-motif-response-cluster" class="section level3">
<h3>RNA MRC (motif response cluster)</h3>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
##Here my focus is to find nearest genes that are MRC and see if they are DEGs
toplist_n45 &lt;- readRDS(&quot;data/toplist_n45.RDS&quot;)

# toplist_n45 %&gt;% 
#   dplyr::filter(peak==&quot;chr4.173529475.173530537&quot;) %&gt;% 
#  ggplot(., aes(x=trt, y=logFC))+
#   geom_boxplot(aes(fill=trt))+
#   ggpubr::fill_palette(palette =drug_pal)+
#   guides(fill=guide_legend(title = &quot;Treatment&quot;))+
#   # facet_wrap(sigcount~time)+
#   theme_bw()+
#   xlab(&quot;&quot;)+
#   ggtitle(&quot;chr4.173529475.173530537 region of  HAND2-AS1&quot;)+
#   ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
#   theme_bw()+
#   facet_wrap(~time)+
#   theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
#         axis.title = element_text(size = 15, color = &quot;black&quot;),
#         # axis.ticks = element_line(linewidth = 1.5),
#         # axis.line = element_line(linewidth = 1.5),
#         strip.background = element_rect(fill = &quot;transparent&quot;),
#         axis.text.x = element_blank(),
#         strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))


toplist_n45 %&gt;% 
  dplyr::filter(peak==&quot;chr6.36678380.36679788&quot;) %&gt;%
 ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;chr6  36678380-36679788 CDKN1a&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/MRC%20and%20RNAseq-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-MRC-and-RNAseq-1">
Past versions of "MRC and RNAseq-1.png"
</button>
</p>
<div id="fig-no-spaces-MRC-and-RNAseq-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/MRC and RNAseq-1.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_n45 %&gt;% 
  dplyr::filter(peak %in% EAR_g2_sol$id) %&gt;% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;absolute lfc of MRCs in EAR in cardiac arrhythmia&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/MRC%20and%20RNAseq-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-MRC-and-RNAseq-2">
Past versions of "MRC and RNAseq-2.png"
</button>
</p>
<div id="fig-no-spaces-MRC-and-RNAseq-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/MRC and RNAseq-2.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_n45 %&gt;% 
  dplyr::filter(peak %in% ESR_g2_sol$id) %&gt;% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;absolute lfc of MRCs in ESR in cardiac arrhythmia&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/MRC%20and%20RNAseq-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-MRC-and-RNAseq-3">
Past versions of "MRC and RNAseq-3.png"
</button>
</p>
<div id="fig-no-spaces-MRC-and-RNAseq-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/MRC and RNAseq-3.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_n45 %&gt;% 
  dplyr::filter(peak %in% LR_g2_sol$id) %&gt;% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;absolute lfc of MRCs in LR in cardiac arrhythmia&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/MRC%20and%20RNAseq-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-MRC-and-RNAseq-4">
Past versions of "MRC and RNAseq-4.png"
</button>
</p>
<div id="fig-no-spaces-MRC-and-RNAseq-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/MRC and RNAseq-4.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pie-charts-of-the-snps" class="section level3">
<h3>Pie charts of the SNPs</h3>
<pre class="r"><code>Set &lt;- c(&quot;EAR&quot;, &quot;ESR&quot;,&quot;LR&quot;, &quot;NR&quot;)
gene_num_ARR &lt;- c(2,13,18,48)
gene_num_HD &lt;- c(12,29,99,170)
gene_num_CAD &lt;- c(21,12,59,84)
gene_num_MI &lt;- c(1,0,10,13)
gene_num_HF &lt;- c(0,2,15,10)
gene_num_CTX &lt;- c(0,0,2,0)
gene_num_ACresp &lt;- c(0,0,4,0)

fills &lt;- c(&quot;#F8766D&quot;,  &quot;#00BFC4&quot;,&quot;#7CAE00&quot;, &quot;#C77CFF&quot;)

pie_chartdata_ARR &lt;- data.frame(Set, gene_num_ARR, fills)
pie_chartdata_HD &lt;- data.frame(Set, gene_num_HD, fills)
pie_chartdata_CAD &lt;- data.frame(Set, gene_num_CAD, fills)
pie_chartdata_MI &lt;- data.frame(Set, gene_num_MI, fills)
pie_chartdata_HF &lt;- data.frame(Set, gene_num_HF, fills)
pie_chartdata_CTX &lt;- data.frame(Set, gene_num_CTX, fills)
pie_chartdata_ACresp &lt;- data.frame(Set, gene_num_ACresp, fills)

 
    # pie(gene_num,labels =Set )      
pie_chartdata_ARR &lt;- pie_chartdata_ARR %&gt;% 
   mutate(prop = gene_num_ARR / sum(pie_chartdata_ARR$gene_num_ARR) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_HD &lt;- pie_chartdata_HD %&gt;% 
   mutate(prop = gene_num_HD / sum(pie_chartdata_HD$gene_num_HD) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_CAD &lt;- pie_chartdata_CAD %&gt;% 
   mutate(prop = gene_num_CAD / sum(pie_chartdata_CAD$gene_num_CAD) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_MI &lt;- pie_chartdata_MI %&gt;% 
   mutate(prop = gene_num_MI / sum(pie_chartdata_MI$gene_num_MI) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_HF &lt;- pie_chartdata_HF %&gt;% 
   mutate(prop = gene_num_HF / sum(pie_chartdata_HF$gene_num_HF) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_CTX &lt;- pie_chartdata_CTX %&gt;% 
   mutate(prop = gene_num_CTX / sum(pie_chartdata_CTX$gene_num_CTX) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_ACresp &lt;- pie_chartdata_ACresp %&gt;% 
   mutate(prop = gene_num_ACresp / sum(pie_chartdata_ACresp$gene_num_ACresp) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )





pie_chartdata_ARR %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_ARR, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of cardiac arrhythmia SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_ARR,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-piechart-noext-1">
Past versions of "piechart noext-1.png"
</button>
</p>
<div id="fig-no-spaces-piechart-noext-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/029478119a221db3b22f4c9d91e3636c0dbdea22/docs/figure/Enrichment_motif.Rmd/piechart noext-1.png" target="_blank">0294781</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/piechart noext-1.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/16adf5b48db844434089da978b910b181bd5fcba/docs/figure/Enrichment_motif.Rmd/piechart noext-1.png" target="_blank">16adf5b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pie_chartdata_HD %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_HD, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of heart disease SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_HD,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-piechart-noext-2">
Past versions of "piechart noext-2.png"
</button>
</p>
<div id="fig-no-spaces-piechart-noext-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/029478119a221db3b22f4c9d91e3636c0dbdea22/docs/figure/Enrichment_motif.Rmd/piechart noext-2.png" target="_blank">0294781</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/piechart noext-2.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/16adf5b48db844434089da978b910b181bd5fcba/docs/figure/Enrichment_motif.Rmd/piechart noext-2.png" target="_blank">16adf5b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pie_chartdata_CAD %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_CAD, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of coronary artery disease SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_CAD,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-piechart-noext-3">
Past versions of "piechart noext-3.png"
</button>
</p>
<div id="fig-no-spaces-piechart-noext-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/029478119a221db3b22f4c9d91e3636c0dbdea22/docs/figure/Enrichment_motif.Rmd/piechart noext-3.png" target="_blank">0294781</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/piechart noext-3.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/16adf5b48db844434089da978b910b181bd5fcba/docs/figure/Enrichment_motif.Rmd/piechart noext-3.png" target="_blank">16adf5b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pie_chartdata_MI %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_MI, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of myocardial infarction SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_MI,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-piechart-noext-4">
Past versions of "piechart noext-4.png"
</button>
</p>
<div id="fig-no-spaces-piechart-noext-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/029478119a221db3b22f4c9d91e3636c0dbdea22/docs/figure/Enrichment_motif.Rmd/piechart noext-4.png" target="_blank">0294781</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/piechart noext-4.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/16adf5b48db844434089da978b910b181bd5fcba/docs/figure/Enrichment_motif.Rmd/piechart noext-4.png" target="_blank">16adf5b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pie_chartdata_HF %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_HF, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of Heart failure SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_HF,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-piechart-noext-5">
Past versions of "piechart noext-5.png"
</button>
</p>
<div id="fig-no-spaces-piechart-noext-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/029478119a221db3b22f4c9d91e3636c0dbdea22/docs/figure/Enrichment_motif.Rmd/piechart noext-5.png" target="_blank">0294781</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-05
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/445c02d2621e069f2eea464ad301a82ee583a0b5/docs/figure/Enrichment_motif.Rmd/piechart noext-5.png" target="_blank">445c02d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pie_chartdata_CTX %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_CTX, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of Cardiotoxicity SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_CTX,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>pie_chartdata_ACresp %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_ACresp, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of response to AC-based chemotherapy  SNPs in MRCs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_ACresp,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>### Code to express # of DE vs non DE genes associated with SNPs found in MRCs

# toplist_n45 &lt;- readRDS(&quot;data/toplist_n45.RDS&quot;)
# 
# toplist_n45 %&gt;% 
#   mutate(DAR= if_else(adj.P.Val&lt;0.05,&quot;DAR&quot;,&quot;notDAR&quot;)) 
         


GWAS_2_DAR_df &lt;-
  EAR_g2_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
ESR_g2_sol %&gt;% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
LR_g2_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
NR_g2_sol %&gt;%
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;NR_ATAC&quot;))%&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))
         
    
GWAS_4_DAR_df &lt;- EAR_g4_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
  ESR_g4_sol %&gt;% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
    mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
    LR_g4_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
    NR_g4_sol %&gt;%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;NR_ATAC&quot;))%&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))



GWAS_1_DAR_df &lt;- EAR_g1_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
  ESR_g1_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
    mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
    LR_g1_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
    NR_g1_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;NR_ATAC&quot;))%&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))

GWAS_3_DAR_df &lt;- EAR_g3_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
  ESR_g3_sol %&gt;% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
    mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
    LR_g3_sol %&gt;%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
    NR_g3_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;NR_ATAC&quot;))%&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))

GWAS_5_DAR_df &lt;- EAR_g5_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
  ESR_g5_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
    mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
    LR_g5_sol %&gt;%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
    NR_g5_sol %&gt;%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;NR_ATAC&quot;))%&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))



GWAS_MI_DAR_df &lt;- EAR_MI_sol %&gt;%
 mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
  ESR_MI_sol %&gt;% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
    mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
    LR_MI_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
    NR_MI_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      mutate(ATAC_MRC=&quot;NR_ATAC&quot;))%&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))




GWAS_CAD_DAR_df &lt;-
  EAR_CAD_sol %&gt;%
    mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
                        # if_else(geneId %in% 
                        #           intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
                        #         if_else(geneId %in% 
                        #                   intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),&quot;DE&quot;,
                        #                 if_else(geneId %in% 
                        #                           intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(ATAC_MRC=&quot;EAR_ATAC&quot;) %&gt;% 
bind_rows(
  ESR_CAD_sol %&gt;% 
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
     mutate(ATAC_MRC=&quot;ESR_ATAC&quot;)) %&gt;% 
  bind_rows(
    LR_CAD_sol %&gt;%
       mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      # mutate(DE_g2= if_else(geneId %in% 
      #                         intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),&quot;DE&quot;,
      #                   if_else(geneId %in% 
      #                             intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
      #                           if_else(geneId %in% 
      #                                   intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),&quot;DE&quot;,
      #                                   if_else(geneId %in% 
      #                                             intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(ATAC_MRC=&quot;LR_ATAC&quot;)) %&gt;% 
  bind_rows(
    NR_CAD_sol %&gt;%
       mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;% 
      # mutate(DE_g2= if_else(geneId %in% 
      #                         intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),&quot;DE&quot;,
      #                   if_else(geneId %in% 
      #                             intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
      #                           if_else(geneId %in% 
      #                                     intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),&quot;DE&quot;,
      #                                   if_else(geneId %in% 
      #                                             intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), &quot;notDE_ATAC&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(ATAC_MRC=&quot;NR_ATAC&quot;)) %&gt;% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,&quot;EAR_RNA&quot;,
                          if_else(geneId  %in% ESR_RNA$ENTREZID,&quot;ESR_RNA&quot;,
                                  if_else(geneId %in% LR_RNA$ENTREZID,&quot;LR_RNA&quot;,
         if_else(geneId %in% NR_RNA$ENTREZID, &quot;NR_RNA&quot;,&quot;non-MRC_RNA&quot;)))))</code></pre>
</div>
<div id="arr-hd-hf-mi-cad-snp-overlaps-and-mrc-status"
class="section level2">
<h2>ARR, HD, HF, MI, CAD snp overlaps and MRC status</h2>
<pre class="r"><code>GWAS_1_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;)%&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS) %&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with Cardiotoxicity SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)

GWAS_3_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;)%&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS) %&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with response to AC-based chemo SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)


GWAS_5_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;)%&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS) %&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with heart disease SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)




GWAS_2_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;)%&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS) %&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with cardiac arrhythmia SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)

GWAS_4_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;)%&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC,RNA_MRC,annotation:distanceToTSS)%&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with heart disease SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)

GWAS_MI_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;)%&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS)%&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with myocardial infarction SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)

GWAS_CAD_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;) %&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS)%&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with coronary artery disease SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)


GWAS_5_DAR_df %&gt;% 
  # dplyr::filter(DE_g2==&quot;DE&quot;) %&gt;% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS)%&gt;% 
  kable(.,caption=&quot;ATAC peaks that overlap with Heart SNPs, annotated with RNA MRC and DE status&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)</code></pre>
<pre class="r"><code>my_hc_filtered_counts_n45 &lt;- readRDS(&quot;data/my_hc_filt_counts_n45.RDS&quot;)
log_filt_hc_n45 &lt;- cpm(my_hc_filtered_counts_n45, log = TRUE) %&gt;% as.data.frame()
drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

log_filt_hc_n45 %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(row.names(.)==&quot;chr6.36678380.36679788&quot;) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm in peak with SNP CDKN1a&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/CDKn1a%20log2cpm%20MRC-RNa-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-CDKn1a-log2cpm-MRC-RNa-1">
Past versions of "CDKn1a log2cpm MRC-RNa-1.png"
</button>
</p>
<div id="fig-no-spaces-CDKn1a-log2cpm-MRC-RNa-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/CDKn1a log2cpm MRC-RNa-1.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Counts_RNA_ERMatthews &lt;- readRDS(&quot;data/other_papers/Counts_RNA_ERMatthews.RDS&quot;)
# Counts_RNA_ERMatthews &lt;-  
Counts_RNA_ERMatthews %&gt;% 
  column_to_rownames(&quot;ENTREZID&quot;) %&gt;% 
  cpm(.,log = TRUE) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column(&quot;expr&quot;) %&gt;%
  filter(expr == 1026) %&gt;% 
  pivot_longer(cols =!expr , names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;trt&quot;,&quot;indv&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm RNA in ESR CDKN1a&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/CDKn1a%20log2cpm%20MRC-RNa-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-CDKn1a-log2cpm-MRC-RNa-2">
Past versions of "CDKn1a log2cpm MRC-RNa-2.png"
</button>
</p>
<div id="fig-no-spaces-CDKn1a-log2cpm-MRC-RNa-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4e9ea56984d0374571fcaf591676a5f79fb68d09/docs/figure/Enrichment_motif.Rmd/CDKn1a log2cpm MRC-RNa-2.png" target="_blank">4e9ea56</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<!-- ### KEGG pathway enrichment by peaks location -->
<pre class="r"><code># library(clusterProfiler)
# genes = lapply(peakAnnoList_n45_motif[1:4], function(i) as.data.frame(i)$geneId)
# names(genes) = sub(&quot;_&quot;, &quot;\n&quot;, names(genes))
# compKEGG &lt;- compareCluster(geneCluster   = genes,
#                          fun           = &quot;enrichGO&quot;,
#                          pvalueCutoff  = 0.05,
#                          OrgDb=&#39;org.Hs.eg.db&#39;,
#                          pAdjustMethod = &quot;BH&quot;)
# dotplot(compKEGG, showCategory = 15, title = &quot;KEGG Pathway Enrichment Analysis&quot;)</code></pre>
<div id="mapping-peaks-to-nearest-rna-expressed-genes"
class="section level3">
<h3>Mapping peaks to nearest RNA-expressed genes</h3>
<p>The code below is the documentation of what I did to make the
RNA-gene list from my expressed genes.</p>
<p>I first used <code>biomaRt</code> package to retrieve the chromosome
information for each ENTREZID. This game me 13114 rows, with &gt; 500
rows containing string-only chromosomal locations. I then tried using
the hgnc symbol to pull up gene information and retrieved
‘chromosome_name’, ‘start_position’, ‘end_position’,‘entrezgene_id’,
‘ensembl_gene_id’, ‘hgnc_symbol’ for 15,497 rows of genes from my
expressed gene list. This list again contained multiple string-only
chromosomal locations. I filtered out non-numeric chromosomes after
looking up several genes that had multiple entries with at least one of
the entries containing no ENTREZID or incorrect coordinates or symbols,
leaving me with 13,426 rows of unique genes, but several contained
<code>NAs</code> in the ENTREZID column. Since I generally use ENTREZIDs
to connect my data, I removed those genes and was left with 12,984
unique genes from my RNA expressed gene list.</p>
<p>The next steps involved exporting this file as a .bed file and using
<code>bedtools closest</code> function to map the nearest expressed gene
to each peak. I used
<code>bedtools closest -a background.bed -b Expressedgenes.bed -d &gt; assigned_closest_gene.bed</code>
to map accessible peaks, ie 151,675 peaks. Some peaks were assigned more
than one neargene, and I kept all neargenes assigned in one file with
156,431 rows. I can collapse these into a comma deliminated file, but
for now I am keeping separated lines.</p>
<p>NOTE FOR MYSELF (are we base 0 or base 1!!!!!)<br />
The peak file names do not match the bed file names due to the 0/1
issue. This means to match up MRC categories by name, I need to alter
the first start position by +1 to get the “new” names to match the old
names!</p>
<pre class="r"><code># get RNA gene list:  (14,084)  Can be done using the S13 from my previous data-  i just went to UCSC browser and 
# followed this post on Biostars:  https://bioinformatics.stackexchange.com/questions/17647/create-bed-file-from-list-with-gene-symbols

## I then took the background peaks list and ran &quot;bedtools closest -a background.bed -b Expressedgenes.bed -d -k 2 &gt; assigned_closest_gene.bed&quot;

### Now i can load the file here intersect information
# assigned_closest_gene &lt;- read_delim(&quot;data/n45_bedfiles/assigned_closest_gene.bed&quot;, 
#     delim = &quot;\t&quot;, escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# gene_ref &lt;- assigned_closest_gene$X10
# gene_ref &lt;- unique(gene_ref)
### here i am trying to link the transcript to gene location.   This did not work well so now I am trying to use biomart to retrieve the start and end coordinates of the genes from the gene list, then use that as a .bed file for the closet function.

#  ensembl &lt;- useEnsembl(biomart = &quot;genes&quot;, dataset = &quot;hsapiens_gene_ensembl&quot;)
# my_attributes &lt;- c(&#39;chromosome_name&#39;, &#39;start_position&#39;, &#39;end_position&#39;,&#39;entrezgene_id&#39;, &#39;ensembl_gene_id&#39;, &#39;hgnc_symbol&#39;)
# # searchAttributes(mart = ensembl, pattern = &quot;refseq&quot;)
# # searchDatasets(mart=ensembl,pattern=&quot;NCBI&quot;)
# gene_list &lt;- getBM(attributes=my_attributes,
#       filters = &quot;hgnc_symbol&quot;,
#         values = S13Table$SYMBOL,
#       mart = ensembl)
# write.csv(gene_ref,&quot;data/gene_ref.csv&quot;)
# listFilters(ensembl)

####

gene_full_list &lt;- gene_list %&gt;% 
  mutate(chromosome_name=as.numeric(chromosome_name))%&gt;% 
  # filter(!is.na(chromosome_name)) #%&gt;% 
  na.omit(.)
  dplyr::select(chromosome_name,start_position,end_position,entrezgene_id,ensembl_gene_id)

### this conversion left me with 13114 genes. some Ids were lost in the conversion, so I will try to figure those locations out.  This led me to adjusting the biomart retrieval by using hgnc symbol, then removing nas from 15497 to 13426.  This was further cut to 12984 using na.omit which excluded those ncbi gene ids (entrezgeneids) that were empty.   I went with the na.omit option overall.
write_delim(gene_full_list,delim = &quot;\t&quot;,file=&quot;data/testnames.txt&quot;)

assigned_closest_gene &lt;- read_delim(&quot;data/n45_bedfiles/assigned_closest_gene.bed&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)

expressed_NEARGENE_table &lt;- assigned_closest_gene %&gt;%
#  NOTE FOR MYSELF  (are we base 0 or base 1!!!!!)  
# The peak file names do not match the bed file names due to the 0/1 issue.  This means to match up MRC categories by name, I need to alter the first start position by +1! 
  mutate(X2 = X2+1) %&gt;% 
  unite(&quot;peakid&quot;,X1:X3, sep = &quot;.&quot;, remove= FALSE) %&gt;% 
  rename(&quot;chr&quot;=X1,
         &quot;start&quot;=X2,
         &quot;end&quot;= X3,
         &quot;NG_chr&quot;=X7,
         &quot;NG_start&quot;=X8 ,
         &quot;NG_end&quot;=X9 ,
         &quot;ENTREZID&quot;= X10,
         &quot;ensembl_id&quot;= X11,
         &quot;SYMBOL&quot;=X12,
         &quot;dist_to_NG&quot;=X13) %&gt;%
  dplyr::select(chr, start,end, peakid,NG_chr:dist_to_NG) 

write_tsv(expressed_NEARGENE_table,&quot;data/n45_bedfiles/exp_neargene_table.tsv&quot;)

##test code for collapsing the list later
  group_by(chr, start, end, peakid) %&gt;%
    summarise(NCBI_gene = paste(unique(ENTREZID),collapse=&quot;,&quot;),
              ensembl_ID= paste(unique(ensembl_id),collapse = &quot;,&quot;),
              SYMBOL= paste(unique(SYMBOL),collapse = &quot;,&quot;) , 
              dist_to_NG =min(dist_to_NG));</code></pre>
</div>
<div id="near-gene-and-mrc-peaks" class="section level3">
<h3>near-gene and MRC peaks</h3>
<pre class="r"><code>##Assume file not loaded
exp_neargene_table &lt;- read_delim(&quot;data/n45_bedfiles/exp_neargene_table.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

col_ng_peak &lt;- exp_neargene_table %&gt;%
  group_by(chr, start, end, peakid) %&gt;%
    summarise(NCBI_gene = paste(unique(ENTREZID),collapse=&quot;,&quot;),
              ensembl_ID= paste(unique(ensembl_id),collapse = &quot;,&quot;),
              SYMBOL= paste(unique(SYMBOL),collapse = &quot;,&quot;) , 
              dist_to_NG =min(dist_to_NG))


EAR_peak_list &lt;- exp_neargene_table %&gt;% 
    dplyr::filter(peakid %in% EAR_df$id) %&gt;% 
  dplyr::select(ENTREZID, peakid) %&gt;% 
  distinct()

ESR_peak_list &lt;- exp_neargene_table %&gt;% 
    dplyr::filter(peakid %in% ESR_df$id) %&gt;% 
  dplyr::select(ENTREZID, peakid) %&gt;% 
  distinct()

LR_peak_list &lt;- exp_neargene_table %&gt;% 
    dplyr::filter(peakid %in% LR_df$id) %&gt;% 
  dplyr::select(ENTREZID, peakid) %&gt;% 
  distinct()

NR_peak_list &lt;- exp_neargene_table %&gt;% 
    dplyr::filter(peakid %in% NR_df$id) %&gt;% 
  dplyr::select(ENTREZID, peakid) %&gt;% 
  distinct()

# length(unique(expressed_NEARGENE_table$ENTREZID))</code></pre>
<div id="venn-of-peaks" class="section level4">
<h4>VENN of peaks</h4>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(EAR_peak_list$ENTREZID,ESR_peak_list$ENTREZID,LR_peak_list$ENTREZID, NR_peak_list$ENTREZID),
              category.names = c(&quot;EAR&quot;,&quot;ESR&quot;,&quot;LR&quot;,&quot;NR&quot;), label = &quot;count&quot;)+
  labs(title = &quot;Distribution of NEAR genes by motif response regions&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/venn%20of%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-venn-of-peaks-1">
Past versions of "venn of peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-venn-of-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ee638023f02994f959e6d10d75bb1079ffae3558/docs/figure/Enrichment_motif.Rmd/venn of peaks-1.png" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(EAR_peak_list$ENTREZID,EAR_RNA$ENTREZID),
              category.names = c(&quot;EAR_ATAC&quot;,&quot;EAR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;EAR MRC-neargenes and expressed-RNA_MRC&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/venn%20of%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-venn-of-peaks-2">
Past versions of "venn of peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-venn-of-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ee638023f02994f959e6d10d75bb1079ffae3558/docs/figure/Enrichment_motif.Rmd/venn of peaks-2.png" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(ESR_peak_list$ENTREZID,ESR_RNA$ENTREZID),
              category.names = c(&quot;ESR_ATAC&quot;,&quot;ESR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;ESR MRC-neargenes and expressed-RNA_MRC&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/venn%20of%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-venn-of-peaks-3">
Past versions of "venn of peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-venn-of-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ee638023f02994f959e6d10d75bb1079ffae3558/docs/figure/Enrichment_motif.Rmd/venn of peaks-3.png" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(LR_peak_list$ENTREZID,LR_RNA$ENTREZID),
              category.names = c(&quot;LR_ATAC&quot;,&quot;LR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;LR MRC-neargenes and expressed-RNA_MRC&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/venn%20of%20peaks-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-venn-of-peaks-4">
Past versions of "venn of peaks-4.png"
</button>
</p>
<div id="fig-no-spaces-venn-of-peaks-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ee638023f02994f959e6d10d75bb1079ffae3558/docs/figure/Enrichment_motif.Rmd/venn of peaks-4.png" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggVennDiagram::ggVennDiagram(list(NR_peak_list$ENTREZID,NR_RNA$ENTREZID),
              category.names = c(&quot;NR_ATAC&quot;,&quot;NR_RNA&quot;), label = &quot;count&quot;)+
  labs(title = &quot;NR MRC-neargenes and expressed-RNA_MRC&quot;)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/venn%20of%20peaks-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-venn-of-peaks-5">
Past versions of "venn of peaks-5.png"
</button>
</p>
<div id="fig-no-spaces-venn-of-peaks-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ee638023f02994f959e6d10d75bb1079ffae3558/docs/figure/Enrichment_motif.Rmd/venn of peaks-5.png" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="znf384-chip-overlaps" class="section level3">
<h3>ZNF384 ChIP overlaps</h3>
<pre class="r"><code>### files loaded on line ~ 151


length(findOverlaps(EAR_df_gr, ZNF384_chip))</code></pre>
<pre><code>[1] 302</code></pre>
<pre class="r"><code>length(findOverlaps(ESR_df_gr,  ZNF384_chip))</code></pre>
<pre><code>[1] 319</code></pre>
<pre class="r"><code>length(findOverlaps(LR_df_gr,  ZNF384_chip))</code></pre>
<pre><code>[1] 742</code></pre>
<pre class="r"><code>length(findOverlaps(NR_df_gr,  ZNF384_chip))</code></pre>
<pre><code>[1] 3252</code></pre>
<pre class="r"><code>ZNF_chip_data &lt;- data.frame(&quot;CHIP_peak_num&quot;=c(length(findOverlaps(EAR_df_gr, ZNF384_chip)), 
                            length(findOverlaps(ESR_df_gr,  ZNF384_chip)),
                            length(findOverlaps(LR_df_gr,  ZNF384_chip)),
                            length(findOverlaps(NR_df_gr,  ZNF384_chip))),
                            &quot;MRC&quot;= c(&quot;EAR&quot;,&quot;ESR&quot;,&quot;LR&quot;,&quot;NR&quot;), 
                            &quot;tot_MRC&quot; = c(length(EAR_df$seqnames), 
                                          length(ESR_df$seqnames), 
                                          length(LR_df$seqnames), 
                                          length(NR_df$seqnames))) %&gt;% 
  mutate(non_chip = tot_MRC-CHIP_peak_num) %&gt;% 
  mutate(MRC= factor(MRC, levels = c(&quot;NR&quot;,&quot;EAR&quot;,&quot;ESR&quot;,&quot;LR&quot;)))

ZNF_chip_data %&gt;% 
  pivot_longer(cols= c(CHIP_peak_num, non_chip), values_to = &quot;counts&quot;, names_to = &quot;type&quot;) %&gt;% 
   mutate(percent=counts/tot_MRC*100) %&gt;% 
  ggplot(., aes(x=MRC,y= percent, fill= type)) +
  geom_col()+
   geom_text(aes(label =sprintf(&quot;%.1f %%&quot;,percent)), position = &quot;stack&quot;,vjust= 1
             )+
  ggtitle(&quot;MRC peaks found in ZNF384 ChIP data&quot;)+

  theme_classic()</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/ZNF384-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ZNF384-1">
Past versions of ZNF384-1.png
</button>
</p>
<div id="fig-ZNF384-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ee638023f02994f959e6d10d75bb1079ffae3558/docs/figure/Enrichment_motif.Rmd/ZNF384-1.png" target="_blank">ee63802</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Because of the occurrence of ZNF384 enrichment in accessibility of
ZNF384 motifs, I used a CHIP pull down of ZNF384 to understand how many
of these peaks may be occupied by the TF in my MRCs.</p>
</div>
</div>
<div id="snps-by-category" class="section level2">
<h2>SNPs by category</h2>
<div id="cardiotoxicity" class="section level3">
<h3>Cardiotoxicity</h3>
<pre class="r"><code>## Heart failure is gwas #5

EAR_CTX_SNP &lt;- as.data.frame(join_overlap_intersect(EAR_df_gr, gwas_1_gr_noext))
ESR_CTX_SNP &lt;- as.data.frame(join_overlap_intersect(ESR_df_gr, gwas_1_gr_noext))
LR_CTX_SNP &lt;- as.data.frame(join_overlap_intersect(LR_df_gr,  gwas_1_gr_noext))
NR_CTX_SNP &lt;- as.data.frame(join_overlap_intersect(NR_df_gr, gwas_1_gr_noext))

CTX_SNP_table &lt;- EAR_CTX_SNP %&gt;% 
  mutate(MRC=&quot;EAR&quot;) %&gt;% 
  bind_rows((ESR_CTX_SNP %&gt;% 
              mutate(MRC=&quot;ESR&quot;))) %&gt;% 
  bind_rows((LR_CTX_SNP %&gt;% 
              mutate(MRC=&quot;LR&quot;))) %&gt;% 
  bind_rows((NR_CTX_SNP %&gt;% 
              mutate(MRC=&quot;NR&quot;)))  %&gt;% 
  distinct()
CTX_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  kable(., caption= &quot;SNPs for Cardiotoxicity found in MRC peaks&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs for Cardiotoxicity found in MRC peaks
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
annotation
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CHR_POS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MRC
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NG_chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dist_to_NG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr9.76808694.76808955
</td>
<td style="text-align:left;">
Exon (ENST00000643823.1/50652, exon 4 of 7)
</td>
<td style="text-align:left;">
rs117299725
</td>
<td style="text-align:right;">
76808695
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
76808694
</td>
<td style="text-align:right;">
76808955
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
76611376
</td>
<td style="text-align:right;">
76906114
</td>
<td style="text-align:left;">
158471
</td>
<td style="text-align:left;">
ENSG00000106772
</td>
<td style="text-align:left;">
PRUNE2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.83599067.83599350
</td>
<td style="text-align:left;">
Intron (ENST00000268613.14/1012, intron 8 of 14)
</td>
<td style="text-align:left;">
rs147631684
</td>
<td style="text-align:right;">
83599215
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
83599067
</td>
<td style="text-align:right;">
83599350
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
82626965
</td>
<td style="text-align:right;">
83800640
</td>
<td style="text-align:left;">
1012
</td>
<td style="text-align:left;">
ENSG00000140945
</td>
<td style="text-align:left;">
CDH13
</td>
<td style="text-align:right;">
0
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>CTX_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  write.csv(.,&quot;data/CTX_SNP_table.csv&quot;)</code></pre>
</div>
<div id="response-to-ac-based-chemo" class="section level3">
<h3>response to AC-based chemo</h3>
<pre class="r"><code>## Heart failure is gwas #5

EAR_ACresp_SNP &lt;- as.data.frame(join_overlap_intersect(EAR_df_gr, gwas_3_gr_noext))
ESR_ACresp_SNP &lt;- as.data.frame(join_overlap_intersect(ESR_df_gr, gwas_3_gr_noext))
LR_ACresp_SNP &lt;- as.data.frame(join_overlap_intersect(LR_df_gr,  gwas_3_gr_noext))
NR_ACresp_SNP &lt;- as.data.frame(join_overlap_intersect(NR_df_gr, gwas_3_gr_noext))

ACresp_SNP_table &lt;- EAR_ACresp_SNP %&gt;% 
  mutate(MRC=&quot;EAR&quot;) %&gt;% 
  bind_rows((ESR_ACresp_SNP %&gt;% 
              mutate(MRC=&quot;ESR&quot;))) %&gt;% 
  bind_rows((LR_ACresp_SNP %&gt;% 
              mutate(MRC=&quot;LR&quot;))) %&gt;% 
  bind_rows((NR_ACresp_SNP %&gt;% 
              mutate(MRC=&quot;NR&quot;)))  %&gt;% 
  distinct()
ACresp_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  kable(., caption= &quot;SNPs for response to AC-based chemo found in MRC peaks&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs for response to AC-based chemo found in MRC peaks
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
annotation
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CHR_POS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MRC
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NG_chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dist_to_NG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr9.76808694.76808955
</td>
<td style="text-align:left;">
Exon (ENST00000643823.1/50652, exon 4 of 7)
</td>
<td style="text-align:left;">
rs117299725
</td>
<td style="text-align:right;">
76808695
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
76808694
</td>
<td style="text-align:right;">
76808955
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
76611376
</td>
<td style="text-align:right;">
76906114
</td>
<td style="text-align:left;">
158471
</td>
<td style="text-align:left;">
ENSG00000106772
</td>
<td style="text-align:left;">
PRUNE2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.123933461.123933812
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs10818894
</td>
<td style="text-align:right;">
123933491
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
123933461
</td>
<td style="text-align:right;">
123933812
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
123379654
</td>
<td style="text-align:right;">
123930152
</td>
<td style="text-align:left;">
57706
</td>
<td style="text-align:left;">
ENSG00000119522
</td>
<td style="text-align:left;">
DENND1A
</td>
<td style="text-align:right;">
3309
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.30740138.30740672
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs3885907
</td>
<td style="text-align:right;">
30740318
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
30740138
</td>
<td style="text-align:right;">
30740672
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
30617693
</td>
<td style="text-align:right;">
30660770
</td>
<td style="text-align:left;">
10208
</td>
<td style="text-align:left;">
ENSG00000132952
</td>
<td style="text-align:left;">
USPL1
</td>
<td style="text-align:right;">
79368
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.83599067.83599350
</td>
<td style="text-align:left;">
Intron (ENST00000268613.14/1012, intron 8 of 14)
</td>
<td style="text-align:left;">
rs147631684
</td>
<td style="text-align:right;">
83599215
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
83599067
</td>
<td style="text-align:right;">
83599350
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
82626965
</td>
<td style="text-align:right;">
83800640
</td>
<td style="text-align:left;">
1012
</td>
<td style="text-align:left;">
ENSG00000140945
</td>
<td style="text-align:left;">
CDH13
</td>
<td style="text-align:right;">
0
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>ACresp_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  write.csv(.,&quot;data/ACresp_SNP_table.csv&quot;)</code></pre>
</div>
<div id="heart-failure" class="section level3">
<h3>Heart Failure</h3>
<pre class="r"><code>## Heart failure is gwas #5

EAR_HF_SNP &lt;- as.data.frame(join_overlap_intersect(EAR_df_gr, gwas_5_gr_noext))
ESR_HF_SNP &lt;- as.data.frame(join_overlap_intersect(ESR_df_gr, gwas_5_gr_noext))
LR_HF_SNP &lt;- as.data.frame(join_overlap_intersect(LR_df_gr,  gwas_5_gr_noext))
NR_HF_SNP &lt;- as.data.frame(join_overlap_intersect(NR_df_gr, gwas_5_gr_noext))

HF_SNP_table &lt;- EAR_HF_SNP %&gt;% 
  mutate(MRC=&quot;EAR&quot;) %&gt;% 
  bind_rows((ESR_HF_SNP %&gt;% 
              mutate(MRC=&quot;ESR&quot;))) %&gt;% 
  bind_rows((LR_HF_SNP %&gt;% 
              mutate(MRC=&quot;LR&quot;))) %&gt;% 
  bind_rows((NR_HF_SNP %&gt;% 
              mutate(MRC=&quot;NR&quot;)))  %&gt;% 
  distinct()
HF_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  kable(., caption= &quot;SNPs for Heart Failure found in MRC peaks&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs for Heart Failure found in MRC peaks
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
annotation
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CHR_POS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MRC
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NG_chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dist_to_NG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr3.49173142.49174143
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs7617480
</td>
<td style="text-align:right;">
49173299
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
49173142
</td>
<td style="text-align:right;">
49174143
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
49171598
</td>
<td style="text-align:right;">
49176486
</td>
<td style="text-align:left;">
200942
</td>
<td style="text-align:left;">
ENSG00000185909
</td>
<td style="text-align:left;">
KLHDC8B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.36678380.36679788
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3176326
</td>
<td style="text-align:right;">
36679512
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
36678380
</td>
<td style="text-align:right;">
36679788
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
36676460
</td>
<td style="text-align:right;">
36687337
</td>
<td style="text-align:left;">
1026
</td>
<td style="text-align:left;">
ENSG00000124762
</td>
<td style="text-align:left;">
CDKN1A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs660240
</td>
<td style="text-align:right;">
109275216
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs629301
</td>
<td style="text-align:right;">
109275684
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.236688731.236689178
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs12724121
</td>
<td style="text-align:right;">
236688982
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
236688731
</td>
<td style="text-align:right;">
236689178
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
236664141
</td>
<td style="text-align:right;">
236764631
</td>
<td style="text-align:left;">
88
</td>
<td style="text-align:left;">
ENSG00000077522
</td>
<td style="text-align:left;">
ACTN2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.178846382.178846751
</td>
<td style="text-align:left;">
Intron (ENST00000652826.1/105373766, intron 1 of 5)
</td>
<td style="text-align:left;">
rs2220127
</td>
<td style="text-align:right;">
178846649
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
178846382
</td>
<td style="text-align:right;">
178846751
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
178829757
</td>
<td style="text-align:right;">
179050137
</td>
<td style="text-align:left;">
285025
</td>
<td style="text-align:left;">
ENSG00000163492
</td>
<td style="text-align:left;">
CCDC141
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.14232390.14233136
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs56281979
</td>
<td style="text-align:right;">
14232793
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
14232390
</td>
<td style="text-align:right;">
14233136
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
14178817
</td>
<td style="text-align:right;">
14201122
</td>
<td style="text-align:left;">
27258
</td>
<td style="text-align:left;">
ENSG00000170860
</td>
<td style="text-align:left;">
LSM3
</td>
<td style="text-align:right;">
31268
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.22625204.22625843
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs112577387
</td>
<td style="text-align:right;">
22625658
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
22625204
</td>
<td style="text-align:right;">
22625843
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
22345071
</td>
<td style="text-align:right;">
22516066
</td>
<td style="text-align:left;">
166647
</td>
<td style="text-align:left;">
ENSG00000152990
</td>
<td style="text-align:left;">
ADGRA3
</td>
<td style="text-align:right;">
109138
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.139426247.139426836
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11242465
</td>
<td style="text-align:right;">
139426616
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
139426247
</td>
<td style="text-align:right;">
139426836
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
139408588
</td>
<td style="text-align:right;">
139444491
</td>
<td style="text-align:left;">
202052
</td>
<td style="text-align:left;">
ENSG00000170464
</td>
<td style="text-align:left;">
DNAJC18
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.117037235.117037719
</td>
<td style="text-align:left;">
Intron (ENST00000445177.6/23387, intron 1 of 24)
</td>
<td style="text-align:left;">
rs7115242
</td>
<td style="text-align:right;">
117037567
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
117037235
</td>
<td style="text-align:right;">
117037719
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
116843402
</td>
<td style="text-align:right;">
117098437
</td>
<td style="text-align:left;">
23387
</td>
<td style="text-align:left;">
ENSG00000160584
</td>
<td style="text-align:left;">
SIK3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.106865628.106866040
</td>
<td style="text-align:left;">
Intron (ENST00000392839.6/55188, intron 8 of 8)
</td>
<td style="text-align:left;">
rs7977247
</td>
<td style="text-align:right;">
106865692
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
106865628
</td>
<td style="text-align:right;">
106866040
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
106774621
</td>
<td style="text-align:right;">
106889316
</td>
<td style="text-align:left;">
55188
</td>
<td style="text-align:left;">
ENSG00000111785
</td>
<td style="text-align:left;">
RIC8B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.719886.721785
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs76064792
</td>
<td style="text-align:right;">
720886
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
719886
</td>
<td style="text-align:right;">
721785
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
720581
</td>
<td style="text-align:right;">
722590
</td>
<td style="text-align:left;">
65990
</td>
<td style="text-align:left;">
ENSG00000103254
</td>
<td style="text-align:left;">
ANTKMT
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.53768577.53768944
</td>
<td style="text-align:left;">
Intron (ENST00000636491.1/79068, intron 2 of 9)
</td>
<td style="text-align:left;">
rs11642015
</td>
<td style="text-align:right;">
53768582
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
53768577
</td>
<td style="text-align:right;">
53768944
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
53701692
</td>
<td style="text-align:right;">
54158512
</td>
<td style="text-align:left;">
79068
</td>
<td style="text-align:left;">
ENSG00000140718
</td>
<td style="text-align:left;">
FTO
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.58289003.58291257
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs10871753
</td>
<td style="text-align:right;">
58289633
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
58289003
</td>
<td style="text-align:right;">
58291257
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
58044226
</td>
<td style="text-align:right;">
58401540
</td>
<td style="text-align:left;">
23327
</td>
<td style="text-align:left;">
ENSG00000049759
</td>
<td style="text-align:left;">
NEDD4L
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.60067561.60068151
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7234864
</td>
<td style="text-align:right;">
60067625
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
60067561
</td>
<td style="text-align:right;">
60068151
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
59899996
</td>
<td style="text-align:right;">
59904305
</td>
<td style="text-align:left;">
5366
</td>
<td style="text-align:left;">
ENSG00000141682
</td>
<td style="text-align:left;">
PMAIP1
</td>
<td style="text-align:right;">
163256
</td>
</tr>
<tr>
<td style="text-align:left;">
chr22.46417011.46418228
</td>
<td style="text-align:left;">
Intron (ENST00000262738.9/9620, intron 5 of 34)
</td>
<td style="text-align:left;">
rs190258023
</td>
<td style="text-align:right;">
46417046
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
46417011
</td>
<td style="text-align:right;">
46418228
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
46360834
</td>
<td style="text-align:right;">
46537620
</td>
<td style="text-align:left;">
9620
</td>
<td style="text-align:left;">
ENSG00000075275
</td>
<td style="text-align:left;">
CELSR1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr22.46417011.46418228
</td>
<td style="text-align:left;">
Intron (ENST00000262738.9/9620, intron 5 of 34)
</td>
<td style="text-align:left;">
rs148416395
</td>
<td style="text-align:right;">
46417215
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
46417011
</td>
<td style="text-align:right;">
46418228
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
46360834
</td>
<td style="text-align:right;">
46537620
</td>
<td style="text-align:left;">
9620
</td>
<td style="text-align:left;">
ENSG00000075275
</td>
<td style="text-align:left;">
CELSR1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.27262237.27263756
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6759518
</td>
<td style="text-align:right;">
27263727
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
27262237
</td>
<td style="text-align:right;">
27263756
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
27253684
</td>
<td style="text-align:right;">
27275817
</td>
<td style="text-align:left;">
7781
</td>
<td style="text-align:left;">
ENSG00000115194
</td>
<td style="text-align:left;">
SLC30A3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.36922199.36922493
</td>
<td style="text-align:left;">
Intron (ENST00000263918.9/6801, intron 2 of 17)
</td>
<td style="text-align:left;">
rs11124554
</td>
<td style="text-align:right;">
36922355
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
36922199
</td>
<td style="text-align:right;">
36922493
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
36837698
</td>
<td style="text-align:right;">
36966536
</td>
<td style="text-align:left;">
6801
</td>
<td style="text-align:left;">
ENSG00000115808
</td>
<td style="text-align:left;">
STRN
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.123019226.123019547
</td>
<td style="text-align:left;">
Intron (ENST00000357599.8/54437, intron 1 of 22)
</td>
<td style="text-align:left;">
rs7632505
</td>
<td style="text-align:right;">
123019460
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
123019226
</td>
<td style="text-align:right;">
123019547
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
122909082
</td>
<td style="text-align:right;">
123028605
</td>
<td style="text-align:left;">
54437
</td>
<td style="text-align:left;">
ENSG00000082684
</td>
<td style="text-align:left;">
SEMA5B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.42087754.42088697
</td>
<td style="text-align:left;">
Downstream (&lt;=300bp)
</td>
<td style="text-align:left;">
rs79661299
</td>
<td style="text-align:right;">
42088268
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
42087754
</td>
<td style="text-align:right;">
42088697
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
42050513
</td>
<td style="text-align:right;">
42087461
</td>
<td style="text-align:left;">
129685
</td>
<td style="text-align:left;">
ENSG00000137413
</td>
<td style="text-align:left;">
TAF8
</td>
<td style="text-align:right;">
293
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.124847104.124848853
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs34866937
</td>
<td style="text-align:right;">
124847608
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
124847104
</td>
<td style="text-align:right;">
124848853
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
124550784
</td>
<td style="text-align:right;">
124728473
</td>
<td style="text-align:left;">
9788
</td>
<td style="text-align:left;">
ENSG00000170873
</td>
<td style="text-align:left;">
MTSS1
</td>
<td style="text-align:right;">
118631
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.26194928.26196665
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs113819537
</td>
<td style="text-align:right;">
26195496
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
26194928
</td>
<td style="text-align:right;">
26196665
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
26121991
</td>
<td style="text-align:right;">
26299290
</td>
<td style="text-align:left;">
8082
</td>
<td style="text-align:left;">
ENSG00000123096
</td>
<td style="text-align:left;">
SSPN
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.75895398.75895669
</td>
<td style="text-align:left;">
Intron (ENST00000552856.1/105369844, intron 1 of 4)
</td>
<td style="text-align:left;">
rs114782882
</td>
<td style="text-align:right;">
75895517
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
75895398
</td>
<td style="text-align:right;">
75895669
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
76025447
</td>
<td style="text-align:right;">
76031776
</td>
<td style="text-align:left;">
22822
</td>
<td style="text-align:left;">
ENSG00000139289
</td>
<td style="text-align:left;">
PHLDA1
</td>
<td style="text-align:right;">
129779
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.23574050.23574545
</td>
<td style="text-align:left;">
Intron (ENST00000269228.10/4864, intron 1 of 24)
</td>
<td style="text-align:left;">
rs1788826
</td>
<td style="text-align:right;">
23574060
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
23574050
</td>
<td style="text-align:right;">
23574545
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
23506184
</td>
<td style="text-align:right;">
23586506
</td>
<td style="text-align:left;">
4864
</td>
<td style="text-align:left;">
ENSG00000141458
</td>
<td style="text-align:left;">
NPC1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41439501.41440141
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs13346603
</td>
<td style="text-align:right;">
41439932
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41439501
</td>
<td style="text-align:right;">
41440141
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41431318
</td>
<td style="text-align:right;">
41440717
</td>
<td style="text-align:left;">
55101
</td>
<td style="text-align:left;">
ENSG00000105341
</td>
<td style="text-align:left;">
DMAC2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.14118943.14119390
</td>
<td style="text-align:left;">
Intron (ENST00000690295.1/105377134, intron 3 of 3)
</td>
<td style="text-align:left;">
rs57346421
</td>
<td style="text-align:right;">
14119015
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
14118943
</td>
<td style="text-align:right;">
14119390
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
14371115
</td>
<td style="text-align:right;">
14383484
</td>
<td style="text-align:left;">
6782
</td>
<td style="text-align:left;">
ENSG00000155304
</td>
<td style="text-align:left;">
HSPA13
</td>
<td style="text-align:right;">
251726
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>HF_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  write.csv(.,&quot;data/HF_SNP_table.csv&quot;)</code></pre>
</div>
<div id="arrhythmia" class="section level3">
<h3>Arrhythmia</h3>
<pre class="r"><code>## Heart failure is gwas #5

EAR_ARR_SNP &lt;- as.data.frame(join_overlap_intersect(EAR_df_gr, gwas_2_gr_noext))
ESR_ARR_SNP &lt;- as.data.frame(join_overlap_intersect(ESR_df_gr, gwas_2_gr_noext))
LR_ARR_SNP &lt;- as.data.frame(join_overlap_intersect(LR_df_gr, gwas_2_gr_noext))
NR_ARR_SNP &lt;- as.data.frame(join_overlap_intersect(NR_df_gr, gwas_2_gr_noext))

ARR_SNP_table &lt;- ESR_ARR_SNP %&gt;% 
  mutate(MRC=&quot;ESR&quot;) %&gt;% 
  bind_rows((EAR_ARR_SNP %&gt;% 
              mutate(MRC=&quot;EAR&quot;))) %&gt;% 
  bind_rows((LR_ARR_SNP %&gt;% 
              mutate(MRC=&quot;LR&quot;))) %&gt;% 
  bind_rows((NR_ARR_SNP %&gt;% 
              mutate(MRC=&quot;NR&quot;))) %&gt;% 
  distinct()
ARR_SNP_table %&gt;%  
  distinct() %&gt;% 
 # rename( id=&quot;peak_id&quot;) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  kable(., caption= &quot;SNPs for Arrhythmia found in MRC peaks&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs for Arrhythmia found in MRC peaks
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
annotation
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CHR_POS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MRC
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NG_chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dist_to_NG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr1.10736684.10737808
</td>
<td style="text-align:left;">
Intron (ENST00000377022.8/54897, intron 2 of 20)
</td>
<td style="text-align:left;">
rs880315
</td>
<td style="text-align:right;">
10736809
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
10736684
</td>
<td style="text-align:right;">
10737808
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
10636604
</td>
<td style="text-align:right;">
10796650
</td>
<td style="text-align:left;">
54897
</td>
<td style="text-align:left;">
ENSG00000130940
</td>
<td style="text-align:left;">
CASZ1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.11792123.11792871
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs17375901
</td>
<td style="text-align:right;">
11792459
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
11792123
</td>
<td style="text-align:right;">
11792871
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
11785723
</td>
<td style="text-align:right;">
11806455
</td>
<td style="text-align:left;">
4524
</td>
<td style="text-align:left;">
ENSG00000177000
</td>
<td style="text-align:left;">
MTHFR
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.154839629.154839995
</td>
<td style="text-align:left;">
Intron (ENST00000271915.9/3782, intron 1 of 7)
</td>
<td style="text-align:left;">
rs34292822
</td>
<td style="text-align:right;">
154839924
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
154839629
</td>
<td style="text-align:right;">
154839995
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
154697455
</td>
<td style="text-align:right;">
154870281
</td>
<td style="text-align:left;">
3782
</td>
<td style="text-align:left;">
ENSG00000143603
</td>
<td style="text-align:left;">
KCNN3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.86367009.86367716
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs72926475
</td>
<td style="text-align:right;">
86367364
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
86367009
</td>
<td style="text-align:right;">
86367716
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
86213993
</td>
<td style="text-align:right;">
86338083
</td>
<td style="text-align:left;">
65055
</td>
<td style="text-align:left;">
ENSG00000068615
</td>
<td style="text-align:left;">
REEP1
</td>
<td style="text-align:right;">
28926
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.178547784.178549172
</td>
<td style="text-align:left;">
Exon (ENST00000603415.1/ENST00000603415.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs3731748
</td>
<td style="text-align:right;">
178548383
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
178547784
</td>
<td style="text-align:right;">
178549172
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
178525989
</td>
<td style="text-align:right;">
178830802
</td>
<td style="text-align:left;">
7273
</td>
<td style="text-align:left;">
ENSG00000155657
</td>
<td style="text-align:left;">
TTN
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.10102598.10102907
</td>
<td style="text-align:left;">
Intron (ENST00000699786.1/9948, intron 2 of 16)
</td>
<td style="text-align:left;">
rs12640611
</td>
<td style="text-align:right;">
10102854
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
10102598
</td>
<td style="text-align:right;">
10102907
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
10068089
</td>
<td style="text-align:right;">
10116972
</td>
<td style="text-align:left;">
9948
</td>
<td style="text-align:left;">
ENSG00000071127
</td>
<td style="text-align:left;">
WDR1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.36678380.36679788
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3176326
</td>
<td style="text-align:right;">
36679512
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
36678380
</td>
<td style="text-align:right;">
36679788
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
36676460
</td>
<td style="text-align:right;">
36687337
</td>
<td style="text-align:left;">
1026
</td>
<td style="text-align:left;">
ENSG00000124762
</td>
<td style="text-align:left;">
CDKN1A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.116545872.116546373
</td>
<td style="text-align:left;">
Intron (ENST00000614113.5/857, intron 3 of 3)
</td>
<td style="text-align:left;">
rs3807989
</td>
<td style="text-align:right;">
116546187
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
116545872
</td>
<td style="text-align:right;">
116546373
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
116524994
</td>
<td style="text-align:right;">
116561179
</td>
<td style="text-align:left;">
857
</td>
<td style="text-align:left;">
ENSG00000105974
</td>
<td style="text-align:left;">
CAV1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.150954727.150956459
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2269001
</td>
<td style="text-align:right;">
150954728
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
150954727
</td>
<td style="text-align:right;">
150956459
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
150944961
</td>
<td style="text-align:right;">
150978321
</td>
<td style="text-align:left;">
3757
</td>
<td style="text-align:left;">
ENSG00000055118
</td>
<td style="text-align:left;">
KCNH2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.103720444.103721025
</td>
<td style="text-align:left;">
Intron (ENST00000369774.9/9644, intron 5 of 14)
</td>
<td style="text-align:left;">
rs35176054
</td>
<td style="text-align:right;">
103720629
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
103720444
</td>
<td style="text-align:right;">
103721025
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
103594027
</td>
<td style="text-align:right;">
103855584
</td>
<td style="text-align:left;">
9644
</td>
<td style="text-align:left;">
ENSG00000107957
</td>
<td style="text-align:left;">
SH3PXD2A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.73374622.73375213
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7172038
</td>
<td style="text-align:right;">
73374914
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73374622
</td>
<td style="text-align:right;">
73375213
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73319859
</td>
<td style="text-align:right;">
73368958
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
ENSG00000138622
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:right;">
5664
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.7510984.7512636
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2071502
</td>
<td style="text-align:right;">
7511639
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
7510984
</td>
<td style="text-align:right;">
7512636
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
7484366
</td>
<td style="text-align:right;">
7514616
</td>
<td style="text-align:left;">
5430
</td>
<td style="text-align:left;">
ENSG00000181222
</td>
<td style="text-align:left;">
POLR2A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.34746498.34746999
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2834618
</td>
<td style="text-align:right;">
34746814
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
34746498
</td>
<td style="text-align:right;">
34746999
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
34787801
</td>
<td style="text-align:right;">
36004667
</td>
<td style="text-align:left;">
100506403
</td>
<td style="text-align:left;">
ENSG00000159216
</td>
<td style="text-align:left;">
RUNX1
</td>
<td style="text-align:right;">
40803
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.34746498.34746999
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2834618
</td>
<td style="text-align:right;">
34746814
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
34746498
</td>
<td style="text-align:right;">
34746999
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
34787801
</td>
<td style="text-align:right;">
36004667
</td>
<td style="text-align:left;">
861
</td>
<td style="text-align:left;">
ENSG00000159216
</td>
<td style="text-align:left;">
RUNX1
</td>
<td style="text-align:right;">
40803
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.173529475.173530537
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3822127
</td>
<td style="text-align:right;">
173530418
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
173529475
</td>
<td style="text-align:right;">
173530537
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
173524969
</td>
<td style="text-align:right;">
173530229
</td>
<td style="text-align:left;">
9464
</td>
<td style="text-align:left;">
ENSG00000164107
</td>
<td style="text-align:left;">
HAND2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.173529475.173530537
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3822127
</td>
<td style="text-align:right;">
173530418
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
173529475
</td>
<td style="text-align:right;">
173530537
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
173527270
</td>
<td style="text-align:right;">
173659696
</td>
<td style="text-align:left;">
79804
</td>
<td style="text-align:left;">
ENSG00000237125
</td>
<td style="text-align:left;">
HAND2-AS1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.116560073.116560553
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs1049334
</td>
<td style="text-align:right;">
116560326
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
116560073
</td>
<td style="text-align:right;">
116560553
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
116524994
</td>
<td style="text-align:right;">
116561179
</td>
<td style="text-align:left;">
857
</td>
<td style="text-align:left;">
ENSG00000105974
</td>
<td style="text-align:left;">
CAV1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs629301
</td>
<td style="text-align:right;">
109275684
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.170224576.170224916
</td>
<td style="text-align:left;">
Intron (ENST00000661719.1/105371609, intron 1 of 1)
</td>
<td style="text-align:left;">
rs72700114
</td>
<td style="text-align:right;">
170224684
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
170224576
</td>
<td style="text-align:right;">
170224916
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
170145959
</td>
<td style="text-align:right;">
170168866
</td>
<td style="text-align:left;">
149281
</td>
<td style="text-align:left;">
ENSG00000203740
</td>
<td style="text-align:left;">
NTMT2
</td>
<td style="text-align:right;">
55710
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.170224576.170224916
</td>
<td style="text-align:left;">
Intron (ENST00000661719.1/105371609, intron 1 of 1)
</td>
<td style="text-align:left;">
rs12122060
</td>
<td style="text-align:right;">
170224718
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
170224576
</td>
<td style="text-align:right;">
170224916
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
170145959
</td>
<td style="text-align:right;">
170168866
</td>
<td style="text-align:left;">
149281
</td>
<td style="text-align:left;">
ENSG00000203740
</td>
<td style="text-align:left;">
NTMT2
</td>
<td style="text-align:right;">
55710
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.179701478.179702086
</td>
<td style="text-align:left;">
Intron (ENST00000410066.7/151126, intron 3 of 9)
</td>
<td style="text-align:left;">
rs16866933
</td>
<td style="text-align:right;">
179701951
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
179701478
</td>
<td style="text-align:right;">
179702086
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
179441982
</td>
<td style="text-align:right;">
179861612
</td>
<td style="text-align:left;">
151126
</td>
<td style="text-align:left;">
ENSG00000144331
</td>
<td style="text-align:left;">
ZNF385B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.38725644.38726351
</td>
<td style="text-align:left;">
Intron (ENST00000655275.1/6336, intron 17 of 27)
</td>
<td style="text-align:left;">
rs6801957
</td>
<td style="text-align:right;">
38725824
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
38725644
</td>
<td style="text-align:right;">
38726351
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
38548057
</td>
<td style="text-align:right;">
38649687
</td>
<td style="text-align:left;">
6331
</td>
<td style="text-align:left;">
ENSG00000183873
</td>
<td style="text-align:left;">
SCN5A
</td>
<td style="text-align:right;">
75957
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.110793293.110793943
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs2220427
</td>
<td style="text-align:right;">
110793733
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
110793293
</td>
<td style="text-align:right;">
110793943
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
110617423
</td>
<td style="text-align:right;">
110642123
</td>
<td style="text-align:left;">
5308
</td>
<td style="text-align:left;">
ENSG00000164093
</td>
<td style="text-align:left;">
PITX2
</td>
<td style="text-align:right;">
151170
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.11641900.11643102
</td>
<td style="text-align:left;">
Intron (ENST00000653416.2/105379242, intron 1 of 2)
</td>
<td style="text-align:left;">
rs35620480
</td>
<td style="text-align:right;">
11642399
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
11641900
</td>
<td style="text-align:right;">
11643102
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
11676959
</td>
<td style="text-align:right;">
11760002
</td>
<td style="text-align:left;">
2626
</td>
<td style="text-align:left;">
ENSG00000136574
</td>
<td style="text-align:left;">
GATA4
</td>
<td style="text-align:right;">
33858
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.20953148.20953475
</td>
<td style="text-align:left;">
Intron (ENST00000675700.1/10529, intron 4 of 6)
</td>
<td style="text-align:left;">
rs7910227
</td>
<td style="text-align:right;">
20953453
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
20953148
</td>
<td style="text-align:right;">
20953475
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
20779973
</td>
<td style="text-align:right;">
21293011
</td>
<td style="text-align:left;">
10529
</td>
<td style="text-align:left;">
ENSG00000078114
</td>
<td style="text-align:left;">
NEBL
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.73660865.73661568
</td>
<td style="text-align:left;">
Intron (ENST00000668336.1/105378360, intron 1 of 1)
</td>
<td style="text-align:left;">
rs10824026
</td>
<td style="text-align:right;">
73661450
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
73660865
</td>
<td style="text-align:right;">
73661568
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
73644881
</td>
<td style="text-align:right;">
73663803
</td>
<td style="text-align:left;">
79933
</td>
<td style="text-align:left;">
ENSG00000166317
</td>
<td style="text-align:left;">
SYNPO2L
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.79139073.79139940
</td>
<td style="text-align:left;">
5’ UTR
</td>
<td style="text-align:left;">
rs1769758
</td>
<td style="text-align:right;">
79139212
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
79139073
</td>
<td style="text-align:right;">
79139940
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
79068966
</td>
<td style="text-align:right;">
79316519
</td>
<td style="text-align:left;">
57178
</td>
<td style="text-align:left;">
ENSG00000108175
</td>
<td style="text-align:left;">
ZMIZ1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
Intron (ENST00000360655.8/89797, intron 11 of 37)
</td>
<td style="text-align:left;">
rs4757877
</td>
<td style="text-align:right;">
19988745
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
19988504
</td>
<td style="text-align:right;">
19989024
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
19350724
</td>
<td style="text-align:right;">
20121601
</td>
<td style="text-align:left;">
89797
</td>
<td style="text-align:left;">
ENSG00000166833
</td>
<td style="text-align:left;">
NAV2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
Intron (ENST00000360655.8/89797, intron 11 of 37)
</td>
<td style="text-align:left;">
rs2625322
</td>
<td style="text-align:right;">
19988805
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
19988504
</td>
<td style="text-align:right;">
19989024
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
19350724
</td>
<td style="text-align:right;">
20121601
</td>
<td style="text-align:left;">
89797
</td>
<td style="text-align:left;">
ENSG00000166833
</td>
<td style="text-align:left;">
NAV2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
Intron (ENST00000360655.8/89797, intron 11 of 37)
</td>
<td style="text-align:left;">
rs1822273
</td>
<td style="text-align:right;">
19988967
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
19988504
</td>
<td style="text-align:right;">
19989024
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
19350724
</td>
<td style="text-align:right;">
20121601
</td>
<td style="text-align:left;">
89797
</td>
<td style="text-align:left;">
ENSG00000166833
</td>
<td style="text-align:left;">
NAV2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.117037235.117037719
</td>
<td style="text-align:left;">
Intron (ENST00000445177.6/23387, intron 1 of 24)
</td>
<td style="text-align:left;">
rs7115242
</td>
<td style="text-align:right;">
117037567
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
117037235
</td>
<td style="text-align:right;">
117037719
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
116843402
</td>
<td style="text-align:right;">
117098437
</td>
<td style="text-align:left;">
23387
</td>
<td style="text-align:left;">
ENSG00000160584
</td>
<td style="text-align:left;">
SIK3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.21536977.21537596
</td>
<td style="text-align:left;">
Intron (ENST00000382374.9/221154, intron 4 of 11)
</td>
<td style="text-align:left;">
rs11841562
</td>
<td style="text-align:right;">
21537382
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
21536977
</td>
<td style="text-align:right;">
21537596
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
21492691
</td>
<td style="text-align:right;">
21604181
</td>
<td style="text-align:left;">
221154
</td>
<td style="text-align:left;">
ENSG00000165487
</td>
<td style="text-align:left;">
MICU2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.2214229.2215681
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs77316573
</td>
<td style="text-align:right;">
2215270
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
2214229
</td>
<td style="text-align:right;">
2215681
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
2211593
</td>
<td style="text-align:right;">
2214840
</td>
<td style="text-align:left;">
283871
</td>
<td style="text-align:left;">
ENSG00000184207
</td>
<td style="text-align:left;">
PGP
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.72877300.72877905
</td>
<td style="text-align:left;">
Intron (ENST00000268489.10/463, intron 4 of 9)
</td>
<td style="text-align:left;">
rs7197197
</td>
<td style="text-align:right;">
72877507
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
72877300
</td>
<td style="text-align:right;">
72877905
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
72782885
</td>
<td style="text-align:right;">
73891871
</td>
<td style="text-align:left;">
463
</td>
<td style="text-align:left;">
ENSG00000140836
</td>
<td style="text-align:left;">
ZFHX3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr20.6590860.6591642
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs2145274
</td>
<td style="text-align:right;">
6591367
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr20
</td>
<td style="text-align:right;">
6590860
</td>
<td style="text-align:right;">
6591642
</td>
<td style="text-align:left;">
chr20
</td>
<td style="text-align:right;">
6767686
</td>
<td style="text-align:right;">
6780246
</td>
<td style="text-align:left;">
650
</td>
<td style="text-align:left;">
ENSG00000125845
</td>
<td style="text-align:left;">
BMP2
</td>
<td style="text-align:right;">
176045
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.21955557.21956277
</td>
<td style="text-align:left;">
Intron (ENST00000637118.1/ENST00000637118.1, intron 1 of 1)
</td>
<td style="text-align:left;">
rs7529220
</td>
<td style="text-align:right;">
21956126
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
21955557
</td>
<td style="text-align:right;">
21956277
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
21822244
</td>
<td style="text-align:right;">
21937310
</td>
<td style="text-align:left;">
3339
</td>
<td style="text-align:left;">
ENSG00000142798
</td>
<td style="text-align:left;">
HSPG2
</td>
<td style="text-align:right;">
18247
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.154741375.154741854
</td>
<td style="text-align:left;">
Intron (ENST00000271915.9/3782, intron 3 of 7)
</td>
<td style="text-align:left;">
rs4999127
</td>
<td style="text-align:right;">
154741530
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
154741375
</td>
<td style="text-align:right;">
154741854
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
154697455
</td>
<td style="text-align:right;">
154870281
</td>
<td style="text-align:left;">
3782
</td>
<td style="text-align:left;">
ENSG00000143603
</td>
<td style="text-align:left;">
KCNN3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.205748526.205750556
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs4951261
</td>
<td style="text-align:right;">
205748695
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
205748526
</td>
<td style="text-align:right;">
205750556
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
205712822
</td>
<td style="text-align:right;">
205750182
</td>
<td style="text-align:left;">
64710
</td>
<td style="text-align:left;">
ENSG00000069275
</td>
<td style="text-align:left;">
NUCKS1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.25936965.25937466
</td>
<td style="text-align:left;">
Intron (ENST00000264712.8/3797, intron 5 of 7)
</td>
<td style="text-align:left;">
rs6546620
</td>
<td style="text-align:right;">
25937071
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
25936965
</td>
<td style="text-align:right;">
25937466
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
25926598
</td>
<td style="text-align:right;">
25982749
</td>
<td style="text-align:left;">
3797
</td>
<td style="text-align:left;">
ENSG00000084731
</td>
<td style="text-align:left;">
KIF3C
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.27262237.27263756
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6759518
</td>
<td style="text-align:right;">
27263727
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
27262237
</td>
<td style="text-align:right;">
27263756
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
27253684
</td>
<td style="text-align:right;">
27275817
</td>
<td style="text-align:left;">
7781
</td>
<td style="text-align:left;">
ENSG00000115194
</td>
<td style="text-align:left;">
SLC30A3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.65055695.65057220
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs74181299
</td>
<td style="text-align:right;">
65056838
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
65055695
</td>
<td style="text-align:right;">
65057220
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
65056354
</td>
<td style="text-align:right;">
65087004
</td>
<td style="text-align:left;">
23177
</td>
<td style="text-align:left;">
ENSG00000011523
</td>
<td style="text-align:left;">
CEP68
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.65055695.65057220
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2540949
</td>
<td style="text-align:right;">
65057097
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
65055695
</td>
<td style="text-align:right;">
65057220
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
65056354
</td>
<td style="text-align:right;">
65087004
</td>
<td style="text-align:left;">
23177
</td>
<td style="text-align:left;">
ENSG00000011523
</td>
<td style="text-align:left;">
CEP68
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.135149084.135149706
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6730157
</td>
<td style="text-align:right;">
135149518
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135149084
</td>
<td style="text-align:right;">
135149706
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135052289
</td>
<td style="text-align:right;">
135176667
</td>
<td style="text-align:left;">
22930
</td>
<td style="text-align:left;">
ENSG00000115839
</td>
<td style="text-align:left;">
RAB3GAP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.135149084.135149706
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6730157
</td>
<td style="text-align:right;">
135149518
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135149084
</td>
<td style="text-align:right;">
135149706
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135136916
</td>
<td style="text-align:right;">
135531218
</td>
<td style="text-align:left;">
84083
</td>
<td style="text-align:left;">
ENSG00000121988
</td>
<td style="text-align:left;">
ZRANB3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.200305915.200307710
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs10931898
</td>
<td style="text-align:right;">
200306468
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
200305915
</td>
<td style="text-align:right;">
200307710
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
200305881
</td>
<td style="text-align:right;">
200482264
</td>
<td style="text-align:left;">
26010
</td>
<td style="text-align:left;">
ENSG00000196141
</td>
<td style="text-align:left;">
SPATS2L
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.200330546.200331118
</td>
<td style="text-align:left;">
Intron (ENST00000439084.5/26010, intron 1 of 4)
</td>
<td style="text-align:left;">
rs295114
</td>
<td style="text-align:right;">
200330879
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
200330546
</td>
<td style="text-align:right;">
200331118
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
200305881
</td>
<td style="text-align:right;">
200482264
</td>
<td style="text-align:left;">
26010
</td>
<td style="text-align:left;">
ENSG00000196141
</td>
<td style="text-align:left;">
SPATS2L
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.123019226.123019547
</td>
<td style="text-align:left;">
Intron (ENST00000357599.8/54437, intron 1 of 22)
</td>
<td style="text-align:left;">
rs7632505
</td>
<td style="text-align:right;">
123019460
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
123019226
</td>
<td style="text-align:right;">
123019547
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
122909082
</td>
<td style="text-align:right;">
123028605
</td>
<td style="text-align:left;">
54437
</td>
<td style="text-align:left;">
ENSG00000082684
</td>
<td style="text-align:left;">
SEMA5B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.179454943.179455262
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7612445
</td>
<td style="text-align:right;">
179455191
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
179454943
</td>
<td style="text-align:right;">
179455262
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
179396088
</td>
<td style="text-align:right;">
179451476
</td>
<td style="text-align:left;">
59345
</td>
<td style="text-align:left;">
ENSG00000114450
</td>
<td style="text-align:left;">
GNB4
</td>
<td style="text-align:right;">
3467
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.10115366.10117581
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3822259
</td>
<td style="text-align:right;">
10117121
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
10115366
</td>
<td style="text-align:right;">
10117581
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
10068089
</td>
<td style="text-align:right;">
10116972
</td>
<td style="text-align:left;">
9948
</td>
<td style="text-align:left;">
ENSG00000071127
</td>
<td style="text-align:left;">
WDR1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.110631655.110632575
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2595104
</td>
<td style="text-align:right;">
110631977
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
110631655
</td>
<td style="text-align:right;">
110632575
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
110617423
</td>
<td style="text-align:right;">
110642123
</td>
<td style="text-align:left;">
5308
</td>
<td style="text-align:left;">
ENSG00000164093
</td>
<td style="text-align:left;">
PITX2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.148015002.148015364
</td>
<td style="text-align:left;">
Intron (ENST00000336498.8/79658, intron 18 of 22)
</td>
<td style="text-align:left;">
rs10213376
</td>
<td style="text-align:right;">
148015263
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
148015002
</td>
<td style="text-align:right;">
148015364
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
147732063
</td>
<td style="text-align:right;">
148072776
</td>
<td style="text-align:left;">
79658
</td>
<td style="text-align:left;">
ENSG00000071205
</td>
<td style="text-align:left;">
ARHGAP10
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.138068694.138070268
</td>
<td style="text-align:left;">
Exon (ENST00000410992.1/ENST00000410992.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs141654122
</td>
<td style="text-align:right;">
138070140
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
138068694
</td>
<td style="text-align:right;">
138070268
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
137937960
</td>
<td style="text-align:right;">
138051961
</td>
<td style="text-align:left;">
51306
</td>
<td style="text-align:left;">
ENSG00000031003
</td>
<td style="text-align:left;">
FAM13B
</td>
<td style="text-align:right;">
16733
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.140322792.140323772
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs17118812
</td>
<td style="text-align:right;">
140323701
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
140322792
</td>
<td style="text-align:right;">
140323772
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
140332843
</td>
<td style="text-align:right;">
140346603
</td>
<td style="text-align:left;">
1839
</td>
<td style="text-align:left;">
ENSG00000113070
</td>
<td style="text-align:left;">
HBEGF
</td>
<td style="text-align:right;">
9072
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.18209595.18210025
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs34969716
</td>
<td style="text-align:right;">
18209878
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
18209595
</td>
<td style="text-align:right;">
18210025
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
18155329
</td>
<td style="text-align:right;">
18223854
</td>
<td style="text-align:left;">
221656
</td>
<td style="text-align:left;">
ENSG00000165097
</td>
<td style="text-align:left;">
KDM1B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.134797479.134798124
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs4896104
</td>
<td style="text-align:right;">
134797951
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
134797479
</td>
<td style="text-align:right;">
134798124
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
134960378
</td>
<td style="text-align:right;">
135103056
</td>
<td style="text-align:left;">
10767
</td>
<td style="text-align:left;">
ENSG00000112339
</td>
<td style="text-align:left;">
HBS1L
</td>
<td style="text-align:right;">
162255
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.836506.836873
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs11768850
</td>
<td style="text-align:right;">
836590
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
836506
</td>
<td style="text-align:right;">
836873
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
816615
</td>
<td style="text-align:right;">
896435
</td>
<td style="text-align:left;">
23353
</td>
<td style="text-align:left;">
ENSG00000164828
</td>
<td style="text-align:left;">
SUN1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.14331921.14332483
</td>
<td style="text-align:left;">
Intron (ENST00000399322.7/1607, intron 22 of 24)
</td>
<td style="text-align:left;">
rs55734480
</td>
<td style="text-align:right;">
14332384
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
14331921
</td>
<td style="text-align:right;">
14332483
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
14145049
</td>
<td style="text-align:right;">
14974777
</td>
<td style="text-align:left;">
1607
</td>
<td style="text-align:left;">
ENSG00000136267
</td>
<td style="text-align:left;">
DGKB
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.92620778.92621787
</td>
<td style="text-align:left;">
Intron (ENST00000265734.8/1021, intron 6 of 7)
</td>
<td style="text-align:left;">
rs42044
</td>
<td style="text-align:right;">
92620826
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
92620778
</td>
<td style="text-align:right;">
92621787
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
92604921
</td>
<td style="text-align:right;">
92836573
</td>
<td style="text-align:left;">
1021
</td>
<td style="text-align:left;">
ENSG00000105810
</td>
<td style="text-align:left;">
CDK6
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.48898749.48899940
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs12155623
</td>
<td style="text-align:right;">
48899642
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
48898749
</td>
<td style="text-align:right;">
48899940
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
48917598
</td>
<td style="text-align:right;">
48921740
</td>
<td style="text-align:left;">
6591
</td>
<td style="text-align:left;">
ENSG00000019549
</td>
<td style="text-align:left;">
SNAI2
</td>
<td style="text-align:right;">
17659
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.123622558.123623378
</td>
<td style="text-align:left;">
Intron (ENST00000652905.1/105375738, intron 2 of 2)
</td>
<td style="text-align:left;">
rs4871397
</td>
<td style="text-align:right;">
123622957
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
123622558
</td>
<td style="text-align:right;">
123623378
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
123644442
</td>
<td style="text-align:right;">
123653801
</td>
<td style="text-align:left;">
340359
</td>
<td style="text-align:left;">
ENSG00000175946
</td>
<td style="text-align:left;">
KLHL38
</td>
<td style="text-align:right;">
21065
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.124847104.124848853
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs35006907
</td>
<td style="text-align:right;">
124847575
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
124847104
</td>
<td style="text-align:right;">
124848853
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
124550784
</td>
<td style="text-align:right;">
124728473
</td>
<td style="text-align:left;">
9788
</td>
<td style="text-align:left;">
ENSG00000170873
</td>
<td style="text-align:left;">
MTSS1
</td>
<td style="text-align:right;">
118631
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.94950339.94951513
</td>
<td style="text-align:left;">
Intron (ENST00000375315.8/84909, intron 7 of 16)
</td>
<td style="text-align:left;">
rs10821415
</td>
<td style="text-align:right;">
94951177
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
94950339
</td>
<td style="text-align:right;">
94951513
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
94726604
</td>
<td style="text-align:right;">
95151793
</td>
<td style="text-align:left;">
84909
</td>
<td style="text-align:left;">
ENSG00000148120
</td>
<td style="text-align:left;">
AOPEP
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.95006342.95007055
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs147288039
</td>
<td style="text-align:right;">
95006476
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
95006342
</td>
<td style="text-align:right;">
95007055
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
94726604
</td>
<td style="text-align:right;">
95151793
</td>
<td style="text-align:left;">
84909
</td>
<td style="text-align:left;">
ENSG00000148120
</td>
<td style="text-align:left;">
AOPEP
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.102229686.102230890
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs10786662
</td>
<td style="text-align:right;">
102230055
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
102229686
</td>
<td style="text-align:right;">
102230890
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
102245371
</td>
<td style="text-align:right;">
102382899
</td>
<td style="text-align:left;">
8729
</td>
<td style="text-align:left;">
ENSG00000107862
</td>
<td style="text-align:left;">
GBF1
</td>
<td style="text-align:right;">
14482
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.103712814.103713354
</td>
<td style="text-align:left;">
Intron (ENST00000369774.9/9644, intron 5 of 14)
</td>
<td style="text-align:left;">
rs10509768
</td>
<td style="text-align:right;">
103713072
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
103712814
</td>
<td style="text-align:right;">
103713354
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
103594027
</td>
<td style="text-align:right;">
103855584
</td>
<td style="text-align:left;">
9644
</td>
<td style="text-align:left;">
ENSG00000107957
</td>
<td style="text-align:left;">
SH3PXD2A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.128894447.128895096
</td>
<td style="text-align:left;">
Intron (ENST00000529694.6/3762, intron 1 of 2)
</td>
<td style="text-align:left;">
rs76097649
</td>
<td style="text-align:right;">
128894675
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
128894447
</td>
<td style="text-align:right;">
128895096
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
128891356
</td>
<td style="text-align:right;">
128921163
</td>
<td style="text-align:left;">
3762
</td>
<td style="text-align:left;">
ENSG00000120457
</td>
<td style="text-align:left;">
KCNJ5
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.128894447.128895096
</td>
<td style="text-align:left;">
Intron (ENST00000529694.6/3762, intron 1 of 2)
</td>
<td style="text-align:left;">
rs75190942
</td>
<td style="text-align:right;">
128894676
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
128894447
</td>
<td style="text-align:right;">
128895096
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
128891356
</td>
<td style="text-align:right;">
128921163
</td>
<td style="text-align:left;">
3762
</td>
<td style="text-align:left;">
ENSG00000120457
</td>
<td style="text-align:left;">
KCNJ5
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.24561685.24562685
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2291437
</td>
<td style="text-align:right;">
24562114
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
24561685
</td>
<td style="text-align:right;">
24562685
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
23529504
</td>
<td style="text-align:right;">
24562544
</td>
<td style="text-align:left;">
6660
</td>
<td style="text-align:left;">
ENSG00000134532
</td>
<td style="text-align:left;">
SOX5
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.26194928.26196665
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs113819537
</td>
<td style="text-align:right;">
26195496
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
26194928
</td>
<td style="text-align:right;">
26196665
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
26121991
</td>
<td style="text-align:right;">
26299290
</td>
<td style="text-align:left;">
8082
</td>
<td style="text-align:left;">
ENSG00000123096
</td>
<td style="text-align:left;">
SSPN
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.22794559.22794879
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs3904323
</td>
<td style="text-align:right;">
22794695
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
22794559
</td>
<td style="text-align:right;">
22794879
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
23180979
</td>
<td style="text-align:right;">
23325162
</td>
<td style="text-align:left;">
6445
</td>
<td style="text-align:left;">
ENSG00000102683
</td>
<td style="text-align:left;">
SGCG
</td>
<td style="text-align:right;">
386101
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.22794559.22794879
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs9506925
</td>
<td style="text-align:right;">
22794804
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
22794559
</td>
<td style="text-align:right;">
22794879
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
23180979
</td>
<td style="text-align:right;">
23325162
</td>
<td style="text-align:left;">
6445
</td>
<td style="text-align:left;">
ENSG00000102683
</td>
<td style="text-align:left;">
SGCG
</td>
<td style="text-align:right;">
386101
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.72893758.72895206
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3814864
</td>
<td style="text-align:right;">
72894562
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
72893758
</td>
<td style="text-align:right;">
72895206
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
72609034
</td>
<td style="text-align:right;">
72894101
</td>
<td style="text-align:left;">
8110
</td>
<td style="text-align:left;">
ENSG00000205683
</td>
<td style="text-align:left;">
DPF3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.100158874.100160055
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs7157599
</td>
<td style="text-align:right;">
100159565
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
100158874
</td>
<td style="text-align:right;">
100160055
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99971449
</td>
<td style="text-align:right;">
100144236
</td>
<td style="text-align:left;">
51466
</td>
<td style="text-align:left;">
ENSG00000196405
</td>
<td style="text-align:left;">
EVL
</td>
<td style="text-align:right;">
14638
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.70171390.70172398
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs2415081
</td>
<td style="text-align:right;">
70171653
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
70171390
</td>
<td style="text-align:right;">
70172398
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
70047790
</td>
<td style="text-align:right;">
70098176
</td>
<td style="text-align:left;">
7090
</td>
<td style="text-align:left;">
ENSG00000140332
</td>
<td style="text-align:left;">
TLE3
</td>
<td style="text-align:right;">
73214
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.73375634.73377363
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7178084
</td>
<td style="text-align:right;">
73375705
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73375634
</td>
<td style="text-align:right;">
73377363
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73319859
</td>
<td style="text-align:right;">
73368958
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
ENSG00000138622
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:right;">
6676
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.73375634.73377363
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7183206
</td>
<td style="text-align:right;">
73376665
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73375634
</td>
<td style="text-align:right;">
73377363
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73319859
</td>
<td style="text-align:right;">
73368958
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
ENSG00000138622
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:right;">
6676
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.98727491.98728563
</td>
<td style="text-align:left;">
Intron (ENST00000649865.1/3480, intron 2 of 20)
</td>
<td style="text-align:left;">
rs6598541
</td>
<td style="text-align:right;">
98727906
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
98727491
</td>
<td style="text-align:right;">
98728563
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
98648539
</td>
<td style="text-align:right;">
98964530
</td>
<td style="text-align:left;">
3480
</td>
<td style="text-align:left;">
ENSG00000140443
</td>
<td style="text-align:left;">
IGF1R
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.1963596.1965464
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2286466
</td>
<td style="text-align:right;">
1964282
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
1963596
</td>
<td style="text-align:right;">
1965464
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
1962058
</td>
<td style="text-align:right;">
1964841
</td>
<td style="text-align:left;">
6187
</td>
<td style="text-align:left;">
ENSG00000140988
</td>
<td style="text-align:left;">
RPS2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.73018778.73020183
</td>
<td style="text-align:left;">
Intron (ENST00000268489.10/463, intron 1 of 9)
</td>
<td style="text-align:left;">
rs2359171
</td>
<td style="text-align:right;">
73019123
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
73018778
</td>
<td style="text-align:right;">
73020183
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
72782885
</td>
<td style="text-align:right;">
73891871
</td>
<td style="text-align:left;">
463
</td>
<td style="text-align:left;">
ENSG00000140836
</td>
<td style="text-align:left;">
ZFHX3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.2867997.2868307
</td>
<td style="text-align:left;">
Intron (ENST00000637138.1/23108, intron 3 of 25)
</td>
<td style="text-align:left;">
rs12603284
</td>
<td style="text-align:right;">
2868218
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
2867997
</td>
<td style="text-align:right;">
2868307
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
2755705
</td>
<td style="text-align:right;">
3037741
</td>
<td style="text-align:left;">
23108
</td>
<td style="text-align:left;">
ENSG00000132359
</td>
<td style="text-align:left;">
RAP1GAP2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.7548665.7550717
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs9899183
</td>
<td style="text-align:right;">
7549660
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
7548665
</td>
<td style="text-align:right;">
7550717
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
7561919
</td>
<td style="text-align:right;">
7571969
</td>
<td style="text-align:left;">
26168
</td>
<td style="text-align:left;">
ENSG00000161956
</td>
<td style="text-align:left;">
SENP3
</td>
<td style="text-align:right;">
11203
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.79396485.79396813
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs8096658
</td>
<td style="text-align:right;">
79396537
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
79396485
</td>
<td style="text-align:right;">
79396813
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
79395856
</td>
<td style="text-align:right;">
79529325
</td>
<td style="text-align:left;">
4772
</td>
<td style="text-align:left;">
ENSG00000131196
</td>
<td style="text-align:left;">
NFATC1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.29151834.29152614
</td>
<td style="text-align:left;">
Intron (ENST00000710354.1/56911, intron 7 of 8)
</td>
<td style="text-align:left;">
rs3787662
</td>
<td style="text-align:right;">
29151972
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
29151834
</td>
<td style="text-align:right;">
29152614
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
29077221
</td>
<td style="text-align:right;">
29175889
</td>
<td style="text-align:left;">
56911
</td>
<td style="text-align:left;">
ENSG00000156265
</td>
<td style="text-align:left;">
MAP3K7CL
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chrX.138708359.138708598
</td>
<td style="text-align:left;">
Intron (ENST00000315930.11/2258, intron 2 of 4)
</td>
<td style="text-align:left;">
rs778479352
</td>
<td style="text-align:right;">
138708418
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chrX
</td>
<td style="text-align:right;">
138708359
</td>
<td style="text-align:right;">
138708598
</td>
<td style="text-align:left;">
.
</td>
<td style="text-align:right;">
-1
</td>
<td style="text-align:right;">
-1
</td>
<td style="text-align:left;">
.
</td>
<td style="text-align:left;">
.
</td>
<td style="text-align:left;">
.
</td>
<td style="text-align:right;">
-1
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>ARR_SNP_table %&gt;%  
 # rename(., &quot;peak_id&quot;=id) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  write.csv(.,&quot;data/ARR_SNP_table.csv&quot;)</code></pre>
</div>
<div id="myocardial-infarction" class="section level3">
<h3>Myocardial Infarction</h3>
<pre class="r"><code>## Heart failure is gwas #5

EAR_MI_SNP &lt;- as.data.frame(join_overlap_intersect(EAR_df_gr,MI_gwas_gr_noext))
ESR_MI_SNP &lt;- as.data.frame(join_overlap_intersect(ESR_df_gr,MI_gwas_gr_noext))
LR_MI_SNP &lt;- as.data.frame(join_overlap_intersect(LR_df_gr, MI_gwas_gr_noext))
NR_MI_SNP &lt;- as.data.frame(join_overlap_intersect(NR_df_gr, MI_gwas_gr_noext))

MI_SNP_table &lt;- EAR_MI_SNP %&gt;% 
  mutate(MRC=&quot;EAR&quot;) %&gt;% 
  bind_rows((ESR_MI_SNP %&gt;% 
              mutate(MRC=&quot;ESR&quot;))) %&gt;% 
  bind_rows((LR_MI_SNP %&gt;% 
              mutate(MRC=&quot;LR&quot;))) %&gt;% 
  bind_rows((NR_MI_SNP %&gt;% 
              mutate(MRC=&quot;NR&quot;)))  %&gt;% 
  distinct()
MI_SNP_table %&gt;%  
 # rename( id=&quot;peak_id&quot;) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  kable(., caption= &quot;SNPs for MI found in MRC peaks&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs for MI found in MRC peaks
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
annotation
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CHR_POS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MRC
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NG_chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dist_to_NG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr6.73694726.73695115
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs79717953
</td>
<td style="text-align:right;">
73695048
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
73694726
</td>
<td style="text-align:right;">
73695115
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
73695785
</td>
<td style="text-align:right;">
73828316
</td>
<td style="text-align:left;">
135228
</td>
<td style="text-align:left;">
ENSG00000156535
</td>
<td style="text-align:left;">
CD109
</td>
<td style="text-align:right;">
671
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs12740374
</td>
<td style="text-align:right;">
109274968
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs646776
</td>
<td style="text-align:right;">
109275908
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.99657657.99657978
</td>
<td style="text-align:left;">
Intron (ENST00000330710.10/84439, intron 3 of 8)
</td>
<td style="text-align:left;">
rs12897285
</td>
<td style="text-align:right;">
99657718
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99657657
</td>
<td style="text-align:right;">
99657978
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99645129
</td>
<td style="text-align:right;">
99680569
</td>
<td style="text-align:left;">
84439
</td>
<td style="text-align:left;">
ENSG00000182218
</td>
<td style="text-align:left;">
HHIPL1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7165042
</td>
<td style="text-align:right;">
78830996
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.86664502.86665119
</td>
<td style="text-align:left;">
Intron (ENST00000565822.1/ENST00000565822.1, intron 2 of 2)
</td>
<td style="text-align:left;">
rs62042066
</td>
<td style="text-align:right;">
86664884
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
86664502
</td>
<td style="text-align:right;">
86665119
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
86576368
</td>
<td style="text-align:right;">
86583478
</td>
<td style="text-align:left;">
2300
</td>
<td style="text-align:left;">
ENSG00000176678
</td>
<td style="text-align:left;">
FOXL1
</td>
<td style="text-align:right;">
81024
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.46940653.46941215
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs1052586
</td>
<td style="text-align:right;">
46941097
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
46940653
</td>
<td style="text-align:right;">
46941215
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
46923075
</td>
<td style="text-align:right;">
46975524
</td>
<td style="text-align:left;">
9570
</td>
<td style="text-align:left;">
ENSG00000108433
</td>
<td style="text-align:left;">
GOSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.17747415.17747946
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs12983897
</td>
<td style="text-align:right;">
17747755
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
17747415
</td>
<td style="text-align:right;">
17747946
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
17719242
</td>
<td style="text-align:right;">
17734513
</td>
<td style="text-align:left;">
55201
</td>
<td style="text-align:left;">
ENSG00000130479
</td>
<td style="text-align:left;">
MAP1S
</td>
<td style="text-align:right;">
12902
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.44908220.44909100
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs429358
</td>
<td style="text-align:right;">
44908684
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44908220
</td>
<td style="text-align:right;">
44909100
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44905791
</td>
<td style="text-align:right;">
44909393
</td>
<td style="text-align:left;">
348
</td>
<td style="text-align:left;">
ENSG00000130203
</td>
<td style="text-align:left;">
APOE
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.44908220.44909100
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs7412
</td>
<td style="text-align:right;">
44908822
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44908220
</td>
<td style="text-align:right;">
44909100
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44905791
</td>
<td style="text-align:right;">
44909393
</td>
<td style="text-align:left;">
348
</td>
<td style="text-align:left;">
ENSG00000130203
</td>
<td style="text-align:left;">
APOE
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr22.24262224.24262913
</td>
<td style="text-align:left;">
Exon (ENST00000414583.6/29774, exon 6 of 7)
</td>
<td style="text-align:left;">
rs180803
</td>
<td style="text-align:right;">
24262890
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
24262224
</td>
<td style="text-align:right;">
24262913
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
24270817
</td>
<td style="text-align:right;">
24417739
</td>
<td style="text-align:left;">
23384
</td>
<td style="text-align:left;">
ENSG00000100014
</td>
<td style="text-align:left;">
SPECC1L
</td>
<td style="text-align:right;">
7905
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.27601778.27602613
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs113716316
</td>
<td style="text-align:right;">
27602129
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
27601778
</td>
<td style="text-align:right;">
27602613
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
27534035
</td>
<td style="text-align:right;">
27604431
</td>
<td style="text-align:left;">
27245
</td>
<td style="text-align:left;">
ENSG00000126705
</td>
<td style="text-align:left;">
AHDC1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.37931148.37932171
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs35267671
</td>
<td style="text-align:right;">
37931697
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
37931148
</td>
<td style="text-align:right;">
37932171
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
37860697
</td>
<td style="text-align:right;">
37947053
</td>
<td style="text-align:left;">
3633
</td>
<td style="text-align:left;">
ENSG00000204084
</td>
<td style="text-align:left;">
INPP5B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.55039222.55040233
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11591147
</td>
<td style="text-align:right;">
55039974
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
55039222
</td>
<td style="text-align:right;">
55040233
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
55066359
</td>
<td style="text-align:right;">
55215378
</td>
<td style="text-align:left;">
23358
</td>
<td style="text-align:left;">
ENSG00000162402
</td>
<td style="text-align:left;">
USP24
</td>
<td style="text-align:right;">
26127
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.113630172.113630986
</td>
<td style="text-align:left;">
Intron (ENST00000307546.14/260425, intron 9 of 20)
</td>
<td style="text-align:left;">
rs1230666
</td>
<td style="text-align:right;">
113630788
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
113630172
</td>
<td style="text-align:right;">
113630986
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
113390515
</td>
<td style="text-align:right;">
113685923
</td>
<td style="text-align:left;">
260425
</td>
<td style="text-align:left;">
ENSG00000081026
</td>
<td style="text-align:left;">
MAGI3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.62727824.62728333
</td>
<td style="text-align:left;">
Intron (ENST00000405015.7/23301, intron 2 of 22)
</td>
<td style="text-align:left;">
rs360801
</td>
<td style="text-align:right;">
62728252
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
62727824
</td>
<td style="text-align:right;">
62728333
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
62673851
</td>
<td style="text-align:right;">
63046487
</td>
<td style="text-align:left;">
23301
</td>
<td style="text-align:left;">
ENSG00000115504
</td>
<td style="text-align:left;">
EHBP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.215435176.215436848
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1250258
</td>
<td style="text-align:right;">
215435462
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
215435176
</td>
<td style="text-align:right;">
215436848
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
215360440
</td>
<td style="text-align:right;">
215436073
</td>
<td style="text-align:left;">
2335
</td>
<td style="text-align:left;">
ENSG00000115414
</td>
<td style="text-align:left;">
FN1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.121651049.121653070
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs10818579
</td>
<td style="text-align:right;">
121652626
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
121651049
</td>
<td style="text-align:right;">
121653070
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
121567057
</td>
<td style="text-align:right;">
121785530
</td>
<td style="text-align:left;">
153090
</td>
<td style="text-align:left;">
ENSG00000136848
</td>
<td style="text-align:left;">
DAB2IP
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.110366109.110367343
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs75112503
</td>
<td style="text-align:right;">
110366813
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
110366109
</td>
<td style="text-align:right;">
110367343
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
110429948
</td>
<td style="text-align:right;">
110464884
</td>
<td style="text-align:left;">
2230
</td>
<td style="text-align:left;">
ENSG00000137714
</td>
<td style="text-align:left;">
FDX1
</td>
<td style="text-align:right;">
62606
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.20004923.20005338
</td>
<td style="text-align:left;">
Intron (ENST00000535764.1/ENST00000535764.1, intron 3 of 4)
</td>
<td style="text-align:left;">
rs11044977
</td>
<td style="text-align:right;">
20005226
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
20004923
</td>
<td style="text-align:right;">
20005338
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
19404045
</td>
<td style="text-align:right;">
19720801
</td>
<td style="text-align:left;">
121536
</td>
<td style="text-align:left;">
ENSG00000139154
</td>
<td style="text-align:left;">
AEBP2
</td>
<td style="text-align:right;">
284122
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.111569574.111570394
</td>
<td style="text-align:left;">
Intron (ENST00000642389.2/6311, intron 1 of 26)
</td>
<td style="text-align:left;">
rs653178
</td>
<td style="text-align:right;">
111569952
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
111569574
</td>
<td style="text-align:right;">
111570394
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
111443485
</td>
<td style="text-align:right;">
111599676
</td>
<td style="text-align:left;">
6311
</td>
<td style="text-align:left;">
ENSG00000204842
</td>
<td style="text-align:left;">
ATXN2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.67071010.67071262
</td>
<td style="text-align:left;">
Intron (ENST00000559460.6/4088, intron 1 of 8)
</td>
<td style="text-align:left;">
rs11632964
</td>
<td style="text-align:right;">
67071235
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
67071010
</td>
<td style="text-align:right;">
67071262
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
67063763
</td>
<td style="text-align:right;">
67195173
</td>
<td style="text-align:left;">
4088
</td>
<td style="text-align:left;">
ENSG00000166949
</td>
<td style="text-align:left;">
SMAD3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.8363631.8364760
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs116843064
</td>
<td style="text-align:right;">
8364439
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
8363631
</td>
<td style="text-align:right;">
8364760
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
8363289
</td>
<td style="text-align:right;">
8374370
</td>
<td style="text-align:left;">
51129
</td>
<td style="text-align:left;">
ENSG00000167772
</td>
<td style="text-align:left;">
ANGPTL4
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.11052376.11052994
</td>
<td style="text-align:left;">
Intron (ENST00000643549.1/6597, intron 30 of 34)
</td>
<td style="text-align:left;">
rs1122608
</td>
<td style="text-align:right;">
11052925
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
11052376
</td>
<td style="text-align:right;">
11052994
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
10960932
</td>
<td style="text-align:right;">
11079426
</td>
<td style="text-align:left;">
6597
</td>
<td style="text-align:left;">
ENSG00000127616
</td>
<td style="text-align:left;">
SMARCA4
</td>
<td style="text-align:right;">
0
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="coronary-artery-disease" class="section level3">
<h3>Coronary Artery Disease</h3>
<pre class="r"><code>## Heart failure is gwas #5

EAR_CAD_SNP &lt;- as.data.frame(join_overlap_intersect(EAR_df_gr,CAD_gwas_gr_noext))
ESR_CAD_SNP &lt;- as.data.frame(join_overlap_intersect(ESR_df_gr,CAD_gwas_gr_noext))
LR_CAD_SNP &lt;- as.data.frame(join_overlap_intersect(LR_df_gr, CAD_gwas_gr_noext))
NR_CAD_SNP &lt;- as.data.frame(join_overlap_intersect(NR_df_gr, CAD_gwas_gr_noext))

CAD_SNP_table &lt;- EAR_CAD_SNP %&gt;% 
  mutate(MRC=&quot;EAR&quot;) %&gt;% 
  bind_rows((ESR_CAD_SNP %&gt;% 
              mutate(MRC=&quot;ESR&quot;))) %&gt;% 
  bind_rows((LR_CAD_SNP %&gt;% 
              mutate(MRC=&quot;LR&quot;))) %&gt;% 
  bind_rows((NR_CAD_SNP %&gt;% 
              mutate(MRC=&quot;NR&quot;))) %&gt;% 
  distinct()
CAD_SNP_table %&gt;% 
  distinct() %&gt;% 
 # rename( id=&quot;peak_id&quot;) %&gt;% 
  select(id, annotation,SNPS,CHR_POS,MRC) %&gt;% 
  left_join(., exp_neargene_table, by=c(&quot;id&quot;=&quot;peakid&quot;)) %&gt;% 
  kable(., caption= &quot;SNPs for CAD found in MRC peaks&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs for CAD found in MRC peaks
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
annotation
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CHR_POS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MRC
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NG_chr
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_start
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
NG_end
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ensembl_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dist_to_NG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr2.217804449.217806257
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs61741262
</td>
<td style="text-align:right;">
217804502
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
217804449
</td>
<td style="text-align:right;">
217806257
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
217799588
</td>
<td style="text-align:right;">
218033982
</td>
<td style="text-align:left;">
7145
</td>
<td style="text-align:left;">
ENSG00000079308
</td>
<td style="text-align:left;">
TNS1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.94575649.94576017
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs2452600
</td>
<td style="text-align:right;">
94575731
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
94575649
</td>
<td style="text-align:right;">
94576017
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
94451857
</td>
<td style="text-align:right;">
94668227
</td>
<td style="text-align:left;">
10611
</td>
<td style="text-align:left;">
ENSG00000163110
</td>
<td style="text-align:left;">
PDLIM5
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.36156365.36157193
</td>
<td style="text-align:left;">
Intron (ENST00000274254.9/6502, intron 2 of 9)
</td>
<td style="text-align:left;">
rs5867305
</td>
<td style="text-align:right;">
36157161
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
36156365
</td>
<td style="text-align:right;">
36157193
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
36151989
</td>
<td style="text-align:right;">
36196849
</td>
<td style="text-align:left;">
6502
</td>
<td style="text-align:left;">
ENSG00000145604
</td>
<td style="text-align:left;">
SKP2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.73694726.73695115
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs79717953
</td>
<td style="text-align:right;">
73695048
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
73694726
</td>
<td style="text-align:right;">
73695115
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
73695785
</td>
<td style="text-align:right;">
73828316
</td>
<td style="text-align:left;">
135228
</td>
<td style="text-align:left;">
ENSG00000156535
</td>
<td style="text-align:left;">
CD109
</td>
<td style="text-align:right;">
671
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.36459049.36459918
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs192407614
</td>
<td style="text-align:right;">
36459695
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
36459049
</td>
<td style="text-align:right;">
36459918
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
36389821
</td>
<td style="text-align:right;">
36453791
</td>
<td style="text-align:left;">
54443
</td>
<td style="text-align:left;">
ENSG00000011426
</td>
<td style="text-align:left;">
ANLN
</td>
<td style="text-align:right;">
5258
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.100243501.100243833
</td>
<td style="text-align:left;">
Intron (ENST00000328453.9/352954, intron 1 of 7)
</td>
<td style="text-align:left;">
rs117038461
</td>
<td style="text-align:right;">
100243731
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
100243501
</td>
<td style="text-align:right;">
100243833
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
100218241
</td>
<td style="text-align:right;">
100221490
</td>
<td style="text-align:left;">
79037
</td>
<td style="text-align:left;">
ENSG00000213413
</td>
<td style="text-align:left;">
PVRIG
</td>
<td style="text-align:right;">
22011
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.58437614.58438322
</td>
<td style="text-align:left;">
Intron (ENST00000356113.10/3990, intron 3 of 10)
</td>
<td style="text-align:left;">
rs588136
</td>
<td style="text-align:right;">
58438299
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
58437614
</td>
<td style="text-align:right;">
58438322
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
58588809
</td>
<td style="text-align:right;">
58749791
</td>
<td style="text-align:left;">
102
</td>
<td style="text-align:left;">
ENSG00000137845
</td>
<td style="text-align:left;">
ADAM10
</td>
<td style="text-align:right;">
150488
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.46935408.46936158
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs17608766
</td>
<td style="text-align:right;">
46935905
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
46935408
</td>
<td style="text-align:right;">
46936158
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
46923075
</td>
<td style="text-align:right;">
46975524
</td>
<td style="text-align:left;">
9570
</td>
<td style="text-align:left;">
ENSG00000108433
</td>
<td style="text-align:left;">
GOSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.11142349.11143639
</td>
<td style="text-align:left;">
Intron (ENST00000585567.5/147841, intron 4 of 5)
</td>
<td style="text-align:left;">
rs34774090
</td>
<td style="text-align:right;">
11143211
</td>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
11142349
</td>
<td style="text-align:right;">
11143639
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
11131520
</td>
<td style="text-align:right;">
11155807
</td>
<td style="text-align:left;">
147841
</td>
<td style="text-align:left;">
ENSG00000161888
</td>
<td style="text-align:left;">
SPC24
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.151483793.151484643
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs4383490
</td>
<td style="text-align:right;">
151483898
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
151483793
</td>
<td style="text-align:right;">
151484643
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
151425384
</td>
<td style="text-align:right;">
151461061
</td>
<td style="text-align:left;">
285313
</td>
<td style="text-align:left;">
ENSG00000152580
</td>
<td style="text-align:left;">
IGSF10
</td>
<td style="text-align:right;">
22732
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.151483793.151484643
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs4518099
</td>
<td style="text-align:right;">
151483986
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
151483793
</td>
<td style="text-align:right;">
151484643
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
151425384
</td>
<td style="text-align:right;">
151461061
</td>
<td style="text-align:left;">
285313
</td>
<td style="text-align:left;">
ENSG00000152580
</td>
<td style="text-align:left;">
IGSF10
</td>
<td style="text-align:right;">
22732
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.151483793.151484643
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs4387942
</td>
<td style="text-align:right;">
151484399
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
151483793
</td>
<td style="text-align:right;">
151484643
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
151425384
</td>
<td style="text-align:right;">
151461061
</td>
<td style="text-align:left;">
285313
</td>
<td style="text-align:left;">
ENSG00000152580
</td>
<td style="text-align:left;">
IGSF10
</td>
<td style="text-align:right;">
22732
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.110334421.110335161
</td>
<td style="text-align:left;">
Intron (ENST00000338882.5/728464, intron 1 of 4)
</td>
<td style="text-align:left;">
rs72935945
</td>
<td style="text-align:right;">
110334472
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
110334421
</td>
<td style="text-align:right;">
110335161
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
110180141
</td>
<td style="text-align:right;">
110254275
</td>
<td style="text-align:left;">
51362
</td>
<td style="text-align:left;">
ENSG00000168438
</td>
<td style="text-align:left;">
CDC40
</td>
<td style="text-align:right;">
80146
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.121650071.121650712
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs10818576
</td>
<td style="text-align:right;">
121650669
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
121650071
</td>
<td style="text-align:right;">
121650712
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
121567057
</td>
<td style="text-align:right;">
121785530
</td>
<td style="text-align:left;">
153090
</td>
<td style="text-align:left;">
ENSG00000136848
</td>
<td style="text-align:left;">
DAB2IP
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.44261009.44262169
</td>
<td style="text-align:left;">
Exon (ENST00000437014.1/ENST00000437014.1, exon 1 of 2)
</td>
<td style="text-align:left;">
rs523297
</td>
<td style="text-align:right;">
44261109
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
44261009
</td>
<td style="text-align:right;">
44262169
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
44370165
</td>
<td style="text-align:right;">
44386493
</td>
<td style="text-align:left;">
6387
</td>
<td style="text-align:left;">
ENSG00000107562
</td>
<td style="text-align:left;">
CXCL12
</td>
<td style="text-align:right;">
107997
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.44261009.44262169
</td>
<td style="text-align:left;">
Exon (ENST00000437014.1/ENST00000437014.1, exon 1 of 2)
</td>
<td style="text-align:left;">
rs518594
</td>
<td style="text-align:right;">
44261659
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
44261009
</td>
<td style="text-align:right;">
44262169
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
44370165
</td>
<td style="text-align:right;">
44386493
</td>
<td style="text-align:left;">
6387
</td>
<td style="text-align:left;">
ENSG00000107562
</td>
<td style="text-align:left;">
CXCL12
</td>
<td style="text-align:right;">
107997
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78726891.78727425
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs899997
</td>
<td style="text-align:right;">
78727236
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78726891
</td>
<td style="text-align:right;">
78727425
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78759206
</td>
<td style="text-align:right;">
78811464
</td>
<td style="text-align:left;">
11173
</td>
<td style="text-align:left;">
ENSG00000136378
</td>
<td style="text-align:left;">
ADAMTS7
</td>
<td style="text-align:right;">
31782
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.17823556.17824258
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs12936927
</td>
<td style="text-align:right;">
17823651
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
17823556
</td>
<td style="text-align:right;">
17824258
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
17810399
</td>
<td style="text-align:right;">
17837002
</td>
<td style="text-align:left;">
6720
</td>
<td style="text-align:left;">
ENSG00000072310
</td>
<td style="text-align:left;">
SREBF1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.80795708.80796334
</td>
<td style="text-align:left;">
Intron (ENST00000574767.5/57521, intron 6 of 11)
</td>
<td style="text-align:left;">
rs729743
</td>
<td style="text-align:right;">
80796014
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
80795708
</td>
<td style="text-align:right;">
80796334
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
80544819
</td>
<td style="text-align:right;">
80966371
</td>
<td style="text-align:left;">
57521
</td>
<td style="text-align:left;">
ENSG00000141564
</td>
<td style="text-align:left;">
RPTOR
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41324959.41326640
</td>
<td style="text-align:left;">
Intron (ENST00000598758.5/7040, intron 2 of 3)
</td>
<td style="text-align:left;">
rs12980942
</td>
<td style="text-align:right;">
41326326
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41324959
</td>
<td style="text-align:right;">
41326640
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41301587
</td>
<td style="text-align:right;">
41353922
</td>
<td style="text-align:left;">
7040
</td>
<td style="text-align:left;">
ENSG00000105329
</td>
<td style="text-align:left;">
TGFB1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.44911105.44911471
</td>
<td style="text-align:left;">
Exon (ENST00000623895.1/ENST00000623895.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs72654473
</td>
<td style="text-align:right;">
44911142
</td>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44911105
</td>
<td style="text-align:right;">
44911471
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44905791
</td>
<td style="text-align:right;">
44909393
</td>
<td style="text-align:left;">
348
</td>
<td style="text-align:left;">
ENSG00000130203
</td>
<td style="text-align:left;">
APOE
</td>
<td style="text-align:right;">
1712
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.3408398.3409463
</td>
<td style="text-align:left;">
Intron (ENST00000511072.5/63976, intron 8 of 15)
</td>
<td style="text-align:left;">
rs2493298
</td>
<td style="text-align:right;">
3409348
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
3408398
</td>
<td style="text-align:right;">
3409463
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
3069168
</td>
<td style="text-align:right;">
3438621
</td>
<td style="text-align:left;">
63976
</td>
<td style="text-align:left;">
ENSG00000142611
</td>
<td style="text-align:left;">
PRDM16
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.56521199.56521723
</td>
<td style="text-align:left;">
Intron (ENST00000641109.1/124904185, intron 1 of 5)
</td>
<td style="text-align:left;">
rs72664335
</td>
<td style="text-align:right;">
56521306
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
56521199
</td>
<td style="text-align:right;">
56521723
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
56494761
</td>
<td style="text-align:right;">
56645301
</td>
<td style="text-align:left;">
8613
</td>
<td style="text-align:left;">
ENSG00000162407
</td>
<td style="text-align:left;">
PLPP3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs12740374
</td>
<td style="text-align:right;">
109274968
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs660240
</td>
<td style="text-align:right;">
109275216
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs629301
</td>
<td style="text-align:right;">
109275684
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs646776
</td>
<td style="text-align:right;">
109275908
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109274953
</td>
<td style="text-align:right;">
109276202
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
109249539
</td>
<td style="text-align:right;">
109275751
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
ENSG00000143126
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.24818716.24819139
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs1550115
</td>
<td style="text-align:right;">
24818751
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
24818716
</td>
<td style="text-align:right;">
24819139
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
24793136
</td>
<td style="text-align:right;">
24822376
</td>
<td style="text-align:left;">
79172
</td>
<td style="text-align:left;">
ENSG00000138092
</td>
<td style="text-align:left;">
CENPO
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.39631404.39632344
</td>
<td style="text-align:left;">
Intron (ENST00000415640.1/728730, intron 1 of 2)
</td>
<td style="text-align:left;">
rs12712649
</td>
<td style="text-align:right;">
39631458
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
39631404
</td>
<td style="text-align:right;">
39632344
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
39664982
</td>
<td style="text-align:right;">
39717963
</td>
<td style="text-align:left;">
130733
</td>
<td style="text-align:left;">
ENSG00000152154
</td>
<td style="text-align:left;">
TMEM178A
</td>
<td style="text-align:right;">
32639
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.43184483.43184778
</td>
<td style="text-align:left;">
Exon (ENST00000705257.1/100506047, exon 2 of 3)
</td>
<td style="text-align:left;">
rs2052923
</td>
<td style="text-align:right;">
43184681
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
43184483
</td>
<td style="text-align:right;">
43184778
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
43222402
</td>
<td style="text-align:right;">
43226606
</td>
<td style="text-align:left;">
678
</td>
<td style="text-align:left;">
ENSG00000152518
</td>
<td style="text-align:left;">
ZFP36L2
</td>
<td style="text-align:right;">
37625
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.45668749.45669787
</td>
<td style="text-align:left;">
Intron (ENST00000421201.1/5581, intron 2 of 2)
</td>
<td style="text-align:left;">
rs582384
</td>
<td style="text-align:right;">
45669298
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
45668749
</td>
<td style="text-align:right;">
45669787
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
45651345
</td>
<td style="text-align:right;">
46187990
</td>
<td style="text-align:left;">
5581
</td>
<td style="text-align:left;">
ENSG00000171132
</td>
<td style="text-align:left;">
PRKCE
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.60391661.60392192
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs243071
</td>
<td style="text-align:right;">
60391893
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
60391661
</td>
<td style="text-align:right;">
60392192
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
60450520
</td>
<td style="text-align:right;">
60554467
</td>
<td style="text-align:left;">
53335
</td>
<td style="text-align:left;">
ENSG00000119866
</td>
<td style="text-align:left;">
BCL11A
</td>
<td style="text-align:right;">
58329
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.217868752.217869273
</td>
<td style="text-align:left;">
Intron (ENST00000646520.1/7145, intron 18 of 32)
</td>
<td style="text-align:left;">
rs112941127
</td>
<td style="text-align:right;">
217869242
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
217868752
</td>
<td style="text-align:right;">
217869273
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
217799588
</td>
<td style="text-align:right;">
218033982
</td>
<td style="text-align:left;">
7145
</td>
<td style="text-align:left;">
ENSG00000079308
</td>
<td style="text-align:left;">
TNS1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.237310830.237311051
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7580831
</td>
<td style="text-align:right;">
237310856
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
237310830
</td>
<td style="text-align:right;">
237311051
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
237324003
</td>
<td style="text-align:right;">
237414328
</td>
<td style="text-align:left;">
1293
</td>
<td style="text-align:left;">
ENSG00000163359
</td>
<td style="text-align:left;">
COL6A3
</td>
<td style="text-align:right;">
12953
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.237315068.237315792
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs11677932
</td>
<td style="text-align:right;">
237315312
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
237315068
</td>
<td style="text-align:right;">
237315792
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
237324003
</td>
<td style="text-align:right;">
237414328
</td>
<td style="text-align:left;">
1293
</td>
<td style="text-align:left;">
ENSG00000163359
</td>
<td style="text-align:left;">
COL6A3
</td>
<td style="text-align:right;">
8212
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.46646587.46647198
</td>
<td style="text-align:left;">
Exon (ENST00000669501.1/ENST00000669501.1, exon 5 of 5)
</td>
<td style="text-align:left;">
rs7633770
</td>
<td style="text-align:right;">
46647072
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
46646587
</td>
<td style="text-align:right;">
46647198
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
46668995
</td>
<td style="text-align:right;">
46693704
</td>
<td style="text-align:left;">
259173
</td>
<td style="text-align:left;">
ENSG00000178038
</td>
<td style="text-align:left;">
ALS2CL
</td>
<td style="text-align:right;">
21798
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.48151277.48152429
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs7617773
</td>
<td style="text-align:right;">
48152025
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
48151277
</td>
<td style="text-align:right;">
48152429
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
48157146
</td>
<td style="text-align:right;">
48188417
</td>
<td style="text-align:left;">
993
</td>
<td style="text-align:left;">
ENSG00000164045
</td>
<td style="text-align:left;">
CDC25A
</td>
<td style="text-align:right;">
4718
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.155512047.155512545
</td>
<td style="text-align:left;">
Exon (ENST00000693509.1/105377505, exon 2 of 2)
</td>
<td style="text-align:left;">
rs7440763
</td>
<td style="text-align:right;">
155512368
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
155512047
</td>
<td style="text-align:right;">
155512545
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
155342658
</td>
<td style="text-align:right;">
155376970
</td>
<td style="text-align:left;">
79884
</td>
<td style="text-align:left;">
ENSG00000164114
</td>
<td style="text-align:left;">
MAP9
</td>
<td style="text-align:right;">
135077
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.168766280.168767270
</td>
<td style="text-align:left;">
Intron (ENST00000505667.6/23022, intron 10 of 21)
</td>
<td style="text-align:left;">
rs7696431
</td>
<td style="text-align:right;">
168766574
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
168766280
</td>
<td style="text-align:right;">
168767270
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
168497052
</td>
<td style="text-align:right;">
168928457
</td>
<td style="text-align:left;">
23022
</td>
<td style="text-align:left;">
ENSG00000129116
</td>
<td style="text-align:left;">
PALLD
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.168766280.168767270
</td>
<td style="text-align:left;">
Intron (ENST00000505667.6/23022, intron 10 of 21)
</td>
<td style="text-align:left;">
rs869396
</td>
<td style="text-align:right;">
168766849
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
168766280
</td>
<td style="text-align:right;">
168767270
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
168497052
</td>
<td style="text-align:right;">
168928457
</td>
<td style="text-align:left;">
23022
</td>
<td style="text-align:left;">
ENSG00000129116
</td>
<td style="text-align:left;">
PALLD
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.17118580.17119099
</td>
<td style="text-align:left;">
Exon (ENST00000665920.1/285696, exon 3 of 3)
</td>
<td style="text-align:left;">
rs1032763
</td>
<td style="text-align:right;">
17118821
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
17118580
</td>
<td style="text-align:right;">
17119099
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
17065598
</td>
<td style="text-align:right;">
17276843
</td>
<td style="text-align:left;">
10409
</td>
<td style="text-align:left;">
ENSG00000176788
</td>
<td style="text-align:left;">
BASP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.122076732.122078641
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1800449
</td>
<td style="text-align:right;">
122077513
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
122076732
</td>
<td style="text-align:right;">
122078641
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
122063195
</td>
<td style="text-align:right;">
122078413
</td>
<td style="text-align:left;">
4015
</td>
<td style="text-align:left;">
ENSG00000113083
</td>
<td style="text-align:left;">
LOX
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.28269507.28270003
</td>
<td style="text-align:left;">
Intron (ENST00000316606.10/7741, intron 1 of 3)
</td>
<td style="text-align:left;">
rs1225600
</td>
<td style="text-align:right;">
28269667
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
28269507
</td>
<td style="text-align:right;">
28270003
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
28267058
</td>
<td style="text-align:right;">
28278224
</td>
<td style="text-align:left;">
7741
</td>
<td style="text-align:left;">
ENSG00000197062
</td>
<td style="text-align:left;">
ZSCAN26
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.39214962.39215733
</td>
<td style="text-align:left;">
Intron (ENST00000359534.4/8645, intron 1 of 4)
</td>
<td style="text-align:left;">
rs10456100
</td>
<td style="text-align:right;">
39215694
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
39214962
</td>
<td style="text-align:right;">
39215733
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
39104063
</td>
<td style="text-align:right;">
39115193
</td>
<td style="text-align:left;">
55776
</td>
<td style="text-align:left;">
ENSG00000112167
</td>
<td style="text-align:left;">
SAYSD1
</td>
<td style="text-align:right;">
99769
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.99822872.99823551
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs62471956
</td>
<td style="text-align:right;">
99823462
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
99822872
</td>
<td style="text-align:right;">
99823551
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
99876958
</td>
<td style="text-align:right;">
99919531
</td>
<td style="text-align:left;">
89122
</td>
<td style="text-align:left;">
ENSG00000146833
</td>
<td style="text-align:left;">
TRIM4
</td>
<td style="text-align:right;">
53408
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.104903653.104904005
</td>
<td style="text-align:left;">
5’ UTR
</td>
<td style="text-align:left;">
rs1800978
</td>
<td style="text-align:right;">
104903697
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
104903653
</td>
<td style="text-align:right;">
104904005
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
104781006
</td>
<td style="text-align:right;">
104928155
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
ENSG00000165029
</td>
<td style="text-align:left;">
ABCA1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.107755276.107755816
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs944172
</td>
<td style="text-align:right;">
107755513
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
107755276
</td>
<td style="text-align:right;">
107755816
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
107484852
</td>
<td style="text-align:right;">
107490482
</td>
<td style="text-align:left;">
9314
</td>
<td style="text-align:left;">
ENSG00000136826
</td>
<td style="text-align:left;">
KLF4
</td>
<td style="text-align:right;">
264794
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.116482574.116483181
</td>
<td style="text-align:left;">
Intron (ENST00000313400.9/23245, intron 20 of 22)
</td>
<td style="text-align:left;">
rs7873551
</td>
<td style="text-align:right;">
116482848
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
116482574
</td>
<td style="text-align:right;">
116483181
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
116423112
</td>
<td style="text-align:right;">
117415070
</td>
<td style="text-align:left;">
23245
</td>
<td style="text-align:left;">
ENSG00000148219
</td>
<td style="text-align:left;">
ASTN2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.9740217.9740734
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs360153
</td>
<td style="text-align:right;">
9740727
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9740217
</td>
<td style="text-align:right;">
9740734
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9664077
</td>
<td style="text-align:right;">
9752993
</td>
<td style="text-align:left;">
23075
</td>
<td style="text-align:left;">
ENSG00000133789
</td>
<td style="text-align:left;">
SWAP70
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.9748359.9748943
</td>
<td style="text-align:left;">
Intron (ENST00000447399.6/23075, intron 9 of 10)
</td>
<td style="text-align:left;">
rs472109
</td>
<td style="text-align:right;">
9748771
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9748359
</td>
<td style="text-align:right;">
9748943
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9664077
</td>
<td style="text-align:right;">
9752993
</td>
<td style="text-align:left;">
23075
</td>
<td style="text-align:left;">
ENSG00000133789
</td>
<td style="text-align:left;">
SWAP70
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.117037235.117037719
</td>
<td style="text-align:left;">
Intron (ENST00000445177.6/23387, intron 1 of 24)
</td>
<td style="text-align:left;">
rs7115242
</td>
<td style="text-align:right;">
117037567
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
117037235
</td>
<td style="text-align:right;">
117037719
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
116843402
</td>
<td style="text-align:right;">
117098437
</td>
<td style="text-align:left;">
23387
</td>
<td style="text-align:left;">
ENSG00000160584
</td>
<td style="text-align:left;">
SIK3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.98207008.98208056
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs9556903
</td>
<td style="text-align:right;">
98207081
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
98207008
</td>
<td style="text-align:right;">
98208056
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
98142562
</td>
<td style="text-align:right;">
98455176
</td>
<td style="text-align:left;">
10160
</td>
<td style="text-align:left;">
ENSG00000152767
</td>
<td style="text-align:left;">
FARP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.99657657.99657978
</td>
<td style="text-align:left;">
Intron (ENST00000330710.10/84439, intron 3 of 8)
</td>
<td style="text-align:left;">
rs12897285
</td>
<td style="text-align:right;">
99657718
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99657657
</td>
<td style="text-align:right;">
99657978
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99645129
</td>
<td style="text-align:right;">
99680569
</td>
<td style="text-align:left;">
84439
</td>
<td style="text-align:left;">
ENSG00000182218
</td>
<td style="text-align:left;">
HHIPL1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.41490412.41490650
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs12440045
</td>
<td style="text-align:right;">
41490486
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
41490412
</td>
<td style="text-align:right;">
41490650
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
41493393
</td>
<td style="text-align:right;">
41503551
</td>
<td style="text-align:left;">
3706
</td>
<td style="text-align:left;">
ENSG00000137825
</td>
<td style="text-align:left;">
ITPKA
</td>
<td style="text-align:right;">
2744
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.67165920.67166709
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs17228212
</td>
<td style="text-align:right;">
67166301
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
67165920
</td>
<td style="text-align:right;">
67166709
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
67063763
</td>
<td style="text-align:right;">
67195173
</td>
<td style="text-align:left;">
4088
</td>
<td style="text-align:left;">
ENSG00000166949
</td>
<td style="text-align:left;">
SMAD3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.73353612.73354309
</td>
<td style="text-align:left;">
Intron (ENST00000261917.4/10021, intron 1 of 7)
</td>
<td style="text-align:left;">
rs11631816
</td>
<td style="text-align:right;">
73354253
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73353612
</td>
<td style="text-align:right;">
73354309
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73319859
</td>
<td style="text-align:right;">
73368958
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
ENSG00000138622
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7165042
</td>
<td style="text-align:right;">
78830996
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7165081
</td>
<td style="text-align:right;">
78831054
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7165733
</td>
<td style="text-align:right;">
78831163
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7166764
</td>
<td style="text-align:right;">
78831167
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs11398961
</td>
<td style="text-align:right;">
78831220
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7182529
</td>
<td style="text-align:right;">
78831289
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7182716
</td>
<td style="text-align:right;">
78831411
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7181432
</td>
<td style="text-align:right;">
78831437
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7403708
</td>
<td style="text-align:right;">
78831543
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs7182103
</td>
<td style="text-align:right;">
78831604
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs10083696
</td>
<td style="text-align:right;">
78831674
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs10083697
</td>
<td style="text-align:right;">
78831697
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78830933
</td>
<td style="text-align:right;">
78832028
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
78810487
</td>
<td style="text-align:right;">
78898139
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
ENSG00000185787
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.90885954.90886257
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs4932373
</td>
<td style="text-align:right;">
90886057
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
90885954
</td>
<td style="text-align:right;">
90886257
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
90883695
</td>
<td style="text-align:right;">
90895776
</td>
<td style="text-align:left;">
2242
</td>
<td style="text-align:left;">
ENSG00000182511
</td>
<td style="text-align:left;">
FES
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.72008437.72009100
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs3213420
</td>
<td style="text-align:right;">
72008534
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
72008437
</td>
<td style="text-align:right;">
72009100
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
72008588
</td>
<td style="text-align:right;">
72027664
</td>
<td style="text-align:left;">
1723
</td>
<td style="text-align:left;">
ENSG00000102967
</td>
<td style="text-align:left;">
DHODH
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.2222861.2223544
</td>
<td style="text-align:left;">
Intron (ENST00000263073.11/23293, intron 10 of 18)
</td>
<td style="text-align:left;">
rs216172
</td>
<td style="text-align:right;">
2223210
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
2222861
</td>
<td style="text-align:right;">
2223544
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
2059839
</td>
<td style="text-align:right;">
2303785
</td>
<td style="text-align:left;">
23293
</td>
<td style="text-align:left;">
ENSG00000070366
</td>
<td style="text-align:left;">
SMG6
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.12584681.12585327
</td>
<td style="text-align:left;">
Exon (ENST00000660226.2/284034, exon 3 of 4)
</td>
<td style="text-align:left;">
rs12150051
</td>
<td style="text-align:right;">
12584797
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
12584681
</td>
<td style="text-align:right;">
12585327
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
12665890
</td>
<td style="text-align:right;">
12768949
</td>
<td style="text-align:left;">
93649
</td>
<td style="text-align:left;">
ENSG00000141052
</td>
<td style="text-align:left;">
MYOCD
</td>
<td style="text-align:right;">
80564
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.75844492.75845572
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2410859
</td>
<td style="text-align:right;">
75845204
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
75844492
</td>
<td style="text-align:right;">
75845572
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
75827225
</td>
<td style="text-align:right;">
75844785
</td>
<td style="text-align:left;">
201294
</td>
<td style="text-align:left;">
ENSG00000092929
</td>
<td style="text-align:left;">
UNC13D
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.22457586.22457998
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs3813127
</td>
<td style="text-align:right;">
22457634
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
22457586
</td>
<td style="text-align:right;">
22457998
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
22413599
</td>
<td style="text-align:right;">
22417915
</td>
<td style="text-align:left;">
64693
</td>
<td style="text-align:left;">
ENSG00000212710
</td>
<td style="text-align:left;">
CTAGE1
</td>
<td style="text-align:right;">
39671
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.44733224.44733992
</td>
<td style="text-align:left;">
Exon (ENST00000624404.1/ENST00000624404.1, exon 1 of 1)
</td>
<td style="text-align:left;">
rs17782904
</td>
<td style="text-align:right;">
44733745
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
44733224
</td>
<td style="text-align:right;">
44733992
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
44680173
</td>
<td style="text-align:right;">
45068510
</td>
<td style="text-align:left;">
26040
</td>
<td style="text-align:left;">
ENSG00000152217
</td>
<td style="text-align:left;">
SETBP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.18478653.18479269
</td>
<td style="text-align:left;">
Intron (ENST00000262809.9/8178, intron 1 of 11)
</td>
<td style="text-align:left;">
rs11670056
</td>
<td style="text-align:right;">
18479133
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
18478653
</td>
<td style="text-align:right;">
18479269
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
18442663
</td>
<td style="text-align:right;">
18522116
</td>
<td style="text-align:left;">
8178
</td>
<td style="text-align:left;">
ENSG00000105656
</td>
<td style="text-align:left;">
ELL
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41318858.41319532
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs73045269
</td>
<td style="text-align:right;">
41319286
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41318858
</td>
<td style="text-align:right;">
41319532
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41301587
</td>
<td style="text-align:right;">
41353922
</td>
<td style="text-align:left;">
7040
</td>
<td style="text-align:left;">
ENSG00000105329
</td>
<td style="text-align:left;">
TGFB1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41318858.41319532
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs73045269
</td>
<td style="text-align:right;">
41319286
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41318858
</td>
<td style="text-align:right;">
41319532
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41310172
</td>
<td style="text-align:right;">
41324873
</td>
<td style="text-align:left;">
90324
</td>
<td style="text-align:left;">
ENSG00000142039
</td>
<td style="text-align:left;">
CCDC97
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.44908220.44909100
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs429358
</td>
<td style="text-align:right;">
44908684
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44908220
</td>
<td style="text-align:right;">
44909100
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44905791
</td>
<td style="text-align:right;">
44909393
</td>
<td style="text-align:left;">
348
</td>
<td style="text-align:left;">
ENSG00000130203
</td>
<td style="text-align:left;">
APOE
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.44908220.44909100
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs7412
</td>
<td style="text-align:right;">
44908822
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44908220
</td>
<td style="text-align:right;">
44909100
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
44905791
</td>
<td style="text-align:right;">
44909393
</td>
<td style="text-align:left;">
348
</td>
<td style="text-align:left;">
ENSG00000130203
</td>
<td style="text-align:left;">
APOE
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr22.24262224.24262913
</td>
<td style="text-align:left;">
Exon (ENST00000414583.6/29774, exon 6 of 7)
</td>
<td style="text-align:left;">
rs180803
</td>
<td style="text-align:right;">
24262890
</td>
<td style="text-align:left;">
LR
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
24262224
</td>
<td style="text-align:right;">
24262913
</td>
<td style="text-align:left;">
chr22
</td>
<td style="text-align:right;">
24270817
</td>
<td style="text-align:right;">
24417739
</td>
<td style="text-align:left;">
23384
</td>
<td style="text-align:left;">
ENSG00000100014
</td>
<td style="text-align:left;">
SPECC1L
</td>
<td style="text-align:right;">
7905
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.27601778.27602613
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs113716316
</td>
<td style="text-align:right;">
27602129
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
27601778
</td>
<td style="text-align:right;">
27602613
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
27534035
</td>
<td style="text-align:right;">
27604431
</td>
<td style="text-align:left;">
27245
</td>
<td style="text-align:left;">
ENSG00000126705
</td>
<td style="text-align:left;">
AHDC1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.55039222.55040233
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs564427867
</td>
<td style="text-align:right;">
55039931
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
55039222
</td>
<td style="text-align:right;">
55040233
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
55066359
</td>
<td style="text-align:right;">
55215378
</td>
<td style="text-align:left;">
23358
</td>
<td style="text-align:left;">
ENSG00000162402
</td>
<td style="text-align:left;">
USP24
</td>
<td style="text-align:right;">
26127
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.55039222.55040233
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11591147
</td>
<td style="text-align:right;">
55039974
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
55039222
</td>
<td style="text-align:right;">
55040233
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
55066359
</td>
<td style="text-align:right;">
55215378
</td>
<td style="text-align:left;">
23358
</td>
<td style="text-align:left;">
ENSG00000162402
</td>
<td style="text-align:left;">
USP24
</td>
<td style="text-align:right;">
26127
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.113630172.113630986
</td>
<td style="text-align:left;">
Intron (ENST00000307546.14/260425, intron 9 of 20)
</td>
<td style="text-align:left;">
rs1230666
</td>
<td style="text-align:right;">
113630788
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
113630172
</td>
<td style="text-align:right;">
113630986
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
113390515
</td>
<td style="text-align:right;">
113685923
</td>
<td style="text-align:left;">
260425
</td>
<td style="text-align:left;">
ENSG00000081026
</td>
<td style="text-align:left;">
MAGI3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.203795403.203796615
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs145306069
</td>
<td style="text-align:right;">
203796035
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
203795403
</td>
<td style="text-align:right;">
203796615
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
203795623
</td>
<td style="text-align:right;">
203854124
</td>
<td style="text-align:left;">
100381270
</td>
<td style="text-align:left;">
ENSG00000257315
</td>
<td style="text-align:left;">
ZBED6
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.203795403.203796615
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs145306069
</td>
<td style="text-align:right;">
203796035
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
203795403
</td>
<td style="text-align:right;">
203796615
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
203795623
</td>
<td style="text-align:right;">
203854999
</td>
<td style="text-align:left;">
9877
</td>
<td style="text-align:left;">
ENSG00000058673
</td>
<td style="text-align:left;">
ZC3H11A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.226353539.226354600
</td>
<td style="text-align:left;">
Exon (ENST00000690647.1/ENST00000690647.1, exon 2 of 2)
</td>
<td style="text-align:left;">
rs6426551
</td>
<td style="text-align:right;">
226354348
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
226353539
</td>
<td style="text-align:right;">
226354600
</td>
<td style="text-align:left;">
chr1
</td>
<td style="text-align:right;">
226360210
</td>
<td style="text-align:right;">
226408154
</td>
<td style="text-align:left;">
142
</td>
<td style="text-align:left;">
ENSG00000143799
</td>
<td style="text-align:left;">
PARP1
</td>
<td style="text-align:right;">
5611
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.21047256.21047719
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs585967
</td>
<td style="text-align:right;">
21047682
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
21047256
</td>
<td style="text-align:right;">
21047719
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
20684014
</td>
<td style="text-align:right;">
20823130
</td>
<td style="text-align:left;">
60526
</td>
<td style="text-align:left;">
ENSG00000118961
</td>
<td style="text-align:left;">
LDAH
</td>
<td style="text-align:right;">
224126
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.27262237.27263756
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6759518
</td>
<td style="text-align:right;">
27263727
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
27262237
</td>
<td style="text-align:right;">
27263756
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
27253684
</td>
<td style="text-align:right;">
27275817
</td>
<td style="text-align:left;">
7781
</td>
<td style="text-align:left;">
ENSG00000115194
</td>
<td style="text-align:left;">
SLC30A3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.85560977.85561973
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs7568458
</td>
<td style="text-align:right;">
85561052
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
85560977
</td>
<td style="text-align:right;">
85561973
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
85544720
</td>
<td style="text-align:right;">
85561532
</td>
<td style="text-align:left;">
2677
</td>
<td style="text-align:left;">
ENSG00000115486
</td>
<td style="text-align:left;">
GGCX
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.85560977.85561973
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs7568458
</td>
<td style="text-align:right;">
85561052
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
85560977
</td>
<td style="text-align:right;">
85561973
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
85561562
</td>
<td style="text-align:right;">
85582031
</td>
<td style="text-align:left;">
8673
</td>
<td style="text-align:left;">
ENSG00000118640
</td>
<td style="text-align:left;">
VAMP8
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.111897773.111899676
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs7604403
</td>
<td style="text-align:right;">
111899075
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
111897773
</td>
<td style="text-align:right;">
111899676
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
111898607
</td>
<td style="text-align:right;">
112029561
</td>
<td style="text-align:left;">
10461
</td>
<td style="text-align:left;">
ENSG00000153208
</td>
<td style="text-align:left;">
MERTK
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.128027137.128028647
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs114192718
</td>
<td style="text-align:right;">
128028089
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
128027137
</td>
<td style="text-align:right;">
128028647
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
127941217
</td>
<td style="text-align:right;">
128028120
</td>
<td style="text-align:left;">
79595
</td>
<td style="text-align:left;">
ENSG00000136715
</td>
<td style="text-align:left;">
SAP130
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.135149084.135149706
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6730157
</td>
<td style="text-align:right;">
135149518
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135149084
</td>
<td style="text-align:right;">
135149706
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135052289
</td>
<td style="text-align:right;">
135176667
</td>
<td style="text-align:left;">
22930
</td>
<td style="text-align:left;">
ENSG00000115839
</td>
<td style="text-align:left;">
RAB3GAP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.135149084.135149706
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs6730157
</td>
<td style="text-align:right;">
135149518
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135149084
</td>
<td style="text-align:right;">
135149706
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
135136916
</td>
<td style="text-align:right;">
135531218
</td>
<td style="text-align:left;">
84083
</td>
<td style="text-align:left;">
ENSG00000121988
</td>
<td style="text-align:left;">
ZRANB3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.215420519.215420812
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs17458018
</td>
<td style="text-align:right;">
215420652
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
215420519
</td>
<td style="text-align:right;">
215420812
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
215360440
</td>
<td style="text-align:right;">
215436073
</td>
<td style="text-align:left;">
2335
</td>
<td style="text-align:left;">
ENSG00000115414
</td>
<td style="text-align:left;">
FN1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.215435176.215436848
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1250259
</td>
<td style="text-align:right;">
215435759
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
215435176
</td>
<td style="text-align:right;">
215436848
</td>
<td style="text-align:left;">
chr2
</td>
<td style="text-align:right;">
215360440
</td>
<td style="text-align:right;">
215436073
</td>
<td style="text-align:left;">
2335
</td>
<td style="text-align:left;">
ENSG00000115414
</td>
<td style="text-align:left;">
FN1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.123019226.123019547
</td>
<td style="text-align:left;">
Intron (ENST00000357599.8/54437, intron 1 of 22)
</td>
<td style="text-align:left;">
rs7632505
</td>
<td style="text-align:right;">
123019460
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
123019226
</td>
<td style="text-align:right;">
123019547
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
122909082
</td>
<td style="text-align:right;">
123028605
</td>
<td style="text-align:left;">
54437
</td>
<td style="text-align:left;">
ENSG00000082684
</td>
<td style="text-align:left;">
SEMA5B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.136350085.136351911
</td>
<td style="text-align:left;">
Intron (ENST00000236698.9/10274, intron 28 of 32)
</td>
<td style="text-align:left;">
rs667920
</td>
<td style="text-align:right;">
136350630
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
136350085
</td>
<td style="text-align:right;">
136351911
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
136336236
</td>
<td style="text-align:right;">
136752403
</td>
<td style="text-align:left;">
10274
</td>
<td style="text-align:left;">
ENSG00000118007
</td>
<td style="text-align:left;">
STAG1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.138347976.138349043
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11713141
</td>
<td style="text-align:right;">
138348784
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
138347976
</td>
<td style="text-align:right;">
138349043
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
138347648
</td>
<td style="text-align:right;">
138405534
</td>
<td style="text-align:left;">
22808
</td>
<td style="text-align:left;">
ENSG00000158186
</td>
<td style="text-align:left;">
MRAS
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.138400627.138401136
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2306374
</td>
<td style="text-align:right;">
138401110
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
138400627
</td>
<td style="text-align:right;">
138401136
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
138347648
</td>
<td style="text-align:right;">
138405534
</td>
<td style="text-align:left;">
22808
</td>
<td style="text-align:left;">
ENSG00000158186
</td>
<td style="text-align:left;">
MRAS
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.172082306.172082889
</td>
<td style="text-align:left;">
Intron (ENST00000415807.7/64778, intron 1 of 25)
</td>
<td style="text-align:left;">
rs4894803
</td>
<td style="text-align:right;">
172082466
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
172082306
</td>
<td style="text-align:right;">
172082889
</td>
<td style="text-align:left;">
chr3
</td>
<td style="text-align:right;">
172039578
</td>
<td style="text-align:right;">
172401669
</td>
<td style="text-align:left;">
64778
</td>
<td style="text-align:left;">
ENSG00000075420
</td>
<td style="text-align:left;">
FNDC3B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.40629425.40630609
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs7690530
</td>
<td style="text-align:right;">
40630591
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
40629425
</td>
<td style="text-align:right;">
40630609
</td>
<td style="text-align:left;">
chr4
</td>
<td style="text-align:right;">
40423267
</td>
<td style="text-align:right;">
40630875
</td>
<td style="text-align:left;">
54502
</td>
<td style="text-align:left;">
ENSG00000163694
</td>
<td style="text-align:left;">
RBM47
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.9544368.9546915
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs112941079
</td>
<td style="text-align:right;">
9545986
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
9544368
</td>
<td style="text-align:right;">
9546915
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
9035033
</td>
<td style="text-align:right;">
9546075
</td>
<td style="text-align:left;">
9037
</td>
<td style="text-align:left;">
ENSG00000112902
</td>
<td style="text-align:left;">
SEMA5A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.9544368.9546915
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs113140904
</td>
<td style="text-align:right;">
9546023
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
9544368
</td>
<td style="text-align:right;">
9546915
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
9035033
</td>
<td style="text-align:right;">
9546075
</td>
<td style="text-align:left;">
9037
</td>
<td style="text-align:left;">
ENSG00000112902
</td>
<td style="text-align:left;">
SEMA5A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.52859657.52860199
</td>
<td style="text-align:left;">
Intron (ENST00000282588.7/3672, intron 2 of 28)
</td>
<td style="text-align:left;">
rs73102285
</td>
<td style="text-align:right;">
52859808
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
52859657
</td>
<td style="text-align:right;">
52860199
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
52787916
</td>
<td style="text-align:right;">
52959209
</td>
<td style="text-align:left;">
3672
</td>
<td style="text-align:left;">
ENSG00000213949
</td>
<td style="text-align:left;">
ITGA1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.108747709.108749572
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs112949822
</td>
<td style="text-align:right;">
108749489
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
108747709
</td>
<td style="text-align:right;">
108749572
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
108747841
</td>
<td style="text-align:right;">
109196841
</td>
<td style="text-align:left;">
2241
</td>
<td style="text-align:left;">
ENSG00000151422
</td>
<td style="text-align:left;">
FER
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.135458929.135459424
</td>
<td style="text-align:left;">
Downstream (&lt;=300bp)
</td>
<td style="text-align:left;">
rs899162
</td>
<td style="text-align:right;">
135459219
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
135458929
</td>
<td style="text-align:right;">
135459424
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
135334381
</td>
<td style="text-align:right;">
135399914
</td>
<td style="text-align:left;">
9555
</td>
<td style="text-align:left;">
ENSG00000113648
</td>
<td style="text-align:left;">
MACROH2A1
</td>
<td style="text-align:right;">
59015
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.157575011.157576352
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11465228
</td>
<td style="text-align:right;">
157575687
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
157575011
</td>
<td style="text-align:right;">
157576352
</td>
<td style="text-align:left;">
chr5
</td>
<td style="text-align:right;">
157395534
</td>
<td style="text-align:right;">
157575775
</td>
<td style="text-align:left;">
8728
</td>
<td style="text-align:left;">
ENSG00000135074
</td>
<td style="text-align:left;">
ADAM19
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.7974642.7975583
</td>
<td style="text-align:left;">
Intron (ENST00000439343.2/100526836, intron 4 of 12)
</td>
<td style="text-align:left;">
rs6597292
</td>
<td style="text-align:right;">
7975026
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
7974642
</td>
<td style="text-align:right;">
7975583
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
7881517
</td>
<td style="text-align:right;">
7910788
</td>
<td style="text-align:left;">
81567
</td>
<td style="text-align:left;">
ENSG00000239264
</td>
<td style="text-align:left;">
TXNDC5
</td>
<td style="text-align:right;">
63854
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.19836783.19839260
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11759102
</td>
<td style="text-align:right;">
19837543
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
19836783
</td>
<td style="text-align:right;">
19839260
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
19837370
</td>
<td style="text-align:right;">
19842197
</td>
<td style="text-align:left;">
3400
</td>
<td style="text-align:left;">
ENSG00000172201
</td>
<td style="text-align:left;">
ID4
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.22612314.22612576
</td>
<td style="text-align:left;">
Intron (ENST00000652081.1/105374973, intron 4 of 7)
</td>
<td style="text-align:left;">
rs6909752
</td>
<td style="text-align:right;">
22612400
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
22612314
</td>
<td style="text-align:right;">
22612576
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
21593751
</td>
<td style="text-align:right;">
21598619
</td>
<td style="text-align:left;">
6659
</td>
<td style="text-align:left;">
ENSG00000124766
</td>
<td style="text-align:left;">
SOX4
</td>
<td style="text-align:right;">
1013695
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.34796116.34796383
</td>
<td style="text-align:left;">
Intron (ENST00000192788.6/54887, intron 1 of 20)
</td>
<td style="text-align:left;">
rs9469890
</td>
<td style="text-align:right;">
34796205
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
34796116
</td>
<td style="text-align:right;">
34796383
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
34757505
</td>
<td style="text-align:right;">
34773857
</td>
<td style="text-align:left;">
6631
</td>
<td style="text-align:left;">
ENSG00000124562
</td>
<td style="text-align:left;">
SNRPC
</td>
<td style="text-align:right;">
22259
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.39166177.39166915
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs56336142
</td>
<td style="text-align:right;">
39166323
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
39166177
</td>
<td style="text-align:right;">
39166915
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
39104063
</td>
<td style="text-align:right;">
39115193
</td>
<td style="text-align:left;">
55776
</td>
<td style="text-align:left;">
ENSG00000112167
</td>
<td style="text-align:left;">
SAYSD1
</td>
<td style="text-align:right;">
50984
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.39203639.39204133
</td>
<td style="text-align:left;">
Intron (ENST00000359534.4/8645, intron 1 of 4)
</td>
<td style="text-align:left;">
rs733701
</td>
<td style="text-align:right;">
39204086
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
39203639
</td>
<td style="text-align:right;">
39204133
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
39104063
</td>
<td style="text-align:right;">
39115193
</td>
<td style="text-align:left;">
55776
</td>
<td style="text-align:left;">
ENSG00000112167
</td>
<td style="text-align:left;">
SAYSD1
</td>
<td style="text-align:right;">
88446
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.130602114.130602839
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs9388813
</td>
<td style="text-align:right;">
130602182
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
130602114
</td>
<td style="text-align:right;">
130602839
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
130366017
</td>
<td style="text-align:right;">
130443067
</td>
<td style="text-align:left;">
114801
</td>
<td style="text-align:left;">
ENSG00000164484
</td>
<td style="text-align:left;">
TMEM200A
</td>
<td style="text-align:right;">
159047
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.134047673.134047938
</td>
<td style="text-align:left;">
Intron (ENST00000275230.6/154091, intron 1 of 4)
</td>
<td style="text-align:left;">
rs965652
</td>
<td style="text-align:right;">
134047815
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
134047673
</td>
<td style="text-align:right;">
134047938
</td>
<td style="text-align:left;">
chr6
</td>
<td style="text-align:right;">
133987581
</td>
<td style="text-align:right;">
134052624
</td>
<td style="text-align:left;">
154091
</td>
<td style="text-align:left;">
ENSG00000146411
</td>
<td style="text-align:left;">
SLC2A12
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.13989380.13991798
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2073533
</td>
<td style="text-align:right;">
13990114
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
13989380
</td>
<td style="text-align:right;">
13991798
</td>
<td style="text-align:left;">
chr7
</td>
<td style="text-align:right;">
13891229
</td>
<td style="text-align:right;">
13991425
</td>
<td style="text-align:left;">
2115
</td>
<td style="text-align:left;">
ENSG00000006468
</td>
<td style="text-align:left;">
ETV1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.19961869.19963021
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs326
</td>
<td style="text-align:right;">
19961928
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
19961869
</td>
<td style="text-align:right;">
19963021
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
19901717
</td>
<td style="text-align:right;">
19967259
</td>
<td style="text-align:left;">
4023
</td>
<td style="text-align:left;">
ENSG00000175445
</td>
<td style="text-align:left;">
LPL
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.19961869.19963021
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs328
</td>
<td style="text-align:right;">
19962213
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
19961869
</td>
<td style="text-align:right;">
19963021
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
19901717
</td>
<td style="text-align:right;">
19967259
</td>
<td style="text-align:left;">
4023
</td>
<td style="text-align:left;">
ENSG00000175445
</td>
<td style="text-align:left;">
LPL
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.20007264.20008372
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs2083636
</td>
<td style="text-align:right;">
20007752
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
20007264
</td>
<td style="text-align:right;">
20008372
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
19901717
</td>
<td style="text-align:right;">
19967259
</td>
<td style="text-align:left;">
4023
</td>
<td style="text-align:left;">
ENSG00000175445
</td>
<td style="text-align:left;">
LPL
</td>
<td style="text-align:right;">
40005
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.20007264.20008372
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs894211
</td>
<td style="text-align:right;">
20008236
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
20007264
</td>
<td style="text-align:right;">
20008372
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
19901717
</td>
<td style="text-align:right;">
19967259
</td>
<td style="text-align:left;">
4023
</td>
<td style="text-align:left;">
ENSG00000175445
</td>
<td style="text-align:left;">
LPL
</td>
<td style="text-align:right;">
40005
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.125466567.125467336
</td>
<td style="text-align:left;">
Exon (ENST00000522815.1/ENST00000522815.1, exon 1 of 4)
</td>
<td style="text-align:left;">
rs6982502
</td>
<td style="text-align:right;">
125467120
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
125466567
</td>
<td style="text-align:right;">
125467336
</td>
<td style="text-align:left;">
chr8
</td>
<td style="text-align:right;">
125430358
</td>
<td style="text-align:right;">
125438403
</td>
<td style="text-align:left;">
10221
</td>
<td style="text-align:left;">
ENSG00000173334
</td>
<td style="text-align:left;">
TRIB1
</td>
<td style="text-align:right;">
28164
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.22102751.22103879
</td>
<td style="text-align:left;">
Intron (ENST00000585267.5/100048912, intron 4 of 8)
</td>
<td style="text-align:left;">
rs1537372
</td>
<td style="text-align:right;">
22103184
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
22102751
</td>
<td style="text-align:right;">
22103879
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
22002903
</td>
<td style="text-align:right;">
22009305
</td>
<td style="text-align:left;">
1030
</td>
<td style="text-align:left;">
ENSG00000147883
</td>
<td style="text-align:left;">
CDKN2B
</td>
<td style="text-align:right;">
93446
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.22102751.22103879
</td>
<td style="text-align:left;">
Intron (ENST00000585267.5/100048912, intron 4 of 8)
</td>
<td style="text-align:left;">
rs1537373
</td>
<td style="text-align:right;">
22103342
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
22102751
</td>
<td style="text-align:right;">
22103879
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
22002903
</td>
<td style="text-align:right;">
22009305
</td>
<td style="text-align:left;">
1030
</td>
<td style="text-align:left;">
ENSG00000147883
</td>
<td style="text-align:left;">
CDKN2B
</td>
<td style="text-align:right;">
93446
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.22102751.22103879
</td>
<td style="text-align:left;">
Intron (ENST00000585267.5/100048912, intron 4 of 8)
</td>
<td style="text-align:left;">
rs1333042
</td>
<td style="text-align:right;">
22103814
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
22102751
</td>
<td style="text-align:right;">
22103879
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
22002903
</td>
<td style="text-align:right;">
22009305
</td>
<td style="text-align:left;">
1030
</td>
<td style="text-align:left;">
ENSG00000147883
</td>
<td style="text-align:left;">
CDKN2B
</td>
<td style="text-align:right;">
93446
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.77011996.77012519
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs62568141
</td>
<td style="text-align:right;">
77012344
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
77011996
</td>
<td style="text-align:right;">
77012519
</td>
<td style="text-align:left;">
chr9
</td>
<td style="text-align:right;">
76611376
</td>
<td style="text-align:right;">
76906114
</td>
<td style="text-align:left;">
158471
</td>
<td style="text-align:left;">
ENSG00000106772
</td>
<td style="text-align:left;">
PRUNE2
</td>
<td style="text-align:right;">
105882
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.12234804.12236216
</td>
<td style="text-align:left;">
Exon (ENST00000455773.7/8872, exon 8 of 11)
</td>
<td style="text-align:left;">
rs7090277
</td>
<td style="text-align:right;">
12236022
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
12234804
</td>
<td style="text-align:right;">
12236216
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
12195965
</td>
<td style="text-align:right;">
12250589
</td>
<td style="text-align:left;">
8872
</td>
<td style="text-align:left;">
ENSG00000151465
</td>
<td style="text-align:left;">
CDC123
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.44245450.44245918
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs494207
</td>
<td style="text-align:right;">
44245808
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
44245450
</td>
<td style="text-align:right;">
44245918
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
44370165
</td>
<td style="text-align:right;">
44386493
</td>
<td style="text-align:left;">
6387
</td>
<td style="text-align:left;">
ENSG00000107562
</td>
<td style="text-align:left;">
CXCL12
</td>
<td style="text-align:right;">
124248
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.103069381.103070029
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1926032
</td>
<td style="text-align:right;">
103069712
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
103069381
</td>
<td style="text-align:right;">
103070029
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
102918294
</td>
<td style="text-align:right;">
103090222
</td>
<td style="text-align:left;">
54805
</td>
<td style="text-align:left;">
ENSG00000148842
</td>
<td style="text-align:left;">
CNNM2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.121155583.121156319
</td>
<td style="text-align:left;">
Distal Intergenic
</td>
<td style="text-align:left;">
rs17101521
</td>
<td style="text-align:right;">
121156039
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
121155583
</td>
<td style="text-align:right;">
121156319
</td>
<td style="text-align:left;">
chr10
</td>
<td style="text-align:right;">
120851305
</td>
<td style="text-align:right;">
120909524
</td>
<td style="text-align:left;">
55717
</td>
<td style="text-align:left;">
ENSG00000120008
</td>
<td style="text-align:left;">
WDR11
</td>
<td style="text-align:right;">
246059
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.8767540.8768078
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs56210063
</td>
<td style="text-align:right;">
8767618
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
8767540
</td>
<td style="text-align:right;">
8768078
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
8693351
</td>
<td style="text-align:right;">
8910951
</td>
<td style="text-align:left;">
6764
</td>
<td style="text-align:left;">
ENSG00000166444
</td>
<td style="text-align:left;">
DENND2B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.9758098.9760569
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs10734649
</td>
<td style="text-align:right;">
9759344
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9758098
</td>
<td style="text-align:right;">
9760569
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9754770
</td>
<td style="text-align:right;">
9759533
</td>
<td style="text-align:left;">
440028
</td>
<td style="text-align:left;">
ENSG00000245522
</td>
<td style="text-align:left;">
LINC02709
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.9758098.9760569
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs10734649
</td>
<td style="text-align:right;">
9759344
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9758098
</td>
<td style="text-align:right;">
9760569
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
9758268
</td>
<td style="text-align:right;">
9811335
</td>
<td style="text-align:left;">
283104
</td>
<td style="text-align:left;">
ENSG00000246273
</td>
<td style="text-align:left;">
SBF2-AS1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.65637814.65638912
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2306363
</td>
<td style="text-align:right;">
65638129
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
65637814
</td>
<td style="text-align:right;">
65638912
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
65638101
</td>
<td style="text-align:right;">
65650918
</td>
<td style="text-align:left;">
6494
</td>
<td style="text-align:left;">
ENSG00000213445
</td>
<td style="text-align:left;">
SIPA1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.72821745.72822900
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11235604
</td>
<td style="text-align:right;">
72822491
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
72821745
</td>
<td style="text-align:right;">
72822900
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
72814406
</td>
<td style="text-align:right;">
72843674
</td>
<td style="text-align:left;">
89849
</td>
<td style="text-align:left;">
ENSG00000168010
</td>
<td style="text-align:left;">
ATG16L2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.75561990.75564409
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs590121
</td>
<td style="text-align:right;">
75563105
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
75561990
</td>
<td style="text-align:right;">
75564409
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
75562056
</td>
<td style="text-align:right;">
75572783
</td>
<td style="text-align:left;">
871
</td>
<td style="text-align:left;">
ENSG00000149257
</td>
<td style="text-align:left;">
SERPINH1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.90455310.90455981
</td>
<td style="text-align:left;">
Intron (ENST00000649150.1/101929222, intron 2 of 7)
</td>
<td style="text-align:left;">
rs534128177
</td>
<td style="text-align:right;">
90455679
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
90455310
</td>
<td style="text-align:right;">
90455981
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
90200429
</td>
<td style="text-align:right;">
90223077
</td>
<td style="text-align:left;">
26973
</td>
<td style="text-align:left;">
ENSG00000110172
</td>
<td style="text-align:left;">
CHORDC1
</td>
<td style="text-align:right;">
232233
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.100739719.100740030
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs604723
</td>
<td style="text-align:right;">
100739815
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
100739719
</td>
<td style="text-align:right;">
100740030
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
100687288
</td>
<td style="text-align:right;">
100993941
</td>
<td style="text-align:left;">
143872
</td>
<td style="text-align:left;">
ENSG00000165895
</td>
<td style="text-align:left;">
ARHGAP42
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.100753128.100753902
</td>
<td style="text-align:left;">
Intron (ENST00000298815.13/143872, intron 1 of 23)
</td>
<td style="text-align:left;">
rs7947761
</td>
<td style="text-align:right;">
100753868
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
100753128
</td>
<td style="text-align:right;">
100753902
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
100687288
</td>
<td style="text-align:right;">
100993941
</td>
<td style="text-align:left;">
143872
</td>
<td style="text-align:left;">
ENSG00000165895
</td>
<td style="text-align:left;">
ARHGAP42
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.118911406.118912630
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11606719
</td>
<td style="text-align:right;">
118911765
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
118911406
</td>
<td style="text-align:right;">
118912630
</td>
<td style="text-align:left;">
chr11
</td>
<td style="text-align:right;">
118893875
</td>
<td style="text-align:right;">
118925926
</td>
<td style="text-align:left;">
283149
</td>
<td style="text-align:left;">
ENSG00000186174
</td>
<td style="text-align:left;">
BCL9L
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.7068459.7069322
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs11838267
</td>
<td style="text-align:right;">
7068568
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
7068459
</td>
<td style="text-align:right;">
7069322
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
6988259
</td>
<td style="text-align:right;">
7071032
</td>
<td style="text-align:left;">
716
</td>
<td style="text-align:left;">
ENSG00000182326
</td>
<td style="text-align:left;">
C1S
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.12349737.12350701
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs11054833
</td>
<td style="text-align:right;">
12350535
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
12349737
</td>
<td style="text-align:right;">
12350701
</td>
<td style="text-align:left;">
chr12
</td>
<td style="text-align:right;">
12326056
</td>
<td style="text-align:right;">
12350242
</td>
<td style="text-align:left;">
54682
</td>
<td style="text-align:left;">
ENSG00000111261
</td>
<td style="text-align:left;">
MANSC1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110181936.110182458
</td>
<td style="text-align:left;">
Intron (ENST00000375820.10/1282, intron 28 of 51)
</td>
<td style="text-align:left;">
rs3783113
</td>
<td style="text-align:right;">
110182399
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110181936
</td>
<td style="text-align:right;">
110182458
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110148963
</td>
<td style="text-align:right;">
110307157
</td>
<td style="text-align:left;">
1282
</td>
<td style="text-align:left;">
ENSG00000187498
</td>
<td style="text-align:left;">
COL4A1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110301444.110302036
</td>
<td style="text-align:left;">
Intron (ENST00000375820.10/1282, intron 1 of 51)
</td>
<td style="text-align:left;">
rs4773140
</td>
<td style="text-align:right;">
110301890
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110301444
</td>
<td style="text-align:right;">
110302036
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110148963
</td>
<td style="text-align:right;">
110307157
</td>
<td style="text-align:left;">
1282
</td>
<td style="text-align:left;">
ENSG00000187498
</td>
<td style="text-align:left;">
COL4A1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110301444.110302036
</td>
<td style="text-align:left;">
Intron (ENST00000375820.10/1282, intron 1 of 51)
</td>
<td style="text-align:left;">
rs4773141
</td>
<td style="text-align:right;">
110302006
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110301444
</td>
<td style="text-align:right;">
110302036
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110148963
</td>
<td style="text-align:right;">
110307157
</td>
<td style="text-align:left;">
1282
</td>
<td style="text-align:left;">
ENSG00000187498
</td>
<td style="text-align:left;">
COL4A1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110305804.110309074
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs4773144
</td>
<td style="text-align:right;">
110308365
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110305804
</td>
<td style="text-align:right;">
110309074
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110148963
</td>
<td style="text-align:right;">
110307157
</td>
<td style="text-align:left;">
1282
</td>
<td style="text-align:left;">
ENSG00000187498
</td>
<td style="text-align:left;">
COL4A1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110305804.110309074
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs4773144
</td>
<td style="text-align:right;">
110308365
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110305804
</td>
<td style="text-align:right;">
110309074
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110305812
</td>
<td style="text-align:right;">
110513209
</td>
<td style="text-align:left;">
1284
</td>
<td style="text-align:left;">
ENSG00000134871
</td>
<td style="text-align:left;">
COL4A2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110387344.110389032
</td>
<td style="text-align:left;">
Intron (ENST00000400163.7/1284, intron 4 of 4)
</td>
<td style="text-align:left;">
rs11838776
</td>
<td style="text-align:right;">
110388334
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110387344
</td>
<td style="text-align:right;">
110389032
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110305812
</td>
<td style="text-align:right;">
110513209
</td>
<td style="text-align:left;">
1284
</td>
<td style="text-align:left;">
ENSG00000134871
</td>
<td style="text-align:left;">
COL4A2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110455785.110456535
</td>
<td style="text-align:left;">
Exon (ENST00000458403.2/100129836, exon 5 of 5)
</td>
<td style="text-align:left;">
rs7333991
</td>
<td style="text-align:right;">
110456089
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110455785
</td>
<td style="text-align:right;">
110456535
</td>
<td style="text-align:left;">
chr13
</td>
<td style="text-align:right;">
110305812
</td>
<td style="text-align:right;">
110513209
</td>
<td style="text-align:left;">
1284
</td>
<td style="text-align:left;">
ENSG00000134871
</td>
<td style="text-align:left;">
COL4A2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.58326948.58327320
</td>
<td style="text-align:left;">
Intron (ENST00000348476.7/5926, intron 8 of 22)
</td>
<td style="text-align:left;">
rs2145598
</td>
<td style="text-align:right;">
58327283
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
58326948
</td>
<td style="text-align:right;">
58327320
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
58298504
</td>
<td style="text-align:right;">
58373887
</td>
<td style="text-align:left;">
5926
</td>
<td style="text-align:left;">
ENSG00000032219
</td>
<td style="text-align:left;">
ARID4A
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.75002369.75003794
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs175040
</td>
<td style="text-align:right;">
75002852
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
75002369
</td>
<td style="text-align:right;">
75003794
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
75002921
</td>
<td style="text-align:right;">
75012366
</td>
<td style="text-align:left;">
8892
</td>
<td style="text-align:left;">
ENSG00000119718
</td>
<td style="text-align:left;">
EIF2B2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.99644968.99645471
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs12893887
</td>
<td style="text-align:right;">
99645228
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99644968
</td>
<td style="text-align:right;">
99645471
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99645129
</td>
<td style="text-align:right;">
99680569
</td>
<td style="text-align:left;">
84439
</td>
<td style="text-align:left;">
ENSG00000182218
</td>
<td style="text-align:left;">
HHIPL1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.99644968.99645471
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs759098931
</td>
<td style="text-align:right;">
99645335
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99644968
</td>
<td style="text-align:right;">
99645471
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99645129
</td>
<td style="text-align:right;">
99680569
</td>
<td style="text-align:left;">
84439
</td>
<td style="text-align:left;">
ENSG00000182218
</td>
<td style="text-align:left;">
HHIPL1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.99682572.99684943
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs8003602
</td>
<td style="text-align:right;">
99682624
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99682572
</td>
<td style="text-align:right;">
99684943
</td>
<td style="text-align:left;">
chr14
</td>
<td style="text-align:right;">
99684298
</td>
<td style="text-align:right;">
99727301
</td>
<td style="text-align:left;">
10858
</td>
<td style="text-align:left;">
ENSG00000036530
</td>
<td style="text-align:left;">
CYP46A1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.73983047.73983757
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs2507
</td>
<td style="text-align:right;">
73983335
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73983047
</td>
<td style="text-align:right;">
73983757
</td>
<td style="text-align:left;">
chr15
</td>
<td style="text-align:right;">
73978926
</td>
<td style="text-align:right;">
73994622
</td>
<td style="text-align:left;">
9399
</td>
<td style="text-align:left;">
ENSG00000067221
</td>
<td style="text-align:left;">
STOML1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr16.81873989.81874357
</td>
<td style="text-align:left;">
Intron (ENST00000697580.1/5336, intron 7 of 32)
</td>
<td style="text-align:left;">
rs7189462
</td>
<td style="text-align:right;">
81874262
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
81873989
</td>
<td style="text-align:right;">
81874357
</td>
<td style="text-align:left;">
chr16
</td>
<td style="text-align:right;">
81779279
</td>
<td style="text-align:right;">
81962685
</td>
<td style="text-align:left;">
5336
</td>
<td style="text-align:left;">
ENSG00000197943
</td>
<td style="text-align:left;">
PLCG2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.2214040.2216112
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs2281727
</td>
<td style="text-align:right;">
2214651
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
2214040
</td>
<td style="text-align:right;">
2216112
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
2059839
</td>
<td style="text-align:right;">
2303785
</td>
<td style="text-align:left;">
23293
</td>
<td style="text-align:left;">
ENSG00000070366
</td>
<td style="text-align:left;">
SMG6
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.43021036.43021565
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs9912587
</td>
<td style="text-align:right;">
43021069
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
43021036
</td>
<td style="text-align:right;">
43021565
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
43014607
</td>
<td style="text-align:right;">
43025123
</td>
<td style="text-align:left;">
10493
</td>
<td style="text-align:left;">
ENSG00000108828
</td>
<td style="text-align:left;">
VAT1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.48892143.48893499
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1962412
</td>
<td style="text-align:right;">
48892897
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
48892143
</td>
<td style="text-align:right;">
48893499
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
48892765
</td>
<td style="text-align:right;">
48895871
</td>
<td style="text-align:left;">
516
</td>
<td style="text-align:left;">
ENSG00000159199
</td>
<td style="text-align:left;">
ATP5MC1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.48970173.48970597
</td>
<td style="text-align:left;">
Promoter (1-2kb)
</td>
<td style="text-align:left;">
rs3895874
</td>
<td style="text-align:right;">
48970506
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
48970173
</td>
<td style="text-align:right;">
48970597
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
48929316
</td>
<td style="text-align:right;">
48944842
</td>
<td style="text-align:left;">
11267
</td>
<td style="text-align:left;">
ENSG00000159210
</td>
<td style="text-align:left;">
SNF8
</td>
<td style="text-align:right;">
25331
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.49012959.49014932
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs9906944
</td>
<td style="text-align:right;">
49014058
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
49012959
</td>
<td style="text-align:right;">
49014932
</td>
<td style="text-align:left;">
chr17
</td>
<td style="text-align:right;">
48997385
</td>
<td style="text-align:right;">
49056145
</td>
<td style="text-align:left;">
10642
</td>
<td style="text-align:left;">
ENSG00000159217
</td>
<td style="text-align:left;">
IGF2BP1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.45879810.45880598
</td>
<td style="text-align:left;">
3’ UTR
</td>
<td style="text-align:left;">
rs148241618
</td>
<td style="text-align:right;">
45880124
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
45879810
</td>
<td style="text-align:right;">
45880598
</td>
<td style="text-align:left;">
chr18
</td>
<td style="text-align:right;">
45800581
</td>
<td style="text-align:right;">
45967329
</td>
<td style="text-align:left;">
57724
</td>
<td style="text-align:left;">
ENSG00000152223
</td>
<td style="text-align:left;">
EPG5
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.2235309.2237183
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs191615952
</td>
<td style="text-align:right;">
2236098
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
2235309
</td>
<td style="text-align:right;">
2237183
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
2230084
</td>
<td style="text-align:right;">
2237704
</td>
<td style="text-align:left;">
55111
</td>
<td style="text-align:left;">
ENSG00000104886
</td>
<td style="text-align:left;">
PLEKHJ1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.2235309.2237183
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs191615952
</td>
<td style="text-align:right;">
2236098
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
2235309
</td>
<td style="text-align:right;">
2237183
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
2236824
</td>
<td style="text-align:right;">
2248655
</td>
<td style="text-align:left;">
8175
</td>
<td style="text-align:left;">
ENSG00000104897
</td>
<td style="text-align:left;">
SF3A2
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.8363631.8364760
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs116843064
</td>
<td style="text-align:right;">
8364439
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
8363631
</td>
<td style="text-align:right;">
8364760
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
8363289
</td>
<td style="text-align:right;">
8374370
</td>
<td style="text-align:left;">
51129
</td>
<td style="text-align:left;">
ENSG00000167772
</td>
<td style="text-align:left;">
ANGPTL4
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.11052376.11052994
</td>
<td style="text-align:left;">
Intron (ENST00000643549.1/6597, intron 30 of 34)
</td>
<td style="text-align:left;">
rs1122608
</td>
<td style="text-align:right;">
11052925
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
11052376
</td>
<td style="text-align:right;">
11052994
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
10960932
</td>
<td style="text-align:right;">
11079426
</td>
<td style="text-align:left;">
6597
</td>
<td style="text-align:left;">
ENSG00000127616
</td>
<td style="text-align:left;">
SMARCA4
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.17107804.17108454
</td>
<td style="text-align:left;">
Intron (ENST00000682292.1/4650, intron 2 of 39)
</td>
<td style="text-align:left;">
rs7246865
</td>
<td style="text-align:right;">
17108295
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
17107804
</td>
<td style="text-align:right;">
17108454
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
17075777
</td>
<td style="text-align:right;">
17214537
</td>
<td style="text-align:left;">
4650
</td>
<td style="text-align:left;">
ENSG00000099331
</td>
<td style="text-align:left;">
MYO9B
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41352378.41354231
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1800470
</td>
<td style="text-align:right;">
41353016
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41352378
</td>
<td style="text-align:right;">
41354231
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41301587
</td>
<td style="text-align:right;">
41353922
</td>
<td style="text-align:left;">
7040
</td>
<td style="text-align:left;">
ENSG00000105329
</td>
<td style="text-align:left;">
TGFB1
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41352378.41354231
</td>
<td style="text-align:left;">
Promoter (&lt;=1kb)
</td>
<td style="text-align:left;">
rs1800470
</td>
<td style="text-align:right;">
41353016
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41352378
</td>
<td style="text-align:right;">
41354231
</td>
<td style="text-align:left;">
chr19
</td>
<td style="text-align:right;">
41350911
</td>
<td style="text-align:right;">
41384083
</td>
<td style="text-align:left;">
641649
</td>
<td style="text-align:left;">
ENSG00000142046
</td>
<td style="text-align:left;">
TMEM91
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.29162621.29163485
</td>
<td style="text-align:left;">
Intron (ENST00000710354.1/56911, intron 8 of 8)
</td>
<td style="text-align:left;">
rs73193808
</td>
<td style="text-align:right;">
29162981
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
29162621
</td>
<td style="text-align:right;">
29163485
</td>
<td style="text-align:left;">
chr21
</td>
<td style="text-align:right;">
29077221
</td>
<td style="text-align:right;">
29175889
</td>
<td style="text-align:left;">
56911
</td>
<td style="text-align:left;">
ENSG00000156265
</td>
<td style="text-align:left;">
MAP3K7CL
</td>
<td style="text-align:right;">
0
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>## arrythmia is gwas #2</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] genomation_1.34.0                       
 [2] plyranges_1.22.0                        
 [3] biomaRt_2.58.2                          
 [4] devtools_2.4.5                          
 [5] usethis_2.2.3                           
 [6] ggpubr_0.6.0                            
 [7] BiocParallel_1.36.0                     
 [8] Cormotif_1.48.0                         
 [9] affy_1.80.0                             
[10] scales_1.3.0                            
[11] VennDiagram_1.7.3                       
[12] futile.logger_1.4.3                     
[13] gridExtra_2.3                           
[14] ggfortify_0.4.17                        
[15] edgeR_4.0.16                            
[16] limma_3.58.1                            
[17] rtracklayer_1.62.0                      
[18] org.Hs.eg.db_3.18.0                     
[19] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[20] GenomicFeatures_1.54.4                  
[21] AnnotationDbi_1.64.1                    
[22] Biobase_2.62.0                          
[23] GenomicRanges_1.54.1                    
[24] GenomeInfoDb_1.38.8                     
[25] IRanges_2.36.0                          
[26] S4Vectors_0.40.2                        
[27] BiocGenerics_0.48.1                     
[28] ChIPseeker_1.38.0                       
[29] RColorBrewer_1.1-3                      
[30] broom_1.0.6                             
[31] kableExtra_1.4.0                        
[32] lubridate_1.9.3                         
[33] forcats_1.0.0                           
[34] stringr_1.5.1                           
[35] dplyr_1.1.4                             
[36] purrr_1.0.2                             
[37] readr_2.1.5                             
[38] tidyr_1.3.1                             
[39] tibble_3.2.1                            
[40] ggplot2_3.5.1                           
[41] tidyverse_2.0.0                         
[42] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.4                               
  [2] matrixStats_1.3.0                      
  [3] bitops_1.0-7                           
  [4] enrichplot_1.22.0                      
  [5] HDO.db_0.99.1                          
  [6] httr_1.4.7                             
  [7] profvis_0.3.8                          
  [8] tools_4.3.1                            
  [9] backports_1.5.0                        
 [10] utf8_1.2.4                             
 [11] R6_2.5.1                               
 [12] lazyeval_0.2.2                         
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.0                            
 [15] prettyunits_1.2.0                      
 [16] preprocessCore_1.64.0                  
 [17] cli_3.6.2                              
 [18] formatR_1.14                           
 [19] scatterpie_0.2.2                       
 [20] labeling_0.4.3                         
 [21] sass_0.4.9                             
 [22] Rsamtools_2.18.0                       
 [23] systemfonts_1.1.0                      
 [24] yulab.utils_0.1.4                      
 [25] DOSE_3.28.2                            
 [26] svglite_2.1.3                          
 [27] sessioninfo_1.2.2                      
 [28] plotrix_3.8-4                          
 [29] BSgenome_1.70.2                        
 [30] impute_1.76.0                          
 [31] rstudioapi_0.16.0                      
 [32] RSQLite_2.3.7                          
 [33] ggVennDiagram_1.5.2                    
 [34] generics_0.1.3                         
 [35] gridGraphics_0.5-1                     
 [36] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [37] BiocIO_1.12.0                          
 [38] vroom_1.6.5                            
 [39] gtools_3.9.5                           
 [40] car_3.1-2                              
 [41] GO.db_3.18.0                           
 [42] Matrix_1.6-5                           
 [43] fansi_1.0.6                            
 [44] abind_1.4-5                            
 [45] lifecycle_1.0.4                        
 [46] whisker_0.4.1                          
 [47] yaml_2.3.8                             
 [48] carData_3.0-5                          
 [49] SummarizedExperiment_1.32.0            
 [50] gplots_3.1.3.1                         
 [51] qvalue_2.34.0                          
 [52] SparseArray_1.2.4                      
 [53] BiocFileCache_2.10.2                   
 [54] blob_1.2.4                             
 [55] promises_1.3.0                         
 [56] crayon_1.5.2                           
 [57] miniUI_0.1.1.1                         
 [58] lattice_0.22-6                         
 [59] cowplot_1.1.3                          
 [60] KEGGREST_1.42.0                        
 [61] pillar_1.9.0                           
 [62] knitr_1.47                             
 [63] fgsea_1.28.0                           
 [64] rjson_0.2.21                           
 [65] boot_1.3-30                            
 [66] codetools_0.2-20                       
 [67] fastmatch_1.1-4                        
 [68] glue_1.7.0                             
 [69] getPass_0.2-4                          
 [70] ggfun_0.1.5                            
 [71] remotes_2.5.0                          
 [72] data.table_1.15.4                      
 [73] vctrs_0.6.5                            
 [74] png_0.1-8                              
 [75] treeio_1.26.0                          
 [76] gtable_0.3.5                           
 [77] cachem_1.1.0                           
 [78] xfun_0.44                              
 [79] S4Arrays_1.2.1                         
 [80] mime_0.12                              
 [81] tidygraph_1.3.1                        
 [82] statmod_1.5.0                          
 [83] ellipsis_0.3.2                         
 [84] interactiveDisplayBase_1.40.0          
 [85] nlme_3.1-164                           
 [86] ggtree_3.10.1                          
 [87] bit64_4.0.5                            
 [88] progress_1.2.3                         
 [89] filelock_1.0.3                         
 [90] rprojroot_2.0.4                        
 [91] bslib_0.7.0                            
 [92] affyio_1.72.0                          
 [93] KernSmooth_2.23-24                     
 [94] colorspace_2.1-0                       
 [95] DBI_1.2.3                              
 [96] seqPattern_1.34.0                      
 [97] tidyselect_1.2.1                       
 [98] processx_3.8.4                         
 [99] bit_4.0.5                              
[100] compiler_4.3.1                         
[101] curl_5.2.1                             
[102] git2r_0.33.0                           
[103] xml2_1.3.6                             
[104] DelayedArray_0.28.0                    
[105] shadowtext_0.1.3                       
[106] caTools_1.18.2                         
[107] callr_3.7.6                            
[108] rappdirs_0.3.3                         
[109] digest_0.6.35                          
[110] rmarkdown_2.27                         
[111] XVector_0.42.0                         
[112] htmltools_0.5.8.1                      
[113] pkgconfig_2.0.3                        
[114] MatrixGenerics_1.14.0                  
[115] highr_0.11                             
[116] dbplyr_2.5.0                           
[117] fastmap_1.2.0                          
[118] htmlwidgets_1.6.4                      
[119] rlang_1.1.3                            
[120] shiny_1.8.1.1                          
[121] farver_2.1.2                           
[122] jquerylib_0.1.4                        
[123] jsonlite_1.8.8                         
[124] GOSemSim_2.28.1                        
[125] RCurl_1.98-1.14                        
[126] magrittr_2.0.3                         
[127] GenomeInfoDbData_1.2.11                
[128] ggplotify_0.1.2                        
[129] patchwork_1.2.0                        
[130] munsell_0.5.1                          
[131] Rcpp_1.0.12                            
[132] ape_5.8                                
[133] viridis_0.6.5                          
[134] stringi_1.8.4                          
[135] ggraph_2.2.1                           
[136] zlibbioc_1.48.2                        
[137] MASS_7.3-60.0.1                        
[138] pkgbuild_1.4.4                         
[139] AnnotationHub_3.10.1                   
[140] plyr_1.8.9                             
[141] parallel_4.3.1                         
[142] ggrepel_0.9.5                          
[143] Biostrings_2.70.3                      
[144] graphlayouts_1.1.1                     
[145] splines_4.3.1                          
[146] hms_1.1.3                              
[147] locfit_1.5-9.9                         
[148] ps_1.7.6                               
[149] igraph_2.0.3                           
[150] ggsignif_0.6.4                         
[151] pkgload_1.3.4                          
[152] reshape2_1.4.4                         
[153] futile.options_1.0.1                   
[154] BiocVersion_3.18.1                     
[155] XML_3.99-0.16.1                        
[156] evaluate_0.23                          
[157] lambda.r_1.2.4                         
[158] BiocManager_1.30.23                    
[159] tzdb_0.4.0                             
[160] tweenr_2.0.3                           
[161] httpuv_1.6.15                          
[162] polyclip_1.10-6                        
[163] gridBase_0.4-7                         
[164] ggforce_0.4.2                          
[165] xtable_1.8-4                           
[166] restfulr_0.0.15                        
[167] tidytree_0.4.6                         
[168] rstatix_0.7.2                          
[169] later_1.3.2                            
[170] viridisLite_0.4.2                      
[171] aplot_0.2.2                            
[172] memoise_2.0.1                          
[173] GenomicAlignments_1.38.2               
[174] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
