<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-04-23" />

<title>Enrichment_motif</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enrichment_motif</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-04-23</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-23
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree8bfb48f4dc9db3a9095a1c814ed2ffbff9fa7dc6targetblank8bfb48fa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/8bfb48f4dc9db3a9095a1c814ed2ffbff9fa7dc6" target="_blank">8bfb48f</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree8bfb48f4dc9db3a9095a1c814ed2ffbff9fa7dc6targetblank8bfb48fa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/8bfb48f4dc9db3a9095a1c814ed2ffbff9fa7dc6" target="_blank">8bfb48f</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/Jaspar_motif.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/corMotifcustom.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/CorMotif_data_n45.Rmd
    Modified:   analysis/Peak_analysis.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Enrichment_motif.Rmd</code>) and
HTML (<code>docs/Enrichment_motif.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8bfb48f4dc9db3a9095a1c814ed2ffbff9fa7dc6/analysis/Enrichment_motif.Rmd" target="_blank">8bfb48f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-23
</td>
<td>
adding other info
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/2ddd597cc8c6de11b96b1cd15e59f3a73490cacf/docs/Enrichment_motif.html" target="_blank">2ddd597</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/244cfc0ea215043ba095fa3a3f6af876da837f27/analysis/Enrichment_motif.Rmd" target="_blank">244cfc0</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-15
</td>
<td>
Adding in no extension
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/5115c784fbe50ada8ceba26aa953838cbeb79174/docs/Enrichment_motif.html" target="_blank">5115c78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1cc42d79947bff39826b90c3f33bf9d7f50a784d/analysis/Enrichment_motif.Rmd" target="_blank">1cc42d7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
updates to code to fix broken string
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/8d44ffbe4c257ea851be5795189adfaac98ee86a/docs/Enrichment_motif.html" target="_blank">8d44ffb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/caecf0befbdf2efe22332b08aad7bf561cf41c27/analysis/Enrichment_motif.Rmd" target="_blank">caecf0b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
adding new page
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
library(bedr)</code></pre>
<div id="knowles-supp-data-check" class="section level3">
<h3>Knowles supp data check</h3>
<pre class="r"><code>Knowles_4&lt;- readRDS(&quot;data/Knowles_4.RDS&quot;)
Knowles_5 &lt;- readRDS(&quot;data/Knowles_5.RDS&quot;)
Knowles_6 &lt;- readRDS(&quot;data/Knowles_6.RDS&quot;)
SNP_supp_schneider &lt;- readRDS(&quot;data/SNP_supp_schneider.RDS&quot;)
peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)
list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)</code></pre>
<pre><code>&lt;environment: R_GlobalEnv&gt;</code></pre>
<pre class="r"><code>###How I made the granges files for overlaps
Knowles_4_noext &lt;- Knowles_4 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()


Knowles_4 &lt;- Knowles_4 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-25, end=pos+25) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()


# 
Knowles_5_noext &lt;- Knowles_5 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()

Knowles_5 &lt;- Knowles_5 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-25, end=pos+25) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()

Knowles_6_noext &lt;- Knowles_6 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, gene,pos) %&gt;%
  GRanges()
 
 Knowles_6 &lt;- Knowles_6 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-25, end=pos+25) %&gt;%
  dplyr::select(chr, start,end, gene,pos) %&gt;%
  GRanges()
 
 
  SNP_supp_schneider &lt;-  SNP_supp_schneider %&gt;%
    na.omit() %&gt;%
     rename(X1=&quot;chr&quot;,X2= &quot;pos&quot;,X4=&quot;RSID&quot;) %&gt;%
    # rename(&quot;chr&quot;=X1, &quot;pos&quot; =X2,&quot;RSID&quot; =X4) %&gt;%
    mutate(start=pos-25, end=pos+25) %&gt;%
   dplyr::select(chr, start,end, RSID,pos) %&gt;%
    GRanges()

##split into granges objects
###make dataframe of each EAR,ESR,LR,NR promoters

EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_promotor &lt;- EAR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
EAR_df_gr &lt;-  GRanges(EAR_df)

# rtracklayer::export.bed(EAR_promo_gr, con=&quot;data/n45_bedfiles/EAR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_promotor &lt;- ESR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
ESR_df_gr &lt;-  GRanges(ESR_df)
 # rtracklayer::export.bed(ESR_promo_gr, con=&quot;data/n45_bedfiles/ESR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
 
LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_promotor &lt;- LR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
LR_df_gr &lt;-  GRanges(LR_df)
 # rtracklayer::export.bed(LR_promo_gr, con=&quot;data/n45_bedfiles/LR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
 # LR_promotor %&gt;% 
 #   dplyr::filter(geneChr==6)

NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_promotor &lt;- NR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
  
NR_df_gr &lt;-  GRanges(NR_df)</code></pre>
<p>Checking for enrichment:</p>
<p>I have made 25 bp +/- files for each of the data sets using the eQTL
or SNP location. I also made a 1bp no extensin file for each of the data
sets using the eQTL or SNP location as start and end as +1.</p>
<p>report:</p>
<table>
<caption>number of intersections between groups</caption>
<thead>
<tr class="header">
<th>motif</th>
<th>K_4(500)</th>
<th>K_5(359)</th>
<th>K_6(447)</th>
<th>schn_g(112)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>3</td>
<td>2</td>
<td>2</td>
<td>0</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>14</td>
<td>10</td>
<td>9</td>
<td>1</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>32</td>
<td>15</td>
<td>19</td>
<td>2</td>
</tr>
</tbody>
</table>
<p>report:</p>
<table>
<caption>number of intersections between groups without
extension</caption>
<thead>
<tr class="header">
<th>motif</th>
<th>K_4(500)</th>
<th>K_5(359)</th>
<th>K_6(447)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>1</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>3</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>13</td>
<td>9</td>
<td>8</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>30</td>
<td>15</td>
<td>19</td>
</tr>
</tbody>
</table>
<p>These numbers are different than the original lists from which they
derived. This is normally due to running <code>na.omit()</code> on the
data frame before GRanges transformation.<br />
</p>
</div>
<div id="gwas" class="section level3">
<h3>GWAS</h3>
<p>I downloaded 5 different trait associated lists from the GWAS
catalog</p>
<p>trait 1, 11 associations with trait
<strong>cardiotoxicity</strong></p>
<p>trait 2, 1068 associations with trait <strong>cardiac
arrhythmia</strong></p>
<p>trait 3, 40 associations with trait <strong>response to
anthracycline-based chemotherapy</strong></p>
<p>trait 4, 6139 associations with trait <strong>heart
disease</strong></p>
<p>trait 5, 447 associations with trait <strong>heart
failure</strong></p>
<pre class="r"><code>##finding GWAS for Hearfailure: I used several traits from the GWAS database:
##1_cardiotoxicity  11 associations   gwas_1
gwas_1 &lt;- readRDS(&quot;data/gwas_1_dataframe.RDS&quot;)
##2_ cardiac arrhythmia  1068 associations   gwas_2
gwas_2 &lt;- readRDS(&quot;data/gwas_2_dataframe.RDS&quot;)
##3_ Anthracycline_response   40 gwas_3
gwas_3 &lt;- readRDS(&quot;data/gwas_3_dataframe.RDS&quot;)

##4_ heart disease (biggest # of associations 6139) 
gwas_4 &lt;- readRDS(&quot;data/gwas_4_dataframe.RDS&quot;)

##5_ heart_failure gwas_5
gwas_5 &lt;- readRDS(&quot;data/gwas_5_dataframe.RDS&quot;)

##6 Coronary artery disease
CAD_gwas &lt;- readRDS( &quot;data/CAD_gwas_dataframe.RDS&quot;)

MI_gwas &lt;- readRDS(&quot;data/MI_gwas.RDS&quot;)

CAD_gwas_gr &lt;- CAD_gwas %&gt;%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)

CAD_gwas_gr_noext &lt;- CAD_gwas %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)


MI_gwas_gr &lt;- MI_gwas %&gt;%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)

MI_gwas_gr_noext &lt;- MI_gwas %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit(.) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   GRanges(.)


gwas_1_gr &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_1_gr_noext &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_2_gr &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()


gwas_2_gr_noext &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()

gwas_3_gr &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
gwas_3_gr_noext &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_4_gr &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID))%&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_4_gr_noext &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_5_gr &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_5_gr_noext &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

# gwas_2[(is.na(gwas_2$CHR_POS)),]</code></pre>
<div class="line-block">motif | gwas_1 (11) | gwas_2 (1061) | gwas_3
(39) | gwas_4 (6052) | gwas_5 (445) | CAD_gwas (3482) |
|———|———|———|———|———|———|———|———|<br />
EAR (7321) | 0 | 2 | 0 | 17 | 1 | 12 |<br />
ESR (15665) | 0 | 30 | 0 | 55 | 2 | 17 |<br />
LR (41759) | 2 | 29 | 4 | 186 | 16 | 116 |<br />
NR (83658) | 0 | 91 | 0 | 292 | 11 | 146 |</div>
<table style="width:100%;">
<caption>gwas trait enrichment</caption>
<colgroup>
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
<col width="11%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>gwas_1 (11)</th>
<th>gwas_2 (1061)</th>
<th>gwas_3 (39)</th>
<th>gwas_4 (6052)</th>
<th>gwas_5 (445)</th>
<th>CAD_gwas (3482)</th>
<th>MI_gwas (399)</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>2</td>
<td>0</td>
<td>17</td>
<td>0</td>
<td>12</td>
<td>1</td>
<td></td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>0</td>
<td>30</td>
<td>0</td>
<td>55</td>
<td>2</td>
<td>17</td>
<td>0</td>
<td></td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>2</td>
<td>29</td>
<td>4</td>
<td>182</td>
<td>15</td>
<td>116</td>
<td>15</td>
<td></td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>0</td>
<td>90</td>
<td>0</td>
<td>278</td>
<td>11</td>
<td>146</td>
<td>19</td>
<td></td>
</tr>
</tbody>
</table>
<p>: gwas trait enrichment with no extension</p>
<p>These numbers are different than the original lists from which they
derived. This is normally due to running <code>na.omit()</code> on the
data frame before GRanges transformation. #### chisquare tests</p>
<pre class="r"><code>chitest_gwas2_EAR &lt;- matrix(c(2,7319,90,83568), ncol=2, byrow=TRUE)
chitest_gwas2_ESR &lt;- matrix(c(30,15635,90,83568), ncol=2, byrow=TRUE)
chitest_gwas2_LR &lt;- matrix(c(20,41739,90,83568), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas2_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas2_EAR
X-squared = 3.5352, df = 1, p-value = 0.06008</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas2_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas2_ESR
X-squared = 7.0222, df = 1, p-value = 0.008051</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas2_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas2_LR
X-squared = 10.653, df = 1, p-value = 0.001099</code></pre>
<pre class="r"><code>chitest_gwas4_EAR &lt;- matrix(c(17,7304,292,83366), ncol=2, byrow=TRUE)
chitest_gwas4_ESR &lt;- matrix(c(55,15610,292,83366), ncol=2, byrow=TRUE)
chitest_gwas4_LR &lt;- matrix(c(186,41573,292,83366), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas4_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas4_EAR
X-squared = 2.3805, df = 1, p-value = 0.1229</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas4_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas4_ESR
X-squared = 1.2433e-27, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chisq.test(chitest_gwas4_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_gwas4_LR
X-squared = 6.5625, df = 1, p-value = 0.01042</code></pre>
<pre class="r"><code>chitest_K4_EAR &lt;- matrix(c(0,7321,32,83626), ncol=2, byrow=TRUE)
chitest_K4_ESR &lt;- matrix(c(3,15662,32,83626), ncol=2, byrow=TRUE)
chitest_K4_LR &lt;- matrix(c(14,417545,32,83626), ncol=2, byrow=TRUE)
chisq.test(chitest_K4_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K4_EAR
X-squared = 1.8191, df = 1, p-value = 0.1774</code></pre>
<pre class="r"><code>chisq.test(chitest_K4_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K4_ESR
X-squared = 0.87801, df = 1, p-value = 0.3487</code></pre>
<pre class="r"><code>chisq.test(chitest_K4_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K4_LR
X-squared = 88.73, df = 1, p-value &lt; 2.2e-16</code></pre>
<pre class="r"><code>chitest_K5_EAR &lt;- matrix(c(1,7320,15,83643), ncol=2, byrow=TRUE)
chitest_K5_ESR &lt;- matrix(c(2,15663,15,83643), ncol=2, byrow=TRUE)
chitest_K5_LR &lt;- matrix(c(10,41749,15,83643), ncol=2, byrow=TRUE)
chisq.test(chitest_K5_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K5_EAR
X-squared = 1.5949e-23, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chisq.test(chitest_K5_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K5_ESR
X-squared = 0.014542, df = 1, p-value = 0.904</code></pre>
<pre class="r"><code>chisq.test(chitest_K5_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K5_LR
X-squared = 0.24911, df = 1, p-value = 0.6177</code></pre>
<pre class="r"><code>chitest_K6_EAR &lt;- matrix(c(0,7321,19,83639), ncol=2, byrow=TRUE)
chitest_K6_ESR &lt;- matrix(c(2,15663,19,83639), ncol=2, byrow=TRUE)
chitest_K6_LR &lt;- matrix(c(9,41750,19,83639), ncol=2, byrow=TRUE)
chisq.test(chitest_K6_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K6_EAR
X-squared = 0.75318, df = 1, p-value = 0.3855</code></pre>
<pre class="r"><code>chisq.test(chitest_K6_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K6_ESR
X-squared = 0.23644, df = 1, p-value = 0.6268</code></pre>
<pre class="r"><code>chisq.test(chitest_K6_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_K6_LR
X-squared = 2.4408e-24, df = 1, p-value = 1</code></pre>
<pre class="r"><code>chitest_CAD_EAR &lt;- matrix(c(12,7309,146,83512), ncol=2, byrow=TRUE)
chitest_CAD_ESR &lt;- matrix(c(17,15648,146,83512), ncol=2, byrow=TRUE)
chitest_CAD_LR &lt;- matrix(c(116,41643,146,83512), ncol=2, byrow=TRUE)
chisq.test(chitest_CAD_EAR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_CAD_EAR
X-squared = 0.0039284, df = 1, p-value = 0.95</code></pre>
<pre class="r"><code>chisq.test(chitest_CAD_ESR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_CAD_ESR
X-squared = 3.1165, df = 1, p-value = 0.07751</code></pre>
<pre class="r"><code>chisq.test(chitest_CAD_LR)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_CAD_LR
X-squared = 13.757, df = 1, p-value = 0.000208</code></pre>
<pre class="r"><code>##sub-setting  peaks that overlap with at least 1 SNP in gwas2 noext

EAR_g2_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_2_gr_noext))
# subsetByOverlaps(gwas_2_gr_noext, ESR_df_gr)
ESR_g2_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_2_gr_noext))
LR_g2_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_2_gr_noext))
NR_g2_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_2_gr_noext))

EAR_g4_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_4_gr_noext))
ESR_g4_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_4_gr_noext))
LR_g4_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_4_gr_noext))
NR_g4_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_4_gr_noext))

EAR_g1_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_1_gr_noext))

ESR_g1_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_1_gr_noext))
LR_g1_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_1_gr_noext))
NR_g1_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_1_gr_noext))


EAR_g3_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_3_gr_noext))

ESR_g3_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_3_gr_noext))
LR_g3_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_3_gr_noext))
NR_g3_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_3_gr_noext))

EAR_g5_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_5_gr_noext))

ESR_g5_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_5_gr_noext))
LR_g5_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_5_gr_noext))
NR_g5_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_5_gr_noext))

EAR_MI_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, MI_gwas_gr))

ESR_MI_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, MI_gwas_gr))
LR_MI_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  MI_gwas_gr))
NR_MI_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, MI_gwas_gr))

EAR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(EAR_df_gr, CAD_gwas_gr))

ESR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(ESR_df_gr, CAD_gwas_gr))
LR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(LR_df_gr,  CAD_gwas_gr))
NR_CAD_sol &lt;- as.data.frame(subsetByOverlaps(NR_df_gr, CAD_gwas_gr))

###Now I can pull the geneID list and see if genes are DE from RNAseq data

toplistall_RNA &lt;- readRDS(&quot;data/other_papers/toplistall_RNA.RDS&quot;) 

###Because of how I applied the DEG system in RNA-seq analysis, the lFC is opposite of the 
###counts.   I did trt-veh instead of veh-trt.  therefore I need to multiply lfc by -1 to get t
###the right correlation.

toplistall_RNA &lt;- toplistall_RNA %&gt;% 
  mutate(logFC = logFC*(-1))

RNA_expresed_genes &lt;- toplistall_RNA %&gt;% 
  # dplyr::filter(adj.P.Val &lt;0.05) %&gt;% 
   mutate(expression = if_else(logFC&lt;0,&quot;down&quot;,&quot;up&quot;)) %&gt;% 
  dplyr::select(ENTREZID,SYMBOL,expression) %&gt;% 
  # dplyr::select(ENTREZID,SYMBOL) %&gt;% 
  unique(.)  
RNA_expresed_genes_DE &lt;- toplistall_RNA %&gt;% 
  dplyr::filter(adj.P.Val &lt;0.05) %&gt;%
  mutate(expression = if_else(logFC&lt;0,&quot;down&quot;,&quot;up&quot;)) %&gt;% 
  dplyr::select(ENTREZID,SYMBOL,expression) %&gt;% 
  unique(.)  

RNA_expresed_genes %&gt;% 
  dplyr::filter(ENTREZID %in% NR_g4_sol$geneID)</code></pre>
<pre><code>[1] ENTREZID   SYMBOL     expression
&lt;0 rows&gt; (or 0-length row.names)</code></pre>
<pre class="r"><code>NR_g2_list &lt;- intersect(RNA_expresed_genes$ENTREZID,NR_g2_sol$geneId)
LR_g2_list &lt;- intersect(RNA_expresed_genes$ENTREZID,LR_g2_sol$geneId)
ESR_g2_list &lt;- intersect(RNA_expresed_genes$ENTREZID,ESR_g2_sol$geneId)
EAR_g2_list &lt;- intersect(RNA_expresed_genes$ENTREZID,EAR_g2_sol$geneId)
g2_motif_list &lt;-list(NR=data.frame(NR_g2_list), LR=data.frame(LR_g2_list),ESR=data.frame(ESR_g2_list),EAR=data.frame(EAR_g2_list))
g2_motif_df &lt;- g2_motif_list %&gt;%  map_df(c, .id = &quot;src&quot;) %&gt;%
  mutate(ENTREZID=if_else(!is.na(EAR_g2_list),EAR_g2_list,if_else(!is.na(ESR_g2_list),ESR_g2_list, if_else(!is.na(LR_g2_list), LR_g2_list,NR_g2_list))))%&gt;%
  dplyr::select(src,ENTREZID)
NR_g4_list &lt;-intersect(RNA_expresed_genes$ENTREZID,NR_g4_sol$geneId)
LR_g4_list &lt;- intersect(RNA_expresed_genes$ENTREZID,LR_g4_sol$geneId)
ESR_g4_list &lt;- intersect(RNA_expresed_genes$ENTREZID,ESR_g4_sol$geneId)
EAR_g4_list &lt;- intersect(RNA_expresed_genes$ENTREZID,EAR_g4_sol$geneId)

g4_motif_list &lt;-list(NR=data.frame(NR_g4_list), LR=data.frame(LR_g4_list),ESR=data.frame(ESR_g4_list),EAR=data.frame(EAR_g4_list))
g4_motif_df &lt;- g4_motif_list %&gt;%  map_df(c, .id = &quot;src&quot;) %&gt;%
  mutate(ENTREZID=if_else(!is.na(EAR_g4_list),EAR_g4_list,if_else(!is.na(ESR_g4_list),ESR_g4_list, if_else(!is.na(LR_g4_list), LR_g4_list,NR_g4_list))))%&gt;%
  dplyr::select(src,ENTREZID)

RNA_expresed_genes_DE %&gt;% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g4_sol$geneId)) %&gt;% 
   rowid_to_column(&quot;row_name&quot;) %&gt;% 
  kable(., caption= &quot;RNA DE genes expresssed in LR DARs that contain SNPs associated with Heart disease&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
RNA DE genes expresssed in LR DARs that contain SNPs associated with
Heart disease
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
expression
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
6331
</td>
<td style="text-align:left;">
SCN5A
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
1723
</td>
<td style="text-align:left;">
DHODH
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
8178
</td>
<td style="text-align:left;">
ELL
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
5366
</td>
<td style="text-align:left;">
PMAIP1
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
79068
</td>
<td style="text-align:left;">
FTO
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
993
</td>
<td style="text-align:left;">
CDC25A
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
10933
</td>
<td style="text-align:left;">
MORF4L1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
130733
</td>
<td style="text-align:left;">
TMEM178A
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
10529
</td>
<td style="text-align:left;">
NEBL
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
9314
</td>
<td style="text-align:left;">
KLF4
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
7145
</td>
<td style="text-align:left;">
TNS1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
13
</td>
<td style="text-align:left;">
4088
</td>
<td style="text-align:left;">
SMAD3
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;">
23387
</td>
<td style="text-align:left;">
SIK3
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
15
</td>
<td style="text-align:left;">
9537
</td>
<td style="text-align:left;">
TP53I11
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
16
</td>
<td style="text-align:left;">
27258
</td>
<td style="text-align:left;">
LSM3
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
17
</td>
<td style="text-align:left;">
285025
</td>
<td style="text-align:left;">
CCDC141
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
18
</td>
<td style="text-align:left;">
63976
</td>
<td style="text-align:left;">
PRDM16
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
23293
</td>
<td style="text-align:left;">
SMG6
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
20
</td>
<td style="text-align:left;">
65990
</td>
<td style="text-align:left;">
ANTKMT
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
21
</td>
<td style="text-align:left;">
283871
</td>
<td style="text-align:left;">
PGP
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
22
</td>
<td style="text-align:left;">
463
</td>
<td style="text-align:left;">
ZFHX3
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
23
</td>
<td style="text-align:left;">
88
</td>
<td style="text-align:left;">
ACTN2
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
24
</td>
<td style="text-align:left;">
23022
</td>
<td style="text-align:left;">
PALLD
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
25
</td>
<td style="text-align:left;">
201294
</td>
<td style="text-align:left;">
UNC13D
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
26
</td>
<td style="text-align:left;">
23075
</td>
<td style="text-align:left;">
SWAP70
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
27
</td>
<td style="text-align:left;">
10409
</td>
<td style="text-align:left;">
BASP1
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
28
</td>
<td style="text-align:left;">
9620
</td>
<td style="text-align:left;">
CELSR1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
29
</td>
<td style="text-align:left;">
151126
</td>
<td style="text-align:left;">
ZNF385B
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
30
</td>
<td style="text-align:left;">
4952
</td>
<td style="text-align:left;">
OCRL
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
31
</td>
<td style="text-align:left;">
109
</td>
<td style="text-align:left;">
ADCY3
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
32
</td>
<td style="text-align:left;">
221154
</td>
<td style="text-align:left;">
MICU2
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
33
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
34
</td>
<td style="text-align:left;">
7741
</td>
<td style="text-align:left;">
ZSCAN26
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
35
</td>
<td style="text-align:left;">
26040
</td>
<td style="text-align:left;">
SETBP1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
36
</td>
<td style="text-align:left;">
4088
</td>
<td style="text-align:left;">
SMAD3
</td>
<td style="text-align:left;">
down
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>RNA_expresed_genes_DE %&gt;% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g2_sol$geneId)) %&gt;% 
   rowid_to_column(&quot;row_name&quot;) %&gt;% 
  kable(., caption= &quot;RNA DE genes expresssed in LR DARs that contain SNPs associated with Cardiac arrhythmia&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
RNA DE genes expresssed in LR DARs that contain SNPs associated with
Cardiac arrhythmia
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
expression
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
1952
</td>
<td style="text-align:left;">
CELSR2
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
10529
</td>
<td style="text-align:left;">
NEBL
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
23387
</td>
<td style="text-align:left;">
SIK3
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
283871
</td>
<td style="text-align:left;">
PGP
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
463
</td>
<td style="text-align:left;">
ZFHX3
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
151126
</td>
<td style="text-align:left;">
ZNF385B
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
221154
</td>
<td style="text-align:left;">
MICU2
</td>
<td style="text-align:left;">
up
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>RNA_expresed_genes_DE %&gt;% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g4_sol$geneId)) %&gt;% 
   rowid_to_column(&quot;row_name&quot;) %&gt;% 
  kable(., caption= &quot;RNA DE genes expresssed in ESR DARs that contain SNPs associated with Heart disease&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
RNA DE genes expresssed in ESR DARs that contain SNPs associated with
Heart disease
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
expression
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
3757
</td>
<td style="text-align:left;">
KCNH2
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
5430
</td>
<td style="text-align:left;">
POLR2A
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
1026
</td>
<td style="text-align:left;">
CDKN1A
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
3782
</td>
<td style="text-align:left;">
KCNN3
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
285313
</td>
<td style="text-align:left;">
IGSF10
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
27129
</td>
<td style="text-align:left;">
HSPB7
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
153090
</td>
<td style="text-align:left;">
DAB2IP
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
341
</td>
<td style="text-align:left;">
APOC1
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
54897
</td>
<td style="text-align:left;">
CASZ1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
9644
</td>
<td style="text-align:left;">
SH3PXD2A
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:left;">
down
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>RNA_expresed_genes_DE %&gt;% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g2_sol$geneId)) %&gt;% 
   rowid_to_column(&quot;row_name&quot;) %&gt;% 
  kable(., caption= &quot;RNA DE genes expresssed in ESR DARs that contain SNPs associated with Cardiac arrhythmia&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
RNA DE genes expresssed in ESR DARs that contain SNPs associated with
Cardiac arrhythmia
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
expression
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
3757
</td>
<td style="text-align:left;">
KCNH2
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
5430
</td>
<td style="text-align:left;">
POLR2A
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
1026
</td>
<td style="text-align:left;">
CDKN1A
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
3782
</td>
<td style="text-align:left;">
KCNN3
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
54897
</td>
<td style="text-align:left;">
CASZ1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
9644
</td>
<td style="text-align:left;">
SH3PXD2A
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
10021
</td>
<td style="text-align:left;">
HCN4
</td>
<td style="text-align:left;">
down
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>RNA_expresed_genes_DE %&gt;% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g4_sol$geneId)) %&gt;% 
   rowid_to_column(&quot;row_name&quot;) %&gt;% 
  kable(., caption= &quot;RNA DE genes expresssed in EAR DARs that contain SNPs associated with Heart disease&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
RNA DE genes expresssed in EAR DARs that contain SNPs associated with
Heart disease
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
expression
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
54443
</td>
<td style="text-align:left;">
ANLN
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
10611
</td>
<td style="text-align:left;">
PDLIM5
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
7145
</td>
<td style="text-align:left;">
TNS1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
352954
</td>
<td style="text-align:left;">
CASTOR3
</td>
<td style="text-align:left;">
up
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
147841
</td>
<td style="text-align:left;">
SPC24
</td>
<td style="text-align:left;">
down
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>RNA_expresed_genes%&gt;% 
  # dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes$ENTREZID,EAR_g2_sol$geneId)) %&gt;% 
   dplyr::filter(ENTREZID ==&quot;79804&quot;) %&gt;% 
   rowid_to_column(&quot;row_name&quot;) %&gt;%
  kable(., caption= &quot;RNA genes expresssed in EAR DARs that contain SNPs associated with Cardiac arrhythmia&quot;) %&gt;% 
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
RNA genes expresssed in EAR DARs that contain SNPs associated with
Cardiac arrhythmia
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ENTREZID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SYMBOL
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
expression
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
79804
</td>
<td style="text-align:left;">
HAND2-AS1
</td>
<td style="text-align:left;">
down
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
79804
</td>
<td style="text-align:left;">
HAND2-AS1
</td>
<td style="text-align:left;">
up
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
##Here my focus is to find nearest genes that are DAR and see if they are DEGs
toplist_n45 &lt;- readRDS(&quot;data/toplist_n45.RDS&quot;)

# toplist_n45 %&gt;% 
#   dplyr::filter(peak==&quot;chr4.173529475.173530537&quot;) %&gt;% 
#  ggplot(., aes(x=trt, y=logFC))+
#   geom_boxplot(aes(fill=trt))+
#   ggpubr::fill_palette(palette =drug_pal)+
#   guides(fill=guide_legend(title = &quot;Treatment&quot;))+
#   # facet_wrap(sigcount~time)+
#   theme_bw()+
#   xlab(&quot;&quot;)+
#   ggtitle(&quot;chr4.173529475.173530537 region of  HAND2-AS1&quot;)+
#   ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
#   theme_bw()+
#   facet_wrap(~time)+
#   theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
#         axis.title = element_text(size = 15, color = &quot;black&quot;),
#         # axis.ticks = element_line(linewidth = 1.5),
#         # axis.line = element_line(linewidth = 1.5),
#         strip.background = element_rect(fill = &quot;transparent&quot;),
#         axis.text.x = element_blank(),
#         strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))


toplist_n45 %&gt;% 
  dplyr::filter(peak==&quot;chr6.36678380.36679788&quot;) %&gt;%
 ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;chr6  36678380-36679788 CDKN1a&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/DAR%20and%20RNAseq-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>toplist_n45 %&gt;% 
  dplyr::filter(peak %in% EAR_g2_sol$id) %&gt;% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;absolute lfc of DARs in EAR in cardiac arrhythmia&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/DAR%20and%20RNAseq-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>toplist_n45 %&gt;% 
  dplyr::filter(peak %in% ESR_g2_sol$id) %&gt;% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;absolute lfc of DARs in ESR in cardiac arrhythmia&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/DAR%20and%20RNAseq-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>toplist_n45 %&gt;% 
  dplyr::filter(peak %in% LR_g2_sol$id) %&gt;% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ggtitle(&quot;absolute lfc of DARs in LR in cardiac arrhythmia&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/DAR%20and%20RNAseq-4.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="pie-charts-of-the-snps" class="section level3">
<h3>Pie charts of the SNPs</h3>
<pre class="r"><code>Set &lt;- c(&quot;EAR&quot;, &quot;ESR&quot;,&quot;LR&quot;, &quot;NR&quot;)
gene_num_ARR &lt;- c(2,30,29,90)
gene_num_HD &lt;- c(17,55,182,278)
gene_num_CAD &lt;- c(12,17,116,146)
gene_num_MI &lt;- c(1,0,15,19)
fills &lt;- c(&quot;#F8766D&quot;,  &quot;#00BFC4&quot;,&quot;#7CAE00&quot;, &quot;#C77CFF&quot;)

pie_chartdata_ARR &lt;- data.frame(Set, gene_num_ARR, fills)
pie_chartdata_HD &lt;- data.frame(Set, gene_num_HD, fills)
pie_chartdata_CAD &lt;- data.frame(Set, gene_num_CAD, fills)
pie_chartdata_MI &lt;- data.frame(Set, gene_num_MI, fills)

 
    # pie(gene_num,labels =Set )      
pie_chartdata_ARR &lt;- pie_chartdata_ARR %&gt;% 
   mutate(prop = gene_num_ARR / sum(pie_chartdata_ARR$gene_num_ARR) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_HD &lt;- pie_chartdata_HD %&gt;% 
   mutate(prop = gene_num_HD / sum(pie_chartdata_HD$gene_num_HD) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_CAD &lt;- pie_chartdata_CAD %&gt;% 
   mutate(prop = gene_num_CAD / sum(pie_chartdata_CAD$gene_num_CAD) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_MI &lt;- pie_chartdata_MI %&gt;% 
   mutate(prop = gene_num_MI / sum(pie_chartdata_MI$gene_num_MI) *100) %&gt;%
  mutate(ypos = (prop)+ 0.5*prop )

pie_chartdata_ARR %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_ARR, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of cardiac arrhythmia SNPs in DARs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_ARR,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>pie_chartdata_HD %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_HD, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of heart disease SNPs in DARs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_HD,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>pie_chartdata_CAD %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_CAD, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of coronary artery disease SNPs in DARs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_CAD,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>pie_chartdata_MI %&gt;% 
  ggplot(.,aes(x=&quot;&quot;,y=gene_num_MI, fill=Set))+
  geom_col(width =1) +
  coord_polar(&quot;y&quot;, pi/2)+
  theme_void()+
  ggtitle(&quot;Distribution of myocardial infarction SNPs in DARs&quot;)+
  geom_text(aes(label = paste0(Set,&quot; (&quot;,gene_num_MI,&quot;)&quot;)),
                position = position_stack(vjust =.45)) +
  theme(legend.position=&quot;none&quot;) +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/piechart%20noext-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>### Code to express # of DE vs non DE genes associated with SNPs found in DARs

# toplist_n45 &lt;- readRDS(&quot;data/toplist_n45.RDS&quot;)
# 
# toplist_n45 %&gt;% 
#   mutate(DAR= if_else(adj.P.Val&lt;0.05,&quot;DAR&quot;,&quot;notDAR&quot;)) 
         


GWAS_2_DAR_df &lt;- EAR_g2_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g2_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g2_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g2_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,NR_g2_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 

bind_rows(

ESR_g2_sol %&gt;% 
   mutate(DE_g2= if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g2_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g2_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g2_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,NR_g2_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
  mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
LR_g2_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g2_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g2_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g2_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,NR_g2_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
  mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
NR_g2_sol %&gt;%
   mutate(DE_g2= if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g2_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g2_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g2_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% intersect(RNA_expresed_genes_DE$ENTREZID,NR_g2_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
  mutate(id=&quot;NR&quot;))
         
    
GWAS_4_DAR_df &lt;- EAR_g4_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% 
                          intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g4_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g4_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g4_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g4_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 
bind_rows(
  ESR_g4_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in%
                            intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g4_sol$geneId),&quot;DE&quot;,
                          if_else(geneId %in% 
                                    intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g4_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                      intersect(RNA_expresed_genes_DE$ENTREZID,LR_g4_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID,NR_g4_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
    mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
    LR_g4_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g4_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g4_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                        intersect(RNA_expresed_genes_DE$ENTREZID, LR_g4_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g4_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
    NR_g4_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g4_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g4_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g4_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g4_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;NR&quot;))



GWAS_1_DAR_df &lt;- EAR_g1_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% 
                          intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g1_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g1_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g1_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g1_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 
bind_rows(
  ESR_g1_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in%
                            intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g1_sol$geneId),&quot;DE&quot;,
                          if_else(geneId %in% 
                                    intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g1_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                      intersect(RNA_expresed_genes_DE$ENTREZID,LR_g1_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID,NR_g1_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
    mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
    LR_g1_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g1_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g1_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                        intersect(RNA_expresed_genes_DE$ENTREZID, LR_g1_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g1_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
    NR_g1_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g1_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g1_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g1_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g1_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;NR&quot;))

GWAS_3_DAR_df &lt;- EAR_g3_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% 
                          intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g3_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g3_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g3_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g3_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 
bind_rows(
  ESR_g3_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in%
                            intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g3_sol$geneId),&quot;DE&quot;,
                          if_else(geneId %in% 
                                    intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g3_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                      intersect(RNA_expresed_genes_DE$ENTREZID,LR_g3_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID,NR_g3_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
    mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
    LR_g3_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g3_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g3_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                        intersect(RNA_expresed_genes_DE$ENTREZID, LR_g3_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g3_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
    NR_g3_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g3_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g3_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g3_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g3_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;NR&quot;))

GWAS_5_DAR_df &lt;- EAR_g5_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% 
                          intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g5_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g5_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g5_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g5_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 
bind_rows(
  ESR_g5_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in%
                            intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g5_sol$geneId),&quot;DE&quot;,
                          if_else(geneId %in% 
                                    intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g5_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                      intersect(RNA_expresed_genes_DE$ENTREZID,LR_g5_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID,NR_g5_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
    mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
    LR_g5_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g5_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g5_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                        intersect(RNA_expresed_genes_DE$ENTREZID, LR_g5_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g5_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
    NR_g5_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g5_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g5_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_g5_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_g5_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;NR&quot;))



GWAS_MI_DAR_df &lt;- EAR_MI_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% 
                          intersect(RNA_expresed_genes_DE$ENTREZID,ESR_MI_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_MI_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_MI_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_MI_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 
bind_rows(
  ESR_MI_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in%
                            intersect(RNA_expresed_genes_DE$ENTREZID,ESR_MI_sol$geneId),&quot;DE&quot;,
                          if_else(geneId %in% 
                                    intersect(RNA_expresed_genes_DE$ENTREZID,EAR_MI_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                      intersect(RNA_expresed_genes_DE$ENTREZID,LR_MI_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID,NR_MI_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
    mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
    LR_MI_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_MI_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_MI_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                        intersect(RNA_expresed_genes_DE$ENTREZID, LR_MI_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_MI_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
    NR_MI_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_MI_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_MI_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_MI_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_MI_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;NR&quot;))




GWAS_CAD_DAR_df &lt;- EAR_CAD_sol %&gt;%
  mutate(DE_g2= if_else(geneId %in% 
                          intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;))))) %&gt;%
  mutate(id=&quot;EAR&quot;) %&gt;% 
bind_rows(
  ESR_CAD_sol %&gt;% 
    mutate(DE_g2= if_else(geneId %in%
                            intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),&quot;DE&quot;,
                          if_else(geneId %in% 
                                    intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                      intersect(RNA_expresed_genes_DE$ENTREZID,LR_CAD_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID,NR_CAD_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
    mutate(id=&quot;ESR&quot;)) %&gt;% 
  bind_rows(
    LR_CAD_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                        intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;LR&quot;)) %&gt;% 
  bind_rows(
    NR_CAD_sol %&gt;%
      mutate(DE_g2= if_else(geneId %in% 
                              intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),&quot;DE&quot;,
                        if_else(geneId %in% 
                                  intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),&quot;DE&quot;,
                                if_else(geneId %in% 
                                          intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),&quot;DE&quot;,
                                        if_else(geneId %in% 
                                                  intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), &quot;notDE&quot;,&quot;NE&quot;)))))%&gt;%
      mutate(id=&quot;NR&quot;))</code></pre>
<pre class="r"><code>GWAS_2_DAR_df %&gt;% 
  dplyr::filter(DE_g2==&quot;DE&quot;)</code></pre>
<pre><code>   seqnames     start       end width strand  id
1      chr1  10736684  10737808  1125      * ESR
2      chr1 154839629 154839995   367      * ESR
3      chr6  36678380  36679788  1409      * ESR
4      chr7 150954727 150956459  1733      * ESR
5     chr10 103720444 103721025   582      * ESR
6     chr15  73374622  73375213   592      * ESR
7     chr17   7510984   7512636  1653      * ESR
8      chr1 109274953 109276202  1250      *  LR
9      chr2 179701478 179702086   609      *  LR
10    chr10  20953148  20953475   328      *  LR
11    chr11 117037235 117037719   485      *  LR
12    chr13  21536977  21537596   620      *  LR
13    chr16   2214229   2215681  1453      *  LR
14    chr16  72877300  72877905   606      *  LR
15    chr10 103712814 103713354   541      *  NR
16    chr15  73375634  73377363  1730      *  NR
17    chr16  73018778  73020183  1406      *  NR
                                          annotation geneChr geneStart
1   Intron (ENST00000377022.8/54897, intron 2 of 20)       1  10704196
2     Intron (ENST00000271915.9/3782, intron 1 of 7)       1 154707426
3                                   Promoter (&lt;=1kb)       6  36678710
4                                   Promoter (&lt;=1kb)       7 150944961
5    Intron (ENST00000369774.9/9644, intron 5 of 14)      10 103600292
6                                  Distal Intergenic      15  73319859
7                                   Promoter (&lt;=1kb)      17   7511515
8                                             3&#39; UTR       1 109270824
9   Intron (ENST00000410066.7/151126, intron 3 of 9)       2 179544827
10   Intron (ENST00000675700.1/10529, intron 4 of 6)      10  20859283
11  Intron (ENST00000445177.6/23387, intron 1 of 24)      11 116845593
12 Intron (ENST00000382374.9/221154, intron 4 of 11)      13  21502827
13                                  Promoter (&lt;=1kb)      16   2211593
14    Intron (ENST00000268489.10/463, intron 4 of 9)      16  72829509
15   Intron (ENST00000369774.9/9644, intron 5 of 14)      10 103599520
16                                 Distal Intergenic      15  73319859
17    Intron (ENST00000268489.10/463, intron 1 of 9)      16  72782885
     geneEnd geneLength geneStrand geneId       transcriptId distanceToTSS
1   10705575       1380          2  54897  ENST00000472814.1        -31109
2  154859843     152418          2   3782  ENST00000361147.8         19848
3   36687337       8628          1   1026  ENST00000405375.5             0
4  150955804      10844          2   3757  ENST00000330883.9             0
5  103736866     136575          2   9644  ENST00000692756.1         15841
6   73368958      49100          2  10021  ENST00000261917.4         -5664
7    7512481        967          1   5430  ENST00000576553.1             0
8  109275607       4784          1   1952  ENST00000498157.1          4129
9  179660182     115356          2 151126  ENST00000463918.1        -41296
10  20960012     100730          2  10529  ENST00000674540.1          6537
11 117098271     252679          2  23387  ENST00000415541.5         60552
12  21515661      12835          2 221154  ENST00000480341.5        -21316
13   2214840       3248          2 283871  ENST00000333503.8             0
14  72889834      60326          2    463  ENST00000563625.1         11929
15 103693623      94104          2   9644  ENST00000315994.6        -19191
16  73368958      49100          2  10021  ENST00000261917.4         -6676
17  73048128     265244          2    463 ENST00000268489.10         27945
   DE_g2
1     DE
2     DE
3     DE
4     DE
5     DE
6     DE
7     DE
8     DE
9     DE
10    DE
11    DE
12    DE
13    DE
14    DE
15    DE
16    DE
17    DE</code></pre>
<pre class="r"><code>GWAS_4_DAR_df %&gt;% 
  dplyr::filter(DE_g2==&quot;DE&quot;)</code></pre>
<pre><code>   seqnames     start       end width strand  id
1      chr2 217804449 217806257  1809      * EAR
2      chr4  94575649  94576017   369      * EAR
3      chr7  36459049  36459918   870      * EAR
4      chr7 100243501 100243833   333      * EAR
5     chr19  11142349  11143639  1291      * EAR
6      chr1  10736684  10737808  1125      * ESR
7      chr1  16012567  16013729  1163      * ESR
8      chr1 154839629 154839995   367      * ESR
9      chr3 151483793 151484643   851      * ESR
10     chr6  36678380  36679788  1409      * ESR
11     chr7 150954727 150956459  1733      * ESR
12     chr9 121650071 121650712   642      * ESR
13    chr10 103720444 103721025   582      * ESR
14    chr15  73374622  73375213   592      * ESR
15    chr17   7510984   7512636  1653      * ESR
16    chr19  44911105  44911471   367      * ESR
17     chr1   3408398   3409463  1066      *  LR
18     chr1 109274953 109276202  1250      *  LR
19     chr1 236688731 236689178   448      *  LR
20     chr2  24818716  24819139   424      *  LR
21     chr2  39631404  39632344   941      *  LR
22     chr2 178846382 178846751   370      *  LR
23     chr2 179701478 179702086   609      *  LR
24     chr2 217868752 217869273   522      *  LR
25     chr3  14232390  14233136   747      *  LR
26     chr3  38615932  38616880   949      *  LR
27     chr3  48151277  48152429  1153      *  LR
28     chr4 168766280 168767270   991      *  LR
29     chr5  17118580  17119099   520      *  LR
30     chr6  28269507  28270003   497      *  LR
31     chr9 107755276 107755816   541      *  LR
32    chr10  20953148  20953475   328      *  LR
33    chr11   9740217   9740734   518      *  LR
34    chr11  44955010  44955513   504      *  LR
35    chr11 117037235 117037719   485      *  LR
36    chr13  21536977  21537596   620      *  LR
37    chr15  67165920  67166709   790      *  LR
38    chr15  73353612  73354309   698      *  LR
39    chr15  78830933  78832028  1096      *  LR
40    chr16    719886    721785  1900      *  LR
41    chr16   2214229   2215681  1453      *  LR
42    chr16  53768577  53768944   368      *  LR
43    chr16  72008437  72009100   664      *  LR
44    chr16  72877300  72877905   606      *  LR
45    chr17   2222861   2223544   684      *  LR
46    chr17  75844492  75845572  1081      *  LR
47    chr18  44733224  44733992   769      *  LR
48    chr18  60067561  60068151   591      *  LR
49    chr19  18478653  18479269   617      *  LR
50    chr22  46417011  46418228  1218      *  LR
51     chrX 129601003 129601369   367      *  LR
52     chr3  14216199  14216968   770      *  NR
53     chr9 121651049 121653070  2022      *  NR
54    chr10 103712814 103713354   541      *  NR
55    chr15  67071010  67071262   253      *  NR
56    chr15  73375634  73377363  1730      *  NR
57    chr16  73018778  73020183  1406      *  NR
58    chr17   2214040   2216112  2073      *  NR
                                                annotation geneChr geneStart
1                                                   3&#39; UTR       2 217809714
2                                                   3&#39; UTR       4  94579221
3                                        Distal Intergenic       7  36448939
4         Intron (ENST00000328453.9/352954, intron 1 of 7)       7 100222645
5         Intron (ENST00000585567.5/147841, intron 4 of 5)      19  11147134
6         Intron (ENST00000377022.8/54897, intron 2 of 20)       1  10704196
7                                        Distal Intergenic       1  16015374
8           Intron (ENST00000271915.9/3782, intron 1 of 7)       1 154707426
9                                        Distal Intergenic       3 151436203
10                                        Promoter (&lt;=1kb)       6  36678710
11                                        Promoter (&lt;=1kb)       7 150944961
12                                        Promoter (&lt;=1kb)       9 121651634
13         Intron (ENST00000369774.9/9644, intron 5 of 14)      10 103600292
14                                       Distal Intergenic      15  73319859
15                                        Promoter (&lt;=1kb)      17   7511515
16 Exon (ENST00000623895.1/ENST00000623895.1, exon 1 of 1)      19  44914247
17        Intron (ENST00000511072.5/63976, intron 8 of 15)       1   3396491
18                                                  3&#39; UTR       1 109270824
19                                        Promoter (1-2kb)       1 236686735
20                                        Promoter (1-2kb)       2  24819369
21        Intron (ENST00000415640.1/728730, intron 1 of 2)       2  39664982
22     Intron (ENST00000652826.1/105373766, intron 1 of 5)       2 178830269
23        Intron (ENST00000410066.7/151126, intron 3 of 9)       2 179544827
24        Intron (ENST00000646520.1/7145, intron 18 of 32)       2 217848472
25                                       Distal Intergenic       3  14178817
26         Intron (ENST00000414099.6/6331, intron 4 of 25)       3  38629842
27                                       Distal Intergenic       3  48164402
28       Intron (ENST00000505667.6/23022, intron 10 of 21)       4 168712159
29            Exon (ENST00000665920.1/285696, exon 3 of 3)       5  17065598
30         Intron (ENST00000316606.10/7741, intron 1 of 3)       6  28267359
31                                       Distal Intergenic       9 107488989
32         Intron (ENST00000675700.1/10529, intron 4 of 6)      10  20859283
33                                                  3&#39; UTR      11   9726895
34                                       Distal Intergenic      11  44932348
35        Intron (ENST00000445177.6/23387, intron 1 of 24)      11 116845593
36       Intron (ENST00000382374.9/221154, intron 4 of 11)      13  21502827
37                                        Promoter (&lt;=1kb)      15  67166019
38         Intron (ENST00000261917.4/10021, intron 1 of 7)      15  73319859
39 Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)      15  78847667
40                                        Promoter (&lt;=1kb)      16    720581
41                                        Promoter (&lt;=1kb)      16   2211593
42         Intron (ENST00000636491.1/79068, intron 2 of 9)      16  53826361
43                                        Promoter (&lt;=1kb)      16  72008588
44          Intron (ENST00000268489.10/463, intron 4 of 9)      16  72829509
45      Intron (ENST00000263073.11/23293, intron 10 of 18)      17   2172737
46                                        Promoter (&lt;=1kb)      17  75842427
47 Exon (ENST00000624404.1/ENST00000624404.1, exon 1 of 1)      18  44701338
48                                       Distal Intergenic      18  59900036
49         Intron (ENST00000262809.9/8178, intron 1 of 11)      19  18444751
50         Intron (ENST00000262738.9/9620, intron 5 of 34)      22  46390528
51                                       Distal Intergenic      23 129589589
52                                       Distal Intergenic       3  14178817
53                                        Promoter (&lt;=1kb)       9 121651634
54         Intron (ENST00000369774.9/9644, intron 5 of 14)      10 103599520
55          Intron (ENST00000559460.6/4088, intron 1 of 8)      15  67067785
56                                       Distal Intergenic      15  73319859
57          Intron (ENST00000268489.10/463, intron 1 of 9)      16  72782885
58                                        Promoter (&lt;=1kb)      17   2172668
     geneEnd geneLength geneStrand geneId       transcriptId distanceToTSS
1  217810304        591          2   7145  ENST00000480416.1          4047
2   94618191      38971          1  10611  ENST00000511586.5         -3204
3   36453655       4717          1  54443  ENST00000491782.1         10110
4  100246548      23904          2 352954  ENST00000538588.2          2715
5   11155761       8628          2 147841  ENST00000586708.1         12122
6   10705575       1380          2  54897  ENST00000472814.1        -31109
7   16017825       2452          2  27129  ENST00000463576.5          4096
8  154859843     152418          2   3782  ENST00000361147.8         19848
9  151461061      24859          2 285313  ENST00000282466.4        -22732
10  36687337       8628          1   1026  ENST00000405375.5             0
11 150955804      10844          2   3757  ENST00000330883.9             0
12 121785530     133897          1 153090  ENST00000408936.8          -922
13 103736866     136575          2   9644  ENST00000692756.1         15841
14  73368958      49100          2  10021  ENST00000261917.4         -5664
15   7512481        967          1   5430  ENST00000576553.1             0
16  44919342       5096          1    341  ENST00000588750.5         -2776
17   3434293      37803          1  63976  ENST00000509860.1         11907
18 109275607       4784          1   1952  ENST00000498157.1          4129
19 236720974      34240          1     88  ENST00000683075.1          1996
20  24820796       1428          2    109  ENST00000498288.1          1657
21  39707136      42155          1 130733  ENST00000437068.5        -32638
22 178837137       6869          2 285025  ENST00000472828.1         -9245
23 179660182     115356          2 151126  ENST00000463918.1        -41296
24 217858633      10162          2   7145  ENST00000480665.1        -10119
25  14201122      22306          1  27258  ENST00000306024.4         53573
26  38633220       3379          2   6331  ENST00000476683.1         16340
27  48174753      10352          2    993  ENST00000459900.1         22324
28 168898686     186528          1  23022  ENST00000510998.5         54121
29  17275353     209756          1  10409  ENST00000606445.1         52982
30  28278224      10866          1   7741  ENST00000619937.4          2148
31 107490482       1494          2   9314  ENST00000420475.1       -264794
32  20960012     100730          2  10529  ENST00000674540.1          6537
33   9750491      23597          1  23075  ENST00000534562.1         13322
34  44951306      18959          2   9537  ENST00000616990.4         -3704
35 117098271     252679          2  23387  ENST00000415541.5         60552
36  21515661      12835          2 221154  ENST00000480341.5        -21316
37  67190762      24744          1   4088  ENST00000537194.6             0
38  73368958      49100          2  10021  ENST00000261917.4         14649
39  78886161      38495          1  10933  ENST00000559697.5        -15639
40    722589       2009          1  65990  ENST00000566525.5             0
41   2214840       3248          2 283871  ENST00000333503.8             0
42  54040726     214366          1  79068  ENST00000563011.2        -57417
43  72027664      19077          1   1723  ENST00000572887.5             0
44  72889834      60326          2    463  ENST00000563625.1         11929
45   2229464      56728          2  23293  ENST00000572205.5          5920
46  75844773       2347          2 201294  ENST00000699512.1             0
47  44872146     170809          1  26040  ENST00000645568.1         31886
48  59902963       2928          1   5366  ENST00000590596.1        167525
49  18475933      31183          2   8178  ENST00000596124.3         -2720
50  46409826      19299          2   9620  ENST00000674359.1         -7185
51 129590633       1045          1   4952  ENST00000463271.1         11414
52  14201122      22306          1  27258  ENST00000306024.4         37382
53 121785530     133897          1 153090  ENST00000408936.8             0
54 103693623      94104          2   9644  ENST00000315994.6        -19191
55  67195173     127389          1   4088  ENST00000679624.1          3225
56  73368958      49100          2  10021  ENST00000261917.4         -6676
57  73048128     265244          2    463 ENST00000268489.10         27945
58   2214349      41682          2  23293  ENST00000572369.5             0
   DE_g2
1     DE
2     DE
3     DE
4     DE
5     DE
6     DE
7     DE
8     DE
9     DE
10    DE
11    DE
12    DE
13    DE
14    DE
15    DE
16    DE
17    DE
18    DE
19    DE
20    DE
21    DE
22    DE
23    DE
24    DE
25    DE
26    DE
27    DE
28    DE
29    DE
30    DE
31    DE
32    DE
33    DE
34    DE
35    DE
36    DE
37    DE
38    DE
39    DE
40    DE
41    DE
42    DE
43    DE
44    DE
45    DE
46    DE
47    DE
48    DE
49    DE
50    DE
51    DE
52    DE
53    DE
54    DE
55    DE
56    DE
57    DE
58    DE</code></pre>
<pre class="r"><code>GWAS_MI_DAR_df %&gt;% 
  dplyr::filter(DE_g2==&quot;DE&quot;)</code></pre>
<pre><code>  seqnames     start       end width strand id
1     chr1 109274953 109276202  1250      * LR
2    chr15  78830933  78832028  1096      * LR
                                               annotation geneChr geneStart
1                                                  3&#39; UTR       1 109270824
2 Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)      15  78847667
    geneEnd geneLength geneStrand geneId      transcriptId distanceToTSS DE_g2
1 109275607       4784          1   1952 ENST00000498157.1          4129    DE
2  78886161      38495          1  10933 ENST00000559697.5        -15639    DE</code></pre>
<pre class="r"><code>GWAS_CAD_DAR_df %&gt;% 
  dplyr::filter(DE_g2==&quot;DE&quot;)</code></pre>
<pre><code>   seqnames     start       end width strand  id
1      chr2 217804449 217806257  1809      * EAR
2      chr4  94575649  94576017   369      * EAR
3      chr7  36459049  36459918   870      * EAR
4      chr7 100243501 100243833   333      * EAR
5     chr17   2222321   2222533   213      * EAR
6     chr19  11142349  11143639  1291      * EAR
7      chr3 151483793 151484643   851      * ESR
8      chr9 121650071 121650712   642      * ESR
9     chr19  44911105  44911471   367      * ESR
10     chr1   3408398   3409463  1066      *  LR
11     chr1 109274953 109276202  1250      *  LR
12     chr2  24818716  24819139   424      *  LR
13     chr2  39631404  39632344   941      *  LR
14     chr2 217868752 217869273   522      *  LR
15     chr3  48151277  48152429  1153      *  LR
16     chr4 168766280 168767270   991      *  LR
17     chr5  17118580  17119099   520      *  LR
18     chr6  28269507  28270003   497      *  LR
19     chr9 107755276 107755816   541      *  LR
20    chr11   9740217   9740734   518      *  LR
21    chr11 117037235 117037719   485      *  LR
22    chr15  67165920  67166709   790      *  LR
23    chr15  73353612  73354309   698      *  LR
24    chr15  78830933  78832028  1096      *  LR
25    chr16  72008437  72009100   664      *  LR
26    chr17   2222861   2223544   684      *  LR
27    chr17  75844492  75845572  1081      *  LR
28    chr18  44733224  44733992   769      *  LR
29    chr19  18478653  18479269   617      *  LR
30    chr17   2214040   2216112  2073      *  NR
                                                annotation geneChr geneStart
1                                                   3&#39; UTR       2 217809714
2                                                   3&#39; UTR       4  94579221
3                                        Distal Intergenic       7  36448939
4         Intron (ENST00000328453.9/352954, intron 1 of 7)       7 100222645
5       Intron (ENST00000263073.11/23293, intron 10 of 18)      17   2172737
6         Intron (ENST00000585567.5/147841, intron 4 of 5)      19  11147134
7                                        Distal Intergenic       3 151436203
8                                         Promoter (&lt;=1kb)       9 121651634
9  Exon (ENST00000623895.1/ENST00000623895.1, exon 1 of 1)      19  44914247
10        Intron (ENST00000511072.5/63976, intron 8 of 15)       1   3396491
11                                                  3&#39; UTR       1 109270824
12                                        Promoter (1-2kb)       2  24819369
13        Intron (ENST00000415640.1/728730, intron 1 of 2)       2  39664982
14        Intron (ENST00000646520.1/7145, intron 18 of 32)       2 217848472
15                                       Distal Intergenic       3  48164402
16       Intron (ENST00000505667.6/23022, intron 10 of 21)       4 168712159
17            Exon (ENST00000665920.1/285696, exon 3 of 3)       5  17065598
18         Intron (ENST00000316606.10/7741, intron 1 of 3)       6  28267359
19                                       Distal Intergenic       9 107488989
20                                                  3&#39; UTR      11   9726895
21        Intron (ENST00000445177.6/23387, intron 1 of 24)      11 116845593
22                                        Promoter (&lt;=1kb)      15  67166019
23         Intron (ENST00000261917.4/10021, intron 1 of 7)      15  73319859
24 Exon (ENST00000674105.1/ENST00000674105.1, exon 1 of 1)      15  78847667
25                                        Promoter (&lt;=1kb)      16  72008588
26      Intron (ENST00000263073.11/23293, intron 10 of 18)      17   2172737
27                                        Promoter (&lt;=1kb)      17  75842427
28 Exon (ENST00000624404.1/ENST00000624404.1, exon 1 of 1)      18  44701338
29         Intron (ENST00000262809.9/8178, intron 1 of 11)      19  18444751
30                                        Promoter (&lt;=1kb)      17   2172668
     geneEnd geneLength geneStrand geneId      transcriptId distanceToTSS DE_g2
1  217810304        591          2   7145 ENST00000480416.1          4047    DE
2   94618191      38971          1  10611 ENST00000511586.5         -3204    DE
3   36453655       4717          1  54443 ENST00000491782.1         10110    DE
4  100246548      23904          2 352954 ENST00000538588.2          2715    DE
5    2229464      56728          2  23293 ENST00000572205.5          6931    DE
6   11155761       8628          2 147841 ENST00000586708.1         12122    DE
7  151461061      24859          2 285313 ENST00000282466.4        -22732    DE
8  121785530     133897          1 153090 ENST00000408936.8          -922    DE
9   44919342       5096          1    341 ENST00000588750.5         -2776    DE
10   3434293      37803          1  63976 ENST00000509860.1         11907    DE
11 109275607       4784          1   1952 ENST00000498157.1          4129    DE
12  24820796       1428          2    109 ENST00000498288.1          1657    DE
13  39707136      42155          1 130733 ENST00000437068.5        -32638    DE
14 217858633      10162          2   7145 ENST00000480665.1        -10119    DE
15  48174753      10352          2    993 ENST00000459900.1         22324    DE
16 168898686     186528          1  23022 ENST00000510998.5         54121    DE
17  17275353     209756          1  10409 ENST00000606445.1         52982    DE
18  28278224      10866          1   7741 ENST00000619937.4          2148    DE
19 107490482       1494          2   9314 ENST00000420475.1       -264794    DE
20   9750491      23597          1  23075 ENST00000534562.1         13322    DE
21 117098271     252679          2  23387 ENST00000415541.5         60552    DE
22  67190762      24744          1   4088 ENST00000537194.6             0    DE
23  73368958      49100          2  10021 ENST00000261917.4         14649    DE
24  78886161      38495          1  10933 ENST00000559697.5        -15639    DE
25  72027664      19077          1   1723 ENST00000572887.5             0    DE
26   2229464      56728          2  23293 ENST00000572205.5          5920    DE
27  75844773       2347          2 201294 ENST00000699512.1             0    DE
28  44872146     170809          1  26040 ENST00000645568.1         31886    DE
29  18475933      31183          2   8178 ENST00000596124.3         -2720    DE
30   2214349      41682          2  23293 ENST00000572369.5             0    DE</code></pre>
<pre class="r"><code>my_hc_filtered_counts_n45 &lt;- readRDS(&quot;data/my_hc_filt_counts_n45.RDS&quot;)
log_filt_hc_n45 &lt;- cpm(my_hc_filtered_counts_n45, log = TRUE) %&gt;% as.data.frame()
drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

log_filt_hc_n45 %&gt;% 
 #  rownames_to_column(&quot;peak&quot;) %&gt;% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %&gt;% 
  dplyr::filter(row.names(.)==&quot;chr6.36678380.36679788&quot;) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm in peak with SNP CDKN1a&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/CDKn1a%20log2cpm%20DAR-RNa-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>Counts_RNA_ERMatthews &lt;- readRDS(&quot;data/other_papers/Counts_RNA_ERMatthews.RDS&quot;)
# Counts_RNA_ERMatthews &lt;-  
Counts_RNA_ERMatthews %&gt;% 
  column_to_rownames(&quot;ENTREZID&quot;) %&gt;% 
  cpm(.,log = TRUE) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column(&quot;expr&quot;) %&gt;%
  filter(expr == 1026) %&gt;% 
  pivot_longer(cols =!expr , names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;trt&quot;,&quot;indv&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle(&quot;log2cpm RNA in ESR CDKN1a&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Enrichment_motif.Rmd/CDKn1a%20log2cpm%20DAR-RNa-2.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] bedr_1.0.7                              
 [2] devtools_2.4.5                          
 [3] usethis_2.2.3                           
 [4] ggpubr_0.6.0                            
 [5] BiocParallel_1.36.0                     
 [6] Cormotif_1.48.0                         
 [7] affy_1.80.0                             
 [8] scales_1.3.0                            
 [9] VennDiagram_1.7.3                       
[10] futile.logger_1.4.3                     
[11] gridExtra_2.3                           
[12] ggfortify_0.4.16                        
[13] edgeR_4.0.16                            
[14] limma_3.58.1                            
[15] rtracklayer_1.62.0                      
[16] org.Hs.eg.db_3.18.0                     
[17] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[18] GenomicFeatures_1.54.4                  
[19] AnnotationDbi_1.64.1                    
[20] Biobase_2.62.0                          
[21] GenomicRanges_1.54.1                    
[22] GenomeInfoDb_1.38.8                     
[23] IRanges_2.36.0                          
[24] S4Vectors_0.40.2                        
[25] BiocGenerics_0.48.1                     
[26] ChIPseeker_1.38.0                       
[27] RColorBrewer_1.1-3                      
[28] broom_1.0.5                             
[29] kableExtra_1.4.0                        
[30] lubridate_1.9.3                         
[31] forcats_1.0.0                           
[32] stringr_1.5.1                           
[33] dplyr_1.1.4                             
[34] purrr_1.0.2                             
[35] readr_2.1.5                             
[36] tidyr_1.3.1                             
[37] tibble_3.2.1                            
[38] ggplot2_3.5.0                           
[39] tidyverse_2.0.0                         
[40] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.3                               
  [2] matrixStats_1.3.0                      
  [3] bitops_1.0-7                           
  [4] enrichplot_1.22.0                      
  [5] HDO.db_0.99.1                          
  [6] httr_1.4.7                             
  [7] profvis_0.3.8                          
  [8] tools_4.3.1                            
  [9] backports_1.4.1                        
 [10] utf8_1.2.4                             
 [11] R6_2.5.1                               
 [12] lazyeval_0.2.2                         
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.0                            
 [15] prettyunits_1.2.0                      
 [16] preprocessCore_1.64.0                  
 [17] cli_3.6.2                              
 [18] formatR_1.14                           
 [19] scatterpie_0.2.2                       
 [20] labeling_0.4.3                         
 [21] sass_0.4.9                             
 [22] Rsamtools_2.18.0                       
 [23] systemfonts_1.0.6                      
 [24] yulab.utils_0.1.4                      
 [25] R.utils_2.12.3                         
 [26] DOSE_3.28.2                            
 [27] svglite_2.1.3                          
 [28] sessioninfo_1.2.2                      
 [29] plotrix_3.8-4                          
 [30] rstudioapi_0.16.0                      
 [31] RSQLite_2.3.6                          
 [32] generics_0.1.3                         
 [33] gridGraphics_0.5-1                     
 [34] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [35] BiocIO_1.12.0                          
 [36] gtools_3.9.5                           
 [37] car_3.1-2                              
 [38] GO.db_3.18.0                           
 [39] Matrix_1.6-5                           
 [40] fansi_1.0.6                            
 [41] abind_1.4-5                            
 [42] R.methodsS3_1.8.2                      
 [43] lifecycle_1.0.4                        
 [44] whisker_0.4.1                          
 [45] yaml_2.3.8                             
 [46] carData_3.0-5                          
 [47] SummarizedExperiment_1.32.0            
 [48] gplots_3.1.3.1                         
 [49] qvalue_2.34.0                          
 [50] SparseArray_1.2.4                      
 [51] BiocFileCache_2.10.2                   
 [52] blob_1.2.4                             
 [53] promises_1.3.0                         
 [54] crayon_1.5.2                           
 [55] miniUI_0.1.1.1                         
 [56] lattice_0.22-6                         
 [57] cowplot_1.1.3                          
 [58] KEGGREST_1.42.0                        
 [59] pillar_1.9.0                           
 [60] knitr_1.46                             
 [61] fgsea_1.28.0                           
 [62] rjson_0.2.21                           
 [63] boot_1.3-30                            
 [64] codetools_0.2-20                       
 [65] fastmatch_1.1-4                        
 [66] glue_1.7.0                             
 [67] getPass_0.2-4                          
 [68] ggfun_0.1.4                            
 [69] remotes_2.5.0                          
 [70] data.table_1.15.4                      
 [71] vctrs_0.6.5                            
 [72] png_0.1-8                              
 [73] treeio_1.26.0                          
 [74] testthat_3.2.1.1                       
 [75] gtable_0.3.4                           
 [76] cachem_1.0.8                           
 [77] xfun_0.43                              
 [78] S4Arrays_1.2.1                         
 [79] mime_0.12                              
 [80] tidygraph_1.3.1                        
 [81] statmod_1.5.0                          
 [82] ellipsis_0.3.2                         
 [83] interactiveDisplayBase_1.40.0          
 [84] nlme_3.1-164                           
 [85] ggtree_3.10.1                          
 [86] bit64_4.0.5                            
 [87] progress_1.2.3                         
 [88] filelock_1.0.3                         
 [89] rprojroot_2.0.4                        
 [90] bslib_0.7.0                            
 [91] affyio_1.72.0                          
 [92] KernSmooth_2.23-22                     
 [93] colorspace_2.1-0                       
 [94] DBI_1.2.2                              
 [95] tidyselect_1.2.1                       
 [96] processx_3.8.4                         
 [97] bit_4.0.5                              
 [98] compiler_4.3.1                         
 [99] curl_5.2.1                             
[100] git2r_0.33.0                           
[101] xml2_1.3.6                             
[102] DelayedArray_0.28.0                    
[103] shadowtext_0.1.3                       
[104] caTools_1.18.2                         
[105] callr_3.7.6                            
[106] rappdirs_0.3.3                         
[107] digest_0.6.35                          
[108] rmarkdown_2.26                         
[109] XVector_0.42.0                         
[110] htmltools_0.5.8.1                      
[111] pkgconfig_2.0.3                        
[112] MatrixGenerics_1.14.0                  
[113] highr_0.10                             
[114] dbplyr_2.5.0                           
[115] fastmap_1.1.1                          
[116] htmlwidgets_1.6.4                      
[117] rlang_1.1.3                            
[118] shiny_1.8.1.1                          
[119] farver_2.1.1                           
[120] jquerylib_0.1.4                        
[121] jsonlite_1.8.8                         
[122] R.oo_1.26.0                            
[123] GOSemSim_2.28.1                        
[124] RCurl_1.98-1.14                        
[125] magrittr_2.0.3                         
[126] GenomeInfoDbData_1.2.11                
[127] ggplotify_0.1.2                        
[128] patchwork_1.2.0                        
[129] munsell_0.5.1                          
[130] Rcpp_1.0.12                            
[131] ape_5.8                                
[132] viridis_0.6.5                          
[133] stringi_1.8.3                          
[134] ggraph_2.2.1                           
[135] brio_1.1.4                             
[136] zlibbioc_1.48.2                        
[137] MASS_7.3-60.0.1                        
[138] pkgbuild_1.4.4                         
[139] AnnotationHub_3.10.1                   
[140] plyr_1.8.9                             
[141] parallel_4.3.1                         
[142] ggrepel_0.9.5                          
[143] Biostrings_2.70.3                      
[144] graphlayouts_1.1.1                     
[145] splines_4.3.1                          
[146] hms_1.1.3                              
[147] locfit_1.5-9.9                         
[148] ps_1.7.6                               
[149] igraph_2.0.3                           
[150] ggsignif_0.6.4                         
[151] pkgload_1.3.4                          
[152] reshape2_1.4.4                         
[153] biomaRt_2.58.2                         
[154] futile.options_1.0.1                   
[155] BiocVersion_3.18.1                     
[156] XML_3.99-0.16.1                        
[157] evaluate_0.23                          
[158] lambda.r_1.2.4                         
[159] BiocManager_1.30.22                    
[160] tzdb_0.4.0                             
[161] tweenr_2.0.3                           
[162] httpuv_1.6.15                          
[163] polyclip_1.10-6                        
[164] ggforce_0.4.2                          
[165] xtable_1.8-4                           
[166] restfulr_0.0.15                        
[167] tidytree_0.4.6                         
[168] rstatix_0.7.2                          
[169] later_1.3.2                            
[170] viridisLite_0.4.2                      
[171] aplot_0.2.2                            
[172] memoise_2.0.1                          
[173] GenomicAlignments_1.38.2               
[174] timechange_0.3.0                       </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
