<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-04-15" />

<title>Enrichment_motif</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enrichment_motif</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-04-15</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-15
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree244cfc0ea215043ba095fa3a3f6af876da837f27targetblank244cfc0a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/244cfc0ea215043ba095fa3a3f6af876da837f27" target="_blank">244cfc0</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree244cfc0ea215043ba095fa3a3f6af876da837f27targetblank244cfc0a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/244cfc0ea215043ba095fa3a3f6af876da837f27" target="_blank">244cfc0</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/Jaspar_motif.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/TSSE.R
    Untracked:  code/corMotifcustom.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/CorMotif_data_n45.Rmd
    Modified:   analysis/Peak_analysis.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Enrichment_motif.Rmd</code>) and
HTML (<code>docs/Enrichment_motif.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/244cfc0ea215043ba095fa3a3f6af876da837f27/analysis/Enrichment_motif.Rmd" target="_blank">244cfc0</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-15
</td>
<td>
Adding in no extension
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/5115c784fbe50ada8ceba26aa953838cbeb79174/docs/Enrichment_motif.html" target="_blank">5115c78</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1cc42d79947bff39826b90c3f33bf9d7f50a784d/analysis/Enrichment_motif.Rmd" target="_blank">1cc42d7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
updates to code to fix broken string
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/8d44ffbe4c257ea851be5795189adfaac98ee86a/docs/Enrichment_motif.html" target="_blank">8d44ffb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/caecf0befbdf2efe22332b08aad7bf561cf41c27/analysis/Enrichment_motif.Rmd" target="_blank">caecf0b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-04-12
</td>
<td>
adding new page
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
library(bedr)</code></pre>
<div id="knowles-supp-data-check" class="section level3">
<h3>Knowles supp data check</h3>
<pre class="r"><code>Knowles_4&lt;- readRDS(&quot;data/Knowles_4.RDS&quot;)
Knowles_5 &lt;- readRDS(&quot;data/Knowles_5.RDS&quot;)
Knowles_6 &lt;- readRDS(&quot;data/Knowles_6.RDS&quot;)
SNP_supp_schneider &lt;- readRDS(&quot;data/SNP_supp_schneider.RDS&quot;)
peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)
list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)</code></pre>
<pre><code>&lt;environment: R_GlobalEnv&gt;</code></pre>
<pre class="r"><code>###How I made the granges files for overlaps
Knowles_4_noext &lt;- Knowles_4 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()


Knowles_4 &lt;- Knowles_4 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-25, end=pos+25) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()


# 
Knowles_5_noext &lt;- Knowles_5 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()

Knowles_5 &lt;- Knowles_5 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-25, end=pos+25) %&gt;%
  dplyr::select(chr, start,end, RSID, gene,pos) %&gt;%
  GRanges()

Knowles_6_noext &lt;- Knowles_6 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos, end=pos+1) %&gt;%
  dplyr::select(chr, start,end, gene,pos) %&gt;%
  GRanges()
 
 Knowles_6 &lt;- Knowles_6 %&gt;%
  na.omit() %&gt;%
  mutate(start=pos-25, end=pos+25) %&gt;%
  dplyr::select(chr, start,end, gene,pos) %&gt;%
  GRanges()
 
 
  SNP_supp_schneider &lt;-  SNP_supp_schneider %&gt;%
    na.omit() %&gt;%
    rename(X1=&quot;chr&quot;, X2 = &quot;pos&quot;, X4=&quot;RSID&quot;) %&gt;%
    mutate(start=pos-25, end=pos+25) %&gt;%
   dplyr::select(chr, start,end, RSID,pos) %&gt;%
    GRanges()

##split into granges objects
###make dataframe of each EAR,ESR,LR,NR promoters

EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_promotor &lt;- EAR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
EAR_df_gr &lt;-  GRanges(EAR_df)

# rtracklayer::export.bed(EAR_promo_gr, con=&quot;data/n45_bedfiles/EAR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_promotor &lt;- ESR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
ESR_df_gr &lt;-  GRanges(ESR_df)
 # rtracklayer::export.bed(ESR_promo_gr, con=&quot;data/n45_bedfiles/ESR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
 
LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_promotor &lt;- LR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
LR_df_gr &lt;-  GRanges(LR_df)
 # rtracklayer::export.bed(LR_promo_gr, con=&quot;data/n45_bedfiles/LR_promo.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
 # LR_promotor %&gt;% 
 #   dplyr::filter(geneChr==6)

NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_promotor &lt;- NR_df %&gt;% 
#   dplyr::filter(annotation==&quot;Promoter (&lt;=1kb)&quot;)
  
NR_df_gr &lt;-  GRanges(NR_df)</code></pre>
<p>Checking for enrichment:</p>
<p>I have made 25 bp +/- files for each of the data sets using the eQTL
or SNP location. I also made a 1bp no extensin file for each of the data
sets using the eQTL or SNP location as start and end as +1.</p>
<p>report:</p>
<table>
<caption>number of intersections between groups</caption>
<thead>
<tr class="header">
<th>motif</th>
<th>K_4(500)</th>
<th>K_5(359)</th>
<th>K_6(447)</th>
<th>schn_g(112)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>1</td>
<td>0</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>3</td>
<td>2</td>
<td>2</td>
<td>0</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>14</td>
<td>10</td>
<td>9</td>
<td>1</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>32</td>
<td>15</td>
<td>19</td>
<td>2</td>
</tr>
</tbody>
</table>
<p>report:</p>
<table>
<caption>number of intersections between groups without
extension</caption>
<thead>
<tr class="header">
<th>motif</th>
<th>K_4(500)</th>
<th>K_5(359)</th>
<th>K_6(447)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>1</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>3</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>13</td>
<td>9</td>
<td>8</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>30</td>
<td>15</td>
<td>19</td>
</tr>
</tbody>
</table>
<p>These numbers are different than the original lists from which they
derived. This is normally due to running <code>na.omit()</code> on the
data frame before GRanges transformation.<br />
</p>
</div>
<div id="gwas" class="section level3">
<h3>GWAS</h3>
<p>I downloaded 5 different trait associated lists from the GWAS
catalog</p>
<p>trait 1, 11 associations with trait
<strong>cardiotoxicity</strong></p>
<p>trait 2, 1068 associations with trait <strong>cardiac
arrhythmia</strong></p>
<p>trait 3, 40 associations with trait <strong>response to
anthracycline-based chemotherapy</strong></p>
<p>trait 4, 6139 associations with trait <strong>heart
disease</strong></p>
<p>trait 5, 447 associations with trait <strong>heart
failure</strong></p>
<pre class="r"><code>##finding GWAS for Hearfailure: I used several traits from the GWAS database:
##1_cardiotoxicity  11 associations   gwas_1
gwas_1 &lt;- readRDS(&quot;data/gwas_1_dataframe.RDS&quot;)
##2_ cardiac arrhythmia  1068 associations   gwas_2
gwas_2 &lt;- readRDS(&quot;data/gwas_2_dataframe.RDS&quot;)
##3_ Anthracycline_response   40 gwas_3
gwas_3 &lt;- readRDS(&quot;data/gwas_3_dataframe.RDS&quot;)

##4_ heart disease (biggest # of associations 6139) 
gwas_4 &lt;- readRDS(&quot;data/gwas_4_dataframe.RDS&quot;)

##5_ heart_failure gwas_5
gwas_5 &lt;- readRDS(&quot;data/gwas_5_dataframe.RDS&quot;)

gwas_1_gr &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_1_gr_noext &lt;- gwas_1 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_2_gr &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()


gwas_2_gr_noext &lt;- gwas_2 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
  na.omit() %&gt;% 
  GRanges()

gwas_3_gr &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()
gwas_3_gr_noext &lt;- gwas_3 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_4_gr &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID))%&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_4_gr_noext &lt;-  gwas_4 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(CHR_POS= as.integer(CHR_POS)) %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_5_gr &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS-25, end=CHR_POS+25, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

gwas_5_gr_noext &lt;- gwas_5 %&gt;% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %&gt;% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %&gt;%
   na.omit() %&gt;% 
  GRanges()

# gwas_2[(is.na(gwas_2$CHR_POS)),]</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_1_gr)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 7321 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_1_gr)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 15665 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_1_gr)</code></pre>
<pre><code>Hits object with 2 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]     22537           8
  [2]     34839           9
  -------
  queryLength: 41759 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_1_gr)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 83658 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_2_gr)</code></pre>
<pre><code>Hits object with 2 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]      2201          88
  [2]      3355         148
  -------
  queryLength: 7321 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_2_gr)</code></pre>
<pre><code>Hits object with 30 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]       147         159
   [2]       147         851
   [3]       162         707
   [4]       935         889
   [5]      1964         418
   ...       ...         ...
  [26]     14755         273
  [27]     14755         510
  [28]     14755         692
  [29]     14755        1030
  [30]     14755        1031
  -------
  queryLength: 15665 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr,  gwas_2_gr)</code></pre>
<pre><code>Hits object with 29 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      1956         741
   [2]      2627         367
   [3]      2627         461
   [4]      2627         526
   [5]      2627         959
   ...       ...         ...
  [25]     30072         201
  [26]     33887         396
  [27]     34674         122
  [28]     38383        1013
  [29]     38383        1014
  -------
  queryLength: 41759 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_2_gr)</code></pre>
<pre><code>Hits object with 91 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]       964         160
   [2]       964         403
   [3]       964         694
   [4]      4665         448
   [5]      4665         945
   ...       ...         ...
  [87]     73639         204
  [88]     78512          59
  [89]     82971         544
  [90]     83378         328
  [91]     83378         349
  -------
  queryLength: 83658 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_3_gr)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 7321 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_3_gr)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 15665 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_3_gr)</code></pre>
<pre><code>Hits object with 4 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]     22537          34
  [2]     23362          17
  [3]     30254          11
  [4]     34839          35
  -------
  queryLength: 41759 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_3_gr)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 83658 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_4_gr)</code></pre>
<pre><code>Hits object with 17 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      1257        2474
   [2]      2043        2885
   [3]      2201         908
   [4]      2298        5449
   [5]      2845        1241
   ...       ...         ...
  [13]      6165        2990
  [14]      6165        5301
  [15]      6274        2229
  [16]      6398        1107
  [17]      6531        4226
  -------
  queryLength: 7321 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_4_gr)</code></pre>
<pre><code>Hits object with 55 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]       147        1408
   [2]       147        4800
   [3]       162        3082
   [4]       194        1618
   [5]       194        1633
   ...       ...         ...
  [51]     14755        1522
  [52]     14755        1991
  [53]     14755        2173
  [54]     14755        4979
  [55]     14755        4980
  -------
  queryLength: 15665 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_4_gr)</code></pre>
<pre><code>Hits object with 186 hits and 0 metadata columns:
        queryHits subjectHits
        &lt;integer&gt;   &lt;integer&gt;
    [1]        50        2529
    [2]        50        2709
    [3]        50        2774
    [4]        50        5180
    [5]      1205        4287
    ...       ...         ...
  [182]     39817        5225
  [183]     39817        6030
  [184]     40321        2255
  [185]     40321        2291
  [186]     41510         142
  -------
  queryLength: 41759 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_4_gr)</code></pre>
<pre><code>Hits object with 292 hits and 0 metadata columns:
        queryHits subjectHits
        &lt;integer&gt;   &lt;integer&gt;
    [1]       113        1193
    [2]       113        5179
    [3]       964        1409
    [4]       964        1884
    [5]       964        2175
    ...       ...         ...
  [288]     80131         520
  [289]     82166        5778
  [290]     82971        2025
  [291]     83378        1577
  [292]     83378        1598
  -------
  queryLength: 83658 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_5_gr)</code></pre>
<pre><code>Hits object with 1 hit and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]      6274         190
  -------
  queryLength: 7321 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_5_gr)</code></pre>
<pre><code>Hits object with 2 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]      2987          54
  [2]      5429         117
  -------
  queryLength: 15665 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_5_gr)</code></pre>
<pre><code>Hits object with 16 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      1398         112
   [2]      1956         432
   [3]      1956         393
   [4]      3834          75
   [5]      6545          47
   ...       ...         ...
  [12]     34394         115
  [13]     37166          44
  [14]     37223           6
  [15]     40321         216
  [16]     40321         252
  -------
  queryLength: 41759 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_5_gr)</code></pre>
<pre><code>Hits object with 11 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      8661         397
   [2]      8957          66
   [3]     18025         139
   [4]     18025         402
   [5]     31172         372
   [6]     41740          78
   [7]     55143          40
   [8]     56612         211
   [9]     72255         162
  [10]     75280          64
  [11]     78282         333
  -------
  queryLength: 83658 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_1_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 7321 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_1_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 15665 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_1_gr_noext)</code></pre>
<pre><code>Hits object with 2 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]     22537           8
  [2]     34839           9
  -------
  queryLength: 41759 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_1_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 83658 / subjectLength: 11</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_2_gr_noext)</code></pre>
<pre><code>Hits object with 2 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]      2201          88
  [2]      3355         148
  -------
  queryLength: 7321 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_2_gr_noext)</code></pre>
<pre><code>Hits object with 30 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]       147         159
   [2]       147         851
   [3]       162         707
   [4]       935         889
   [5]      1964         418
   ...       ...         ...
  [26]     14755         273
  [27]     14755         510
  [28]     14755         692
  [29]     14755        1030
  [30]     14755        1031
  -------
  queryLength: 15665 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr,  gwas_2_gr_noext)</code></pre>
<pre><code>Hits object with 29 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      1956         741
   [2]      2627         367
   [3]      2627         461
   [4]      2627         526
   [5]      2627         959
   ...       ...         ...
  [25]     30072         201
  [26]     33887         396
  [27]     34674         122
  [28]     38383        1013
  [29]     38383        1014
  -------
  queryLength: 41759 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_2_gr_noext)</code></pre>
<pre><code>Hits object with 90 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]       964         160
   [2]       964         403
   [3]       964         694
   [4]      4665         448
   [5]      4665         945
   ...       ...         ...
  [86]     68837         690
  [87]     73639         204
  [88]     78512          59
  [89]     83378         328
  [90]     83378         349
  -------
  queryLength: 83658 / subjectLength: 1061</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_3_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 7321 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_3_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 15665 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_3_gr_noext)</code></pre>
<pre><code>Hits object with 4 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]     22537          34
  [2]     23362          17
  [3]     30254          11
  [4]     34839          35
  -------
  queryLength: 41759 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_3_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 83658 / subjectLength: 39</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_4_gr_noext)</code></pre>
<pre><code>Hits object with 15 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      1257        2474
   [2]      2043        2885
   [3]      2201         908
   [4]      2298        5449
   [5]      2845        1241
   ...       ...         ...
  [11]      6165        2842
  [12]      6165        2990
  [13]      6165        5301
  [14]      6398        1107
  [15]      6531        4226
  -------
  queryLength: 7321 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_4_gr_noext)</code></pre>
<pre><code>Hits object with 55 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]       147        1408
   [2]       147        4800
   [3]       162        3082
   [4]       194        1618
   [5]       194        1633
   ...       ...         ...
  [51]     14755        1522
  [52]     14755        1991
  [53]     14755        2173
  [54]     14755        4979
  [55]     14755        4980
  -------
  queryLength: 15665 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_4_gr_noext)</code></pre>
<pre><code>Hits object with 182 hits and 0 metadata columns:
        queryHits subjectHits
        &lt;integer&gt;   &lt;integer&gt;
    [1]        50        2529
    [2]        50        2709
    [3]        50        2774
    [4]        50        5180
    [5]      1205        4287
    ...       ...         ...
  [178]     39817        5225
  [179]     39817        6030
  [180]     40321        2255
  [181]     40321        2291
  [182]     41510         142
  -------
  queryLength: 41759 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_4_gr_noext)</code></pre>
<pre><code>Hits object with 278 hits and 0 metadata columns:
        queryHits subjectHits
        &lt;integer&gt;   &lt;integer&gt;
    [1]       964        1409
    [2]       964        1884
    [3]       964        2175
    [4]      1318        1760
    [5]      1318        2416
    ...       ...         ...
  [274]     79616         653
  [275]     80131         520
  [276]     82166        5778
  [277]     83378        1577
  [278]     83378        1598
  -------
  queryLength: 83658 / subjectLength: 6052</code></pre>
<pre class="r"><code>findOverlaps(EAR_df_gr, gwas_5_gr_noext)</code></pre>
<pre><code>Hits object with 0 hits and 0 metadata columns:
   queryHits subjectHits
   &lt;integer&gt;   &lt;integer&gt;
  -------
  queryLength: 7321 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(ESR_df_gr, gwas_5_gr_noext)</code></pre>
<pre><code>Hits object with 2 hits and 0 metadata columns:
      queryHits subjectHits
      &lt;integer&gt;   &lt;integer&gt;
  [1]      2987          54
  [2]      5429         117
  -------
  queryLength: 15665 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(LR_df_gr, gwas_5_gr_noext)</code></pre>
<pre><code>Hits object with 15 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      1956         432
   [2]      1956         393
   [3]      3834          75
   [4]      6545          47
   [5]      7714          89
   ...       ...         ...
  [11]     34394         115
  [12]     37166          44
  [13]     37223           6
  [14]     40321         216
  [15]     40321         252
  -------
  queryLength: 41759 / subjectLength: 445</code></pre>
<pre class="r"><code>findOverlaps(NR_df_gr, gwas_5_gr_noext)</code></pre>
<pre><code>Hits object with 11 hits and 0 metadata columns:
       queryHits subjectHits
       &lt;integer&gt;   &lt;integer&gt;
   [1]      8661         397
   [2]      8957          66
   [3]     18025         139
   [4]     18025         402
   [5]     31172         372
   [6]     41740          78
   [7]     55143          40
   [8]     56612         211
   [9]     72255         162
  [10]     75280          64
  [11]     78282         333
  -------
  queryLength: 83658 / subjectLength: 445</code></pre>
<table>
<caption>gwas trait enrichment</caption>
<colgroup>
<col width="15%" />
<col width="15%" />
<col width="18%" />
<col width="15%" />
<col width="18%" />
<col width="16%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>gwas_1 (11)</th>
<th>gwas_2 (1061)</th>
<th>gwas_3 (39)</th>
<th>gwas_4 (6052)</th>
<th>gwas_5 (445)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>2</td>
<td>0</td>
<td>17</td>
<td>1</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>0</td>
<td>30</td>
<td>0</td>
<td>55</td>
<td>2</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>2</td>
<td>29</td>
<td>4</td>
<td>186</td>
<td>16</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>0</td>
<td>91</td>
<td>0</td>
<td>292</td>
<td>11</td>
</tr>
</tbody>
</table>
<table>
<caption>gwas trait enrichment with no extension</caption>
<colgroup>
<col width="15%" />
<col width="15%" />
<col width="18%" />
<col width="15%" />
<col width="18%" />
<col width="16%" />
</colgroup>
<thead>
<tr class="header">
<th>motif</th>
<th>gwas_1 (11)</th>
<th>gwas_2 (1061)</th>
<th>gwas_3 (39)</th>
<th>gwas_4 (6052)</th>
<th>gwas_5 (445)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>EAR (7321)</td>
<td>0</td>
<td>2</td>
<td>0</td>
<td>17</td>
<td>0</td>
</tr>
<tr class="even">
<td>ESR (15665)</td>
<td>0</td>
<td>30</td>
<td>0</td>
<td>55</td>
<td>2</td>
</tr>
<tr class="odd">
<td>LR (41759)</td>
<td>2</td>
<td>29</td>
<td>4</td>
<td>182</td>
<td>15</td>
</tr>
<tr class="even">
<td>NR (83658)</td>
<td>0</td>
<td>90</td>
<td>0</td>
<td>278</td>
<td>11</td>
</tr>
</tbody>
</table>
<p>These numbers are different than the original lists from which they
derived. This is normally due to running <code>na.omit()</code> on the
data frame before GRanges transformation.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] bedr_1.0.7                              
 [2] devtools_2.4.5                          
 [3] usethis_2.2.3                           
 [4] ggpubr_0.6.0                            
 [5] BiocParallel_1.36.0                     
 [6] Cormotif_1.48.0                         
 [7] affy_1.80.0                             
 [8] scales_1.3.0                            
 [9] VennDiagram_1.7.3                       
[10] futile.logger_1.4.3                     
[11] gridExtra_2.3                           
[12] ggfortify_0.4.16                        
[13] edgeR_4.0.16                            
[14] limma_3.58.1                            
[15] rtracklayer_1.62.0                      
[16] org.Hs.eg.db_3.18.0                     
[17] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[18] GenomicFeatures_1.54.3                  
[19] AnnotationDbi_1.64.1                    
[20] Biobase_2.62.0                          
[21] GenomicRanges_1.54.1                    
[22] GenomeInfoDb_1.38.7                     
[23] IRanges_2.36.0                          
[24] S4Vectors_0.40.2                        
[25] BiocGenerics_0.48.1                     
[26] ChIPseeker_1.38.0                       
[27] RColorBrewer_1.1-3                      
[28] broom_1.0.5                             
[29] kableExtra_1.4.0                        
[30] lubridate_1.9.3                         
[31] forcats_1.0.0                           
[32] stringr_1.5.1                           
[33] dplyr_1.1.4                             
[34] purrr_1.0.2                             
[35] readr_2.1.5                             
[36] tidyr_1.3.1                             
[37] tibble_3.2.1                            
[38] ggplot2_3.5.0                           
[39] tidyverse_2.0.0                         
[40] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.3                               
  [2] matrixStats_1.2.0                      
  [3] bitops_1.0-7                           
  [4] enrichplot_1.22.0                      
  [5] HDO.db_0.99.1                          
  [6] httr_1.4.7                             
  [7] profvis_0.3.8                          
  [8] tools_4.3.1                            
  [9] backports_1.4.1                        
 [10] utf8_1.2.4                             
 [11] R6_2.5.1                               
 [12] lazyeval_0.2.2                         
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.0                            
 [15] prettyunits_1.2.0                      
 [16] preprocessCore_1.64.0                  
 [17] cli_3.6.2                              
 [18] formatR_1.14                           
 [19] scatterpie_0.2.2                       
 [20] sass_0.4.9                             
 [21] Rsamtools_2.18.0                       
 [22] systemfonts_1.0.6                      
 [23] yulab.utils_0.1.4                      
 [24] R.utils_2.12.3                         
 [25] DOSE_3.28.2                            
 [26] svglite_2.1.3                          
 [27] sessioninfo_1.2.2                      
 [28] plotrix_3.8-4                          
 [29] rstudioapi_0.16.0                      
 [30] RSQLite_2.3.6                          
 [31] generics_0.1.3                         
 [32] gridGraphics_0.5-1                     
 [33] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [34] BiocIO_1.12.0                          
 [35] gtools_3.9.5                           
 [36] car_3.1-2                              
 [37] GO.db_3.18.0                           
 [38] Matrix_1.6-5                           
 [39] fansi_1.0.6                            
 [40] abind_1.4-5                            
 [41] R.methodsS3_1.8.2                      
 [42] lifecycle_1.0.4                        
 [43] whisker_0.4.1                          
 [44] yaml_2.3.8                             
 [45] carData_3.0-5                          
 [46] SummarizedExperiment_1.32.0            
 [47] gplots_3.1.3.1                         
 [48] qvalue_2.34.0                          
 [49] SparseArray_1.2.4                      
 [50] BiocFileCache_2.10.1                   
 [51] blob_1.2.4                             
 [52] promises_1.3.0                         
 [53] crayon_1.5.2                           
 [54] miniUI_0.1.1.1                         
 [55] lattice_0.22-6                         
 [56] cowplot_1.1.3                          
 [57] KEGGREST_1.42.0                        
 [58] pillar_1.9.0                           
 [59] knitr_1.46                             
 [60] fgsea_1.28.0                           
 [61] rjson_0.2.21                           
 [62] boot_1.3-30                            
 [63] codetools_0.2-20                       
 [64] fastmatch_1.1-4                        
 [65] glue_1.7.0                             
 [66] getPass_0.2-4                          
 [67] ggfun_0.1.4                            
 [68] remotes_2.5.0                          
 [69] data.table_1.15.4                      
 [70] vctrs_0.6.5                            
 [71] png_0.1-8                              
 [72] treeio_1.26.0                          
 [73] testthat_3.2.1                         
 [74] gtable_0.3.4                           
 [75] cachem_1.0.8                           
 [76] xfun_0.43                              
 [77] S4Arrays_1.2.1                         
 [78] mime_0.12                              
 [79] tidygraph_1.3.1                        
 [80] statmod_1.5.0                          
 [81] ellipsis_0.3.2                         
 [82] interactiveDisplayBase_1.40.0          
 [83] nlme_3.1-164                           
 [84] ggtree_3.10.1                          
 [85] bit64_4.0.5                            
 [86] progress_1.2.3                         
 [87] filelock_1.0.3                         
 [88] rprojroot_2.0.4                        
 [89] bslib_0.7.0                            
 [90] affyio_1.72.0                          
 [91] KernSmooth_2.23-22                     
 [92] colorspace_2.1-0                       
 [93] DBI_1.2.2                              
 [94] tidyselect_1.2.1                       
 [95] processx_3.8.4                         
 [96] bit_4.0.5                              
 [97] compiler_4.3.1                         
 [98] curl_5.2.1                             
 [99] git2r_0.33.0                           
[100] xml2_1.3.6                             
[101] DelayedArray_0.28.0                    
[102] shadowtext_0.1.3                       
[103] caTools_1.18.2                         
[104] callr_3.7.6                            
[105] rappdirs_0.3.3                         
[106] digest_0.6.35                          
[107] rmarkdown_2.26                         
[108] XVector_0.42.0                         
[109] htmltools_0.5.8.1                      
[110] pkgconfig_2.0.3                        
[111] MatrixGenerics_1.14.0                  
[112] dbplyr_2.5.0                           
[113] fastmap_1.1.1                          
[114] htmlwidgets_1.6.4                      
[115] rlang_1.1.3                            
[116] shiny_1.8.1.1                          
[117] farver_2.1.1                           
[118] jquerylib_0.1.4                        
[119] jsonlite_1.8.8                         
[120] R.oo_1.26.0                            
[121] GOSemSim_2.28.1                        
[122] RCurl_1.98-1.14                        
[123] magrittr_2.0.3                         
[124] GenomeInfoDbData_1.2.11                
[125] ggplotify_0.1.2                        
[126] patchwork_1.2.0                        
[127] munsell_0.5.1                          
[128] Rcpp_1.0.12                            
[129] ape_5.7-1                              
[130] viridis_0.6.5                          
[131] stringi_1.8.3                          
[132] ggraph_2.2.1                           
[133] brio_1.1.4                             
[134] zlibbioc_1.48.0                        
[135] MASS_7.3-60.0.1                        
[136] pkgbuild_1.4.4                         
[137] AnnotationHub_3.10.0                   
[138] plyr_1.8.9                             
[139] parallel_4.3.1                         
[140] ggrepel_0.9.5                          
[141] Biostrings_2.70.2                      
[142] graphlayouts_1.1.1                     
[143] splines_4.3.1                          
[144] hms_1.1.3                              
[145] locfit_1.5-9.9                         
[146] ps_1.7.6                               
[147] igraph_2.0.3                           
[148] ggsignif_0.6.4                         
[149] pkgload_1.3.4                          
[150] reshape2_1.4.4                         
[151] biomaRt_2.58.2                         
[152] futile.options_1.0.1                   
[153] BiocVersion_3.18.1                     
[154] XML_3.99-0.16.1                        
[155] evaluate_0.23                          
[156] lambda.r_1.2.4                         
[157] BiocManager_1.30.22                    
[158] tzdb_0.4.0                             
[159] tweenr_2.0.3                           
[160] httpuv_1.6.15                          
[161] polyclip_1.10-6                        
[162] ggforce_0.4.2                          
[163] xtable_1.8-4                           
[164] restfulr_0.0.15                        
[165] tidytree_0.4.6                         
[166] rstatix_0.7.2                          
[167] later_1.3.2                            
[168] viridisLite_0.4.2                      
[169] aplot_0.2.2                            
[170] memoise_2.0.1                          
[171] GenomicAlignments_1.38.2               
[172] timechange_0.3.0                       </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
