<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2025-05-01" />

<title>ATAC_fastqc</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">ATAC_fastqc</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2025-05-01</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-05-01
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treeba4d3e00770510a81288061a6e3a0c6125c777b5targetblankba4d3e0a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/ba4d3e00770510a81288061a6e3a0c6125c777b5" target="_blank">ba4d3e0</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treeba4d3e00770510a81288061a6e3a0c6125c777b5targetblankba4d3e0a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/ba4d3e00770510a81288061a6e3a0c6125c777b5" target="_blank">ba4d3e0</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/Correlation_of_SNPnPEAK.Rmd
    Modified:   analysis/GO_KEGG_analysis.Rmd
    Modified:   analysis/Odds_ratios_ff.Rmd
    Modified:   analysis/Raodah_mycount.Rmd
    Modified:   analysis/TE_analysis_ff.Rmd
    Modified:   analysis/final_plot_attempt.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Fastqc_results.Rmd</code>) and
HTML (<code>docs/Fastqc_results.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/ba4d3e00770510a81288061a6e3a0c6125c777b5/analysis/Fastqc_results.Rmd" target="_blank">ba4d3e0</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-01
</td>
<td>
updates to webpage
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/9f3ac8f232a042a5b33e1be8dce5cc2654eeea53/analysis/Fastqc_results.Rmd" target="_blank">9f3ac8f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-25
</td>
<td>
after updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/Fastqc_results.html" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/cca2022bff19b2920d690fea18c0521f607819ff/analysis/Fastqc_results.Rmd" target="_blank">cca2022</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
<td>
updates to reads graphs
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/2910c4d5fa151b6e00c0982e640268cbafcc56ee/analysis/Fastqc_results.Rmd" target="_blank">2910c4d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
<td>
updates to code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/de853f8b60001dc9abf34c4080e57bc30c369198/docs/Fastqc_results.html" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/60c7f41b0b301db3c4285e33dfbff20f886c4999/analysis/Fastqc_results.Rmd" target="_blank">60c7f41</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
<td>
updates to number of reads and summaries
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/1f95eb9c1061623dfef1e5661b66a0140ee861cc/docs/Fastqc_results.html" target="_blank">1f95eb9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/82d757ceb9cc75694083b053518db92ccbc2be56/analysis/Fastqc_results.Rmd" target="_blank">82d757c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
<td>
adding in fragment length
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/3e9762bda264188db5d38cb4fcb7a245f18b5d99/docs/Fastqc_results.html" target="_blank">3e9762b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/778ce47a7c4e47fbe2e06ea6bc9f7c057bd38662/analysis/Fastqc_results.Rmd" target="_blank">778ce47</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
<td>
adding more graphs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/dce7d59112d1fb43d37814a8221ef437f919949d/docs/Fastqc_results.html" target="_blank">dce7d59</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/83c0d791fc0c63cf1c68de257041f0751af8df02/docs/Fastqc_results.html" target="_blank">83c0d79</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/ccb3f282386a1a2cb3f657eb98c961519b273709/analysis/Fastqc_results.Rmd" target="_blank">ccb3f28</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
<td>
first updates
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)</code></pre>
<div id="total-sequences-before-mapping" class="section level2">
<h2>Total Sequences before mapping</h2>
<p>This code takes the multiqc fastqc output file and: splits by rows to
trimmed and non trimmed, then separates the trimmed file names into
categories I want, then adds back in the non trimmed data rows (while
also splitting file name like the trimmed file name). after rbind, I
split treatmenttime by position, fix the names of the time column,
remove numbers from trt column, add a new column called “trimmed” where
I add in a vector that lets me group by trimmed file verses non trimmed
file, the select only those columns containing the columns I want to
keep.</p>
<pre class="r"><code>multiqc_fastqc2 &lt;- read_csv(&quot;data/multiqc_fastqc_run2.txt&quot;)
multiqc_general_stats2 &lt;- read_csv(&quot;data/multiqc_genestat_run2.txt&quot;)


fastqc_full &lt;- multiqc_fastqc2 %&gt;% 
  slice_tail(n=144) %&gt;% 
  separate(Filename, into = c(NA,&quot;ind&quot;,&quot;treatmenttime&quot;,NA,&quot;read&quot;)) %&gt;% 
  rbind(., (multiqc_fastqc2 %&gt;% slice_head(n=144) %&gt;% separate(Filename, into = c(&quot;ind&quot;,&quot;treatmenttime&quot;,NA,&quot;read&quot;)))) %&gt;% 
  separate_wider_position(., col =treatmenttime,c(2,trt=2,time=3),too_few = &quot;align_start&quot;) %&gt;% 
  mutate(time=case_match(trt,&quot;E2&quot;~&quot;24h&quot;,&quot;E3&quot;~&quot;3h&quot;,&quot;M2&quot;~&quot;24h&quot;, &quot;M3&quot;~&quot;3h&quot;,&quot;T2&quot;~&quot;24h&quot;,&quot;T3&quot;~&quot;3h&quot;,&quot;V2&quot;~&quot;24h&quot;,&quot;V3&quot;~&quot;3h&quot;,.default = time)) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;, trt) ) %&gt;% 
  mutate(trimmed = if_else(grepl(pattern =&quot;^trim&quot;, x = Sample)==TRUE, &quot;yes&quot;,&quot;no&quot;)) %&gt;% 
  dplyr::select(Sample:read, trimmed,`Total Sequences`:avg_sequence_length) %&gt;% 
  full_join(., multiqc_general_stats2, join_by(Sample)) %&gt;% 
   dplyr::rename(&quot;percent_gc&quot;=&quot;FastQC_mqc-generalstats-fastqc-percent_gc&quot;,
         &quot;avg_seq_len&quot;= &quot;FastQC_mqc-generalstats-fastqc-avg_sequence_length&quot;,
         &quot;percent_dup&quot;= &quot;FastQC_mqc-generalstats-fastqc-percent_duplicates&quot;,
         &quot;percent_fails&quot;= &quot;FastQC_mqc-generalstats-fastqc-percent_fails&quot;,
         &quot;total_sequences&quot;= &quot;FastQC_mqc-generalstats-fastqc-total_sequences&quot;) %&gt;% 
  mutate(ind = factor(ind, levels = c(&quot;Ind1&quot;, &quot;Ind2&quot;, &quot;Ind3&quot;, &quot;Ind4&quot;, &quot;Ind5&quot;, &quot;Ind6&quot;))) %&gt;%
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DX&quot;,&quot;E&quot;, &quot;DA&quot;,&quot;M&quot;, &quot;T&quot;, &quot;V&quot;), labels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) </code></pre>
<p>(in this case, Sample, ind, trt, time read, trimmed, Total sequences,
Flagged poor quality, sequence length, %GC,total deduplicated %,and avg
sequence length) I also then addin the gen_stats file and rename the
columns to normal things.</p>
<pre class="r"><code># fastqc_full
drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
drug_pal_fac &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
fastqc_full %&gt;% 
  filter(trimmed==&quot;no&quot;) %&gt;%
  ggplot(., aes(x=trt, y= `Total Sequences`))+
  geom_col(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle(&quot;Total Sequences, untrimmed&quot;)+
      # ylab(ylab)+
      xlab(&quot;&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/ggplot%20Total%20sequences-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ggplot-Total-sequences-1">
Past versions of "ggplot Total sequences-1.png"
</button>
</p>
<div id="fig-no-spaces-ggplot-Total-sequences-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/83c0d791fc0c63cf1c68de257041f0751af8df02/docs/figure/Fastqc_results.Rmd/ggplot Total sequences-1.png" target="_blank">83c0d79</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fastqc_full %&gt;% 
  filter(trimmed==&quot;yes&quot;) %&gt;% 
  ggplot(., aes(x=trt, y= `Total Sequences`))+
  geom_col(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle(&quot;Total Sequences, trimmed&quot;)+
      # ylab(ylab)+
      xlab(&quot;&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/ggplot%20Total%20sequences-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ggplot-Total-sequences-2">
Past versions of "ggplot Total sequences-2.png"
</button>
</p>
<div id="fig-no-spaces-ggplot-Total-sequences-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/83c0d791fc0c63cf1c68de257041f0751af8df02/docs/figure/Fastqc_results.Rmd/ggplot Total sequences-2.png" target="_blank">83c0d79</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>totseq &lt;- fastqc_full %&gt;% 
  dplyr::filter(read ==&#39;R1&#39;) %&gt;% 
  # group_by(ind,trt,time) %&gt;% 
dplyr::select(Sample, ind, trt, time, trimmed, `Total Sequences`) %&gt;% 
  pivot_wider(id_cols = c(ind,trt,time), names_from = trimmed, values_from = `Total Sequences`) %&gt;% 
  mutate(perc_removed=(no-yes)/no*100) #%&gt;% 
 # kable(list(totseq[1:36,], totseq[37:72,]),caption= &quot;Summary of Total sequences before and after trimming, with percentage of removed sequences&quot;) %&gt;%
 #  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
 #  kable_styling(full_width = FALSE,font_size = 18) #%&gt;%
 #   # scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)
  
  totseq %&gt;% 
    ggplot(.,aes(x=trt,y=perc_removed) )+
    geom_col(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle(&quot;Total Sequences, percent removed&quot;)+
      # ylab(ylab)+
      xlab(&quot;&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/ggplot%20Total%20sequences-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ggplot-Total-sequences-3">
Past versions of "ggplot Total sequences-3.png"
</button>
</p>
<div id="fig-no-spaces-ggplot-Total-sequences-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/83c0d791fc0c63cf1c68de257041f0751af8df02/docs/figure/Fastqc_results.Rmd/ggplot Total sequences-3.png" target="_blank">83c0d79</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>##Average sequence length</p>
<pre class="r"><code>fastqc_full %&gt;% 
  
  filter(trimmed==&quot;no&quot;) %&gt;%
  mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %&gt;% 
  ggplot(., aes(x=read, y= avg_seq_len))+
  geom_boxplot(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  # ylim(0,55)+
  ggtitle(&quot;Average sequence length, untrimmed&quot;)+
      # ylab(ylab)+
      # xlab(&quot;&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/avg%20seq%20length-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-avg-seq-length-1">
Past versions of "avg seq length-1.png"
</button>
</p>
<div id="fig-no-spaces-avg-seq-length-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/3e9762bda264188db5d38cb4fcb7a245f18b5d99/docs/figure/Fastqc_results.Rmd/avg seq length-1.png" target="_blank">3e9762b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fastqc_full %&gt;% 
  filter(trimmed==&quot;yes&quot;) %&gt;% 
   mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %&gt;% 
  ggplot(., aes(x=read, y= avg_seq_len))+
  geom_boxplot(aes(col= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  scale_color_manual(values = drug_pal)+
  theme_bw()+
  ggtitle(&quot;Average sequence length, trimmed&quot;)+
      # ylab(ylab)+
      # xlab(&quot;&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/avg%20seq%20length-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-avg-seq-length-2">
Past versions of "avg seq length-2.png"
</button>
</p>
<div id="fig-no-spaces-avg-seq-length-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/3e9762bda264188db5d38cb4fcb7a245f18b5d99/docs/figure/Fastqc_results.Rmd/avg seq length-2.png" target="_blank">3e9762b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fastqc_full %&gt;% 
  
  filter(trimmed==&quot;no&quot;) %&gt;%
  group_by(trt) %&gt;% 
  # mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %&gt;% 
  ggplot(., aes(x=read, y= percent_dup))+
  geom_col(position= &quot;dodge&quot;,aes(fill= trt))+
  geom_text(aes(group=trt,label = sprintf(&quot;%.1f&quot;,percent_dup)),
            position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  # ylim(0,55)+
  ggtitle(&quot;Percent duplicated, untrimmed&quot;)+
      # ylab(ylab)+
      # xlab(&quot;&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/percentdup-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-percentdup-1">
Past versions of percentdup-1.png
</button>
</p>
<div id="fig-percentdup-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/3e9762bda264188db5d38cb4fcb7a245f18b5d99/docs/figure/Fastqc_results.Rmd/percentdup-1.png" target="_blank">3e9762b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fastqc_full %&gt;% 
  filter(trimmed==&quot;yes&quot;) %&gt;% 
   group_by(trt) %&gt;% 
  # mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %&gt;% 
  ggplot(., aes(x=read, y= percent_dup))+
  geom_col(position= &quot;dodge&quot;,aes(fill= trt))+
  geom_text(aes(group=trt,label = sprintf(&quot;%.1f&quot;,percent_dup)),
            position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/percentdup-2.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-percentdup-2">
Past versions of percentdup-2.png
</button>
</p>
<div id="fig-percentdup-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/3e9762bda264188db5d38cb4fcb7a245f18b5d99/docs/figure/Fastqc_results.Rmd/percentdup-2.png" target="_blank">3e9762b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="trimmed-sequence-lengths" class="section level2">
<h2>Trimmed sequence lengths</h2>
<pre class="r"><code>###omitted the first288 lines (this was a duplicate of the first multiqc run... changed files to iterate over in the furture)


trimmed_seq_length &lt;- read.csv(&quot;data/trimmed_seq_length.csv&quot;, row.names = 1, col.names = 0:25)
# save &lt;-
trimmed_seq_length &lt;- trimmed_seq_length %&gt;%
  dplyr::rename( &quot;0bp&quot;=X0,&quot;2bp&quot;=X1,&quot;4bp&quot;=X2,&quot;6bp&quot;=X3,&quot;8bp&quot;=X4,&quot;10bp&quot;=X5,&quot;12bp&quot;=X6, &quot;14bp&quot;=X7, &quot;16bp&quot;=X8, &quot;18bp&quot;=X9, &quot;20bp&quot;=X10, &quot;22bp&quot;=X11, &quot;24bp&quot;=X12, &quot;26bp&quot;=X13, &quot;28bp&quot;=X14, &quot;30bp&quot;=X15, &quot;32bp&quot;=X16, &quot;34bp&quot;=X17, &quot;36bp&quot;=X18, &quot;38bp&quot;=X19, &quot;40bp&quot;=X20, &quot;42bp&quot;=X21, &quot;44bp&quot;=X22, &quot;46bp&quot;=X23, &quot;48bp&quot;=X24, &quot;50bp&quot;=X25)# %&gt;%
  # column_to_rownames(&quot;samples&quot;) %&gt;% write.csv(.,&quot;data/trimmed_seq_length.csv&quot;)
#   t() %&gt;% 
# as.numeric()
# trimmed_seq_length %&gt;% 
#   rownames_to_column(&quot;samples&quot;) %&gt;% 
#   pivot_longer(., col=!samples, names_to = &quot;frag_length&quot;, values_to = &quot;counts&quot;)
# 
# 
# 
# pivot_longer(everything(), names_to = &quot;lelsngth&quot;, values_to =&quot;counts&quot;)
# str(save)</code></pre>
<pre class="r"><code>aln_results &lt;- read.csv(&quot;data/aln_run1_results.txt&quot;, row.names = 1)
aln_results %&gt;% 
  mutate(treatment=factor(treatment, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;%
  mutate(time=factor(time, levels = c(&quot;3&quot;,&quot;24&quot;),labels=c(&quot;3 hours&quot;, &quot;24 hours&quot;))) %&gt;% 
  mutate(indv=factor(indv, levels =c (&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;))) %&gt;% 
  
  ggplot(., aes(x =time, y= reads), group= time)+
  geom_boxplot(position = &quot;dodge&quot;,aes(fill=treatment))+
  geom_point(aes(col=indv))+
  facet_wrap(~treatment)+
  theme_bw()+
  ggtitle(&quot;number of read-pairs before mapping&quot;)+
        scale_fill_manual(values=drug_pal)   +
  scale_color_brewer(palette =&quot;Dark2&quot;)+
  theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/trimmed%20readpairs-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-trimmed-readpairs-1">
Past versions of "trimmed readpairs-1.png"
</button>
</p>
<div id="fig-no-spaces-trimmed-readpairs-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/trimmed readpairs-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/1f95eb9c1061623dfef1e5661b66a0140ee861cc/docs/figure/Fastqc_results.Rmd/trimmed readpairs-1.png" target="_blank">1f95eb9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>aln_results %&gt;% 
  mutate(treatment=factor(treatment, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;%
  mutate(time=factor(time, levels = c(&quot;3&quot;,&quot;24&quot;),labels=c(&quot;3 hours&quot;, &quot;24 hours&quot;))) %&gt;% 
  mutate(indv=factor(indv, levels =c (&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;))) %&gt;% 
  # mutate(perc_aln_1_con= aln_1_con/reads * 100) %&gt;% 
  
  ggplot(., aes(x =time, y= aln_1_con), group= time)+
  geom_col(position= &quot;dodge&quot;,aes(fill= treatment))+
  # geom_text(aes(group=treatment,label = sprintf(&quot;%.1f&quot;,(aln_overall*100)),
  #           position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 ))+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle(&quot;number of concordantly aligned reads = 1&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/number%20of%20aln%20is%201-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-number-of-aln-is-1-1">
Past versions of "number of aln is 1-1.png"
</button>
</p>
<div id="fig-no-spaces-number-of-aln-is-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/1f95eb9c1061623dfef1e5661b66a0140ee861cc/docs/figure/Fastqc_results.Rmd/number of aln is 1-1.png" target="_blank">1f95eb9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>aln_results %&gt;% 
  mutate(treatment=factor(treatment, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;%
  mutate(time=factor(time, levels = c(&quot;3&quot;,&quot;24&quot;),labels=c(&quot;3 hours&quot;, &quot;24 hours&quot;))) %&gt;% 
  mutate(indv=factor(indv, levels =c (&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;))) %&gt;% 
  mutate(perc_aln_1_con= aln_1_con/reads * 100) %&gt;% 
  
  ggplot(., aes(x =time, y= perc_aln_1_con), group= time)+
  geom_col(position= &quot;dodge&quot;,aes(fill= treatment))+
  geom_text(aes(group=treatment,label = sprintf(&quot;%.1f&quot;,perc_aln_1_con)),
            position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle(&quot;Percent of concordantly aligned reads = 1&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/percent%20conc%20aln%20reads-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-percent-conc-aln-reads-1">
Past versions of "percent conc aln reads-1.png"
</button>
</p>
<div id="fig-no-spaces-percent-conc-aln-reads-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/1f95eb9c1061623dfef1e5661b66a0140ee861cc/docs/figure/Fastqc_results.Rmd/percent conc aln reads-1.png" target="_blank">1f95eb9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>aln_results %&gt;% 
  mutate(treatment=factor(treatment, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;%
  mutate(time=factor(time, levels = c(&quot;3&quot;,&quot;24&quot;),labels=c(&quot;3 hours&quot;, &quot;24 hours&quot;))) %&gt;% 
  mutate(indv=factor(indv, levels =c (&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;))) %&gt;% 
  mutate(perc_multimapped= aln_.1_con/reads * 100) %&gt;% 
  
  ggplot(., aes(x =time, y= perc_multimapped), group= time)+
  geom_col(position= &quot;dodge&quot;,aes(fill= treatment))+
  geom_text(aes(group=treatment,label = sprintf(&quot;%.1f&quot;,perc_multimapped)),
            position=position_dodge(width =.93),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle(&quot;Percent of aligned reads &gt; 1&quot;)+
      theme(strip.background = element_rect(fill = &quot;white&quot;,linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face=&quot;bold&quot;),
          axis.title = element_text(size = 10, color = &quot;black&quot;),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, &quot;pt&quot;),face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Fastqc_results.Rmd/perc%20multimaped%20reads-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-perc-multimaped-reads-1">
Past versions of "perc multimaped reads-1.png"
</button>
</p>
<div id="fig-no-spaces-perc-multimaped-reads-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/1f95eb9c1061623dfef1e5661b66a0140ee861cc/docs/figure/Fastqc_results.Rmd/perc multimaped reads-1.png" target="_blank">1f95eb9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="after-mapping-and-filtering" class="section level2">
<h2>After Mapping and filtering</h2>
<ul>
<li><p> Currently the steps were are as follows:</p>
<ul>
<li><p>Basic Fastqc followed by adapter trimming and Fastqc analysis on
the leftover fragments.</p></li>
<li><p>Trimmed reads were aligned to the hg38 human genome.</p></li>
<li><p>Mitochondrial reads (chrM) were removed</p></li>
<li><p>samtools was used to removed non-paired, discordantly paired, and
multi-mapped reads from the .bam.</p></li>
<li><p>Markduplicates function from Picard was used to mark optical and
PCR duplicates, with samtools used to remove these reads using the flag
-F 1024.</p></li>
</ul></li>
</ul>
<pre class="r"><code>Ind1_summary  &lt;- read.csv(&quot;data/Ind1_summary.txt&quot;, row.names = 1) %&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;reads&quot;=X2,&quot;mapped&quot;=X3)
Ind2_summary  &lt;- read.csv(&quot;data/Ind2_summary.txt&quot;, row.names = 1)%&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;reads&quot;=X2,&quot;mapped&quot;=X3)
Ind3_summary  &lt;- read.csv(&quot;data/Ind3_summary.txt&quot;, row.names = 1)%&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;reads&quot;=X2,&quot;mapped&quot;=X3)
Ind4_summary  &lt;- read.csv(&quot;data/Ind4_summary.txt&quot;, row.names = 1)%&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;reads&quot;=X2,&quot;mapped&quot;=X3)
Ind5_summary  &lt;- read.csv(&quot;data/Ind5_summary.txt&quot;, row.names = 1)%&gt;% 
 dplyr::rename(&quot;sample&quot;=X1,&quot;reads&quot;=X2,&quot;mapped&quot;=X3)
Ind6_summary  &lt;- read.csv(&quot;data/Ind6_summary.txt&quot;, row.names = 1)%&gt;% 
  dplyr::rename(&quot;sample&quot;=X1,&quot;reads&quot;=X2,&quot;mapped&quot;=X3)</code></pre>
<div id="individual-1-read-summary" class="section level3">
<h3>Individual 1 read summary</h3>
<pre class="r"><code>Ind1_reads_summary &lt;- 
Ind1_summary %&gt;%                          
 separate(reads,into=c(&quot;reads&quot;,NA),sep= &quot; &quot;) %&gt;% 
   mutate(reads=as.numeric(reads)) %&gt;% 
   separate(mapped, into= c(&quot;mapped_reads&quot;, NA,NA, &quot;percent_mapped_reads&quot;,NA)) %&gt;% 
   separate(sample, into=c(&quot;one&quot;,&quot;two&quot;,NA, &quot;4&quot;,&quot;sample&quot;,NA,&quot;seven&quot;,&quot;8&quot;)) %&gt;% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %&gt;% 
   mutate(type = if_else(two==&quot;first&quot;, &quot;total&quot;,
                         if_else(two==&quot;noM&quot;,&quot;nuclear&quot;,
                                 if_else((seven == &quot;fin&quot;&amp; two==&quot;files&quot;), &quot;unique&quot;, &quot;dedup&quot;))))%&gt;% 
   dplyr::select(sample,type,  reads, mapped_reads)%&gt;% 
   pivot_longer(cols=reads:mapped_reads, names_to = &quot;read_info&quot;, values_to = &quot;reads&quot;) %&gt;% 
   unite(&quot;type&quot;,type:read_info, sep=&quot;_&quot;) %&gt;% 
   distinct() %&gt;% 
   pivot_wider(id_cols = sample, names_from = &quot;type&quot;, values_from = &quot;reads&quot;) %&gt;% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %&gt;% 
  mutate(dedup_read_pairs=dedup_reads/2) %&gt;% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind1_reads_summary %&gt;% 
   kable(., caption= &quot;Reads summary from Ind 1&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 18) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 18px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Reads summary from Ind 1
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
sample
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
per_nuclear_mapped
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unique_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_read_pairs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
75DA24h
</td>
<td style="text-align:right;">
85102446
</td>
<td style="text-align:right;">
83436153
</td>
<td style="text-align:right;">
66053400
</td>
<td style="text-align:right;">
64409401
</td>
<td style="text-align:right;">
77.19603
</td>
<td style="text-align:right;">
49798536
</td>
<td style="text-align:right;">
42171858
</td>
<td style="text-align:right;">
21085929
</td>
</tr>
<tr>
<td style="text-align:left;">
75DA3h
</td>
<td style="text-align:right;">
96460622
</td>
<td style="text-align:right;">
94743255
</td>
<td style="text-align:right;">
58141974
</td>
<td style="text-align:right;">
56469027
</td>
<td style="text-align:right;">
59.60216
</td>
<td style="text-align:right;">
42013916
</td>
<td style="text-align:right;">
34375528
</td>
<td style="text-align:right;">
17187764
</td>
</tr>
<tr>
<td style="text-align:left;">
75DX24h
</td>
<td style="text-align:right;">
86568932
</td>
<td style="text-align:right;">
85201226
</td>
<td style="text-align:right;">
64404238
</td>
<td style="text-align:right;">
63062470
</td>
<td style="text-align:right;">
74.01592
</td>
<td style="text-align:right;">
50177516
</td>
<td style="text-align:right;">
39698738
</td>
<td style="text-align:right;">
19849369
</td>
</tr>
<tr>
<td style="text-align:left;">
75DX3h
</td>
<td style="text-align:right;">
88961680
</td>
<td style="text-align:right;">
87544345
</td>
<td style="text-align:right;">
57721936
</td>
<td style="text-align:right;">
56338902
</td>
<td style="text-align:right;">
64.35470
</td>
<td style="text-align:right;">
43476750
</td>
<td style="text-align:right;">
35305350
</td>
<td style="text-align:right;">
17652675
</td>
</tr>
<tr>
<td style="text-align:left;">
75E24h
</td>
<td style="text-align:right;">
85515534
</td>
<td style="text-align:right;">
84156301
</td>
<td style="text-align:right;">
64314630
</td>
<td style="text-align:right;">
62981693
</td>
<td style="text-align:right;">
74.83895
</td>
<td style="text-align:right;">
50758792
</td>
<td style="text-align:right;">
38466006
</td>
<td style="text-align:right;">
19233003
</td>
</tr>
<tr>
<td style="text-align:left;">
75E3h
</td>
<td style="text-align:right;">
86850434
</td>
<td style="text-align:right;">
85065673
</td>
<td style="text-align:right;">
46706014
</td>
<td style="text-align:right;">
44956986
</td>
<td style="text-align:right;">
52.84974
</td>
<td style="text-align:right;">
33433984
</td>
<td style="text-align:right;">
28499906
</td>
<td style="text-align:right;">
14249953
</td>
</tr>
<tr>
<td style="text-align:left;">
75M24h
</td>
<td style="text-align:right;">
84253310
</td>
<td style="text-align:right;">
82788715
</td>
<td style="text-align:right;">
63010726
</td>
<td style="text-align:right;">
61559907
</td>
<td style="text-align:right;">
74.35785
</td>
<td style="text-align:right;">
48792426
</td>
<td style="text-align:right;">
39199816
</td>
<td style="text-align:right;">
19599908
</td>
</tr>
<tr>
<td style="text-align:left;">
75M3h
</td>
<td style="text-align:right;">
68695646
</td>
<td style="text-align:right;">
67291306
</td>
<td style="text-align:right;">
42935278
</td>
<td style="text-align:right;">
41548693
</td>
<td style="text-align:right;">
61.74452
</td>
<td style="text-align:right;">
31769450
</td>
<td style="text-align:right;">
27750384
</td>
<td style="text-align:right;">
13875192
</td>
</tr>
<tr>
<td style="text-align:left;">
75T24h
</td>
<td style="text-align:right;">
79185864
</td>
<td style="text-align:right;">
77440692
</td>
<td style="text-align:right;">
53324884
</td>
<td style="text-align:right;">
51609067
</td>
<td style="text-align:right;">
66.64334
</td>
<td style="text-align:right;">
40379014
</td>
<td style="text-align:right;">
26761992
</td>
<td style="text-align:right;">
13380996
</td>
</tr>
<tr>
<td style="text-align:left;">
75T3h
</td>
<td style="text-align:right;">
65007048
</td>
<td style="text-align:right;">
64223127
</td>
<td style="text-align:right;">
39631422
</td>
<td style="text-align:right;">
38860422
</td>
<td style="text-align:right;">
60.50846
</td>
<td style="text-align:right;">
29963178
</td>
<td style="text-align:right;">
25078330
</td>
<td style="text-align:right;">
12539165
</td>
</tr>
<tr>
<td style="text-align:left;">
75V24h
</td>
<td style="text-align:right;">
87628722
</td>
<td style="text-align:right;">
86228282
</td>
<td style="text-align:right;">
61813336
</td>
<td style="text-align:right;">
60441680
</td>
<td style="text-align:right;">
70.09496
</td>
<td style="text-align:right;">
48093816
</td>
<td style="text-align:right;">
35969854
</td>
<td style="text-align:right;">
17984927
</td>
</tr>
<tr>
<td style="text-align:left;">
75V3h
</td>
<td style="text-align:right;">
67368518
</td>
<td style="text-align:right;">
66122877
</td>
<td style="text-align:right;">
32655484
</td>
<td style="text-align:right;">
31448184
</td>
<td style="text-align:right;">
47.56022
</td>
<td style="text-align:right;">
22740426
</td>
<td style="text-align:right;">
19004626
</td>
<td style="text-align:right;">
9502313
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>Ind1_reads_summary %&gt;% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to=&quot;type&quot;, values_to = &quot;reads&quot;) %&gt;% 
  dplyr::filter(type %in% list(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;, &quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)) %&gt;% 
  mutate(type=factor(type, levels=c(&quot;total_reads&quot;, &quot;total_mapped_reads&quot;, &quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;))) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;,sample)) %&gt;% 
  dplyr::filter (type != &quot;total_reads&quot;) %&gt;% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl(&quot;24h$&quot;, sample) ==TRUE, &quot;24_hours&quot;,&quot;3_hours&quot;)) %&gt;% 
   mutate(time=factor(time, levels=c(&quot;3_hours&quot;,&quot;24_hours&quot;))) %&gt;% 
  mutate(trt = case_match(trt,&quot;DAh&quot;~&quot;DNR&quot;,&quot;DXh&quot;~&quot;DOX&quot;,&quot;Eh&quot;~&quot;EPI&quot;, &quot;Mh&quot; ~ &quot;MTX&quot;, &quot;Th&quot; ~ &quot;TRZ&quot;, &quot;Vh&quot; ~&quot;VEH&quot;,.default = trt)) %&gt;% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position=&quot;dodge&quot;,aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Ind 1&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/frag%20length-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-frag-length-1">
Past versions of "frag length-1.png"
</button>
</p>
<div id="fig-no-spaces-frag-length-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/frag length-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/frag length-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/1f95eb9c1061623dfef1e5661b66a0140ee861cc/docs/figure/Fastqc_results.Rmd/frag length-1.png" target="_blank">1f95eb9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-02-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="individual-2-read-summary" class="section level3">
<h3>Individual 2 read summary</h3>
<pre class="r"><code>Ind2_reads_summary &lt;- 
Ind2_summary %&gt;%                          
 separate(reads,into=c(&quot;reads&quot;,NA),sep= &quot; &quot;) %&gt;% 
   mutate(reads=as.numeric(reads)) %&gt;% 
   separate(mapped, into= c(&quot;mapped_reads&quot;, NA,NA, &quot;percent_mapped_reads&quot;,NA)) %&gt;% 
   separate(sample, into=c(&quot;one&quot;,&quot;two&quot;,NA, &quot;4&quot;,&quot;sample&quot;,NA,&quot;seven&quot;,&quot;8&quot;)) %&gt;% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %&gt;% 
   mutate(type = if_else(two==&quot;first&quot;, &quot;total&quot;,
                         if_else(two==&quot;noM&quot;,&quot;nuclear&quot;,
                                 if_else((seven == &quot;fin&quot;&amp; two==&quot;files&quot;), &quot;unique&quot;, &quot;dedup&quot;))))%&gt;% 
   dplyr::select(sample,type,  reads, mapped_reads)%&gt;% 
  
   pivot_longer(cols=reads:mapped_reads, names_to = &quot;read_info&quot;, values_to = &quot;reads&quot;) %&gt;% 
   unite(&quot;type&quot;,type:read_info, sep=&quot;_&quot;) %&gt;% 
   distinct() %&gt;% 
   pivot_wider(id_cols = sample, names_from = &quot;type&quot;, values_from = &quot;reads&quot;) %&gt;% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %&gt;% 
  mutate(dedup_read_pairs=dedup_reads/2) %&gt;% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind2_reads_summary %&gt;% 
   kable(., caption= &quot;Reads summary from Ind 2&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 18) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 18px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Reads summary from Ind 2
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
sample
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
per_nuclear_mapped
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unique_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_read_pairs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
87DA24h
</td>
<td style="text-align:right;">
91284490
</td>
<td style="text-align:right;">
90726302
</td>
<td style="text-align:right;">
50438182
</td>
<td style="text-align:right;">
49917009
</td>
<td style="text-align:right;">
55.01934
</td>
<td style="text-align:right;">
38845730
</td>
<td style="text-align:right;">
29862542
</td>
<td style="text-align:right;">
14931271
</td>
</tr>
<tr>
<td style="text-align:left;">
87DA3h
</td>
<td style="text-align:right;">
99530062
</td>
<td style="text-align:right;">
98512274
</td>
<td style="text-align:right;">
69173490
</td>
<td style="text-align:right;">
68181108
</td>
<td style="text-align:right;">
69.21077
</td>
<td style="text-align:right;">
53701246
</td>
<td style="text-align:right;">
33518302
</td>
<td style="text-align:right;">
16759151
</td>
</tr>
<tr>
<td style="text-align:left;">
87DX24h
</td>
<td style="text-align:right;">
95926258
</td>
<td style="text-align:right;">
95529135
</td>
<td style="text-align:right;">
45925936
</td>
<td style="text-align:right;">
45576255
</td>
<td style="text-align:right;">
47.70927
</td>
<td style="text-align:right;">
35048106
</td>
<td style="text-align:right;">
28145618
</td>
<td style="text-align:right;">
14072809
</td>
</tr>
<tr>
<td style="text-align:left;">
87DX3h
</td>
<td style="text-align:right;">
86437590
</td>
<td style="text-align:right;">
85672915
</td>
<td style="text-align:right;">
56277016
</td>
<td style="text-align:right;">
55536889
</td>
<td style="text-align:right;">
64.82433
</td>
<td style="text-align:right;">
44108178
</td>
<td style="text-align:right;">
29633114
</td>
<td style="text-align:right;">
14816557
</td>
</tr>
<tr>
<td style="text-align:left;">
87E24h
</td>
<td style="text-align:right;">
75753486
</td>
<td style="text-align:right;">
75405936
</td>
<td style="text-align:right;">
42828630
</td>
<td style="text-align:right;">
42514478
</td>
<td style="text-align:right;">
56.38081
</td>
<td style="text-align:right;">
33142100
</td>
<td style="text-align:right;">
26866466
</td>
<td style="text-align:right;">
13433233
</td>
</tr>
<tr>
<td style="text-align:left;">
87E3h
</td>
<td style="text-align:right;">
93018986
</td>
<td style="text-align:right;">
92423041
</td>
<td style="text-align:right;">
62694510
</td>
<td style="text-align:right;">
62125220
</td>
<td style="text-align:right;">
67.21832
</td>
<td style="text-align:right;">
49108624
</td>
<td style="text-align:right;">
36723324
</td>
<td style="text-align:right;">
18361662
</td>
</tr>
<tr>
<td style="text-align:left;">
87M24h
</td>
<td style="text-align:right;">
95131906
</td>
<td style="text-align:right;">
94542825
</td>
<td style="text-align:right;">
56081116
</td>
<td style="text-align:right;">
55527509
</td>
<td style="text-align:right;">
58.73265
</td>
<td style="text-align:right;">
43614330
</td>
<td style="text-align:right;">
33827912
</td>
<td style="text-align:right;">
16913956
</td>
</tr>
<tr>
<td style="text-align:left;">
87M3h
</td>
<td style="text-align:right;">
93095744
</td>
<td style="text-align:right;">
92544851
</td>
<td style="text-align:right;">
70021308
</td>
<td style="text-align:right;">
69490912
</td>
<td style="text-align:right;">
75.08890
</td>
<td style="text-align:right;">
57350454
</td>
<td style="text-align:right;">
40556564
</td>
<td style="text-align:right;">
20278282
</td>
</tr>
<tr>
<td style="text-align:left;">
87T24h
</td>
<td style="text-align:right;">
84449704
</td>
<td style="text-align:right;">
83992710
</td>
<td style="text-align:right;">
38660624
</td>
<td style="text-align:right;">
38236404
</td>
<td style="text-align:right;">
45.52348
</td>
<td style="text-align:right;">
29595164
</td>
<td style="text-align:right;">
22302276
</td>
<td style="text-align:right;">
11151138
</td>
</tr>
<tr>
<td style="text-align:left;">
87T3h
</td>
<td style="text-align:right;">
77262934
</td>
<td style="text-align:right;">
76647754
</td>
<td style="text-align:right;">
46790664
</td>
<td style="text-align:right;">
46202260
</td>
<td style="text-align:right;">
60.27869
</td>
<td style="text-align:right;">
37014986
</td>
<td style="text-align:right;">
24758586
</td>
<td style="text-align:right;">
12379293
</td>
</tr>
<tr>
<td style="text-align:left;">
87V24h
</td>
<td style="text-align:right;">
76257666
</td>
<td style="text-align:right;">
75404837
</td>
<td style="text-align:right;">
43814448
</td>
<td style="text-align:right;">
42990898
</td>
<td style="text-align:right;">
57.01345
</td>
<td style="text-align:right;">
34879692
</td>
<td style="text-align:right;">
20674790
</td>
<td style="text-align:right;">
10337395
</td>
</tr>
<tr>
<td style="text-align:left;">
87V3h
</td>
<td style="text-align:right;">
78106046
</td>
<td style="text-align:right;">
77578683
</td>
<td style="text-align:right;">
53170414
</td>
<td style="text-align:right;">
52664767
</td>
<td style="text-align:right;">
67.88562
</td>
<td style="text-align:right;">
42211678
</td>
<td style="text-align:right;">
32184480
</td>
<td style="text-align:right;">
16092240
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>Ind2_reads_summary %&gt;% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to=&quot;type&quot;, values_to = &quot;reads&quot;) %&gt;% 
  dplyr::filter(type %in% list(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;, &quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)) %&gt;% 
  mutate(type=factor(type, levels=c(&quot;total_reads&quot;, &quot;total_mapped_reads&quot;, &quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;))) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;,sample)) %&gt;% 
  dplyr::filter (type != &quot;total_reads&quot;) %&gt;% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl(&quot;24h$&quot;, sample) ==TRUE, &quot;24_hours&quot;,&quot;3_hours&quot;)) %&gt;% 
   mutate(time=factor(time, levels=c(&quot;3_hours&quot;,&quot;24_hours&quot;))) %&gt;% 
  mutate(trt = case_match(trt,&quot;DAh&quot;~&quot;DNR&quot;,&quot;DXh&quot;~&quot;DOX&quot;,&quot;Eh&quot;~&quot;EPI&quot;, &quot;Mh&quot; ~ &quot;MTX&quot;, &quot;Th&quot; ~ &quot;TRZ&quot;, &quot;Vh&quot; ~&quot;VEH&quot;,.default = trt)) %&gt;% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position=&quot;dodge&quot;,aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Ind 2&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/Readssummary2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Readssummary2-1">
Past versions of Readssummary2-1.png
</button>
</p>
<div id="fig-Readssummary2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/Readssummary2-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/Readssummary2-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="individual-3-read-summary" class="section level3">
<h3>Individual 3 read summary</h3>
<pre class="r"><code>Ind3_reads_summary &lt;- 
Ind3_summary %&gt;%                          
 separate(reads,into=c(&quot;reads&quot;,NA),sep= &quot; &quot;) %&gt;% 
   mutate(reads=as.numeric(reads)) %&gt;% 
   separate(mapped, into= c(&quot;mapped_reads&quot;, NA,NA, &quot;percent_mapped_reads&quot;,NA)) %&gt;% 
   separate(sample, into=c(&quot;one&quot;,&quot;two&quot;,NA, &quot;4&quot;,&quot;sample&quot;,NA,&quot;seven&quot;,&quot;8&quot;)) %&gt;% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %&gt;% 
   mutate(type = if_else(two==&quot;first&quot;, &quot;total&quot;,
                         if_else(two==&quot;noM&quot;,&quot;nuclear&quot;,
                                 if_else((seven == &quot;fin&quot;&amp; two==&quot;files&quot;), &quot;unique&quot;, &quot;dedup&quot;))))%&gt;% 
   dplyr::select(sample,type,  reads, mapped_reads)%&gt;% 
   pivot_longer(cols=reads:mapped_reads, names_to = &quot;read_info&quot;, values_to = &quot;reads&quot;) %&gt;% 
   unite(&quot;type&quot;,type:read_info, sep=&quot;_&quot;) %&gt;% 
   distinct() %&gt;% 
   pivot_wider(id_cols = sample, names_from = &quot;type&quot;, values_from = &quot;reads&quot;) %&gt;% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %&gt;% 
  mutate(dedup_read_pairs=dedup_reads/2) %&gt;% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind3_reads_summary %&gt;% 
   kable(., caption= &quot;Reads summary from Ind 3&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 18) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 18px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Reads summary from Ind 3
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
sample
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
per_nuclear_mapped
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unique_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_read_pairs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
77DA24h
</td>
<td style="text-align:right;">
75363518
</td>
<td style="text-align:right;">
74999111
</td>
<td style="text-align:right;">
35094224
</td>
<td style="text-align:right;">
34752650
</td>
<td style="text-align:right;">
46.33742
</td>
<td style="text-align:right;">
27025868
</td>
<td style="text-align:right;">
21869650
</td>
<td style="text-align:right;">
10934825
</td>
</tr>
<tr>
<td style="text-align:left;">
77DA3h
</td>
<td style="text-align:right;">
142295378
</td>
<td style="text-align:right;">
141426527
</td>
<td style="text-align:right;">
62143706
</td>
<td style="text-align:right;">
61349463
</td>
<td style="text-align:right;">
43.37904
</td>
<td style="text-align:right;">
46664938
</td>
<td style="text-align:right;">
34279586
</td>
<td style="text-align:right;">
17139793
</td>
</tr>
<tr>
<td style="text-align:left;">
77DX24h
</td>
<td style="text-align:right;">
78097470
</td>
<td style="text-align:right;">
77592243
</td>
<td style="text-align:right;">
37675048
</td>
<td style="text-align:right;">
37212379
</td>
<td style="text-align:right;">
47.95889
</td>
<td style="text-align:right;">
29360006
</td>
<td style="text-align:right;">
19207264
</td>
<td style="text-align:right;">
9603632
</td>
</tr>
<tr>
<td style="text-align:left;">
77DX3h
</td>
<td style="text-align:right;">
87512178
</td>
<td style="text-align:right;">
86896129
</td>
<td style="text-align:right;">
35332636
</td>
<td style="text-align:right;">
34767930
</td>
<td style="text-align:right;">
40.01091
</td>
<td style="text-align:right;">
26347392
</td>
<td style="text-align:right;">
18957010
</td>
<td style="text-align:right;">
9478505
</td>
</tr>
<tr>
<td style="text-align:left;">
77E24h
</td>
<td style="text-align:right;">
82977864
</td>
<td style="text-align:right;">
82385624
</td>
<td style="text-align:right;">
40566624
</td>
<td style="text-align:right;">
40017951
</td>
<td style="text-align:right;">
48.57395
</td>
<td style="text-align:right;">
31314418
</td>
<td style="text-align:right;">
18551820
</td>
<td style="text-align:right;">
9275910
</td>
</tr>
<tr>
<td style="text-align:left;">
77E3h
</td>
<td style="text-align:right;">
100276150
</td>
<td style="text-align:right;">
99649171
</td>
<td style="text-align:right;">
44913542
</td>
<td style="text-align:right;">
44340553
</td>
<td style="text-align:right;">
44.49666
</td>
<td style="text-align:right;">
33759792
</td>
<td style="text-align:right;">
25465236
</td>
<td style="text-align:right;">
12732618
</td>
</tr>
<tr>
<td style="text-align:left;">
77M24h
</td>
<td style="text-align:right;">
72997484
</td>
<td style="text-align:right;">
72456264
</td>
<td style="text-align:right;">
39085452
</td>
<td style="text-align:right;">
38577089
</td>
<td style="text-align:right;">
53.24190
</td>
<td style="text-align:right;">
30258994
</td>
<td style="text-align:right;">
21749104
</td>
<td style="text-align:right;">
10874552
</td>
</tr>
<tr>
<td style="text-align:left;">
77M3h
</td>
<td style="text-align:right;">
98862906
</td>
<td style="text-align:right;">
98372193
</td>
<td style="text-align:right;">
42466218
</td>
<td style="text-align:right;">
42029163
</td>
<td style="text-align:right;">
42.72464
</td>
<td style="text-align:right;">
32457148
</td>
<td style="text-align:right;">
24872086
</td>
<td style="text-align:right;">
12436043
</td>
</tr>
<tr>
<td style="text-align:left;">
77T24h
</td>
<td style="text-align:right;">
81855304
</td>
<td style="text-align:right;">
81363048
</td>
<td style="text-align:right;">
31562706
</td>
<td style="text-align:right;">
31121534
</td>
<td style="text-align:right;">
38.25021
</td>
<td style="text-align:right;">
22935466
</td>
<td style="text-align:right;">
17397726
</td>
<td style="text-align:right;">
8698863
</td>
</tr>
<tr>
<td style="text-align:left;">
77T3h
</td>
<td style="text-align:right;">
77560582
</td>
<td style="text-align:right;">
77137107
</td>
<td style="text-align:right;">
27982800
</td>
<td style="text-align:right;">
27608032
</td>
<td style="text-align:right;">
35.79086
</td>
<td style="text-align:right;">
20240194
</td>
<td style="text-align:right;">
16008756
</td>
<td style="text-align:right;">
8004378
</td>
</tr>
<tr>
<td style="text-align:left;">
77V24h
</td>
<td style="text-align:right;">
62649778
</td>
<td style="text-align:right;">
61652700
</td>
<td style="text-align:right;">
33205184
</td>
<td style="text-align:right;">
32226761
</td>
<td style="text-align:right;">
52.27145
</td>
<td style="text-align:right;">
25488894
</td>
<td style="text-align:right;">
7986090
</td>
<td style="text-align:right;">
3993045
</td>
</tr>
<tr>
<td style="text-align:left;">
77V3h
</td>
<td style="text-align:right;">
89706546
</td>
<td style="text-align:right;">
89075188
</td>
<td style="text-align:right;">
35232316
</td>
<td style="text-align:right;">
34649803
</td>
<td style="text-align:right;">
38.89950
</td>
<td style="text-align:right;">
25704912
</td>
<td style="text-align:right;">
18970988
</td>
<td style="text-align:right;">
9485494
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code> Ind3_reads_summary %&gt;% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to=&quot;type&quot;, values_to = &quot;reads&quot;) %&gt;% 
  dplyr::filter(type %in% list(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;, &quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)) %&gt;% 
  mutate(type=factor(type, levels=c(&quot;total_reads&quot;, &quot;total_mapped_reads&quot;, &quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;))) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;,sample)) %&gt;% 
  dplyr::filter (type != &quot;total_reads&quot;) %&gt;% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl(&quot;24h$&quot;, sample) ==TRUE, &quot;24_hours&quot;,&quot;3_hours&quot;)) %&gt;% 
   mutate(time=factor(time, levels=c(&quot;3_hours&quot;,&quot;24_hours&quot;))) %&gt;% 
  mutate(trt = case_match(trt,&quot;DAh&quot;~&quot;DNR&quot;,&quot;DXh&quot;~&quot;DOX&quot;,&quot;Eh&quot;~&quot;EPI&quot;, &quot;Mh&quot; ~ &quot;MTX&quot;, &quot;Th&quot; ~ &quot;TRZ&quot;, &quot;Vh&quot; ~&quot;VEH&quot;,.default = trt)) %&gt;% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position=&quot;dodge&quot;,aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Ind 3&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/Readssummary3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Readssummary3-1">
Past versions of Readssummary3-1.png
</button>
</p>
<div id="fig-Readssummary3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/Readssummary3-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/Readssummary3-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="individual-4-read-summary" class="section level3">
<h3>Individual 4 read summary</h3>
<pre class="r"><code>Ind4_reads_summary &lt;- 
Ind4_summary %&gt;%                          
 separate(reads,into=c(&quot;reads&quot;,NA),sep= &quot; &quot;) %&gt;% 
   mutate(reads=as.numeric(reads)) %&gt;% 
   separate(mapped, into= c(&quot;mapped_reads&quot;, NA,NA, &quot;percent_mapped_reads&quot;,NA)) %&gt;% 
   separate(sample, into=c(&quot;one&quot;,&quot;two&quot;,NA, &quot;4&quot;,&quot;sample&quot;,NA,&quot;seven&quot;,&quot;8&quot;)) %&gt;% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %&gt;% 
   mutate(type = if_else(two==&quot;first&quot;, &quot;total&quot;,
                         if_else(two==&quot;noM&quot;,&quot;nuclear&quot;,
                                 if_else((seven == &quot;fin&quot;&amp; two==&quot;files&quot;), &quot;unique&quot;, &quot;dedup&quot;))))%&gt;% 
   dplyr::select(sample,type,  reads, mapped_reads)%&gt;% 
   pivot_longer(cols=reads:mapped_reads, names_to = &quot;read_info&quot;, values_to = &quot;reads&quot;) %&gt;% 
   unite(&quot;type&quot;,type:read_info, sep=&quot;_&quot;) %&gt;% 
   distinct() %&gt;% 
   pivot_wider(id_cols = sample, names_from = &quot;type&quot;, values_from = &quot;reads&quot;) %&gt;% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %&gt;% 
  mutate(dedup_read_pairs=dedup_reads/2) %&gt;% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind4_reads_summary %&gt;% 
   kable(., caption= &quot;Reads summary from Ind 4&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 18) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 18px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Reads summary from Ind 4
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
sample
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
per_nuclear_mapped
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unique_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_read_pairs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
79DA24h
</td>
<td style="text-align:right;">
83930268
</td>
<td style="text-align:right;">
83140985
</td>
<td style="text-align:right;">
39561574
</td>
<td style="text-align:right;">
38811928
</td>
<td style="text-align:right;">
46.68206
</td>
<td style="text-align:right;">
29459906
</td>
<td style="text-align:right;">
17862648
</td>
<td style="text-align:right;">
8931324
</td>
</tr>
<tr>
<td style="text-align:left;">
79DA3h
</td>
<td style="text-align:right;">
114610072
</td>
<td style="text-align:right;">
112679708
</td>
<td style="text-align:right;">
74283554
</td>
<td style="text-align:right;">
72386250
</td>
<td style="text-align:right;">
64.24071
</td>
<td style="text-align:right;">
55956416
</td>
<td style="text-align:right;">
26487126
</td>
<td style="text-align:right;">
13243563
</td>
</tr>
<tr>
<td style="text-align:left;">
79DX24h
</td>
<td style="text-align:right;">
78395382
</td>
<td style="text-align:right;">
77692597
</td>
<td style="text-align:right;">
25391928
</td>
<td style="text-align:right;">
24733417
</td>
<td style="text-align:right;">
31.83497
</td>
<td style="text-align:right;">
17146642
</td>
<td style="text-align:right;">
11544950
</td>
<td style="text-align:right;">
5772475
</td>
</tr>
<tr>
<td style="text-align:left;">
79DX3h
</td>
<td style="text-align:right;">
77601292
</td>
<td style="text-align:right;">
76058132
</td>
<td style="text-align:right;">
47116936
</td>
<td style="text-align:right;">
45593581
</td>
<td style="text-align:right;">
59.94570
</td>
<td style="text-align:right;">
35460994
</td>
<td style="text-align:right;">
18908344
</td>
<td style="text-align:right;">
9454172
</td>
</tr>
<tr>
<td style="text-align:left;">
79E24h
</td>
<td style="text-align:right;">
86039200
</td>
<td style="text-align:right;">
85324746
</td>
<td style="text-align:right;">
29965542
</td>
<td style="text-align:right;">
29300484
</td>
<td style="text-align:right;">
34.33996
</td>
<td style="text-align:right;">
21004354
</td>
<td style="text-align:right;">
12495788
</td>
<td style="text-align:right;">
6247894
</td>
</tr>
<tr>
<td style="text-align:left;">
79E3h
</td>
<td style="text-align:right;">
86993222
</td>
<td style="text-align:right;">
85671962
</td>
<td style="text-align:right;">
46372712
</td>
<td style="text-align:right;">
45084280
</td>
<td style="text-align:right;">
52.62431
</td>
<td style="text-align:right;">
34578548
</td>
<td style="text-align:right;">
19109612
</td>
<td style="text-align:right;">
9554806
</td>
</tr>
<tr>
<td style="text-align:left;">
79M24h
</td>
<td style="text-align:right;">
82061002
</td>
<td style="text-align:right;">
81424543
</td>
<td style="text-align:right;">
27148372
</td>
<td style="text-align:right;">
26562054
</td>
<td style="text-align:right;">
32.62168
</td>
<td style="text-align:right;">
19085838
</td>
<td style="text-align:right;">
9894182
</td>
<td style="text-align:right;">
4947091
</td>
</tr>
<tr>
<td style="text-align:left;">
79M3h
</td>
<td style="text-align:right;">
83929214
</td>
<td style="text-align:right;">
82368935
</td>
<td style="text-align:right;">
48985626
</td>
<td style="text-align:right;">
47454435
</td>
<td style="text-align:right;">
57.61205
</td>
<td style="text-align:right;">
36937534
</td>
<td style="text-align:right;">
16384910
</td>
<td style="text-align:right;">
8192455
</td>
</tr>
<tr>
<td style="text-align:left;">
79T24h
</td>
<td style="text-align:right;">
90875858
</td>
<td style="text-align:right;">
89680567
</td>
<td style="text-align:right;">
31347532
</td>
<td style="text-align:right;">
30204755
</td>
<td style="text-align:right;">
33.68038
</td>
<td style="text-align:right;">
21789834
</td>
<td style="text-align:right;">
8948490
</td>
<td style="text-align:right;">
4474245
</td>
</tr>
<tr>
<td style="text-align:left;">
79T3h
</td>
<td style="text-align:right;">
106856444
</td>
<td style="text-align:right;">
105027081
</td>
<td style="text-align:right;">
65690664
</td>
<td style="text-align:right;">
63898507
</td>
<td style="text-align:right;">
60.84003
</td>
<td style="text-align:right;">
49669024
</td>
<td style="text-align:right;">
28930176
</td>
<td style="text-align:right;">
14465088
</td>
</tr>
<tr>
<td style="text-align:left;">
79V24h
</td>
<td style="text-align:right;">
77013488
</td>
<td style="text-align:right;">
75962835
</td>
<td style="text-align:right;">
27439164
</td>
<td style="text-align:right;">
26415467
</td>
<td style="text-align:right;">
34.77420
</td>
<td style="text-align:right;">
18870010
</td>
<td style="text-align:right;">
10276972
</td>
<td style="text-align:right;">
5138486
</td>
</tr>
<tr>
<td style="text-align:left;">
79V3h
</td>
<td style="text-align:right;">
74863328
</td>
<td style="text-align:right;">
73508067
</td>
<td style="text-align:right;">
52535548
</td>
<td style="text-align:right;">
51196256
</td>
<td style="text-align:right;">
69.64713
</td>
<td style="text-align:right;">
40497298
</td>
<td style="text-align:right;">
25112288
</td>
<td style="text-align:right;">
12556144
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code> Ind4_reads_summary %&gt;% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to=&quot;type&quot;, values_to = &quot;reads&quot;) %&gt;% 
  dplyr::filter(type %in% list(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;, &quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)) %&gt;% 
  mutate(type=factor(type, levels=c(&quot;total_reads&quot;, &quot;total_mapped_reads&quot;, &quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;))) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;,sample)) %&gt;% 
  dplyr::filter (type != &quot;total_reads&quot;) %&gt;% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl(&quot;24h$&quot;, sample) ==TRUE, &quot;24_hours&quot;,&quot;3_hours&quot;)) %&gt;% 
   mutate(time=factor(time, levels=c(&quot;3_hours&quot;,&quot;24_hours&quot;))) %&gt;% 
  mutate(trt = case_match(trt,&quot;DAh&quot;~&quot;DNR&quot;,&quot;DXh&quot;~&quot;DOX&quot;,&quot;Eh&quot;~&quot;EPI&quot;, &quot;Mh&quot; ~ &quot;MTX&quot;, &quot;Th&quot; ~ &quot;TRZ&quot;, &quot;Vh&quot; ~&quot;VEH&quot;,.default = trt)) %&gt;% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position=&quot;dodge&quot;,aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Ind 4&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/Readssummary4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Readssummary4-1">
Past versions of Readssummary4-1.png
</button>
</p>
<div id="fig-Readssummary4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/Readssummary4-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/Readssummary4-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><em>note: reprocessed reads for Individual 4 vary from previous
output. This is due a file truncation that was detected in the process.
When the files were reprocessed, the truncated bam file was fixed,
resulting in a larger number of overall reads. Yay, I figured it
out!</em></p>
</div>
<div id="individual-5-read-summary" class="section level3">
<h3>Individual 5 read summary</h3>
<pre class="r"><code>Ind5_reads_summary &lt;- 
Ind5_summary %&gt;%                          
 separate(reads,into=c(&quot;reads&quot;,NA),sep= &quot; &quot;) %&gt;% 
   mutate(reads=as.numeric(reads)) %&gt;% 
   separate(mapped, into= c(&quot;mapped_reads&quot;, NA,NA, &quot;percent_mapped_reads&quot;,NA)) %&gt;% 
   separate(sample, into=c(&quot;one&quot;,&quot;two&quot;,NA, &quot;4&quot;,&quot;sample&quot;,NA,&quot;seven&quot;,&quot;8&quot;)) %&gt;% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %&gt;% 
   mutate(type = if_else(two==&quot;first&quot;, &quot;total&quot;,
                         if_else(two==&quot;noM&quot;,&quot;nuclear&quot;,
                                 if_else((seven == &quot;fin&quot;&amp; two==&quot;files&quot;), &quot;unique&quot;, &quot;dedup&quot;))))%&gt;% 
   dplyr::select(sample,type,  reads, mapped_reads)%&gt;% 
   pivot_longer(cols=reads:mapped_reads, names_to = &quot;read_info&quot;, values_to = &quot;reads&quot;) %&gt;% 
   unite(&quot;type&quot;,type:read_info, sep=&quot;_&quot;) %&gt;% 
   distinct() %&gt;% 
   pivot_wider(id_cols = sample, names_from = &quot;type&quot;, values_from = &quot;reads&quot;) %&gt;% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %&gt;% 
  mutate(dedup_read_pairs=dedup_reads/2) %&gt;% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind5_reads_summary %&gt;% 
   kable(., caption= &quot;Reads summary from Ind 5&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 18) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 18px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Reads summary from Ind 5
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
sample
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
per_nuclear_mapped
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unique_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_read_pairs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
78DA24h
</td>
<td style="text-align:right;">
93798644
</td>
<td style="text-align:right;">
93247559
</td>
<td style="text-align:right;">
40654316
</td>
<td style="text-align:right;">
40144022
</td>
<td style="text-align:right;">
43.05102
</td>
<td style="text-align:right;">
29016844
</td>
<td style="text-align:right;">
15985676
</td>
<td style="text-align:right;">
7992838
</td>
</tr>
<tr>
<td style="text-align:left;">
78DA3h
</td>
<td style="text-align:right;">
99686934
</td>
<td style="text-align:right;">
99001020
</td>
<td style="text-align:right;">
39593518
</td>
<td style="text-align:right;">
38953971
</td>
<td style="text-align:right;">
39.34704
</td>
<td style="text-align:right;">
27391084
</td>
<td style="text-align:right;">
19301222
</td>
<td style="text-align:right;">
9650611
</td>
</tr>
<tr>
<td style="text-align:left;">
78DX24h
</td>
<td style="text-align:right;">
73732812
</td>
<td style="text-align:right;">
73320774
</td>
<td style="text-align:right;">
28057954
</td>
<td style="text-align:right;">
27671754
</td>
<td style="text-align:right;">
37.74067
</td>
<td style="text-align:right;">
19662022
</td>
<td style="text-align:right;">
13718342
</td>
<td style="text-align:right;">
6859171
</td>
</tr>
<tr>
<td style="text-align:left;">
78DX3h
</td>
<td style="text-align:right;">
100368976
</td>
<td style="text-align:right;">
99588099
</td>
<td style="text-align:right;">
38244760
</td>
<td style="text-align:right;">
37513055
</td>
<td style="text-align:right;">
37.66821
</td>
<td style="text-align:right;">
25968162
</td>
<td style="text-align:right;">
16753844
</td>
<td style="text-align:right;">
8376922
</td>
</tr>
<tr>
<td style="text-align:left;">
78E24h
</td>
<td style="text-align:right;">
80101516
</td>
<td style="text-align:right;">
79632167
</td>
<td style="text-align:right;">
31905942
</td>
<td style="text-align:right;">
31467541
</td>
<td style="text-align:right;">
39.51612
</td>
<td style="text-align:right;">
22102498
</td>
<td style="text-align:right;">
16872182
</td>
<td style="text-align:right;">
8436091
</td>
</tr>
<tr>
<td style="text-align:left;">
78E3h
</td>
<td style="text-align:right;">
91523420
</td>
<td style="text-align:right;">
90956218
</td>
<td style="text-align:right;">
32894252
</td>
<td style="text-align:right;">
32371141
</td>
<td style="text-align:right;">
35.58981
</td>
<td style="text-align:right;">
22531754
</td>
<td style="text-align:right;">
15019734
</td>
<td style="text-align:right;">
7509867
</td>
</tr>
<tr>
<td style="text-align:left;">
78M24h
</td>
<td style="text-align:right;">
94307714
</td>
<td style="text-align:right;">
93638536
</td>
<td style="text-align:right;">
48887176
</td>
<td style="text-align:right;">
48253791
</td>
<td style="text-align:right;">
51.53198
</td>
<td style="text-align:right;">
37330604
</td>
<td style="text-align:right;">
21133380
</td>
<td style="text-align:right;">
10566690
</td>
</tr>
<tr>
<td style="text-align:left;">
78M3h
</td>
<td style="text-align:right;">
71296912
</td>
<td style="text-align:right;">
70156341
</td>
<td style="text-align:right;">
34691864
</td>
<td style="text-align:right;">
33569035
</td>
<td style="text-align:right;">
47.84890
</td>
<td style="text-align:right;">
26212720
</td>
<td style="text-align:right;">
16100264
</td>
<td style="text-align:right;">
8050132
</td>
</tr>
<tr>
<td style="text-align:left;">
78T24h
</td>
<td style="text-align:right;">
88253908
</td>
<td style="text-align:right;">
87620321
</td>
<td style="text-align:right;">
36281030
</td>
<td style="text-align:right;">
35687616
</td>
<td style="text-align:right;">
40.72984
</td>
<td style="text-align:right;">
25942984
</td>
<td style="text-align:right;">
14776178
</td>
<td style="text-align:right;">
7388089
</td>
</tr>
<tr>
<td style="text-align:left;">
78T3h
</td>
<td style="text-align:right;">
76629092
</td>
<td style="text-align:right;">
76156525
</td>
<td style="text-align:right;">
27160272
</td>
<td style="text-align:right;">
26724144
</td>
<td style="text-align:right;">
35.09108
</td>
<td style="text-align:right;">
19214446
</td>
<td style="text-align:right;">
10123244
</td>
<td style="text-align:right;">
5061622
</td>
</tr>
<tr>
<td style="text-align:left;">
78V24h
</td>
<td style="text-align:right;">
99083200
</td>
<td style="text-align:right;">
98184830
</td>
<td style="text-align:right;">
32756682
</td>
<td style="text-align:right;">
31912089
</td>
<td style="text-align:right;">
32.50206
</td>
<td style="text-align:right;">
21660782
</td>
<td style="text-align:right;">
10088962
</td>
<td style="text-align:right;">
5044481
</td>
</tr>
<tr>
<td style="text-align:left;">
78V3h
</td>
<td style="text-align:right;">
82686104
</td>
<td style="text-align:right;">
82045556
</td>
<td style="text-align:right;">
32865770
</td>
<td style="text-align:right;">
32260193
</td>
<td style="text-align:right;">
39.31985
</td>
<td style="text-align:right;">
22506718
</td>
<td style="text-align:right;">
11067518
</td>
<td style="text-align:right;">
5533759
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code> Ind5_reads_summary %&gt;% 
   pivot_longer(cols = total_reads:dedup_read_pairs, names_to=&quot;type&quot;, values_to = &quot;reads&quot;) %&gt;% 
  dplyr::filter(type %in% list(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;, &quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)) %&gt;% 
  mutate(type=factor(type, levels=c(&quot;total_reads&quot;, &quot;total_mapped_reads&quot;, &quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;))) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;,sample)) %&gt;% 
  dplyr::filter (type != &quot;total_reads&quot;) %&gt;% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl(&quot;24h$&quot;, sample) ==TRUE, &quot;24_hours&quot;,&quot;3_hours&quot;)) %&gt;% 
   mutate(time=factor(time, levels=c(&quot;3_hours&quot;,&quot;24_hours&quot;))) %&gt;% 
  mutate(trt = case_match(trt,&quot;DAh&quot;~&quot;DNR&quot;,&quot;DXh&quot;~&quot;DOX&quot;,&quot;Eh&quot;~&quot;EPI&quot;, &quot;Mh&quot; ~ &quot;MTX&quot;, &quot;Th&quot; ~ &quot;TRZ&quot;, &quot;Vh&quot; ~&quot;VEH&quot;,.default = trt)) %&gt;% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position=&quot;dodge&quot;,aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Ind 5&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/Readssummary5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Readssummary5-1">
Past versions of Readssummary5-1.png
</button>
</p>
<div id="fig-Readssummary5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/Readssummary5-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/Readssummary5-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="individual-6-read-summary" class="section level3">
<h3>Individual 6 read summary</h3>
<pre class="r"><code>Ind6_reads_summary &lt;- 
Ind6_summary %&gt;%                          
 separate(reads,into=c(&quot;reads&quot;,NA),sep= &quot; &quot;) %&gt;% 
   mutate(reads=as.numeric(reads)) %&gt;% 
   separate(mapped, into= c(&quot;mapped_reads&quot;, NA,NA, &quot;percent_mapped_reads&quot;,NA)) %&gt;% 
   separate(sample, into=c(&quot;one&quot;,&quot;two&quot;,NA, &quot;4&quot;,&quot;sample&quot;,NA,&quot;seven&quot;,&quot;8&quot;)) %&gt;% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %&gt;% 
   mutate(type = if_else(two==&quot;first&quot;, &quot;total&quot;,
                         if_else(two==&quot;noM&quot;,&quot;nuclear&quot;,
                                 if_else((seven == &quot;fin&quot;&amp; two==&quot;files&quot;), &quot;unique&quot;, &quot;dedup&quot;))))%&gt;% 
   dplyr::select(sample,type,  reads, mapped_reads)%&gt;% 
   pivot_longer(cols=reads:mapped_reads, names_to = &quot;read_info&quot;, values_to = &quot;reads&quot;) %&gt;% 
   unite(&quot;type&quot;,type:read_info, sep=&quot;_&quot;) %&gt;% 
   distinct() %&gt;% 
   pivot_wider(id_cols = sample, names_from = &quot;type&quot;, values_from = &quot;reads&quot;) %&gt;% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %&gt;% 
  mutate(dedup_read_pairs=dedup_reads/2) %&gt;% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind6_reads_summary %&gt;% 
   kable(., caption= &quot;Reads summary from Ind 6&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(full_width = FALSE,font_size = 18) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; font-size: 18px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Reads summary from Ind 6
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
sample
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
nuclear_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
per_nuclear_mapped
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
unique_mapped_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_reads
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
dedup_read_pairs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
71DA24h
</td>
<td style="text-align:right;">
82118778
</td>
<td style="text-align:right;">
81809775
</td>
<td style="text-align:right;">
43159318
</td>
<td style="text-align:right;">
42879967
</td>
<td style="text-align:right;">
52.41423
</td>
<td style="text-align:right;">
33579444
</td>
<td style="text-align:right;">
25938306
</td>
<td style="text-align:right;">
12969153
</td>
</tr>
<tr>
<td style="text-align:left;">
71DA3h
</td>
<td style="text-align:right;">
88190684
</td>
<td style="text-align:right;">
87826538
</td>
<td style="text-align:right;">
48938930
</td>
<td style="text-align:right;">
48604831
</td>
<td style="text-align:right;">
55.34185
</td>
<td style="text-align:right;">
37469828
</td>
<td style="text-align:right;">
29611060
</td>
<td style="text-align:right;">
14805530
</td>
</tr>
<tr>
<td style="text-align:left;">
71DX24h
</td>
<td style="text-align:right;">
75155380
</td>
<td style="text-align:right;">
74868610
</td>
<td style="text-align:right;">
31677782
</td>
<td style="text-align:right;">
31423548
</td>
<td style="text-align:right;">
41.97159
</td>
<td style="text-align:right;">
22967970
</td>
<td style="text-align:right;">
18070632
</td>
<td style="text-align:right;">
9035316
</td>
</tr>
<tr>
<td style="text-align:left;">
71DX3h
</td>
<td style="text-align:right;">
90235848
</td>
<td style="text-align:right;">
89816458
</td>
<td style="text-align:right;">
42018780
</td>
<td style="text-align:right;">
41633173
</td>
<td style="text-align:right;">
46.35361
</td>
<td style="text-align:right;">
31108924
</td>
<td style="text-align:right;">
23632420
</td>
<td style="text-align:right;">
11816210
</td>
</tr>
<tr>
<td style="text-align:left;">
71E24h
</td>
<td style="text-align:right;">
86730082
</td>
<td style="text-align:right;">
86449747
</td>
<td style="text-align:right;">
37387316
</td>
<td style="text-align:right;">
37144236
</td>
<td style="text-align:right;">
42.96628
</td>
<td style="text-align:right;">
28138936
</td>
<td style="text-align:right;">
17335200
</td>
<td style="text-align:right;">
8667600
</td>
</tr>
<tr>
<td style="text-align:left;">
71E3h
</td>
<td style="text-align:right;">
87292426
</td>
<td style="text-align:right;">
86821698
</td>
<td style="text-align:right;">
44283140
</td>
<td style="text-align:right;">
43845617
</td>
<td style="text-align:right;">
50.50076
</td>
<td style="text-align:right;">
32917338
</td>
<td style="text-align:right;">
25378234
</td>
<td style="text-align:right;">
12689117
</td>
</tr>
<tr>
<td style="text-align:left;">
71M24h
</td>
<td style="text-align:right;">
82011248
</td>
<td style="text-align:right;">
81628718
</td>
<td style="text-align:right;">
45399152
</td>
<td style="text-align:right;">
45045409
</td>
<td style="text-align:right;">
55.18329
</td>
<td style="text-align:right;">
34342106
</td>
<td style="text-align:right;">
26140848
</td>
<td style="text-align:right;">
13070424
</td>
</tr>
<tr>
<td style="text-align:left;">
71M3h
</td>
<td style="text-align:right;">
90916110
</td>
<td style="text-align:right;">
90521870
</td>
<td style="text-align:right;">
42962278
</td>
<td style="text-align:right;">
42602678
</td>
<td style="text-align:right;">
47.06341
</td>
<td style="text-align:right;">
32093512
</td>
<td style="text-align:right;">
23624888
</td>
<td style="text-align:right;">
11812444
</td>
</tr>
<tr>
<td style="text-align:left;">
71T24h
</td>
<td style="text-align:right;">
71401066
</td>
<td style="text-align:right;">
71211530
</td>
<td style="text-align:right;">
30697278
</td>
<td style="text-align:right;">
30530941
</td>
<td style="text-align:right;">
42.87359
</td>
<td style="text-align:right;">
22901380
</td>
<td style="text-align:right;">
17278728
</td>
<td style="text-align:right;">
8639364
</td>
</tr>
<tr>
<td style="text-align:left;">
71T3h
</td>
<td style="text-align:right;">
71750048
</td>
<td style="text-align:right;">
71481397
</td>
<td style="text-align:right;">
34683166
</td>
<td style="text-align:right;">
34436567
</td>
<td style="text-align:right;">
48.17557
</td>
<td style="text-align:right;">
25868334
</td>
<td style="text-align:right;">
20269094
</td>
<td style="text-align:right;">
10134547
</td>
</tr>
<tr>
<td style="text-align:left;">
71V24h
</td>
<td style="text-align:right;">
79633316
</td>
<td style="text-align:right;">
79337448
</td>
<td style="text-align:right;">
37768500
</td>
<td style="text-align:right;">
37501106
</td>
<td style="text-align:right;">
47.26785
</td>
<td style="text-align:right;">
28483866
</td>
<td style="text-align:right;">
20922744
</td>
<td style="text-align:right;">
10461372
</td>
</tr>
<tr>
<td style="text-align:left;">
71V3h
</td>
<td style="text-align:right;">
85216880
</td>
<td style="text-align:right;">
84814985
</td>
<td style="text-align:right;">
35805146
</td>
<td style="text-align:right;">
35439842
</td>
<td style="text-align:right;">
41.78488
</td>
<td style="text-align:right;">
25807810
</td>
<td style="text-align:right;">
19449074
</td>
<td style="text-align:right;">
9724537
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code> Ind6_reads_summary %&gt;% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to=&quot;type&quot;, values_to = &quot;reads&quot;) %&gt;% 
  dplyr::filter(type %in% list(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;, &quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)) %&gt;% 
  mutate(type=factor(type, levels=c(&quot;total_reads&quot;, &quot;total_mapped_reads&quot;, &quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;))) %&gt;% 
  mutate(trt=gsub(&quot;[[:digit:]]&quot;, &quot;&quot;,sample)) %&gt;% 
  dplyr::filter (type != &quot;total_reads&quot;) %&gt;% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl(&quot;24h$&quot;, sample) ==TRUE, &quot;24_hours&quot;,&quot;3_hours&quot;)) %&gt;% 
   mutate(time=factor(time, levels=c(&quot;3_hours&quot;,&quot;24_hours&quot;))) %&gt;% 
  mutate(trt = case_match(trt,&quot;DAh&quot;~&quot;DNR&quot;,&quot;DXh&quot;~&quot;DOX&quot;,&quot;Eh&quot;~&quot;EPI&quot;, &quot;Mh&quot; ~ &quot;MTX&quot;, &quot;Th&quot; ~ &quot;TRZ&quot;, &quot;Vh&quot; ~&quot;VEH&quot;,.default = trt)) %&gt;% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position=&quot;dodge&quot;,aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle(&quot;Ind 6&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/Readssummary6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Readssummary6-1">
Past versions of Readssummary6-1.png
</button>
</p>
<div id="fig-Readssummary6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/Readssummary6-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/de853f8b60001dc9abf34c4080e57bc30c369198/docs/figure/Fastqc_results.Rmd/Readssummary6-1.png" target="_blank">de853f8</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="total-reads-summary" class="section level3">
<h3>Total reads summary</h3>
<pre class="r"><code>total_reads_summary &lt;- Ind1_reads_summary %&gt;% 
 mutate(sample=gsub(&quot;75&quot;,&quot;1_&quot;,sample)) %&gt;% 
  rbind((Ind2_reads_summary %&gt;%
           mutate(sample=gsub(&quot;87&quot;,&quot;2_&quot;,sample)))) %&gt;% 
  rbind((Ind3_reads_summary %&gt;%
           mutate(sample=gsub(&quot;77&quot;,&quot;3_&quot;,sample)))) %&gt;% 
  rbind((Ind4_reads_summary %&gt;%
           mutate(sample=gsub(&quot;79&quot;,&quot;4_&quot;,sample)))) %&gt;% 
  rbind((Ind5_reads_summary %&gt;%
           mutate(sample=gsub(&quot;78&quot;,&quot;5_&quot;,sample)))) %&gt;% 
  rbind((Ind6_reads_summary %&gt;%
           mutate(sample=gsub(&quot;71&quot;,&quot;6_&quot;,sample)))) %&gt;% 
  mutate(sample = gsub(&quot;24h&quot;,&quot;_24h&quot;,sample), 
       sample = gsub(&quot;3h&quot;,&quot;_3h&quot;,sample)) %&gt;%
  separate(sample, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels=c(&quot;DX&quot;,&quot;E&quot;,&quot;DA&quot;,&quot;M&quot;,&quot;T&quot;,&quot;V&quot;), labels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;)))
         
total_reads_summary %&gt;% 
  ggplot(., aes(x=trt, y=total_reads ))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;total number of reads by time and treatment&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/readsummary%201-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-readsummary-1-1">
Past versions of "readsummary 1-1.png"
</button>
</p>
<div id="fig-no-spaces-readsummary-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/readsummary 1-1.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>total_reads_summary %&gt;% 
  ggplot(., aes(x=trt, y=nuclear_reads ))+
  geom_boxplot(aes(fill=trt))+
 geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;total number of nuclear reads by time and treatment&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/readsummary%201-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-readsummary-1-2">
Past versions of "readsummary 1-2.png"
</button>
</p>
<div id="fig-no-spaces-readsummary-1-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/readsummary 1-2.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>total_reads_summary %&gt;% 
  ggplot(., aes(x=trt, y=unique_mapped_reads ))+
  geom_boxplot(aes(fill=trt))+
 geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;total number of unique-nuclear reads by time and treatment&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/readsummary%201-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-readsummary-1-3">
Past versions of "readsummary 1-3.png"
</button>
</p>
<div id="fig-no-spaces-readsummary-1-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/readsummary 1-3.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>total_reads_summary %&gt;% 
  ggplot(., aes(x=trt, y=dedup_reads ))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;total unique, deduplicated nuclear mapped reads by time and treatment&quot;)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/readsummary%201-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-readsummary-1-4">
Past versions of "readsummary 1-4.png"
</button>
</p>
<div id="fig-no-spaces-readsummary-1-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/d4db64b2acb47f8fb1e8f973c61b26ebcbdf6db6/docs/figure/Fastqc_results.Rmd/readsummary 1-4.png" target="_blank">d4db64b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>total_reads_summary %&gt;% 
  pivot_longer(., cols=c(total_mapped_reads,nuclear_mapped_reads,unique_mapped_reads,dedup_reads), names_to=&quot;plotting_data&quot;, values_to = &quot;counts&quot;) %&gt;% 
  dplyr::filter(indv !=&quot;4&quot;) %&gt;% 
  dplyr::filter(indv !=&quot;5&quot;) %&gt;% 
  mutate(plotting_data=factor(plotting_data, levels =c(&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)))  %&gt;% 
  ggplot(., aes(x=plotting_data, y=counts))+
  geom_boxplot(aes(fill=indv))+
  theme_bw() +
 scale_fill_brewer(palette = &quot;Dark2&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  
  facet_wrap(~time)</code></pre>
<p><img src="figure/Fastqc_results.Rmd/total%20reads%20adding%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>total_reads_summary %&gt;% 
  pivot_longer(., cols=c(total_mapped_reads,nuclear_mapped_reads,unique_mapped_reads,dedup_reads), names_to=&quot;plotting_data&quot;, values_to = &quot;counts&quot;) %&gt;% 
  dplyr::filter(indv !=&quot;4&quot;) %&gt;% 
  dplyr::filter(indv !=&quot;5&quot;) %&gt;% 
  mutate(plotting_data=factor(plotting_data, levels =c(&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;dedup_reads&quot;)))  %&gt;% 
  ggplot(., aes(x=trt, y=dedup_read_pairs))+
  geom_boxplot(aes(fill=trt))+
 geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;Deduplicated read pairs&quot;)+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  
  theme_bw()</code></pre>
<p><img src="figure/Fastqc_results.Rmd/total%20reads%20adding%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>total_reads_summary %&gt;% 
    dplyr::filter(indv !=&quot;4&quot;) %&gt;% 
  dplyr::filter(indv !=&quot;5&quot;) %&gt;% 
 summary()</code></pre>
<pre><code>     indv            trt     time     total_reads        total_mapped_reads 
 Length:48          DOX:8   3h :24   Min.   : 62649778   Min.   : 61652700  
 Class :character   EPI:8   24h:24   1st Qu.: 77486170   1st Qu.: 77014769  
 Mode  :character   DNR:8            Median : 85159663   Median : 84074506  
                    MTX:8            Mean   : 84763764   Mean   : 84017027  
                    TRZ:8            3rd Qu.: 89838872   3rd Qu.: 89260506  
                    VEH:8            Max.   :142295378   Max.   :141426527  
 nuclear_reads      nuclear_mapped_reads per_nuclear_mapped unique_mapped_reads
 Min.   :27982800   Min.   :27608032     Min.   :35.79      Min.   :20240194   
 1st Qu.:37603115   1st Qu.:37195343     1st Qu.:46.13      1st Qu.:28397634   
 Median :43486883   Median :42935432     Median :52.63      Median :33288042   
 Mean   :46388144   Mean   :45674820     Mean   :54.43      Mean   :35359491   
 3rd Qu.:56130091   3rd Qu.:55529854     3rd Qu.:62.40      3rd Qu.:42527946   
 Max.   :70021308   Max.   :69490912     Max.   :77.20      Max.   :57350454   
  dedup_reads       dedup_read_pairs  
 Min.   : 7986090   Min.   : 3993045  
 1st Qu.:20064089   1st Qu.:10032044  
 Median :25421735   Median :12710868  
 Mean   :26339644   Mean   :13169822  
 3rd Qu.:32517936   3rd Qu.:16258968  
 Max.   :42171858   Max.   :21085929  </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] RColorBrewer_1.1-3 kableExtra_1.4.0   lubridate_1.9.4    forcats_1.0.0     
 [5] stringr_1.5.1      dplyr_1.1.4        purrr_1.0.4        readr_2.1.5       
 [9] tidyr_1.3.1        tibble_3.2.1       ggplot2_3.5.1      tidyverse_2.0.0   
[13] workflowr_1.7.1   

loaded via a namespace (and not attached):
 [1] gtable_0.3.6      xfun_0.51         bslib_0.9.0       processx_3.8.6   
 [5] callr_3.7.6       tzdb_0.4.0        vctrs_0.6.5       tools_4.4.2      
 [9] ps_1.9.0          generics_0.1.3    parallel_4.4.2    pkgconfig_2.0.3  
[13] lifecycle_1.0.4   farver_2.1.2      compiler_4.4.2    git2r_0.35.0     
[17] munsell_0.5.1     getPass_0.2-4     httpuv_1.6.15     htmltools_0.5.8.1
[21] sass_0.4.9        yaml_2.3.10       later_1.4.1       pillar_1.10.1    
[25] crayon_1.5.3      jquerylib_0.1.4   whisker_0.4.1     cachem_1.1.0     
[29] tidyselect_1.2.1  digest_0.6.37     stringi_1.8.4     labeling_0.4.3   
[33] rprojroot_2.0.4   fastmap_1.2.0     grid_4.4.2        colorspace_2.1-1 
[37] cli_3.6.4         magrittr_2.0.3    withr_3.0.2       scales_1.3.0     
[41] promises_1.3.2    bit64_4.6.0-1     timechange_0.3.0  rmarkdown_2.29   
[45] httr_1.4.7        bit_4.6.0         hms_1.1.3         evaluate_1.0.3   
[49] knitr_1.49        viridisLite_0.4.2 rlang_1.1.5       Rcpp_1.0.14      
[53] glue_1.8.0        xml2_1.3.7        svglite_2.1.3     rstudioapi_0.17.1
[57] vroom_1.6.5       jsonlite_1.9.1    R6_2.6.1          systemfonts_1.2.1
[61] fs_1.6.5         </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
