<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-10-07" />

<title>GO and KEGG pathway analysis</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">GO and KEGG pathway analysis</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-10-07</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-10-07
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree2f0dc7bfb29cbe1fa6f27a0d2def3e0db2252b92targetblank2f0dc7ba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/2f0dc7bfb29cbe1fa6f27a0d2def3e0db2252b92" target="_blank">2f0dc7b</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree2f0dc7bfb29cbe1fa6f27a0d2def3e0db2252b92targetblank2f0dc7ba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/2f0dc7bfb29cbe1fa6f27a0d2def3e0db2252b92" target="_blank">2f0dc7b</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/output_100_MA0835.2.txt
    Ignored:    data/output_101_MA0018.4.txt
    Ignored:    data/output_102_MA1102.2.txt
    Ignored:    data/output_103_MA0162.4.txt
    Ignored:    data/output_104_MA0598.3.txt
    Ignored:    data/output_105_MA0473.3.txt
    Ignored:    data/output_106_MA0640.2.txt
    Ignored:    data/output_107_MA0761.2.txt
    Ignored:    data/output_108_MA0477.2.txt
    Ignored:    data/output_109_MA0062.3.txt
    Ignored:    data/output_10_MA0076.2.txt
    Ignored:    data/output_110_MA0035.4.txt
    Ignored:    data/output_111_MA0482.2.txt
    Ignored:    data/output_112_MA1104.2.txt
    Ignored:    data/output_113_MA0490.2.txt
    Ignored:    data/output_114_MA0491.2.txt
    Ignored:    data/output_115_MA1107.2.txt
    Ignored:    data/output_116_MA0496.3.txt
    Ignored:    data/output_117_MA0620.3.txt
    Ignored:    data/output_118_MA0500.2.txt
    Ignored:    data/output_119_MA0089.2.txt
    Ignored:    data/output_11_MA0258.2.txt
    Ignored:    data/output_120_MA0782.2.txt
    Ignored:    data/output_121_MA0090.3.txt
    Ignored:    data/output_122_MA0809.2.txt
    Ignored:    data/output_123_MA0093.3.txt
    Ignored:    data/output_124_MA0609.2.txt
    Ignored:    data/output_125_MA1684.1.txt
    Ignored:    data/output_126_MA1928.1.txt
    Ignored:    data/output_127_MA1929.1.txt
    Ignored:    data/output_128_MA1930.1.txt
    Ignored:    data/output_129_MA1934.1.txt
    Ignored:    data/output_12_MA0150.2.txt
    Ignored:    data/output_130_MA1940.1.txt
    Ignored:    data/output_131_MA1942.1.txt
    Ignored:    data/output_132_MA1944.1.txt
    Ignored:    data/output_133_MA1951.1.txt
    Ignored:    data/output_134_MA1959.1.txt
    Ignored:    data/output_135_MA1964.1.txt
    Ignored:    data/output_136_MA1970.1.txt
    Ignored:    data/output_137_MA1988.1.txt
    Ignored:    data/output_138_MA1992.1.txt
    Ignored:    data/output_139_MA1993.1.txt
    Ignored:    data/output_13_MA0140.2.txt
    Ignored:    data/output_140_MA1995.1.txt
    Ignored:    data/output_141_MA1997.1.txt
    Ignored:    data/output_142_MA2002.1.txt
    Ignored:    data/output_143_MA0037.4.txt
    Ignored:    data/output_144_MA0156.3.txt
    Ignored:    data/output_145_MA0474.3.txt
    Ignored:    data/output_146_MA0489.2.txt
    Ignored:    data/output_147_MA0493.2.txt
    Ignored:    data/output_148_MA0516.3.txt
    Ignored:    data/output_149_MA0521.2.txt
    Ignored:    data/output_14_MA0591.1.txt
    Ignored:    data/output_150_MA0633.2.txt
    Ignored:    data/output_151_MA0659.3.txt
    Ignored:    data/output_152_MA0685.2.txt
    Ignored:    data/output_153_MA0697.2.txt
    Ignored:    data/output_154_MA0740.2.txt
    Ignored:    data/output_155_MA0742.2.txt
    Ignored:    data/output_156_MA0764.3.txt
    Ignored:    data/output_157_MA0831.3.txt
    Ignored:    data/output_158_MA1472.2.txt
    Ignored:    data/output_159_MA1483.2.txt
    Ignored:    data/output_15_MA0595.1.txt
    Ignored:    data/output_160_MA1511.2.txt
    Ignored:    data/output_161_MA1633.2.txt
    Ignored:    data/output_16_MA0596.1.txt
    Ignored:    data/output_17_MA0599.1.txt
    Ignored:    data/output_18_MA0603.1.txt
    Ignored:    data/output_19_MA0645.1.txt
    Ignored:    data/output_1_MA0029.1.txt
    Ignored:    data/output_1_Mecom.txt
    Ignored:    data/output_20_MA0475.2.txt
    Ignored:    data/output_21_MA0655.1.txt
    Ignored:    data/output_22_MA0656.1.txt
    Ignored:    data/output_23_MA0660.1.txt
    Ignored:    data/output_24_MA0741.1.txt
    Ignored:    data/output_25_MA0747.1.txt
    Ignored:    data/output_26_MA0751.1.txt
    Ignored:    data/output_27_MA0760.1.txt
    Ignored:    data/output_28_MA0098.3.txt
    Ignored:    data/output_29_MA0762.1.txt
    Ignored:    data/output_2_MA0119.1.txt
    Ignored:    data/output_30_MA0498.2.txt
    Ignored:    data/output_31_MA0774.1.txt
    Ignored:    data/output_32_MA0775.1.txt
    Ignored:    data/output_33_MA0783.1.txt
    Ignored:    data/output_34_MA0797.1.txt
    Ignored:    data/output_35_MA0510.2.txt
    Ignored:    data/output_36_MA0808.1.txt
    Ignored:    data/output_37_MA0816.1.txt
    Ignored:    data/output_38_MA0840.1.txt
    Ignored:    data/output_39_MA0117.2.txt
    Ignored:    data/output_3_MA0139.1.txt
    Ignored:    data/output_40_MA0841.1.txt
    Ignored:    data/output_41_MA0036.3.txt
    Ignored:    data/output_42_MA1114.1.txt
    Ignored:    data/output_43_MA1121.1.txt
    Ignored:    data/output_44_MA0750.2.txt
    Ignored:    data/output_45_MA0099.3.txt
    Ignored:    data/output_46_MA1126.1.txt
    Ignored:    data/output_47_MA1128.1.txt
    Ignored:    data/output_48_MA1129.1.txt
    Ignored:    data/output_49_MA1130.1.txt
    Ignored:    data/output_4_MA0145.2.txt
    Ignored:    data/output_50_MA1131.1.txt
    Ignored:    data/output_51_MA1132.1.txt
    Ignored:    data/output_52_MA1133.1.txt
    Ignored:    data/output_53_MA1134.1.txt
    Ignored:    data/output_54_MA1135.1.txt
    Ignored:    data/output_55_MA1136.1.txt
    Ignored:    data/output_56_MA1137.1.txt
    Ignored:    data/output_57_MA1138.1.txt
    Ignored:    data/output_58_MA1139.1.txt
    Ignored:    data/output_59_MA1141.1.txt
    Ignored:    data/output_5_MA0476.1.txt
    Ignored:    data/output_60_MA1142.1.txt
    Ignored:    data/output_61_MA1143.1.txt
    Ignored:    data/output_62_MA1144.1.txt
    Ignored:    data/output_63_MA1145.1.txt
    Ignored:    data/output_64_MA1101.2.txt
    Ignored:    data/output_65_MA1474.1.txt
    Ignored:    data/output_66_MA1475.1.txt
    Ignored:    data/output_67_MA1484.1.txt
    Ignored:    data/output_68_MA1512.1.txt
    Ignored:    data/output_69_MA1516.1.txt
    Ignored:    data/output_6_MA0478.1.txt
    Ignored:    data/output_70_MA1517.1.txt
    Ignored:    data/output_71_MA1520.1.txt
    Ignored:    data/output_72_MA1521.1.txt
    Ignored:    data/output_73_MA1523.1.txt
    Ignored:    data/output_74_MA1527.1.txt
    Ignored:    data/output_75_MA1528.1.txt
    Ignored:    data/output_76_MA1564.1.txt
    Ignored:    data/output_77_MA1571.1.txt
    Ignored:    data/output_78_MA1572.1.txt
    Ignored:    data/output_79_MA1593.1.txt
    Ignored:    data/output_7_MA0488.1.txt
    Ignored:    data/output_80_MA1599.1.txt
    Ignored:    data/output_81_MA1628.1.txt
    Ignored:    data/output_82_MA1629.1.txt
    Ignored:    data/output_83_MA1100.2.txt
    Ignored:    data/output_84_MA0462.2.txt
    Ignored:    data/output_85_MA0766.2.txt
    Ignored:    data/output_86_MA1140.2.txt
    Ignored:    data/output_87_MA0842.2.txt
    Ignored:    data/output_88_MA0746.2.txt
    Ignored:    data/output_89_MA1713.1.txt
    Ignored:    data/output_8_MA0492.1.txt
    Ignored:    data/output_90_MA1721.1.txt
    Ignored:    data/output_91_MA1632.1.txt
    Ignored:    data/output_92_MA1634.1.txt
    Ignored:    data/output_93_MA1635.1.txt
    Ignored:    data/output_94_MA1641.1.txt
    Ignored:    data/output_95_MA1643.1.txt
    Ignored:    data/output_96_MA1645.1.txt
    Ignored:    data/output_97_MA1656.1.txt
    Ignored:    data/output_98_MA1726.1.txt
    Ignored:    data/output_99_MA1728.1.txt
    Ignored:    data/output_9_MA0501.1.txt
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  Correlationplot_scaled.pdf
    Untracked:  DOX_DAR_assess.Rmd
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/ReHeat_analysis.Rmd
    Untracked:  analysis/TE_analysis_old.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/MRC_clusterlog2cpm.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  output_1_Mecom.txt
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/CorMotif_data_n45.Rmd
    Modified:   analysis/Enhancer_files_ff.Rmd
    Modified:   analysis/Enrichment_motif.Rmd
    Modified:   analysis/Jaspar_motif_ff.Rmd
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Raodah.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd
    Modified:   analysis/TE_analysis.Rmd
    Modified:   analysis/final_four_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/GO_KEGG_analysis.Rmd</code>) and
HTML (<code>docs/GO_KEGG_analysis.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/2f0dc7bfb29cbe1fa6f27a0d2def3e0db2252b92/analysis/GO_KEGG_analysis.Rmd" target="_blank">2f0dc7b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
adding pdf
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/GO_KEGG_analysis.html" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/19b7e2b46dde3e905e5285c7265b322f73f478e6/analysis/GO_KEGG_analysis.Rmd" target="_blank">19b7e2b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
adding pdf
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/GO_KEGG_analysis.html" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/df3bc4e9832b58bf4f48c82cb0876aa657471244/analysis/GO_KEGG_analysis.Rmd" target="_blank">df3bc4e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
updated group of eight
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ec629e184d193d035c76231eefda455868047fb0/analysis/GO_KEGG_analysis.Rmd" target="_blank">ec629e1</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-20
</td>
<td>
before adding in 2kb and KEGG filters to data, removing all none-useful
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/128c4f731ab5119b81b2f8daa22b95842fd8e7a0/analysis/GO_KEGG_analysis.Rmd" target="_blank">128c4f7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-20
</td>
<td>
before adding in 2kb and KEGG filters to data, removing all none-useful
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1d8660b296e524e2a3b0b45385bf082a51d9a199/analysis/GO_KEGG_analysis.Rmd" target="_blank">1d8660b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-19
</td>
<td>
updates to code
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/55dcc724bd763db8d0c1330b1b91c4e2531e7e7b/analysis/GO_KEGG_analysis.Rmd" target="_blank">55dcc72</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-06
</td>
<td>
using median lfc update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/3115a559d23530cfb44f1cbaa0d6576725bf257c/docs/GO_KEGG_analysis.html" target="_blank">3115a55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5193b51ba1e3bac53956a7cb6423096352d07cb0/analysis/GO_KEGG_analysis.Rmd" target="_blank">5193b51</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-05
</td>
<td>
updates to barcharts
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/1933f92ec3f5c96b47380f7a65555d2351e88bfd/docs/GO_KEGG_analysis.html" target="_blank">1933f92</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50cc96f71825164ace9bfe928fd2628ecd88598b/analysis/GO_KEGG_analysis.Rmd" target="_blank">50cc96f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-02
</td>
<td>
updates using new guidelines
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5d5401708c50e87c3abf2b4c7e2a6b5acb142b07/analysis/GO_KEGG_analysis.Rmd" target="_blank">5d54017</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-12
</td>
<td>
adding commit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/bd3317516fa2261ca011fbddd3d00cee03c9d3a3/docs/GO_KEGG_analysis.html" target="_blank">bd33175</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5e8a1e4a0ce08989bdb89191a23a368c1ca07bf/analysis/GO_KEGG_analysis.Rmd" target="_blank">b5e8a1e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-07
</td>
<td>
updates and new analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/021ca5f348a08afdb373698776ccf34cb3423d6d/docs/GO_KEGG_analysis.html" target="_blank">021ca5f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/95e041556424446e7e9d42fb09014ab7cdebbae6/analysis/GO_KEGG_analysis.Rmd" target="_blank">95e0415</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-07
</td>
<td>
updates, i think
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/eea09763a0a28f3cd9406245fd356e92c7743a12/docs/GO_KEGG_analysis.html" target="_blank">eea0976</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0672654e63bcc949adbb430a1eee3f455ff7be3e/analysis/GO_KEGG_analysis.Rmd" target="_blank">0672654</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-03
</td>
<td>
updates to gene lists
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/897af2369a6cba17a7ff5781af490c9924d0ec2b/docs/GO_KEGG_analysis.html" target="_blank">897af23</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/5c6ef838be394b8e520989b052b285bb88be246a/analysis/GO_KEGG_analysis.Rmd" target="_blank">5c6ef83</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-28
</td>
<td>
updates to KEGG/GO 20kb lists
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/d1a3829a1e5eb1e72c5c0c9dad3c59e3a0671b5f/docs/GO_KEGG_analysis.html" target="_blank">d1a3829</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9fb4f9a53ef385137e999fc72feb272bc9e7c469/analysis/GO_KEGG_analysis.Rmd" target="_blank">9fb4f9a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-17
</td>
<td>
adding in updates from Neargenes and enhncers
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/2d991d77abbfc9924acaba263dab895e2aeadd43/docs/GO_KEGG_analysis.html" target="_blank">2d991d7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/f57f2ee662bb848a02e677a0566d7707cac03606/analysis/GO_KEGG_analysis.Rmd" target="_blank">f57f2ee</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
first update
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(gprofiler2)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(biomaRt)</code></pre>
<pre class="r"><code>GO_KEGG_analysis &lt;- function(df,type, setp_value=1){
# Extract the name of the dataframe
  a &lt;- deparse(substitute(df)) %&gt;% 
  gsub(&quot;_NG_2kresgenes&quot;,&quot;&quot;,.)

 # Prepare the table output
  df_nomtable &lt;-  df$result %&gt;% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))
# Create a color palette for different dataframe names
  # You can customize this color palette as needed  
  
  mrc_palette &lt;- c(
    &quot;EAR_open&quot; = &quot;#F8766D&quot;,
    &quot;EAR_close&quot; = &quot;#f6483c&quot;,
    &quot;ESR_open&quot; = &quot;#7CAE00&quot;,
    &quot;ESR_close&quot; = &quot;#587b00&quot;,
     &quot;ESR_OC&quot; = &quot;#6a9500&quot;,
    &quot;ESR_opcl&quot; = &quot;cornflowerblue&quot;,
    &quot;ESR_clop&quot;= &quot;grey60&quot;,
     &quot;LR_open&quot; = &quot;#00BFC4&quot;,
     &quot;LR_close&quot; = &quot;#008d91&quot;,
     &quot;NR&quot; = &quot;#C77CFF&quot;
  )
   # Use the dataframe name as the fill color (fallback to black if name is not in the palette)
   fill_color &lt;- mrc_palette[a] %||% &quot;darkgrey&quot;
   
table &lt;- df_nomtable %&gt;%
  dplyr::filter(source==type) %&gt;% 
  dplyr::filter(p_value &lt; setp_value) %&gt;% 
  mutate_at(.vars = 6, .funs = scientific_format()) %&gt;%
  kable(.,caption = paste0(&quot;Significant (p &lt; &quot;,setp_value,&quot; ) &quot;,type,&quot; terms found in &quot;,a,&quot; neargenes&quot;)) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = FALSE) %&gt;%
  kable_styling(
    full_width = FALSE,
    position = &quot;left&quot;,
    bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)
  ) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)

# plot the table as ggplot object
plot &lt;- df_nomtable %&gt;%
  filter(source==type) %&gt;% 
  # slice_head(n=10) %&gt;% 
    dplyr::filter(p_value &lt; setp_value) %&gt;% 
  mutate(log10_p=(-log(p_value, base=10))) %&gt;% 
  ggplot(., aes(x=log10_p, y=reorder(term_name, log10_p)))+
    geom_col(fill=fill_color)+
  geom_vline(aes(xintercept=-log(0.05,base = 10)), col=&quot;red&quot;,linetype = 2)+
  theme_bw()+
  ylab(&quot;&quot;)+
  xlab(paste0(&quot;-log10 p-value&quot;))+
  guides(fill=&quot;none&quot;)+
  theme(axis.text.y = element_text(color=&quot;black&quot;))+
  ggtitle(paste0(&quot;(p &lt; &quot;,setp_value,&quot; ) &quot;,type,&quot; terms found in\n &quot;,a,&quot; neargenes&quot;))


print(table)
print(plot)
}</code></pre>
<div id="data-loading" class="section level5">
<h5>data loading</h5>
<pre class="r"><code>col_ng_peak &lt;- read.delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;)
TSS_NG_data &lt;- read_delim(&quot;data/Final_four_data/TSS_assigned_NG.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)

# scale_fill_mrc &lt;-  function(...){
#   ggplot2:::manual_scale(
#     &#39;fill&#39;,
#     values = setNames(c(&quot;#F8766D&quot;,&quot;#f6483c&quot;,&quot;#7CAE00&quot;,&quot;#587b00&quot;,&quot;#6a9500&quot;,  &quot;#00BFC4&quot;,&quot;#008d91&quot;, &quot;#C77CFF&quot;), c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;,&quot;ESR_close&quot;,&quot;ESR_OC&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;)),
#     ...
#   )
# }
peak_list_all_mrc &lt;- readRDS( &quot;data/Final_four_data/Peak_list_all_mrc_NG.RDS&quot;)

background_NGs &lt;- TSS_NG_data %&gt;% distinct(ENTREZID,SYMBOL)



mrc_palette &lt;- c(
    &quot;EAR_open&quot; = &quot;#F8766D&quot;,
    &quot;EAR_close&quot; = &quot;#f6483c&quot;,
    &quot;ESR_open&quot; = &quot;#7CAE00&quot;,
    &quot;ESR_close&quot; = &quot;#587b00&quot;,
     &quot;ESR_OC&quot; = &quot;#6a9500&quot;,
    &quot;ESR_opcl&quot; = &quot;cornflowerblue&quot;,
    &quot;ESR_clop&quot;= &quot;grey60&quot;,
     &quot;LR_open&quot; = &quot;#00BFC4&quot;,
     &quot;LR_close&quot; = &quot;#008d91&quot;,
     &quot;NR&quot; = &quot;#C77CFF&quot;
  )</code></pre>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>This is an updated page of GO/KEGG analysis based on previous
analysis. To get this analysis, I took the 8 sets of peaks from my data
and filtered each group by peaks that are within 2000 kb of a TSS of a
near-gene. I used the background of all expressed RNA genes (14,084)
(the genes used to generate the expressed near-gene list) to test for
enrichment of <a href="GO:Biological" class="uri">GO:Biological</a>
Processes and KEGG Pathways.</p>
<p>This page is broken up into the following sections:<br />
1. Creation of NG lists 2. List of terms from <a href="GO:BP"
class="uri">GO:BP</a> and KEGG by each MRC 3. Bargraphs of top
KEGG/<del>GO terms</del></p>
<div id="making-the-lists" class="section level3">
<h3>Making the lists</h3>
<div id="ear_open-to-background-list" class="section level4">
<h4>EAR_open to background list</h4>
<pre class="r"><code>EAR_open_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;EAR_open&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
EAR_close_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;EAR_close&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
  
ESR_open_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;ESR_open&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
ESR_close_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;ESR_close&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
ESR_OC_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;ESR_OC&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)

LR_open_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;LR_open&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)
LR_close_NG_2k&lt;-  peak_list_all_mrc %&gt;% 
      dplyr::filter(mrc ==&quot;LR_close&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene:SYMBOL, delim= &quot;,&quot;) %&gt;% 
    distinct(NCBI_gene,SYMBOL)

NR_NG_2k &lt;-   peak_list_all_mrc %&gt;%
      dplyr::filter(mrc ==&quot;NR&quot;) %&gt;% 
    dplyr::filter(dist_to_NG &gt;-2000&amp;dist_to_NG&lt;2000) %&gt;% 
    dplyr::select(Peakid, NCBI_gene:SYMBOL,dist_to_NG, mrc) %&gt;% 
    separate_longer_delim(., cols=NCBI_gene, delim= &quot;,&quot;) %&gt;% 
     separate_longer_delim(., cols=,SYMBOL, delim= &quot;,&quot;)  
 
# NR_NG_2kresgenes &lt;- gost(query = NR_NG_2k$NCBI_gene,
#                     organism = &quot;hsapiens&quot;,
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = &quot;custom&quot;,
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c(&quot;fdr&quot;),
#                     custom_bg = background_NGs$ENTREZID,
#                     sources=c(&quot;GO:BP&quot;,&quot;KEGG&quot;))



# saveRDS(NR_NG_2k,&quot;data/Final_four_data/GO_analysis/NR_NG_2kresgenes.RDS&quot;)
# saveRDS(EAR_close_NG_2kresgenes,&quot;data/Final_four_data/GO_analysis/EAR_close_NG_2kresgenes.RDS&quot;)
# saveRDS(ESR_close_NG_2kresgenes,&quot;data/Final_four_data/GO_analysis/ESR_close_NG_2kresgenes.RDS&quot;)
# saveRDS(ESR_OC_NG_2kresgenes,&quot;data/Final_four_data/GO_analysis/ESR_OC_NG_2kresgenes.RDS&quot;)
# saveRDS(LR_open_NG_2kresgenes,&quot;data/Final_four_data/GO_analysis/LR_open_NG_2kresgenes.RDS&quot;)
# saveRDS(ESR_open_NG_2kresgenes,&quot;data/Final_four_data/GO_analysis/ESR_open_NG_2kresgenes.RDS&quot;)
# saveRDS(EAR_open_NG_2kresgenes,&quot;data/Final_four_data/GO_analysis/EAR_open_NG_2kresgenes.RDS&quot;)
NR_NG_2k &lt;- readRDS(&quot;data/Final_four_data/GO_analysis/NR_NG_2kresgenes.RDS&quot;)
EAR_close_NG_2kresgenes &lt;- readRDS(&quot;data/Final_four_data/GO_analysis/EAR_close_NG_2kresgenes.RDS&quot;)
ESR_close_NG_2kresgenes &lt;- readRDS(&quot;data/Final_four_data/GO_analysis/ESR_close_NG_2kresgenes.RDS&quot;)
ESR_OC_NG_2kresgenes &lt;- readRDS(&quot;data/Final_four_data/GO_analysis/ESR_OC_NG_2kresgenes.RDS&quot;)
LR_open_NG_2kresgenes &lt;- readRDS(&quot;data/Final_four_data/GO_analysis/LR_open_NG_2kresgenes.RDS&quot;)
LR_close_NG_2kresgenes &lt;- readRDS(&quot;data/Final_four_data/GO_analysis/LR_close_NG_2kresgenes.RDS&quot;)
ESR_open_NG_2kresgenes &lt;-readRDS (&quot;data/Final_four_data/GO_analysis/ESR_open_NG_2kresgenes.RDS&quot;)
EAR_open_NG_2kresgenes&lt;-readRDS  (&quot;data/Final_four_data/GO_analysis/EAR_open_NG_2kresgenes.RDS&quot;)</code></pre>
</div>
</div>
</div>
<div id="experimental-go" class="section level2">
<h2>experimental GO</h2>
<pre class="r"><code>GO_KEGG_analysis(EAR_open_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in EAR_open neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0090304 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; nucleic acid metabolic process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 198 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 3690 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.45e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0046483 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; heterocycle metabolic process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 220 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4193 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.45e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0006139 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; nucleobase-containing compound metabolic process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 216 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4084 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.45e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0034641 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; cellular nitrogen compound metabolic process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 238 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4538 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.45e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0034660 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; ncRNA metabolic process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 42 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 525 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.88e-02 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-1">
Past versions of GOBP-1.png
</button>
</p>
<div id="fig-GOBP-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-1.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-1.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(EAR_close_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in EAR_close neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0072359 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; circulatory system development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 20 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 852 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.51e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0046688 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; response to copper ion &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 22 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.51e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0048513 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; animal organ development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 33 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 1975 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.51e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0007507 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; heart development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 14 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 494 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 3.31e-02 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-2">
Past versions of GOBP-2.png
</button>
</p>
<div id="fig-GOBP-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-2.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-2.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(ESR_open_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in ESR_open neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0048732 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; gland development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 24 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 305 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 6.40e-03 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:2000736 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; regulation of stem cell differentiation &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 11 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 71 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 6.40e-03 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0032502 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; developmental process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 148 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4119 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.08e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0060425 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; lung morphogenesis &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 8 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 41 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.08e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0003278 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; apoptotic process involved in heart morphogenesis &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 6 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.08e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0048568 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; embryonic organ development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 22 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 301 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.68e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0048839 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; inner ear development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 13 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 128 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.08e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0048856 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; anatomical structure development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 137 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 3812 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.08e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0043583 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; ear development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 14 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 146 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.08e-02 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-3">
Past versions of GOBP-3.png
</button>
</p>
<div id="fig-GOBP-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-3.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-3.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(ESR_close_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in ESR_close neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0060047 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; heart contraction &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 19 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 192 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.94e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0030001 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; metal ion transport &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 36 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 520 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.94e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0006812 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; monoatomic cation transport &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 38 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 609 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0003015 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; heart process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 19 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 202 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0045760 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; positive regulation of action potential &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 3 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 3 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0030239 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; myofibril assembly &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 10 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 66 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0045214 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; sarcomere organization &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 8 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 43 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0055002 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; striated muscle cell development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 10 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 68 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0006811 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; monoatomic ion transport &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 43 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 712 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0008016 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; regulation of heart contraction &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 16 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 163 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.22e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0010959 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; regulation of metal ion transport &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 20 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 238 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 2.25e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0001738 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; morphogenesis of a polarized epithelium &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 8 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 47 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 3.44e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0032501 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; multicellular organismal process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 173 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4439 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 3.62e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0061061 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; muscle structure development &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 33 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 531 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 3.86e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0098660 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; inorganic ion transmembrane transport &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 32 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 516 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.76e-02 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-4">
Past versions of GOBP-4.png
</button>
</p>
<div id="fig-GOBP-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-4.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-4.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(ESR_OC_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in ESR_OC neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-5">
Past versions of GOBP-5.png
</button>
</p>
<div id="fig-GOBP-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-5.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-5.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(LR_open_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in LR_open neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0023052 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; signaling &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 256 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 3865 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.58e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0007154 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; cell communication &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 258 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 3929 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 1.58e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:BP &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; GO:0032501 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; multicellular organismal process &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 282 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 4439 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.60e-02 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-6">
Past versions of GOBP-6.png
</button>
</p>
<div id="fig-GOBP-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-6.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-6.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(LR_close_NG_2kresgenes,&quot;GO:BP&quot;, 0.049)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.049 ) GO:BP terms found in LR_close neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/GOBP-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GOBP-7">
Past versions of GOBP-7.png
</button>
</p>
<div id="fig-GOBP-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/GOBP-7.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/GOBP-7.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># GO_KEGG_analysis(NR_NG_2k,&quot;GO:BP&quot;, 0.049)</code></pre>
<div id="kegg-not-informative" class="section level5">
<h5>KEGG (not informative)</h5>
<pre class="r"><code>GO_KEGG_analysis(EAR_open_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in EAR_open neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-1">
Past versions of "KEGG analysis-1.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-1.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-1.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(EAR_close_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in EAR_close neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-2">
Past versions of "KEGG analysis-2.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-2.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-2.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(ESR_open_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in ESR_open neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-3">
Past versions of "KEGG analysis-3.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-3.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-3.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(ESR_close_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in ESR_close neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; KEGG &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; KEGG:05414 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; Dilated cardiomyopathy &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 10 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 82 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.93e-02 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; KEGG &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; KEGG:04261 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; Adrenergic signaling in cardiomyocytes &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 12 &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; 121 &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; 4.93e-02 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-4">
Past versions of "KEGG analysis-4.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-4.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-4.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(ESR_OC_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in ESR_OC neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-5">
Past versions of "KEGG analysis-5.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-5.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-5.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(LR_open_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in LR_open neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-6">
Past versions of "KEGG analysis-6.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-6.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-6.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>GO_KEGG_analysis(LR_close_NG_2kresgenes,&quot;KEGG&quot;, 0.05)</code></pre>
<pre><code>&lt;div style=&quot;border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; &quot;&gt;&lt;table class=&quot; lightable-paper lightable-striped table table-striped table-hover&quot; style=&#39;font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; &#39;&gt;
&lt;caption&gt;Significant (p &amp;lt; 0.05 ) KEGG terms found in LR_close neargenes&lt;/caption&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; source &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_id &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_name &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; intersection_size &lt;/th&gt;
   &lt;th style=&quot;text-align:right;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; term_size &lt;/th&gt;
   &lt;th style=&quot;text-align:left;position: sticky; top:0; background-color: #FFFFFF;&quot;&gt; p_value &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; NA &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; NA &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------ &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :--------- &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; -----------------: &lt;/td&gt;
   &lt;td style=&quot;text-align:right;&quot;&gt; ---------: &lt;/td&gt;
   &lt;td style=&quot;text-align:left;&quot;&gt; :------- &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;&lt;/div&gt;</code></pre>
<p><img src="figure/GO_KEGG_analysis.Rmd/KEGG%20analysis-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-KEGG-analysis-7">
Past versions of "KEGG analysis-7.png"
</button>
</p>
<div id="fig-no-spaces-KEGG-analysis-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8ea25f4215139feba3bd061b686bbd3f74b0d9ae/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-7.png" target="_blank">8ea25f4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fd5c82d963d05a35715d5b4a84f912e8475fdfb2/docs/figure/GO_KEGG_analysis.Rmd/KEGG analysis-7.png" target="_blank">fd5c82d</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># GO_KEGG_analysis(NR_NG_2k,&quot;KEGG&quot;, 0.05)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] biomaRt_2.60.1                          
 [2] ggpubr_0.6.0                            
 [3] BiocParallel_1.38.0                     
 [4] ggVennDiagram_1.5.2                     
 [5] scales_1.3.0                            
 [6] VennDiagram_1.7.3                       
 [7] futile.logger_1.4.3                     
 [8] gridExtra_2.3                           
 [9] ggfortify_0.4.17                        
[10] edgeR_4.2.1                             
[11] limma_3.60.4                            
[12] rtracklayer_1.64.0                      
[13] org.Hs.eg.db_3.19.1                     
[14] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[15] GenomicFeatures_1.56.0                  
[16] AnnotationDbi_1.66.0                    
[17] Biobase_2.64.0                          
[18] GenomicRanges_1.56.1                    
[19] GenomeInfoDb_1.40.1                     
[20] IRanges_2.38.1                          
[21] S4Vectors_0.42.1                        
[22] BiocGenerics_0.50.0                     
[23] ChIPseeker_1.40.0                       
[24] gprofiler2_0.2.3                        
[25] RColorBrewer_1.1-3                      
[26] broom_1.0.7                             
[27] kableExtra_1.4.0                        
[28] lubridate_1.9.3                         
[29] forcats_1.0.0                           
[30] stringr_1.5.1                           
[31] dplyr_1.1.4                             
[32] purrr_1.0.2                             
[33] readr_2.1.5                             
[34] tidyr_1.3.1                             
[35] tibble_3.2.1                            
[36] ggplot2_3.5.1                           
[37] tidyverse_2.0.0                         
[38] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.4                               
  [2] matrixStats_1.4.1                      
  [3] bitops_1.0-8                           
  [4] enrichplot_1.24.4                      
  [5] httr_1.4.7                             
  [6] tools_4.4.1                            
  [7] backports_1.5.0                        
  [8] utf8_1.2.4                             
  [9] R6_2.5.1                               
 [10] lazyeval_0.2.2                         
 [11] withr_3.0.1                            
 [12] prettyunits_1.2.0                      
 [13] cli_3.6.3                              
 [14] formatR_1.14                           
 [15] scatterpie_0.2.4                       
 [16] labeling_0.4.3                         
 [17] sass_0.4.9                             
 [18] Rsamtools_2.20.0                       
 [19] systemfonts_1.1.0                      
 [20] yulab.utils_0.1.7                      
 [21] DOSE_3.30.5                            
 [22] svglite_2.1.3                          
 [23] R.utils_2.12.3                         
 [24] plotrix_3.8-4                          
 [25] rstudioapi_0.16.0                      
 [26] RSQLite_2.3.7                          
 [27] generics_0.1.3                         
 [28] gridGraphics_0.5-1                     
 [29] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [30] BiocIO_1.14.0                          
 [31] vroom_1.6.5                            
 [32] gtools_3.9.5                           
 [33] car_3.1-3                              
 [34] GO.db_3.19.1                           
 [35] Matrix_1.7-0                           
 [36] fansi_1.0.6                            
 [37] abind_1.4-8                            
 [38] R.methodsS3_1.8.2                      
 [39] lifecycle_1.0.4                        
 [40] whisker_0.4.1                          
 [41] yaml_2.3.10                            
 [42] carData_3.0-5                          
 [43] SummarizedExperiment_1.34.0            
 [44] gplots_3.1.3.1                         
 [45] qvalue_2.36.0                          
 [46] SparseArray_1.4.8                      
 [47] BiocFileCache_2.12.0                   
 [48] blob_1.2.4                             
 [49] promises_1.3.0                         
 [50] crayon_1.5.3                           
 [51] lattice_0.22-6                         
 [52] cowplot_1.1.3                          
 [53] KEGGREST_1.44.1                        
 [54] pillar_1.9.0                           
 [55] knitr_1.48                             
 [56] fgsea_1.30.0                           
 [57] rjson_0.2.23                           
 [58] boot_1.3-31                            
 [59] codetools_0.2-20                       
 [60] fastmatch_1.1-4                        
 [61] glue_1.8.0                             
 [62] getPass_0.2-4                          
 [63] ggfun_0.1.6                            
 [64] data.table_1.16.0                      
 [65] vctrs_0.6.5                            
 [66] png_0.1-8                              
 [67] treeio_1.28.0                          
 [68] gtable_0.3.5                           
 [69] cachem_1.1.0                           
 [70] xfun_0.47                              
 [71] S4Arrays_1.4.1                         
 [72] tidygraph_1.3.1                        
 [73] statmod_1.5.0                          
 [74] nlme_3.1-166                           
 [75] ggtree_3.12.0                          
 [76] bit64_4.5.2                            
 [77] filelock_1.0.3                         
 [78] progress_1.2.3                         
 [79] rprojroot_2.0.4                        
 [80] bslib_0.8.0                            
 [81] KernSmooth_2.23-24                     
 [82] colorspace_2.1-1                       
 [83] DBI_1.2.3                              
 [84] tidyselect_1.2.1                       
 [85] processx_3.8.4                         
 [86] bit_4.5.0                              
 [87] compiler_4.4.1                         
 [88] curl_5.2.3                             
 [89] git2r_0.33.0                           
 [90] httr2_1.0.5                            
 [91] xml2_1.3.6                             
 [92] DelayedArray_0.30.1                    
 [93] plotly_4.10.4                          
 [94] shadowtext_0.1.4                       
 [95] caTools_1.18.3                         
 [96] callr_3.7.6                            
 [97] rappdirs_0.3.3                         
 [98] digest_0.6.37                          
 [99] rmarkdown_2.28                         
[100] XVector_0.44.0                         
[101] htmltools_0.5.8.1                      
[102] pkgconfig_2.0.3                        
[103] MatrixGenerics_1.16.0                  
[104] highr_0.11                             
[105] dbplyr_2.5.0                           
[106] fastmap_1.2.0                          
[107] rlang_1.1.4                            
[108] htmlwidgets_1.6.4                      
[109] UCSC.utils_1.0.0                       
[110] farver_2.1.2                           
[111] jquerylib_0.1.4                        
[112] jsonlite_1.8.9                         
[113] GOSemSim_2.30.2                        
[114] R.oo_1.26.0                            
[115] RCurl_1.98-1.16                        
[116] magrittr_2.0.3                         
[117] Formula_1.2-5                          
[118] GenomeInfoDbData_1.2.12                
[119] ggplotify_0.1.2                        
[120] patchwork_1.3.0                        
[121] munsell_0.5.1                          
[122] Rcpp_1.0.13                            
[123] ape_5.8                                
[124] viridis_0.6.5                          
[125] stringi_1.8.4                          
[126] ggraph_2.2.1                           
[127] zlibbioc_1.50.0                        
[128] MASS_7.3-61                            
[129] plyr_1.8.9                             
[130] parallel_4.4.1                         
[131] ggrepel_0.9.6                          
[132] Biostrings_2.72.1                      
[133] graphlayouts_1.2.0                     
[134] splines_4.4.1                          
[135] hms_1.1.3                              
[136] locfit_1.5-9.10                        
[137] ps_1.8.0                               
[138] igraph_2.0.3                           
[139] ggsignif_0.6.4                         
[140] reshape2_1.4.4                         
[141] futile.options_1.0.1                   
[142] XML_3.99-0.17                          
[143] evaluate_1.0.0                         
[144] lambda.r_1.2.4                         
[145] tzdb_0.4.0                             
[146] tweenr_2.0.3                           
[147] httpuv_1.6.15                          
[148] polyclip_1.10-7                        
[149] ggforce_0.4.2                          
[150] restfulr_0.0.15                        
[151] tidytree_0.4.6                         
[152] rstatix_0.7.2                          
[153] later_1.3.2                            
[154] viridisLite_0.4.2                      
[155] aplot_0.2.3                            
[156] memoise_2.0.1                          
[157] GenomicAlignments_1.40.0               
[158] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
