<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-05-09" />

<title>H3K27ac_data_inital_QC</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">H3K27ac_data_inital_QC</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-05-09</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-05-14
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025tree1bb4020a7fee041516ec405271426ad269757387targetblank1bb4020a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/1bb4020a7fee041516ec405271426ad269757387" target="_blank">1bb4020</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025tree1bb4020a7fee041516ec405271426ad269757387targetblank1bb4020a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/1bb4020a7fee041516ec405271426ad269757387" target="_blank">1bb4020</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  RNA_seq_integration.Rmd
    Untracked:  analysis/Diagnosis-tmm.Rmd
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/H3K27ac_integration_noM.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/making_master_peaks_list.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/AF_HF_SNPs.Rmd
    Modified:   analysis/Cardiotox_SNPs.Rmd
    Modified:   analysis/Cormotif_analysis.Rmd
    Modified:   analysis/Counts_matrix.Rmd
    Modified:   analysis/Enhancer_enrichment.Rmd
    Modified:   analysis/H3K27ac_cormotif.Rmd
    Modified:   analysis/H3K27ac_integration.Rmd
    Modified:   analysis/Jaspar_motif.Rmd
    Modified:   analysis/Jaspar_motif_ff.Rmd
    Modified:   analysis/RNA_seq_integration.Rmd
    Modified:   analysis/TSS_and_CUG.Rmd
    Modified:   analysis/final_four_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/H3K27ac_initial_QC.Rmd</code>)
and HTML (<code>docs/H3K27ac_initial_QC.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/1bb4020a7fee041516ec405271426ad269757387/analysis/H3K27ac_initial_QC.Rmd" target="_blank">1bb4020</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-14
</td>
<td>
taking out extra section
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/H3K27ac_initial_QC.html" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/2b4cfbd2b9df2bcd11e4322aac87f114b3f3d216/analysis/H3K27ac_initial_QC.Rmd" target="_blank">2b4cfbd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
<td>
wflow_publish("analysis/H3K27ac_initial_QC.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/H3K27ac_initial_QC.html" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/15e8bfc74c898053c2229adc05e9789c0a9f42c5/analysis/H3K27ac_initial_QC.Rmd" target="_blank">15e8bfc</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
<td>
adding H3K27ac reanalysis.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
# library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
# library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(eulerr)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(smplot2)
library(stringr)
library(cowplot)</code></pre>
<p>Function loading</p>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#41B333&quot;)


prop_var_percent &lt;- function(pca_result){ 
  # Ensure the input is a PCA result object
  if (!inherits(pca_result, &quot;prcomp&quot;)) {
    stop(&quot;Input must be a result from prcomp()&quot;)
  }
  
  # Get the standard deviations from the PCA result
  sdev &lt;- pca_result$sdev
  
  # Calculate the proportion of variance
  proportion_variance &lt;- (sdev^2) / sum(sdev^2)*100
  
  return(proportion_variance)
}</code></pre>
<p>Importing the count file and editing names</p>
<pre class="r"><code>### file originally loaded from ~/ATAC_files/peaks_folder/Raodah_data/H3K27ac_count.txt
### Blacklisted regions were removed
H3K27ac_raw_count &lt;- read_delim(&quot;C:/Users/renee/ATAC_folder/peaks folder/Raodah_data/H3K27ac_count.txt&quot;,
                            delim = &quot;\t&quot;, 
                            escape_double = FALSE,
                            trim_ws = TRUE, 
                            skip = 1)
 ### simplifying names to paper names
 names(H3K27ac_raw_count) &lt;- names(H3K27ac_raw_count) |&gt;
  gsub(&quot;_final\\.bam.*&quot;, &quot;&quot;, x = _) |&gt;
  gsub(&quot;^Individual_data/(77-1|87-1|71-1)/bamFiles/&quot;, &quot;&quot;, x = _) |&gt;
  gsub(&quot;87_1_&quot;, &quot;A_&quot;, x = _) |&gt;
  gsub(&quot;77_1_&quot;, &quot;B_&quot;, x = _) |&gt;
  gsub(&quot;71_1_&quot;, &quot;C_&quot;, x = _) |&gt;
  gsub(&quot;3$&quot;, &quot;_3&quot;, x = _) |&gt;
  gsub(&quot;24$&quot;, &quot;_24&quot;, x = _)

 sample_peaks_locations &lt;- H3K27ac_raw_count[1:6]
 raw_counts_only &lt;- H3K27ac_raw_count[,c(1,7:31)]
 sample_peaks_locations %&gt;% 
   GRanges() %&gt;% 
   saveRDS(&quot;data/Final_four_data/re_analysis/H3K27ac_granges_df.RDS&quot;)</code></pre>
<p>Looking at raw counts QC</p>
<pre class="r"><code>PCA_H3_mat &lt;- raw_counts_only%&gt;% column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()
hist(PCA_H3_mat, main= &quot;Raw counts across samples&quot;,
     xlab = &quot;Raw counts&quot;,
     col=4)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/examine%20inital%20data-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-inital-data-1">
Past versions of "examine inital data-1.png"
</button>
</p>
<div id="fig-no-spaces-examine-inital-data-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/examine inital data-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist(cpm(PCA_H3_mat, log=TRUE), 
     main = expression(&quot;Histogram of Log&quot;[2]*&quot; cpm unfiltered&quot;),
     xlab = expression(&quot;Log&quot;[2]*&quot; counts-per-million&quot;),
     col=4)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/examine%20inital%20data-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-inital-data-2">
Past versions of "examine inital data-2.png"
</button>
</p>
<div id="fig-no-spaces-examine-inital-data-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/examine inital data-2.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>boxplot(cpm(PCA_H3_mat, log=TRUE),
        main=expression(&quot;Boxplot of Log&quot;[2]*&quot; counts-per-million&quot;),
                        col=4,
        names=colnames(PCA_H3_mat),
        las=2, cex.axis=.7)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/examine%20inital%20data-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-inital-data-3">
Past versions of "examine inital data-3.png"
</button>
</p>
<div id="fig-no-spaces-examine-inital-data-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/examine inital data-3.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="heatmap-of-unfiltered-log2-cpm-counts" class="section level3">
<h3>Heatmap of unfiltered log2 cpm counts</h3>
<pre class="r"><code>cor_raw_counts &lt;- PCA_H3_mat %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor(.,method = &quot;pearson&quot;)

anno_raw_counts &lt;- data.frame(timeset = colnames(PCA_H3_mat))

counts_corr_mat &lt;-anno_raw_counts %&gt;%
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, 
                         if_else(trt == &quot;DOX&quot;, &quot;AC&quot;, 
                                 if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, 
                         if_else(trt == &quot;DOX&quot;, &quot;yes&quot;, 
                                 if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, 
                                         if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) 

                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#41B333&quot;),
   indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)


htanno_full &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat, col = mat_colors)
Heatmap(cor_raw_counts, top_annotation = htanno_full, column_title = &quot;H3K27ac log2 cpm unfiltered counts correlation&quot;)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/full%20heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-full-heatmap-1">
Past versions of "full heatmap-1.png"
</button>
</p>
<div id="fig-no-spaces-full-heatmap-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/full heatmap-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Filtering out low count enriched regions</p>
<pre class="r"><code>lcpm &lt;- cpm(PCA_H3_mat, log= TRUE)
  ### for determining the basic cutoffs
filt_raw_counts &lt;- PCA_H3_mat[rowMeans(lcpm)&gt; 0,]
dim(filt_raw_counts)</code></pre>
<pre><code>[1] 20137    25</code></pre>
<pre class="r"><code>tail(rownames(filt_raw_counts),n=10)</code></pre>
<pre><code> [1] &quot;chrX.154515375.154517834&quot; &quot;chrX.154526840.154527801&quot;
 [3] &quot;chrX.154531568.154532847&quot; &quot;chrX.154546431.154547623&quot;
 [5] &quot;chrX.154762614.154763946&quot; &quot;chrX.155026613.155027943&quot;
 [7] &quot;chrX.155070633.155072047&quot; &quot;chrX.155215803.155217295&quot;
 [9] &quot;chrX.155612083.155613144&quot; &quot;chrX.155719034.155719529&quot;</code></pre>
<p>There is no change in filtering out regions using
rowMeans(log2cpm)&gt;0, but I will still use the filtered matrix name in
the following PCA plots</p>
</div>
<div id="examining-pca" class="section level3">
<h3>Examining PCA</h3>
<pre class="r"><code>filt_matrix_lcpm &lt;- cpm(filt_raw_counts , log=TRUE)
## store PRcomp
PCA_H3K27ac_info_filter &lt;- (prcomp(t(filt_matrix_lcpm), scale. = TRUE))
###make annotation dataframe
annotation_mat &lt;- data.frame(timeset=colnames(filt_matrix_lcpm)) %&gt;%
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) 
### join together for plotting
pca_H3K27ac_anno &lt;- data.frame(annotation_mat, PCA_H3K27ac_info_filter$x)
plotting_var_names &lt;- prop_var_percent(PCA_H3K27ac_info_filter)

pca_H3K27ac_anno %&gt;%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm) H3K27ac filtered enriched regions&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = paste0(&quot;PC 2 (&quot;,round(plotting_var_names[2],2),&quot;%)&quot;)
       , x =paste0(&quot;PC 1 (&quot;,round(plotting_var_names[1],2),&quot;%)&quot;))+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/Examining%20PCA-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Examining-PCA-1">
Past versions of "Examining PCA-1.png"
</button>
</p>
<div id="fig-no-spaces-Examining-PCA-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/Examining PCA-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_H3K27ac_anno %&gt;%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm)  H3K27ac filtered enriched regions&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = paste0(&quot;PC 4 (&quot;,round(plotting_var_names[4],1),&quot;%)&quot;)
       , x =paste0(&quot;PC 3 (&quot;,round(plotting_var_names[3],1),&quot;%)&quot;))+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/Examining%20PCA-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Examining-PCA-2">
Past versions of "Examining PCA-2.png"
</button>
</p>
<div id="fig-no-spaces-Examining-PCA-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/Examining PCA-2.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Like the correlation plots, B-24hour VEH and C 3 hour VEH are not
aliging with the rest of the data. This is in line with the number of
peaks called by each set (data not shown here). This led us to remove
the two outliers for further analysis</p>
<div id="removing-outliers" class="section level4">
<h4>Removing outliers</h4>
<p>Removing C_VEH_3 and B_VEH_24 from matrix and reanalyzing</p>
<pre class="r"><code>final_23_mat &lt;- PCA_H3_mat %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(!C_VEH_3) %&gt;% 
   dplyr::select(!B_VEH_24) %&gt;% 
  
    as.matrix()

hist(final_23_mat, main= &quot;Raw counts across final samples&quot;,
     xlab = &quot;Raw counts&quot;,
     col=6)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-1.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hist(cpm(final_23_mat, log=TRUE), 
     main = expression(&quot;Histogram of Log&quot;[2]*&quot; cpm unfiltered final counts H3K27ac&quot;),
     xlab = expression(&quot;Log&quot;[2]*&quot; counts-per-million&quot;),
     col=6)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-2">
Past versions of unnamed-chunk-2-2.png
</button>
</p>
<div id="fig-unnamed-chunk-2-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-2.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-2.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>boxplot(cpm(final_23_mat, log=TRUE),
        main=expression(&quot;Boxplot of Log&quot;[2]*&quot; counts-per-million final H3K27ac samples&quot;),
                        col=6,
        names=colnames(final_23_mat),
        las=2, cex.axis=.7)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-3">
Past versions of unnamed-chunk-2-3.png
</button>
</p>
<div id="fig-unnamed-chunk-2-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-3.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/unnamed-chunk-2-3.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>saveRDS(final_23_mat, &quot;data/Final_four_data/re_analysis/H3K27ac_final_23_raw_counts.RDS&quot;)</code></pre>
</div>
<div id="filtering-check" class="section level4">
<h4>Filtering check</h4>
<pre class="r"><code>lcpm_f &lt;- cpm(final_23_mat, log= TRUE)
  ### for determining the basic cutoffs
filt_final_raw_counts &lt;- final_23_mat[rowMeans(lcpm_f)&gt; 0,]
dim(filt_final_raw_counts)</code></pre>
<pre><code>[1] 20137    23</code></pre>
<p>Still no change in row number. Number of enriched regions is 20137,
23 (rows and columns)</p>
</div>
<div id="differentially-acetylated-enriched-regions"
class="section level4">
<h4>Differentially acetylated enriched regions</h4>
<pre class="r"><code>groupset &lt;- colnames(final_23_mat)
split_parts &lt;- strsplit(groupset, &quot;_&quot;)
group &lt;- sapply(split_parts, function(x) paste(x[2], x[3], sep = &quot;_&quot;))
indv &lt;- sapply(split_parts, function(x) paste(x[1]))

group &lt;- factor(group, levels=c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;VEH_24&quot;,&quot;VEH_3&quot;))

dge &lt;-  DGEList.data.frame(counts = final_23_mat, group = group, genes = row.names(final_23_mat))
dge &lt;- calcNormFactors(dge)
dge$samples</code></pre>
<pre><code>          group lib.size norm.factors
C_DNR_24 DNR_24   662075    0.9880860
C_DNR_3   DNR_3   304401    0.9449674
C_DOX_24 DOX_24  1184054    1.1516301
C_EPI_24 EPI_24   582422    1.0381793
C_EPI_3   EPI_3   344951    0.9248089
C_MTX_24 MTX_24   454798    0.8269981
C_MTX_3   MTX_3   625668    1.0524885
C_VEH_24 VEH_24  1297229    1.1877036
B_DNR_24 DNR_24  1637644    1.1481949
B_DNR_3   DNR_3  1693158    1.0600627
B_DOX_3   DOX_3  1397016    1.0510043
B_EPI_24 EPI_24   675946    0.9361002
B_EPI_3   EPI_3   492082    0.7423355
B_MTX_24 MTX_24  1124918    1.0782328
B_VEH_3   VEH_3   926454    0.9588624
A_DNR_24 DNR_24  1231409    0.9933291
A_DNR_3   DNR_3   894507    0.9522172
A_DOX_24 DOX_24   762252    0.9612265
A_DOX_3   DOX_3   619348    0.8824246
A_MTX_24 MTX_24  2236590    1.0893364
A_MTX_3   MTX_3   868211    1.0179427
A_VEH_24 VEH_24  1539759    1.1458191
A_VEH_3   VEH_3   753791    1.0001017</code></pre>
<p>Making model matrix</p>
<pre class="r"><code> mm &lt;- model.matrix(~0 +group)
colnames(mm) &lt;-  c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
mm</code></pre>
<pre><code>   DNR_24 DNR_3 DOX_24 DOX_3 EPI_24 EPI_3 MTX_24 MTX_3 VEH_24 VEH_3
1       1     0      0     0      0     0      0     0      0     0
2       0     1      0     0      0     0      0     0      0     0
3       0     0      1     0      0     0      0     0      0     0
4       0     0      0     0      1     0      0     0      0     0
5       0     0      0     0      0     1      0     0      0     0
6       0     0      0     0      0     0      1     0      0     0
7       0     0      0     0      0     0      0     1      0     0
8       0     0      0     0      0     0      0     0      1     0
9       1     0      0     0      0     0      0     0      0     0
10      0     1      0     0      0     0      0     0      0     0
11      0     0      0     1      0     0      0     0      0     0
12      0     0      0     0      1     0      0     0      0     0
13      0     0      0     0      0     1      0     0      0     0
14      0     0      0     0      0     0      1     0      0     0
15      0     0      0     0      0     0      0     0      0     1
16      1     0      0     0      0     0      0     0      0     0
17      0     1      0     0      0     0      0     0      0     0
18      0     0      1     0      0     0      0     0      0     0
19      0     0      0     1      0     0      0     0      0     0
20      0     0      0     0      0     0      1     0      0     0
21      0     0      0     0      0     0      0     1      0     0
22      0     0      0     0      0     0      0     0      1     0
23      0     0      0     0      0     0      0     0      0     1
attr(,&quot;assign&quot;)
 [1] 1 1 1 1 1 1 1 1 1 1
attr(,&quot;contrasts&quot;)
attr(,&quot;contrasts&quot;)$group
[1] &quot;contr.treatment&quot;</code></pre>
<pre class="r"><code>y &lt;- voom(dge, mm,plot =FALSE)

corfit &lt;- duplicateCorrelation(y, mm, block = indv)
 
v &lt;- voom(dge, mm, block = indv, correlation = corfit$consensus)

fit &lt;- lmFit(v, mm, block = indv, correlation = corfit$consensus)


cm &lt;- makeContrasts(
  DNR_3.VEH_3 = DNR_3-VEH_3,
  DOX_3.VEH_3 = DOX_3-VEH_3,
  EPI_3.VEH_3 = EPI_3-VEH_3,
  MTX_3.VEH_3 = MTX_3-VEH_3,
  DNR_24.VEH_24 =DNR_24-VEH_24,
  DOX_24.VEH_24= DOX_24-VEH_24,
  EPI_24.VEH_24= EPI_24-VEH_24,
  MTX_24.VEH_24= MTX_24-VEH_24,
  levels = mm)


fit2&lt;- contrasts.fit(fit, contrasts=cm)

efit2 &lt;- eBayes(fit2)

results = decideTests(efit2)

summary(results)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 DNR_24.VEH_24
Down           867          86           0           0          1752
NotSig       18860       20023       20137       20137         16317
Up             410          28           0           0          2068
       DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24
Down             188            48             0
NotSig         19297         19754         20136
Up               652           335             1</code></pre>
<pre class="r"><code>plotSA(efit2, main=&quot;Mean-Variance trend for final model of H3K27ac&quot;)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/run%20pipeline-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-run-pipeline-1">
Past versions of "run pipeline-1.png"
</button>
</p>
<div id="fig-no-spaces-run-pipeline-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/run pipeline-1.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/run pipeline-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>V.DNR_3.top= topTable(efit2, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_3.top= topTable(efit2, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_3.top= topTable(efit2, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_3.top= topTable(efit2, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DNR_24.top= topTable(efit2, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_24.top= topTable(efit2, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_24.top= topTable(efit2, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_24.top= topTable(efit2, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)


# plot_filenames &lt;- c(&quot;V.DNR_3.top&quot;,&quot;V.DOX_3.top&quot;,&quot;V.EPI_3.top&quot;,&quot;V.MTX_3.top&quot;,
#                     &quot;V.TRZ_.top&quot;,&quot;V.DNR_24.top&quot;,&quot;V.DOX_24.top&quot;,&quot;V.EPI_24.top&quot;,
#                     &quot;V.MTX_24.top&quot;,&quot;V.TRZ_24.top&quot;)
# plot_files &lt;- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
#                     V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
#                     V.MTX_24.top,V.TRZ_24.top)

save_list &lt;- list(&quot;DNR_3&quot;=V.DNR_3.top,&quot;DOX_3&quot;=V.DOX_3.top,&quot;EPI_3&quot;=V.EPI_3.top,&quot;MTX_3&quot;=V.MTX_3.top,&quot;DNR_24&quot;=V.DNR_24.top,&quot;DOX_24&quot;=V.DOX_24.top,&quot;EPI_24&quot;=V.EPI_24.top,&quot;MTX_24&quot;= V.MTX_24.top)

saveRDS(save_list,&quot;data/Final_four_data/re_analysis/Toptable_results_H3K27ac_data.RDS&quot;)</code></pre>
<pre class="r"><code>volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
      # ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC &gt;= 0,&quot;B&quot;, &quot;C&quot;)))
    ##This is where I could add labels, but I have taken out
    # df &lt;- df %&gt;% mutate(genelabels = &quot;&quot;)
    # df$genelabels[1:topg] &lt;- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    ggrastr::geom_point_rast(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, &quot;npc&quot;)), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    ylim(0,15)+
    xlim(-7,7)+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}

v1 &lt;- volcanosig(V.DNR_3.top, 0.05)+ ggtitle(&quot;DNR 3 hour&quot;)
v2 &lt;- volcanosig(V.DNR_24.top, 0.05)+ ggtitle(&quot;DNR 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(V.DOX_3.top, 0.05)+ ggtitle(&quot;DOX 3 hour&quot;)
v4 &lt;- volcanosig(V.DOX_24.top, 0.05)+ ggtitle(&quot;DOX 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(V.EPI_3.top, 0.05)+ ggtitle(&quot;EPI 3 hour&quot;)
v6 &lt;- volcanosig(V.EPI_24.top, 0.05)+ ggtitle(&quot;EPI 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(V.MTX_3.top, 0.05)+ ggtitle(&quot;MTX 3 hour&quot;)
v8 &lt;- volcanosig(V.MTX_24.top, 0.05)+ ggtitle(&quot;MTX 24 hour&quot;)+ylab(&quot;&quot;)


plot_grid(v1,v2,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/Volcano%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-1">
Past versions of "Volcano plots-1.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-1.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3,v4,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/Volcano%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-2">
Past versions of "Volcano plots-2.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-2.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-2.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5,v6,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/Volcano%20plots-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-3">
Past versions of "Volcano plots-3.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-3.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-3.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7,v8,  rel_widths =c(1,1))</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/Volcano%20plots-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Volcano-plots-4">
Past versions of "Volcano plots-4.png"
</button>
</p>
<div id="fig-no-spaces-Volcano-plots-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-4.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/Volcano plots-4.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>H3K27ac enriched regions median dataframe calculations</p>
<pre class="r"><code>all_results &lt;- bind_rows(save_list, .id = &quot;group&quot;)

median_df &lt;- all_results %&gt;% 
  separate(group, into=c(&quot;trt&quot;,&quot;time&quot;),sep = &quot;_&quot;) %&gt;% 
  pivot_wider(., id_cols=c(time,genes), names_from = trt, values_from = logFC) %&gt;% 
  rowwise() %&gt;% 
  mutate(median_H3K27ac_lfc= median(c_across(DNR:MTX)))

median_3_lfc &lt;-   median_df %&gt;%
    dplyr::filter(time == &quot;3&quot;) %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(time, genes,median_H3K27ac_lfc) %&gt;% 
  dplyr::rename(&quot;med_Kac_3h_lfc&quot;=median_H3K27ac_lfc, &quot;H3K27ac_peak&quot;=genes)
  

median_24_lfc &lt;- median_df %&gt;%
    dplyr::filter(time == &quot;24&quot;) %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(time, genes,median_H3K27ac_lfc) %&gt;% 
  dplyr::rename(&quot;med_Kac_24h_lfc&quot;=median_H3K27ac_lfc,, &quot;H3K27ac_peak&quot;=genes)
  
write_csv(median_3_lfc, &quot;data/Final_four_data/re_analysis/median_3_lfc_H3K27ac_norm.csv&quot;)
write_csv(median_24_lfc, &quot;data/Final_four_data/re_analysis/median_24_lfc_H3K27ac_norm.csv&quot;)</code></pre>
</div>
<div id="correlation-of-lfc-between-treatments" class="section level4">
<h4>Correlation of LFC between treatments</h4>
<pre class="r"><code>AC_matrix_ff &lt;- subset(efit2$coefficients)

colnames(AC_matrix_ff) &lt;-
  c(&quot;DNR\n3h&quot;,
    &quot;DOX\n3h&quot;,
    &quot;EPI\n3h&quot;,
    &quot;MTX\n3h&quot;,
    &quot;DNR\n24h&quot;,
    &quot;DOX\n24h&quot;,
    &quot;EPI\n24h&quot;,
    &quot;MTX\n24h&quot;
     )


mat_col_ac &lt;-
  data.frame(
    time = c(rep(&quot;3 hours&quot;, 4), rep(&quot;24 hours&quot;, 4)),
    class = (c(
      &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;,&quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot; 
    )))
rownames(mat_col_ac) &lt;- colnames(AC_matrix_ff)

mat_colors_ac &lt;-
  list(
    time = c(&quot;pink&quot;, &quot;chocolate4&quot;),
    class = c(&quot;yellow1&quot;, &quot;lightgreen&quot;))

names(mat_colors_ac$time) &lt;- unique(mat_col_ac$time)
names(mat_colors_ac$class) &lt;- unique(mat_col_ac$class)
# names(mat_colors_FC$TOP2i) &lt;- unique(mat_col_FC$TOP2i)
pearson_corr_AC&lt;- cor(AC_matrix_ff, method = &quot;pearson&quot;)
spearman_corr_AC &lt;- cor(AC_matrix_ff, method = &quot;spearman&quot;)

htanno_AC &lt;-  HeatmapAnnotation(df = mat_col_ac, col = mat_colors_ac)
Heatmap(pearson_corr_AC, top_annotation = htanno_AC,
        column_title = &quot;H3K27ac LFC Pearson Correlation&quot;)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/LFC%20correlation-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LFC-correlation-1">
Past versions of "LFC correlation-1.png"
</button>
</p>
<div id="fig-no-spaces-LFC-correlation-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/LFC correlation-1.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/LFC correlation-1.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Heatmap(spearman_corr_AC, top_annotation = htanno_AC,
        column_title = &quot;H3K27ac LFC Spearman Correlation&quot;)</code></pre>
<p><img src="figure/H3K27ac_initial_QC.Rmd/LFC%20correlation-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LFC-correlation-2">
Past versions of "LFC correlation-2.png"
</button>
</p>
<div id="fig-no-spaces-LFC-correlation-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/7fcb892c7e83a601d6e97ddae1e4bffe796baaeb/docs/figure/H3K27ac_initial_QC.Rmd/LFC correlation-2.png" target="_blank">7fcb892</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/035f37b150fbfbacfece2bfef45cd6cedac0b631/docs/figure/H3K27ac_initial_QC.Rmd/LFC correlation-2.png" target="_blank">035f37b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Raw counts without chrM (not doing)</p>
<pre class="r"><code>PCA_H3_mat_noM &lt;-raw_counts_only%&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  dplyr::filter(!grepl(&quot;^chrM&quot;, rownames(.))) %&gt;% 
  as.matrix()

hist(PCA_H3_mat_noM, main= &quot;Raw counts across samples&quot;,
     xlab = &quot;Raw counts&quot;,
     col=4)
hist(cpm(PCA_H3_mat_noM, log=TRUE), 
     main = expression(&quot;Histogram of Log&quot;[2]*&quot; cpm unfiltered&quot;),
     xlab = expression(&quot;Log&quot;[2]*&quot; counts-per-million&quot;),
     col=4)

boxplot(cpm(PCA_H3_mat_noM, log=TRUE),
        main=expression(&quot;Boxplot of Log&quot;[2]*&quot; counts-per-million without chrM&quot;),
                        col=4,
        names=colnames(PCA_H3_mat_noM),
        las=2, cex.axis=.7)</code></pre>
</div>
<div id="removing-outlier-nom" class="section level4">
<h4>Removing outlier noM</h4>
<p>Removing C_VEH_3 and B_VEH_24 from matrix and reanalyzing</p>
<pre class="r"><code>final_23_mat_noM &lt;- PCA_H3_mat_noM %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(!C_VEH_3) %&gt;% 
   dplyr::select(!B_VEH_24) %&gt;% 
  
    as.matrix()

hist(final_23_mat_noM, main= &quot;Raw counts across final samples_noM&quot;,
     xlab = &quot;Raw counts&quot;,
     col=6)
hist(cpm(final_23_mat_noM, log=TRUE), 
     main = expression(&quot;Histogram of Log&quot;[2]*&quot; cpm unfiltered final counts H3K27ac_noM&quot;),
     xlab = expression(&quot;Log&quot;[2]*&quot; counts-per-million&quot;),
     col=6)

boxplot(cpm(final_23_mat_noM, log=TRUE),
        main=expression(&quot;Boxplot of Log&quot;[2]*&quot; counts-per-million final H3K27ac samples_noM&quot;),
                        col=6,
        names=colnames(final_23_mat_noM),
        las=2, cex.axis=.7)

saveRDS(final_23_mat_noM, &quot;data/Final_four_data/re_analysis/H3K27ac_final_23_raw_counts_noM.RDS&quot;)</code></pre>
<pre class="r"><code>cor_raw_counts_noM &lt;- PCA_H3_mat_noM %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor(.,method = &quot;pearson&quot;)

anno_raw_counts_noM &lt;- data.frame(timeset = colnames(PCA_H3_mat))

counts_corr_mat_noM &lt;-anno_raw_counts_noM %&gt;%
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, 
                         if_else(trt == &quot;DOX&quot;, &quot;AC&quot;, 
                                 if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, 
                         if_else(trt == &quot;DOX&quot;, &quot;yes&quot;, 
                                 if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, 
                                         if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) 

                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#41B333&quot;),
   indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)


htanno_full &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat_noM, col = mat_colors)
Heatmap(cor_raw_counts_noM, top_annotation = htanno_full, column_title = &quot;H3K27ac log2 cpm unfiltered counts correlation noM&quot;)</code></pre>
</div>
<div id="filtering-check-1" class="section level4">
<h4>Filtering check</h4>
<pre class="r"><code>lcpm_f_noM &lt;- cpm(final_23_mat_noM, log= TRUE)
  ### for determining the basic cutoffs
filt_final_raw_counts_noM &lt;- final_23_mat_noM[rowMeans(lcpm_f_noM)&gt; 0,]
dim(filt_final_raw_counts_noM)</code></pre>
<p>Still no change in row number. Number of enriched regions is 20137,
23 (rows and columns)</p>
</div>
</div>
<div id="differentially-acetylated-enriched-regions-nom"
class="section level3">
<h3>Differentially acetylated enriched regions noM</h3>
<pre class="r"><code>groupset &lt;- colnames(final_23_mat_noM)
split_parts &lt;- strsplit(groupset, &quot;_&quot;)
group &lt;- sapply(split_parts, function(x) paste(x[2], x[3], sep = &quot;_&quot;))
indv &lt;- sapply(split_parts, function(x) paste(x[1]))

group &lt;- factor(group, levels=c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;VEH_24&quot;,&quot;VEH_3&quot;))

dge_noM &lt;-  DGEList.data.frame(counts = final_23_mat_noM, group = group, genes = row.names(final_23_mat_noM))
dge_noM &lt;- calcNormFactors(dge_noM)
dge_noM$samples</code></pre>
<p>Making model matrix</p>
<pre class="r"><code> mm &lt;- model.matrix(~0 +group)
colnames(mm) &lt;-  c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
mm</code></pre>
<pre class="r"><code>y &lt;- voom(dge_noM, mm,plot =FALSE)

corfit &lt;- duplicateCorrelation(y, mm, block = indv)
 
v &lt;- voom(dge_noM, mm, block = indv, correlation = corfit$consensus)

fit &lt;- lmFit(v, mm, block = indv, correlation = corfit$consensus)


cm &lt;- makeContrasts(
  DNR_3.VEH_3 = DNR_3-VEH_3,
  DOX_3.VEH_3 = DOX_3-VEH_3,
  EPI_3.VEH_3 = EPI_3-VEH_3,
  MTX_3.VEH_3 = MTX_3-VEH_3,
  DNR_24.VEH_24 =DNR_24-VEH_24,
  DOX_24.VEH_24= DOX_24-VEH_24,
  EPI_24.VEH_24= EPI_24-VEH_24,
  MTX_24.VEH_24= MTX_24-VEH_24,
  levels = mm)


fit2&lt;- contrasts.fit(fit, contrasts=cm)

efit2_noM &lt;- eBayes(fit2)

results_noM = decideTests(efit2_noM)

summary(results_noM)
plotSA(efit2_noM, main=&quot;Mean-Variance trend for final model of H3K27ac&quot;)</code></pre>
<pre class="r"><code>V.DNR_3.top= topTable(efit2_noM, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_3.top= topTable(efit2_noM, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_3.top= topTable(efit2_noM, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_3.top= topTable(efit2_noM, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DNR_24.top= topTable(efit2_noM, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_24.top= topTable(efit2_noM, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_24.top= topTable(efit2_noM, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_24.top= topTable(efit2_noM, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)


# plot_filenames &lt;- c(&quot;V.DNR_3.top&quot;,&quot;V.DOX_3.top&quot;,&quot;V.EPI_3.top&quot;,&quot;V.MTX_3.top&quot;,
#                     &quot;V.TRZ_.top&quot;,&quot;V.DNR_24.top&quot;,&quot;V.DOX_24.top&quot;,&quot;V.EPI_24.top&quot;,
#                     &quot;V.MTX_24.top&quot;,&quot;V.TRZ_24.top&quot;)
# plot_files &lt;- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
#                     V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
#                     V.MTX_24.top,V.TRZ_24.top)

save_list &lt;- list(&quot;DNR_3&quot;=V.DNR_3.top,&quot;DOX_3&quot;=V.DOX_3.top,&quot;EPI_3&quot;=V.EPI_3.top,&quot;MTX_3&quot;=V.MTX_3.top,&quot;DNR_24&quot;=V.DNR_24.top,&quot;DOX_24&quot;=V.DOX_24.top,&quot;EPI_24&quot;=V.EPI_24.top,&quot;MTX_24&quot;= V.MTX_24.top)

saveRDS(save_list,&quot;data/Final_four_data/re_analysis/Toptable_results_H3K27ac_data_noM.RDS&quot;)</code></pre>
<pre class="r"><code>volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
      # ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC &gt;= 0,&quot;B&quot;, &quot;C&quot;)))
    ##This is where I could add labels, but I have taken out
    # df &lt;- df %&gt;% mutate(genelabels = &quot;&quot;)
    # df$genelabels[1:topg] &lt;- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    geom_point(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, &quot;npc&quot;)), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    ylim(0,20)+
    xlim(-6,6)+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}

v1 &lt;- volcanosig(V.DNR_3.top, 0.05)+ ggtitle(&quot;DNR 3 hour&quot;)
v2 &lt;- volcanosig(V.DNR_24.top, 0.05)+ ggtitle(&quot;DNR 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(V.DOX_3.top, 0.05)+ ggtitle(&quot;DOX 3 hour&quot;)
v4 &lt;- volcanosig(V.DOX_24.top, 0.05)+ ggtitle(&quot;DOX 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(V.EPI_3.top, 0.05)+ ggtitle(&quot;EPI 3 hour&quot;)
v6 &lt;- volcanosig(V.EPI_24.top, 0.05)+ ggtitle(&quot;EPI 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(V.MTX_3.top, 0.05)+ ggtitle(&quot;MTX 3 hour&quot;)
v8 &lt;- volcanosig(V.MTX_24.top, 0.05)+ ggtitle(&quot;MTX 24 hour&quot;)+ylab(&quot;&quot;)


plot_grid(v1,v2,  rel_widths =c(1,1))
plot_grid(v3,v4,  rel_widths =c(1,1))
plot_grid(v5,v6,  rel_widths =c(1,1))
plot_grid(v7,v8,  rel_widths =c(1,1))</code></pre>
<div id="correlation-of-lfc-between-treatments-nom"
class="section level4">
<h4>Correlation of LFC between treatments noM</h4>
<pre class="r"><code>AC_matrix_ff_noM &lt;- subset(efit2_noM$coefficients)

colnames(AC_matrix_ff_noM) &lt;-
  c(&quot;DNR\n3h&quot;,
    &quot;DOX\n3h&quot;,
    &quot;EPI\n3h&quot;,
    &quot;MTX\n3h&quot;,
    &quot;DNR\n24h&quot;,
    &quot;DOX\n24h&quot;,
    &quot;EPI\n24h&quot;,
    &quot;MTX\n24h&quot;
     )


mat_col_ac_noM &lt;-
  data.frame(
    time = c(rep(&quot;3 hours&quot;, 4), rep(&quot;24 hours&quot;, 4)),
    class = (c(
      &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;,&quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot; 
    )))
rownames(mat_col_ac_noM) &lt;- colnames(AC_matrix_ff_noM)

mat_colors_ac_noM &lt;-
  list(
    time = c(&quot;pink&quot;, &quot;chocolate4&quot;),
    class = c(&quot;yellow1&quot;, &quot;lightgreen&quot;))

names(mat_colors_ac_noM$time) &lt;- unique(mat_col_ac_noM$time)
names(mat_colors_ac_noM$class) &lt;- unique(mat_col_ac_noM$class)
# names(mat_colors_FC$TOP2i) &lt;- unique(mat_col_FC$TOP2i)
pearson_corr_AC_noM&lt;- cor(AC_matrix_ff_noM, method = &quot;pearson&quot;)
spearman_corr_AC_noM &lt;- cor(AC_matrix_ff_noM, method = &quot;spearman&quot;)

htanno_AC_noM &lt;-  HeatmapAnnotation(df = mat_col_ac_noM, col = mat_colors_ac_noM)
Heatmap(pearson_corr_AC_noM, top_annotation = htanno_AC_noM,
        column_title = &quot;H3K27ac LFC Pearson Correlation&quot;)

Heatmap(spearman_corr_AC_noM, top_annotation = htanno_AC_noM,
        column_title = &quot;H3K27ac LFC Spearman Correlation&quot;)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] cowplot_1.1.3                           
 [2] smplot2_0.2.5                           
 [3] ComplexHeatmap_2.22.0                   
 [4] ggrepel_0.9.6                           
 [5] plyranges_1.26.0                        
 [6] ggsignif_0.6.4                          
 [7] genomation_1.38.0                       
 [8] eulerr_7.0.2                            
 [9] devtools_2.4.5                          
[10] usethis_3.1.0                           
[11] ggpubr_0.6.0                            
[12] BiocParallel_1.40.0                     
[13] scales_1.3.0                            
[14] VennDiagram_1.7.3                       
[15] futile.logger_1.4.3                     
[16] gridExtra_2.3                           
[17] edgeR_4.4.2                             
[18] limma_3.62.2                            
[19] rtracklayer_1.66.0                      
[20] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[21] GenomicFeatures_1.58.0                  
[22] AnnotationDbi_1.68.0                    
[23] Biobase_2.66.0                          
[24] GenomicRanges_1.58.0                    
[25] GenomeInfoDb_1.42.3                     
[26] IRanges_2.40.1                          
[27] S4Vectors_0.44.0                        
[28] BiocGenerics_0.52.0                     
[29] ChIPseeker_1.42.1                       
[30] RColorBrewer_1.1-3                      
[31] broom_1.0.7                             
[32] kableExtra_1.4.0                        
[33] lubridate_1.9.4                         
[34] forcats_1.0.0                           
[35] stringr_1.5.1                           
[36] dplyr_1.1.4                             
[37] purrr_1.0.4                             
[38] readr_2.1.5                             
[39] tidyr_1.3.1                             
[40] tibble_3.2.1                            
[41] ggplot2_3.5.1                           
[42] tidyverse_2.0.0                         
[43] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.5                               
  [2] matrixStats_1.5.0                      
  [3] bitops_1.0-9                           
  [4] enrichplot_1.26.6                      
  [5] httr_1.4.7                             
  [6] doParallel_1.0.17                      
  [7] profvis_0.4.0                          
  [8] tools_4.4.2                            
  [9] backports_1.5.0                        
 [10] R6_2.6.1                               
 [11] lazyeval_0.2.2                         
 [12] GetoptLong_1.0.5                       
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.2                            
 [15] cli_3.6.4                              
 [16] formatR_1.14                           
 [17] Cairo_1.6-2                            
 [18] labeling_0.4.3                         
 [19] sass_0.4.9                             
 [20] Rsamtools_2.22.0                       
 [21] systemfonts_1.2.1                      
 [22] yulab.utils_0.2.0                      
 [23] foreign_0.8-88                         
 [24] DOSE_4.0.0                             
 [25] svglite_2.1.3                          
 [26] R.utils_2.13.0                         
 [27] sessioninfo_1.2.3                      
 [28] plotrix_3.8-4                          
 [29] BSgenome_1.74.0                        
 [30] pwr_1.3-0                              
 [31] rstudioapi_0.17.1                      
 [32] impute_1.80.0                          
 [33] RSQLite_2.3.9                          
 [34] shape_1.4.6.1                          
 [35] generics_0.1.3                         
 [36] gridGraphics_0.5-1                     
 [37] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [38] BiocIO_1.16.0                          
 [39] vroom_1.6.5                            
 [40] gtools_3.9.5                           
 [41] car_3.1-3                              
 [42] GO.db_3.20.0                           
 [43] Matrix_1.7-3                           
 [44] ggbeeswarm_0.7.2                       
 [45] abind_1.4-8                            
 [46] R.methodsS3_1.8.2                      
 [47] lifecycle_1.0.4                        
 [48] whisker_0.4.1                          
 [49] yaml_2.3.10                            
 [50] carData_3.0-5                          
 [51] SummarizedExperiment_1.36.0            
 [52] gplots_3.2.0                           
 [53] qvalue_2.38.0                          
 [54] SparseArray_1.6.2                      
 [55] blob_1.2.4                             
 [56] promises_1.3.2                         
 [57] crayon_1.5.3                           
 [58] miniUI_0.1.1.1                         
 [59] ggtangle_0.0.6                         
 [60] lattice_0.22-6                         
 [61] KEGGREST_1.46.0                        
 [62] magick_2.8.5                           
 [63] pillar_1.10.1                          
 [64] knitr_1.49                             
 [65] fgsea_1.32.2                           
 [66] rjson_0.2.23                           
 [67] boot_1.3-31                            
 [68] codetools_0.2-20                       
 [69] fastmatch_1.1-6                        
 [70] glue_1.8.0                             
 [71] getPass_0.2-4                          
 [72] ggfun_0.1.8                            
 [73] data.table_1.17.0                      
 [74] remotes_2.5.0                          
 [75] vctrs_0.6.5                            
 [76] png_0.1-8                              
 [77] treeio_1.30.0                          
 [78] gtable_0.3.6                           
 [79] cachem_1.1.0                           
 [80] xfun_0.51                              
 [81] S4Arrays_1.6.0                         
 [82] mime_0.12                              
 [83] iterators_1.0.14                       
 [84] statmod_1.5.0                          
 [85] ellipsis_0.3.2                         
 [86] nlme_3.1-167                           
 [87] ggtree_3.14.0                          
 [88] bit64_4.6.0-1                          
 [89] rprojroot_2.0.4                        
 [90] bslib_0.9.0                            
 [91] vipor_0.4.7                            
 [92] rpart_4.1.24                           
 [93] KernSmooth_2.23-26                     
 [94] Hmisc_5.2-2                            
 [95] colorspace_2.1-1                       
 [96] DBI_1.2.3                              
 [97] nnet_7.3-20                            
 [98] seqPattern_1.38.0                      
 [99] ggrastr_1.0.2                          
[100] tidyselect_1.2.1                       
[101] processx_3.8.6                         
[102] bit_4.6.0                              
[103] compiler_4.4.2                         
[104] curl_6.2.1                             
[105] git2r_0.35.0                           
[106] htmlTable_2.4.3                        
[107] xml2_1.3.7                             
[108] DelayedArray_0.32.0                    
[109] checkmate_2.3.2                        
[110] caTools_1.18.3                         
[111] callr_3.7.6                            
[112] digest_0.6.37                          
[113] rmarkdown_2.29                         
[114] XVector_0.46.0                         
[115] base64enc_0.1-3                        
[116] htmltools_0.5.8.1                      
[117] pkgconfig_2.0.3                        
[118] MatrixGenerics_1.18.1                  
[119] fastmap_1.2.0                          
[120] GlobalOptions_0.1.2                    
[121] rlang_1.1.5                            
[122] htmlwidgets_1.6.4                      
[123] UCSC.utils_1.2.0                       
[124] shiny_1.10.0                           
[125] farver_2.1.2                           
[126] jquerylib_0.1.4                        
[127] zoo_1.8-13                             
[128] jsonlite_1.9.1                         
[129] GOSemSim_2.32.0                        
[130] R.oo_1.27.0                            
[131] RCurl_1.98-1.16                        
[132] magrittr_2.0.3                         
[133] Formula_1.2-5                          
[134] GenomeInfoDbData_1.2.13                
[135] ggplotify_0.1.2                        
[136] patchwork_1.3.0                        
[137] munsell_0.5.1                          
[138] Rcpp_1.0.14                            
[139] ape_5.8-1                              
[140] stringi_1.8.4                          
[141] zlibbioc_1.52.0                        
[142] plyr_1.8.9                             
[143] pkgbuild_1.4.6                         
[144] parallel_4.4.2                         
[145] Biostrings_2.74.1                      
[146] splines_4.4.2                          
[147] circlize_0.4.16                        
[148] hms_1.1.3                              
[149] locfit_1.5-9.12                        
[150] ps_1.9.0                               
[151] igraph_2.1.4                           
[152] reshape2_1.4.4                         
[153] pkgload_1.4.0                          
[154] futile.options_1.0.1                   
[155] XML_3.99-0.18                          
[156] evaluate_1.0.3                         
[157] lambda.r_1.2.4                         
[158] foreach_1.5.2                          
[159] tzdb_0.4.0                             
[160] httpuv_1.6.15                          
[161] clue_0.3-66                            
[162] gridBase_0.4-7                         
[163] xtable_1.8-4                           
[164] restfulr_0.0.15                        
[165] tidytree_0.4.6                         
[166] rstatix_0.7.2                          
[167] later_1.4.1                            
[168] viridisLite_0.4.2                      
[169] aplot_0.2.5                            
[170] beeswarm_0.4.0                         
[171] memoise_2.0.1                          
[172] GenomicAlignments_1.42.0               
[173] cluster_2.1.8.1                        
[174] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
