<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-06-13" />

<title>Jaspar_enrichment</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Jaspar_enrichment</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-06-13</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-06-13
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree3f7e1eb6f594dea759fb73fcb88dfc1daf65dbcftargetblank3f7e1eba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/3f7e1eb6f594dea759fb73fcb88dfc1daf65dbcf" target="_blank">3f7e1eb</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree3f7e1eb6f594dea759fb73fcb88dfc1daf65dbcftargetblank3f7e1eba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/3f7e1eb6f594dea759fb73fcb88dfc1daf65dbcf" target="_blank">3f7e1eb</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/Fastqc_results.Rmd
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Jaspar_motif.Rmd</code>) and HTML
(<code>docs/Jaspar_motif.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3f7e1eb6f594dea759fb73fcb88dfc1daf65dbcf/analysis/Jaspar_motif.Rmd" target="_blank">3f7e1eb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-13
</td>
<td>
adding updates to motifs
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/2d6d3d702badb7220f70140805e9e80dcd392ab7/analysis/Jaspar_motif.Rmd" target="_blank">2d6d3d7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-12
</td>
<td>
adding commit
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ba8611bae735472bb1aba81fbca238aa9f8ecc23/analysis/Jaspar_motif.Rmd" target="_blank">ba8611b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-02
</td>
<td>
updates of code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/af5dd6ae89a4654857ead44e7738d1b7056d384c/docs/Jaspar_motif.html" target="_blank">af5dd6a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0485646631e215e0067ec937c239b35c4dec7071/analysis/Jaspar_motif.Rmd" target="_blank">0485646</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
updated commit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/8b26fb3962797fec69567f429e4e31048b28d463/docs/Jaspar_motif.html" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/efbd424dde9c21c7a01bf6f170b0a01f7c6a303d/analysis/Jaspar_motif.Rmd" target="_blank">efbd424</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
first commit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/Jaspar_motif.html" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/c58f48cadf8044ec5f0185a23ba57a8607e76c32/analysis/Jaspar_motif.Rmd" target="_blank">c58f48c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
<td>
first updates
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
library(bedr)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)</code></pre>
<pre class="r"><code># xstreme_ESR_v_NR &lt;-ESR_v_NR_full_xstreme %&gt;%
#   slice_head(n = length(.$ID)-3)
# saveRDS(xstreme_ESR_v_NR, &quot;data/xstreme/xstreme_ESR_v_NR.RDS&quot;)
# sea_EAR &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/full peak set Jaspar core vertebrates/EAR_v_NR_appXSTREME_5.5.51713891248048-1903497910.tar/appXSTREME_5.5.51713891248048-1903497910/sea_disc_out/sea.tsv&quot;, 
#     delim = &quot;\t&quot;, escape_double = FALSE, 
#     trim_ws = TRUE)%&gt;%
#   slice_head(n = length(.$ID)-3)
# saveRDS(sea_EAR, &quot;data/xstreme/sea_EAR.RDS&quot;)
# sea_ESR &lt;- sea_ESR%&gt;%
#   slice_head(n = length(.$ID)-3)
# saveRDS(sea_ESR, &quot;data/xstreme/sea_ESR.RDS&quot;)
# sea_LR &lt;- sea_LR%&gt;%
#   slice_head(n = length(.$ID)-3)
# saveRDS(sea_LR, &quot;data/xstreme/sea_LR.RDS&quot;)

sea_EAR &lt;- readRDS(&quot;data/xstreme/sea_EAR.RDS&quot;)
sea_ESR &lt;- readRDS(&quot;data/xstreme/sea_ESR.RDS&quot;)
sea_LR &lt;- readRDS(&quot;data/xstreme/sea_LR.RDS&quot;)


xstreme_EAR_v_NR &lt;- readRDS(&quot;data/xstreme/xstreme_EAR_v_NR.RDS&quot;)

xstreme_ESR_v_NR &lt;-readRDS(&quot;data/xstreme/xstreme_ESR_v_NR.RDS&quot;) 

xstreme_LR_v_NR &lt;- readRDS(&quot;data/xstreme/xstreme_LR_v_NR.RDS&quot;)</code></pre>
<pre class="r"><code>xstreme_EAR_v_NR %&gt;% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
  # separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  # dplyr::filter(., EVALUE&lt;0.05) %&gt;% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  kable(., caption = &quot;Enriched motifs in EAR v NR&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Enriched motifs in EAR v NR
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SIM_MOTIF
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ALT_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SEA_PVALUE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
EVALUE
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
TGTGTRTGTGTGTGT
</td>
<td style="text-align:left;">
MEME-2
</td>
<td style="text-align:left;">
TGTGTRTGTGTGTGT
</td>
<td style="text-align:left;">
0.000919
</td>
<td style="text-align:left;">
2.3e-82
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0682.2 (PITX1)
</td>
<td style="text-align:left;">
MEME-1
</td>
<td style="text-align:left;">
GCTGGGATTACAGGC
</td>
<td style="text-align:left;">
0.00423
</td>
<td style="text-align:left;">
6.2e-47
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1125.1 (ZNF384)
</td>
<td style="text-align:left;">
MEME-3
</td>
<td style="text-align:left;">
TTTTTTTTTTTTWKT
</td>
<td style="text-align:left;">
0.655
</td>
<td style="text-align:left;">
3.1e-45
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1596.1 (ZNF460)
</td>
<td style="text-align:left;">
MEME-4
</td>
<td style="text-align:left;">
CTCRGCCTCCCRARK
</td>
<td style="text-align:left;">
0.00022
</td>
<td style="text-align:left;">
1.5e-29
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0508.3 (PRDM1)
</td>
<td style="text-align:left;">
MEME-5
</td>
<td style="text-align:left;">
RGAGRGAGRGA
</td>
<td style="text-align:left;">
0.145
</td>
<td style="text-align:left;">
5.3e-26
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1973.1 (ZKSCAN3)
</td>
<td style="text-align:left;">
MEME-6
</td>
<td style="text-align:left;">
CCAGCCTGGGCRACA
</td>
<td style="text-align:left;">
3.13e-09
</td>
<td style="text-align:left;">
5.2e-22
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0119.1 (NFIC::TLX1)
</td>
<td style="text-align:left;">
MEME-8
</td>
<td style="text-align:left;">
GGCRYGGTGGCTCAY
</td>
<td style="text-align:left;">
0.0256
</td>
<td style="text-align:left;">
1e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1721.1 (ZNF93)
</td>
<td style="text-align:left;">
MEME-7
</td>
<td style="text-align:left;">
GSGGCGGSGGCGGSG
</td>
<td style="text-align:left;">
0.219
</td>
<td style="text-align:left;">
7.8e-07
</td>
</tr>
<tr>
<td style="text-align:left;">
TTGCAGTGAGCYGAG
</td>
<td style="text-align:left;">
MEME-9
</td>
<td style="text-align:left;">
TTGCAGTGAGCYGAG
</td>
<td style="text-align:left;">
1.94e-09
</td>
<td style="text-align:left;">
2.4e-05
</td>
</tr>
<tr>
<td style="text-align:left;">
1-AGATGACA
</td>
<td style="text-align:left;">
STREME-1
</td>
<td style="text-align:left;">
1-AGATGACA
</td>
<td style="text-align:left;">
1.65e-07
</td>
<td style="text-align:left;">
0.0355
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0805.1 (TBX1)
</td>
<td style="text-align:left;">
STREME-2
</td>
<td style="text-align:left;">
2-TCACACAT
</td>
<td style="text-align:left;">
2.24e-07
</td>
<td style="text-align:left;">
0.0373
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0036.3 (GATA2)
</td>
<td style="text-align:left;">
STREME-3
</td>
<td style="text-align:left;">
3-AKAGATA
</td>
<td style="text-align:left;">
0.000699
</td>
<td style="text-align:left;">
0.0593
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0038.2 (GFI1)
</td>
<td style="text-align:left;">
STREME-4
</td>
<td style="text-align:left;">
4-CTCACTGCAA
</td>
<td style="text-align:left;">
3.94e-14
</td>
<td style="text-align:left;">
2.21
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1645.1 (NKX2-2)
</td>
<td style="text-align:left;">
STREME-5
</td>
<td style="text-align:left;">
5-AGGCTTGAGT
</td>
<td style="text-align:left;">
4.5e-09
</td>
<td style="text-align:left;">
2.98
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1988.1 (Atf3)
</td>
<td style="text-align:left;">
STREME-6
</td>
<td style="text-align:left;">
6-GTKACTYRNC
</td>
<td style="text-align:left;">
0.000199
</td>
<td style="text-align:left;">
4.26
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>xstreme_ESR_v_NR %&gt;% 
 dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %&gt;% 
   arrange(.,EVALUE) %&gt;% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  # separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  # dplyr::filter(., EVALUE&lt;0.05) %&gt;% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  kable(., caption = &quot;Enriched motifs in ESR v NR&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Enriched motifs in ESR v NR
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SIM_MOTIF
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ALT_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SEA_PVALUE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
EVALUE
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
GTGTGTGTRTGTGTG
</td>
<td style="text-align:left;">
MEME-3
</td>
<td style="text-align:left;">
GTGTGTGTRTGTGTG
</td>
<td style="text-align:left;">
9.06e-52
</td>
<td style="text-align:left;">
1.2e-165
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1990.1 (Gli1)
</td>
<td style="text-align:left;">
MEME-1
</td>
<td style="text-align:left;">
TGGTGTGTGGTGTGT
</td>
<td style="text-align:left;">
1.44e-41
</td>
<td style="text-align:left;">
4.2e-135
</td>
</tr>
<tr>
<td style="text-align:left;">
TGTGTGYGTGTGTGT
</td>
<td style="text-align:left;">
MEME-5
</td>
<td style="text-align:left;">
TGTGTGYGTGTGTGT
</td>
<td style="text-align:left;">
3e-41
</td>
<td style="text-align:left;">
2.7e-100
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1155.1 (ZSCAN4)
</td>
<td style="text-align:left;">
MEME-2
</td>
<td style="text-align:left;">
GTGTGTGTGTGTG
</td>
<td style="text-align:left;">
1.66e-53
</td>
<td style="text-align:left;">
1.7e-84
</td>
</tr>
<tr>
<td style="text-align:left;">
CCCTCAYCRAGWCCC
</td>
<td style="text-align:left;">
MEME-6
</td>
<td style="text-align:left;">
CCCTCAYCRAGWCCC
</td>
<td style="text-align:left;">
7.39e-05
</td>
<td style="text-align:left;">
9.9e-75
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1125.1 (ZNF384)
</td>
<td style="text-align:left;">
MEME-4
</td>
<td style="text-align:left;">
MAAAAAAAAAAAAAA
</td>
<td style="text-align:left;">
0.839
</td>
<td style="text-align:left;">
7.2e-49
</td>
</tr>
<tr>
<td style="text-align:left;">
TGTGTGYATGTGTGT
</td>
<td style="text-align:left;">
MEME-9
</td>
<td style="text-align:left;">
TGTGTGYATGTGTGT
</td>
<td style="text-align:left;">
1.17e-43
</td>
<td style="text-align:left;">
2.1e-40
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0812.1 (TFAP2B)
</td>
<td style="text-align:left;">
MEME-10
</td>
<td style="text-align:left;">
CCWGCCTSRGCCWCM
</td>
<td style="text-align:left;">
0.00224
</td>
<td style="text-align:left;">
7.2e-29
</td>
</tr>
<tr>
<td style="text-align:left;">
AGRCGCTCCTCACTT
</td>
<td style="text-align:left;">
MEME-7
</td>
<td style="text-align:left;">
AGRCGCTCCTCACTT
</td>
<td style="text-align:left;">
4.12e-10
</td>
<td style="text-align:left;">
1.4e-16
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0682.2 (PITX1)
</td>
<td style="text-align:left;">
MEME-8
</td>
<td style="text-align:left;">
GCTGGGATTACAGGC
</td>
<td style="text-align:left;">
1.03e-12
</td>
<td style="text-align:left;">
8.8e-16
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1107.2 (KLF9)
</td>
<td style="text-align:left;">
STREME-1
</td>
<td style="text-align:left;">
1-CCCACACC
</td>
<td style="text-align:left;">
5e-45
</td>
<td style="text-align:left;">
0.0208
</td>
</tr>
<tr>
<td style="text-align:left;">
2-CAGACACAT
</td>
<td style="text-align:left;">
STREME-2
</td>
<td style="text-align:left;">
2-CAGACACAT
</td>
<td style="text-align:left;">
1.09e-32
</td>
<td style="text-align:left;">
0.046
</td>
</tr>
<tr>
<td style="text-align:left;">
3-GACTGCCTCCTCA
</td>
<td style="text-align:left;">
STREME-3
</td>
<td style="text-align:left;">
3-GACTGCCTCCTCA
</td>
<td style="text-align:left;">
3.48e-10
</td>
<td style="text-align:left;">
0.047
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1710.1 (ZNF257)
</td>
<td style="text-align:left;">
STREME-4
</td>
<td style="text-align:left;">
4-CCTCCTGCCTCAKYM
</td>
<td style="text-align:left;">
7.33e-08
</td>
<td style="text-align:left;">
0.047
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1155.1 (ZSCAN4)
</td>
<td style="text-align:left;">
STREME-5
</td>
<td style="text-align:left;">
5-GTGTGTGA
</td>
<td style="text-align:left;">
2.18e-27
</td>
<td style="text-align:left;">
0.0616
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1990.1 (Gli1)
</td>
<td style="text-align:left;">
STREME-6
</td>
<td style="text-align:left;">
6-CACMCATS
</td>
<td style="text-align:left;">
1.36e-64
</td>
<td style="text-align:left;">
0.0682
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1643.1 (NFIB)
</td>
<td style="text-align:left;">
STREME-7
</td>
<td style="text-align:left;">
7-CTGTGCCA
</td>
<td style="text-align:left;">
4.08e-60
</td>
<td style="text-align:left;">
0.0703
</td>
</tr>
<tr>
<td style="text-align:left;">
8-ACATGCACA
</td>
<td style="text-align:left;">
STREME-8
</td>
<td style="text-align:left;">
8-ACATGCACA
</td>
<td style="text-align:left;">
1.35e-39
</td>
<td style="text-align:left;">
0.269
</td>
</tr>
<tr>
<td style="text-align:left;">
9-KGGSCCM
</td>
<td style="text-align:left;">
STREME-9
</td>
<td style="text-align:left;">
9-KGGSCCM
</td>
<td style="text-align:left;">
1.03e-37
</td>
<td style="text-align:left;">
1.84
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>xstreme_LR_v_NR %&gt;% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  # separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  # dplyr::filter(., EVALUE&lt;0.05) %&gt;% 
  kable(., caption = &quot;Enriched motifs  (EVALUE &lt; 0.05) in LR v NR&quot;) %&gt;%
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;800px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:800px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Enriched motifs (EVALUE &lt; 0.05) in LR v NR
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SIM_MOTIF
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ALT_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SEA_PVALUE
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
EVALUE
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MA1125.1 (ZNF384)
</td>
<td style="text-align:left;">
MEME-1
</td>
<td style="text-align:left;">
AAAAWAAAAAAAWWA
</td>
<td style="text-align:left;">
3.2e-21
</td>
<td style="text-align:left;">
6.5e-70
</td>
</tr>
<tr>
<td style="text-align:left;">
AYACACATAYACACA
</td>
<td style="text-align:left;">
MEME-3
</td>
<td style="text-align:left;">
AYACACATAYACACA
</td>
<td style="text-align:left;">
2.89e-31
</td>
<td style="text-align:left;">
1.6e-57
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1125.1 (ZNF384)
</td>
<td style="text-align:left;">
MEME-2
</td>
<td style="text-align:left;">
AAAAAAAAAAAAAAA
</td>
<td style="text-align:left;">
5.64e-10
</td>
<td style="text-align:left;">
1.2e-43
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1547.2 (PITX2)
</td>
<td style="text-align:left;">
MEME-5
</td>
<td style="text-align:left;">
TAATCCCAGCACTTT
</td>
<td style="text-align:left;">
6.86e-72
</td>
<td style="text-align:left;">
5.9e-35
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1125.1 (ZNF384)
</td>
<td style="text-align:left;">
MEME-4
</td>
<td style="text-align:left;">
AAAAADAAAAA
</td>
<td style="text-align:left;">
5.53e-34
</td>
<td style="text-align:left;">
8.8e-30
</td>
</tr>
<tr>
<td style="text-align:left;">
TGTGTGGTRTGKKGT
</td>
<td style="text-align:left;">
MEME-7
</td>
<td style="text-align:left;">
TGTGTGGTRTGKKGT
</td>
<td style="text-align:left;">
3.86e-49
</td>
<td style="text-align:left;">
5e-26
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0477.2 (FOSL1)
</td>
<td style="text-align:left;">
STREME-1
</td>
<td style="text-align:left;">
1-DDATGASTCATHH
</td>
<td style="text-align:left;">
9.94e-305
</td>
<td style="text-align:left;">
2.38e-23
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1101.2 (BACH2)
</td>
<td style="text-align:left;">
MEME-6
</td>
<td style="text-align:left;">
AAWGGGATGAKTCAY
</td>
<td style="text-align:left;">
1.96e-305
</td>
<td style="text-align:left;">
1.8e-18
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0052.4 (MEF2A)
</td>
<td style="text-align:left;">
MEME-8
</td>
<td style="text-align:left;">
ATTTATTTTWA
</td>
<td style="text-align:left;">
2.68e-53
</td>
<td style="text-align:left;">
3e-04
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0610.1 (DMRT3)
</td>
<td style="text-align:left;">
STREME-2
</td>
<td style="text-align:left;">
2-AATACA
</td>
<td style="text-align:left;">
1.39e-232
</td>
<td style="text-align:left;">
0.000485
</td>
</tr>
<tr>
<td style="text-align:left;">
3-WCATGW
</td>
<td style="text-align:left;">
STREME-3
</td>
<td style="text-align:left;">
3-WCATGW
</td>
<td style="text-align:left;">
5.83e-245
</td>
<td style="text-align:left;">
0.000669
</td>
</tr>
<tr>
<td style="text-align:left;">
4-ATAKMTAT
</td>
<td style="text-align:left;">
STREME-4
</td>
<td style="text-align:left;">
4-ATAKMTAT
</td>
<td style="text-align:left;">
1.19e-167
</td>
<td style="text-align:left;">
0.00769
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1124.1 (ZNF24)
</td>
<td style="text-align:left;">
STREME-5
</td>
<td style="text-align:left;">
5-TSATKCA
</td>
<td style="text-align:left;">
2.04e-248
</td>
<td style="text-align:left;">
0.00806
</td>
</tr>
<tr>
<td style="text-align:left;">
MA1487.2 (FOXE1)
</td>
<td style="text-align:left;">
STREME-6
</td>
<td style="text-align:left;">
6-CAAACAA
</td>
<td style="text-align:left;">
4.32e-102
</td>
<td style="text-align:left;">
0.00999
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0018.4 (CREB1)
</td>
<td style="text-align:left;">
STREME-7
</td>
<td style="text-align:left;">
7-ACATCA
</td>
<td style="text-align:left;">
1.01e-223
</td>
<td style="text-align:left;">
0.0263
</td>
</tr>
<tr>
<td style="text-align:left;">
8-ACTTAT
</td>
<td style="text-align:left;">
STREME-8
</td>
<td style="text-align:left;">
8-ACTTAT
</td>
<td style="text-align:left;">
2.5e-238
</td>
<td style="text-align:left;">
0.0446
</td>
</tr>
<tr>
<td style="text-align:left;">
9-ATGATA
</td>
<td style="text-align:left;">
STREME-9
</td>
<td style="text-align:left;">
9-ATGATA
</td>
<td style="text-align:left;">
1.24e-175
</td>
<td style="text-align:left;">
0.0509
</td>
</tr>
<tr>
<td style="text-align:left;">
MA0083.3 (SRF)
</td>
<td style="text-align:left;">
STREME-10
</td>
<td style="text-align:left;">
10-TATATG
</td>
<td style="text-align:left;">
5.41e-217
</td>
<td style="text-align:left;">
0.0874
</td>
</tr>
<tr>
<td style="text-align:left;">
11-TATWATA
</td>
<td style="text-align:left;">
STREME-11
</td>
<td style="text-align:left;">
11-TATWATA
</td>
<td style="text-align:left;">
7.44e-205
</td>
<td style="text-align:left;">
0.0914
</td>
</tr>
<tr>
<td style="text-align:left;">
12-ATATGAR
</td>
<td style="text-align:left;">
STREME-12
</td>
<td style="text-align:left;">
12-ATATGAR
</td>
<td style="text-align:left;">
8.12e-205
</td>
<td style="text-align:left;">
0.185
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>### selecting just the list of names and checking for expression


gene_list_SEA &lt;- xstreme_EAR_v_NR %&gt;% 
 dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  mutate(MCR=&quot;EAR&quot;) %&gt;% 
  bind_rows(xstreme_ESR_v_NR %&gt;% 
 dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
 mutate(MCR=&quot;ESR&quot;)) %&gt;% 
  bind_rows(xstreme_LR_v_NR %&gt;% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
    mutate(MCR=&quot;LR&quot;)) %&gt;% 
  dplyr::filter(!is.na(NAME)) %&gt;%  
  mutate(NAME= gsub(&quot;[()]&quot;,&quot;&quot;,NAME)) %&gt;% 
 distinct(NAME)</code></pre>
<pre class="r"><code>EAR_sea &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/EAR_sea.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
   trim_ws = TRUE)
ESR_sea &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/ESR_sea.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
LR_sea &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/LR_sea.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)


# length(intersect(EAR_sea$ID,ESR_sea$ID))
# length(unique(EAR_sea$ID))
# length(unique(ESR_sea$ID))
# length(unique(LR_sea$ID))
# head(EAR_sea,n=20)
EAR_sea %&gt;% 
  slice_head(n=20) %&gt;% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
  geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+
  ggtitle(&quot;EAR SEA top 20 motifs&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/SEA-checks%20notthe%20same%20as%20sea_EAR!-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ESR_sea %&gt;% 
  slice_head(n=20) %&gt;% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
   geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+
   ggtitle(&quot;ESR SEA top 20 motifs&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/SEA-checks%20notthe%20same%20as%20sea_EAR!-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>LR_sea %&gt;% 
  slice_head(n=20) %&gt;% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
   geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+ 
  ggtitle(&quot;LR SEA top 20 motifs&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/SEA-checks%20notthe%20same%20as%20sea_EAR!-3.png" width="672" style="display: block; margin: auto;" /></p>
<div id="looking-at-sea-data" class="section level3">
<h3>Looking at SEA data</h3>
<pre class="r"><code>toplistall_RNA &lt;- readRDS(&quot;data/other_papers/toplistall_RNA.RDS&quot;) 
###Because of how I applied the DEG system in RNA-seq analysis, the lFC is opposite of the 
###counts.   I did trt-veh instead of veh-trt.  therefore I need to multiply lfc by -1 to get t
###the right correlation.

toplistall_RNA &lt;- toplistall_RNA %&gt;% 
  mutate(logFC = logFC*(-1))

RNA_expresed_genes &lt;- toplistall_RNA %&gt;% 
  # dplyr::filter(adj.P.Val &lt;0.05) %&gt;% 
   mutate(expression = if_else(logFC&lt;0,&quot;down&quot;,&quot;up&quot;)) %&gt;% 
  dplyr::select(ENTREZID,SYMBOL,expression) %&gt;% 
  # dplyr::select(ENTREZID,SYMBOL) %&gt;% 
  unique(.)  
RNA_expresed_genes_DE &lt;- toplistall_RNA %&gt;% 
  dplyr::filter(adj.P.Val &lt;0.05) %&gt;%
  mutate(expression = if_else(logFC&lt;0,&quot;down&quot;,&quot;up&quot;)) %&gt;% 
  dplyr::select(ENTREZID,SYMBOL,expression) %&gt;% 
  unique(.) 

motifgenes_EAR_v_NR &lt;- xstreme_EAR_v_NR %&gt;% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SITES,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  dplyr::filter(., EVALUE&lt;0.05) %&gt;% 
  mutate(motif_name= gsub(&quot;[()]&quot;,&quot;&quot;,NAME)) %&gt;% 
   dplyr::select(SIM_MOTIF,motif_name,SITES) %&gt;% 
  unique()

motifgenes_ESR_v_NR &lt;- xstreme_ESR_v_NR %&gt;% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SITES,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  dplyr::filter(., EVALUE&lt;0.05) %&gt;% 
  mutate(motif_name= gsub(&quot;[()]&quot;,&quot;&quot;,NAME)) %&gt;% 
   dplyr::select(SIM_MOTIF,motif_name,SITES) %&gt;% 
  unique()


motifgenes_LR_v_NR &lt;- xstreme_LR_v_NR %&gt;% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SITES,EVALUE) %&gt;% 
  arrange(.,EVALUE) %&gt;% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  dplyr::filter(., EVALUE&lt;0.05) %&gt;% 
  mutate(motif_name= gsub(&quot;[()]&quot;,&quot;&quot;,NAME)) %&gt;% 
   dplyr::select(SIM_MOTIF,motif_name) %&gt;% 
  unique()

gene_list_SEA %&gt;%
  mutate(express_RNA=if_else(NAME %in%  RNA_expresed_genes$SYMBOL,&quot;exp&quot;,&quot;not_exp&quot;),DE_RNA=if_else(NAME %in% RNA_expresed_genes_DE$SYMBOL,&quot;DE&quot;,&quot;notDE&quot;)) %&gt;%
  dplyr::filter(express_RNA != &quot;not_exp&quot;) %&gt;%
  rowid_to_column(&quot;row_name&quot;) %&gt;%
  kable(., caption= &quot;Enriched motifs annotated with RNA expression status&quot;) %&gt;%
    kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 16) %&gt;%
  scroll_box(height = &quot;1000px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:1000px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 16px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Enriched motifs annotated with RNA expression status
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
row_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
NAME
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
express_RNA
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE_RNA
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
ZNF384
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
notDE
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
ZNF460
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
notDE
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
PRDM1
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
ZKSCAN3
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
ZNF93
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
GATA2
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
KLF9
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
notDE
</td>
</tr>
<tr>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
NFIB
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
PITX2
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
notDE
</td>
</tr>
<tr>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
FOSL1
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
BACH2
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
MEF2A
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
notDE
</td>
</tr>
<tr>
<td style="text-align:right;">
13
</td>
<td style="text-align:left;">
ZNF24
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
notDE
</td>
</tr>
<tr>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;">
CREB1
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
<tr>
<td style="text-align:right;">
15
</td>
<td style="text-align:left;">
SRF
</td>
<td style="text-align:left;">
exp
</td>
<td style="text-align:left;">
DE
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)
# list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)
EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr &lt;-  GRanges(EAR_df)

ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr &lt;-  GRanges(ESR_df)

LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr &lt;-  GRanges(LR_df)
 
NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr &lt;-  GRanges(NR_df)


# ESR_peaks_resized = resize(ESR_df_gr, width = 2000, fix=&#39;center&#39;)
# EAR_peaks_resized = resize(EAR_df_gr, width = 2000, fix=&#39;center&#39;)
# LR_peaks_resized = resize(LR_df_gr, width = 2000, fix=&#39;center&#39;)
# NR_peaks_resized = resize(NR_df_gr, width = 2000, fix=&#39;center&#39;)

ESR_peaks_resized_250 = resize(ESR_df_gr, width = 500, fix=&#39;center&#39;)
EAR_peaks_resized_250 = resize(EAR_df_gr, width = 500, fix=&#39;center&#39;)
LR_peaks_resized_250 = resize(LR_df_gr, width = 500, fix=&#39;center&#39;)
NR_peaks_resized_250 = resize(NR_df_gr, width =500, fix=&#39;center&#39;)

seq_ESR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_peaks_resized_250)
seq_EAR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_peaks_resized_250)
seq_LR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_peaks_resized_250)
seq_NR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_peaks_resized_250)

# 
# ESR_peaks_resized_500 = resize(ESR_df_gr, width = 1000, fix=&#39;center&#39;)
# EAR_peaks_resized_500 = resize(EAR_df_gr, width = 1000, fix=&#39;center&#39;)
# LR_peaks_resized_500 = resize(LR_df_gr, width = 1000, fix=&#39;center&#39;)
# NR_peaks_resized_500 = resize(NR_df_gr, width = 1000, fix=&#39;center&#39;)
# ESR_peaks_resized_site = resize(ESR_df_gr,  fix=&#39;center&#39;)

# seq_ESR = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_peaks_resized)
# seq_EAR = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_peaks_resized)
# seq_LR = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_peaks_resized)
# seq_NR = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_peaks_resized)
# 
# seq_ESR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_peaks_resized_500)
# seq_EAR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_peaks_resized_500)
# seq_LR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_peaks_resized_500)
# seq_NR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_peaks_resized_500)
# 
# seq_ESR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_df_gr)
# seq_EAR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_df_gr)
# seq_LR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_df_gr)
# seq_NR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_df_gr)</code></pre>
<pre class="r"><code>EAR_passing_sequences &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/EAR_passing_sequences.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_types = cols(motif_DB = col_skip()), 
    trim_ws = TRUE)

motif_EAR_list &lt;- EAR_passing_sequences %&gt;% 
   distinct(motif_ID, .keep_all = TRUE) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(motif_ID, motif_ALT_ID)
            
expr_EAR_tfs &lt;- data.frame(EAR_TFs=intersect(motif_EAR_list$motif_ALT_ID, unique(RNA_expresed_genes$SYMBOL))) 

ESR_passing_sequences &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/ESR_passing_sequences.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_types = cols(motif_DB = col_skip()), 
    trim_ws = TRUE)
motif_ESR_list &lt;- ESR_passing_sequences %&gt;% 
   distinct(motif_ID, .keep_all = TRUE) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(motif_ID, motif_ALT_ID)


expr_ESR_tfs &lt;- data.frame(ESR_TFs=intersect(motif_ESR_list$motif_ALT_ID, unique(RNA_expresed_genes$SYMBOL))) 

intersect(expr_ESR_tfs$ESR_TFs, expr_EAR_tfs$EAR_TFs)</code></pre>
<pre><code>  [1] &quot;MEF2A&quot;   &quot;FOS&quot;     &quot;FOSL2&quot;   &quot;ZNF460&quot;  &quot;FOSL1&quot;   &quot;MEF2D&quot;   &quot;JUNB&quot;   
  [8] &quot;BACH1&quot;   &quot;MEF2C&quot;   &quot;JUND&quot;    &quot;BNC2&quot;    &quot;BACH2&quot;   &quot;JDP2&quot;    &quot;ZNF135&quot; 
 [15] &quot;ZNF281&quot;  &quot;KLF4&quot;    &quot;KLF5&quot;    &quot;GATA2&quot;   &quot;KLF7&quot;    &quot;GATA4&quot;   &quot;KLF10&quot;  
 [22] &quot;SP1&quot;     &quot;SP4&quot;     &quot;SP2&quot;     &quot;KLF12&quot;   &quot;TEAD1&quot;   &quot;TEAD2&quot;   &quot;TRPS1&quot;  
 [29] &quot;KLF9&quot;    &quot;GATA5&quot;   &quot;RREB1&quot;   &quot;NFIB&quot;    &quot;ZNF384&quot;  &quot;MAFK&quot;    &quot;TEAD4&quot;  
 [36] &quot;TEAD3&quot;   &quot;GATA6&quot;   &quot;KLF15&quot;   &quot;ESRRA&quot;   &quot;FOXL1&quot;   &quot;ZNF148&quot;  &quot;IRF3&quot;   
 [43] &quot;CTCF&quot;    &quot;ESRRB&quot;   &quot;KLF11&quot;   &quot;SP3&quot;     &quot;FOXC1&quot;   &quot;KLF2&quot;    &quot;PATZ1&quot;  
 [50] &quot;FOXO4&quot;   &quot;ZNF530&quot;  &quot;NR2C1&quot;   &quot;ATF2&quot;    &quot;ZNF354A&quot; &quot;FOXP1&quot;   &quot;FOXC2&quot;  
 [57] &quot;KLF16&quot;   &quot;EGR1&quot;    &quot;MAZ&quot;     &quot;NFIC&quot;    &quot;ZKSCAN5&quot; &quot;CREB1&quot;   &quot;ZNF93&quot;  
 [64] &quot;NR2F2&quot;   &quot;ELF3&quot;    &quot;RFX2&quot;    &quot;THRA&quot;    &quot;ZNF24&quot;   &quot;RFX1&quot;    &quot;NFIX&quot;   
 [71] &quot;ZNF263&quot;  &quot;JUN&quot;     &quot;FOXO6&quot;   &quot;RFX3&quot;    &quot;KLF6&quot;    &quot;FOXK2&quot;   &quot;NR2C2&quot;  
 [78] &quot;FOXK1&quot;   &quot;KLF3&quot;    &quot;E2F6&quot;    &quot;RFX5&quot;    &quot;RORA&quot;    &quot;MLX&quot;     &quot;CREM&quot;   
 [85] &quot;ELF1&quot;    &quot;GABPA&quot;   &quot;NR6A1&quot;   &quot;IRF7&quot;    &quot;POU6F1&quot;  &quot;ETV1&quot;    &quot;ZNF449&quot; 
 [92] &quot;PBX2&quot;    &quot;MAFF&quot;    &quot;MYC&quot;     &quot;RFX7&quot;    &quot;ZBTB6&quot;   &quot;POU3F1&quot;  &quot;MYCN&quot;   
 [99] &quot;USF1&quot;    &quot;ATF7&quot;    &quot;MITF&quot;    &quot;PKNOX1&quot;  &quot;ATF3&quot;    &quot;LIN54&quot;   &quot;ZNF740&quot; 
[106] &quot;PBX3&quot;    &quot;ZNF701&quot;  &quot;SOX4&quot;    &quot;PRDM1&quot;   &quot;ZFP14&quot;   &quot;STAT3&quot;   &quot;THRB&quot;   
[113] &quot;MEIS1&quot;   &quot;THAP1&quot;   &quot;TCF7L1&quot;  &quot;CLOCK&quot;   &quot;MEIS2&quot;   &quot;RBPJ&quot;    &quot;SREBF2&quot; 
[120] &quot;HEY1&quot;    &quot;ELK4&quot;    &quot;PITX2&quot;   &quot;CEBPG&quot;   &quot;TFE3&quot;    &quot;ZNF324&quot;  &quot;ATF4&quot;   
[127] &quot;RARA&quot;    &quot;CREB3L4&quot; &quot;EGR2&quot;    &quot;ZNF416&quot;  &quot;MNT&quot;     &quot;USF2&quot;    &quot;TFEB&quot;   
[134] &quot;MAFA&quot;    &quot;REST&quot;    &quot;MAX&quot;     &quot;IRF2&quot;    &quot;POU2F2&quot;  &quot;NRL&quot;     &quot;ARNT2&quot;  
[141] &quot;IRF9&quot;    &quot;HEY2&quot;    &quot;BHLHE41&quot; &quot;EGR3&quot;    &quot;TCF7&quot;    &quot;TCF7L2&quot;  &quot;POU2F1&quot; 
[148] &quot;HOXB5&quot;   &quot;MAF&quot;     &quot;ZNF669&quot;  &quot;ELF4&quot;    &quot;TEF&quot;     &quot;NR4A2&quot;   &quot;ETV6&quot;   
[155] &quot;NFIL3&quot;   &quot;DMRTA1&quot;  &quot;ZNF680&quot;  &quot;ZNF610&quot;  &quot;SATB1&quot;   &quot;DBP&quot;     &quot;FOXD1&quot;  
[162] &quot;ZNF320&quot;  &quot;STAT1&quot;   &quot;ZNF282&quot;  &quot;HES1&quot;    &quot;FOXN3&quot;   &quot;ELF2&quot;    &quot;NR4A1&quot;  </code></pre>
<pre class="r"><code>LR_passing_sequences &lt;- read_delim(&quot;~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/LR_passing_sequences.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    col_types = cols(motif_DB = col_skip()), 
    trim_ws = TRUE)
motif_LR_list &lt;- LR_passing_sequences %&gt;% 
   distinct(motif_ID, .keep_all = TRUE) %&gt;% 
  na.omit() %&gt;% 
  dplyr::select(motif_ID, motif_ALT_ID)


expr_LR_tfs &lt;- data.frame(LR_TFs=intersect(motif_LR_list$motif_ALT_ID, unique(RNA_expresed_genes$SYMBOL))) 

# ggVennDiagram(list(motif_EAR_list$motif_ID, motif_ESR_list$motif_ID, motif_LR_list$motif_ID))


# ggVennDiagram::ggVennDiagram(list(expr_EAR_tfs,expr_ESR_tfs, expr_LR_tfs))</code></pre>
<p>From SEA of EAR peaks:</p>
<p>I obtained a list of all tf motifs that were found enriched based on
shuffled sequences of the peak sequences. The EAR cluster has a total of
7321 peaks with a minimum width of 100 bps and a maximum width of 4430
bps. The average width and median width are 622.7464827 and 512 bps
respectively. There were a total of 417 unique motifs found. Of those
motifs found, 196 TFs were expressed in my data.</p>
<p>From SEA of ESR peaks:</p>
<p>I obtained a list of all tf motifs that were found enriched based on
shuffled sequences of the peak sequences. The ESR cluster has a total of
15665 peaks with a minimum width of 112 bps and a maximum width of 4027
bps. The average width and median width are 591.1618257 and 522 bps
respectively. There were a total of 466 unique motifs found. Of those
motifs found, 191 TFs were expressed in my data.</p>
<p>From SEA of LR peaks:</p>
<p>I obtained a list of all tf motifs that were found enriched based on
shuffled sequences of the peak sequences. The ESR cluster has a total of
41759 peaks with a minimum width of 92 bps and a maximum width of 3308
bps. The average width and median width are 572.4592303 and 515 bps
respectively. There were a total of 501 unique motifs found. Of those
motifs found, 221 TFs were expressed in my data.</p>
<p>The EAR and ESR sets share 362 motifs out of a total of 521 total
motifs between them.</p>
<pre class="r"><code># EAR_passing_sequences %&gt;% 
#   group_by(motif_ALT_ID) %&gt;% 
#   dplyr::filter(motif_ALT_ID %in% expr_EAR_tfs$EAR_TFs)

##top 20 motifs with expressedTFS

EAR_sea %&gt;% 
  dplyr::filter(ALT_ID %in% expr_EAR_tfs$EAR_TFs) %&gt;% 
  slice_head(n=20) %&gt;% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
  geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+
 theme_bw()+
  ggtitle(&quot;expressed EAR enriched motifs&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/filterSEA_EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ESR_sea %&gt;% 
  dplyr::filter(ALT_ID %in% expr_ESR_tfs$ESR_TFs) %&gt;% 
  slice_head(n=20) %&gt;% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
  geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+
 theme_bw()+
  ggtitle(&quot;expressed ESR enriched motifs&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/filterSEA_EAR-2.png" width="672" style="display: block; margin: auto;" />
### TF motifs in each response cluster</p>
</div>
<div id="enriched-motifs" class="section level3">
<h3>enriched motifs</h3>
<pre class="r"><code>xstreme_EAR_v_NR %&gt;% 
  left_join(., sea_EAR, by= c(&quot;ALT_ID&quot;=&quot;ALT_ID&quot;,&quot;CONSENSUS&quot;=&quot;CONSENSUS&quot;, &quot;ID&quot;=&quot;ID&quot;)) %&gt;% 
  dplyr::select(CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:&#39;FP%&#39;)%&gt;% 
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  arrange(.,EVALUE.x) %&gt;%
 # dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3))))#%&gt;%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %&gt;% 
  mutate(motif_name= gsub(&quot;[()]&quot;,&quot;&quot;,NAME), mrc=&quot;EAR&quot;) %&gt;% 
  mutate(motif_name=if_else(is.na(motif_name),ID,motif_name)) %&gt;% 
# dplyr::filter(!is.na(motif_name)) %&gt;% 
  # group_by(mrc) %&gt;% 
   # dplyr::arrange(.,desc(log(EVALUE.y), base = 10)) %&gt;% 
ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=&quot;pink2&quot;) +
   geom_point(aes(x=`TP%`), size =5)+
   geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(sec.axis = dup_axis(name=&quot;Percent of peaks with motif&quot;))+
  # geom_text(aes())
  theme_classic()+
  ylab(&quot;Enriched TF motif&quot;)+
  ggtitle(&quot;Early-acute response peaks&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/enriched%20motifs%20EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-enriched-motifs-EAR-1">
Past versions of "enriched motifs EAR-1.png"
</button>
</p>
<div id="fig-no-spaces-enriched-motifs-EAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/enriched motifs EAR-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>xstreme_ESR_v_NR %&gt;% 
   left_join(., sea_ESR, by= c(&quot;ALT_ID&quot;=&quot;ALT_ID&quot;,&quot;CONSENSUS&quot;=&quot;CONSENSUS&quot;, &quot;ID&quot;=&quot;ID&quot;))%&gt;% 
  dplyr::select(CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:&#39;FP%&#39;)%&gt;% 
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  arrange(.,EVALUE.x) %&gt;%
 # dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3))))#%&gt;%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %&gt;% 
  mutate(motif_name= gsub(&quot;[()]&quot;,&quot;&quot;,NAME), mrc=&quot;EAR&quot;) %&gt;% 
  mutate(motif_name=if_else(is.na(motif_name),ID,motif_name)) %&gt;% 
# dplyr::filter(!is.na(motif_name)) %&gt;% 
  # group_by(mrc) %&gt;% 
ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=&quot;lightgreen&quot;) +
   geom_point(aes(x=`TP%`), size =5)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  # geom_text(aes())
  scale_x_continuous(sec.axis = dup_axis(name=&quot;Percent of peaks with motif&quot;))+
  theme_classic()+
  ylab(&quot;Enriched TF motif&quot;)+
  ggtitle(&quot;Early-sustained response peaks&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/enriched%20motifs%20ESR-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>xstreme_LR_v_NR %&gt;% 
    left_join(., sea_LR, by= c(&quot;ALT_ID&quot;=&quot;ALT_ID&quot;, &quot;ID&quot;=&quot;ID&quot;)) %&gt;% 
  dplyr::select(CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:&#39;FP%&#39;)%&gt;% 
  separate(SIM_MOTIF, into= c(&quot;SIM_MOTIF&quot;, &quot;NAME&quot;), sep= &quot; &quot;) %&gt;% 
  arrange(.,EVALUE.x) %&gt;%
 # dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3))))#%&gt;%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %&gt;% 
   group_by(NAME) %&gt;% 
  mutate(RowGroup=cumsum(NAME != 0)) %&gt;% 
  ungroup() %&gt;% 
  mutate(motif_name= gsub(&quot;[()]&quot;,&quot;&quot;,NAME), mrc=&quot;LR&quot;) %&gt;% 
   mutate(motif_name=if_else(is.na(motif_name),ID,if_else(motif_name==&quot;ZNF384&quot;,paste(motif_name, RowGroup, sep=&quot;_&quot;),motif_name)))%&gt;%
 
  mutate(`TP%`=as.numeric(`TP%`)) %&gt;%
# dplyr::filter(!is.na(motif_name)) %&gt;% 
  # group_by(mrc) %&gt;% 
ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=&quot;cyan&quot;) +
    geom_point(aes(x=`TP%`), size =5)+
   # geom_line(aes(x=TP, group=mrc))+
  # geom_text(aes())
  scale_x_continuous(sec.axis = dup_axis(name=&quot;Percent of peaks with motif&quot;))+
  theme_classic()+
  ylab(&quot;Enriched TF motif&quot;)+
  ggtitle(&quot;Late response peaks&quot;)</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/enriched%20motifs%20LR-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="genes-and-expression" class="section level3">
<h3>genes and expression</h3>
<div id="bach2" class="section level4">
<h4>BACH2</h4>
</div>
<div id="znf384" class="section level4">
<h4>ZNF384</h4>
<p>enriched in EAR, ESR, LR</p>
<pre class="r"><code># motifs_ZNF384 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;ZNF384&#39;)
# ZNF384_motif &lt;- motifs_ZNF384$`Hsapiens-jaspar2022-ZNF384-MA1125.1`
# # 
# # 
# ZNF384_pwm = PWMatrix(
#     ID = &#39;ZNF384&#39;,
#     profileMatrix = ZNF384_motif)
hits_ESR_ZNF384 &lt;- readRDS(&quot;data/hits_files/ZNF384_hits_ESR.RDS&quot;)
# hits = searchSeq(ZNF384_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_ESR &lt;- hits
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/ZNF384_hits_ESR.RDS&quot;)
# hits_ESR_ZNF384 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the ZNF384 peaks&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;ESR: ZNF384 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_ZNF384 &lt;- readRDS(&quot;data/hits_files/ZNF384_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(ZNF384_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/ZNF384_hits_EAR.RDS&quot;)

# hits_EAR_ZNF384 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the ZNF384 peaks&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;EAR: ZNF384 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_LR_ZNF384 &lt;- readRDS(&quot;data/hits_files/ZNF384_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(ZNF384_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/ZNF384_hits_LR.RDS&quot;)

# hits_LR_ZNF384 %&gt;% 
#   mutate(position=start-1000) %&gt;% 
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the ZNF384 peaks&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: ZNF384 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))
hits_NR_ZNF384 &lt;- readRDS(&quot;data/hits_files/ZNF384_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(ZNF384_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/ZNF384_hits_NR.RDS&quot;)

hits_NR_ZNF384 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_ZNF384 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_ZNF384 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_ZNF384 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_ZNF384, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the ZNF384 motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around ZNF384 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/ZNF384-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ZNF384-1">
Past versions of ZNF384-1.png
</button>
</p>
<div id="fig-ZNF384-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8b26fb3962797fec69567f429e4e31048b28d463/docs/figure/Jaspar_motif.Rmd/ZNF384-1.png" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/ZNF384-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="zkscan3" class="section level4">
<h4>ZKSCAN3</h4>
<p>enriched in EAR</p>
<pre class="r"><code># 
# motifs_ZKSCAN3 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;ZKSCAN3&#39;)
# ZKSCAN3_motif &lt;- motifs_ZKSCAN3$`Hsapiens-jaspar2022-ZKSCAN3-MA1973.1`
# 
# 
# ZKSCAN3_pwm = PWMatrix(
#     ID = &#39;ZKSCAN3&#39;,
#     profileMatrix = ZKSCAN3_motif)

# hits_ESR_250 = searchSeq(ZKSCAN3_pwm, seq_ESR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_df_ESR_250 &lt;- as(hits_ESR_250,&quot;data.frame&quot;)
# hits_ESR_250 &lt;- as.data.frame(hits_ESR_250)
# saveRDS(hits_ESR_250,&quot;data/hits_files/ZKSCAN3_hits_ESR_250.RDS&quot;)
hits_ESR_ZKSCAN3_250 &lt;- readRDS(&quot;data/hits_files/ZKSCAN3_hits_ESR_250.RDS&quot;)



# 
# 
# hits_EAR_250 &lt;- searchSeq(ZKSCAN3_pwm, seq_EAR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR_250 &lt;- as.data.frame(hits_EAR_250)
# saveRDS(hits_EAR_250,&quot;data/hits_files/ZKSCAN3_hits_EAR_250.RDS&quot;)
# 
hits_EAR_ZKSCAN3_250 &lt;- readRDS(&quot;data/hits_files/ZKSCAN3_hits_EAR_250.RDS&quot;)
# 
# hits_LR_250 &lt;- searchSeq(ZKSCAN3_pwm, seq_LR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR_250 &lt;- as.data.frame(hits_LR_250)
# saveRDS(hits_LR_250,&quot;data/hits_files/ZKSCAN3_hits_LR_250.RDS&quot;)
hits_LR_ZKSCAN3_250 &lt;- readRDS(&quot;data/hits_files/ZKSCAN3_hits_LR_250.RDS&quot;)

# 
# hits_NR_250 &lt;- searchSeq(ZKSCAN3_pwm, seq_NR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR_250 &lt;- as.data.frame(hits_NR_250)
# saveRDS(hits_NR_250,&quot;data/hits_files/ZKSCAN3_hits_NR_250.RDS&quot;)
hits_NR_ZKSCAN3_250 &lt;- readRDS(&quot;data/hits_files/ZKSCAN3_hits_NR_250.RDS&quot;)


hits_NR_ZKSCAN3_250 %&gt;% 
  mutate(position=start-250) %&gt;% 
  mutate(id_mrc=&quot;NR&quot;) %&gt;% 
  rbind(.,(hits_LR_ZKSCAN3_250 %&gt;% mutate(position=start-250) %&gt;% 
  mutate(id_mrc=&quot;LR&quot;))) %&gt;% 
   rbind(.,(hits_ESR_ZKSCAN3_250 %&gt;% mutate(position=start-250) %&gt;% 
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;% 
  rbind(.,(hits_EAR_ZKSCAN3_250 %&gt;% mutate(position=start-250) %&gt;% 
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;% 
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the ZKSCAN3 motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around ZKSCAN3 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/ZKSCAN3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ZKSCAN3-1">
Past versions of ZKSCAN3-1.png
</button>
</p>
<div id="fig-ZKSCAN3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8b26fb3962797fec69567f429e4e31048b28d463/docs/figure/Jaspar_motif.Rmd/ZKSCAN3-1.png" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/ZKSCAN3-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pitx1" class="section level4">
<h4>PITX1</h4>
<p>enriched in EAR and ESR</p>
<pre class="r"><code># 
# motifs_PITX1 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;PITX1&#39;)
#     PITX1_motif &lt;- motifs_PITX1$`Hsapiens-jaspar2022-PITX1-MA0682.2`


# PITX1_pwm = PWMatrix(
#     ID = &#39;PITX1&#39;,
#     profileMatrix = PITX1_motif)
hits_ESR_PITX1 &lt;- readRDS(&quot;data/hits_files/PITX1_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(PITX1_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_ESR &lt;- hits
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/PITX1_hits_ESR.RDS&quot;)
hits_ESR_PITX1 %&gt;%
  mutate(position=start-1000) %&gt;%
  ggplot(.,aes(position))+
  geom_density(size=2) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the PITX1 peaks&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;ESR: PITX1 binding&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/PITX1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PITX1-1">
Past versions of PITX1-1.png
</button>
</p>
<div id="fig-PITX1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/PITX1-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hits_EAR_PITX1 &lt;- readRDS(&quot;data/hits_files/PITX1_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(PITX1_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/PITX1_hits_EAR.RDS&quot;)

hits_EAR_PITX1 %&gt;%
  mutate(position=start-1000) %&gt;%
  ggplot(.,aes(position))+
  geom_density(size=2) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the PITX1 peaks&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;EAR: PITX1 binding&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/PITX1-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PITX1-2">
Past versions of PITX1-2.png
</button>
</p>
<div id="fig-PITX1-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/PITX1-2.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hits_LR_PITX1 &lt;- readRDS(&quot;data/hits_files/PITX1_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(PITX1_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/PITX1_hits_LR.RDS&quot;)

hits_LR_PITX1 %&gt;%
  mutate(position=start-1000) %&gt;%
  ggplot(.,aes(position))+
  geom_density(size=2) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the PITX1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;LR: PITX1 binding&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/PITX1-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PITX1-3">
Past versions of PITX1-3.png
</button>
</p>
<div id="fig-PITX1-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/PITX1-3.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>hits_NR_PITX1 &lt;- readRDS(&quot;data/hits_files/PITX1_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(PITX1_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/PITX1_hits_NR.RDS&quot;)

hits_NR_PITX1 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_PITX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_PITX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_PITX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_PITX1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the PITX1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around PITX1 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/PITX1-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PITX1-4">
Past versions of PITX1-4.png
</button>
</p>
<div id="fig-PITX1-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/PITX1-4.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tp53" class="section level4">
<h4>TP53</h4>
<p>not enriched, just curious</p>
<pre class="r"><code># # 
# motifs_TP53 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;TP53&#39;)
#     TP53_motif &lt;- motifs_TP53$`Hsapiens-JASPAR_CORE-TP53-MA0106.1`
# 
# 
# TP53_pwm = PWMatrix(
#     ID = &#39;TP53&#39;,
#     profileMatrix = TP53_motif)
hits_ESR_TP53 &lt;- readRDS(&quot;data/hits_files/TP53_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(TP53_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_ESR &lt;- hits
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/TP53_hits_ESR.RDS&quot;)
# hits_ESR_TP53 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the TP53  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;ESR: TP53 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_TP53 &lt;- readRDS(&quot;data/hits_files/TP53_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(TP53_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/TP53_hits_EAR.RDS&quot;)

# hits_EAR_TP53 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the TP53  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;EAR: TP53 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_LR_TP53 &lt;- readRDS(&quot;data/hits_files/TP53_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(TP53_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/TP53_hits_LR.RDS&quot;)

# hits_LR_TP53 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the TP53  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: TP53 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_NR_TP53 &lt;- readRDS(&quot;data/hits_files/TP53_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(TP53_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/TP53_hits_NR.RDS&quot;)

hits_NR_TP53 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_TP53 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_TP53 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_TP53 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_TP53, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the TP53 motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around TP53 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/TP53-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-TP53-1">
Past versions of TP53-1.png
</button>
</p>
<div id="fig-TP53-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/TP53-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># </code></pre>
</div>
<div id="nrf1" class="section level4">
<h4>NRF1</h4>
<pre class="r"><code># 
# motifs_NRF1 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;NRF1&#39;)
#     NRF1_motif &lt;- motifs_NRF1$`Hsapiens-jaspar2018-NRF1-MA0506.1`
# # 
# # 
# NRF1_pwm = PWMatrix(
#     ID = &#39;NRF1&#39;,
#     profileMatrix = NRF1_motif)
hits_ESR_NRF1 &lt;- readRDS(&quot;data/hits_files/NRF1_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(NRF1_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_ESR &lt;- hits
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/NRF1_hits_ESR.RDS&quot;)
# hits_ESR_NRF1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the NRF1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;ESR: NRF1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_NRF1 &lt;- readRDS(&quot;data/hits_files/NRF1_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(NRF1_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/NRF1_hits_EAR.RDS&quot;)

# hits_EAR_NRF1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the NRF1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;EAR: NRF1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_NRF1 &lt;- readRDS(&quot;data/hits_files/NRF1_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(NRF1_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/NRF1_hits_LR.RDS&quot;)

# hits_LR_NRF1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the NRF1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: NRF1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_NR_NRF1 &lt;- readRDS(&quot;data/hits_files/NRF1_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(NRF1_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/NRF1_hits_NR.RDS&quot;)

hits_NR_NRF1 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_NRF1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_NRF1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_NRF1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_NRF1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the NRF1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around NRF1 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/NRF1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-NRF1-1">
Past versions of NRF1-1.png
</button>
</p>
<div id="fig-NRF1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/NRF1-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nfic_tlx1" class="section level4">
<h4>NFIC_TLX1</h4>
<pre class="r"><code># 
# motifs_NFIC_TLX1 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;NFIC::TLX1&#39;)
#     NFIC_TLX1_motif &lt;- motifs_NFIC_TLX1$`Hsapiens-jaspar2022-NFIC::TLX1-MA0119.1`
# #
# # # 
# NFIC_TLX1_pwm = PWMatrix(
#     ID = &#39;NFIC_TLX1&#39;,
#     profileMatrix = NFIC_TLX1_motif)
hits_ESR_NFIC_TLX1 &lt;- readRDS(&quot;data/hits_files/NFIC_TLX1_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(NFIC_TLX1_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# 
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/NFIC_TLX1_hits_ESR.RDS&quot;)
# hits_ESR_NFIC_TLX1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the NFIC_TLX1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;ESR: NFIC_TLX1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_NFIC_TLX1 &lt;- readRDS(&quot;data/hits_files/NFIC_TLX1_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(NFIC_TLX1_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/NFIC_TLX1_hits_EAR.RDS&quot;)

# hits_EAR_NFIC_TLX1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the NFIC_TLX1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;EAR: NFIC_TLX1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_NFIC_TLX1 &lt;- readRDS(&quot;data/hits_files/NFIC_TLX1_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(NFIC_TLX1_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/NFIC_TLX1_hits_LR.RDS&quot;)

# hits_LR_NFIC_TLX1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the NFIC_TLX1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: NFIC_TLX1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_NR_NFIC_TLX1 &lt;- readRDS(&quot;data/hits_files/NFIC_TLX1_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(NFIC_TLX1_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/NFIC_TLX1_hits_NR.RDS&quot;)

hits_NR_NFIC_TLX1 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_NFIC_TLX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_NFIC_TLX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_NFIC_TLX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_NFIC_TLX1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the NFIC_TLX1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around NFIC_TLX1 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))#+</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/NFIC_TLX1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-NFIC_TLX1-1">
Past versions of NFIC_TLX1-1.png
</button>
</p>
<div id="fig-NFIC_TLX1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/NFIC_TLX1-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>  # coord_cartesian(xlim= c(-500,500))</code></pre>
</div>
<div id="prdm1" class="section level4">
<h4>PRDM1</h4>
<pre class="r"><code># 
# motifs_PRDM1 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;PRDM1&#39;)
#     PRDM1_motif &lt;- motifs_PRDM1$`Hsapiens-jaspar2022-PRDM1-MA0508.3`
# #
# PRDM1_pwm = PWMatrix(
#     ID = &#39;PRDM1&#39;,
#     profileMatrix = PRDM1_motif)
hits_ESR_PRDM1 &lt;- readRDS(&quot;data/hits_files/PRDM1_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(PRDM1_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# 
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/PRDM1_hits_ESR.RDS&quot;)

hits_EAR_PRDM1 &lt;- readRDS(&quot;data/hits_files/PRDM1_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(PRDM1_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/PRDM1_hits_EAR.RDS&quot;)


hits_LR_PRDM1 &lt;- readRDS(&quot;data/hits_files/PRDM1_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(PRDM1_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/PRDM1_hits_LR.RDS&quot;)



hits_NR_PRDM1 &lt;- readRDS(&quot;data/hits_files/PRDM1_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(PRDM1_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/PRDM1_hits_NR.RDS&quot;)

hits_NR_PRDM1 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_PRDM1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_PRDM1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_PRDM1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_PRDM1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the PRDM1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around PRDM1 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/PRDM1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-PRDM1-1">
Past versions of PRDM1-1.png
</button>
</p>
<div id="fig-PRDM1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8b26fb3962797fec69567f429e4e31048b28d463/docs/figure/Jaspar_motif.Rmd/PRDM1-1.png" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/PRDM1-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># +
  # coord_cartesian(xlim= c(-500,500))</code></pre>
</div>
<div id="tbx1" class="section level4">
<h4>TBX1</h4>
<pre class="r"><code># 
# motifs_TBX1 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;TBX1&#39;)
#     TBX1_motif &lt;- motifs_TBX1$`Hsapiens-jaspar2022-TBX1-MA0805.1`
# #
# #
# TBX1_pwm = PWMatrix(
#     ID = &#39;TBX1&#39;,
#     profileMatrix = TBX1_motif)
hits_ESR_TBX1 &lt;- readRDS(&quot;data/hits_files/TBX1_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(TBX1_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# 
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/TBX1_hits_ESR.RDS&quot;)
# hits_ESR_TBX1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the TBX1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;ESR: TBX1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_TBX1 &lt;- readRDS(&quot;data/hits_files/TBX1_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(TBX1_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/TBX1_hits_EAR.RDS&quot;)

# hits_EAR_TBX1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the TBX1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;EAR: TBX1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_TBX1 &lt;- readRDS(&quot;data/hits_files/TBX1_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(TBX1_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/TBX1_hits_LR.RDS&quot;)

# hits_LR_TBX1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the TBX1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: TBX1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_NR_TBX1 &lt;- readRDS(&quot;data/hits_files/TBX1_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(TBX1_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/TBX1_hits_NR.RDS&quot;)

hits_NR_TBX1 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_TBX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_TBX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_TBX1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_TBX1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the TBX1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around TBX1 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))+
  coord_cartesian(xlim= c(-500,500))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/TBX1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-TBX1-1">
Past versions of TBX1-1.png
</button>
</p>
<div id="fig-TBX1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8b26fb3962797fec69567f429e4e31048b28d463/docs/figure/Jaspar_motif.Rmd/TBX1-1.png" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/TBX1-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="gli1" class="section level4">
<h4>GLI1</h4>
<pre class="r"><code># 
# motifs_GLI1 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;GLI1&#39;)
#     GLI1_motif &lt;- motifs_GLI1$`Hsapiens-HOCOMOCOv10-GLI1_HUMAN.H10MO.C`
# # # 
# # # 
# GLI1_pwm = PWMatrix(
#     ID = &#39;GLI1&#39;,
#     profileMatrix = GLI1_motif)
hits_ESR_GLI1 &lt;- readRDS(&quot;data/hits_files/GLI1_hits_ESR.RDS&quot;)
# hits_ESR = searchSeq(GLI1_pwm, seq_ESR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# 
# hits_ESR &lt;- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,&quot;data/hits_files/GLI1_hits_ESR.RDS&quot;)
# hits_ESR_GLI1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the GLI1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;ESR: GLI1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_GLI1 &lt;- readRDS(&quot;data/hits_files/GLI1_hits_EAR.RDS&quot;)
# hits_EAR &lt;- searchSeq(GLI1_pwm, seq_EAR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR &lt;- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,&quot;data/hits_files/GLI1_hits_EAR.RDS&quot;)

# hits_EAR_GLI1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the GLI1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;EAR: GLI1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_GLI1 &lt;- readRDS(&quot;data/hits_files/GLI1_hits_LR.RDS&quot;)
# hits_LR &lt;- searchSeq(GLI1_pwm, seq_LR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR &lt;- as.data.frame(hits_LR)
# saveRDS(hits_LR,&quot;data/hits_files/GLI1_hits_LR.RDS&quot;)

# hits_LR_GLI1 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the GLI1  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: GLI1 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

hits_NR_GLI1 &lt;- readRDS(&quot;data/hits_files/GLI1_hits_NR.RDS&quot;)
# hits_NR &lt;- searchSeq(GLI1_pwm, seq_NR, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR &lt;- as.data.frame(hits_NR)
# saveRDS(hits_NR,&quot;data/hits_files/GLI1_hits_NR.RDS&quot;)

hits_NR_GLI1 %&gt;%
  mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_GLI1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_GLI1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_GLI1 %&gt;% mutate(position=start-1000) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_GLI1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the GLI1  motif&#39;) +
  ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
  ggtitle(&quot;Chromatin accessibility around GLI1 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))+
  coord_cartesian(xlim= c(-500,500))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/GLI1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-GLI1-1">
Past versions of GLI1-1.png
</button>
</p>
<div id="fig-GLI1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8b26fb3962797fec69567f429e4e31048b28d463/docs/figure/Jaspar_motif.Rmd/GLI1-1.png" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/59f30bb6f750b769fa0ba657e2d2a6231423c7ac/docs/figure/Jaspar_motif.Rmd/GLI1-1.png" target="_blank">59f30bb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># 
# sites_LR_v_NR %&gt;% 
#   dplyr::filter(motif_ALT_ID==&quot;STREME-1&quot;, seq_ID==&quot;chr3&quot;) %&gt;% 
#   # mutate(position)
#   ggplot(., aes(site_Score))+
#   geom_density()</code></pre>
</div>
<div id="znf460" class="section level4">
<h4>ZNF460</h4>
<pre class="r"><code># # 
# motifs_ZNF460 = query(query(MotifDb, &#39;Hsapiens&#39;), &#39;ZNF460&#39;)
#     ZNF460_motif &lt;- motifs_ZNF460$`Hsapiens-jaspar2022-ZNF460-MA1596.1`
# 
# ZNF460_pwm = PWMatrix(
#     ID = &#39;ZNF460&#39;,
#     profileMatrix = ZNF460_motif)

# hits_ESR_250 = searchSeq(ZNF460_pwm, seq_ESR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_ESR_250 &lt;- as.data.frame(hits_ESR_250)
# saveRDS(hits_ESR_250,&quot;data/hits_files/ZNF460_hits_ESR_250.RDS&quot;)
hits_ESR_ZNF460_250 &lt;- readRDS(&quot;data/hits_files/ZNF460_hits_ESR_250.RDS&quot;)

# 
# hits_EAR_250 &lt;- searchSeq(ZNF460_pwm, seq_EAR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_EAR_250 &lt;- as.data.frame(hits_EAR_250)
# saveRDS(hits_EAR_250,&quot;data/hits_files/ZNF460_hits_EAR_250.RDS&quot;)
hits_EAR_ZNF460_250 &lt;- readRDS(&quot;data/hits_files/ZNF460_hits_EAR_250.RDS&quot;)



# hits_LR_250 &lt;- searchSeq(ZNF460_pwm, seq_LR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_LR_250 &lt;- as.data.frame(hits_LR_250)
# saveRDS(hits_LR_250,&quot;data/hits_files/ZNF460_hits_LR_250.RDS&quot;)
hits_LR_ZNF460_250 &lt;- readRDS(&quot;data/hits_files/ZNF460_hits_LR_250.RDS&quot;)
# hits_LR_ZNF460 %&gt;%
#   mutate(position=start-1000) %&gt;%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
#   xlab(&#39;Position around the ZNF460  motif&#39;) +
#   ylab(&#39;Per position percentage\nof motif occurrence&#39;) +
#   ggtitle(&quot;LR: ZNF460 binding&quot;)+
#   theme(
#     axis.text = element_text(size=10, face=&#39;bold&#39;),
#     axis.title = element_text(size=14,face=&quot;bold&quot;),
#     plot.title = element_text(hjust = 0.5))

# 
# hits_NR_250 &lt;- searchSeq(ZNF460_pwm, seq_NR_250, min.score=&quot;80%&quot;, strand=&quot;*&quot;)
# hits_NR_250 &lt;- as.data.frame(hits_NR_250)
# saveRDS(hits_NR_250,&quot;data/hits_files/ZNF460_hits_NR_250.RDS&quot;)
hits_NR_ZNF460_250 &lt;- readRDS(&quot;data/hits_files/ZNF460_hits_NR_250.RDS&quot;)
hits_NR_ZNF460_250 %&gt;%
  mutate(position=start-250) %&gt;%
  mutate(id_mrc=&quot;NR&quot;) %&gt;%
  rbind(.,(hits_LR_ZNF460_250 %&gt;% mutate(position=start-250) %&gt;%
  mutate(id_mrc=&quot;LR&quot;))) %&gt;%
   rbind(.,(hits_ESR_ZNF460_250 %&gt;% mutate(position=start-250) %&gt;%
  mutate(id_mrc=&quot;ESR&quot;))) %&gt;%
   rbind(.,(hits_EAR_ZNF460_250 %&gt;% mutate(position=start-250) %&gt;%
  mutate(id_mrc=&quot;EAR&quot;))) %&gt;%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_ZNF460, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color=&#39;red&#39;, size=2) +
  xlab(&#39;Position around the ZNF460  motif&#39;) +
  ylab(&#39;ZNF460&#39;) +
  # ggtitle(&quot;Chromatin accessibility around ZNF460 binding site&quot;)+
  theme(
    axis.text = element_text(size=10, face=&#39;bold&#39;),
    axis.title = element_text(size=14,face=&quot;bold&quot;),
    plot.title = element_text(hjust = 0.5))</code></pre>
<p><img src="figure/Jaspar_motif.Rmd/ZNF460-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ZNF460-1">
Past versions of ZNF460-1.png
</button>
</p>
<div id="fig-ZNF460-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8b26fb3962797fec69567f429e4e31048b28d463/docs/figure/Jaspar_motif.Rmd/ZNF460-1.png" target="_blank">8b26fb3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-05-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># +
  # coord_cartesian(xlim= c(-500,500))</code></pre>
</div>
<div id="mef2a" class="section level4">
<h4>MEF2A</h4>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] BSgenome.Hsapiens.UCSC.hg38_1.4.5       
 [2] BSgenome_1.70.2                         
 [3] BiocIO_1.12.0                           
 [4] MotifDb_1.44.0                          
 [5] Biostrings_2.70.3                       
 [6] XVector_0.42.0                          
 [7] TFBSTools_1.40.0                        
 [8] JASPAR2022_0.99.8                       
 [9] BiocFileCache_2.10.2                    
[10] dbplyr_2.5.0                            
[11] bedr_1.0.7                              
[12] devtools_2.4.5                          
[13] usethis_2.2.3                           
[14] ggpubr_0.6.0                            
[15] BiocParallel_1.36.0                     
[16] Cormotif_1.48.0                         
[17] affy_1.80.0                             
[18] scales_1.3.0                            
[19] VennDiagram_1.7.3                       
[20] futile.logger_1.4.3                     
[21] gridExtra_2.3                           
[22] ggfortify_0.4.17                        
[23] edgeR_4.0.16                            
[24] limma_3.58.1                            
[25] rtracklayer_1.62.0                      
[26] org.Hs.eg.db_3.18.0                     
[27] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[28] GenomicFeatures_1.54.4                  
[29] AnnotationDbi_1.64.1                    
[30] Biobase_2.62.0                          
[31] GenomicRanges_1.54.1                    
[32] GenomeInfoDb_1.38.8                     
[33] IRanges_2.36.0                          
[34] S4Vectors_0.40.2                        
[35] BiocGenerics_0.48.1                     
[36] ChIPseeker_1.38.0                       
[37] RColorBrewer_1.1-3                      
[38] broom_1.0.6                             
[39] kableExtra_1.4.0                        
[40] lubridate_1.9.3                         
[41] forcats_1.0.0                           
[42] stringr_1.5.1                           
[43] dplyr_1.1.4                             
[44] purrr_1.0.2                             
[45] readr_2.1.5                             
[46] tidyr_1.3.1                             
[47] tibble_3.2.1                            
[48] ggplot2_3.5.1                           
[49] tidyverse_2.0.0                         
[50] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.4                               
  [2] matrixStats_1.3.0                      
  [3] bitops_1.0-7                           
  [4] DirichletMultinomial_1.44.0            
  [5] enrichplot_1.22.0                      
  [6] HDO.db_0.99.1                          
  [7] httr_1.4.7                             
  [8] profvis_0.3.8                          
  [9] tools_4.3.1                            
 [10] backports_1.5.0                        
 [11] utf8_1.2.4                             
 [12] R6_2.5.1                               
 [13] lazyeval_0.2.2                         
 [14] urlchecker_1.0.1                       
 [15] withr_3.0.0                            
 [16] prettyunits_1.2.0                      
 [17] preprocessCore_1.64.0                  
 [18] cli_3.6.2                              
 [19] formatR_1.14                           
 [20] scatterpie_0.2.2                       
 [21] labeling_0.4.3                         
 [22] sass_0.4.9                             
 [23] Rsamtools_2.18.0                       
 [24] systemfonts_1.1.0                      
 [25] yulab.utils_0.1.4                      
 [26] R.utils_2.12.3                         
 [27] DOSE_3.28.2                            
 [28] svglite_2.1.3                          
 [29] sessioninfo_1.2.2                      
 [30] plotrix_3.8-4                          
 [31] rstudioapi_0.16.0                      
 [32] RSQLite_2.3.7                          
 [33] generics_0.1.3                         
 [34] gridGraphics_0.5-1                     
 [35] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [36] vroom_1.6.5                            
 [37] gtools_3.9.5                           
 [38] car_3.1-2                              
 [39] GO.db_3.18.0                           
 [40] Matrix_1.6-5                           
 [41] fansi_1.0.6                            
 [42] abind_1.4-5                            
 [43] R.methodsS3_1.8.2                      
 [44] lifecycle_1.0.4                        
 [45] whisker_0.4.1                          
 [46] yaml_2.3.8                             
 [47] carData_3.0-5                          
 [48] SummarizedExperiment_1.32.0            
 [49] gplots_3.1.3.1                         
 [50] qvalue_2.34.0                          
 [51] SparseArray_1.2.4                      
 [52] blob_1.2.4                             
 [53] promises_1.3.0                         
 [54] crayon_1.5.2                           
 [55] miniUI_0.1.1.1                         
 [56] lattice_0.22-6                         
 [57] cowplot_1.1.3                          
 [58] annotate_1.80.0                        
 [59] KEGGREST_1.42.0                        
 [60] pillar_1.9.0                           
 [61] knitr_1.47                             
 [62] fgsea_1.28.0                           
 [63] rjson_0.2.21                           
 [64] boot_1.3-30                            
 [65] codetools_0.2-20                       
 [66] fastmatch_1.1-4                        
 [67] glue_1.7.0                             
 [68] getPass_0.2-4                          
 [69] ggfun_0.1.5                            
 [70] remotes_2.5.0                          
 [71] data.table_1.15.4                      
 [72] vctrs_0.6.5                            
 [73] png_0.1-8                              
 [74] treeio_1.26.0                          
 [75] testthat_3.2.1.1                       
 [76] poweRlaw_0.80.0                        
 [77] gtable_0.3.5                           
 [78] cachem_1.1.0                           
 [79] xfun_0.44                              
 [80] S4Arrays_1.2.1                         
 [81] mime_0.12                              
 [82] tidygraph_1.3.1                        
 [83] pracma_2.4.4                           
 [84] statmod_1.5.0                          
 [85] ellipsis_0.3.2                         
 [86] interactiveDisplayBase_1.40.0          
 [87] nlme_3.1-164                           
 [88] ggtree_3.10.1                          
 [89] bit64_4.0.5                            
 [90] progress_1.2.3                         
 [91] filelock_1.0.3                         
 [92] rprojroot_2.0.4                        
 [93] bslib_0.7.0                            
 [94] affyio_1.72.0                          
 [95] KernSmooth_2.23-24                     
 [96] splitstackshape_1.4.8                  
 [97] seqLogo_1.68.0                         
 [98] colorspace_2.1-0                       
 [99] DBI_1.2.3                              
[100] tidyselect_1.2.1                       
[101] processx_3.8.4                         
[102] bit_4.0.5                              
[103] compiler_4.3.1                         
[104] curl_5.2.1                             
[105] git2r_0.33.0                           
[106] xml2_1.3.6                             
[107] DelayedArray_0.28.0                    
[108] shadowtext_0.1.3                       
[109] caTools_1.18.2                         
[110] callr_3.7.6                            
[111] rappdirs_0.3.3                         
[112] digest_0.6.35                          
[113] rmarkdown_2.27                         
[114] htmltools_0.5.8.1                      
[115] pkgconfig_2.0.3                        
[116] MatrixGenerics_1.14.0                  
[117] highr_0.11                             
[118] fastmap_1.2.0                          
[119] htmlwidgets_1.6.4                      
[120] rlang_1.1.3                            
[121] shiny_1.8.1.1                          
[122] farver_2.1.2                           
[123] jquerylib_0.1.4                        
[124] jsonlite_1.8.8                         
[125] R.oo_1.26.0                            
[126] GOSemSim_2.28.1                        
[127] RCurl_1.98-1.14                        
[128] magrittr_2.0.3                         
[129] GenomeInfoDbData_1.2.11                
[130] ggplotify_0.1.2                        
[131] patchwork_1.2.0                        
[132] munsell_0.5.1                          
[133] Rcpp_1.0.12                            
[134] ape_5.8                                
[135] viridis_0.6.5                          
[136] stringi_1.8.4                          
[137] ggraph_2.2.1                           
[138] brio_1.1.5                             
[139] zlibbioc_1.48.2                        
[140] MASS_7.3-60.0.1                        
[141] pkgbuild_1.4.4                         
[142] AnnotationHub_3.10.1                   
[143] plyr_1.8.9                             
[144] parallel_4.3.1                         
[145] ggrepel_0.9.5                          
[146] CNEr_1.38.0                            
[147] graphlayouts_1.1.1                     
[148] splines_4.3.1                          
[149] hms_1.1.3                              
[150] locfit_1.5-9.9                         
[151] ps_1.7.6                               
[152] igraph_2.0.3                           
[153] ggsignif_0.6.4                         
[154] pkgload_1.3.4                          
[155] reshape2_1.4.4                         
[156] biomaRt_2.58.2                         
[157] TFMPvalue_0.0.9                        
[158] futile.options_1.0.1                   
[159] BiocVersion_3.18.1                     
[160] XML_3.99-0.16.1                        
[161] evaluate_0.23                          
[162] lambda.r_1.2.4                         
[163] BiocManager_1.30.23                    
[164] tzdb_0.4.0                             
[165] tweenr_2.0.3                           
[166] httpuv_1.6.15                          
[167] polyclip_1.10-6                        
[168] ggforce_0.4.2                          
[169] xtable_1.8-4                           
[170] restfulr_0.0.15                        
[171] tidytree_0.4.6                         
[172] rstatix_0.7.2                          
[173] later_1.3.2                            
[174] viridisLite_0.4.2                      
[175] aplot_0.2.2                            
[176] memoise_2.0.1                          
[177] GenomicAlignments_1.38.2               
[178] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
