<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2025-05-01" />

<title>Calculating Odds ratios</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Calculating Odds ratios</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2025-05-01</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-05-01
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025tree02acfd6808d889ca0006d23d5cfe59f586624565targetblank02acfd6a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/02acfd6808d889ca0006d23d5cfe59f586624565" target="_blank">02acfd6</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025tree02acfd6808d889ca0006d23d5cfe59f586624565targetblank02acfd6a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/02acfd6808d889ca0006d23d5cfe59f586624565" target="_blank">02acfd6</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/Correlation_of_SNPnPEAK.Rmd
    Modified:   analysis/TE_analysis_ff.Rmd
    Modified:   analysis/final_plot_attempt.Rmd
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Odds_ratios_ff.Rmd</code>) and
HTML (<code>docs/Odds_ratios_ff.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/02acfd6808d889ca0006d23d5cfe59f586624565/analysis/Odds_ratios_ff.Rmd" target="_blank">02acfd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-05-01
</td>
<td>
update webpage
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/Odds_ratios_ff.html" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/35ce20b6a30e39ab46ca3ed71967951b9442bc67/analysis/Odds_ratios_ff.Rmd" target="_blank">35ce20b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
<td>
redo on TSS enrichment Odds ratio
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(epitools)
library(circlize)
library(ggridges)</code></pre>
<div id="loading-data" class="section level5">
<h5>loading data</h5>
<p>This is where I pull in the repeatmasker file taken from UCSC genome
browser, the peaks assigned with the closest expressed genes as
‘neargenes’ by TSS, the same peaks list, only condensing to unique peaks
(some are assigned to two neargene), I call the collapsed_peaks and the
peaks assigned to each MRC (EAR, etc…).<br />
With the TSS data and the collapsed data, I made granges objects. I also
separate out the LINEs, SINEs, LTRs, DNAs, and Retroposons from the
repeatmasker to make granges objects from those sets.</p>
<pre class="r"><code>repeatmasker &lt;- read.delim(&quot;data/other_papers/repeatmasker.tsv&quot;)</code></pre>
<pre class="r"><code>### With new TSS_ngdata frame
TSS_NG_data &lt;- read_delim(&quot;data/Final_four_data/TSS_assigned_NG.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
Collapsed_peaks &lt;- read_delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;,
                              delim = &quot;\t&quot;, 
                              escape_double = FALSE, 
                              trim_ws = TRUE)

reClass_list &lt;- repeatmasker %&gt;% 
  distinct(repClass)

Line_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

Sine_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

LTR_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LTR&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

DNA_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

retroposon_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


all_TEs_gr &lt;- repeatmasker %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


peakAnnoList_ff_motif &lt;- readRDS(&quot;data/Final_four_data/peakAnnoList_ff_motif.RDS&quot;)

background_peaks &lt;- as.data.frame(peakAnnoList_ff_motif$background) 
EAR_df &lt;- as.data.frame(peakAnnoList_ff_motif$EAR)
# EAR_df_gr &lt;-  EAR_df %&gt;%  GRanges()
ESR_df &lt;- as.data.frame(peakAnnoList_ff_motif$ESR)
# ESR_df_gr &lt;-ESR_df %&gt;%  GRanges()

LR_df &lt;- as.data.frame(peakAnnoList_ff_motif$LR)
# LR_df_gr &lt;-LR_df %&gt;%  GRanges()

NR_df &lt;- as.data.frame(peakAnnoList_ff_motif$NR)
# NR_df_gr &lt;-NR_df %&gt;%  GRanges()

TSS_data_gr &lt;- TSS_NG_data %&gt;% 
  # dplyr::filter(chr != &quot;chrX&quot;) %&gt;%
  dplyr::filter(chr != &quot;chrY&quot;) %&gt;%
  dplyr::filter(Peakid %in% background_peaks$Peakid) %&gt;% 
  GRanges()

Col_TSS_data_gr &lt;- Collapsed_peaks %&gt;% 
  # dplyr::filter(chr != &quot;chrX&quot;) %&gt;%
  dplyr::filter(chr != &quot;chrY&quot;) %&gt;%
  dplyr::filter(Peakid %in% background_peaks$Peakid) %&gt;% 
  GRanges()</code></pre>
<p>this code contains the fill functions for each of the plots that
needed similar colors.</p>
<pre class="r"><code># scale fill repeat, 2nd set ----------------------------------------------
rep_other_names&lt;- repeatmasker %&gt;% 
  distinct(repClass) %&gt;% 
  rbind(&quot;Other&quot;)

scale_fill_repeat &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;,
                         &quot;darkgrey&quot;), unique(rep_other_names$repClass)), 
    ...
  )
}


# scale fill LTRs ---------------------------------------------------------

LTR_df &lt;- LTR_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_LTRs &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(LTR_df$repFamily)), 
    ...
  )
}



scale_fill_DNA_family &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;, &quot;#FFFFB3&quot;, &quot;#BEBADA&quot; ,&quot;#FB8072&quot;, &quot;#80B1D3&quot;, &quot;#FDB462&quot;, &quot;#B3DE69&quot;, &quot;#FCCDE5&quot;, &quot;purple4&quot;), unique(DNA_family$repFamily)), 
    ...
  )
}


# # scale fill repeat first -------------------------------------------------
# 
# scale_fill_repeat &lt;-  function(...){
#   ggplot2:::manual_scale(
#     &#39;fill&#39;, 
#     values = setNames(c( &quot;#8DD3C7&quot;,
#                          &quot;#FFFFB3&quot;,
#                          &quot;#BEBADA&quot; ,
#                          &quot;#FB8072&quot;,
#                          &quot;#80B1D3&quot;,
#                          &quot;#FDB462&quot;,
#                          &quot;#B3DE69&quot;,
#                          &quot;#FCCDE5&quot;,
#                          &quot;#D9D9D9&quot;,
#                          &quot;#BC80BD&quot;,
#                          &quot;#CCEBC5&quot;,
#                          &quot;pink4&quot;,
#                          &quot;cornflowerblue&quot;,
#                          &quot;chocolate&quot;,
#                          &quot;brown&quot;,
#                          &quot;green&quot;,
#                          &quot;yellow4&quot;,
#                          &quot;purple&quot;,
#                          &quot;darkorchid4&quot;,
#                          &quot;coral4&quot;,
#                          &quot;darkolivegreen4&quot;,
#                          &quot;darkorange&quot;), unique(repeatmasker$repClass)), 
#     ...
#   )
# }

# scale lines -------------------------------------------------------------
Line_df &lt;- Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_lines &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(Line_df$repFamily)), 
    ...
  )
}


# scale fill L2 family ----------------------------------------------------
L2_line_df&lt;- Line_df %&gt;% 
  dplyr::filter(repFamily==&quot;L2&quot;)


scale_fill_L2 &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(L2_line_df$repName)), 
    ...
  )
}

# scale fill sines --------------------------------------------------------
Sine_df &lt;- Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_sines &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(Sine_df$repFamily)), 
    ...
  )
}


# scale fill DNAs ---------------------------------------------------------
DNA_df &lt;- DNA_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_DNAs &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;,
                         &quot;blue&quot;,
                         &quot;grey&quot;,
                         &quot;lightgrey&quot;), unique(DNA_df$repFamily)), 
    ...
  )
}


# scale fill retroposons --------------------------------------------------

retroposon_df &lt;- retroposon_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repName=factor(repName))

scale_fill_retroposons &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(retroposon_df$repName)), 
    ...
  )
}</code></pre>
<pre class="r"><code>######################################################

Col_fullDF_overlap &lt;- readRDS(&quot;data/Final_four_data/Col_full_DF_overlap.RDS&quot;)
# all_TEs_gr$TE_width &lt;- width(all_TEs_gr)
# Col_TSS_data_gr$peak_width &lt;- width(Col_TSS_data_gr)
# Col_fullDF_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,all_TEs_gr)
Col_fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot; Table 2: Count of peaks by TE class; overlap at least 1 bp; using one:one df &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 2: Count of peaks by TE class; overlap at least 1 bp; using
one:one df
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
16960
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
138
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
40440
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
27844
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
257
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5731
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
52
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
295
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
52755
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
191
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
30184
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
288
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
54
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
144
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
302
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Col_fullDF_overlap %&gt;% 
   as.data.frame %&gt;% 
  mutate(per_ol= width/TE_width) %&gt;% 
  dplyr::filter(per_ol&gt;0.5) %&gt;%
  group_by(repClass) %&gt;% 
  tally() %&gt;% 
  kable(., caption=&quot; Table 3:Count of peaks by TE class; overlap of &gt;50% of TE; newway &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 3:Count of peaks by TE class; overlap of &gt;50% of TE; newway
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
11821
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
118
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
25548
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
18403
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
199
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5305
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
81
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
31936
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
81
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
28143
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
242
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
124
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
291
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Filter_TE_list &lt;- Col_fullDF_overlap %&gt;% 
   as.data.frame %&gt;% 
  mutate(per_ol= width/TE_width) 
  # dplyr::filter(per_ol&gt;0.5)

Unique_peak_overlap &lt;- Col_fullDF_overlap %&gt;%
  as.data.frame() %&gt;%
  distinct(Peakid)

peak_overlap_50unique &lt;-  Filter_TE_list %&gt;%
   dplyr::filter(per_ol&gt;0.5) %&gt;% 
  distinct(Peakid)</code></pre>
<p>Note that the counts of peaks total to more than the total number of
peaks that overlap a TE because many peaks overlap multiple elements
(generally the very small TEs).</p>
<p>A summary of numbers of peaks is below:</p>
<ul>
<li>Total number of peaks = 172481</li>
<li>Total number of peaks overlapping at least 1 TE = 104149<br />
</li>
<li>Total number of peaks overlapping by &gt;50% TE length = 81185<br />
<strong>note</strong> these numbers include peaks that are not
classified by an MRC.</li>
</ul>
<p>Tables 3 above used the better way of sub-setting (keeping all
metadata organized vs the sub-setting code I found on the internet).</p>
</div>
<div id="distribution-of-te-overlaps-by-peaks-according-to-class"
class="section level3">
<h3>Distribution of TE overlaps by peaks according to class</h3>
<p>Below are plots of the distribution of overlapping widths between
peaks and TEs. The first plot is all TE Class, and the 2nd plot limits
the classes to LINEs, SINEs, LTRs, DNAs, and Retroposons.</p>
<pre class="r"><code>Col_fullDF_overlap %&gt;%  
  as.data.frame %&gt;% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color=&quot;darkblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping widths in my data&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,750))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/another%20set%20of%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-1">
Past versions of "another set of plots-1.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/another set of plots-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Col_fullDF_overlap %&gt;%  
  as.data.frame %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass ==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass ==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color=&quot;darkblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths&quot;,subtitle = &quot;Just LINE, SINE,LTR, DNA, Retroposons; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,1550))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/another%20set%20of%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-2">
Past versions of "another set of plots-2.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/another set of plots-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Getting the numbers of peaks containing an overlap of &gt; 50% of the
TE required creation of a data frame. The first step of the dataframe
was to use all high confidence Peaks and join with the TE_Overlapping
dataframe.</p>
<pre class="r"><code>Peak_TE_overlapbreakdown &lt;-  Col_TSS_data_gr %&gt;% 
  as.data.frame %&gt;%
  distinct(Peakid) %&gt;%
  left_join(.,(Col_fullDF_overlap %&gt;% as.data.frame)) %&gt;%
  dplyr::select(Peakid, repName:repFamily,TE_width,width) %&gt;% 
  left_join(.,(TSS_data_gr %&gt;% as.data.frame), by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(mrc=if_else(Peakid %in% NR_df$Peakid, &quot;NR&quot;,
                    if_else(Peakid %in% ESR_df$Peakid,&quot;ESR&quot;,
                       if_else(Peakid %in% LR_df$Peakid,&quot;LR&quot;,
  if_else(Peakid %in% EAR_df$Peakid, &quot;EAR&quot;, &quot;not_mrc&quot;))))) %&gt;%   mutate(per_ol= width.x/TE_width)

Peak_TE_overlapbreakdown %&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
distinct(Peakid,.keep_all = TRUE) %&gt;% 
  group_by(mrc,TEstatus) %&gt;% tally() </code></pre>
<pre><code># A tibble: 10 × 3
# Groups:   mrc [5]
   mrc     TEstatus        n
   &lt;chr&gt;   &lt;chr&gt;       &lt;int&gt;
 1 EAR     TE_peak      4858
 2 EAR     not_TE_peak  2596
 3 ESR     TE_peak     10256
 4 ESR     not_TE_peak  5525
 5 LR      TE_peak     29288
 6 LR      not_TE_peak 13321
 7 NR      TE_peak     57561
 8 NR      not_TE_peak 28767
 9 not_mrc TE_peak      2186
10 not_mrc not_TE_peak  1199</code></pre>
<pre class="r"><code># TE_mrc_status_list %&gt;% 
#   distinct(Peakid, .keep_all = TRUE) %&gt;% 
#   group_by(TEstatus, mrc) %&gt;%  
#   tally #%&gt;% 
#   dplyr::filter(n&gt;=4&amp;mrc==&quot;EAR&quot;)
  
  
  # Col_fullDF_cug_overlap %&gt;% as.data.frame() %&gt;% group_by(Peakid) %&gt;% tally %&gt;% dplyr::filter(n&gt;=4)</code></pre>
<p>This increased the number of rows from 172481 to 228784 indicating
many peaks contained more than one TE. The column <code>per_ol</code> is
representing the proportion of TE covered within the peak. I next
decided to only count a peak to contain a TE, if it covered &gt;50% of
the length of the TE.</p>
<pre class="r"><code>per_cov=0
TE_mrc_status_list &lt;- Peak_TE_overlapbreakdown %&gt;%
   mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) 
 
TE_mrc_status_list%&gt;% 
   distinct(Peakid,TEstatus,.keep_all = TRUE) %&gt;%
   mutate(mrc=&quot;all_peaks&quot;)%&gt;% 
   rbind((TE_mrc_status_list %&gt;% distinct(Peakid,TEstatus,.keep_all = TRUE))) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
     ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+ 
   theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family using the &quot;,per_cov*100,&quot;% cutoff&quot;))</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Making%20of%204group%20TE%20dataframe-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Making-of-4group-TE-dataframe-1">
Past versions of "Making of 4group TE dataframe-1.png"
</button>
</p>
<div id="fig-no-spaces-Making-of-4group-TE-dataframe-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Making of 4group TE dataframe-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The table below represents the numbers for each of the four groups
when I include the &gt;50% stringency cutoff filter.</p>
<pre class="r"><code>per_cov=0
 
TE_mrc_status_list %&gt;% 
   distinct(Peakid,.keep_all = TRUE) %&gt;%
   mutate(mrc=&quot;all_peaks&quot;)%&gt;% 
   rbind((TE_mrc_status_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  group_by(mrc, TEstatus) %&gt;% 
  count() %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100) %&gt;% 
      kable(., caption=&quot;Table 4: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC using strigency cutoff of 50%.&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 4: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC using strigency cutoff of 50%.
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
summary
</th>
<th style="text-align:left;">
TEstatus
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
4858
</td>
<td style="text-align:right;">
65.17306
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
2596
</td>
<td style="text-align:right;">
34.82694
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
10256
</td>
<td style="text-align:right;">
64.98954
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
5525
</td>
<td style="text-align:right;">
35.01046
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
29288
</td>
<td style="text-align:right;">
68.73665
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
13321
</td>
<td style="text-align:right;">
31.26335
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
57561
</td>
<td style="text-align:right;">
66.67709
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
28767
</td>
<td style="text-align:right;">
33.32291
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
104149
</td>
<td style="text-align:right;">
66.95231
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
51408
</td>
<td style="text-align:right;">
33.04769
</td>
</tr>
</tbody>
</table>
<p>Problem with summary of TE ## Human genome TE breakdown</p>
<p>I first wanted to know the distribution of TEs across the human
genome compared to each other. Below are pie plots with all classes from
repeatmasker, and then pie plots with ONLY the LINES, SINES, LTRs, DNAs,
and Retroposon classes.</p>
<pre class="r"><code>repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/repeatmasker%20genome%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-1">
Past versions of "repeatmasker genome breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/repeatmasker genome breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>LiSiLTDNRe &lt;- repeatmasker %&gt;%
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;)


repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon only&quot;, subtitle=paste(length(LiSiLTDNRe$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/repeatmasker%20genome%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-2">
Past versions of "repeatmasker genome breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/repeatmasker genome breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>repeatmasker  %&gt;% 
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  # dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon focused with other&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/repeatmasker%20genome%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-3">
Past versions of "repeatmasker genome breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-3"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/repeatmasker genome breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># saveRDS(TE_mrc_status_list,&quot;data/TE_info/TE_mrc_status_list.RDS&quot;)
# TE_ALL_count &lt;- TE_mrc_status_list %&gt;%
#   dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
#   dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;%
#   distinct(Peakid) %&gt;% 
#     count
# TE_ALL_count_filt &lt;- TE_mrc_status_list %&gt;%
#   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
#   dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
#   dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
#   distinct(Peakid) %&gt;% 
#     count
TE_50_count &lt;- TE_mrc_status_list %&gt;%
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(Peakid) %&gt;% 
    count
TE_50_count_filt &lt;- TE_mrc_status_list %&gt;%
   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  # dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(Peakid) %&gt;% 
    count

TE_mrc_status_list %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;% 
   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35)+
  # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of Just LINEs, SINEs, LTRs, DNAs, and Retroposons&quot;,subtitle = paste(TE_50_count_filt$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TE%20in%20all%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-1">
Past versions of "TE in all peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TE in all peaks-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
    # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
    #                  position = position_stack(vjust = .3),
    #                  show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks using 50% cutoff&quot;,subtitle = paste(TE_50_count$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TE%20in%20all%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-2">
Past versions of "TE in all peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TE in all peaks-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;%
  count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks using 50% cutoff&quot;,subtitle = paste(TE_50_count_filt$n, &quot;peaks that contain LINEs, SINEs, LTRs, DNAs, or Retroposons&quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TE%20in%20all%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-3">
Past versions of "TE in all peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TE in all peaks-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>### note the change in data frame Renee, this is where you flipped back to the original without filters.
Peak_TE_overlapbreakdown%&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TEs%20in%20all%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-1">
Past versions of "TEs in all peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TEs in all peaks-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown%&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths &gt;50%&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TEs%20in%20all%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-2">
Past versions of "TEs in all peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TEs in all peaks-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown%&gt;%
  mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;%
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths &gt;50% of TE length&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TEs%20in%20all%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-3">
Past versions of "TEs in all peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TEs in all peaks-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown %&gt;% 
  mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  distinct() %&gt;% 
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  scale_fill_repeat()+
  theme_classic()+
  ggtitle(&quot;Filtered Distribution of all overlapping peak-TE widths &gt;50 TE length%&quot;)</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TEs%20in%20all%20peaks-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-4">
Past versions of "TEs in all peaks-4.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TEs in all peaks-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="line-repeats" class="section level4">
<h4>Line repeats</h4>
<pre class="r"><code>Line_df %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LINE breakdown&quot;, subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LINEclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-1">
Past versions of "LINEclass breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LINEclass breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;) %&gt;%
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;%
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE breakdown of peaks &quot;,per_cov),subtitle=paste(TE_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LINEclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-2">
Past versions of "LINEclass breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LINEclass breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count
ESR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count
LR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count
NR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;EAR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(EAR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LINEclass%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-3">
Past versions of "LINEclass breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LINEclass breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;ESR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(ESR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LINEclass%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-4">
Past versions of "LINEclass breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LINEclass breakdown-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(LR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LINEclass%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-5">
Past versions of "LINEclass breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LINEclass breakdown-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;NR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(NR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LINEclass%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-6">
Past versions of "LINEclass breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LINEclass breakdown-6.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;) %&gt;% 
  tally
EAR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;EAR&quot;) %&gt;% 
  tally
ESR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;ESR&quot;) %&gt;% 
  tally
LR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;LR&quot;) %&gt;% 
  tally
NR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;NR&quot;) %&gt;% 
  tally


TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for all peaks &quot;,per_cov),subtitle=paste(all_L2_count,&quot; total LINEs&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/L2breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-1">
Past versions of L2breakdown-1.png
</button>
</p>
<div id="fig-L2breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/L2breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;%
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for EAR &quot;,per_cov),subtitle=paste(EAR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/L2breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-2">
Past versions of L2breakdown-2.png
</button>
</p>
<div id="fig-L2breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/L2breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for ESR &quot;,per_cov),subtitle=paste(ESR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/L2breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-3">
Past versions of L2breakdown-3.png
</button>
</p>
<div id="fig-L2breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/L2breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for LR &quot;,per_cov),subtitle=paste(LR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/L2breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-4">
Past versions of L2breakdown-4.png
</button>
</p>
<div id="fig-L2breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/L2breakdown-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for NR &quot;,per_cov),subtitle=paste(NR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/L2breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-5">
Past versions of L2breakdown-5.png
</button>
</p>
<div id="fig-L2breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/L2breakdown-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="sine-repeats" class="section level4">
<h4>Sine repeats</h4>
<pre class="r"><code>Sine_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome SINE breakdown&quot;, subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Sineclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-1">
Past versions of "Sineclass breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Sineclass breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;SINE breakdown of peaks &quot;,per_cov),subtitle=paste(TE_SINE_count$n,&quot; total SINEs found&quot;))+
  scale_fill_sines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Sineclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-2">
Past versions of "Sineclass breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Sineclass breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count
ESR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count
LR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count
NR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;EAR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(EAR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Sineclass%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-3">
Past versions of "Sineclass breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Sineclass breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  # 
  theme_void()+
  ggtitle(paste0(&quot;ESR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(ESR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Sineclass%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-4">
Past versions of "Sineclass breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Sineclass breakdown-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(LR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Sineclass%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-5">
Past versions of "Sineclass breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Sineclass breakdown-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;NR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(NR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/Sineclass%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-6">
Past versions of "Sineclass breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/Sineclass breakdown-6.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="ltr-repeats" class="section level4">
<h4>LTR repeats</h4>
<pre class="r"><code>per_cov &lt;- 0.0


LTR_df%&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LTR breakdown&quot;, subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LTR%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-1">
Past versions of "LTR breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LTR breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  count
EAR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count
ESR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count
LR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count
NR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;All peaks LTR breakdown of peaks&quot;,per_cov),subtitle=paste(LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LTR%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-2">
Past versions of "LTR breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LTR breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;EAR LTR breakdown of peaks&quot;,per_cov),subtitle=paste(EAR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LTR%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-3">
Past versions of "LTR breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LTR breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR LTR breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LTR%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-4">
Past versions of "LTR breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LTR breakdown-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR LTR breakdown of peaks &quot;,per_cov),subtitle=paste(LR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LTR%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-5">
Past versions of "LTR breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LTR breakdown-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR LTR breakdown of peaks&quot;, per_cov),subtitle=paste(NR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/LTR%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-6">
Past versions of "LTR breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/LTR breakdown-6.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dna-tes" class="section level4">
<h4>DNA TEs</h4>
<pre class="r"><code>per_cov &lt;- 0.0

DNA_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;Human genome DNA breakdown&quot;, subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/DNA%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-1">
Past versions of "DNA breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/DNA breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;DNA breakdown of peaks&quot;, per_cov),subtitle=paste(TE_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/DNA%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-2">
Past versions of "DNA breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/DNA breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count
ESR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count
LR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count
NR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count


TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;EAR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(EAR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/DNA%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-3">
Past versions of "DNA breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/DNA breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/DNA%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-4">
Past versions of "DNA breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/DNA breakdown-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(LR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/DNA%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-5">
Past versions of "DNA breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/DNA breakdown-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR DNA breakdown of peaks&quot;, per_cov),subtitle=paste(NR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/DNA%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-6">
Past versions of "DNA breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/DNA breakdown-6.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="retroposon" class="section level4">
<h4>retroposon</h4>
<pre class="r"><code>per_cov &lt;- 0.0

retroposon_df%&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label =  paste0(repName ,&quot;\n&quot;, sprintf(&quot;%.2f&quot;, perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome retroposon breakdown&quot;, subtitle= paste( length(retroposon_df$milliIns)))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/retroposon%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-1">
Past versions of "retroposon breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/retroposon breakdown-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;) %&gt;% 
   mutate(repName=factor(repName)) %&gt;% 
     # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(TE_retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/retroposon%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-2">
Past versions of "retroposon breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/retroposon breakdown-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count
ESR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count
LR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count
NR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count


TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/retroposon%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-3">
Past versions of "retroposon breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/retroposon breakdown-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(LR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/retroposon%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-4">
Past versions of "retroposon breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/retroposon breakdown-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(NR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/retroposon%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-5">
Past versions of "retroposon breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/retroposon breakdown-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="te-by-response-cluster-bargraph-summaries"
class="section level3">
<h3>TE by response cluster bargraph summaries</h3>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                          if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) %&gt;%
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
  rbind((TE_mrc_status_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;%
  mutate(repClass=factor(repClass)) %&gt;%
  mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  # dplyr::filter(is.na(per_ol)| per_ol&gt;per_cov) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;,&quot;&gt;&quot;, per_cov*100,&quot;%&quot;))</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/TE%20status%20old%20clusters-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-status-old-clusters-1">
Past versions of "TE status old clusters-1.png"
</button>
</p>
<div id="fig-no-spaces-TE-status-old-clusters-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/TE status old clusters-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Lines_Etc &lt;-   TE_mrc_status_list %&gt;%
  mutate(repClass_org = repClass) %&gt;%  
  # dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
              if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol)%&gt;% 
    distinct(Peakid,mrc, .keep_all = TRUE) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;)</code></pre>
<pre class="r"><code>LiSiLTDNRe_TE &lt;-TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
        if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) %&gt;%
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  rbind(Lines_Etc) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;%
  mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  group_by(mrc, TEstatus) %&gt;% 
  count() %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
  rowwise() %&gt;% 
  mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
  ungroup() %&gt;% 
  pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
  mutate(percent_mrc= n/summary*100)

LiSiLTDNRe_TE %&gt;% 
  kable(., caption=&quot;Table 5: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC in TE_peak count&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 5: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC in TE_peak count
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
summary
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
TEstatus
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
n
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
4858
</td>
<td style="text-align:right;">
65.17306
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
2596
</td>
<td style="text-align:right;">
34.82694
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
10256
</td>
<td style="text-align:right;">
64.98954
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
5525
</td>
<td style="text-align:right;">
35.01046
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
29288
</td>
<td style="text-align:right;">
68.73665
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
13321
</td>
<td style="text-align:right;">
31.26335
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
57561
</td>
<td style="text-align:right;">
66.67709
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
28767
</td>
<td style="text-align:right;">
33.32291
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
104149
</td>
<td style="text-align:right;">
66.95231
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
51408
</td>
<td style="text-align:right;">
33.04769
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="breakdown-of-classes-by-4-member-clusters"
class="section level3">
<h3>Breakdown of Classes by 4 member clusters</h3>
<pre class="r"><code>per_cov &lt;- 0.0
 

  ggline_df &lt;-   TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;LINE&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

ggsine_df &lt;-TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;SINE&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 
ggLTR_df &lt;- TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;LTR&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 
ggDNA_df &lt;-TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;DNA&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 
ggretroposon_df &lt;-TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;Retroposons&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 
plot1 &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggline_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LINE breakdown by MRC and Family&quot;, per_cov))+
  scale_fill_lines()
plot1</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/breakdown%20by%20cluster-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-1">
Past versions of "breakdown by cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/breakdown by cluster-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot2 &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;SINE breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_sines()
plot2</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/breakdown%20by%20cluster-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-2">
Past versions of "breakdown by cluster-2.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/breakdown by cluster-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggLTR_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LTR breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/breakdown%20by%20cluster-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-3">
Past versions of "breakdown by cluster-3.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/breakdown by cluster-3.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggDNA_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;DNA breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/breakdown%20by%20cluster-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-4">
Past versions of "breakdown by cluster-4.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/breakdown by cluster-4.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggretroposon_df) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;Retroposon breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/breakdown%20by%20cluster-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-5">
Past versions of "breakdown by cluster-5.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/breakdown by cluster-5.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="adding-in-the-new-categories" class="section level3">
<h3>Adding in the new categories</h3>
<p>Here I break up the categories using median LFC</p>
<ul>
<li>EAR_open = median 3 hour LFC &gt; 0<br />
</li>
<li>EAR_close = median 3 hour LFC &lt; 0</li>
<li>ESR_A = median 3 hour &gt; 0 and median 24 hour &gt;0</li>
<li>ESR_B = median 3 hour &lt; 0 and median 24 hour &lt;0</li>
<li>ESR_opcl [C] = median 3 hour &gt; 0 and median 24 hour &lt;0</li>
<li>ESR_clop [D] = median 3 hour &lt; 0 and median 24 hour &gt;0</li>
<li>LR_open = median 24 hour LFC &gt; 0<br />
</li>
<li>LR_close = median 24 hour LFC &lt; 0<br />
</li>
<li>NR = all NR classified peaks</li>
</ul>
<pre class="r"><code>per_cov=0
median_24_lfc &lt;- read_csv(&quot;data/Final_four_data/median_24_lfc.csv&quot;) 
median_3_lfc &lt;- read_csv(&quot;data/Final_four_data/median_3_lfc.csv&quot;)

open_3med &lt;- median_3_lfc %&gt;% 
  dplyr::filter(med_3h_lfc &gt; 0)

close_3med &lt;- median_3_lfc %&gt;% 
  dplyr::filter(med_3h_lfc &lt; 0)

open_24med &lt;- median_24_lfc %&gt;% 
  dplyr::filter(med_24h_lfc &gt; 0)

close_24med &lt;- median_24_lfc %&gt;% 
  dplyr::filter(med_24h_lfc &lt; 0)

medA &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(med_3h_lfc &gt; 0 &amp; med_24h_lfc&gt;0)

medB &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(med_3h_lfc &lt; 0 &amp; med_24h_lfc &lt; 0)
 
medC &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(med_3h_lfc &gt; 0&amp; med_24h_lfc &lt;0)
  

medD &lt;- median_3_lfc %&gt;% 
 left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;))%&gt;% 
  dplyr::filter(med_3h_lfc &lt; 0 &amp; med_24h_lfc &gt; 0)
 

EAR_open &lt;- EAR_df %&gt;%
  dplyr::filter(Peakid %in% open_3med$peak)
  
EAR_open_gr &lt;- EAR_open %&gt;% GRanges()

EAR_close &lt;- EAR_df %&gt;%
  dplyr::filter(Peakid %in% close_3med$peak) 

EAR_close_gr &lt;- EAR_close %&gt;% GRanges()

LR_open &lt;- LR_df %&gt;%
  dplyr::filter(Peakid %in% open_24med$peak) 

LR_open_gr &lt;- LR_open %&gt;% GRanges()

LR_close &lt;- LR_df %&gt;%
  dplyr::filter(Peakid %in% close_24med$peak) 

LR_close_gr &lt;- LR_close %&gt;% GRanges()

NR_gr &lt;- NR_df %&gt;% 
   GRanges()

ESR_open &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medA$peak)  
 
ESR_open_gr &lt;- ESR_open %&gt;% GRanges()

ESR_close &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medB$peak)  

ESR_close_gr &lt;- ESR_close %&gt;% GRanges()

ESR_C &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medC$peak) 
ESR_opcl &lt;- ESR_C
ESR_D &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medD$peak) 
ESR_clop &lt;- ESR_D

ESR_OC &lt;- ESR_C %&gt;% 
  rbind(ESR_D)
ESR_OC_gr &lt;- ESR_OC %&gt;% GRanges()

Nine_group_TE &lt;-Col_TSS_data_gr %&gt;%
  as.data.frame %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  left_join(.,(Col_fullDF_overlap %&gt;% 
                 as.data.frame)) %&gt;% 
   mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_opcl$Peakid,&quot;ESR_opcl&quot;,
                                           if_else(Peakid %in% ESR_clop$Peakid,&quot;ESR_clop&quot;,                   if_else(Peakid %in% LR_open$Peakid, &quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid, &quot;LR_close&quot;,
                                     if_else(Peakid %in% NR_df$Peakid, &quot;NR&quot;, &quot;not_mrc&quot;)))))))))) %&gt;%  
                                       mutate(per_ol= width/TE_width) %&gt;% 
  mutate(repClass_org=repClass) %&gt;% 
  mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  mutate(repClass=factor(repClass)) %&gt;%
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org, &quot;Other&quot;))))))) %&gt;% 
  dplyr::select(Peakid, repName,repClass,repClass_org, repFamily, width, TEstatus, mrc, per_ol)</code></pre>
</div>
<div id="sub-set-breakdown-with-new-classes" class="section level3">
<h3>sub-set breakdown with new classes</h3>
<p>This section reclassifies any TE that is NOT a LINE, SINE, LTR, DNA,
or retroposon as “other”. Still using &gt; 50% coverage cutoff</p>
<pre class="r"><code>Class_status_df &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
    # mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
  # rbind(subsetall_df) %&gt;% 
   mutate(Sine_status = if_else(is.na(repClass),&quot;not_sine&quot;,
                               if_else(repClass==&quot;SINE&quot;,&quot;sine_peak&quot;, &quot;not_sine&quot;))) %&gt;% 
   mutate(Line_status = if_else(is.na(repClass),&quot;not_line&quot;,
                                if_else(repClass==&quot;LINE&quot;,&quot;line_peak&quot;, &quot;not_line&quot;))) %&gt;%
   mutate(LTR_status = if_else(is.na(repClass),&quot;not_LTR&quot;,
                               if_else(repClass==&quot;LTR&quot;,&quot;LTR_peak&quot;, &quot;not_LTR&quot;))) %&gt;% 
   mutate(DNA_status = if_else(is.na(repClass),&quot;not_DNA&quot;,
                                if_else(repClass==&quot;DNA&quot;,&quot;DNA_peak&quot;, &quot;not_DNA&quot;))) %&gt;% 
   mutate(Retro_status = if_else(is.na(repClass)&amp;is.na(per_ol),&quot;not_Retro&quot;,
                                if_else(repClass==&quot;Retroposon&quot;,&quot;Retro_peak&quot;, &quot;not_Retro&quot;))) %&gt;% 
     mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
    mutate(Sine_status=factor(Sine_status, levels = c(&quot;sine_peak&quot;,&quot;not_sine&quot;)),
       Line_status=factor(Line_status, levels =c(&quot;line_peak&quot;,&quot;not_line&quot;)),
       LTR_status=factor(LTR_status, levels =c(&quot;LTR_peak&quot;,&quot;not_LTR&quot;)),
       DNA_status=factor(DNA_status, levels =c(&quot;DNA_peak&quot;,&quot;not_DNA&quot;)),
       Retro_status=factor(Retro_status, levels =c(&quot;Retro_peak&quot;,&quot;not_Retro&quot;))) %&gt;% 
     mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_opcl&quot;,&quot;ESR_clop&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;)))</code></pre>
<pre class="r"><code>Class_status_df  %&gt;% distinct(Peakid,.keep_all = TRUE) %&gt;% 
  group_by(mrc, TEstatus) %&gt;% 
  tally</code></pre>
<pre><code># A tibble: 18 × 3
# Groups:   mrc [9]
   mrc       TEstatus        n
   &lt;fct&gt;     &lt;fct&gt;       &lt;int&gt;
 1 EAR_open  TE_peak      2054
 2 EAR_open  not_TE_peak  1104
 3 EAR_close TE_peak      2804
 4 EAR_close not_TE_peak  1492
 5 ESR_open  TE_peak      3519
 6 ESR_open  not_TE_peak  1753
 7 ESR_close TE_peak      5934
 8 ESR_close not_TE_peak  3415
 9 ESR_opcl  TE_peak        68
10 ESR_opcl  not_TE_peak    31
11 ESR_clop  TE_peak       735
12 ESR_clop  not_TE_peak   326
13 LR_open   TE_peak     20072
14 LR_open   not_TE_peak  8465
15 LR_close  TE_peak      9216
16 LR_close  not_TE_peak  4856
17 NR        TE_peak     57561
18 NR        not_TE_peak 28767</code></pre>
</div>
<div id="cg-islands" class="section level3">
<h3>CG islands!</h3>
<pre class="r"><code>cpgislands_df &lt;- read.delim(&quot;data/other_papers/cpg_islands.tsv&quot;)
cpg_cCREs_df &lt;- read_delim(&quot;data/other_papers/cpg_cCREs.tsv&quot;, delim=&quot;\t&quot;)


CPG_promoters_gr &lt;- cpg_cCREs_df %&gt;% 
    dplyr::rename(.,&quot;seqnames&quot;=X1,&quot;start&quot;=X2,&quot;end&quot;=X3,&quot;promotor_name&quot;=X4,&quot;length&quot;=X5,&quot;strand&quot;=X6,&quot;color&quot;=X9) %&gt;% 
  dplyr::filter(color ==25500) %&gt;% 
  dplyr::select(seqnames:strand,color) %&gt;% 
  GRanges() 


Peaks_v_cpgpromo &lt;- join_overlap_intersect(CPG_promoters_gr,Col_TSS_data_gr) %&gt;% as.data.frame

cpg_island_gr &lt;- cpgislands_df %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;chrom&quot;, start.field = &quot;chromStart&quot;, end.field = &quot;chromEnd&quot;,starts.in.df.are.0based=TRUE)
Col_TSS_data_gr$peak_width &lt;- width(Col_TSS_data_gr)
cpg_island_gr$cpg_width &lt;- width(cpg_island_gr)</code></pre>
<pre class="r"><code>Col_fullDF_cug_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,cpg_island_gr)
Col_fullDF_cug_overlap &lt;-Col_fullDF_cug_overlap %&gt;% 
  as.data.frame %&gt;% 
  mutate(per_ol=width/cpg_width)

CUG_mrc_nine_list &lt;-
Col_TSS_data_gr%&gt;% as.data.frame() %&gt;%
  left_join(., (Col_fullDF_cug_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;Peakid&quot;=&quot;Peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;,  &quot;SYMBOL&quot; = &quot;SYMBOL&quot;, &quot;peak_width&quot;=&quot;peak_width&quot;)) %&gt;% 
  dplyr::select(Peakid, name,cpgNum:per_ol) %&gt;% 
  mutate(cugstatus=if_else(is.na(cpgNum),&quot;not_CGi_peak&quot;,&quot;CGi_peak&quot;)) %&gt;% 
    mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_clop$Peakid,&quot;ESR_clop&quot;,if_else(Peakid %in% ESR_opcl,&quot;ESR_opcl&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR_df$Peakid,&quot;NR&quot;,&quot;not_mrc&quot;)))))))))) %&gt;%
  distinct()</code></pre>
<pre class="r"><code>Col_fullDF_cug_overlap %&gt;% 
        as.data.frame %&gt;% 
        ggplot(., aes (x = width))+
        geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of CGisland overlap widths&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/cug%20eight%20stuff-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cug-eight-stuff-1">
Past versions of "cug eight stuff-1.png"
</button>
</p>
<div id="fig-no-spaces-cug-eight-stuff-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/cug eight stuff-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>cpg_island_gr %&gt;% 
  as.data.frame() %&gt;% 
  ggplot(., aes (x = cpg_width))+
        geom_density(aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of overlap widths of CGislands &quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/cug%20eight%20stuff-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cug-eight-stuff-2">
Past versions of "cug eight stuff-2.png"
</button>
</p>
<div id="fig-no-spaces-cug-eight-stuff-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/cug eight stuff-2.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="te-odds-ratio-matrix" class="section level3">
<h3>TE odds ratio matrix</h3>
<pre class="r"><code>## full peak set as all peaks
# subsetall_df
### making of nine set

TE_mat&lt;- Nine_group_TE %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(TEstatus, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = TEstatus,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  as.matrix(.)
  
# saveRDS(Nine_te_df,&quot;data/Final_four_data/Nine_group_TE_df.RDS&quot;)</code></pre>
<pre class="r"><code>### makeing of nine set
Nine_class_status_df &lt;-
Class_status_df %&gt;% 
  mutate(mrc=if_else(Peakid %in% ESR_C$Peakid,&quot;ESR_opcl&quot;,                 if_else(Peakid %in% ESR_D$Peakid,&quot;ESR_clop&quot;,mrc)))

Sine_mat&lt;- Nine_class_status_df %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  group_by(Sine_status, mrc) %&gt;% 
  tally %&gt;% 
  
  pivot_wider(id_cols = mrc, names_from = Sine_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

Line_mat&lt;- Nine_class_status_df %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(Line_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = Line_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

LTR_mat&lt;- Nine_class_status_df %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(LTR_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = LTR_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)

Retro_mat&lt;- Nine_class_status_df %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(Retro_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = Retro_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;%
  na.omit(.) %&gt;% 
  as.matrix(.)

DNA_mat&lt;- Nine_class_status_df %&gt;% 
 dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(DNA_status, mrc) %&gt;% 
  tally %&gt;% 
  mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  pivot_wider(id_cols = mrc, names_from = DNA_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit(.) %&gt;% 
  as.matrix(.)


CUG_mat &lt;- CUG_mrc_nine_list %&gt;% 
  distinct(Peakid,.keep_all = TRUE) %&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=if_else(Peakid %in% ESR_C$Peakid,&quot;ESR_opcl&quot;,                 if_else(Peakid %in% ESR_D$Peakid,&quot;ESR_clop&quot;,mrc))) %&gt;% 
    mutate(mrc= factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;, &quot;ESR_opcl&quot;, &quot;ESR_clop&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;))) %&gt;% 
  group_by(cugstatus, mrc) %&gt;% 
  tally  %&gt;% 
  pivot_wider(id_cols = mrc, names_from = cugstatus,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;% 
  na.omit() %&gt;% 
  as.matrix(.)</code></pre>
<pre class="r"><code>txdb &lt;- TxDb.Hsapiens.UCSC.hg38.knownGene
transcripts &lt;- transcripts(txdb)
tss &lt;- resize(transcripts, width = 1, fix = &quot;start&quot;)

overlaps &lt;- findOverlaps(Col_TSS_data_gr,tss)
overlapping_regions &lt;- Col_TSS_data_gr[queryHits(overlaps)]
TSS_overlap_list &lt;- overlapping_regions %&gt;% as.data.frame() %&gt;% 
  distinct(Peakid)

TSS_mat &lt;-
Col_TSS_data_gr %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  mutate(TSS_status= if_else(Peakid %in% TSS_overlap_list$Peakid,&quot;TSS_peak&quot;,&quot;not_TSS_peak&quot;)) %&gt;% 
   mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_C$Peakid,&quot;ESR_opcl&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR_df$Peakid,&quot;NR&quot;,if_else(Peakid %in% ESR_D,&quot;ESR_clop&quot;,&quot;not_mrc&quot;))))))))) ) %&gt;% 
  group_by(TSS_status, mrc) %&gt;% 
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
  tally  %&gt;% 
  pivot_wider(id_cols = mrc, names_from = TSS_status,values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) %&gt;%
  na.omit(.) %&gt;% 
  as.matrix(.)</code></pre>
</div>
<div id="chi-square-results" class="section level3">
<h3>chi square results</h3>
<pre class="r"><code>matrix_list &lt;- list(&quot;TE&quot;=TE_mat, &quot;Sines&quot;=Sine_mat, &quot;Lines&quot;=Line_mat,&quot;DNA&quot;= DNA_mat,&quot;Retro&quot;= Retro_mat,&quot;LTR&quot;= LTR_mat,&quot;CGI&quot;=CUG_mat,&quot;TSS&quot;=TSS_mat)
results &lt;- data.frame(Matrix_Name = character(),
                      Row_Compared = character(),
                      Chi_Square_Statistic = numeric(),
                      P_Value = numeric(),
                      stringsAsFactors = FALSE)
names_list &lt;- names(matrix_list)
# Loop through each matrix in the list
for (matrix_name in names_list) {
  current_matrix &lt;- matrix_list[[matrix_name]]
  n_rows &lt;- nrow(current_matrix)
  
  # Print matrix details for debugging
  cat(&quot;Processing Matrix:&quot;, matrix_name, &quot;\n&quot;)
  print(current_matrix)
  
  # Loop through each row of the current matrix (except the last row)
  for (i in 1:(n_rows - 1)) {
    # Print row details for debugging
    cat(&quot;Comparing row&quot;, i, &quot;with last row:\n&quot;)
    print(current_matrix[i, ])
    print(current_matrix[n_rows, ])
    
    # Perform chi-square test between row i and the last row
    test_result &lt;- tryCatch({
      chisq.test(rbind(current_matrix[i, ], current_matrix[n_rows, ]))
    }, error = function(e) {
      cat(&quot;Error in chi-square test:&quot;, e$message, &quot;\n&quot;)
      return(NULL)
    })
    
    # Only store the result if test_result is valid (i.e., not NULL)
    if (!is.null(test_result)) {
      results &lt;- rbind(results, data.frame(Matrix_Name = matrix_name,
                                           Row_Compared = rownames(current_matrix)[i],
                                           Chi_Square_Statistic = test_result$statistic,
                                           P_Value = test_result$p.value))
    }
  }
}</code></pre>
<pre><code>Processing Matrix: TE 
          TE_peak not_TE_peak
EAR_open     3736        1104
EAR_close    4436        1492
ESR_open     6092        1753
ESR_close    9352        3415
ESR_opcl      107          31
ESR_clop     1208         326
LR_open     34312        8465
LR_close    14908        4856
NR          98024       28767
Comparing row 1 with last row:
    TE_peak not_TE_peak 
       3736        1104 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 2 with last row:
    TE_peak not_TE_peak 
       4436        1492 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 3 with last row:
    TE_peak not_TE_peak 
       6092        1753 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 4 with last row:
    TE_peak not_TE_peak 
       9352        3415 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 5 with last row:
    TE_peak not_TE_peak 
        107          31 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 6 with last row:
    TE_peak not_TE_peak 
       1208         326 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 7 with last row:
    TE_peak not_TE_peak 
      34312        8465 
    TE_peak not_TE_peak 
      98024       28767 
Comparing row 8 with last row:
    TE_peak not_TE_peak 
      14908        4856 
    TE_peak not_TE_peak 
      98024       28767 
Processing Matrix: Sines 
          sine_peak not_sine
EAR_open       1003     3837
EAR_close      1545     4383
ESR_open       1753     6092
ESR_close      3010     9757
ESR_opcl         30      108
ESR_clop        458     1076
LR_open       10996    31781
LR_close       4367    15397
NR            28532    98259
Comparing row 1 with last row:
sine_peak  not_sine 
     1003      3837 
sine_peak  not_sine 
    28532     98259 
Comparing row 2 with last row:
sine_peak  not_sine 
     1545      4383 
sine_peak  not_sine 
    28532     98259 
Comparing row 3 with last row:
sine_peak  not_sine 
     1753      6092 
sine_peak  not_sine 
    28532     98259 
Comparing row 4 with last row:
sine_peak  not_sine 
     3010      9757 
sine_peak  not_sine 
    28532     98259 
Comparing row 5 with last row:
sine_peak  not_sine 
       30       108 
sine_peak  not_sine 
    28532     98259 
Comparing row 6 with last row:
sine_peak  not_sine 
      458      1076 
sine_peak  not_sine 
    28532     98259 
Comparing row 7 with last row:
sine_peak  not_sine 
    10996     31781 
sine_peak  not_sine 
    28532     98259 
Comparing row 8 with last row:
sine_peak  not_sine 
     4367     15397 
sine_peak  not_sine 
    28532     98259 
Processing Matrix: Lines 
          line_peak not_line
EAR_close      1100     4828
EAR_open        739     4101
ESR_clop        290     1244
ESR_close      2233    10534
ESR_opcl         34      104
ESR_open       1403     6442
LR_close       3777    15987
LR_open        8569    34208
NR            21423   105368
Comparing row 1 with last row:
line_peak  not_line 
     1100      4828 
line_peak  not_line 
    21423    105368 
Comparing row 2 with last row:
line_peak  not_line 
      739      4101 
line_peak  not_line 
    21423    105368 
Comparing row 3 with last row:
line_peak  not_line 
      290      1244 
line_peak  not_line 
    21423    105368 
Comparing row 4 with last row:
line_peak  not_line 
     2233     10534 
line_peak  not_line 
    21423    105368 
Comparing row 5 with last row:
line_peak  not_line 
       34       104 
line_peak  not_line 
    21423    105368 
Comparing row 6 with last row:
line_peak  not_line 
     1403      6442 
line_peak  not_line 
    21423    105368 
Comparing row 7 with last row:
line_peak  not_line 
     3777     15987 
line_peak  not_line 
    21423    105368 
Comparing row 8 with last row:
line_peak  not_line 
     8569     34208 
line_peak  not_line 
    21423    105368 
Processing Matrix: DNA 
          DNA_peak not_DNA
EAR_close      408    5520
EAR_open       301    4539
ESR_clop       128    1406
ESR_close      854   11913
ESR_opcl         8     130
ESR_open       546    7299
LR_close      1522   18242
LR_open       3809   38968
NR            9040  117751
Comparing row 1 with last row:
DNA_peak  not_DNA 
     408     5520 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 2 with last row:
DNA_peak  not_DNA 
     301     4539 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 3 with last row:
DNA_peak  not_DNA 
     128     1406 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 4 with last row:
DNA_peak  not_DNA 
     854    11913 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 5 with last row:
DNA_peak  not_DNA 
       8      130 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 6 with last row:
DNA_peak  not_DNA 
     546     7299 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 7 with last row:
DNA_peak  not_DNA 
    1522    18242 
DNA_peak  not_DNA 
    9040   117751 
Comparing row 8 with last row:
DNA_peak  not_DNA 
    3809    38968 
DNA_peak  not_DNA 
    9040   117751 
Processing Matrix: Retro 
          Retro_peak not_Retro
EAR_close          7      5921
EAR_open          22      4818
ESR_clop           2      1532
ESR_close         11     12756
ESR_open          41      7804
LR_close          18     19746
LR_open           38     42739
NR               148    126643
Comparing row 1 with last row:
Retro_peak  not_Retro 
         7       5921 
Retro_peak  not_Retro 
       148     126643 
Comparing row 2 with last row:
Retro_peak  not_Retro 
        22       4818 
Retro_peak  not_Retro 
       148     126643 
Comparing row 3 with last row:
Retro_peak  not_Retro 
         2       1532 
Retro_peak  not_Retro 
       148     126643 </code></pre>
<pre><code>Comparing row 4 with last row:
Retro_peak  not_Retro 
        11      12756 
Retro_peak  not_Retro 
       148     126643 
Comparing row 5 with last row:
Retro_peak  not_Retro 
        41       7804 
Retro_peak  not_Retro 
       148     126643 
Comparing row 6 with last row:
Retro_peak  not_Retro 
        18      19746 
Retro_peak  not_Retro 
       148     126643 
Comparing row 7 with last row:
Retro_peak  not_Retro 
        38      42739 
Retro_peak  not_Retro 
       148     126643 
Processing Matrix: LTR 
          LTR_peak not_LTR
EAR_close      787    5141
EAR_open       427    4413
ESR_clop       225    1309
ESR_close     1508   11259
ESR_opcl        20     118
ESR_open       776    7069
LR_close      2684   17080
LR_open       6116   36661
NR           14757  112034
Comparing row 1 with last row:
LTR_peak  not_LTR 
     787     5141 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 2 with last row:
LTR_peak  not_LTR 
     427     4413 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 3 with last row:
LTR_peak  not_LTR 
     225     1309 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 4 with last row:
LTR_peak  not_LTR 
    1508    11259 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 5 with last row:
LTR_peak  not_LTR 
      20      118 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 6 with last row:
LTR_peak  not_LTR 
     776     7069 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 7 with last row:
LTR_peak  not_LTR 
    2684    17080 
LTR_peak  not_LTR 
   14757   112034 
Comparing row 8 with last row:
LTR_peak  not_LTR 
    6116    36661 
LTR_peak  not_LTR 
   14757   112034 
Processing Matrix: CGI 
          CGi_peak not_CGi_peak
EAR_open        81         3077
EAR_close       17         4279
ESR_open       139         5133
ESR_close       71         9278
LR_open        307        28230
LR_close       102        13970
NR            1265        85063
Comparing row 1 with last row:
    CGi_peak not_CGi_peak 
          81         3077 
    CGi_peak not_CGi_peak 
        1265        85063 
Comparing row 2 with last row:
    CGi_peak not_CGi_peak 
          17         4279 
    CGi_peak not_CGi_peak 
        1265        85063 
Comparing row 3 with last row:
    CGi_peak not_CGi_peak 
         139         5133 
    CGi_peak not_CGi_peak 
        1265        85063 
Comparing row 4 with last row:
    CGi_peak not_CGi_peak 
          71         9278 
    CGi_peak not_CGi_peak 
        1265        85063 
Comparing row 5 with last row:
    CGi_peak not_CGi_peak 
         307        28230 
    CGi_peak not_CGi_peak 
        1265        85063 
Comparing row 6 with last row:
    CGi_peak not_CGi_peak 
         102        13970 
    CGi_peak not_CGi_peak 
        1265        85063 
Processing Matrix: TSS 
          TSS_peak not_TSS_peak
EAR_close      255         4041
EAR_open       940         2218
ESR_close      855         8494
ESR_opcl         7           92
ESR_open       575         4697
LR_close      1407        12665
LR_open       1843        26694
NR           15502        70826
Comparing row 1 with last row:
    TSS_peak not_TSS_peak 
         255         4041 
    TSS_peak not_TSS_peak 
       15502        70826 
Comparing row 2 with last row:
    TSS_peak not_TSS_peak 
         940         2218 
    TSS_peak not_TSS_peak 
       15502        70826 
Comparing row 3 with last row:
    TSS_peak not_TSS_peak 
         855         8494 
    TSS_peak not_TSS_peak 
       15502        70826 
Comparing row 4 with last row:
    TSS_peak not_TSS_peak 
           7           92 
    TSS_peak not_TSS_peak 
       15502        70826 
Comparing row 5 with last row:
    TSS_peak not_TSS_peak 
         575         4697 
    TSS_peak not_TSS_peak 
       15502        70826 
Comparing row 6 with last row:
    TSS_peak not_TSS_peak 
        1407        12665 
    TSS_peak not_TSS_peak 
       15502        70826 
Comparing row 7 with last row:
    TSS_peak not_TSS_peak 
        1843        26694 
    TSS_peak not_TSS_peak 
       15502        70826 </code></pre>
<pre class="r"><code># Print the resulting dataframe
# saveRDS(results,&quot;data/Final_four_data/chisquare_results_TE_df.RDS&quot;)
print(results) %&gt;% 
   kable(., caption = &quot; Chi squre results and significance values of TE enrichment compared to No response group&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<pre><code>            Matrix_Name Row_Compared Chi_Square_Statistic       P_Value
X-squared            TE     EAR_open         3.255012e-02  8.568256e-01
X-squared1           TE    EAR_close         1.965110e+01  9.294970e-06
X-squared2           TE     ESR_open         4.766672e-01  4.899358e-01
X-squared3           TE    ESR_close         1.075397e+02  3.390396e-25
X-squared4           TE     ESR_opcl         1.366893e-25  1.000000e+00
X-squared5           TE     ESR_clop         1.703952e+00  1.917720e-01
X-squared6           TE      LR_open         1.567943e+02  5.676780e-36
X-squared7           TE     LR_close         3.412907e+01  5.157487e-09
X-squared8        Sines     EAR_open         8.385977e+00  3.781268e-03
X-squared9        Sines    EAR_close         4.073898e+01  1.739814e-10
X-squared10       Sines     ESR_open         9.657475e-02  7.559794e-01
X-squared11       Sines    ESR_close         7.576128e+00  5.914635e-03
X-squared12       Sines     ESR_opcl         1.273197e-02  9.101606e-01
X-squared13       Sines     ESR_clop         4.644582e+01  9.418709e-12
X-squared14       Sines      LR_open         1.832910e+02  9.265952e-42
X-squared15       Sines     LR_close         1.607202e+00  2.048856e-01
X-squared16       Lines    EAR_close         1.095372e+01  9.341575e-04
X-squared17       Lines     EAR_open         8.705754e+00  3.172072e-03
X-squared18       Lines     ESR_clop         4.207890e+00  4.023635e-02
X-squared19       Lines    ESR_close         2.866069e+00  9.046560e-02
X-squared20       Lines     ESR_opcl         5.342899e+00  2.080684e-02
X-squared21       Lines     ESR_open         5.048981e+00  2.464042e-02
X-squared22       Lines     LR_close         5.872044e+01  1.817440e-14
X-squared23       Lines      LR_open         2.157756e+02  7.548848e-49
X-squared24         DNA    EAR_close         4.868862e-01  4.853196e-01
X-squared25         DNA     EAR_open         5.729269e+00  1.668443e-02
X-squared26         DNA     ESR_clop         3.188597e+00  7.415364e-02
X-squared27         DNA    ESR_close         3.353784e+00  6.705071e-02
X-squared28         DNA     ESR_opcl         1.959302e-01  6.580261e-01
X-squared29         DNA     ESR_open         2.976560e-01  5.853556e-01
X-squared30         DNA     LR_close         8.251911e+00  4.070912e-03
X-squared31         DNA      LR_open         1.435574e+02  4.439819e-33
X-squared32       Retro    EAR_close         2.269176e-24  1.000000e+00
X-squared33       Retro     EAR_open         3.867129e+01  5.015332e-10
X-squared34       Retro     ESR_clop         5.740842e-29  1.000000e+00
X-squared35       Retro    ESR_close         7.027035e-01  4.018768e-01
X-squared36       Retro     ESR_open         8.395750e+01  5.055255e-20
X-squared37       Retro     LR_close         7.807185e-01  3.769215e-01
X-squared38       Retro      LR_open         2.024021e+00  1.548287e-01
X-squared39         LTR    EAR_close         1.452134e+01  1.385811e-04
X-squared40         LTR     EAR_open         3.596589e+01  2.008021e-09
X-squared41         LTR     ESR_clop         1.319045e+01  2.813794e-04
X-squared42         LTR    ESR_close         3.200447e-01  5.715808e-01
X-squared43         LTR     ESR_opcl         8.316735e-01  3.617891e-01
X-squared44         LTR     ESR_open         2.192667e+01  2.832692e-06
X-squared45         LTR     LR_close         6.128351e+01  4.942022e-15
X-squared46         LTR      LR_open         2.091919e+02  2.061473e-47
X-squared47         CGI     EAR_open         2.412608e+01  9.022978e-07
X-squared48         CGI    EAR_close         3.280949e+01  1.016472e-08
X-squared49         CGI     ESR_open         4.438709e+01  2.694584e-11
X-squared50         CGI    ESR_close         3.001802e+01  4.280499e-08
X-squared51         CGI      LR_open         2.382286e+01  1.056207e-06
X-squared52         CGI     LR_close         4.885169e+01  2.760671e-12
X-squared53         TSS    EAR_close         4.108826e+02  2.355135e-91
X-squared54         TSS     EAR_open         2.824666e+02  2.178182e-63
X-squared55         TSS    ESR_close         4.615046e+02 2.261902e-102
X-squared56         TSS     ESR_opcl         7.236925e+00  7.141918e-03
X-squared57         TSS     ESR_open         1.701874e+02  6.733593e-39
X-squared58         TSS     LR_close         5.466340e+02 6.795877e-121
X-squared59         TSS      LR_open         2.211106e+03  0.000000e+00</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Chi squre results and significance values of TE enrichment compared to
No response group
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Matrix_Name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Row_Compared
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Chi_Square_Statistic
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P_Value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
X-squared
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.0325501
</td>
<td style="text-align:right;">
0.8568256
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared1
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
19.6511019
</td>
<td style="text-align:right;">
0.0000093
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared2
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.4766672
</td>
<td style="text-align:right;">
0.4899358
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared3
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
107.5397339
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared4
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared5
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.7039519
</td>
<td style="text-align:right;">
0.1917720
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared6
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
156.7942828
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared7
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
34.1290747
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared8
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
8.3859774
</td>
<td style="text-align:right;">
0.0037813
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared9
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
40.7389846
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared10
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.0965747
</td>
<td style="text-align:right;">
0.7559794
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared11
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
7.5761282
</td>
<td style="text-align:right;">
0.0059146
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared12
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.0127320
</td>
<td style="text-align:right;">
0.9101606
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared13
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
46.4458194
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared14
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
183.2910301
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared15
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1.6072016
</td>
<td style="text-align:right;">
0.2048856
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared16
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
10.9537226
</td>
<td style="text-align:right;">
0.0009342
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared17
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
8.7057537
</td>
<td style="text-align:right;">
0.0031721
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared18
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
4.2078901
</td>
<td style="text-align:right;">
0.0402364
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared19
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
2.8660688
</td>
<td style="text-align:right;">
0.0904656
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared20
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
5.3428993
</td>
<td style="text-align:right;">
0.0208068
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared21
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
5.0489812
</td>
<td style="text-align:right;">
0.0246404
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared22
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
58.7204361
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared23
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
215.7755637
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared24
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.4868862
</td>
<td style="text-align:right;">
0.4853196
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared25
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
5.7292691
</td>
<td style="text-align:right;">
0.0166844
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared26
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
3.1885968
</td>
<td style="text-align:right;">
0.0741536
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared27
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
3.3537841
</td>
<td style="text-align:right;">
0.0670507
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared28
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.1959302
</td>
<td style="text-align:right;">
0.6580261
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared29
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.2976560
</td>
<td style="text-align:right;">
0.5853556
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared30
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
8.2519112
</td>
<td style="text-align:right;">
0.0040709
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared31
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
143.5573743
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared32
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared33
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
38.6712858
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared34
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared35
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.7027035
</td>
<td style="text-align:right;">
0.4018768
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared36
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
83.9575031
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared37
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.7807185
</td>
<td style="text-align:right;">
0.3769215
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared38
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
2.0240211
</td>
<td style="text-align:right;">
0.1548287
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared39
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
14.5213360
</td>
<td style="text-align:right;">
0.0001386
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared40
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
35.9658900
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared41
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
13.1904496
</td>
<td style="text-align:right;">
0.0002814
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared42
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.3200447
</td>
<td style="text-align:right;">
0.5715808
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared43
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.8316735
</td>
<td style="text-align:right;">
0.3617891
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared44
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
21.9266729
</td>
<td style="text-align:right;">
0.0000028
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared45
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
61.2835101
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared46
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
209.1918737
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared47
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
24.1260844
</td>
<td style="text-align:right;">
0.0000009
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared48
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
32.8094943
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared49
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
44.3870882
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared50
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
30.0180203
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared51
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
23.8228586
</td>
<td style="text-align:right;">
0.0000011
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared52
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
48.8516928
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared53
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
410.8825623
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared54
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
282.4665788
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared55
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
461.5045614
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared56
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
7.2369254
</td>
<td style="text-align:right;">
0.0071419
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared57
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
170.1873529
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared58
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
546.6339562
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
X-squared59
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
2211.1057673
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="odds-ratio-results" class="section level3">
<h3>odds ratio results</h3>
<pre class="r"><code>results_or &lt;- data.frame(Matrix_Name = character(),
                      Row_Compared = character(),
                      Odds_Ratio = numeric(),
                      Lower_CI = numeric(),
                      Upper_CI = numeric(),
                      P_Value = numeric(),
                      stringsAsFactors = FALSE)

# Loop through each matrix in the list
for (matrix_name in names(matrix_list)) {
  current_matrix &lt;- matrix_list[[matrix_name]]
  n_rows &lt;- nrow(current_matrix)
  
  # Loop through each row of the current matrix (except the last row)
  for (i in 1:(n_rows - 1)) {
    # Perform odds ratio test between row i and the last row using epitools
    test_result &lt;- tryCatch({
      contingency_table &lt;- rbind(current_matrix[i, ], current_matrix[n_rows, ])
      
      # Check if any row in the contingency table contains only zeros
      if (any(rowSums(contingency_table) == 0)) {
        stop(&quot;Contingency table contains empty rows.&quot;)
      }
      
      oddsratio_result &lt;- oddsratio(contingency_table)
       # Ensure the oddsratio result has at least 2 rows
      if (nrow(oddsratio_result$measure) &lt; 2) {
        stop(&quot;oddsratio result does not have enough data.&quot;)
      }
      
     list(oddsratio = oddsratio_result, p.value = oddsratio_result$p.value[2,&quot;chi.square&quot;])
      
    }, error = function(e) {
      cat(&quot;Error in odds ratio test for row&quot;, i, &quot;in matrix&quot;, matrix_name, &quot;:&quot;, e$message, &quot;\n&quot;)
      return(NULL)
    })
    
    # Only store the result if test_result is valid (i.e., not NULL)
    if (!is.null(test_result)) {
      or_value &lt;- test_result$oddsratio$measure[2, &quot;estimate&quot;]
      lower_ci &lt;- test_result$oddsratio$measure[2, &quot;lower&quot;]
      upper_ci &lt;- test_result$oddsratio$measure[2, &quot;upper&quot;]
      p_value &lt;- test_result$oddsratio$p.value[2,&quot;chi.square&quot;]
      
      # Check if the values are numeric and valid (not NA)
      if (!is.na(or_value) &amp;&amp; !is.na(lower_ci) &amp;&amp; !is.na(upper_ci) &amp;&amp; !is.na(p_value)) {
        # Store the results in the dataframe
        results_or &lt;- rbind(results_or, data.frame(Matrix_Name = matrix_name,
                                             Row_Compared = rownames(current_matrix)[i],
                                             Odds_Ratio = or_value,
                                             Lower_CI = lower_ci,
                                             Upper_CI = upper_ci,
                                             P_Value = p_value))
      }
    }
  }
}

# saveRDS(results_or,&quot;data/Final_four_data/OR_results_TE_df.RDS&quot;)
# results_or &lt;- readRDS(&quot;data/Final_four_data/OR_results_TE_df.RDS&quot;) # Print the resulting dataframe

results_or &lt;- results_or %&gt;% 
  mutate(Matrix_Name=factor(Matrix_Name, levels=c(&quot;TE&quot;,&quot;Sines&quot;,&quot;Lines&quot;, &quot;DNA&quot;,&quot;LTR&quot;,&quot;Retro&quot;,&quot;CGI&quot;,&quot;TSS&quot;))) %&gt;% 
    mutate(Row_Compared=factor(Row_Compared, levels = c(&quot;EAR_open&quot;,&quot;ESR_open&quot;, &quot;LR_open&quot;,&quot;ESR_opcl&quot;, &quot;EAR_close&quot;, &quot;ESR_close&quot;, &quot;LR_close&quot;, &quot;ESR_clop&quot;))) %&gt;% 
  arrange(Matrix_Name,Row_Compared)  
print(results_or) %&gt;% 
  kable(., caption = &quot;Odd ratio results and significance values of TE enrichment compared to No response group&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(width = &quot;100%&quot;, height = &quot;400px&quot;)</code></pre>
<pre><code>           Matrix_Name Row_Compared Odds_Ratio  Lower_CI  Upper_CI
estimate            TE     EAR_open  0.9930149 0.9276918 1.0637550
estimate2           TE     ESR_open  1.0198043 0.9657073 1.0774383
estimate6           TE      LR_open  1.1895330 1.1576918 1.2223494
estimate4           TE     ESR_opcl  1.0091461 0.6849717 1.5314210
estimate1           TE    EAR_close  0.8724990 0.8217932 0.9267896
estimate3           TE    ESR_close  0.8036436 0.7711656 0.8376684
estimate7           TE     LR_close  0.9009153 0.8700517 0.9330668
estimate5           TE     ESR_clop  1.0870568 0.9625132 1.2311723
estimate8        Sines     EAR_open  0.9003151 0.8384455 0.9659169
estimate10       Sines     ESR_open  0.9910389 0.9380274 1.0465405
estimate14       Sines      LR_open  1.1915424 1.1616378 1.2221370
estimate12       Sines     ESR_opcl  0.9604240 0.6293681 1.4206259
estimate9        Sines    EAR_close  1.2140218 1.1436499 1.2880365
estimate11       Sines    ESR_close  1.0624427 1.0176585 1.1088832
estimate15       Sines     LR_close  0.9767914 0.9420791 1.0125838
estimate13       Sines     ESR_clop  1.4661622 1.3121527 1.6356040
estimate17       Lines     EAR_open  0.8864872 0.8180134 0.9593484
estimate21       Lines     ESR_open  1.0713070 1.0089789 1.1366501
estimate23       Lines      LR_open  1.2320451 1.1981776 1.2668316
estimate20       Lines     ESR_opcl  1.6132125 1.0788300 2.3514816
estimate16       Lines    EAR_close  1.1207354 1.0475169 1.1980391
estimate19       Lines    ESR_close  1.0426688 0.9936261 1.0936497
estimate22       Lines     LR_close  1.1620391 1.1181608 1.2074006
estimate18       Lines     ESR_clop  1.1470621 1.0068554 1.3024806
estimate25         DNA     EAR_open  0.8642179 0.7660552 0.9712230
estimate29         DNA     ESR_open  0.9746323 0.8902416 1.0648643
estimate31         DNA      LR_open  1.2732597 1.2237553 1.3244921
estimate28         DNA     ESR_opcl  0.8169463 0.3652356 1.5628711
estimate24         DNA    EAR_close  0.9631076 0.8677809 1.0659409
estimate27         DNA    ESR_close  0.9339086 0.8679018 1.0036802
estimate30         DNA     LR_close  1.0868636 1.0268235 1.1496416
estimate26         DNA     ESR_clop  1.1872063 0.9848197 1.4183896
estimate40         LTR     EAR_open  0.7348741 0.6634605 0.8117356
estimate44         LTR     ESR_open  0.8335422 0.7719410 0.8988335
estimate46         LTR      LR_open  1.2665403 1.2265514 1.3077553
estimate43         LTR     ESR_opcl  1.2955805 0.7818817 2.0337293
estimate39         LTR    EAR_close  1.1623912 1.0755693 1.2545921
estimate42         LTR    ESR_close  1.0169326 0.9608134 1.0756303
estimate45         LTR     LR_close  1.1930868 1.1413249 1.2467341
estimate41         LTR     ESR_clop  1.3057464 1.1295958 1.5022764
estimate33       Retro     EAR_open  3.9321127 2.4405460 6.0252674
estimate36       Retro     ESR_open  4.5085891 3.1471538 6.3127730
estimate38       Retro      LR_open  0.7633061 0.5267756 1.0783561
estimate32       Retro    EAR_close  1.0340804 0.4355106 2.0463892
estimate35       Retro    ESR_close  0.7481210 0.3804652 1.3164846
estimate37       Retro     LR_close  0.7863199 0.4649896 1.2476868
estimate34       Retro     ESR_clop  1.2027275 0.1855626 3.7511651
estimate47         CGI     EAR_open  1.7734414 1.4018570 2.2113095
estimate49         CGI     ESR_open  1.8228090 1.5201218 2.1684240
estimate51         CGI      LR_open  0.7315696 0.6442733 0.8280861
estimate48         CGI    EAR_close  0.2697156 0.1602209 0.4210667
estimate50         CGI    ESR_close  0.5156835 0.4020549 0.6503579
estimate52         CGI     LR_close  0.4917024 0.3991197 0.5988032
estimate54         TSS     EAR_open  1.9364655 1.7900763 2.0932235
estimate57         TSS     ESR_open  0.5594316 0.5116390 0.6105511
estimate59         TSS      LR_open  0.3154702 0.2998791 0.3316466
estimate56         TSS     ESR_opcl  0.3551397 0.1482999 0.7122690
estimate53         TSS    EAR_close  0.2884772 0.2532677 0.3270568
estimate55         TSS    ESR_close  0.4599742 0.4275569 0.4942230
estimate58         TSS     LR_close  0.5076196 0.4789560 0.5376113
                 P_Value
estimate    8.431233e-01
estimate2   4.812431e-01
estimate6   5.213551e-36
estimate4   9.497618e-01
estimate1   8.635414e-06
estimate3   3.020870e-25
estimate7   4.883218e-09
estimate5   1.815400e-01
estimate8   3.575062e-03
estimate10  7.454107e-01
estimate14  8.468238e-42
estimate12  8.299120e-01
estimate9   1.568282e-10
estimate11  5.717164e-03
estimate15  2.016309e-01
estimate13  7.603440e-12
estimate17  2.976821e-03
estimate21  2.366672e-02
estimate23  6.775038e-49
estimate20  1.530432e-02
estimate16  8.767903e-04
estimate19  8.813468e-02
estimate22  1.679414e-14
estimate18  3.701771e-02
estimate25  1.543072e-02
estimate29  5.698995e-01
estimate31  3.908336e-33
estimate28  5.430867e-01
estimate24  4.693046e-01
estimate27  6.439674e-02
estimate30  3.884398e-03
estimate26  6.642753e-02
estimate40  1.743304e-09
estimate44  2.591425e-06
estimate46  1.821622e-47
estimate43  2.961428e-01
estimate39  1.274425e-04
estimate42  5.617843e-01
estimate45  4.498738e-15
estimate41  2.407915e-04
estimate33  1.340329e-10
estimate36  1.183286e-20
estimate38  1.317748e-01
estimate32  9.761622e-01
estimate35  3.291164e-01
estimate37  3.186374e-01
estimate34  8.764030e-01
estimate47  6.156094e-07
estimate49  1.815962e-11
estimate51  9.095905e-07
estimate48  6.867152e-09
estimate50  3.289884e-08
estimate52  2.085836e-12
estimate54  1.467716e-63
estimate57  5.271225e-39
estimate59  0.000000e+00
estimate56  4.784639e-03
estimate53  1.548593e-91
estimate55 1.656628e-102
estimate58 5.112792e-121</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:400px; overflow-x: scroll; width:100%; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Odd ratio results and significance values of TE enrichment compared to
No response group
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Matrix_Name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Row_Compared
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Odds_Ratio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Lower_CI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Upper_CI
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P_Value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
estimate
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.9930149
</td>
<td style="text-align:right;">
0.9276918
</td>
<td style="text-align:right;">
1.0637550
</td>
<td style="text-align:right;">
0.8431233
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate2
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
1.0198043
</td>
<td style="text-align:right;">
0.9657073
</td>
<td style="text-align:right;">
1.0774383
</td>
<td style="text-align:right;">
0.4812431
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate6
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
1.1895330
</td>
<td style="text-align:right;">
1.1576918
</td>
<td style="text-align:right;">
1.2223494
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate4
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
1.0091461
</td>
<td style="text-align:right;">
0.6849717
</td>
<td style="text-align:right;">
1.5314210
</td>
<td style="text-align:right;">
0.9497618
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate1
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.8724990
</td>
<td style="text-align:right;">
0.8217932
</td>
<td style="text-align:right;">
0.9267896
</td>
<td style="text-align:right;">
0.0000086
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate3
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.8036436
</td>
<td style="text-align:right;">
0.7711656
</td>
<td style="text-align:right;">
0.8376684
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate7
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.9009153
</td>
<td style="text-align:right;">
0.8700517
</td>
<td style="text-align:right;">
0.9330668
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate5
</td>
<td style="text-align:left;">
TE
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.0870568
</td>
<td style="text-align:right;">
0.9625132
</td>
<td style="text-align:right;">
1.2311723
</td>
<td style="text-align:right;">
0.1815400
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate8
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.9003151
</td>
<td style="text-align:right;">
0.8384455
</td>
<td style="text-align:right;">
0.9659169
</td>
<td style="text-align:right;">
0.0035751
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate10
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.9910389
</td>
<td style="text-align:right;">
0.9380274
</td>
<td style="text-align:right;">
1.0465405
</td>
<td style="text-align:right;">
0.7454107
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate14
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
1.1915424
</td>
<td style="text-align:right;">
1.1616378
</td>
<td style="text-align:right;">
1.2221370
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate12
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.9604240
</td>
<td style="text-align:right;">
0.6293681
</td>
<td style="text-align:right;">
1.4206259
</td>
<td style="text-align:right;">
0.8299120
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate9
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1.2140218
</td>
<td style="text-align:right;">
1.1436499
</td>
<td style="text-align:right;">
1.2880365
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate11
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
1.0624427
</td>
<td style="text-align:right;">
1.0176585
</td>
<td style="text-align:right;">
1.1088832
</td>
<td style="text-align:right;">
0.0057172
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate15
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.9767914
</td>
<td style="text-align:right;">
0.9420791
</td>
<td style="text-align:right;">
1.0125838
</td>
<td style="text-align:right;">
0.2016309
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate13
</td>
<td style="text-align:left;">
Sines
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.4661622
</td>
<td style="text-align:right;">
1.3121527
</td>
<td style="text-align:right;">
1.6356040
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate17
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.8864872
</td>
<td style="text-align:right;">
0.8180134
</td>
<td style="text-align:right;">
0.9593484
</td>
<td style="text-align:right;">
0.0029768
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate21
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
1.0713070
</td>
<td style="text-align:right;">
1.0089789
</td>
<td style="text-align:right;">
1.1366501
</td>
<td style="text-align:right;">
0.0236667
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate23
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
1.2320451
</td>
<td style="text-align:right;">
1.1981776
</td>
<td style="text-align:right;">
1.2668316
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate20
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
1.6132125
</td>
<td style="text-align:right;">
1.0788300
</td>
<td style="text-align:right;">
2.3514816
</td>
<td style="text-align:right;">
0.0153043
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate16
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1.1207354
</td>
<td style="text-align:right;">
1.0475169
</td>
<td style="text-align:right;">
1.1980391
</td>
<td style="text-align:right;">
0.0008768
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate19
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
1.0426688
</td>
<td style="text-align:right;">
0.9936261
</td>
<td style="text-align:right;">
1.0936497
</td>
<td style="text-align:right;">
0.0881347
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate22
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1.1620391
</td>
<td style="text-align:right;">
1.1181608
</td>
<td style="text-align:right;">
1.2074006
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate18
</td>
<td style="text-align:left;">
Lines
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.1470621
</td>
<td style="text-align:right;">
1.0068554
</td>
<td style="text-align:right;">
1.3024806
</td>
<td style="text-align:right;">
0.0370177
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate25
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.8642179
</td>
<td style="text-align:right;">
0.7660552
</td>
<td style="text-align:right;">
0.9712230
</td>
<td style="text-align:right;">
0.0154307
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate29
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.9746323
</td>
<td style="text-align:right;">
0.8902416
</td>
<td style="text-align:right;">
1.0648643
</td>
<td style="text-align:right;">
0.5698995
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate31
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
1.2732597
</td>
<td style="text-align:right;">
1.2237553
</td>
<td style="text-align:right;">
1.3244921
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate28
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.8169463
</td>
<td style="text-align:right;">
0.3652356
</td>
<td style="text-align:right;">
1.5628711
</td>
<td style="text-align:right;">
0.5430867
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate24
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.9631076
</td>
<td style="text-align:right;">
0.8677809
</td>
<td style="text-align:right;">
1.0659409
</td>
<td style="text-align:right;">
0.4693046
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate27
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.9339086
</td>
<td style="text-align:right;">
0.8679018
</td>
<td style="text-align:right;">
1.0036802
</td>
<td style="text-align:right;">
0.0643967
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate30
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1.0868636
</td>
<td style="text-align:right;">
1.0268235
</td>
<td style="text-align:right;">
1.1496416
</td>
<td style="text-align:right;">
0.0038844
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate26
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.1872063
</td>
<td style="text-align:right;">
0.9848197
</td>
<td style="text-align:right;">
1.4183896
</td>
<td style="text-align:right;">
0.0664275
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate40
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
0.7348741
</td>
<td style="text-align:right;">
0.6634605
</td>
<td style="text-align:right;">
0.8117356
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate44
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.8335422
</td>
<td style="text-align:right;">
0.7719410
</td>
<td style="text-align:right;">
0.8988335
</td>
<td style="text-align:right;">
0.0000026
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate46
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
1.2665403
</td>
<td style="text-align:right;">
1.2265514
</td>
<td style="text-align:right;">
1.3077553
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate43
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
1.2955805
</td>
<td style="text-align:right;">
0.7818817
</td>
<td style="text-align:right;">
2.0337293
</td>
<td style="text-align:right;">
0.2961428
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate39
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1.1623912
</td>
<td style="text-align:right;">
1.0755693
</td>
<td style="text-align:right;">
1.2545921
</td>
<td style="text-align:right;">
0.0001274
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate42
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
1.0169326
</td>
<td style="text-align:right;">
0.9608134
</td>
<td style="text-align:right;">
1.0756303
</td>
<td style="text-align:right;">
0.5617843
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate45
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1.1930868
</td>
<td style="text-align:right;">
1.1413249
</td>
<td style="text-align:right;">
1.2467341
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate41
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.3057464
</td>
<td style="text-align:right;">
1.1295958
</td>
<td style="text-align:right;">
1.5022764
</td>
<td style="text-align:right;">
0.0002408
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate33
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
3.9321127
</td>
<td style="text-align:right;">
2.4405460
</td>
<td style="text-align:right;">
6.0252674
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate36
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
4.5085891
</td>
<td style="text-align:right;">
3.1471538
</td>
<td style="text-align:right;">
6.3127730
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate38
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.7633061
</td>
<td style="text-align:right;">
0.5267756
</td>
<td style="text-align:right;">
1.0783561
</td>
<td style="text-align:right;">
0.1317748
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate32
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1.0340804
</td>
<td style="text-align:right;">
0.4355106
</td>
<td style="text-align:right;">
2.0463892
</td>
<td style="text-align:right;">
0.9761622
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate35
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.7481210
</td>
<td style="text-align:right;">
0.3804652
</td>
<td style="text-align:right;">
1.3164846
</td>
<td style="text-align:right;">
0.3291164
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate37
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.7863199
</td>
<td style="text-align:right;">
0.4649896
</td>
<td style="text-align:right;">
1.2476868
</td>
<td style="text-align:right;">
0.3186374
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate34
</td>
<td style="text-align:left;">
Retro
</td>
<td style="text-align:left;">
ESR_clop
</td>
<td style="text-align:right;">
1.2027275
</td>
<td style="text-align:right;">
0.1855626
</td>
<td style="text-align:right;">
3.7511651
</td>
<td style="text-align:right;">
0.8764030
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate47
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1.7734414
</td>
<td style="text-align:right;">
1.4018570
</td>
<td style="text-align:right;">
2.2113095
</td>
<td style="text-align:right;">
0.0000006
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate49
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
1.8228090
</td>
<td style="text-align:right;">
1.5201218
</td>
<td style="text-align:right;">
2.1684240
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate51
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.7315696
</td>
<td style="text-align:right;">
0.6442733
</td>
<td style="text-align:right;">
0.8280861
</td>
<td style="text-align:right;">
0.0000009
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate48
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.2697156
</td>
<td style="text-align:right;">
0.1602209
</td>
<td style="text-align:right;">
0.4210667
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate50
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.5156835
</td>
<td style="text-align:right;">
0.4020549
</td>
<td style="text-align:right;">
0.6503579
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate52
</td>
<td style="text-align:left;">
CGI
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.4917024
</td>
<td style="text-align:right;">
0.3991197
</td>
<td style="text-align:right;">
0.5988032
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate54
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1.9364655
</td>
<td style="text-align:right;">
1.7900763
</td>
<td style="text-align:right;">
2.0932235
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate57
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
0.5594316
</td>
<td style="text-align:right;">
0.5116390
</td>
<td style="text-align:right;">
0.6105511
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate59
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
0.3154702
</td>
<td style="text-align:right;">
0.2998791
</td>
<td style="text-align:right;">
0.3316466
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate56
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
ESR_opcl
</td>
<td style="text-align:right;">
0.3551397
</td>
<td style="text-align:right;">
0.1482999
</td>
<td style="text-align:right;">
0.7122690
</td>
<td style="text-align:right;">
0.0047846
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate53
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
0.2884772
</td>
<td style="text-align:right;">
0.2532677
</td>
<td style="text-align:right;">
0.3270568
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate55
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
0.4599742
</td>
<td style="text-align:right;">
0.4275569
</td>
<td style="text-align:right;">
0.4942230
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
estimate58
</td>
<td style="text-align:left;">
TSS
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
0.5076196
</td>
<td style="text-align:right;">
0.4789560
</td>
<td style="text-align:right;">
0.5376113
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code># breaks &lt;- seq(0, 5, length.out = 12)
# # Create a color palette with 12 colors from &quot;purple3&quot; to &quot;green3&quot;, passing through &quot;white&quot;
# colors &lt;- colorRampPalette(c(&quot;purple3&quot;, &quot;white&quot;, &quot;green3&quot;))(12)
# 
# # Create the color function with 12 bins
# col_fun_OR &lt;- colorRamp2(breaks, colors)

col_fun_OR = colorRamp2(c(0,1,1.5,3,4), c(&quot;#BC9BFF&quot;,&quot;white&quot;,&quot;lightgreen&quot;,&quot;green3&quot;,&quot;green3&quot; ))
sig_mat_OR &lt;- results_or %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select( Matrix_Name,Row_Compared,P_Value) %&gt;% 
  pivot_wider(., id_cols = Matrix_Name, names_from = Row_Compared, values_from = P_Value) %&gt;% 
  column_to_rownames(&quot;Matrix_Name&quot;) %&gt;% 
  as.matrix() 


results_or %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select( Matrix_Name,Row_Compared,Odds_Ratio) %&gt;% 
  pivot_wider(., id_cols = Matrix_Name, names_from = Row_Compared, values_from = Odds_Ratio) %&gt;% 
  column_to_rownames(&quot;Matrix_Name&quot;) %&gt;% 
  as.matrix() %&gt;% 
  ComplexHeatmap::Heatmap(. ,col = col_fun_OR, 
                          cluster_rows=FALSE, 
                          cluster_columns=FALSE, 
                          column_names_side = &quot;top&quot;, 
                          column_names_rot = 45,
                          # na_col = &quot;black&quot;,
                          cell_fun = function(j, i, x, y, width, height, fill) {if (!is.na(sig_mat_OR[i, j]) &amp;&amp; sig_mat_OR[i, j] &lt; 0.05)
            grid.text(&quot;*&quot;, x, y, gp = gpar(fontsize = 20))})</code></pre>
<p><img src="figure/Odds_ratios_ff.Rmd/odds%20ratio%20test-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-odds-ratio-test-1">
Past versions of "odds ratio test-1.png"
</button>
</p>
<div id="fig-no-spaces-odds-ratio-test-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/632278d84d50e437f66d68cd91b1adc0a4e2354b/docs/figure/Odds_ratios_ff.Rmd/odds ratio test-1.png" target="_blank">632278d</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The final image for Figure2A was created using
<code>code/MRC_clusterlog2cpm.R</code> script. This added the BH
correction for multiple testing across all tests. (the Odds ratio for
TEs above and for cREs)</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggridges_0.5.6                          
 [2] circlize_0.4.16                         
 [3] epitools_0.5-10.1                       
 [4] ggrepel_0.9.6                           
 [5] plyranges_1.26.0                        
 [6] ggsignif_0.6.4                          
 [7] genomation_1.38.0                       
 [8] smplot2_0.2.5                           
 [9] eulerr_7.0.2                            
[10] biomaRt_2.62.1                          
[11] devtools_2.4.5                          
[12] usethis_3.1.0                           
[13] ggpubr_0.6.0                            
[14] BiocParallel_1.40.0                     
[15] scales_1.3.0                            
[16] VennDiagram_1.7.3                       
[17] futile.logger_1.4.3                     
[18] gridExtra_2.3                           
[19] ggfortify_0.4.17                        
[20] edgeR_4.4.2                             
[21] limma_3.62.2                            
[22] rtracklayer_1.66.0                      
[23] org.Hs.eg.db_3.20.0                     
[24] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[25] GenomicFeatures_1.58.0                  
[26] AnnotationDbi_1.68.0                    
[27] Biobase_2.66.0                          
[28] GenomicRanges_1.58.0                    
[29] GenomeInfoDb_1.42.3                     
[30] IRanges_2.40.1                          
[31] S4Vectors_0.44.0                        
[32] BiocGenerics_0.52.0                     
[33] ChIPseeker_1.42.1                       
[34] RColorBrewer_1.1-3                      
[35] broom_1.0.7                             
[36] kableExtra_1.4.0                        
[37] lubridate_1.9.4                         
[38] forcats_1.0.0                           
[39] stringr_1.5.1                           
[40] dplyr_1.1.4                             
[41] purrr_1.0.4                             
[42] readr_2.1.5                             
[43] tidyr_1.3.1                             
[44] tibble_3.2.1                            
[45] ggplot2_3.5.1                           
[46] tidyverse_2.0.0                         
[47] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.5                               
  [2] matrixStats_1.5.0                      
  [3] bitops_1.0-9                           
  [4] enrichplot_1.26.6                      
  [5] doParallel_1.0.17                      
  [6] httr_1.4.7                             
  [7] profvis_0.4.0                          
  [8] tools_4.4.2                            
  [9] backports_1.5.0                        
 [10] utf8_1.2.4                             
 [11] R6_2.6.1                               
 [12] lazyeval_0.2.2                         
 [13] GetoptLong_1.0.5                       
 [14] urlchecker_1.0.1                       
 [15] withr_3.0.2                            
 [16] prettyunits_1.2.0                      
 [17] cli_3.6.4                              
 [18] formatR_1.14                           
 [19] labeling_0.4.3                         
 [20] sass_0.4.9                             
 [21] Rsamtools_2.22.0                       
 [22] systemfonts_1.2.1                      
 [23] yulab.utils_0.2.0                      
 [24] foreign_0.8-88                         
 [25] DOSE_4.0.0                             
 [26] svglite_2.1.3                          
 [27] R.utils_2.13.0                         
 [28] sessioninfo_1.2.3                      
 [29] plotrix_3.8-4                          
 [30] BSgenome_1.74.0                        
 [31] pwr_1.3-0                              
 [32] impute_1.80.0                          
 [33] rstudioapi_0.17.1                      
 [34] RSQLite_2.3.9                          
 [35] shape_1.4.6.1                          
 [36] generics_0.1.3                         
 [37] gridGraphics_0.5-1                     
 [38] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [39] BiocIO_1.16.0                          
 [40] vroom_1.6.5                            
 [41] gtools_3.9.5                           
 [42] car_3.1-3                              
 [43] GO.db_3.20.0                           
 [44] Matrix_1.7-3                           
 [45] abind_1.4-8                            
 [46] R.methodsS3_1.8.2                      
 [47] lifecycle_1.0.4                        
 [48] whisker_0.4.1                          
 [49] yaml_2.3.10                            
 [50] carData_3.0-5                          
 [51] SummarizedExperiment_1.36.0            
 [52] gplots_3.2.0                           
 [53] qvalue_2.38.0                          
 [54] SparseArray_1.6.2                      
 [55] BiocFileCache_2.14.0                   
 [56] blob_1.2.4                             
 [57] promises_1.3.2                         
 [58] crayon_1.5.3                           
 [59] miniUI_0.1.1.1                         
 [60] ggtangle_0.0.6                         
 [61] lattice_0.22-6                         
 [62] cowplot_1.1.3                          
 [63] KEGGREST_1.46.0                        
 [64] magick_2.8.5                           
 [65] ComplexHeatmap_2.22.0                  
 [66] pillar_1.10.1                          
 [67] knitr_1.49                             
 [68] fgsea_1.32.2                           
 [69] rjson_0.2.23                           
 [70] boot_1.3-31                            
 [71] codetools_0.2-20                       
 [72] fastmatch_1.1-6                        
 [73] glue_1.8.0                             
 [74] getPass_0.2-4                          
 [75] ggfun_0.1.8                            
 [76] data.table_1.17.0                      
 [77] remotes_2.5.0                          
 [78] vctrs_0.6.5                            
 [79] png_0.1-8                              
 [80] treeio_1.30.0                          
 [81] gtable_0.3.6                           
 [82] cachem_1.1.0                           
 [83] xfun_0.51                              
 [84] S4Arrays_1.6.0                         
 [85] mime_0.12                              
 [86] iterators_1.0.14                       
 [87] statmod_1.5.0                          
 [88] ellipsis_0.3.2                         
 [89] nlme_3.1-167                           
 [90] ggtree_3.14.0                          
 [91] bit64_4.6.0-1                          
 [92] filelock_1.0.3                         
 [93] progress_1.2.3                         
 [94] rprojroot_2.0.4                        
 [95] bslib_0.9.0                            
 [96] rpart_4.1.24                           
 [97] KernSmooth_2.23-26                     
 [98] Hmisc_5.2-2                            
 [99] colorspace_2.1-1                       
[100] DBI_1.2.3                              
[101] seqPattern_1.38.0                      
[102] nnet_7.3-20                            
[103] tidyselect_1.2.1                       
[104] processx_3.8.6                         
[105] bit_4.6.0                              
[106] compiler_4.4.2                         
[107] curl_6.2.1                             
[108] git2r_0.35.0                           
[109] httr2_1.1.1                            
[110] htmlTable_2.4.3                        
[111] xml2_1.3.7                             
[112] DelayedArray_0.32.0                    
[113] checkmate_2.3.2                        
[114] caTools_1.18.3                         
[115] callr_3.7.6                            
[116] rappdirs_0.3.3                         
[117] digest_0.6.37                          
[118] rmarkdown_2.29                         
[119] XVector_0.46.0                         
[120] base64enc_0.1-3                        
[121] htmltools_0.5.8.1                      
[122] pkgconfig_2.0.3                        
[123] MatrixGenerics_1.18.1                  
[124] dbplyr_2.5.0                           
[125] fastmap_1.2.0                          
[126] GlobalOptions_0.1.2                    
[127] rlang_1.1.5                            
[128] htmlwidgets_1.6.4                      
[129] UCSC.utils_1.2.0                       
[130] shiny_1.10.0                           
[131] farver_2.1.2                           
[132] jquerylib_0.1.4                        
[133] zoo_1.8-13                             
[134] jsonlite_1.9.1                         
[135] GOSemSim_2.32.0                        
[136] R.oo_1.27.0                            
[137] RCurl_1.98-1.16                        
[138] magrittr_2.0.3                         
[139] Formula_1.2-5                          
[140] GenomeInfoDbData_1.2.13                
[141] ggplotify_0.1.2                        
[142] patchwork_1.3.0                        
[143] munsell_0.5.1                          
[144] Rcpp_1.0.14                            
[145] ape_5.8-1                              
[146] stringi_1.8.4                          
[147] zlibbioc_1.52.0                        
[148] plyr_1.8.9                             
[149] pkgbuild_1.4.6                         
[150] parallel_4.4.2                         
[151] Biostrings_2.74.1                      
[152] splines_4.4.2                          
[153] hms_1.1.3                              
[154] locfit_1.5-9.12                        
[155] ps_1.9.0                               
[156] igraph_2.1.4                           
[157] reshape2_1.4.4                         
[158] pkgload_1.4.0                          
[159] futile.options_1.0.1                   
[160] XML_3.99-0.18                          
[161] evaluate_1.0.3                         
[162] lambda.r_1.2.4                         
[163] foreach_1.5.2                          
[164] tzdb_0.4.0                             
[165] httpuv_1.6.15                          
[166] clue_0.3-66                            
[167] gridBase_0.4-7                         
[168] xtable_1.8-4                           
[169] restfulr_0.0.15                        
[170] tidytree_0.4.6                         
[171] rstatix_0.7.2                          
[172] later_1.4.1                            
[173] viridisLite_0.4.2                      
[174] aplot_0.2.5                            
[175] memoise_2.0.1                          
[176] GenomicAlignments_1.42.0               
[177] cluster_2.1.8.1                        
[178] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
