<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-03-29" />

<title>Peak_analysis</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Peak_analysis</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-03-29</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-03-29
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree64ec010850628a78e0ae7b270c00e9322d9987b7targetblank64ec010a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/64ec010850628a78e0ae7b270c00e9322d9987b7" target="_blank">64ec010</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree64ec010850628a78e0ae7b270c00e9322d9987b7targetblank64ec010a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/64ec010850628a78e0ae7b270c00e9322d9987b7" target="_blank">64ec010</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/TSSE.R
    Untracked:  code/corMotifcustom.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Peak_analysis.Rmd</code>) and
HTML (<code>docs/Peak_analysis.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/64ec010850628a78e0ae7b270c00e9322d9987b7/analysis/Peak_analysis.Rmd" target="_blank">64ec010</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-29
</td>
<td>
updates to code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/Peak_analysis.html" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1b50eea1261e8a8f29102e8da10c4baecff5ac34/analysis/Peak_analysis.Rmd" target="_blank">1b50eea</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
<td>
adding in more DAR analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/Peak_analysis.html" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1a83bd63a756bd951fd77edb0c8c141f41e1ea49/analysis/Peak_analysis.Rmd" target="_blank">1a83bd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
adding PCA
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/Peak_analysis.html" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3a4202a028caca9505a95fc99c24aed052e4b3c6/analysis/Peak_analysis.Rmd" target="_blank">3a4202a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
adding in DAR w/o 4 5
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cf2d4069cebb83f2cb80285db29220f628e246bb/analysis/Peak_analysis.Rmd" target="_blank">cf2d406</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
<td>
adding cormotif
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/Peak_analysis.html" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/930ee3cecd3d46cf7444c89f84f72ba0bb183c71/analysis/Peak_analysis.Rmd" target="_blank">930ee3c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
<td>
updates to filtering of peaks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/78795a75745e18937e4dc541e9c1f9be5eecb5b1/docs/Peak_analysis.html" target="_blank">78795a7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0e9506a5ac281b578bd829fc15a408b7d08a0772/analysis/Peak_analysis.Rmd" target="_blank">0e9506a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-14
</td>
<td>
adding DAR initial
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/aa06c1a7a4a11f7b090d06ceb4510569e8406cfe/docs/Peak_analysis.html" target="_blank">aa06c1a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/7906ccf13a1f0fd51703332a14e88d37448542df/analysis/Peak_analysis.Rmd" target="_blank">7906ccf</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-08
</td>
<td>
adding new page
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(ggsignif)
library(cowplot)
library(ggpubr)
library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
# library(ChIPseeker)
# library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
# library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
# library(rtracklayer)
library(gridExtra)
library(edgeR)
library(ggfortify)
library(limma)</code></pre>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
pca_plot &lt;-
  function(df,
           col_var = NULL,
           shape_var = NULL,
           title = &quot;&quot;) {
    ggplot(df) + geom_point(aes_string(
      x = &quot;PC1&quot;,
      y = &quot;PC2&quot;,
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = &quot;PC 1&quot;, y = &quot;PC 2&quot;) +
      scale_color_manual(values = c(
        &quot;#8B006D&quot;,
        &quot;#DF707E&quot;,
        &quot;#F1B72B&quot;,
        &quot;#3386DD&quot;,
        &quot;#707031&quot;,
        &quot;#41B333&quot;
      ))
  }
pca_var_plot &lt;- function(pca) {
  # x: class == prcomp
  pca.var &lt;- pca$sdev ^ 2
  pca.prop &lt;- pca.var / sum(pca.var)
  var.plot &lt;-
    qplot(PC, prop, data = data.frame(PC = 1:length(pca.prop),
                                      prop = pca.prop)) +
    labs(title = &#39;Variance contributed by each PC&#39;,
         x = &#39;PC&#39;, y = &#39;Proportion of variance&#39;)
}

calc_pca &lt;- function(x) {
  # Performs principal components analysis with prcomp
  # x: a sample-by-gene numeric matrix
  prcomp(x, scale. = TRUE, retx = TRUE)
}

get_regr_pval &lt;- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == &quot;lm&quot;)
  fstat &lt;- summary(mod)$fstatistic
  pval &lt;- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc &lt;- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char &lt;- paste0(&quot;PC&quot;, pc_num)
  # Calculate F-statistic p-value for linear model
  pval &lt;- get_regr_pval(lm(df[, pc_char] ~ df[, fac]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = &quot;lm&quot;) + labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  }
}
x_axis_labels = function(labels, every_nth = 1, ...) {
  axis(side = 1,
       at = seq_along(labels),
       labels = F)
  text(
    x = (seq_along(labels))[seq_len(every_nth) == 1],
    y = par(&quot;usr&quot;)[3] - 0.075 * (par(&quot;usr&quot;)[4] - par(&quot;usr&quot;)[3]),
    labels = labels[seq_len(every_nth) == 1],
    xpd = TRUE,
    ...
  )
}</code></pre>
<div id="initial-peak-calling-heatmap" class="section level3">
<h3>Initial peak calling heatmap</h3>
<pre class="r"><code>first_run_frag_counts &lt;- read.csv(&quot;data/first_run_frag_counts.txt&quot;, row.names = 1) 
Frag_cor &lt;- first_run_frag_counts %&gt;% 
  dplyr::select(Ind1_75DA24h:Ind6_71V3h)  %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
  
filmat_groupmat_col &lt;- data.frame(timeset = colnames(Frag_cor))

counts_corr_mat &lt;-filmat_groupmat_col %&gt;% 
  mutate(timeset=gsub(&quot;75&quot;,&quot;1_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;87&quot;,&quot;2_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;77&quot;,&quot;3_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;79&quot;,&quot;4_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;78&quot;,&quot;5_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;71&quot;,&quot;6_&quot;,timeset)) %&gt;% 
  mutate(timeset = gsub(&quot;24h&quot;,&quot;_24h&quot;,timeset), 
       timeset = gsub(&quot;3h&quot;,&quot;_3h&quot;,timeset)) %&gt;%
  separate(timeset, into = c(NA,&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
  ))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))
  ))) 
                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;),
   indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;, &quot;#E7298A&quot; ,&quot;#66A61E&quot;, &quot;#E6AB02&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)


ComplexHeatmap::pheatmap(Frag_cor,
                         # column_title=(paste0(&quot;RNA-seq log&quot;[2]~&quot;cpm correlation&quot;)),
        annotation_col = counts_corr_mat,
        annotation_colors = mat_colors,
        heatmap_legend_param = mat_colors,
        fontsize=10,
        fontsize_row = 8,
        angle_col=&quot;90&quot;,
        treeheight_row=25,
        fontsize_col = 8,
        treeheight_col = 20)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/cpm%20and%20corr%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cpm-and-corr-plots-1">
Past versions of "cpm and corr plots-1.png"
</button>
</p>
<div id="fig-no-spaces-cpm-and-corr-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/aa06c1a7a4a11f7b090d06ceb4510569e8406cfe/docs/figure/Peak_analysis.Rmd/cpm and corr plots-1.png" target="_blank">aa06c1a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This correlation is after log2 of the counts in peaks. The next
correlation will filter out rowMeans &gt;0.</p>
<pre class="r"><code>first_run_frag_counts &lt;- read.csv(&quot;data/first_run_frag_counts.txt&quot;, row.names = 1) 

##loading of the counts matrix
##then separting off the non-counts columns
PCAmat &lt;- first_run_frag_counts %&gt;% 
  dplyr::select(Ind1_75DA24h:Ind6_71V3h) %&gt;% as.matrix()

annotation_mat &lt;- data.frame(timeset=colnames(PCAmat)) %&gt;% 
  mutate(sample = timeset) %&gt;% 
  mutate(timeset=gsub(&quot;Ind1_75&quot;,&quot;1_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;Ind2_87&quot;,&quot;2_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;Ind3_77&quot;,&quot;3_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;Ind4_79&quot;,&quot;4_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;Ind5_78&quot;,&quot;5_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;Ind6_71&quot;,&quot;6_&quot;,timeset)) %&gt;% 
  mutate(timeset = gsub(&quot;24h&quot;,&quot;_24h&quot;,timeset), 
       timeset = gsub(&quot;3h&quot;,&quot;_3h&quot;,timeset)) %&gt;%
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) %&gt;% 
  # mutate(indv = factor(indv, levels = c(&quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;, &quot;6&quot;))) %&gt;%
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) 

PCA_info &lt;- (prcomp(t(PCAmat), scale. = TRUE)) 
PCA_info_anno &lt;- PCA_info$x %&gt;% cbind(.,annotation_mat)
# autoplot(PCA_info)
summary(PCA_info)</code></pre>
<pre><code>Importance of components:
                            PC1       PC2       PC3       PC4       PC5
Standard deviation     376.9018 203.90212 167.84380 130.89150 111.62091
Proportion of Variance   0.3062   0.08961   0.06072   0.03693   0.02685
Cumulative Proportion    0.3062   0.39580   0.45652   0.49345   0.52031
                            PC6     PC7      PC8      PC9     PC10     PC11
Standard deviation     98.43631 92.6486 84.97646 81.63429 79.45811 77.59140
Proportion of Variance  0.02089  0.0185  0.01556  0.01436  0.01361  0.01298
Cumulative Proportion   0.54119  0.5597  0.57526  0.58962  0.60323  0.61621
                           PC12     PC13     PC14     PC15     PC16     PC17
Standard deviation     74.48303 73.59252 71.32544 68.95613 67.61537 66.62440
Proportion of Variance  0.01196  0.01167  0.01097  0.01025  0.00985  0.00957
Cumulative Proportion   0.62816  0.63984  0.65080  0.66105  0.67091  0.68047
                           PC18     PC19     PC20     PC21     PC22     PC23
Standard deviation     65.89180 65.52139 65.01338 64.59942 63.83943 62.55323
Proportion of Variance  0.00936  0.00925  0.00911  0.00899  0.00878  0.00843
Cumulative Proportion   0.68983  0.69908  0.70820  0.71719  0.72597  0.73441
                           PC24     PC25     PC26     PC27     PC28     PC29
Standard deviation     62.11037 61.85108 61.03154 60.79074 60.08203 59.53870
Proportion of Variance  0.00831  0.00825  0.00803  0.00797  0.00778  0.00764
Cumulative Proportion   0.74272  0.75097  0.75900  0.76696  0.77474  0.78238
                           PC30     PC31     PC32     PC33     PC34     PC35
Standard deviation     59.28043 58.34606 57.34933 56.48724 55.83691 55.01438
Proportion of Variance  0.00757  0.00734  0.00709  0.00688  0.00672  0.00652
Cumulative Proportion   0.78996  0.79730  0.80439  0.81126  0.81798  0.82451
                           PC36     PC37     PC38     PC39     PC40     PC41
Standard deviation     53.98905 53.89602 53.55217 53.14725 52.91971 52.68384
Proportion of Variance  0.00628  0.00626  0.00618  0.00609  0.00604  0.00598
Cumulative Proportion   0.83079  0.83705  0.84323  0.84932  0.85536  0.86134
                           PC42     PC43     PC44     PC45    PC46     PC47
Standard deviation     52.40497 52.08406 51.99267 51.63864 51.4089 50.93515
Proportion of Variance  0.00592  0.00585  0.00583  0.00575  0.0057  0.00559
Cumulative Proportion   0.86726  0.87311  0.87893  0.88468  0.8904  0.89597
                           PC48     PC49     PC50     PC51    PC52     PC53
Standard deviation     50.46722 50.23329 49.78971 49.54472 48.6485 48.33228
Proportion of Variance  0.00549  0.00544  0.00534  0.00529  0.0051  0.00504
Cumulative Proportion   0.90146  0.90690  0.91224  0.91753  0.9226  0.92767
                           PC54     PC55     PC56     PC57    PC58     PC59
Standard deviation     48.22246 47.52685 47.33857 46.97043 46.7177 46.25328
Proportion of Variance  0.00501  0.00487  0.00483  0.00476  0.0047  0.00461
Cumulative Proportion   0.93268  0.93755  0.94238  0.94713  0.9518  0.95645
                           PC60     PC61     PC62    PC63     PC64     PC65
Standard deviation     45.96486 45.31000 45.12708 44.1664 43.29335 42.38188
Proportion of Variance  0.00455  0.00443  0.00439  0.0042  0.00404  0.00387
Cumulative Proportion   0.96100  0.96543  0.96982  0.9740  0.97806  0.98193
                           PC66     PC67     PC68     PC69     PC70     PC71
Standard deviation     41.31984 40.12882 38.78887 35.99534 34.36816 32.90725
Proportion of Variance  0.00368  0.00347  0.00324  0.00279  0.00255  0.00233
Cumulative Proportion   0.98561  0.98908  0.99233  0.99512  0.99767  1.00000
                            PC72
Standard deviation     1.156e-12
Proportion of Variance 0.000e+00
Cumulative Proportion  1.000e+00</code></pre>
<pre class="r"><code># cpm(PCAmat, log=TRUE)
pca_plot(PCA_info_anno, col_var=&#39;trt&#39;, shape_var = &#39;time&#39;)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/inital%20analysis-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-inital-analysis-1">
Past versions of "inital analysis-1.png"
</button>
</p>
<div id="fig-no-spaces-inital-analysis-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/aa06c1a7a4a11f7b090d06ceb4510569e8406cfe/docs/figure/Peak_analysis.Rmd/inital analysis-1.png" target="_blank">aa06c1a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_plot(PCA_info_anno, col_var=&#39;trt&#39;, shape_var = &#39;indv&#39;)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/inital%20analysis-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-inital-analysis-2">
Past versions of "inital analysis-2.png"
</button>
</p>
<div id="fig-no-spaces-inital-analysis-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/aa06c1a7a4a11f7b090d06ceb4510569e8406cfe/docs/figure/Peak_analysis.Rmd/inital analysis-2.png" target="_blank">aa06c1a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>This PCA is impacted by the number of reads per sample too.<br />
### Log2 cpm of initial peak counts</p>
<pre class="r"><code>lcpm &lt;- cpm(PCAmat, log=TRUE)  ### for determining the basic cutoffs
dim(lcpm)</code></pre>
<pre><code>[1] 463947     72</code></pre>
<pre class="r"><code>row_means &lt;- rowMeans(lcpm)
x_filtered &lt;- PCAmat[row_means &gt; 0,]
dim(x_filtered)</code></pre>
<pre><code>[1] 170488     72</code></pre>
<pre class="r"><code>filt_matrix_lcpm &lt;- cpm(x_filtered, log=TRUE)
# hist(lcpm,  main = &quot;Histogram of total counts (unfiltered)&quot;, 
#      xlab =expression(&quot;Log&quot;[2]*&quot; counts-per-million&quot;), col =4 )
# 
# hist(filt_matrix_lcpm, main = &quot;Histogram of total counts (filtered)&quot;, 
#      xlab =expression(&quot;Log&quot;[2]*&quot; counts-per-million&quot;), col =4 )
PCA_info_filter &lt;- (prcomp(t(filt_matrix_lcpm), scale. = TRUE))
summary(PCA_info_filter)</code></pre>
<pre><code>Importance of components:
                            PC1      PC2       PC3     PC4      PC5      PC6
Standard deviation     161.9618 155.4965 113.48780 93.0643 76.76785 70.41775
Proportion of Variance   0.1539   0.1418   0.07554  0.0508  0.03457  0.02909
Cumulative Proportion    0.1539   0.2957   0.37123  0.4220  0.45660  0.48568
                            PC7      PC8      PC9     PC10     PC11     PC12
Standard deviation     61.86703 60.77059 58.99124 56.07243 55.68977 55.40875
Proportion of Variance  0.02245  0.02166  0.02041  0.01844  0.01819  0.01801
Cumulative Proportion   0.50813  0.52980  0.55021  0.56865  0.58684  0.60485
                           PC13     PC14     PC15    PC16     PC17     PC18
Standard deviation     53.38239 50.88937 50.16247 48.3365 47.21993 45.90407
Proportion of Variance  0.01671  0.01519  0.01476  0.0137  0.01308  0.01236
Cumulative Proportion   0.62156  0.63675  0.65151  0.6652  0.67830  0.69065
                           PC19     PC20     PC21     PC22     PC23     PC24
Standard deviation     45.81551 44.29971 43.22211 42.77785 42.28063 41.71540
Proportion of Variance  0.01231  0.01151  0.01096  0.01073  0.01049  0.01021
Cumulative Proportion   0.70297  0.71448  0.72544  0.73617  0.74665  0.75686
                           PC25     PC26     PC27     PC28     PC29     PC30
Standard deviation     40.78697 39.34892 38.62606 37.67205 37.43678 36.83664
Proportion of Variance  0.00976  0.00908  0.00875  0.00832  0.00822  0.00796
Cumulative Proportion   0.76662  0.77570  0.78445  0.79278  0.80100  0.80896
                          PC31     PC32     PC33     PC34     PC35     PC36
Standard deviation     36.2335 35.49546 34.96088 34.58537 34.42430 33.42100
Proportion of Variance  0.0077  0.00739  0.00717  0.00702  0.00695  0.00655
Cumulative Proportion   0.8167  0.82405  0.83122  0.83823  0.84518  0.85173
                           PC37     PC38     PC39     PC40     PC41     PC42
Standard deviation     32.71478 32.39574 32.28072 31.62075 31.55443 31.28032
Proportion of Variance  0.00628  0.00616  0.00611  0.00586  0.00584  0.00574
Cumulative Proportion   0.85801  0.86417  0.87028  0.87614  0.88199  0.88772
                           PC43    PC44     PC45     PC46    PC47     PC48
Standard deviation     30.78303 30.0529 29.89864 29.37093 29.1872 28.71515
Proportion of Variance  0.00556  0.0053  0.00524  0.00506  0.0050  0.00484
Cumulative Proportion   0.89328  0.8986  0.90382  0.90888  0.9139  0.91872
                           PC49     PC50     PC51     PC52     PC53     PC54
Standard deviation     28.57247 27.83316 27.63750 27.55010 27.25222 26.85689
Proportion of Variance  0.00479  0.00454  0.00448  0.00445  0.00436  0.00423
Cumulative Proportion   0.92350  0.92805  0.93253  0.93698  0.94134  0.94557
                           PC55     PC56     PC57     PC58    PC59    PC60
Standard deviation     26.32749 25.88742 25.84616 25.37313 25.1207 24.7672
Proportion of Variance  0.00407  0.00393  0.00392  0.00378  0.0037  0.0036
Cumulative Proportion   0.94963  0.95356  0.95748  0.96126  0.9650  0.9686
                           PC61    PC62     PC63     PC64     PC65     PC66
Standard deviation     24.37194 24.0858 23.76387 23.13286 22.32128 21.95522
Proportion of Variance  0.00348  0.0034  0.00331  0.00314  0.00292  0.00283
Cumulative Proportion   0.97204  0.9755  0.97876  0.98190  0.98482  0.98765
                           PC67     PC68     PC69     PC70     PC71      PC72
Standard deviation     21.70357 21.41262 20.61163 19.99292 18.76363 5.069e-13
Proportion of Variance  0.00276  0.00269  0.00249  0.00234  0.00207 0.000e+00
Cumulative Proportion   0.99041  0.99310  0.99559  0.99793  1.00000 1.000e+00</code></pre>
<pre class="r"><code># autoplot(PCA_info_filter)

pca_var_plot(PCA_info_filter)

pca_n45 &lt;- calc_pca(t(filt_matrix_lcpm))
pca_n45_anno &lt;- data.frame(annotation_mat, pca_n45$x)


head(pca_n45_anno)[,1:12]</code></pre>
<pre><code>             indv trt     time       sample        PC1       PC2         PC3
Ind1_75DA24h    1 DNR 24 hours Ind1_75DA24h -310.51421  35.67950 -115.422874
Ind1_75DA3h     1 DNR  3 hours  Ind1_75DA3h -138.03488 -51.56883   84.677561
Ind1_75DX24h    1 DOX 24 hours Ind1_75DX24h -220.03002 207.36404    2.333076
Ind1_75DX3h     1 DOX  3 hours  Ind1_75DX3h  -64.31471  69.74401   45.932581
Ind1_75E24h     1 EPI 24 hours  Ind1_75E24h -230.03135 233.95335   16.889693
Ind1_75E3h      1 EPI  3 hours   Ind1_75E3h  -97.54951  47.26703   65.389711
                  PC4        PC5        PC6        PC7        PC8
Ind1_75DA24h  9.63590  -3.296965  13.547571 -10.728337  17.978191
Ind1_75DA3h  35.40794 133.803475 -61.057095  13.572833   4.248771
Ind1_75DX24h 40.08605   7.836060  10.825420 -16.104147  -9.787441
Ind1_75DX3h  16.23451 119.584048 -48.873227   5.326085  -6.740912
Ind1_75E24h  50.91447   9.958310   8.998017 -14.855373 -16.007582
Ind1_75E3h   13.10312 128.342249 -59.441331  17.413143  -9.700106</code></pre>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

pca_n45_anno %&gt;%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
  #scale_shape_manual(name = &quot;Time&quot;,values= c(&quot;3h&quot;=0,&quot;24h&quot;=1))+
  ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm)&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  # labs(y = &quot;PC 2 (15.76%)&quot;, x =&quot;PC 1 (29.06%)&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20initial%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-initial-peaks-1">
Past versions of "filtering step initial peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-initial-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-1.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-1.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-1.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/78795a75745e18937e4dc541e9c1f9be5eecb5b1/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-1.png" target="_blank">78795a7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_n45_anno %&gt;%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
  #scale_shape_manual(name = &quot;Time&quot;,values= c(&quot;3h&quot;=0,&quot;24h&quot;=1))+
  ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(peaks)&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  # labs(y = &quot;PC 2 (15.76%)&quot;, x =&quot;PC 1 (29.06%)&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20initial%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-initial-peaks-2">
Past versions of "filtering step initial peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-initial-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-2.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-2.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-2.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-2.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/78795a75745e18937e4dc541e9c1f9be5eecb5b1/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-2.png" target="_blank">78795a7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Frag_cor_filter &lt;- filt_matrix_lcpm %&gt;% cor()
ComplexHeatmap::pheatmap(Frag_cor_filter,
                         # column_title=(paste0(&quot;RNA-seq log&quot;[2]~&quot;cpm correlation&quot;)),
        annotation_col = counts_corr_mat,
        annotation_colors = mat_colors,
        heatmap_legend_param = mat_colors,
        fontsize=10,
        fontsize_row = 8,
        angle_col=&quot;90&quot;,
        treeheight_row=25,
        fontsize_col = 8,
        treeheight_col = 20)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20initial%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-initial-peaks-3">
Past versions of "filtering step initial peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-initial-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/78795a75745e18937e4dc541e9c1f9be5eecb5b1/docs/figure/Peak_analysis.Rmd/filtering step initial peaks-3.png" target="_blank">78795a7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We go from 463947, 72 peaks to170488, 72 peaks using rowMeans
(log2cpm) &gt; 0 across all samples. The heatmap shows good clustering
in treatments, with individual 5 and some of individual 4 at 3 hours
clustering outside. This is inline with QC metrics (FRiP) and Fragment
lengths graphs.</p>
</div>
<div id="high-confidence-peak-set" class="section level3">
<h3>High confidence peak set</h3>
<p>Based on the initial peaks, I wanted to know if they same thing is
seen in a high confidence set of peaks. To this end, I did several steps
in bedtools to create a high confidence set.</p>
<p>-First I moved all .narrowPeak files into the same folder and ran
<code>bedtools multiinter -i ./* &gt;log.file.txt</code> to create an
intersection of all peaks. I then was only interested in segments that
had a count of more than 4 (intersection existed in at least 4 of the
data sets) in all files. I filtered column #4 of the
<code>log.file.txt</code> output by
<code>awk -F"\t" '$4 &gt; 4 {print $1"\t"$2"\t"$3 }' log.file.txt &gt; all_filt_peaks.bed</code>
and printed the results of anything &gt;4 in bed format to the
<code>all_filt_peaks.bed</code> file. Upon further reading of bedtools
documents, I realized the number of “peaks” was actually fragments of
peaks that were intersected amoung all files. This was not the final
output I wanted so I intersected these high counted segments back with
the very first initial mergedPeaks.bed file using
<code>bedtools intersect -a mergedPeaks.bed -b all_filt_peaks.bed -wa -u &gt; merged_filtered_peaks.bed</code>.
using the <code>-wa -u</code> flags allowed me to filter the first
mergedPeaks file, keeping only those high confidence peaks that
overlapped the <code>all_filt_peaks.bed</code> and only reporting the
unique calls. This left me with a file that went from 470,000+ to
162,264 high confidence peaks that I will analyze below.</p>
<p>** I added a new step on the 15th. I filtered out the blacklisted
regions using the following code:
<code>intersectBed -v -a merged_filtered_peaks.bed -b Blacklist/hg38.blacklist.bed.gz &gt; final_bl_filt_peaks.bed</code>
My final peaks file contains 162243 peaks. Not much of a difference, but
may have an impact later after I reanalyze.</p>
<pre class="r"><code>##the file was called all_filt_peaks before I imported into this library
##I then saved as high_conf_peaks_counts.txt

###this removes that pesky long file names
# names(high_conf_peak_counts) = gsub(pattern = &quot;_S.*&quot;, replacement = &quot;&quot;, x = names(high_conf_peak_counts))
# # 
# names(high_conf_peak_counts)=gsub(pattern = &quot;^ind6.trimmed.filt_files.trimmed_&quot;, replacement = &quot;&quot;, x = names(high_conf_peak_counts))

# write.csv(high_conf_peak_counts, &quot;data/high_conf_peaks_bl_counts.txt&quot;)
## write_delim(all_bl_filt_peaks, &quot;data/high_conf_peaks_bl_counts.txt&quot;)
high_conf_peak_counts &lt;- read.csv(&quot;data/high_conf_peaks_bl_counts.txt&quot;, row.names = 1)


Frag_cor_filt &lt;- high_conf_peak_counts %&gt;% 
  dplyr::select(Ind1_75DA24h:Ind6_71V3h)  %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
  
filmat_groupmat_col1 &lt;- data.frame(timeset = colnames(Frag_cor_filt))

counts_corr_mat1 &lt;-filmat_groupmat_col1 %&gt;% 
  mutate(timeset=gsub(&quot;75&quot;,&quot;1_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;87&quot;,&quot;2_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;77&quot;,&quot;3_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;79&quot;,&quot;4_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;78&quot;,&quot;5_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;71&quot;,&quot;6_&quot;,timeset)) %&gt;% 
  mutate(timeset = gsub(&quot;24h&quot;,&quot;_24h&quot;,timeset), 
       timeset = gsub(&quot;3h&quot;,&quot;_3h&quot;,timeset)) %&gt;%
  separate(timeset, into = c(NA,&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
  ))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))
  ))) 
                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;),
   indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;, &quot;#E7298A&quot; ,&quot;#66A61E&quot;, &quot;#E6AB02&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat1$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat1$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat1$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat1$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat1$TOP2i)


ComplexHeatmap::pheatmap(Frag_cor_filt,
                         # column_title=(paste0(&quot;RNA-seq log&quot;[2]~&quot;cpm correlation&quot;)),
        annotation_col = counts_corr_mat1,
        annotation_colors = mat_colors,
        heatmap_legend_param = mat_colors,
        fontsize=10,
        fontsize_row = 8,
        angle_col=&quot;90&quot;,
        treeheight_row=25,
        fontsize_col = 8,
        treeheight_col = 20)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/highconf%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-highconf-peaks-1">
Past versions of "highconf peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-highconf-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/highconf peaks-1.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/78795a75745e18937e4dc541e9c1f9be5eecb5b1/docs/figure/Peak_analysis.Rmd/highconf peaks-1.png" target="_blank">78795a7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Changes in the heatmap are as follows:<br />
Ind4 and Ind5 segregate out of the full correlation. 3 hour treatments
AC and non-AC cluster out into the lower right. Very sharp AC signal
seen in 3 hours and 24 hours so far.</p>
<p>Next I will try my initial Differential accessibility analysis.</p>
</div>
<div id="dar-high-confidence-peak-set1" class="section level3">
<h3>DAR high confidence peak set1</h3>
<pre class="r"><code>##filter log cpm counts file
my_hc_counts &lt;- high_conf_peak_counts %&gt;% 
  dplyr::select(Geneid,Ind1_75DA24h:Ind6_71V3h) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;)

lcpm &lt;- cpm(my_hc_counts, log=TRUE)  ### for determining the basic cutoffs



row_means &lt;- rowMeans(lcpm)
my_hc_filtered_counts &lt;- my_hc_counts[row_means &gt; 0,]
dim(my_hc_filtered_counts)</code></pre>
<pre><code>[1] 153348     72</code></pre>
<pre class="r"><code>##3 now have 153,348 high conf peaks
group &lt;- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),6))
group &lt;- factor(group, levels =c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;,&quot;7&quot;,&quot;8&quot;,&quot;9&quot;,&quot;10&quot;,&quot;11&quot;,&quot;12&quot;))
short_names &lt;- paste0(counts_corr_mat1$indv,&quot;_&quot;,counts_corr_mat1$trt,&quot;_&quot;,counts_corr_mat1$time)
# saveRDS(my_hc_filtered_counts,&quot;data/my_hc_filt_counts.RDS&quot;)
dge &lt;- DGEList.data.frame(counts = my_hc_filtered_counts, group = group, genes = row.names(my_hc_filtered_counts))

##renaming colnames
colnames(my_hc_filtered_counts) &lt;- short_names
dge$group$indv &lt;- counts_corr_mat1$indv
dge$group$time &lt;- counts_corr_mat1$time
dge$group$trt &lt;- counts_corr_mat1$trt




indv &lt;- counts_corr_mat1$indv
# indv &lt;- factor(indv, levels = c(1,2,3,4,5,6))
time &lt;- counts_corr_mat1$time
# time &lt;- factor(time, levels =c(&quot;3h&quot;,&quot;24&quot;))
trt &lt;- counts_corr_mat1$trt
# trt &lt;- factor(trt, levels = c(&quot;VEH&quot;,&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,))

##commented out to save time for future processing

efit2 &lt;- readRDS(&quot;data/filt_Peaks_efit2_bl.RDS&quot;)

group_1 &lt;- c(rep(c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;),6))


mm &lt;- model.matrix(~0 +group_1)  
# colnames(mm) &lt;- c(&quot;DNR_24&quot;, &quot;DNtrt_n45# colnames(mm) &lt;- c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;, &quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)

# y &lt;- voom(dge$counts, mm,plot =TRUE)
# 
# corfit &lt;- duplicateCorrelation(y, mm, block = indv)
# 
# v &lt;- voom(dge$counts, mm, block = indv, correlation = corfit$consensus)
# 
# fit &lt;- lmFit(v, mm, block = indv, correlation = corfit$consensus)
# colnames(mm) &lt;- c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
#
#
# cm &lt;- makeContrasts(
#   DNR_3.VEH_3 = DNR_3-VEH_3,
#   DOX_3.VEH_3 = DOX_3-VEH_3,
#   EPI_3.VEH_3 = EPI_3-VEH_3,
#   MTX_3.VEH_3 = MTX_3-VEH_3,
#   TRZ_3.VEH_3 = TRZ_3-VEH_3,
#   DNR_24.VEH_24 =DNR_24-VEH_24,
#   DOX_24.VEH_24= DOX_24-VEH_24,
#   EPI_24.VEH_24= EPI_24-VEH_24,
#   MTX_24.VEH_24= MTX_24-VEH_24,
#   TRZ_24.VEH_24= TRZ_24-VEH_24,
#   levels = mm)

# vfit &lt;- lmFit(y, mm)

# vfit&lt;- contrasts.fit(vfit, contrasts=cm)
# 
# efit2 &lt;- eBayes(vfit)
# saveRDS(efit2,&quot;data/filt_Peaks_efit2_bl.RDS&quot;)

results = decideTests(efit2)
summary(results)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 TRZ_3.VEH_3
Down         10075        1289        7363         362           0
NotSig      138053      151776      142388      152727      153348
Up            5220         283        3597         259           0
       DNR_24.VEH_24 DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24 TRZ_24.VEH_24
Down           29112         24984         25991          3654             0
NotSig         89482        102741        102410        144334        153348
Up             34754         25623         24947          5360             0</code></pre>
</div>
<div id="evaluation-of-change-in-peaks" class="section level3">
<h3>Evaluation of change in peaks</h3>
<pre class="r"><code>V.DNR_3.top= topTable(efit2, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_3.top= topTable(efit2, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_3.top= topTable(efit2, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_3.top= topTable(efit2, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_3.top= topTable(efit2, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DNR_24.top= topTable(efit2, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_24.top= topTable(efit2, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_24.top= topTable(efit2, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_24.top= topTable(efit2, coef=9, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_24.top= topTable(efit2, coef=10, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)

# toplist_full  &lt;- list(V.DNR_3.top, V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,V.TRZ_3.top,V.DNR_24.top, V.DOX_24.top,V.EPI_24.top,V.MTX_24.top,V.TRZ_24.top)
# names(toplist_full) &lt;- c(&quot;DNR_3&quot;, &quot;DOX_3&quot;,&quot;EPI_3&quot;,&quot;MTX_3&quot;,&quot;TRZ_3&quot;,&quot;DNR_24&quot;, &quot;DOX_24&quot;,&quot;EPI_24&quot;,&quot;MTX_24&quot;,&quot;TRZ_24&quot;)
# toplist_6 &lt;-map_df(toplist_full, ~as.data.frame(.x), .id=&quot;trt_time&quot;)
# 
# toplist_6 &lt;- toplist_6 %&gt;% 
#   separate(trt_time, into= c(&quot;trt&quot;,&quot;time&quot;), sep = &quot;_&quot;) %&gt;% 
#   mutate(trt=factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;))) %&gt;% 
#   mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels = c(&quot;3 hours&quot;, &quot;24 hours&quot;)))
# saveRDS(toplist_6, &quot;data/toplist_6.RDS&quot;)</code></pre>
</div>
<div id="hour-boxplots-full-set" class="section level3">
<h3>3 hour boxplots full set</h3>
<pre class="r"><code>DNR_3_top3 &lt;- row.names(V.DNR_3.top[1:3,])

log_filt_hc &lt;- my_hc_filtered_counts %&gt;% 
  cpm(., log=TRUE) %&gt;% as.data.frame()

row.names(log_filt_hc) &lt;- row.names(my_hc_filtered_counts)
log_filt_hc %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h-1">
Past versions of "examine top3_3h-1.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/examine top3_3h-1.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_3_top3 &lt;- row.names(V.DOX_3.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% DOX_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h-2">
Past versions of "examine top3_3h-2.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h-2.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/examine top3_3h-2.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_3_top3 &lt;- row.names(V.EPI_3.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% EPI_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h-3">
Past versions of "examine top3_3h-3.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h-3.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/examine top3_3h-3.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_3_top3 &lt;- row.names(V.MTX_3.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% MTX_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h-4">
Past versions of "examine top3_3h-4.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h-4.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/examine top3_3h-4.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_3_top3 &lt;- row.names(V.TRZ_3.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% TRZ_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h-5">
Past versions of "examine top3_3h-5.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h-5.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/examine top3_3h-5.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="hour-boxplots-full-set-1" class="section level2">
<h2>24 hour boxplots full set</h2>
<pre class="r"><code>DNR_24_top3 &lt;- row.names(V.DNR_24.top[1:3,])

log_filt_hc &lt;- my_hc_filtered_counts %&gt;% 
  cpm(., log=TRUE) %&gt;% as.data.frame()
row.names(log_filt_hc) &lt;- row.names(my_hc_filtered_counts)
log_filt_hc %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hourtopexpression-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-24hourtopexpression-1">
Past versions of 24hourtopexpression-1.png
</button>
</p>
<div id="fig-24hourtopexpression-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hourtopexpression-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/24hourtopexpression-1.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_24_top3 &lt;- row.names(V.DOX_24.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% DOX_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hourtopexpression-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-24hourtopexpression-2">
Past versions of 24hourtopexpression-2.png
</button>
</p>
<div id="fig-24hourtopexpression-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hourtopexpression-2.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/24hourtopexpression-2.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_24_top3 &lt;- row.names(V.EPI_24.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% EPI_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hourtopexpression-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-24hourtopexpression-3">
Past versions of 24hourtopexpression-3.png
</button>
</p>
<div id="fig-24hourtopexpression-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hourtopexpression-3.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/24hourtopexpression-3.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_24_top3 &lt;- row.names(V.MTX_24.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% MTX_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hourtopexpression-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-24hourtopexpression-4">
Past versions of 24hourtopexpression-4.png
</button>
</p>
<div id="fig-24hourtopexpression-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hourtopexpression-4.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/24hourtopexpression-4.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_24_top3 &lt;- row.names(V.TRZ_24.top[1:3,])

log_filt_hc %&gt;%  
  dplyr::filter(row.names(.) %in% TRZ_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hourtopexpression-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-24hourtopexpression-5">
Past versions of 24hourtopexpression-5.png
</button>
</p>
<div id="fig-24hourtopexpression-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hourtopexpression-5.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8799b87fd20e65a9a7fb8173c9c57470a42c118d/docs/figure/Peak_analysis.Rmd/24hourtopexpression-5.png" target="_blank">8799b87</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="volcano-plots-of-peaks" class="section level3">
<h3>Volcano plots of peaks</h3>
<pre class="r"><code>library(cowplot)
efit2 &lt;- readRDS(&quot;data/filt_Peaks_efit2_bl.RDS&quot;)
# volcanoplot(efit2,coef = c(1,2,3,4,5,6),style = &quot;p-value&quot;,
#                           # highlight = 8,
#                           # names = efit2$genes$SYMBOL,
#                           hl.col = &quot;red&quot;,xlab = &quot;Log2 Fold Change&quot;,
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = &quot;test&quot;)
# 

plot_filenames &lt;- c(&quot;V.DNR_3.top&quot;,&quot;V.DOX_3.top&quot;,&quot;V.EPI_3.top&quot;,&quot;V.MTX_3.top&quot;,
                    &quot;V.TRZ_.top&quot;,&quot;V.DNR_24.top&quot;,&quot;V.DOX_24.top&quot;,&quot;V.EPI_24.top&quot;,
                    &quot;V.MTX_24.top&quot;,&quot;V.TRZ_24.top&quot;)
plot_files &lt;- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
                    V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
                    V.MTX_24.top,V.TRZ_24.top)

volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
      # ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC &gt;= 0,&quot;B&quot;, &quot;C&quot;)))
    ##This is where I could add labels, but I have taken out
    # df &lt;- df %&gt;% mutate(genelabels = &quot;&quot;)
    # df$genelabels[1:topg] &lt;- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(adj.P.Val))) + 
    geom_point(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, &quot;npc&quot;)), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;adj. P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}
#v1&lt;- volcanosig(V.DA24.top, 0.01,0)
v1 &lt;- volcanosig(V.DNR_3.top, 0.01)+ ggtitle(&quot;Daunorubicin 3 hour&quot;)
v2 &lt;- volcanosig(V.DNR_24.top, 0.01)+ ggtitle(&quot;Daunorubicin 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(V.DOX_3.top, 0.01)+ ggtitle(&quot;Doxorubicin 3 hour&quot;)
v4 &lt;- volcanosig(V.DOX_24.top, 0.01)+ ggtitle(&quot;Doxorubicin 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(V.EPI_3.top, 0.01)+ ggtitle(&quot;Epirubicin 3 hour&quot;)
v6 &lt;- volcanosig(V.EPI_24.top, 0.01)+ ggtitle(&quot;Epirubicin 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(V.MTX_3.top, 0.01)+ ggtitle(&quot;Mitoxantrone 3 hour&quot;)
v8 &lt;- volcanosig(V.MTX_24.top, 0.01)+ ggtitle(&quot;Mitoxantrone 24 hour&quot;)+ylab(&quot;&quot;)
v9 &lt;- volcanosig(V.TRZ_3.top, 0.01)+ ggtitle(&quot;Trastuzumab 3 hour&quot;)
v10 &lt;- volcanosig(V.TRZ_24.top, 0.01)+ ggtitle(&quot;Trastuzumab 24 hour&quot;)+ylab(&quot;&quot;)
# volcanoplot(efit2,coef = 10,style = &quot;p-value&quot;,
#                           highlight = 8,
#                           names = efit2$genes$SYMBOL,
#                           hl.col = &quot;red&quot;,xlab = &quot;Log2 Fold Change&quot;,
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = &quot;Using Trastuzumab 24 hour data and volcanoplot function&quot;)
plot_grid(v1,v2,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-1">
Past versions of "volcano plots-1.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/volcano plots-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3,v4,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-2">
Past versions of "volcano plots-2.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/volcano plots-2.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5,v6,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-3">
Past versions of "volcano plots-3.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/volcano plots-3.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7,v8,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-4">
Past versions of "volcano plots-4.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/volcano plots-4.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v9,v10,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-5">
Past versions of "volcano plots-5.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/volcano plots-5.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-analysis-of-full-set" class="section level3">
<h3>PCA analysis of full set</h3>
<p>This is the PCA above redone on the filtered matrix of
my_hc_filtered_counts. The “low’ expression</p>
<pre class="r"><code>my_hc_filtered_counts &lt;- readRDS(&quot;data/my_hc_filt_counts.RDS&quot;)
PCAmat_all &lt;- my_hc_filtered_counts %&gt;%
  cpm(., log = TRUE) %&gt;% 
  as.matrix()

annotation_mat_all  &lt;-
  data.frame(timeset=colnames(PCAmat_all )) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) %&gt;% 
  mutate(indv = factor(indv, levels = c(&quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;, &quot;6&quot;))) %&gt;%
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) 

PCA_info_all  &lt;- (prcomp(t(PCAmat_all ), scale. = TRUE)) 
PCA_info_anno_all  &lt;- PCA_info_all$x %&gt;% cbind(.,annotation_mat_all )
# autoplot(PCA_info)
summary(PCA_info_all)</code></pre>
<pre><code>Importance of components:
                            PC1      PC2       PC3     PC4     PC5      PC6
Standard deviation     159.4523 145.5401 100.97737 90.1550 74.0920 66.96354
Proportion of Variance   0.1658   0.1381   0.06649  0.0530  0.0358  0.02924
Cumulative Proportion    0.1658   0.3039   0.37042  0.4234  0.4592  0.48846
                            PC7      PC8      PC9     PC10     PC11     PC12
Standard deviation     58.67103 57.98376 56.58485 53.67913 53.06653 52.94208
Proportion of Variance  0.02245  0.02192  0.02088  0.01879  0.01836  0.01828
Cumulative Proportion   0.51091  0.53284  0.55372  0.57251  0.59087  0.60915
                           PC13     PC14    PC15     PC16     PC17     PC18
Standard deviation     51.10291 48.56176 47.6350 46.43015 45.49040 43.58899
Proportion of Variance  0.01703  0.01538  0.0148  0.01406  0.01349  0.01239
Cumulative Proportion   0.62618  0.64156  0.6563  0.67041  0.68391  0.69630
                           PC19     PC20     PC21     PC22     PC23     PC24
Standard deviation     43.40574 42.09537 41.09479 40.95717 39.98403 39.61774
Proportion of Variance  0.01229  0.01156  0.01101  0.01094  0.01043  0.01024
Cumulative Proportion   0.70858  0.72014  0.73115  0.74209  0.75252  0.76275
                           PC25     PC26    PC27     PC28     PC29     PC30
Standard deviation     38.41466 37.62660 36.9328 35.74224 35.64233 35.04227
Proportion of Variance  0.00962  0.00923  0.0089  0.00833  0.00828  0.00801
Cumulative Proportion   0.77237  0.78161  0.7905  0.79883  0.80712  0.81512
                           PC31     PC32     PC33     PC34     PC35     PC36
Standard deviation     34.03350 33.41242 32.85185 32.45692 32.30540 31.21309
Proportion of Variance  0.00755  0.00728  0.00704  0.00687  0.00681  0.00635
Cumulative Proportion   0.82268  0.82996  0.83700  0.84386  0.85067  0.85702
                           PC37     PC38     PC39     PC40     PC41     PC42
Standard deviation     30.47407 30.19149 30.00867 29.51048 29.41676 29.21973
Proportion of Variance  0.00606  0.00594  0.00587  0.00568  0.00564  0.00557
Cumulative Proportion   0.86308  0.86902  0.87490  0.88058  0.88622  0.89179
                           PC43     PC44     PC45     PC46     PC47    PC48
Standard deviation     28.83057 28.17079 27.91791 27.29198 27.08496 26.8400
Proportion of Variance  0.00542  0.00518  0.00508  0.00486  0.00478  0.0047
Cumulative Proportion   0.89721  0.90238  0.90746  0.91232  0.91711  0.9218
                           PC49    PC50    PC51     PC52     PC53     PC54
Standard deviation     26.73380 25.9682 25.6932 25.46750 25.27056 24.94023
Proportion of Variance  0.00466  0.0044  0.0043  0.00423  0.00416  0.00406
Cumulative Proportion   0.92646  0.9309  0.9352  0.93940  0.94356  0.94762
                           PC55     PC56     PC57     PC58     PC59     PC60
Standard deviation     24.51715 24.21432 24.02275 23.51529 23.39981 23.00520
Proportion of Variance  0.00392  0.00382  0.00376  0.00361  0.00357  0.00345
Cumulative Proportion   0.95154  0.95536  0.95912  0.96273  0.96630  0.96975
                           PC61     PC62     PC63     PC64    PC65    PC66
Standard deviation     22.64560 22.41860 22.32560 21.66939 20.7228 20.7064
Proportion of Variance  0.00334  0.00328  0.00325  0.00306  0.0028  0.0028
Cumulative Proportion   0.97309  0.97637  0.97962  0.98268  0.9855  0.9883
                           PC67    PC68     PC69     PC70     PC71      PC72
Standard deviation     20.19775 19.9668 18.99846 18.32628 17.13658 5.366e-13
Proportion of Variance  0.00266  0.0026  0.00235  0.00219  0.00192 0.000e+00
Cumulative Proportion   0.99094  0.9935  0.99589  0.99808  1.00000 1.000e+00</code></pre>
<pre class="r"><code># cpm(PCAmat, log=TRUE)
# pca_plot(PCA_info_all , col_var=&#39;trt&#39;, shape_var = &#39;time&#39;)
#         
# pca_plot(PCA_info_all , col_var=&#39;trt&#39;, shape_var = &#39;indv&#39;)


drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

PCA_info_anno_all  %&gt;%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
  #scale_shape_manual(name = &quot;Time&quot;,values= c(&quot;3h&quot;=0,&quot;24h&quot;=1))+
  ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm)&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 2 (13.81%)&quot;, x =&quot;PC 1 (16.58%)&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20hcpeaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-hcpeaks-1">
Past versions of "filtering step hcpeaks-1.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-hcpeaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/filtering step hcpeaks-1.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/filtering step hcpeaks-1.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>PCA_info_anno_all  %&gt;%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
  #scale_shape_manual(name = &quot;Time&quot;,values= c(&quot;3h&quot;=0,&quot;24h&quot;=1))+
  ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(peaks)&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 4 (5.3%)&quot;, x =&quot;PC 3 (6.65%)&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20hcpeaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-hcpeaks-2">
Past versions of "filtering step hcpeaks-2.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-hcpeaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/filtering step hcpeaks-2.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/filtering step hcpeaks-2.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-of-trt-time-indv-for-full-set" class="section level3">
<h3>PCA of trt, time, indv for full set</h3>
<pre class="r"><code>facs &lt;- c(&quot;indv&quot;, &quot;trt&quot;, &quot;time&quot;)
names(facs) &lt;- c(&quot;Individual&quot;, &quot;Treatment&quot;, &quot;Time&quot;)
drug1 &lt;- c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;)##for changing shapes and colors
time &lt;- rep(c(&quot;24h&quot;, &quot;3h&quot;),36) %&gt;% factor(., levels = c(&quot;3h&quot;,&quot;24h&quot;))
##gglistmaking
for (f in facs) {
  # PC1 v PC2
  pca_plot(PCA_info_anno_all, col_var = f, shape_var = time,
           title = names(facs)[which(facs == f)])
  print(last_plot())
  
  # Plot f versus PC1 and PC2
  f_v_pc1 &lt;- arrangeGrob(plot_versus_pc(PCA_info_anno_all, 1, f))
  f_v_pc2 &lt;- arrangeGrob(plot_versus_pc(PCA_info_anno_all, 2, f))
  grid.arrange(f_v_pc1, f_v_pc2, ncol = 2, top = names(facs)[which(facs == f)])
  # summary(plot_versus_pc(PCA_info_anno_all, 1, f))
  # summary(plot_versus_pc(PCA_info_anno_all, 2, f))
 
}</code></pre>
<p><img src="figure/Peak_analysis.Rmd/PCAfull%20segments-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAfull-segments-1">
Past versions of "PCAfull segments-1.png"
</button>
</p>
<div id="fig-no-spaces-PCAfull-segments-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAfull segments-1.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAfull%20segments-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAfull-segments-2">
Past versions of "PCAfull segments-2.png"
</button>
</p>
<div id="fig-no-spaces-PCAfull-segments-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAfull segments-2.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAfull%20segments-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAfull-segments-3">
Past versions of "PCAfull segments-3.png"
</button>
</p>
<div id="fig-no-spaces-PCAfull-segments-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAfull segments-3.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAfull%20segments-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAfull-segments-4">
Past versions of "PCAfull segments-4.png"
</button>
</p>
<div id="fig-no-spaces-PCAfull-segments-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAfull segments-4.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAfull%20segments-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAfull-segments-5">
Past versions of "PCAfull segments-5.png"
</button>
</p>
<div id="fig-no-spaces-PCAfull-segments-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAfull segments-5.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAfull%20segments-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAfull-segments-6">
Past versions of "PCAfull segments-6.png"
</button>
</p>
<div id="fig-no-spaces-PCAfull-segments-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAfull segments-6.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="magnitude-of-response-full-set" class="section level3">
<h3>Magnitude of Response full set</h3>
<pre class="r"><code>toplist_6 &lt;- readRDS(&quot;data/toplist_6.RDS&quot;)

toplist_6 %&gt;% 
  group_by(time, trt) %&gt;% 
  mutate(sigcount = if_else(adj.P.Val &lt; 0.05,&#39;sig&#39;,&#39;notsig&#39;))%&gt;%
  count(sigcount) %&gt;% 
  pivot_wider(id_cols = c(time,trt), names_from=sigcount, values_from=n) %&gt;% 
  mutate(prop = sig/(sig+notsig)*100) %&gt;% 
  mutate(prop=if_else(is.na(prop),0,prop)) %&gt;% 
  ggplot(., aes(x=trt, y= prop))+
  geom_col(aes(fill=trt))+
  geom_text(aes(label = sprintf(&quot;%.2f&quot;,prop)),
            position=position_dodge(0.9),vjust=-.2 )+
  scale_fill_manual(values =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  facet_wrap(~time)+#labeller = (time = facettimelabel) )+
  theme_bw()+
  xlab(&quot;&quot;)+
  ylab(&quot;Percentage DAR peaks&quot;)+
  theme_bw()+
  ggtitle(&quot;Percent DARs (adj. P value &lt;0.05)&quot;)+
  scale_y_continuous(expand=expansion(c(0.02,.2)))+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_text(size = 8, color = &quot;white&quot;, angle = 0),
        axis.text.y = element_text(size = 8, color = &quot;black&quot;, angle = 0),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/Magresp%20full%20set-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Magresp-full-set-1">
Past versions of "Magresp full set-1.png"
</button>
</p>
<div id="fig-no-spaces-Magresp-full-set-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/Magresp full set-1.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_6 %&gt;% 
  group_by(time, trt) %&gt;% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/Magresp%20full%20set-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Magresp-full-set-2">
Past versions of "Magresp full set-2.png"
</button>
</p>
<div id="fig-no-spaces-Magresp-full-set-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/Magresp full set-2.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_6 %&gt;% 
  group_by(time, trt) %&gt;% 
  dplyr::filter(trt != &quot;TRZ&quot;) %&gt;%
  ggplot(., aes( x=(adj.P.Val)))+
  geom_histogram(aes(y=..density..), colour=&quot;black&quot;, fill=&quot;white&quot;)+
  geom_density(alpha=.2, fill=&quot;#FF6666&quot;)+
  theme_bw()+
  ggtitle(&quot;adj.P.Value density by time/trt&quot;)+
  facet_wrap(time~trt)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/Magresp%20full%20set-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Magresp-full-set-3">
Past versions of "Magresp full set-3.png"
</button>
</p>
<div id="fig-no-spaces-Magresp-full-set-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/Magresp full set-3.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_6 %&gt;% 
  group_by(time, trt) %&gt;% 
  # dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  ggplot(., aes( x=(P.Value)))+
  geom_histogram(aes(y=..density..), colour=&quot;black&quot;, fill=&quot;white&quot;)+
  geom_density(alpha=.2, fill=&quot;#FF6666&quot;)+
  theme_bw()+
  ggtitle(&quot;P.Value density by time/trt&quot;)+
  facet_wrap(time~trt)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/Magresp%20full%20set-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Magresp-full-set-4">
Past versions of "Magresp full set-4.png"
</button>
</p>
<div id="fig-no-spaces-Magresp-full-set-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/Magresp full set-4.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="analysis-without-indv-4-and-5" class="section level3">
<h3>Analysis without Indv 4 and 5</h3>
<div id="heatmap-log2cpm" class="section level4">
<h4>Heatmap log2cpm</h4>
<pre class="r"><code>high_conf_peak_counts &lt;- read.csv(&quot;data/high_conf_peaks_bl_counts.txt&quot;, row.names = 1)


Frag_cor_filt_n45 &lt;- high_conf_peak_counts %&gt;% 
  dplyr::select(Ind1_75DA24h:Ind3_77V3h,Ind6_71DA24h:Ind6_71V3h)  %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
  
filmat_groupmat_col_n45 &lt;- data.frame(timeset = colnames(Frag_cor_filt_n45))

counts_corr_mat_n45 &lt;-filmat_groupmat_col_n45 %&gt;% 
  mutate(timeset=gsub(&quot;75&quot;,&quot;1_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;87&quot;,&quot;2_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;77&quot;,&quot;3_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;79&quot;,&quot;4_&quot;,timeset)) %&gt;% 
  mutate(timeset=gsub(&quot;78&quot;,&quot;5_&quot;,timeset)) %&gt;%
  mutate(timeset=gsub(&quot;71&quot;,&quot;6_&quot;,timeset)) %&gt;% 
  mutate(timeset = gsub(&quot;24h&quot;,&quot;_24h&quot;,timeset), 
       timeset = gsub(&quot;3h&quot;,&quot;_3h&quot;,timeset)) %&gt;%
  separate(timeset, into = c(NA,&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
  ))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))
  ))) 
                         
 mat_colors_n45 &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;),
   indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;, &quot;#E6AB02&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors_n45$trt)   &lt;- unique(counts_corr_mat_n45$trt)                      
names(mat_colors_n45$indv) &lt;- unique(counts_corr_mat_n45$indv)
names(mat_colors_n45$time) &lt;- unique(counts_corr_mat_n45$time)
names(mat_colors_n45$class) &lt;- unique(counts_corr_mat_n45$class)
names(mat_colors_n45$TOP2i) &lt;- unique(counts_corr_mat_n45$TOP2i)


ComplexHeatmap::pheatmap(Frag_cor_filt_n45,
                         # column_title=(paste0(&quot;RNA-seq log&quot;[2]~&quot;cpm correlation&quot;)),
        annotation_col = counts_corr_mat_n45,
        annotation_colors = mat_colors_n45,
        heatmap_legend_param = mat_colors_n45,
        fontsize=10,
        fontsize_row = 8,
        angle_col=&quot;90&quot;,
        treeheight_row=25,
        fontsize_col = 8,
        treeheight_col = 20)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/no%204%20or%205%20log2cpm-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-no-4-or-5-log2cpm-1">
Past versions of "no 4 or 5 log2cpm-1.png"
</button>
</p>
<div id="fig-no-spaces-no-4-or-5-log2cpm-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/no 4 or 5 log2cpm-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="deg-using-1236" class="section level4">
<h4>DEG using 1,2,3,6</h4>
<pre class="r"><code>my_hc_counts_n45 &lt;- high_conf_peak_counts %&gt;% 
  dplyr::select(Geneid,Ind1_75DA24h:Ind3_77V3h,Ind6_71DA24h:Ind6_71V3h) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;)

lcpm_n45 &lt;- cpm(my_hc_counts_n45, log=TRUE)  ### for determining the basic cutoffs



row_means_n45 &lt;- rowMeans(lcpm_n45)
my_hc_filtered_counts_n45 &lt;- my_hc_counts_n45[row_means_n45 &gt; 0,]
dim(my_hc_filtered_counts_n45)</code></pre>
<pre><code>[1] 151800     48</code></pre>
<pre class="r"><code># saveRDS(my_hc_filtered_counts_n45,&quot;data/my_hc_filt_counts_n45.RDS&quot;)
##3 now have 151,800 high conf peaks
group_n45 &lt;- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group_n45 &lt;- factor(group_n45, levels =c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;,&quot;7&quot;,&quot;8&quot;,&quot;9&quot;,&quot;10&quot;,&quot;11&quot;,&quot;12&quot;))
short_names_n45 &lt;- paste0(counts_corr_mat_n45$indv,&quot;_&quot;,counts_corr_mat_n45$trt,&quot;_&quot;,counts_corr_mat_n45$time)

dge_n45 &lt;- DGEList.data.frame(counts = my_hc_filtered_counts_n45, group = group_n45, genes = row.names(my_hc_filtered_counts_n45))

##renaming colnames
colnames(my_hc_filtered_counts_n45) &lt;- short_names_n45
# dge_n45$group$indv &lt;- counts_corr_mat_n45$indv
# dge_n45$group$time &lt;- counts_corr_mat_n45$time
# dge_n45$group$trt &lt;- counts_corr_mat_n45$trt



indv_n45 &lt;- counts_corr_mat_n45$indv
# indv &lt;- factor(indv, levels = c(1,2,3,4,5,6))
time_n45 &lt;- counts_corr_mat_n45$time
# time &lt;- factor(time, levels =c(&quot;3h&quot;,&quot;24&quot;))
trt_n45 &lt;- counts_corr_mat_n45$trt
# trt &lt;- factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))

efit2_n45 &lt;- readRDS(&quot;data/filt_Peaks_efit2_n45.RDS&quot;)
##commented out to save time for future processing

group_n45 &lt;- c(rep(c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;),4))

mm_n4 &lt;- model.matrix(~0 +group_n45)
# mm_n45 &lt;- model.matrix(~0 + indv_n45 +trt_n45+time_n45)
# colnames(mm) &lt;- c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;, &quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)

# 
# y_n4 &lt;- voom(dge_n45$counts,mm_n4)
# 
# 
# corfit_n4 &lt;- duplicateCorrelation(y_n4, mm_n4, block = indv_n45)
# 
# v_n4 &lt;- voom(dge_n45$counts, mm_n4, block = indv_n45, correlation = corfit_n4$consensus)
# 
# # 
# # 
# fit_n4 &lt;- lmFit(v_n4, mm_n45, block = indv_n45, correlation = corfit_n4$consensus)
# 
# colnames(mm_n4) &lt;- c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)


# cm_n4 &lt;-makeContrasts(
#   DNR_3.VEH_3 = DNR_3-VEH_3,
#   DOX_3.VEH_3 = DOX_3-VEH_3,
#   EPI_3.VEH_3 = EPI_3-VEH_3,
#   MTX_3.VEH_3 = MTX_3-VEH_3,
#   TRZ_3.VEH_3 = TRZ_3-VEH_3,
#   DNR_24.VEH_24 =DNR_24-VEH_24,
#   DOX_24.VEH_24= DOX_24-VEH_24,
#   EPI_24.VEH_24= EPI_24-VEH_24,
#   MTX_24.VEH_24= MTX_24-VEH_24,
#   TRZ_24.VEH_24= TRZ_24-VEH_24,
#   levels = mm_n4)

# vfit_n4 &lt;- lmFit(y_n4, mm_n4)

# vfit_n4&lt;- contrasts.fit(vfit_n4, contrasts=cm_n4)
#
# efit2_n45 &lt;- eBayes(vfit_n4)

# saveRDS(efit2_n45,&quot;data/filt_Peaks_efit2_n45.RDS&quot;)

results_n45 = decideTests(efit2_n45)
summary(results_n45)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 TRZ_3.VEH_3
Down          5933         449        2998          20           0
NotSig      142909      151284      147699      151778      151800
Up            2958          67        1103           2           0
       DNR_24.VEH_24 DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24 TRZ_24.VEH_24
Down           27068         21257         22831          1571             0
NotSig         89455        105170        104309        146126        151800
Up             35277         25373         24660          4103             0</code></pre>
</div>
</div>
<div id="evaluation-of-change-in-peaks-1" class="section level3">
<h3>Evaluation of change in peaks</h3>
<pre class="r"><code>V.DNR_3.top_n45= topTable(efit2_n45, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_3.top_n45= topTable(efit2_n45, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_3.top_n45= topTable(efit2_n45, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_3.top_n45= topTable(efit2_n45, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_3.top_n45= topTable(efit2_n45, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DNR_24.top_n45= topTable(efit2_n45, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_24.top_n45= topTable(efit2_n45, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_24.top_n45= topTable(efit2_n45, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_24.top_n45= topTable(efit2_n45, coef=9, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_24.top_n45= topTable(efit2_n45, coef=10, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)

# toplist_full_n45  &lt;- list(V.DNR_3.top_n45, V.DOX_3.top_n45,V.EPI_3.top_n45,V.MTX_3.top_n45,V.TRZ_3.top_n45,V.DNR_24.top_n45, V.DOX_24.top_n45,V.EPI_24.top_n45,V.MTX_24.top_n45,V.TRZ_24.top_n45)
# names(toplist_full_n45) &lt;- c(&quot;DNR_3&quot;, &quot;DOX_3&quot;,&quot;EPI_3&quot;,&quot;MTX_3&quot;,&quot;TRZ_3&quot;,&quot;DNR_24&quot;, &quot;DOX_24&quot;,&quot;EPI_24&quot;,&quot;MTX_24&quot;,&quot;TRZ_24&quot;)
# toplist_n45 &lt;-map_df(toplist_full_n45, ~as.data.frame(.x), .id=&quot;trt_time&quot;)
# 
# toplist_n45 &lt;- toplist_n45 %&gt;%
#   separate(trt_time, into= c(&quot;trt&quot;,&quot;time&quot;), sep = &quot;_&quot;) %&gt;%
#   mutate(trt=factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;))) %&gt;%
#   mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels = c(&quot;3 hours&quot;, &quot;24 hours&quot;)))
# saveRDS(toplist_n45, &quot;data/toplist_n45.RDS&quot;)</code></pre>
</div>
<div id="hour-boxplots-without-4-and-5" class="section level3">
<h3>3 hour boxplots without 4 and 5</h3>
<pre class="r"><code>DNR_3_top3_n45 &lt;- row.names(V.DNR_3.top_n45[1:3,])

log_filt_hc_n45 &lt;- my_hc_filtered_counts_n45 %&gt;% 
  cpm(., log=TRUE) %&gt;% as.data.frame()

row.names(log_filt_hc_n45) &lt;- row.names(my_hc_filtered_counts_n45)
log_filt_hc_n45 %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_3_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h_n45-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h_n45-1">
Past versions of "examine top3_3h_n45-1.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h_n45-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h_n45-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_3_top3_n45 &lt;- row.names(V.DOX_3.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% DOX_3_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h_n45-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h_n45-2">
Past versions of "examine top3_3h_n45-2.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h_n45-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h_n45-2.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_3_top3_n45 &lt;- row.names(V.EPI_3.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% EPI_3_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h_n45-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h_n45-3">
Past versions of "examine top3_3h_n45-3.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h_n45-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h_n45-3.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_3_top3_n45 &lt;- row.names(V.MTX_3.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% MTX_3_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h_n45-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h_n45-4">
Past versions of "examine top3_3h_n45-4.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h_n45-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h_n45-4.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_3_top3_n45 &lt;- row.names(V.TRZ_3.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% TRZ_3_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/examine%20top3_3h_n45-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-top3_3h_n45-5">
Past versions of "examine top3_3h_n45-5.png"
</button>
</p>
<div id="fig-no-spaces-examine-top3_3h_n45-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/examine top3_3h_n45-5.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="hour-boxplots-without-4-and-5-1" class="section level3">
<h3>24 hour boxplots without 4 and 5</h3>
<pre class="r"><code>DNR_24_top3_n45 &lt;- row.names(V.DNR_24.top_n45[1:3,])

log_filt_hc_n45 &lt;- my_hc_filtered_counts_n45 %&gt;% 
  cpm(., log=TRUE) %&gt;% as.data.frame()
row.names(log_filt_hc_n45) &lt;- row.names(my_hc_filtered_counts_n45)
log_filt_hc_n45 %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_24_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hour%20n45-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-24hour-n45-1">
Past versions of "24hour n45-1.png"
</button>
</p>
<div id="fig-no-spaces-24hour-n45-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hour n45-1.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_24_top3_n45 &lt;- row.names(V.DOX_24.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% DOX_24_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hour%20n45-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-24hour-n45-2">
Past versions of "24hour n45-2.png"
</button>
</p>
<div id="fig-no-spaces-24hour-n45-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hour n45-2.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_24_top3_n45 &lt;- row.names(V.EPI_24.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% EPI_24_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hour%20n45-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-24hour-n45-3">
Past versions of "24hour n45-3.png"
</button>
</p>
<div id="fig-no-spaces-24hour-n45-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hour n45-3.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_24_top3_n45 &lt;- row.names(V.MTX_24.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% MTX_24_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hour%20n45-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-24hour-n45-4">
Past versions of "24hour n45-4.png"
</button>
</p>
<div id="fig-no-spaces-24hour-n45-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hour n45-4.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_24_top3_n45 &lt;- row.names(V.TRZ_24.top_n45[1:3,])

log_filt_hc_n45 %&gt;%  
  dplyr::filter(row.names(.) %in% TRZ_24_top3_n45) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/24hour%20n45-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-24hour-n45-5">
Past versions of "24hour n45-5.png"
</button>
</p>
<div id="fig-no-spaces-24hour-n45-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/88e30e524cf6113f735ea580047b0ae3a64f6b28/docs/figure/Peak_analysis.Rmd/24hour n45-5.png" target="_blank">88e30e5</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="volcano-plots-of-peaks-n45" class="section level3">
<h3>Volcano plots of peaks (n45)</h3>
<pre class="r"><code>library(cowplot)
efit2_n45 &lt;- readRDS(&quot;data/filt_Peaks_efit2_n45.RDS&quot;)


# plot_filenames &lt;- c(&quot;V.DNR_3.top&quot;,&quot;V.DOX_3.top&quot;,&quot;V.EPI_3.top&quot;,&quot;V.MTX_3.top&quot;,
#                     &quot;V.TRZ_.top&quot;,&quot;V.DNR_24.top&quot;,&quot;V.DOX_24.top&quot;,&quot;V.EPI_24.top&quot;,
#                     &quot;V.MTX_24.top&quot;,&quot;V.TRZ_24.top&quot;)
# plot_files &lt;- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
#                     V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
#                     V.MTX_24.top,V.TRZ_24.top)

volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
      # ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC &gt;= 0,&quot;B&quot;, &quot;C&quot;)))
    ##This is where I could add labels, but I have taken out
    # df &lt;- df %&gt;% mutate(genelabels = &quot;&quot;)
    # df$genelabels[1:topg] &lt;- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(adj.P.Val))) + 
    geom_point(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, &quot;npc&quot;)), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;adj. P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}
#v1&lt;- volcanosig(V.DA24.top, 0.01,0)
v1n &lt;- volcanosig(V.DNR_3.top_n45, 0.01)+ ggtitle(&quot;Daunorubicin 3 hour&quot;)+ylim(0,16)
v2n &lt;- volcanosig(V.DNR_24.top_n45, 0.01)+ ggtitle(&quot;Daunorubicin 24 hour&quot;)+ylab(&quot;&quot;)+ ylim(0,16)
v3n &lt;- volcanosig(V.DOX_3.top_n45, 0.01)+ ggtitle(&quot;Doxorubicin 3 hour&quot;)+ ylim(0,16)
v4n &lt;- volcanosig(V.DOX_24.top_n45, 0.01)+ ggtitle(&quot;Doxorubicin 24 hour&quot;)+ylab(&quot;&quot;)+ ylim(0,16)
v5n &lt;- volcanosig(V.EPI_3.top_n45, 0.01)+ ggtitle(&quot;Epirubicin 3 hour&quot;)+ ylim(0,16)
v6n &lt;- volcanosig(V.EPI_24.top_n45, 0.01)+ ggtitle(&quot;Epirubicin 24 hour&quot;)+ylab(&quot;&quot;)+ ylim(0,16)
v7n &lt;- volcanosig(V.MTX_3.top_n45, 0.01)+ ggtitle(&quot;Mitoxantrone 3 hour&quot;)+ ylim(0,16)
v8n &lt;- volcanosig(V.MTX_24.top_n45, 0.01)+ ggtitle(&quot;Mitoxantrone 24 hour&quot;)+ylab(&quot;&quot;)+ ylim(0,16)
v9n &lt;- volcanosig(V.TRZ_3.top_n45, 0.01)+ ggtitle(&quot;Trastuzumab 3 hour&quot;)+ ylim(0,16)
v10n &lt;- volcanosig(V.TRZ_24.top_n45, 0.01)+ ggtitle(&quot;Trastuzumab 24 hour&quot;)+ylab(&quot;&quot;)+ ylim(0,16)
# volcanoplot(efit2,coef = 10,style = &quot;p-value&quot;,
#                           highlight = 8,
#                           names = efit2$genes$SYMBOL,
#                           hl.col = &quot;red&quot;,xlab = &quot;Log2 Fold Change&quot;,
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = &quot;Using Trastuzumab 24 hour data and volcanoplot function&quot;)
plot_grid(v1n,v2n,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots%20n45-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-n45-1">
Past versions of "volcano plots n45-1.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-n45-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/volcano plots n45-1.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3n,v4n,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots%20n45-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-n45-2">
Past versions of "volcano plots n45-2.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-n45-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/volcano plots n45-2.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5n,v6n,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots%20n45-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-n45-3">
Past versions of "volcano plots n45-3.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-n45-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/volcano plots n45-3.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7n,v8n,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots%20n45-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-n45-4">
Past versions of "volcano plots n45-4.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-n45-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/volcano plots n45-4.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v9n,v10n,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/volcano%20plots%20n45-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-volcano-plots-n45-5">
Past versions of "volcano plots n45-5.png"
</button>
</p>
<div id="fig-no-spaces-volcano-plots-n45-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/volcano plots n45-5.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># plot_grid(v1n)</code></pre>
</div>
<div id="pca-analysis-without-4-and-5" class="section level3">
<h3>PCA analysis without 4 and 5</h3>
<pre class="r"><code>my_hc_filtered_counts_n45 &lt;- readRDS(&quot;data/my_hc_filt_counts_n45.RDS&quot;)
PCAmat_n45 &lt;- my_hc_filtered_counts_n45 %&gt;%
  cpm(., log = TRUE) %&gt;% 
  as.matrix()

annotation_mat_n45  &lt;-
  data.frame(timeset=colnames(PCAmat_n45 )) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(trt= case_match(trt, &#39;DX&#39; ~&#39;DOX&#39;, &#39;E&#39;~&#39;EPI&#39;, &#39;DA&#39;~&#39;DNR&#39;, &#39;M&#39;~&#39;MTX&#39;, &#39;T&#39;~&#39;TRZ&#39;, &#39;V&#39;~&#39;VEH&#39;,.default = trt)) %&gt;% 
  mutate(indv = factor(indv, levels = c(&quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;, &quot;6&quot;))) %&gt;%
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) 

PCA_info_n45  &lt;- (prcomp(t(PCAmat_n45 ), scale. = TRUE)) 
PCA_info_anno_n45  &lt;- PCA_info_n45$x %&gt;% cbind(.,annotation_mat_n45 )
# autoplot(PCA_info)
summary(PCA_info_n45)</code></pre>
<pre><code>Importance of components:
                            PC1      PC2       PC3      PC4      PC5      PC6
Standard deviation     184.6606 144.7343 111.55919 91.50947 73.90333 64.39062
Proportion of Variance   0.2246   0.1380   0.08199  0.05516  0.03598  0.02731
Cumulative Proportion    0.2246   0.3626   0.44462  0.49978  0.53576  0.56308
                            PC7      PC8      PC9     PC10     PC11     PC12
Standard deviation     61.44960 60.02665 54.45457 51.12195 49.40730 48.68554
Proportion of Variance  0.02488  0.02374  0.01953  0.01722  0.01608  0.01561
Cumulative Proportion   0.58795  0.61169  0.63122  0.64844  0.66452  0.68013
                           PC13     PC14     PC15     PC16     PC17     PC18
Standard deviation     47.98591 45.74783 45.23358 44.39518 43.94967 43.34120
Proportion of Variance  0.01517  0.01379  0.01348  0.01298  0.01272  0.01237
Cumulative Proportion   0.69530  0.70909  0.72257  0.73555  0.74828  0.76065
                           PC19     PC20     PC21     PC22     PC23     PC24
Standard deviation     42.90497 42.66800 41.57356 40.92157 40.77075 39.64620
Proportion of Variance  0.01213  0.01199  0.01139  0.01103  0.01095  0.01035
Cumulative Proportion   0.77278  0.78477  0.79616  0.80719  0.81814  0.82849
                           PC25     PC26     PC27     PC28     PC29    PC30
Standard deviation     39.38247 38.44709 38.02843 37.75943 37.10861 36.7643
Proportion of Variance  0.01022  0.00974  0.00953  0.00939  0.00907  0.0089
Cumulative Proportion   0.83871  0.84845  0.85797  0.86737  0.87644  0.8853
                           PC31     PC32     PC33     PC34     PC35     PC36
Standard deviation     36.07143 35.56752 35.43143 34.68374 34.53339 33.67581
Proportion of Variance  0.00857  0.00833  0.00827  0.00792  0.00786  0.00747
Cumulative Proportion   0.89391  0.90225  0.91052  0.91844  0.92630  0.93377
                          PC37     PC38     PC39     PC40     PC41     PC42
Standard deviation     33.2774 32.94509 32.89226 31.75694 30.54799 30.36175
Proportion of Variance  0.0073  0.00715  0.00713  0.00664  0.00615  0.00607
Cumulative Proportion   0.9411  0.94821  0.95534  0.96198  0.96813  0.97420
                           PC43     PC44     PC45     PC46     PC47      PC48
Standard deviation     30.00405 29.48075 28.18776 26.88753 25.07756 5.673e-13
Proportion of Variance  0.00593  0.00573  0.00523  0.00476  0.00414 0.000e+00
Cumulative Proportion   0.98014  0.98586  0.99109  0.99586  1.00000 1.000e+00</code></pre>
<pre class="r"><code># cpm(PCAmat, log=TRUE)
# pca_plot(PCA_info_n45 , col_var=&#39;trt&#39;, shape_var = &#39;time&#39;)
#         
# pca_plot(PCA_info_n45 , col_var=&#39;trt&#39;, shape_var = &#39;indv&#39;)


drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

PCA_info_anno_n45  %&gt;%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
  #scale_shape_manual(name = &quot;Time&quot;,values= c(&quot;3h&quot;=0,&quot;24h&quot;=1))+
  ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm)&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 2 (13.81%)&quot;, x =&quot;PC 1 (22.46%)&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20wo%204-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-wo-4-5-1">
Past versions of "filtering step wo 4-5-1.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-wo-4-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/filtering step wo 4-5-1.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>PCA_info_anno_n45  %&gt;%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
  #scale_shape_manual(name = &quot;Time&quot;,values= c(&quot;3h&quot;=0,&quot;24h&quot;=1))+
  ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(peaks)&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 4 (5.52%)&quot;, x =&quot;PC 3 (8.2%)&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/filtering%20step%20wo%204-5-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filtering-step-wo-4-5-2">
Past versions of "filtering step wo 4-5-2.png"
</button>
</p>
<div id="fig-no-spaces-filtering-step-wo-4-5-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/51d55e3180998ecab0df168ddc10509c6efcfff6/docs/figure/Peak_analysis.Rmd/filtering step wo 4-5-2.png" target="_blank">51d55e3</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pca-of-trt-time-indv-wo-4-and-5" class="section level3">
<h3>PCA of trt, time, indv w/o 4 and 5</h3>
<pre class="r"><code>facs &lt;- c(&quot;indv&quot;, &quot;trt&quot;, &quot;time&quot;)
names(facs) &lt;- c(&quot;Individual&quot;, &quot;Treatment&quot;, &quot;Time&quot;)
drug1 &lt;- c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;)##for changing shapes and colors
time &lt;- rep(c(&quot;24h&quot;, &quot;3h&quot;),24) %&gt;% factor(., levels = c(&quot;3h&quot;,&quot;24h&quot;))
##gglistmaking
for (f in facs) {
  # PC1 v PC2
  pca_plot(PCA_info_anno_n45, col_var = f, shape_var = time,
           title = names(facs)[which(facs == f)])
  print(last_plot())
  
  # Plot f versus PC1 and PC2
  f_v_pc1 &lt;- arrangeGrob(plot_versus_pc(PCA_info_anno_n45, 1, f))
  f_v_pc2 &lt;- arrangeGrob(plot_versus_pc(PCA_info_anno_n45, 2, f))
  grid.arrange(f_v_pc1, f_v_pc2, ncol = 2, top = names(facs)[which(facs == f)])
  # summary(plot_versus_pc(PCA_info_anno_all, 1, f))
  # summary(plot_versus_pc(PCA_info_anno_all, 2, f))
 
}</code></pre>
<p><img src="figure/Peak_analysis.Rmd/PCAtrt%20time%20indv%20n45-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAtrt-time-indv-n45-1">
Past versions of "PCAtrt time indv n45-1.png"
</button>
</p>
<div id="fig-no-spaces-PCAtrt-time-indv-n45-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAtrt time indv n45-1.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAtrt%20time%20indv%20n45-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAtrt-time-indv-n45-2">
Past versions of "PCAtrt time indv n45-2.png"
</button>
</p>
<div id="fig-no-spaces-PCAtrt-time-indv-n45-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAtrt time indv n45-2.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAtrt%20time%20indv%20n45-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAtrt-time-indv-n45-3">
Past versions of "PCAtrt time indv n45-3.png"
</button>
</p>
<div id="fig-no-spaces-PCAtrt-time-indv-n45-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAtrt time indv n45-3.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAtrt%20time%20indv%20n45-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAtrt-time-indv-n45-4">
Past versions of "PCAtrt time indv n45-4.png"
</button>
</p>
<div id="fig-no-spaces-PCAtrt-time-indv-n45-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAtrt time indv n45-4.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAtrt%20time%20indv%20n45-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAtrt-time-indv-n45-5">
Past versions of "PCAtrt time indv n45-5.png"
</button>
</p>
<div id="fig-no-spaces-PCAtrt-time-indv-n45-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAtrt time indv n45-5.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Peak_analysis.Rmd/PCAtrt%20time%20indv%20n45-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-PCAtrt-time-indv-n45-6">
Past versions of "PCAtrt time indv n45-6.png"
</button>
</p>
<div id="fig-no-spaces-PCAtrt-time-indv-n45-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/PCAtrt time indv n45-6.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="magnitude-of-response" class="section level3">
<h3>Magnitude of Response</h3>
<pre class="r"><code>toplist_n45 &lt;- readRDS(&quot;data/toplist_n45.RDS&quot;)
toplist_n45 %&gt;% 
  group_by(time, trt) %&gt;% 
  mutate(sigcount = if_else(adj.P.Val &lt; 0.05,&#39;sig&#39;,&#39;notsig&#39;))%&gt;%
  count(sigcount) %&gt;% 
  pivot_wider(id_cols = c(time,trt), names_from=sigcount, values_from=n) %&gt;% 
  mutate(prop = sig/(sig+notsig)*100) %&gt;% 
  mutate(prop=if_else(is.na(prop),0,prop)) %&gt;% 
  ggplot(., aes(x=trt, y= prop))+
  geom_col(aes(fill=trt))+
  geom_text(aes(label = sprintf(&quot;%.2f&quot;,prop)),
            position=position_dodge(0.9),vjust=-.2 )+
  scale_fill_manual(values =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  facet_wrap(~time)+#labeller = (time = facettimelabel) )+
  theme_bw()+
  xlab(&quot;&quot;)+
  ylab(&quot;Percentage DAR peaks&quot;)+
  theme_bw()+
  ggtitle(&quot;Percent DARs (adj. P value &lt;0.05)&quot;)+
  scale_y_continuous(expand=expansion(c(0.02,.2)))+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_text(size = 8, color = &quot;white&quot;, angle = 0),
        axis.text.y = element_text(size = 8, color = &quot;black&quot;, angle = 0),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/magresp_45-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-magresp_45-1">
Past versions of magresp_45-1.png
</button>
</p>
<div id="fig-magresp_45-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/magresp_45-1.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_n45 %&gt;% 
  group_by(time, trt) %&gt;% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = &quot;Treatment&quot;))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab(&quot;&quot;)+
  ylab(expression(&quot;Log&quot;[2]*&quot; fold change&quot;))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = &quot;transparent&quot;),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = &quot;black&quot;, face = &quot;bold&quot;))</code></pre>
<p><img src="figure/Peak_analysis.Rmd/magresp_45-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-magresp_45-2">
Past versions of magresp_45-2.png
</button>
</p>
<div id="fig-magresp_45-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/magresp_45-2.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_n45 %&gt;% 
  group_by(time, trt) %&gt;% 
  dplyr::filter(trt != &quot;TRZ&quot;) %&gt;%
  ggplot(., aes( x=(adj.P.Val)))+
 geom_histogram(aes(y=..density..), colour=&quot;black&quot;, fill=&quot;white&quot;)+
  geom_density(alpha=.2, fill=&quot;#FF6666&quot;)+
  theme_bw()+
  ggtitle(&quot;adj.P.Value density by time/trt&quot;)+
  facet_wrap(time~trt)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/magresp_45-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-magresp_45-3">
Past versions of magresp_45-3.png
</button>
</p>
<div id="fig-magresp_45-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/magresp_45-3.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>toplist_n45 %&gt;% 
  group_by(time, trt) %&gt;% 
  # dplyr::filter(trt == &quot;DOX&quot;) %&gt;% 
  ggplot(., aes( x=(P.Value)))+
  geom_histogram(aes(y=..density..), colour=&quot;black&quot;, fill=&quot;white&quot;)+
  geom_density(alpha=.2, fill=&quot;#FF6666&quot;)+
  theme_bw()+
  ggtitle(&quot;P.Value density by time/trt&quot;)+
  facet_wrap(time~trt)</code></pre>
<p><img src="figure/Peak_analysis.Rmd/magresp_45-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-magresp_45-4">
Past versions of magresp_45-4.png
</button>
</p>
<div id="fig-magresp_45-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/fb1abb772e1ab9c3ba4f7fb738498963ff1e7dfd/docs/figure/Peak_analysis.Rmd/magresp_45-4.png" target="_blank">fb1abb7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-03-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggfortify_0.4.16   edgeR_4.0.16       limma_3.58.1       gridExtra_2.3     
 [5] RColorBrewer_1.1-3 kableExtra_1.4.0   scales_1.3.0       ggpubr_0.6.0      
 [9] cowplot_1.1.3      ggsignif_0.6.4     lubridate_1.9.3    forcats_1.0.0     
[13] stringr_1.5.1      dplyr_1.1.4        purrr_1.0.2        readr_2.1.5       
[17] tidyr_1.3.1        tibble_3.2.1       ggplot2_3.5.0      tidyverse_2.0.0   
[21] workflowr_1.7.1   

loaded via a namespace (and not attached):
 [1] rlang_1.1.3           magrittr_2.0.3        clue_0.3-65          
 [4] GetoptLong_1.0.5      git2r_0.33.0          matrixStats_1.2.0    
 [7] compiler_4.3.1        getPass_0.2-4         png_0.1-8            
[10] systemfonts_1.0.6     callr_3.7.6           vctrs_0.6.5          
[13] pkgconfig_2.0.3       shape_1.4.6.1         crayon_1.5.2         
[16] fastmap_1.1.1         backports_1.4.1       magick_2.8.3         
[19] labeling_0.4.3        utf8_1.2.4            promises_1.2.1       
[22] rmarkdown_2.26        tzdb_0.4.0            ps_1.7.6             
[25] xfun_0.43             cachem_1.0.8          jsonlite_1.8.8       
[28] highr_0.10            later_1.3.2           broom_1.0.5          
[31] parallel_4.3.1        cluster_2.1.6         R6_2.5.1             
[34] bslib_0.7.0           stringi_1.8.3         car_3.1-2            
[37] jquerylib_0.1.4       Rcpp_1.0.12           iterators_1.0.14     
[40] knitr_1.45            IRanges_2.36.0        httpuv_1.6.15        
[43] timechange_0.3.0      tidyselect_1.2.1      rstudioapi_0.16.0    
[46] abind_1.4-5           yaml_2.3.8            doParallel_1.0.17    
[49] codetools_0.2-19      processx_3.8.4        lattice_0.22-6       
[52] withr_3.0.0           evaluate_0.23         xml2_1.3.6           
[55] circlize_0.4.16       pillar_1.9.0          carData_3.0-5        
[58] whisker_0.4.1         foreach_1.5.2         stats4_4.3.1         
[61] generics_0.1.3        rprojroot_2.0.4       hms_1.1.3            
[64] S4Vectors_0.40.2      munsell_0.5.0         glue_1.7.0           
[67] tools_4.3.1           locfit_1.5-9.9        fs_1.6.3             
[70] grid_4.3.1            colorspace_2.1-0      cli_3.6.2            
[73] fansi_1.0.6           viridisLite_0.4.2     svglite_2.1.3        
[76] ComplexHeatmap_2.18.0 gtable_0.3.4          rstatix_0.7.2        
[79] sass_0.4.9            digest_0.6.35         BiocGenerics_0.48.1  
[82] ggrepel_0.9.5         rjson_0.2.21          farver_2.1.1         
[85] htmltools_0.5.8       lifecycle_1.0.4       httr_1.4.7           
[88] GlobalOptions_0.1.2   statmod_1.5.0        </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
