<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-12-16" />

<title>Raodah’s data</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Raodah’s data</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-12-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree3267bc81a0bb95e671deccbe8665e487c7a874ebtargetblank3267bc8a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/3267bc81a0bb95e671deccbe8665e487c7a874eb" target="_blank">3267bc8</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree3267bc81a0bb95e671deccbe8665e487c7a874ebtargetblank3267bc8a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/3267bc81a0bb95e671deccbe8665e487c7a874eb" target="_blank">3267bc8</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  Correlationplot_scaled.pdf
    Untracked:  DOX_DAR_assess.Rmd
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/MRC_clusterlog2cpm.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  dataredo.RData
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  output_1_Mecom.txt
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/JustTE_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Raodah_mycount.Rmd</code>) and
HTML (<code>docs/Raodah_mycount.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/cfe4ce97125a9e1e6db6873c575ccf81eaca41ec/docs/Raodah_mycount.html" target="_blank">cfe4ce9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/a9d87d259b9a350df0a84416abb48a8be64b9723/analysis/Raodah_mycount.Rmd" target="_blank">a9d87d2</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-23
</td>
<td>
wflow_publish("analysis/Raodah_mycount.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/Raodah_mycount.html" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/628986a260599f273f245c4ea98f77e854245228/analysis/Raodah_mycount.Rmd" target="_blank">628986a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
<td>
pdf output
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/50f603e982b067deee87030676647f1b97853dde/docs/Raodah_mycount.html" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/a55255a95d9bf15ab3e985412724d1d0dba6bad0/analysis/Raodah_mycount.Rmd" target="_blank">a55255a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
adding in Raodah evaluation
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/82f20273302012c4ec5324fbc0d204d0778c854d/analysis/Raodah_mycount.Rmd" target="_blank">82f2027</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-16
</td>
<td>
updates to plots
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
# library(ChIPseeker)
# library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
# library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
# library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(eulerr)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(smplot2)
library(stringr)
library(cowplot)</code></pre>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#41B333&quot;)
pca_plot &lt;-
  function(df,
           col_var = NULL,
           shape_var = NULL,
           title = &quot;&quot;) {
    ggplot(df) + geom_point(aes_string(
      x = &quot;PC1&quot;,
      y = &quot;PC2&quot;,
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = &quot;PC 1&quot;, y = &quot;PC 2&quot;) +
      scale_color_manual(values = c(
        &quot;#8B006D&quot;,
        &quot;#DF707E&quot;,
        &quot;#F1B72B&quot;,
        &quot;#3386DD&quot;,
        
        &quot;#41B333&quot;
      ))
  }
pca_var_plot &lt;- function(pca) {
  # x: class == prcomp
  pca.var &lt;- pca$sdev ^ 2
  pca.prop &lt;- pca.var / sum(pca.var)
  var.plot &lt;-
    qplot(PC, prop, data = data.frame(PC = 1:length(pca.prop),
                                      prop = pca.prop)) +
    labs(title = &#39;Variance contributed by each PC&#39;,
         x = &#39;PC&#39;, y = &#39;Proportion of variance&#39;)
  plot(var.plot)
}

calc_pca &lt;- function(x) {
  # Performs principal components analysis with prcomp
  # x: a sample-by-gene numeric matrix
  prcomp(x, scale. = TRUE, retx = TRUE)
}

get_regr_pval &lt;- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == &quot;lm&quot;)
  fstat &lt;- summary(mod)$fstatistic
  pval &lt;- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc &lt;- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char &lt;- paste0(&quot;PC&quot;, pc_num)
  # Calculate F-statistic p-value for linear model
  pval &lt;- get_regr_pval(lm(df[, pc_char] ~ df[, fac]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = &quot;lm&quot;) + labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  }
}</code></pre>
<pre class="r"><code># saveRDS(cardiotox_highConf_3location, &quot;data/Final_four_data/Raodah_location_counts.RDS&quot;)
# saveRDS(H3K27ac_countstable,&quot;data/Final_four_data/Raodah_counts_fixed.RDS&quot; )
H3K27ac_countstable&lt;- readRDS(&quot;data/Final_four_data/Raodah_counts.RDS&quot;)
cardiotox_highConf_3location &lt;-  readRDS(&quot;data/Final_four_data/Raodah_location_counts.RDS&quot;)
# saveRDS(roadah_counts,  &quot;data/Final_four_data/New_raodah_counts.RDS&quot;)
raodah_counts &lt;- readRDS(&quot;data/Final_four_data/New_raodah_counts.RDS&quot;)
 
names(raodah_counts ) = gsub(pattern = &quot;_final.bam*&quot;, replacement = &quot;&quot;, x = names(raodah_counts ))
names(raodah_counts ) = gsub(pattern = &quot;^Raodah_data/77-1/bamFiles/*&quot;, replacement = &quot;&quot;, x = names(raodah_counts ))
names(raodah_counts ) = gsub(pattern = &quot;^Raodah_data/87-1/bamFiles/*&quot;, replacement = &quot;&quot;, x = names(raodah_counts ))
names(raodah_counts ) = gsub(pattern = &quot;^Raodah_data/71-1/bamFiles/*&quot;, replacement = &quot;&quot;, x = names(raodah_counts ))


names(raodah_counts ) = gsub(pattern = &quot;*87_1_*&quot;, replacement = &quot;A_&quot;, x = names(raodah_counts ))
 names(raodah_counts ) = gsub(pattern = &quot;77_1_*&quot;, replacement = &quot;B_&quot;, x = names(raodah_counts ))
 # names(roadah_counts) = gsub(pattern = &quot;*79-1 *&quot;, replacement = &quot;4_&quot;, x = names(roadah_counts))
 # names(roadah_counts) = gsub(pattern = &quot;*78-1 *&quot;, replacement = &quot;5_&quot;, x = names(roadah_counts))
 names(raodah_counts ) = gsub(pattern = &quot;*71_1_*&quot;, replacement = &quot;C_&quot;, x = names(raodah_counts ))
 names(raodah_counts ) = gsub(pattern = &quot;3$&quot;, replacement = &quot;_3&quot;, x = names(raodah_counts ))
 names(raodah_counts ) = gsub(pattern = &quot;24$&quot;, replacement = &quot;_24&quot;, x = names(raodah_counts ))
raodah_counts_list &lt;- raodah_counts[1:6]
raodah_counts_file &lt;- raodah_counts[,c(1,7:31)]
# saveRDS(raodah_counts_list, &quot;data/Final_four_data/All_Raodahpeaks.RDS&quot;)

# rtracklayer::export.bed(raodah_counts_list,con=&quot;data/Final_four_data/Raodah_ac_peaks.bed&quot;,format=&quot;bed&quot;)</code></pre>
<pre class="r"><code>PCA_H3_mat &lt;- raodah_counts_file%&gt;% column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

annotation_H3_mat &lt;-  data.frame(timeset=colnames(PCA_H3_mat)) %&gt;% 
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;VEH&quot;))) 

PCA_H3_info &lt;- (prcomp(t(PCA_H3_mat), scale. = TRUE)) 
PCA_H3_info_anno &lt;- PCA_H3_info$x %&gt;% cbind(.,annotation_H3_mat)
# autoplot(PCA_info)
summary(PCA_H3_info)</code></pre>
<pre><code>Importance of components:
                            PC1      PC2      PC3      PC4      PC5      PC6
Standard deviation     118.6544 38.53744 32.93705 23.03431 19.97846 17.65678
Proportion of Variance   0.6992  0.07375  0.05387  0.02635  0.01982  0.01548
Cumulative Proportion    0.6992  0.77291  0.82678  0.85313  0.87295  0.88843
                            PC7      PC8      PC9     PC10     PC11     PC12
Standard deviation     16.24997 15.43698 15.28003 13.60632 12.62621 12.05130
Proportion of Variance  0.01311  0.01183  0.01159  0.00919  0.00792  0.00721
Cumulative Proportion   0.90154  0.91338  0.92497  0.93417  0.94208  0.94929
                           PC13     PC14     PC15     PC16     PC17    PC18
Standard deviation     11.83553 11.20249 10.86818 10.31780 10.14317 9.56714
Proportion of Variance  0.00696  0.00623  0.00587  0.00529  0.00511 0.00455
Cumulative Proportion   0.95625  0.96248  0.96835  0.97364  0.97874 0.98329
                          PC19    PC20    PC21    PC22    PC23    PC24
Standard deviation     8.94188 7.84366 7.63001 7.26326 6.59195 6.37024
Proportion of Variance 0.00397 0.00306 0.00289 0.00262 0.00216 0.00202
Cumulative Proportion  0.98726 0.99032 0.99321 0.99583 0.99798 1.00000
                            PC25
Standard deviation     5.364e-14
Proportion of Variance 0.000e+00
Cumulative Proportion  1.000e+00</code></pre>
<pre class="r"><code># %&gt;% as.tibble() %&gt;% 
#   kable(., caption = &quot;Summary of PCA variables&quot;) %&gt;% 
#   kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
#   kable_styling(full_width = FALSE, font_size = 16) %&gt;%
#   scroll_box(height = &quot;500px&quot;)
# cpm(PCAmat, log=TRUE)
pca_plot(PCA_H3_info_anno, col_var=&#39;trt&#39;, shape_var = &#39;time&#39;)+ggtitle(&quot;PCA of raw counts by time and treatment&quot;)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/examine%20inital%20data-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-examine-inital-data-1">
Past versions of "examine inital data-1.png"
</button>
</p>
<div id="fig-no-spaces-examine-inital-data-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/examine inital data-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/examine inital data-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># pca_plot(PCA_H3_info_anno, col_var=&#39;trt&#39;, shape_var = &#39;indv&#39;)+ggtitle(&quot;PCA of raw counts by treatment and individual&quot;)</code></pre>
<pre class="r"><code># PCA_H3_mat_t &lt;- H3K27ac_countstable %&gt;% 
#   dplyr::select(&#39;2_DNR_3&#39;:&#39;3_VEH_24&#39;,&#39;6_DNR_3&#39;:&#39;6_VEH_24&#39;)

anno_H3_mat &lt;-  data.frame(timeset=colnames(PCA_H3_mat)) %&gt;%
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;)))
  
lcpm_h3 &lt;- cpm(PCA_H3_mat, log=TRUE)  ### for determining the basic cutoffs
dim(lcpm_h3)</code></pre>
<pre><code>[1] 20137    25</code></pre>
<pre class="r"><code>row_means &lt;- rowMeans(lcpm_h3)
lcpm_h3_filtered &lt;- PCA_H3_mat[row_means &gt;0,]
dim(lcpm_h3_filtered)</code></pre>
<pre><code>[1] 20137    25</code></pre>
<pre class="r"><code>filt_H3_matrix_lcpm &lt;- cpm(lcpm_h3_filtered, log=TRUE)

PCA_H3_info_filter &lt;- (prcomp(t(filt_H3_matrix_lcpm), scale. = TRUE))
summary(PCA_H3_info_filter)</code></pre>
<pre><code>Importance of components:
                           PC1     PC2     PC3      PC4      PC5     PC6
Standard deviation     56.8156 48.4324 45.6952 36.87354 34.89198 33.2180
Proportion of Variance  0.1603  0.1165  0.1037  0.06752  0.06046  0.0548
Cumulative Proportion   0.1603  0.2768  0.3805  0.44800  0.50846  0.5633
                            PC7      PC8      PC9     PC10     PC11     PC12
Standard deviation     31.95473 28.59825 27.71405 24.76717 24.16232 24.07542
Proportion of Variance  0.05071  0.04061  0.03814  0.03046  0.02899  0.02878
Cumulative Proportion   0.61397  0.65458  0.69272  0.72318  0.75218  0.78096
                           PC13     PC14     PC15     PC16     PC17     PC18
Standard deviation     23.56663 22.47242 21.30002 20.85296 20.40028 19.42485
Proportion of Variance  0.02758  0.02508  0.02253  0.02159  0.02067  0.01874
Cumulative Proportion   0.80854  0.83362  0.85615  0.87774  0.89841  0.91715
                           PC19     PC20     PC21     PC22     PC23     PC24
Standard deviation     18.81143 18.30230 16.42981 16.14845 15.23137 14.72480
Proportion of Variance  0.01757  0.01663  0.01341  0.01295  0.01152  0.01077
Cumulative Proportion   0.93472  0.95136  0.96476  0.97771  0.98923  1.00000
                            PC25
Standard deviation     1.529e-13
Proportion of Variance 0.000e+00
Cumulative Proportion  1.000e+00</code></pre>
<pre class="r"><code>pca_var_plot(PCA_H3_info_filter)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/filter%20and%20do%20again-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filter-and-do-again-1">
Past versions of "filter and do again-1.png"
</button>
</p>
<div id="fig-no-spaces-filter-and-do-again-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/filter and do again-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/filter and do again-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_H3 &lt;- calc_pca(t(filt_H3_matrix_lcpm))
pca_H3_anno &lt;- data.frame(anno_H3_mat, pca_H3$x)


pca_H3_anno %&gt;%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm) Raodah data, same three individuals&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 2 11.65%&quot;, x =&quot;PC 1 16.3%&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/filter%20and%20do%20again-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filter-and-do-again-2">
Past versions of "filter and do again-2.png"
</button>
</p>
<div id="fig-no-spaces-filter-and-do-again-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cfe4ce97125a9e1e6db6873c575ccf81eaca41ec/docs/figure/Raodah_mycount.Rmd/filter and do again-2.png" target="_blank">cfe4ce9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/filter and do again-2.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/filter and do again-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_H3_anno %&gt;%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm)  Raodah data, same three individuals&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 4 6.7% &quot;, x =&quot;PC 3 10.4%&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/filter%20and%20do%20again-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-filter-and-do-again-3">
Past versions of "filter and do again-3.png"
</button>
</p>
<div id="fig-no-spaces-filter-and-do-again-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cfe4ce97125a9e1e6db6873c575ccf81eaca41ec/docs/figure/Raodah_mycount.Rmd/filter and do again-3.png" target="_blank">cfe4ce9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/filter and do again-3.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/filter and do again-3.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="diff-analysis-3-individual-and-5-individuals"
class="section level3">
<h3>Diff analysis, 3 individual and 5 individuals</h3>
<p>3 is 2,3&amp; 6 5 is 1 ,2, 3, 4, 5</p>
<pre class="r"><code>###group for 3 individuals
group &lt;- c( 1,2,4,5,6,7,9,10,1,2,3,4,7,8,9,10,1,2,3,5,6,7,8,9,10)

##group for 5 individuals
# group &lt;- c(1,2,3,4,7,8,9,10,1,2,3,5,6,8,9,10, 1,2,3,4,5,6,7,8,9,10,1,2,3,4,5,6,7,8,9,10,1,2,4,5,6,7,8,9,10)
# group &lt;- factor(group, levels =c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;,&quot;7&quot;,&quot;8&quot;,&quot;9&quot;,&quot;10&quot;))
# short_names &lt;- paste0(pca_final_four_anno$indv,&quot;_&quot;,pca_final_four_anno$trt,&quot;_&quot;,pca_final_four_anno$time)

for_group &lt;- data.frame(timeset=colnames(PCA_H3_mat)) %&gt;% 
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  unite(&quot;test&quot;, trt:time,sep=&quot;_&quot;, remove = FALSE)


dge &lt;- DGEList.data.frame(counts = PCA_H3_mat, group = group, genes = lcpm_h3_filtered)

group_1 &lt;- for_group$test
# efit_raodah1 &lt;- readRDS(&quot;data/Final_four_data/efit4_raodah1.RDS&quot;)
# efit_raodah_shared &lt;- readRDS(&quot;data/Final_four_data/efit4_raodah_shared.RDS&quot;)
dge$group$indv &lt;- for_group$indv
dge$group$time &lt;- for_group$time
dge$group$trt &lt;- for_group$trt
# 
indv &lt;- for_group$indv
# efit4 &lt;- readRDS(&quot;data/Final_four_data/efit4_filt_bl.RDS&quot;)
mm &lt;- model.matrix(~0 +group_1)
colnames(mm) &lt;-  c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
# 
# y &lt;- voom(dge$counts, mm,plot =TRUE)
# 
# corfit &lt;- duplicateCorrelation(y, mm, block = indv)
# 
# v &lt;- voom(dge$counts, mm, block = indv, correlation = corfit$consensus)
# 
# fit &lt;- lmFit(v, mm, block = indv, correlation = corfit$consensus)
# # colnames(mm) &lt;- c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
# #
# #
# cm &lt;- makeContrasts(
#   DNR_3.VEH_3 = DNR_3-VEH_3,
#   DOX_3.VEH_3 = DOX_3-VEH_3,
#   EPI_3.VEH_3 = EPI_3-VEH_3,
#   MTX_3.VEH_3 = MTX_3-VEH_3,
#   DNR_24.VEH_24 =DNR_24-VEH_24,
#   DOX_24.VEH_24= DOX_24-VEH_24,
#   EPI_24.VEH_24= EPI_24-VEH_24,
#   MTX_24.VEH_24= MTX_24-VEH_24,
#     levels = mm)
# # 
# vfit &lt;- lmFit(y, mm)
# # 
# vfit&lt;- contrasts.fit(vfit, contrasts=cm)
# 
# efit_raodah_new &lt;- eBayes(vfit)

# saveRDS(efit_raodah_new,&quot;data/Final_four_data/efit4_raodah_new.RDS&quot;)
efit_raodah_new &lt;- readRDS(&quot;data/Final_four_data/efit4_raodah_new.RDS&quot;)
results = decideTests(efit_raodah_new)
summary(results)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 DNR_24.VEH_24
Down           186           0           0           0           671
NotSig       19895       20137       20137       20137         18115
Up              56           0           0           0          1351
       DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24
Down              13             8             0
NotSig         19900         20041         20137
Up               224            88             0</code></pre>
</div>
<div id="removing-low-peak-number-samples" class="section level3">
<h3>removing low peak number samples:</h3>
<p>Veh-3hr-71 (C) and Veh-24hr-77 (B)</p>
<pre class="r"><code>PCA_H3_mat_23s &lt;- raodah_counts_file %&gt;% 
  dplyr::select(Geneid,B_DNR_24:B_MTX_24, B_VEH_3:C_VEH_24) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

anno_H3_mat_23s &lt;-
data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;%
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;)))%&gt;% 
  # dplyr::filter(sample != &quot;C_VEH_3&quot;) %&gt;% 
  dplyr::filter(sample != &quot;B_VEH_24&quot;)

  
lcpm_h3_23s &lt;- cpm(PCA_H3_mat_23s, log=TRUE)  ### for determining the basic cutoffs
# dim(lcpm_h3_23s)

row_means23s &lt;- rowMeans(lcpm_h3_23s)
lcpm_h3_23s_filtered &lt;- PCA_H3_mat_23s[row_means23s &gt;0,]
dim(lcpm_h3_23s_filtered)</code></pre>
<pre><code>[1] 20137    23</code></pre>
<pre class="r"><code>filt_H3_matrix_lcpm_23s &lt;- cpm(lcpm_h3_23s_filtered, log=TRUE)

PCA_H3_info_filter_23s &lt;- (prcomp(t(filt_H3_matrix_lcpm_23s), scale. = TRUE))
summary(PCA_H3_info_filter_23s)</code></pre>
<pre><code>Importance of components:
                           PC1     PC2      PC3     PC4      PC5      PC6
Standard deviation     52.2001 50.2346 44.78209 40.1125 36.76144 31.78530
Proportion of Variance  0.1353  0.1253  0.09959  0.0799  0.06711  0.05017
Cumulative Proportion   0.1353  0.2606  0.36022  0.4401  0.50724  0.55741
                            PC7      PC8      PC9     PC10     PC11     PC12
Standard deviation     31.62265 29.25073 27.61829 26.82506 26.35383 25.46783
Proportion of Variance  0.04966  0.04249  0.03788  0.03573  0.03449  0.03221
Cumulative Proportion   0.60707  0.64956  0.68744  0.72317  0.75766  0.78987
                           PC13    PC14     PC15     PC16     PC17     PC18
Standard deviation     23.81706 23.4901 23.20255 21.85550 21.05976 20.47942
Proportion of Variance  0.02817  0.0274  0.02673  0.02372  0.02202  0.02083
Cumulative Proportion   0.81804  0.8454  0.87218  0.89590  0.91792  0.93875
                           PC19     PC20     PC21     PC22     PC23
Standard deviation     18.49516 18.09649 17.04140 16.53649 1.27e-13
Proportion of Variance  0.01699  0.01626  0.01442  0.01358 0.00e+00
Cumulative Proportion   0.95574  0.97200  0.98642  1.00000 1.00e+00</code></pre>
<pre class="r"><code>pca_var_plot(PCA_H3_info_filter_23s)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/removing%20low%20reads-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-removing-low-reads-1">
Past versions of "removing low reads-1.png"
</button>
</p>
<div id="fig-no-spaces-removing-low-reads-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/removing low reads-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/removing low reads-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_H3_23s &lt;- calc_pca(t(filt_H3_matrix_lcpm_23s))
pca_H3_anno_23s &lt;- data.frame(anno_H3_mat_23s, pca_H3_23s$x)


pca_H3_anno_23s %&gt;%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm) Raodah data, same three individuals without low peak samples&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 2 12.5%&quot;, x =&quot;PC 1 13.5%&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;)) </code></pre>
<p><img src="figure/Raodah_mycount.Rmd/removing%20low%20reads-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-removing-low-reads-2">
Past versions of "removing low reads-2.png"
</button>
</p>
<div id="fig-no-spaces-removing-low-reads-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cfe4ce97125a9e1e6db6873c575ccf81eaca41ec/docs/figure/Raodah_mycount.Rmd/removing low reads-2.png" target="_blank">cfe4ce9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/removing low reads-2.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/removing low reads-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pca_H3_anno_23s %&gt;%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression(&quot;PCA of log&quot;[2]*&quot;(cpm)  Raodah data, same three individuals without low peak samples&quot;))+
  theme_bw()+
  guides(col=&quot;none&quot;, size =4)+
  labs(y = &quot;PC 4 7.5 % &quot;, x =&quot;PC 3 10.%&quot;)+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = &quot;black&quot;))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/removing%20low%20reads-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-removing-low-reads-3">
Past versions of "removing low reads-3.png"
</button>
</p>
<div id="fig-no-spaces-removing-low-reads-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cfe4ce97125a9e1e6db6873c575ccf81eaca41ec/docs/figure/Raodah_mycount.Rmd/removing low reads-3.png" target="_blank">cfe4ce9</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/removing low reads-3.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/removing low reads-3.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>### group for -veh3hr and 24hr individual samples
# group_23s &lt;- c( 1,2,4,5,6,7,10,1,2,3,4,7,8,9,10,1,2,3,5,6,7,8,9)
### group for -24hr individual sample
# group_24s &lt;- c( 1,2,4,5,6,7,9,10,1,2,3,4,7,8,9,10,1,2,3,5,6,7,8,9)
# for_group_23s &lt;- data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;% 
#   mutate(sample = timeset) %&gt;% 
#   separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
#   unite(&quot;test&quot;, trt:time,sep=&quot;_&quot;, remove = FALSE)

# dge_23s &lt;- DGEList.data.frame(counts = PCA_H3_mat_23s, group = group_23s, genes = lcpm_h3_23s_filtered)

# group_1 &lt;- for_group_23s$test
# 
# dge_23s$group$indv &lt;- for_group_23s$indv
# dge_23s$group$time &lt;- for_group_23s$time
# dge_23s$group$trt &lt;- for_group_23s$trt
# # 
# indv_23s &lt;- for_group_23s$indv
# mm &lt;- model.matrix(~0 +group_1)
# colnames(mm) &lt;-  c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
# 
# y &lt;- voom(dge_23s$counts, mm,plot =TRUE)
# 
# corfit &lt;- duplicateCorrelation(y, mm, block = indv_23s)
# 
# v &lt;- voom(dge_23s$counts, mm, block = indv_23s, correlation = corfit$consensus)
# 
# fit &lt;- lmFit(v, mm, block = indv_23s, correlation = corfit$consensus)
# # colnames(mm) &lt;- c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
# #
# #
# cm &lt;- makeContrasts(
#   DNR_3.VEH_3 = DNR_3-VEH_3,
#   DOX_3.VEH_3 = DOX_3-VEH_3,
#   EPI_3.VEH_3 = EPI_3-VEH_3,
#   MTX_3.VEH_3 = MTX_3-VEH_3,
#   DNR_24.VEH_24 =DNR_24-VEH_24,
#   DOX_24.VEH_24= DOX_24-VEH_24,
#   EPI_24.VEH_24= EPI_24-VEH_24,
#   MTX_24.VEH_24= MTX_24-VEH_24,
#     levels = mm)
#
# vfit &lt;- lmFit(y, mm)
# # 
# vfit&lt;- contrasts.fit(vfit, contrasts=cm)
# 
# efit_raodah_24s &lt;- eBayes(vfit)

# saveRDS(efit_raodah_24s,&quot;data/Final_four_data/efit4_raodah_24s.RDS&quot;)
efit_raodah_23s &lt;- readRDS(&quot;data/Final_four_data/efit4_raodah_23s.RDS&quot;)
# efit_raodah_24s &lt;- readRDS(&quot;data/Final_four_data/efit4_raodah_24s.RDS&quot;)
results = decideTests(efit_raodah_23s)

summary(results)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 DNR_24.VEH_24
Down           424           2           0           0          2007
NotSig       19502       20135       20137       20137         16669
Up             211           0           0           0          1461
       DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24
Down             126            55             0
NotSig         19756         19997         20137
Up               255            85             0</code></pre>
<pre class="r"><code>Frag_cor &lt;- raodah_counts_file %&gt;% 
  dplyr::select(Geneid,B_DNR_24:B_MTX_24, B_VEH_3:C_VEH_24) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
  
filmat_groupmat_col &lt;- data.frame(timeset = colnames(Frag_cor))

counts_corr_mat &lt;-  filmat_groupmat_col %&gt;% 
 # mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
  ))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) %&gt;% 
  mutate(indv=factor(indv, levels = c(&quot;A&quot;,&quot;B&quot;,&quot;C&quot;))) 
                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#41B333&quot;),
   # indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;, &quot;#E7298A&quot; ,&quot;#66A61E&quot;, &quot;#E6AB02&quot;),
  indv=c(A=&quot;#1B9E77&quot;,B= &quot;#D95F02&quot; ,C=&quot;#7570B3&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)

htanno &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat, col = mat_colors)
ComplexHeatmap::Heatmap(Frag_cor, top_annotation = htanno)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/correlation%2024%20samples-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-correlation-24-samples-1">
Past versions of "correlation 24 samples-1.png"
</button>
</p>
<div id="fig-no-spaces-correlation-24-samples-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/correlation 24 samples-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Frag_cor_all &lt;- raodah_counts_file %&gt;% 
  # dplyr::select(Geneid,B_DNR_24:B_MTX_24, B_VEH_3:C_VEH_3) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(., log = TRUE) %&gt;% 
  cor()
  
filmat_groupmat_col_all &lt;- data.frame(timeset = colnames(Frag_cor_all))

counts_corr_mat_all &lt;-  filmat_groupmat_col_all %&gt;% 
 # mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
  ))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) %&gt;% 
  mutate(indv=factor(indv, levels = c(&quot;A&quot;,&quot;B&quot;,&quot;C&quot;))) 
                         

htanno &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat_all, col = mat_colors)
ComplexHeatmap::Heatmap(Frag_cor_all, top_annotation = htanno)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/correlation%2024%20samples-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-correlation-24-samples-2">
Past versions of "correlation 24 samples-2.png"
</button>
</p>
<div id="fig-no-spaces-correlation-24-samples-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/correlation 24 samples-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FCmatrix_23s &lt;- subset(efit_raodah_23s$coefficients)

colnames(FCmatrix_23s) &lt;-
  c(&quot;DNR\n3h&quot;,
    &quot;DOX\n3h&quot;,
    &quot;EPI\n3h&quot;,
    &quot;MTX\n3h&quot;,
    &quot;DNR\n24h&quot;,
    &quot;DOX\n24h&quot;,
    &quot;EPI\n24h&quot;,
    &quot;MTX\n24h&quot;
     )


mat_col_FC &lt;-
  data.frame(
    time = c(rep(&quot;3 hours&quot;, 4), rep(&quot;24 hours&quot;, 4)),
    class = (c(
      &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;
    )))
rownames(mat_col_FC) &lt;- colnames(FCmatrix_23s)

mat_colors_FC &lt;-
  list(
    time = c(&quot;pink&quot;, &quot;chocolate4&quot;),
    class = c(&quot;yellow1&quot;, &quot;lightgreen&quot;))

names(mat_colors_FC$time) &lt;- unique(mat_col_FC$time)
names(mat_colors_FC$class) &lt;- unique(mat_col_FC$class)
# names(mat_colors_FC$TOP2i) &lt;- unique(mat_col_FC$TOP2i)
corrFC_23s &lt;- cor(FCmatrix_23s)

htanno_23s &lt;-  HeatmapAnnotation(df = mat_col_FC, col = mat_colors_FC)
Heatmap(corrFC_23s, top_annotation = htanno_23s)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/log%20fold%20change%20correlation-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-log-fold-change-correlation-1">
Past versions of "log fold change correlation-1.png"
</button>
</p>
<div id="fig-no-spaces-log-fold-change-correlation-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/log fold change correlation-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>AC.DNR_3.top= topTable(efit_raodah_23s, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.DOX_3.top= topTable(efit_raodah_23s, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.EPI_3.top= topTable(efit_raodah_23s, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.MTX_3.top= topTable(efit_raodah_23s, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)

AC.DNR_24.top= topTable(efit_raodah_23s, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.DOX_24.top= topTable(efit_raodah_23s, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.EPI_24.top= topTable(efit_raodah_23s, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.MTX_24.top= topTable(efit_raodah_23s, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)

# volcanoplot(efit2,coef = c(1,2,3,4,5,6),style = &quot;p-value&quot;,
#                           # highlight = 8,
#                           # names = efit2$genes$SYMBOL,
#                           hl.col = &quot;red&quot;,xlab = &quot;Log2 Fold Change&quot;,
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = &quot;test&quot;)
# 

plot_filenames &lt;- c(&quot;AC.DNR_3.top&quot;,&quot;AC.DOX_3.top&quot;,&quot;AC.EPI_3.top&quot;,&quot;AC.MTX_3.top&quot;,
                    &quot;AC.DNR_24.top&quot;,&quot;AC.DOX_24.top&quot;,&quot;AC.EPI_24.top&quot;,
                    &quot;AC.MTX_24.top&quot;)
plot_files &lt;- c( AC.DNR_3.top,AC.DOX_3.top,AC.EPI_3.top,AC.MTX_3.top,
                    AC.DNR_24.top,AC.DOX_24.top,AC.EPI_24.top,
                    AC.MTX_24.top)

volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
      # ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC &gt;= 0,&quot;B&quot;, &quot;C&quot;)))
    ##This is where I could add labels, but I have taken out
    # df &lt;- df %&gt;% mutate(genelabels = &quot;&quot;)
    # df$genelabels[1:topg] &lt;- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    geom_point(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, &quot;npc&quot;)), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}
#v1&lt;- volcanosig(V.DA24.top, 0.01,0)
v1 &lt;- volcanosig(AC.DNR_3.top, 0.01)+ ggtitle(&quot;H3K27ac DNR 3 hour&quot;)
v2 &lt;- volcanosig(AC.DNR_24.top, 0.01)+ ggtitle(&quot;H3K27ac DNR 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(AC.DOX_3.top, 0.01)+ ggtitle(&quot;H3K27ac DOX 3 hour&quot;)
v4 &lt;- volcanosig(AC.DOX_24.top, 0.01)+ ggtitle(&quot;H3K27ac DOX 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(AC.EPI_3.top, 0.01)+ ggtitle(&quot;H3K27ac EPI 3 hour&quot;)
v6 &lt;- volcanosig(AC.EPI_24.top, 0.01)+ ggtitle(&quot;H3K27ac EPI 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(AC.MTX_3.top, 0.01)+ ggtitle(&quot;H3K27ac MTX 3 hour&quot;)
v8 &lt;- volcanosig(AC.MTX_24.top, 0.01)+ ggtitle(&quot;H3K27ac MTX 24 hour&quot;)+ylab(&quot;&quot;)
# v9 &lt;- volcanosig(V.TRZ_3.top, 0.01)+ ggtitle(&quot;Trastuzumab 3 hour&quot;)
# v10 &lt;- volcanosig(V.TRZ_24.top, 0.01)+ ggtitle(&quot;Trastuzumab 24 hour&quot;)+ylab(&quot;&quot;)
# volcanoplot(efit2,coef = 10,style = &quot;p-value&quot;,
#                           highlight = 8,
#                           names = efit2$genes$SYMBOL,
#                           hl.col = &quot;red&quot;,xlab = &quot;Log2 Fold Change&quot;,
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = &quot;Using Trastuzumab 24 hour data and volcanoplot function&quot;)
plot_grid(v1,v2,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/Volcano-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Volcano-1">
Past versions of Volcano-1.png
</button>
</p>
<div id="fig-Volcano-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/Volcano-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3,v4,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/Volcano-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Volcano-2">
Past versions of Volcano-2.png
</button>
</p>
<div id="fig-Volcano-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/Volcano-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5,v6,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/Volcano-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Volcano-3">
Past versions of Volcano-3.png
</button>
</p>
<div id="fig-Volcano-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/Volcano-3.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7,v8,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/Volcano-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Volcano-4">
Past versions of Volcano-4.png
</button>
</p>
<div id="fig-Volcano-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/Volcano-4.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># plot_grid(v9,v10,  rel_widths =c(.8,1))</code></pre>
<pre class="r"><code># AC.DNR_3.top= topTable(efit_raodah_23s, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# AC.DOX_3.top= topTable(efit_raodah_23s, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# AC.EPI_3.top= topTable(efit_raodah_23s, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# AC.MTX_3.top= topTable(efit_raodah_23s, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# 
# AC.DNR_24.top= topTable(efit_raodah_23s, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# AC.DOX_24.top= topTable(efit_raodah_23s, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# AC.EPI_24.top= topTable(efit_raodah_23s, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# AC.MTX_24.top= topTable(efit_raodah_23s, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
# toplist_ac_23  &lt;- list(AC.DNR_3.top,  AC.DOX_3.top, AC.EPI_3.top, AC.MTX_3.top, AC.DNR_24.top, AC.DOX_24.top, AC.EPI_24.top, AC.MTX_24.top)
# names(toplist_ac_23 ) &lt;- c(&quot;DNR_3&quot;, &quot;DOX_3&quot;,&quot;EPI_3&quot;,&quot;MTX_3&quot;,&quot;DNR_24&quot;, &quot;DOX_24&quot;,&quot;EPI_24&quot;,&quot;MTX_24&quot;)
# toplist_ac_lfc &lt;-map_df(toplist_ac_23, ~as.data.frame(.x), .id=&quot;trt_time&quot;)
# # # #
# toplist_ac_lfc &lt;- toplist_ac_lfc %&gt;%
#   # rownames_to_column(&quot;Peakid&quot;)%&gt;%
  # mutate(Peakid=(gsub(&quot;\\.\\.\\.{5}.$&quot;,&quot;&quot;,Peakid)))
#   separate(trt_time, into= c(&quot;trt&quot;,&quot;time&quot;), sep = &quot;_&quot;) %&gt;%
#   mutate(trt=factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;))) %&gt;%
#   mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels = c(&quot;3 hours&quot;, &quot;24 hours&quot;)))
# saveRDS(toplist_ac_lfc,&quot;data/Final_four_data/toplist_ac_lfc.RDS&quot;)
toplist_ac_lfc &lt;- readRDS(&quot;data/Final_four_data/toplist_ac_lfc.RDS&quot;) #%&gt;% 
  # rownames_to_column(&quot;Peakid&quot;) %&gt;% 
  # mutate(Peakid=(gsub(&quot;\\.\\.\\.*&quot;,&quot;&quot;,Peakid)))</code></pre>
</div>
<div id="making-of-median-dataframe-code-only" class="section level3">
<h3>making of median dataframe, code only</h3>
<pre class="r"><code>### making median lfc
library(stringr)
# 
AC_median_df &lt;- toplist_ac_lfc %&gt;%
    dplyr::select(Peakid:logFC)%&gt;% 
   mutate(Peakid=str_replace(Peakid, &quot;\\.\\.\\.[0=9]*&quot;,&quot;/&quot;)) %&gt;% separate(Peakid, into=c(&quot;Peakid&quot;,NA),sep=&quot;/&quot;) %&gt;% 
  pivot_wider(., id_cols=c(time,Peakid), names_from = &quot;trt&quot;, values_from = &quot;logFC&quot;) %&gt;% 
  rowwise() %&gt;% 
  mutate(median_lfc= median(c_across(DNR:MTX))) %&gt;% 
  ungroup()

AC_median_3_lfc &lt;-  AC_median_df %&gt;%
    dplyr::filter(time == &quot;3&quot;) %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(time, Peakid,median_lfc) %&gt;% 
  dplyr::rename(&quot;AC_3h_lfc&quot;=median_lfc)
  

AC_median_24_lfc &lt;- AC_median_df %&gt;%
    dplyr::filter(time == &quot;24&quot;) %&gt;% 
  ungroup() %&gt;% 
  dplyr::select(time, Peakid,median_lfc) %&gt;% 
  dplyr::rename(&quot;AC_24h_lfc&quot;=median_lfc)
  
write_csv(AC_median_3_lfc, &quot;data/Final_four_data/AC_median_3_lfc.csv&quot;)
write_csv(AC_median_24_lfc, &quot;data/Final_four_data/AC_median_24_lfc.csv&quot;)</code></pre>
</div>
<div id="examples-of-dars" class="section level3">
<h3>Examples of DARs</h3>
<pre class="r"><code>DNR_3_top3 &lt;- row.names(AC.DNR_3.top[1:3,])

log_filt_23s &lt;- lcpm_h3_23s_filtered %&gt;% 
  cpm(., log=TRUE) %&gt;% as.data.frame()

row.names(log_filt_23s) &lt;- row.names(lcpm_h3_23s_filtered)
log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%203hr%20expression-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-expression-1">
Past versions of "example 3hr expression-1.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-expression-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 3hr expression-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 3hr expression-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_3_top3 &lt;- row.names(AC.DOX_3.top[1:3,])

log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%203hr%20expression-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-expression-2">
Past versions of "example 3hr expression-2.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-expression-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 3hr expression-2.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 3hr expression-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_3_top3 &lt;- row.names(AC.EPI_3.top[1:3,])

log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%203hr%20expression-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-expression-3">
Past versions of "example 3hr expression-3.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-expression-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 3hr expression-3.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 3hr expression-3.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_3_top3 &lt;- row.names(AC.MTX_3.top[1:3,])

log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_3_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%203hr%20expression-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-3hr-expression-4">
Past versions of "example 3hr expression-4.png"
</button>
</p>
<div id="fig-no-spaces-example-3hr-expression-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 3hr expression-4.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 3hr expression-4.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DNR_24_top3 &lt;- row.names(AC.DNR_24.top[1:3,])

log_filt_23s &lt;- lcpm_h3_23s_filtered %&gt;% 
  cpm(., log=TRUE) %&gt;% as.data.frame()

row.names(log_filt_23s) &lt;- row.names(lcpm_h3_23s_filtered)
log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%2024hr%20expression-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-expression-1">
Past versions of "example 24hr expression-1.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-expression-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 24hr expression-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 24hr expression-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_24_top3 &lt;- row.names(AC.DOX_24.top[1:3,])

log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%2024hr%20expression-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-expression-2">
Past versions of "example 24hr expression-2.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-expression-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 24hr expression-2.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 24hr expression-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_24_top3 &lt;- row.names(AC.EPI_24.top[1:3,])

log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%2024hr%20expression-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-expression-3">
Past versions of "example 24hr expression-3.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-expression-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 24hr expression-3.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 24hr expression-3.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_24_top3 &lt;- row.names(AC.MTX_24.top[1:3,])

log_filt_23s %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_24_top3) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/example%2024hr%20expression-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-example-24hr-expression-4">
Past versions of "example 24hr expression-4.png"
</button>
</p>
<div id="fig-no-spaces-example-24hr-expression-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/example 24hr expression-4.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/example 24hr expression-4.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="correlation-to-atac-data" class="section level3">
<h3>Correlation to ATAC data</h3>
<pre class="r"><code>atac_median_3_lfc &lt;- read_csv(&quot;data/Final_four_data/median_3_lfc.csv&quot;)
atac_median_24_lfc &lt;- read_csv( &quot;data/Final_four_data/median_24_lfc.csv&quot;)

AC_median_3_lfc &lt;- read_csv(&quot;data/Final_four_data/AC_median_3_lfc.csv&quot;)
AC_median_24_lfc &lt;- read_csv(&quot;data/Final_four_data/AC_median_24_lfc.csv&quot;)

overlap_atac_ac_peaks &lt;- readRDS( &quot;data/Final_four_data/overlapping_ac_atac_peaks.RDS&quot;)

# threehroverlap &lt;- 
 overlap_df_ggplot &lt;-  overlap_atac_ac_peaks %&gt;% 
    dplyr::select(peakid, Geneid) %&gt;% 
    left_join(AC_median_3_lfc,by = c(&quot;Geneid&quot;=&quot;Peakid&quot;)) %&gt;% 
    left_join(AC_median_24_lfc,by = c(&quot;Geneid&quot;=&quot;Peakid&quot;)) %&gt;% 
    dplyr::select(peakid:Geneid,AC_3h_lfc,AC_24h_lfc) %&gt;% 
    left_join(., (atac_median_3_lfc %&gt;% dplyr::select(peak,med_3h_lfc)), by = c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
    left_join(., (atac_median_24_lfc %&gt;% dplyr::select(peak,med_24h_lfc)), by = c(&quot;peakid&quot;=&quot;peak&quot;))
 
 overlap_df_ggplot %&gt;% 
   ggplot(., aes(x=AC_3h_lfc,y=med_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = &#39;pearson&#39;)+
   ggtitle(&quot;Correlation of H3K27ac peaks and ATAC peaks that overlap at 3 hours&quot;)+
  xlab(&quot;H3K27ac peak LFC&quot;)+
   ylab(&quot;ATAC peak LFC&quot;)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/comparions%20mapping%20to%20ATAC%20seq-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-comparions-mapping-to-ATAC-seq-1">
Past versions of "comparions mapping to ATAC seq-1.png"
</button>
</p>
<div id="fig-no-spaces-comparions-mapping-to-ATAC-seq-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/comparions mapping to ATAC seq-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/comparions mapping to ATAC seq-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code> overlap_df_ggplot %&gt;% 
   ggplot(., aes(x=AC_24h_lfc,y=med_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = &#39;pearson&#39;)+
   
   ggtitle(&quot;Correlation of H3K27ac peaks and ATAC peaks that overlap at 24 hours&quot;)+
  xlab(&quot;H3K27ac peak LFC&quot;)+
   ylab(&quot;ATAC peak LFC&quot;)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/comparions%20mapping%20to%20ATAC%20seq-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-comparions-mapping-to-ATAC-seq-2">
Past versions of "comparions mapping to ATAC seq-2.png"
</button>
</p>
<div id="fig-no-spaces-comparions-mapping-to-ATAC-seq-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/comparions mapping to ATAC seq-2.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/comparions mapping to ATAC seq-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code> # saveRDS(overlap_df_ggplot,&quot;data/Final_four_data/LFC_ATAC_K27ac.RDS&quot;)</code></pre>
<pre class="r"><code>efit4 &lt;- readRDS(&quot;data/Final_four_data/efit4_filt_bl.RDS&quot;)

ATAC.DNR_3.top_ff= topTable(efit4, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;) %&gt;% rownames_to_column(&quot;peak&quot;)
ATAC.DOX_3.top_ff= topTable(efit4, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)
ATAC.EPI_3.top_ff= topTable(efit4, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)
ATAC.MTX_3.top_ff= topTable(efit4, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)

ATAC.DNR_24.top_ff= topTable(efit4, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)
ATAC.DOX_24.top_ff= topTable(efit4, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)
ATAC.EPI_24.top_ff= topTable(efit4, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)
ATAC.MTX_24.top_ff= topTable(efit4, coef=9, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)%&gt;% rownames_to_column(&quot;peak&quot;)
ATAC_LFC_data&lt;- ATAC.DNR_3.top_ff %&gt;% 
  dplyr::select(peak,logFC) %&gt;% 
  dplyr::rename(&quot;ATAC_DNR_3&quot;=logFC) %&gt;% 
  left_join(., (ATAC.DOX_3.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_DOX_3&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., (ATAC.EPI_3.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_EPI_3&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;%
  left_join(., (ATAC.MTX_3.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_MTX_3&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;%
  left_join(., (ATAC.DNR_24.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_DNR_24&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;%
  left_join(., (ATAC.DOX_24.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_DOX_24&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;%
  left_join(., (ATAC.EPI_24.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_EPI_24&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;%
  left_join(., (ATAC.MTX_24.top_ff %&gt;% select(peak,logFC) %&gt;%
                  dplyr::rename(&quot;ATAC_MTX_24&quot;=logFC)), by=c(&quot;peak&quot;=&quot;peak&quot;)) 


AC_LFC_data &lt;- AC.DNR_3.top %&gt;% 
  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
  dplyr::select(Geneid,logFC) %&gt;% 
  dplyr::rename(&quot;AC_DNR_3&quot;=logFC) %&gt;% 
  left_join(., (AC.DOX_3.top %&gt;%
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_DOX_3&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;% 
  left_join(., (AC.EPI_3.top %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_EPI_3&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;%
  left_join(., (AC.MTX_3.top %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_MTX_3&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;%
  left_join(., (AC.DNR_24.top %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_DNR_24&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;%
  left_join(., (AC.DOX_24.top %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_DOX_24&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;%
  left_join(., (AC.EPI_24.top %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_EPI_24&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;%
  left_join(., (AC.MTX_24.top %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;) %&gt;% 
                  select(Geneid,logFC) %&gt;%
                  dplyr::rename(&quot;AC_MTX_24&quot;=logFC)), by=c(&quot;Geneid&quot;=&quot;Geneid&quot;)) 

LFC_corr &lt;- overlap_df_ggplot %&gt;% 
  select(peakid,Geneid) %&gt;% 
  left_join(., ATAC_LFC_data, by = c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., AC_LFC_data, by = c (&quot;Geneid&quot;=&quot;Geneid&quot;)) %&gt;% 
  unite(&quot;name&quot;,peakid:Geneid) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  cor()
  
ComplexHeatmap::Heatmap(LFC_corr)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/corr%20heatmaps-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-heatmaps-1">
Past versions of "corr heatmaps-1.png"
</button>
</p>
<div id="fig-no-spaces-corr-heatmaps-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/corr heatmaps-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># filmat_groupmat_col &lt;- data.frame(timeset = colnames(LFC_corr))
# 
# counts_corr_mat &lt;-  filmat_groupmat_col %&gt;% 
#  # mutate(sample = timeset) %&gt;% 
#   separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
#   mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
#   mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
#   mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
#     trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
#   ))) %&gt;%
#   mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
#     trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) %&gt;% 
#   mutate(indv=factor(indv, levels = c(&quot;A&quot;,&quot;B&quot;,&quot;C&quot;))) 
#                          
#  mat_colors &lt;- list( 
#    trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#41B333&quot;),
#    # indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;, &quot;#E7298A&quot; ,&quot;#66A61E&quot;, &quot;#E6AB02&quot;),
#   indv=c(A=&quot;#1B9E77&quot;,B= &quot;#D95F02&quot; ,C=&quot;#7570B3&quot;),
#    time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
#    class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
#    TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
#                          
# names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
# names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
# names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
# names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
# names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)
# 
# htanno &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat, col = mat_colors)
# ComplexHeatmap::Heatmap(Frag_cor, top_annotation = htanno)</code></pre>
<pre class="r"><code>ATAC_Full_raw_counts&lt;- readRDS(&quot;data/Final_four_data/ATAC_filtered_raw_counts_allsamples.RDS&quot;)
ATAC_lcpm_specific &lt;- ATAC_Full_raw_counts %&gt;% as.data.frame() %&gt;% 
  dplyr::select(Ind2_87DA24h:Ind2_87M3h,Ind2_87V24h:Ind3_77M3h,Ind3_77V24h:Ind3_77V3h,Ind6_71DA24h:Ind6_71M3h,Ind6_71V24h,Ind6_71V3h) %&gt;% 
  cpm(., log=TRUE) 


K27_ac_lcpm &lt;- filt_H3_matrix_lcpm_23s
 
counts_corr &lt;-
  overlap_df_ggplot %&gt;% 
  select(peakid,Geneid) %&gt;% 
  left_join(., (ATAC_lcpm_specific %&gt;% as.data.frame() %&gt;% 
                  rownames_to_column(&quot;peak&quot;)), by = c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., (K27_ac_lcpm %&gt;% as.data.frame() %&gt;% 
                  rownames_to_column(&quot;Geneid&quot;)), by = c (&quot;Geneid&quot;=&quot;Geneid&quot;))%&gt;% 
  unite(&quot;name&quot;,peakid:Geneid) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  cor()
ComplexHeatmap::Heatmap(counts_corr)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/corr%20of%20log2cpm-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-corr-of-log2cpm-1">
Past versions of "corr of log2cpm-1.png"
</button>
</p>
<div id="fig-no-spaces-corr-of-log2cpm-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/corr of log2cpm-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cormotif" class="section level3">
<h3>Cormotif</h3>
<pre class="r"><code>###now adjusted for 23 samples:
# notes for self in future: I have adjusted this and run on both 23 samples and 25 samples of all 3 shared individuals.  A is individual 87, B is individual 77 and C is individual 71.
for_group3 &lt;- data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;% 
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  unite(&quot;test&quot;, trt:time,sep=&quot;_&quot;, remove = FALSE)
### for 3 individuals {2,3,6}
group_3 &lt;- c( 1,2,4,5,6,7,10,1,2,3,4,7,8,9,10,1,2,3,5,6,7,8,9)
group_fac_3 &lt;- group_3
groupid_3 &lt;- as.numeric(group_fac_3)
label &lt;- for_group3$sample
compid_3 &lt;- data.frame(c1= c(2,4,6,8,1,3,5,7), c2 = c( 10,10,10,10,9,9,9,9))

y_TMM_cpm_3 &lt;- cpm(PCA_H3_mat_23s, log = TRUE)

colnames(y_TMM_cpm_3) &lt;- label

# set.seed(31415)
# cormotif_initial_3 &lt;- cormotiffit(exprs = y_TMM_cpm_3, groupid = groupid_3, compid = compid_3, K=1:6, max.iter = 500, runtype = &quot;logCPM&quot;)
cormotif_initial_23s &lt;- readRDS(&quot;data/Final_four_data/cormotif_3_raodah_run_23s.RDS&quot;)
# saveRDS(cormotif_initial_3,&quot;data/Final_four_data/cormotif_3_raodah_run_23s.RDS&quot;)
plotIC(cormotif_initial_23s)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/cormotif3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-cormotif3-1">
Past versions of cormotif3-1.png
</button>
</p>
<div id="fig-cormotif3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/cormotif3-1.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/cormotif3-1.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plotMotif(cormotif_initial_23s)</code></pre>
<p><img src="figure/Raodah_mycount.Rmd/cormotif3-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-cormotif3-2">
Past versions of cormotif3-2.png
</button>
</p>
<div id="fig-cormotif3-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/4881ad6c88f2eeb9b6d446958d5b0b378f4e01aa/docs/figure/Raodah_mycount.Rmd/cormotif3-2.png" target="_blank">4881ad6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-10-07
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/50f603e982b067deee87030676647f1b97853dde/docs/figure/Raodah_mycount.Rmd/cormotif3-2.png" target="_blank">50f603e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># head(cormotif_initial_3$bestmotif)</code></pre>
<pre class="r"><code>motif_prob_23s &lt;- cormotif_initial_23s$bestmotif$clustlike
rownames(motif_prob_23s) &lt;- rownames(y_TMM_cpm_3)

background_AC_peaks &lt;- motif_prob_23s %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column(&quot;Peakid&quot;) %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  separate(Peakid, into=c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;),remove = FALSE)

NR_23s &lt;- motif_prob_23s %&gt;%
  as.data.frame() %&gt;% 
  dplyr::filter(V1&gt;.5 &amp; V2&lt;.5 &amp; V3 &lt;.5&amp; V4&lt;0.5) %&gt;% 
  rownames_to_column(&quot;Peakid&quot;) %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  separate(Peakid, into=c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;),remove = FALSE)
 
ESR_23s &lt;- motif_prob_23s %&gt;%
   as.data.frame() %&gt;% 
  dplyr::filter(V1&lt;.5 &amp; V2&gt;.5 &amp; V3 &lt;.5&amp; V4&lt;0.5) %&gt;% 
  rownames_to_column(&quot;Peakid&quot;) %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  separate(Peakid, into=c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;),remove = FALSE)
 
 
EAR_23s &lt;- motif_prob_23s %&gt;%
  as.data.frame() %&gt;% 
  dplyr::filter(V1&lt;.5 &amp; V2&lt;.5 &amp; V3 &gt;.5&amp; V4&lt;0.5) %&gt;% 
  rownames_to_column(&quot;Peakid&quot;) %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  separate(Peakid, into=c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;),remove = FALSE)
 

LR_23s &lt;- motif_prob_23s %&gt;%
  as.data.frame() %&gt;% 
  dplyr::filter(V1&lt;.5 &amp; V2&lt;.5 &amp; V3 &lt;.5&amp; V4&gt;0.5) %&gt;% 
  rownames_to_column(&quot;Peakid&quot;) %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  separate(Peakid, into=c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;),remove = FALSE)
 
# saveRDS(background_AC_peaks,&quot;data/Final_four_data/H3K27ac_files/background_AC_peaks.RDS&quot;)
# 
# saveRDS(NR_23s,&quot;data/Final_four_data/H3K27ac_files/NR_23s.RDS&quot;)
# 
# saveRDS(ESR_23s,&quot;data/Final_four_data/H3K27ac_files/ESR_23s.RDS&quot;)
# 
# saveRDS(EAR_23s,&quot;data/Final_four_data/H3K27ac_files/EAR_23s.RDS&quot;)
# 
# saveRDS(LR_23s,&quot;data/Final_four_data/H3K27ac_files/LR_23s.RDS&quot;)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] cowplot_1.1.3         smplot2_0.2.4         ComplexHeatmap_2.20.0
 [4] ggrepel_0.9.6         plyranges_1.24.0      ggsignif_0.6.4       
 [7] genomation_1.36.0     eulerr_7.0.2          devtools_2.4.5       
[10] usethis_3.0.0         ggpubr_0.6.0          BiocParallel_1.38.0  
[13] Cormotif_1.50.0       affy_1.82.0           Biobase_2.64.0       
[16] scales_1.3.0          VennDiagram_1.7.3     futile.logger_1.4.3  
[19] gridExtra_2.3         edgeR_4.2.1           limma_3.60.4         
[22] rtracklayer_1.64.0    GenomicRanges_1.56.1  GenomeInfoDb_1.40.1  
[25] IRanges_2.38.1        S4Vectors_0.42.1      BiocGenerics_0.50.0  
[28] RColorBrewer_1.1-3    broom_1.0.7           kableExtra_1.4.0     
[31] lubridate_1.9.3       forcats_1.0.0         stringr_1.5.1        
[34] dplyr_1.1.4           purrr_1.0.2           readr_2.1.5          
[37] tidyr_1.3.1           tibble_3.2.1          ggplot2_3.5.1        
[40] tidyverse_2.0.0       workflowr_1.7.1      

loaded via a namespace (and not attached):
  [1] splines_4.4.1               later_1.3.2                
  [3] BiocIO_1.14.0               bitops_1.0-9               
  [5] preprocessCore_1.66.0       rpart_4.1.23               
  [7] XML_3.99-0.17               lifecycle_1.0.4            
  [9] rstatix_0.7.2               doParallel_1.0.17          
 [11] rprojroot_2.0.4             vroom_1.6.5                
 [13] processx_3.8.4              lattice_0.22-6             
 [15] backports_1.5.0             magrittr_2.0.3             
 [17] Hmisc_5.2-0                 sass_0.4.9                 
 [19] rmarkdown_2.29              jquerylib_0.1.4            
 [21] yaml_2.3.10                 remotes_2.5.0              
 [23] plotrix_3.8-4               httpuv_1.6.15              
 [25] sessioninfo_1.2.2           pkgbuild_1.4.5             
 [27] abind_1.4-8                 pkgload_1.4.0              
 [29] zlibbioc_1.50.0             RCurl_1.98-1.16            
 [31] nnet_7.3-19                 git2r_0.35.0               
 [33] circlize_0.4.16             GenomeInfoDbData_1.2.12    
 [35] svglite_2.1.3               codetools_0.2-20           
 [37] DelayedArray_0.30.1         xml2_1.3.6                 
 [39] shape_1.4.6.1               tidyselect_1.2.1           
 [41] farver_2.1.2                UCSC.utils_1.0.0           
 [43] base64enc_0.1-3             matrixStats_1.4.1          
 [45] GenomicAlignments_1.40.0    jsonlite_1.8.9             
 [47] GetoptLong_1.0.5            ellipsis_0.3.2             
 [49] Formula_1.2-5               iterators_1.0.14           
 [51] systemfonts_1.1.0           foreach_1.5.2              
 [53] tools_4.4.1                 Rcpp_1.0.13-1              
 [55] glue_1.8.0                  SparseArray_1.4.8          
 [57] mgcv_1.9-1                  xfun_0.49                  
 [59] MatrixGenerics_1.16.0       withr_3.0.2                
 [61] formatR_1.14                BiocManager_1.30.25        
 [63] fastmap_1.2.0               fansi_1.0.6                
 [65] callr_3.7.6                 digest_0.6.37              
 [67] timechange_0.3.0            R6_2.5.1                   
 [69] mime_0.12                   seqPattern_1.36.0          
 [71] colorspace_2.1-1            utf8_1.2.4                 
 [73] generics_0.1.3              data.table_1.16.2          
 [75] httr_1.4.7                  htmlwidgets_1.6.4          
 [77] S4Arrays_1.4.1              whisker_0.4.1              
 [79] pkgconfig_2.0.3             gtable_0.3.6               
 [81] impute_1.78.0               XVector_0.44.0             
 [83] htmltools_0.5.8.1           carData_3.0-5              
 [85] pwr_1.3-0                   profvis_0.4.0              
 [87] clue_0.3-66                 png_0.1-8                  
 [89] knitr_1.49                  lambda.r_1.2.4             
 [91] rstudioapi_0.17.1           tzdb_0.4.0                 
 [93] reshape2_1.4.4              rjson_0.2.23               
 [95] nlme_3.1-166                checkmate_2.3.2            
 [97] curl_6.0.1                  zoo_1.8-12                 
 [99] GlobalOptions_0.1.2         cachem_1.1.0               
[101] KernSmooth_2.23-24          parallel_4.4.1             
[103] miniUI_0.1.1.1              foreign_0.8-87             
[105] restfulr_0.0.15             pillar_1.9.0               
[107] vctrs_0.6.5                 urlchecker_1.0.1           
[109] promises_1.3.0              car_3.1-3                  
[111] cluster_2.1.6               xtable_1.8-4               
[113] htmlTable_2.4.3             evaluate_1.0.1             
[115] magick_2.8.5                cli_3.6.3                  
[117] locfit_1.5-9.10             compiler_4.4.1             
[119] futile.options_1.0.1        Rsamtools_2.20.0           
[121] rlang_1.1.4                 crayon_1.5.3               
[123] labeling_0.4.3              ps_1.8.1                   
[125] getPass_0.2-4               plyr_1.8.9                 
[127] fs_1.6.5                    stringi_1.8.4              
[129] viridisLite_0.4.2           gridBase_0.4-7             
[131] munsell_0.5.1               Biostrings_2.72.1          
[133] Matrix_1.7-1                BSgenome_1.72.0            
[135] patchwork_1.3.0             hms_1.1.3                  
[137] bit64_4.5.2                 statmod_1.5.0              
[139] shiny_1.9.1                 SummarizedExperiment_1.34.0
[141] memoise_2.0.1               affyio_1.74.0              
[143] bslib_0.8.0                 bit_4.5.0                  </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
