<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-02-27" />

<title>Supplemental Figures 12-19</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Supplemental Figures 12-19</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-02-27</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-05-01
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treea197856075034a08febd5e8e91448c9e1053c25etargetblanka197856a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/a197856075034a08febd5e8e91448c9e1053c25e" target="_blank">a197856</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treea197856075034a08febd5e8e91448c9e1053c25etargetblanka197856a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/a197856075034a08febd5e8e91448c9e1053c25e" target="_blank">a197856</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/Correlation_of_SNPnPEAK.Rmd
    Modified:   analysis/GO_KEGG_analysis.Rmd
    Modified:   analysis/Raodah_mycount.Rmd
    Modified:   analysis/TE_analysis_ff.Rmd
    Modified:   analysis/final_plot_attempt.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Supp_Fig_12-19.Rmd</code>) and
HTML (<code>docs/Supp_Fig_12-19.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/b5ac214417ceb405f16f2867c8d83aeb81f33ea2/docs/Supp_Fig_12-19.html" target="_blank">b5ac214</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/58be8ae5925df0c666369b9479a2259c54566eca/analysis/Supp_Fig_12-19.Rmd" target="_blank">58be8ae</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
updates to supplementary files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/ea368e679f92ae9df9756bd152527b2e7568c837/analysis/Supp_Fig_12-19.Rmd" target="_blank">ea368e6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
updates to supplementary files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/4a22bff3c5b77d452482bbd8d5fa5785ba402ae4/docs/Supp_Fig_12-19.html" target="_blank">4a22bff</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/Supp_Fig_12-19.html" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/8435f1f95dee538bcf9f241e60881cbd3c8466fa/analysis/Supp_Fig_12-19.Rmd" target="_blank">8435f1f</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
adding figures
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/fcc142550d14ab7dd9c4011f022a77ad716d3442/analysis/Supp_Fig_12-19.Rmd" target="_blank">fcc1425</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/337980a0aad6c527eff1f70e938e39b2f3e9bd68/analysis/Supp_Fig_12-19.Rmd" target="_blank">337980a</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
updates to plot
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/37c7d47cabc39c1e27a6b8e888b17433b8bf5f94/analysis/Supp_Fig_12-19.Rmd" target="_blank">37c7d47</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
wflow_git_commit("analysis/Supp_Fig_12-19.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(ggfortify)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(edgeR)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(readxl)
library(devtools)
library(vargen)
library(eulerr)</code></pre>
<div
id="figure-s12-h3k27-acetylated-regions-are-enriched-at-transcription-start-sites."
class="section level3">
<h3>Figure S12: H3K27-acetylated regions are enriched at transcription
start sites.</h3>
<pre class="r"><code>knitr::include_graphics(&quot;assets/Fig\ S12.png&quot;, error=FALSE)</code></pre>
<p><img src="assets/Fig S12.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Fig-S12">
Past versions of "Fig S12.png"
</button>
</p>
<div id="fig-no-spaces-Fig-S12" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/50f3de933860a0ff78053606691ee072e12461ed/docs/assets/Fig S12.png" target="_blank">50f3de9</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>knitr::include_graphics(&quot;docs/assets/Fig\ S12.png&quot;,error = FALSE)</code></pre>
<p><img src="docs/assets/Fig S12.png" style="display: block; margin: auto;" /></p>
<pre class="r"><code>txdb &lt;- TxDb.Hsapiens.UCSC.hg38.knownGene
###taken from Peak_calling rmd

# first get peakfiles (using .narrowPeak files from MACS2 calling) and upload functions

TSS = getBioRegion(TxDb=txdb, upstream=2000, downstream=2000, by = &quot;gene&quot;, 
                   type = &quot;start_site&quot;)
#### EXAMPLE OF CODE #####
# ind4_V24hpeaks_gr &lt;- prepGRangeObj(ind4_V24hpeaks)
# ind1_DA24hpeaks_gr &lt;- prepGRangeObj((ind1_DA24hpeaks))
# H3K27ac_list &lt;- GRangesList(ind1_DA24hpeaks_gr, ind4_V24hpeaks_gr)
# # ##plotting the TSS average window (making an overlap of each using Epi_list as list holder)
# H3K27ac_list_tagMatrix = lapply(H3K27ac_list, getTagMatrix, windows = TSS)
# plotAvgProf(H3K27ac_list_tagMatrix, xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)
#plotPeakProf(H3K27ac_list_tagMatrix, facet = &quot;none&quot;, conf = 0.95)

## What I did here:  I called all my narrowpeak files
peakfiles1 &lt;- choose.files()

##these were practice for getting file names and shortening for the for loop below
# testname &lt;- basename(peakfiles1[1])
# str_split_i(testname, &quot;_&quot;,3)

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
IndA_peaks &lt;- list()
for (file in 1:8){
    testname &lt;- basename(peakfiles1[file])
  banana_peel &lt;- str_split_i(testname, &quot;_&quot;,3)
 IndA_peaks[[banana_peel]] &lt;- readPeakFile(peakfiles1[file])
}
saveRDS(IndA_peaks, &quot;data/Final_four_data/H3K27ac_files/IndA_peaks_list.RDS&quot;)
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)
peakAnnoList_1 &lt;- lapply(IndA_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
saveRDS(peakAnnoList_1, &quot;data/Final_four_data/H3K27ac_files/IndA_peakAnnoList.RDS&quot;)
promoter &lt;- getPromoters(TxDb=txdb, upstream=3000, downstream=3000)
tagMatrix_C &lt;- lapply(IndC_peaks,getTagMatrix, windows=promoter)
plotAvgProf(tagMatrix_C, xlim = c(-3000,3000),xlab = &quot;Genomic Region (5&#39;-&gt;3&#39;)&quot;, ylab = &quot;Read Count Frequency&quot;)

saveRDS(tagMatrix_C, &quot;data/Final_four_data/H3K27ac_files/IndC_tagMatrix.RDS&quot;)</code></pre>
<div id="making-the-plot" class="section level4">
<h4>Making the plot</h4>
<pre class="r"><code>##load tagMatrix files from above
tagMatrix_A &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/IndA_tagMatrix.RDS&quot;)
tagMatrix_B &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/IndB_tagMatrix.RDS&quot;)
tagMatrix_C &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/IndC_tagMatrix.RDS&quot;)
###making the plots and storing the 3 hour as an object
a1&lt;- plotAvgProf(tagMatrix_A[c(1,3,5,7)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual A&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:40:56 AM </code></pre>
<pre class="r"><code>b1 &lt;- plotAvgProf(tagMatrix_B[c(1,3,4,7)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual B&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:40:56 AM </code></pre>
<pre class="r"><code>c1 &lt;- plotAvgProf(tagMatrix_C[c(1,4,6,8)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual C&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:40:57 AM </code></pre>
<pre class="r"><code>### making the plots and storing the 24 hour as an object
a2&lt;- plotAvgProf(tagMatrix_A[c(2,4,6,8)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual A&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:40:57 AM </code></pre>
<pre class="r"><code>b2 &lt;- plotAvgProf(tagMatrix_B[c(2,5,6,8)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual B&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:40:57 AM </code></pre>
<pre class="r"><code>c2 &lt;- plotAvgProf(tagMatrix_C[c(2,3,5,7,9)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual C&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:40:57 AM </code></pre>
<pre class="r"><code>plot_grid(a1,a2, b1,b2,c1,c2, axis=&quot;l&quot;,align = &quot;hv&quot;,nrow=3, ncol=2)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS12-1">
Past versions of FigS12-1.png
</button>
</p>
<div id="fig-FigS12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS12-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="figure-s13-h3k27ac-cuttag-sequencing-data-separate-by-time-and-drug-treatment."
class="section level3">
<h3>Figure S13: H3K27ac CUT&amp;Tag sequencing data separate by time and
drug treatment.</h3>
<pre class="r"><code>H3K27ac_counts &lt;- read_delim(&quot;data/Final_four_data/H3K27ac_files/H3K27ac_counts_file.txt&quot;, delim= &quot;\t&quot;)
corr_lcpmH3K27ac &lt;- H3K27ac_counts %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  cpm(.,log=TRUE) %&gt;% 
  cor()
  
filmat_groupmat_col &lt;- data.frame(timeset = colnames(corr_lcpmH3K27ac))

counts_corr_mat &lt;-  filmat_groupmat_col %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;))) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;AC&quot;, if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)
  ))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, if_else(
    trt == &quot;DOX&quot;, &quot;yes&quot;, if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) %&gt;% 
  mutate(indv=factor(indv, levels = c(&quot;A&quot;,&quot;B&quot;,&quot;C&quot;))) 
                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#41B333&quot;),
   indv=c(A=&quot;#1B9E77&quot;,B= &quot;#D95F02&quot; ,C=&quot;#7570B3&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)

htanno &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat, col = mat_colors)
ComplexHeatmap::Heatmap(corr_lcpmH3K27ac, top_annotation = htanno)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS13-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS13-1">
Past versions of FigS13-1.png
</button>
</p>
<div id="fig-FigS13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS13-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s14-change-in-h3k27ac-enrichment-in-response-to-drugs-is-highly-correlated-across-acs."
class="section level3">
<h3>Figure S14: Change in H3K27ac enrichment in response to drugs is
highly correlated across ACs.</h3>
<div
id="figure-s14a-differentially-enriched-h3k27ac-regions-across-drug-treatments"
class="section level4">
<h4>Figure S14A: Differentially enriched H3K27ac regions across drug
treatments</h4>
<p>Code for Differential analysis</p>
<pre class="r"><code>### group  without veh3hr and veh24hr individual samples
# group_23s &lt;- c( 1,2,4,5,6,7,10,1,2,3,4,7,8,9,10,1,2,3,5,6,7,8,9)
H3K27ac_counts_file &lt;- read_delim(&quot;data/Final_four_data/H3K27ac_files/H3K27ac_counts_file.txt&quot;, delim= &quot;\t&quot;)
PCA_H3_mat_23s &lt;- H3K27ac_counts_file %&gt;% 
  ##removing C_VEH_3 and B_VEH_24 columns 
  dplyr::select(Geneid,B_DNR_24:B_MTX_24, B_VEH_3:C_VEH_24) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

anno_H3_mat_23s &lt;-
data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;%
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;))) 
  

  
lcpm_h3_23s &lt;- cpm(PCA_H3_mat_23s, log=TRUE)  ### for determining the basic cutoffs
row_means23s &lt;- rowMeans(lcpm_h3_23s)
h3_23s_filtered &lt;- PCA_H3_mat_23s[row_means23s &gt;0,]
filt_H3_matrix_lcpm_23s &lt;- cpm(h3_23s_filtered, log=TRUE)

### no difference in between filtered and unfiltered numbers

for_group_23s &lt;- data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;%
  mutate(sample = timeset) %&gt;%
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;%
  unite(&quot;test&quot;, trt:time,sep=&quot;_&quot;, remove = FALSE)

# dge_23s &lt;- DGEList.data.frame(counts = PCA_H3_mat_23s, group = group_23s, genes = lcpm_h3_23s_filtered)

# group_1 &lt;- for_group_23s$test
# 
# dge_23s$group$indv &lt;- for_group_23s$indv
# dge_23s$group$time &lt;- for_group_23s$time
# dge_23s$group$trt &lt;- for_group_23s$trt
# # 
# indv_23s &lt;- for_group_23s$indv
# mm &lt;- model.matrix(~0 +group_1)
# colnames(mm) &lt;-  c(&quot;DNR_24&quot;, &quot;DNR_3&quot;, &quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;, &quot;EPI_3&quot;,&quot;MTX_24&quot;, &quot;MTX_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
# 
# y &lt;- voom(dge_23s$counts, mm,plot =TRUE)
# 
# corfit &lt;- duplicateCorrelation(y, mm, block = indv_23s)
# 
# v &lt;- voom(dge_23s$counts, mm, block = indv_23s, correlation = corfit$consensus)
# 
# fit &lt;- lmFit(v, mm, block = indv_23s, correlation = corfit$consensus)
# # colnames(mm) &lt;- c(&quot;DNR_24&quot;,&quot;DNR_3&quot;,&quot;DOX_24&quot;,&quot;DOX_3&quot;,&quot;EPI_24&quot;,&quot;EPI_3&quot;,&quot;MTX_24&quot;,&quot;MTX_3&quot;,&quot;TRZ_24&quot;,&quot;TRZ_3&quot;,&quot;VEH_24&quot;, &quot;VEH_3&quot;)
# #
# #
# cm &lt;- makeContrasts(
#   DNR_3.VEH_3 = DNR_3-VEH_3,
#   DOX_3.VEH_3 = DOX_3-VEH_3,
#   EPI_3.VEH_3 = EPI_3-VEH_3,
#   MTX_3.VEH_3 = MTX_3-VEH_3,
#   DNR_24.VEH_24 =DNR_24-VEH_24,
#   DOX_24.VEH_24= DOX_24-VEH_24,
#   EPI_24.VEH_24= EPI_24-VEH_24,
#   MTX_24.VEH_24= MTX_24-VEH_24,
#     levels = mm)
#
# vfit &lt;- lmFit(y, mm)
# # 
# vfit&lt;- contrasts.fit(vfit, contrasts=cm)
# 
# efit4_raodah_23s &lt;- eBayes(vfit)</code></pre>
<pre class="r"><code>efit_raodah_23s &lt;- readRDS(&quot;data/Final_four_data/efit4_raodah_23s.RDS&quot;)
results = decideTests(efit_raodah_23s)
summary(results)</code></pre>
<pre><code>       DNR_3.VEH_3 DOX_3.VEH_3 EPI_3.VEH_3 MTX_3.VEH_3 DNR_24.VEH_24
Down           424           2           0           0          2007
NotSig       19502       20135       20137       20137         16669
Up             211           0           0           0          1461
       DOX_24.VEH_24 EPI_24.VEH_24 MTX_24.VEH_24
Down             126            55             0
NotSig         19756         19997         20137
Up               255            85             0</code></pre>
<pre class="r"><code>AC.DNR_3.top= topTable(efit_raodah_23s, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.DOX_3.top= topTable(efit_raodah_23s, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.EPI_3.top= topTable(efit_raodah_23s, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.MTX_3.top= topTable(efit_raodah_23s, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)

AC.DNR_24.top= topTable(efit_raodah_23s, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.DOX_24.top= topTable(efit_raodah_23s, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.EPI_24.top= topTable(efit_raodah_23s, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
AC.MTX_24.top= topTable(efit_raodah_23s, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)


plot_filenames &lt;- c(&quot;AC.DNR_3.top&quot;,&quot;AC.DOX_3.top&quot;,&quot;AC.EPI_3.top&quot;,&quot;AC.MTX_3.top&quot;,
                    &quot;AC.DNR_24.top&quot;,&quot;AC.DOX_24.top&quot;,&quot;AC.EPI_24.top&quot;,
                    &quot;AC.MTX_24.top&quot;)
plot_files &lt;- c( AC.DNR_3.top,AC.DOX_3.top,AC.EPI_3.top,AC.MTX_3.top,
                    AC.DNR_24.top,AC.DOX_24.top,AC.EPI_24.top,
                    AC.MTX_24.top)

volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
     
    ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
      geom_point(aes(color=threshold))+
      xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
      ylab(expression(&quot;-log&quot;[10]*&quot;P Value&quot;))+
      scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
      ylim(0,12)+
      xlim(-5,5)+
      theme_cowplot()+
      theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}

v1 &lt;- volcanosig(AC.DNR_3.top, 0.05)+ ggtitle(&quot;H3K27ac DNR 3 hour&quot;)
v2 &lt;- volcanosig(AC.DNR_24.top, 0.05)+ ggtitle(&quot;H3K27ac DNR 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(AC.DOX_3.top, 0.05)+ ggtitle(&quot;H3K27ac DOX 3 hour&quot;)
v4 &lt;- volcanosig(AC.DOX_24.top, 0.05)+ ggtitle(&quot;H3K27ac DOX 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(AC.EPI_3.top, 0.05)+ ggtitle(&quot;H3K27ac EPI 3 hour&quot;)
v6 &lt;- volcanosig(AC.EPI_24.top, 0.05)+ ggtitle(&quot;H3K27ac EPI 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(AC.MTX_3.top, 0.05)+ ggtitle(&quot;H3K27ac MTX 3 hour&quot;)
v8 &lt;- volcanosig(AC.MTX_24.top, 0.05)+ ggtitle(&quot;H3K27ac MTX 24 hour&quot;)+ylab(&quot;&quot;)

plot_grid(v1,v2,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS14A-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS14A-1">
Past versions of FigS14A-1.png
</button>
</p>
<div id="fig-FigS14A-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS14A-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3,v4,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS14A-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS14A-2">
Past versions of FigS14A-2.png
</button>
</p>
<div id="fig-FigS14A-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS14A-2.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5,v6,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS14A-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS14A-3">
Past versions of FigS14A-3.png
</button>
</p>
<div id="fig-FigS14A-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS14A-3.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7,v8,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS14A-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS14A-4">
Past versions of FigS14A-4.png
</button>
</p>
<div id="fig-FigS14A-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS14A-4.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s14b-correlation-of-h3k27ac-log-fold-change-across-all-regions."
class="section level4">
<h4>Figure S14B: Correlation of H3K27ac log fold change across all
regions.</h4>
<pre class="r"><code>FCmatrix_23s &lt;- subset(efit_raodah_23s$coefficients)

colnames(FCmatrix_23s) &lt;-
  c(&quot;DNR\n3h&quot;,
    &quot;DOX\n3h&quot;,
    &quot;EPI\n3h&quot;,
    &quot;MTX\n3h&quot;,
    &quot;DNR\n24h&quot;,
    &quot;DOX\n24h&quot;,
    &quot;EPI\n24h&quot;,
    &quot;MTX\n24h&quot;
     )


mat_col_FC &lt;-
  data.frame(
    time = c(rep(&quot;3 hours&quot;, 4), rep(&quot;24 hours&quot;, 4)),
    class = (c(
      &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;
    )))
rownames(mat_col_FC) &lt;- colnames(FCmatrix_23s)

mat_colors_FC &lt;-
  list(
    time = c(&quot;pink&quot;, &quot;chocolate4&quot;),
    class = c(&quot;yellow1&quot;, &quot;lightgreen&quot;))

names(mat_colors_FC$time) &lt;- unique(mat_col_FC$time)
names(mat_colors_FC$class) &lt;- unique(mat_col_FC$class)
# names(mat_colors_FC$TOP2i) &lt;- unique(mat_col_FC$TOP2i)
corrFC_23s &lt;- cor(FCmatrix_23s)

htanno_23s &lt;-  HeatmapAnnotation(df = mat_col_FC, col = mat_colors_FC)
Heatmap(corrFC_23s, top_annotation = htanno_23s)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS14B-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS14B-1">
Past versions of FigS14B-1.png
</button>
</p>
<div id="fig-FigS14B-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS14B-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="figure-s15-four-h3k27ac-enrichment-signatures-capture-the-response-to-top2i-over-time."
class="section level3">
<h3>Figure S15: Four H3K27ac enrichment signatures capture the response
to TOP2i over time.</h3>
<div
id="figure-s15-a-b-bic-and-aic-for-h3k27ac-following-joint-modeling-of-pairs-of-tests."
class="section level4">
<h4>Figure S15 A &amp; B: BIC and AIC for H3K27ac following joint
modeling of pairs of tests.</h4>
<pre class="r"><code>###now adjusted for 23 samples:
H3K27ac_counts_file &lt;- read_delim(&quot;data/Final_four_data/H3K27ac_files/H3K27ac_counts_file.txt&quot;, delim= &quot;\t&quot;)
PCA_H3_mat_23s &lt;- H3K27ac_counts_file %&gt;% 
  ##removing C_VEH_3 and B_VEH_24 columns 
  dplyr::select(Geneid,B_DNR_24:B_MTX_24, B_VEH_3:C_VEH_24) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix()

anno_H3_mat_23s &lt;-
data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;%
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;, &quot;24&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;,&quot;VEH&quot;))) 

for_group3 &lt;- data.frame(timeset=colnames(PCA_H3_mat_23s)) %&gt;% 
  mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  tidyr::unite(&quot;test&quot;, trt:time,sep=&quot;_&quot;, remove = FALSE)
### for 3 individuals {2,3,6}
group_3 &lt;- c( 1,2,4,5,6,7,10,1,2,3,4,7,8,9,10,1,2,3,5,6,7,8,9)
group_fac_3 &lt;- group_3
groupid_3 &lt;- as.numeric(group_fac_3)
label &lt;- for_group3$sample
compid_3 &lt;- data.frame(c1= c(2,4,6,8,1,3,5,7), c2 = c( 10,10,10,10,9,9,9,9))

y_TMM_cpm_3 &lt;- cpm(PCA_H3_mat_23s, log = TRUE)

colnames(y_TMM_cpm_3) &lt;- label

# set.seed(31415)
# cormotif_initial_3 &lt;- cormotiffit(exprs = y_TMM_cpm_3, groupid = groupid_3, compid = compid_3, K=1:6, max.iter = 500, runtype = &quot;logCPM&quot;)

##results from the K1:6 run:
cormotif_initial_23s &lt;- readRDS(&quot;data/Final_four_data/cormotif_3_raodah_run_23s.RDS&quot;)

Cormotif::plotIC(cormotif_initial_23s)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/Figs15AB-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Figs15AB-1">
Past versions of Figs15AB-1.png
</button>
</p>
<div id="fig-Figs15AB-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/Figs15AB-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="figure-s15-c-four-drug-response-motifs-are-identified."
class="section level4">
<h4>Figure S15 C: Four drug response motifs are identified.</h4>
<pre class="r"><code>Cormotif::plotMotif(cormotif_initial_23s)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS15C-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS15C-1">
Past versions of FigS15C-1.png
</button>
</p>
<div id="fig-FigS15C-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS15C-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s15-d-h3k27ac-enrichment-at-each-of-the-response-classes-at-three-and-24-hours-across-treatments."
class="section level4">
<h4>Figure S15 D: H3K27ac enrichment at each of the response classes at
three and 24 hours across treatments.</h4>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#41B333&quot;)
H3K27ac_counts_file &lt;- read_delim(&quot;data/Final_four_data/H3K27ac_files/H3K27ac_counts_file.txt&quot;, delim= &quot;\t&quot;)
lcpm_h3_23s &lt;- H3K27ac_counts_file %&gt;% 
  ##removing C_VEH_3 and B_VEH_24 columns 
  dplyr::select(Geneid,B_DNR_24:B_MTX_24, B_VEH_3:C_VEH_24) %&gt;% 
  column_to_rownames(&quot;Geneid&quot;) %&gt;% 
  as.matrix() %&gt;% 
 cpm(., log = TRUE) 

NR_ac &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/NR_23s.RDS&quot;)

ESR_ac &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/ESR_23s.RDS&quot;)

EAR_ac &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/EAR_23s.RDS&quot;)

LR_ac &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/LR_23s.RDS&quot;)

set.seed(31415)
sample_peaks &lt;- rbind(
  &quot;NR_1&quot;=sample_n(NR_ac,size = 2),
  &quot;ESR_2&quot;=sample_n(ESR_ac,size=2),
  &quot;EAR_3&quot;=sample_n(EAR_ac,size=2),
  &quot;LR_4&quot;=sample_n(LR_ac, size=2)) 
sample_peaks_choice &lt;- sample_peaks %&gt;% 
  dplyr::filter(rownames(.)==&quot;NR_1.2&quot;|
                  rownames(.)==&quot;ESR_2.1&quot;|
                  rownames(.)==&quot;EAR_3.1&quot;|
                  rownames(.)==&quot;LR_4.1&quot;)
lcpm_h3_23s%&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column(&quot;Peakid&quot;)%&gt;% 
  dplyr::filter(Peakid %in% sample_peaks_choice$Peakid) %&gt;%
  pivot_longer(., cols = !Peakid, names_to = &quot;samples&quot;,values_to = &quot;logcpm&quot;) %&gt;% 
  separate_wider_delim(., samples, delim = &quot;_&quot;,names=c(&quot;ind&quot;,&quot;trt&quot;,&quot;time&quot;), cols_remove = FALSE) %&gt;% 
  left_join(., (sample_peaks %&gt;% 
              rownames_to_column(&quot;cluster&quot;) %&gt;% 
              dplyr::select(cluster,Peakid)),
            by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(trt=factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;, &quot;VEH&quot;)),
         time= factor(time, levels = c(&quot;3&quot;,&quot;24&quot;))) %&gt;% 
  ggplot(aes(x=time,y=logcpm))+
  geom_boxplot(aes(fill=trt))+
  # geom_point(aes(color=ind))+
  facet_wrap(~Peakid+cluster,scales=&quot;free_y&quot;, nrow=4, ncol=2)+
  ggtitle(&quot; H3K27ac acetylation&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab(&quot;log2 cpm H3K27ac&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS15D-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS15D-1">
Past versions of FigS15D-1.png
</button>
</p>
<div id="fig-FigS15D-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS15D-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="figure-s16-shared-atac-and-h3k27-acetylation-region-response-to-top2i-clusters-by-time."
class="section level3">
<h3>Figure S16: Shared ATAC and H3K27-acetylation region response to
TOP2i clusters by time.</h3>
<pre class="r"><code>toplist_atac_lfc &lt;- readRDS(&quot;data/Final_four_data/toplist_ff.RDS&quot;) 

hr3_ATAC &lt;- toplist_atac_lfc %&gt;% 
  dplyr::select(trt:logFC) %&gt;% 
  dplyr::filter(time==&quot;3 hours&quot;) %&gt;% 
  pivot_wider(., id_cols = c(&quot;peak&quot;),names_from = trt,values_from = logFC) %&gt;% 
   dplyr::rename(DOX_3_ATAC=DOX, DNR_3_ATAC=DNR,EPI_3_ATAC=EPI,MTX_3_ATAC=MTX, TRZ_3_ATAC=TRZ)

hr24_ATAC &lt;- toplist_atac_lfc %&gt;% 
  dplyr::select(trt:logFC) %&gt;% 
  dplyr::filter(time==&quot;24 hours&quot;) %&gt;% 
  pivot_wider(., id_cols = c(&quot;peak&quot;),names_from = trt,values_from = logFC) %&gt;% 
   dplyr::rename(DOX_24_ATAC=DOX, DNR_24_ATAC=DNR,EPI_24_ATAC=EPI,MTX_24_ATAC=MTX, TRZ_24_ATAC=TRZ)

ATAC_LFC_df &lt;- hr3_ATAC %&gt;% 
  left_join(., hr24_ATAC, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;%
   dplyr::select(peak:MTX_3_ATAC,DNR_24_ATAC:MTX_24_ATAC)

toplist_ac_lfc &lt;- readRDS(&quot;data/Final_four_data/toplist_ac_lfc.RDS&quot;)

hr3_K27 &lt;- toplist_ac_lfc %&gt;% 
  dplyr::filter(time==&quot;3&quot;) %&gt;% 
  dplyr::select(Peakid:logFC) %&gt;% 
  ###something weird in peaknames... additional numbers? so I will remove them
   mutate(Peakid=str_replace(Peakid, &quot;\\.\\.\\.[0=9]*&quot;,&quot;/&quot;)) %&gt;% 
  separate(Peakid, into=c(&quot;Geneid&quot;,NA),sep=&quot;/&quot;) %&gt;% 
  pivot_wider(., id_cols=c(Geneid), names_from = &quot;trt&quot;, values_from = &quot;logFC&quot;) %&gt;% 
  dplyr::rename(DOX_3_K27=DOX, DNR_3_K27=DNR,EPI_3_K27=EPI,MTX_3_K27=MTX) 

hr24_K27 &lt;- toplist_ac_lfc %&gt;% 
  dplyr::filter(time==&quot;24&quot;) %&gt;% 
  dplyr::select(Peakid:logFC) %&gt;% 
  ###something weird in peaknames... additional numbers? so I will remove them
   mutate(Peakid=str_replace(Peakid, &quot;\\.\\.\\.[0=9]*&quot;,&quot;/&quot;)) %&gt;% 
  separate(Peakid, into=c(&quot;Geneid&quot;,NA),sep=&quot;/&quot;) %&gt;% 
  pivot_wider(., id_cols=c(Geneid), names_from = &quot;trt&quot;, values_from = &quot;logFC&quot;) %&gt;% 
  dplyr::rename(DOX_24_K27=DOX, DNR_24_K27=DNR,EPI_24_K27=EPI,MTX_24_K27=MTX)


K27_LFC_df &lt;- hr3_K27 %&gt;% 
  left_join(., hr24_K27, by=c(&quot;Geneid&quot;=&quot;Geneid&quot;))  



overlap_df_ggplot &lt;- readRDS(&quot;data/Final_four_data/LFC_ATAC_K27ac.RDS&quot;) %&gt;% 
  dplyr::select(peakid,Geneid) %&gt;% 
  left_join(.,ATAC_LFC_df, by=c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,K27_LFC_df, by= c(&quot;Geneid&quot;=&quot;Geneid&quot;))
 
counts_corr &lt;-
  overlap_df_ggplot %&gt;% 
    tidyr::unite(.,&quot;name&quot;,peakid,Geneid,sep=&quot;_&quot;) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  cor()
ComplexHeatmap::Heatmap(counts_corr)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS16-1">
Past versions of FigS16-1.png
</button>
</p>
<div id="fig-FigS16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS16-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s17-shared-accessible-and-h3k27-acetylated-chromatin-regions-have-a-significant-correlation-with-nearby-gene-expression-at-24-hours."
class="section level3">
<h3>Figure S17: Shared accessible and H3K27-acetylated chromatin regions
have a significant correlation with nearby gene expression at 24
hours.</h3>
<pre class="r"><code>RNA_ATAC_overlap_df_ggplot &lt;- readRDS(&quot;data/Final_four_data/LFC_ATAC_K27ac.RDS&quot;) %&gt;% 
  dplyr::select(peakid,Geneid)
Collapsed_new_peaks &lt;- read_delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;, delim = &quot;\t&quot;, col_names = TRUE)

RNA_median_3_lfc &lt;- readRDS(&quot;data/other_papers/RNA_median_3_lfc.RDS&quot;)
RNA_median_24_lfc &lt;- readRDS(&quot;data/other_papers/RNA_median_24_lfc.RDS&quot;)
overlap_df_ggplot &lt;- readRDS(&quot;data/Final_four_data/LFC_ATAC_K27ac.RDS&quot;)

ATAC_median_24_lfc &lt;- read_csv(&quot;data/Final_four_data/median_24_lfc.csv&quot;) 
ATAC_median_3_lfc &lt;- read_csv(&quot;data/Final_four_data/median_3_lfc.csv&quot;)

anti_joined_lfc &lt;- ATAC_median_24_lfc %&gt;% 
  dplyr::filter(!peak %in% RNA_ATAC_overlap_df_ggplot$peakid)  %&gt;% 
  left_join(., (Collapsed_new_peaks %&gt;% dplyr::select(Peakid, NCBI_gene:dist_to_NG)), by = c(&quot;peak&quot;=&quot;Peakid&quot;)) %&gt;% 
  left_join(.,ATAC_median_3_lfc,by = c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(., RNA_median_3_lfc ,
                by=c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;, &quot;NCBI_gene&quot;=&quot;ENTREZID&quot;)) %&gt;%
  left_join(., RNA_median_24_lfc,
                 by=c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;, &quot;NCBI_gene&quot;=&quot;ENTREZID&quot;)) 

joined_LFC_df &lt;-
RNA_ATAC_overlap_df_ggplot %&gt;%
  left_join(.,(Collapsed_new_peaks %&gt;%
                 dplyr::select(Peakid,dist_to_NG, NCBI_gene:SYMBOL)),
            by=c(&quot;peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  left_join(., RNA_median_3_lfc ,
                 by=c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;, &quot;NCBI_gene&quot;=&quot;ENTREZID&quot;)) %&gt;%
  left_join(., RNA_median_24_lfc,
             by=c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;, &quot;NCBI_gene&quot;=&quot;ENTREZID&quot;)) %&gt;% 
  left_join(.,ATAC_median_3_lfc,by = c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;%
  left_join(.,ATAC_median_24_lfc,by = c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(dist_to_NG &gt; -2000| dist_to_NG &lt;2000)

 
joined_LFC_df %&gt;% 
dplyr::filter(dist_to_NG&gt;-2000 &amp; dist_to_NG&lt;2000) %&gt;% 
  ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = &#39;pearson&#39;)+
  ggtitle(&quot;Correlation of shared 2kb H3K27ac peaks and ATAC_peaks at 3 hours&quot;)+
  ylab(&quot;H3K27ac peak LFC&quot;)+
   xlab(&quot;ATAC-seq log FC&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS17-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS17-1">
Past versions of FigS17-1.png
</button>
</p>
<div id="fig-FigS17-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS17-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>joined_LFC_df %&gt;% 
dplyr::filter(dist_to_NG&gt;-2000 &amp; dist_to_NG&lt;2000) %&gt;% 
  ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = &#39;pearson&#39;)+
  ggtitle(&quot;Correlation of shared 2kb H3K27ac peaks and ATAC_peaks at 24 hours&quot;)+
  ylab(&quot;H3K27ac peak LFC&quot;)+
   xlab(&quot;ATAC-seq log FC&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS17-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS17-2">
Past versions of FigS17-2.png
</button>
</p>
<div id="fig-FigS17-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS17-2.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code> anti_joined_lfc %&gt;% 
 dplyr::filter(dist_to_NG&gt;-2000 &amp; dist_to_NG&lt;2000) %&gt;% 
   # dplyr::filter(peak %in% ATAC_LFC_df$peak) %&gt;% 
  ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = &#39;pearson&#39;)+
  ggtitle(&quot;ATAC-seq regions without H3K27ac overlap at 3 hours&quot;)+
  ylab(&quot;RNA expression LFC&quot;)+
   xlab(&quot;ATAC-seq log FC&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS17-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS17-3">
Past versions of FigS17-3.png
</button>
</p>
<div id="fig-FigS17-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS17-3.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code> anti_joined_lfc %&gt;% 
  dplyr::filter(dist_to_NG&gt;-2000 &amp; dist_to_NG&lt;2000) %&gt;% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = &#39;pearson&#39;)+
   ggtitle(&quot;Correlation of ATAC peaks not shared with H3K27ac and RNA-NG at 24 hours&quot;)+
  xlab(&quot;ATAC peak LFC&quot;)+
   ylab(&quot;RNA LFC&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS17-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS17-4">
Past versions of FigS17-4.png
</button>
</p>
<div id="fig-FigS17-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS17-4.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s18-drug-response-associates-with-time-and-molecular-phenotype."
class="section level3">
<h3>Figure S18: Drug response associates with time and molecular
phenotype.</h3>
<pre class="r"><code>toplistall_RNA &lt;- readRDS(&quot;data/other_papers/toplistall_RNA.RDS&quot;) 
toplistall_RNA &lt;- toplistall_RNA %&gt;% 
  mutate(logFC = logFC*(-1))

hr3_RNA &lt;- toplistall_RNA %&gt;% 
  dplyr::select(time:logFC) %&gt;% 
  dplyr::filter(time==&quot;3_hours&quot;) %&gt;% 
  pivot_wider(., id_cols=c(ENTREZID,SYMBOL), names_from = id, values_from = logFC) %&gt;% 
  dplyr::rename(DOX_3_RNA=DOX, DNR_3_RNA=DNR,EPI_3_RNA=EPI,MTX_3_RNA=MTX, TRZ_3_RNA=TRZ)
  

hr24_RNA &lt;- toplistall_RNA %&gt;% 
  dplyr::select(time:logFC) %&gt;% 
  dplyr::filter(time==&quot;24_hours&quot;) %&gt;% 
  pivot_wider(., id_cols=c(ENTREZID,SYMBOL), names_from = id, values_from = logFC) %&gt;% 
  dplyr::rename(DOX_24_RNA=DOX, DNR_24_RNA=DNR,EPI_24_RNA=EPI,MTX_24_RNA=MTX, TRZ_24_RNA=TRZ)
RNA_LFC_df &lt;- hr3_RNA %&gt;% 
  left_join(., hr24_RNA, by=c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;,&quot;ENTREZID&quot;=&quot;ENTREZID&quot;)) %&gt;% 
  dplyr::select(ENTREZID:MTX_3_RNA,DNR_24_RNA:MTX_24_RNA)



combo_lfc &lt;- readRDS(&quot;data/Final_four_data/LFC_ATAC_K27ac.RDS&quot;) %&gt;% 
  dplyr::select(peakid,Geneid) %&gt;% 
  left_join(.,ATAC_LFC_df, by=c(&quot;peakid&quot;=&quot;peak&quot;)) %&gt;% 
  left_join(.,K27_LFC_df, by= c(&quot;Geneid&quot;=&quot;Geneid&quot;))

combo_corr &lt;- combo_lfc %&gt;% 
  left_join(.,(Collapsed_new_peaks %&gt;% 
  dplyr::select(Peakid, NCBI_gene,SYMBOL)), by=c(&quot;peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  left_join(., RNA_LFC_df, by=c(&quot;SYMBOL&quot;=&quot;SYMBOL&quot;,&quot;NCBI_gene&quot;=&quot;ENTREZID&quot;)) %&gt;% 
  tidyr::unite(., &quot;name&quot;,peakid,Geneid,NCBI_gene,SYMBOL,sep = &quot;_&quot;) %&gt;% 
  column_to_rownames(&quot;name&quot;) %&gt;% 
  na.omit() %&gt;% 
  cor()
ComplexHeatmap::Heatmap(combo_corr)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS18-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS18-1">
Past versions of FigS18-1.png
</button>
</p>
<div id="fig-FigS18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS18-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s19-expressed-genes-near-snp-containing-drug-responsive-chromatin-regions-reflect-chromatin-accessibility-changes."
class="section level3">
<h3>Figure S19: Expressed genes near SNP-containing drug-responsive
chromatin regions reflect chromatin accessibility changes.</h3>
<div id="figure-s19a-prdx6-rna-expression" class="section level4">
<h4>Figure S19A: <em>PRDX6</em> RNA expression</h4>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
PRDX6_info &lt;- Collapsed_new_peaks %&gt;% dplyr::filter(SYMBOL==&quot;PRDX6&quot;) %&gt;% 
  distinct(NCBI_gene,SYMBOL)

RNA_counts &lt;- readRDS(&quot;data/other_papers/cpmcount.RDS&quot;) %&gt;%
  dplyr::rename_with(.,~gsub(pattern=&quot;Da&quot;,replacement=&quot;DNR&quot;,.)) %&gt;% 
 dplyr::rename_with(.,~gsub(pattern=&quot;Do&quot;,replacement=&quot;DOX&quot;,.)) %&gt;% 
  dplyr::rename_with(.,~gsub(pattern=&quot;Ep&quot;,replacement=&quot;EPI&quot;,.)) %&gt;% 
   dplyr::rename_with(.,~gsub(pattern=&quot;Mi&quot;,replacement=&quot;MTX&quot;,.)) %&gt;% 
    dplyr::rename_with(.,~gsub(pattern=&quot;Tr&quot;,replacement=&quot;TRZ&quot;,.)) %&gt;% 
       dplyr::rename_with(.,~gsub(pattern=&quot;Ve&quot;,replacement=&quot;VEH&quot;,.)) %&gt;% 
  rownames_to_column(&quot;ENTREZID&quot;) 


RNA_counts %&gt;% 
  dplyr::filter(ENTREZID == PRDX6_info$NCBI_gene) %&gt;% 
  pivot_longer(cols = !ENTREZID, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  left_join(., PRDX6_info, by =c(&quot;ENTREZID&quot;=&quot;NCBI_gene&quot;)) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;trt&quot;,&quot;ind&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL, scales=&quot;free_y&quot;)+
  ggtitle(&quot;RNA log 2 cpm of expressed gene&quot;)+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab(&quot;log2 cpm RNA&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/FigS19A-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS19A-1">
Past versions of FigS19A-1.png
</button>
</p>
<div id="fig-FigS19A-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/FigS19A-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="figure-s19b-prune2-rna-expression" class="section level4">
<h4>Figure S19B: <em>PRUNE2</em> RNA expression</h4>
<pre class="r"><code>PRUNE2_info &lt;- Collapsed_new_peaks %&gt;% dplyr::filter(SYMBOL==&quot;PRUNE2&quot;) %&gt;% 
  distinct(NCBI_gene,SYMBOL)


RNA_counts %&gt;% 
  dplyr::filter(ENTREZID == PRUNE2_info$NCBI_gene) %&gt;% 
  pivot_longer(cols = !ENTREZID, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  left_join(., PRUNE2_info, by =c(&quot;ENTREZID&quot;=&quot;NCBI_gene&quot;)) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;trt&quot;,&quot;ind&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL, scales=&quot;free_y&quot;)+
  ggtitle(&quot;RNA log 2 cpm of expressed gene&quot;)+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab(&quot;log2 cpm RNA&quot;)</code></pre>
<p><img src="figure/Supp_Fig_12-19.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/77a569b5edb2bb41d7502bf700fdc8d40477fff2/docs/figure/Supp_Fig_12-19.Rmd/unnamed-chunk-3-1.png" target="_blank">77a569b</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] eulerr_7.0.2                            
 [2] vargen_0.2.3                            
 [3] devtools_2.4.5                          
 [4] usethis_3.1.0                           
 [5] readxl_1.4.5                            
 [6] smplot2_0.2.5                           
 [7] cowplot_1.1.3                           
 [8] ComplexHeatmap_2.22.0                   
 [9] ggrepel_0.9.6                           
[10] plyranges_1.26.0                        
[11] ggsignif_0.6.4                          
[12] genomation_1.38.0                       
[13] edgeR_4.4.2                             
[14] limma_3.62.2                            
[15] ggpubr_0.6.0                            
[16] BiocParallel_1.40.0                     
[17] ggVennDiagram_1.5.2                     
[18] scales_1.3.0                            
[19] VennDiagram_1.7.3                       
[20] futile.logger_1.4.3                     
[21] gridExtra_2.3                           
[22] ggfortify_0.4.17                        
[23] rtracklayer_1.66.0                      
[24] org.Hs.eg.db_3.20.0                     
[25] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[26] GenomicFeatures_1.58.0                  
[27] AnnotationDbi_1.68.0                    
[28] Biobase_2.66.0                          
[29] GenomicRanges_1.58.0                    
[30] GenomeInfoDb_1.42.3                     
[31] IRanges_2.40.1                          
[32] S4Vectors_0.44.0                        
[33] BiocGenerics_0.52.0                     
[34] ChIPseeker_1.42.1                       
[35] RColorBrewer_1.1-3                      
[36] broom_1.0.7                             
[37] kableExtra_1.4.0                        
[38] lubridate_1.9.4                         
[39] forcats_1.0.0                           
[40] stringr_1.5.1                           
[41] dplyr_1.1.4                             
[42] purrr_1.0.4                             
[43] readr_2.1.5                             
[44] tidyr_1.3.1                             
[45] tibble_3.2.1                            
[46] ggplot2_3.5.1                           
[47] tidyverse_2.0.0                         
[48] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.5                               
  [2] matrixStats_1.5.0                      
  [3] bitops_1.0-9                           
  [4] enrichplot_1.26.6                      
  [5] httr_1.4.7                             
  [6] doParallel_1.0.17                      
  [7] profvis_0.4.0                          
  [8] tools_4.4.2                            
  [9] backports_1.5.0                        
 [10] R6_2.6.1                               
 [11] mgcv_1.9-1                             
 [12] lazyeval_0.2.2                         
 [13] GetoptLong_1.0.5                       
 [14] urlchecker_1.0.1                       
 [15] withr_3.0.2                            
 [16] preprocessCore_1.68.0                  
 [17] cli_3.6.4                              
 [18] formatR_1.14                           
 [19] Cormotif_1.52.0                        
 [20] labeling_0.4.3                         
 [21] sass_0.4.9                             
 [22] Rsamtools_2.22.0                       
 [23] systemfonts_1.2.1                      
 [24] yulab.utils_0.2.0                      
 [25] foreign_0.8-88                         
 [26] DOSE_4.0.0                             
 [27] svglite_2.1.3                          
 [28] R.utils_2.13.0                         
 [29] sessioninfo_1.2.3                      
 [30] plotrix_3.8-4                          
 [31] BSgenome_1.74.0                        
 [32] pwr_1.3-0                              
 [33] rstudioapi_0.17.1                      
 [34] impute_1.80.0                          
 [35] RSQLite_2.3.9                          
 [36] generics_0.1.3                         
 [37] gridGraphics_0.5-1                     
 [38] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [39] shape_1.4.6.1                          
 [40] BiocIO_1.16.0                          
 [41] vroom_1.6.5                            
 [42] gtools_3.9.5                           
 [43] car_3.1-3                              
 [44] GO.db_3.20.0                           
 [45] Matrix_1.7-3                           
 [46] abind_1.4-8                            
 [47] R.methodsS3_1.8.2                      
 [48] lifecycle_1.0.4                        
 [49] whisker_0.4.1                          
 [50] yaml_2.3.10                            
 [51] carData_3.0-5                          
 [52] SummarizedExperiment_1.36.0            
 [53] gplots_3.2.0                           
 [54] qvalue_2.38.0                          
 [55] SparseArray_1.6.2                      
 [56] blob_1.2.4                             
 [57] promises_1.3.2                         
 [58] crayon_1.5.3                           
 [59] miniUI_0.1.1.1                         
 [60] ggtangle_0.0.6                         
 [61] lattice_0.22-6                         
 [62] KEGGREST_1.46.0                        
 [63] magick_2.8.5                           
 [64] pillar_1.10.1                          
 [65] knitr_1.49                             
 [66] fgsea_1.32.2                           
 [67] rjson_0.2.23                           
 [68] boot_1.3-31                            
 [69] codetools_0.2-20                       
 [70] fastmatch_1.1-6                        
 [71] glue_1.8.0                             
 [72] getPass_0.2-4                          
 [73] ggfun_0.1.8                            
 [74] remotes_2.5.0                          
 [75] data.table_1.17.0                      
 [76] vctrs_0.6.5                            
 [77] png_0.1-8                              
 [78] treeio_1.30.0                          
 [79] cellranger_1.1.0                       
 [80] gtable_0.3.6                           
 [81] cachem_1.1.0                           
 [82] xfun_0.51                              
 [83] mime_0.12                              
 [84] S4Arrays_1.6.0                         
 [85] iterators_1.0.14                       
 [86] statmod_1.5.0                          
 [87] ellipsis_0.3.2                         
 [88] nlme_3.1-167                           
 [89] ggtree_3.14.0                          
 [90] bit64_4.6.0-1                          
 [91] rprojroot_2.0.4                        
 [92] bslib_0.9.0                            
 [93] affyio_1.76.0                          
 [94] rpart_4.1.24                           
 [95] KernSmooth_2.23-26                     
 [96] colorspace_2.1-1                       
 [97] DBI_1.2.3                              
 [98] Hmisc_5.2-2                            
 [99] nnet_7.3-20                            
[100] seqPattern_1.38.0                      
[101] tidyselect_1.2.1                       
[102] processx_3.8.6                         
[103] bit_4.6.0                              
[104] compiler_4.4.2                         
[105] curl_6.2.1                             
[106] git2r_0.35.0                           
[107] htmlTable_2.4.3                        
[108] xml2_1.3.7                             
[109] DelayedArray_0.32.0                    
[110] checkmate_2.3.2                        
[111] caTools_1.18.3                         
[112] affy_1.84.0                            
[113] callr_3.7.6                            
[114] digest_0.6.37                          
[115] rmarkdown_2.29                         
[116] XVector_0.46.0                         
[117] htmltools_0.5.8.1                      
[118] pkgconfig_2.0.3                        
[119] base64enc_0.1-3                        
[120] MatrixGenerics_1.18.1                  
[121] fastmap_1.2.0                          
[122] htmlwidgets_1.6.4                      
[123] rlang_1.1.5                            
[124] GlobalOptions_0.1.2                    
[125] UCSC.utils_1.2.0                       
[126] shiny_1.10.0                           
[127] farver_2.1.2                           
[128] jquerylib_0.1.4                        
[129] zoo_1.8-13                             
[130] jsonlite_1.9.1                         
[131] GOSemSim_2.32.0                        
[132] R.oo_1.27.0                            
[133] RCurl_1.98-1.16                        
[134] magrittr_2.0.3                         
[135] Formula_1.2-5                          
[136] GenomeInfoDbData_1.2.13                
[137] ggplotify_0.1.2                        
[138] patchwork_1.3.0                        
[139] munsell_0.5.1                          
[140] Rcpp_1.0.14                            
[141] ape_5.8-1                              
[142] stringi_1.8.4                          
[143] zlibbioc_1.52.0                        
[144] pkgbuild_1.4.6                         
[145] plyr_1.8.9                             
[146] parallel_4.4.2                         
[147] Biostrings_2.74.1                      
[148] splines_4.4.2                          
[149] hms_1.1.3                              
[150] circlize_0.4.16                        
[151] locfit_1.5-9.12                        
[152] ps_1.9.0                               
[153] igraph_2.1.4                           
[154] pkgload_1.4.0                          
[155] reshape2_1.4.4                         
[156] futile.options_1.0.1                   
[157] XML_3.99-0.18                          
[158] evaluate_1.0.3                         
[159] BiocManager_1.30.25                    
[160] lambda.r_1.2.4                         
[161] tzdb_0.4.0                             
[162] foreach_1.5.2                          
[163] httpuv_1.6.15                          
[164] clue_0.3-66                            
[165] gridBase_0.4-7                         
[166] xtable_1.8-4                           
[167] restfulr_0.0.15                        
[168] tidytree_0.4.6                         
[169] rstatix_0.7.2                          
[170] later_1.4.1                            
[171] viridisLite_0.4.2                      
[172] aplot_0.2.5                            
[173] memoise_2.0.1                          
[174] GenomicAlignments_1.42.0               
[175] cluster_2.1.8.1                        
[176] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
