<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Renee Matthews" />

<meta name="date" content="2025-02-26" />

<title>Supplemental Figures 1-11</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Matthews_cardiotox_ATAC_2025</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Supplemental Figures 1-11</h1>
<h4 class="author">Renee Matthews</h4>
<h4 class="date">2025-02-26</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-05-01
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treea197856075034a08febd5e8e91448c9e1053c25etargetblanka197856a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/a197856075034a08febd5e8e91448c9e1053c25e" target="_blank">a197856</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcommwardlabMatthewscardiotoxATAC2025treea197856075034a08febd5e8e91448c9e1053c25etargetblanka197856a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/tree/a197856075034a08febd5e8e91448c9e1053c25e" target="_blank">a197856</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/TFmapnames.RDS
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/datasave.RDS
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summarized_peaks_dataframe.txt
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/

Untracked files:
    Untracked:  analysis/Expressed_RNA_associations.Rmd
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/SVA.Rmd
    Untracked:  analysis/Tan2020.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/just_for_Fun.R
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  setup.RData

Unstaged changes:
    Modified:   ATAC_learning.Rproj
    Modified:   analysis/Correlation_of_SNPnPEAK.Rmd
    Modified:   analysis/GO_KEGG_analysis.Rmd
    Modified:   analysis/Raodah_mycount.Rmd
    Modified:   analysis/TE_analysis_ff.Rmd
    Modified:   analysis/final_plot_attempt.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Supp_Fig_1-11.Rmd</code>) and
HTML (<code>docs/Supp_Fig_1-11.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/b5ac214417ceb405f16f2867c8d83aeb81f33ea2/docs/Supp_Fig_1-11.html" target="_blank">b5ac214</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/58be8ae5925df0c666369b9479a2259c54566eca/analysis/Supp_Fig_1-11.Rmd" target="_blank">58be8ae</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
updates to supplementary files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/ea368e679f92ae9df9756bd152527b2e7568c837/analysis/Supp_Fig_1-11.Rmd" target="_blank">ea368e6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
<td>
updates to supplementary files
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/35ff04fd59899698b3ddb0d0dab9c7259ad8907b/docs/Supp_Fig_1-11.html" target="_blank">35ff04f</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-03-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/Supp_Fig_1-11.html" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/bb8d8a88f3786dc0fb4cf4978f5e72d9cfee80b3/analysis/Supp_Fig_1-11.Rmd" target="_blank">bb8d8a8</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
updates to plot
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/634732c52356d3b833440d2872e05a9194476192/analysis/Supp_Fig_1-11.Rmd" target="_blank">634732c</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
<td>
updates to volcano plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/mward-lab/Matthews_cardiotox_ATAC_2025/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/Supp_Fig_1-11.html" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/785ca3a2346733b1b5b0ac5bf63f5bb471bab7f8/analysis/Supp_Fig_1-11.Rmd" target="_blank">785ca3a</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
<td>
updating supplemental figures
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/faa2861eaabd909901087c889b840bad9c3c3c88/analysis/Supp_Fig_1-11.Rmd" target="_blank">faa2861</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
<td>
end of day
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/66d9e61ce9bd2ac362f363dee71e5d136fdea1a1/analysis/Supp_Fig_1-11.Rmd" target="_blank">66d9e61</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
<td>
first open commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(ggfortify)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(edgeR)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(readxl)
library(devtools)
library(vargen)
library(eulerr)</code></pre>
<div
id="figure-s1-read-numbers-are-similar-across-time-and-drug-treatments."
class="section level3">
<h3>Figure S1: Read numbers are similar across time and drug
treatments.</h3>
<pre class="r"><code>drug_pal &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)
read_summary &lt;- read_delim(file=&quot;data/Final_four_data/reads_summary_FF.txt&quot;,delim=&quot;\t&quot;)
read_summary %&gt;% 
  pivot_longer(., cols=c(total_reads:unique_mapped_reads), names_to = &quot;read_type&quot;,values_to = &quot;counts&quot;) %&gt;% 
  dplyr::mutate(trt=factor(trt, levels = c(&quot;DOX&quot;, &quot;EPI&quot;,&quot;DNR&quot;, &quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  mutate(time=factor(time, levels =c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(indv=gsub(&quot;1&quot;,&quot;D&quot;,indv), indv=gsub(&quot;2&quot;,&quot;A&quot;,indv), indv=gsub(&quot;3&quot;,&quot;B&quot;,indv), indv=gsub(&quot;6&quot;,&quot;C&quot;,indv))%&gt;% 
  mutate(indv=factor(indv, levels=c(&quot;IndD&quot;,&quot;IndA&quot;,&quot;IndB&quot;,&quot;IndC&quot;))) %&gt;% 
  mutate(read_type=factor(read_type, levels =c(&quot;total_reads&quot;,&quot;total_mapped_reads&quot;,&quot;nuclear_mapped_reads&quot;,&quot;unique_mapped_reads&quot;,&quot;nodup_mapped_reads&quot;))) %&gt;% 
  ggplot(., aes(x=read_type, y=counts))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv))+
   theme_bw()+
  facet_wrap(~trt+time,nrow = 3, ncol = 6 )+
 scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  theme(strip.text = element_text(face = &quot;bold&quot;,  hjust = 0, size = 8),
        strip.background = element_rect(fill = &quot;white&quot;, linetype = &quot;solid&quot;,
                                        color = &quot;black&quot;, linewidth = 1),
        panel.spacing = unit(1, &#39;points&#39;),
        axis.text.x=element_text(angle = 90, vjust = 0.5, hjust=1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/S1%20dataframe-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-S1-dataframe-1">
Past versions of "S1 dataframe-1.png"
</button>
</p>
<div id="fig-no-spaces-S1-dataframe-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/S1 dataframe-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-2-peak-numbers-are-similar-across-time-and-drug-treatments."
class="section level3">
<h3>Figure 2: Peak numbers are similar across time and drug
treatments.</h3>
<div id="figure-s2a-read-numbers-across-treatment-and-time"
class="section level4">
<h4>Figure S2A: Read numbers across treatment and time</h4>
<pre class="r"><code>read_summary %&gt;% 
  dplyr::select(sample:time, nodup_mapped_reads) %&gt;% 
  dplyr::mutate(trt=factor(trt, levels = c(&quot;DOX&quot;, &quot;EPI&quot;,&quot;DNR&quot;, &quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  mutate(time=factor(time, levels =c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(indv=gsub(&quot;1&quot;,&quot;D&quot;,indv), 
         indv=gsub(&quot;2&quot;,&quot;A&quot;,indv), 
         indv=gsub(&quot;3&quot;,&quot;B&quot;,indv), 
         indv=gsub(&quot;6&quot;,&quot;C&quot;,indv))%&gt;% 
  mutate(indv=factor(indv, levels=c(&quot;IndD&quot;,&quot;IndA&quot;,&quot;IndB&quot;,&quot;IndC&quot;))) %&gt;% 
  ggplot(., aes(x=trt,y=nodup_mapped_reads,group=(interaction(time,trt))))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
  scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;Reads across treatment and time&quot;)+
  theme_bw()+
  theme(strip.text = element_text(face = &quot;bold&quot;,  hjust = .5, size = 8),
        strip.background = element_rect(fill = &quot;white&quot;, linetype = &quot;solid&quot;,
                                        color = &quot;black&quot;, linewidth = 1),
        panel.spacing = unit(1, &#39;points&#39;))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/fig2a-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-fig2a-1">
Past versions of fig2a-1.png
</button>
</p>
<div id="fig-fig2a-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/fig2a-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="figure-s2b-peak-number-across-treatment-and-time"
class="section level4">
<h4>Figure S2B: Peak number across treatment and time</h4>
<pre class="r"><code>peakcount_ff &lt;- read_delim(&quot;data/Final_four_data/Peak_count_ff.txt&quot;,delim= &quot;\t&quot;)
peakcount_ff %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;% 
   mutate(indv=gsub(&quot;1&quot;,&quot;D&quot;,indv), 
         indv=gsub(&quot;2&quot;,&quot;A&quot;,indv), 
         indv=gsub(&quot;3&quot;,&quot;B&quot;,indv), 
         indv=gsub(&quot;6&quot;,&quot;C&quot;,indv))%&gt;% 
  mutate(indv=factor(indv, levels=c(&quot;D&quot;,&quot;A&quot;,&quot;B&quot;,&quot;C&quot;))) %&gt;% 
   ggplot(., aes(x=trt,y=peak_number))+
  geom_boxplot(aes(fill=trt))+
   geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
     scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;Peaks across treatment and time&quot;)+
  theme_bw()+
  theme(strip.text = element_text(face = &quot;bold&quot;,  hjust = .5, size = 8),
        strip.background = element_rect(fill = &quot;white&quot;, linetype = &quot;solid&quot;,
                                        color = &quot;black&quot;, linewidth = 1),
        panel.spacing = unit(1, &#39;points&#39;))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/Figs2b-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Figs2b-1">
Past versions of Figs2b-1.png
</button>
</p>
<div id="fig-Figs2b-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/b5ac214417ceb405f16f2867c8d83aeb81f33ea2/docs/figure/Supp_Fig_1-11.Rmd/Figs2b-1.png" target="_blank">b5ac214</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2025-03-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/Figs2b-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="figure-s3-samples-have-a-high-fraction-of-read-fragments-in-high-confidence-open-chromatin-regions."
class="section level3">
<h3>Figure S3: Samples have a high fraction of read-fragments in
high-confidence open chromatin regions.</h3>
<pre class="r"><code>frip_newpeaks &lt;- c(38.8,36.3,46.0,38.9,49.6,40.0,39.2,30.2,52.1,39.8,51.1,28.0,
                   42.3,40.3,39.7,38.7,37.9,36.6,36.0,48.7,50.4,44.2,52.0,31.9,
                   40.5,34.1,41.2,33.7,43.5,28.6,34.7,42.8,38.1,40.3,44.6,26.4,
                   46.5,23.9,46.9,25.8,46.7,23.8,21.8,39.2,33.2,22.8,36.8,34.8)
peakcount_ff$frip_newpeaks &lt;- frip_newpeaks

peakcount_ff %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) %&gt;% 
   mutate(indv=gsub(&quot;1&quot;,&quot;D&quot;,indv), 
         indv=gsub(&quot;2&quot;,&quot;A&quot;,indv), 
         indv=gsub(&quot;3&quot;,&quot;B&quot;,indv), 
         indv=gsub(&quot;6&quot;,&quot;C&quot;,indv))%&gt;% 
  mutate(indv=factor(indv, levels=c(&quot;D&quot;,&quot;A&quot;,&quot;B&quot;,&quot;C&quot;))) %&gt;% 
   ggplot(., aes(x=trt,y=frip_newpeaks))+
  geom_boxplot(aes(fill=trt))+
   geom_point(aes(col=indv, size =3))+
  geom_hline(aes(yintercept = 20), linetype=2, color=&quot;red&quot;)+
  facet_wrap(time~.)+
     scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = &quot;Dark2&quot;)+
  ggtitle(&quot;Fraction of fragments in high-confidence regions&quot;)+
  theme_bw()+
  theme(strip.text = element_text(face = &quot;bold&quot;,  hjust = .5, size = 8),
        strip.background = element_rect(fill = &quot;white&quot;, linetype = &quot;solid&quot;,
                                        color = &quot;black&quot;, linewidth = 1),
        panel.spacing = unit(1, &#39;points&#39;))+
  coord_cartesian(ylim = c(0,100))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/figure/Supp_Fig_1-11.Rmd/unnamed-chunk-2-1.png" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s4-ipsc-cm-open-chromatin-regions-are-shared-with-human-heart-left-ventricle-open-chromatin-regions."
class="section level3">
<h3>Figure S4: iPSC-CM open chromatin regions are shared with human
heart-left ventricle open chromatin regions.</h3>
<pre class="r"><code>Snyder_41peaks &lt;- read.delim(&quot;data/other_papers/ENCFF966JZT_bed_Snyder_41peaks.bed&quot;,header=TRUE) %&gt;% 
  GRanges()

filtered_hc_regions &lt;- read_delim(&quot;data/Final_four_data/LCPM_matrix_ff.txt&quot;,delim = &quot;/&quot;) %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  separate_wider_delim(., cols =Peakid,
                       names=c(&quot;chr&quot;,&quot;start&quot;,&quot;end&quot;), 
                       delim = &quot;.&quot;, 
                       cols_remove = FALSE) 

filtered_hc_regions_gr &lt;- filtered_hc_regions %&gt;%
  dplyr::filter(chr!=&quot;chrY&quot;) %&gt;% 
  GRanges() %&gt;% 
  keepStandardChromosomes(., pruning.mode = &quot;coarse&quot;)

heart_overlap &lt;- join_overlap_intersect(Snyder_41peaks,filtered_hc_regions_gr)
 length(unique(heart_overlap$Peakid))</code></pre>
<pre><code>[1] 66927</code></pre>
<pre class="r"><code>fit &lt;- euler(c(&quot;This study&quot; = length(filtered_hc_regions_gr) - length(unique(heart_overlap$Peakid)),        
               &quot;Snyder study&quot; = length(Snyder_41peaks) - length(unique(heart_overlap$name)),        
               &quot;This study&amp;Snyder study&quot; = 66927))
plot(fit, fills = list(fill = c(&quot;skyblue&quot;, &quot;lightcoral&quot;), alpha = 0.6),
     labels = FALSE, edges = TRUE, quantities = TRUE,
     main = &quot;Euler diagram between this study and Snyder&#39;s study&quot;)</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-figS4-1">
Past versions of figS4-1.png
</button>
</p>
<div id="fig-figS4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/figS4-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s5-open-chromatin-regions-are-enriched-at-transcription-start-sites."
class="section level3">
<h3>Figure S5: Open chromatin regions are enriched at transcription
start sites.</h3>
<div id="figure-s5a-enrichment-of-accessible-chromatin-at-tss"
class="section level4">
<h4>Figure S5A: Enrichment of accessible chromatin at TSS</h4>
<pre class="r"><code>## What I did here:  I called all my narrowpeak files made by MACS2 callpeaks

# peakfiles1 &lt;- choose.files()
# 
# ##This loop first established a list then (because I already knew the list had 12 files)
# ## I then imported each of these onto that list.  Once I had the list, I stored it as
# ## an R object, 
# Ind1_peaks &lt;- list()
# for (file in 1:12){
#     testname &lt;- basename(peakfiles1[file])
#   banana_peel &lt;- str_split_i(testname, &quot;_&quot;,3)
#  Ind1_peaks[[banana_peel]] &lt;- readPeakFile(peakfiles1[file])
# }
# saveRDS(Ind4_peaks, &quot;data/Ind4_peaks_list.RDS&quot;)
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

# peakAnnoList_1 &lt;- lapply(Ind1_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_1, &quot;data/peakAnnoList_1.RDS&quot;)



IndD_TSS_peaks_plot &lt;- readRDS(&quot;data/Ind1_TSS_peaks.RDS&quot;)
IndA_TSS_peaks_plot &lt;- readRDS(&quot;data/Ind2_TSS_peaks.RDS&quot;)
IndB_TSS_peaks_plot &lt;- readRDS(&quot;data/Ind3_TSS_peaks.RDS&quot;)
IndC_TSS_peaks_plot &lt;- readRDS(&quot;data/Ind6_TSS_peaks.RDS&quot;)

d1&lt;- plotAvgProf(IndD_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual D&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:46 AM </code></pre>
<pre class="r"><code>a1 &lt;- plotAvgProf(IndA_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual A&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:47 AM </code></pre>
<pre class="r"><code>b1 &lt;- plotAvgProf(IndB_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual B&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:48 AM </code></pre>
<pre class="r"><code>c1 &lt;- plotAvgProf(IndC_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;3 hour Individual C&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:49 AM </code></pre>
<pre class="r"><code>d2 &lt;- plotAvgProf(IndD_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual D&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:50 AM </code></pre>
<pre class="r"><code>a2 &lt;- plotAvgProf(IndA_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual A&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:52 AM </code></pre>
<pre class="r"><code>b2 &lt;- plotAvgProf(IndB_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual B&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:53 AM </code></pre>
<pre class="r"><code>c2 &lt;- plotAvgProf(IndC_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = &quot;Count Frequency&quot;)+ ggtitle(&quot;24 hour Individual C&quot; )+coord_cartesian(xlim=c(-2000,2000))</code></pre>
<pre><code>&gt;&gt; plotting figure...            2025-05-01 10:38:54 AM </code></pre>
<pre class="r"><code>plot_grid(a1,a2, b1,b2,c1,c2,d1,d2, axis=&quot;l&quot;,align = &quot;hv&quot;,nrow=4, ncol=2)</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS5A-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-figS5A-1">
Past versions of figS5A-1.png
</button>
</p>
<div id="fig-figS5A-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/figS5A-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Code used to calculate fig S5B enrichment scores</p>
<pre class="r"><code>library(GenomicRanges)
library(ATACseqQC)
bamfilelist &lt;- choose.files()

list1 &lt;- lapply(bamfilelist, readBamFile,bigFile=TRUE)
# bamfilenames &lt;- lapply(bamfilelist, basename)

# gal1 &lt;- readBamFile(bamFile=bamfile, tag=character(0),
                                # asMates=FALSE)
library(TxDb.Hsapiens.UCSC.hg38.knownGene)
txs &lt;- transcripts(TxDb.Hsapiens.UCSC.hg38.knownGene)
# tsse &lt;- TSSEscore(gal1, txs)
indA_TSSE &lt;-  lapply(list1,TSSEscore, txs=txs)

# saveRDS(indC_TSSE, &quot;data/Final_four_data/H3K27ac_files/indC_TSSE.RDS&quot;)
# saveRDS(indB_TSSE, &quot;data/Final_four_data/H3K27ac_files/indB_TSSE.RDS&quot;)
# saveRDS(indA_TSSE, &quot;data/Final_four_data/H3K27ac_files/indA_TSSE.RDS&quot;)
# saveRDS(ind6_TSSE, &quot;data/ind6_TSSE.RDS&quot;)
# saveRDS(ind4_TSSE, &quot;data/ind4_TSSE.RDS&quot;)
# saveRDS(ind5_TSSE, &quot;data/ind5_TSSE.RDS&quot;)
# saveRDS(ind2_TSSE, &quot;data/ind2_TSSE.RDS&quot;)
# saveRDS(ind3_TSSE, &quot;data/ind3_TSSE.RDS&quot;)
# saveRDS(ind1_TSSE,&quot;data/ind1_TSSE.RDS&quot;)
# ind1_TSSE &lt;- tribble(
#   ~sample, ~TSSE,
#   &quot;1_DNR_3&quot;,16.89282,
# &quot;1_DOX_3&quot;,19.43605,
# &quot;1_EPI_3&quot;,18.97398,
# &quot;1_MTX_3&quot;,14.93388,
# &quot;1_TRZ_3&quot;,21.0788,
# &quot;1_VEH_3&quot;,12.46743,
# &quot;1_DNR_24&quot;,16.56416,
# &quot;1_DOX_24&quot;,21.6031,
# &quot;1_EPI_24&quot;, 21.75785,
# &quot;1_MTX_24&quot;,17.63624,
# &quot;1_TRZ_24&quot;, 28.37166,
# &quot;1_VEH_24&quot;,34.34781)
##now I can ccombine them all!

ind1_TSSE &lt;- readRDS(&quot;data/ind1_TSSE.RDS&quot;)
ind2_TSSE &lt;- readRDS(&quot;data/ind2_TSSE.RDS&quot;)
ind3_TSSE &lt;- readRDS(&quot;data/ind3_TSSE.RDS&quot;)
ind4_TSSE &lt;- readRDS(&quot;data/ind4_TSSE.RDS&quot;)
ind5_TSSE &lt;- readRDS(&quot;data/ind5_TSSE.RDS&quot;)
ind6_TSSE &lt;- readRDS(&quot;data/ind6_TSSE.RDS&quot;)


ind1 &lt;- lapply(ind1_TSSE, &#39;[[&#39;,2)
names(ind1) &lt;- c(&quot;1_DNR_3&quot;, &quot;1_DNR_24&quot;,&quot;1_DOX_3&quot;,
&quot;1_DOX_24&quot;,&quot;1_EPI_3&quot;,&quot;1_EPI_24&quot;,&quot;1_MTX_3&quot;,
&quot;1_MTX_24&quot;,&quot;1_TRZ_3&quot; , &quot;1_TRZ_24&quot;,&quot;1_VEH_3&quot;,&quot;1_VEH_24&quot;)

ind1 &lt;- lapply(ind1_TSSE, &#39;[[&#39;,2)
names(ind1) &lt;- c(&quot;1_DNR_3&quot;, &quot;1_DNR_24&quot;,&quot;1_DOX_3&quot;,
                 &quot;1_DOX_24&quot;,&quot;1_EPI_3&quot;,&quot;1_EPI_24&quot;,&quot;1_MTX_3&quot;,
                 &quot;1_MTX_24&quot;,&quot;1_TRZ_3&quot; , &quot;1_TRZ_24&quot;,&quot;1_VEH_3&quot;,&quot;1_VEH_24&quot;)

ind2 &lt;- lapply(ind2_TSSE, &#39;[[&#39;,2)
names(ind2) &lt;- c(&quot;2_DNR_3&quot;, &quot;2_DNR_24&quot;,&quot;2_DOX_3&quot;,
                 &quot;2_DOX_24&quot;,&quot;2_EPI_3&quot;,&quot;2_EPI_24&quot;,&quot;2_MTX_3&quot;,
                 &quot;2_MTX_24&quot;,&quot;2_TRZ_3&quot; , &quot;2_TRZ_24&quot;,&quot;2_VEH_3&quot;,&quot;2_VEH_24&quot;)


ind3 &lt;- lapply(ind3_TSSE, &#39;[[&#39;,2)
names(ind3) &lt;- c(&quot;3_DNR_3&quot;, &quot;3_DNR_24&quot;,&quot;3_DOX_3&quot;,
                 &quot;3_DOX_24&quot;,&quot;3_EPI_3&quot;,&quot;3_EPI_24&quot;,&quot;3_MTX_3&quot;,
                 &quot;3_MTX_24&quot;,&quot;3_TRZ_3&quot; , &quot;3_TRZ_24&quot;,&quot;3_VEH_3&quot;,&quot;3_VEH_24&quot;)


ind4 &lt;- lapply(ind4_TSSE, &#39;[[&#39;,2)
names(ind4) &lt;- c(&quot;4_DNR_3&quot;, &quot;4_DNR_24&quot;,&quot;4_DOX_3&quot;,
                 &quot;4_DOX_24&quot;,&quot;4_EPI_3&quot;,&quot;4_EPI_24&quot;,&quot;4_MTX_3&quot;,
                 &quot;4_MTX_24&quot;,&quot;4_TRZ_3&quot; , &quot;4_TRZ_24&quot;,&quot;4_VEH_3&quot;,&quot;4_VEH_24&quot;)


ind5 &lt;- lapply(ind5_TSSE, &#39;[[&#39;,2)
names(ind5) &lt;- c(&quot;5_DNR_3&quot;, &quot;5_DNR_24&quot;,&quot;5_DOX_3&quot;,
                 &quot;5_DOX_24&quot;,&quot;5_EPI_3&quot;,&quot;5_EPI_24&quot;,&quot;5_MTX_3&quot;,
                 &quot;5_MTX_24&quot;,&quot;5_TRZ_3&quot; , &quot;5_TRZ_24&quot;,&quot;5_VEH_3&quot;,&quot;5_VEH_24&quot;)


ind6 &lt;- lapply(ind6_TSSE, &#39;[[&#39;,2)
names(ind6) &lt;- c(&quot;6_DNR_3&quot;, &quot;6_DNR_24&quot;,&quot;6_DOX_3&quot;,
                 &quot;6_DOX_24&quot;,&quot;6_EPI_3&quot;,&quot;6_EPI_24&quot;,&quot;6_MTX_3&quot;,
                 &quot;6_MTX_24&quot;,&quot;6_TRZ_3&quot; , &quot;6_TRZ_24&quot;,&quot;6_VEH_3&quot;,&quot;6_VEH_24&quot;)
allTSSE &lt;- c(ind1, ind2, ind3, ind4, ind5, ind6)

allTSSE &lt;- do.call(rbind, allTSSE)
saveRDS(allTSSE, &quot;data/all_TSSE_scores.RDS&quot;)


############################################################
###Adding H3K27 combos

indC_TSSE &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/indC_TSSE.RDS&quot;)
indB_TSSE &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/indB_TSSE.RDS&quot;)
indA_TSSE &lt;- readRDS(&quot;data/Final_four_data/H3K27ac_files/indA_TSSE.RDS&quot;)


indA &lt;- lapply(indA_TSSE, &#39;[[&#39;,2)
names(indA) &lt;- c(&quot;A_DNR_3&quot;, &quot;A_DNR_24&quot;,&quot;A_DOX_3&quot;,
                 &quot;A_DOX_24&quot;,&quot;A_MTX_3&quot;,
                 &quot;A_MTX_24&quot;,&quot;A_VEH_3&quot;,&quot;A_VEH_24&quot;)

indB &lt;- lapply(indB_TSSE, &#39;[[&#39;,2)
names(indB) &lt;- c(&quot;B_DNR_3&quot;, &quot;B_DNR_24&quot;,&quot;B_DOX_3&quot;,&quot;B_EPI_3&quot;,
                 &quot;B_EPI_24&quot;,&quot;B_MTX_24&quot;,&quot;B_VEH_3&quot;,&quot;B_VEH_24&quot;)

indC &lt;- lapply(indC_TSSE, &#39;[[&#39;,2)
names(indC) &lt;- c(&quot;C_DNR_3&quot;, &quot;C_DNR_24&quot;,&quot;C_DOX_24&quot;,&quot;C_EPI_3&quot;,
                 &quot;C_EPI_24&quot;,&quot;C_MTX_3&quot;,&quot;C_MTX_24&quot;,&quot;C_VEH_3&quot;,&quot;C_VEH_24&quot;)

allTSSE_ac &lt;- c(indA, indB, indC)

allTSSE_ac &lt;- do.call(rbind, allTSSE_ac)
saveRDS(allTSSE_ac, &quot;data/Final_four_data/H3K27ac_files/H3K27ac_TSSE_scores.RDS&quot;)</code></pre>
</div>
<div id="figure-s5b-tss-enrichement-scores" class="section level4">
<h4>Figure S5B: TSS enrichement scores</h4>
<pre class="r"><code>allTSSE &lt;- readRDS( &quot;data/all_TSSE_scores.RDS&quot;)
allTSSE %&gt;% as.data.frame() %&gt;% 
  rownames_to_column(&quot;sample&quot;) %&gt;% 
  separate(sample, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;%
  mutate(trt= factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3&quot;,&quot;24&quot;),labels = c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  dplyr::filter(indv !=4 &amp;indv !=5) %&gt;% 
  mutate(indv=gsub(&quot;1&quot;,&quot;D&quot;,indv), 
         indv=gsub(&quot;2&quot;,&quot;A&quot;,indv), 
         indv=gsub(&quot;3&quot;,&quot;B&quot;,indv), 
         indv=gsub(&quot;6&quot;,&quot;C&quot;,indv))%&gt;% 
  ggplot(., aes(x= time, y= V1, group = indv))+
  geom_jitter(aes(col = trt, size = 1.5, alpha = 0.5) ,  position=position_jitter(0.25))+
  geom_hline(yintercept=5, linetype = 3)+
    geom_hline(yintercept=7, col = &quot;blue&quot;)+
  facet_wrap(~indv)+
   theme_bw()+
  ylab(&quot;score&quot;)+
  ggtitle(&quot;TSS enrichment scores&quot;)+
   scale_color_manual(values=drug_pal)+
   theme(strip.text = element_text(face = &quot;bold&quot;,  hjust = .5, size = 8),
        strip.background = element_rect(fill = &quot;white&quot;, linetype = &quot;solid&quot;,
                                        color = &quot;black&quot;, linewidth = 1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS5B-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-figS5B-1">
Past versions of figS5B-1.png
</button>
</p>
<div id="fig-figS5B-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/figS5B-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="figure-s6-genome-coverage-is-similar-across-samples-at-the-tss-of-the-cardiac-gene-tnnt2."
class="section level3">
<h3>Figure S6: Genome coverage is similar across samples at the TSS of
the cardiac gene TNNT2.</h3>
<pre class="r"><code>knitr::include_graphics(&quot;assets/Fig\ S6.png&quot;, error=FALSE)</code></pre>
<p><img src="assets/Fig S6.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Fig-S6">
Past versions of "Fig S6.png"
</button>
</p>
<div id="fig-no-spaces-Fig-S6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/50f3de933860a0ff78053606691ee072e12461ed/docs/assets/Fig S6.png" target="_blank">50f3de9</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>knitr::include_graphics(&quot;docs/assets/Fig\ S6.png&quot;,error = FALSE)</code></pre>
<p><img src="docs/assets/Fig S6.png" style="display: block; margin: auto;" />
### Figure S7: ATAC-seq samples cluster by time and treatment.</p>
<pre class="r"><code>ATAC_counts &lt;- readRDS(&quot;data/Final_four_data/ATAC_filtered_raw_counts_allsamples.RDS&quot;) %&gt;%   as.data.frame() %&gt;% 
  rename_with(.,~gsub(pattern = &quot;Ind1_75&quot;, replacement = &quot;D_&quot;,.)) %&gt;% 
  rename_with(.,~gsub(pattern = &quot;Ind2_87&quot;, replacement = &quot;A_&quot;,.)) %&gt;% 
  rename_with(.,~gsub(pattern = &quot;Ind3_77&quot;, replacement = &quot;B_&quot;,.)) %&gt;% 
  rename_with(.,~gsub(pattern = &quot;Ind6_71&quot;, replacement = &quot;C_&quot;,.)) %&gt;% 
  rename_with(.,~gsub( &quot;DX&quot; ,&#39;DOX&#39;,.)) %&gt;% 
  rename_with(.,~gsub( &quot;DA&quot; ,&#39;DNR&#39;,.)) %&gt;% 
  rename_with(.,~gsub( &quot;E&quot; ,&#39;EPI&#39;,.)) %&gt;% 
  rename_with(.,~gsub( &quot;T&quot; ,&#39;TRZ&#39;,.)) %&gt;%
  rename_with(.,~gsub( &quot;M&quot; ,&#39;MTX&#39;,.)) %&gt;% 
  rename_with(.,~gsub( &quot;V&quot; ,&#39;VEH&#39;,.)) %&gt;% 
  rename_with(.,~gsub(&quot;24h&quot;,&quot;_24h&quot;,.)) %&gt;% 
  rename_with(.,~gsub(&quot;3h&quot;,&quot;_3h&quot;,.)) %&gt;% 
  cpm(., log = TRUE) 

FCmatrix_full &lt;-   ATAC_counts %&gt;%
  as.matrix() %&gt;% 
  cor()

filmat_groupmat_col &lt;- data.frame(timeset = colnames(FCmatrix_full))

counts_corr_mat &lt;-filmat_groupmat_col %&gt;%
  # mutate(sample = timeset) %&gt;% 
  separate(timeset, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), sep= &quot;_&quot;) %&gt;% 
  mutate(class = if_else(trt == &quot;DNR&quot;, &quot;AC&quot;, 
                         if_else(trt == &quot;DOX&quot;, &quot;AC&quot;, 
                                 if_else(trt == &quot;EPI&quot;, &quot;AC&quot;, &quot;nAC&quot;)))) %&gt;%
  mutate(TOP2i = if_else(trt == &quot;DNR&quot;, &quot;yes&quot;, 
                         if_else(trt == &quot;DOX&quot;, &quot;yes&quot;, 
                                 if_else(trt == &quot;EPI&quot;, &quot;yes&quot;, 
                                         if_else(trt == &quot;MTX&quot;, &quot;yes&quot;, &quot;no&quot;))))) 

                         
 mat_colors &lt;- list( 
   trt= c(&quot;#F1B72B&quot;,&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;),
   indv=c(&quot;#1B9E77&quot;, &quot;#D95F02&quot; ,&quot;#7570B3&quot;, &quot;#E6AB02&quot;),
   time=c(&quot;pink&quot;, &quot;chocolate4&quot;),
   class=c(&quot;yellow1&quot;,&quot;darkorange1&quot;), 
   TOP2i =c(&quot;darkgreen&quot;,&quot;lightgreen&quot;))                        
                         
names(mat_colors$trt)   &lt;- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) &lt;- unique(counts_corr_mat$indv)
names(mat_colors$time) &lt;- unique(counts_corr_mat$time)
names(mat_colors$class) &lt;- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) &lt;- unique(counts_corr_mat$TOP2i)

htanno_full &lt;-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat, col = mat_colors)
Heatmap(FCmatrix_full, top_annotation = htanno_full)</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS7-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS7-1">
Past versions of FigS7-1.png
</button>
</p>
<div id="fig-FigS7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS7-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s8-pc1-associates-with-drug-treatment-and-pc2-associates-with-individual."
class="section level3">
<h3>Figure S8: PC1 associates with drug treatment and PC2 associates
with individual.</h3>
<pre class="r"><code>pca_final_four &lt;- (prcomp(t(ATAC_counts), scale. = TRUE))

pca_final_four_anno &lt;- pca_final_four$x %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column(&quot;sample&quot;) %&gt;% 
  separate_wider_delim(., cols =sample,
                       names=c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;), 
                       delim = &quot;_&quot;, 
                       cols_remove = FALSE) %&gt;% 
  mutate(time = factor(time, levels = c(&quot;3h&quot;, &quot;24h&quot;), labels= c(&quot;3 hours&quot;,&quot;24 hours&quot;))) %&gt;% 
  mutate(trt = factor(trt, levels = c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;))) 
pca_plot &lt;-
  function(df, col_var = NULL, shape_var = NULL, title = &quot;&quot;) {
    ggplot(df) + geom_point(aes(
      x = PC1,
      y = PC2,
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = &quot;PC 1&quot;, y = &quot;PC 2&quot;) +
      scale_color_manual(values = c(
        &quot;#8B006D&quot;,
        &quot;#DF707E&quot;,
        &quot;#F1B72B&quot;,
        &quot;#3386DD&quot;,
        &quot;#707031&quot;,
        &quot;#41B333&quot;
      ))
  }
get_regr_pval &lt;- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == &quot;lm&quot;)
  fstat &lt;- summary(mod)$fstatistic
  pval &lt;- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc &lt;- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char &lt;- paste0(&quot;PC&quot;, pc_num)
  # Calculate F-statistic p-value for linear model
  pval &lt;- get_regr_pval(lm(df[[ pc_char]] ~ df[[ fac]]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = &quot;lm&quot;) + labs(title = sprintf(&quot;p-val: %.2f&quot;, pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf(&quot;p-val: %.3f&quot;, pval))
  }
}
  
facs &lt;- c(&quot;indv&quot;, &quot;trt&quot;, &quot;time&quot;)
names(facs) &lt;- c(&quot;Individual&quot;, &quot;Treatment&quot;, &quot;Time&quot;)
drug1 &lt;- c(&quot;DOX&quot;,&quot;EPI&quot;, &quot;DNR&quot;, &quot;MTX&quot;, &quot;TRZ&quot;, &quot;VEH&quot;)##for changing shapes and colors
time &lt;- rep(c(&quot;24h&quot;, &quot;3h&quot;),24) %&gt;% factor(., levels = c(&quot;3h&quot;,&quot;24h&quot;))
##gglistmaking
for (f in facs) {
  # PC1 v PC2
  pca_plot(pca_final_four_anno, col_var = f, shape_var = time,
           title = names(facs)[which(facs == f)])
  # print(last_plot())
  
  # Plot f versus PC1 and PC2
  f_v_pc1 &lt;- arrangeGrob(plot_versus_pc(pca_final_four_anno, 1, f))
  f_v_pc2 &lt;- arrangeGrob(plot_versus_pc(pca_final_four_anno, 2, f))
  grid.arrange(f_v_pc1, f_v_pc2, ncol = 2, top = names(facs)[which(facs == f)])
  # summary(plot_versus_pc(PCA_info_anno_all, 1, f))
  # summary(plot_versus_pc(PCA_info_anno_all, 2, f))
}</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS8-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS8-1">
Past versions of FigS8-1.png
</button>
</p>
<div id="fig-FigS8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS8-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS8-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS8-2">
Past versions of FigS8-2.png
</button>
</p>
<div id="fig-FigS8-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS8-2.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS8-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS8-3">
Past versions of FigS8-3.png
</button>
</p>
<div id="fig-FigS8-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS8-3.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s9-thousands-of-chromatin-regions-show-changes-in-accessibility-in-response-to-top2i-treatment."
class="section level3">
<h3>Figure S9: Thousands of chromatin regions show changes in
accessibility in response to TOP2i treatment.</h3>
<pre class="r"><code>### results from diff analysis check out the final_four_analysis.html file for detailed steps
efit4 &lt;- readRDS(&quot;data/Final_four_data/efit4_filt_bl.RDS&quot;)

V.DNR_3.top= topTable(efit4, coef=1, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_3.top= topTable(efit4, coef=2, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_3.top= topTable(efit4, coef=3, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_3.top= topTable(efit4, coef=4, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_3.top= topTable(efit4, coef=5, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DNR_24.top= topTable(efit4, coef=6, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.DOX_24.top= topTable(efit4, coef=7, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.EPI_24.top= topTable(efit4, coef=8, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.MTX_24.top= topTable(efit4, coef=9, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)
V.TRZ_24.top= topTable(efit4, coef=10, adjust.method=&quot;BH&quot;, number=Inf, sort.by=&quot;p&quot;)

plot_filenames &lt;- c(&quot;V.DNR_3.top&quot;,&quot;V.DOX_3.top&quot;,&quot;V.EPI_3.top&quot;,&quot;V.MTX_3.top&quot;,
                    &quot;V.TRZ_.top&quot;,&quot;V.DNR_24.top&quot;,&quot;V.DOX_24.top&quot;,&quot;V.EPI_24.top&quot;,
                    &quot;V.MTX_24.top&quot;,&quot;V.TRZ_24.top&quot;)
plot_files &lt;- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
                    V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
                    V.MTX_24.top,V.TRZ_24.top)

volcanosig &lt;- function(df, psig.lvl) {
    df &lt;- df %&gt;% 
    mutate(threshold = ifelse(adj.P.Val &gt; psig.lvl, &quot;A&quot;, ifelse(adj.P.Val &lt;= psig.lvl &amp; logFC&lt;=0,&quot;B&quot;,&quot;C&quot;)))
  
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    geom_point(aes(color=threshold))+
       xlab(expression(&quot;Log&quot;[2]*&quot; FC&quot;))+
    ylab(expression(&quot;-log&quot;[10]*&quot;P Value&quot;))+
    scale_color_manual(values = c(&quot;black&quot;, &quot;red&quot;,&quot;blue&quot;))+
    theme_cowplot()+
    ylim(0,20)+
    xlim(-6,6)+
    theme(legend.position = &quot;none&quot;,
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}

v1 &lt;- volcanosig(V.DNR_3.top, 0.05)+ ggtitle(&quot;DNR 3 hour&quot;)
v2 &lt;- volcanosig(V.DNR_24.top, 0.05)+ ggtitle(&quot;DNR 24 hour&quot;)+ylab(&quot;&quot;)
v3 &lt;- volcanosig(V.DOX_3.top, 0.05)+ ggtitle(&quot;DOX 3 hour&quot;)
v4 &lt;- volcanosig(V.DOX_24.top, 0.05)+ ggtitle(&quot;DOX 24 hour&quot;)+ylab(&quot;&quot;)
v5 &lt;- volcanosig(V.EPI_3.top, 0.05)+ ggtitle(&quot;EPI 3 hour&quot;)
v6 &lt;- volcanosig(V.EPI_24.top, 0.05)+ ggtitle(&quot;EPI 24 hour&quot;)+ylab(&quot;&quot;)
v7 &lt;- volcanosig(V.MTX_3.top, 0.05)+ ggtitle(&quot;MTX 3 hour&quot;)
v8 &lt;- volcanosig(V.MTX_24.top, 0.05)+ ggtitle(&quot;MTX 24 hour&quot;)+ylab(&quot;&quot;)
v9 &lt;- volcanosig(V.TRZ_3.top, 0.05)+ ggtitle(&quot;TRZ 3 hour&quot;)
v10 &lt;- volcanosig(V.TRZ_24.top, 0.05)+ ggtitle(&quot;TRZ 24 hour&quot;)+ylab(&quot;&quot;)

plot_grid(v1,v2,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS9%20updated-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-figS9-updated-1">
Past versions of "figS9 updated-1.png"
</button>
</p>
<div id="fig-no-spaces-figS9-updated-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/figure/Supp_Fig_1-11.Rmd/figS9 updated-1.png" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v3,v4,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS9%20updated-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-figS9-updated-2">
Past versions of "figS9 updated-2.png"
</button>
</p>
<div id="fig-no-spaces-figS9-updated-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/figure/Supp_Fig_1-11.Rmd/figS9 updated-2.png" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v5,v6,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS9%20updated-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-figS9-updated-3">
Past versions of "figS9 updated-3.png"
</button>
</p>
<div id="fig-no-spaces-figS9-updated-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/figure/Supp_Fig_1-11.Rmd/figS9 updated-3.png" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v7,v8,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS9%20updated-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-figS9-updated-4">
Past versions of "figS9 updated-4.png"
</button>
</p>
<div id="fig-no-spaces-figS9-updated-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/figure/Supp_Fig_1-11.Rmd/figS9 updated-4.png" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot_grid(v9,v10,  rel_widths =c(.8,1))</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/figS9%20updated-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-figS9-updated-5">
Past versions of "figS9 updated-5.png"
</button>
</p>
<div id="fig-no-spaces-figS9-updated-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/6b0cfc36d7e35fe071ccb5939163a11aa9a2a7df/docs/figure/Supp_Fig_1-11.Rmd/figS9 updated-5.png" target="_blank">6b0cfc3</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="figure-s10-top-differentiall-accesible-regions-are-shared-across-anthracyclines"
class="section level3">
<h3>Figure S10: Top differentiall accesible regions are shared across
anthracyclines</h3>
<div id="figure-s10a-examples-at-3-and-24-hours" class="section level4">
<h4>Figure S10A: examples at 3 and 24 hours</h4>
<pre class="r"><code>DNR_3_top3_ff &lt;- row.names(V.DNR_3.top[1:3,])

log_filt_ff &lt;- ATAC_counts%&gt;% 
  as.data.frame()  
  
row.names(log_filt_ff) &lt;- row.names(ATAC_counts)

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-1">
Past versions of FigS10A-1.png
</button>
</p>
<div id="fig-FigS10A-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_3_top3_ff &lt;- row.names(V.DOX_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-2">
Past versions of FigS10A-2.png
</button>
</p>
<div id="fig-FigS10A-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-2.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_3_top3_ff &lt;- row.names(V.EPI_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-3">
Past versions of FigS10A-3.png
</button>
</p>
<div id="fig-FigS10A-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-3.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_3_top3_ff &lt;- row.names(V.MTX_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-4">
Past versions of FigS10A-4.png
</button>
</p>
<div id="fig-FigS10A-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-4.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_3_top3_ff &lt;- row.names(V.TRZ_3.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% TRZ_3_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 3 hour TRZ&quot;)+
 scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-5">
Past versions of FigS10A-5.png
</button>
</p>
<div id="fig-FigS10A-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-5.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DNR_24_top3_ff &lt;- row.names(V.DNR_24.top[1:3,])



log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DNR_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DNR&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-6">
Past versions of FigS10A-6.png
</button>
</p>
<div id="fig-FigS10A-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-6.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>DOX_24_top3_ff &lt;- row.names(V.DOX_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% DOX_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour DOX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-7">
Past versions of FigS10A-7.png
</button>
</p>
<div id="fig-FigS10A-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-7.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EPI_24_top3_ff &lt;- row.names(V.EPI_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% EPI_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour EPI&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-8">
Past versions of FigS10A-8.png
</button>
</p>
<div id="fig-FigS10A-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-8.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>MTX_24_top3_ff &lt;- row.names(V.MTX_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% MTX_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour MTX&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-9">
Past versions of FigS10A-9.png
</button>
</p>
<div id="fig-FigS10A-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-9.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TRZ_24_top3_ff &lt;- row.names(V.TRZ_24.top[1:3,])

log_filt_ff %&gt;% 
  dplyr::filter(row.names(.) %in% TRZ_24_top3_ff) %&gt;% 
  mutate(Peak = row.names(.)) %&gt;% 
  pivot_longer(cols = !Peak, names_to = &quot;sample&quot;, values_to = &quot;counts&quot;) %&gt;% 
  separate(&quot;sample&quot;, into = c(&quot;indv&quot;,&quot;trt&quot;,&quot;time&quot;)) %&gt;% 
  mutate(time=factor(time, levels = c(&quot;3h&quot;,&quot;24h&quot;))) %&gt;% 
  mutate(trt=factor(trt, levels= c(&quot;DOX&quot;,&quot;EPI&quot;,&quot;DNR&quot;,&quot;MTX&quot;,&quot;TRZ&quot;,&quot;VEH&quot;))) %&gt;% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle(&quot;top 3 DAR in 24 hour TRZ&quot;)+
  scale_fill_manual(values = drug_pal)+
  theme_bw()</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10A-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10A-10">
Past versions of FigS10A-10.png
</button>
</p>
<div id="fig-FigS10A-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10A-10.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="figure-s10b-lfc-correlation" class="section level4">
<h4>Figure S10B: LFC correlation</h4>
<pre class="r"><code>FCmatrix_ff &lt;- subset(efit4$coefficients)

colnames(FCmatrix_ff) &lt;-
  c(&quot;DNR\n3h&quot;,
    &quot;DOX\n3h&quot;,
    &quot;EPI\n3h&quot;,
    &quot;MTX\n3h&quot;,
    &quot;TRZ\n3h&quot;,
    &quot;DNR\n24h&quot;,
    &quot;DOX\n24h&quot;,
    &quot;EPI\n24h&quot;,
    &quot;MTX\n24h&quot;,
    &quot;TRZ\n24h&quot;
     )


mat_col_ff &lt;-
  data.frame(
    time = c(rep(&quot;3 hours&quot;, 5), rep(&quot;24 hours&quot;, 5)),
    class = (c(
      &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;,&quot;nAC&quot;,  &quot;AC&quot;, &quot;AC&quot;, &quot;AC&quot;, &quot;nAC&quot;,&quot;nAC&quot; 
    )))
rownames(mat_col_ff) &lt;- colnames(FCmatrix_ff)

mat_colors_ff &lt;-
  list(
    time = c(&quot;pink&quot;, &quot;chocolate4&quot;),
    class = c(&quot;yellow1&quot;, &quot;lightgreen&quot;))

names(mat_colors_ff$time) &lt;- unique(mat_col_ff$time)
names(mat_colors_ff$class) &lt;- unique(mat_col_ff$class)
# names(mat_colors_FC$TOP2i) &lt;- unique(mat_col_FC$TOP2i)
corrFC_ff &lt;- cor(FCmatrix_ff)

htanno_ff &lt;-  HeatmapAnnotation(df = mat_col_ff, col = mat_colors_ff)
Heatmap(corrFC_ff, top_annotation = htanno_ff)</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS10B-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS10B-1">
Past versions of FigS10B-1.png
</button>
</p>
<div id="fig-FigS10B-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS10B-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="figure-s11-four-differentially-accessible-signatures-capture-the-response-to-treatment-over-time."
class="section level3">
<h3>Figure S11: Four differentially accessible signatures capture the
response to treatment over time.</h3>
<div id="figure-s11a-bayesian-information-criterion"
class="section level4">
<h4>Figure S11A: Bayesian information criterion</h4>
<pre class="r"><code>## Fit limma model using code as it is found in the original cormotif code. It has
## only been modified to add names to the matrix of t values, as well as the
## limma fits

limmafit.default &lt;- function(exprs,groupid,compid) {
  limmafits  &lt;- list()
  compnum    &lt;- nrow(compid)
  genenum    &lt;- nrow(exprs)
  limmat     &lt;- matrix(0,genenum,compnum)
  limmas2    &lt;- rep(0,compnum)
  limmadf    &lt;- rep(0,compnum)
  limmav0    &lt;- rep(0,compnum)
  limmag1num &lt;- rep(0,compnum)
  limmag2num &lt;- rep(0,compnum)

  rownames(limmat)  &lt;- rownames(exprs)
  colnames(limmat)  &lt;- rownames(compid)
  names(limmas2)    &lt;- rownames(compid)
  names(limmadf)    &lt;- rownames(compid)
  names(limmav0)    &lt;- rownames(compid)
  names(limmag1num) &lt;- rownames(compid)
  names(limmag2num) &lt;- rownames(compid)

  for(i in 1:compnum) {
    selid1 &lt;- which(groupid == compid[i,1])
    selid2 &lt;- which(groupid == compid[i,2])
    eset   &lt;- new(&quot;ExpressionSet&quot;, exprs=cbind(exprs[,selid1],exprs[,selid2]))
    g1num  &lt;- length(selid1)
    g2num  &lt;- length(selid2)
    designmat &lt;- cbind(base=rep(1,(g1num+g2num)), delta=c(rep(0,g1num),rep(1,g2num)))
    fit &lt;- lmFit(eset,designmat)
    fit &lt;- eBayes(fit)
    limmat[,i] &lt;- fit$t[,2]
    limmas2[i] &lt;- fit$s2.prior
    limmadf[i] &lt;- fit$df.prior
    limmav0[i] &lt;- fit$var.prior[2]
    limmag1num[i] &lt;- g1num
    limmag2num[i] &lt;- g2num
    limmafits[[i]] &lt;- fit

    # log odds
    # w&lt;-sqrt(1+fit$var.prior[2]/(1/g1num+1/g2num))
    # log(0.99)+dt(fit$t[1,2],g1num+g2num-2+fit$df.prior,log=TRUE)-log(0.01)-dt(fit$t[1,2]/w, g1num+g2num-2+fit$df.prior, log=TRUE)+log(w)
  }
  names(limmafits) &lt;- rownames(compid)
  limmacompnum&lt;-nrow(compid)
  result&lt;-list(t       = limmat,
               v0      = limmav0,
               df0     = limmadf,
               s20     = limmas2,
               g1num   = limmag1num,
               g2num   = limmag2num,
               compnum = limmacompnum,
               fits    = limmafits)
}

limmafit.counts &lt;-
  function (exprs, groupid, compid, norm.factor.method = &quot;TMM&quot;, voom.normalize.method = &quot;none&quot;)
  {
    limmafits  &lt;- list()
    compnum    &lt;- nrow(compid)
    genenum    &lt;- nrow(exprs)
    limmat     &lt;- matrix(NA,genenum,compnum)
    limmas2    &lt;- rep(0,compnum)
    limmadf    &lt;- rep(0,compnum)
    limmav0    &lt;- rep(0,compnum)
    limmag1num &lt;- rep(0,compnum)
    limmag2num &lt;- rep(0,compnum)

    rownames(limmat)  &lt;- rownames(exprs)
    colnames(limmat)  &lt;- rownames(compid)
    names(limmas2)    &lt;- rownames(compid)
    names(limmadf)    &lt;- rownames(compid)
    names(limmav0)    &lt;- rownames(compid)
    names(limmag1num) &lt;- rownames(compid)
    names(limmag2num) &lt;- rownames(compid)

    for (i in 1:compnum) {
      message(paste(&quot;Running limma for comparision&quot;,i,&quot;/&quot;,compnum))
      selid1 &lt;- which(groupid == compid[i, 1])
      selid2 &lt;- which(groupid == compid[i, 2])
      # make a new count data frame
      counts &lt;- cbind(exprs[, selid1], exprs[, selid2])

      # remove NAs
      not.nas &lt;- which(apply(counts, 1, function(x) !any(is.na(x))) == TRUE)

      # runn voom/limma
      d &lt;- DGEList(counts[not.nas,])
      d &lt;- calcNormFactors(d, method = norm.factor.method)
      g1num &lt;- length(selid1)
      g2num &lt;- length(selid2)
      designmat &lt;- cbind(base = rep(1, (g1num + g2num)), delta = c(rep(0,
                                                                       g1num), rep(1, g2num)))

      y &lt;- voom(d, designmat, normalize.method = voom.normalize.method)
      fit &lt;- lmFit(y, designmat)
      fit &lt;- eBayes(fit)

      limmafits[[i]] &lt;- fit
      limmat[not.nas, i] &lt;- fit$t[, 2]
      limmas2[i] &lt;- fit$s2.prior
      limmadf[i] &lt;- fit$df.prior
      limmav0[i] &lt;- fit$var.prior[2]
      limmag1num[i] &lt;- g1num
      limmag2num[i] &lt;- g2num
    }
    limmacompnum &lt;- nrow(compid)
    names(limmafits) &lt;- rownames(compid)
    result &lt;- list(t       = limmat,
                   v0      = limmav0,
                   df0     = limmadf,
                   s20     = limmas2,
                   g1num   = limmag1num,
                   g2num   = limmag2num,
                   compnum = limmacompnum,
                   fits    = limmafits)
  }

limmafit.list &lt;-
  function (fitlist, cmp.idx=2)
  {
    compnum    &lt;- length(fitlist)

    genes &lt;- c()
    for (i in 1:compnum) genes &lt;- unique(c(genes, rownames(fitlist[[i]])))

    genenum    &lt;- length(genes)
    limmat     &lt;- matrix(NA,genenum,compnum)
    limmas2    &lt;- rep(0,compnum)
    limmadf    &lt;- rep(0,compnum)
    limmav0    &lt;- rep(0,compnum)
    limmag1num &lt;- rep(0,compnum)
    limmag2num &lt;- rep(0,compnum)

    rownames(limmat)  &lt;- genes
    colnames(limmat)  &lt;- names(fitlist)
    names(limmas2)    &lt;- names(fitlist)
    names(limmadf)    &lt;- names(fitlist)
    names(limmav0)    &lt;- names(fitlist)
    names(limmag1num) &lt;- names(fitlist)
    names(limmag2num) &lt;- names(fitlist)

    for (i in 1:compnum) {
      this.t &lt;- fitlist[[i]]$t[,cmp.idx]
      limmat[names(this.t),i] &lt;- this.t

      limmas2[i]    &lt;- fitlist[[i]]$s2.prior
      limmadf[i]    &lt;- fitlist[[i]]$df.prior
      limmav0[i]    &lt;- fitlist[[i]]$var.prior[cmp.idx]
      limmag1num[i] &lt;- sum(fitlist[[i]]$design[,cmp.idx]==0)
      limmag2num[i] &lt;- sum(fitlist[[i]]$design[,cmp.idx]==1)
    }

    limmacompnum &lt;- compnum
    result &lt;- list(t       = limmat,
                   v0      = limmav0,
                   df0     = limmadf,
                   s20     = limmas2,
                   g1num   = limmag1num,
                   g2num   = limmag2num,
                   compnum = limmacompnum,
                   fits    = limmafits)

  }

## Rank genes based on statistics
generank&lt;-function(x) {
  xcol&lt;-ncol(x)
  xrow&lt;-nrow(x)
  result&lt;-matrix(0,xrow,xcol)
  z&lt;-(1:1:xrow)
  for(i in 1:xcol) {
    y&lt;-sort(x[,i],decreasing=TRUE,na.last=TRUE)
    result[,i]&lt;-match(x[,i],y)
    result[,i]&lt;-order(result[,i])
  }
  result
}

## Log-likelihood for moderated t under H0
modt.f0.loglike&lt;-function(x,df) {
  a&lt;-dt(x, df, log=TRUE)
  result&lt;-as.vector(a)
  flag&lt;-which(is.na(result)==TRUE)
  result[flag]&lt;-0
  result
}

## Log-likelihood for moderated t under H1
## param=c(df,g1num,g2num,v0)
modt.f1.loglike&lt;-function(x,param) {
  df&lt;-param[1]
  g1num&lt;-param[2]
  g2num&lt;-param[3]
  v0&lt;-param[4]
  w&lt;-sqrt(1+v0/(1/g1num+1/g2num))
  dt(x/w, df, log=TRUE)-log(w)
  a&lt;-dt(x/w, df, log=TRUE)-log(w)
  result&lt;-as.vector(a)
  flag&lt;-which(is.na(result)==TRUE)
  result[flag]&lt;-0
  result
}

## Correlation Motif Fit
cmfit.X&lt;-function(x, type, K=1, tol=1e-3, max.iter=100) {
  ## initialize
  xrow &lt;- nrow(x)
  xcol &lt;- ncol(x)
  loglike0 &lt;- list()
  loglike1 &lt;- list()
  p &lt;- rep(1, K)/K
  q &lt;- matrix(runif(K * xcol), K, xcol)
  q[1, ] &lt;- rep(0.01, xcol)
  for (i in 1:xcol) {
    f0 &lt;- type[[i]][[1]]
    f0param &lt;- type[[i]][[2]]
    f1 &lt;- type[[i]][[3]]
    f1param &lt;- type[[i]][[4]]
    loglike0[[i]] &lt;- f0(x[, i], f0param)
    loglike1[[i]] &lt;- f1(x[, i], f1param)
  }
  condlike &lt;- list()
  for (i in 1:xcol) {
    condlike[[i]] &lt;- matrix(0, xrow, K)
  }
  loglike.old &lt;- -1e+10
  for (i.iter in 1:max.iter) {
    if ((i.iter%%50) == 0) {
      print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations for K=&quot;,
                  K, sep = &quot;&quot;))
    }
    err &lt;- tol + 1
    clustlike &lt;- matrix(0, xrow, K)
    #templike &lt;- matrix(0, xrow, 2)
    templike1 &lt;- rep(0, xrow)
    templike2 &lt;- rep(0, xrow)
    for (j in 1:K) {
      for (i in 1:xcol) {
        templike1 &lt;- log(q[j, i]) + loglike1[[i]]
        templike2 &lt;- log(1 - q[j, i]) + loglike0[[i]]
        tempmax &lt;- Rfast::Pmax(templike1, templike2)

        templike1 &lt;- exp(templike1 - tempmax)
        templike2 &lt;- exp(templike2 - tempmax)

        tempsum &lt;- templike1 + templike2
        clustlike[, j] &lt;- clustlike[, j] + tempmax +
          log(tempsum)
        condlike[[i]][, j] &lt;- templike1/tempsum
      }
      clustlike[, j] &lt;- clustlike[, j] + log(p[j])
    }
    #tempmax &lt;- apply(clustlike, 1, max)
    tempmax &lt;- Rfast::rowMaxs(clustlike, value=TRUE)
    for (j in 1:K) {
      clustlike[, j] &lt;- exp(clustlike[, j] - tempmax)
    }
    #tempsum &lt;- apply(clustlike, 1, sum)
    tempsum &lt;- Rfast::rowsums(clustlike)
    for (j in 1:K) {
      clustlike[, j] &lt;- clustlike[, j]/tempsum
    }
    #p.new &lt;- (apply(clustlike, 2, sum) + 1)/(xrow + K)
    p.new &lt;- (Rfast::colsums(clustlike) + 1)/(xrow + K)
    q.new &lt;- matrix(0, K, xcol)
    for (j in 1:K) {
      clustpsum &lt;- sum(clustlike[, j])
      for (i in 1:xcol) {
        q.new[j, i] &lt;- (sum(clustlike[, j] * condlike[[i]][,
                                                           j]) + 1)/(clustpsum + 2)
      }
    }
    err.p &lt;- max(abs(p.new - p)/p)
    err.q &lt;- max(abs(q.new - q)/q)
    err &lt;- max(err.p, err.q)
    loglike.new &lt;- (sum(tempmax + log(tempsum)) + sum(log(p.new)) +
                      sum(log(q.new) + log(1 - q.new)))/xrow
    p &lt;- p.new
    q &lt;- q.new
    loglike.old &lt;- loglike.new
    if (err &lt; tol) {
      break
    }
  }
  clustlike &lt;- matrix(0, xrow, K)
  for (j in 1:K) {
    for (i in 1:xcol) {
      templike1 &lt;- log(q[j, i]) + loglike1[[i]]
      templike2 &lt;- log(1 - q[j, i]) + loglike0[[i]]
      tempmax &lt;- Rfast::Pmax(templike1, templike2)

      templike1 &lt;- exp(templike1 - tempmax)
      templike2 &lt;- exp(templike2 - tempmax)

      tempsum &lt;- templike1 + templike2
      clustlike[, j] &lt;- clustlike[, j] + tempmax + log(tempsum)
      condlike[[i]][, j] &lt;- templike1/tempsum
    }
    clustlike[, j] &lt;- clustlike[, j] + log(p[j])
  }
  #tempmax &lt;- apply(clustlike, 1, max)
  tempmax &lt;- Rfast::rowMaxs(clustlike, value=TRUE)
  for (j in 1:K) {
    clustlike[, j] &lt;- exp(clustlike[, j] - tempmax)
  }
  #tempsum &lt;- apply(clustlike, 1, sum)
  tempsum &lt;- Rfast::rowsums(clustlike)
  for (j in 1:K) {
    clustlike[, j] &lt;- clustlike[, j]/tempsum
  }
  p.post &lt;- matrix(0, xrow, xcol)
  for (j in 1:K) {
    for (i in 1:xcol) {
      p.post[, i] &lt;- p.post[, i] + clustlike[, j] * condlike[[i]][,
                                                                  j]
    }
  }
  loglike.old &lt;- loglike.old - (sum(log(p)) + sum(log(q) +
                                                    log(1 - q)))/xrow
  loglike.old &lt;- loglike.old * xrow
  result &lt;- list(p.post = p.post, motif.prior = p, motif.q = q,
                 loglike = loglike.old, clustlike=clustlike, condlike=condlike)
}

## Fit using (0,0,...,0) and (1,1,...,1)
cmfitall&lt;-function(x, type, tol=1e-3, max.iter=100) {
  ## initialize
  xrow&lt;-nrow(x)
  xcol&lt;-ncol(x)
  loglike0&lt;-list()
  loglike1&lt;-list()
  p&lt;-0.01

  ## compute loglikelihood
  L0&lt;-matrix(0,xrow,1)
  L1&lt;-matrix(0,xrow,1)
  for(i in 1:xcol) {
    f0&lt;-type[[i]][[1]]
    f0param&lt;-type[[i]][[2]]
    f1&lt;-type[[i]][[3]]
    f1param&lt;-type[[i]][[4]]
    loglike0[[i]]&lt;-f0(x[,i],f0param)
    loglike1[[i]]&lt;-f1(x[,i],f1param)
    L0&lt;-L0+loglike0[[i]]
    L1&lt;-L1+loglike1[[i]]
  }


  ## EM algorithm to get MLE of p and q
  loglike.old &lt;- -1e10
  for(i.iter in 1:max.iter) {
    if((i.iter%%50) == 0) {
      print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations&quot;,sep=&quot;&quot;))
    }
    err&lt;-tol+1

    ## compute posterior cluster membership
    clustlike&lt;-matrix(0,xrow,2)
    clustlike[,1]&lt;-log(1-p)+L0
    clustlike[,2]&lt;-log(p)+L1

    tempmax&lt;-apply(clustlike,1,max)
    for(j in 1:2) {
      clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
    }
    tempsum&lt;-apply(clustlike,1,sum)

    ## update motif occurrence rate
    for(j in 1:2) {
      clustlike[,j]&lt;-clustlike[,j]/tempsum
    }

    p.new&lt;-(sum(clustlike[,2])+1)/(xrow+2)

    ## evaluate convergence
    err&lt;-abs(p.new-p)/p

    ## evaluate whether the log.likelihood increases
    loglike.new&lt;-(sum(tempmax+log(tempsum))+log(p.new)+log(1-p.new))/xrow

    loglike.old&lt;-loglike.new
    p&lt;-p.new

    if(err&lt;tol) {
      break;
    }
  }

  ## compute posterior p
  clustlike&lt;-matrix(0,xrow,2)
  clustlike[,1]&lt;-log(1-p)+L0
  clustlike[,2]&lt;-log(p)+L1

  tempmax&lt;-apply(clustlike,1,max)
  for(j in 1:2) {
    clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
  }
  tempsum&lt;-apply(clustlike,1,sum)

  for(j in 1:2) {
    clustlike[,j]&lt;-clustlike[,j]/tempsum
  }

  p.post&lt;-matrix(0,xrow,xcol)
  for(i in 1:xcol) {
    p.post[,i]&lt;-clustlike[,2]
  }

  ## return

  #calculate back loglikelihood
  loglike.old&lt;-loglike.old-(log(p)+log(1-p))/xrow
  loglike.old&lt;-loglike.old*xrow
  result&lt;-list(p.post=p.post, motif.prior=p, loglike=loglike.old)
}

## Fit each dataset separately
cmfitsep&lt;-function(x, type, tol=1e-3, max.iter=100) {
  ## initialize
  xrow&lt;-nrow(x)
  xcol&lt;-ncol(x)
  loglike0&lt;-list()
  loglike1&lt;-list()
  p&lt;-0.01*rep(1,xcol)
  loglike.final&lt;-rep(0,xcol)

  ## compute loglikelihood
  for(i in 1:xcol) {
    f0&lt;-type[[i]][[1]]
    f0param&lt;-type[[i]][[2]]
    f1&lt;-type[[i]][[3]]
    f1param&lt;-type[[i]][[4]]
    loglike0[[i]]&lt;-f0(x[,i],f0param)
    loglike1[[i]]&lt;-f1(x[,i],f1param)
  }

  p.post&lt;-matrix(0,xrow,xcol)

  ## EM algorithm to get MLE of p
  for(coli in 1:xcol) {
    loglike.old &lt;- -1e10
    for(i.iter in 1:max.iter) {
      if((i.iter%%50) == 0) {
        print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations&quot;,sep=&quot;&quot;))
      }
      err&lt;-tol+1

      ## compute posterior cluster membership
      clustlike&lt;-matrix(0,xrow,2)
      clustlike[,1]&lt;-log(1-p[coli])+loglike0[[coli]]
      clustlike[,2]&lt;-log(p[coli])+loglike1[[coli]]

      tempmax&lt;-apply(clustlike,1,max)
      for(j in 1:2) {
        clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
      }
      tempsum&lt;-apply(clustlike,1,sum)

      ## evaluate whether the log.likelihood increases
      loglike.new&lt;-sum(tempmax+log(tempsum))/xrow

      ## update motif occurrence rate
      for(j in 1:2) {
        clustlike[,j]&lt;-clustlike[,j]/tempsum
      }

      p.new&lt;-(sum(clustlike[,2]))/(xrow)

      ## evaluate convergence
      err&lt;-abs(p.new-p[coli])/p[coli]
      loglike.old&lt;-loglike.new
      p[coli]&lt;-p.new

      if(err&lt;tol) {
        break;
      }
    }

    ## compute posterior p
    clustlike&lt;-matrix(0,xrow,2)
    clustlike[,1]&lt;-log(1-p[coli])+loglike0[[coli]]
    clustlike[,2]&lt;-log(p[coli])+loglike1[[coli]]

    tempmax&lt;-apply(clustlike,1,max)
    for(j in 1:2) {
      clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
    }
    tempsum&lt;-apply(clustlike,1,sum)

    for(j in 1:2) {
      clustlike[,j]&lt;-clustlike[,j]/tempsum
    }

    p.post[,coli]&lt;-clustlike[,2]
    loglike.final[coli]&lt;-loglike.old
  }


  ## return
  loglike.final&lt;-loglike.final*xrow
  result&lt;-list(p.post=p.post, motif.prior=p, loglike=loglike.final)
}

## Fit the full model
cmfitfull&lt;-function(x, type, tol=1e-3, max.iter=100) {
  ## initialize
  xrow&lt;-nrow(x)
  xcol&lt;-ncol(x)
  loglike0&lt;-list()
  loglike1&lt;-list()
  K&lt;-2^xcol
  p&lt;-rep(1,K)/K
  pattern&lt;-rep(0,xcol)
  patid&lt;-matrix(0,K,xcol)

  ## compute loglikelihood
  for(i in 1:xcol) {
    f0&lt;-type[[i]][[1]]
    f0param&lt;-type[[i]][[2]]
    f1&lt;-type[[i]][[3]]
    f1param&lt;-type[[i]][[4]]
    loglike0[[i]]&lt;-f0(x[,i],f0param)
    loglike1[[i]]&lt;-f1(x[,i],f1param)
  }
  L&lt;-matrix(0,xrow,K)
  for(i in 1:K)
  {
    patid[i,]&lt;-pattern
    for(j in 1:xcol) {
      if(pattern[j] &lt; 0.5) {
        L[,i]&lt;-L[,i]+loglike0[[j]]
      } else {
        L[,i]&lt;-L[,i]+loglike1[[j]]
      }
    }

    if(i &lt; K) {
      pattern[xcol]&lt;-pattern[xcol]+1
      j&lt;-xcol
      while(pattern[j] &gt; 1) {
        pattern[j]&lt;-0
        j&lt;-j-1
        pattern[j]&lt;-pattern[j]+1
      }
    }
  }

  ## EM algorithm to get MLE of p and q
  loglike.old &lt;- -1e10
  for(i.iter in 1:max.iter) {
    if((i.iter%%50) == 0) {
      print(paste(&quot;We have run the first &quot;, i.iter, &quot; iterations&quot;,sep=&quot;&quot;))
    }
    err&lt;-tol+1

    ## compute posterior cluster membership
    clustlike&lt;-matrix(0,xrow,K)
    for(j in 1:K) {
      clustlike[,j]&lt;-log(p[j])+L[,j]
    }

    tempmax&lt;-apply(clustlike,1,max)
    for(j in 1:K) {
      clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
    }
    tempsum&lt;-apply(clustlike,1,sum)

    ## update motif occurrence rate
    for(j in 1:K) {
      clustlike[,j]&lt;-clustlike[,j]/tempsum
    }

    p.new&lt;-(apply(clustlike,2,sum)+1)/(xrow+K)

    ## evaluate convergence
    err&lt;-max(abs(p.new-p)/p)

    ## evaluate whether the log.likelihood increases
    loglike.new&lt;-(sum(tempmax+log(tempsum))+sum(log(p.new)))/xrow

    loglike.old&lt;-loglike.new
    p&lt;-p.new

    if(err&lt;tol) {
      break;
    }
  }

  ## compute posterior p
  clustlike&lt;-matrix(0,xrow,K)
  for(j in 1:K) {
    clustlike[,j]&lt;-log(p[j])+L[,j]
  }

  tempmax&lt;-apply(clustlike,1,max)
  for(j in 1:K) {
    clustlike[,j]&lt;-exp(clustlike[,j]-tempmax)
  }
  tempsum&lt;-apply(clustlike,1,sum)

  for(j in 1:K) {
    clustlike[,j]&lt;-clustlike[,j]/tempsum
  }

  p.post&lt;-matrix(0,xrow,xcol)
  for(j in 1:K) {
    for(i in 1:xcol) {
      if(patid[j,i] &gt; 0.5) {
        p.post[,i]&lt;-p.post[,i]+clustlike[,j]
      }
    }
  }

  ## return
  #calculate back loglikelihood
  loglike.old&lt;-loglike.old-sum(log(p))/xrow
  loglike.old&lt;-loglike.old*xrow
  result&lt;-list(p.post=p.post, motif.prior=p, loglike=loglike.old)
}

generatetype&lt;-function(limfitted)
{
  jtype&lt;-list()
  df&lt;-limfitted$g1num+limfitted$g2num-2+limfitted$df0
  for(j in 1:limfitted$compnum)
  {
    jtype[[j]]&lt;-list(f0=modt.f0.loglike, f0.param=df[j], f1=modt.f1.loglike, f1.param=c(df[j],limfitted$g1num[j],limfitted$g2num[j],limfitted$v0[j]))
  }
  jtype
}

cormotiffit &lt;- function(exprs, groupid=NULL, compid=NULL, K=1, tol=1e-3,
                        max.iter=100, BIC=TRUE, norm.factor.method=&quot;TMM&quot;,
                        voom.normalize.method = &quot;none&quot;, runtype=c(&quot;logCPM&quot;,&quot;counts&quot;,&quot;limmafits&quot;), each=3)
{
  # first I want to do some typechecking. Input can be either a normalized
  # matrix, a count matrix, or a list of limma fits. Dispatch the correct
  # limmafit accordingly.
  # todo: add some typechecking here
  limfitted &lt;- list()
  if (runtype==&quot;counts&quot;) {
    limfitted &lt;- limmafit.counts(exprs,groupid,compid, norm.factor.method, voom.normalize.method)
  } else if (runtype==&quot;logCPM&quot;) {
    limfitted &lt;- limmafit.default(exprs,groupid,compid)
  } else if (runtype==&quot;limmafits&quot;) {
    limfitted &lt;- limmafit.list(exprs)
  } else {
    stop(&quot;runtype must be one of &#39;logCPM&#39;, &#39;counts&#39;, or &#39;limmafits&#39;&quot;)
  }


  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-list()
  ks &lt;- rep(K, each = each)
  fitresult &lt;- bplapply(1:length(ks), function(i, x, type, ks, tol, max.iter) {
    cmfit.X(x, type, K = ks[i], tol = tol, max.iter = max.iter)
  }, x=limfitted$t, type=jtype, ks=ks, tol=tol, max.iter=max.iter)

  best.fitresults &lt;- list()
  for (i in 1:length(K)) {
    w.k &lt;- which(ks==K[i])
    this.bic &lt;- c()
    for (j in w.k) this.bic[j] &lt;- -2 * fitresult[[j]]$loglike + (K[i] - 1 + K[i] * limfitted$compnum) * log(dim(limfitted$t)[1])
    w.min &lt;- which(this.bic == min(this.bic, na.rm = TRUE))[1]
    best.fitresults[[i]] &lt;- fitresult[[w.min]]
  }
  fitresult &lt;- best.fitresults

  bic &lt;- rep(0, length(K))
  aic &lt;- rep(0, length(K))
  loglike &lt;- rep(0, length(K))
  for (i in 1:length(K)) loglike[i] &lt;- fitresult[[i]]$loglike
  for (i in 1:length(K)) bic[i] &lt;- -2 * fitresult[[i]]$loglike + (K[i] - 1 + K[i] * limfitted$compnum) * log(dim(limfitted$t)[1])
  for (i in 1:length(K)) aic[i] &lt;- -2 * fitresult[[i]]$loglike + 2 * (K[i] - 1 + K[i] * limfitted$compnum)
  if(BIC==TRUE) {
    bestflag=which(bic==min(bic))
  }
  else {
    bestflag=which(aic==min(aic))
  }
  result&lt;-list(bestmotif=fitresult[[bestflag]],bic=cbind(K,bic),
               aic=cbind(K,aic),loglike=cbind(K,loglike), allmotifs=fitresult)

}

cormotiffitall&lt;-function(exprs,groupid,compid, tol=1e-3, max.iter=100)
{
  limfitted&lt;-limmafit(exprs,groupid,compid)
  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-cmfitall(limfitted$t,type=jtype,tol=1e-3,max.iter=max.iter)
}

cormotiffitsep&lt;-function(exprs,groupid,compid, tol=1e-3, max.iter=100)
{
  limfitted&lt;-limmafit(exprs,groupid,compid)
  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-cmfitsep(limfitted$t,type=jtype,tol=1e-3,max.iter=max.iter)
}

cormotiffitfull&lt;-function(exprs,groupid,compid, tol=1e-3, max.iter=100)
{
  limfitted&lt;-limmafit(exprs,groupid,compid)
  jtype&lt;-generatetype(limfitted)
  fitresult&lt;-cmfitfull(limfitted$t,type=jtype,tol=1e-3,max.iter=max.iter)
}

plotIC&lt;-function(fitted_cormotif)
{
  oldpar&lt;-par(mfrow=c(1,2))
  plot(fitted_cormotif$bic[,1], fitted_cormotif$bic[,2], type=&quot;b&quot;,xlab=&quot;Motif Number&quot;, ylab=&quot;BIC&quot;, main=&quot;BIC&quot;)
  plot(fitted_cormotif$aic[,1], fitted_cormotif$aic[,2], type=&quot;b&quot;,xlab=&quot;Motif Number&quot;, ylab=&quot;AIC&quot;, main=&quot;AIC&quot;)
}

plotMotif&lt;-function(fitted_cormotif,title=&quot;&quot;)
{
  layout(matrix(1:2,ncol=2))
  u&lt;-1:dim(fitted_cormotif$bestmotif$motif.q)[2]
  v&lt;-1:dim(fitted_cormotif$bestmotif$motif.q)[1]
  image(u,v,t(fitted_cormotif$bestmotif$motif.q),
        col=gray(seq(from=1,to=0,by=-0.1)),xlab=&quot;Study&quot;,yaxt = &quot;n&quot;,
        ylab=&quot;Corr. Motifs&quot;,main=paste(title,&quot;pattern&quot;,sep=&quot; &quot;))
  axis(2,at=1:length(v))
  for(i in 1:(length(u)+1))
  {
    abline(v=(i-0.5))
  }
  for(i in 1:(length(v)+1))
  {
    abline(h=(i-0.5))
  }
  Ng=10000
  if(is.null(fitted_cormotif$bestmotif$p.post)!=TRUE)
    Ng=nrow(fitted_cormotif$bestmotif$p.post)
  genecount=floor(fitted_cormotif$bestmotif$motif.p*Ng)
  NK=nrow(fitted_cormotif$bestmotif$motif.q)
  plot(0,0.7,pch=&quot;.&quot;,xlim=c(0,1.2),ylim=c(0.75,NK+0.25),
       frame.plot=FALSE,axes=FALSE,xlab=&quot;No. of genes&quot;,ylab=&quot;&quot;, main=paste(title,&quot;frequency&quot;,sep=&quot; &quot;))
  segments(0,0.7,fitted_cormotif$bestmotif$motif.p[1],0.7)
  rect(0,1:NK-0.3,fitted_cormotif$bestmotif$motif.p,1:NK+0.3,
       col=&quot;dark grey&quot;)
  mtext(1:NK,at=1:NK,side=2,cex=0.8)
  text(fitted_cormotif$bestmotif$motif.p+0.15,1:NK,
       labels=floor(fitted_cormotif$bestmotif$motif.p*Ng))
}

plotMotifnew&lt;-function(fitted_cormotif,title=&quot;&quot;)
{
  layout(matrix(1:2,ncol=2))
  u&lt;-1:dim(fitted_cormotif$motif.q)[2]
  v&lt;-1:dim(fitted_cormotif$motif.q)[1]
  image(u,v,t(fitted_cormotif$motif.q),
        col=gray(seq(from=1,to=0,by=-0.1)),xlab=&quot;Experiment&quot;,yaxt = &quot;n&quot;,
        ylab=&quot;Corr. Motifs&quot;,main=paste(title,&quot;pattern&quot;,sep=&quot; &quot;))
  axis(2,at=1:length(v))
  for(i in 1:(length(u)+1))
  {
    abline(v=(i-0.5))
  }
  for(i in 1:(length(v)+1))
  {
    abline(h=(i-0.5))
  }
  Ng=10000
  if(is.null(fitted_cormotif$p.post)!=TRUE)
    Ng=nrow(fitted_cormotif$p.post)
  genecount=floor(fitted_cormotif$motif.p*Ng)
  NK=nrow(fitted_cormotif$motif.q)
  plot(0,0.7,pch=&quot;.&quot;,xlim=c(0,1.2),ylim=c(0.75,NK+0.25),
       frame.plot=FALSE,axes=FALSE,xlab=&quot;No. of regions&quot;,ylab=&quot;&quot;, main=paste(title,&quot;frequency&quot;,sep=&quot; &quot;))
  segments(0,0.7,fitted_cormotif$motif.p[1],0.7)
  rect(0,1:NK-0.3,fitted_cormotif$motif.p,1:NK+0.3,
       col=&quot;dark grey&quot;)
  mtext(1:NK,at=1:NK,side=2,cex=0.8)
  text(fitted_cormotif$motif.p+0.15,1:NK,
       labels=floor(fitted_cormotif$motif.p*Ng))
}</code></pre>
<pre class="r"><code>group &lt;- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group &lt;- factor(group, levels =c(&quot;1&quot;,&quot;2&quot;,&quot;3&quot;,&quot;4&quot;,&quot;5&quot;,&quot;6&quot;,&quot;7&quot;,&quot;8&quot;,&quot;9&quot;,&quot;10&quot;,&quot;11&quot;,&quot;12&quot;))
group_fac_ff &lt;- group
groupid_ff &lt;- as.numeric(group_fac_ff)

compid_ff &lt;- data.frame(c1= c(2,4,6,8,10,1,3,5,7,9), c2 = c( 12,12,12,12,12,11,11,11,11,11))
y_TMM_cpm_ff &lt;- ATAC_counts

# set.seed(31415)
# cormotif_initial_ff &lt;- cormotiffit(exprs = y_TMM_cpm_ff, groupid = groupid_ff, compid = compid_ff, K=1:8, max.iter = 500, runtype = &quot;logCPM&quot;)
# 
# saveRDS(cormotif_initial_ff,&quot;data/Final_four_data/cormotif_ff_4_run.RDS&quot;)

# cormotif_four_ff &lt;- cormotiffit(exprs = y_TMM_cpm_ff, groupid = groupid_ff, compid = compid_ff, K=4, max.iter = 500, runtype = &quot;logCPM&quot;)

# saveRDS(cormotif_four_ff,&quot;data/Final_four_data/cormotif_only4_run.RDS&quot;)
cormotif_initial_ff &lt;-  readRDS(&quot;data/Final_four_data/cormotif_ff_4_run.RDS&quot;)
Cormotif::plotIC(cormotif_initial_ff)</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS11A-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS11A-1">
Past versions of FigS11A-1.png
</button>
</p>
<div id="fig-FigS11A-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS11A-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="figure-s11b-akaike-information-criterion"
class="section level4">
<h4>Figure S11B: Akaike information criterion</h4>
<pre class="r"><code>Cormotif::plotIC(cormotif_initial_ff)</code></pre>
<p><img src="figure/Supp_Fig_1-11.Rmd/FigS11B-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-FigS11B-1">
Past versions of FigS11B-1.png
</button>
</p>
<div id="fig-FigS11B-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/mward-lab/Matthews_cardiotox_ATAC_2025/blob/e446dec1e6b130ea8e9f149a61310a13ef426bb9/docs/figure/Supp_Fig_1-11.Rmd/FigS11B-1.png" target="_blank">e446dec</a>
</td>
<td>
E. Renee Matthews
</td>
<td>
2025-02-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] eulerr_7.0.2                            
 [2] vargen_0.2.3                            
 [3] devtools_2.4.5                          
 [4] usethis_3.1.0                           
 [5] readxl_1.4.5                            
 [6] smplot2_0.2.5                           
 [7] cowplot_1.1.3                           
 [8] ComplexHeatmap_2.22.0                   
 [9] ggrepel_0.9.6                           
[10] plyranges_1.26.0                        
[11] ggsignif_0.6.4                          
[12] genomation_1.38.0                       
[13] edgeR_4.4.2                             
[14] limma_3.62.2                            
[15] ggpubr_0.6.0                            
[16] BiocParallel_1.40.0                     
[17] ggVennDiagram_1.5.2                     
[18] scales_1.3.0                            
[19] VennDiagram_1.7.3                       
[20] futile.logger_1.4.3                     
[21] gridExtra_2.3                           
[22] ggfortify_0.4.17                        
[23] rtracklayer_1.66.0                      
[24] org.Hs.eg.db_3.20.0                     
[25] TxDb.Hsapiens.UCSC.hg38.knownGene_3.20.0
[26] GenomicFeatures_1.58.0                  
[27] AnnotationDbi_1.68.0                    
[28] Biobase_2.66.0                          
[29] GenomicRanges_1.58.0                    
[30] GenomeInfoDb_1.42.3                     
[31] IRanges_2.40.1                          
[32] S4Vectors_0.44.0                        
[33] BiocGenerics_0.52.0                     
[34] ChIPseeker_1.42.1                       
[35] RColorBrewer_1.1-3                      
[36] broom_1.0.7                             
[37] kableExtra_1.4.0                        
[38] lubridate_1.9.4                         
[39] forcats_1.0.0                           
[40] stringr_1.5.1                           
[41] dplyr_1.1.4                             
[42] purrr_1.0.4                             
[43] readr_2.1.5                             
[44] tidyr_1.3.1                             
[45] tibble_3.2.1                            
[46] ggplot2_3.5.1                           
[47] tidyverse_2.0.0                         
[48] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.5                               
  [2] matrixStats_1.5.0                      
  [3] bitops_1.0-9                           
  [4] enrichplot_1.26.6                      
  [5] httr_1.4.7                             
  [6] doParallel_1.0.17                      
  [7] profvis_0.4.0                          
  [8] tools_4.4.2                            
  [9] backports_1.5.0                        
 [10] R6_2.6.1                               
 [11] lazyeval_0.2.2                         
 [12] GetoptLong_1.0.5                       
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.2                            
 [15] preprocessCore_1.68.0                  
 [16] cli_3.6.4                              
 [17] formatR_1.14                           
 [18] Cormotif_1.52.0                        
 [19] labeling_0.4.3                         
 [20] sass_0.4.9                             
 [21] Rsamtools_2.22.0                       
 [22] systemfonts_1.2.1                      
 [23] yulab.utils_0.2.0                      
 [24] foreign_0.8-88                         
 [25] DOSE_4.0.0                             
 [26] svglite_2.1.3                          
 [27] R.utils_2.13.0                         
 [28] sessioninfo_1.2.3                      
 [29] plotrix_3.8-4                          
 [30] BSgenome_1.74.0                        
 [31] pwr_1.3-0                              
 [32] rstudioapi_0.17.1                      
 [33] impute_1.80.0                          
 [34] RSQLite_2.3.9                          
 [35] generics_0.1.3                         
 [36] gridGraphics_0.5-1                     
 [37] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [38] shape_1.4.6.1                          
 [39] BiocIO_1.16.0                          
 [40] vroom_1.6.5                            
 [41] gtools_3.9.5                           
 [42] car_3.1-3                              
 [43] GO.db_3.20.0                           
 [44] Matrix_1.7-3                           
 [45] abind_1.4-8                            
 [46] R.methodsS3_1.8.2                      
 [47] lifecycle_1.0.4                        
 [48] whisker_0.4.1                          
 [49] yaml_2.3.10                            
 [50] carData_3.0-5                          
 [51] SummarizedExperiment_1.36.0            
 [52] gplots_3.2.0                           
 [53] qvalue_2.38.0                          
 [54] SparseArray_1.6.2                      
 [55] blob_1.2.4                             
 [56] promises_1.3.2                         
 [57] crayon_1.5.3                           
 [58] miniUI_0.1.1.1                         
 [59] ggtangle_0.0.6                         
 [60] lattice_0.22-6                         
 [61] KEGGREST_1.46.0                        
 [62] magick_2.8.5                           
 [63] pillar_1.10.1                          
 [64] knitr_1.49                             
 [65] fgsea_1.32.2                           
 [66] rjson_0.2.23                           
 [67] boot_1.3-31                            
 [68] codetools_0.2-20                       
 [69] fastmatch_1.1-6                        
 [70] glue_1.8.0                             
 [71] getPass_0.2-4                          
 [72] ggfun_0.1.8                            
 [73] remotes_2.5.0                          
 [74] data.table_1.17.0                      
 [75] vctrs_0.6.5                            
 [76] png_0.1-8                              
 [77] treeio_1.30.0                          
 [78] cellranger_1.1.0                       
 [79] gtable_0.3.6                           
 [80] cachem_1.1.0                           
 [81] xfun_0.51                              
 [82] mime_0.12                              
 [83] S4Arrays_1.6.0                         
 [84] iterators_1.0.14                       
 [85] statmod_1.5.0                          
 [86] ellipsis_0.3.2                         
 [87] nlme_3.1-167                           
 [88] ggtree_3.14.0                          
 [89] bit64_4.6.0-1                          
 [90] rprojroot_2.0.4                        
 [91] bslib_0.9.0                            
 [92] affyio_1.76.0                          
 [93] rpart_4.1.24                           
 [94] KernSmooth_2.23-26                     
 [95] colorspace_2.1-1                       
 [96] DBI_1.2.3                              
 [97] Hmisc_5.2-2                            
 [98] nnet_7.3-20                            
 [99] seqPattern_1.38.0                      
[100] tidyselect_1.2.1                       
[101] processx_3.8.6                         
[102] bit_4.6.0                              
[103] compiler_4.4.2                         
[104] curl_6.2.1                             
[105] git2r_0.35.0                           
[106] htmlTable_2.4.3                        
[107] xml2_1.3.7                             
[108] DelayedArray_0.32.0                    
[109] checkmate_2.3.2                        
[110] caTools_1.18.3                         
[111] affy_1.84.0                            
[112] callr_3.7.6                            
[113] digest_0.6.37                          
[114] rmarkdown_2.29                         
[115] XVector_0.46.0                         
[116] htmltools_0.5.8.1                      
[117] pkgconfig_2.0.3                        
[118] base64enc_0.1-3                        
[119] MatrixGenerics_1.18.1                  
[120] fastmap_1.2.0                          
[121] htmlwidgets_1.6.4                      
[122] rlang_1.1.5                            
[123] GlobalOptions_0.1.2                    
[124] UCSC.utils_1.2.0                       
[125] shiny_1.10.0                           
[126] farver_2.1.2                           
[127] jquerylib_0.1.4                        
[128] zoo_1.8-13                             
[129] jsonlite_1.9.1                         
[130] GOSemSim_2.32.0                        
[131] R.oo_1.27.0                            
[132] RCurl_1.98-1.16                        
[133] magrittr_2.0.3                         
[134] Formula_1.2-5                          
[135] GenomeInfoDbData_1.2.13                
[136] ggplotify_0.1.2                        
[137] patchwork_1.3.0                        
[138] munsell_0.5.1                          
[139] Rcpp_1.0.14                            
[140] ape_5.8-1                              
[141] stringi_1.8.4                          
[142] zlibbioc_1.52.0                        
[143] pkgbuild_1.4.6                         
[144] plyr_1.8.9                             
[145] parallel_4.4.2                         
[146] Biostrings_2.74.1                      
[147] splines_4.4.2                          
[148] hms_1.1.3                              
[149] circlize_0.4.16                        
[150] polylabelr_0.3.0                       
[151] locfit_1.5-9.12                        
[152] ps_1.9.0                               
[153] igraph_2.1.4                           
[154] pkgload_1.4.0                          
[155] reshape2_1.4.4                         
[156] futile.options_1.0.1                   
[157] XML_3.99-0.18                          
[158] evaluate_1.0.3                         
[159] BiocManager_1.30.25                    
[160] lambda.r_1.2.4                         
[161] tzdb_0.4.0                             
[162] foreach_1.5.2                          
[163] httpuv_1.6.15                          
[164] polyclip_1.10-7                        
[165] clue_0.3-66                            
[166] gridBase_0.4-7                         
[167] xtable_1.8-4                           
[168] restfulr_0.0.15                        
[169] tidytree_0.4.6                         
[170] rstatix_0.7.2                          
[171] later_1.4.1                            
[172] viridisLite_0.4.2                      
[173] aplot_0.2.5                            
[174] memoise_2.0.1                          
[175] GenomicAlignments_1.42.0               
[176] cluster_2.1.8.1                        
[177] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
