<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-07-02" />

<title>TEs and my data</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">TEs and my data</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-07-02</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-07-02
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree71d70f7eb07e38bf3855614c92d74f65c88c7b59targetblank71d70f7a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/71d70f7eb07e38bf3855614c92d74f65c88c7b59" target="_blank">71d70f7</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree71d70f7eb07e38bf3855614c92d74f65c88c7b59targetblank71d70f7a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/71d70f7eb07e38bf3855614c92d74f65c88c7b59" target="_blank">71d70f7</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/ReHeat_analysis.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/Enrichment_motif.Rmd
    Modified:   analysis/Jaspar_motif.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/TE_analysis.Rmd</code>) and HTML
(<code>docs/TE_analysis.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/71d70f7eb07e38bf3855614c92d74f65c88c7b59/analysis/TE_analysis.Rmd" target="_blank">71d70f7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
<td>
updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/TE_analysis.html" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b60b42cad1a362232dc4b0ce06be8c1cad8de465/analysis/TE_analysis.Rmd" target="_blank">b60b42c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github(&#39;davetang/bedr&#39;)
# library(bedr)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)</code></pre>
<div id="loading-data" class="section level5">
<h5>loading data</h5>
<pre class="r"><code>repeatmasker &lt;- read.delim(&quot;data/other_papers/repeatmasker.tsv&quot;)

TSS_NG_data &lt;- read_delim(&quot;data/n45_bedfiles/TSS_NG_data.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
Collapsed_peaks &lt;- read_delim(&quot;data/n45_bedfiles/TSS_NG_data_collapsed_peaks.tsv&quot;,
                              delim = &quot;\t&quot;, 
                              escape_double = FALSE, 
                              trim_ws = TRUE)
TSS_data_gf &lt;- TSS_NG_data %&gt;% 
  dplyr::filter(seqnames != &quot;chrX&quot;) %&gt;% 
  dplyr::filter(seqnames != &quot;chrY&quot;) %&gt;%
  GRanges()

Col_TSS_data_gr &lt;- Collapsed_peaks %&gt;% 
  dplyr::filter(seqnames != &quot;chrX&quot;) %&gt;% 
  dplyr::filter(seqnames != &quot;chrY&quot;) %&gt;%
  GRanges()

reClass_list &lt;- repeatmasker %&gt;% 
  distinct(repClass)

Line_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

Sine_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

LTR_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LTR&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

DNA_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

retroposon_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


all_TEs_gr &lt;- repeatmasker %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)

EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)


ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_df_gr &lt;- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)</code></pre>
</div>
<div id="te-distrubution" class="section level2">
<h2>TE distrubution:</h2>
<p>This code below was my first attempt at finding how many peaks
overlap more than 50% of TEs from the repeatmasker list. The first
output</p>
<pre class="r"><code># length(retroposon_repeats)
# ##5974
# length(Sine_repeats)
# ##1910631
# length(Line_repeats)
# ## 1614481
# length(LTR_repeats)
# ## 770551
# length(DNA_repeats)
# ## 512404
# intersect_ranges(TSS_data_gf,retroposon_repeats)
# intersect_ranges(TSS_data_gf,Line_repeats)
# intersect_ranges(TSS_data_gf,Sine_repeats)
# intersect_ranges(TSS_data_gf,LTR_repeats)
# intersect_ranges(TSS_data_gf,DNA_repeats)
# intersect_ranges(TSS_data_gf,all_TEs_gr)
fullDF_overlap &lt;- join_overlap_intersect(TSS_data_gf,all_TEs_gr)
fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks by TE class; overlap 1 bp or greater&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks by TE class; overlap 1 bp or greater
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
16528
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
125
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
39199
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
26934
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
253
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5627
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
48
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
334
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
51990
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
227
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
29958
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
280
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
53
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
191
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
403
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>### This is how I subset only those peaks who cover &gt;50% of TEs
hits &lt;- findOverlaps(TSS_data_gf,all_TEs_gr)
overlaps &lt;- pintersect(TSS_data_gf[queryHits(hits)], all_TEs_gr[subjectHits(hits)])
percentOverlap &lt;- width(overlaps) / width(all_TEs_gr[subjectHits(hits)])
hits &lt;- hits[percentOverlap &gt; 0.5]

testingol &lt;- TSS_data_gf[queryHits(hits)]
testingol %&gt;% as.data.frame() %&gt;% 
  left_join(., (fullDF_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;peakid&quot;=&quot;peakid&quot;, &quot;NG_start&quot;=&quot;NG_start&quot;, &quot;end_position&quot;=&quot;end_position&quot;, &quot;entrezgene_id&quot;=&quot;entrezgene_id&quot;, &quot;ensembl_gene_id&quot;=&quot;ensembl_gene_id&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;, &quot;width&quot;=&quot;width&quot;, &quot;strand&quot;=&quot;strand&quot;, &quot;hgnc_symbol&quot; = &quot;hgnc_symbol&quot;)) %&gt;% 
  group_by(repClass) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks by TE class; overlap&gt; 50%&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks by TE class; overlap&gt; 50%
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
268
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
1079
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
1789
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
195
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
103
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
117556
</td>
</tr>
</tbody>
</table>
<p>After the above data frames were run, I realized I ran the data using
a dataframe where a peak is listed more than once because it was
assigned more than one neargene ( one-to-many relationships). I changed
the code to use the ‘collapsed’ data frame. This means the data frame
was simplified to only include peaks one time, but those peaks that were
assigned to more than one neargene had the assigned neargenes condesed
and separated by a comma into the same column to create a one-to-one
relationship dataframe. (yes, wordy i know)</p>
<pre class="r"><code>######################################################
Col_fullDF_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,all_TEs_gr)
Col_fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks by TE class; overlap at least 1 bp; using one:one df &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks by TE class; overlap at least 1 bp; using one:one df
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
16088
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
125
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
38468
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
26306
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
244
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5527
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
45
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
322
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
50933
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
193
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
29334
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
271
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
138
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
297
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>hits_col &lt;- findOverlaps(Col_TSS_data_gr,all_TEs_gr)

overlaps_col &lt;- pintersect(Col_TSS_data_gr[queryHits(hits_col)], all_TEs_gr[subjectHits(hits_col)])
percentOverlap_col &lt;- width(overlaps_col) / width(all_TEs_gr[subjectHits(hits_col)])
hits_col2 &lt;- hits_col[percentOverlap_col &gt; 0.5]

testingol_col2 &lt;- Col_TSS_data_gr[queryHits(hits_col2)]
testingol_col2 %&gt;% as.data.frame() %&gt;% 
  left_join(., (Col_fullDF_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;peakid&quot;=&quot;peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;ensembl_ID&quot;=&quot;ensembl_ID&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;, &quot;width&quot;=&quot;width&quot;, &quot;strand&quot;=&quot;strand&quot;, &quot;SYMBOL&quot; = &quot;SYMBOL&quot;)) %&gt;% 
  group_by(repClass) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks by TE class; overlap&gt; 50%; using one:one df&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks by TE class; overlap&gt; 50%; using one:one df
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
226
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
1020
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
1716
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
188
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
82
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
114966
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>testingol_col2 %&gt;% 
  as.data.frame() %&gt;% 
  group_by(peakid) %&gt;% 
  tally %&gt;% 
  summary()</code></pre>
<pre><code>    peakid                n        
 Length:77759       Min.   : 1.00  
 Class :character   1st Qu.: 1.00  
 Mode  :character   Median : 1.00  
                    Mean   : 1.52  
                    3rd Qu.: 2.00  
                    Max.   :12.00  </code></pre>
<p>The data above shows the unique peak count of TE overlaps, followed
by the table that includes only those TEs that overlap a peak by more
than 50%. The third summary shows how many peaks overlap more than one
TE. Most peaks only overlap 1 TE, but several peaks overlap 2 or more
TEs.</p>
<p>This next data is a breakdown of the number of TEs by response
group.</p>
<pre class="r"><code>TE_mrc_status_list &lt;- testingol_col2 %&gt;% as.data.frame() %&gt;% 
  left_join(., (Col_fullDF_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;peakid&quot;=&quot;peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;ensembl_ID&quot;=&quot;ensembl_ID&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;, &quot;width&quot;=&quot;width&quot;, &quot;strand&quot;=&quot;strand&quot;, &quot;SYMBOL&quot; = &quot;SYMBOL&quot;)) %&gt;% 
    dplyr::select(peakid,repName,repClass,repFamily) %&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;% 
   mutate(mrc=if_else(peakid %in% EAR_df$id, &quot;EAR&quot;,
                     if_else(peakid %in% ESR_df$id,&quot;ESR&quot;,
                             if_else(peakid %in% LR_df$id,&quot;LR&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;))))) 

scale_fill_TE &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;, &quot;#FFFFB3&quot;, &quot;#BEBADA&quot; ,&quot;#FB8072&quot;, &quot;#80B1D3&quot;, &quot;#FDB462&quot;, &quot;#B3DE69&quot;, &quot;#FCCDE5&quot;, &quot;#D9D9D9&quot;,&quot;#BC80BD&quot;, &quot;#CCEBC5&quot;), unique(TE_mrc_status_list$repClass)), 
        ...
    )
}

# saveRDS(TE_mrc_status_list,&quot;data/TE_info/TE_mrc_status_list.RDS&quot;)
TE_ALL_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  count
TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
   geom_label_repel(aes(label = repClass),
                     position = position_stack(vjust = .3),
                     show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks&quot;,subtitle = paste(TE_ALL_count$n))+
  scale_fill_TE()</code></pre>
<p><img src="figure/TE_analysis.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/unnamed-chunk-2-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="te-by-response-cluster" class="section level3">
<h3>TE by response cluster</h3>
<pre class="r"><code>TE_EAR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;EAR&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;EAR&quot;) %&gt;% 
   count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
   geom_label_repel(aes(label = n),
                     position = position_stack(vjust = .3),
                     show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;EAR-TE peak breakdown&quot;,subtitle = paste(TE_EAR_count$n))+
  scale_fill_TE()</code></pre>
<p><img src="figure/TE_analysis.Rmd/EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-EAR-1">
Past versions of EAR-1.png
</button>
</p>
<div id="fig-EAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/EAR-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>LTR_family &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;LTR&quot;) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
    distinct(repFamily) 

scale_fill_LTR_family &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;, &quot;#FFFFB3&quot;, &quot;#BEBADA&quot; ,&quot;#FB8072&quot;, &quot;#80B1D3&quot;, &quot;#FDB462&quot;, &quot;#B3DE69&quot;, &quot;#FCCDE5&quot;), unique(LTR_family$repFamily)), 
        ...
    )
}

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;LTR&quot;) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
   geom_label_repel(aes(label = n),
                     position = position_stack(vjust = .3),
                     show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;EAR-LTR peak breakdown&quot;,subtitle = paste(TE_EAR_count$n))+
  scale_fill_LTR_family()</code></pre>
<p><img src="figure/TE_analysis.Rmd/EAR-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>TE_ESR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;ESR&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;ESR&quot;) %&gt;% 
   count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
   geom_label_repel(aes(label = n),
                     position = position_stack(vjust = .3),
                     show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;ESR-TE peak breakdown&quot;, subtitle = paste(TE_ESR_count$n))+
   scale_fill_TE()</code></pre>
<p><img src="figure/TE_analysis.Rmd/ESR-te-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-1">
Past versions of ESR-te-1.png
</button>
</p>
<div id="fig-ESR-te-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/ESR-te-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_LR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;LR&quot;) %&gt;% 
  count
TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;LR&quot;) %&gt;% 
   count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
   geom_label_repel(aes(label = n),
                     position = position_stack(vjust = .3),
                     show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;LR-TE peak breakdown&quot;,subtitle=paste(TE_LR_count$n))+
   scale_fill_TE()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LRTE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LRTE-1">
Past versions of LRTE-1.png
</button>
</p>
<div id="fig-LRTE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/LRTE-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_NR_count &lt;- TE_mrc_status_list %&gt;% 
  #  mutate(repClass=factor(repClass)) %&gt;% 
  # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;NR&quot;) %&gt;% 
  count
TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;mrc==&quot;NR&quot;) %&gt;% 
   count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
   geom_label_repel(aes(label = n),
                     position = position_stack(vjust = .3),
                     show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;NR-TE peak breakdown&quot;, subtitle=paste(TE_NR_count$n))+
   scale_fill_TE()</code></pre>
<p><img src="figure/TE_analysis.Rmd/NR-te-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-NR-te-1">
Past versions of NR-te-1.png
</button>
</p>
<div id="fig-NR-te-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/NR-te-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="human-genome-te-breakdown" class="section level3">
<h3>Human genome TE breakdown</h3>
<pre class="r"><code>scale_fill_repeat &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;,
                             &quot;#FFFFB3&quot;,
                             &quot;#BEBADA&quot; ,
                             &quot;#FB8072&quot;,
                             &quot;#80B1D3&quot;,
                             &quot;#FDB462&quot;,
                             &quot;#B3DE69&quot;,
                             &quot;#FCCDE5&quot;,
                             &quot;#D9D9D9&quot;,
                             &quot;#BC80BD&quot;,
                             &quot;#CCEBC5&quot;,
                             &quot;pink4&quot;,
                             &quot;cornflowerblue&quot;,
                             &quot;chocolate&quot;,
                             &quot;brown&quot;,
                             &quot;green&quot;,
                             &quot;yellow4&quot;,
                             &quot;purple&quot;,
                             &quot;darkorchid4&quot;,
                             &quot;coral4&quot;,
                             &quot;darkolivegreen4&quot;,
                             &quot;darkorange&quot;), unique(repeatmasker$repClass)), 
      ...
    )
}


repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/repeatmasker%20genome%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
  #                    position = position_stack(vjust = .5),
  #                    force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown without labels&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/repeatmasker%20genome%20breakdown-2.png" width="672" style="display: block; margin: auto;" />
#### Line repeats</p>
<pre class="r"><code>Line_df &lt;- Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))

scale_fill_lines &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;,
                             &quot;#FFFFB3&quot;,
                             &quot;#BEBADA&quot; ,
                             &quot;#FB8072&quot;,
                             &quot;#80B1D3&quot;,
                             &quot;#FDB462&quot;,
                             &quot;#B3DE69&quot;,
                             &quot;#FCCDE5&quot;,
                             &quot;#D9D9D9&quot;,
                             &quot;#BC80BD&quot;,
                             &quot;#CCEBC5&quot;,
                             &quot;pink4&quot;,
                             &quot;cornflowerblue&quot;,
                             &quot;chocolate&quot;,
                             &quot;brown&quot;,
                             &quot;green&quot;,
                             &quot;yellow4&quot;,
                             &quot;purple&quot;,
                             &quot;darkorchid4&quot;,
                             &quot;coral4&quot;,
                             &quot;darkolivegreen4&quot;,
                             &quot;darkorange&quot;), unique(Line_df$repFamily)), 
      ...
    )
}

Line_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LINE breakdown&quot;, subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>Line_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  theme_void()+
  ggtitle(&quot;Human genome LINE breakdown without labels&quot;, subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" />
#### Sine repeats</p>
<pre class="r"><code>Sine_df &lt;- Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))

scale_fill_sines &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;,
                             &quot;#FFFFB3&quot;,
                             &quot;#BEBADA&quot; ,
                             &quot;#FB8072&quot;,
                             &quot;#80B1D3&quot;,
                             &quot;#FDB462&quot;,
                             &quot;#B3DE69&quot;,
                             &quot;#FCCDE5&quot;,
                             &quot;#D9D9D9&quot;,
                             &quot;#BC80BD&quot;,
                             &quot;#CCEBC5&quot;,
                             &quot;pink4&quot;,
                             &quot;cornflowerblue&quot;,
                             &quot;chocolate&quot;,
                             &quot;brown&quot;,
                             &quot;green&quot;,
                             &quot;yellow4&quot;,
                             &quot;purple&quot;,
                             &quot;darkorchid4&quot;,
                             &quot;coral4&quot;,
                             &quot;darkolivegreen4&quot;,
                             &quot;darkorange&quot;), unique(Sine_df$repFamily)), 
      ...
    )
}

Sine_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome SINE breakdown&quot;, subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>Sine_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  theme_void()+
  ggtitle(&quot;Human genome SINE breakdown without labels&quot;, subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" />
#### LTR repeats</p>
<pre class="r"><code>LTR_df &lt;- LTR_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))

scale_fill_LTRs &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;,
                             &quot;#FFFFB3&quot;,
                             &quot;#BEBADA&quot; ,
                             &quot;#FB8072&quot;,
                             &quot;#80B1D3&quot;,
                             &quot;#FDB462&quot;,
                             &quot;#B3DE69&quot;,
                             &quot;#FCCDE5&quot;,
                             &quot;#D9D9D9&quot;,
                             &quot;#BC80BD&quot;,
                             &quot;#CCEBC5&quot;,
                             &quot;pink4&quot;,
                             &quot;cornflowerblue&quot;,
                             &quot;chocolate&quot;,
                             &quot;brown&quot;,
                             &quot;green&quot;,
                             &quot;yellow4&quot;,
                             &quot;purple&quot;,
                             &quot;darkorchid4&quot;,
                             &quot;coral4&quot;,
                             &quot;darkolivegreen4&quot;,
                             &quot;darkorange&quot;), unique(LTR_df$repFamily)), 
      ...
    )
}

LTR_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LTR breakdown&quot;, subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>LTR_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  theme_void()+
  ggtitle(&quot;Human genome LTR breakdown without labels&quot;, subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-2.png" width="672" style="display: block; margin: auto;" />
#### DNA TEs</p>
<pre class="r"><code>DNA_df &lt;- DNA_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))
DNA_df %&gt;% distinct(repFamily)</code></pre>
<pre><code>       repFamily
1    hAT-Charlie
2   TcMar-Tigger
3     hAT-Tip100
4      MULE-MuDR
5      TcMar-Tc2
6  TcMar-Mariner
7  hAT-Blackjack
8            hAT
9         hAT-Ac
10      PiggyBac
11          hAT?
12   hAT-Tip100?
13           DNA
14     PiggyBac?
15        TcMar?
16         TcMar
17      hAT-Tag1
18        Merlin
19    TcMar-Pogo
20 PIF-Harbinger
21    MULE-MuDR?
22     TcMar-Tc1
23       Crypton
24     Crypton-A
25       Kolobok</code></pre>
<pre class="r"><code>scale_fill_DNAs &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;,
                             &quot;#FFFFB3&quot;,
                             &quot;#BEBADA&quot; ,
                             &quot;#FB8072&quot;,
                             &quot;#80B1D3&quot;,
                             &quot;#FDB462&quot;,
                             &quot;#B3DE69&quot;,
                             &quot;#FCCDE5&quot;,
                             &quot;#D9D9D9&quot;,
                             &quot;#BC80BD&quot;,
                             &quot;#CCEBC5&quot;,
                             &quot;pink4&quot;,
                             &quot;cornflowerblue&quot;,
                             &quot;chocolate&quot;,
                             &quot;brown&quot;,
                             &quot;green&quot;,
                             &quot;yellow4&quot;,
                             &quot;purple&quot;,
                             &quot;darkorchid4&quot;,
                             &quot;coral4&quot;,
                             &quot;darkolivegreen4&quot;,
                             &quot;darkorange&quot;,
                             &quot;blue&quot;,
                             &quot;grey&quot;,
                             &quot;lightgrey&quot;), unique(DNA_df$repFamily)), 
      ...
    )
}


DNA_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;Human genome DNA breakdown&quot;, subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>DNA_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  theme_void()+
  ggtitle(&quot;Human genome DNA breakdown without labels&quot;, subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<div id="retroposon" class="section level4">
<h4>retroposon</h4>
<pre class="r"><code>retroposon_df &lt;- retroposon_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repName=factor(repName))

length(unique(retroposon_df$repName))</code></pre>
<pre><code>[1] 6</code></pre>
<pre class="r"><code>scale_fill_retroposons &lt;-  function(...){
    ggplot2:::manual_scale(
        &#39;fill&#39;, 
        values = setNames(c( &quot;#8DD3C7&quot;,
                             &quot;#FFFFB3&quot;,
                             &quot;#BEBADA&quot; ,
                             &quot;#FB8072&quot;,
                             &quot;#80B1D3&quot;,
                             &quot;#FDB462&quot;,
                             &quot;#B3DE69&quot;,
                             &quot;#FCCDE5&quot;,
                             &quot;#D9D9D9&quot;,
                             &quot;#BC80BD&quot;,
                             &quot;#CCEBC5&quot;,
                             &quot;pink4&quot;,
                             &quot;cornflowerblue&quot;,
                             &quot;chocolate&quot;,
                             &quot;brown&quot;,
                             &quot;green&quot;,
                             &quot;yellow4&quot;,
                             &quot;purple&quot;,
                             &quot;darkorchid4&quot;,
                             &quot;coral4&quot;,
                             &quot;darkolivegreen4&quot;,
                             &quot;darkorange&quot;), unique(retroposon_df$repName)), 
      ...
    )
}

retroposon_df%&gt;% 
count(repName) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome retroposon breakdown&quot;, subtitle=paste(length(retroposon_df$milliIns)))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>retroposon_df%&gt;% 
count(repName) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  theme_void()+
  ggtitle(&quot;Human genome retroposon breakdown without labels&quot;, subtitle=paste(length(retroposon_df$milliIns)))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggrepel_0.9.5                           
 [2] plyranges_1.22.0                        
 [3] ggsignif_0.6.4                          
 [4] genomation_1.34.0                       
 [5] smplot2_0.2.3                           
 [6] eulerr_7.0.2                            
 [7] biomaRt_2.58.2                          
 [8] devtools_2.4.5                          
 [9] usethis_2.2.3                           
[10] ggpubr_0.6.0                            
[11] BiocParallel_1.36.0                     
[12] Cormotif_1.48.0                         
[13] affy_1.80.0                             
[14] scales_1.3.0                            
[15] VennDiagram_1.7.3                       
[16] futile.logger_1.4.3                     
[17] gridExtra_2.3                           
[18] ggfortify_0.4.17                        
[19] edgeR_4.0.16                            
[20] limma_3.58.1                            
[21] rtracklayer_1.62.0                      
[22] org.Hs.eg.db_3.18.0                     
[23] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[24] GenomicFeatures_1.54.4                  
[25] AnnotationDbi_1.64.1                    
[26] Biobase_2.62.0                          
[27] GenomicRanges_1.54.1                    
[28] GenomeInfoDb_1.38.8                     
[29] IRanges_2.36.0                          
[30] S4Vectors_0.40.2                        
[31] BiocGenerics_0.48.1                     
[32] ChIPseeker_1.38.0                       
[33] RColorBrewer_1.1-3                      
[34] broom_1.0.6                             
[35] kableExtra_1.4.0                        
[36] lubridate_1.9.3                         
[37] forcats_1.0.0                           
[38] stringr_1.5.1                           
[39] dplyr_1.1.4                             
[40] purrr_1.0.2                             
[41] readr_2.1.5                             
[42] tidyr_1.3.1                             
[43] tibble_3.2.1                            
[44] ggplot2_3.5.1                           
[45] tidyverse_2.0.0                         
[46] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.4                               
  [2] matrixStats_1.3.0                      
  [3] bitops_1.0-7                           
  [4] enrichplot_1.22.0                      
  [5] HDO.db_0.99.1                          
  [6] httr_1.4.7                             
  [7] profvis_0.3.8                          
  [8] tools_4.3.1                            
  [9] backports_1.5.0                        
 [10] utf8_1.2.4                             
 [11] R6_2.5.1                               
 [12] lazyeval_0.2.2                         
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.0                            
 [15] prettyunits_1.2.0                      
 [16] preprocessCore_1.64.0                  
 [17] cli_3.6.2                              
 [18] formatR_1.14                           
 [19] scatterpie_0.2.3                       
 [20] labeling_0.4.3                         
 [21] sass_0.4.9                             
 [22] Rsamtools_2.18.0                       
 [23] systemfonts_1.1.0                      
 [24] yulab.utils_0.1.4                      
 [25] foreign_0.8-86                         
 [26] DOSE_3.28.2                            
 [27] svglite_2.1.3                          
 [28] sessioninfo_1.2.2                      
 [29] plotrix_3.8-4                          
 [30] BSgenome_1.70.2                        
 [31] pwr_1.3-0                              
 [32] impute_1.76.0                          
 [33] rstudioapi_0.16.0                      
 [34] RSQLite_2.3.7                          
 [35] generics_0.1.3                         
 [36] gridGraphics_0.5-1                     
 [37] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [38] BiocIO_1.12.0                          
 [39] vroom_1.6.5                            
 [40] gtools_3.9.5                           
 [41] car_3.1-2                              
 [42] GO.db_3.18.0                           
 [43] Matrix_1.6-5                           
 [44] fansi_1.0.6                            
 [45] abind_1.4-5                            
 [46] lifecycle_1.0.4                        
 [47] whisker_0.4.1                          
 [48] yaml_2.3.8                             
 [49] carData_3.0-5                          
 [50] SummarizedExperiment_1.32.0            
 [51] gplots_3.1.3.1                         
 [52] qvalue_2.34.0                          
 [53] SparseArray_1.2.4                      
 [54] BiocFileCache_2.10.2                   
 [55] blob_1.2.4                             
 [56] promises_1.3.0                         
 [57] crayon_1.5.2                           
 [58] miniUI_0.1.1.1                         
 [59] lattice_0.22-6                         
 [60] cowplot_1.1.3                          
 [61] KEGGREST_1.42.0                        
 [62] pillar_1.9.0                           
 [63] knitr_1.47                             
 [64] fgsea_1.28.0                           
 [65] rjson_0.2.21                           
 [66] boot_1.3-30                            
 [67] codetools_0.2-20                       
 [68] fastmatch_1.1-4                        
 [69] glue_1.7.0                             
 [70] getPass_0.2-4                          
 [71] ggfun_0.1.5                            
 [72] remotes_2.5.0                          
 [73] data.table_1.15.4                      
 [74] vctrs_0.6.5                            
 [75] png_0.1-8                              
 [76] treeio_1.26.0                          
 [77] gtable_0.3.5                           
 [78] cachem_1.1.0                           
 [79] xfun_0.44                              
 [80] S4Arrays_1.2.1                         
 [81] mime_0.12                              
 [82] tidygraph_1.3.1                        
 [83] statmod_1.5.0                          
 [84] ellipsis_0.3.2                         
 [85] interactiveDisplayBase_1.40.0          
 [86] nlme_3.1-165                           
 [87] ggtree_3.10.1                          
 [88] bit64_4.0.5                            
 [89] progress_1.2.3                         
 [90] filelock_1.0.3                         
 [91] rprojroot_2.0.4                        
 [92] bslib_0.7.0                            
 [93] affyio_1.72.0                          
 [94] rpart_4.1.23                           
 [95] KernSmooth_2.23-24                     
 [96] Hmisc_5.1-3                            
 [97] colorspace_2.1-0                       
 [98] DBI_1.2.3                              
 [99] seqPattern_1.34.0                      
[100] nnet_7.3-19                            
[101] tidyselect_1.2.1                       
[102] processx_3.8.4                         
[103] bit_4.0.5                              
[104] compiler_4.3.1                         
[105] curl_5.2.1                             
[106] git2r_0.33.0                           
[107] htmlTable_2.4.2                        
[108] xml2_1.3.6                             
[109] DelayedArray_0.28.0                    
[110] shadowtext_0.1.3                       
[111] checkmate_2.3.1                        
[112] caTools_1.18.2                         
[113] callr_3.7.6                            
[114] rappdirs_0.3.3                         
[115] digest_0.6.35                          
[116] rmarkdown_2.27                         
[117] XVector_0.42.0                         
[118] base64enc_0.1-3                        
[119] htmltools_0.5.8.1                      
[120] pkgconfig_2.0.3                        
[121] MatrixGenerics_1.14.0                  
[122] highr_0.11                             
[123] dbplyr_2.5.0                           
[124] fastmap_1.2.0                          
[125] htmlwidgets_1.6.4                      
[126] rlang_1.1.4                            
[127] shiny_1.8.1.1                          
[128] farver_2.1.2                           
[129] jquerylib_0.1.4                        
[130] zoo_1.8-12                             
[131] jsonlite_1.8.8                         
[132] GOSemSim_2.28.1                        
[133] RCurl_1.98-1.14                        
[134] magrittr_2.0.3                         
[135] Formula_1.2-5                          
[136] GenomeInfoDbData_1.2.11                
[137] ggplotify_0.1.2                        
[138] patchwork_1.2.0                        
[139] munsell_0.5.1                          
[140] Rcpp_1.0.12                            
[141] ape_5.8                                
[142] viridis_0.6.5                          
[143] stringi_1.8.4                          
[144] ggraph_2.2.1                           
[145] zlibbioc_1.48.2                        
[146] MASS_7.3-60.0.1                        
[147] pkgbuild_1.4.4                         
[148] AnnotationHub_3.10.1                   
[149] plyr_1.8.9                             
[150] parallel_4.3.1                         
[151] Biostrings_2.70.3                      
[152] graphlayouts_1.1.1                     
[153] splines_4.3.1                          
[154] hms_1.1.3                              
[155] locfit_1.5-9.9                         
[156] ps_1.7.6                               
[157] igraph_2.0.3                           
[158] pkgload_1.3.4                          
[159] reshape2_1.4.4                         
[160] futile.options_1.0.1                   
[161] BiocVersion_3.18.1                     
[162] XML_3.99-0.16.1                        
[163] evaluate_0.24.0                        
[164] lambda.r_1.2.4                         
[165] BiocManager_1.30.23                    
[166] tzdb_0.4.0                             
[167] tweenr_2.0.3                           
[168] httpuv_1.6.15                          
[169] polyclip_1.10-6                        
[170] gridBase_0.4-7                         
[171] ggforce_0.4.2                          
[172] xtable_1.8-4                           
[173] restfulr_0.0.15                        
[174] tidytree_0.4.6                         
[175] rstatix_0.7.2                          
[176] later_1.3.2                            
[177] viridisLite_0.4.2                      
[178] aplot_0.2.2                            
[179] memoise_2.0.1                          
[180] GenomicAlignments_1.38.2               
[181] cluster_2.1.6                          
[182] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
