<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-09-23" />

<title>TEs and my data</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">TEs and my data</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-09-23</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-09-23
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree365e5573f00ce6e74ce7a3cb3fac1a71b9cb9c9etargetblank365e557a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/365e5573f00ce6e74ce7a3cb3fac1a71b9cb9c9e" target="_blank">365e557</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree365e5573f00ce6e74ce7a3cb3fac1a71b9cb9c9etargetblank365e557a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/365e5573f00ce6e74ce7a3cb3fac1a71b9cb9c9e" target="_blank">365e557</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Final_four_data/
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_GWAS_PEAK_MRC_id
    Ignored:    data/SNP_GWAS_PEAK_MRC_id.csv
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/all_four_filtered_counts.txt
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/annotated_gwas_SNPS.csv
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_Eight.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/summary_peakIDandReHeat.csv
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  DOX_DAR_assess.Rmd
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/ReHeat_analysis.Rmd
    Untracked:  analysis/TE_analysis_old.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/CorMotif_data_n45.Rmd
    Modified:   analysis/Enhancer_files_ff.Rmd
    Modified:   analysis/Enrichment_motif.Rmd
    Modified:   analysis/GO_KEGG_analysis.Rmd
    Modified:   analysis/Peak_calling.Rmd
    Modified:   analysis/Raodah.Rmd
    Modified:   analysis/Raodah_mycount.Rmd
    Modified:   analysis/Smaller_set_DAR.Rmd
    Modified:   analysis/TE_analysis.Rmd
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/TE_analysis_ff.Rmd</code>) and
HTML (<code>docs/TE_analysis_ff.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/365e5573f00ce6e74ce7a3cb3fac1a71b9cb9c9e/analysis/TE_analysis_ff.Rmd" target="_blank">365e557</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-23
</td>
<td>
updated group of eight based on 4 individuals
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/TE_analysis_ff.html" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/617aa1e7c420f4083683fe4f334a68eda683ca43/analysis/TE_analysis_ff.Rmd" target="_blank">617aa1e</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
<td>
updated peaklist filtering fixed
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ff2f378f8ed4fbadf5fbef7957c0ceae62f3cca3/analysis/TE_analysis_ff.Rmd" target="_blank">ff2f378</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-12
</td>
<td>
fixing additional calculation problems
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b7f8719756287376f4a037f04401da7165bcd8dc/analysis/TE_analysis_ff.Rmd" target="_blank">b7f8719</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-12
</td>
<td>
fixing additional calculation problems
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1ae2a6369282d67fd02f5f248800137df8fd03ad/analysis/TE_analysis_ff.Rmd" target="_blank">1ae2a63</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-10
</td>
<td>
updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/TE_analysis_ff.html" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/cc49b3bd594098d6f8c3e77eeb86fc5c394a9beb/analysis/TE_analysis_ff.Rmd" target="_blank">cc49b3b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
<td>
updated with new peak file
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6e09040ea4e10a9e728407c7df35276f203d3fc0/analysis/TE_analysis_ff.Rmd" target="_blank">6e09040</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-08
</td>
<td>
updated with new peaks
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)</code></pre>
<div id="loading-data" class="section level5">
<h5>loading data</h5>
<p>This is where I pull in the repeatmasker file taken from UCSC
genomebrowser, the peaks assigned with the closest expressed genes as
‘neargenes’ by TSS, the same peaks list, only condensing to unique peaks
(some are assigned to two neargene), I call the collapsed_peaks and the
peaks assigned to each MRC (EAR, etc…).<br />
With the TSS data and the collapsed data, I made granges objects. I also
separate out the LINEs, SINEs, LTRs, DNAs, and Retroposons from the
repeatmasker to make granges objects from those sets.</p>
<pre class="r"><code>repeatmasker &lt;- read.delim(&quot;data/other_papers/repeatmasker.tsv&quot;)</code></pre>
<pre class="r"><code>### With new TSS_ngdata frame
TSS_NG_data &lt;- read_delim(&quot;data/Final_four_data/TSS_assigned_NG.txt&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
Collapsed_peaks &lt;- read_delim(&quot;data/Final_four_data/collapsed_new_peaks.txt&quot;,
                              delim = &quot;\t&quot;, 
                              escape_double = FALSE, 
                              trim_ws = TRUE)

reClass_list &lt;- repeatmasker %&gt;% 
  distinct(repClass)

Line_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

Sine_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

LTR_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LTR&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

DNA_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

retroposon_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


all_TEs_gr &lt;- repeatmasker %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


peakAnnoList_ff_motif &lt;- readRDS(&quot;data/Final_four_data/peakAnnoList_ff_motif.RDS&quot;)

background_peaks &lt;- as.data.frame(peakAnnoList_ff_motif$background) 
EAR_df &lt;- as.data.frame(peakAnnoList_ff_motif$EAR)
# EAR_df_gr &lt;-  EAR_df %&gt;%  GRanges()
ESR_df &lt;- as.data.frame(peakAnnoList_ff_motif$ESR)
# ESR_df_gr &lt;-ESR_df %&gt;%  GRanges()

LR_df &lt;- as.data.frame(peakAnnoList_ff_motif$LR)
# LR_df_gr &lt;-LR_df %&gt;%  GRanges()

NR_df &lt;- as.data.frame(peakAnnoList_ff_motif$NR)
# NR_df_gr &lt;-NR_df %&gt;%  GRanges()

TSS_data_gr &lt;- TSS_NG_data %&gt;% 
  # dplyr::filter(chr != &quot;chrX&quot;) %&gt;%
  dplyr::filter(chr != &quot;chrY&quot;) %&gt;%
  dplyr::filter(Peakid %in% background_peaks$Peakid) %&gt;% 
  GRanges()

Col_TSS_data_gr &lt;- Collapsed_peaks %&gt;% 
  # dplyr::filter(chr != &quot;chrX&quot;) %&gt;%
  dplyr::filter(chr != &quot;chrY&quot;) %&gt;%
  dplyr::filter(Peakid %in% background_peaks$Peakid) %&gt;% 
  GRanges()</code></pre>
<p>this code contains the fill functions for each of the plots that
needed similar colors.</p>
<pre class="r"><code># scale fill repeat, 2nd set ----------------------------------------------
rep_other_names&lt;- repeatmasker %&gt;% 
  distinct(repClass) %&gt;% 
  rbind(&quot;Other&quot;)

scale_fill_repeat &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;,
                         &quot;darkgrey&quot;), unique(rep_other_names$repClass)), 
    ...
  )
}


# scale fill LTRs ---------------------------------------------------------

LTR_df &lt;- LTR_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_LTRs &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(LTR_df$repFamily)), 
    ...
  )
}



scale_fill_DNA_family &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;, &quot;#FFFFB3&quot;, &quot;#BEBADA&quot; ,&quot;#FB8072&quot;, &quot;#80B1D3&quot;, &quot;#FDB462&quot;, &quot;#B3DE69&quot;, &quot;#FCCDE5&quot;, &quot;purple4&quot;), unique(DNA_family$repFamily)), 
    ...
  )
}


# # scale fill repeat first -------------------------------------------------
# 
# scale_fill_repeat &lt;-  function(...){
#   ggplot2:::manual_scale(
#     &#39;fill&#39;, 
#     values = setNames(c( &quot;#8DD3C7&quot;,
#                          &quot;#FFFFB3&quot;,
#                          &quot;#BEBADA&quot; ,
#                          &quot;#FB8072&quot;,
#                          &quot;#80B1D3&quot;,
#                          &quot;#FDB462&quot;,
#                          &quot;#B3DE69&quot;,
#                          &quot;#FCCDE5&quot;,
#                          &quot;#D9D9D9&quot;,
#                          &quot;#BC80BD&quot;,
#                          &quot;#CCEBC5&quot;,
#                          &quot;pink4&quot;,
#                          &quot;cornflowerblue&quot;,
#                          &quot;chocolate&quot;,
#                          &quot;brown&quot;,
#                          &quot;green&quot;,
#                          &quot;yellow4&quot;,
#                          &quot;purple&quot;,
#                          &quot;darkorchid4&quot;,
#                          &quot;coral4&quot;,
#                          &quot;darkolivegreen4&quot;,
#                          &quot;darkorange&quot;), unique(repeatmasker$repClass)), 
#     ...
#   )
# }

# scale lines -------------------------------------------------------------
Line_df &lt;- Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_lines &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(Line_df$repFamily)), 
    ...
  )
}


# scale fill L2 family ----------------------------------------------------
L2_line_df&lt;- Line_df %&gt;% 
  dplyr::filter(repFamily==&quot;L2&quot;)


scale_fill_L2 &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(L2_line_df$repName)), 
    ...
  )
}

# scale fill sines --------------------------------------------------------
Sine_df &lt;- Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_sines &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(Sine_df$repFamily)), 
    ...
  )
}


# scale fill DNAs ---------------------------------------------------------
DNA_df &lt;- DNA_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_DNAs &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;,
                         &quot;blue&quot;,
                         &quot;grey&quot;,
                         &quot;lightgrey&quot;), unique(DNA_df$repFamily)), 
    ...
  )
}


# scale fill retroposons --------------------------------------------------

retroposon_df &lt;- retroposon_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repName=factor(repName))

scale_fill_retroposons &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(retroposon_df$repName)), 
    ...
  )
}</code></pre>
</div>
<div id="te-distrubution" class="section level2">
<h2>TE distrubution:</h2>
<p>The code below uses the TE data sets to create dataframes that
contain all peaks that overlap a TE. Additionally, I wanted to know size
distribution of the overlaps(width) between TEs and my peaks, the size
of the TEs and their size distribution, and the size distribution of my
peaks. This was to determine a cutoff across TEs to minimize size bias.
We used this data to apply a inclusion stringency cutoff for TEs of
&gt;50% of the full TE needed to be covered by a peak to call the peak
“TE-containing”. In the plots, the median of each density distribution
is shown by the dotted line.</p>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all LINEs across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/data%20about%20TEs-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-1">
Past versions of "data about TEs-1.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/data about TEs-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all SINEs across human genome&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/data%20about%20TEs-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-2">
Past versions of "data about TEs-2.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/data about TEs-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all LTRs across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/data%20about%20TEs-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-3">
Past versions of "data about TEs-3.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/data about TEs-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all DNA-TEs across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/data%20about%20TEs-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-4">
Past versions of "data about TEs-4.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/data about TEs-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass,alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all Retroposons across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
   coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/data%20about%20TEs-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-5">
Past versions of "data about TEs-5.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/data about TEs-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(repClass, width) %&gt;% 
  # dplyr::filter(repClass==&quot;LTR&quot;) %&gt;%
  group_by(repClass) %&gt;% 
  summarise(med.width=median(width),mean.width=mean(width)) %&gt;% 
  kable(., caption=&quot;Table 1: Summary of mean and median length of each TE Class&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 1: Summary of mean and median length of each TE Class
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
med.width
</th>
<th style="text-align:right;">
mean.width
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
155.0
</td>
<td style="text-align:right;">
210.61264
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
125.0
</td>
<td style="text-align:right;">
137.59542
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
219.0
</td>
<td style="text-align:right;">
421.99752
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
329.0
</td>
<td style="text-align:right;">
371.36349
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
170.0
</td>
<td style="text-align:right;">
208.63493
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
44.0
</td>
<td style="text-align:right;">
61.55669
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
171.0
</td>
<td style="text-align:right;">
210.89231
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
144.0
</td>
<td style="text-align:right;">
150.47242
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
165.0
</td>
<td style="text-align:right;">
163.94730
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
646.0
</td>
<td style="text-align:right;">
776.24171
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
258.0
</td>
<td style="text-align:right;">
221.43075
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
83.5
</td>
<td style="text-align:right;">
74.34211
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
496.0
</td>
<td style="text-align:right;">
8741.48243
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
36.0
</td>
<td style="text-align:right;">
55.83912
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
119.0
</td>
<td style="text-align:right;">
136.70621
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
86.0
</td>
<td style="text-align:right;">
140.04608
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
99.0
</td>
<td style="text-align:right;">
96.07143
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
77.5
</td>
<td style="text-align:right;">
79.82394
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
136.0
</td>
<td style="text-align:right;">
169.81834
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
69.0
</td>
<td style="text-align:right;">
60.12893
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Col_TSS_data_gr %&gt;% 
  as.data.frame %&gt;% 
   ggplot(., aes(x=width))+
    geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
  geom_vline(data=Col_TSS_data_gr$width., aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of peak length (bps) across all experimentally derived peaks&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/data%20about%20TEs-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-6">
Past versions of "data about TEs-6.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/data about TEs-6.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/data about TEs-6.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fullDF_overlap &lt;- join_overlap_intersect(TSS_data_gr,all_TEs_gr)

fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot;Table 2: Count of peaks by TE class; overlap 1 bp or greater&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)

subsetByOverlaps(TSS_data_gr,all_TEs_gr) %&gt;% as.data.frame %&gt;% 
   ggplot(., aes(x=width))+
    geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of overlaps between all TEs and all my peaks&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))

### This is how I subset only those peaks who cover &gt;50% of TEs
# hits &lt;- findOverlaps(TSS_data_gr,all_TEs_gr)
# overlaps &lt;- pintersect(TSS_data_gr[queryHits(hits)], all_TEs_gr[subjectHits(hits)])
# percentOverlap &lt;- width(overlaps) / width(all_TEs_gr[subjectHits(hits)])
# hits &lt;- hits[percentOverlap &gt; 0.5]
# ### THis actually did not work well---I ended up losing data.  What I wanted was a data frame that had metadata from both the TE overlap and the peak metadata, so I could easily sort and manipulation.  
# testingol &lt;- TSS_data_gr[queryHits(hits)]
# testingol %&gt;% as.data.frame() %&gt;% 
#   left_join(., (fullDF_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;Peakid&quot;=&quot;Peakid&quot;, &quot;NG_start&quot;=&quot;NG_start&quot;, &quot;end_position&quot;=&quot;end_position&quot;, &quot;entrezgene_id&quot;=&quot;entrezgene_id&quot;, &quot;ensembl_gene_id&quot;=&quot;ensembl_gene_id&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;, &quot;width&quot;=&quot;width&quot;, &quot;strand&quot;=&quot;strand&quot;, &quot;hgnc_symbol&quot; = &quot;hgnc_symbol&quot;)) %&gt;% 
#   group_by(repClass) %&gt;% 
#   tally %&gt;% 
#   kable(., caption=&quot; Table 3: Count of peaks by TE class; overlap&gt; 50%&quot;) %&gt;% 
#   kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
#   kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<p>The first dataframe to subset peaks that overlap &gt;50% of a TE, was
using the full neargene dataframe, where a peak is listed more than once
because it was assigned more than one neargene ( one-to-many
relationships). I changed the code to use the ‘collapsed’ data frame.
This means the data frame was simplified to only include peaks one time,
but those peaks that were assigned to more than one neargene had the
assigned neargenes condensed and separated by a comma into the same
column to create a one-to-one relationship dataframe. (yes, wordy I
know)</p>
<pre class="r"><code>######################################################
all_TEs_gr$TE_width &lt;- width(all_TEs_gr)
Col_TSS_data_gr$peak_width &lt;- width(Col_TSS_data_gr)
Col_fullDF_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,all_TEs_gr)
Col_fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot; Table 2: Count of peaks by TE class; overlap at least 1 bp; using one:one df &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 2: Count of peaks by TE class; overlap at least 1 bp; using
one:one df
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
16960
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
138
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
40440
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
27844
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
257
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5731
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
52
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
295
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
52755
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
191
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
30184
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
288
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
54
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
144
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
302
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Col_fullDF_overlap %&gt;% 
   as.data.frame %&gt;% 
  mutate(per_ol= width/TE_width) %&gt;% 
  dplyr::filter(per_ol&gt;0.5) %&gt;%
  group_by(repClass) %&gt;% 
  tally() %&gt;% 
  kable(., caption=&quot; Table 3:Count of peaks by TE class; overlap of &gt;50% of TE; newway &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 3:Count of peaks by TE class; overlap of &gt;50% of TE; newway
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
11821
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
118
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
25548
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
18403
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
199
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5305
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
81
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
31936
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
81
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
28143
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
242
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
124
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
291
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Filter_TE_list &lt;- Col_fullDF_overlap %&gt;% 
   as.data.frame %&gt;% 
  mutate(per_ol= width/TE_width) 
  # dplyr::filter(per_ol&gt;0.5)

Unique_peak_overlap &lt;- Col_fullDF_overlap %&gt;%
  as.data.frame() %&gt;%
  distinct(Peakid)

peak_overlap_50unique &lt;-  Filter_TE_list %&gt;%
   dplyr::filter(per_ol&gt;0.5) %&gt;% 
  distinct(Peakid)</code></pre>
<p>Note that the counts of peaks total to more than the total number of
peaks that overlap a TE because many peaks overlap multiple elements
(generally the very small TEs).</p>
<p>A summary of numbers of peaks is below:</p>
<ul>
<li>Total number of peaks = 172481</li>
<li>Total number of peaks overlapping at least 1 TE = 104149<br />
</li>
<li>Total number of peaks overlapping by &gt;50% TE length = 81185<br />
<strong>note</strong> these numbers include peaks that are not
classified by an MRC.</li>
</ul>
<p>Tables 3 above used the better way of sub-setting (keeping all
metadata organized vs the sub-setting code I found on the internet).</p>
<div id="distribution-of-te-overlaps-by-peaks-according-to-class"
class="section level3">
<h3>Distribution of TE overlaps by peaks according to class</h3>
<p>Below are plots of the distribution of overlapping widths between
peaks and TEs. The first plot is all TE Class, and the 2nd plot limits
the classes to LINEs, SINEs, LTRs, DNAs, and Retroposons.</p>
<pre class="r"><code>Col_fullDF_overlap %&gt;%  
  as.data.frame %&gt;% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color=&quot;darkblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping widths in my data&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,750))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/another%20set%20of%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-1">
Past versions of "another set of plots-1.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/another set of plots-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/another set of plots-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Col_fullDF_overlap %&gt;%  
  as.data.frame %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass ==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass ==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color=&quot;darkblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths&quot;,subtitle = &quot;Just LINE, SINE,LTR, DNA, Retroposons; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,1550))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/another%20set%20of%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-2">
Past versions of "another set of plots-2.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/another set of plots-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/another set of plots-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Getting the numbers of peaks containing an overlap of &gt; 50% of the
TE required creation of a data frame. The first step of the dataframe
was to use all high confidence Peaks and join with the TE_Overlapping
dataframe.</p>
<pre class="r"><code>Peak_TE_overlapbreakdown &lt;-  Col_TSS_data_gr %&gt;% 
  as.data.frame %&gt;%
  distinct(Peakid) %&gt;%
  left_join(.,(Col_fullDF_overlap %&gt;% as.data.frame)) %&gt;%
  dplyr::select(Peakid, repName:repFamily,TE_width,width) %&gt;% 
  left_join(.,(TSS_data_gr %&gt;% as.data.frame), by=c(&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  mutate(mrc=if_else(Peakid %in% NR_df$Peakid, &quot;NR&quot;,
                    if_else(Peakid %in% ESR_df$Peakid,&quot;ESR&quot;,
                       if_else(Peakid %in% LR_df$Peakid,&quot;LR&quot;,
  if_else(Peakid %in% EAR_df$Peakid, &quot;EAR&quot;, &quot;not_mrc&quot;))))) %&gt;%   mutate(per_ol= width.x/TE_width)

Peak_TE_overlapbreakdown %&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
distinct(Peakid,.keep_all = TRUE) %&gt;% 
  group_by(mrc,TEstatus) %&gt;% tally() </code></pre>
<pre><code># A tibble: 10 × 3
# Groups:   mrc [5]
   mrc     TEstatus        n
   &lt;chr&gt;   &lt;chr&gt;       &lt;int&gt;
 1 EAR     TE_peak      4858
 2 EAR     not_TE_peak  2596
 3 ESR     TE_peak     10256
 4 ESR     not_TE_peak  5525
 5 LR      TE_peak     29288
 6 LR      not_TE_peak 13321
 7 NR      TE_peak     57561
 8 NR      not_TE_peak 28767
 9 not_mrc TE_peak      2186
10 not_mrc not_TE_peak  1199</code></pre>
<pre class="r"><code># TE_mrc_status_list %&gt;% 
#   distinct(Peakid, .keep_all = TRUE) %&gt;% 
#   group_by(TEstatus, mrc) %&gt;%  
#   tally #%&gt;% 
#   dplyr::filter(n&gt;=4&amp;mrc==&quot;EAR&quot;)
  
  
  # Col_fullDF_cug_overlap %&gt;% as.data.frame() %&gt;% group_by(Peakid) %&gt;% tally %&gt;% dplyr::filter(n&gt;=4)</code></pre>
<p>This increased the number of rows from 172481 to 232225 indicating
many peaks contained more than one TE. The column <code>per_ol</code> is
representing the proportion of TE covered within the peak. I next
decided to only count a peak to contain a TE, if it covered &gt;50% of
the length of the TE.</p>
<pre class="r"><code>per_cov=0.5
TE_mrc_status_list &lt;- Peak_TE_overlapbreakdown %&gt;%
   mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) 
 
TE_mrc_status_list%&gt;% 
   distinct(Peakid,TEstatus,.keep_all = TRUE) %&gt;%
   mutate(mrc=&quot;all_peaks&quot;)%&gt;% 
   rbind((TE_mrc_status_list %&gt;% distinct(Peakid,TEstatus,.keep_all = TRUE))) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
     ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+ 
   theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family using the &quot;,per_cov*100,&quot;% cutoff&quot;))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Making%20of%204group%20TE%20dataframe-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Making-of-4group-TE-dataframe-1">
Past versions of "Making of 4group TE dataframe-1.png"
</button>
</p>
<div id="fig-no-spaces-Making-of-4group-TE-dataframe-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Making of 4group TE dataframe-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The table below represents the numbers for each of the four groups
when I include the &gt;50% stringency cutoff filter.</p>
<pre class="r"><code>per_cov=0.5
 
TE_mrc_status_list %&gt;% 
   distinct(Peakid,.keep_all = TRUE) %&gt;%
   mutate(mrc=&quot;all_peaks&quot;)%&gt;% 
   rbind((TE_mrc_status_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  group_by(mrc, TEstatus) %&gt;% 
  count() %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100) %&gt;% 
      kable(., caption=&quot;Table 4: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC using strigency cutoff of 50%.&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 4: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC using strigency cutoff of 50%.
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
summary
</th>
<th style="text-align:left;">
TEstatus
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
3126
</td>
<td style="text-align:right;">
41.93721
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
4328
</td>
<td style="text-align:right;">
58.06279
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
6594
</td>
<td style="text-align:right;">
41.78442
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
9187
</td>
<td style="text-align:right;">
58.21558
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
18855
</td>
<td style="text-align:right;">
44.25121
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
23754
</td>
<td style="text-align:right;">
55.74879
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
37456
</td>
<td style="text-align:right;">
43.38801
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
48872
</td>
<td style="text-align:right;">
56.61199
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
67439
</td>
<td style="text-align:right;">
43.35324
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
88118
</td>
<td style="text-align:right;">
56.64676
</td>
</tr>
</tbody>
</table>
<p>Problem with summary of TE ## Human genome TE breakdown</p>
<p>I first wanted to know the distribution of TEs across the human
genome compared to each other. Below are pie plots with all classes from
repeatmasker, and then pie plots with ONLY the LINES, SINES, LTRs, DNAs,
and Retroposon classes.</p>
<pre class="r"><code>repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/repeatmasker%20genome%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-1">
Past versions of "repeatmasker genome breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/repeatmasker genome breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>LiSiLTDNRe &lt;- repeatmasker %&gt;%
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;)


repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon only&quot;, subtitle=paste(length(LiSiLTDNRe$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/repeatmasker%20genome%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-2">
Past versions of "repeatmasker genome breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/repeatmasker genome breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>repeatmasker  %&gt;% 
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  # dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon focused with other&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/repeatmasker%20genome%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-3">
Past versions of "repeatmasker genome breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-3"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/repeatmasker genome breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># saveRDS(TE_mrc_status_list,&quot;data/TE_info/TE_mrc_status_list.RDS&quot;)
# TE_ALL_count &lt;- TE_mrc_status_list %&gt;%
#   dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
#   dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;%
#   distinct(Peakid) %&gt;% 
#     count
# TE_ALL_count_filt &lt;- TE_mrc_status_list %&gt;%
#   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
#   dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
#   dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
#   distinct(Peakid) %&gt;% 
#     count
TE_50_count &lt;- TE_mrc_status_list %&gt;%
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(Peakid) %&gt;% 
    count
TE_50_count_filt &lt;- TE_mrc_status_list %&gt;%
   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  # dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(Peakid) %&gt;% 
    count

TE_mrc_status_list %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;% 
   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35)+
  # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of Just LINEs, SINEs, LTRs, DNAs, and Retroposons&quot;,subtitle = paste(TE_50_count_filt$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TE%20in%20all%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-1">
Past versions of "TE in all peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TE in all peaks-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/TE in all peaks-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(repClass=factor(repClass)) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
    # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
    #                  position = position_stack(vjust = .3),
    #                  show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks using 50% cutoff&quot;,subtitle = paste(TE_50_count$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TE%20in%20all%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-2">
Past versions of "TE in all peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TE in all peaks-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/TE in all peaks-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;%
  count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks using 50% cutoff&quot;,subtitle = paste(TE_50_count_filt$n, &quot;peaks that contain LINEs, SINEs, LTRs, DNAs, or Retroposons&quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TE%20in%20all%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-3">
Past versions of "TE in all peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TE in all peaks-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/TE in all peaks-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>### note the change in data frame Renee, this is where you flipped back to the original without filters.
Peak_TE_overlapbreakdown%&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TEs%20in%20all%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-1">
Past versions of "TEs in all peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TEs in all peaks-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown%&gt;% 
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths &gt;50%&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TEs%20in%20all%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-2">
Past versions of "TEs in all peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TEs in all peaks-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown%&gt;%
  mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;%
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths &gt;50% of TE length&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TEs%20in%20all%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-3">
Past versions of "TEs in all peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TEs in all peaks-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown %&gt;% 
  mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  distinct() %&gt;% 
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width.x))+
  geom_density(aes(fill=repClass, alpha = 0.5))+
  scale_fill_repeat()+
  theme_classic()+
  ggtitle(&quot;Filtered Distribution of all overlapping peak-TE widths &gt;50 TE length%&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TEs%20in%20all%20peaks-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-in-all-peaks-4">
Past versions of "TEs in all peaks-4.png"
</button>
</p>
<div id="fig-no-spaces-TEs-in-all-peaks-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TEs in all peaks-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="line-repeats" class="section level4">
<h4>Line repeats</h4>
<pre class="r"><code>Line_df %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LINE breakdown&quot;, subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LINEclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-1">
Past versions of "LINEclass breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;) %&gt;%
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;%
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE breakdown of peaks &quot;,per_cov),subtitle=paste(TE_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LINEclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-2">
Past versions of "LINEclass breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count
ESR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count
LR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count
NR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;EAR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(EAR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LINEclass%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-3">
Past versions of "LINEclass breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;ESR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(ESR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LINEclass%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-4">
Past versions of "LINEclass breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(LR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LINEclass%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-5">
Past versions of "LINEclass breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;NR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(NR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LINEclass%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-6">
Past versions of "LINEclass breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-6.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LINEclass breakdown-6.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;) %&gt;% 
  tally
EAR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;EAR&quot;) %&gt;% 
  tally
ESR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;ESR&quot;) %&gt;% 
  tally
LR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;LR&quot;) %&gt;% 
  tally
NR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;NR&quot;) %&gt;% 
  tally


TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for all peaks &quot;,per_cov),subtitle=paste(all_L2_count,&quot; total LINEs&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/L2breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-1">
Past versions of L2breakdown-1.png
</button>
</p>
<div id="fig-L2breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/L2breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/L2breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;%
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for EAR &quot;,per_cov),subtitle=paste(EAR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/L2breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-2">
Past versions of L2breakdown-2.png
</button>
</p>
<div id="fig-L2breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/L2breakdown-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/L2breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for ESR &quot;,per_cov),subtitle=paste(ESR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/L2breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-3">
Past versions of L2breakdown-3.png
</button>
</p>
<div id="fig-L2breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/L2breakdown-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/L2breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for LR &quot;,per_cov),subtitle=paste(LR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/L2breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-4">
Past versions of L2breakdown-4.png
</button>
</p>
<div id="fig-L2breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/L2breakdown-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/L2breakdown-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for NR &quot;,per_cov),subtitle=paste(NR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/L2breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-5">
Past versions of L2breakdown-5.png
</button>
</p>
<div id="fig-L2breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/L2breakdown-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/L2breakdown-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="sine-repeats" class="section level4">
<h4>Sine repeats</h4>
<pre class="r"><code>Sine_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome SINE breakdown&quot;, subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sineclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-1">
Past versions of "Sineclass breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;SINE breakdown of peaks &quot;,per_cov),subtitle=paste(TE_SINE_count$n,&quot; total SINEs found&quot;))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sineclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-2">
Past versions of "Sineclass breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count
ESR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count
LR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count
NR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;EAR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(EAR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sineclass%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-3">
Past versions of "Sineclass breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  # 
  theme_void()+
  ggtitle(paste0(&quot;ESR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(ESR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sineclass%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-4">
Past versions of "Sineclass breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(LR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sineclass%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-5">
Past versions of "Sineclass breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;SINE&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;NR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(NR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sineclass%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-6">
Past versions of "Sineclass breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-6.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sineclass breakdown-6.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="ltr-repeats" class="section level4">
<h4>LTR repeats</h4>
<pre class="r"><code>per_cov &lt;- 0.5


LTR_df%&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LTR breakdown&quot;, subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-1">
Past versions of "LTR breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  count
EAR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count
ESR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count
LR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count
NR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;All peaks LTR breakdown of peaks&quot;,per_cov),subtitle=paste(LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-2">
Past versions of "LTR breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;EAR LTR breakdown of peaks&quot;,per_cov),subtitle=paste(EAR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-3">
Past versions of "LTR breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR LTR breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-4">
Past versions of "LTR breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR LTR breakdown of peaks &quot;,per_cov),subtitle=paste(LR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-5">
Past versions of "LTR breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LTR&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR LTR breakdown of peaks&quot;, per_cov),subtitle=paste(NR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-6">
Past versions of "LTR breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-6.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR breakdown-6.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dna-tes" class="section level4">
<h4>DNA TEs</h4>
<pre class="r"><code>per_cov &lt;- 0.5

DNA_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;Human genome DNA breakdown&quot;, subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-1">
Past versions of "DNA breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;DNA breakdown of peaks&quot;, per_cov),subtitle=paste(TE_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-2">
Past versions of "DNA breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count
ESR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count
LR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count
NR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  count


TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;EAR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(EAR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-3">
Past versions of "DNA breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-4">
Past versions of "DNA breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(LR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-5">
Past versions of "DNA breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;DNA&quot;) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR DNA breakdown of peaks&quot;, per_cov),subtitle=paste(NR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-6">
Past versions of "DNA breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-6.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA breakdown-6.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="retroposon" class="section level4">
<h4>retroposon</h4>
<pre class="r"><code>per_cov &lt;- 0.5

retroposon_df%&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label =  paste0(repName ,&quot;\n&quot;, sprintf(&quot;%.2f&quot;, perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome retroposon breakdown&quot;, subtitle= paste( length(retroposon_df$milliIns)))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/retroposon%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-1">
Past versions of "retroposon breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;) %&gt;% 
   mutate(repName=factor(repName)) %&gt;% 
     # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(TE_retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/retroposon%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-2">
Past versions of "retroposon breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count
ESR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count
LR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count
NR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  count


TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/retroposon%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-3">
Past versions of "retroposon breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(LR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/retroposon%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-4">
Past versions of "retroposon breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(NR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/retroposon%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-5">
Past versions of "retroposon breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/retroposon breakdown-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="te-by-response-cluster-bargraph-summaries"
class="section level3">
<h3>TE by response cluster bargraph summaries</h3>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                          if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) %&gt;%
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
  rbind((TE_mrc_status_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;%
  mutate(repClass=factor(repClass)) %&gt;%
  mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  # dplyr::filter(is.na(per_ol)| per_ol&gt;per_cov) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;,&quot;&gt;&quot;, per_cov*100,&quot;%&quot;))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TE%20status%20old%20clusters-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-status-old-clusters-1">
Past versions of "TE status old clusters-1.png"
</button>
</p>
<div id="fig-no-spaces-TE-status-old-clusters-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TE status old clusters-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/TE status old clusters-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># TE_counts_nofilt &lt;- TE_mrc_status_list %&gt;% 
#    mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
#   mutate(repClass=if_else(##relable repClass with other
#     repClass_org==&quot;LINE&quot;, repClass_org,if_else(repClass_org==&quot;SINE&quot;,repClass_org,if_else(repClass_org==&quot;LTR&quot;, repClass_org, if_else(repClass_org==&quot;DNA&quot;, repClass_org, if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
#    dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) %&gt;% distinct(Peakid, .keep_all = TRUE) %&gt;% 
#    mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
#   rbind((TE_mrc_status_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;% 
#   # mutate(repClass=factor(repClass)) %&gt;%
#    mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
#    dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
#   mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
#      group_by(mrc, TEstatus) %&gt;% 
#      count() %&gt;% 
#      pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
#      rowwise() %&gt;% 
#      mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
#      ungroup() %&gt;% 
#      pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
#      mutate(percent_mrc= n/summary*100)

# notinterested_list &lt;- c(&quot;Simple_repeat&quot;,&quot;Satellite&quot;,&quot;Low_complexity&quot;,&quot;DNA?&quot;,&quot;snRNA&quot;,&quot;tRNA&quot;,&quot;Unknown&quot;,&quot;RC&quot;,&quot;LTR?&quot;,&quot;srpRNA&quot;,&quot;scRNA&quot;,&quot;rRNA&quot;,&quot;RC?&quot;,&quot;SINE?&quot;)

Lines_Etc &lt;-   TE_mrc_status_list %&gt;%
  mutate(repClass_org = repClass) %&gt;%  
  # dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
              if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol)%&gt;% 
    distinct(Peakid,mrc, .keep_all = TRUE) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;)</code></pre>
<pre class="r"><code>LiSiLTDNRe_TE &lt;-TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
        if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) %&gt;%
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  rbind(Lines_Etc) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;%
  mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  group_by(mrc, TEstatus) %&gt;% 
  count() %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
  rowwise() %&gt;% 
  mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
  ungroup() %&gt;% 
  pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
  mutate(percent_mrc= n/summary*100)

LiSiLTDNRe_TE %&gt;% 
  kable(., caption=&quot;Table 5: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC in TE_peak count&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 5: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC in TE_peak count
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
summary
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
TEstatus
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
n
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
3126
</td>
<td style="text-align:right;">
41.93721
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
7454
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
4328
</td>
<td style="text-align:right;">
58.06279
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
6594
</td>
<td style="text-align:right;">
41.78442
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
15781
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
9187
</td>
<td style="text-align:right;">
58.21558
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
18855
</td>
<td style="text-align:right;">
44.25121
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
42609
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
23754
</td>
<td style="text-align:right;">
55.74879
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
37456
</td>
<td style="text-align:right;">
43.38801
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
86328
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
48872
</td>
<td style="text-align:right;">
56.61199
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
67439
</td>
<td style="text-align:right;">
43.35324
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
155557
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
88118
</td>
<td style="text-align:right;">
56.64676
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code># TE_mrc_status_list %&gt;% 
#    mutate(repClass_org = repClass) %&gt;% 
#     mutate(repClass=if_else(##relable repClass with other
#     repClass_org==&quot;LINE&quot;, repClass_org,
#     if_else(repClass_org==&quot;SINE&quot;,repClass_org,
#             if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
#                     if_else(repClass_org==&quot;DNA&quot;, repClass_org,                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
#    dplyr::select(Peakid,repName,repClass,repFamily,TEstatus, mrc, per_ol) %&gt;%
#   distinct(Peakid, .keep_all = TRUE) %&gt;% 
#   rbind(Lines_Etc) %&gt;% 
#   dplyr::filter(is.na(per_ol)| per_ol &gt; per_cov) %&gt;% 
#    mutate(repClass=factor(repClass)) %&gt;% 
#    mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
#    dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
#   mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
#   ggplot(., aes(x=mrc, fill= TEstatus))+
#   geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
#   theme_classic()+
#   ggtitle(paste(&quot;TE status by MRC and Family&quot;, per_cov), subtitle = &quot;Just LINEs, SINEs,LTRs, DNAs, Retroposons, and all peaks in the MRC families&quot;)</code></pre>
<div id="chi-square-tests" class="section level4">
<h4>Chi square tests</h4>
<pre class="r"><code># ##complete counts()
# print(&quot;Chi square tests without filtering&quot;)
# chitest_LRvNRTE_nf &lt;- matrix(c(TE_counts_nofilt$n[5],TE_counts_nofilt$n[6],TE_counts_nofilt$n[7],TE_counts_nofilt$n[8]), ncol=2, byrow=TRUE)
# chisq.test(chitest_LRvNRTE_nf)
# 
# chitest_EARvNRTE_nf &lt;- matrix(c(TE_counts_nofilt$n[1],TE_counts_nofilt$n[2],TE_counts_nofilt$n[7],TE_counts_nofilt$n[8]), ncol=2, byrow=TRUE)
# chisq.test(chitest_EARvNRTE_nf)
# 
# chitest_ESRvNRTE_nf &lt;- matrix(c(TE_counts_nofilt$n[3],TE_counts_nofilt$n[4],TE_counts_nofilt$n[7],TE_counts_nofilt$n[8]), ncol=2, byrow=TRUE)
# chisq.test(chitest_ESRvNRTE_nf)

### just subsets

# print(&quot;chi test on subsets without &gt;50% overlap cut off&quot;)
# chitest_LRvNRTE_nc &lt;- matrix(c(LiSiLTDNRe_TE_no_cut$n[5],LiSiLTDNRe_TE_no_cut$n[6],LiSiLTDNRe_TE_no_cut$n[7],LiSiLTDNRe_TE_no_cut$n[8]), ncol=2, byrow=TRUE)
# chisq.test(chitest_LRvNRTE_nc)
# 
# chitest_EARvNRTE_nc &lt;- matrix(c(LiSiLTDNRe_TE_no_cut$n[1],LiSiLTDNRe_TE_no_cut$n[2],LiSiLTDNRe_TE_no_cut$n[7],LiSiLTDNRe_TE_no_cut$n[8]), ncol=2, byrow=TRUE)
# chisq.test(chitest_EARvNRTE_nc)
# 
# chitest_ESRvNRTE_nc &lt;- matrix(c(LiSiLTDNRe_TE_no_cut$n[3],LiSiLTDNRe_TE_no_cut$n[4],LiSiLTDNRe_TE_no_cut$n[7],LiSiLTDNRe_TE_no_cut$n[8]), ncol=2, byrow=TRUE)
# chisq.test(chitest_ESRvNRTE_nc)



##### JUst using the .5 overlap cutoff
print(&quot; chi tests using &gt;50% cutoff&quot;)</code></pre>
<pre><code>[1] &quot; chi tests using &gt;50% cutoff&quot;</code></pre>
<pre class="r"><code>chitest_LRvNRTE &lt;- matrix(c(LiSiLTDNRe_TE$n[5],LiSiLTDNRe_TE$n[6],LiSiLTDNRe_TE$n[7],LiSiLTDNRe_TE$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_LRvNRTE)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_LRvNRTE
X-squared = 8.6061, df = 1, p-value = 0.00335</code></pre>
<pre class="r"><code>chitest_EARvNRTE &lt;- matrix(c(LiSiLTDNRe_TE$n[1],LiSiLTDNRe_TE$n[2],LiSiLTDNRe_TE$n[7],LiSiLTDNRe_TE$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_EARvNRTE)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_EARvNRTE
X-squared = 5.8244, df = 1, p-value = 0.01581</code></pre>
<pre class="r"><code>chitest_ESRvNRTE &lt;- matrix(c(LiSiLTDNRe_TE$n[3],LiSiLTDNRe_TE$n[4],LiSiLTDNRe_TE$n[7],LiSiLTDNRe_TE$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_ESRvNRTE)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_ESRvNRTE
X-squared = 13.922, df = 1, p-value = 0.0001906</code></pre>
</div>
</div>
<div id="breakdown-of-classes-by-4-member-clusters"
class="section level3">
<h3>Breakdown of Classes by 4 member clusters</h3>
<pre class="r"><code>per_cov &lt;- 0.5
 
# ggline_df &lt;-   Line_repeats %&gt;%
#   as.data.frame() %&gt;% 
#  tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
#   dplyr::select(Peakid,repName,repClass, repFamily) %&gt;% 
#   mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;, per_ol = &quot;NA&quot;) %&gt;% 
  ggline_df &lt;-   TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;LINE&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

ggsine_df &lt;-TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;SINE&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 #  Sine_repeats %&gt;% 
 #  as.data.frame() %&gt;% 
 # tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
 #  dplyr::select(Peakid,repName,repClass, repFamily) %&gt;% 
 #  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;% 
 #    rbind((TE_mrc_status_list %&gt;%  
 #             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
 #             dplyr::filter(repClass==&quot;SINE&quot;))) %&gt;%
 #  mutate(mrc=&quot;all_peaks&quot;)

ggLTR_df &lt;- TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;LTR&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 #  LTR_repeats %&gt;% 
 #  as.data.frame() %&gt;% 
 # tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
 #  dplyr::select(Peakid,repName,repClass, repFamily) %&gt;% 
 #  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
 #    rbind((TE_mrc_status_list %&gt;%  
 #             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
 #             dplyr::filter(repClass==&quot;LTR&quot;))) %&gt;%
 #  mutate(mrc=&quot;all_peaks&quot;)


ggDNA_df &lt;-TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;DNA&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

 #  DNA_repeats %&gt;% 
 #  as.data.frame() %&gt;% 
 # tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
 #  dplyr::select(Peakid,repName,repClass, repFamily) %&gt;% 
 #  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
 #     rbind((TE_mrc_status_list %&gt;%  
 #             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
 #             dplyr::filter(repClass==&quot;DNA&quot;))) %&gt;%
 #  mutate(mrc=&quot;all_peaks&quot;)

ggretroposon_df &lt;-TE_mrc_status_list %&gt;%  
             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
             dplyr::filter(repClass==&quot;Retroposons&quot;) %&gt;%
  distinct(Peakid, TEstatus,repClass,.keep_all = TRUE) %&gt;% 
  mutate(mrc=&quot;all_peaks&quot;)

  # retroposon_repeats %&gt;% 
 #  as.data.frame() %&gt;% 
 # tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
 #  dplyr::select(Peakid,repName,repClass, repFamily) %&gt;% 
 #  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
 #     rbind((TE_mrc_status_list %&gt;%  
 #             mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
 #             dplyr::filter(repClass==&quot;Retroposon&quot;))) %&gt;%
 #  mutate(mrc=&quot;all_peaks&quot;)


plot1 &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggline_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LINE breakdown by MRC and Family&quot;, per_cov))+
  scale_fill_lines()
plot1</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/breakdown%20by%20cluster-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-1">
Past versions of "breakdown by cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot2 &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;SINE breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_sines()
plot2</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/breakdown%20by%20cluster-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-2">
Past versions of "breakdown by cluster-2.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggLTR_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LTR breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/breakdown%20by%20cluster-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-3">
Past versions of "breakdown by cluster-3.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggDNA_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;DNA breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/breakdown%20by%20cluster-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-4">
Past versions of "breakdown by cluster-4.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-4.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-4.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggretroposon_df) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;Retroposon breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/breakdown%20by%20cluster-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-5">
Past versions of "breakdown by cluster-5.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-5.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/breakdown by cluster-5.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="adding-in-the-new-categories" class="section level3">
<h3>Adding in the new categories</h3>
<p>Here I break up the categories using median LFC</p>
<ul>
<li>EAR_open = median 3 hour LFC &gt; 0<br />
</li>
<li>EAR_close = median 3 hour LFC &lt; 0</li>
<li>ESR_A = median 3 hour &gt; 0 and median 24 hour &gt;0</li>
<li>ESR_B = median 3 hour &lt; 0 and median 24 hour &lt;0</li>
<li>ESR_C = median 3 hour &gt; 0 and median 24 hour &lt;0</li>
<li>ESR_D = median 3 hour &lt; 0 and median 24 hour &gt;0</li>
<li>LR_open = median 24 hour LFC &gt; 0<br />
</li>
<li>LR_close = median 24 hour LFC &lt; 0<br />
</li>
<li>NR = all NR classified peaks</li>
</ul>
<pre class="r"><code>per_cov=0.5
median_24_lfc &lt;- read_csv(&quot;data/Final_four_data/median_24_lfc.csv&quot;) 
median_3_lfc &lt;- read_csv(&quot;data/Final_four_data/median_3_lfc.csv&quot;)

open_3med &lt;- median_3_lfc %&gt;% 
  dplyr::filter(med_3h_lfc &gt; 0)

close_3med &lt;- median_3_lfc %&gt;% 
  dplyr::filter(med_3h_lfc &lt; 0)

open_24med &lt;- median_24_lfc %&gt;% 
  dplyr::filter(med_24h_lfc &gt; 0)

close_24med &lt;- median_24_lfc %&gt;% 
  dplyr::filter(med_24h_lfc &lt; 0)

medA &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(med_3h_lfc &gt; 0 &amp; med_24h_lfc&gt;0)

medB &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(med_3h_lfc &lt; 0 &amp; med_24h_lfc &lt; 0)
 
medC &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::filter(med_3h_lfc &gt; 0&amp; med_24h_lfc &lt;0)
  

medD &lt;- median_3_lfc %&gt;% 
 left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;))%&gt;% 
  dplyr::filter(med_3h_lfc &lt; 0 &amp; med_24h_lfc &gt; 0)
 

EAR_open &lt;- EAR_df %&gt;%
  dplyr::filter(Peakid %in% open_3med$peak)
  
EAR_open_gr &lt;- EAR_open %&gt;% GRanges()

EAR_close &lt;- EAR_df %&gt;%
  dplyr::filter(Peakid %in% close_3med$peak) 

EAR_close_gr &lt;- EAR_close %&gt;% GRanges()

LR_open &lt;- LR_df %&gt;%
  dplyr::filter(Peakid %in% open_24med$peak) 

LR_open_gr &lt;- LR_open %&gt;% GRanges()

LR_close &lt;- LR_df %&gt;%
  dplyr::filter(Peakid %in% close_24med$peak) 

LR_close_gr &lt;- LR_close %&gt;% GRanges()

NR_gr &lt;- NR_df %&gt;% 
   GRanges()

ESR_open &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medA$peak)  
 
ESR_open_gr &lt;- ESR_open %&gt;% GRanges()

ESR_close &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medB$peak)  

ESR_close_gr &lt;- ESR_close %&gt;% GRanges()

ESR_C &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medC$peak) 

ESR_D &lt;- ESR_df %&gt;% 
  dplyr::filter(Peakid %in% medD$peak) 


ESR_OC &lt;- ESR_C %&gt;% 
  rbind(ESR_D)
ESR_OC_gr &lt;- ESR_OC %&gt;% GRanges()

Eight_group_TE &lt;-  Col_TSS_data_gr %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(Peakid) %&gt;% 
  left_join(.,(Col_fullDF_overlap %&gt;% 
                 as.data.frame)) %&gt;% 
   # mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;% 
   mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_OC$Peakid,&quot;ESR_OC&quot;,
                                                              if_else(Peakid %in% LR_open$Peakid, &quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid, &quot;LR_close&quot;,
                                     if_else(Peakid %in% NR_df$Peakid, &quot;NR&quot;, &quot;not_mrc&quot;))))))))) %&gt;%  
                                       mutate(per_ol= width/TE_width) %&gt;% 
  mutate(repClass_org=repClass) %&gt;% 
  mutate(repClass=if_else(per_ol&gt;per_cov, repClass,                          if_else(per_ol&lt;per_cov,NA,repClass))) %&gt;%
  mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  mutate(repClass=factor(repClass)) %&gt;%
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org, &quot;Other&quot;))))))) %&gt;% 
  dplyr::select(Peakid, repName,repClass,repClass_org, repFamily, width, TEstatus, mrc, per_ol)


Eight_group_TE %&gt;% distinct(Peakid,.keep_all = TRUE) %&gt;% 
  group_by(mrc, TEstatus) %&gt;% 
  tally</code></pre>
<pre><code># A tibble: 18 × 3
# Groups:   mrc [9]
   mrc       TEstatus        n
   &lt;chr&gt;     &lt;chr&gt;       &lt;int&gt;
 1 EAR_close TE_peak      1728
 2 EAR_close not_TE_peak  2568
 3 EAR_open  TE_peak      1398
 4 EAR_open  not_TE_peak  1760
 5 ESR_OC    TE_peak       496
 6 ESR_OC    not_TE_peak   664
 7 ESR_close TE_peak      3657
 8 ESR_close not_TE_peak  5692
 9 ESR_open  TE_peak      2441
10 ESR_open  not_TE_peak  2831
11 LR_close  TE_peak      5833
12 LR_close  not_TE_peak  8239
13 LR_open   TE_peak     13022
14 LR_open   not_TE_peak 15515
15 NR        TE_peak     37456
16 NR        not_TE_peak 48872
17 not_mrc   TE_peak      1408
18 not_mrc   not_TE_peak  1977</code></pre>
<div id="ear" class="section level4">
<h4>EAR</h4>
</div>
<div id="esr" class="section level4">
<h4>ESR</h4>
<p><img src="figure/TE_analysis_ff.Rmd/ESR-te-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-1">
Past versions of ESR-te-1.png
</button>
</p>
<div id="fig-ESR-te-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/ESR-te-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/TE_analysis_ff.Rmd/ESR-te-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-2">
Past versions of ESR-te-2.png
</button>
</p>
<div id="fig-ESR-te-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/ESR-te-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/TE_analysis_ff.Rmd/ESR-te-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-3">
Past versions of ESR-te-3.png
</button>
</p>
<div id="fig-ESR-te-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/ESR-te-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lr" class="section level4">
<h4>LR</h4>
<p><img src="figure/TE_analysis_ff.Rmd/LRTE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LRTE-1">
Past versions of LRTE-1.png
</button>
</p>
<div id="fig-LRTE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LRTE-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/TE_analysis_ff.Rmd/LRTE-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LRTE-2">
Past versions of LRTE-2.png
</button>
</p>
<div id="fig-LRTE-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LRTE-2.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nr" class="section level4">
<h4>NR</h4>
<p><img src="figure/TE_analysis_ff.Rmd/NR-te-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-NR-te-1">
Past versions of NR-te-1.png
</button>
</p>
<div id="fig-NR-te-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/NR-te-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="sub-set-breakdown-with-new-classes" class="section level3">
<h3>sub-set breakdown with new classes</h3>
<p>This section reclassifies any TE that is NOT a LINE, SINE, LTR, DNA,
or retroposon as “other”. Still using &gt; 50% coverage cutoff</p>
<pre class="r"><code>per_cov &lt;- 0.5
subsetall_df &lt;-  Eight_group_TE %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
  mutate(mrc=&quot;all_peaks&quot;) 

# h.genome_df &lt;- LiSiLTDNRe %&gt;% 
h.genome_df &lt;- repeatmasker %&gt;% 
  mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,if_else(repClass_org==&quot;SINE&quot;,repClass_org,if_else(repClass_org==&quot;LTR&quot;, repClass_org, if_else(repClass_org==&quot;DNA&quot;, repClass_org, if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  mutate(Peakid=paste0(rownames(.),&quot;_TE&quot;)) %&gt;% 
  dplyr::select(Peakid,repName,repClass, repFamily,repClass_org) %&gt;%
   mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;, width=&quot;NA&quot;)


Eight_group_TE %&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
  # dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  rbind(subsetall_df) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repClass))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;Repeat breakdown across eight clusters&quot;, per_cov))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/breakdown%20by%20specific%20cluster-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-specific-cluster-1">
Past versions of "breakdown by specific cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-specific-cluster-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/breakdown by specific cluster-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/breakdown by specific cluster-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Now I will display without the “other” and regraph with new
groups</p>
<pre class="r"><code>Eight_group_TE %&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
  dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  rbind(subsetall_df) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(is.na(repClass)|repClass != &quot;Other&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repClass))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
   ggtitle(paste(&quot;Repeat breakdown across eight clusters without &#39;Other&#39;&quot;, per_cov))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/eight%20group%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-eight-group-breakdown-1">
Past versions of "eight group breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-eight-group-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/eight group breakdown-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/eight group breakdown-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Eight_group_TE %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
  # mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,repClass_org,width,TEstatus, mrc, per_ol)%&gt;%
  distinct(Peakid, .keep_all = TRUE)%&gt;% 
  rbind((subsetall_df %&gt;% distinct(Peakid,TEstatus,.keep_all = TRUE))) %&gt;% 
    mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  dplyr::filter(is.na(repClass)|repClass != &quot;Other&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status for eight MRCs and Family &gt; 50 %&quot;))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/TEs%20by%20%20without%20other-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TEs-by--without-other-1">
Past versions of "TEs by without other-1.png"
</button>
</p>
<div id="fig-no-spaces-TEs-by--without-other-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/TEs by  without other-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Eight_group_TE %&gt;% 
  mutate(repClass_org = repClass) %&gt;% 
  mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,repClass_org,width,TEstatus, mrc, per_ol)%&gt;%
  distinct(Peakid, .keep_all = TRUE)%&gt;% 
  rbind((subsetall_df %&gt;% distinct(Peakid,TEstatus, .keep_all = TRUE))) %&gt;% 
    mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
  dplyr::filter(is.na(repClass)|repClass != &quot;Other&quot;) %&gt;% 
  group_by(TEstatus, mrc) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
  rowwise() %&gt;% 
  mutate(perc_TE= (TE_peak/sum(c_across(TE_peak:not_TE_peak)))*100) %&gt;% 
  kable(., caption = &quot;Unique Peak TE status counts for each using &gt;50% of TE LINEs,SINEs, LTRs,DNAs,and Retroposons&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Unique Peak TE status counts for each using &gt;50% of TE LINEs,SINEs,
LTRs,DNAs,and Retroposons
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TE_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
not_TE_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
perc_TE
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1415
</td>
<td style="text-align:right;">
2574
</td>
<td style="text-align:right;">
35.47255
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
878
</td>
<td style="text-align:right;">
1761
</td>
<td style="text-align:right;">
33.27018
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
433
</td>
<td style="text-align:right;">
667
</td>
<td style="text-align:right;">
39.36364
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
2739
</td>
<td style="text-align:right;">
5698
</td>
<td style="text-align:right;">
32.46415
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
1648
</td>
<td style="text-align:right;">
2841
</td>
<td style="text-align:right;">
36.71196
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
4581
</td>
<td style="text-align:right;">
8251
</td>
<td style="text-align:right;">
35.69981
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
10659
</td>
<td style="text-align:right;">
15560
</td>
<td style="text-align:right;">
40.65372
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
26417
</td>
<td style="text-align:right;">
48978
</td>
<td style="text-align:right;">
35.03813
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
59855
</td>
<td style="text-align:right;">
97170
</td>
<td style="text-align:right;">
38.11813
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>Eight_group_TE %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
  # mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,repClass_org,width,TEstatus, mrc, per_ol)%&gt;%
  distinct(Peakid, .keep_all = TRUE)%&gt;% 
  rbind(subsetall_df ) %&gt;% 
    mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status for eight MRCs and Family &gt; 50 %&quot;))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/with%20out%20the%20other-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-with-out-the-other-1">
Past versions of "with out the other-1.png"
</button>
</p>
<div id="fig-no-spaces-with-out-the-other-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/with out the other-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Eight_group_TE %&gt;% 
  mutate(repClass_org = repClass) %&gt;% 
  mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   dplyr::select(Peakid,repName,repClass,repFamily,repClass_org,width,TEstatus, mrc, per_ol)%&gt;%
  distinct(Peakid, .keep_all = TRUE)%&gt;% 
  rbind(subsetall_df) %&gt;% 
    mutate(repClass=if_else(per_ol&gt;0.5, repClass,NA)) %&gt;% 
   mutate(TEstatus= if_else(is.na(repClass), &quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  group_by(TEstatus, mrc) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
  rowwise() %&gt;% 
  mutate(perc_TE= (TE_peak/sum(c_across(TE_peak:not_TE_peak)))*100) %&gt;% 
  kable(., caption = &quot;Unique Peak TE status counts for each  using &gt;50% of TE&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Unique Peak TE status counts for each using &gt;50% of TE
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TE_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
not_TE_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
perc_TE
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1722
</td>
<td style="text-align:right;">
2574
</td>
<td style="text-align:right;">
40.08380
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1397
</td>
<td style="text-align:right;">
1761
</td>
<td style="text-align:right;">
44.23686
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
493
</td>
<td style="text-align:right;">
667
</td>
<td style="text-align:right;">
42.50000
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
3651
</td>
<td style="text-align:right;">
5698
</td>
<td style="text-align:right;">
39.05231
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
2431
</td>
<td style="text-align:right;">
2841
</td>
<td style="text-align:right;">
46.11153
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
5821
</td>
<td style="text-align:right;">
8251
</td>
<td style="text-align:right;">
41.36583
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
12977
</td>
<td style="text-align:right;">
15560
</td>
<td style="text-align:right;">
45.47430
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
37350
</td>
<td style="text-align:right;">
48978
</td>
<td style="text-align:right;">
43.26522
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
120003
</td>
<td style="text-align:right;">
102381
</td>
<td style="text-align:right;">
53.96207
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>Class_status_df &lt;-
  Eight_group_TE %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
    # mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
  # rbind(subsetall_df) %&gt;% 
   mutate(Sine_status = if_else(is.na(repClass),&quot;not_sine&quot;,
                               if_else(repClass==&quot;SINE&quot;,&quot;sine_peak&quot;, &quot;not_sine&quot;))) %&gt;% 
   mutate(Line_status = if_else(is.na(repClass),&quot;not_line&quot;,
                                if_else(repClass==&quot;LINE&quot;,&quot;line_peak&quot;, &quot;not_line&quot;))) %&gt;%
   mutate(LTR_status = if_else(is.na(repClass),&quot;not_LTR&quot;,
                               if_else(repClass==&quot;LTR&quot;,&quot;LTR_peak&quot;, &quot;not_LTR&quot;))) %&gt;% 
   mutate(DNA_status = if_else(is.na(repClass),&quot;not_DNA&quot;,
                                if_else(repClass==&quot;DNA&quot;,&quot;DNA_peak&quot;, &quot;not_DNA&quot;))) %&gt;% 
   mutate(Retro_status = if_else(is.na(repClass)&amp;is.na(per_ol),&quot;not_Retro&quot;,
                                if_else(repClass==&quot;Retroposon&quot;,&quot;Retro_peak&quot;, &quot;not_Retro&quot;))) %&gt;% 
     # mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;%
     mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   # dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
 mutate(Sine_status=factor(Sine_status, levels = c(&quot;sine_peak&quot;,&quot;not_sine&quot;)),
       Line_status=factor(Line_status, levels =c(&quot;line_peak&quot;,&quot;not_line&quot;)),
       LTR_status=factor(LTR_status, levels =c(&quot;LTR_peak&quot;,&quot;not_LTR&quot;)),
       DNA_status=factor(DNA_status, levels =c(&quot;DNA_peak&quot;,&quot;not_DNA&quot;)),
       Retro_status=factor(Retro_status, levels =c(&quot;Retro_peak&quot;,&quot;not_Retro&quot;))) %&gt;% 
  # dplyr::filter(per_ol&gt;per_cov|is.na(per_ol)) %&gt;% 
   mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;)))</code></pre>
<pre class="r"><code>Class_status_df  %&gt;% distinct(Peakid,.keep_all = TRUE) %&gt;% 
  group_by(mrc, TEstatus) %&gt;% 
  tally</code></pre>
<pre><code># A tibble: 16 × 3
# Groups:   mrc [8]
   mrc       TEstatus        n
   &lt;fct&gt;     &lt;fct&gt;       &lt;int&gt;
 1 EAR_open  TE_peak      1398
 2 EAR_open  not_TE_peak  1760
 3 EAR_close TE_peak      1728
 4 EAR_close not_TE_peak  2568
 5 ESR_open  TE_peak      2441
 6 ESR_open  not_TE_peak  2831
 7 ESR_close TE_peak      3657
 8 ESR_close not_TE_peak  5692
 9 ESR_OC    TE_peak       496
10 ESR_OC    not_TE_peak   664
11 LR_open   TE_peak     13022
12 LR_open   not_TE_peak 15515
13 LR_close  TE_peak      5833
14 LR_close  not_TE_peak  8239
15 NR        TE_peak     37456
16 NR        not_TE_peak 48872</code></pre>
<div id="sine-status" class="section level4">
<h4>Sine status</h4>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, Sine_status) %&gt;% 
  ggplot(., aes(x=mrc, fill= Sine_status))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(&quot;Sine_status&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/status%20plots%20sine-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-status-plots-sine-1">
Past versions of "status plots sine-1.png"
</button>
</p>
<div id="fig-no-spaces-status-plots-sine-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/status plots sine-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, Sine_status) %&gt;% 
  group_by(mrc,Sine_status) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = Sine_status, values_from = n) %&gt;% 
  mutate(Per_peak = sine_peak/(not_sine+sine_peak)*100) %&gt;% 
  kable(., caption= &quot;Sine status of unique Peaks&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) </code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Sine status of unique Peaks
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
sine_peak
</th>
<th style="text-align:right;">
not_sine
</th>
<th style="text-align:right;">
Per_peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
800
</td>
<td style="text-align:right;">
2785
</td>
<td style="text-align:right;">
22.31520
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1232
</td>
<td style="text-align:right;">
3638
</td>
<td style="text-align:right;">
25.29774
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
1379
</td>
<td style="text-align:right;">
4605
</td>
<td style="text-align:right;">
23.04479
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
2458
</td>
<td style="text-align:right;">
8046
</td>
<td style="text-align:right;">
23.40061
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
388
</td>
<td style="text-align:right;">
956
</td>
<td style="text-align:right;">
28.86905
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
8817
</td>
<td style="text-align:right;">
24248
</td>
<td style="text-align:right;">
26.66566
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
3626
</td>
<td style="text-align:right;">
12252
</td>
<td style="text-align:right;">
22.83663
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
23227
</td>
<td style="text-align:right;">
74915
</td>
<td style="text-align:right;">
23.66673
</td>
</tr>
</tbody>
</table>
</div>
<div id="line-status" class="section level4">
<h4>Line status</h4>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, Line_status) %&gt;% 
  ggplot(., aes(x=mrc, fill= Line_status))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(&quot;Line_status&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/status%20plots%20lines-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-status-plots-lines-1">
Past versions of "status plots lines-1.png"
</button>
</p>
<div id="fig-no-spaces-status-plots-lines-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/status plots lines-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, Line_status) %&gt;% 
  group_by(mrc,Line_status) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = Line_status, values_from = n) %&gt;% 
  mutate(Per_peak = line_peak/(not_line+line_peak)*100) %&gt;% 
  kable(., caption= &quot;Line status of unique Peaks&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) </code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Line status of unique Peaks
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
line_peak
</th>
<th style="text-align:right;">
not_line
</th>
<th style="text-align:right;">
Per_peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
603
</td>
<td style="text-align:right;">
2882
</td>
<td style="text-align:right;">
17.30273
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
950
</td>
<td style="text-align:right;">
3763
</td>
<td style="text-align:right;">
20.15701
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
1126
</td>
<td style="text-align:right;">
4749
</td>
<td style="text-align:right;">
19.16596
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
1939
</td>
<td style="text-align:right;">
8278
</td>
<td style="text-align:right;">
18.97817
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
276
</td>
<td style="text-align:right;">
1023
</td>
<td style="text-align:right;">
21.24711
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
7051
</td>
<td style="text-align:right;">
25214
</td>
<td style="text-align:right;">
21.85340
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
3240
</td>
<td style="text-align:right;">
12339
</td>
<td style="text-align:right;">
20.79723
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
18221
</td>
<td style="text-align:right;">
77137
</td>
<td style="text-align:right;">
19.10799
</td>
</tr>
</tbody>
</table>
</div>
<div id="ltr-status" class="section level4">
<h4>LTR status</h4>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc,LTR_status) %&gt;% 
  ggplot(., aes(x=mrc, fill= LTR_status))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(&quot;LTR_status&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/status%20plots%20ltrs-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-status-plots-ltrs-1">
Past versions of "status plots ltrs-1.png"
</button>
</p>
<div id="fig-no-spaces-status-plots-ltrs-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/status plots ltrs-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, LTR_status) %&gt;% 
  group_by(mrc,LTR_status) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = LTR_status, values_from = n) %&gt;% 
  mutate(Per_peak = LTR_peak/(not_LTR+LTR_peak)*100) %&gt;% 
  kable(., caption= &quot;LTR status of unique Peaks&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) </code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
LTR status of unique Peaks
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
LTR_peak
</th>
<th style="text-align:right;">
not_LTR
</th>
<th style="text-align:right;">
Per_peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
356
</td>
<td style="text-align:right;">
2972
</td>
<td style="text-align:right;">
10.69712
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
674
</td>
<td style="text-align:right;">
3905
</td>
<td style="text-align:right;">
14.71937
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
621
</td>
<td style="text-align:right;">
4969
</td>
<td style="text-align:right;">
11.10912
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
1264
</td>
<td style="text-align:right;">
8614
</td>
<td style="text-align:right;">
12.79611
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
198
</td>
<td style="text-align:right;">
1053
</td>
<td style="text-align:right;">
15.82734
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
4871
</td>
<td style="text-align:right;">
26033
</td>
<td style="text-align:right;">
15.76171
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
2242
</td>
<td style="text-align:right;">
12784
</td>
<td style="text-align:right;">
14.92080
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
12012
</td>
<td style="text-align:right;">
79598
</td>
<td style="text-align:right;">
13.11211
</td>
</tr>
</tbody>
</table>
</div>
<div id="dna-status" class="section level4">
<h4>DNA status</h4>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, DNA_status) %&gt;% 
  ggplot(., aes(x=mrc, fill= DNA_status))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(&quot;DNA_status&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/status%20plots%20dna-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-status-plots-dna-1">
Past versions of "status plots dna-1.png"
</button>
</p>
<div id="fig-no-spaces-status-plots-dna-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/status plots dna-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, DNA_status) %&gt;% 
  group_by(mrc,DNA_status) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = DNA_status, values_from = n) %&gt;% 
  mutate(Per_peak = DNA_peak/(not_DNA+DNA_peak)*100) %&gt;% 
  kable(., caption= &quot;DNA status of unique Peaks&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) </code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
DNA status of unique Peaks
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
DNA_peak
</th>
<th style="text-align:right;">
not_DNA
</th>
<th style="text-align:right;">
Per_peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
269
</td>
<td style="text-align:right;">
3045
</td>
<td style="text-align:right;">
8.117079
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
363
</td>
<td style="text-align:right;">
4137
</td>
<td style="text-align:right;">
8.066667
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
486
</td>
<td style="text-align:right;">
5091
</td>
<td style="text-align:right;">
8.714363
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
777
</td>
<td style="text-align:right;">
9017
</td>
<td style="text-align:right;">
7.933429
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
124
</td>
<td style="text-align:right;">
1112
</td>
<td style="text-align:right;">
10.032362
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
3311
</td>
<td style="text-align:right;">
27192
</td>
<td style="text-align:right;">
10.854670
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
1363
</td>
<td style="text-align:right;">
13484
</td>
<td style="text-align:right;">
9.180306
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
8093
</td>
<td style="text-align:right;">
83036
</td>
<td style="text-align:right;">
8.880817
</td>
</tr>
</tbody>
</table>
</div>
<div id="retroposon-status" class="section level4">
<h4>Retroposon status</h4>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, Retro_status) %&gt;% 
  ggplot(., aes(x=mrc, fill= Retro_status))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(&quot;Retroposon status&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/status%20plots%20Retroposons-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-status-plots-Retroposons-1">
Past versions of "status plots Retroposons-1.png"
</button>
</p>
<div id="fig-no-spaces-status-plots-Retroposons-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/status plots Retroposons-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Class_status_df %&gt;% 
  distinct(Peakid, mrc, Retro_status) %&gt;% 
  group_by(mrc,Retro_status) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = Retro_status, values_from = n) %&gt;% 
  mutate(Per_peak = Retro_peak/(not_Retro+Retro_peak)*100) %&gt;% 
  kable(., caption= &quot;Retro status of unique Peaks&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) </code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Retro status of unique Peaks
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
Retro_peak
</th>
<th style="text-align:right;">
not_Retro
</th>
<th style="text-align:right;">
Per_peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
3149
</td>
<td style="text-align:right;">
0.6624606
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
4292
</td>
<td style="text-align:right;">
0.1628286
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
40
</td>
<td style="text-align:right;">
5260
</td>
<td style="text-align:right;">
0.7547170
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
9346
</td>
<td style="text-align:right;">
0.1175590
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1159
</td>
<td style="text-align:right;">
0.1722653
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
28529
</td>
<td style="text-align:right;">
0.1260284
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
14067
</td>
<td style="text-align:right;">
0.1277955
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
148
</td>
<td style="text-align:right;">
86286
</td>
<td style="text-align:right;">
0.1712289
</td>
</tr>
</tbody>
</table>
<div id="this-code-is-for-chi-analysis" class="section level5">
<h5>This code is for Chi analysis</h5>
<pre class="r"><code>####### LOoKing at the results from TE counts across clusters##### with cutoff

chi_TE_Eight &lt;- Eight_group_TE %&gt;% 
  # dplyr::filter(is.na(per_ol)|per_ol&gt;.5) %&gt;% 
  dplyr::select(Peakid,TEstatus, mrc)%&gt;% 
  distinct(Peakid, .keep_all = TRUE) %&gt;% 
  group_by(mrc,TEstatus) %&gt;% 
  tally %&gt;% 
  ungroup() %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
  column_to_rownames(&quot;mrc&quot;) 

chi_TE_nine_mat &lt;- chi_TE_Eight %&gt;% as.matrix

EARopenvNR &lt;- matrix(c(chi_TE_nine_mat[2,1],chi_TE_nine_mat[2,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE)%&gt;% chisq.test(.)

EARclosevNR &lt;- matrix(c(chi_TE_nine_mat[1,1],chi_TE_nine_mat[1,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE)%&gt;% chisq.test(.)

ESRopenvNR &lt;- matrix(c(chi_TE_nine_mat[5,1],chi_TE_nine_mat[5,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE)%&gt;% chisq.test(.)

ESRclosevNR &lt;- matrix(c(chi_TE_nine_mat[4,1],chi_TE_nine_mat[4,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE)%&gt;% chisq.test(.)

ESROCvNR &lt;- matrix(c(chi_TE_nine_mat[3,1],chi_TE_nine_mat[3,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE)%&gt;% chisq.test(.)


LRopenvNR &lt;- matrix(c(chi_TE_nine_mat[7,1],chi_TE_nine_mat[7,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE) %&gt;% chisq.test(.)

LRclosevNR &lt;- matrix(c(chi_TE_nine_mat[6,1],chi_TE_nine_mat[6,2],chi_TE_nine_mat[8,1],chi_TE_nine_mat[8,2]),ncol=2, byrow=TRUE)%&gt;% chisq.test(.)

pvalue &lt;- c(EARclosevNR$p.value, 
            EARopenvNR$p.value,
            ESROCvNR$p.value,
            ESRclosevNR$p.value,
            ESRopenvNR$p.value,
            LRclosevNR$p.value, 
            LRopenvNR$p.value, 
            &quot;na&quot;,
            &quot;not checked&quot;)

chi_TE_Eight %&gt;% 
  cbind(pvalue) %&gt;% 
  mutate(pvalue= as.numeric(pvalue)) %&gt;% 
  mutate(signif=if_else(pvalue&lt;0.005,&quot;***&quot;,if_else(pvalue&lt;0.01,&quot;**&quot;,if_else(pvalue&lt;0.05,&quot;*&quot;,&quot;ns&quot;)))) %&gt;% 
  mutate(per_TE=TE_peak/(TE_peak+not_TE_peak)*100) %&gt;% 
  mutate(per_TE=sprintf(&quot;%.2f%% &quot;,per_TE)) %&gt;% 
    kable(., caption = &quot;Unique Peak TE status with all TEs for each MRC, &gt;50% TE coverage&quot;) %&gt;% 
   kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Unique Peak TE status with all TEs for each MRC, &gt;50% TE coverage
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TE_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
not_TE_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
signif
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
per_TE
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1728
</td>
<td style="text-align:right;">
2568
</td>
<td style="text-align:right;">
0.0000469
</td>
<td style="text-align:left;">
***
</td>
<td style="text-align:left;">
40.22%
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1398
</td>
<td style="text-align:right;">
1760
</td>
<td style="text-align:right;">
0.3359162
</td>
<td style="text-align:left;">
ns
</td>
<td style="text-align:left;">
44.27%
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
496
</td>
<td style="text-align:right;">
664
</td>
<td style="text-align:right;">
0.6892784
</td>
<td style="text-align:left;">
ns
</td>
<td style="text-align:left;">
42.76%
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
3657
</td>
<td style="text-align:right;">
5692
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:left;">
***
</td>
<td style="text-align:left;">
39.12%
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
2441
</td>
<td style="text-align:right;">
2831
</td>
<td style="text-align:right;">
0.0000367
</td>
<td style="text-align:left;">
***
</td>
<td style="text-align:left;">
46.30%
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
5833
</td>
<td style="text-align:right;">
8239
</td>
<td style="text-align:right;">
0.0000176
</td>
<td style="text-align:left;">
***
</td>
<td style="text-align:left;">
41.45%
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
13022
</td>
<td style="text-align:right;">
15515
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:left;">
***
</td>
<td style="text-align:left;">
45.63%
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
37456
</td>
<td style="text-align:right;">
48872
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
43.39%
</td>
</tr>
<tr>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:right;">
1408
</td>
<td style="text-align:right;">
1977
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
41.60%
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="sub-class-breakdown" class="section level3">
<h3>sub-Class breakdown</h3>
<pre class="r"><code>per_cov &lt;- 0.5
 
ggline_df &lt;-Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(Peakid,repName,repClass, repFamily,width) %&gt;%
   mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;, per_ol = &quot;NA&quot;) %&gt;% 
    rbind(Eight_group_TE %&gt;% dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;%  mutate(repClass_org=repClass)) 
  

ggsine_df &lt;-
  Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(Peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;%
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
    rbind(Eight_group_TE %&gt;% dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;% mutate(repClass_org=repClass))

ggLTR_df &lt;-LTR_repeats %&gt;% 
   as.data.frame() %&gt;% 
 tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(Peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;%
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
    rbind(Eight_group_TE %&gt;% dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;% mutate(repClass_org=repClass))


ggDNA_df &lt;-DNA_repeats %&gt;% 
 
  as.data.frame() %&gt;% 
 tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(Peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;%
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
    rbind(Eight_group_TE %&gt;% dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;% mutate(repClass_org=repClass))


ggretroposon_df &lt;-retroposon_repeats %&gt;% 
 
  as.data.frame() %&gt;% 
 tidyr::unite(Peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(Peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;%
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
    rbind(Eight_group_TE %&gt;% dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;% mutate(repClass_org=repClass)) </code></pre>
</div>
<div id="eight-group-by-subset-te-type" class="section level3">
<h3>Eight group by subset TE type</h3>
<pre class="r"><code>eight_lines &lt;- Eight_group_TE %&gt;% 
    dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% 
  # distinct(mrc)
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggline_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;%
  dplyr::filter(mrc != &quot;h.genome&quot;)

eight_lines %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LINE breakdown by eight-clusters and Family&quot;, per_cov))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LinesLTRand%20eight-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LinesLTRand-eight-1">
Past versions of "LinesLTRand eight-1.png"
</button>
</p>
<div id="fig-no-spaces-LinesLTRand-eight-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LinesLTRand eight-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LinesLTRand eight-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>eight_lines %&gt;% 
  group_by(mrc,repFamily) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = repFamily, values_from = n) %&gt;% 
  rowwise() %&gt;% 
 mutate(total= sum(c_across(&quot;CR1&quot;:&quot;RTE-X&quot;),na.rm =TRUE)) %&gt;% 
  kable(., caption=&quot;Breakdown of SINE counts by Family&quot;) %&gt;% 
kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of SINE counts by Family
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CR1
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Dong-R4
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
L1
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
L2
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Penelope
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
RTE-BovB
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
RTE-X
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
39
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
253
</td>
<td style="text-align:right;">
427
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
739
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
81
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
348
</td>
<td style="text-align:right;">
633
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
1100
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
112
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
492
</td>
<td style="text-align:right;">
748
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
1403
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
165
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
717
</td>
<td style="text-align:right;">
1285
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
49
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
2233
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
110
</td>
<td style="text-align:right;">
179
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
324
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
710
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
2724
</td>
<td style="text-align:right;">
4849
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
142
</td>
<td style="text-align:right;">
128
</td>
<td style="text-align:right;">
8569
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
237
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1335
</td>
<td style="text-align:right;">
2058
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
81
</td>
<td style="text-align:right;">
62
</td>
<td style="text-align:right;">
3777
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
1660
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
7395
</td>
<td style="text-align:right;">
11769
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:right;">
290
</td>
<td style="text-align:right;">
276
</td>
<td style="text-align:right;">
21423
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
3101
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
13664
</td>
<td style="text-align:right;">
22418
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
649
</td>
<td style="text-align:right;">
546
</td>
<td style="text-align:right;">
40440
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>eight_sines &lt;- Eight_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;%
  dplyr::filter(mrc != &quot;h.genome&quot;) 

eight_sines %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;SINE breakdown by breakdown by eight-clusters and Family&quot;,per_cov))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Sines%20and%20eight-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sines-and-eight-1">
Past versions of "Sines and eight-1.png"
</button>
</p>
<div id="fig-no-spaces-Sines-and-eight-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Sines and eight-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Sines and eight-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>eight_sines %&gt;% 
  group_by(mrc,repFamily) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = repFamily, values_from = n) %&gt;% 
  rowwise() %&gt;% 
 mutate(total= sum(c_across(1:6),na.rm =TRUE)) %&gt;% 
  kable(., caption=&quot;Breakdown of SINE counts by Family&quot;) %&gt;% 
kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of SINE counts by Family
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
5S-Deu-L2
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Alu
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
MIR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
tRNA
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
tRNA-Deu
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
tRNA-RTE
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
459
</td>
<td style="text-align:right;">
530
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1003
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
709
</td>
<td style="text-align:right;">
822
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
1545
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
896
</td>
<td style="text-align:right;">
829
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
1753
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
1247
</td>
<td style="text-align:right;">
1727
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
3010
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
247
</td>
<td style="text-align:right;">
237
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
488
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
5631
</td>
<td style="text-align:right;">
5179
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
10996
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
35
</td>
<td style="text-align:right;">
1842
</td>
<td style="text-align:right;">
2440
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
4367
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
127
</td>
<td style="text-align:right;">
12831
</td>
<td style="text-align:right;">
15289
</td>
<td style="text-align:right;">
84
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
171
</td>
<td style="text-align:right;">
28532
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
239
</td>
<td style="text-align:right;">
24295
</td>
<td style="text-align:right;">
27669
</td>
<td style="text-align:right;">
154
</td>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
345
</td>
<td style="text-align:right;">
52755
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>eight_LTRs &lt;- Eight_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggLTR_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;%
  dplyr::filter(mrc != &quot;h.genome&quot;)

eight_LTRs %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LTR breakdown by breakdown by eight-clusters and Family&quot;,per_cov))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/LTR%20and%20eight-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-and-eight-1">
Past versions of "LTR and eight-1.png"
</button>
</p>
<div id="fig-no-spaces-LTR-and-eight-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/LTR and eight-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/LTR and eight-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>eight_LTRs %&gt;% 
  group_by(mrc,repFamily) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = repFamily, values_from = n) %&gt;% 
  rowwise() %&gt;% 
 mutate(total= sum(c_across(1:9),na.rm =TRUE)) %&gt;% 
  kable(., caption=&quot;Breakdown of LTR counts by Family&quot;) %&gt;% 
kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of LTR counts by Family
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
ERV1
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
ERVK
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
ERVL
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
ERVL-MaLR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
ERVL?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Gypsy
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Gypsy?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
LTR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
ERV1?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
122
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
108
</td>
<td style="text-align:right;">
157
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
427
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
201
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
202
</td>
<td style="text-align:right;">
335
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
787
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
239
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
184
</td>
<td style="text-align:right;">
305
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
776
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
353
</td>
<td style="text-align:right;">
40
</td>
<td style="text-align:right;">
428
</td>
<td style="text-align:right;">
618
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
40
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1508
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
57
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
79
</td>
<td style="text-align:right;">
92
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
245
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
1547
</td>
<td style="text-align:right;">
79
</td>
<td style="text-align:right;">
1602
</td>
<td style="text-align:right;">
2603
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
142
</td>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
6116
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
614
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
843
</td>
<td style="text-align:right;">
1046
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
69
</td>
<td style="text-align:right;">
40
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2684
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
4010
</td>
<td style="text-align:right;">
526
</td>
<td style="text-align:right;">
4215
</td>
<td style="text-align:right;">
5212
</td>
<td style="text-align:right;">
97
</td>
<td style="text-align:right;">
405
</td>
<td style="text-align:right;">
212
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
14757
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
7266
</td>
<td style="text-align:right;">
755
</td>
<td style="text-align:right;">
7821
</td>
<td style="text-align:right;">
10588
</td>
<td style="text-align:right;">
147
</td>
<td style="text-align:right;">
719
</td>
<td style="text-align:right;">
390
</td>
<td style="text-align:right;">
116
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
27844
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>eight_DNA &lt;- Eight_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggDNA_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;%
  dplyr::filter(mrc != &quot;h.genome&quot;) 

eight_DNA %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;DNA breakdown by breakdown by eight-clusters and Family&quot;,per_cov))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/DNA%20and%20eight-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-and-eight-1">
Past versions of "DNA and eight-1.png"
</button>
</p>
<div id="fig-no-spaces-DNA-and-eight-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/DNA and eight-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/DNA and eight-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>eight_DNA %&gt;% 
  group_by(mrc,repFamily) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = repFamily, values_from = n) %&gt;% 
  rowwise() %&gt;% 
 mutate(total= sum(c_across(1:18),na.rm =TRUE)) %&gt;% 
  kable(., caption=&quot;Breakdown of DNA counts by Family&quot;) %&gt;% 
kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of DNA counts by Family
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
DNA
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT-Ac
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT-Blackjack
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT-Charlie
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT-Tip100
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PiggyBac
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PiggyBac?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TcMar-Mariner
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TcMar-Tc2
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TcMar-Tigger
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT-Tip100?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
hAT?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
MULE-MuDR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TcMar?
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
PIF-Harbinger
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TcMar
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
TcMar-Pogo
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
165
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
67
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
301
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
228
</td>
<td style="text-align:right;">
52
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
81
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
408
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:right;">
262
</td>
<td style="text-align:right;">
65
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
141
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
546
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:right;">
496
</td>
<td style="text-align:right;">
136
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
117
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
854
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
75
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
136
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
52
</td>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
131
</td>
<td style="text-align:right;">
1873
</td>
<td style="text-align:right;">
370
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
1155
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
3809
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
813
</td>
<td style="text-align:right;">
261
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
273
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1522
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
135
</td>
<td style="text-align:right;">
142
</td>
<td style="text-align:right;">
371
</td>
<td style="text-align:right;">
4804
</td>
<td style="text-align:right;">
1132
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
106
</td>
<td style="text-align:right;">
82
</td>
<td style="text-align:right;">
2023
</td>
<td style="text-align:right;">
65
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
9040
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
127
</td>
<td style="text-align:right;">
276
</td>
<td style="text-align:right;">
250
</td>
<td style="text-align:right;">
670
</td>
<td style="text-align:right;">
8893
</td>
<td style="text-align:right;">
2097
</td>
<td style="text-align:right;">
89
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
219
</td>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
3979
</td>
<td style="text-align:right;">
95
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
16960
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>eight_retro &lt;- Eight_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggretroposon_df) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
mutate(mrc=factor(mrc, levels = c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;, &quot;ESR_close&quot;,&quot;ESR_OC&quot;, &quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;%
  dplyr::filter(mrc != &quot;h.genome&quot;) 

eight_retro %&gt;% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;Retroposon breakdown by by eight-clusters and Family&quot;,per_cov))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/Retroposon%20and%20eight-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Retroposon-and-eight-1">
Past versions of "Retroposon and eight-1.png"
</button>
</p>
<div id="fig-no-spaces-Retroposon-and-eight-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/Retroposon and eight-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/Retroposon and eight-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>eight_retro %&gt;% 
  group_by(mrc,repName) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = repName, values_from = n) %&gt;% 
  rowwise() %&gt;% 
 mutate(total= sum(c_across(1:6),na.rm =TRUE)) %&gt;% 
  kable(., caption=&quot;Breakdown of Retroposon counts by Name&quot;) %&gt;% 
kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of Retroposon counts by Name
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
SVA_A
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
SVA_B
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
SVA_D
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
SVA_E
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
SVA_F
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
SVA_C
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
total
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
38
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
148
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:right;">
121
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
295
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="cg-islands" class="section level3">
<h3>CG islands!</h3>
<pre class="r"><code>cpgislands_df &lt;- read.delim(&quot;data/other_papers/cpg_islands.tsv&quot;)
cpg_cCREs_df &lt;- read_delim(&quot;data/other_papers/cpg_cCREs.tsv&quot;, delim=&quot;\t&quot;)
# aligncre &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed&quot;) %&gt;% as.data.frame

CPG_promoters_gr &lt;- cpg_cCREs_df %&gt;% 
    dplyr::rename(.,&quot;seqnames&quot;=X1,&quot;start&quot;=X2,&quot;end&quot;=X3,&quot;promotor_name&quot;=X4,&quot;length&quot;=X5,&quot;strand&quot;=X6,&quot;color&quot;=X9) %&gt;% 
  dplyr::filter(color ==25500) %&gt;% 
  dplyr::select(seqnames:strand,color) %&gt;% 
  GRanges() 


Peaks_v_cpgpromo &lt;- join_overlap_intersect(CPG_promoters_gr,Col_TSS_data_gr) %&gt;% as.data.frame

cpg_island_gr &lt;- cpgislands_df %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;chrom&quot;, start.field = &quot;chromStart&quot;, end.field = &quot;chromEnd&quot;,starts.in.df.are.0based=TRUE)
Col_TSS_data_gr$peak_width &lt;- width(Col_TSS_data_gr)
cpg_island_gr$cpg_width &lt;- width(cpg_island_gr)</code></pre>
<pre class="r"><code>Col_fullDF_cug_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,cpg_island_gr)
Col_fullDF_cug_overlap &lt;-Col_fullDF_cug_overlap %&gt;% 
  as.data.frame %&gt;% 
  mutate(per_ol=width/cpg_width)


Col_fullDF_cug_overlap %&gt;% 
  as.data.frame() %&gt;% 
    # group_by(name) %&gt;%
  distinct(Peakid) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks with CG islands&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks with CG islands
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
18769
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>Col_fullDF_cug_overlap %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::filter(per_ol&gt;0.5) %&gt;% 
  distinct(Peakid) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks with &gt;50% of CG islands&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks with &gt;50% of CG islands
</caption>
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
15310
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code># CUG_mrc_status_list &lt;-
# Col_TSS_data_gr%&gt;% 
#   as.data.frame() %&gt;%
#   left_join(., (Col_fullDF_cug_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;Peakid&quot;=&quot;Peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;ensembl_ID&quot;=&quot;ensembl_ID&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;,  &quot;SYMBOL&quot; = &quot;SYMBOL&quot;, &quot;peak_width&quot;=&quot;peak_width&quot;)) %&gt;% 
#   left_join(., (Peaks_v_cpgpromo %&gt;% dplyr::select(promotor_name,Peakid,peak_width)), by=c(&quot;peak_width&quot;=&quot;peak_width&quot;,&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
#   dplyr::select(Peakid, name,cpgNum:promotor_name) %&gt;% 
#   mutate(cugstatus=if_else(is.na(cpgNum),&quot;not_CGi_peak&quot;,&quot;CGi_peak&quot;)) %&gt;% 
#   mutate(prom_status= if_else(is.na(promotor_name),&quot;not_CpGpromo&quot;,&quot;CpGpromo&quot;)) %&gt;% 
#    mutate(mrc=if_else(Peakid %in% EAR_df$id, &quot;EAR&quot;,
#                      if_else(Peakid %in% ESR_df$id,&quot;ESR&quot;,
#                              if_else(Peakid %in% LR_df$id,&quot;LR&quot;,
#                                      if_else(Peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;))))) %&gt;% distinct()
# 


CUG_mrc_nine_list &lt;-
Col_TSS_data_gr%&gt;% as.data.frame() %&gt;%
  left_join(., (Col_fullDF_cug_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;Peakid&quot;=&quot;Peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;ensembl_ID&quot;=&quot;ensembl_ID&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;,  &quot;SYMBOL&quot; = &quot;SYMBOL&quot;, &quot;peak_width&quot;=&quot;peak_width&quot;)) %&gt;% 
  left_join(., (Peaks_v_cpgpromo %&gt;% dplyr::select(promotor_name,Peakid,peak_width)), by=c(&quot;peak_width&quot;=&quot;peak_width&quot;,&quot;Peakid&quot;=&quot;Peakid&quot;)) %&gt;% 
  dplyr::select(Peakid, name,cpgNum:promotor_name) %&gt;% 
  mutate(cugstatus=if_else(is.na(cpgNum),&quot;not_CGi_peak&quot;,&quot;CGi_peak&quot;)) %&gt;% 
  mutate(prom_status= if_else(is.na(promotor_name),&quot;not_CpGpromo&quot;,&quot;CpGpromo&quot;)) %&gt;% 
   mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_OC$Peakid,&quot;ESR_OC&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR_df$Peakid,&quot;NR&quot;,&quot;not_mrc&quot;))))))))) %&gt;%
  distinct()

# 
# CUG_mrc_status_list %&gt;% 
#  group_by(cugstatus, mrc) %&gt;% 
#   distinct(Peakid) %&gt;% 
#  count %&gt;% 
#   pivot_wider(., id_cols = mrc, names_from = cugstatus, values_from = n) %&gt;% 
#   kable(., caption=&quot;Breakdown of CG islands overlap four groups&quot;) %&gt;% 
# kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
#   kable_styling(full_width = FALSE, font_size = 14)

# CUG_mrc_status_list %&gt;% 
#   mutate(mrc=factor(mrc, levels = c(&quot;NR&quot;, &quot;EAR&quot;, &quot;ESR&quot;,&quot;LR&quot;,&quot;not_mrc&quot;))) %&gt;%
#   group_by(cugstatus, mrc) %&gt;% 
#   ggplot(., aes(x = mrc,  fill = cugstatus)) +
#   geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
#   theme_bw()+
#   ggtitle(&quot;CG islands by mrc&quot;)
          # subtitle=paste((CUG_mrc_status_list %&gt;% dplyr::filter(cugstatus==&quot;CGi_peak&quot;) %&gt;% count(cugstatus))$n))
  
CUG_mrc_nine_list %&gt;%
  distinct(Peakid,.keep_all = TRUE) %&gt;%
  mutate(mrc=&quot;full_list&quot;) %&gt;%
  rbind(., (CUG_mrc_nine_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;%
   mutate(mrc=factor(mrc, levels=c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;,&quot;ESR_close&quot;,&quot;ESR_OC&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;%
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;%
  group_by(cugstatus, mrc) %&gt;%
  ggplot(., aes(x = mrc,  fill = cugstatus)) +
  geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
  theme_bw()+
  ggtitle(&quot;CG islands by mrc&quot;)</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/cug%20development-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cug-development-1">
Past versions of "cug development-1.png"
</button>
</p>
<div id="fig-no-spaces-cug-development-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/cug development-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/cug development-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># subtitle=paste((CUG_mrc_nine_list %&gt;% dplyr::filter(cugstatus==&quot;CGi_peak&quot;) %&gt;% count(cugstatus))$n, &quot;CG island peaks&quot;))
    
CUG_mrc_nine_list %&gt;% 
  distinct(Peakid,.keep_all = TRUE) %&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=&quot;full_list&quot;) %&gt;% 
  rbind(., (CUG_mrc_nine_list %&gt;% distinct(Peakid,.keep_all = TRUE))) %&gt;% 
   mutate(mrc=factor(mrc, levels=c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;,&quot;ESR_close&quot;,&quot;ESR_OC&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;%
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
  group_by(cugstatus, mrc) %&gt;% 
  tally %&gt;% 
  pivot_wider(., id_cols = mrc, names_from = cugstatus, values_from = n) %&gt;% 
  kable(., caption=&quot;Breakdown of CG islands overlap with not_mrc removed&quot;) %&gt;% 
kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)%&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Breakdown of CG islands overlap with not_mrc removed
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
CGi_peak
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
not_CGi_peak
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:right;">
81
</td>
<td style="text-align:right;">
3077
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
4279
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:right;">
139
</td>
<td style="text-align:right;">
5133
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:right;">
71
</td>
<td style="text-align:right;">
9278
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:right;">
307
</td>
<td style="text-align:right;">
28230
</td>
</tr>
<tr>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:right;">
102
</td>
<td style="text-align:right;">
13970
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
1265
</td>
<td style="text-align:right;">
85063
</td>
</tr>
<tr>
<td style="text-align:left;">
full_list
</td>
<td style="text-align:right;">
1982
</td>
<td style="text-align:right;">
150190
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
1160
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>Col_fullDF_cug_overlap %&gt;% 
        as.data.frame %&gt;% 
        ggplot(., aes (x = width))+
        geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of CGisland overlap widths&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/cug%20eight%20stuff-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cug-eight-stuff-1">
Past versions of "cug eight stuff-1.png"
</button>
</p>
<div id="fig-no-spaces-cug-eight-stuff-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/cug eight stuff-1.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/cug eight stuff-1.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>cpg_island_gr %&gt;% 
  as.data.frame() %&gt;% 
  ggplot(., aes (x = cpg_width))+
        geom_density(aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of overlap widths of CGislands &quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/cug%20eight%20stuff-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cug-eight-stuff-2">
Past versions of "cug eight stuff-2.png"
</button>
</p>
<div id="fig-no-spaces-cug-eight-stuff-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/cug eight stuff-2.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># CUG_mrc_status_list %&gt;% 
#    mutate(mrc=&quot;full_list&quot;) %&gt;% 
#   rbind(., CUG_mrc_status_list) %&gt;% 
#    mutate(mrc=factor(mrc, levels=c(&quot;EAR&quot;,&quot;ESR&quot;,&quot;LR&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;% 
#   dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
#   dplyr::filter(cugstatus==&quot;CGi_peak&quot;) %&gt;% 
#   ggplot(., aes(x = mrc,  fill = prom_status)) +
#   geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
#   theme_bw()+
#   ggtitle(&quot;CG islands promotors vs non-promotor CpGi by mrc&quot;, subtitle=paste(length(CUG_mrc_status_list$Peakid )))


promo_count_list &lt;- CUG_mrc_nine_list %&gt;% 
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
  dplyr::filter(cugstatus==&quot;CGi_peak&quot;)  %&gt;%
  group_by(prom_status) %&gt;% 
  tally()
  

CUG_mrc_nine_list %&gt;%
   mutate(mrc=&quot;full_list&quot;) %&gt;%
  rbind(., CUG_mrc_nine_list) %&gt;%
   mutate(mrc=factor(mrc, levels=c(&quot;EAR_open&quot;,&quot;EAR_close&quot;,&quot;ESR_open&quot;,&quot;ESR_close&quot;,&quot;ESR_OC&quot;,&quot;LR_open&quot;,&quot;LR_close&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;%
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;%
  dplyr::filter(cugstatus==&quot;CGi_peak&quot;) %&gt;%
  ggplot(., aes(x = mrc,  fill = prom_status)) +
  geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
  theme_bw()+
  ggtitle(&quot;CG islands promotors vs non-promotor CpGi by mrc&quot;, subtitle=paste(promo_count_list[1,1],promo_count_list[1,2],&quot;\n&quot;, promo_count_list [2,1],promo_count_list [2,2]))</code></pre>
<p><img src="figure/TE_analysis_ff.Rmd/cug%20eight%20stuff-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cug-eight-stuff-3">
Past versions of "cug eight stuff-3.png"
</button>
</p>
<div id="fig-no-spaces-cug-eight-stuff-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/23374bd9aeeacd337019f904091fa508d3a9b9a6/docs/figure/TE_analysis_ff.Rmd/cug eight stuff-3.png" target="_blank">23374bd</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/6d65dd60e07b20c2e2aac9fde164cb8368980f09/docs/figure/TE_analysis_ff.Rmd/cug eight stuff-3.png" target="_blank">6d65dd6</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-09-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="snp" class="section level3">
<h3>SNP</h3>
<pre class="r"><code>gwas_ACtox &lt;- readRDS(&quot;data/gwas_1_dataframe.RDS&quot;)  
gwas_ARR &lt;- readRDS(&quot;data/gwas_2_dataframe.RDS&quot;)
gwas_ACresp &lt;- readRDS(&quot;data/gwas_3_dataframe.RDS&quot;)
gwas_HD &lt;- readRDS(&quot;data/gwas_4_dataframe.RDS&quot;)
gwas_HF &lt;- readRDS(&quot;data/gwas_5_dataframe.RDS&quot;)
gwas_CAD &lt;- readRDS( &quot;data/CAD_gwas_dataframe.RDS&quot;)
gwas_MI &lt;- readRDS(&quot;data/MI_gwas.RDS&quot;)

gwas_snp_list &lt;- gwas_ACtox %&gt;%
  distinct(SNPS,.keep_all = TRUE) %&gt;% 
  dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
  mutate(gwas=&quot;ACtox&quot;) %&gt;% 
  rbind(gwas_ARR %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ARR&quot;)) %&gt;% 
  rbind(gwas_ACresp %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ACresp&quot;)) %&gt;% 
  rbind(gwas_HD %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HD&quot;)) %&gt;% 
  rbind(gwas_HF %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HF&quot;)) %&gt;% 
  rbind(gwas_CAD %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;CAD&quot;)) %&gt;% 
  rbind(gwas_MI %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;MI&quot;)) %&gt;% 
  separate_longer_delim(.,col= c(CHR_ID,CHR_POS,SNPS), delim= &quot;;&quot;)  

gwas_snp_gr &lt;- gwas_snp_list %&gt;% 
   mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit() %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;% 
  GRanges()

# rtracklayer::export.bed(gwas_snp_gr,con=&quot;data/full_bedfiles/GWAS_SNP.bed&quot;,format=&quot;bed&quot;)

findOverlaps(gwas_snp_gr, all_TEs_gr)</code></pre>
<pre><code>Hits object with 3432 hits and 0 metadata columns:
         queryHits subjectHits
         &lt;integer&gt;   &lt;integer&gt;
     [1]         1     4910496
     [2]         4     1047158
     [3]         7     4718029
     [4]        11      116853
     [5]        12     4826568
     ...       ...         ...
  [3428]      7804     5009192
  [3429]      7805     3904058
  [3430]      7807     4735830
  [3431]      7809     4615754
  [3432]      7812     1872218
  -------
  queryLength: 7816 / subjectLength: 5683690</code></pre>
<pre class="r"><code>test &lt;- join_overlap_intersect(gwas_snp_gr, all_TEs_gr) %&gt;% GRanges
##3413- (#3432 after separate)

test_new &lt;-  test %&gt;% 
   as.data.frame %&gt;% 
   dplyr::select(seqnames:gwas,repName:repFamily) %&gt;% 
   GRanges()
peaks &lt;-
  Collapsed_peaks %&gt;% 
    dplyr::select(chr:Peakid) %&gt;% 
    GRanges()
peak_test &lt;- join_overlap_intersect(test_new, peaks)

# peak_test 135

Col_fullDF_overlap %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(seqnames:Peakid,repName:repFamily) %&gt;% 
  GRanges() %&gt;% 
  join_overlap_intersect(.,gwas_snp_gr)</code></pre>
<pre><code>GRanges object with 131 ranges and 8 metadata columns:
        seqnames    ranges strand |                 Peakid     repName
           &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; |            &lt;character&gt; &lt;character&gt;
    [1]     chr1  16012818      * | chr1.16012567.16013729        MIRb
    [2]     chr1 170224718      * | chr1.170224576.17022..       L1MA7
    [3]     chr1 170224718      * | chr1.170224576.17022..       L1MA7
    [4]    chr10  20953453      * | chr10.20953148.20953..       MLT1B
    [5]    chr10  20953453      * | chr10.20953148.20953..       MLT1B
    ...      ...       ...    ... .                    ...         ...
  [127]     chr8 124847608      * | chr8.124847104.12484..     MamTip2
  [128]     chr8 124847608      * | chr8.124847104.12484..     MamTip2
  [129]     chr9 107755513      * | chr9.107755276.10775..  Charlie13a
  [130]     chr9 107755513      * | chr9.107755276.10775..  Charlie13a
  [131]     chr9 123933491      * | chr9.123933461.12393..       MLT1J
           repClass   repFamily    CHR_ID   CHR_POS        SNPS        gwas
        &lt;character&gt; &lt;character&gt; &lt;numeric&gt; &lt;numeric&gt; &lt;character&gt; &lt;character&gt;
    [1]        SINE         MIR         1  16012818  rs10927886          HD
    [2]        LINE          L1         1 170224718  rs12122060         ARR
    [3]        LINE          L1         1 170224718  rs12122060          HD
    [4]         LTR   ERVL-MaLR        10  20953453   rs7910227         ARR
    [5]         LTR   ERVL-MaLR        10  20953453   rs7910227          HD
    ...         ...         ...       ...       ...         ...         ...
  [127]         DNA  hAT-Tip100         8 124847608  rs34866937          HD
  [128]         DNA  hAT-Tip100         8 124847608  rs34866937          HF
  [129]         DNA hAT-Charlie         9 107755513    rs944172          HD
  [130]         DNA hAT-Charlie         9 107755513    rs944172         CAD
  [131]         LTR   ERVL-MaLR         9 123933491  rs10818894      ACresp
  -------
  seqinfo: 23 sequences from an unspecified genome; no seqlengths</code></pre>
<pre class="r"><code>test2 &lt;- join_overlap_intersect(peaks, gwas_snp_gr) 
  
test2 %&gt;% 
   join_overlap_intersect(., all_TEs_gr) %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(seqnames:gwas,repName:repFamily) %&gt;% 
  mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_OC$Peakid,&quot;ESR_OC&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR_df$Peakid,&quot;NR&quot;,&quot;not_mrc&quot;))))))))) %&gt;%
  # distinct(Peakid, .keep_all = TRUE) %&gt;% 
  group_by(gwas,mrc) %&gt;% 
  tally</code></pre>
<pre><code># A tibble: 24 × 3
# Groups:   gwas [6]
   gwas   mrc           n
   &lt;chr&gt;  &lt;chr&gt;     &lt;int&gt;
 1 ACresp LR_open       2
 2 ARR    ESR_close     2
 3 ARR    LR_close      4
 4 ARR    LR_open       3
 5 ARR    NR           12
 6 ARR    not_mrc       1
 7 CAD    EAR_close     1
 8 CAD    EAR_open      1
 9 CAD    ESR_close     2
10 CAD    LR_close      4
# ℹ 14 more rows</code></pre>
<pre class="r"><code>SNP_df &lt;- test2 %&gt;% 
   join_overlap_intersect(., all_TEs_gr) %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(seqnames:gwas,repName:repFamily) %&gt;% 
  mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_OC$Peakid,&quot;ESR_OC&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR_df$Peakid,&quot;NR&quot;,&quot;not_mrc&quot;))))))))) 

SNP_df %&gt;%  group_by(Peakid) %&gt;%
    summarise(snp_id=paste(unique(SNPS), collapse = &quot;,&quot;),
              gwas=paste(unique(gwas),collapse=&quot;,&quot;),
              repName = paste(unique(repName),collapse=&quot;,&quot;),
              mrc =paste(unique(mrc),collapse=&quot;,&quot;),
              repFamily= paste(unique(repFamily),collapse = &quot;,&quot;),
              location_chr= paste(unique(CHR_ID),collapse = &quot;,&quot;) ,
              location= paste(unique(CHR_POS),collapse = &quot;,&quot;) ,
              ) %&gt;% 
  kable(., caption = &quot;The output of all SNPs and their SNP category followed by location and peak&quot;) %&gt;% 
  
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
The output of all SNPs and their SNP category followed by location and
peak
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Peakid
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
snp_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gwas
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
repName
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
repFamily
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
location_chr
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
location
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
chr1.16012567.16013729
</td>
<td style="text-align:left;">
rs10927886
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
16012818
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.170224576.170224916
</td>
<td style="text-align:left;">
rs12122060
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
L1MA7
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
170224718
</td>
</tr>
<tr>
<td style="text-align:left;">
chr1.55055575.55055793
</td>
<td style="text-align:left;">
rs472495
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MLT2D
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
55055640
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.103712814.103713354
</td>
<td style="text-align:left;">
rs10509768
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
103713072
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.121155583.121156319
</td>
<td style="text-align:left;">
rs17101521
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LTR16A2
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
121156039
</td>
</tr>
<tr>
<td style="text-align:left;">
chr10.20953148.20953475
</td>
<td style="text-align:left;">
rs7910227
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MLT1B
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
20953453
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.100753128.100753902
</td>
<td style="text-align:left;">
rs7947761
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L1PBa
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
100753868
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.118911506.118912630
</td>
<td style="text-align:left;">
rs11606719
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
118911765
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.128759383.128760154
</td>
<td style="text-align:left;">
rs672149
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
128759487
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
rs4757877,rs2625322
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
AmnSINE1
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
5S-Deu-L2
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
19988745,19988805
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.90455310.90455981
</td>
<td style="text-align:left;">
rs534128177
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LTR14B
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERVK
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
90455679
</td>
</tr>
<tr>
<td style="text-align:left;">
chr11.9748359.9748943
</td>
<td style="text-align:left;">
rs472109
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LTR41
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
9748771
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.106865628.106866040
</td>
<td style="text-align:left;">
rs7977247
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
L2a
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
106865692
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.109538192.109538640
</td>
<td style="text-align:left;">
rs75524776
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
109538471
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.111171885.111172119
</td>
<td style="text-align:left;">
rs3809297
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
(TGCGTG)n
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
111171923
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.111569574.111570394
</td>
<td style="text-align:left;">
rs653178
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
MER3
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
111569952
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.20004923.20005338
</td>
<td style="text-align:left;">
rs11044977
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
L2a
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
20005226
</td>
</tr>
<tr>
<td style="text-align:left;">
chr12.24561685.24562685
</td>
<td style="text-align:left;">
rs2291437
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
(CCACCTC)n
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
24562114
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.110301444.110302036
</td>
<td style="text-align:left;">
rs4773141
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
110302006
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.21536977.21537596
</td>
<td style="text-align:left;">
rs11841562
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
21537382
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.22794559.22794879
</td>
<td style="text-align:left;">
rs9506925
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
OldhAT1
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
hAT-Ac
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
22794804
</td>
</tr>
<tr>
<td style="text-align:left;">
chr13.30740138.30740672
</td>
<td style="text-align:left;">
rs3885907
</td>
<td style="text-align:left;">
ACresp
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
30740318
</td>
</tr>
<tr>
<td style="text-align:left;">
chr14.58326948.58327320
</td>
<td style="text-align:left;">
rs2145598
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MER3
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
58327283
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.70171390.70172398
</td>
<td style="text-align:left;">
rs2415081
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
70171653
</td>
</tr>
<tr>
<td style="text-align:left;">
chr15.90885954.90886257
</td>
<td style="text-align:left;">
rs4932373
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
90886057
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.17823556.17824258
</td>
<td style="text-align:left;">
rs12936927
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
(CGCCC)n
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
17823651
</td>
</tr>
<tr>
<td style="text-align:left;">
chr17.2867997.2868307
</td>
<td style="text-align:left;">
rs12603284
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MamRTE1
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
RTE-BovB
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
2868218
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.35089781.35090661
</td>
<td style="text-align:left;">
rs476348
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
L1MB7
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
35090057
</td>
</tr>
<tr>
<td style="text-align:left;">
chr18.45565618.45565968
</td>
<td style="text-align:left;">
rs2852306
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
MIR3
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
45565696
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.18478653.18479216
</td>
<td style="text-align:left;">
rs11670056
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MER20
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
18479133
</td>
</tr>
<tr>
<td style="text-align:left;">
chr19.41352378.41354231
</td>
<td style="text-align:left;">
rs1800470
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
(CAGCAG)n
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
41353016
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.25936965.25937466
</td>
<td style="text-align:left;">
rs6546620
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
25937071
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.27262237.27263756
</td>
<td style="text-align:left;">
rs6759518
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
27263727
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.36922199.36922493
</td>
<td style="text-align:left;">
rs11124554
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
AluJr
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
36922355
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.60391661.60392192
</td>
<td style="text-align:left;">
rs243071
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
60391893
</td>
</tr>
<tr>
<td style="text-align:left;">
chr2.86367009.86367615
</td>
<td style="text-align:left;">
rs72926475
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
86367364
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.14118943.14119390
</td>
<td style="text-align:left;">
rs57346421
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
HERV1_I-int
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
14119015
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.29162621.29163485
</td>
<td style="text-align:left;">
rs73193808
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
AluSz6
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
29162981
</td>
</tr>
<tr>
<td style="text-align:left;">
chr21.34746498.34746999
</td>
<td style="text-align:left;">
rs2834618
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
34746814
</td>
</tr>
<tr>
<td style="text-align:left;">
chr22.38435884.38436418
</td>
<td style="text-align:left;">
rs2267386
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
38436107
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.123019226.123019547
</td>
<td style="text-align:left;">
rs7632505
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
AluJb
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
123019460
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.136350085.136351911
</td>
<td style="text-align:left;">
rs667920
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L1M4b
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
136350630
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.14216199.14216968
</td>
<td style="text-align:left;">
rs62232870
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
MLT1K
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
14216209
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.151483988.151484643
</td>
<td style="text-align:left;">
rs4387942
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L1PA7
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
151484399
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.169478628.169479748
</td>
<td style="text-align:left;">
rs2421649
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
L2b
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
169479545
</td>
</tr>
<tr>
<td style="text-align:left;">
chr3.57959935.57960395
</td>
<td style="text-align:left;">
rs1522387
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
MLT1K
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
57960369
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.148015916.148016422
</td>
<td style="text-align:left;">
rs10213171
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
AluJr4
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
148016386
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.168766280.168767270
</td>
<td style="text-align:left;">
rs7696431,rs869396
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CR1
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
168766574,168766849
</td>
</tr>
<tr>
<td style="text-align:left;">
chr4.76495482.76496240
</td>
<td style="text-align:left;">
rs2068063
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
76496050
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.115543502.115545233
</td>
<td style="text-align:left;">
rs13177180
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
G-rich
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
115544896
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.135458929.135459424
</td>
<td style="text-align:left;">
rs899162
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L2c
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
135459219
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.138068694.138070268
</td>
<td style="text-align:left;">
rs141654122
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
SVA_D
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
SVA
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
138070140
</td>
</tr>
<tr>
<td style="text-align:left;">
chr5.52859657.52860199
</td>
<td style="text-align:left;">
rs73102285
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CR1
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
52859808
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.118252411.118253042
</td>
<td style="text-align:left;">
rs281868
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
L2b
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
118252898
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.134047673.134047938
</td>
<td style="text-align:left;">
rs965652
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MamRep1894
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
hAT
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
134047815
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.28269507.28270003
</td>
<td style="text-align:left;">
rs1225600
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MLT2B4
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
28269667
</td>
</tr>
<tr>
<td style="text-align:left;">
chr6.39166177.39166915
</td>
<td style="text-align:left;">
rs56336142
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
39166323
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.100243501.100243833
</td>
<td style="text-align:left;">
rs117038461
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
L1M5
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
100243731
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.36459124.36459918
</td>
<td style="text-align:left;">
rs192407614
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LTR38B
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
36459695
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.836506.836873
</td>
<td style="text-align:left;">
rs11768850
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
MLT2B4
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
836590
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.92620778.92621787
</td>
<td style="text-align:left;">
rs42044
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
AluJb
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
92620826
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.93713592.93714106
</td>
<td style="text-align:left;">
rs376825901
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
Tigger1
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
TcMar-Tigger
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
93714028
</td>
</tr>
<tr>
<td style="text-align:left;">
chr7.99822872.99823551
</td>
<td style="text-align:left;">
rs62471956
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MER52A
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
99823462
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.124847104.124848853
</td>
<td style="text-align:left;">
rs35006907,rs34866937
</td>
<td style="text-align:left;">
ARR,HD,HF
</td>
<td style="text-align:left;">
MamTip2
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
hAT-Tip100
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
124847575,124847608
</td>
</tr>
<tr>
<td style="text-align:left;">
chr8.20007264.20008372
</td>
<td style="text-align:left;">
rs2083636,rs894211
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
MER34A,LTR48
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
20007752,20008236
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.107755276.107755816
</td>
<td style="text-align:left;">
rs944172
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
Charlie13a
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
107755513
</td>
</tr>
<tr>
<td style="text-align:left;">
chr9.123933461.123933812
</td>
<td style="text-align:left;">
rs10818894
</td>
<td style="text-align:left;">
ACresp
</td>
<td style="text-align:left;">
MLT1J
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
123933491
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code># SNP_df %&gt;%  group_by(Peakid) %&gt;%
#     summarise(snp_id=paste(unique(SNPS), collapse = &quot;,&quot;),
#               gwas=paste(unique(gwas),collapse=&quot;,&quot;),
#               repName = paste(unique(repName),collapse=&quot;,&quot;),
#               mrc =paste(unique(mrc),collapse=&quot;,&quot;),
#               repFamily= paste(unique(repFamily),collapse = &quot;,&quot;),
#               location_chr= paste(unique(CHR_ID),collapse = &quot;;&quot;) ,
#               location= paste(unique(CHR_POS),collapse = &quot;;&quot;) ,
#               ) %&gt;%
# write.csv(.,&quot;data/SNP_GWAS_PEAK_MRC_id.csv&quot;)
# rtracklayer::export(ESR_open,con = &quot;data/Final_four_data/meme_bed/ESR_open.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# 
# rtracklayer::export(ESR_close,con = &quot;data/Final_four_data/meme_bed/ESR_close.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# 
# rtracklayer::export(ESR_OC,con = &quot;data/Final_four_data/meme_bed/ESR_OC.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# rtracklayer::export(EAR_open,con = &quot;data/Final_four_data/meme_bed/EAR_open.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# rtracklayer::export(EAR_close,con = &quot;data/Final_four_data/meme_bed/EAR_close.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# 
# rtracklayer::export(LR_open,con = &quot;data/Final_four_data/meme_bed/LR_open.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# rtracklayer::export(LR_close,con = &quot;data/Final_four_data/meme_bed/LR_close.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)
# 
# rtracklayer::export(NR_df,con = &quot;data/Final_four_data/meme_bed/NR_df.bed&quot;, format=&quot;bed&quot;, ignore.strand = FALSE)</code></pre>
<div id="snp-annotation-file" class="section level5">
<h5>SNP annotation file</h5>
<pre class="r"><code>SNPanno &lt;- 
  test2 %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(Peakid,SNPS,gwas) %&gt;% 
 mutate(mrc=if_else(Peakid %in% EAR_open$Peakid, &quot;EAR_open&quot;,
                      if_else(Peakid %in% EAR_close$Peakid, &quot;EAR_close&quot;,
                              if_else(Peakid %in% ESR_open$Peakid,&quot;ESR_open&quot;,
                                      if_else(Peakid %in% ESR_close$Peakid,&quot;ESR_close&quot;,
                                              if_else(Peakid %in% ESR_OC$Peakid,&quot;ESR_OC&quot;,
                                                      if_else(Peakid %in% LR_open$Peakid,&quot;LR_open&quot;,
                                                                      if_else(Peakid %in% LR_close$Peakid,&quot;LR_close&quot;,
                                                                              if_else(Peakid %in% NR_df$Peakid,&quot;NR&quot;,&quot;not_mrc&quot;))))))))) %&gt;%
  left_join(Col_fullDF_cug_overlap) %&gt;% 
  mutate(has_cpg= if_else(is.na(name),&quot;not_CpGi&quot;,&quot;CpGi&quot;)) %&gt;% 
  dplyr::select(Peakid:mrc,has_cpg,name,per_ol) %&gt;% 
  dplyr::rename(&quot;CpG&quot;=&quot;has_cpg&quot;,&quot;cpg_per_ol&quot;=&quot;per_ol&quot;,&quot;CpG_name&quot;=&quot;name&quot;) %&gt;%
  left_join(., Peaks_v_cpgpromo, by=c(&quot;Peakid&quot;=&quot;Peakid&quot;))%&gt;% 
    dplyr::select(Peakid:cpg_per_ol,promotor_name) %&gt;% 
    mutate(CpG_promo=if_else(is.na(promotor_name),&quot;not_CpGpromo&quot;,&quot;CpGpromo&quot;)) %&gt;% 
  left_join(.,Filter_TE_list, by=c(Peakid=&quot;Peakid&quot;))%&gt;% 
  dplyr::select(SNPS, Peakid,gwas:CpG_promo,repName:repFamily, per_ol) %&gt;% 
  dplyr::rename(&quot;TE_per_ol&quot;=&quot;per_ol&quot;) %&gt;% 
    group_by(SNPS) %&gt;% 
    summarise(Peakid=paste(unique(Peakid), collapse = &quot;,&quot;),
              gwas=paste(unique(gwas),collapse=&quot;,&quot;),
              mrc=paste(unique(mrc),collapse=&quot;,&quot;),
               CpG=paste(unique(CpG),collapse=&quot;,&quot;),
               CpG_name=paste(unique(CpG_name),collapse=&quot;,&quot;),
               cpg_per_ol=paste(unique(cpg_per_ol),collapse=&quot;,&quot;),
               promotor_name=paste(unique(promotor_name),collapse=&quot;,&quot;),
               CpG_promo=paste(unique(CpG_promo),collapse=&quot;,&quot;),
               repName=paste(unique(repName),collapse=&quot;,&quot;),
               repClass=paste(unique(repClass),collapse=&quot;,&quot;),
               repFamily=paste(unique(repFamily),collapse=&quot;,&quot;),
               TE_per_ol=paste(unique(TE_per_ol),collapse=&quot;;&quot;))
              
# write.csv(SNPanno,&quot;data/Final_four_data/annotated_gwas_SNPS.csv&quot;)

SNPanno %&gt;% 
  kable(., caption = &quot;SNPs and their annotations from my data&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14) %&gt;% 
  scroll_box(height = &quot;500px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; ">
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
SNPs and their annotations from my data
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SNPS
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Peakid
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gwas
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mrc
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
CpG
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
CpG_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
cpg_per_ol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
promotor_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
CpG_promo
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
repName
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
repClass
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
repFamily
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
TE_per_ol
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
rs10083696
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10083697
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10121140
</td>
<td style="text-align:left;">
chr9.14292492.14292732
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10213171
</td>
<td style="text-align:left;">
chr4.148015916.148016422
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJr4
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.673400673400673
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10213376
</td>
<td style="text-align:left;">
chr4.148015002.148015341
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TCTCCTG)n,MIR3
</td>
<td style="text-align:left;">
Simple_repeat,SINE
</td>
<td style="text-align:left;">
Simple_repeat,MIR
</td>
<td style="text-align:left;">
1;0.135714285714286
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1032763
</td>
<td style="text-align:left;">
chr5.17118580.17119099
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10456100
</td>
<td style="text-align:left;">
chr6.39214962.39215733
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.820809248554913
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1049334
</td>
<td style="text-align:left;">
chr7.116560073.116560553
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10509768
</td>
<td style="text-align:left;">
chr10.103712814.103713354
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb,MIR3
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1;0.0384615384615385
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1052586
</td>
<td style="text-align:left;">
chr17.46940653.46941215
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10734649
</td>
<td style="text-align:left;">
chr11.9758098.9760569
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 72
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(GTGGC)n,MIR,GA-rich,L2c
</td>
<td style="text-align:left;">
Simple_repeat,SINE,Low_complexity,LINE
</td>
<td style="text-align:left;">
Simple_repeat,MIR,Low_complexity,L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10786662
</td>
<td style="text-align:left;">
chr10.102229686.102230890
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 141
</td>
<td style="text-align:left;">
0.268564356435644
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,(GGC)n,(GCCCGG)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10811650
</td>
<td style="text-align:left;">
chr9.22067279.22067652
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10818576
</td>
<td style="text-align:left;">
chr9.121650071.121650712
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER30,MIR3
</td>
<td style="text-align:left;">
DNA,SINE
</td>
<td style="text-align:left;">
hAT-Charlie,MIR
</td>
<td style="text-align:left;">
0.256198347107438;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10818579
</td>
<td style="text-align:left;">
chr9.121651049.121653070
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 78
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2724036
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
GA-rich,MIRb,L3
</td>
<td style="text-align:left;">
Low_complexity,SINE,LINE
</td>
<td style="text-align:left;">
Low_complexity,MIR,CR1
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10818894
</td>
<td style="text-align:left;">
chr9.123933461.123933812
</td>
<td style="text-align:left;">
ACresp
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1J
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
0.698412698412698
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10821415
</td>
<td style="text-align:left;">
chr9.94950339.94951513
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc,MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.746543778801843;1;0.633484162895928
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10824026
</td>
<td style="text-align:left;">
chr10.73660865.73661568
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluY,L1MA4A,AluSz
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
Alu,L1
</td>
<td style="text-align:left;">
0.342948717948718;1;0.164634146341463
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10871753
</td>
<td style="text-align:left;">
chr18.58289003.58291154
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2a,MLT1I
</td>
<td style="text-align:left;">
LINE,LTR
</td>
<td style="text-align:left;">
L2,ERVL-MaLR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10927886
</td>
<td style="text-align:left;">
chr1.16012567.16013729
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb,MIR3,L2b
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
MIR,L2
</td>
<td style="text-align:left;">
1;0.686567164179104
</td>
</tr>
<tr>
<td style="text-align:left;">
rs10931898
</td>
<td style="text-align:left;">
chr2.200305915.200307710
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 150
</td>
<td style="text-align:left;">
0.954334365325077
</td>
<td style="text-align:left;">
EH38E2065239
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(GGCG)n,(CGGCCC)n,(CGCCCG)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11038225
</td>
<td style="text-align:left;">
chr11.44955010.44955513
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.121848739495798
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11044977
</td>
<td style="text-align:left;">
chr12.20004923.20005338
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER85,L2a
</td>
<td style="text-align:left;">
DNA,LINE
</td>
<td style="text-align:left;">
PiggyBac,L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11054833
</td>
<td style="text-align:left;">
chr12.12349737.12350701
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 61
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1593399
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
L1ME4b
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11124554
</td>
<td style="text-align:left;">
chr2.36922199.36922493
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1MC4,AluJr
</td>
<td style="text-align:left;">
LINE,SINE
</td>
<td style="text-align:left;">
L1,Alu
</td>
<td style="text-align:left;">
0.0381231671554252;0.842622950819672
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11180610
</td>
<td style="text-align:left;">
chr12.75646380.75646944
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
Tigger19b
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
TcMar-Tigger
</td>
<td style="text-align:left;">
0.802083333333333
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1122608
</td>
<td style="text-align:left;">
chr19.11052376.11052994
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJr
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.244897959183673
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11235604
</td>
<td style="text-align:left;">
chr11.72821745.72822900
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 111
</td>
<td style="text-align:left;">
0.847676419965577
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11242465
</td>
<td style="text-align:left;">
chr5.139426247.139426836
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
Alu,AluSz6
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
1;0.72463768115942
</td>
</tr>
<tr>
<td style="text-align:left;">
rs112577387
</td>
<td style="text-align:left;">
chr4.22625204.22625843
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.811188811188811
</td>
</tr>
<tr>
<td style="text-align:left;">
rs112941079
</td>
<td style="text-align:left;">
chr5.9544368.9546915
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 138,CpG: 25
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(GGAGCGG)n,(CCG)n,(GCCCC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs112941127
</td>
<td style="text-align:left;">
chr2.217868752.217869273
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.220883534136546
</td>
</tr>
<tr>
<td style="text-align:left;">
rs112949822
</td>
<td style="text-align:left;">
chr5.108747709.108749572
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 110
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs113140904
</td>
<td style="text-align:left;">
chr5.9544368.9546915
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 138,CpG: 25
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(GGAGCGG)n,(CCG)n,(GCCCC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs113452171
</td>
<td style="text-align:left;">
chr7.91784539.91784870
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs113716316
</td>
<td style="text-align:left;">
chr1.27601871.27602613
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CCAGC)n,(TCCCGC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs113819537
</td>
<td style="text-align:left;">
chr12.26194928.26196665
</td>
<td style="text-align:left;">
ARR,HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 74
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1600228
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
AluJr,(CACC)n,(CTG)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
Alu,Simple_repeat
</td>
<td style="text-align:left;">
0.229452054794521;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs113920486
</td>
<td style="text-align:left;">
chr10.52539479.52540028
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamGypLTR1d
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
Gypsy
</td>
<td style="text-align:left;">
0.175572519083969
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11398961
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs114192718
</td>
<td style="text-align:left;">
chr2.128027137.128028647
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 165
</td>
<td style="text-align:left;">
0.86441647597254
</td>
<td style="text-align:left;">
EH38E2031716,EH38E2031718
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CGG)n,(CCGC)n,G-rich
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
1;0.780487804878049
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11465228
</td>
<td style="text-align:left;">
chr5.157575011.157576352
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 95
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2424756
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIR3,(AG)n,(T)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
0.191489361702128;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs114782882
</td>
<td style="text-align:left;">
chr12.75895398.75895669
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11554495
</td>
<td style="text-align:left;">
chr12.52903311.52905516
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 41
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb,(CCACC)n,(A)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
0.311827956989247;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11591147
</td>
<td style="text-align:left;">
chr1.55039222.55040233
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 85
</td>
<td style="text-align:left;">
0.88586387434555
</td>
<td style="text-align:left;">
EH38E1349458,EH38E1349459
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(GCT)n,MIR3
</td>
<td style="text-align:left;">
Simple_repeat,SINE
</td>
<td style="text-align:left;">
Simple_repeat,MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11606719
</td>
<td style="text-align:left;">
chr11.118911506.118912630
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11631816
</td>
<td style="text-align:left;">
chr15.73353612.73354309
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11642015
</td>
<td style="text-align:left;">
chr16.53768577.53768944
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
UCON8
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11643207
</td>
<td style="text-align:left;">
chr16.75463626.75465107
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 113
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1828262,EH38E1828263,EH38E1828264
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11670056
</td>
<td style="text-align:left;">
chr19.18478653.18479216
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER20
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
0.552941176470588
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11677932
</td>
<td style="text-align:left;">
chr2.237315068.237315792
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs116843064
</td>
<td style="text-align:left;">
chr19.8363631.8364760
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 52
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSz,(CCCCGAAT)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
Alu,Simple_repeat
</td>
<td style="text-align:left;">
0.869747899159664;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs117038461
</td>
<td style="text-align:left;">
chr7.100243501.100243833
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc,L1M5
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
MIR,L1
</td>
<td style="text-align:left;">
0.521276595744681;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11705555
</td>
<td style="text-align:left;">
chr22.27810496.27811281
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TGCACA)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
0.95
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11713141
</td>
<td style="text-align:left;">
chr3.138347976.138349043
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 132
</td>
<td style="text-align:left;">
0.979225684608121
</td>
<td style="text-align:left;">
EH38E2241845,EH38E2241846
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs117299725
</td>
<td style="text-align:left;">
chr9.76808694.76808955
</td>
<td style="text-align:left;">
ACtox,ACresp,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11759102
</td>
<td style="text-align:left;">
chr6.19836789.19839260
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 247
</td>
<td style="text-align:left;">
0.997826086956522
</td>
<td style="text-align:left;">
EH38E2452004
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
AmnSINE2
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
tRNA-Deu
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11768850
</td>
<td style="text-align:left;">
chr7.836506.836873
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT2B4
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
0.651612903225806
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11838267
</td>
<td style="text-align:left;">
chr12.7068459.7069322
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11838776
</td>
<td style="text-align:left;">
chr13.110387344.110389032
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs11841562
</td>
<td style="text-align:left;">
chr13.21536977.21537596
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2,AluSq2
</td>
<td style="text-align:left;">
LINE,SINE
</td>
<td style="text-align:left;">
L2,Alu
</td>
<td style="text-align:left;">
0.176470588235294;1;0.440251572327044
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12122060
</td>
<td style="text-align:left;">
chr1.170224576.170224916
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER53,MIRb,L1MA7
</td>
<td style="text-align:left;">
DNA,SINE,LINE
</td>
<td style="text-align:left;">
hAT,MIR,L1
</td>
<td style="text-align:left;">
0.0173913043478261;1;0.288571428571429
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12150051
</td>
<td style="text-align:left;">
chr17.12584681.12585327
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1M7
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
0.331269349845201
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12155623
</td>
<td style="text-align:left;">
chr8.48898785.48899940
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CA)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
0.787037037037037
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1225600
</td>
<td style="text-align:left;">
chr6.28269507.28270003
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT2B4,AluSc8
</td>
<td style="text-align:left;">
LTR,SINE
</td>
<td style="text-align:left;">
ERVL,Alu
</td>
<td style="text-align:left;">
0.82565130260521;0.259259259259259
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12440045
</td>
<td style="text-align:left;">
chr15.41490412.41490650
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1250258
</td>
<td style="text-align:left;">
chr2.215435176.215436848
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 43
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2072836
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CCCG)n,G-rich
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1250259
</td>
<td style="text-align:left;">
chr2.215435176.215436848
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 43
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2072836
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CCCG)n,G-rich
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1250566
</td>
<td style="text-align:left;">
chr10.79286266.79286796
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12603284
</td>
<td style="text-align:left;">
chr17.2867997.2868307
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamRTE1
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
RTE-BovB
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12640611
</td>
<td style="text-align:left;">
chr4.10102598.10102907
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MSTD,MIRb
</td>
<td style="text-align:left;">
LTR,SINE
</td>
<td style="text-align:left;">
ERVL-MaLR,MIR
</td>
<td style="text-align:left;">
0.197707736389685;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12712649
</td>
<td style="text-align:left;">
chr2.39631404.39632344
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER5A,MLT1O
</td>
<td style="text-align:left;">
DNA,LTR
</td>
<td style="text-align:left;">
hAT-Charlie,ERVL-MaLR
</td>
<td style="text-align:left;">
1;0.738095238095238
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12724121
</td>
<td style="text-align:left;">
chr1.236688731.236689166
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12740374
</td>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TCCTC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12893887
</td>
<td style="text-align:left;">
chr14.99644968.99645471
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 93
</td>
<td style="text-align:left;">
0.641221374045801
</td>
<td style="text-align:left;">
EH38E1741629,EH38E1741630
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12906125
</td>
<td style="text-align:left;">
chr15.90884159.90884726
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 45
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1787901
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
G-rich
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12936927
</td>
<td style="text-align:left;">
chr17.17823556.17824258
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 47
</td>
<td style="text-align:left;">
0.967213114754098
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CGCCC)n,MIRb
</td>
<td style="text-align:left;">
Simple_repeat,SINE
</td>
<td style="text-align:left;">
Simple_repeat,MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12980942
</td>
<td style="text-align:left;">
chr19.41324959.41326640
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CTCC)n,(CCCTG)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs12983897
</td>
<td style="text-align:left;">
chr19.17747415.17747946
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 64
</td>
<td style="text-align:left;">
0.738461538461539
</td>
<td style="text-align:left;">
EH38E1943617
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.0130718954248366
</td>
</tr>
<tr>
<td style="text-align:left;">
rs13176353
</td>
<td style="text-align:left;">
chr5.1800893.1802029
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 185
</td>
<td style="text-align:left;">
0.367839607201309
</td>
<td style="text-align:left;">
EH38E2353141
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs13177180
</td>
<td style="text-align:left;">
chr5.115543502.115545233
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 65
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2400588
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
G-rich,THE1B
</td>
<td style="text-align:left;">
Low_complexity,LTR
</td>
<td style="text-align:left;">
Low_complexity,ERVL-MaLR
</td>
<td style="text-align:left;">
1;0.304093567251462
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1333042
</td>
<td style="text-align:left;">
chr9.22102751.22103879
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,(TG)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs13346603
</td>
<td style="text-align:left;">
chr19.41439501.41440141
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 18
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJr
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.148936170212766
</td>
</tr>
<tr>
<td style="text-align:left;">
rs13402621
</td>
<td style="text-align:left;">
chr2.43231283.43231596
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs141654122
</td>
<td style="text-align:left;">
chr5.138068694.138070268
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,U6,MamTip2,AluSp,SVA_D
</td>
<td style="text-align:left;">
SINE,snRNA,DNA,Retroposon
</td>
<td style="text-align:left;">
MIR,snRNA,hAT-Tip100,Alu,SVA
</td>
<td style="text-align:left;">
1;0.177004538577912
</td>
</tr>
<tr>
<td style="text-align:left;">
rs145306069
</td>
<td style="text-align:left;">
chr1.203795403.203796608
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 57
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1413938
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1468498
</td>
<td style="text-align:left;">
chr10.114306394.114306913
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs147288039
</td>
<td style="text-align:left;">
chr9.95006342.95007055
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSc8
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.516778523489933
</td>
</tr>
<tr>
<td style="text-align:left;">
rs147631684
</td>
<td style="text-align:left;">
chr16.83599141.83599350
</td>
<td style="text-align:left;">
ACtox,ACresp,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1K
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
0.0246045694200351
</td>
</tr>
<tr>
<td style="text-align:left;">
rs148241618
</td>
<td style="text-align:left;">
chr18.45879810.45880598
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs148416395
</td>
<td style="text-align:left;">
chr22.46417011.46418228
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc,MIRb,AluSx1
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR,Alu
</td>
<td style="text-align:left;">
1;0.0946372239747634
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1522387
</td>
<td style="text-align:left;">
chr3.57959935.57960395
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1K
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
0.677083333333333
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1522388
</td>
<td style="text-align:left;">
chr3.57959935.57960395
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1K
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
0.677083333333333
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1537372
</td>
<td style="text-align:left;">
chr9.22102751.22103879
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,(TG)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1537373
</td>
<td style="text-align:left;">
chr9.22102751.22103879
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,(TG)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs16866933
</td>
<td style="text-align:left;">
chr2.179701478.179702086
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.367741935483871
</td>
</tr>
<tr>
<td style="text-align:left;">
rs170041
</td>
<td style="text-align:left;">
chr17.2266828.2267276
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.222996515679443
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17101521
</td>
<td style="text-align:left;">
chr10.121155583.121156319
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L3,LTR16A2
</td>
<td style="text-align:left;">
LINE,LTR
</td>
<td style="text-align:left;">
CR1,ERVL
</td>
<td style="text-align:left;">
0.363247863247863;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17118812
</td>
<td style="text-align:left;">
chr5.140322792.140323772
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17228212
</td>
<td style="text-align:left;">
chr15.67165920.67166678
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2b
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17375901
</td>
<td style="text-align:left;">
chr1.11792123.11792871
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c,MIRc
</td>
<td style="text-align:left;">
LINE,SINE
</td>
<td style="text-align:left;">
L2,MIR
</td>
<td style="text-align:left;">
0.3625;0.56
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17458018
</td>
<td style="text-align:left;">
chr2.215420519.215420812
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs175040
</td>
<td style="text-align:left;">
chr14.75002369.75003794
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 53
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1728281
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17608766
</td>
<td style="text-align:left;">
chr17.46935408.46936158
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1769758
</td>
<td style="text-align:left;">
chr10.79139073.79139940
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs17782904
</td>
<td style="text-align:left;">
chr18.44733224.44733992
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1788826
</td>
<td style="text-align:left;">
chr18.23574050.23574545
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1800449
</td>
<td style="text-align:left;">
chr5.122076732.122078641
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 137
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1800470
</td>
<td style="text-align:left;">
chr19.41352378.41354231
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 139
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1954757,EH38E1954758
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CAGCAG)n,(TCC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1800797
</td>
<td style="text-align:left;">
chr7.22726398.22726697
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1800978
</td>
<td style="text-align:left;">
chr9.104903653.104904005
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1822273
</td>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AmnSINE1
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
5S-Deu-L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs190258023
</td>
<td style="text-align:left;">
chr22.46417011.46418228
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc,MIRb,AluSx1
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR,Alu
</td>
<td style="text-align:left;">
1;0.0946372239747634
</td>
</tr>
<tr>
<td style="text-align:left;">
rs191615952
</td>
<td style="text-align:left;">
chr19.2235309.2237165
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 157
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1932153,EH38E1932154
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MLT1C,AluJr
</td>
<td style="text-align:left;">
LTR,SINE
</td>
<td style="text-align:left;">
ERVL-MaLR,Alu
</td>
<td style="text-align:left;">
0.0374149659863946;0.102040816326531
</td>
</tr>
<tr>
<td style="text-align:left;">
rs192407614
</td>
<td style="text-align:left;">
chr7.36459124.36459918
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR38B
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1926032
</td>
<td style="text-align:left;">
chr10.103069381.103070029
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1ME3A
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
0.2
</td>
</tr>
<tr>
<td style="text-align:left;">
rs1962412
</td>
<td style="text-align:left;">
chr17.48892152.48893499
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 35
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1868507
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
AluSx
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.0487012987012987
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2052923
</td>
<td style="text-align:left;">
chr2.43184483.43184778
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb,Cheshire
</td>
<td style="text-align:left;">
SINE,DNA
</td>
<td style="text-align:left;">
MIR,hAT-Charlie
</td>
<td style="text-align:left;">
1;0.100917431192661
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2068063
</td>
<td style="text-align:left;">
chr4.76495482.76496240
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR16A1,L2
</td>
<td style="text-align:left;">
LTR,LINE
</td>
<td style="text-align:left;">
ERVL,L2
</td>
<td style="text-align:left;">
0.337264150943396;0.889583333333333
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2071502
</td>
<td style="text-align:left;">
chr17.7510984.7512510
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSx
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.902654867256637
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2073533
</td>
<td style="text-align:left;">
chr7.13989380.13991798
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 18,CpG: 21
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2535294
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(AACA)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2076380
</td>
<td style="text-align:left;">
chr20.38164364.38165984
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 35
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2111384
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIRc,(AC)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2083636
</td>
<td style="text-align:left;">
chr8.20007264.20008372
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR32,MER34A,LTR48
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL,ERV1
</td>
<td style="text-align:left;">
0.875294117647059;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2099684
</td>
<td style="text-align:left;">
chr1.161530078.161531531
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 32
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
tRNA-Leu-CTG,tRNA-Gly-GGA,(TTCC)n,Charlie5
</td>
<td style="text-align:left;">
tRNA,Simple_repeat,DNA
</td>
<td style="text-align:left;">
tRNA,Simple_repeat,hAT-Charlie
</td>
<td style="text-align:left;">
1;0.204301075268817
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2145274
</td>
<td style="text-align:left;">
chr20.6590860.6591642
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
Penelope1_Vert
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
Penelope
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2145598
</td>
<td style="text-align:left;">
chr14.58326948.58327320
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c,MER3
</td>
<td style="text-align:left;">
LINE,DNA
</td>
<td style="text-align:left;">
L2,hAT-Charlie
</td>
<td style="text-align:left;">
0.219123505976096;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs216172
</td>
<td style="text-align:left;">
chr17.2222861.2223544
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2220127
</td>
<td style="text-align:left;">
chr2.178846382.178846751
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.38655462184874
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2220427
</td>
<td style="text-align:left;">
chr4.110793293.110793943
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2267386
</td>
<td style="text-align:left;">
chr22.38435884.38436418
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2269001
</td>
<td style="text-align:left;">
chr7.150954727.150956459
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 23
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2600998
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2281727
</td>
<td style="text-align:left;">
chr17.2214040.2216112
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(AC)n,GA-rich
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2286466
</td>
<td style="text-align:left;">
chr16.1963596.1965464
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1796197,EH38E1796198,EH38E1796199
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
AluSx3,(CCCCGG)n,(CTGACCC)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
Alu,Simple_repeat
</td>
<td style="text-align:left;">
0.0114068441064639;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2291437
</td>
<td style="text-align:left;">
chr12.24561685.24562685
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 92
</td>
<td style="text-align:left;">
0.495515695067265
</td>
<td style="text-align:left;">
EH38E1599170,EH38E1599171
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
G-rich,(CCACCTC)n,(TG)n,(TCGC)n
</td>
<td style="text-align:left;">
Low_complexity,Simple_repeat
</td>
<td style="text-align:left;">
Low_complexity,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2306363
</td>
<td style="text-align:left;">
chr11.65637814.65638912
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 36
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1545418
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.252873563218391
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2306374
</td>
<td style="text-align:left;">
chr3.138400627.138401136
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2359171
</td>
<td style="text-align:left;">
chr16.73018778.73020183
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(GCGTGT)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2410859
</td>
<td style="text-align:left;">
chr17.75844492.75845572
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(GTGGG)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2415081
</td>
<td style="text-align:left;">
chr15.70171390.70172398
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER102c,MIRb,MIR3
</td>
<td style="text-align:left;">
DNA,SINE
</td>
<td style="text-align:left;">
hAT-Charlie,MIR
</td>
<td style="text-align:left;">
0.788793103448276;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2421649
</td>
<td style="text-align:left;">
chr3.169478628.169479748
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2b,(A)n
</td>
<td style="text-align:left;">
LINE,Simple_repeat
</td>
<td style="text-align:left;">
L2,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs243071
</td>
<td style="text-align:left;">
chr2.60391661.60392192
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2452600
</td>
<td style="text-align:left;">
chr4.94575665.94576013
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2493298
</td>
<td style="text-align:left;">
chr1.3408398.3409463
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 37
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2507
</td>
<td style="text-align:left;">
chr15.73983047.73983757
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CCTTC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2540949
</td>
<td style="text-align:left;">
chr2.65055695.65057220
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 132
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2004398
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2595104
</td>
<td style="text-align:left;">
chr4.110631655.110632575
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 100
</td>
<td style="text-align:left;">
0.576480990274094
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2625322
</td>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AmnSINE1
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
5S-Deu-L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2660739
</td>
<td style="text-align:left;">
chr3.78644690.78645022
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs281868
</td>
<td style="text-align:left;">
chr6.118252411.118253042
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2a,L2b
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.492152466367713;0.26080476900149
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2834618
</td>
<td style="text-align:left;">
chr21.34746498.34746999
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSx,MIRc,(CTCC)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
Alu,MIR,Simple_repeat
</td>
<td style="text-align:left;">
0.201320132013201;1;0.526315789473684
</td>
</tr>
<tr>
<td style="text-align:left;">
rs2852306
</td>
<td style="text-align:left;">
chr18.45565618.45565968
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.614814814814815
</td>
</tr>
<tr>
<td style="text-align:left;">
rs295114
</td>
<td style="text-align:left;">
chr2.200330546.200331118
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3176326
</td>
<td style="text-align:left;">
chr6.36678380.36679788
</td>
<td style="text-align:left;">
ARR,HD,HF
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 204
</td>
<td style="text-align:left;">
0.594777127420081
</td>
<td style="text-align:left;">
EH38E2463466
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3213420
</td>
<td style="text-align:left;">
chr16.72008437.72009100
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 50
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1826130
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs326
</td>
<td style="text-align:left;">
chr8.19961869.19963021
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs328
</td>
<td style="text-align:left;">
chr8.19961869.19963021
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs337711
</td>
<td style="text-align:left;">
chr5.114412835.114413212
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs34292822
</td>
<td style="text-align:left;">
chr1.154839629.154839995
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs34774090
</td>
<td style="text-align:left;">
chr19.11142349.11143639
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
EAR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR1_Amn,G-rich,MER20
</td>
<td style="text-align:left;">
SINE,Low_complexity,DNA
</td>
<td style="text-align:left;">
MIR,Low_complexity,hAT-Charlie
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs34866937
</td>
<td style="text-align:left;">
chr8.124847104.124848853
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamTip2,AluSx
</td>
<td style="text-align:left;">
DNA,SINE
</td>
<td style="text-align:left;">
hAT-Tip100,Alu
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs34871776
</td>
<td style="text-align:left;">
chr3.12773796.12774087
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs34969716
</td>
<td style="text-align:left;">
chr6.18209665.18210014
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJr,MER5B
</td>
<td style="text-align:left;">
SINE,DNA
</td>
<td style="text-align:left;">
Alu,hAT-Charlie
</td>
<td style="text-align:left;">
0.636904761904762;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs35006907
</td>
<td style="text-align:left;">
chr8.124847104.124848853
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamTip2,AluSx
</td>
<td style="text-align:left;">
DNA,SINE
</td>
<td style="text-align:left;">
hAT-Tip100,Alu
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs35176054
</td>
<td style="text-align:left;">
chr10.103720444.103721025
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs35267671
</td>
<td style="text-align:left;">
chr1.37931148.37932171
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 53
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1338382
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIR3
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs35620480
</td>
<td style="text-align:left;">
chr8.11641900.11643102
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1K,MIRb,(C)n
</td>
<td style="text-align:left;">
LTR,SINE,Simple_repeat
</td>
<td style="text-align:left;">
ERVL-MaLR,MIR,Simple_repeat
</td>
<td style="text-align:left;">
0.588832487309645;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs35946663
</td>
<td style="text-align:left;">
chr14.96362201.96364292
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 101
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1739803,EH38E1739804
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CGGCTC)n,MIR
</td>
<td style="text-align:left;">
Simple_repeat,SINE
</td>
<td style="text-align:left;">
Simple_repeat,MIR
</td>
<td style="text-align:left;">
1;0.248780487804878
</td>
</tr>
<tr>
<td style="text-align:left;">
rs360153
</td>
<td style="text-align:left;">
chr11.9740217.9740734
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs360801
</td>
<td style="text-align:left;">
chr2.62727824.62728333
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR35,L1MB8
</td>
<td style="text-align:left;">
LTR,LINE
</td>
<td style="text-align:left;">
ERV1,L1
</td>
<td style="text-align:left;">
0.886917960088692;0.107569721115538
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3731748
</td>
<td style="text-align:left;">
chr2.178547784.178549172
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_OC
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
OldhAT1
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Ac
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3734634
</td>
<td style="text-align:left;">
chr6.125789773.125791822
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 107
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2500543,EH38E2500544
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CCT)n,(CGC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs376825901
</td>
<td style="text-align:left;">
chr7.93713592.93714106
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
Tigger1
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
TcMar-Tigger
</td>
<td style="text-align:left;">
0.213959285417532
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3787662
</td>
<td style="text-align:left;">
chr21.29151834.29152614
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2b
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3803802
</td>
<td style="text-align:left;">
chr17.7718045.7718289
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3807989
</td>
<td style="text-align:left;">
chr7.116545872.116546373
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3809297
</td>
<td style="text-align:left;">
chr12.111171885.111172119
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3813127
</td>
<td style="text-align:left;">
chr18.22457586.22457998
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.418439716312057
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3814864
</td>
<td style="text-align:left;">
chr14.72893758.72895206
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 165
</td>
<td style="text-align:left;">
0.218648018648019
</td>
<td style="text-align:left;">
EH38E1726798,EH38E1726799
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
GA-rich
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3822127
</td>
<td style="text-align:left;">
chr4.173529475.173530537
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 120
</td>
<td style="text-align:left;">
0.593113141250878
</td>
<td style="text-align:left;">
EH38E2344608,EH38E2344610
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(CCGGCT)n,(CCCTC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3822259
</td>
<td style="text-align:left;">
chr4.10115366.10117581
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 136
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2280769,EH38E2280771,EH38E2280772
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIRc,5S,(GGCAG)n,G-rich,(CCGGCC)n
</td>
<td style="text-align:left;">
SINE,rRNA,Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
MIR,rRNA,Simple_repeat,Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3853445
</td>
<td style="text-align:left;">
chr4.110840197.110840561
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3885907
</td>
<td style="text-align:left;">
chr13.30740138.30740672
</td>
<td style="text-align:left;">
ACresp
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L3,MIR,(TC)n,L2
</td>
<td style="text-align:left;">
LINE,SINE,Simple_repeat
</td>
<td style="text-align:left;">
CR1,MIR,Simple_repeat,L2
</td>
<td style="text-align:left;">
0.385869565217391;1;0.27319587628866
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3895874
</td>
<td style="text-align:left;">
chr17.48970223.48970597
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs3904323
</td>
<td style="text-align:left;">
chr13.22794559.22794879
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
OldhAT1
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Ac
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs42044
</td>
<td style="text-align:left;">
chr7.92620778.92621787
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJb,MamRTE1
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
Alu,RTE-BovB
</td>
<td style="text-align:left;">
0.506802721088435;0.298449612403101
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4234323
</td>
<td style="text-align:left;">
chr3.151479569.151479846
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4234324
</td>
<td style="text-align:left;">
chr3.151479569.151479846
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4387942
</td>
<td style="text-align:left;">
chr3.151483988.151484643
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1PA7
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
0.101516558341071
</td>
</tr>
<tr>
<td style="text-align:left;">
rs472109
</td>
<td style="text-align:left;">
chr11.9748359.9748943
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR41,MIRb
</td>
<td style="text-align:left;">
LTR,SINE
</td>
<td style="text-align:left;">
ERVL,MIR
</td>
<td style="text-align:left;">
0.61864406779661;1;0.703703703703704
</td>
</tr>
<tr>
<td style="text-align:left;">
rs472495
</td>
<td style="text-align:left;">
chr1.55055575.55055793
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4757877
</td>
<td style="text-align:left;">
chr11.19988504.19989024
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AmnSINE1
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
5S-Deu-L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs476348
</td>
<td style="text-align:left;">
chr18.35089781.35090661
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1MB7,L1MA6,AluSx1
</td>
<td style="text-align:left;">
LINE,SINE
</td>
<td style="text-align:left;">
L1,Alu
</td>
<td style="text-align:left;">
0.478102189781022;1;0.67948717948718
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4773140
</td>
<td style="text-align:left;">
chr13.110301444.110302036
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.210682492581602
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4773141
</td>
<td style="text-align:left;">
chr13.110301444.110302036
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.210682492581602
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4773144
</td>
<td style="text-align:left;">
chr13.110305804.110309074
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 178,CpG: 19
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1697729,EH38E1697730
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(AAGAAC)n,(GGC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4871397
</td>
<td style="text-align:left;">
chr8.123622558.123623378
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4894803
</td>
<td style="text-align:left;">
chr3.172082306.172082889
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4896104
</td>
<td style="text-align:left;">
chr6.134797479.134798124
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4932373
</td>
<td style="text-align:left;">
chr15.90885954.90886257
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR,MER58A
</td>
<td style="text-align:left;">
SINE,DNA
</td>
<td style="text-align:left;">
MIR,hAT-Charlie
</td>
<td style="text-align:left;">
1;0.0588235294117647
</td>
</tr>
<tr>
<td style="text-align:left;">
rs494207
</td>
<td style="text-align:left;">
chr10.44245450.44245918
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1MB2
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
0.566343042071197
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4951261
</td>
<td style="text-align:left;">
chr1.205748526.205750556
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 100
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4977575
</td>
<td style="text-align:left;">
chr9.22124697.22124840
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs4999127
</td>
<td style="text-align:left;">
chr1.154741375.154741854
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs518594
</td>
<td style="text-align:left;">
chr10.44261009.44262169
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs523297
</td>
<td style="text-align:left;">
chr10.44261009.44262169
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs534128177
</td>
<td style="text-align:left;">
chr11.90455310.90455981
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1PA16,LTR14B
</td>
<td style="text-align:left;">
LINE,LTR
</td>
<td style="text-align:left;">
L1,ERVK
</td>
<td style="text-align:left;">
0.0403825717321998;1;0.10625
</td>
</tr>
<tr>
<td style="text-align:left;">
rs55734480
</td>
<td style="text-align:left;">
chr7.14331921.14332483
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(GTTT)n,MIR,MER113
</td>
<td style="text-align:left;">
Simple_repeat,SINE,DNA
</td>
<td style="text-align:left;">
Simple_repeat,MIR,hAT-Charlie
</td>
<td style="text-align:left;">
1;0.00966183574879227
</td>
</tr>
<tr>
<td style="text-align:left;">
rs56210063
</td>
<td style="text-align:left;">
chr11.8767540.8768078
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2a,L3
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2,CR1
</td>
<td style="text-align:left;">
0.422077922077922;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs56281979
</td>
<td style="text-align:left;">
chr3.14232390.14233136
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,L3,AluSz6
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
MIR,CR1,Alu
</td>
<td style="text-align:left;">
0.72;1;0.130281690140845
</td>
</tr>
<tr>
<td style="text-align:left;">
rs56336142
</td>
<td style="text-align:left;">
chr6.39166177.39166915
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs564427867
</td>
<td style="text-align:left;">
chr1.55039222.55040233
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 85
</td>
<td style="text-align:left;">
0.88586387434555
</td>
<td style="text-align:left;">
EH38E1349458,EH38E1349459
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(GCT)n,MIR3
</td>
<td style="text-align:left;">
Simple_repeat,SINE
</td>
<td style="text-align:left;">
Simple_repeat,MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs57346421
</td>
<td style="text-align:left;">
chr21.14118943.14119390
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
HERV1_I-int
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
0.0510483135824977
</td>
</tr>
<tr>
<td style="text-align:left;">
rs582384
</td>
<td style="text-align:left;">
chr2.45668749.45669787
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc,L1M5,AluSx1
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
MIR,L1,Alu
</td>
<td style="text-align:left;">
0.260869565217391;1;0.196013289036545
</td>
</tr>
<tr>
<td style="text-align:left;">
rs585967
</td>
<td style="text-align:left;">
chr2.21047256.21047719
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CA)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs5867305
</td>
<td style="text-align:left;">
chr5.36156365.36157193
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER5A,MER112
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs587606498
</td>
<td style="text-align:left;">
chr1.121213617.121213779
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs588136
</td>
<td style="text-align:left;">
chr15.58437614.58438322
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs590121
</td>
<td style="text-align:left;">
chr11.75561990.75564409
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 53
</td>
<td style="text-align:left;">
0.873873873873874
</td>
<td style="text-align:left;">
EH38E1553275
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
G-rich
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs60280851
</td>
<td style="text-align:left;">
chr15.68666479.68666914
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
5S
</td>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs604723
</td>
<td style="text-align:left;">
chr11.100739719.100740030
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs62042066
</td>
<td style="text-align:left;">
chr16.86664502.86665119
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AmnSINE1,LTR16A2
</td>
<td style="text-align:left;">
SINE,LTR
</td>
<td style="text-align:left;">
5S-Deu-L2,ERVL
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs62139062
</td>
<td style="text-align:left;">
chr2.65271531.65272160
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSp
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.385665529010239
</td>
</tr>
<tr>
<td style="text-align:left;">
rs62232870
</td>
<td style="text-align:left;">
chr3.14216199.14216968
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1K,MER117,MIRb
</td>
<td style="text-align:left;">
LTR,DNA,SINE
</td>
<td style="text-align:left;">
ERVL-MaLR,hAT-Charlie,MIR
</td>
<td style="text-align:left;">
0.767313019390582;1;0.248587570621469
</td>
</tr>
<tr>
<td style="text-align:left;">
rs62471956
</td>
<td style="text-align:left;">
chr7.99822872.99823551
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER52A
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERV1
</td>
<td style="text-align:left;">
0.416922133660331
</td>
</tr>
<tr>
<td style="text-align:left;">
rs62568141
</td>
<td style="text-align:left;">
chr9.77011996.77012519
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs629301
</td>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TCCTC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6426551
</td>
<td style="text-align:left;">
chr1.226353539.226354600
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR78,Charlie4z,MLT1L
</td>
<td style="text-align:left;">
LTR,DNA
</td>
<td style="text-align:left;">
ERV1,hAT-Charlie,ERVL-MaLR
</td>
<td style="text-align:left;">
0.652284263959391;1;0.0392156862745098
</td>
</tr>
<tr>
<td style="text-align:left;">
rs646776
</td>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TCCTC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs653178
</td>
<td style="text-align:left;">
chr12.111569574.111570394
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluJb,(CAC)n,MER3,L3,L1MC5
</td>
<td style="text-align:left;">
SINE,Simple_repeat,DNA,LINE
</td>
<td style="text-align:left;">
Alu,Simple_repeat,hAT-Charlie,CR1,L1
</td>
<td style="text-align:left;">
0.534798534798535;1;0.768707482993197
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6542647
</td>
<td style="text-align:left;">
chr2.4898614.4899128
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1PB3,MER113B
</td>
<td style="text-align:left;">
LINE,DNA
</td>
<td style="text-align:left;">
L1,hAT-Charlie
</td>
<td style="text-align:left;">
0.0321543408360129;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6546620
</td>
<td style="text-align:left;">
chr2.25936965.25937466
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb,(GGCAGT)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
MIR,Simple_repeat
</td>
<td style="text-align:left;">
0.691176470588235;0.0487804878048781
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6597292
</td>
<td style="text-align:left;">
chr6.7974642.7975583
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6598541
</td>
<td style="text-align:left;">
chr15.98727491.98728563
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(T)n,LTR12F
</td>
<td style="text-align:left;">
Simple_repeat,LTR
</td>
<td style="text-align:left;">
Simple_repeat,ERV1
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs660240
</td>
<td style="text-align:left;">
chr1.109274953.109276202
</td>
<td style="text-align:left;">
HD,HF,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TCCTC)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs667920
</td>
<td style="text-align:left;">
chr3.136350085.136351911
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSz,L1M4b,L1MA9,L1M4a2
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
Alu,L1
</td>
<td style="text-align:left;">
0.25;1;0.300291545189504
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6702619
</td>
<td style="text-align:left;">
chr1.99580395.99580980
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs672149
</td>
<td style="text-align:left;">
chr11.128759383.128760154
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.927433628318584
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6730157
</td>
<td style="text-align:left;">
chr2.135149084.135149706
</td>
<td style="text-align:left;">
ARR,HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR,AluY
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR,Alu
</td>
<td style="text-align:left;">
1;0.152542372881356
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6759518
</td>
<td style="text-align:left;">
chr2.27262237.27263756
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 257
</td>
<td style="text-align:left;">
0.479192938209332
</td>
<td style="text-align:left;">
EH38E1982668
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.84297520661157
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6801957
</td>
<td style="text-align:left;">
chr3.38725644.38726351
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.15668202764977
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6807275
</td>
<td style="text-align:left;">
chr3.14232390.14233136
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR3,L3,AluSz6
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
MIR,CR1,Alu
</td>
<td style="text-align:left;">
0.72;1;0.130281690140845
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6909752
</td>
<td style="text-align:left;">
chr6.22612314.22612576
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs6982502
</td>
<td style="text-align:left;">
chr8.125466567.125467336
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7090277
</td>
<td style="text-align:left;">
chr10.12234804.12236216
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSp,MER117
</td>
<td style="text-align:left;">
SINE,DNA
</td>
<td style="text-align:left;">
Alu,hAT-Charlie
</td>
<td style="text-align:left;">
0.678787878787879;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7115242
</td>
<td style="text-align:left;">
chr11.117037235.117037719
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7157599
</td>
<td style="text-align:left;">
chr14.100158874.100160055
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 131
</td>
<td style="text-align:left;">
0.939579684763573
</td>
<td style="text-align:left;">
EH38E1742143
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
G-rich
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7165042
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7165081
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7165733
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7166764
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7172038
</td>
<td style="text-align:left;">
chr15.73374622.73375198
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7178084
</td>
<td style="text-align:left;">
chr15.73375634.73376745
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c,(CAAGCCC)n
</td>
<td style="text-align:left;">
LINE,Simple_repeat
</td>
<td style="text-align:left;">
L2,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7181432
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7182103
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7182529
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7182716
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7183206
</td>
<td style="text-align:left;">
chr15.73375634.73376745
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c,(CAAGCCC)n
</td>
<td style="text-align:left;">
LINE,Simple_repeat
</td>
<td style="text-align:left;">
L2,Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7189462
</td>
<td style="text-align:left;">
chr16.81873989.81874357
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER102a
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7197197
</td>
<td style="text-align:left;">
chr16.72877300.72877905
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(TGT)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7234864
</td>
<td style="text-align:left;">
chr18.60067561.60068151
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.0103092783505155
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7246865
</td>
<td style="text-align:left;">
chr19.17107804.17108454
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1C,MER20
</td>
<td style="text-align:left;">
LTR,DNA
</td>
<td style="text-align:left;">
ERVL-MaLR,hAT-Charlie
</td>
<td style="text-align:left;">
0.321867321867322;0.541666666666667
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72654473
</td>
<td style="text-align:left;">
chr19.44911105.44911471
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.393939393939394
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72658867
</td>
<td style="text-align:left;">
chr19.11119936.11120563
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSq,(AAAC)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
Alu,Simple_repeat
</td>
<td style="text-align:left;">
0.135231316725979;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72664335
</td>
<td style="text-align:left;">
chr1.56521199.56521723
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSz6,(TG)n,FLAM_C,(CTCC)n
</td>
<td style="text-align:left;">
SINE,Simple_repeat
</td>
<td style="text-align:left;">
Alu,Simple_repeat
</td>
<td style="text-align:left;">
0.205787781350482;1;0.219178082191781
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72671655
</td>
<td style="text-align:left;">
chr8.105335411.105336052
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72700114
</td>
<td style="text-align:left;">
chr1.170224576.170224916
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER53,MIRb,L1MA7
</td>
<td style="text-align:left;">
DNA,SINE,LINE
</td>
<td style="text-align:left;">
hAT,MIR,L1
</td>
<td style="text-align:left;">
0.0173913043478261;1;0.288571428571429
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72926475
</td>
<td style="text-align:left;">
chr2.86367009.86367615
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.407608695652174;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs72935945
</td>
<td style="text-align:left;">
chr6.110334421.110335161
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs729743
</td>
<td style="text-align:left;">
chr17.80795708.80796334
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
THE1D,MLT2A2
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL-MaLR,ERVL
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs73045269
</td>
<td style="text-align:left;">
chr19.41318885.41319481
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs73102285
</td>
<td style="text-align:left;">
chr5.52859657.52860199
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L3
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
CR1
</td>
<td style="text-align:left;">
0.763713080168776
</td>
</tr>
<tr>
<td style="text-align:left;">
rs73123536
</td>
<td style="text-align:left;">
chr4.22630084.22630437
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
not_mrc
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
0.32258064516129
</td>
</tr>
<tr>
<td style="text-align:left;">
rs73193808
</td>
<td style="text-align:left;">
chr21.29162621.29163485
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluSq2,AluSz6
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.263333333333333;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7333991
</td>
<td style="text-align:left;">
chr13.110455785.110456535
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs733701
</td>
<td style="text-align:left;">
chr6.39203639.39204133
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER5B
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7403708
</td>
<td style="text-align:left;">
chr15.78830933.78832028
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 122
</td>
<td style="text-align:left;">
0.692356285533797
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs74181299
</td>
<td style="text-align:left;">
chr2.65055695.65057220
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 132
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2004398
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7433206
</td>
<td style="text-align:left;">
chr3.38615932.38616880
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7440763
</td>
<td style="text-align:left;">
chr4.155512047.155512545
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
Charlie4z
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
0.42483660130719
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7486169
</td>
<td style="text-align:left;">
chr12.74338024.74338393
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs75112503
</td>
<td style="text-align:left;">
chr11.110366109.110367343
</td>
<td style="text-align:left;">
HD,MI
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
AluY,L1MB3,L2b,AluSx
</td>
<td style="text-align:left;">
SINE,LINE
</td>
<td style="text-align:left;">
Alu,L1,L2
</td>
<td style="text-align:left;">
0.277227722772277;1;0.598425196850394
</td>
</tr>
<tr>
<td style="text-align:left;">
rs75190942
</td>
<td style="text-align:left;">
chr11.128894447.128895096
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7529220
</td>
<td style="text-align:left;">
chr1.21955557.21956277
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs75524776
</td>
<td style="text-align:left;">
chr12.109538192.109538640
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7568458
</td>
<td style="text-align:left;">
chr2.85560977.85561973
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 37
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2013886
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
AluJr
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.0138888888888889
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7580831
</td>
<td style="text-align:left;">
chr2.237310830.237311051
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs759098931
</td>
<td style="text-align:left;">
chr14.99644968.99645471
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 93
</td>
<td style="text-align:left;">
0.641221374045801
</td>
<td style="text-align:left;">
EH38E1741629,EH38E1741630
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7604403
</td>
<td style="text-align:left;">
chr2.111897773.111899676
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 137
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2024982
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
AluSc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
Alu
</td>
<td style="text-align:left;">
0.533783783783784
</td>
</tr>
<tr>
<td style="text-align:left;">
rs76064792
</td>
<td style="text-align:left;">
chr16.719886.721785
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 169
</td>
<td style="text-align:left;">
0.757477243172952
</td>
<td style="text-align:left;">
EH38E1795024,EH38E1795025
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
(GCAGG)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs76097649
</td>
<td style="text-align:left;">
chr11.128894447.128895096
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7612445
</td>
<td style="text-align:left;">
chr3.179454943.179455262
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7617480
</td>
<td style="text-align:left;">
chr3.49173142.49174143
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CCATCT)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7617773
</td>
<td style="text-align:left;">
chr3.48151277.48152429
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamTip1
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Tip100
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7632505
</td>
<td style="text-align:left;">
chr3.123019226.123019547
</td>
<td style="text-align:left;">
ARR,HD,HF,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIR,AluJb
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR,Alu
</td>
<td style="text-align:left;">
0.771929824561403;0.829787234042553
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7633770
</td>
<td style="text-align:left;">
chr3.46646587.46647198
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER74A
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL
</td>
<td style="text-align:left;">
0.641025641025641
</td>
</tr>
<tr>
<td style="text-align:left;">
rs76600480
</td>
<td style="text-align:left;">
chr7.128909544.128910271
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.1875
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7690530
</td>
<td style="text-align:left;">
chr4.40629425.40630609
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 66
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E2292822
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
G-rich
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7696431
</td>
<td style="text-align:left;">
chr4.168766280.168767270
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER41A,L3,MIRc
</td>
<td style="text-align:left;">
LTR,LINE,SINE
</td>
<td style="text-align:left;">
ERV1,CR1,MIR
</td>
<td style="text-align:left;">
0.842105263157895;1;0.883018867924528
</td>
</tr>
<tr>
<td style="text-align:left;">
rs77316573
</td>
<td style="text-align:left;">
chr16.2214289.2215681
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 162
</td>
<td style="text-align:left;">
0.75131926121372
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CCG)n,L1ME3,AluY
</td>
<td style="text-align:left;">
Simple_repeat,LINE,SINE
</td>
<td style="text-align:left;">
Simple_repeat,L1,Alu
</td>
<td style="text-align:left;">
1;0.205787781350482
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7873551
</td>
<td style="text-align:left;">
chr9.116482574.116483181
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7910227
</td>
<td style="text-align:left;">
chr10.20953148.20953475
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MLT1B
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL-MaLR
</td>
<td style="text-align:left;">
0.824120603015075
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7947761
</td>
<td style="text-align:left;">
chr11.100753128.100753902
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L1PBa
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L1
</td>
<td style="text-align:left;">
0.395206527281999
</td>
</tr>
<tr>
<td style="text-align:left;">
rs79661299
</td>
<td style="text-align:left;">
chr6.42087754.42088697
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MIRb,AluSc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR,Alu
</td>
<td style="text-align:left;">
0.706666666666667;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs79717953
</td>
<td style="text-align:left;">
chr6.73694726.73695115
</td>
<td style="text-align:left;">
HD,CAD,MI
</td>
<td style="text-align:left;">
EAR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER5A1
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Charlie
</td>
<td style="text-align:left;">
0.264367816091954
</td>
</tr>
<tr>
<td style="text-align:left;">
rs7977247
</td>
<td style="text-align:left;">
chr12.106865628.106866040
</td>
<td style="text-align:left;">
HD,HF
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2a
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.568870523415978
</td>
</tr>
<tr>
<td style="text-align:left;">
rs79825511
</td>
<td style="text-align:left;">
chr11.69700398.69700987
</td>
<td style="text-align:left;">
HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER113B,(CCAGG)n
</td>
<td style="text-align:left;">
DNA,Simple_repeat
</td>
<td style="text-align:left;">
hAT-Charlie,Simple_repeat
</td>
<td style="text-align:left;">
0.713235294117647;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs8003602
</td>
<td style="text-align:left;">
chr14.99682572.99684938
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 127
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1741669,EH38E1741670
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MER50,MamGypLTR3,MamGypLTR2b,(CCCGG)n,G-rich,MIR
</td>
<td style="text-align:left;">
LTR,Simple_repeat,Low_complexity,SINE
</td>
<td style="text-align:left;">
ERV1,Gypsy,Simple_repeat,Low_complexity,MIR
</td>
<td style="text-align:left;">
0.0142857142857143;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs8096658
</td>
<td style="text-align:left;">
chr18.79396485.79396813
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 544
</td>
<td style="text-align:left;">
0.0474336793540946
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs869396
</td>
<td style="text-align:left;">
chr4.168766280.168767270
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MER41A,L3,MIRc
</td>
<td style="text-align:left;">
LTR,LINE,SINE
</td>
<td style="text-align:left;">
ERV1,CR1,MIR
</td>
<td style="text-align:left;">
0.842105263157895;1;0.883018867924528
</td>
</tr>
<tr>
<td style="text-align:left;">
rs880315
</td>
<td style="text-align:left;">
chr1.10736684.10737808
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.931034482758621;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs894211
</td>
<td style="text-align:left;">
chr8.20007264.20008372
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
LTR32,MER34A,LTR48
</td>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:left;">
ERVL,ERV1
</td>
<td style="text-align:left;">
0.875294117647059;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs899162
</td>
<td style="text-align:left;">
chr5.135458929.135459424
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2c
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs899997
</td>
<td style="text-align:left;">
chr15.78726891.78727425
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
ESR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9388813
</td>
<td style="text-align:left;">
chr6.130602114.130602839
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamSINE1
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
tRNA-RTE
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs944172
</td>
<td style="text-align:left;">
chr9.107755276.107755816
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_open
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
Charlie13a,MLT2C1
</td>
<td style="text-align:left;">
DNA,LTR
</td>
<td style="text-align:left;">
hAT-Charlie,ERVL
</td>
<td style="text-align:left;">
0.71658615136876;0.307420494699647
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9469890
</td>
<td style="text-align:left;">
chr6.34796116.34796383
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
L2a
</td>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:left;">
L2
</td>
<td style="text-align:left;">
0.528089887640449
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9506925
</td>
<td style="text-align:left;">
chr13.22794559.22794879
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
OldhAT1
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT-Ac
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9556903
</td>
<td style="text-align:left;">
chr13.98207008.98208056
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
LR_close
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
(CA)n
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs965652
</td>
<td style="text-align:left;">
chr6.134047673.134047938
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
MamRep1894
</td>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:left;">
hAT
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9899183
</td>
<td style="text-align:left;">
chr17.7548665.7550717
</td>
<td style="text-align:left;">
ARR,HD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 50
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1844469
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MER21A,G-rich,MIRc,MER94
</td>
<td style="text-align:left;">
LTR,Low_complexity,SINE,DNA
</td>
<td style="text-align:left;">
ERVL,Low_complexity,MIR,hAT-Blackjack
</td>
<td style="text-align:left;">
0.748815165876777;1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9906944
</td>
<td style="text-align:left;">
chr17.49012959.49014932
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
CpGi
</td>
<td style="text-align:left;">
CpG: 48,CpG: 18
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
EH38E1868609
</td>
<td style="text-align:left;">
CpGpromo
</td>
<td style="text-align:left;">
MIRc
</td>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:left;">
MIR
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
rs9912587
</td>
<td style="text-align:left;">
chr17.43021036.43021565
</td>
<td style="text-align:left;">
HD,CAD
</td>
<td style="text-align:left;">
NR
</td>
<td style="text-align:left;">
not_CpGi
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
not_CpGpromo
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="peak-annotation-file" class="section level5">
<h5>Peak annotation file</h5>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggrepel_0.9.6                           
 [2] plyranges_1.24.0                        
 [3] ggsignif_0.6.4                          
 [4] genomation_1.36.0                       
 [5] smplot2_0.2.4                           
 [6] eulerr_7.0.2                            
 [7] biomaRt_2.60.1                          
 [8] devtools_2.4.5                          
 [9] usethis_3.0.0                           
[10] ggpubr_0.6.0                            
[11] BiocParallel_1.38.0                     
[12] scales_1.3.0                            
[13] VennDiagram_1.7.3                       
[14] futile.logger_1.4.3                     
[15] gridExtra_2.3                           
[16] ggfortify_0.4.17                        
[17] edgeR_4.2.1                             
[18] limma_3.60.4                            
[19] rtracklayer_1.64.0                      
[20] org.Hs.eg.db_3.19.1                     
[21] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[22] GenomicFeatures_1.56.0                  
[23] AnnotationDbi_1.66.0                    
[24] Biobase_2.64.0                          
[25] GenomicRanges_1.56.1                    
[26] GenomeInfoDb_1.40.1                     
[27] IRanges_2.38.1                          
[28] S4Vectors_0.42.1                        
[29] BiocGenerics_0.50.0                     
[30] ChIPseeker_1.40.0                       
[31] RColorBrewer_1.1-3                      
[32] broom_1.0.6                             
[33] kableExtra_1.4.0                        
[34] lubridate_1.9.3                         
[35] forcats_1.0.0                           
[36] stringr_1.5.1                           
[37] dplyr_1.1.4                             
[38] purrr_1.0.2                             
[39] readr_2.1.5                             
[40] tidyr_1.3.1                             
[41] tibble_3.2.1                            
[42] ggplot2_3.5.1                           
[43] tidyverse_2.0.0                         
[44] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.4                               
  [2] matrixStats_1.4.1                      
  [3] bitops_1.0-8                           
  [4] enrichplot_1.24.4                      
  [5] httr_1.4.7                             
  [6] profvis_0.3.8                          
  [7] tools_4.4.1                            
  [8] backports_1.5.0                        
  [9] utf8_1.2.4                             
 [10] R6_2.5.1                               
 [11] lazyeval_0.2.2                         
 [12] urlchecker_1.0.1                       
 [13] withr_3.0.1                            
 [14] prettyunits_1.2.0                      
 [15] cli_3.6.3                              
 [16] formatR_1.14                           
 [17] scatterpie_0.2.4                       
 [18] labeling_0.4.3                         
 [19] sass_0.4.9                             
 [20] Rsamtools_2.20.0                       
 [21] systemfonts_1.1.0                      
 [22] yulab.utils_0.1.7                      
 [23] foreign_0.8-87                         
 [24] DOSE_3.30.5                            
 [25] svglite_2.1.3                          
 [26] R.utils_2.12.3                         
 [27] sessioninfo_1.2.2                      
 [28] plotrix_3.8-4                          
 [29] BSgenome_1.72.0                        
 [30] pwr_1.3-0                              
 [31] impute_1.78.0                          
 [32] rstudioapi_0.16.0                      
 [33] RSQLite_2.3.7                          
 [34] generics_0.1.3                         
 [35] gridGraphics_0.5-1                     
 [36] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [37] BiocIO_1.14.0                          
 [38] vroom_1.6.5                            
 [39] gtools_3.9.5                           
 [40] car_3.1-2                              
 [41] GO.db_3.19.1                           
 [42] Matrix_1.7-0                           
 [43] fansi_1.0.6                            
 [44] abind_1.4-8                            
 [45] R.methodsS3_1.8.2                      
 [46] lifecycle_1.0.4                        
 [47] whisker_0.4.1                          
 [48] yaml_2.3.10                            
 [49] carData_3.0-5                          
 [50] SummarizedExperiment_1.34.0            
 [51] BiocFileCache_2.12.0                   
 [52] gplots_3.1.3.1                         
 [53] qvalue_2.36.0                          
 [54] SparseArray_1.4.8                      
 [55] blob_1.2.4                             
 [56] promises_1.3.0                         
 [57] crayon_1.5.3                           
 [58] miniUI_0.1.1.1                         
 [59] lattice_0.22-6                         
 [60] cowplot_1.1.3                          
 [61] KEGGREST_1.44.1                        
 [62] pillar_1.9.0                           
 [63] knitr_1.48                             
 [64] fgsea_1.30.0                           
 [65] rjson_0.2.23                           
 [66] boot_1.3-31                            
 [67] codetools_0.2-20                       
 [68] fastmatch_1.1-4                        
 [69] glue_1.7.0                             
 [70] getPass_0.2-4                          
 [71] ggfun_0.1.6                            
 [72] remotes_2.5.0                          
 [73] data.table_1.16.0                      
 [74] vctrs_0.6.5                            
 [75] png_0.1-8                              
 [76] treeio_1.28.0                          
 [77] gtable_0.3.5                           
 [78] cachem_1.1.0                           
 [79] xfun_0.47                              
 [80] S4Arrays_1.4.1                         
 [81] mime_0.12                              
 [82] tidygraph_1.3.1                        
 [83] statmod_1.5.0                          
 [84] ellipsis_0.3.2                         
 [85] nlme_3.1-166                           
 [86] ggtree_3.12.0                          
 [87] bit64_4.0.5                            
 [88] filelock_1.0.3                         
 [89] progress_1.2.3                         
 [90] rprojroot_2.0.4                        
 [91] bslib_0.8.0                            
 [92] rpart_4.1.23                           
 [93] KernSmooth_2.23-24                     
 [94] Hmisc_5.1-3                            
 [95] colorspace_2.1-1                       
 [96] DBI_1.2.3                              
 [97] seqPattern_1.36.0                      
 [98] nnet_7.3-19                            
 [99] tidyselect_1.2.1                       
[100] processx_3.8.4                         
[101] bit_4.0.5                              
[102] compiler_4.4.1                         
[103] curl_5.2.2                             
[104] git2r_0.33.0                           
[105] httr2_1.0.4                            
[106] htmlTable_2.4.3                        
[107] xml2_1.3.6                             
[108] DelayedArray_0.30.1                    
[109] shadowtext_0.1.4                       
[110] checkmate_2.3.2                        
[111] caTools_1.18.3                         
[112] callr_3.7.6                            
[113] rappdirs_0.3.3                         
[114] digest_0.6.37                          
[115] rmarkdown_2.28                         
[116] XVector_0.44.0                         
[117] base64enc_0.1-3                        
[118] htmltools_0.5.8.1                      
[119] pkgconfig_2.0.3                        
[120] MatrixGenerics_1.16.0                  
[121] highr_0.11                             
[122] dbplyr_2.5.0                           
[123] fastmap_1.2.0                          
[124] htmlwidgets_1.6.4                      
[125] rlang_1.1.4                            
[126] UCSC.utils_1.0.0                       
[127] shiny_1.9.1                            
[128] farver_2.1.2                           
[129] jquerylib_0.1.4                        
[130] zoo_1.8-12                             
[131] jsonlite_1.8.8                         
[132] GOSemSim_2.30.2                        
[133] R.oo_1.26.0                            
[134] RCurl_1.98-1.16                        
[135] magrittr_2.0.3                         
[136] Formula_1.2-5                          
[137] GenomeInfoDbData_1.2.12                
[138] ggplotify_0.1.2                        
[139] patchwork_1.3.0                        
[140] munsell_0.5.1                          
[141] Rcpp_1.0.13                            
[142] ape_5.8                                
[143] viridis_0.6.5                          
[144] stringi_1.8.4                          
[145] ggraph_2.2.1                           
[146] zlibbioc_1.50.0                        
[147] MASS_7.3-61                            
[148] plyr_1.8.9                             
[149] pkgbuild_1.4.4                         
[150] parallel_4.4.1                         
[151] Biostrings_2.72.1                      
[152] graphlayouts_1.1.1                     
[153] splines_4.4.1                          
[154] hms_1.1.3                              
[155] locfit_1.5-9.10                        
[156] ps_1.8.0                               
[157] igraph_2.0.3                           
[158] reshape2_1.4.4                         
[159] pkgload_1.4.0                          
[160] futile.options_1.0.1                   
[161] XML_3.99-0.17                          
[162] evaluate_1.0.0                         
[163] lambda.r_1.2.4                         
[164] tzdb_0.4.0                             
[165] tweenr_2.0.3                           
[166] httpuv_1.6.15                          
[167] polyclip_1.10-7                        
[168] gridBase_0.4-7                         
[169] ggforce_0.4.2                          
[170] xtable_1.8-4                           
[171] restfulr_0.0.15                        
[172] tidytree_0.4.6                         
[173] rstatix_0.7.2                          
[174] later_1.3.2                            
[175] viridisLite_0.4.2                      
[176] aplot_0.2.3                            
[177] memoise_2.0.1                          
[178] GenomicAlignments_1.40.0               
[179] cluster_2.1.6                          
[180] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
