<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="ERM" />

<meta name="date" content="2024-08-23" />

<title>TEs and my data</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ATAC_learning</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/reneeisnowhere/ATAC_learning">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">TEs and my data</h1>
<h4 class="author">ERM</h4>
<h4 class="date">2024-08-23</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-08-23
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ATAC_learning/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20231016code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20231016)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20231016code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20231016)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree0fa2caeadba1e9a808edf1b3fab9166df66ebe88targetblank0fa2caea">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/0fa2caeadba1e9a808edf1b3fab9166df66ebe88" target="_blank">0fa2cae</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomreneeisnowhereATAClearningtree0fa2caeadba1e9a808edf1b3fab9166df66ebe88targetblank0fa2caea"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/reneeisnowhere/ATAC_learning/tree/0fa2caeadba1e9a808edf1b3fab9166df66ebe88" target="_blank">0fa2cae</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .RData
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/ACresp_SNP_table.csv
    Ignored:    data/ARR_SNP_table.csv
    Ignored:    data/All_merged_peaks.tsv
    Ignored:    data/CAD_gwas_dataframe.RDS
    Ignored:    data/CTX_SNP_table.csv
    Ignored:    data/Collapsed_expressed_NG_peak_table.csv
    Ignored:    data/DEG_toplist_sep_n45.RDS
    Ignored:    data/FRiP_first_run.txt
    Ignored:    data/Frip_1_reads.csv
    Ignored:    data/Frip_2_reads.csv
    Ignored:    data/Frip_3_reads.csv
    Ignored:    data/Frip_4_reads.csv
    Ignored:    data/Frip_5_reads.csv
    Ignored:    data/Frip_6_reads.csv
    Ignored:    data/GO_KEGG_analysis/
    Ignored:    data/HF_SNP_table.csv
    Ignored:    data/Ind1_75DA24h_dedup_peaks.csv
    Ignored:    data/Ind1_TSS_peaks.RDS
    Ignored:    data/Ind1_firstfragment_files.txt
    Ignored:    data/Ind1_fragment_files.txt
    Ignored:    data/Ind1_peaks_list.RDS
    Ignored:    data/Ind1_summary.txt
    Ignored:    data/Ind2_TSS_peaks.RDS
    Ignored:    data/Ind2_fragment_files.txt
    Ignored:    data/Ind2_peaks_list.RDS
    Ignored:    data/Ind2_summary.txt
    Ignored:    data/Ind3_TSS_peaks.RDS
    Ignored:    data/Ind3_fragment_files.txt
    Ignored:    data/Ind3_peaks_list.RDS
    Ignored:    data/Ind3_summary.txt
    Ignored:    data/Ind4_79B24h_dedup_peaks.csv
    Ignored:    data/Ind4_TSS_peaks.RDS
    Ignored:    data/Ind4_V24h_fraglength.txt
    Ignored:    data/Ind4_fragment_files.txt
    Ignored:    data/Ind4_fragment_filesN.txt
    Ignored:    data/Ind4_peaks_list.RDS
    Ignored:    data/Ind4_summary.txt
    Ignored:    data/Ind5_TSS_peaks.RDS
    Ignored:    data/Ind5_fragment_files.txt
    Ignored:    data/Ind5_fragment_filesN.txt
    Ignored:    data/Ind5_peaks_list.RDS
    Ignored:    data/Ind5_summary.txt
    Ignored:    data/Ind6_TSS_peaks.RDS
    Ignored:    data/Ind6_fragment_files.txt
    Ignored:    data/Ind6_peaks_list.RDS
    Ignored:    data/Ind6_summary.txt
    Ignored:    data/Knowles_4.RDS
    Ignored:    data/Knowles_5.RDS
    Ignored:    data/Knowles_6.RDS
    Ignored:    data/LiSiLTDNRe_TE_df.RDS
    Ignored:    data/MI_gwas.RDS
    Ignored:    data/SNP_gene_cat_list.tsv
    Ignored:    data/SNP_supp_schneider.RDS
    Ignored:    data/TE_info/
    Ignored:    data/all_TSSE_scores.RDS
    Ignored:    data/aln_run1_results.txt
    Ignored:    data/anno_ind1_DA24h.RDS
    Ignored:    data/anno_ind4_V24h.RDS
    Ignored:    data/background_n45_he_peaks.RDS
    Ignored:    data/cardiac_muscle_FRIP.csv
    Ignored:    data/cardiomyocyte_FRIP.csv
    Ignored:    data/col_ng_peak.csv
    Ignored:    data/cormotif_full_4_run.RDS
    Ignored:    data/cormotif_full_4_run_he.RDS
    Ignored:    data/cormotif_full_6_run.RDS
    Ignored:    data/cormotif_full_6_run_he.RDS
    Ignored:    data/cormotif_probability_45_list.csv
    Ignored:    data/cormotif_probability_45_list_he.csv
    Ignored:    data/cormotif_probability_all_6_list.csv
    Ignored:    data/cormotif_probability_all_6_list_he.csv
    Ignored:    data/embryo_heart_FRIP.csv
    Ignored:    data/enhancer_list_ENCFF126UHK.bed
    Ignored:    data/enhancerdata/
    Ignored:    data/filt_Peaks_efit2.RDS
    Ignored:    data/filt_Peaks_efit2_bl.RDS
    Ignored:    data/filt_Peaks_efit2_n45.RDS
    Ignored:    data/first_Peaksummarycounts.csv
    Ignored:    data/first_run_frag_counts.txt
    Ignored:    data/full_bedfiles/
    Ignored:    data/gene_ref.csv
    Ignored:    data/gwas_1_dataframe.RDS
    Ignored:    data/gwas_2_dataframe.RDS
    Ignored:    data/gwas_3_dataframe.RDS
    Ignored:    data/gwas_4_dataframe.RDS
    Ignored:    data/gwas_5_dataframe.RDS
    Ignored:    data/high_conf_peak_counts.csv
    Ignored:    data/high_conf_peak_counts.txt
    Ignored:    data/high_conf_peaks_bl_counts.txt
    Ignored:    data/high_conf_peaks_counts.txt
    Ignored:    data/hits_files/
    Ignored:    data/hyper_files/
    Ignored:    data/hypo_files/
    Ignored:    data/ind1_DA24hpeaks.RDS
    Ignored:    data/ind1_TSSE.RDS
    Ignored:    data/ind2_TSSE.RDS
    Ignored:    data/ind3_TSSE.RDS
    Ignored:    data/ind4_TSSE.RDS
    Ignored:    data/ind4_V24hpeaks.RDS
    Ignored:    data/ind5_TSSE.RDS
    Ignored:    data/ind6_TSSE.RDS
    Ignored:    data/initial_complete_stats_run1.txt
    Ignored:    data/left_ventricle_FRIP.csv
    Ignored:    data/median_24_lfc.RDS
    Ignored:    data/median_3_lfc.RDS
    Ignored:    data/mergedPeads.gff
    Ignored:    data/mergedPeaks.gff
    Ignored:    data/motif_list_full
    Ignored:    data/motif_list_n45
    Ignored:    data/motif_list_n45.RDS
    Ignored:    data/multiqc_fastqc_run1.txt
    Ignored:    data/multiqc_fastqc_run2.txt
    Ignored:    data/multiqc_genestat_run1.txt
    Ignored:    data/multiqc_genestat_run2.txt
    Ignored:    data/my_hc_filt_counts.RDS
    Ignored:    data/my_hc_filt_counts_n45.RDS
    Ignored:    data/n45_bedfiles/
    Ignored:    data/n45_files
    Ignored:    data/other_papers/
    Ignored:    data/peakAnnoList_1.RDS
    Ignored:    data/peakAnnoList_2.RDS
    Ignored:    data/peakAnnoList_24_full.RDS
    Ignored:    data/peakAnnoList_24_n45.RDS
    Ignored:    data/peakAnnoList_3.RDS
    Ignored:    data/peakAnnoList_3_full.RDS
    Ignored:    data/peakAnnoList_3_n45.RDS
    Ignored:    data/peakAnnoList_4.RDS
    Ignored:    data/peakAnnoList_5.RDS
    Ignored:    data/peakAnnoList_6.RDS
    Ignored:    data/peakAnnoList_full_motif.RDS
    Ignored:    data/peakAnnoList_n45_motif.RDS
    Ignored:    data/siglist_full.RDS
    Ignored:    data/siglist_n45.RDS
    Ignored:    data/test.list.RDS
    Ignored:    data/testnames.txt
    Ignored:    data/toplist_6.RDS
    Ignored:    data/toplist_full.RDS
    Ignored:    data/toplist_full_DAR_6.RDS
    Ignored:    data/toplist_n45.RDS
    Ignored:    data/trimmed_seq_length.csv
    Ignored:    data/unclassified_full_set_peaks.RDS
    Ignored:    data/unclassified_n45_set_peaks.RDS
    Ignored:    data/xstreme/
    Ignored:    trimmed_Ind1_75DA24h_S7.nodup.splited.bam/

Untracked files:
    Untracked:  DOX_DAR_assess.Rmd
    Untracked:  EAR_2_plot.pdf
    Untracked:  ESR_1_plot.pdf
    Untracked:  Firstcorr plotATAC.pdf
    Untracked:  IND1_2_3_6_corrplot.pdf
    Untracked:  LR_3_plot.pdf
    Untracked:  NR_1_plot.pdf
    Untracked:  analysis/LFC_corr.Rmd
    Untracked:  analysis/ReHeat_analysis.Rmd
    Untracked:  analysis/TE_analysis_old.Rmd
    Untracked:  analysis/my_hc_filt_counts.csv
    Untracked:  analysis/nucleosome_explore.Rmd
    Untracked:  code/IGV_snapshot_code.R
    Untracked:  code/LongDARlist.R
    Untracked:  code/TSSE.R
    Untracked:  code/just_for_Fun.R
    Untracked:  code/toplist_assembly.R
    Untracked:  lcpm_filtered_corplot.pdf
    Untracked:  log2cpmfragcount.pdf
    Untracked:  output/cormotif_probability_45_list.csv
    Untracked:  output/cormotif_probability_all_6_list.csv
    Untracked:  splited/
    Untracked:  trimmed_Ind1_75DA24h_S7.nodup.fragment.size.distribution.pdf
    Untracked:  trimmed_Ind1_75DA3h_S1.nodup.fragment.size.distribution.pdf

Unstaged changes:
    Modified:   analysis/GO_KEGG_analysis.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/TE_analysis.Rmd</code>) and HTML
(<code>docs/TE_analysis.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/0fa2caeadba1e9a808edf1b3fab9166df66ebe88/analysis/TE_analysis.Rmd" target="_blank">0fa2cae</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-23
</td>
<td>
update CpG islands
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/TE_analysis.html" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/8174c677e06863b47d9264a146a4f597fe32192d/analysis/TE_analysis.Rmd" target="_blank">8174c67</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
<td>
updates to barplots and 50% cutoff added
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/TE_analysis.html" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/ec629e184d193d035c76231eefda455868047fb0/analysis/TE_analysis.Rmd" target="_blank">ec629e1</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-20
</td>
<td>
before adding in 2kb and KEGG filters to data, removing all none-useful
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/128c4f731ab5119b81b2f8daa22b95842fd8e7a0/analysis/TE_analysis.Rmd" target="_blank">128c4f7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-20
</td>
<td>
before adding in 2kb and KEGG filters to data, removing all none-useful
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/1d8660b296e524e2a3b0b45385bf082a51d9a199/analysis/TE_analysis.Rmd" target="_blank">1d8660b</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-19
</td>
<td>
updates to code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/TE_analysis.html" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9658fcba6df8add6c69d25c602682e78b7bd6b6f/analysis/TE_analysis.Rmd" target="_blank">9658fcb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
<td>
updates to TE lists and filtering
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/2b02bbb4ce2276d1ecb0e20060d3a4cf82d51004/analysis/TE_analysis.Rmd" target="_blank">2b02bbb</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-09
</td>
<td>
capturing snapshot before change
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/7b5f189025a49f33189d3cf4f049ac1a485e34da/analysis/TE_analysis.Rmd" target="_blank">7b5f189</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-30
</td>
<td>
updates to TE
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/4c818dc0ab49ca1b36d141ee1577dc836c8e0820/docs/TE_analysis.html" target="_blank">4c818dc</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/db2e632d6754dc04231f75472e6b3da986afd7ac/analysis/TE_analysis.Rmd" target="_blank">db2e632</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
<td>
renaming
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/b0aadc4c2205273739162480034a992e3be9fe97/docs/TE_analysis.html" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/71bd20f7363d5b935cbfcb1bfd74efe02672f773/analysis/TE_analysis.Rmd" target="_blank">71bd20f</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
<td>
adding in cpg data
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3e8a9bfb7ac72c0066085c44665c702f52611fba/analysis/TE_analysis.Rmd" target="_blank">3e8a9bf</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
<td>
cpg updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/d3797736ec4026f7d6eb1898e7339d5259a8e2d3/docs/TE_analysis.html" target="_blank">d379773</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/a4655323a410bca32fc56137f55b69e64dec2af4/analysis/TE_analysis.Rmd" target="_blank">a465532</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
<td>
updates agian
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/9678f55cd1d509320ab30612921307a0426e90ea/docs/TE_analysis.html" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/71d70f7eb07e38bf3855614c92d74f65c88c7b59/analysis/TE_analysis.Rmd" target="_blank">71d70f7</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
<td>
updates
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/reneeisnowhere/ATAC_learning/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/TE_analysis.html" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b60b42cad1a362232dc4b0ce06be8c1cad8de465/analysis/TE_analysis.Rmd" target="_blank">b60b42c</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
</style>
<pre class="r"><code>library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library(&quot;TxDb.Hsapiens.UCSC.hg38.knownGene&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)</code></pre>
<div id="loading-data" class="section level5">
<h5>loading data</h5>
<p>This is where I pull in the repeatmasker file taken from UCSC
genomebrowser, the peaks assigned with the closest expressed genes as
‘neargenes’ by TSS, the same peaks list, only condensing to unique peaks
(some are assigned to two neargene), I call the collapsed_peaks and the
peaks assigned to each MRC (EAR, etc…).<br />
With the TSS data and the collapsed data, I made granges objects. I also
separate out the LINEs, SINEs, LTRs, DNAs, and Retroposons from the
repeatmasker to make granges objects from those sets.</p>
<pre class="r"><code>repeatmasker &lt;- read.delim(&quot;data/other_papers/repeatmasker.tsv&quot;)</code></pre>
<pre class="r"><code>TSS_NG_data &lt;- read_delim(&quot;data/n45_bedfiles/TSS_NG_data.tsv&quot;, 
    delim = &quot;\t&quot;, escape_double = FALSE, 
    trim_ws = TRUE)
Collapsed_peaks &lt;- read_delim(&quot;data/n45_bedfiles/TSS_NG_data_collapsed_peaks.tsv&quot;,
                              delim = &quot;\t&quot;, 
                              escape_double = FALSE, 
                              trim_ws = TRUE)
TSS_data_gr &lt;- TSS_NG_data %&gt;% 
  dplyr::filter(seqnames != &quot;chrX&quot;) %&gt;% 
  dplyr::filter(seqnames != &quot;chrY&quot;) %&gt;%
  GRanges()

Col_TSS_data_gr &lt;- Collapsed_peaks %&gt;% 
  dplyr::filter(seqnames != &quot;chrX&quot;) %&gt;% 
  dplyr::filter(seqnames != &quot;chrY&quot;) %&gt;%
  GRanges()

reClass_list &lt;- repeatmasker %&gt;% 
  distinct(repClass)

Line_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

Sine_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

LTR_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;LTR&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

DNA_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)

retroposon_repeats &lt;- repeatmasker %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;) %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


all_TEs_gr &lt;- repeatmasker %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;genoName&quot;, start.field = &quot;genoStart&quot;, end.field = &quot;genoEnd&quot;,starts.in.df.are.0based=TRUE)


peakAnnoList_n45_motif &lt;- readRDS(&quot;data/peakAnnoList_n45_motif.RDS&quot;)

EAR_df &lt;- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)

ESR_df &lt;- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_df_gr &lt;- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df &lt;- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df &lt;- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_df_gr &lt;-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)</code></pre>
<p>this code contains the fill functions for each of the plots that
needed similar colors.</p>
<pre class="r"><code># scale fill repeat, 2nd set ----------------------------------------------
rep_other_names&lt;- repeatmasker %&gt;% 
  distinct(repClass) %&gt;% 
  rbind(&quot;Other&quot;)

scale_fill_repeat &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;,
                         &quot;darkgrey&quot;), unique(rep_other_names$repClass)), 
    ...
  )
}


# scale fill LTRs ---------------------------------------------------------

LTR_df &lt;- LTR_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_LTRs &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(LTR_df$repFamily)), 
    ...
  )
}



scale_fill_DNA_family &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;, &quot;#FFFFB3&quot;, &quot;#BEBADA&quot; ,&quot;#FB8072&quot;, &quot;#80B1D3&quot;, &quot;#FDB462&quot;, &quot;#B3DE69&quot;, &quot;#FCCDE5&quot;, &quot;purple4&quot;), unique(DNA_family$repFamily)), 
    ...
  )
}


# # scale fill repeat first -------------------------------------------------
# 
# scale_fill_repeat &lt;-  function(...){
#   ggplot2:::manual_scale(
#     &#39;fill&#39;, 
#     values = setNames(c( &quot;#8DD3C7&quot;,
#                          &quot;#FFFFB3&quot;,
#                          &quot;#BEBADA&quot; ,
#                          &quot;#FB8072&quot;,
#                          &quot;#80B1D3&quot;,
#                          &quot;#FDB462&quot;,
#                          &quot;#B3DE69&quot;,
#                          &quot;#FCCDE5&quot;,
#                          &quot;#D9D9D9&quot;,
#                          &quot;#BC80BD&quot;,
#                          &quot;#CCEBC5&quot;,
#                          &quot;pink4&quot;,
#                          &quot;cornflowerblue&quot;,
#                          &quot;chocolate&quot;,
#                          &quot;brown&quot;,
#                          &quot;green&quot;,
#                          &quot;yellow4&quot;,
#                          &quot;purple&quot;,
#                          &quot;darkorchid4&quot;,
#                          &quot;coral4&quot;,
#                          &quot;darkolivegreen4&quot;,
#                          &quot;darkorange&quot;), unique(repeatmasker$repClass)), 
#     ...
#   )
# }

# scale lines -------------------------------------------------------------
Line_df &lt;- Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_lines &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(Line_df$repFamily)), 
    ...
  )
}


# scale fill L2 family ----------------------------------------------------
L2_line_df&lt;- Line_df %&gt;% 
  dplyr::filter(repFamily==&quot;L2&quot;)


scale_fill_L2 &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(L2_line_df$repName)), 
    ...
  )
}

# scale fill sines --------------------------------------------------------
Sine_df &lt;- Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_sines &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(Sine_df$repFamily)), 
    ...
  )
}


# scale fill DNAs ---------------------------------------------------------
DNA_df &lt;- DNA_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repFamily=factor(repFamily))


scale_fill_DNAs &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;,
                         &quot;blue&quot;,
                         &quot;grey&quot;,
                         &quot;lightgrey&quot;), unique(DNA_df$repFamily)), 
    ...
  )
}


# scale fill retroposons --------------------------------------------------

retroposon_df &lt;- retroposon_repeats %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repName=factor(repName))

scale_fill_retroposons &lt;-  function(...){
  ggplot2:::manual_scale(
    &#39;fill&#39;, 
    values = setNames(c( &quot;#8DD3C7&quot;,
                         &quot;#FFFFB3&quot;,
                         &quot;#BEBADA&quot; ,
                         &quot;#FB8072&quot;,
                         &quot;#80B1D3&quot;,
                         &quot;#FDB462&quot;,
                         &quot;#B3DE69&quot;,
                         &quot;#FCCDE5&quot;,
                         &quot;#D9D9D9&quot;,
                         &quot;#BC80BD&quot;,
                         &quot;#CCEBC5&quot;,
                         &quot;pink4&quot;,
                         &quot;cornflowerblue&quot;,
                         &quot;chocolate&quot;,
                         &quot;brown&quot;,
                         &quot;green&quot;,
                         &quot;yellow4&quot;,
                         &quot;purple&quot;,
                         &quot;darkorchid4&quot;,
                         &quot;coral4&quot;,
                         &quot;darkolivegreen4&quot;,
                         &quot;darkorange&quot;), unique(retroposon_df$repName)), 
    ...
  )
}</code></pre>
</div>
<div id="te-distrubution" class="section level2">
<h2>TE distrubution:</h2>
<p>The code below uses the TE data sets to create dataframes that
contain all peaks that overlap a TE. Additionally, I wanted to know size
distribution of the overlaps(width) between TEs and my peaks, the size
of the TEs and their size distribution, and the size distribution of my
peaks. This was to determine a cutoff across TEs to minimize size bias.
We used this data to apply a inclusion stringency cutoff for TEs of
&gt;50% of the full TE needed to be covered by a peak to call the peak
“TE-containing”. In the plots, the median of each density distribution
is shown by the dotted line.</p>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all LINEs across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/data%20about%20TEs-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-1">
Past versions of "data about TEs-1.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/data about TEs-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/data about TEs-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all SINEs across human genome&quot;)</code></pre>
<p><img src="figure/TE_analysis.Rmd/data%20about%20TEs-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-2">
Past versions of "data about TEs-2.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/data about TEs-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/data about TEs-2.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/data about TEs-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all LTRs across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/data%20about%20TEs-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-3">
Past versions of "data about TEs-3.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/data about TEs-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/data about TEs-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/data about TEs-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all DNA-TEs across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/data%20about%20TEs-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-4">
Past versions of "data about TEs-4.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/data about TEs-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/data about TEs-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/data about TEs-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass,alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of lengths of all Retroposons across human genome&quot;,subtitle = &quot; limited x axis&quot;)+
   coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/data%20about%20TEs-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-5">
Past versions of "data about TEs-5.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/data about TEs-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/data about TEs-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/data about TEs-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_TEs_gr %&gt;% 
  as.data.frame() %&gt;% 
  dplyr::select(repClass, width) %&gt;% 
  # dplyr::filter(repClass==&quot;LTR&quot;) %&gt;%
  group_by(repClass) %&gt;% 
  summarise(med.width=median(width),mean.width=mean(width)) %&gt;% 
  kable(., caption=&quot;Table 1: Summary of mean and median length of each TE Class&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 1: Summary of mean and median length of each TE Class
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
med.width
</th>
<th style="text-align:right;">
mean.width
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
155.0
</td>
<td style="text-align:right;">
210.61264
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
125.0
</td>
<td style="text-align:right;">
137.59542
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
219.0
</td>
<td style="text-align:right;">
421.99752
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
329.0
</td>
<td style="text-align:right;">
371.36349
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
170.0
</td>
<td style="text-align:right;">
208.63493
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
44.0
</td>
<td style="text-align:right;">
61.55669
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
171.0
</td>
<td style="text-align:right;">
210.89231
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
144.0
</td>
<td style="text-align:right;">
150.47242
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
165.0
</td>
<td style="text-align:right;">
163.94730
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
646.0
</td>
<td style="text-align:right;">
776.24171
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
258.0
</td>
<td style="text-align:right;">
221.43075
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
83.5
</td>
<td style="text-align:right;">
74.34211
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
496.0
</td>
<td style="text-align:right;">
8741.48243
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
36.0
</td>
<td style="text-align:right;">
55.83912
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
119.0
</td>
<td style="text-align:right;">
136.70621
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
86.0
</td>
<td style="text-align:right;">
140.04608
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
99.0
</td>
<td style="text-align:right;">
96.07143
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
77.5
</td>
<td style="text-align:right;">
79.82394
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
136.0
</td>
<td style="text-align:right;">
169.81834
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
69.0
</td>
<td style="text-align:right;">
60.12893
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Col_TSS_data_gr %&gt;% 
  as.data.frame %&gt;% 
   ggplot(., aes(x=width))+
    geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
  geom_vline(data=Col_TSS_data_gr$width., aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of peak length (bps) across all experimentally derived peaks&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/data%20about%20TEs-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-data-about-TEs-6">
Past versions of "data about TEs-6.png"
</button>
</p>
<div id="fig-no-spaces-data-about-TEs-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/data about TEs-6.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/data about TEs-6.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/data about TEs-6.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>fullDF_overlap &lt;- join_overlap_intersect(TSS_data_gr,all_TEs_gr)
fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot;Table 2: Count of peaks by TE class; overlap 1 bp or greater&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 2: Count of peaks by TE class; overlap 1 bp or greater
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
16528
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
125
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
39199
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
26934
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
253
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5627
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
48
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
334
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
51990
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
227
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
29958
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
280
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
53
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
191
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
403
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>subsetByOverlaps(TSS_data_gr,all_TEs_gr) %&gt;% as.data.frame %&gt;% 
   ggplot(., aes(x=width))+
    geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle(&quot;Distribution of overlaps between all TEs and all my peaks&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/overlapping%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>### This is how I subset only those peaks who cover &gt;50% of TEs
hits &lt;- findOverlaps(TSS_data_gr,all_TEs_gr)
overlaps &lt;- pintersect(TSS_data_gr[queryHits(hits)], all_TEs_gr[subjectHits(hits)])
percentOverlap &lt;- width(overlaps) / width(all_TEs_gr[subjectHits(hits)])
hits &lt;- hits[percentOverlap &gt; 0.5]
### THis actually did not work well---I ended up losing data.  What I wanted was a data frame that had metadata from both the TE overlap and the peak metadata, so I could easily sort and manipulation.  
testingol &lt;- TSS_data_gr[queryHits(hits)]
testingol %&gt;% as.data.frame() %&gt;% 
  left_join(., (fullDF_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;peakid&quot;=&quot;peakid&quot;, &quot;NG_start&quot;=&quot;NG_start&quot;, &quot;end_position&quot;=&quot;end_position&quot;, &quot;entrezgene_id&quot;=&quot;entrezgene_id&quot;, &quot;ensembl_gene_id&quot;=&quot;ensembl_gene_id&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;, &quot;width&quot;=&quot;width&quot;, &quot;strand&quot;=&quot;strand&quot;, &quot;hgnc_symbol&quot; = &quot;hgnc_symbol&quot;)) %&gt;% 
  group_by(repClass) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot; Table 3: Count of peaks by TE class; overlap&gt; 50%&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 3: Count of peaks by TE class; overlap&gt; 50%
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
268
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
1079
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
1789
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
195
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
103
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:right;">
117556
</td>
</tr>
</tbody>
</table>
<p>The first dataframe to subset peaks that overlap &gt;50% of a TE, was
using the full neargene dataframe, where a peak is listed more than once
because it was assigned more than one neargene ( one-to-many
relationships). I changed the code to use the ‘collapsed’ data frame.
This means the data frame was simplified to only include peaks one time,
but those peaks that were assigned to more than one neargene had the
assigned neargenes condensed and separated by a comma into the same
column to create a one-to-one relationship dataframe. (yes, wordy I
know)</p>
<p>This yeilded a new observation about how I was subsetting the peaks
that covered &gt;50% of the specific TE. I was using the
<code>queryHits</code> function of GRanges and later joining the peaks
back to a dataframe, forgetting there were peaks that hit more than one
TE. I lost the metadata between the data sets in this process and missed
the other possible sets of peak-TE overlap due to how I joined the data
frame. The last table (Table 3) gave the initial hint I did something
wrong (NA=117,556). I checked this by subsetting using subjectHits data
and started realizing where the error was. Below is how I began to fix
the lost data issue.</p>
<pre class="r"><code>######################################################
all_TEs_gr$TE_width &lt;- width(all_TEs_gr)
Col_TSS_data_gr$peak_width &lt;- width(Col_TSS_data_gr)
Col_fullDF_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,all_TEs_gr)
Col_fullDF_overlap %&gt;% 
  as.data.frame() %&gt;% 
  group_by(repClass) %&gt;%  
  tally %&gt;% 
  kable(., caption=&quot; Table 4: Count of peaks by TE class; overlap at least 1 bp; using one:one df &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 4: Count of peaks by TE class; overlap at least 1 bp; using
one:one df
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
16088
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
125
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
38468
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
26306
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
244
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5527
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
45
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
322
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
50933
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
193
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
29334
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
271
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
138
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
297
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Col_fullDF_overlap %&gt;% 
   as.data.frame %&gt;% 
  mutate(per_ol= width/TE_width) %&gt;% 
  dplyr::filter(per_ol&gt;0.5) %&gt;% 
  group_by(repClass) %&gt;% 
  tally() %&gt;% 
  kable(., caption=&quot; Table 5:Count of peaks by TE class; overlap of &gt;50% of TE; newway &quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 5:Count of peaks by TE class; overlap of &gt;50% of TE; newway
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
11274
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
108
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
24586
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
17532
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
189
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5136
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
92
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
31006
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
84
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
27424
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
228
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
119
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
287
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>Filter_TE_list &lt;- Col_fullDF_overlap %&gt;% 
   as.data.frame %&gt;% 
  mutate(per_ol= width/TE_width) 
  # dplyr::filter(per_ol&gt;0.5)
### Old way
hits_col &lt;- findOverlaps(Col_TSS_data_gr,all_TEs_gr)

overlaps_col &lt;- pintersect(Col_TSS_data_gr[queryHits(hits_col)], all_TEs_gr[subjectHits(hits_col)])
percentOverlap_col &lt;- width(overlaps_col) / width(all_TEs_gr[subjectHits(hits_col)])
hits_col2 &lt;- hits_col[percentOverlap_col &gt; 0.5]
## bad dataframe
testingol_col2 &lt;- Col_TSS_data_gr[queryHits(hits_col2)]
## better alternative way
just_TEdata &lt;- all_TEs_gr[subjectHits(hits_col2)]


just_TEdata %&gt;% 
  as.data.frame %&gt;% 
  distinct(., .keep_all = TRUE) %&gt;% 
  group_by(repClass) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot; Table 6: Count of peaks by TE class Just TE list; overlap&gt; 50%; using one:one df&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 6: Count of peaks by TE class Just TE list; overlap&gt; 50%; using
one:one df
</caption>
<thead>
<tr>
<th style="text-align:left;">
repClass
</th>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DNA
</td>
<td style="text-align:right;">
11274
</td>
</tr>
<tr>
<td style="text-align:left;">
DNA?
</td>
<td style="text-align:right;">
108
</td>
</tr>
<tr>
<td style="text-align:left;">
LINE
</td>
<td style="text-align:right;">
24586
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR
</td>
<td style="text-align:right;">
17532
</td>
</tr>
<tr>
<td style="text-align:left;">
LTR?
</td>
<td style="text-align:right;">
189
</td>
</tr>
<tr>
<td style="text-align:left;">
Low_complexity
</td>
<td style="text-align:right;">
5136
</td>
</tr>
<tr>
<td style="text-align:left;">
RC
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:left;">
RC?
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
RNA
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
Retroposon
</td>
<td style="text-align:right;">
92
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE
</td>
<td style="text-align:right;">
31006
</td>
</tr>
<tr>
<td style="text-align:left;">
SINE?
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
Satellite
</td>
<td style="text-align:right;">
84
</td>
</tr>
<tr>
<td style="text-align:left;">
Simple_repeat
</td>
<td style="text-align:right;">
27424
</td>
</tr>
<tr>
<td style="text-align:left;">
Unknown
</td>
<td style="text-align:right;">
228
</td>
</tr>
<tr>
<td style="text-align:left;">
rRNA
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
scRNA
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
snRNA
</td>
<td style="text-align:right;">
119
</td>
</tr>
<tr>
<td style="text-align:left;">
srpRNA
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
tRNA
</td>
<td style="text-align:right;">
287
</td>
</tr>
</tbody>
</table>
<pre class="r"><code># 
# 
# testingol_col2 %&gt;% 
#   as.data.frame() %&gt;% 
#   group_by(peakid) %&gt;% 
#   tally %&gt;% 
#   summary()
Unique_peak_overlap &lt;- Col_fullDF_overlap %&gt;%
  as.data.frame() %&gt;%
  distinct(peakid)

peak_overlap_50unique &lt;- testingol_col2 %&gt;%
  as.data.frame() %&gt;%
  distinct(peakid) </code></pre>
<p>Note that the counts of peaks total to more than the total number of
peaks that overlap a TE because many peaks overlap multiple elements
(generally the very small TEs).</p>
<p>A summary of numbers of peaks is below:</p>
<ul>
<li>Total number of peaks = 151675</li>
<li>Total number of peaks overlapping at least 1 TE = 98927<br />
</li>
<li>Total number of peaks overlapping by &gt;50% TE length = 77759</li>
</ul>
<p>Tables 4 and 5 above used the “new” way of sub-setting (keeping all
metadata organized vs the sub-setting code I found on the internet).
Just to verify the numbers were the same, I ran the data using the old
method and compared the numbers. They are identical (Table 6) to each
other. Success!</p>
<div id="distribution-of-te-overlaps-by-peaks-according-to-class"
class="section level3">
<h3>Distribution of TE overlaps by peaks according to class</h3>
<p>Below are plots of the distribution of overlapping widths between
peaks and TEs. The first plot is all TE Class, and the 2nd plot limits
the classes to LINEs, SINEs, LTRs, DNAs, and Retroposons.</p>
<pre class="r"><code>Col_fullDF_overlap %&gt;%  
  as.data.frame %&gt;% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color=&quot;darkblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping widths in my data&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,750))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/another%20set%20of%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-1">
Past versions of "another set of plots-1.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/another set of plots-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Col_fullDF_overlap %&gt;%  
  as.data.frame %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass ==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass ==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color=&quot;darkblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths&quot;,subtitle = &quot;Just LINE, SINE,LTR, DNA, Retroposons; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,1550))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/another%20set%20of%20plots-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-2">
Past versions of "another set of plots-2.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/another set of plots-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown &lt;- Col_TSS_data_gr %&gt;% as.data.frame %&gt;% 
  dplyr::select(peakid) %&gt;% 
  left_join(.,(Col_fullDF_overlap %&gt;% as.data.frame)) %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;% 
   mutate(mrc=if_else(peakid %in% EAR_df$id, &quot;EAR&quot;,
                     if_else(peakid %in% ESR_df$id,&quot;ESR&quot;,
                             if_else(peakid %in% LR_df$id,&quot;LR&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;))))) %&gt;%  mutate(per_ol= width/TE_width) 
  
 # Peak_TE_overlapbreakdown 
 ## takes too long to render as a kable table
  #  kable(., caption=&quot; Table 6: complete data frame annotated with metadata from TE and peak joined data frames.  per_ol column is the percent overlap for the indicated peaks&quot;) %&gt;% 
  # kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  # kable_styling(full_width = FALSE, font_size = 14)
 
 TE_mrc_status_list &lt;- Peak_TE_overlapbreakdown %&gt;% 
      dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol)
 
 Peak_TE_overlapbreakdown %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;,&quot;all&quot;))</code></pre>
<p><img src="figure/TE_analysis.Rmd/another%20set%20of%20plots-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-plots-3">
Past versions of "another set of plots-3.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-plots-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/another set of plots-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>   Peak_TE_overlapbreakdown %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
     group_by(mrc, TEstatus) %&gt;% 
     count() %&gt;% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100) %&gt;% 
      kable(., caption=&quot;Table 7: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC.&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 7: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC.
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
summary
</th>
<th style="text-align:left;">
TEstatus
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
10435
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
8029
</td>
<td style="text-align:right;">
76.94298
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
10435
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
2406
</td>
<td style="text-align:right;">
23.05702
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
21763
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
16503
</td>
<td style="text-align:right;">
75.83054
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
21763
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
5260
</td>
<td style="text-align:right;">
24.16946
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
59727
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
47268
</td>
<td style="text-align:right;">
79.14009
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
59727
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
12459
</td>
<td style="text-align:right;">
20.85991
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
119719
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
93310
</td>
<td style="text-align:right;">
77.94084
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
119719
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
26409
</td>
<td style="text-align:right;">
22.05916
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
216091
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
168446
</td>
<td style="text-align:right;">
77.95142
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
216091
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
47645
</td>
<td style="text-align:right;">
22.04858
</td>
</tr>
</tbody>
</table>
<p>The plots below represent numbers when I include the &gt;50%
stringency cutoff filter.</p>
<pre class="r"><code>per_cov=0.5
TE_mrc_status_list &lt;- Peak_TE_overlapbreakdown %&gt;% 
      dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol)
 
 Peak_TE_overlapbreakdown %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
   dplyr::filter(per_ol&gt;per_cov| is.na(per_ol)) %&gt;% 
   ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;,&quot;&gt;&quot;,per_cov*100,&quot;% covered&quot;))</code></pre>
<p><img src="figure/TE_analysis.Rmd/another%20set%20of%20limited%20plots-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-another-set-of-limited-plots-1">
Past versions of "another set of limited plots-1.png"
</button>
</p>
<div id="fig-no-spaces-another-set-of-limited-plots-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/another set of limited plots-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/another set of limited plots-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>   Peak_TE_overlapbreakdown %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
      dplyr::filter(per_ol&gt;per_cov| is.na(per_ol)) %&gt;% 
     group_by(mrc, TEstatus) %&gt;% 
     count() %&gt;% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100) %&gt;% 
      kable(., caption=&quot;Table 8: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC using strigency cutoff of 50%.&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 8: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC using strigency cutoff of 50%.
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
summary
</th>
<th style="text-align:left;">
TEstatus
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
8022
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
5616
</td>
<td style="text-align:right;">
70.00748
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
8022
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
2406
</td>
<td style="text-align:right;">
29.99252
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
16696
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
11436
</td>
<td style="text-align:right;">
68.49545
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
16696
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
5260
</td>
<td style="text-align:right;">
31.50455
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
45117
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
32658
</td>
<td style="text-align:right;">
72.38513
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
45117
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
12459
</td>
<td style="text-align:right;">
27.61487
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
92616
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
66207
</td>
<td style="text-align:right;">
71.48549
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
92616
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
26409
</td>
<td style="text-align:right;">
28.51451
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
165868
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
118223
</td>
<td style="text-align:right;">
71.27535
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
165868
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
47645
</td>
<td style="text-align:right;">
28.72465
</td>
</tr>
</tbody>
</table>
</div>
<div id="human-genome-te-breakdown" class="section level3">
<h3>Human genome TE breakdown</h3>
<p>I first wanted to know the distribution of TEs across the human
genome compared to each other. Below are pie plots with all classes from
repeatmasker, and then pie plots with ONLY the LINES, SINES, LTRs, DNAs,
and Retroposon classes.</p>
<pre class="r"><code>repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/repeatmasker%20genome%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-1">
Past versions of "repeatmasker genome breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># repeatmasker  %&gt;% 
#    mutate(repClass=factor(repClass)) %&gt;% 
# count(repClass) %&gt;% 
#    mutate(perc= n/sum(n)) %&gt;% 
#    arrange(desc(n)) %&gt;% 
#   mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
#   mutate(text_y = cumsum(n) - n/2) %&gt;% 
#    ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   # geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
#   #                    position = position_stack(vjust = .5),
#   #                    force=.9,show.legend = FALSE,max.overlaps = 35) +
#   theme_void()+
#   ggtitle(&quot;Human genome TE breakdown without labels&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
#   scale_fill_repeat()
LiSiLTDNRe &lt;- repeatmasker %&gt;%
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;)


repeatmasker  %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon only&quot;, subtitle=paste(length(LiSiLTDNRe$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/repeatmasker%20genome%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-2">
Past versions of "repeatmasker genome breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-2"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>repeatmasker  %&gt;% 
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  # dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
count(repClass) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon focused with other&quot;, subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/repeatmasker%20genome%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-repeatmasker-genome-breakdown-3">
Past versions of "repeatmasker genome breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-repeatmasker-genome-breakdown-3"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/repeatmasker genome breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># saveRDS(TE_mrc_status_list,&quot;data/TE_info/TE_mrc_status_list.RDS&quot;)
TE_ALL_count &lt;- Peak_TE_overlapbreakdown %&gt;%
  distinct() %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(peakid) %&gt;% 
    count
TE_ALL_count_filt &lt;- Peak_TE_overlapbreakdown %&gt;%
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(peakid) %&gt;% 
    count
TE_50_count &lt;- Peak_TE_overlapbreakdown %&gt;% 
  # distinct() %&gt;% 
  dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(peakid) %&gt;% 
    count
TE_50_count_filt &lt;- Peak_TE_overlapbreakdown %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
  dplyr::filter(mrc!=&quot;not_mrc&quot;) %&gt;% 
  distinct(peakid) %&gt;% 
    count


TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks&quot;,subtitle = paste(TE_ALL_count$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-1">
Past versions of "TE in all peaks-1.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE in all peaks-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/TE in all peaks-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35)+
  # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks&quot;,subtitle = paste(TE_ALL_count_filt$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-2">
Past versions of "TE in all peaks-2.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE in all peaks-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/TE in all peaks-2.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(per_ol&gt;.5) %&gt;% 
  # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
    # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
    #                  position = position_stack(vjust = .3),
    #                  show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks using 50% cutoff&quot;,subtitle = paste(TE_50_count$n))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-3">
Past versions of "TE in all peaks-3.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE in all peaks-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/TE in all peaks-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  #  mutate(repClass=factor(repClass)) %&gt;% 
  # dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
   mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(per_ol&gt;per_cov) %&gt;% 
  # group_by(repClass) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;) %&gt;% 
   count(repClass) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  
  geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = paste(n,&quot;\n&quot;, repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;TE breakdown of all peaks using 50% cutoff&quot;,subtitle = paste(TE_50_count_filt$n, &quot;peaks that contain LINEs, SINEs, LTRs, DNAs, or Retroposons&quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-4">
Past versions of "TE in all peaks-4.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE in all peaks-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/TE in all peaks-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown%&gt;% 
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
    theme_classic()+
   # coord_cartesian(xlim= c(0,500))+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-5">
Past versions of "TE in all peaks-5.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Peak_TE_overlapbreakdown%&gt;% 
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  distinct() %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
    theme_classic()+
   # coord_cartesian(xlim= c(0,500))+
  ggtitle(&quot;Filtered Distribution of all overlapping peak-TE widths&quot;)+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-6">
Past versions of "TE in all peaks-6.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-6.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(per_ol&gt;0.5) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  scale_fill_repeat()+
  theme_classic()+
  ggtitle(&quot;Distribution of all overlapping peak-TE widths &gt;50%&quot;)</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-7">
Past versions of "TE in all peaks-7.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-7.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(per_ol&gt;0.5) %&gt;% 
  distinct() %&gt;% 
  dplyr::filter(TEstatus==&quot;TE_peak&quot;) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;|repClass==&quot;SINE&quot;|repClass==&quot;LTR&quot;|repClass==&quot;DNA&quot;|repClass==&quot;Retroposon&quot;) %&gt;% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  scale_fill_repeat()+
  theme_classic()+
  ggtitle(&quot;Filtered Distribution of all overlapping peak-TE widths &gt;50%&quot;)</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20in%20all%20peaks-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-in-all-peaks-8">
Past versions of "TE in all peaks-8.png"
</button>
</p>
<div id="fig-no-spaces-TE-in-all-peaks-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/TE in all peaks-8.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="line-repeats" class="section level4">
<h4>Line repeats</h4>
<pre class="r"><code>per_cov &lt;- 0.5


Line_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LINE breakdown&quot;, subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-1">
Past versions of "LINEclass breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LINEclass breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LINEclass breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/LINEclass breakdown-1.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/LINEclass breakdown-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Line_df%&gt;%
# count(repFamily) %&gt;%
#    mutate(perc= n/sum(n)) %&gt;%
#    arrange(desc(n)) %&gt;%
#   mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;%
#   mutate(text_y = cumsum(n) - n/2) %&gt;%
#    ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   theme_void()+
#   ggtitle(&quot;Human genome LINE breakdown without labels&quot;, subtitle=paste(length(Line_df$milliIns)))+
#   scale_fill_lines()

TE_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;%
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;LINE breakdown of peaks &quot;,per_cov),subtitle=paste(TE_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-2">
Past versions of "LINEclass breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LINEclass breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/LINEclass breakdown-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
ESR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
LR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
NR_LINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;EAR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(EAR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-3">
Past versions of "LINEclass breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LINEclass breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LINEclass breakdown-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LINEclass breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/LINEclass breakdown-3.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;ESR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(ESR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-4">
Past versions of "LINEclass breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LINEclass breakdown-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LINEclass breakdown-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LINEclass breakdown-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;LR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(LR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-5">
Past versions of "LINEclass breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LINEclass breakdown-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LINEclass breakdown-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LINEclass breakdown-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;NR LINE breakdown of peaks &quot;,per_cov),subtitle=paste(NR_LINE_count$n))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LINEclass%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LINEclass-breakdown-6">
Past versions of "LINEclass breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-LINEclass-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LINEclass breakdown-6.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LINEclass breakdown-6.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LINEclass breakdown-6.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>all_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  tally
EAR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;EAR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  tally
ESR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;ESR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  tally
LR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;LR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  tally
NR_L2_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;, mrc==&quot;NR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  tally


TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;&amp;per_ol&gt;per_cov)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for all peaks &quot;,per_cov),subtitle=paste(all_L2_count,&quot; total LINEs&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis.Rmd/L2breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-1">
Past versions of L2breakdown-1.png
</button>
</p>
<div id="fig-L2breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/L2breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/L2breakdown-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/L2breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;&amp;per_ol&gt;per_cov)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;%
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for EAR &quot;,per_cov),subtitle=paste(EAR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis.Rmd/L2breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-2">
Past versions of L2breakdown-2.png
</button>
</p>
<div id="fig-L2breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/L2breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/L2breakdown-2.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/L2breakdown-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;&amp;per_ol&gt;per_cov)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for ESR &quot;,per_cov),subtitle=paste(ESR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis.Rmd/L2breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-3">
Past versions of L2breakdown-3.png
</button>
</p>
<div id="fig-L2breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/L2breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/L2breakdown-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/L2breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;&amp;per_ol&gt;per_cov)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for LR &quot;,per_cov),subtitle=paste(LR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis.Rmd/L2breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-4">
Past versions of L2breakdown-4.png
</button>
</p>
<div id="fig-L2breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/L2breakdown-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/L2breakdown-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/L2breakdown-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LINE&quot;&amp;repFamily==&quot;L2&quot;&amp;per_ol&gt;per_cov)%&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LINE-L2 breakdown for NR &quot;,per_cov),subtitle=paste(NR_L2_count$n,&quot; total L2s&quot;))+
  scale_fill_L2()</code></pre>
<p><img src="figure/TE_analysis.Rmd/L2breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-L2breakdown-5">
Past versions of L2breakdown-5.png
</button>
</p>
<div id="fig-L2breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/L2breakdown-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/L2breakdown-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/L2breakdown-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="sine-repeats" class="section level4">
<h4>Sine repeats</h4>
<pre class="r"><code>Sine_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome SINE breakdown&quot;, subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-1">
Past versions of "Sineclass breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/Sineclass breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/Sineclass breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/Sineclass breakdown-1.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/Sineclass breakdown-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># Sine_df%&gt;% 
# count(repFamily) %&gt;% 
#    mutate(perc= n/sum(n)) %&gt;% 
#    arrange(desc(n)) %&gt;% 
#   mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
#   mutate(text_y = cumsum(n) - n/2) %&gt;% 
#    ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   theme_void()+
#   ggtitle(&quot;Human genome SINE breakdown without labels&quot;, subtitle=paste(length(Sine_df$milliIns)))+
#   scale_fill_sines()


TE_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0(&quot;SINE breakdown of peaks &quot;,per_cov),subtitle=paste(TE_SINE_count$n,&quot; total SINEs found&quot;))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-2">
Past versions of "Sineclass breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/Sineclass breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/Sineclass breakdown-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
ESR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
LR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
NR_SINE_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;EAR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(EAR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-3">
Past versions of "Sineclass breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/Sineclass breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/Sineclass breakdown-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/Sineclass breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/Sineclass breakdown-3.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  # 
  theme_void()+
  ggtitle(paste0(&quot;ESR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(ESR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-4">
Past versions of "Sineclass breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/Sineclass breakdown-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/Sineclass breakdown-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/Sineclass breakdown-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;LR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(LR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-5">
Past versions of "Sineclass breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/Sineclass breakdown-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/Sineclass breakdown-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/Sineclass breakdown-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0(&quot;NR SINE breakdown of peaks &quot;,per_cov),subtitle=paste(NR_SINE_count$n))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/Sineclass%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-Sineclass-breakdown-6">
Past versions of "Sineclass breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-Sineclass-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/Sineclass breakdown-6.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/Sineclass breakdown-6.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/Sineclass breakdown-6.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="ltr-repeats" class="section level4">
<h4>LTR repeats</h4>
<pre class="r"><code>per_cov &lt;- 0.5


LTR_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome LTR breakdown&quot;, subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-1">
Past versions of "LTR breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LTR breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LTR breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/LTR breakdown-1.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/LTR breakdown-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># LTR_df%&gt;% 
# count(repFamily) %&gt;% 
#    mutate(perc= n/sum(n)) %&gt;% 
#    arrange(desc(n)) %&gt;% 
#   mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
#   mutate(text_y = cumsum(n) - n/2) %&gt;% 
#    ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   theme_void()+
#   ggtitle(&quot;Human genome LTR breakdown without labels&quot;, subtitle=paste(length(LTR_df$milliIns)))+
#   scale_fill_LTRs()

LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  count
EAR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  count
ESR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  count
LR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  count
NR_LTR_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;All peaks LTR breakdown of peaks&quot;,per_cov),subtitle=paste(LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-2">
Past versions of "LTR breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LTR breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/LTR breakdown-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;EAR LTR breakdown of peaks&quot;,per_cov),subtitle=paste(EAR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-3">
Past versions of "LTR breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LTR breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LTR breakdown-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LTR breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/LTR breakdown-3.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR LTR breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-4">
Past versions of "LTR breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LTR breakdown-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LTR breakdown-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LTR breakdown-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR LTR breakdown of peaks &quot;,per_cov),subtitle=paste(LR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-5">
Past versions of "LTR breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LTR breakdown-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LTR breakdown-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LTR breakdown-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;LTR&quot;&amp;per_ol&gt; per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR LTR breakdown of peaks&quot;, per_cov),subtitle=paste(NR_LTR_count$n))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LTR%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LTR-breakdown-6">
Past versions of "LTR breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-LTR-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LTR breakdown-6.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LTR breakdown-6.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LTR breakdown-6.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dna-tes" class="section level4">
<h4>DNA TEs</h4>
<pre class="r"><code>per_cov &lt;- 0.5

DNA_df%&gt;% 
count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(&quot;Human genome DNA breakdown&quot;, subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-1">
Past versions of "DNA breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/DNA breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/DNA breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/DNA breakdown-1.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/DNA breakdown-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># DNA_df%&gt;% 
# count(repFamily) %&gt;% 
#    mutate(perc= n/sum(n)) %&gt;% 
#    arrange(desc(n)) %&gt;% 
#   mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
#   mutate(text_y = cumsum(n) - n/2) %&gt;% 
#    ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   theme_void()+
#   ggtitle(&quot;Human genome DNA breakdown without labels&quot;, subtitle=paste(length(DNA_df$milliIns)))+
#   scale_fill_DNAs()


TE_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
   mutate(repFamily=factor(repFamily)) %&gt;% 
     # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;DNA breakdown of peaks&quot;, per_cov),subtitle=paste(TE_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-2">
Past versions of "DNA breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/DNA breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/DNA breakdown-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/DNA breakdown-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
ESR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
LR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
NR_DNA_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count


TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;EAR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(EAR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-3">
Past versions of "DNA breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/DNA breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/DNA breakdown-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/DNA breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/DNA breakdown-3.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-4">
Past versions of "DNA breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/DNA breakdown-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/DNA breakdown-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/DNA breakdown-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR DNA breakdown of peaks&quot;,per_cov),subtitle=paste(LR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-5">
Past versions of "DNA breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/DNA breakdown-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/DNA breakdown-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/DNA breakdown-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  # group_by(repFamily) %&gt;% 
  count(repFamily) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR DNA breakdown of peaks&quot;, per_cov),subtitle=paste(NR_DNA_count$n))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/DNA%20breakdown-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DNA-breakdown-6">
Past versions of "DNA breakdown-6.png"
</button>
</p>
<div id="fig-no-spaces-DNA-breakdown-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/DNA breakdown-6.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/DNA breakdown-6.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/DNA breakdown-6.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="retroposon" class="section level4">
<h4>retroposon</h4>
<pre class="r"><code>per_cov &lt;- 0.5

retroposon_df%&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label =  paste0(repName ,&quot;\n&quot;, sprintf(&quot;%.2f&quot;, perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;Human genome retroposon breakdown&quot;, subtitle= paste( length(retroposon_df$milliIns)))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-1">
Past versions of "retroposon breakdown-1.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/retroposon breakdown-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/retroposon breakdown-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/retroposon breakdown-1.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/retroposon breakdown-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code># retroposon_df%&gt;% 
# count(repName) %&gt;% 
#    mutate(perc= n/sum(n)) %&gt;% 
#    arrange(desc(n)) %&gt;% 
#   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
#   mutate(text_y = cumsum(n) - n/2) %&gt;% 
#    ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   theme_void()+
#   ggtitle(&quot;Human genome retroposon breakdown without labels&quot;, subtitle=paste(length(retroposon_df$milliIns)))+
#   scale_fill_retroposons()

TE_retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass == &quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
   mutate(repName=factor(repName)) %&gt;% 
     # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
   mutate(perc= n/sum(n)) %&gt;% 
   arrange(desc(n)) %&gt;% 
  mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(TE_retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-2">
Past versions of "retroposon breakdown-2.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/retroposon breakdown-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/retroposon breakdown-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>EAR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
ESR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
LR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count
NR_Retroposon_count &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  count

# TE_mrc_status_list %&gt;% 
#   dplyr::filter(mrc ==&quot;EAR&quot;&amp;repClass==&quot;Retroposon&quot;) %&gt;% 
#   mutate(repName=factor(repName)) %&gt;% 
#   group_by(repName) %&gt;% 
#   count(repName) %&gt;% 
#   mutate(perc= n/sum(n)) %&gt;% 
#   arrange(desc(n)) %&gt;% 
#    mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
#   mutate(text_y = cumsum(n) - n/2) %&gt;% 
#   ggplot(., aes(x = &quot;&quot;, y = n, fill = repFamily)) +
#   geom_col(color = &quot;black&quot;) +
#   coord_polar(theta = &quot;y&quot;, start = 0)+
#   geom_label_repel(aes(label = n),
#                      position = position_stack(vjust = .3),
#                      show.legend = FALSE,max.overlaps = 50) +
#   theme_void()+
#   ggtitle(&quot;EAR Retroposon breakdown of peaks&quot;,subtitle=paste(EAR_Retroposon_count$n))+
#   scale_fill_retroposons()

TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;ESR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;ESR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(ESR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-3">
Past versions of "retroposon breakdown-3.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/retroposon breakdown-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/retroposon breakdown-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/retroposon breakdown-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b0aadc4c2205273739162480034a992e3be9fe97/docs/figure/TE_analysis.Rmd/retroposon breakdown-3.png" target="_blank">b0aadc4</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;LR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;LR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(LR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-4">
Past versions of "retroposon breakdown-4.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/retroposon breakdown-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/retroposon breakdown-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/retroposon breakdown-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(mrc ==&quot;NR&quot;&amp;repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  # group_by(repName) %&gt;% 
  count(repName) %&gt;% 
  mutate(perc= n/sum(n)) %&gt;% 
  arrange(desc(n)) %&gt;% 
   mutate(repName = fct_rev(fct_inorder(repName))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repName)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
  geom_label_repel(aes(label = paste0(repName,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste(&quot;NR Retroposon breakdown of peaks&quot;,per_cov),subtitle=paste(NR_Retroposon_count$n))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/retroposon%20breakdown-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-retroposon-breakdown-5">
Past versions of "retroposon breakdown-5.png"
</button>
</p>
<div id="fig-no-spaces-retroposon-breakdown-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/retroposon breakdown-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/retroposon breakdown-5.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/retroposon breakdown-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="te-by-response-cluster-bargraph-summaries"
class="section level3">
<h3>TE by response cluster bargraph summaries</h3>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,if_else(repClass_org==&quot;SINE&quot;,repClass_org,if_else(repClass_org==&quot;LTR&quot;, repClass_org, if_else(repClass_org==&quot;DNA&quot;, repClass_org, if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;,&quot;all&quot;))</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20status%20old%20clusters-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-status-old-clusters-1">
Past versions of "TE status old clusters-1.png"
</button>
</p>
<div id="fig-no-spaces-TE-status-old-clusters-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE status old clusters-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,if_else(repClass_org==&quot;SINE&quot;,repClass_org,if_else(repClass_org==&quot;LTR&quot;, repClass_org, if_else(repClass_org==&quot;DNA&quot;, repClass_org, if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  dplyr::filter(is.na(per_ol)| per_ol&gt;per_cov) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;,&quot;&gt;&quot;, per_cov*100,&quot;%&quot;))</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20status%20old%20clusters-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-status-old-clusters-2">
Past versions of "TE status old clusters-2.png"
</button>
</p>
<div id="fig-no-spaces-TE-status-old-clusters-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE status old clusters-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_counts_nofilt &lt;- TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,if_else(repClass_org==&quot;SINE&quot;,repClass_org,if_else(repClass_org==&quot;LTR&quot;, repClass_org, if_else(repClass_org==&quot;DNA&quot;, repClass_org, if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) %&gt;% 
   rbind(TE_mrc_status_list) %&gt;% 
  # mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
     group_by(mrc, TEstatus) %&gt;% 
     count() %&gt;% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100)

notinterested_list &lt;- c(&quot;Simple_repeat&quot;,&quot;Satellite&quot;,&quot;Low_complexity&quot;,&quot;DNA?&quot;,&quot;snRNA&quot;,&quot;tRNA&quot;,&quot;Unknown&quot;,&quot;RC&quot;,&quot;LTR?&quot;,&quot;srpRNA&quot;,&quot;scRNA&quot;,&quot;rRNA&quot;,&quot;RC?&quot;,&quot;SINE?&quot;)

Lines_Etc &lt;- TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
  dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org, if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol)%&gt;% 
   mutate(mrc=&quot;all_peaks&quot;) 
   


LiSiLTDNRe_TE_no_cut &lt;-
TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
  dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
        if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
  rbind(Lines_Etc) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  # dplyr::filter(is.na(per_ol)|per_ol&gt;per_cov) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
     group_by(mrc, TEstatus) %&gt;% 
     count() %&gt;% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100)

LiSiLTDNRe_TE_no_cut %&gt;% 
  kable(., caption=&quot;Table 9: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC with LINEs/SINEs/etc only in TE_peak count&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 9: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC with LINEs/SINEs/etc only in TE_peak count
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
summary
</th>
<th style="text-align:left;">
TEstatus
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
8640
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
6234
</td>
<td style="text-align:right;">
72.15278
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
8640
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
2406
</td>
<td style="text-align:right;">
27.84722
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
18255
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
12995
</td>
<td style="text-align:right;">
71.18598
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
18255
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
5260
</td>
<td style="text-align:right;">
28.81402
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
52594
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
40135
</td>
<td style="text-align:right;">
76.31099
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
52594
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
12459
</td>
<td style="text-align:right;">
23.68901
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
96554
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
70145
</td>
<td style="text-align:right;">
72.64847
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
96554
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
26409
</td>
<td style="text-align:right;">
27.35153
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
179789
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
132144
</td>
<td style="text-align:right;">
73.49949
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
179789
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
47645
</td>
<td style="text-align:right;">
26.50051
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>LiSiLTDNRe_TE_df &lt;- readRDS(&quot;data/LiSiLTDNRe_TE_df.RDS&quot;)

LiSiLTDNRe_TE_cut &lt;-
TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
  dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
        if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,if_else(is.na(repClass_org),repClass_org,&quot;Other&quot;))))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
  rbind(Lines_Etc) %&gt;% 
  mutate(repClass=factor(repClass)) %&gt;%
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  dplyr::filter(is.na(per_ol)|per_ol&gt;per_cov) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
     group_by(mrc, TEstatus) %&gt;% 
     count() %&gt;% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %&gt;% 
     rowwise() %&gt;% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %&gt;% 
     ungroup() %&gt;% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c(&quot;TEstatus&quot;), values_to = &quot;n&quot;) %&gt;% 
     mutate(percent_mrc= n/summary*100)


TE_mrc_status_list %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
  dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
  rbind(Lines_Etc) %&gt;% 
  dplyr::filter(is.na(per_ol)| per_ol &gt; per_cov) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family&quot;, per_cov), subtitle = &quot;Just LINEs, SINEs,LTRs, DNAs, Retroposons, and all peaks in the MRC families&quot;)</code></pre>
<p><img src="figure/TE_analysis.Rmd/TE%20status%20old%20clusters-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-TE-status-old-clusters-3">
Past versions of "TE status old clusters-3.png"
</button>
</p>
<div id="fig-no-spaces-TE-status-old-clusters-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/TE status old clusters-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>LiSiLTDNRe_TE_cut %&gt;% 
  kable(., caption=&quot;Table 10: Summary of peak numbers overlapping and not overlapping TEs by each basic MRC using stringency cutoff of 50%. LINEs/SINEs/etc only in TE_peak count&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 10: Summary of peak numbers overlapping and not overlapping TEs by
each basic MRC using stringency cutoff of 50%. LINEs/SINEs/etc only in
TE_peak count
</caption>
<thead>
<tr>
<th style="text-align:left;">
mrc
</th>
<th style="text-align:right;">
summary
</th>
<th style="text-align:left;">
TEstatus
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
percent_mrc
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
6356
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
3950
</td>
<td style="text-align:right;">
62.14600
</td>
</tr>
<tr>
<td style="text-align:left;">
EAR
</td>
<td style="text-align:right;">
6356
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
2406
</td>
<td style="text-align:right;">
37.85400
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
13546
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
8286
</td>
<td style="text-align:right;">
61.16935
</td>
</tr>
<tr>
<td style="text-align:left;">
ESR
</td>
<td style="text-align:right;">
13546
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
5260
</td>
<td style="text-align:right;">
38.83065
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
38563
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
26104
</td>
<td style="text-align:right;">
67.69183
</td>
</tr>
<tr>
<td style="text-align:left;">
LR
</td>
<td style="text-align:right;">
38563
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
12459
</td>
<td style="text-align:right;">
32.30817
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
70898
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
44489
</td>
<td style="text-align:right;">
62.75071
</td>
</tr>
<tr>
<td style="text-align:left;">
NR
</td>
<td style="text-align:right;">
70898
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
26409
</td>
<td style="text-align:right;">
37.24929
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
132157
</td>
<td style="text-align:left;">
TE_peak
</td>
<td style="text-align:right;">
84512
</td>
<td style="text-align:right;">
63.94818
</td>
</tr>
<tr>
<td style="text-align:left;">
all_peaks
</td>
<td style="text-align:right;">
132157
</td>
<td style="text-align:left;">
not_TE_peak
</td>
<td style="text-align:right;">
47645
</td>
<td style="text-align:right;">
36.05182
</td>
</tr>
</tbody>
</table>
</div>
<div id="breakdown-of-classes-by-4-member-clusters"
class="section level3">
<h3>Breakdown of Classes by 4 member clusters</h3>
<pre class="r"><code>per_cov &lt;- 0.5
 
ggline_df &lt;- Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;, per_ol = &quot;NA&quot;) %&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;LINE&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;))
 
  

ggsine_df &lt;-
  Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;SINE&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;))

ggLTR_df &lt;-LTR_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;LTR&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;))


ggDNA_df &lt;-DNA_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;DNA&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;))


ggretroposon_df &lt;-retroposon_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;Retroposon&quot;) %&gt;% mutate(mrc=&quot;all_peaks&quot;))


plot1 &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggline_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LINE breakdown by MRC and Family&quot;, per_cov))+
  scale_fill_lines()
plot1</code></pre>
<p><img src="figure/TE_analysis.Rmd/breakdown%20by%20cluster-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-1">
Past versions of "breakdown by cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/breakdown by cluster-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>plot2 &lt;- TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;SINE breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_sines()
plot2</code></pre>
<p><img src="figure/TE_analysis.Rmd/breakdown%20by%20cluster-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-2">
Past versions of "breakdown by cluster-2.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/breakdown by cluster-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggLTR_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LTR breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/breakdown%20by%20cluster-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-3">
Past versions of "breakdown by cluster-3.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/breakdown by cluster-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/breakdown by cluster-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggDNA_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;DNA breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/breakdown%20by%20cluster-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-4">
Past versions of "breakdown by cluster-4.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/breakdown by cluster-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>TE_mrc_status_list %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggretroposon_df) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR&quot;,&quot;ESR&quot;, &quot;LR&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;Retroposon breakdown by MRC and Family&quot;,per_cov))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/breakdown%20by%20cluster-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-cluster-5">
Past versions of "breakdown by cluster-5.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-cluster-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/breakdown by cluster-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="adding-in-the-new-categories" class="section level3">
<h3>Adding in the new categories</h3>
<p>Here I break up the categories using median LFC</p>
<ul>
<li>EAR_hyper = median 3 hour LFC &gt; 0<br />
</li>
<li>EAR_hypo = median 3 hour LFC &lt; 0</li>
<li>ESR_A = median 3 hour &gt; 0 and median 24 hour &gt;0</li>
<li>ESR_B = median 3 hour &lt; 0 and median 24 hour &lt;0</li>
<li>ESR_C = median 3 hour &gt; 0 and median 24 hour &lt;0</li>
<li>ESR_D = median 3 hour &lt; 0 and median 24 hour &gt;0</li>
<li>LR_hyper = median 24 hour LFC &gt; 0<br />
</li>
<li>LR_hypo = median 24 hour LFC &lt; 0<br />
</li>
<li>NR = all NR classified peaks</li>
</ul>
<pre class="r"><code>per_cov=0.5
median_24_lfc &lt;- readRDS(&quot;data/median_24_lfc.RDS&quot;) %&gt;% ungroup()
median_3_lfc &lt;- readRDS(&quot;data/median_3_lfc.RDS&quot;) %&gt;% ungroup()

hyper_3med &lt;- median_3_lfc %&gt;% 
  dplyr::filter(median &gt; 0)

hypo_3med &lt;- median_3_lfc %&gt;% 
  dplyr::filter(median &lt; 0)

hyper_24med &lt;- median_24_lfc %&gt;% 
  dplyr::filter(median &gt; 0)

hypo_24med &lt;- median_24_lfc %&gt;% 
  dplyr::filter(median &lt; 0)

medA &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::rename(&quot;3_hours&quot;=&quot;median.x&quot;, &quot;24_hours&quot;=&quot;median.y&quot;) %&gt;% 
  dplyr::filter(`3_hours`&gt;0 &amp; `24_hours`&gt;0)

medB &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::rename(&quot;3_hours&quot;=&quot;median.x&quot;, &quot;24_hours&quot;=&quot;median.y&quot;) %&gt;% 
  dplyr::filter(`3_hours`&lt;0 &amp; `24_hours`&lt;0)

medC &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::rename(&quot;3_hours&quot;=&quot;median.x&quot;, &quot;24_hours&quot;=&quot;median.y&quot;) %&gt;% 
  dplyr::filter(`3_hours`&gt;0 &amp; `24_hours`&lt;0)

medD &lt;- median_3_lfc %&gt;% 
  left_join(median_24_lfc, by=c(&quot;peak&quot;=&quot;peak&quot;)) %&gt;% 
  dplyr::rename(&quot;3_hours&quot;=&quot;median.x&quot;, &quot;24_hours&quot;=&quot;median.y&quot;) %&gt;% 
  dplyr::filter(`3_hours`&lt;0 &amp; `24_hours`&gt;0)

EAR_hyper &lt;- EAR_df %&gt;%
  dplyr::filter(id %in% hyper_3med$peak) %&gt;% 
  dplyr::select(seqnames:id)
EAR_hyper_gr &lt;- EAR_hyper %&gt;% GRanges()

EAR_hypo &lt;- EAR_df %&gt;%
  dplyr::filter(id %in% hypo_3med$peak) %&gt;% 
  dplyr::select(seqnames:id)
EAR_hypo_gr &lt;- EAR_hypo %&gt;% GRanges()

LR_hyper &lt;- LR_df %&gt;%
  dplyr::filter(id %in% hyper_24med$peak) %&gt;% 
  dplyr::select(seqnames:id)
LR_hyper_gr &lt;- LR_hyper %&gt;% GRanges()

LR_hypo &lt;- LR_df %&gt;%
  dplyr::filter(id %in% hypo_24med$peak) %&gt;% 
  dplyr::select(seqnames:id)
LR_hypo_gr &lt;- LR_hypo %&gt;% GRanges()

ESR_A &lt;- ESR_df %&gt;% 
  dplyr::filter(id %in% medA$peak) %&gt;% 
  dplyr::select(seqnames:id)

ESR_B &lt;- ESR_df %&gt;% 
  dplyr::filter(id %in% medB$peak) %&gt;% 
  dplyr::select(seqnames:id)

ESR_C &lt;- ESR_df %&gt;% 
  dplyr::filter(id %in% medC$peak) %&gt;% 
  dplyr::select(seqnames:id)

ESR_D &lt;- ESR_df %&gt;% 
  dplyr::filter(id %in% medD$peak) %&gt;% 
  dplyr::select(seqnames:id)

Nine_group_TE &lt;- Col_TSS_data_gr %&gt;% as.data.frame %&gt;% 
  dplyr::select(peakid) %&gt;% 
  left_join(.,(Col_fullDF_overlap %&gt;% as.data.frame)) %&gt;% 
   mutate(TEstatus=if_else(is.na(repClass),&quot;not_TE_peak&quot;,&quot;TE_peak&quot;)) %&gt;% 
   mutate(mrc=if_else(peakid %in% EAR_hyper$id, &quot;EAR_hyper&quot;,
                      if_else(peakid %in% EAR_hypo$id, &quot;EAR_hypo&quot;,
                              if_else(peakid %in% ESR_A$id,&quot;ESR_A&quot;,
                                      if_else(peakid %in% ESR_B$id,&quot;ESR_B&quot;,
                                              if_else(peakid %in% ESR_C$id,&quot;ESR_C&quot;,
                                                      if_else(peakid %in% ESR_D$id,&quot;ESR_D&quot;,
                                                              if_else(peakid %in% LR_hyper$id,&quot;LR_hyper&quot;,
                                                                      if_else(peakid %in% LR_hypo$id,&quot;LR_hypo&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;)))))))))) %&gt;%  
                                       mutate(per_ol= width/TE_width) %&gt;% 
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
  dplyr::select(peakid, repName,repClass,repClass_org, repFamily, width, TEstatus, mrc, per_ol) </code></pre>
<div id="ear" class="section level4">
<h4>EAR</h4>
<pre class="r"><code>TE_EAR_hyper_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;EAR_hyper&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  distinct() %&gt;% 
  count
TE_EAR_hypo_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;EAR_hypo&quot;&amp;per_ol&gt;per_cov) %&gt;%
  distinct() %&gt;% 
  count
Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;EAR_hyper&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;EAR_hyper TE peak breakdown&quot;,subtitle = paste(TE_EAR_hyper_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/EAR-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-EAR-1">
Past versions of EAR-1.png
</button>
</p>
<div id="fig-EAR-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/EAR-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/EAR-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/EAR-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/EAR-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/EAR-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;EAR_hypo&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;EAR_hypo TE peak breakdown&quot;,subtitle = paste(TE_EAR_hypo_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/EAR-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-EAR-2">
Past versions of EAR-2.png
</button>
</p>
<div id="fig-EAR-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/EAR-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/EAR-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/EAR-2.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="esr" class="section level4">
<h4>ESR</h4>
<pre class="r"><code>TE_ESR_A_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_A&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  distinct() %&gt;% 
  count
TE_ESR_B_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_B&quot;&amp;per_ol&gt;per_cov) %&gt;%
  distinct() %&gt;% 
  count
TE_ESR_C_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_C&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  distinct() %&gt;% 
  count
TE_ESR_D_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_D&quot;&amp;per_ol&gt;per_cov) %&gt;%
  distinct() %&gt;% 
  count



Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_A&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   theme_void()+
  ggtitle(&quot;ESR_A TE peak breakdown&quot;,subtitle = paste(TE_ESR_A_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/ESR-te-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-1">
Past versions of ESR-te-1.png
</button>
</p>
<div id="fig-ESR-te-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/ESR-te-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/ESR-te-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/ESR-te-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/ESR-te-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/ESR-te-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_B&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   theme_void()+
  ggtitle(&quot;ESR_B TE peak breakdown&quot;,subtitle = paste(TE_ESR_B_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/ESR-te-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-2">
Past versions of ESR-te-2.png
</button>
</p>
<div id="fig-ESR-te-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/ESR-te-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_C&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   theme_void()+
  ggtitle(&quot;ESR_C TE peak breakdown&quot;,subtitle = paste(TE_ESR_C_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/ESR-te-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-3">
Past versions of ESR-te-3.png
</button>
</p>
<div id="fig-ESR-te-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/ESR-te-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;ESR_D&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   theme_void()+
  ggtitle(&quot;ESR_D TE peak breakdown&quot;,subtitle = paste(TE_ESR_D_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/ESR-te-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ESR-te-4">
Past versions of ESR-te-4.png
</button>
</p>
<div id="fig-ESR-te-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/ESR-te-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="lr" class="section level4">
<h4>LR</h4>
<pre class="r"><code>TE_LR_hyper_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;LR_hyper&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  distinct() %&gt;% 
  count
TE_LR_hypo_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;LR_hypo&quot;&amp;per_ol&gt;per_cov) %&gt;%
  distinct() %&gt;% 
  count
Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;LR_hyper&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;LR_hyper TE peak breakdown&quot;,subtitle = paste(TE_LR_hyper_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LRTE-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LRTE-1">
Past versions of LRTE-1.png
</button>
</p>
<div id="fig-LRTE-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LRTE-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/LRTE-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/LRTE-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/LRTE-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/LRTE-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;LR_hypo&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;LR_hypo TE peak breakdown&quot;,subtitle = paste(TE_LR_hypo_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LRTE-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LRTE-2">
Past versions of LRTE-2.png
</button>
</p>
<div id="fig-LRTE-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LRTE-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nr" class="section level4">
<h4>NR</h4>
<pre class="r"><code>TE_NR_count &lt;-
  Nine_group_TE %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;NR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  distinct() %&gt;% 
  count

Nine_group_TE %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
  dplyr::filter(TEstatus ==&quot;TE_peak&quot;&amp; mrc==&quot;NR&quot;&amp;per_ol&gt;per_cov) %&gt;%
  count(repClass) %&gt;% 
  dplyr::mutate(perc = n/(sum(n))) %&gt;% 
  arrange(desc(n)) %&gt;% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %&gt;% 
  mutate(text_y = cumsum(n) - n/2) %&gt;% 
  ggplot(., aes(x = &quot;&quot;, y = n, fill = repClass)) +
  geom_col(color = &quot;black&quot;) +
  coord_polar(theta = &quot;y&quot;, start = 0)+
   geom_label_repel(aes(label = paste0(repClass,&quot;\n&quot;,sprintf(&quot;%.2f&quot;,perc*100),&quot;%&quot;)),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(&quot;NR TE peak breakdown&quot;,subtitle = paste(TE_NR_count$n, &quot;using &gt; 50% coverage &quot;))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/NR-te-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-NR-te-1">
Past versions of NR-te-1.png
</button>
</p>
<div id="fig-NR-te-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/NR-te-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/NR-te-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/NR-te-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/9678f55cd1d509320ab30612921307a0426e90ea/docs/figure/TE_analysis.Rmd/NR-te-1.png" target="_blank">9678f55</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-07-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/NR-te-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="sub-set-breakdown-with-new-classes" class="section level3">
<h3>sub-set breakdown with new classes</h3>
<p>This section reclassifies any TE that is NOT a LINE, SINE, LTR, DNA,
or retroposon as “other”. Still using &gt; 50% coverage cutoff</p>
<pre class="r"><code>per_cov &lt;- 0.5
subsetall_df &lt;-  Nine_group_TE %&gt;% 
    mutate(mrc=&quot;all_peaks&quot;) %&gt;%
    dplyr::filter(per_ol&gt;per_cov) %&gt;% 
    dplyr::filter(mrc != &quot;not_mrc&quot;) 


# h.genome_df &lt;- LiSiLTDNRe %&gt;% 
h.genome_df &lt;- repeatmasker %&gt;% 
  mutate(repClass_org = repClass) %&gt;% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,if_else(repClass_org==&quot;SINE&quot;,repClass_org,if_else(repClass_org==&quot;LTR&quot;, repClass_org, if_else(repClass_org==&quot;DNA&quot;, repClass_org, if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,&quot;Other&quot;)))))) %&gt;% 
  mutate(peakid=paste0(rownames(.),&quot;_TE&quot;)) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,repClass_org) %&gt;%
   mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;, width=&quot;NA&quot;)
 

# Nine_group_TE %&gt;% 
#   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
#   dplyr::filter(per_ol&gt;per_cov) %&gt;% 
#   rbind(h.genome_df)%&gt;% 
#   rbind(subsetall_df) %&gt;% 
#   mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;, &quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;,&quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
#   ggplot(., aes(x=mrc, fill= repClass))+
#   geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
#   theme_bw()+
#   ggtitle(paste(&quot;Repeat breakdown by MRC&quot;, per_cov))+
#   scale_fill_repeat()

Nine_group_TE %&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
  dplyr::filter(per_ol&gt;per_cov) %&gt;%
  rbind(subsetall_df) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;, &quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;,&quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  ggplot(., aes(x=mrc, fill= repClass))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;Repeat breakdown across nine clusters&quot;, per_cov))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/breakdown%20by%20specific%20cluster-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-breakdown-by-specific-cluster-1">
Past versions of "breakdown by specific cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-breakdown-by-specific-cluster-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/breakdown by specific cluster-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/breakdown by specific cluster-1.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Now I will display without the “other” and regraph with new
groups</p>
<pre class="r"><code>Nine_group_TE %&gt;%
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;%
  dplyr::filter(per_ol&gt;per_cov) %&gt;%
  rbind(subsetall_df) %&gt;%
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;, &quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;,&quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;%
  dplyr::filter(repClass!=&quot;Other&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repClass))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;Repeat breakdown across nine clusters without &#39;Other&#39;&quot;, per_cov))+
  scale_fill_repeat()</code></pre>
<p><img src="figure/TE_analysis.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/unnamed-chunk-2-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/unnamed-chunk-2-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/788d19aadf95738b3b33b620b5bb07c5567c2235/docs/figure/TE_analysis.Rmd/unnamed-chunk-2-1.png" target="_blank">788d19a</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-06-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
   mutate(repClass_org = repClass) %&gt;% 
   dplyr::filter(!repClass %in% notinterested_list) %&gt;% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %&gt;% 
  rbind(Lines_Etc) %&gt;% 
  dplyr::filter(per_ol&gt;per_cov|is.na(per_ol)) %&gt;% 
   mutate(repClass=factor(repClass)) %&gt;% 
   mutate(TEstatus=factor(TEstatus, levels = c(&quot;TE_peak&quot;,&quot;not_TE_peak&quot;)))%&gt;% 
   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;))) %&gt;% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;TE status by MRC and Family &gt; 50 %&quot;), subtitle = &quot;Just LINEs, SINEs,LTRs, DNAs, Retroposons, and all peaks in the MRC families&quot;)</code></pre>
<p><img src="figure/TE_analysis.Rmd/unnamed-chunk-2-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="sub-class-breakdown" class="section level3">
<h3>sub-Class breakdown</h3>
<pre class="r"><code>per_cov &lt;- 0.5
 
ggline_df &lt;-
Line_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;%
   mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;, per_ol = &quot;NA&quot;) %&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;%  mutate(repClass_org=repClass)) 
  

ggsine_df &lt;-
  Sine_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;) %&gt;%
  mutate(repClass_org=repClass) %&gt;% 
   mutate(repClass=if_else(##relable repClass with other
    repClass_org==&quot;LINE&quot;, repClass_org,
    if_else(repClass_org==&quot;SINE&quot;,repClass_org,
            if_else(repClass_org==&quot;LTR&quot;, repClass_org, 
                    if_else(repClass_org==&quot;DNA&quot;, repClass_org,
                            if_else(repClass_org==&quot;Retroposon&quot;,repClass_org,
                                    if_else(is.na(repClass_org), repClass_org,&quot;Other&quot;))))))) %&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% mutate(mrc=&quot;all_peaks&quot;) %&gt;% mutate(repClass_org=repClass))

ggLTR_df &lt;-LTR_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;LTR&quot;&amp;per_ol&gt;per_cov) %&gt;% mutate(mrc=&quot;all_peaks&quot;)) %&gt;%  mutate(repClass_org=repClass) 


ggDNA_df &lt;-DNA_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% mutate(mrc=&quot;all_peaks&quot;)) %&gt;% mutate(repClass_org=repClass) 


ggretroposon_df &lt;-retroposon_repeats %&gt;% 
  as.data.frame() %&gt;% 
 tidyr::unite(peakid,seqnames:end, sep= &quot;.&quot;) %&gt;% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %&gt;% 
  mutate(TEstatus =&quot;TE_peak&quot;, mrc=&quot;h.genome&quot;,per_ol = &quot;NA&quot;)%&gt;% 
    rbind(TE_mrc_status_list %&gt;% dplyr::filter(repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% mutate(mrc=&quot;all_peaks&quot;)) %&gt;%  mutate(repClass_org=repClass) 

# Nine_group_TE %&gt;% 
#   dplyr::filter(repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
#   dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
#   rbind(., ggline_df) %&gt;% 
#   mutate(repFamily=factor(repFamily)) %&gt;% 
#   mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
#   ggplot(., aes(x=mrc, fill= repFamily))+
#   geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
#   theme_bw()+
#   ggtitle(paste(&quot;LINE breakdown by MRC and Family&quot;, per_cov))+
#   scale_fill_lines()</code></pre>
<pre class="r"><code>Nine_group_TE %&gt;% 
    dplyr::filter(repClass==&quot;LINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  # distinct(mrc)
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
  rbind(., ggline_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  dplyr::filter(mrc != &quot;h.genome&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LINE breakdown by nine-clusters and Family&quot;, per_cov))+
  scale_fill_lines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LinesLTRand%20nine-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LinesLTRand-nine-1">
Past versions of "LinesLTRand nine-1.png"
</button>
</p>
<div id="fig-no-spaces-LinesLTRand-nine-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LinesLTRand nine-1.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;SINE&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  dplyr::filter(mrc != &quot;h.genome&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;SINE breakdown by breakdown by nine-clusters and Family&quot;,per_cov))+
  scale_fill_sines()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LinesLTRand%20nine-2.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LinesLTRand-nine-2">
Past versions of "LinesLTRand nine-2.png"
</button>
</p>
<div id="fig-no-spaces-LinesLTRand-nine-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LinesLTRand nine-2.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;LTR&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  dplyr::filter(mrc != &quot;h.genome&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;LTR breakdown by breakdown by nine-clusters and Family&quot;,per_cov))+
  scale_fill_LTRs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LinesLTRand%20nine-3.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LinesLTRand-nine-3">
Past versions of "LinesLTRand nine-3.png"
</button>
</p>
<div id="fig-no-spaces-LinesLTRand-nine-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LinesLTRand nine-3.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;DNA&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggsine_df) %&gt;% 
  mutate(repFamily=factor(repFamily)) %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  dplyr::filter(mrc != &quot;h.genome&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_bw()+
  ggtitle(paste(&quot;DNA breakdown by breakdown by nine-clusters and Family&quot;,per_cov))+
  scale_fill_DNAs()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LinesLTRand%20nine-4.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LinesLTRand-nine-4">
Past versions of "LinesLTRand nine-4.png"
</button>
</p>
<div id="fig-no-spaces-LinesLTRand-nine-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LinesLTRand nine-4.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Nine_group_TE %&gt;% 
  dplyr::filter(repClass==&quot;Retroposon&quot;&amp;per_ol&gt;per_cov) %&gt;% 
  dplyr::filter(mrc != &quot;not_mrc&quot;) %&gt;% 
   rbind(., ggretroposon_df) %&gt;% 
  mutate(repName=factor(repName)) %&gt;% 
 mutate(mrc=factor(mrc, levels = c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;, &quot;LR_hyper&quot;,&quot;LR_hypo&quot;, &quot;NR&quot;,&quot;all_peaks&quot;,&quot;h.genome&quot;))) %&gt;% 
  dplyr::filter(mrc != &quot;h.genome&quot;) %&gt;% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position=&quot;fill&quot;, col=&quot;black&quot;)+
  theme_classic()+
  ggtitle(paste(&quot;Retroposon breakdown by by nine-clusters and Family&quot;,per_cov))+
  scale_fill_retroposons()</code></pre>
<p><img src="figure/TE_analysis.Rmd/LinesLTRand%20nine-5.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-LinesLTRand-nine-5">
Past versions of "LinesLTRand nine-5.png"
</button>
</p>
<div id="fig-no-spaces-LinesLTRand-nine-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/3343a3549292f3cfdd41d6ab6f178c19dfbaf711/docs/figure/TE_analysis.Rmd/LinesLTRand nine-5.png" target="_blank">3343a35</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cg-islands" class="section level3">
<h3>CG islands!</h3>
<pre class="r"><code>cpgislands_df &lt;- read.delim(&quot;data/other_papers/cpg_islands.tsv&quot;)
cpg_cCREs_df &lt;- read_delim(&quot;data/other_papers/cpg_cCREs.tsv&quot;, delim=&quot;\t&quot;)
aligncre &lt;- genomation::readBed(&quot;data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed&quot;) %&gt;% as.data.frame

CPG_promoters_gr &lt;- cpg_cCREs_df %&gt;% 
    dplyr::rename(.,&quot;seqnames&quot;=X1,&quot;start&quot;=X2,&quot;end&quot;=X3,&quot;promotor_name&quot;=X4,&quot;length&quot;=X5,&quot;strand&quot;=X6,&quot;color&quot;=X9) %&gt;% 
  dplyr::filter(color ==25500) %&gt;% 
  dplyr::select(seqnames:strand,color) %&gt;% 
  GRanges()


  # classify_cres &lt;- aligncre %&gt;% 
  #   distinct(blockCount,.keep_all = TRUE) %&gt;% 
  #   select(itemRgb,blockCount) %&gt;% 
  #   mutate(number)
Peaks_v_cpgpromo &lt;- join_overlap_intersect(CPG_promoters_gr,Col_TSS_data_gr) %&gt;% as.data.frame

cpg_island_gr &lt;- cpgislands_df %&gt;% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = &quot;chrom&quot;, start.field = &quot;chromStart&quot;, end.field = &quot;chromEnd&quot;,starts.in.df.are.0based=TRUE)
Col_TSS_data_gr$peak_width &lt;- width(Col_TSS_data_gr)
cpg_island_gr$cpg_width &lt;- width(cpg_island_gr)

Col_fullDF_cug_overlap &lt;- join_overlap_intersect(Col_TSS_data_gr,cpg_island_gr)
Col_fullDF_cug_overlap &lt;-Col_fullDF_cug_overlap %&gt;% 
  as.data.frame %&gt;% 
  mutate(per_ol=width/cpg_width)


Col_fullDF_cug_overlap %&gt;% 
  as.data.frame() %&gt;% 
  # group_by(name) %&gt;%
  distinct(peakid) %&gt;% 
  tally %&gt;% 
  kable(., caption=&quot;Count of peaks with CG islands&quot;) %&gt;% 
  kable_paper(&quot;striped&quot;, full_width = TRUE) %&gt;%
  kable_styling(full_width = FALSE, font_size = 14)</code></pre>
<table class=" lightable-paper lightable-striped table" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto; font-size: 14px; width: auto !important; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Count of peaks with CG islands
</caption>
<thead>
<tr>
<th style="text-align:right;">
n
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
18198
</td>
</tr>
</tbody>
</table>
<pre class="r"><code># testingol_cpgCol_fullDF_cug_overlap %&gt;% 
#   mutate(per_ol = width/cpg_width)
  
# hits_cpg &lt;- findOverlaps(Col_TSS_data_gr,cpg_island_gr)
# overlaps_cpg &lt;- pintersect(Col_TSS_data_gr[queryHits(hits_cpg)], cpg_island_gr[subjectHits(hits_cpg)])
# percentOverlap_cpg &lt;- width(overlaps_cpg) / width(cpg_island_gr[subjectHits(hits_cpg)])
# hits_cpg &lt;- hits_cpg[percentOverlap_cpg &gt; 0.5]
# 
# testingol_cpg &lt;- Col_TSS_data_gr[queryHits(hits_cpg)]

CUG_mrc_status_list &lt;-
Col_TSS_data_gr%&gt;% as.data.frame() %&gt;%
  left_join(., (Col_fullDF_cug_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;peakid&quot;=&quot;peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;ensembl_ID&quot;=&quot;ensembl_ID&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;,  &quot;SYMBOL&quot; = &quot;SYMBOL&quot;, &quot;peak_width&quot;=&quot;peak_width&quot;)) %&gt;% 
  left_join(., (Peaks_v_cpgpromo %&gt;% dplyr::select(promotor_name,peakid,peak_width)), by=c(&quot;peak_width&quot;=&quot;peak_width&quot;,&quot;peakid&quot;=&quot;peakid&quot;)) %&gt;% 
  dplyr::select(peakid, name,cpgNum:promotor_name) %&gt;% 
  mutate(cugstatus=if_else(is.na(cpgNum),&quot;not_CGi_peak&quot;,&quot;CGi_peak&quot;)) %&gt;% 
  mutate(prom_status= if_else(is.na(promotor_name),&quot;not_CpGpromo&quot;,&quot;CpGpromo&quot;)) %&gt;% 
   mutate(mrc=if_else(peakid %in% EAR_df$id, &quot;EAR&quot;,
                     if_else(peakid %in% ESR_df$id,&quot;ESR&quot;,
                             if_else(peakid %in% LR_df$id,&quot;LR&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;))))) %&gt;% distinct()



CUG_mrc_nine_list &lt;-
Col_TSS_data_gr%&gt;% as.data.frame() %&gt;%
  left_join(., (Col_fullDF_cug_overlap %&gt;% as.data.frame(.)), by =c(&quot;seqnames&quot;=&quot;seqnames&quot;,&quot;start&quot;=&quot;start&quot;,&quot;end&quot;=&quot;end&quot;,&quot;peakid&quot;=&quot;peakid&quot;, &quot;NCBI_gene&quot;=&quot;NCBI_gene&quot;, &quot;ensembl_ID&quot;=&quot;ensembl_ID&quot;,&quot;dist_to_NG&quot;=&quot;dist_to_NG&quot;,  &quot;SYMBOL&quot; = &quot;SYMBOL&quot;, &quot;peak_width&quot;=&quot;peak_width&quot;)) %&gt;% 
  left_join(., (Peaks_v_cpgpromo %&gt;% dplyr::select(promotor_name,peakid,peak_width)), by=c(&quot;peak_width&quot;=&quot;peak_width&quot;,&quot;peakid&quot;=&quot;peakid&quot;)) %&gt;% 
  dplyr::select(peakid, name,cpgNum:promotor_name) %&gt;% 
  mutate(cugstatus=if_else(is.na(cpgNum),&quot;not_CGi_peak&quot;,&quot;CGi_peak&quot;)) %&gt;% 
  mutate(prom_status= if_else(is.na(promotor_name),&quot;not_CpGpromo&quot;,&quot;CpGpromo&quot;)) %&gt;% 
   mutate(mrc=if_else(peakid %in% EAR_hyper$id, &quot;EAR_hyper&quot;,
                      if_else(peakid %in% EAR_hypo$id, &quot;EAR_hypo&quot;,
                              if_else(peakid %in% ESR_A$id,&quot;ESR_A&quot;,
                                      if_else(peakid %in% ESR_B$id,&quot;ESR_B&quot;,
                                              if_else(peakid %in% ESR_C$id,&quot;ESR_C&quot;,
                                                      if_else(peakid %in% ESR_D$id,&quot;ESR_D&quot;,
                                                              if_else(peakid %in% LR_hyper$id,&quot;LR_hyper&quot;,
                                                                      if_else(peakid %in% LR_hypo$id,&quot;LR_hypo&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;)))))))))) %&gt;%
  distinct()


CUG_mrc_status_list %&gt;% 
 group_by(cugstatus, mrc) %&gt;% 
 count</code></pre>
<pre><code># A tibble: 10 × 3
# Groups:   cugstatus, mrc [10]
   cugstatus    mrc         n
   &lt;chr&gt;        &lt;chr&gt;   &lt;int&gt;
 1 CGi_peak     EAR        91
 2 CGi_peak     ESR       237
 3 CGi_peak     LR        431
 4 CGi_peak     NR       1375
 5 CGi_peak     not_mrc    55
 6 not_CGi_peak EAR      7291
 7 not_CGi_peak ESR     15126
 8 not_CGi_peak LR      40286
 9 not_CGi_peak NR      82863
10 not_CGi_peak not_mrc  3170</code></pre>
<pre class="r"><code>CUG_mrc_status_list %&gt;% 
  mutate(mrc=factor(mrc, levels = c(&quot;NR&quot;, &quot;EAR&quot;, &quot;ESR&quot;,&quot;LR&quot;,&quot;not_mrc&quot;))) %&gt;%
  group_by(cugstatus, mrc) %&gt;% 
  ggplot(., aes(x = mrc,  fill = cugstatus)) +
  geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
  theme_bw()+
  ggtitle(&quot;CG islands by mrc&quot;, subtitle=paste(length(CUG_mrc_status_list$peakid )))</code></pre>
<p><img src="figure/TE_analysis.Rmd/cgisland%20load-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cgisland-load-1">
Past versions of "cgisland load-1.png"
</button>
</p>
<div id="fig-no-spaces-cgisland-load-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/cgisland load-1.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>CUG_mrc_status_list %&gt;% 
  mutate(mrc=&quot;full_list&quot;) %&gt;% 
  rbind(., CUG_mrc_status_list) %&gt;% 
   mutate(mrc=factor(mrc, levels=c(&quot;EAR&quot;,&quot;ESR&quot;,&quot;LR&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;% 
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
  group_by(cugstatus, mrc) %&gt;% 
  ggplot(., aes(x = mrc,  fill = cugstatus)) +
  geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
  theme_bw()+
  ggtitle(&quot;CG islands by mrc&quot;, subtitle=paste(length(CUG_mrc_status_list$peakid )))</code></pre>
<p><img src="figure/TE_analysis.Rmd/cgisland%20load-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cgisland-load-2">
Past versions of "cgisland load-2.png"
</button>
</p>
<div id="fig-no-spaces-cgisland-load-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/cgisland load-2.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>Col_fullDF_cug_overlap %&gt;% 
        as.data.frame %&gt;% 
        ggplot(., aes (x = width))+
        geom_density(color=&quot;darkblue&quot;,fill=&quot;lightblue&quot;,aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of CGisland overlap widths&quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/cgisland%20load-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cgisland-load-3">
Past versions of "cgisland load-3.png"
</button>
</p>
<div id="fig-no-spaces-cgisland-load-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/cgisland load-3.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/cgisland load-3.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>cpg_island_gr %&gt;% 
  as.data.frame() %&gt;% 
  ggplot(., aes (x = cpg_width))+
        geom_density(aes(alpha = 0.5))+
  theme_classic()+
  ggtitle(&quot;Distribution of overlap widths of CGislands &quot;,subtitle = &quot; limited x axis&quot;)+
  coord_cartesian(xlim= c(0,2500))</code></pre>
<p><img src="figure/TE_analysis.Rmd/cgisland%20load-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cgisland-load-4">
Past versions of "cgisland load-4.png"
</button>
</p>
<div id="fig-no-spaces-cgisland-load-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/80ffd81601eac90f287a6b6d6ee3c2bdaae19214/docs/figure/TE_analysis.Rmd/cgisland load-4.png" target="_blank">80ffd81</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/cgisland load-4.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>CUG_mrc_status_list %&gt;% 
   mutate(mrc=&quot;full_list&quot;) %&gt;% 
  rbind(., CUG_mrc_status_list) %&gt;% 
   mutate(mrc=factor(mrc, levels=c(&quot;EAR&quot;,&quot;ESR&quot;,&quot;LR&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;% 
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
  dplyr::filter(cugstatus==&quot;CGi_peak&quot;) %&gt;% 
  ggplot(., aes(x = mrc,  fill = prom_status)) +
  geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
  theme_bw()+
  ggtitle(&quot;CG islands promotors vs non-promotor CpGi by mrc&quot;, subtitle=paste(length(CUG_mrc_status_list$peakid )))</code></pre>
<p><img src="figure/TE_analysis.Rmd/cgisland%20load-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-cgisland-load-5">
Past versions of "cgisland load-5.png"
</button>
</p>
<div id="fig-no-spaces-cgisland-load-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/reneeisnowhere/ATAC_learning/blob/b5b4a54f223c5e158d645d1c2fb33a3054fb3e80/docs/figure/TE_analysis.Rmd/cgisland load-5.png" target="_blank">b5b4a54</a>
</td>
<td>
reneeisnowhere
</td>
<td>
2024-08-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>CUG_mrc_nine_list %&gt;% 
   mutate(mrc=&quot;full_list&quot;) %&gt;% 
  rbind(., CUG_mrc_nine_list) %&gt;% 
   mutate(mrc=factor(mrc, levels=c(&quot;EAR_hyper&quot;,&quot;EAR_hypo&quot;,&quot;ESR_A&quot;,&quot;ESR_B&quot;,&quot;ESR_C&quot;,&quot;ESR_D&quot;,&quot;LR_hyper&quot;,&quot;LR_hypo&quot;,&quot;NR&quot;,&quot;not-mrc&quot;,&quot;full_list&quot;))) %&gt;% 
  dplyr::filter(mrc !=&quot;not_mrc&quot;) %&gt;% 
  dplyr::filter(cugstatus==&quot;CGi_peak&quot;) %&gt;% 
  ggplot(., aes(x = mrc,  fill = prom_status)) +
  geom_bar(position=&quot;fill&quot;,color = &quot;black&quot;)+
  theme_bw()+
  ggtitle(&quot;CG islands promotors vs non-promotor CpGi by mrc&quot;, subtitle=paste(length(CUG_mrc_status_list$peakid )))</code></pre>
<p><img src="figure/TE_analysis.Rmd/cgisland%20load-6.png" width="672" style="display: block; margin: auto;" /></p>
<div id="this-code-is-for-chi-analysis" class="section level5">
<h5>This code is for Chi analysis</h5>
<pre class="r"><code># ##complete counts()

chitest_LRvNRTE_nf &lt;- matrix(c(TE_counts_nofilt$n[5],TE_counts_nofilt$n[6],TE_counts_nofilt$n[7],TE_counts_nofilt$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_LRvNRTE_nf)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_LRvNRTE_nf
X-squared = 33.703, df = 1, p-value = 6.422e-09</code></pre>
<pre class="r"><code>chitest_EARvNRTE_nf &lt;- matrix(c(TE_counts_nofilt$n[1],TE_counts_nofilt$n[2],TE_counts_nofilt$n[7],TE_counts_nofilt$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_EARvNRTE_nf)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_EARvNRTE_nf
X-squared = 5.4867, df = 1, p-value = 0.01916</code></pre>
<pre class="r"><code>chitest_ESRvNRTE_nf &lt;- matrix(c(TE_counts_nofilt$n[3],TE_counts_nofilt$n[4],TE_counts_nofilt$n[7],TE_counts_nofilt$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_ESRvNRTE_nf)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_ESRvNRTE_nf
X-squared = 47.083, df = 1, p-value = 6.803e-12</code></pre>
<pre class="r"><code>### just subsets


chitest_LRvNRTE_nc &lt;- matrix(c(LiSiLTDNRe_TE_no_cut$n[5],LiSiLTDNRe_TE_no_cut$n[6],LiSiLTDNRe_TE_no_cut$n[7],LiSiLTDNRe_TE_no_cut$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_LRvNRTE_nc)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_LRvNRTE_nc
X-squared = 236.84, df = 1, p-value &lt; 2.2e-16</code></pre>
<pre class="r"><code>chitest_EARvNRTE_nc &lt;- matrix(c(LiSiLTDNRe_TE_no_cut$n[1],LiSiLTDNRe_TE_no_cut$n[2],LiSiLTDNRe_TE_no_cut$n[7],LiSiLTDNRe_TE_no_cut$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_EARvNRTE_nc)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_EARvNRTE_nc
X-squared = 0.95495, df = 1, p-value = 0.3285</code></pre>
<pre class="r"><code>chitest_ESRvNRTE_nc &lt;- matrix(c(LiSiLTDNRe_TE_no_cut$n[3],LiSiLTDNRe_TE_no_cut$n[4],LiSiLTDNRe_TE_no_cut$n[7],LiSiLTDNRe_TE_no_cut$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_ESRvNRTE_nc)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_ESRvNRTE_nc
X-squared = 16.366, df = 1, p-value = 5.222e-05</code></pre>
<pre class="r"><code>##### JUst using the .5 overlap cutoff

chitest_LRvNRTE &lt;- matrix(c(LiSiLTDNRe_TE_df$n[5],LiSiLTDNRe_TE_df$n[6],LiSiLTDNRe_TE_df$n[7],LiSiLTDNRe_TE_df$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_LRvNRTE)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_LRvNRTE
X-squared = 266.08, df = 1, p-value &lt; 2.2e-16</code></pre>
<pre class="r"><code>chitest_EARvNRTE &lt;- matrix(c(LiSiLTDNRe_TE_df$n[1],LiSiLTDNRe_TE_df$n[2],LiSiLTDNRe_TE_df$n[7],LiSiLTDNRe_TE_df$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_EARvNRTE)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_EARvNRTE
X-squared = 0.88637, df = 1, p-value = 0.3465</code></pre>
<pre class="r"><code>chitest_ESRvNRTE &lt;- matrix(c(LiSiLTDNRe_TE_df$n[3],LiSiLTDNRe_TE_df$n[4],LiSiLTDNRe_TE_df$n[7],LiSiLTDNRe_TE_df$n[8]), ncol=2, byrow=TRUE)
chisq.test(chitest_ESRvNRTE)</code></pre>
<pre><code>
    Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  chitest_ESRvNRTE
X-squared = 12.067, df = 1, p-value = 0.0005132</code></pre>
</div>
</div>
<div id="snp" class="section level3">
<h3>SNP</h3>
<pre class="r"><code>gwas_ACtox &lt;- readRDS(&quot;data/gwas_1_dataframe.RDS&quot;)  
gwas_ARR &lt;- readRDS(&quot;data/gwas_2_dataframe.RDS&quot;)
gwas_ACresp &lt;- readRDS(&quot;data/gwas_3_dataframe.RDS&quot;)
gwas_HD &lt;- readRDS(&quot;data/gwas_4_dataframe.RDS&quot;)
gwas_HF &lt;- readRDS(&quot;data/gwas_5_dataframe.RDS&quot;)
gwas_CAD &lt;- readRDS( &quot;data/CAD_gwas_dataframe.RDS&quot;)
gwas_MI &lt;- readRDS(&quot;data/MI_gwas.RDS&quot;)

gwas_snp_list &lt;- gwas_ACtox %&gt;%
  distinct(SNPS,.keep_all = TRUE) %&gt;% 
  dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
  mutate(gwas=&quot;ACtox&quot;) %&gt;% 
  rbind(gwas_ARR %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ARR&quot;)) %&gt;% 
  rbind(gwas_ACresp %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;ACresp&quot;)) %&gt;% 
  rbind(gwas_HD %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HD&quot;)) %&gt;% 
  rbind(gwas_HF %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;HF&quot;)) %&gt;% 
  rbind(gwas_CAD %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;CAD&quot;)) %&gt;% 
  rbind(gwas_MI %&gt;% 
          distinct(SNPS,.keep_all = TRUE) %&gt;%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %&gt;% 
          mutate(gwas=&quot;MI&quot;))

gwas_snp_gr &lt;- gwas_snp_list %&gt;% 
   mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %&gt;% 
  na.omit() %&gt;% 
   mutate(start=CHR_POS, end=CHR_POS, chr=paste0(&quot;chr&quot;,CHR_ID)) %&gt;% 
  GRanges()

findOverlaps(gwas_snp_gr, all_TEs_gr)</code></pre>
<pre><code>Hits object with 3413 hits and 0 metadata columns:
         queryHits subjectHits
         &lt;integer&gt;   &lt;integer&gt;
     [1]         1     4910496
     [2]         4     1047158
     [3]         7     4718029
     [4]        11      116853
     [5]        12     4826568
     ...       ...         ...
  [3409]      7750     5009192
  [3410]      7751     3904058
  [3411]      7753     4735830
  [3412]      7755     4615754
  [3413]      7758     1872218
  -------
  queryLength: 7762 / subjectLength: 5683690</code></pre>
<pre class="r"><code>test &lt;- join_overlap_intersect(gwas_snp_gr, all_TEs_gr) %&gt;% GRanges
##3413

test_new &lt;-  test %&gt;% 
   as.data.frame %&gt;% 
   dplyr::select (seqnames:gwas,repName:repFamily) %&gt;% 
   GRanges()
peaks &lt;-
  Collapsed_peaks %&gt;% 
    dplyr::select(seqnames:peakid) %&gt;% 
    GRanges()
peak_test &lt;- join_overlap_intersect(test_new, peaks)

# peak_test

Col_fullDF_overlap %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(seqnames:peakid,repName:repFamily) %&gt;% 
  GRanges() %&gt;% 
  join_overlap_intersect(.,gwas_snp_gr)</code></pre>
<pre><code>GRanges object with 135 ranges and 8 metadata columns:
        seqnames    ranges strand |                 peakid     repName
           &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; |            &lt;character&gt; &lt;character&gt;
    [1]     chr1  16012818      * | chr1.16012567.16013729        MIRb
    [2]     chr1 113630788      * | chr1.113630172.11363..       L1MA3
    [3]     chr1 113630788      * | chr1.113630172.11363..       L1MA3
    [4]     chr1 113630788      * | chr1.113630172.11363..       L1MA3
    [5]     chr1 170224718      * | chr1.170224576.17022..       L1MA7
    ...      ...       ...    ... .                    ...         ...
  [131]    chr21  29162981      * | chr21.29162621.29163..      AluSz6
  [132]    chr21  29162981      * | chr21.29162621.29163..      AluSz6
  [133]    chr21  34746814      * | chr21.34746498.34746..        MIRc
  [134]    chr21  34746814      * | chr21.34746498.34746..        MIRc
  [135]    chr22  38436107      * | chr22.38435884.38436..         MIR
           repClass   repFamily    CHR_ID   CHR_POS        SNPS        gwas
        &lt;character&gt; &lt;character&gt; &lt;numeric&gt; &lt;numeric&gt; &lt;character&gt; &lt;character&gt;
    [1]        SINE         MIR         1  16012818  rs10927886          HD
    [2]        LINE          L1         1 113630788   rs1230666          HD
    [3]        LINE          L1         1 113630788   rs1230666         CAD
    [4]        LINE          L1         1 113630788   rs1230666          MI
    [5]        LINE          L1         1 170224718  rs12122060         ARR
    ...         ...         ...       ...       ...         ...         ...
  [131]        SINE         Alu        21  29162981  rs73193808          HD
  [132]        SINE         Alu        21  29162981  rs73193808         CAD
  [133]        SINE         MIR        21  34746814   rs2834618         ARR
  [134]        SINE         MIR        21  34746814   rs2834618          HD
  [135]        SINE         MIR        22  38436107   rs2267386          HD
  -------
  seqinfo: 22 sequences from an unspecified genome; no seqlengths</code></pre>
<pre class="r"><code>test2 &lt;- join_overlap_intersect(peaks, gwas_snp_gr) 
  
test2 %&gt;% 
   join_overlap_intersect(., all_TEs_gr) %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(seqnames:gwas,repName:repFamily) %&gt;% 
  mutate(mrc=if_else(peakid %in% EAR_hyper$id, &quot;EAR_hyper&quot;,
                      if_else(peakid %in% EAR_hypo$id, &quot;EAR_hypo&quot;,
                              if_else(peakid %in% ESR_A$id,&quot;ESR_A&quot;,
                                      if_else(peakid %in% ESR_B$id,&quot;ESR_B&quot;,
                                              if_else(peakid %in% ESR_C$id,&quot;ESR_C&quot;,
                                                      if_else(peakid %in% ESR_D$id,&quot;ESR_D&quot;,
                                                              if_else(peakid %in% LR_hyper$id,&quot;LR_hyper&quot;,
                                                                      if_else(peakid %in% LR_hypo$id,&quot;LR_hypo&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;)))))))))) %&gt;% 
  # distinct(peakid, .keep_all = TRUE) %&gt;% 
  group_by(gwas,mrc) %&gt;% 
  tally</code></pre>
<pre><code># A tibble: 24 × 3
# Groups:   gwas [6]
   gwas   mrc           n
   &lt;chr&gt;  &lt;chr&gt;     &lt;int&gt;
 1 ACresp LR_hyper      2
 2 ARR    ESR_B         2
 3 ARR    LR_hyper      1
 4 ARR    LR_hypo       4
 5 ARR    NR           12
 6 ARR    not_mrc       1
 7 CAD    EAR_hyper     1
 8 CAD    EAR_hypo      1
 9 CAD    ESR_B         4
10 CAD    LR_hyper      7
# ℹ 14 more rows</code></pre>
<pre class="r"><code>SNP_df &lt;- test2 %&gt;% 
   join_overlap_intersect(., all_TEs_gr) %&gt;% 
  as.data.frame %&gt;% 
  dplyr::select(seqnames:gwas,repName:repFamily) %&gt;% 
  mutate(mrc=if_else(peakid %in% EAR_hyper$id, &quot;EAR_hyper&quot;,
                      if_else(peakid %in% EAR_hypo$id, &quot;EAR_hypo&quot;,
                              if_else(peakid %in% ESR_A$id,&quot;ESR_A&quot;,
                                      if_else(peakid %in% ESR_B$id,&quot;ESR_B&quot;,
                                              if_else(peakid %in% ESR_C$id,&quot;ESR_C&quot;,
                                                      if_else(peakid %in% ESR_D$id,&quot;ESR_D&quot;,
                                                              if_else(peakid %in% LR_hyper$id,&quot;LR_hyper&quot;,
                                                                      if_else(peakid %in% LR_hypo$id,&quot;LR_hypo&quot;,
                                     if_else(peakid %in% NR_df$id,&quot;NR&quot;,&quot;not_mrc&quot;))))))))))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ggrepel_0.9.5                           
 [2] plyranges_1.24.0                        
 [3] ggsignif_0.6.4                          
 [4] genomation_1.36.0                       
 [5] smplot2_0.2.4                           
 [6] eulerr_7.0.2                            
 [7] biomaRt_2.60.1                          
 [8] devtools_2.4.5                          
 [9] usethis_3.0.0                           
[10] ggpubr_0.6.0                            
[11] BiocParallel_1.38.0                     
[12] scales_1.3.0                            
[13] VennDiagram_1.7.3                       
[14] futile.logger_1.4.3                     
[15] gridExtra_2.3                           
[16] ggfortify_0.4.17                        
[17] edgeR_4.2.1                             
[18] limma_3.60.4                            
[19] rtracklayer_1.64.0                      
[20] org.Hs.eg.db_3.19.1                     
[21] TxDb.Hsapiens.UCSC.hg38.knownGene_3.18.0
[22] GenomicFeatures_1.56.0                  
[23] AnnotationDbi_1.66.0                    
[24] Biobase_2.64.0                          
[25] GenomicRanges_1.56.1                    
[26] GenomeInfoDb_1.40.1                     
[27] IRanges_2.38.1                          
[28] S4Vectors_0.42.1                        
[29] BiocGenerics_0.50.0                     
[30] ChIPseeker_1.40.0                       
[31] RColorBrewer_1.1-3                      
[32] broom_1.0.6                             
[33] kableExtra_1.4.0                        
[34] lubridate_1.9.3                         
[35] forcats_1.0.0                           
[36] stringr_1.5.1                           
[37] dplyr_1.1.4                             
[38] purrr_1.0.2                             
[39] readr_2.1.5                             
[40] tidyr_1.3.1                             
[41] tibble_3.2.1                            
[42] ggplot2_3.5.1                           
[43] tidyverse_2.0.0                         
[44] workflowr_1.7.1                         

loaded via a namespace (and not attached):
  [1] fs_1.6.4                               
  [2] matrixStats_1.3.0                      
  [3] bitops_1.0-8                           
  [4] enrichplot_1.24.2                      
  [5] HDO.db_0.99.1                          
  [6] httr_1.4.7                             
  [7] profvis_0.3.8                          
  [8] tools_4.4.1                            
  [9] backports_1.5.0                        
 [10] utf8_1.2.4                             
 [11] R6_2.5.1                               
 [12] lazyeval_0.2.2                         
 [13] urlchecker_1.0.1                       
 [14] withr_3.0.1                            
 [15] prettyunits_1.2.0                      
 [16] cli_3.6.3                              
 [17] formatR_1.14                           
 [18] scatterpie_0.2.3                       
 [19] labeling_0.4.3                         
 [20] sass_0.4.9                             
 [21] Rsamtools_2.20.0                       
 [22] systemfonts_1.1.0                      
 [23] yulab.utils_0.1.5                      
 [24] foreign_0.8-87                         
 [25] DOSE_3.30.2                            
 [26] svglite_2.1.3                          
 [27] sessioninfo_1.2.2                      
 [28] plotrix_3.8-4                          
 [29] BSgenome_1.72.0                        
 [30] pwr_1.3-0                              
 [31] impute_1.78.0                          
 [32] rstudioapi_0.16.0                      
 [33] RSQLite_2.3.7                          
 [34] generics_0.1.3                         
 [35] gridGraphics_0.5-1                     
 [36] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2
 [37] BiocIO_1.14.0                          
 [38] vroom_1.6.5                            
 [39] gtools_3.9.5                           
 [40] car_3.1-2                              
 [41] GO.db_3.19.1                           
 [42] Matrix_1.7-0                           
 [43] fansi_1.0.6                            
 [44] abind_1.4-5                            
 [45] lifecycle_1.0.4                        
 [46] whisker_0.4.1                          
 [47] yaml_2.3.10                            
 [48] carData_3.0-5                          
 [49] SummarizedExperiment_1.34.0            
 [50] BiocFileCache_2.12.0                   
 [51] gplots_3.1.3.1                         
 [52] qvalue_2.36.0                          
 [53] SparseArray_1.4.8                      
 [54] blob_1.2.4                             
 [55] promises_1.3.0                         
 [56] crayon_1.5.3                           
 [57] miniUI_0.1.1.1                         
 [58] lattice_0.22-6                         
 [59] cowplot_1.1.3                          
 [60] KEGGREST_1.44.1                        
 [61] pillar_1.9.0                           
 [62] knitr_1.48                             
 [63] fgsea_1.30.0                           
 [64] rjson_0.2.21                           
 [65] boot_1.3-30                            
 [66] codetools_0.2-20                       
 [67] fastmatch_1.1-4                        
 [68] glue_1.7.0                             
 [69] getPass_0.2-4                          
 [70] ggfun_0.1.5                            
 [71] data.table_1.15.4                      
 [72] remotes_2.5.0                          
 [73] vctrs_0.6.5                            
 [74] png_0.1-8                              
 [75] treeio_1.28.0                          
 [76] gtable_0.3.5                           
 [77] cachem_1.1.0                           
 [78] xfun_0.46                              
 [79] S4Arrays_1.4.1                         
 [80] mime_0.12                              
 [81] tidygraph_1.3.1                        
 [82] statmod_1.5.0                          
 [83] ellipsis_0.3.2                         
 [84] nlme_3.1-165                           
 [85] ggtree_3.12.0                          
 [86] bit64_4.0.5                            
 [87] filelock_1.0.3                         
 [88] progress_1.2.3                         
 [89] rprojroot_2.0.4                        
 [90] bslib_0.8.0                            
 [91] rpart_4.1.23                           
 [92] KernSmooth_2.23-24                     
 [93] Hmisc_5.1-3                            
 [94] colorspace_2.1-1                       
 [95] DBI_1.2.3                              
 [96] seqPattern_1.36.0                      
 [97] nnet_7.3-19                            
 [98] tidyselect_1.2.1                       
 [99] processx_3.8.4                         
[100] bit_4.0.5                              
[101] compiler_4.4.1                         
[102] curl_5.2.1                             
[103] git2r_0.33.0                           
[104] httr2_1.0.2                            
[105] htmlTable_2.4.3                        
[106] xml2_1.3.6                             
[107] DelayedArray_0.30.1                    
[108] shadowtext_0.1.4                       
[109] checkmate_2.3.2                        
[110] caTools_1.18.2                         
[111] callr_3.7.6                            
[112] rappdirs_0.3.3                         
[113] digest_0.6.36                          
[114] rmarkdown_2.27                         
[115] XVector_0.44.0                         
[116] base64enc_0.1-3                        
[117] htmltools_0.5.8.1                      
[118] pkgconfig_2.0.3                        
[119] MatrixGenerics_1.16.0                  
[120] highr_0.11                             
[121] dbplyr_2.5.0                           
[122] fastmap_1.2.0                          
[123] rlang_1.1.4                            
[124] htmlwidgets_1.6.4                      
[125] UCSC.utils_1.0.0                       
[126] shiny_1.9.1                            
[127] farver_2.1.2                           
[128] jquerylib_0.1.4                        
[129] zoo_1.8-12                             
[130] jsonlite_1.8.8                         
[131] GOSemSim_2.30.0                        
[132] RCurl_1.98-1.16                        
[133] magrittr_2.0.3                         
[134] Formula_1.2-5                          
[135] GenomeInfoDbData_1.2.12                
[136] ggplotify_0.1.2                        
[137] patchwork_1.2.0                        
[138] munsell_0.5.1                          
[139] Rcpp_1.0.13                            
[140] ape_5.8                                
[141] viridis_0.6.5                          
[142] stringi_1.8.4                          
[143] ggraph_2.2.1                           
[144] zlibbioc_1.50.0                        
[145] MASS_7.3-61                            
[146] plyr_1.8.9                             
[147] pkgbuild_1.4.4                         
[148] parallel_4.4.1                         
[149] Biostrings_2.72.1                      
[150] graphlayouts_1.1.1                     
[151] splines_4.4.1                          
[152] hms_1.1.3                              
[153] locfit_1.5-9.10                        
[154] ps_1.7.7                               
[155] igraph_2.0.3                           
[156] reshape2_1.4.4                         
[157] pkgload_1.4.0                          
[158] futile.options_1.0.1                   
[159] XML_3.99-0.17                          
[160] evaluate_0.24.0                        
[161] lambda.r_1.2.4                         
[162] tzdb_0.4.0                             
[163] tweenr_2.0.3                           
[164] httpuv_1.6.15                          
[165] polyclip_1.10-7                        
[166] gridBase_0.4-7                         
[167] ggforce_0.4.2                          
[168] xtable_1.8-4                           
[169] restfulr_0.0.15                        
[170] tidytree_0.4.6                         
[171] rstatix_0.7.2                          
[172] later_1.3.2                            
[173] viridisLite_0.4.2                      
[174] aplot_0.2.3                            
[175] memoise_2.0.1                          
[176] GenomicAlignments_1.40.0               
[177] cluster_2.1.6                          
[178] timechange_0.3.0                       </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
