---
title: "CorMotif_data"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)

library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github('davetang/bedr')
library(bedr)
```

```{r echo=TRUE, file='code/corMotifcustom.R'}

```

```{r without4 and 5 setting up data}

high_conf_peak_counts <- read.csv("data/high_conf_peak_counts.csv", row.names = 1)


high_conf_peak_counts_n45 <- high_conf_peak_counts %>% 
  column_to_rownames("Geneid") %>% 
  dplyr::select(Ind1_75DA24h:Ind3_77V3h,Ind6_71DA24h:Ind6_71V3h)

groupmat_names_n45 <- data.frame(timeset = colnames(high_conf_peak_counts_n45))
df_names_n45 <-groupmat_names_n45 %>% 
  mutate(timeset=gsub("75","1_",timeset)) %>% 
  mutate(timeset=gsub("87","2_",timeset)) %>% 
  mutate(timeset=gsub("77","3_",timeset)) %>% 
  mutate(timeset=gsub("79","4_",timeset)) %>% 
  mutate(timeset=gsub("78","5_",timeset)) %>%
  mutate(timeset=gsub("71","6_",timeset)) %>% 
  mutate(timeset = gsub("24h","_24h",timeset), 
       timeset = gsub("3h","_3h",timeset)) %>%
  separate(timeset, into = c(NA,"indv","trt","time"), sep= "_") %>% 
  
  mutate(trt= case_match(trt, 'DX' ~'DOX', 'E'~'EPI', 'DA'~'DNR', 'M'~'MTX', 'T'~'TRZ', 'V'~'VEH',.default = trt)) %>% 
  dplyr::filter(indv %in% c(1,2,3,6))
 
group_n45 <- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group_n45 <- factor(group_n45, levels =c("1","2","3","4","5","6","7","8","9","10","11","12"))                        
indv_n45 <- df_names_n45$indv
# indv <- factor(indv, levels = c(1,2,3,4,5,6))
time_n45 <- df_names_n45$time
# time <- factor(time, levels =c("3h","24"))
trt_n45 <- df_names_n45$trt
label_n45 <- paste0(indv_n45,"_",trt_n45,"_",time_n45)
group_fac_n45 <- group_n45
groupid_n45 <- as.numeric(group_fac_n45)

compid_n45 <- data.frame(c1= c(2,4,6,8,10,1,3,5,7,9), c2 = c( 12,12,12,12,12,11,11,11,11,11))

y_TMM_cpm_n45 <- cpm(high_conf_peak_counts_n45, log = TRUE)

colnames(y_TMM_cpm_n45) <- label_n45
# y_TMM_cpm
set.seed(31415)
# cormotif_initial_n45 <- cormotiffit(exprs = y_TMM_cpm_n45, groupid = groupid_n45, compid = compid_n45, K=1:6, max.iter = 500, runtype = "logCPM")

# saveRDS(cormotif_initial_n45,"data/cormotif_n45_4_run.RDS")

cormotif_initial_n45 <- readRDS("data/cormotif_full_4_run.RDS")
gene_prob_tran_n45 <- cormotif_initial_n45$bestmotif$p.post
rownames(gene_prob_tran_n45) <- rownames(high_conf_peak_counts_n45)
motif_prob_n45 <- cormotif_initial_n45$bestmotif$clustlike
rownames(motif_prob_n45) <- rownames(high_conf_peak_counts_n45)
# write.csv(motif_prob_n45,"data/cormotif_probability_45_list.csv")

Cormotif::plotIC(cormotif_initial_n45)
Cormotif::plotMotif(cormotif_initial_n45)



myColors <-  rev(c("#FFFFFF", "#E6E6E6" ,"#CCCCCC", "#B3B3B3", "#999999", "#808080", "#666666","#4C4C4C", "#333333", "#191919","#000000"))
plot.new()
legend('bottomleft',fill=myColors, legend =rev(c("0", "0.1", "0.2", "0.3", "0.4",  "0.5", "0.6", "0.7", "0.8","0.9", "1")), box.col="white",title = "Probability\nlegend", horiz=FALSE,title.cex=.8)
```

Study breakdown: 1 = DNR_3, 2 = DOX_3, 3 = EPI_3, 4 = MTX_3, 5 = TRZ_3\
6 = DNR_24, 7 = DOX_24, 8 = EPI_24, 9 = MTX_24, 10 = TRZ_24

```{r he peaks n45}

high_conf_peak_counts <- read.csv("data/high_conf_peak_counts.csv", row.names = 1)


high_conf_peak_counts_n45 <- high_conf_peak_counts %>% 
  column_to_rownames("Geneid") %>% 
  dplyr::select(Ind1_75DA24h:Ind3_77V3h,Ind6_71DA24h:Ind6_71V3h)

lcpm_n45 <- cpm(high_conf_peak_counts_n45, log=TRUE)  ### for determining the basic cutoffs

row_means_n45 <- rowMeans(lcpm_n45)
my_hc_filtered_counts_he_n45 <- high_conf_peak_counts_n45[row_means_n45 > 0,]


groupmat_names_n45 <- data.frame(timeset = colnames(my_hc_filtered_counts_he_n45))
df_names_n45 <-groupmat_names_n45 %>% 
  mutate(timeset=gsub("75","1_",timeset)) %>% 
  mutate(timeset=gsub("87","2_",timeset)) %>% 
  mutate(timeset=gsub("77","3_",timeset)) %>% 
  mutate(timeset=gsub("79","4_",timeset)) %>% 
  mutate(timeset=gsub("78","5_",timeset)) %>%
  mutate(timeset=gsub("71","6_",timeset)) %>% 
  mutate(timeset = gsub("24h","_24h",timeset), 
       timeset = gsub("3h","_3h",timeset)) %>%
  separate(timeset, into = c(NA,"indv","trt","time"), sep= "_") %>% 
  mutate(trt= case_match(trt, 'DX' ~'DOX', 'E'~'EPI', 'DA'~'DNR', 'M'~'MTX', 'T'~'TRZ', 'V'~'VEH',.default = trt)) %>% 
  dplyr::filter(indv %in% c(1,2,3,6))
# group_fac <- group
# groupid <- as.nCumeric(group_fac)
indv_n45 <- df_names_n45$indv
time_n45 <- df_names_n45$time
# time <- factor(time, levels =c("3h","24"))
trt_n45 <- df_names_n45$trt
label_n45 <- paste0(indv_n45,"_",trt_n45,"_",time_n45)

compid_n45 <- data.frame(c1= c(2,4,6,8,10,1,3,5,7,9), c2 = c( 12,12,12,12,12,11,11,11,11,11))

y_TMM_cpm_n45_he <- cpm(my_hc_filtered_counts_he_n45, log = TRUE)

# y_TMM_cpm_n45_he <- cpm(high_conf_peak_counts_he_n45, log = TRUE)
rownames(y_TMM_cpm_n45_he) <- rownames(my_hc_filtered_counts_he_n45)
colnames(y_TMM_cpm_n45_he) <- label_n45
# y_TMM_cpm
# set.seed(31415)
# cormotif_initial_n45_he <- cormotiffit(exprs = y_TMM_cpm_n45_he, groupid = groupid_n45, compid = compid_n45, K=1:6, max.iter = 500, runtype = "logCPM")

# saveRDS(cormotif_initial_n45_he,"data/cormotif_full_4_run_he.RDS")

cormotif_initial_n45_he <- readRDS("data/cormotif_full_4_run_he.RDS")
gene_prob_tran_n45_he <- cormotif_initial_n45_he$bestmotif$p.post
rownames(gene_prob_tran_n45_he) <- rownames(my_hc_filtered_counts_he_n45)
motif_prob_n45_he <- cormotif_initial_n45_he$bestmotif$clustlike
rownames(motif_prob_n45_he) <- rownames(my_hc_filtered_counts_he_n45)
# write.csv(motif_prob_n45_he,"data/cormotif_probability_45_list_he.csv")

Cormotif::plotIC(cormotif_initial_n45_he)
Cormotif::plotMotif(cormotif_initial_n45_he)


```

Doing the analysis on the filtered set, just like the full set.

```{r pulling out the groups}

motif_prob_n45 <- read.csv("data/cormotif_probability_45_list_he.csv")
motif_prob_n45 <- motif_prob_n45 %>% 
  column_to_rownames('X')
clust1_n45 <- motif_prob_n45 %>%
  as.data.frame() %>%
  filter(V1>0.5) %>% 
  rownames
clust2_n45 <- motif_prob_n45 %>%
  as.data.frame() %>%
  filter(V2>0.5) %>% 
  rownames
clust3_n45 <- motif_prob_n45 %>%
  as.data.frame() %>%
  filter(V3>0.5) %>% 
  rownames
clust4_n45 <- motif_prob_n45 %>%
  as.data.frame() %>%
  filter(V4>0.5) %>% 
  rownames
# backGL_n45_he <- row.names(motif_prob_n45)
# saveRDS(backGL_n45_he,"data/background_n45_he_peaks.RDS")

# unclassified_n45 <- setdiff(backGL_n45_he,union(clust1_n45,union(clust2_n45,union(clust4_n45,clust3_n45))))
# 
# motif_list_n45 <- list("NR_n45"=clust1_n45,"LR_n45"=clust2_n45,"EAR_n45"=clust3_n45,"ESR_n45"=clust4_n45,"unclassified_n45"=unclassified_n45, "backGL_n45_he"=backGL_n45_he)
# saveRDS(motif_list_n45,"data/motif_list_n45.RDS")
# saveRDS(unclassified_n45,"data/unclassified_n45_set_peaks.RDS")

```

Seems to be grouping like RNA-seq. I am using the same nomenclature. I filter each column of the likelihood of belonging to the cluster by \> 0.5- This causes some number discrepancies in overall.

-   \- Clust1 has `r length(clust1_n45)` regions and is the **No Response** set.

-   \- Clust2 has `r length(clust2_n45)` regions and is the **Late-Response** set.

-   \- Clust3 has `r length(clust3_n45)` regions and is the **Early-Acute Response** set.

-   \- Clust4 has `r length(clust4_n45)` regions and is the **Early-Sustained Response** set.

These contain `r length(clust1_n45)+length(clust2_n45)+length(clust3_n45) + length(clust4_n45)` total peaks out of `r length(my_hc_filtered_counts_he_n45[[1]])` for approximately `r (length(clust1_n45)+length(clust2_n45)+length(clust3_n45)+length(clust4_n45))/length(my_hc_filtered_counts_he_n45[[1]]) *100` percent of all peaks accounted for.

```{r assessment of each cluster}

txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene

# loadFile_peakCall <- function(){
#  file <- choose.files()
#  file <- readPeakFile(file, header = FALSE)
#  return(file)
# }
# 
# prepGRangeObj <- function(seek_object){
#  seek_object$Peaks = seek_object$V4
#  seek_object$level = seek_object$V5
#  seek_object$V4 = seek_object$V5 = NULL
#  return(seek_object)
# }
motif_list_n45 <- readRDS("data/motif_list_n45.RDS")
list2env(motif_list_n45, envir= .GlobalEnv)

TSS = getBioRegion(TxDb=txdb, upstream=2000, downstream=2000, by = "gene", 
                   type = "start_site")
##converting lists to dataframes as bed files
# # EAR %>%
# data.frame("EAR_n45" = EAR_n45) %>%
#   mutate(name =EAR_n45) %>%
# separate(EAR_n45, into = c( "chr","start","end")) %>%
#   # mutate(strand=".") %>%
#   mutate(start=as.integer(start), end=as.integer(end)) %>%
#   dplyr::select(chr, start,end,name) %>%
#   na.omit() %>%
#   # unite(.,col="start-end",start, end,sep = "-") %>%
#   write.table(.,file = "data/n45_bedfiles/EAR_n45.bed", row.names =FALSE,col.name=FALSE,append = TRUE, quote = FALSE)
# 
#   #
#   data.frame("unclassified_n45" = unclassified_n45) %>%
#     mutate(name =unclassified_n45) %>%
#   separate(unclassified_n45, into = c( "chr","start","end")) %>%
#     # mutate(strand=".") %>%
#     mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit() %>%
#     # unite(.,col="start-end",start, end,sep = "-") %>%
#     write.table(.,file = "data/n45_bedfiles/unclassified_n45.bed", row.names =FALSE,col.name=FALSE,append = TRUE, quote = FALSE)
# 
#   #
#   data.frame("background_n45" = backGL_n45_he) %>%
#     mutate(name =background_n45) %>%
#   separate(background_n45, into = c( "chr","start","end")) %>%
#     # mutate(strand=".") %>%
#     mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit() %>%
#     # unite(.,col="start-end",start, end,sep = "-") %>%
#     write.table(.,file = "data/n45_bedfiles/background_n45.bed",row.names =FALSE,col.name=FALSE,append = TRUE, quote = FALSE)
# 
#   # #
#   data.frame("ESR_n45" = ESR_n45) %>%
#     mutate(name =ESR_n45) %>%
#   separate(ESR_n45, into = c( "chr","start","end")) %>%
#     # mutate(strand=".") %>%
#     mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit() %>%
#     # unite(.,col="start-end",start, end,sep = "-") %>%
#     write.table(.,file = "data/n45_bedfiles/ESR_n45.bed", row.names =FALSE,col.name=FALSE,append = TRUE, quote = FALSE)
# 
#   data.frame("LR_n45" = LR_n45) %>%
#     mutate(name =LR_n45) %>%
#   separate(LR_n45, into = c( "chr","start","end")) %>%
#     # mutate(strand=".") %>%
#     mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit() %>%
#     # unite(.,col="start-end",start, end,sep = "-") %>%
#     write.table(.,file = "data/n45_bedfiles/LR_n45.bed", row.names =FALSE,col.name=FALSE,append = TRUE, quote = FALSE)
# 
#   data.frame("NR_n45" = NR_n45) %>%
#     mutate(name =NR_n45) %>%
#   separate(NR_n45, into = c( "chr","start","end")) %>%
#     # mutate(strand=".") %>%
#     mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit() %>%
#     # unite(.,col="start-end",start, end,sep = "-") %>%
#     write.table(.,file = "data/n45_bedfiles/NR_n45.bed", row.names =FALSE,col.name=FALSE,append = TRUE, quote = FALSE)
# 
# # #
# EAR_n45_gr <- bed_to_granges("data/n45_bedfiles/EAR_n45.bed")
# ESR_n45_gr <- bed_to_granges("data/n45_bedfiles/ESR_n45.bed")
# LR_n45_gr <- bed_to_granges("data/n45_bedfiles/LR_n45.bed")
# NR_n45_gr <- bed_to_granges("data/n45_bedfiles/NR_n45.bed")
# unclassified_n45_gr <- bed_to_granges("data/n45_bedfiles/unclassified_n45.bed")
# background_n45_gr <- bed_to_granges("data/n45_bedfiles/background_n45.bed")

# mylist <- list(EAR_n45_gr,ESR_n45_gr,LR_n45_gr,NR_n45_gr,unclassified_gr,background_gr)
# peakAnnoList<- lapply(mylist, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# names(peakAnnoList) <- c("EAR_n45_gr","ESR_n45_gr","LR_n45_gr","NR_n45_gr","unclassified_gr","background_gr")
# saveRDS(peakAnnoList, "data/peakAnnoList_n45_motif.RDS")

peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
plotAnnoBar(peakAnnoList_n45_motif[1:4], main = "Genomic Feature Distribution, CorMotif")+
  ggtitle ("Genomic Feature Distribution, Individuals 1,2,3,6")
##Epi_list_tagMatrix title was just because I was too lazy to change the name
# Epi_list_tagMatrix = lapply(Ind6_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind6_TSS_peaks.RDS")


```



```{r fullset cormotif lfc EAR}

my_hc_filtered_counts_n45 <- readRDS("data/my_hc_filt_counts_n45.RDS")
log_filt_hc_n45 <- cpm(my_hc_filtered_counts_n45, log = TRUE) %>% as.data.frame()
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")

log_filt_hc_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(row.names(.) %in% EAR_n45[c(1:3)]) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("log2cpm in EAR peaks n45")+
  scale_fill_manual(values = drug_pal)


```


```{r mlfc EAR}
toplist_n45<- readRDS("data/toplist_n45.RDS")


toplist_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(peak %in% motif_list_n45$EAR) %>% 
   mutate(logFC= abs(logFC)) %>% 
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle("logFC in EAR peaks")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()


```



```{r ESR boxplot, eval =FALSE}

log_filt_hc_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(row.names(.) %in% motif_list_n45$ESR[c(1,2,3)]) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("log2cpm in ESR peaks")+
  scale_fill_manual(values = drug_pal)


```



```{r mlfc ESR}



toplist_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(peak %in% motif_list_n45$ESR) %>% 
   mutate(logFC= abs(logFC)) %>% 
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle("logFC in ESR peaks")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()


```


```{r LR boxplot, eval = FALSE}

log_filt_hc_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(row.names(.) %in%LR[c(1,2,3)]) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("log2cpm in LR peaks")+
  scale_fill_manual(values = drug_pal)


```


```{r mlfc LR}

toplist_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(peak %in% motif_list_n45$LR) %>% 
   mutate(logFC= abs(logFC)) %>% 
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle("logFC in LR peaks")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()


```



```{r NR boxplot, eval=FALSE}

log_filt_hc_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(row.names(.) %in% NR[c(1,2,3)]) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("log2cpm in NR peaks")+
  scale_fill_manual(values = drug_pal)


```



```{r mlfc NR}



toplist_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(peak %in% motif_list_n45$NR) %>% 
  mutate(logFC= abs(logFC)) %>% 
  
  ggplot(., aes (x = time, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle("logFC in NR peaks")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

```


###lfc
```{r mlfcall}



log_filt_hc_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(row.names(.) %in% NR_n45[c(1,2,3)]) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("log2cpm in NR peaks")+
  scale_fill_manual(values = drug_pal)



mean_lfc <-
  toplist_n45 %>% as.data.frame() %>% 
  mutate(NR = if_else(peak %in% NR_n45,"y","no")) %>% 
  mutate(LR=if_else(peak %in%LR_n45,"y","no")) %>%
  mutate(ESR=if_else(peak %in%ESR_n45,"y","no")) %>%
  mutate(EAR=if_else(peak %in%EAR_n45,"y","no")) %>%
  # mutate(id = as.factor(id)) %>%
  # mutate(time=factor(time, levels=c("3_hours","24_hours"))) #%>% 
  group_by(trt,time) %>%
  mutate(absFC=abs(logFC)) %>% 
  dplyr::select(trt,time,absFC,EAR,ESR,LR,NR) %>% 
  dplyr::summarize(EAR=mean(absFC[EAR=="y"]),ESR=mean(absFC[ESR=="y"]),LR=mean(absFC[LR=="y"]),NR=mean(absFC[NR=="y"])) %>% as.data.frame()



mean_lfc %>% 
    pivot_longer(!c(trt,time), names_to = "Motif",values_to="meanLFC") %>% 
  ggplot(., aes(x=time,y= meanLFC,col=trt,group=trt))+
  geom_point()+
  geom_line(size = 3)+
 scale_fill_manual(values=drug_pal)+
  facet_wrap(~Motif)+
  theme_bw()+
  xlab("")+
  scale_color_manual(values=drug_pal)+
  ylab("|Log Fold Change|")+
  theme_bw()+
  ggtitle(" average |Log Fold| for genes across treatment each motif")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text = element_text(size = 8, color = "black", angle = 0),   
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))

# schneider_closest_output <- read_delim("~/ATAC_downloads/BEDtoolplay/output.bed",
#     delim = "\t", escape_double = FALSE,
#    col_names = c("chrom","start","stop","RSID","Pvalue","chrom_gene","start_gene","end_gene","ENTREZID","ensembl_gene_id"), trim_ws = TRUE)
# 
# setdiff(overlaps,backGL_n45_he)

```




#### Venn
```{r venn of gene overlap}

shorterlist <- peakAnnoList_n45_motif[c(1:4)]
genes= lapply(shorterlist, function(i) as.data.frame(i)$geneId)
vennplot(genes)
plotDistToTSS(peakAnnoList_n45_motif[c(1:4)])+ ggtitle("n45 peaks from TSS")
```



<!-- shorterlist <- peakAnnoList_n45_motif[1:4] -->
<!-- > genes= lapply(shorterlist, function(i) as.data.frame(i)$geneId) -->
<!-- > vennplot(genes) -->
<!-- > shorterlist <- peakAnnoList_n45_motif[c(1:4,6)] -->
<!-- > genes= lapply(shorterlist, function(i) as.data.frame(i)$geneId) -->
<!-- > vennplot(genes) -->
<!-- >  -->
