---
title: "Plot for 6c"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
	dev = c("png","pdf")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(ggfortify)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(edgeR)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(readxl)
library(arrow)

```
Notes to self(and anyone else who is reading this!):  
This is me applying the same code from my correlation_of_SNPnPeak.rmd document.    
Summary of what I am doing:
1: create a list of peaks within +/-20 kb, +/-10 kb, and +/- 5 kb of an RNA expressed gene TSS. (3 separate lists)  
2: making a dataframe that has all ATAC 3 hour and 24 hr LFC by peak for later ease of use.
3: creating lists of gwas SNPs (HF and ARR lists only) that are either 1bp, 10kb, 20kb, or 50kb in length to determine impact of the SNP on surrounding peaks.


```{r loading data}

Collapsed_new_peaks <- read_delim("data/Final_four_data/collapsed_new_peaks.txt", delim = "\t", col_names = TRUE)

Collapsed_new_peaks_gr <- Collapsed_new_peaks %>% dplyr::select(chr:Peakid) %>% GRanges()

peak_10kb_neargenes <-
  Collapsed_new_peaks %>% 
    dplyr::filter(dist_to_NG<5000&dist_to_NG>-5000) %>% 
  distinct(Peakid, .keep_all = TRUE) %>% 
  dplyr::select(Peakid,NCBI_gene,SYMBOL)

peak_20kb_neargenes <-
  Collapsed_new_peaks %>% 
    dplyr::filter(dist_to_NG<10000&dist_to_NG>-10000) %>% 
  distinct(Peakid, .keep_all = TRUE) %>% 
  dplyr::select(Peakid,NCBI_gene,SYMBOL)

peak_40kb_neargenes <-
  Collapsed_new_peaks %>% 
    dplyr::filter(dist_to_NG<20000&dist_to_NG>-20000) %>% 
  distinct(Peakid, .keep_all = TRUE) %>% 
  dplyr::select(Peakid,NCBI_gene,SYMBOL)

RNA_median_3_lfc <- readRDS("data/other_papers/RNA_median_3_lfc.RDS")
RNA_median_24_lfc <- readRDS("data/other_papers/RNA_median_24_lfc.RDS")

ATAC_24_lfc <- read_csv("data/Final_four_data/median_24_lfc.csv") 
ATAC_3_lfc <- read_csv("data/Final_four_data/median_3_lfc.csv")

#### AS of 1/23/24, I am pulling updated gwas for HF and ARR (now the term is Atrial fib)  These are stored as RDS in the other papers folder
# saveRDS(gwas_ud_HF, "data/other_papers/HF_gwas_association_downloaded_2025_01_23_EFO_0003144_withChildTraits.RDS")
# saveRDS(gwas_ud_AF,"data/other_papers/AF_gwas_association_downloaded_2025_01_23_EFO_0000275.RDS")

gwas_HF <- readRDS("data/other_papers/HF_gwas_association_downloaded_2025_01_23_EFO_0003144_withChildTraits.RDS")

gwas_ARR <- readRDS("data/other_papers/AF_gwas_association_downloaded_2025_01_23_EFO_0000275.RDS")
Short_gwas_gr <-
  gwas_ARR %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="AF") %>% 
   rbind(gwas_HF %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="HF")) %>% 
  na.omit() %>% 
 mutate(seqnames=paste0("chr",CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit() %>%
   mutate(start=CHR_POS, end=CHR_POS, width=1) %>% 
  GRanges()


Short_gwas_5k_gr <- 
    gwas_ARR %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="AF") %>% 
   rbind(gwas_HF %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="HF")) %>% 
  na.omit() %>% 
 mutate(seqnames=paste0("chr",CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit() %>%
   mutate(start=CHR_POS-5000, end=CHR_POS+4999) %>% 
  GRanges()


Short_gwas_20k_gr <- 
    gwas_ARR %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="AF") %>% 
   rbind(gwas_HF %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="HF")) %>% 
  na.omit() %>% 
 mutate(seqnames=paste0("chr",CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit() %>%
  mutate(start=(CHR_POS-10000),end=(CHR_POS+9999), width=20000) %>%
  distinct() %>% 
  GRanges()


Short_gwas_50k_gr <- 
    gwas_ARR %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="AF") %>% 
   rbind(gwas_HF %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="HF")) %>% 
  na.omit() %>% 
 mutate(seqnames=paste0("chr",CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit() %>%
  mutate(start=(CHR_POS-25000),end=(CHR_POS+24999), width=50000) %>%
  distinct() %>% 
  GRanges()
 
gwas_peak_check <- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_gr) %>%
  as.data.frame()
# 
gwas_peak_check_10k <- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_5k_gr) %>%
  as.data.frame()
gwas_peak_check_20k <- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_20k_gr) %>% 
  as.data.frame()
 gwas_peak_check_50k <- join_overlap_intersect(Collapsed_new_peaks_gr,Short_gwas_50k_gr) %>% 
  as.data.frame()


ATAC_LFC <- Collapsed_new_peaks %>%
                 dplyr::select(Peakid) %>% 
  left_join(.,(ATAC_3_lfc %>% dplyr::select(peak, med_3h_lfc)), by=c("Peakid"="peak")) %>% 
  left_join(.,(ATAC_24_lfc %>% dplyr::select(peak, med_24h_lfc)), by=c("Peakid"="peak"))
            Short_gwas_gr %>% as.data.frame(
              
            )
 # rtracklayer::export.bed(Short_gwas_gr,"data/Final_four_data/ARR_HF_SNP_local.bed")
 # 
 
 Dox_prot <- readRDS("data/other_papers/Dox_proteome_paper.RDS")
proto_list <- Dox_prot %>% 
  group_by(SYMBOL,logFC) %>% 
  summarize(ENTREZID=paste(unique(ENTREZID),collapse=";"),
            Protein=paste(unique(Protein),collapse=";"))
```

Here I am doing the overlapping of the previous ranges of SNPs and the full ATAC peak set.  I also later create the data frames from the reheat data, the reheat data using the p<0.005 top genes, the cluster names associated with each peak, and the list of TE/notTE associated with each peak.
```{r Creating df lists}

ATAC_peaks_gr <- Collapsed_new_peaks %>% GRanges()

Peaks_cutoff <- read_delim("data/Final_four_data/LCPM_matrix_ff.txt",delim = "/") %>% dplyr::select(Peakid)
  


gwas_short_list <- gwas_peak_check %>% as.data.frame %>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

gwas_10k_list <- gwas_peak_check_10k %>% distinct(SNPS,Peakid)%>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

gwas_20k_list <- gwas_peak_check_20k %>% distinct(SNPS,Peakid)%>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

gwas_50k_list <- gwas_peak_check_50k %>% distinct(SNPS,Peakid)%>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

Reheat_data <- read_excel("data/other_papers/jah36123-sup-0002-tables2.xlsx")
top_reheat <- Reheat_data %>%
  dplyr::filter(fisher_pvalue<0.005)
Nine_te_df <- readRDS("data/Final_four_data/Nine_group_TE_df.RDS")
###needed to change TE status to at least 1 bp overlap
match <- Nine_te_df %>% 
   mutate(TEstatus=if_else(!is.na(per_ol),"TE_peak","not_TE_peak")) %>% 
  distinct(Peakid,TEstatus,mrc,.keep_all = TRUE) 

```


### Peaks within 5kb +/- RNA TSS, gwas range +/- 10 kb

To break down what I am doing here:  I start with the list of peaks that overlap a gwas SNP that has been expanded by 20kb.  I then only add the RNA expressed genes associated with the peaks that are within +/- 5 kb of its TSS.  I join the median LFC data frames for ATAC and RNA at 3 and 24 hours, the TEstatus, the reheat status and exclude any SNP-Peak combinations that do not have RNA assigned. (This effectively is filtering out peaks outside of the 10kb TSS range That would make the list drop from 2019 to 298 rows)
```{r point-10k-20k-50k overlap, fig.height=55, fig.width=7}
gwas_df <-gwas_20k_list%>% 
  as.data.frame() %>%
  left_join(., peak_10kb_neargenes, by=c("Peakid"="Peakid")) %>%
  left_join(., (ATAC_3_lfc %>%
                  dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
                  dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
  left_join(., RNA_median_3_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>%
  left_join(., RNA_median_24_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,"reheat_gene","not_reheat_gene")) %>% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %>% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep ="_",remove=FALSE) %>% 
  left_join(.,(match %>% 
                 group_by(Peakid) %>%
                 filter(!(TEstatus=="not_TE_peak" & any (TEstatus == "TE_peak"))) %>% 
                 ungroup() %>%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c("Peakid"="Peakid")) %>% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &SNPS %in% gwas_50k_list$SNPS ~ 50)) %>% 
  group_by(SNPS,Peakid) %>% 
  summarize(name=unique(name),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          SYMBOL=paste(unique(SYMBOL),collapse=";"),
           reheat=paste(unique(reheat),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP)) %>% 
  na.omit(RNA_3h_lfc)

gwas_mat <- gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
gwas_name_mat <- gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno <- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat$TEstatus,reheat_status=gwas_name_mat$reheat,MRC=gwas_name_mat$mrc,direct_overlap=gwas_name_mat$dist_to_SNP,col= list(TE_status=c("TE_peak"="goldenrod",
                                "TE_peak;not_TE_peak"="goldenrod",
                                "not_TE_peak;TE_peak"="goldenrod",
                                "not_TE_peak"="lightblue"),                                                     MRC = c("EAR_open" = "#F8766D",                                                                    "EAR_close" = "#f6483c","ESR_open" = "#7CAE00",
                            "ESR_close" = "#587b00",
                            "ESR_opcl"="grey40",
                            "ESR_C"="grey40",
                            "ESR_clop"="tan",
                             "ESR_D"="tan",
                               "ESR_OC" = "#6a9500",
                                "LR_open" = "#00BFC4",
                              "LR_close" = "#008d91",
                              "NR" = "#C77CFF",
                                "not_mrc"="black"),
                    reheat_status=c("reheat_gene"="green","not_reheat_gene"="orange"),
                    direct_overlap=c("0"="red","10"="pink","20"="tan2","50"="grey8")))
mat2 <- gwas_mat  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = "")
simply_map <- ComplexHeatmap::Heatmap(gwas_mat,
                        left_annotation = row_anno,
                        show_row_names = TRUE,
                        # row_names_side = "left",
                        row_names_max_width= max_text_width(rownames(gwas_mat),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")


```

### Peaks within 5kb +/- RNA TSS, gwas range +/- 25 kb

For comparison, I went ahead and did the same this as above, but used the +/- 25 kb expanded SNP range.  This left me with 660 ATAC-SNP_RNA sets.
```{r point and 10k only, fig.height=35, fig.width=7}
gwas_df <-
gwas_50k_list%>% 
  as.data.frame() %>%
  left_join(., peak_10kb_neargenes, by=c("Peakid"="Peakid")) %>%
  left_join(., (ATAC_3_lfc %>%
                  dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
                  dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
  left_join(., RNA_median_3_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>%
  left_join(., RNA_median_24_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,"reheat_gene","not_reheat_gene")) %>% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %>% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep ="_",remove=FALSE) %>% 
  left_join(.,(match %>% 
                 group_by(Peakid) %>%
                 filter(!(TEstatus=="not_TE_peak" & any (TEstatus == "TE_peak"))) %>% 
                 ungroup() %>%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c("Peakid"="Peakid")) %>% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &SNPS %in% gwas_50k_list$SNPS ~ 50)) %>% 
  group_by(SNPS,Peakid) %>% 
  # mutate(Keep=case_when(SNPS))
  # group_by(Peakid) %>% 
 summarize(name=unique(name),
           # SNPS=unique(SNPS),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          SYMBOL=paste(unique(SYMBOL),collapse=";"),
           reheat=paste(unique(reheat),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %>% 
  na.omit(RNA_3h_lfc)

gwas_mat <- gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
gwas_name_mat <- gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno <- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat$TEstatus,reheat_status=gwas_name_mat$reheat,MRC=gwas_name_mat$mrc,direct_overlap=gwas_name_mat$dist_to_SNP,col= list(TE_status=c("TE_peak"="goldenrod",
                                "TE_peak;not_TE_peak"="goldenrod",
                                "not_TE_peak;TE_peak"="goldenrod",
                                "not_TE_peak"="lightblue"),                                                     MRC = c("EAR_open" = "#F8766D",                                                                    "EAR_close" = "#f6483c","ESR_open" = "#7CAE00",
                            "ESR_close" = "#587b00",
                            "ESR_opcl"="grey40",
                            "ESR_C"="grey40",
                            "ESR_clop"="tan",
                             "ESR_D"="tan",
                               "ESR_OC" = "#6a9500",
                                "LR_open" = "#00BFC4",
                              "LR_close" = "#008d91",
                              "NR" = "#C77CFF",
                                "not_mrc"="black"),
                    reheat_status=c("reheat_gene"="green","not_reheat_gene"="orange"),
                    direct_overlap=c("0"="red","10"="pink","20"="tan2","50"="grey8")))
mat2 <- gwas_mat  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = "")
simply_map <- ComplexHeatmap::Heatmap(gwas_mat,
                        left_annotation = row_anno,
                        show_row_names = TRUE,
                        # row_names_side = "left",
                        row_names_max_width= max_text_width(rownames(gwas_mat),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")
```
### Peaks within +/-20 kb RNA TSS ,gwas range +/- 10 kb

warning, 702 rows below 

```{r 20kb RNA,fig.height=35, fig.width=7}
gwas_df <-
gwas_20k_list%>% 
  as.data.frame() %>%
  left_join(., peak_40kb_neargenes, by=c("Peakid"="Peakid")) %>%
  left_join(., (ATAC_3_lfc %>%
                  dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
                  dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
  left_join(., RNA_median_3_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>%
  left_join(., RNA_median_24_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,"reheat_gene","not_reheat_gene")) %>% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %>% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep ="_",remove=FALSE) %>% 
  left_join(.,(match %>% 
                 group_by(Peakid) %>%
                 filter(!(TEstatus=="not_TE_peak" & any (TEstatus == "TE_peak"))) %>% 
                 ungroup() %>%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c("Peakid"="Peakid")) %>% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &SNPS %in% gwas_50k_list$SNPS ~ 50)) %>% 
  group_by(SNPS,Peakid) %>% 
  arrange(., Peakid) %>% 
  # mutate(Keep=case_when(SNPS))
  # group_by(Peakid) %>% 
 summarize(name=unique(name),
           # SNPS=unique(SNPS),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          SYMBOL=paste(unique(SYMBOL),collapse=";"),
           reheat=paste(unique(reheat),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %>% 
  na.omit(RNA_3h_lfc)

gwas_mat <- gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
gwas_name_mat <- gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno <- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat$TEstatus,reheat_status=gwas_name_mat$reheat,MRC=gwas_name_mat$mrc,direct_overlap=gwas_name_mat$dist_to_SNP,col= list(TE_status=c("TE_peak"="goldenrod",
                                "TE_peak;not_TE_peak"="goldenrod",
                                "not_TE_peak;TE_peak"="goldenrod",
                                "not_TE_peak"="lightblue"),                                                     MRC = c("EAR_open" = "#F8766D",                                                                    "EAR_close" = "#f6483c","ESR_open" = "#7CAE00",
                            "ESR_close" = "#587b00",
                            "ESR_opcl"="grey40",
                            "ESR_C"="grey40",
                            "ESR_clop"="tan",
                             "ESR_D"="tan",
                               "ESR_OC" = "#6a9500",
                                "LR_open" = "#00BFC4",
                              "LR_close" = "#008d91",
                              "NR" = "#C77CFF",
                                "not_mrc"="black"),
                    reheat_status=c("reheat_gene"="green","not_reheat_gene"="orange"),
                    direct_overlap=c("0"="red","10"="pink","20"="tan2","50"="grey8")))
mat2 <- gwas_mat  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = "")
simply_map <- ComplexHeatmap::Heatmap(gwas_mat,
                        left_annotation = row_anno,
                        show_row_names = TRUE,
                        # row_names_side = "left",
                        row_names_max_width= max_text_width(rownames(gwas_mat),                                                        gp=gpar(fontsize=8)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")

```
### gwas short list only
To make life really easy, or the smallest set that was readable, I used only the peaks that were directly overlapping a SNP, but filtered out peaks that were more than +/- 5 kb from an expressed RNA TSS.  This gave me 33 ATAC-SNP-RNA rows.
```{r RNA,fig.height=10, fig.width=7}

gwas_df_short <-gwas_short_list%>%
  as.data.frame() %>%
  left_join(., peak_10kb_neargenes, by=c("Peakid"="Peakid")) %>%
  left_join(., (ATAC_3_lfc %>%
                  dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
                  dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
  left_join(., RNA_median_3_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>%
  left_join(., RNA_median_24_lfc,by =c("NCBI_gene"="ENTREZID", "SYMBOL"="SYMBOL")) %>% 
  na.omit(RNA_median_24_lfc) %>% 
  mutate(reheat=if_else(SYMBOL %in% Reheat_data$gene,"reheat_gene","not_reheat_gene")) %>% 
  distinct(SNPS,Peakid,.keep_all = TRUE) %>% 
  tidyr::unite(name,SNPS,SYMBOL,Peakid,sep ="_",remove=FALSE) %>% 
  left_join(.,(match %>% 
                 group_by(Peakid) %>%
                 filter(!(TEstatus=="not_TE_peak" & any (TEstatus == "TE_peak"))) %>% 
                 ungroup() %>%
                 distinct(TEstatus,Peakid,.keep_all = TRUE)),
            by = c("Peakid"="Peakid")) %>% 
  mutate(dist_to_SNP=case_when(Peakid %in% gwas_short_list$Peakid &SNPS %in% gwas_short_list$SNPS~ 0,
                               Peakid %in% gwas_10k_list$Peakid &SNPS %in% gwas_10k_list$SNPS~ 10,
    Peakid %in% gwas_20k_list$Peakid &SNPS %in% gwas_20k_list$SNPS~ 20,
     Peakid %in% gwas_50k_list$Peakid &SNPS %in% gwas_50k_list$SNPS ~ 50)) %>% 
  group_by(SNPS,Peakid) %>% 
  
  # mutate(Keep=case_when(SNPS))
  # group_by(Peakid) %>% 
 summarize(name=unique(name),
           # SNPS=unique(SNPS),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           RNA_3h_lfc=unique(RNA_3h_lfc),
           RNA_24h_lfc=unique(RNA_24h_lfc),
          repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          SYMBOL=paste(unique(SYMBOL),collapse=";"),
           reheat=paste(unique(reheat),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP)) %>% 
   arrange(., Peakid)%>% 
  left_join(., proto_list, by=c("SYMBOL"="SYMBOL"))
 
gwas_mat_short <- gwas_df_short %>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc:RNA_24h_lfc,logFC) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
gwas_name_mat_short <- gwas_df_short %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,reheat,dist_to_SNP)

row_anno_short <- ComplexHeatmap::rowAnnotation(TE_status=gwas_name_mat_short$TEstatus,reheat_status=gwas_name_mat_short$reheat,MRC=gwas_name_mat_short$mrc,direct_overlap=gwas_name_mat_short$dist_to_SNP,col= list(TE_status=c("TE_peak"="goldenrod",
                                "TE_peak;not_TE_peak"="goldenrod",
                                "not_TE_peak;TE_peak"="goldenrod",
                                "not_TE_peak"="lightblue"),                                                     MRC = c("EAR_open" = "#F8766D",                                                                    "EAR_close" = "#f6483c","ESR_open" = "#7CAE00",
                            "ESR_close" = "#587b00",
                            "ESR_opcl"="grey40",
                            "ESR_C"="grey40",
                            "ESR_clop"="tan",
                             "ESR_D"="tan",
                               "ESR_OC" = "#6a9500",
                                "LR_open" = "#00BFC4",
                              "LR_close" = "#008d91",
                              "NR" = "#C77CFF",
                                "not_mrc"="black"),
                    reheat_status=c("reheat_gene"="green","not_reheat_gene"="orange"),
                    direct_overlap=c("0"="red","10"="pink","20"="tan2","50"="grey8")))
mat2_short <- gwas_mat_short  
# rownames(mat2)[1] = paste(c(letters, LETTERS), collapse = "")
simply_map_short <- ComplexHeatmap::Heatmap(gwas_mat_short,
                        left_annotation = row_anno_short,
                        # show_row_names = TRUE,
                        # width = 10,
                        # row_names_side = "left",
                        row_names_max_width= max_text_width(rownames(gwas_mat_short),                                                        gp=gpar(fontsize=16)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_short, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")


```

### mapping genes of interest:

```{r}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
# K27_counts <-  readRDS("data/Final_four_data/All_Raodahpeaks.RDS")
ATAC_counts <- readRDS("data/Final_four_data/x4_filtered.RDS")
RNA_counts <- readRDS("data/other_papers/cpmcount.RDS") %>%
  dplyr::rename_with(.,~gsub(pattern="Da",replacement="DNR",.)) %>% 
 dplyr::rename_with(.,~gsub(pattern="Do",replacement="DOX",.)) %>% 
  dplyr::rename_with(.,~gsub(pattern="Ep",replacement="EPI",.)) %>% 
   dplyr::rename_with(.,~gsub(pattern="Mi",replacement="MTX",.)) %>% 
    dplyr::rename_with(.,~gsub(pattern="Tr",replacement="TRZ",.)) %>% 
       dplyr::rename_with(.,~gsub(pattern="Ve",replacement="VEH",.)) %>% 
  rownames_to_column("ENTREZID")

```


```{r, fig.height=3, fig.width=7}

df_gene <- data.frame(SYMBOL=c("PSRC1","CDKN1A","CELSR2"))
df_gene <- df_gene %>% 
left_join(., (RNA_median_24_lfc %>% dplyr::select(ENTREZID,SYMBOL)), by = c ("SYMBOL"="SYMBOL")) %>% 
  left_join(., (gwas_df_short %>% dplyr::select(SNPS,Peakid,mrc,SYMBOL)),by = c("SYMBOL"="SYMBOL")) 

        
RNA_counts %>% 
  dplyr::filter(ENTREZID %in% df_gene$ENTREZID) %>% 
  pivot_longer(cols = !ENTREZID, names_to = "sample", values_to = "counts") %>% 
  left_join(., df_gene, by =c("ENTREZID"="ENTREZID")) %>% 
  separate("sample", into = c("trt","ind","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL+Peakid, scales="free_y")+
  ggtitle("RNA LFC of expressed gene")+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm RNA")



ATAC_counts %>% 
  cpm(., log = TRUE) %>% 
   as.data.frame() %>%
  rename_with(.,~gsub(pattern = "Ind1_75", replacement = "1_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind2_87", replacement = "2_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind3_77", replacement = "3_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind6_71", replacement = "6_",.)) %>%
  rename_with(.,~gsub( "DX" ,'DOX',.)) %>%
  rename_with(.,~gsub( "DA" ,'DNR',.)) %>%
  rename_with(.,~gsub( "E" ,'EPI',.)) %>%
  rename_with(.,~gsub( "T" ,'TRZ',.)) %>%
  rename_with(.,~gsub( "M" ,'MTX',.)) %>%
  rename_with(.,~gsub( "V" ,'VEH',.)) %>%
  rename_with(.,~gsub("24h","_24h",.)) %>%
  rename_with(.,~gsub("3h","_3h",.)) %>% 
  dplyr::filter(row.names(.) %in% df_gene$Peakid) %>% 
  mutate(Peakid = row.names(.)) %>% 
  pivot_longer(cols = !Peakid, names_to = "sample", values_to = "counts") %>% 
  left_join(., df_gene, by =c("Peakid"="Peakid")) %>% 
  separate("sample", into = c("ind","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~Peakid+SYMBOL,scales="free_y")+
  ggtitle(" ATAC accessibility")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm ATAC")
```

```{r additional mapping}

df_gene_next <- data.frame(SYMBOL=c("RAB44","DINOL"), ENTREZID=c("401258","108783646"))
df_gene_next <- df_gene_next %>% 
  left_join(., (gwas_df_short %>% dplyr::select(SNPS,Peakid,mrc,SYMBOL)),by = c("SYMBOL"="SYMBOL")) 


RNA_counts %>% 
  dplyr::filter(ENTREZID %in% df_gene_next$ENTREZID) %>% 
  pivot_longer(cols = !ENTREZID, names_to = "sample", values_to = "counts") %>% 
  # mutate(ENTREZID=as.numeric(ENTREZID)) %>% 
  left_join(., df_gene_next, by =c("ENTREZID"="ENTREZID")) %>% 
  separate("sample", into = c("trt","ind","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(~SYMBOL+Peakid, scales="free_y")+
  ggtitle("RNA LFC of expressed gene")+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm RNA")

```
### Last figure attempt #2

```{r direct overlap new gwas,fig.height=12, fig.width=8}

count_df <- join_overlap_intersect(Collapsed_new_peaks_gr, Short_gwas_gr)

new_gwas_df <- count_df %>% 
  as.data.frame() %>% 
  left_join(., Nine_te_df, by=("Peakid"="Peakid")) %>% 
  left_join(.,(Collapsed_new_peaks %>% 
                 dplyr::select (Peakid, SYMBOL )),by = c ("Peakid"="Peakid")) %>% 
  dplyr::filter(mrc !="NR") %>%
  dplyr::filter(mrc !="not_mrc") %>%
  left_join(., (ATAC_3_lfc %>%
  dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
  dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak")) %>% 
  mutate(dist_to_SNP=0) %>% 
  group_by(Peakid, SNPS) %>% 
  summarize(med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
            SYMBOL=unique(SYMBOL),collapse=";",
           # AC_24h_lfc=unique(AC_24h_lfc),
           repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
            GWAS=paste(unique(gwas),collapse=";"),
           mrc=unique(mrc),
            dist_to_SNP=min(dist_to_SNP)) %>% 
  tidyr::unite(name,Peakid,SNPS,SYMBOL,sep ="_",remove=FALSE) %>% 
   arrange(., Peakid)


new_gwas_mat <- new_gwas_df%>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc, med_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
new_gwas_name_mat <- new_gwas_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,GWAS,dist_to_SNP)

row_anno_gwas <-
  rowAnnotation(
    TE_status=new_gwas_name_mat$TEstatus,
    gwas_status=new_gwas_name_mat$GWAS,
    MRC=new_gwas_name_mat$mrc,
    direct_overlap=new_gwas_name_mat$dist_to_SNP,
    col= list(TE_status=c("TE_peak"="goldenrod",
                          "not_TE_peak"="lightblue"), 
              MRC = c("EAR_open" = "#F8766D",
                      "EAR_close" = "#f6483c",
                      "ESR_open" = "#7CAE00",
                      "ESR_close" = "#587b00",
                      "ESR_opcl"="grey40", 
                      "ESR_clop"="tan",
                      "LR_open" = "#00BFC4",
                      "LR_close" = "#008d91",
                      "NR" = "#C77CFF",
                      "not_mrc"="black"),
              gwas_status=c("AF"="green",
                            "HF"="orange", 
                            "AF;HF"="purple3"),
              direct_overlap=c("0"="red",
                               "10"="pink",
                               "20"="tan2",
                               "50"="grey8")))

simply_map_gwas <- ComplexHeatmap::Heatmap(new_gwas_mat,
                        left_annotation = row_anno_gwas,
                        row_names_max_width = max_text_width(rownames(new_gwas_mat),  
                                                             gp=gpar(fontsize=16)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_gwas, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")



# new_gwas_df %>% 
#   dplyr::filter(GWAS=="AF"|GWAS =="AF;HF") %>% 
#   distinct(Peakid)
#  
# #   distinct(Peakid) %>% 
# #   tally
# Short_gwas_gr %>%
#   as.data.frame() %>%
#   dplyr::filter(gwas=="AF") %>%
#   distinct(SNPS)

```

```{r new heatmap noRNA}
Knowles_respQTL <- readRDS("data/Knowles_5.RDS")
Knowles_alleQTL <- readRDS("data/Knowles_4.RDS")
# file <- read_parquet(choose.files())
Heart_Left_Ventricle_v10_eGenes <- readRDS("data/other_papers/Heart_Left_Ventricle_v10_eGenes_GTEx.RDS")
gwaslist <- new_gwas_df %>% distinct(SNPS) %>% ungroup()
Heart_Left_Ventricle_v10_eGenes %>% dplyr::filter (rs_id_dbSNP155_GRCh38p13 %in% gwaslist$SNPS)
gwaslist$gene <-c("-","PSRC1","PSRC1","CLCN6","-","PRRX1","PRRX1","-","-","-",
"FUT11;SYNPO2l-AS1","-","ENSG00000254851;ENSG00000280143","-","-","-","TMEM263-DT","-","-","BRICD5;PGP",
"-","-","-","-","CHRNB1","-","-","-","FKBP7","-",
"PLGLB1","-","KCNE1","-","-","-","SCN10A","GMPPB;WDR6;NCKIPSD;NICN1;RBM6;AMT;QRICH1;IHO1","WDR1","-",
"-","-","-","DNAJC18;SPATA24;PROB1;SLC23A1","-","CDKN1A","CAV2;ENSG00000279086","-","-")

gwaslist %>% 
  dplyr::filter(SNPS %in% Knowles_alleQTL$RSID)
Knowles_alleQTL %>% 
  dplyr::filter(RSID %in% gwaslist$SNPS)
# write.csv(gwaslist,"data/other_papers/GWAS_eQTL_genes.csv")

```
### Park et al.  


```{r Boram Park etal}
# saveRDS(ParkSNPs,"data/other_papers/ParkSNPs_pull_VEF.RDS")

ParkSNPs <- readRDS("data/other_papers/ParkSNPs_pull_VEF.RDS")


ParkSNP_table <-
  ParkSNPs %>% 
  dplyr::select(1:2) %>% 
    distinct() %>% 
    separate_wider_delim(.,Location,delim=":",names=c("chr","position"), cols_remove=FALSE) %>% 
    separate_wider_delim(.,position,delim="-",names=c("begin","term")) %>%
    mutate(chr=paste0("chr",chr)) 
ParkSNP_gr <- ParkSNP_table %>% 
  mutate("start" = begin, "end"=term) %>% 
    GRanges()
ParkSNP_10k_gr <- ParkSNP_table %>% 
  mutate(begin=as.numeric(begin),term=as.numeric(term)) %>% 
  mutate(start=begin-5000, end=term+5000) %>% 
  GRanges()

ParkSNP_20k_gr <- ParkSNP_table %>% 
  mutate(begin=as.numeric(begin),term=as.numeric(term)) %>% 
  mutate(start=begin-10000, end=term+10000) %>% 
  GRanges()

ParkSNP_gr_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_gr) %>%
  as.data.frame()

ParkSNP_gr_10k_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_10k_gr) %>%
  as.data.frame()

ParkSNP_gr_20k_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_20k_gr) %>%
  as.data.frame()
# rtracklayer::export.bed(ParkSNP_gr,"data/Final_four_data/ParkSNP.bed")
```

```{r masterplot Park}

ParkSNP_gr_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_gr) %>%
  as.data.frame()
Park_df <-ParkSNP_gr_20k_check%>% 
  as.data.frame() %>%
  dplyr::select(Peakid, X.Uploaded_variation) %>% 
  dplyr::rename("SNPS"=X.Uploaded_variation) %>% 
  left_join(., Nine_te_df, by=("Peakid"="Peakid")) %>%
  dplyr::select(Peakid, SNPS,mrc,TEstatus) %>% 
  # left_join(., (Collapsed_new_peaks %>% 
  #             dplyr::select(Peakid,NCBI_gene,SYMBOL)), by=c("Peakid"="Peakid"))
  left_join(., (ATAC_3_lfc %>%
          dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
              dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
    dplyr::filter(mrc !="NR") %>%
  dplyr::filter(mrc !="not_mrc") %>% 
 mutate(dist_to_SNP=case_when(
    Peakid %in% ParkSNP_gr_check$Peakid &SNPS %in% ParkSNP_gr_check$X.Uploaded_variation~ 0,
    Peakid %in% ParkSNP_gr_10k_check$Peakid &SNPS %in% ParkSNP_gr_10k_check$X.Uploaded_variation~ 10,
    Peakid %in% ParkSNP_gr_20k_check$Peakid &SNPS %in% ParkSNP_gr_20k_check$X.Uploaded_variation~ 20)) %>% 
    tidyr::unite(name,Peakid,SNPS, sep = "_", remove = FALSE) %>% 
  group_by(Peakid) %>% 
   summarize(name=unique(name),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           # RNA_3h_lfc=unique(RNA_3h_lfc),
           # RNA_24h_lfc=unique(RNA_24h_lfc),
          # repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          # SYMBOL=paste(unique(SYMBOL),collapse=";"),
           # reheat=paste(unique(reheat),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %>% 
  arrange(., Peakid)
 
new_park_mat <- Park_df%>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc, med_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
new_park_name_mat <- Park_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,dist_to_SNP)

row_anno_park <-
  rowAnnotation(
    TE_status=new_park_name_mat$TEstatus,
    # gwas_status=new_park_name_mat$GWAS,
    MRC=new_park_name_mat$mrc,
    direct_overlap=new_park_name_mat$dist_to_SNP,
    col= list(TE_status=c("TE_peak"="goldenrod",
                          "not_TE_peak"="lightblue"), 
              MRC = c("EAR_open" = "#F8766D",
                      "EAR_close" = "#f6483c",
                      "ESR_open" = "#7CAE00",
                      "ESR_close" = "#587b00",
                      "ESR_opcl"="grey40", 
                      "ESR_clop"="tan",
                      "LR_open" = "#00BFC4",
                      "LR_close" = "#008d91",
                      "NR" = "#C77CFF",
                      "not_mrc"="black"),
              # gwas_status=c("AF"="green",
              #               "HF"="orange", 
              #               "AF;HF"="purple3"),
              direct_overlap=c("0"="red",
                               "10"="pink",
                               "20"="tan2",
                               "50"="grey8")))
col_fun <- circlize::colorRamp2(c(-4, 0, 4), c("blue", "white", "red"))
simply_map_park <- ComplexHeatmap::Heatmap(new_park_mat,
                                           col=col_fun,
                        left_annotation = row_anno_park,
                        row_names_max_width = max_text_width(rownames(new_park_mat),  
                                                             gp=gpar(fontsize=16)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_park, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")



 


```
```{r RNA park, fig.height=4, fig.width=5}
park_gene <- data.frame(SYMBOL=c("PRUNE2","PCA3","CDH13","TMEM237"))
park_gene <- park_gene %>% 
left_join(., (RNA_median_24_lfc %>% dplyr::select(ENTREZID,SYMBOL)), by = c ("SYMBOL"="SYMBOL")) 
  # left_join(., (gwas_df_short %>% dplyr::select(SNPS,Peakid,mrc,SYMBOL)),by = c("SYMBOL"="SYMBOL")) 

        
RNA_counts %>% 
  dplyr::filter(ENTREZID %in% park_gene$ENTREZID) %>% 
  pivot_longer(cols = !ENTREZID, names_to = "sample", values_to = "counts") %>% 
  left_join(., park_gene, by =c("ENTREZID"="ENTREZID")) %>% 
  separate("sample", into = c("trt","ind","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL, scales="free_y")+
  ggtitle("RNA log 2 cpm of expressed gene")+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm RNA")

park_atac <- Park_df %>% dplyr::select(Peakid)

ATAC_counts %>% 
  cpm(., log = TRUE) %>% 
   as.data.frame() %>%
  rename_with(.,~gsub(pattern = "Ind1_75", replacement = "1_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind2_87", replacement = "2_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind3_77", replacement = "3_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind6_71", replacement = "6_",.)) %>%
  rename_with(.,~gsub( "DX" ,'DOX',.)) %>%
  rename_with(.,~gsub( "DA" ,'DNR',.)) %>%
  rename_with(.,~gsub( "E" ,'EPI',.)) %>%
  rename_with(.,~gsub( "T" ,'TRZ',.)) %>%
  rename_with(.,~gsub( "M" ,'MTX',.)) %>%
  rename_with(.,~gsub( "V" ,'VEH',.)) %>%
  rename_with(.,~gsub("24h","_24h",.)) %>%
  rename_with(.,~gsub("3h","_3h",.)) %>% 
  dplyr::filter(row.names(.) %in% park_atac$Peakid) %>% 
  mutate(Peakid = row.names(.)) %>% 
  pivot_longer(cols = !Peakid, names_to = "sample", values_to = "counts") %>% 
  left_join(., park_atac, by =c("Peakid"="Peakid")) %>% 
  separate("sample", into = c("ind","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~Peakid,scales="free_y")+
  ggtitle(" ATAC accessibility")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm ATAC")



```

### nakano, et al
```{r}
nakano_SNPs <- readRDS("data/other_papers/nakano_SNPs_pull_VEF.RDS")

nakano_SNP_table <-
  nakano_SNPs %>% 
  dplyr::select(1:2) %>% 
    distinct() %>% 
    separate_wider_delim(.,Location,delim=":",names=c("chr","position"), cols_remove=FALSE) %>% 
    separate_wider_delim(.,position,delim="-",names=c("begin","term")) %>%
    mutate(chr=paste0("chr",chr)) 
nakano_SNP_gr <- nakano_SNP_table %>% 
  mutate("start" = begin, "end"=term) %>% 
    GRanges()
nakano_SNP_10k_gr <- nakano_SNP_table %>% 
  mutate(begin=as.numeric(begin),term=as.numeric(term)) %>% 
  mutate(start=begin-5000, end=term+5000) %>% 
  GRanges()

nakano_SNP_20k_gr <- nakano_SNP_table %>% 
  mutate(begin=as.numeric(begin),term=as.numeric(term)) %>% 
  mutate(start=begin-10000, end=term+10000) %>% 
  GRanges()

nakano_SNP_gr_check <- join_overlap_intersect(Collapsed_new_peaks_gr,nakano_SNP_gr) %>%
  as.data.frame()

nakano_SNP_gr_10k_check <- join_overlap_intersect(Collapsed_new_peaks_gr,nakano_SNP_10k_gr) %>%
  as.data.frame()

nakano_SNP_gr_20k_check <- join_overlap_intersect(Collapsed_new_peaks_gr,nakano_SNP_20k_gr) %>%
  as.data.frame()

```

```{r}
nakano_SNP_gr_check <- join_overlap_intersect(Collapsed_new_peaks_gr,nakano_SNP_gr) %>%
  as.data.frame()
nakano_df <-nakano_SNP_gr_20k_check%>% 
  as.data.frame() %>%
  dplyr::select(Peakid, X.Uploaded_variation) %>% 
  dplyr::rename("SNPS"=X.Uploaded_variation) %>% 
  left_join(., Nine_te_df, by=("Peakid"="Peakid")) %>%
  dplyr::select(Peakid, SNPS,mrc,TEstatus) %>% 
  # left_join(., (Collapsed_new_peaks %>% 
  #             dplyr::select(Peakid,NCBI_gene,SYMBOL)), by=c("Peakid"="Peakid"))
  left_join(., (ATAC_3_lfc %>%
          dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
              dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
    dplyr::filter(mrc !="NR") %>%
  dplyr::filter(mrc !="not_mrc") %>% 
 mutate(dist_to_SNP=case_when(
    Peakid %in% nakano_SNP_gr_check$Peakid &SNPS %in% nakano_SNP_gr_check$X.Uploaded_variation~ 0,
    Peakid %in% nakano_SNP_gr_10k_check$Peakid &SNPS %in% nakano_SNP_gr_10k_check$X.Uploaded_variation~ 10,
    Peakid %in% nakano_SNP_gr_20k_check$Peakid &SNPS %in% nakano_SNP_gr_20k_check$X.Uploaded_variation~ 20)) %>% 
    tidyr::unite(name,Peakid,SNPS, sep = "_", remove = FALSE) %>% 
  group_by(Peakid) %>% 
   summarize(name=unique(name),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           # AC_3h_lfc=unique(AC_3h_lfc),
           # AC_24h_lfc=unique(AC_24h_lfc),
           # RNA_3h_lfc=unique(RNA_3h_lfc),
           # RNA_24h_lfc=unique(RNA_24h_lfc),
          # repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          # SYMBOL=paste(unique(SYMBOL),collapse=";"),
           # reheat=paste(unique(reheat),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %>% 
  arrange(., Peakid)
 
new_nakano_mat <- nakano_df%>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc, med_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
new_nakano_name_mat <- nakano_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,dist_to_SNP)

row_anno_nakano <-
  rowAnnotation(
    TE_status=new_nakano_name_mat$TEstatus,
    # gwas_status=new_nakano__name_mat$GWAS,
    MRC=new_nakano_name_mat$mrc,
    direct_overlap=new_nakano_name_mat$dist_to_SNP,
    col= list(TE_status=c("TE_peak"="goldenrod",
                          "not_TE_peak"="lightblue"), 
              MRC = c("EAR_open" = "#F8766D",
                      "EAR_close" = "#f6483c",
                      "ESR_open" = "#7CAE00",
                      "ESR_close" = "#587b00",
                      "ESR_opcl"="grey40", 
                      "ESR_clop"="tan",
                      "LR_open" = "#00BFC4",
                      "LR_close" = "#008d91",
                      "NR" = "#C77CFF",
                      "not_mrc"="black"),
              # gwas_status=c("AF"="green",
              #               "HF"="orange", 
              #               "AF;HF"="purple3"),
              direct_overlap=c("0"="red",
                               "10"="pink",
                               "20"="tan2",
                               "50"="grey8")))

simply_map_nakano <- ComplexHeatmap::Heatmap(new_nakano_mat,
                                             col= col_fun,
                        left_annotation = row_anno_nakano,
                        row_names_max_width = max_text_width(rownames(new_nakano_mat),  
                                                             gp=gpar(fontsize=16)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_nakano, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")
```
```{r RNA and accessibility}


nakano_gene <- data.frame(SYMBOL=c("FSCN2","FAAP100"))
nakano_gene <- nakano_gene %>% 
left_join(., (RNA_median_24_lfc %>% dplyr::select(ENTREZID,SYMBOL)), by = c ("SYMBOL"="SYMBOL")) 
  # left_join(., (gwas_df_short %>% dplyr::select(SNPS,Peakid,mrc,SYMBOL)),by = c("SYMBOL"="SYMBOL")) 

        
RNA_counts %>% 
  dplyr::filter(ENTREZID %in% nakano_gene$ENTREZID) %>% 
  pivot_longer(cols = !ENTREZID, names_to = "sample", values_to = "counts") %>% 
  left_join(., nakano_gene, by =c("ENTREZID"="ENTREZID")) %>% 
  separate("sample", into = c("trt","ind","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL, scales="free_y")+
  ggtitle("RNA log 2 cpm of expressed gene")+
    scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm RNA")

nakano_atac <- nakano_df %>% dplyr::select(Peakid)

ATAC_counts %>% 
  cpm(., log = TRUE) %>% 
   as.data.frame() %>%
  rename_with(.,~gsub(pattern = "Ind1_75", replacement = "1_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind2_87", replacement = "2_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind3_77", replacement = "3_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind6_71", replacement = "6_",.)) %>%
  rename_with(.,~gsub( "DX" ,'DOX',.)) %>%
  rename_with(.,~gsub( "DA" ,'DNR',.)) %>%
  rename_with(.,~gsub( "E" ,'EPI',.)) %>%
  rename_with(.,~gsub( "T" ,'TRZ',.)) %>%
  rename_with(.,~gsub( "M" ,'MTX',.)) %>%
  rename_with(.,~gsub( "V" ,'VEH',.)) %>%
  rename_with(.,~gsub("24h","_24h",.)) %>%
  rename_with(.,~gsub("3h","_3h",.)) %>% 
  dplyr::filter(row.names(.) %in% nakano_atac$Peakid) %>% 
  mutate(Peakid = row.names(.)) %>% 
  pivot_longer(cols = !Peakid, names_to = "sample", values_to = "counts") %>% 
  left_join(., nakano_atac, by =c("Peakid"="Peakid")) %>% 
  separate("sample", into = c("ind","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~Peakid,scales="free_y")+
  ggtitle(" ATAC accessibility")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm ATAC")




```

