---
title: "DAR_analysis"
author: "Renee Matthews"
date: "2025-05-06"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
	dev = c("png","pdf")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r packages}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(eulerr)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(data.table)
library(ggVennDiagram)

```


### Differential analysis

Loading counts matrix and making filtered matrix
```{r}
raw_counts <- read_delim("data/Final_four_data/re_analysis/Raw_unfiltered_counts.tsv",delim="\t") %>% 
  column_to_rownames("Peakid") %>% 
  as.matrix()

lcpm <- cpm(raw_counts, log= TRUE)
  ### for determining the basic cutoffs
filt_raw_counts <- raw_counts[rowMeans(lcpm)> 0,]

filt_raw_counts_noY <- filt_raw_counts[!grepl("chrY",rownames(filt_raw_counts)),]
dim(filt_raw_counts_noY)
```

Number of filtered regions without the y chromosome = `r length(rownames(filt_raw_counts_noY))` regions


#### making the metadata form
```{r metadata}

annotation_mat <- data.frame(timeset=colnames(filt_raw_counts_noY)) %>%
  mutate(sample = timeset) %>% 
  separate(timeset, into = c("indv","trt","time"), sep= "_") %>% 
  mutate(time = factor(time, levels = c("3h", "24h"))) %>% 
  mutate(trt = factor(trt, levels = c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH"))) %>% 
  mutate(indv=factor(indv, levels = c("A","B","C","D"))) %>% 
  mutate(trt_time=paste0(trt,"_",time))

```


prepare DGE object

```{r dge object}
group <- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group <- factor(group, levels =c("1","2","3","4","5","6","7","8","9","10","11","12"))
dge <-  DGEList.data.frame(counts = filt_raw_counts_noY, group = group, genes = row.names(filt_raw_counts_noY))
dge <- calcNormFactors(dge)

dge$samples
```
Making model matrix
```{r create mm}
group_1 <- c(rep(c("DNR_24","DNR_3","DOX_24","DOX_3","EPI_24","EPI_3","MTX_24","MTX_3","TRZ_24","TRZ_3","VEH_24", "VEH_3"),4))

 mm <- model.matrix(~0 +group_1)
colnames(mm) <-  c("DNR_24", "DNR_3", "DOX_24","DOX_3","EPI_24", "EPI_3","MTX_24", "MTX_3", "TRZ_24","TRZ_3","VEH_24", "VEH_3")
mm
```

In this pipeline, I first run voom transformation, then estimate the intra-individual correlation.  Next I do voom again with correlation info.
I fit the linear model, define contrasts, then apply the contrasts and perform eBayes to get statistics. 
```{r run pipeline}
y <- voom(dge, mm,plot =FALSE)

corfit <- duplicateCorrelation(y, mm, block = annotation_mat$indv)
 
v <- voom(dge, mm, block = annotation_mat$indv, correlation = corfit$consensus)

fit <- lmFit(v, mm, block = annotation_mat$indv, correlation = corfit$consensus)


cm <- makeContrasts(
  DNR_3.VEH_3 = DNR_3-VEH_3,
  DOX_3.VEH_3 = DOX_3-VEH_3,
  EPI_3.VEH_3 = EPI_3-VEH_3,
  MTX_3.VEH_3 = MTX_3-VEH_3,
  TRZ_3.VEH_3 = TRZ_3-VEH_3,
  DNR_24.VEH_24 =DNR_24-VEH_24,
  DOX_24.VEH_24= DOX_24-VEH_24,
  EPI_24.VEH_24= EPI_24-VEH_24,
  MTX_24.VEH_24= MTX_24-VEH_24,
  TRZ_24.VEH_24= TRZ_24-VEH_24,
  levels = mm)


fit2<- contrasts.fit(fit, contrasts=cm)

efit2 <- eBayes(fit2)

results = decideTests(efit2)

summary(results)

```
```{r}
plotSA(efit2, main="Mean-Variance trend for final model")
```

```{r Extract info from toptable and store as RDS}
V.DNR_3.top= topTable(efit2, coef=1, adjust.method="BH", number=Inf, sort.by="p")
V.DOX_3.top= topTable(efit2, coef=2, adjust.method="BH", number=Inf, sort.by="p")
V.EPI_3.top= topTable(efit2, coef=3, adjust.method="BH", number=Inf, sort.by="p")
V.MTX_3.top= topTable(efit2, coef=4, adjust.method="BH", number=Inf, sort.by="p")
V.TRZ_3.top= topTable(efit2, coef=5, adjust.method="BH", number=Inf, sort.by="p")
V.DNR_24.top= topTable(efit2, coef=6, adjust.method="BH", number=Inf, sort.by="p")
V.DOX_24.top= topTable(efit2, coef=7, adjust.method="BH", number=Inf, sort.by="p")
V.EPI_24.top= topTable(efit2, coef=8, adjust.method="BH", number=Inf, sort.by="p")
V.MTX_24.top= topTable(efit2, coef=9, adjust.method="BH", number=Inf, sort.by="p")
V.TRZ_24.top= topTable(efit2, coef=10, adjust.method="BH", number=Inf, sort.by="p")


# plot_filenames <- c("V.DNR_3.top","V.DOX_3.top","V.EPI_3.top","V.MTX_3.top",
#                     "V.TRZ_.top","V.DNR_24.top","V.DOX_24.top","V.EPI_24.top",
#                     "V.MTX_24.top","V.TRZ_24.top")
# plot_files <- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
#                     V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
#                     V.MTX_24.top,V.TRZ_24.top)

save_list <- list("DNR_3"=V.DNR_3.top,"DOX_3"=V.DOX_3.top,"EPI_3"=V.EPI_3.top,"MTX_3"=V.MTX_3.top,"TRZ_3"=V.TRZ_3.top,"DNR_24"=V.DNR_24.top,"DOX_24"=V.DOX_24.top,"EPI_24"=V.EPI_24.top,"MTX_24"= V.MTX_24.top, "TRZ_24"=V.TRZ_24.top)

saveRDS(save_list,"data/Final_four_data/re_analysis/Toptable_results.RDS")
```

### Volcano Plots
```{r Volcano plots}
volcanosig <- function(df, psig.lvl) {
    df <- df %>% 
    mutate(threshold = ifelse(adj.P.Val > psig.lvl, "A", ifelse(adj.P.Val <= psig.lvl & logFC<=0,"B","C")))
      # ifelse(adj.P.Val <= psig.lvl & logFC >= 0,"B", "C")))
    ##This is where I could add labels, but I have taken out
    # df <- df %>% mutate(genelabels = "")
    # df$genelabels[1:topg] <- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    ggrastr::geom_point_rast(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, "npc")), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression("Log"[2]*" FC"))+
    ylab(expression("-log"[10]*"P Value"))+
    scale_color_manual(values = c("black", "red","blue"))+
    theme_cowplot()+
    ylim(0,25)+
    xlim(-6,6)+
    theme(legend.position = "none",
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}

v1 <- volcanosig(V.DNR_3.top, 0.05)+ ggtitle("DNR 3 hour")
v2 <- volcanosig(V.DNR_24.top, 0.05)+ ggtitle("DNR 24 hour")+ylab("")
v3 <- volcanosig(V.DOX_3.top, 0.05)+ ggtitle("DOX 3 hour")
v4 <- volcanosig(V.DOX_24.top, 0.05)+ ggtitle("DOX 24 hour")+ylab("")
v5 <- volcanosig(V.EPI_3.top, 0.05)+ ggtitle("EPI 3 hour")
v6 <- volcanosig(V.EPI_24.top, 0.05)+ ggtitle("EPI 24 hour")+ylab("")
v7 <- volcanosig(V.MTX_3.top, 0.05)+ ggtitle("MTX 3 hour")
v8 <- volcanosig(V.MTX_24.top, 0.05)+ ggtitle("MTX 24 hour")+ylab("")
v9 <- volcanosig(V.TRZ_3.top, 0.05)+ ggtitle("TRZ 3 hour")
v10 <- volcanosig(V.TRZ_24.top, 0.05)+ ggtitle("TRZ 24 hour")+ylab("")

plot_grid(v1,v2,  rel_widths =c(1,1))
plot_grid(v3,v4,  rel_widths =c(1,1))
plot_grid(v5,v6,  rel_widths =c(1,1))
plot_grid(v7,v8,  rel_widths =c(1,1))
plot_grid(v9,v10,  rel_widths =c(1,1))

```

Making the median dataframes by time.  The files were saved as .csv for future use.

```{r bind all lists, eval=FALSE}

all_results <- bind_rows(save_list, .id = "group")

median_df <- all_results %>% 
  separate(group, into=c("trt","time"),sep = "_") %>% 
  pivot_wider(., id_cols=c(time,genes), names_from = trt, values_from = logFC) %>% 
  rowwise() %>% 
  mutate(median_ATAC_lfc= median(c_across(DNR:TRZ)))

median_3_lfc <-   median_df %>%
    dplyr::filter(time == "3") %>% 
  ungroup() %>% 
  dplyr::select(time, genes,median_ATAC_lfc) %>% 
  dplyr::rename("med_3h_lfc"=median_ATAC_lfc, "peak"=genes)
  

median_24_lfc <- median_df %>%
    dplyr::filter(time == "24") %>% 
  ungroup() %>% 
  dplyr::select(time, genes,median_ATAC_lfc) %>% 
  dplyr::rename("med_24h_lfc"=median_ATAC_lfc,, "peak"=genes)
  
write_csv(median_3_lfc, "data/Final_four_data/re_analysis/median_3_lfc_norm.csv")
write_csv(median_24_lfc, "data/Final_four_data/re_analysis/median_24_lfc_norm.csv")

```
Correlation of LFC between treatments
```{r LFC correlation}
FCmatrix_ff <- subset(efit2$coefficients)

colnames(FCmatrix_ff) <-
  c("DNR\n3h",
    "DOX\n3h",
    "EPI\n3h",
    "MTX\n3h",
    "TRZ\n3h",
    "DNR\n24h",
    "DOX\n24h",
    "EPI\n24h",
    "MTX\n24h",
    "TRZ\n24h"
     )


mat_col_ff <-
  data.frame(
    time = c(rep("3 hours", 5), rep("24 hours", 5)),
    class = (c(
      "AC", "AC", "AC", "nAC","nAC",  "AC", "AC", "AC", "nAC","nAC" 
    )))
rownames(mat_col_ff) <- colnames(FCmatrix_ff)

mat_colors_ff <-
  list(
    time = c("pink", "chocolate4"),
    class = c("yellow1", "lightgreen"))

names(mat_colors_ff$time) <- unique(mat_col_ff$time)
names(mat_colors_ff$class) <- unique(mat_col_ff$class)
# names(mat_colors_FC$TOP2i) <- unique(mat_col_FC$TOP2i)
corrFC_ff <- cor(FCmatrix_ff)

htanno_ff <-  HeatmapAnnotation(df = mat_col_ff, col = mat_colors_ff)
Heatmap(corrFC_ff, top_annotation = htanno_ff)
```

```{r example 3hr accesibility}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")

# all_results <- bind_rows(save_list, .id = "group")
DNR_3_top3_ff <- row.names(V.DNR_3.top[1:3,])

log_filt_ff <-
  filt_raw_counts_noY %>% 
   cpm(., log=TRUE)%>%
  as.data.frame() 
  
row.names(log_filt_ff) <- row.names(filt_raw_counts_noY)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DNR_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour DNR")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

DOX_3_top3_ff <- row.names(V.DOX_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DOX_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour DOX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

EPI_3_top3_ff <- row.names(V.EPI_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% EPI_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour EPI")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

MTX_3_top3_ff <- row.names(V.MTX_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% MTX_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour MTX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

TRZ_3_top3_ff <- row.names(V.TRZ_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% TRZ_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour TRZ")+
 scale_fill_manual(values = drug_pal)+
  theme_bw()
```




```{r example 24hr accesibility}
DNR_24_top3_ff <- row.names(V.DNR_24.top[1:3,])


log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DNR_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour DNR")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

DOX_24_top3_ff <- row.names(V.DOX_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DOX_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour DOX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

EPI_24_top3_ff <- row.names(V.EPI_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% EPI_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour EPI")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

MTX_24_top3_ff <- row.names(V.MTX_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% MTX_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour MTX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()


TRZ_24_top3_ff <- row.names(V.TRZ_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% TRZ_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour TRZ")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()
```


#### Examining around the adj. p value cutoff

```{r bottom slices 3 hour}
DNR_closest <-  V.DNR_3.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)


log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DNR_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("Bottom DAR in 3 hour DNR")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

DOX_closest <-  V.DOX_3.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DOX_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 3 hour DOX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

EPI_closest <-  V.EPI_3.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% EPI_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 3 hour EPI")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

MTX_closest <-  V.MTX_3.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% MTX_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 3 hour MTX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

TRZ_closest <-  V.TRZ_3.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% TRZ_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 3 hour TRZ")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

```



```{r bottom slices 24 hour}
DNR_closest <-  V.DNR_24.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)


log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DNR_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("Bottom DAR in 24 hour DNR")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

DOX_closest <-  V.DOX_24.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DOX_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 24 hour DOX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

EPI_closest <-  V.EPI_24.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% EPI_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 24 hour EPI")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

MTX_closest <-  V.MTX_24.top %>%
  dplyr::filter(adj.P.Val<0.05) %>% 
  slice_tail(n=5)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% MTX_closest$genes) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("bottom 5 DAR in 24 hour MTX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

# TRZ_closest <-  V.TRZ_24.top %>%
#   dplyr::filter(adj.P.Val<0.05) %>% 
#   slice_tail(n=5)
# 
# log_filt_ff %>% 
#   dplyr::filter(row.names(.) %in% TRZ_closest$genes) %>% 
#   mutate(Peak = row.names(.)) %>% 
#   pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
#   separate("sample", into = c("indv","trt","time")) %>% 
#   mutate(time=factor(time, levels = c("3h","24h"))) %>% 
#   mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
#   ggplot(., aes (x = time, y=counts))+
#   geom_boxplot(aes(fill=trt))+
#   facet_wrap(Peak~.)+
#   ggtitle("bottom 5 DAR in 24 hour TRZ")+
#   scale_fill_manual(values = drug_pal)+
#   theme_bw()

```

```{r export tables to excel, eval=FALSE}
toptable_results <- readRDS("data/Final_four_data/re_analysis/Toptable_results.RDS")
library(openxlsx)
output_dir <- "data/Final_four_data/re_analysis/ATAC_excel_outputs"

# Create directory if it doesn't exist
if (!dir.exists(output_dir)) {
  dir.create(output_dir, recursive = TRUE)
}

# Export each data frame to a separate .xlsx file
for (name in names(toptable_results)) {
  # Create a new workbook
  wb <- createWorkbook()
  
  # Add a worksheet (you can use the name as the sheet name too)
  addWorksheet(wb, name)
  
  # Write the data frame to the sheet
  writeData(wb, sheet = name, toptable_results[[name]])
  # Full file path using file.path()
  output_file <- file.path(output_dir, paste0(name, ".xlsx"))
  saveWorkbook(wb, file = output_file, overwrite = TRUE)
}

```

```{r extracting top DARs}
toptable_results <- readRDS("data/Final_four_data/re_analysis/Toptable_results.RDS")

all_results  <- toptable_results %>%
  imap(~ .x %>% tibble::rownames_to_column(var = "rowname") %>%
         mutate(source = .y)) %>%
  bind_rows()
all_results_list  <- toptable_results %>%
  imap(~ .x %>% tibble::rownames_to_column(var = "rowname") %>%
         mutate(source = .y)) 

sig_meta_and_loc <- all_results %>% 
  dplyr::filter(adj.P.Val<0.05) %>%  ## filter by pvalue
##Create parsed dataframe from "rowname" column, "genes column will keep id"
   separate(rowname, into = c("seqnames", "start", "end"), sep = "\\.", convert = TRUE)
###split into lists by DNR_3, etc..
sig_meta_and_loc_split <- split(sig_meta_and_loc, sig_meta_and_loc$source)
### Convert to Granges for downstream
sig_meta_and_loc_split_gr <- lapply(sig_meta_and_loc_split, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %>% select(-seqnames, -start, -end)
  )
})


notsig_meta_and_loc <- all_results %>% 
  dplyr::filter(adj.P.Val>0.05) %>% 
  separate(rowname, into = c("seqnames","start","end"), sep = "\\.", convert=TRUE)
    
notsig_meta_and_loc_split <- split(notsig_meta_and_loc, notsig_meta_and_loc$source)

notsig_meta_and_loc_split_gr <- lapply(notsig_meta_and_loc_split, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %>% select(-seqnames, -start, -end)
  )
})

all_DAR_regions <- all_results %>% 
   separate(rowname, into = c("seqnames", "start", "end"), sep = "\\.", convert = TRUE)
all_DAR_regions_list <- split(all_DAR_regions, all_DAR_regions$source)

all_DAR_regions_gr <- lapply(all_DAR_regions_list, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %>% dplyr::select(-seqnames, -start, -end)
  )
})
```

```{r exporting bed files, eval=FALSE}
# Folder with input BED files

output_dir <- "data/Final_four_data/re_analysis/motif_beds_centered"

# Create output folder if needed
dir.create(output_dir, showWarnings = FALSE)

# Loop through each BED file
for (name in names(sig_meta_and_loc_split_gr)) {
  gr <- sig_meta_and_loc_split_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered <- resize(gr, width = 200, fix = "center")
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, "sig_centered.bed")), format = "BED")
}


### not significant DAR regions for xstreme

for (name in names(notsig_meta_and_loc_split_gr)) {
  gr <- notsig_meta_and_loc_split_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered <- resize(gr, width = 200, fix = "center")
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, "notsig_centered.bed")), format = "BED")
}

```

### Examining regions between sig DARs and trt_time

```{r DAR venn}
sig_venn_list <- sapply(sig_meta_and_loc_split, function(x) x$genes)
sig_venn_3hr <- sig_venn_list[c("DOX_3","EPI_3", "DNR_3","MTX_3")]
sig_venn_24hr <- sig_venn_list[c("DOX_24","EPI_24", "DNR_24","MTX_24")]

ggVennDiagram::ggVennDiagram(sig_venn_3hr)


ggVennDiagram::ggVennDiagram(sig_venn_24hr)
saveRDS(sig_meta_and_loc_split,"Sig_meta")


sig_3hr_obj <- ggVennDiagram::Venn(sig_venn_list[c("DOX_3","EPI_3", "DNR_3","MTX_3")])
sig_24hr_obj <- ggVennDiagram::Venn(sig_venn_list[c("DOX_24","EPI_24", "DNR_24","MTX_24")])
sig_3hr_obj <- ggVennDiagram::process_data(sig_3hr_obj)
sig_24hr_obj <- ggVennDiagram::process_data(sig_24hr_obj)

sig_3hr_regions <- ggVennDiagram::venn_region(sig_3hr_obj)
sig_24hr_regions<- ggVennDiagram::venn_region(sig_24hr_obj)
sig_3hr_shared <- sig_3hr_obj$regionLabel$item[[11]]
sig_24hr_shared <-   sig_24hr_obj$regionLabel$item[[11]]

# saveRDS(sig_3hr_shared,"data/Final_four_data/re_analysis/AC_shared_3hour_DARs.RDS")
# saveRDS(sig_24hr_shared,"data/Final_four_data/re_analysis/AC_shared_24hour_DARs.RDS")


```

#### Proportion of peaks that are DARs

```{r 3hr DAR}
three_hour_df <- all_results %>% 
  dplyr::select(source, genes, logFC,adj.P.Val) %>% 
  mutate(sig_val=if_else(adj.P.Val<0.05,"sig","not_sig")) %>% 
  separate(source, into=c("trt","time"),sep="_") %>% 
  dplyr::filter(time=="3") %>% 
  mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ")))

twentyfour_hour_df <- all_results %>% 
  dplyr::select(source, genes, logFC,adj.P.Val) %>% 
  mutate(sig_val=if_else(adj.P.Val<0.05,"sig","not_sig")) %>% 
  separate(source, into=c("trt","time"),sep="_") %>% 
  dplyr::filter(time=="24") %>% 
  mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ")))

```

```{r 3 hour ggplot}
three_hour_df %>% 
  mutate(sig_val=factor(sig_val,levels = c("not_sig","sig"))) %>% 
  ggplot(., aes(x=trt,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle("Proportion of significant regions by 3 hours")+
  ylab("proportion")


twentyfour_hour_df %>% 
  mutate(sig_val=factor(sig_val,levels = c("not_sig","sig"))) %>% 
  ggplot(., aes(x=trt,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle("Proportion of significant regions by 24 hours")+
  ylab("proportion")
```



```{r doxbed, eval=FALSE}
DOX_sig <- sig_meta_and_loc_split[c("DOX_3", "DOX_24")]

DOXsig_up <- lapply(DOX_sig, function(x) dplyr::filter(x, logFC > 0))
names(DOXsig_up) <- paste0(names(DOXsig_up), "_up")

DOXsig_down <- lapply(DOX_sig, function(x) dplyr::filter(x, logFC < 0))
names(DOXsig_down) <- paste0(names(DOXsig_down), "_down")


DOXsig_up_gr <- lapply(DOXsig_up, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %>% select(-seqnames, -start, -end)
  )
})

DOXsig_down_gr <- lapply(DOXsig_down, function(sub_df) {
  GRanges(
    seqnames = sub_df$seqnames,
    ranges = IRanges(start = sub_df$start, end = sub_df$end),
    mcols = sub_df %>% select(-seqnames, -start, -end)
  )
})

output_dir <- "data/Final_four_data/re_analysis/motif_beds_centered"

# Create output folder if needed
dir.create(output_dir, showWarnings = FALSE)

# Loop through each BED file
for (name in names(DOXsig_down_gr)) {
  gr <- DOXsig_down_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered <- resize(gr, width = 200, fix = "center")
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, "DOXsig_down_centered.bed")), format = "BED")
}

# Loop through each BED file
for (name in names(DOXsig_up_gr)) {
  gr <- DOXsig_up_gr[[name]]
  
  # Recenter each region to 200 bp around its midpoint
  gr_centered <- resize(gr, width = 200, fix = "center")
  
  # Export to BED (auto converts to 0-based)
  export(gr_centered, con = file.path(output_dir, paste0(name, "DOXsig_up_centered.bed")), format = "BED")
}

```

```{r, eval=FALSE}
 filt_DOX24_notup <- all_results %>% 
  dplyr::filter (source=="DOX_24") %>% 
  dplyr::filter(!genes %in% DOXsig_up$DOX_24_up$genes) %>% 
  separate(rowname, into = c("seqnames", "start", "end"), sep = "\\.", convert = TRUE) 

filt_DOX24_notdown <- all_results %>% 
  dplyr::filter (source=="DOX_24") %>% 
  dplyr::filter(!genes %in% DOXsig_down$DOX_24_down$genes) %>% 
  separate(rowname, into = c("seqnames", "start", "end"), sep = "\\.", convert = TRUE) 

# Now convert to GRanges
filt_DOX24_notup_gr <- GRanges(
  seqnames = filt_DOX24_notup$seqnames,
  ranges = IRanges(start = filt_DOX24_notup$start, end = filt_DOX24_notup$end),
  mcols = filt_DOX24_notup %>% select(-seqnames, -start, -end)
)
filt_DOX24_notdown_gr <- GRanges(
  seqnames = filt_DOX24_notdown$seqnames,
  ranges = IRanges(start = filt_DOX24_notdown$start, end = filt_DOX24_notdown$end),
  mcols = filt_DOX24_notdown %>% select(-seqnames, -start, -end)
)
DOX_not_list_gr <- list("DOX24_notup"=filt_DOX24_notup_gr,"DOX24_notdown"=filt_DOX24_notdown_gr)
# for (name in names(DOX_not_list)) {
#   gr <- DOX_not_list[[name]]
#   
#   # Recenter each region to 200 bp around its midpoint
#   gr_centered <- resize(gr, width = 200, fix = "center")
#   
#   # Export to BED (auto converts to 0-based)
#   export(gr_centered, con = file.path(output_dir, paste0(name, "DOX24not_centered.bed")), format = "BED")
# }


```

```{r Genomic Features}
txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene
### maybe use annotatePeakInBatch from ChIPpeakAnno
# peakAnnoList_DOX_DAR <- lapply(all_DAR_regions_gr, annotatePeak, tssRegion =c(-2000,2000), TxDb=txdb)
peakAnnoList_DOX_DAR <- readRDS("data/Final_four_data/re_analysis/DOX_DAR_annotated_peaks_chipannno.RDS")
 # saveRDS, "data/Final_four_data/re_analysis/DOX_DAR_annotated_peaks_chipannno.RDS")
# filt_peakAnnoList_DOX_DAR <- lapply(sig_meta_and_loc_split_gr,annotatePeak, tssRegion =c(-2000,2000), TxDb=txdb)
# saveRDS(filt_peakAnnoList_DOX_DAR, "data/Final_four_data/re_analysis/filt_DOX_DAR_annotated_peaks_chipannno.RDS")
filt_peakAnnoList_DOX_DAR <- readRDS( "data/Final_four_data/re_analysis/filt_DOX_DAR_annotated_peaks_chipannno.RDS")

plotAnnoBar(peakAnnoList_DOX_DAR)+
  ggtitle ("Genomic Feature Distribution, all DAR no filtering\n should look identical")
plotAnnoBar(filt_peakAnnoList_DOX_DAR)+
  ggtitle ("Genomic Feature Distribution, Significant regions \n using adj.P.Val <0.05")

# annotated_peak_TSS_chipanno <-  peakAnnoList_DOX_DAR %>%
#   imap(~ .x %>% tibble::rownames_to_column(var = "rowname") %>%
#          mutate(source = .y)) 

```

```{r looking at proportion of DE genes that have DOX DAR}


toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") %>% 
  mutate(logFC = logFC*(-1))
peakAnnoList_DOX_DAR <- readRDS("data/Final_four_data/re_analysis/DOX_DAR_annotated_peaks_chipannno.RDS")
Assigned_genes_toPeak <- peakAnnoList_DOX_DAR$DOX_24 %>% as.data.frame() %>% 
  dplyr::select(mcols.genes,annotation, geneId, distanceToTSS) %>% 
  dplyr::rename("Peakid"=mcols.genes)


RNA_results <-
toplistall_RNA %>% 
  dplyr::select(time:logFC) %>% 
  tidyr::unite("sample",time, id) %>% 
  pivot_wider(., id_cols = c(ENTREZID,SYMBOL),names_from = sample, values_from = logFC) %>% 
  rename_with(~ str_replace(., "hours", "RNA"))


DOX24_degs <- toplistall_RNA %>%
  dplyr::select(time:logFC,adj.P.Val) %>% 
  dplyr::filter(id=="DOX") %>% 
  tidyr::unite("sample",time, id) %>% 
  dplyr::select(sample:SYMBOL,adj.P.Val) %>% 
  dplyr::filter(adj.P.Val<0.05) %>% 
  dplyr::filter(sample=="24_hours_DOX")

DOX3_degs <- toplistall_RNA %>%
  dplyr::select(time:logFC,adj.P.Val) %>% 
  dplyr::filter(id=="DOX") %>% 
  tidyr::unite("sample",time, id) %>% 
  dplyr::select(sample:SYMBOL,adj.P.Val) %>% 
  dplyr::filter(adj.P.Val<0.05) %>% 
  dplyr::filter(sample=="3_hours_DOX")

RNA_all_expressed <-toplistall_RNA %>% 
  dplyr::select(time:logFC,adj.P.Val) %>% 
  dplyr::filter(id=="DOX") %>% 
  dplyr::filter(time=="24_hours") %>% 
  tidyr::unite("sample",time, id) %>% 
  dplyr::select(ENTREZID, SYMBOL)  

Peak_gene_RNA_LFC <- Assigned_genes_toPeak %>% 
  left_join(., RNA_results, by =c("geneId"="ENTREZID"))


entrez_ids <- Assigned_genes_toPeak$geneId  


gene_info <- AnnotationDbi::select(
  org.Hs.eg.db,
  keys = entrez_ids,
  columns = c("SYMBOL"),
  keytype = "ENTREZID"
)
gene_info_collapsed <- gene_info %>%
  group_by(ENTREZID) %>%
  summarise(SYMBOL = paste(unique(SYMBOL), collapse = ","), .groups = "drop")

```
### Exploring DARs and expressed genes
#### 3 hour DOX DAR and expressed RNA genes 2kb
```{r 3hr expressed 2kb}
three_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-2000 & distanceToTSS<2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle("Expressed genes with and without DOX DARs within 2kb  at 3 hours ")+
  ylab("proportion")

filtered_df_3hr_exp <-
three_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-2000 & distanceToTSS<2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_3hr_exp <- table(filtered_df_3hr_exp$EXP_RNA, filtered_df_3hr_exp$sig_val)
contingency_table_3hr_exp

fisher.test(contingency_table_3hr_exp)


```

#### 3 hour DOX DAR and expressed RNA genes 20kb
```{r 3hr expressed 20kb}
three_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle(" DOX DARs with expressed genes at 3 hours within 20kb")+
  ylab("proportion")

filtered_df_3hr_exp20kb <-
three_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_3hr_exp20kb <- table(filtered_df_3hr_exp20kb$EXP_RNA, filtered_df_3hr_exp20kb$sig_val)
contingency_table_3hr_exp20kb

fisher.test(contingency_table_3hr_exp20kb)
```


#### 24 hour DOX DAR and expressed RNA genes 2kb
```{r 24hr expressed 2kb}
twentyfour_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-2000 & distanceToTSS<2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle("Expressed genes with and without DOX DARs within 2kb  at 24 hours ")+
  ylab("proportion")

filtered_df_24hr_exp <-
twentyfour_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-2000 & distanceToTSS<2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_24hr_exp <- table(filtered_df_24hr_exp$EXP_RNA, filtered_df_24hr_exp$sig_val)
contingency_table_24hr_exp

fisher.test(contingency_table_24hr_exp)


```

#### 24 hour DOX DAR and expressed RNA genes 20kb
```{r 24hr expressed 20kb}
twentyfour_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
  ggplot(., aes(x=sig_val,fill=EXP_RNA))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle(" DOX DARs with expressed genes at 24 hours within 20kb")+
  ylab("proportion")

filtered_df_24hr_exp20kb <-
twentyfour_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(EXP_RNA=if_else(geneId %in% RNA_all_expressed$ENTREZID,"exp","not_exp")) %>% 
  dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_24hr_exp20kb <- table(filtered_df_24hr_exp20kb$EXP_RNA, filtered_df_24hr_exp20kb$sig_val)
contingency_table_24hr_exp20kb

fisher.test(contingency_table_24hr_exp20kb)
```


#### 3 hour DAR and DEGs
Looking at DARs within 2kb of TSS
```{r  2kb 3hr DEGnDAR}
three_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(DEG=if_else(geneId %in% DOX3_degs$ENTREZID,"DEG","not_DEG")) %>% 
  dplyr::filter(distanceToTSS>-2000 & distanceToTSS<2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
  ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
   ggtitle(" DOX DARs and not-DARs within 2kb of TSS of DEGs at 3 hours")+
  ylab("proportion")

filtered_df_3hr <-
three_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(Assigned_genes_toPeak, by = c("genes" = "Peakid")) %>% 
  mutate(DEG = if_else(geneId %in% DOX3_degs$ENTREZID, "DEG", "not_DEG")) %>% 
  dplyr::filter(distanceToTSS> -2000 & distanceToTSS < 2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_3hr <- table(filtered_df_3hr$DEG, filtered_df_3hr$sig_val)
contingency_table_3hr

fisher.test(contingency_table_3hr)
```
Looking at DARs within 20kb of TSS
```{r  20kb 3hr DEGnDAR}
three_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(DEG=if_else(geneId %in% DOX3_degs$ENTREZID,"DEG","not_DEG")) %>% 
  dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle(" DOX DARs and not-DARs within 20kb of TSS of DEGs at 3 hours")+
  ylab("proportion")

filtered_df_3hr20k <-
three_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(Assigned_genes_toPeak, by = c("genes" = "Peakid")) %>% 
  mutate(DEG = if_else(geneId %in% DOX3_degs$ENTREZID, "DEG", "not_DEG")) %>% 
  dplyr::filter(distanceToTSS> -20000 & distanceToTSS < 20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_3hr20k <- table(filtered_df_3hr20k$DEG, filtered_df_3hr20k$sig_val)
contingency_table_3hr20k

fisher.test(contingency_table_3hr20k)
```

Looking at DARs with all distances to TSS

```{r  nodist 3hr DEGnDAR}
three_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(DEG=if_else(geneId %in% DOX3_degs$ENTREZID,"DEG","not_DEG")) %>% 
  # dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 

 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
   ggtitle("DOX DARs and not-DARs within all distance of TSS of DEGs at 3 hours")+
  ylab("proportion")

filtered_df_3hrnodist <-
three_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(Assigned_genes_toPeak, by = c("genes" = "Peakid")) %>% 
  mutate(DEG = if_else(geneId %in% DOX3_degs$ENTREZID, "DEG", "not_DEG")) %>% 
  # dplyr::filter(distanceToTSS> -20000 & distanceToTSS < 20000) %>% 
  mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_3hrnodist <- table(filtered_df_3hrnodist$DEG, filtered_df_3hrnodist$sig_val)
contingency_table_3hrnodist

fisher.test(contingency_table_3hrnodist)
```



#### 24 hour DAR and DEGs
Looking at DARs within 2kb of TSS
```{r  2kb 24hr DEGnDAR}
twentyfour_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(DEG=if_else(geneId %in% DOX24_degs$ENTREZID,"DEG","not_DEG")) %>% 
  dplyr::filter(distanceToTSS>-2000 & distanceToTSS<2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle(" DOX DARs and not-DARs within 2kb of TSS of DEGs at 24 hours")+
  ylab("proportion")

filtered_df_24hr <-
twentyfour_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(Assigned_genes_toPeak, by = c("genes" = "Peakid")) %>% 
  mutate(DEG = if_else(geneId %in% DOX24_degs$ENTREZID, "DEG", "not_DEG")) %>% 
  dplyr::filter(distanceToTSS> -2000 & distanceToTSS < 2000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_24hr <- table(filtered_df_24hr$DEG, filtered_df_24hr$sig_val)
contingency_table_24hr

fisher.test(contingency_table_24hr)
```
Looking at DARs within 20kb of TSS
```{r  20kb 24hr DEGnDAR}
twentyfour_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(DEG=if_else(geneId %in% DOX24_degs$ENTREZID,"DEG","not_DEG")) %>% 
  dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
  mutate(sig_val=factor(sig_val,levels = c("sig","not_sig"))) %>% 
  ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
   ggtitle(" DOX DARs and not-DARs within 20kb of TSS of DEGs at 24 hours")+
  ylab("proportion")

filtered_df_24hr20k <-
twentyfour_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(Assigned_genes_toPeak, by = c("genes" = "Peakid")) %>% 
  mutate(DEG = if_else(geneId %in% DOX24_degs$ENTREZID, "DEG", "not_DEG")) %>% 
  dplyr::filter(distanceToTSS> -20000 & distanceToTSS < 20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))



contingency_table_24hr20k <- table(filtered_df_24hr20k$DEG, filtered_df_24hr20k$sig_val)
contingency_table_24hr20k

fisher.test(contingency_table_24hr20k)
```

Looking at DARs with all distances to TSS

```{r  nodist 24hr DEGnDAR}
twentyfour_hour_df %>% 
  dplyr::filter(trt=="DOX") %>% 
  left_join(., Assigned_genes_toPeak, by=c("genes"="Peakid")) %>% 
  mutate(DEG=if_else(geneId %in% DOX24_degs$ENTREZID,"DEG","not_DEG")) %>% 
  # dplyr::filter(distanceToTSS>-20000 & distanceToTSS<20000) %>% 
   mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig"))) %>% 
 ggplot(., aes(x=DEG,fill=sig_val))+
  geom_bar(position="fill")+
  theme_bw()+
  ggtitle(" DOX DARs and not-DARs associated with TSS DEGs at 24 hours")+
  ylab("proportion")

filtered_df_24hrnodist <-
twentyfour_hour_df %>% 
  dplyr::filter(trt == "DOX") %>% 
  left_join(Assigned_genes_toPeak, by = c("genes" = "Peakid")) %>% 
  mutate(DEG = if_else(geneId %in% DOX24_degs$ENTREZID, "DEG", "not_DEG")) %>% 
  # dplyr::filter(distanceToTSS> -20000 & distanceToTSS < 20000) %>% 
  mutate(sig_val = factor(sig_val, levels = c( "sig","not_sig")))


contingency_table_24hrnodist <- table(filtered_df_24hrnodist$DEG, filtered_df_24hrnodist$sig_val)
contingency_table_24hrnodist

fisher.test(contingency_table_24hrnodist)
```
