---
title: "Enhancer_files"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```
####Loading
```{r package loading, message=FALSE, warning=FALSE}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)
library(plyranges)
library(genomation)

```

Data loading

```{r data loading}

toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") 
S13Table <- read.csv( "data/other_papers/S13Table_Matthews2024.csv",row.names = 1)
##14021

EAR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="EAR") %>% 
  dplyr::select(ENTREZID) %>% 
  mutate(ENTREZID= as.character(ENTREZID))
ESR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="ESR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))
LR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="LR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID=as.character(ENTREZID))
NR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="NR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))

# exp_neargene_table <- read_delim("data/n45_bedfiles/exp_neargene_table.tsv", 
#     delim = "\t", escape_double = FALSE, 
#     trim_ws = TRUE)

TSS_NG_data <- read_delim("data/n45_bedfiles/TSS_NG_data.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

col_ng_peak <- read.delim("data/n45_bedfiles/TSS_NG_data_collapsed_peaks.tsv")

peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
# list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)

peakAnnoList_3_n45 <- readRDS("data/peakAnnoList_3_n45.RDS")
peakAnnoList_24_n45<- readRDS("data/peakAnnoList_24_n45.RDS")

all_peak_gr <- as.GRanges(peakAnnoList_24_n45$background)

Joined_tss_neargene <- all_peak_gr %>%
  as.data.frame() %>% 
  dplyr::select(seqnames:id,geneId,distanceToTSS) %>% 
  dplyr::filter(!grepl("chrX",id)& !grepl("chrY",id))%>%
  dplyr::rename("close_geneId"="geneId") %>% 
  mutate(start=start+1) %>% 
  left_join(., TSS_NG_data, by=c("id"="peakid", "start"="start","end"="end")) %>%
  dplyr::select(seqnames.x:close_geneId,distanceToTSS, entrezgene_id:dist_to_NG)
  


  
### THESE ARE DAR peaks (not total peaks)

# DOX_3hr <- as.data.frame(peakAnnoList_3_n45$DOX_3_n45)
# DOX_3hr_gr <- as.GRanges(peakAnnoList_3_n45$DOX_3_n45)
# EPI_3hr <- as.data.frame(peakAnnoList_3_n45$EPI_3_n45)
# EPI_3hr_gr <- as.GRanges(peakAnnoList_3_n45$EPI_3_n45)
# DNR_3hr <- as.data.frame(peakAnnoList_3_n45$DNR_3_n45)
# DNR_3hr_gr <- as.GRanges(peakAnnoList_3_n45$DNR_3_n45)
# MTX_3hr <- as.data.frame(peakAnnoList_3_n45$MTX_3_n45)
# MTX_3hr_gr <- as.GRanges(peakAnnoList_3_n45$MTX_3_n45)
# # # TRZ_3hr <- as.data.frame(peakAnnoList_3_n45$TRZ_3_n45)
# # TRZ_3hr_gr <- GRanges(TRZ_3hr)
# DOX_3_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% DOX_3hr$id)
# EPI_3_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% EPI_3hr$id)
# DNR_3_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% DNR_3hr$id)
# MTX_3_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% MTX_3hr$id)

### THESE ARE DAR peaks (not total peaks)
# 
# DOX_24hr <- as.data.frame(peakAnnoList_24_n45$DOX_24_n45)
# DOX_24hr_gr <- as.GRanges(peakAnnoList_24_n45$DOX_24_n45)
# EPI_24hr <- as.data.frame(peakAnnoList_24_n45$EPI_24_n45)
# EPI_24hr_gr <-as.GRanges(peakAnnoList_24_n45$EPI_24_n45)
# DNR_24hr <- as.data.frame(peakAnnoList_24_n45$DNR_24_n45)
# DNR_24hr_gr <- as.GRanges(peakAnnoList_24_n45$DNR_24_n45)
# MTX_24hr <- as.data.frame(peakAnnoList_24_n45$MTX_24_n45)
# MTX_24hr_gr <- as.GRanges(peakAnnoList_24_n45$MTX_24_n45)
# # TRZ_24hr <- as.data.frame(peakAnnoList_24_n45$TRZ_24_n45)
# # TRZ_24hr_gr <- as.GRanges(peakAnnoList_24_n45$TRZ_24_n45)
# ## THESE ARE DAR peaks (not total peaks)
# DOX_24_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% DOX_24hr$id)
# EPI_24_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% EPI_24hr$id)
# DNR_24_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% DNR_24hr$id)
# MTX_24_peak_list <-  col_ng_peak %>%
#   # dplyr::filter(dist_to_NG<20000) %>%
#     dplyr::filter(peakid %in% MTX_24hr$id)


EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)


ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr <- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)

EAR_peak_list_all <-  TSS_NG_data %>%
    dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(MRC="EAR") 

ESR_peak_list_all <- TSS_NG_data %>%
    dplyr::filter(peakid %in% ESR_df$id) %>% 
    mutate(MRC="ESR")

LR_peak_list_all <- TSS_NG_data %>%
    dplyr::filter(peakid %in% LR_df$id) %>% 
    mutate(MRC="LR")

NR_peak_list_all <- TSS_NG_data %>%
    dplyr::filter(peakid %in% NR_df$id) %>% 
    mutate(MRC="NR")

peak_list_all_MRC <- EAR_peak_list_all %>% 
  rbind(ESR_peak_list_all) %>% 
  rbind(LR_peak_list_all) %>% 
  rbind(NR_peak_list_all)


gr_EAR_peak_list<-  TSS_NG_data %>%
    dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(MRC="EAR") %>%
  distinct() %>% 
  GRanges()

gr_ESR_peak_list <- TSS_NG_data %>%
    dplyr::filter(peakid %in% ESR_df$id) %>% 
    mutate(MRC="ESR")%>% 
 distinct() %>% 
  GRanges()

gr_LR_peak_list <- TSS_NG_data %>%
    dplyr::filter(peakid %in% LR_df$id) %>% 
    mutate(MRC="LR")%>% 
  distinct() %>% 
  GRanges()

gr_NR_peak_list <- TSS_NG_data %>%
    dplyr::filter(peakid %in% NR_df$id) %>% 
    mutate(MRC="NR")%>% 
  distinct() %>% 
  GRanges()

toplist_ATAC <- readRDS("data/toplist_n45.RDS")

# toplist_DOX24_ATAC <- toplist_ATAC %>% 
#   dplyr::filter(trt=="DOX" & time=="24 hours")
median_24_lfc <- readRDS("data/median_24_lfc.RDS") %>% ungroup()
median_3_lfc <- readRDS("data/median_3_lfc.RDS") %>% ungroup()


EAR_peak_list_20k <-  TSS_NG_data %>%
  dplyr::filter(dist_to_NG<20000) %>% 
              dplyr::filter(dist_to_NG> -20000) %>%
  #  dplyr::filter(dist_to_NG>=0) %>%
    dplyr::filter(peakid %in% EAR_df$id) %>% 
mutate(ENTREZID=as.character(entrezgene_id)) %>% 
  dplyr::select(peakid, entrezgene_id:ENTREZID) %>% 
  distinct() %>% 
  mutate(MRC="EAR")

ESR_peak_list_20k <- TSS_NG_data %>%
   dplyr::filter(dist_to_NG<20000) %>% 
              dplyr::filter(dist_to_NG> -20000) %>%
  dplyr::filter(peakid %in% ESR_df$id) %>%
  mutate(ENTREZID=as.character(entrezgene_id)) %>% 
  dplyr::select(peakid, entrezgene_id:ENTREZID) %>% 
   distinct() %>% 
    mutate(MRC="ESR")

LR_peak_list_20k <- TSS_NG_data %>%
   dplyr::filter(dist_to_NG<20000) %>% 
              dplyr::filter(dist_to_NG> -20000) %>%
  dplyr::filter(peakid %in% LR_df$id) %>%
  mutate(ENTREZID=as.character(entrezgene_id)) %>% 
  dplyr::select(peakid, entrezgene_id:ENTREZID) %>% 
   distinct() %>% 
    mutate(MRC="LR")

NR_peak_list_20k <- TSS_NG_data %>%
   dplyr::filter(dist_to_NG<20000) %>% 
              dplyr::filter(dist_to_NG> -20000) %>%
  dplyr::filter(peakid %in% NR_df$id) %>%
  mutate(ENTREZID=as.character(entrezgene_id)) %>% 
  dplyr::select(peakid, entrezgene_id:ENTREZID) %>% 
   distinct() %>% 
    mutate(MRC="NR")

##master 20kb neargene response peaklist
peak_list_20k_MRC <- EAR_peak_list_20k %>% 
  rbind(ESR_peak_list_20k) %>% 
  rbind(LR_peak_list_20k) %>% 
  rbind(NR_peak_list_20k)

n45_fullpeaks_gr <- readBed("data/n45_bedfiles/DAR_DEG_background.bed")

Top2b_peaks_gr <- readBed("data/n45_bedfiles/TOP2B_CM.bed")
###file for ATACseq replicability
ENCFF544RZR <-readRDS ("data/n45_bedfiles/ENCFF533RZR_gr.RDS")
# ENCFFFF544RZR_bed <- read_delim("data/n45_bedfiles/ENCFF134HBK.bed", 
#      delim = "\t", escape_double = FALSE, 
#      col_names = FALSE, trim_ws = TRUE)
#  ENCFF544RZR <- ENCFF544RZR_bed %>% 
#    rename(X1="chr",
#          X2="start",
#          X3="end",
#          X10="length") %>%
#    GRanges()
 
Snyder_overlaps <- join_overlap_intersect(n45_fullpeaks_gr,ENCFF544RZR)
# 84725
# length(find_overlaps(ENCFF544RZR,n45_fullpeaks_gr))
# test <- (subsetByOverlaps(n45_fullpeaks_gr,ENCFF544RZR))
# test %>% 
#   as.data.frame() %>% 
#   distinct(name)
# head(Snyder_overlaps)
toplist_ATAC <- readRDS("data/toplist_n45.RDS")

# toplist_DOX24_ATAC <- toplist_ATAC %>% 
#   dplyr::filter(trt=="DOX" & time=="24 hours")

```

### MRC venns

These are Venns using the Neargenes between each set of peaks.

This is similar to the data on the TSS_ATAC_MRC_DAR_FC_RNA.html .  That one looked only at peaks that were within 20kb of the near-gene TSS.
```{r Venns of data}
ggVennDiagram::ggVennDiagram(list(unique(EAR_peak_list_all$entrezgene_id), 
                                  unique(ESR_peak_list_all$entrezgene_id),  
                                  unique(LR_peak_list_all$entrezgene_id), 
                                  unique(NR_peak_list_all$entrezgene_id)),
                             category.names = c("EAR_NG","ESR_NG","LR_NG","NR_NG"),
              show_intersect = FALSE,
              set_color = "black",
              label = "count",
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = 1,
              label_color = "black",
              edge_lty = "solid", set_size = 4)+
  ggtitle("Near gene distribution for all peaks")+
  scale_x_continuous(expand = expansion(mult = .2))



ggVennDiagram::ggVennDiagram(list(as.character(EAR_peak_list_all$entrezgene_id),EAR_RNA$ENTREZID),
              category.names = c("EAR_ATAC","EAR_RNA"), label = "count")+
  labs(title = "EAR ATAC-neargenes and\n expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.0), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))+
  

ggVennDiagram::ggVennDiagram(list(as.character(ESR_peak_list_all$entrezgene_id),ESR_RNA$ENTREZID),
              category.names = c("ESR_ATAC","ESR_RNA"), label = "count")+
  labs(title = "ESRATAC-neargenes and\n expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.0), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))+

ggVennDiagram::ggVennDiagram(list(as.character(LR_peak_list_all$entrezgene_id),LR_RNA$ENTREZID),
              category.names = c("LR_ATAC","LR_RNA"), label = "count")+
  labs(title = "LR ATAC-neargenes and\n expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.0), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))+

ggVennDiagram::ggVennDiagram(list(as.character(NR_peak_list_all$entrezgene_id),NR_RNA$ENTREZID),
              category.names = c("NR_ATAC","NR_RNA"), label = "count")+
  labs(title = "NR ATAC-neargenes and\n expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.0), hjust = 0.5))+ 
  scale_x_continuous(expand = expansion(mult = .2))


```

### Enhancers and Response clusters


First:  obtained a list of cis Regulatory Elements from Encode Screen
[(https://screen.encodeproject.org/#)]


```{r  enhancers of the heart}
# BiocManager::install("plyranges")
# enhancers_HLV_46F <- genomation::readBed("C:/Users/renee/Downloads/Supplements folde manuscriptr/ENCODE/heart_left_ventricle_tissue_female_adult_46_years.enhancers.bed")
cREs_HLV_46F <- genomation::readBed("data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed")

# cREs_HLV_53F <- genomation::readBed("data/enhancerdata/ENCFF417JSF_ENCFF651XRK_ENCFF320IPT_ENCFF440RUS.7group.bed") 
 
NR_cREs <- join_overlap_intersect(gr_NR_peak_list,cREs_HLV_46F)
LR_cREs <- join_overlap_intersect(gr_LR_peak_list,cREs_HLV_46F)
ESR_cREs <- join_overlap_intersect(gr_ESR_peak_list,cREs_HLV_46F)
EAR_cREs <- join_overlap_intersect(gr_EAR_peak_list,cREs_HLV_46F)

# NR_cREs_all <- join_overlap_intersect(NR_df_gr,cREs_HLV_46F)
# LR_cREs_all <- join_overlap_intersect(LR_df_gr,cREs_HLV_46F)
# ESR_cREs_all <- join_overlap_intersect(ESR_df_gr,cREs_HLV_46F)
# EAR_cREs_all <- join_overlap_intersect(EAR_df_gr,cREs_HLV_46F)

# Whole_peaks <- join_overlap_intersect(n45_fullpeaks_gr, cREs_HLV_46F)

NR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="NR", per=n/(length(unique(NR_cREs$peakid)))) %>% 
  kable(., caption=paste0("No Response peaks overlapping cREs n= ", length(unique(NR_cREs$peakid)))) %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)


LR_cREs%>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="LR", per=n/(length(unique(LR_cREs$peakid)))) %>% 
kable(., caption=paste0("Late Response peaks overlapping cREs n= ", length(unique(LR_cREs$peakid)))) %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)

ESR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="ESR", per=n/(length(unique(ESR_cREs$peakid)))) %>% 
kable(., caption=paste0("Early-sustained Response peaks overlapping cREs n= ", length(unique(ESR_cREs$peakid)))) %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)


EAR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="EAR", per=n/(length(unique(EAR_cREs$peakid)))) %>% 
kable(., caption=paste0("Early Response peaks overlapping cREs n= ", length(unique(EAR_cREs$peakid)))) %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)


```

### Visualization of categories:

First, filtering cRE set by type to include 'CTCF-only,CTCF-bound', 'PLS', 'PLS,CTCF-bound','dELS', 'dELS,CTCF-bound', 'pELS', 'pELS,CTCF-bound'.

Second, reclassify (new column) to group the CTCF-bound into their respective groups so only 4 groups, ("CTCF-only","PLS","dELS", "pELS") are created.

Third, left-join the median LFC for both 3 hour and 24 hour data to data set and boxplot by group.

Fourth, rbind two more data frames to help with ggplot visualization

PLS= promoter like sequences
pELS= proximal enhancer like sequences
dELS=distal enhancer like sequences
#### EAR
```{r  EAR cRE 3 and 24 hours}
EAR_24h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(type="all_EAR") %>% 
  left_join(.,(median_24_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))
EAR_cRE_peak_list <- EAR_cREs %>%
  as.data.frame() %>%
  distinct(peakid)

not_EAR_24h_cREs <-   median_24_lfc %>%  
  dplyr::filter(peak %in% EAR_df$id) %>%
  dplyr::filter(!peak %in% EAR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

EAR_3h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(type="all_EAR") %>% 
  left_join(.,(median_3_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))

not_EAR_3h_cREs <-   median_3_lfc %>%  
  dplyr::filter(peak %in% EAR_df$id) %>% 
  dplyr::filter(!peak %in% EAR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

EAR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_24_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(EAR_24h_peak_list) %>% 
  rbind(not_EAR_24h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_EAR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(" 24 hour EAR n = ",length(unique(EAR_cREs$peakid))," out of ", length(unique(EAR_df$id)), " total peaks (",sprintf("%.1f",length(unique(EAR_cREs$peakid))/length(unique(EAR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

EAR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_3_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(EAR_3h_peak_list) %>% 
  rbind(not_EAR_3h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_EAR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("3 hour EAR n = ",length(unique(EAR_cREs$peakid))," out of ", length(unique(EAR_df$id)), " total peaks (",sprintf("%.1f",length(unique(EAR_cREs$peakid))/length(unique(EAR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

```

#### ESR

```{r  ESR cRE 3 and 24 hours}
ESR_24h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% ESR_df$id) %>% 
  mutate(type="all_ESR") %>% 
  left_join(.,(median_24_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))
ESR_cRE_peak_list <- ESR_cREs %>%
  as.data.frame() %>%
  distinct(peakid)

not_ESR_24h_cREs <-   median_24_lfc %>%  
  dplyr::filter(peak %in% ESR_df$id) %>%
  dplyr::filter(!peak %in% ESR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

ESR_3h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% ESR_df$id) %>% 
  mutate(type="all_ESR") %>% 
  left_join(.,(median_3_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))

not_ESR_3h_cREs <-   median_3_lfc %>%  
  dplyr::filter(peak %in% ESR_df$id) %>% 
  dplyr::filter(!peak %in% ESR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

ESR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_24_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(ESR_24h_peak_list) %>% 
  rbind(not_ESR_24h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_ESR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(" 24 hour ESR n = ",length(unique(ESR_cREs$peakid))," out of ", length(unique(ESR_df$id)), " total peaks (",sprintf("%.1f",length(unique(ESR_cREs$peakid))/length(unique(ESR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

ESR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_3_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(ESR_3h_peak_list) %>% 
  rbind(not_ESR_3h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_ESR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("3 hour ESR n = ",length(unique(ESR_cREs$peakid))," out of ", length(unique(ESR_df$id)), " total peaks (",sprintf("%.1f",length(unique(ESR_cREs$peakid))/length(unique(ESR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

```
#### LR
```{r  LR cRE 3 and 24 hours}
LR_24h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% LR_df$id) %>% 
  mutate(type="all_LR") %>% 
  left_join(.,(median_24_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))
LR_cRE_peak_list <- LR_cREs %>%
  as.data.frame() %>%
  distinct(peakid)

not_LR_24h_cREs <-   median_24_lfc %>%  
  dplyr::filter(peak %in% LR_df$id) %>%
  dplyr::filter(!peak %in% LR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

LR_3h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% LR_df$id) %>% 
  mutate(type="all_LR") %>% 
  left_join(.,(median_3_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))

not_LR_3h_cREs <-   median_3_lfc %>%  
  dplyr::filter(peak %in% LR_df$id) %>% 
  dplyr::filter(!peak %in% LR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

LR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_24_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(LR_24h_peak_list) %>% 
  rbind(not_LR_24h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_LR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(" 24 hour LR n = ",length(unique(LR_cREs$peakid))," out of ", length(unique(LR_df$id)), " total peaks (",sprintf("%.1f",length(unique(LR_cREs$peakid))/length(unique(LR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

LR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_3_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(LR_3h_peak_list) %>% 
  rbind(not_LR_3h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_LR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("3 hour LR n = ",length(unique(LR_cREs$peakid))," out of ", length(unique(LR_df$id)), " total peaks (",sprintf("%.1f",length(unique(LR_cREs$peakid))/length(unique(LR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

```

#### NR

```{r  NR cRE 3 and 24 hours}
NR_24h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% NR_df$id) %>% 
  mutate(type="all_NR") %>% 
  left_join(.,(median_24_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))
NR_cRE_peak_list <- NR_cREs %>%
  as.data.frame() %>%
  distinct(peakid)

not_NR_24h_cREs <-   median_24_lfc %>%  
  dplyr::filter(peak %in% NR_df$id) %>%
  dplyr::filter(!peak %in% NR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

NR_3h_peak_list <-  col_ng_peak %>% 
  distinct(peakid) %>% 
  dplyr::filter(peakid %in% NR_df$id) %>% 
  mutate(type="all_NR") %>% 
  left_join(.,(median_3_lfc %>% 
                 dplyr::select(peak,median)),by = c("peakid"="peak"))

not_NR_3h_cREs <-   median_3_lfc %>%  
  dplyr::filter(peak %in% NR_df$id) %>% 
  dplyr::filter(!peak %in% NR_cRE_peak_list$peakid) %>% 
  dplyr::rename("peakid"="peak") %>% 
  mutate(type = "not_cRE_peaks") %>% 
  dplyr::select(peakid, median, type)

NR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_24_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(NR_24h_peak_list) %>% 
  rbind(not_NR_24h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_NR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0(" 24 hour NR n = ",length(unique(NR_cREs$peakid))," out of ", length(unique(NR_df$id)), " total peaks (",sprintf("%.1f",length(unique(NR_cREs$peakid))/length(unique(NR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

NR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., median_3_lfc, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(median,peakid,type) %>%
  distinct() %>% 
  rbind(NR_3h_peak_list) %>% 
  rbind(not_NR_3h_cREs) %>% 
  mutate(direction=if_else(median>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only","not_cRE_peaks","all_NR"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("3 hour NR n = ",length(unique(NR_cREs$peakid))," out of ", length(unique(NR_df$id)), " total peaks (",sprintf("%.1f",length(unique(NR_cREs$peakid))/length(unique(NR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

```


| Cluster | eREs                                           | not cREs                                                                |
|------------------------|------------------------|------------------------|
| NR      | `r length(unique(NR_cREs$peakid))` | `r length(unique(NR_df$id))-length(unique(NR_cREs$peakid))`  |
| LR      | `r length(unique(LR_cREs$peakid))` | `r length(unique(LR_df$id))-length(unique(LR_cREs$peakid))` |
| ESR     | `r length(unique(ESR_cREs$peakid))` | `r length(unique(ESR_df$id))-length(unique(ESR_cREs$peakid))` |
| EAR     | `r length(unique(EAR_cREs$peakid))` | `r length(unique(EAR_df$id))-length(unique(EAR_cREs$peakid))` |



```{r making enhancer NG sets}
# EAR_20k_peaks_gr <- GRanges(EAR_peak_list_20k)
# ESR_20k_peaks_gr <- GRanges(ESR_peak_list_20k)
# LR_20k_peaks_gr <- GRanges(LR_peak_list_20k)
# NR_20k_peaks_gr <- GRanges(NR_peak_list_20k)
# 
# EAR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(EAR_20k_peaks_gr,enhancers_HLV_46F))
# ESR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(ESR_20k_peaks_gr,enhancers_HLV_46F))
# LR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(LR_20k_peaks_gr,enhancers_HLV_46F))
# NR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(NR_20k_peaks_gr,enhancers_HLV_46F))
# saveRDS(EAR_enh_peaks_20k,"data/enhancerdata/EAR_enh_peaks_20k.RDS")
# saveRDS(ESR_enh_peaks_20k,"data/enhancerdata/ESR_enh_peaks_20k.RDS")
# saveRDS(LR_enh_peaks_20k,"data/enhancerdata/LR_enh_peaks_20k.RDS")
# saveRDS(NR_enh_peaks_20k,"data/enhancerdata/NR_enh_peaks_20k.RDS")
### making a set that is with all peaks in motif
# EAR_enh_peaks_20k <- subsetByOverlaps(NR_df_gr,enhancers_HLV_46F)

```


###### noodling around
```{r crazy enhancergwas, eval=FALSE, include=FALSE}

fullOL_medtable_3h <- fullOL %>% 
as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
left_join(., median_3_lfc, by =c("id"="peak")) %>% 
  dplyr::select(id, annotation, type, median) %>% 
  mutate(status=if_else(median>=0,"open","closed")) 

fullOL_medtable_3h %>% 
 mutate(status=factor(status, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=median))+
  geom_boxplot(aes(fill=status)) +
  scale_fill_discrete()+
  ggtitle("All peaks and median logFC")+
  # facet_wrap(~direction)+
  theme_bw()

```

### Top2b

```{r Top2b peaks}

### created the marked dataframe with mRC
Peak_MRC_table <- TSS_NG_data %>%
    distinct(peakid, .keep_all = TRUE) %>% 
    mutate(mrc=if_else(peakid %in% EAR_df$id, "EAR",
                     if_else(peakid %in% ESR_df$id,"ESR",
                             if_else(peakid %in% LR_df$id,"LR",
                                     if_else(peakid %in% NR_df$id,"NR","not_mrc")))))%>%
    dplyr::select(peakid:end,entrezgene_id:mrc) 
  
##overlapping and intersection top2b peaks with my peaks
top2Boverlap <- join_overlap_intersect(all_peak_gr,Top2b_peaks_gr)

### TAking the list of peaks dataframe by MRC and joining with marked group above so neargenes by TSS are joined thenchecking for distance to neargene across categories

peak_list_all_MRC %>% 
  dplyr::filter(peakid %in% top2Boverlap$id) %>% 
  dplyr::select(peakid,MRC) %>% 
  left_join(.,Peak_MRC_table, by=c("peakid"="peakid")) %>% 
  ggplot(., aes(x=mrc,y=dist_to_NG))+
  geom_boxplot()
peak_list_all_MRC %>% 
  dplyr::filter(peakid %in% top2Boverlap$id) %>% 
  dplyr::select(peakid,MRC) %>% 
  left_join(.,median_24_lfc, by=c("peakid"="peak")) %>% 
  ggplot(., aes(x=MRC,y=median))+
  geom_boxplot()  +
  theme_bw()+
  ggtitle("24 hr median logFC across response peaks overlapped with TOP2B")+
  coord_cartesian(ylim= c(-3.5,2))

peak_list_all_MRC %>% 
  dplyr::filter(peakid %in% top2Boverlap$id) %>% 
  dplyr::select(peakid,MRC) %>% 
  left_join(.,median_3_lfc, by=c("peakid"="peak")) %>% 
  ggplot(., aes(x=MRC,y=median))+
  geom_boxplot()  +
   theme_bw()+
  ggtitle("3 hr median logFC across response peaks overlapped with TOP2B")+
  coord_cartesian(ylim= c(-3.5,2))

```

### Counts of peaks-

```{r top2b_cluster}
peak_list_all_MRC %>% 
  dplyr::filter(peakid %in% top2Boverlap$id) %>% 
  dplyr::select(peakid,MRC) %>% 
  left_join(.,Peak_MRC_table, by=c("peakid"="peakid")) %>% 
  group_by(MRC) %>% 
  tally


```

