---
title: "Enhancer_files"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r package loading, message=FALSE, warning=FALSE}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github('davetang/bedr')
library(bedr)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)
library(plyranges)
library(genomation)

```

Data loading

```{r data loading}

toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") 
S13Table <- read.csv( "data/other_papers/S13Table_Matthews2024.csv",row.names = 1)
##14021

EAR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="EAR") %>% 
  dplyr::select(ENTREZID) %>% 
  mutate(ENTREZID= as.character(ENTREZID))
ESR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="ESR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))
LR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="LR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID=as.character(ENTREZID))
NR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="NR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))

exp_neargene_table <- read_delim("data/n45_bedfiles/exp_neargene_table.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

# col_ng_peak <- exp_neargene_table %>%
#   group_by(chr, start, end, peakid) %>%
#     summarise(NCBI_gene = paste(unique(ENTREZID),collapse=","),
#               ensembl_ID= paste(unique(ensembl_id),collapse = ","),
#               SYMBOL= paste(unique(SYMBOL),collapse = ",") , 
#               dist_to_NG =min(dist_to_NG)) %>% 
#   ungroup()
col_ng_peak <- read.csv("data/col_ng_peak.csv",row.names = 1)
peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
# list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)

peakAnnoList_3_n45 <- readRDS("data/peakAnnoList_3_n45.RDS")
peakAnnoList_24_n45<- readRDS("data/peakAnnoList_24_n45.RDS")

all_peak_gr <- as.GRanges(peakAnnoList_24_n45$background)

Joined_tss_neargene <- all_peak_gr %>% 
  as.data.frame() %>% 
  dplyr::select(seqnames:id,geneId,distanceToTSS) %>% 
  dplyr::filter(!grepl("chrX",id)& !grepl("chrY",id))%>%
  dplyr::rename("close_geneId"="geneId") %>% 
  mutate(start=start+1) %>% 
  left_join(., col_ng_peak, by=c("id"="peakid", "start"="start","end"="end")) %>% 
  dplyr::rename("NCBI_gene_NG"="NCBI_gene") 

  
### THESE ARE DAR peaks (not total peaks)



DOX_3hr <- as.data.frame(peakAnnoList_3_n45$DOX_3_n45)
DOX_3hr_gr <- as.GRanges(peakAnnoList_3_n45$DOX_3_n45)
EPI_3hr <- as.data.frame(peakAnnoList_3_n45$EPI_3_n45)
EPI_3hr_gr <- as.GRanges(peakAnnoList_3_n45$EPI_3_n45)
DNR_3hr <- as.data.frame(peakAnnoList_3_n45$DNR_3_n45)
DNR_3hr_gr <- as.GRanges(peakAnnoList_3_n45$DNR_3_n45)
MTX_3hr <- as.data.frame(peakAnnoList_3_n45$MTX_3_n45)
MTX_3hr_gr <- as.GRanges(peakAnnoList_3_n45$MTX_3_n45)
# # TRZ_3hr <- as.data.frame(peakAnnoList_3_n45$TRZ_3_n45)
# TRZ_3hr_gr <- GRanges(TRZ_3hr)
DOX_3_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% DOX_3hr$id)
EPI_3_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% EPI_3hr$id)
DNR_3_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% DNR_3hr$id)
MTX_3_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% MTX_3hr$id)

### THESE ARE DAR peaks (not total peaks)

DOX_24hr <- as.data.frame(peakAnnoList_24_n45$DOX_24_n45)
DOX_24hr_gr <- as.GRanges(peakAnnoList_24_n45$DOX_24_n45)
EPI_24hr <- as.data.frame(peakAnnoList_24_n45$EPI_24_n45)
EPI_24hr_gr <-as.GRanges(peakAnnoList_24_n45$EPI_24_n45)
DNR_24hr <- as.data.frame(peakAnnoList_24_n45$DNR_24_n45)
DNR_24hr_gr <- as.GRanges(peakAnnoList_24_n45$DNR_24_n45)
MTX_24hr <- as.data.frame(peakAnnoList_24_n45$MTX_24_n45)
MTX_24hr_gr <- as.GRanges(peakAnnoList_24_n45$MTX_24_n45)
# TRZ_24hr <- as.data.frame(peakAnnoList_24_n45$TRZ_24_n45)
# TRZ_24hr_gr <- as.GRanges(peakAnnoList_24_n45$TRZ_24_n45)
## THESE ARE DAR peaks (not total peaks)
DOX_24_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% DOX_24hr$id)
EPI_24_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% EPI_24hr$id)
DNR_24_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% DNR_24hr$id)
MTX_24_peak_list <-  col_ng_peak %>%
  # dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% MTX_24hr$id)


EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)


ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr <- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)


EAR_peak_list_20k <-  col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(MRC="EAR") 

EAR_peak_list <-  col_ng_peak %>%
    dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(MRC="EAR") 

ESR_peak_list_20k <- col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% ESR_df$id) %>% 
    mutate(MRC="ESR")

ESR_peak_list<- col_ng_peak %>%
    dplyr::filter(peakid %in% ESR_df$id) %>% 
    mutate(MRC="ESR")

LR_peak_list_20k <- col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% LR_df$id) %>% 
    mutate(MRC="LR")

LR_peak_list <- col_ng_peak %>%
    dplyr::filter(peakid %in% LR_df$id) %>% 
    mutate(MRC="LR")

NR_peak_list_20k <- col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>%
    dplyr::filter(peakid %in% NR_df$id) %>% 
    mutate(MRC="NR")
NR_peak_list <- col_ng_peak %>%
    dplyr::filter(peakid %in% NR_df$id) %>% 
    mutate(MRC="NR")

peak_list_20k_MRC <- EAR_peak_list_20k %>% 
  rbind(ESR_peak_list_20k) %>% 
  rbind(LR_peak_list_20k) %>% 
  rbind(NR_peak_list_20k)

peak_list_MRC <- EAR_peak_list %>% 
  rbind(ESR_peak_list) %>% 
  rbind(LR_peak_list) %>% 
  rbind(NR_peak_list)

gr_EAR_peak_list<-  col_ng_peak %>%
    dplyr::filter(peakid %in% EAR_df$id) %>% 
  mutate(MRC="EAR") %>% 
  GRanges()

gr_ESR_peak_list <- col_ng_peak %>%
    dplyr::filter(peakid %in% ESR_df$id) %>% 
    mutate(MRC="ESR")%>% 
  GRanges()

gr_LR_peak_list <- col_ng_peak %>%
    dplyr::filter(peakid %in% LR_df$id) %>% 
    mutate(MRC="LR")%>% 
  GRanges()

gr_NR_peak_list <- col_ng_peak %>%
    dplyr::filter(peakid %in% NR_df$id) %>% 
    mutate(MRC="NR")%>% 

  GRanges()

n45_fullpeaks_gr <- readBed("data/n45_bedfiles/DAR_DEG_background.bed")
###file for ATACseq replicability
ENCFF544RZR <-readRDS ("data/n45_bedfiles/ENCFF533RZR_gr.RDS")
# ENCFFFF544RZR_bed <- read_delim("data/n45_bedfiles/ENCFF134HBK.bed", 
#      delim = "\t", escape_double = FALSE, 
#      col_names = FALSE, trim_ws = TRUE)
#  ENCFF544RZR <- ENCFF544RZR_bed %>% 
#    rename(X1="chr",
#          X2="start",
#          X3="end",
#          X10="length") %>%
#    GRanges()
 
Snyder_overlaps <- join_overlap_intersect(n45_fullpeaks_gr,ENCFF544RZR)
# 84725
# length(find_overlaps(ENCFF544RZR,n45_fullpeaks_gr))
# test <- (subsetByOverlaps(n45_fullpeaks_gr,ENCFF544RZR))
# test %>% 
#   as.data.frame() %>% 
#   distinct(name)
# head(Snyder_overlaps)
toplist_ATAC <- readRDS("data/toplist_n45.RDS")

toplist_DOX24_ATAC <- toplist_ATAC %>% 
  dplyr::filter(trt=="DOX" & time=="24 hours")

```

### MRC venns

These are Venns using the Neargenes between each set of peaks
```{r Venns of data}
ggVennDiagram::ggVennDiagram(list(unique(EAR_peak_list$ensembl_ID), 
                                  unique(ESR_peak_list$ensembl_ID),  
                                  unique(LR_peak_list$ensembl_ID), 
                                  unique(NR_peak_list$ensembl_ID)),
                             category.names = c("EAR_NG","ESR_NG","LR_NG","NR_NG"),
              show_intersect = FALSE,
              set_color = "black",
              label = "count",
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = 1,
              label_color = "black",
              edge_lty = "solid", set_size = 4)+
  ggtitle("Near gene distribution for all peaks")+
  scale_x_continuous(expand = expansion(mult = .2))



ggVennDiagram::ggVennDiagram(list(unique(EAR_peak_list$NCBI_gene),EAR_RNA$ENTREZID),
              category.names = c("EAR_ATAC","EAR_RNA"), label = "count")+
  labs(title = "EAR ATAC-neargenes and expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))
  

ggVennDiagram::ggVennDiagram(list(unique(ESR_peak_list$NCBI_gene),ESR_RNA$ENTREZID),
              category.names = c("ESR_ATAC","ESR_RNA"), label = "count")+
  labs(title = "ESRATAC-neargenes and expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))

ggVennDiagram::ggVennDiagram(list(unique(LR_peak_list$NCBI_gene),LR_RNA$ENTREZID),
              category.names = c("LR_ATAC","LR_RNA"), label = "count")+
  labs(title = "LR ATAC-neargenes and expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+
  scale_x_continuous(expand = expansion(mult = .2))

ggVennDiagram::ggVennDiagram(list(unique(NR_peak_list$NCBI_gene),NR_RNA$ENTREZID),
              category.names = c("NR_ATAC","NR_RNA"), label = "count")+
  labs(title = "NR ATAC-neargenes and expressed response RNA")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))+ 
  scale_x_continuous(expand = expansion(mult = .2))


```

### Enhancers and Response clusters


First:  obtained a list of cis Regulatory Elements from Encode Screen
[(https://screen.encodeproject.org/#)]


```{r  enhancers of the heart}
# BiocManager::install("plyranges")
# enhancers_HLV_46F <- genomation::readBed("C:/Users/renee/Downloads/Supplements folde manuscriptr/ENCODE/heart_left_ventricle_tissue_female_adult_46_years.enhancers.bed")
 cREs_HLV_46F <- genomation::readBed("data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed")

cREs_HLV_53F <- genomation::readBed("data/enhancerdata/ENCFF417JSF_ENCFF651XRK_ENCFF320IPT_ENCFF440RUS.7group.bed") 
 
NR_cREs <- join_overlap_intersect(gr_NR_peak_list,cREs_HLV_46F)
LR_cREs <- join_overlap_intersect(gr_LR_peak_list,cREs_HLV_46F)
ESR_cREs <- join_overlap_intersect(gr_ESR_peak_list,cREs_HLV_46F)
EAR_cREs <- join_overlap_intersect(gr_EAR_peak_list,cREs_HLV_46F)

# NR_cREs_all <- join_overlap_intersect(NR_df_gr,cREs_HLV_46F)
# LR_cREs_all <- join_overlap_intersect(LR_df_gr,cREs_HLV_46F)
# ESR_cREs_all <- join_overlap_intersect(ESR_df_gr,cREs_HLV_46F)
# EAR_cREs_all <- join_overlap_intersect(EAR_df_gr,cREs_HLV_46F)

# Whole_peaks <- join_overlap_intersect(n45_fullpeaks_gr, cREs_HLV_46F)

NR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="NR", per=n/(length(unique(NR_cREs$peakid)))) %>% 
  kable(., caption=paste0("No Response peaks overlapping cREs n= ", length(unique(NR_cREs$peakid)))) %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)


LR_cREs%>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="LR", per=n/25610) %>% 
kable(., caption="Late Response peaks overlapping cREs") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)

ESR_cREs %>%
  as.data.frame() %>%
  # dplyr::filter(blockCount=="CTCF-only,CTCF-bound") %>%
  group_by(blockCount) %>%
 count() %>% 
  mutate(MRC="ESR", per=n/10122) %>% 
  kable(., caption="Early-sustained response peaks overlapping cREs") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)


EAR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="EAR", per=n/4813) %>% 
  kable(., caption="Early acute response peaks overlapping cREs") %>% 
  kable_paper("striped") %>%
  kable_styling(full_width = FALSE, font_size = 14)
```

### Visualization of categories:

First, filtering cRE set by type to include 'CTCF-only,CTCF-bound', 'PLS', 'PLS,CTCF-bound','dELS', 'dELS,CTCF-bound', 'pELS', 'pELS,CTCF-bound'.

Second, reclassify (new column) to group the CTCF-bound into their respective groups so only 4 groups, ("CTCF-only","PLS","dELS", "pELS") are created.

Third, left-join the DOX24hr ATAC LFC to data set and boxplot by group.

PLS= promoter like sequences
pELS= proximal enhancer like sequences
dELS=distal enhancer like sequences

```{r  EARcRE maps}
adj_toplist_DOX24_ATAC <- toplist_DOX24_ATAC%>% 
  dplyr::filter(adj.P.Val<0.05,.keepall=TRUE)

EAR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("EAR n = ",length(unique(EAR_cREs$peakid))," out of ", length(unique(EAR_df$id)), " total peaks (",sprintf("%.1f",length(unique(EAR_cREs$peakid))/length(unique(EAR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)



ESR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
   ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("ESR n = ",length(unique(ESR_cREs$peakid))," out of ", length(unique(ESR_df$id)), " total peaks (",sprintf("%.1f",length(unique(ESR_cREs$peakid))/length(unique(ESR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)


LR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle(paste0("LR n = ",length(unique(LR_cREs$peakid))," out of ", length(unique(LR_df$id)), " total peaks (",sprintf("%.1f",length(unique(LR_cREs$peakid))/length(unique(LR_df$id))*100),"%)"))+
  # facet_wrap(~direction)+
  theme_bw()+
  xlim(-4,4)

NR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
   ggplot(., aes(y=type, x=logFC,fill=direction))+
  geom_boxplot() +
  # facet_wrap(~direction)+
  ggtitle(paste0("NR n = ",length(unique(NR_cREs$peakid))," out of ", length(unique(NR_df$id)), " total peaks (",sprintf("%.1f",length(unique(NR_cREs$peakid))/length(unique(NR_df$id))*100),"%)"))+
  theme_bw()+
  xlim(-4,4)


# cREs_HLV_46F %>%
#   as.data.frame()# %>%
#   dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
#   mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
#                       if_else(grepl("PLS",blockCount),"Promoter-like",
#                               if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
#   right_join(., toplist_DOX24_ATAC, by =c("peakid"="peak")) %>% 
#     



```
### All response (EAR+ESR+LR peaks that overlap enhancer, then split for logFC at DOX24)



```{r logFC all response}

respcREs <-
EAR_cREs %>% 
as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  rbind(ESR_cREs %>% 
as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount)))) %>% 
  rbind(LR_cREs %>% 
as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount)))) %>% 
   left_join(., toplist_DOX24_ATAC, by =c("peakid"="peak")) %>% 
  dplyr::select(peakid, NCBI_gene,ensembl_ID,SYMBOL,name, type, logFC,adj.P.Val) %>% 
  mutate(status=if_else(logFC>=0,"open","closed")) 

respcREs %>% 
  # dplyr::filter(adj.P.Val<0.05) %>% 

  ggplot(., aes(y=type, x=logFC, fill=status))+
  geom_boxplot() +
  # facet_wrap(~direction)+
  ggtitle("Log FC of cREs across response groups ")+
  theme_bw()+
  xlim(-4,4)
  

```
### Overlap of all peaks with enhancers

```{r}
fullOL <- join_overlap_intersect(all_peak_gr,cREs_HLV_46F)
fullOLtable <- fullOL %>% 
as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
left_join(., toplist_DOX24_ATAC, by =c("id"="peak")) %>% 
  dplyr::select(id, annotation, type, logFC,adj.P.Val) %>% 
  mutate(status=if_else(logFC>=0,"open","closed")) 
   
fullOLtable %>% 
ggplot(., aes(y=type, x=logFC, fill=status))+
  geom_boxplot() +
  # facet_wrap(~direction)+
  ggtitle("Log FC across all peaks ")+
  theme_bw()+
  xlim(-4,4)
  
fullOLtable %>% 
  group_by(type,status) %>% 
  count() %>%
  pivot_wider(., id_cols="type",names_from = "status",values_from = n) %>% 
  mutate(total=closed+open) %>% 
  kable(., caption="Breakdown of peaks overlapping cREs") %>% 
  kable_paper("striped") %>%
  kable_styling(full_width = FALSE, font_size = 14)


```


###  EAR cREs with all peaks
```{r enhacerstuffforlater EAR}
toplist_DOX24_ATAC_nop <- toplist_ATAC %>% 
  dplyr::filter(trt=="DOX" & time=="24 hours")
EAR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="EAR", per=n/7321) %>% 
  kable(., caption="Early acute response peaks (7321) overlapping cREs") %>% 
  kable_paper("striped") %>%
  kable_styling(full_width = FALSE, font_size = 14)

EAR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC_nop, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle("EAR")+
  # facet_wrap(~direction)+
  theme_bw()
```



###  ESR cREs with all peaks
```{r enhacerstuffforlater ESR}
toplist_DOX24_ATAC_nop <- toplist_ATAC %>% 
  dplyr::filter(trt=="DOX" & time=="24 hours")
ESR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="ESR", per=n/15665) %>% 
  kable(., caption="Early sustained response peaks (15,665) overlapping cREs") %>% 
  kable_paper("striped") %>%
  kable_styling(full_width = FALSE, font_size = 14)

ESR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC_nop, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle("ESR")+
  # facet_wrap(~direction)+
  theme_bw()
```


###  LR cREs with all peaks
```{r enhacerstuffforlater LR}
toplist_DOX24_ATAC_nop <- toplist_ATAC %>% 
  dplyr::filter(trt=="DOX" & time=="24 hours")
LR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="LR", per=n/41759) %>% 
  kable(., caption="Late response peaks (41,759) overlapping cREs") %>% 
  kable_paper("striped") %>%
  kable_styling(full_width = FALSE, font_size = 14)

LR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC_nop, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle("LR")+
  # facet_wrap(~direction)+
  theme_bw()
```



###  NR cREs with all peaks
```{r enhacerstuffforlater NR}
toplist_DOX24_ATAC_nop <- toplist_ATAC %>% 
  dplyr::filter(trt=="DOX" & time=="24 hours")
NR_cREs %>%
  as.data.frame() %>%
  group_by(blockCount) %>%
  count() %>% 
  mutate(MRC="NR", per=n/83658) %>% 
  kable(., caption="No response peaks (83,658) overlapping cREs") %>% 
  kable_paper("striped") %>%
  kable_styling(full_width = FALSE, font_size = 14)

NR_cREs %>%
  as.data.frame() %>%
  dplyr::filter(blockCount=="CTCF-only,CTCF-bound"|blockCount =="PLS"|blockCount =="PLS,CTCF-bound"|blockCount =="dELS"|blockCount =="dELS,CTCF-bound"|blockCount =="pELS"|blockCount =="pELS,CTCF-bound") %>% 
  mutate(type=if_else(blockCount=="CTCF-only,CTCF-bound","CTCF-only",
                      if_else(grepl("PLS",blockCount),"Promoter-like",
                              if_else(grepl("pELS",blockCount),"Proximal enhancer-like", "Distal enhancer-like"),blockCount))) %>% 
  left_join(., toplist_DOX24_ATAC_nop, by =c("peakid"="peak")) %>% 
  na.omit() %>% 
  dplyr::select(logFC,peakid,type) %>% 
  mutate(direction=if_else(logFC>0,"open","closed")) %>% 
  mutate(type=factor(type, levels = c("Promoter-like","Proximal enhancer-like","Distal enhancer-like","CTCF-only"))) %>%
  mutate(direction=factor(direction, levels=c("open","closed"))) %>% 
  ggplot(., aes(y=type, x=logFC,group=type))+
  geom_boxplot() +
  scale_fill_discrete()+
  ggtitle("NR")+
  # facet_wrap(~direction)+
  theme_bw()
```




| Cluster | eREs                                           | not cREs                                                                |
|------------------------|------------------------|------------------------|
| NR      | `r length(unique(NR_cREs$peakid))` | `r length(unique(NR_df$id))-length(unique(NR_cREs$peakid))`  |
| LR      | `r length(unique(LR_cREs$peakid))` | `r length(unique(LR_df$id))-length(unique(LR_cREs$peakid))` |
| ESR     | `r length(unique(ESR_cREs$peakid))` | `r length(unique(ESR_df$id))-length(unique(ESR_cREs$peakid))` |
| EAR     | `r length(unique(EAR_cREs$peakid))` | `r length(unique(EAR_df$id))-length(unique(EAR_cREs$peakid))` |



```{r making enhancer NG sets}
# EAR_20k_peaks_gr <- GRanges(EAR_peak_list_20k)
# ESR_20k_peaks_gr <- GRanges(ESR_peak_list_20k)
# LR_20k_peaks_gr <- GRanges(LR_peak_list_20k)
# NR_20k_peaks_gr <- GRanges(NR_peak_list_20k)
# 
# EAR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(EAR_20k_peaks_gr,enhancers_HLV_46F))
# ESR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(ESR_20k_peaks_gr,enhancers_HLV_46F))
# LR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(LR_20k_peaks_gr,enhancers_HLV_46F))
# NR_enh_peaks_20k <- as.data.frame(subsetByOverlaps(NR_20k_peaks_gr,enhancers_HLV_46F))
# saveRDS(EAR_enh_peaks_20k,"data/enhancerdata/EAR_enh_peaks_20k.RDS")
# saveRDS(ESR_enh_peaks_20k,"data/enhancerdata/ESR_enh_peaks_20k.RDS")
# saveRDS(LR_enh_peaks_20k,"data/enhancerdata/LR_enh_peaks_20k.RDS")
# saveRDS(NR_enh_peaks_20k,"data/enhancerdata/NR_enh_peaks_20k.RDS")
### making a set that is with all peaks in motif
# EAR_enh_peaks_20k <- subsetByOverlaps(NR_df_gr,enhancers_HLV_46F)

```


### DAR cRE venns 


```{r DAR venns 3hr }
ggVennDiagram::ggVennDiagram(list(DOX_3_peak_list$peakid, 
                                  EPI_3_peak_list$peakid,  
                                  DNR_3_peak_list$peakid, 
                                  MTX_3_peak_list$peakid),
                             category.names = c("DOX peaks\n3 hour","EPI peaks\n3 hour","DNR peaks\n3 hour","MTX peaks\n3 hour"),
              show_intersect = FALSE,
              set_color = "black",
              label = "count",
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = "black",
              edge_lty = "solid", set_size = 4)

ggVennDiagram::ggVennDiagram(list(unique(DOX_3_peak_list$ensembl_ID), 
                                  unique(EPI_3_peak_list$ensembl_ID),  
                                  unique(DNR_3_peak_list$ensembl_ID), 
                                  unique(MTX_3_peak_list$ensembl_ID)),
                             category.names = c("DOX_NG\n3 hour","EPI_NG\n3 hour","DNR_NG\n3 hour","MTX_NG\n3 hour"),
              show_intersect = FALSE,
              set_color = "black",
              label = "count",
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = "black",
              edge_lty = "solid", set_size = 4)

# enhancerdf_DAR_3hr <- data.frame("DAR"=c("DOX_3", "EPI_3","DNR_3","MTX_3"), 
#               "cRE_yes"=c(length(findOverlaps(enhancers_HLV_46F,DOX_3hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,EPI_3hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,DNR_3hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,MTX_3hr_gr))),
#               "cRE_no"=c(length(DOX_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DOX_3hr_gr)),
#                        length(EPI_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,EPI_3hr_gr)), 
#                        length(DNR_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DNR_3hr_gr)),
#                        length(MTX_3hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,MTX_3hr_gr))))



```

#### DAR 24 hour


```{r DAR venns 24hr }
ggVennDiagram::ggVennDiagram(list(DOX_24_peak_list$peakid, 
                                  EPI_24_peak_list$peakid,  
                                  DNR_24_peak_list$peakid, 
                                  MTX_24_peak_list$peakid),
                             category.names = c("DOX peaks","EPI peaks","DNR peaks","MTX peaks"),
              show_intersect = FALSE,
              set_color = "black",
              label = "count",
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = "black",
              edge_lty = "solid", set_size = 4)

ggVennDiagram::ggVennDiagram(list(unique(DOX_24_peak_list$ensembl_ID), 
                                  unique(EPI_24_peak_list$ensembl_ID),  
                                  unique(DNR_24_peak_list$ensembl_ID), 
                                  unique(MTX_24_peak_list$ensembl_ID)),
                             category.names = c("DOX_NG","EPI_NG","DNR_NG","MTX_NG"),
              show_intersect = FALSE,
              set_color = "black",
              label = "count",
              label_percent_digit = 1,
              label_size = 4,
              label_alpha = .8,
              label_color = "black",
              edge_lty = "solid", set_size = 4)

# enhancerdf_DAR_24hr <- data.frame("DAR"=c("DOX_24", "EPI_24","DNR_24","MTX_24"), 
#               "cRE_yes"=c(length(findOverlaps(enhancers_HLV_46F,DOX_24hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,EPI_24hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,DNR_24hr_gr)),
#                         length(findOverlaps(enhancers_HLV_46F,MTX_24hr_gr))),
#               "cRE_no"=c(length(DOX_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DOX_24hr_gr)),
#                        length(EPI_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,EPI_24hr_gr)), 
#                        length(DNR_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,DNR_24hr_gr)),
#                        length(MTX_24hr_gr$geneId)-length(findOverlaps(enhancers_HLV_46F,MTX_24hr_gr))))


```




### Enhancers and DAR 24
```{r  DAR and enhancers}


# testdf <- enhancerdf_DAR_24hr %>% 
#   column_to_rownames("DAR") %>% 
#   as.matrix() 
# paste("Late Response chi")
# chisq.test(testdf[c(1,2),])
# paste("Early sustained Response chi")
# chisq.test(testdf[c(1,3),])
# paste("Early-acute Response chi")
# chisq.test(testdf[c(1,4),])
# enhancerdf_DAR_3hr %>% 
#       pivot_longer(., cols=c(cRE_yes,cRE_no), names_to = "Enh_status", values_to = "count") %>% 
#   mutate(DAR=factor(DAR, levels = c("DOX_3","EPI_3", "DNR_3","MTX_3"))) %>% 
#   ggplot(., aes(x = DAR, y= count ))+
#     geom_bar(position ="fill", stat="identity", aes(fill=Enh_status))
# enhancerdf_DAR_24hr %>% 
#       pivot_longer(., cols=c(cRE_yes,cRE_no), names_to = "Enh_status", values_to = "count") %>% 
#   mutate(DAR=factor(DAR, levels = c("DOX_24","EPI_24", "DNR_24","MTX_24"))) %>% 
#   ggplot(., aes(x = DAR, y= count ))+
#     geom_bar(position ="fill", stat="identity", aes(fill=Enh_status))
 
```


###### noodling around
```{r crazy enhancergwas, eval=FALSE, include=FALSE}

gwas_list <- readRDS("data/test.list.RDS")

list2env(gwas_list, envir = .GlobalEnv)

##gwas2 is ARR,  gwas4 is HD
gwas_2_gr_noext
gwas_4_gr_noext
# findOverlaps(enhancers_HLV_46F,gwas_2_gr_noext)
# ARR_SNPS <- subsetByOverlaps(enhancers_HLV_46F,gwas_2_gr_noext)
# arr_list <- as.data.frame(ARR_SNPS)
# (findOverlaps(ARR_SNPS,NR_df_gr))
# (findOverlaps(ARR_SNPS,LR_df_gr))
# (findOverlaps(ARR_SNPS,ESR_df_gr))
# (findOverlaps(ARR_SNPS,EAR_df_gr))
# 
# subsetByOverlaps(NR_df_gr, ARR_SNPS)
# subsetByOverlaps(LR_df_gr, ARR_SNPS)
# subsetByOverlaps(ESR_df_gr, ARR_SNPS)
# subsetByOverlaps(EAR_df_gr, ARR_SNPS)

# 
# 
# findOverlaps(enhancers_HLV_46F,gwas_4_gr_noext)
# HD_SNPS <- subsetByOverlaps(enhancers_HLV_46F,gwas_4_gr_noext)
# HD_list <- as.data.frame(HD_SNPS)
# (findOverlaps(HD_SNPS,NR_df_gr))
# (findOverlaps(HD_SNPS,LR_df_gr))
# (findOverlaps(HD_SNPS,ESR_df_gr))
# (findOverlaps(HD_SNPS,EAR_df_gr))
# 
# subsetByOverlaps(NR_df_gr,HD_SNPS)
#  LRgene <- subsetByOverlaps(LR_df_gr, HD_SNPS)
#  LRgene$transcriptId
# subsetByOverlaps(ESR_df_gr, HD_SNPS)
# subsetByOverlaps(EAR_df_gr, HD_SNPS)





```
