---
title: "Enrichment_motif"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r package loading, message=FALSE, warning=FALSE}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github('davetang/bedr')
library(bedr)
library(biomaRt)
library(plyranges)
library(genomation)

```

### Knowles supp data check

```{r import data}
Knowles_4<- readRDS("data/Knowles_4.RDS")
Knowles_5 <- readRDS("data/Knowles_5.RDS")
Knowles_6 <- readRDS("data/Knowles_6.RDS")
SNP_supp_schneider <- readRDS("data/SNP_supp_schneider.RDS")
peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)
###How I made the granges files for overlaps
Knowles_4_noext <- Knowles_4 %>%
  na.omit() %>%
  mutate(start=pos, end=pos+1) %>%
  dplyr::select(chr, start,end, RSID, gene,pos) %>%
  GRanges()


Knowles_4 <- Knowles_4 %>%
  na.omit() %>%
  mutate(start=pos-500, end=pos+500) %>%
  dplyr::select(chr, start,end, RSID, gene,pos) %>%
  GRanges()


# 
Knowles_5_noext <- Knowles_5 %>%
  na.omit() %>%
  mutate(start=pos, end=pos+1) %>%
  dplyr::select(chr, start,end, RSID, gene,pos) %>%
  GRanges()

Knowles_5 <- Knowles_5 %>%
  na.omit() %>%
  mutate(start=pos-500, end=pos+500) %>%
  dplyr::select(chr, start,end, RSID, gene,pos) %>%
  GRanges()

Knowles_6_noext <- Knowles_6 %>%
  na.omit() %>%
  mutate(start=pos, end=pos+1) %>%
  dplyr::select(chr, start,end, gene,pos) %>%
  GRanges()
 
 Knowles_6 <- Knowles_6 %>%
  na.omit() %>%
  mutate(start=pos-500, end=pos+500) %>%
  dplyr::select(chr, start,end, gene,pos) %>%
  GRanges()
 
 
SNP_supp_schneider <-  SNP_supp_schneider %>%
    na.omit() %>%
     rename(X1="chr",X2= "pos",X4="RSID") %>%
    # rename("chr"=X1, "pos" =X2,"RSID" =X4) %>%
    mutate(start=pos-500, end=pos+500) %>%
   dplyr::select(chr, start,end, RSID,pos) %>%
    GRanges()

##split into granges objects
###make dataframe of each EAR,ESR,LR,NR promoters
# background_df <- as.data.frame(background_gr)
# background_Gr <- GRanges(background_df)
#  rtracklayer::export.bed(background_Gr, con="data/n45_bedfiles/background_n45.bed", format="bed")
EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_promotor <- EAR_df %>% 
#   dplyr::filter(annotation=="Promoter (<=1kb)")
EAR_df_gr <-  GRanges(EAR_df)

# rtracklayer::export.bed(EAR_promo_gr, con="data/n45_bedfiles/EAR_promo.bed", format="bed", ignore.strand = FALSE)
ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_promotor <- ESR_df %>% 
#   dplyr::filter(annotation=="Promoter (<=1kb)")
ESR_df_gr <-  GRanges(ESR_df)
 # rtracklayer::export.bed(ESR_promo_gr, con="data/n45_bedfiles/ESR_promo.bed", format="bed", ignore.strand = FALSE)
 
LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_promotor <- LR_df %>% 
#   dplyr::filter(annotation=="Promoter (<=1kb)")
LR_df_gr <-  GRanges(LR_df)
 # rtracklayer::export.bed(LR_promo_gr, con="data/n45_bedfiles/LR_promo.bed", format="bed", ignore.strand = FALSE)
 # LR_promotor %>% 
 #   dplyr::filter(geneChr==6)

NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_promotor <- NR_df %>% 
#   dplyr::filter(annotation=="Promoter (<=1kb)")
 NR_df_gr <-  GRanges(NR_df)
 
 ENCFF134HBK <- read_delim("data/n45_bedfiles/ENCFF134HBK.bed", 
     delim = "\t", escape_double = FALSE, 
     col_names = FALSE, trim_ws = TRUE)
 ZNF384_chip <- ENCFF134HBK %>% 
   rename(X1="chr",
         X2="start",
         X3="end",
         X10="length") %>%
   GRanges()
 
 
```

### Checking for enrichment K4-K6 cardiotox:

I have made 500 bp +/- files for each of the data sets using the eQTL or SNP location. I also made a 1bp no extension file for each of the data sets using the eQTL or SNP location as start and end as +1.

```{r overlaps, include=FALSE}
findOverlaps(EAR_df_gr, Knowles_4)
findOverlaps(ESR_df_gr, Knowles_4)
findOverlaps(LR_df_gr, Knowles_4)
findOverlaps(NR_df_gr, Knowles_4)

findOverlaps(EAR_df_gr, Knowles_5)
findOverlaps(ESR_df_gr, Knowles_5)
findOverlaps(LR_df_gr, Knowles_5)
findOverlaps(NR_df_gr, Knowles_5)


findOverlaps(EAR_df_gr, Knowles_6)
findOverlaps(ESR_df_gr, Knowles_6)
findOverlaps(LR_df_gr, Knowles_6)
findOverlaps(NR_df_gr, Knowles_6)

findOverlaps(EAR_df_gr, SNP_supp_schneider)
findOverlaps(ESR_df_gr, SNP_supp_schneider)
findOverlaps(LR_df_gr, SNP_supp_schneider)
findOverlaps(NR_df_gr, SNP_supp_schneider)

subsetByOverlaps(LR_df_gr, SNP_supp_schneider)


findOverlaps(EAR_df_gr, Knowles_4_noext)
findOverlaps(ESR_df_gr, Knowles_4_noext)
findOverlaps(LR_df_gr, Knowles_4_noext)
findOverlaps(NR_df_gr, Knowles_4_noext)

findOverlaps(EAR_df_gr, Knowles_5_noext)
findOverlaps(ESR_df_gr, Knowles_5_noext)
findOverlaps(LR_df_gr, Knowles_5_noext)
findOverlaps(NR_df_gr, Knowles_5_noext)


findOverlaps(EAR_df_gr, Knowles_6_noext)
findOverlaps(ESR_df_gr, Knowles_6_noext)
findOverlaps(LR_df_gr, Knowles_6_noext)
findOverlaps(NR_df_gr, Knowles_6_noext)



```

report:

| motif       | K_4(500)                                       | K_5(359)                                       | K_6(447)                                       | schn_g(112)                                             |
|---------------|---------------|---------------|---------------|---------------|
| EAR (7321)  | `r length(findOverlaps(EAR_df_gr, Knowles_4))` | `r length(findOverlaps(EAR_df_gr, Knowles_5))` | `r length(findOverlaps(EAR_df_gr, Knowles_6))` | `r length(findOverlaps(EAR_df_gr, SNP_supp_schneider))` |
| ESR (15665) | `r length(findOverlaps(ESR_df_gr, Knowles_4))` | `r length(findOverlaps(ESR_df_gr, Knowles_5))` | `r length(findOverlaps(ESR_df_gr, Knowles_6))` | `r length(findOverlaps(ESR_df_gr, SNP_supp_schneider))` |
| LR (41759)  | `r length(findOverlaps(LR_df_gr, Knowles_4))`  | `r length(findOverlaps(LR_df_gr, Knowles_5))`  | `r length(findOverlaps(LR_df_gr, Knowles_6))`  | `r length(findOverlaps(LR_df_gr, SNP_supp_schneider))`  |
| NR (83658)  | `r length(findOverlaps(NR_df_gr, Knowles_4))`  | `r length(findOverlaps(NR_df_gr, Knowles_5))`  | `r length(findOverlaps(NR_df_gr, Knowles_6))`  | `r length(findOverlaps(NR_df_gr, SNP_supp_schneider))`  |

: number of intersections between groups with 500 =/- bp around SNP

report:

| motif       | K_4(500) | K_5(359) | K_6(447) |
|-------------|----------|----------|----------|
| EAR (7321)  | 0        | 1        | 0        |
| ESR (15665) | 3        | 2        | 2        |
| LR (41759)  | 13       | 9        | 8        |
| NR (83658)  | 30       | 15       | 19       |

: number of intersections between groups without extension

These numbers are different than the original lists from which they derived. This is normally due to running `na.omit()` on the data frame before GRanges transformation.\

### GWAS

I downloaded 5 different trait associated lists from the GWAS catalog

trait 1, 11 associations with trait **cardiotoxicity**

trait 2, 1068 associations with trait **cardiac arrhythmia**

trait 3, 40 associations with trait **response to anthracycline-based chemotherapy**

trait 4, 6139 associations with trait **heart disease**

trait 5, 445 associations with trait **heart failure**

```{r GWAS info, message=FALSE, warning=FALSE}


##finding GWAS for Hearfailure: I used several traits from the GWAS database:
##1_cardiotoxicity  11 associations   gwas_1
gwas_1 <- readRDS("data/gwas_1_dataframe.RDS")
##2_ cardiac arrhythmia  1068 associations   gwas_2
gwas_2 <- readRDS("data/gwas_2_dataframe.RDS")

##3_ Anthracycline_response   40 gwas_3
gwas_3 <- readRDS("data/gwas_3_dataframe.RDS")

##4_ heart disease (biggest # of associations 6139) 
gwas_4 <- readRDS("data/gwas_4_dataframe.RDS")

##5_ heart_failure gwas_5
gwas_5 <- readRDS("data/gwas_5_dataframe.RDS")

##6 Coronary artery disease
CAD_gwas <- readRDS( "data/CAD_gwas_dataframe.RDS")

MI_gwas <- readRDS("data/MI_gwas.RDS")

CAD_gwas_gr <- CAD_gwas %>%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit(.) %>% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   GRanges(.)
CAD_gwas_gr_ten <- CAD_gwas %>%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit(.) %>% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   GRanges(.)

CAD_gwas_gr_noext <- CAD_gwas %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit(.) %>% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   GRanges(.)
 # rtracklayer::export.bed(CAD_gwas_gr_noext, con="data/n45_bedfiles/CAD_gwas_gr_noext.bed", format="bed", ignore.strand = FALSE)

MI_gwas_gr <- MI_gwas %>%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit(.) %>% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   GRanges(.)
MI_gwas_gr_ten <- MI_gwas %>%
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit(.) %>% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   GRanges(.)

MI_gwas_gr_noext <- MI_gwas %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit(.) %>% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   GRanges(.)
 # rtracklayer::export.bed(MI_gwas_gr_noext , con="data/n45_bedfiles/MI_gwas_gr_noext.bed", format="bed", ignore.strand = FALSE)

gwas_1_gr <- gwas_1 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()

gwas_1_gr_ten <- gwas_1 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()
# rtracklayer::export.bed(gwas_1_gr , con="data/n45_bedfiles/cardiotox_gwas_gr_ext.bed", format="bed", ignore.strand = FALSE)
gwas_1_gr_noext <- gwas_1 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()

gwas_2_gr <- gwas_2 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
   mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
  na.omit() %>% 
  GRanges()

gwas_2_gr_ten <- gwas_2 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
   mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
  na.omit() %>% 
  GRanges()

 
gwas_2_gr_noext <- gwas_2 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
  na.omit() %>% 
  GRanges()

# rtracklayer::export.bed(gwas_2_gr_noext , con="data/n45_bedfiles/ARR_gwas_gr_noext.bed", format="bed", ignore.strand = FALSE)

gwas_3_gr <- gwas_3 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()


gwas_3_gr_ten <- gwas_3 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()

gwas_3_gr_noext <- gwas_3 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()

gwas_4_gr <-  gwas_4 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_POS= as.integer(CHR_POS)) %>% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID))%>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()
 
gwas_4_gr_ten <-  gwas_4 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_POS= as.integer(CHR_POS)) %>% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID))%>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()
gwas_4_gr_noext <-  gwas_4 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(CHR_POS= as.integer(CHR_POS)) %>% 
   mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()

# rtracklayer::export.bed(gwas_4_gr_noext , con="data/n45_bedfiles/HD_gwas_gr_noext.bed", format="bed", ignore.strand = FALSE)

gwas_5_gr <- gwas_5 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS-500, end=CHR_POS+500, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()
gwas_5_gr_ten <- gwas_5 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS-5000, end=CHR_POS+5000, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()


gwas_5_gr_noext <- gwas_5 %>% 
  dplyr::select(CHR_ID,CHR_POS,SNPS) %>% 
  mutate(start=CHR_POS, end=CHR_POS+1, chr=paste0("chr",CHR_ID)) %>%
  dplyr::select(chr, start,end, SNPS,CHR_POS) %>%
   na.omit() %>% 
  GRanges()
# saveRDS(list("gwas_2_gr_noext"=gwas_2_gr_noext, "gwas_4_gr_noext"= gwas_4_gr_noext),"data/test.list.RDS")
# gwas_2[(is.na(gwas_2$CHR_POS)),]
```


#### GWAS overlap findings

```{r gwas overlaps, include=FALSE}
findOverlaps(EAR_df_gr, gwas_1_gr)
findOverlaps(ESR_df_gr, gwas_1_gr)
findOverlaps(LR_df_gr, gwas_1_gr)
findOverlaps(NR_df_gr, gwas_1_gr)

findOverlaps(EAR_df_gr, gwas_2_gr)
findOverlaps(ESR_df_gr, gwas_2_gr)
findOverlaps(LR_df_gr,  gwas_2_gr)
findOverlaps(NR_df_gr, gwas_2_gr)


findOverlaps(EAR_df_gr, gwas_3_gr)
findOverlaps(ESR_df_gr, gwas_3_gr)
findOverlaps(LR_df_gr, gwas_3_gr)
findOverlaps(NR_df_gr, gwas_3_gr)

findOverlaps(EAR_df_gr, gwas_4_gr)
findOverlaps(ESR_df_gr, gwas_4_gr)
findOverlaps(LR_df_gr, gwas_4_gr)
findOverlaps(NR_df_gr, gwas_4_gr)

findOverlaps(EAR_df_gr, gwas_5_gr)
findOverlaps(ESR_df_gr, gwas_5_gr)
findOverlaps(LR_df_gr, gwas_5_gr)
findOverlaps(NR_df_gr, gwas_5_gr)

findOverlaps(EAR_df_gr, CAD_gwas_gr)
findOverlaps(ESR_df_gr,  CAD_gwas_gr)
findOverlaps(LR_df_gr,  CAD_gwas_gr)
findOverlaps(NR_df_gr,  CAD_gwas_gr)

findOverlaps(EAR_df_gr, MI_gwas_gr)
findOverlaps(ESR_df_gr,  MI_gwas_gr)
findOverlaps(LR_df_gr,  MI_gwas_gr)
findOverlaps(NR_df_gr,  MI_gwas_gr)

EAR_g1_ol <- findOverlaps(EAR_df_gr, gwas_1_gr_noext)
ESR_g1_ol <-findOverlaps(ESR_df_gr, gwas_1_gr_noext)
LR_g1_ol <-findOverlaps(LR_df_gr, gwas_1_gr_noext)
NR_g1_ol <-findOverlaps(NR_df_gr, gwas_1_gr_noext)

EAR_g2_ol <-findOverlaps(EAR_df_gr, gwas_2_gr_noext)
ESR_g2_ol <-findOverlaps(ESR_df_gr, gwas_2_gr_noext)
LR_g2_ol <-findOverlaps(LR_df_gr,  gwas_2_gr_noext)
NR_g2_ol <-findOverlaps(NR_df_gr, gwas_2_gr_noext)


EAR_g3_ol <-findOverlaps(EAR_df_gr, gwas_3_gr_noext)
ESR_g3_ol <-findOverlaps(ESR_df_gr, gwas_3_gr_noext)
LR_g3_ol <-findOverlaps(LR_df_gr, gwas_3_gr_noext)
NR_g3_ol <-findOverlaps(NR_df_gr, gwas_3_gr_noext)

EAR_g4_ol <-findOverlaps(EAR_df_gr, gwas_4_gr_noext)
ESR_g4_ol <-findOverlaps(ESR_df_gr, gwas_4_gr_noext)
LR_g4_ol <-findOverlaps(LR_df_gr, gwas_4_gr_noext)
NR_g4_ol <-findOverlaps(NR_df_gr, gwas_4_gr_noext)

EAR_g5_ol <-findOverlaps(EAR_df_gr, gwas_5_gr_noext)
ESR_g5_ol <-findOverlaps(ESR_df_gr, gwas_5_gr_noext)
LR_g5_ol <-findOverlaps(LR_df_gr, gwas_5_gr_noext)
NR_g5_ol <-findOverlaps(NR_df_gr, gwas_5_gr_noext)




EAR_CAD_ol <-findOverlaps(EAR_df_gr, CAD_gwas_gr_noext)
ESR_CAD_ol <-findOverlaps(ESR_df_gr,  CAD_gwas_gr_noext)
LR_CAD_ol <-findOverlaps(LR_df_gr,  CAD_gwas_gr_noext)
NR_CAD_ol <-findOverlaps(NR_df_gr,  CAD_gwas_gr_noext)

findOverlaps(EAR_df_gr, MI_gwas_gr_noext)
findOverlaps(ESR_df_gr,  MI_gwas_gr_noext)
findOverlaps(LR_df_gr,  MI_gwas_gr_noext)
findOverlaps(NR_df_gr,  MI_gwas_gr_noext)


```

| motif       | gwas_1 (11)                                        | gwas_2 (1061)                                      | gwas_3 (39)                                        | gwas_4 (6052)                                      | Heart Failure (445)                                        | CAD_gwas (3482)                                      |
|-----------|-----------|-----------|-----------|-----------|-----------|-----------|
| EAR (7321)  | `r length(findOverlaps(EAR_df_gr, gwas_1_gr_ten))` | `r length(findOverlaps(EAR_df_gr, gwas_2_gr_ten))` | `r length(findOverlaps(EAR_df_gr, gwas_3_gr_ten))` | `r length(findOverlaps(EAR_df_gr, gwas_4_gr_ten))` | `r  length(findOverlaps(EAR_df_gr, gwas_5_gr_ten))` | `r length(findOverlaps(EAR_df_gr, CAD_gwas_gr_ten))` |
| ESR (15665) | `r length(findOverlaps(ESR_df_gr, gwas_1_gr_ten))` | `r length(findOverlaps(ESR_df_gr, gwas_2_gr_ten))` | `r length(findOverlaps(ESR_df_gr, gwas_3_gr_ten))` | `r length(findOverlaps(ESR_df_gr, gwas_4_gr_ten))` | `r length(findOverlaps(ESR_df_gr, gwas_5_gr_ten))`  | `r length(findOverlaps(ESR_df_gr, CAD_gwas_gr_ten))` |
| LR (41759)  | `r length(findOverlaps(LR_df_gr, gwas_1_gr_ten))`  | `r length(findOverlaps(LR_df_gr, gwas_2_gr_ten))`  | `r length(findOverlaps(LR_df_gr, gwas_3_gr_ten))`  | `r length(findOverlaps(LR_df_gr, gwas_4_gr_ten))`  | `r length(findOverlaps(LR_df_gr, gwas_5_gr_ten))`   | `r length(findOverlaps(LR_df_gr, CAD_gwas_gr_ten))`  |
| NR (83658)  | `r length(findOverlaps(NR_df_gr, gwas_1_gr_ten))`  | `r length(findOverlaps(NR_df_gr, gwas_2_gr_ten))`  | `r length(findOverlaps(NR_df_gr, gwas_3_gr_ten))`  | `r length(findOverlaps(NR_df_gr, gwas_4_gr_ten))`  | `r length(findOverlaps(NR_df_gr, gwas_5_gr_ten))`   | `r length(findOverlaps(NR_df_gr, CAD_gwas_gr_ten))`  |

: gwas trait enrichment with extensions of 5,000 bp +/- SNP location

| motif       | gwas_1 (11)                                    | gwas_2 (1061)                                  | gwas_3 (39)                                    | gwas_4 (6052)                                  | Heart Failure (445)                                   | CAD_gwas (3482)                                  |
|-----------|-----------|-----------|-----------|-----------|-----------|-----------|
| EAR (7321)  | `r length(findOverlaps(EAR_df_gr, gwas_1_gr))` | `r length(findOverlaps(EAR_df_gr, gwas_2_gr))` | `r length(findOverlaps(EAR_df_gr, gwas_3_gr))` | `r length(findOverlaps(EAR_df_gr, gwas_4_gr))` | `r length(findOverlaps(EAR_df_gr, gwas_5_gr))` | `r length(findOverlaps(EAR_df_gr, CAD_gwas_gr))` |
| ESR (15665) | `r length(findOverlaps(ESR_df_gr, gwas_1_gr))` | `r length(findOverlaps(ESR_df_gr, gwas_2_gr))` | `r length(findOverlaps(ESR_df_gr, gwas_3_gr))` | `r length(findOverlaps(ESR_df_gr, gwas_4_gr))` | `r length(findOverlaps(ESR_df_gr, gwas_5_gr))` | `r length(findOverlaps(ESR_df_gr, CAD_gwas_gr))` |
| LR (41759)  | `r length(findOverlaps(LR_df_gr, gwas_1_gr))`  | `r length(findOverlaps(LR_df_gr, gwas_2_gr))`  | `r length(findOverlaps(LR_df_gr, gwas_3_gr))`  | `r length(findOverlaps(LR_df_gr, gwas_4_gr))`  | `r length(findOverlaps(LR_df_gr, gwas_5_gr))`  | `r length(findOverlaps(LR_df_gr, CAD_gwas_gr))`  |
| NR (83658)  | `r length(findOverlaps(NR_df_gr, gwas_1_gr))`  | `r length(findOverlaps(NR_df_gr, gwas_2_gr))`  | `r length(findOverlaps(NR_df_gr, gwas_3_gr))`  | `r length(findOverlaps(NR_df_gr, gwas_4_gr))`  | `r length(findOverlaps(NR_df_gr, gwas_5_gr))`  | `r length(findOverlaps(NR_df_gr, CAD_gwas_gr))`  |

: gwas trait enrichment with extensions of 500 bp +/- SNP location

| motif       | gwas_1 (11) | gwas_2 (1061) | gwas_3 (39) | gwas_4 (6052) | Heart Failure (445) | CAD_gwas (3482) | MI_gwas (399) |     |
|--------|--------|--------|--------|--------|--------|--------|--------|--------|
| EAR (7321)  | 0           | 2             | 0           | 17            | 0            | 12              | 1             |     |
| ESR (15665) | 0           | 30            | 0           | 55            | 2            | 17              | 0             |     |
| LR (41759)  | 2           | 29            | 4           | 182           | 15           | 116             | 15            |     |
| NR (83658)  | 0           | 90            | 0           | 278           | 11           | 146             | 19            |     |

: gwas trait enrichment with no extension

These numbers are different than the original lists from which they derived. This is normally due to running `na.omit()` on the data frame before GRanges transformation. \#### chisquare tests

```{r chi numbers}
chitest_gwas2_EAR <- matrix(c(2,7319,90,83568), ncol=2, byrow=TRUE)
chitest_gwas2_ESR <- matrix(c(30,15635,90,83568), ncol=2, byrow=TRUE)
chitest_gwas2_LR <- matrix(c(20,41739,90,83568), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas2_EAR)
chisq.test(chitest_gwas2_ESR)
chisq.test(chitest_gwas2_LR)



chitest_gwas4_EAR <- matrix(c(17,7304,292,83366), ncol=2, byrow=TRUE)
chitest_gwas4_ESR <- matrix(c(55,15610,292,83366), ncol=2, byrow=TRUE)
chitest_gwas4_LR <- matrix(c(186,41573,292,83366), ncol=2, byrow=TRUE)
chisq.test(chitest_gwas4_EAR)
chisq.test(chitest_gwas4_ESR)
chisq.test(chitest_gwas4_LR)


chitest_K4_EAR <- matrix(c(0,7321,32,83626), ncol=2, byrow=TRUE)
chitest_K4_ESR <- matrix(c(3,15662,32,83626), ncol=2, byrow=TRUE)
chitest_K4_LR <- matrix(c(14,417545,32,83626), ncol=2, byrow=TRUE)
chisq.test(chitest_K4_EAR)
chisq.test(chitest_K4_ESR)
chisq.test(chitest_K4_LR)


chitest_K5_EAR <- matrix(c(1,7320,15,83643), ncol=2, byrow=TRUE)
chitest_K5_ESR <- matrix(c(2,15663,15,83643), ncol=2, byrow=TRUE)
chitest_K5_LR <- matrix(c(10,41749,15,83643), ncol=2, byrow=TRUE)
chisq.test(chitest_K5_EAR)
chisq.test(chitest_K5_ESR)
chisq.test(chitest_K5_LR)


chitest_K6_EAR <- matrix(c(0,7321,19,83639), ncol=2, byrow=TRUE)
chitest_K6_ESR <- matrix(c(2,15663,19,83639), ncol=2, byrow=TRUE)
chitest_K6_LR <- matrix(c(9,41750,19,83639), ncol=2, byrow=TRUE)
chisq.test(chitest_K6_EAR)
chisq.test(chitest_K6_ESR)
chisq.test(chitest_K6_LR)

chitest_CAD_EAR <- matrix(c(12,7309,146,83512), ncol=2, byrow=TRUE)
chitest_CAD_ESR <- matrix(c(17,15648,146,83512), ncol=2, byrow=TRUE)
chitest_CAD_LR <- matrix(c(116,41643,146,83512), ncol=2, byrow=TRUE)
chisq.test(chitest_CAD_EAR)
chisq.test(chitest_CAD_ESR)
chisq.test(chitest_CAD_LR)



```

#### subsetting overlaps

```{r subsetoverlap to check for peaks/nearestgene}
##sub-setting  peaks that overlap with at least 1 SNP in gwas2 noext

EAR_g2_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_2_gr_noext))
# subsetByOverlaps(gwas_2_gr_noext, ESR_df_gr)
ESR_g2_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_2_gr_noext))
LR_g2_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_2_gr_noext))
NR_g2_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_2_gr_noext))

EAR_g4_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_4_gr_noext))
ESR_g4_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_4_gr_noext))
LR_g4_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_4_gr_noext))
NR_g4_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_4_gr_noext))

EAR_g1_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_1_gr_noext))

ESR_g1_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_1_gr_noext))
LR_g1_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_1_gr_noext))
NR_g1_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_1_gr_noext))


EAR_g3_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_3_gr_noext))

ESR_g3_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_3_gr_noext))
LR_g3_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_3_gr_noext))
NR_g3_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_3_gr_noext))

EAR_g5_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, gwas_5_gr_noext))
ESR_g5_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, gwas_5_gr_noext))
LR_g5_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  gwas_5_gr_noext))
NR_g5_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, gwas_5_gr_noext))

EAR_MI_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, MI_gwas_gr))

ESR_MI_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, MI_gwas_gr))
LR_MI_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  MI_gwas_gr))
NR_MI_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, MI_gwas_gr))

EAR_CAD_sol <- as.data.frame(subsetByOverlaps(EAR_df_gr, CAD_gwas_gr))

ESR_CAD_sol <- as.data.frame(subsetByOverlaps(ESR_df_gr, CAD_gwas_gr))
LR_CAD_sol <- as.data.frame(subsetByOverlaps(LR_df_gr,  CAD_gwas_gr))
NR_CAD_sol <- as.data.frame(subsetByOverlaps(NR_df_gr, CAD_gwas_gr))

###Now I can pull the geneID list and see if genes are DE from RNAseq data
```

### RNA MRC (motif response cluster)
```{r adding in RNA genes}
toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") 
S13Table <- read.csv( "data/other_papers/S13Table_Matthews2024.csv",row.names = 1)
##14021

EAR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="EAR") %>% 
  dplyr::select(ENTREZID) %>% 
  mutate(ENTREZID= as.character(ENTREZID))
ESR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="ESR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))
LR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="LR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID=as.character(ENTREZID))
NR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="NR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))

###Because of how I applied the DEG system in RNA-seq analysis, the lFC is opposite of the 
###counts.   I did trt-veh instead of veh-trt.  therefore I need to multiply lfc by -1 to get t
###the right correlation.

toplistall_RNA <- toplistall_RNA %>% 
  mutate(logFC = logFC*(-1))

RNA_expresed_genes <- toplistall_RNA %>% 
  # dplyr::filter(adj.P.Val <0.05) %>% 
   mutate(expression = if_else(logFC<0,"down","up")) %>% 
  dplyr::select(ENTREZID,SYMBOL,expression) %>% 
  # dplyr::select(ENTREZID,SYMBOL) %>% 
  unique(.)  
RNA_expresed_genes_DE <- toplistall_RNA %>% 
  dplyr::filter(adj.P.Val <0.05) %>%
  mutate(expression = if_else(logFC<0,"down","up")) %>% 
  dplyr::select(ENTREZID,SYMBOL,expression) %>% 
  unique(.)  

# RNA_expresed_genes %>% 
#   dplyr::filter(ENTREZID %in% NR_g4_sol$geneID)

NR_g2_list <- intersect(RNA_expresed_genes$ENTREZID,NR_g2_sol$geneId)
LR_g2_list <- intersect(RNA_expresed_genes$ENTREZID,LR_g2_sol$geneId)
ESR_g2_list <- intersect(RNA_expresed_genes$ENTREZID,ESR_g2_sol$geneId)
EAR_g2_list <- intersect(RNA_expresed_genes$ENTREZID,EAR_g2_sol$geneId)
g2_motif_list <-list(NR=data.frame(NR_g2_list), LR=data.frame(LR_g2_list),ESR=data.frame(ESR_g2_list),EAR=data.frame(EAR_g2_list))
g2_motif_df <- g2_motif_list %>%  
  map_df(c, .id = "src") %>%
  mutate(ENTREZID=if_else(!is.na(EAR_g2_list),EAR_g2_list,if_else(!is.na(ESR_g2_list),ESR_g2_list, if_else(!is.na(LR_g2_list), LR_g2_list,NR_g2_list))))%>%
  dplyr::select(src,ENTREZID)
NR_g4_list <-intersect(RNA_expresed_genes$ENTREZID,NR_g4_sol$geneId)
LR_g4_list <- intersect(RNA_expresed_genes$ENTREZID,LR_g4_sol$geneId)
ESR_g4_list <- intersect(RNA_expresed_genes$ENTREZID,ESR_g4_sol$geneId)
EAR_g4_list <- intersect(RNA_expresed_genes$ENTREZID,EAR_g4_sol$geneId)



NR_g5_list <-intersect(RNA_expresed_genes$ENTREZID,NR_g5_sol$geneId)
LR_g5_list <- intersect(RNA_expresed_genes$ENTREZID,LR_g5_sol$geneId)
ESR_g5_list <- intersect(RNA_expresed_genes$ENTREZID,ESR_g5_sol$geneId)
EAR_g5_list <- intersect(RNA_expresed_genes$ENTREZID,EAR_g5_sol$geneId)


g4_motif_list <-list(NR=data.frame(NR_g4_list), LR=data.frame(LR_g4_list),ESR=data.frame(ESR_g4_list),EAR=data.frame(EAR_g4_list))
g4_motif_df <- g4_motif_list %>%  map_df(c, .id = "src") %>%
  mutate(ENTREZID=if_else(!is.na(EAR_g4_list),EAR_g4_list,if_else(!is.na(ESR_g4_list),ESR_g4_list, if_else(!is.na(LR_g4_list), LR_g4_list,NR_g4_list))))%>%
  dplyr::select(src,ENTREZID)

g5_motif_list <-list(NR=data.frame(NR_g5_list), LR=data.frame(LR_g5_list),ESR=data.frame(ESR_g5_list),EAR=data.frame(EAR_g5_list))
g5_motif_df <- g5_motif_list %>%  map_df(c, .id = "src") %>%
  mutate(ENTREZID=if_else(!is.na(EAR_g5_list),EAR_g5_list,if_else(!is.na(ESR_g5_list),ESR_g5_list, if_else(!is.na(LR_g5_list), LR_g5_list,NR_g5_list))))%>%
  dplyr::select(src,ENTREZID)

RNA_expresed_genes_DE %>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g4_sol$geneId)) %>% 
   rowid_to_column("row_name") %>% 
  kable(., caption= "RNA DE genes associated with peaks in ATAC-LR that contain SNPs associated with Heart disease") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
RNA_expresed_genes_DE %>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g5_sol$geneId)) %>% 
   rowid_to_column("row_name") %>% 
  kable(., caption= "RNA DE genes expresssed in LR MRCs that contain SNPs associated with Heart failure") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")


RNA_expresed_genes_DE %>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,LR_g2_sol$geneId)) %>% 
   rowid_to_column("row_name") %>% 
  kable(., caption= "RNA DE genes expresssed in LR MRCs that contain SNPs associated with Cardiac arrhythmia") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")


RNA_expresed_genes_DE %>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g4_sol$geneId)) %>% 
   rowid_to_column("row_name") %>% 
  kable(., caption= "RNA DE genes expresssed in ESR MRCs that contain SNPs associated with Heart disease") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")

RNA_expresed_genes_DE %>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,ESR_g2_sol$geneId)) %>% 
   rowid_to_column("row_name") %>% 
  kable(., caption= "RNA DE genes expresssed in ESR MRCs that contain SNPs associated with Cardiac arrhythmia") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")


RNA_expresed_genes_DE %>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes_DE$ENTREZID,EAR_g4_sol$geneId)) %>% 
   rowid_to_column("row_name") %>% 
  kable(., caption= "RNA DE genes expresssed in EAR MRCs that contain SNPs associated with Heart disease") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")

RNA_expresed_genes%>% 
  dplyr::filter(ENTREZID %in% intersect(RNA_expresed_genes$ENTREZID,EAR_g2_sol$geneId)) %>%
   # dplyr::filter(ENTREZID =="79804") %>% 
   rowid_to_column("row_name") %>%
  kable(., caption= "RNA genes expresssed in EAR MRCs that contain SNPs associated with Cardiac arrhythmia") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")



```


```{r MRC and RNAseq}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
##Here my focus is to find nearest genes that are MRC and see if they are DEGs
toplist_n45 <- readRDS("data/toplist_n45.RDS")

# toplist_n45 %>% 
#   dplyr::filter(peak=="chr4.173529475.173530537") %>% 
#  ggplot(., aes(x=trt, y=logFC))+
#   geom_boxplot(aes(fill=trt))+
#   ggpubr::fill_palette(palette =drug_pal)+
#   guides(fill=guide_legend(title = "Treatment"))+
#   # facet_wrap(sigcount~time)+
#   theme_bw()+
#   xlab("")+
#   ggtitle("chr4.173529475.173530537 region of  HAND2-AS1")+
#   ylab(expression("Log"[2]*" fold change"))+
#   theme_bw()+
#   facet_wrap(~time)+
#   theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
#         axis.title = element_text(size = 15, color = "black"),
#         # axis.ticks = element_line(linewidth = 1.5),
#         # axis.line = element_line(linewidth = 1.5),
#         strip.background = element_rect(fill = "transparent"),
#         axis.text.x = element_blank(),
#         strip.text.x = element_text(size = 12, color = "black", face = "bold"))


toplist_n45 %>% 
  dplyr::filter(peak=="chr6.36678380.36679788") %>%
 ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("chr6  36678380-36679788 CDKN1a")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))



toplist_n45 %>% 
  dplyr::filter(peak %in% EAR_g2_sol$id) %>% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("absolute lfc of MRCs in EAR in cardiac arrhythmia")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))

toplist_n45 %>% 
  dplyr::filter(peak %in% ESR_g2_sol$id) %>% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("absolute lfc of MRCs in ESR in cardiac arrhythmia")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


toplist_n45 %>% 
  dplyr::filter(peak %in% LR_g2_sol$id) %>% 
 ggplot(., aes(x=trt, y=abs(logFC)))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("absolute lfc of MRCs in LR in cardiac arrhythmia")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


```

### Pie charts of the SNPs

```{r  piechart noext }

Set <- c("EAR", "ESR","LR", "NR")
gene_num_ARR <- c(2,30,29,90)
gene_num_HD <- c(17,55,182,278)
gene_num_CAD <- c(12,17,116,146)
gene_num_MI <- c(1,0,15,19)
gene_num_HF <- c(0,2,15,11)
fills <- c("#F8766D",  "#00BFC4","#7CAE00", "#C77CFF")

pie_chartdata_ARR <- data.frame(Set, gene_num_ARR, fills)
pie_chartdata_HD <- data.frame(Set, gene_num_HD, fills)
pie_chartdata_CAD <- data.frame(Set, gene_num_CAD, fills)
pie_chartdata_MI <- data.frame(Set, gene_num_MI, fills)
pie_chartdata_HF <- data.frame(Set, gene_num_HF, fills)

 
    # pie(gene_num,labels =Set )      
pie_chartdata_ARR <- pie_chartdata_ARR %>% 
   mutate(prop = gene_num_ARR / sum(pie_chartdata_ARR$gene_num_ARR) *100) %>%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_HD <- pie_chartdata_HD %>% 
   mutate(prop = gene_num_HD / sum(pie_chartdata_HD$gene_num_HD) *100) %>%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_CAD <- pie_chartdata_CAD %>% 
   mutate(prop = gene_num_CAD / sum(pie_chartdata_CAD$gene_num_CAD) *100) %>%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_MI <- pie_chartdata_MI %>% 
   mutate(prop = gene_num_MI / sum(pie_chartdata_MI$gene_num_MI) *100) %>%
  mutate(ypos = (prop)+ 0.5*prop )
pie_chartdata_HF <- pie_chartdata_HF %>% 
   mutate(prop = gene_num_MI / sum(pie_chartdata_HF$gene_num_HF) *100) %>%
  mutate(ypos = (prop)+ 0.5*prop )




pie_chartdata_ARR %>% 
  ggplot(.,aes(x="",y=gene_num_ARR, fill=Set))+
  geom_col(width =1) +
  coord_polar("y", pi/2)+
  theme_void()+
  ggtitle("Distribution of cardiac arrhythmia SNPs in MRCs")+
  geom_text(aes(label = paste0(Set," (",gene_num_ARR,")")),
                position = position_stack(vjust =.45)) +
  theme(legend.position="none") +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

pie_chartdata_HD %>% 
  ggplot(.,aes(x="",y=gene_num_HD, fill=Set))+
  geom_col(width =1) +
  coord_polar("y", pi/2)+
  theme_void()+
  ggtitle("Distribution of heart disease SNPs in MRCs")+
  geom_text(aes(label = paste0(Set," (",gene_num_HD,")")),
                position = position_stack(vjust =.45)) +
  theme(legend.position="none") +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))


pie_chartdata_CAD %>% 
  ggplot(.,aes(x="",y=gene_num_CAD, fill=Set))+
  geom_col(width =1) +
  coord_polar("y", pi/2)+
  theme_void()+
  ggtitle("Distribution of coronary artery disease SNPs in MRCs")+
  geom_text(aes(label = paste0(Set," (",gene_num_CAD,")")),
                position = position_stack(vjust =.45)) +
  theme(legend.position="none") +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

pie_chartdata_MI %>% 
  ggplot(.,aes(x="",y=gene_num_MI, fill=Set))+
  geom_col(width =1) +
  coord_polar("y", pi/2)+
  theme_void()+
  ggtitle("Distribution of myocardial infarction SNPs in MRCs")+
  geom_text(aes(label = paste0(Set," (",gene_num_MI,")")),
                position = position_stack(vjust =.45)) +
  theme(legend.position="none") +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

pie_chartdata_HF %>% 
  ggplot(.,aes(x="",y=gene_num_HF, fill=Set))+
  geom_col(width =1) +
  coord_polar("y", pi/2)+
  theme_void()+
  ggtitle("Distribution of Heart failure SNPs in MRCs")+
  geom_text(aes(label = paste0(Set," (",gene_num_HF,")")),
                position = position_stack(vjust =.45)) +
  theme(legend.position="none") +
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))
```



```{r expressionDE chart}

### Code to express # of DE vs non DE genes associated with SNPs found in MRCs

# toplist_n45 <- readRDS("data/toplist_n45.RDS")
# 
# toplist_n45 %>% 
#   mutate(DAR= if_else(adj.P.Val<0.05,"DAR","notDAR")) 
         


GWAS_2_DAR_df <-
  EAR_g2_sol %>%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
ESR_g2_sol %>% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
LR_g2_sol %>%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
NR_g2_sol %>%
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="NR_ATAC"))%>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))
         
    
GWAS_4_DAR_df <- EAR_g4_sol %>%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
  ESR_g4_sol %>% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
    mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
    LR_g4_sol %>%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
    NR_g4_sol %>%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="NR_ATAC"))%>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))



GWAS_1_DAR_df <- EAR_g1_sol %>%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
  ESR_g1_sol %>% 
    mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
    mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
    LR_g1_sol %>%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
    NR_g1_sol %>%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="NR_ATAC"))%>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))

GWAS_3_DAR_df <- EAR_g3_sol %>%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
  ESR_g3_sol %>% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
    mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
    LR_g3_sol %>%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
    NR_g3_sol %>%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="NR_ATAC"))%>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))

GWAS_5_DAR_df <- EAR_g5_sol %>%
  mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
  ESR_g5_sol %>% 
    mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
    mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
    LR_g5_sol %>%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
    NR_g5_sol %>%
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="NR_ATAC"))%>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))



GWAS_MI_DAR_df <- EAR_MI_sol %>%
 mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
  ESR_MI_sol %>% 
   mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
    mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
    LR_MI_sol %>%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
    NR_MI_sol %>%
      mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      mutate(ATAC_MRC="NR_ATAC"))%>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))




GWAS_CAD_DAR_df <-
  EAR_CAD_sol %>%
    mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
                        # if_else(geneId %in% 
                        #           intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),"DE",
                        #         if_else(geneId %in% 
                        #                   intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),"DE",
                        #                 if_else(geneId %in% 
                        #                           intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), "notDE","NE"))))) %>%
  mutate(ATAC_MRC="EAR_ATAC") %>% 
bind_rows(
  ESR_CAD_sol %>% 
     mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
    # mutate(DE_g2= if_else(geneId %in%
    #                         intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),"DE",
    #                       if_else(geneId %in% 
    #                                 intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),"DE",
    #                             if_else(geneId %in% 
    #                                   intersect(RNA_expresed_genes_DE$ENTREZID,LR_CAD_sol$geneId),"DE",
    #                                     if_else(geneId %in% 
    #                                       intersect(RNA_expresed_genes_DE$ENTREZID,NR_CAD_sol$geneId), "notDE","NE")))))%>%
    mutate(ATAC_MRC="ESR_ATAC")) %>% 
  bind_rows(
    LR_CAD_sol %>%
       mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      # mutate(DE_g2= if_else(geneId %in% 
      #                         intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),"DE",
      #                   if_else(geneId %in% 
      #                             intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),"DE",
      #                           if_else(geneId %in% 
      #                                   intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),"DE",
      #                                   if_else(geneId %in% 
      #                                             intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), "notDE","NE")))))%>%
      mutate(ATAC_MRC="LR_ATAC")) %>% 
  bind_rows(
    NR_CAD_sol %>%
       mutate(DE_g2= if_else(geneId %in% RNA_expresed_genes_DE$ENTREZID,"DE","notDE")) %>% 
      # mutate(DE_g2= if_else(geneId %in% 
      #                         intersect(RNA_expresed_genes_DE$ENTREZID,ESR_CAD_sol$geneId),"DE",
      #                   if_else(geneId %in% 
      #                             intersect(RNA_expresed_genes_DE$ENTREZID,EAR_CAD_sol$geneId),"DE",
      #                           if_else(geneId %in% 
      #                                     intersect(RNA_expresed_genes_DE$ENTREZID, LR_CAD_sol$geneId),"DE",
      #                                   if_else(geneId %in% 
      #                                             intersect(RNA_expresed_genes_DE$ENTREZID, NR_CAD_sol$geneId), "notDE_ATAC","NE")))))%>%
      mutate(ATAC_MRC="NR_ATAC")) %>% 
  mutate(RNA_MRC= if_else(geneId %in% EAR_RNA$ENTREZID,"EAR_RNA",
                          if_else(geneId  %in% ESR_RNA$ENTREZID,"ESR_RNA",
                                  if_else(geneId %in% LR_RNA$ENTREZID,"LR_RNA",
         if_else(geneId %in% NR_RNA$ENTREZID, "NR_RNA","non-MRC_RNA")))))
         
         

```

## ARR, HD, HF, MI, CAD snp overlaps and MRC status

```{r  motifMRC enrichment}
GWAS_5_DAR_df %>% 
  # dplyr::filter(DE_g2=="DE")%>% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS) %>% 
  kable(.,caption="ATAC peaks that overlap with heart disease SNPs, annotated with RNA MRC and DE status") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")




GWAS_2_DAR_df %>% 
  # dplyr::filter(DE_g2=="DE")%>% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS) %>% 
  kable(.,caption="ATAC peaks that overlap with cardiac arrhythmia SNPs, annotated with RNA MRC and DE status") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")

GWAS_4_DAR_df %>% 
  # dplyr::filter(DE_g2=="DE")%>% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC,RNA_MRC,annotation:distanceToTSS)%>% 
  kable(.,caption="ATAC peaks that overlap with heart disease SNPs, annotated with RNA MRC and DE status") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")

GWAS_MI_DAR_df %>% 
  # dplyr::filter(DE_g2=="DE")%>% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS)%>% 
  kable(.,caption="ATAC peaks that overlap with myocardial infarction SNPs, annotated with RNA MRC and DE status") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")

GWAS_CAD_DAR_df %>% 
  # dplyr::filter(DE_g2=="DE") %>% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS)%>% 
  kable(.,caption="ATAC peaks that overlap with coronary artery disease SNPs, annotated with RNA MRC and DE status") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")


GWAS_5_DAR_df %>% 
  # dplyr::filter(DE_g2=="DE") %>% 
  dplyr::select(seqnames:id,DE_g2,ATAC_MRC, RNA_MRC,annotation:distanceToTSS)%>% 
  kable(.,caption="ATAC peaks that overlap with Heart SNPs, annotated with RNA MRC and DE status") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")


```

```{r CDKn1a log2cpm MRC-RNa}
my_hc_filtered_counts_n45 <- readRDS("data/my_hc_filt_counts_n45.RDS")
log_filt_hc_n45 <- cpm(my_hc_filtered_counts_n45, log = TRUE) %>% as.data.frame()
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")

log_filt_hc_n45 %>% 
 #  rownames_to_column("peak") %>% 
 #  dplyr::filter(peak %in% motif_list_n45$EAR) 
 # log_filt_hc %>% 
  dplyr::filter(row.names(.)=="chr6.36678380.36679788") %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle("log2cpm in peak with SNP CDKN1a")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))
Counts_RNA_ERMatthews <- readRDS("data/other_papers/Counts_RNA_ERMatthews.RDS")
# Counts_RNA_ERMatthews <-  
Counts_RNA_ERMatthews %>% 
  column_to_rownames("ENTREZID") %>% 
  cpm(.,log = TRUE) %>% 
  as.data.frame() %>% 
  rownames_to_column("expr") %>%
  filter(expr == 1026) %>% 
  pivot_longer(cols =!expr , names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("trt","indv","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  # facet_wrap(Peak~.)+
  ggtitle("log2cpm RNA in ESR CDKN1a")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))
  
  



```

### KEGG pathway enrichment by peaks location
```{r function profiles comp}
library(clusterProfiler)
genes = lapply(peakAnnoList_n45_motif[1:4], function(i) as.data.frame(i)$geneId)
names(genes) = sub("_", "\n", names(genes))
compKEGG <- compareCluster(geneCluster   = genes,
                         fun           = "enrichKEGG",
                         pvalueCutoff  = 0.05,
                         pAdjustMethod = "BH")
dotplot(compKEGG, showCategory = 15, title = "KEGG Pathway Enrichment Analysis")

```

### Mapping peaks to nearest RNA-expressed genes

The code below is the documentation of what I did to make the RNA-gene list from my expressed genes.

I first used `biomaRt` package to retrieve the chromosome information for each ENTREZID.  This game me 13114 rows, with > 500 rows containing string-only chromosomal locations.  I then tried using the hgnc symbol to pull up gene information and  retrieved  'chromosome_name', 'start_position', 'end_position','entrezgene_id', 'ensembl_gene_id', 'hgnc_symbol' for 15,497 rows of genes from my expressed gene list. This list again contained multiple string-only chromosomal locations.  I filtered out non-numeric chromosomes after looking up several genes that had multiple entries with at least one of the entries containing no ENTREZID or incorrect coordinates or symbols, leaving me with 13,426 rows of unique genes, but several contained `NAs` in the ENTREZID column.  Since I generally use ENTREZIDs to connect my data, I removed those genes and was left with 12,984 unique genes from my RNA expressed gene list.  
 
The next steps involved exporting this file as a .bed file and using `bedtools closest` function to map the nearest expressed gene to each peak.   I used `bedtools closest -a background.bed -b Expressedgenes.bed -d > assigned_closest_gene.bed`  to map accessible peaks, ie 151,675 peaks.  Some peaks were assigned more than one neargene, and I kept all neargenes assigned in one file with 156,431 rows.  I can collapse these into a comma deliminated file, but for now I am keeping separated lines.


NOTE FOR MYSELF  (are we base 0 or base 1!!!!!)  
The peak file names do not match the bed file names due to the 0/1 issue.  This means to match up MRC categories by name, I need to alter the first start position by +1 to get the "new" names to match the old names!
```{r overlapping closest, eval=FALSE, include=TRUE}

# get RNA gene list:  (14,084)  Can be done using the S13 from my previous data-  i just went to UCSC browser and 
# followed this post on Biostars:  https://bioinformatics.stackexchange.com/questions/17647/create-bed-file-from-list-with-gene-symbols

## I then took the background peaks list and ran "bedtools closest -a background.bed -b Expressedgenes.bed -d -k 2 > assigned_closest_gene.bed"

### Now i can load the file here intersect information
# assigned_closest_gene <- read_delim("data/n45_bedfiles/assigned_closest_gene.bed", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# gene_ref <- assigned_closest_gene$X10
# gene_ref <- unique(gene_ref)
### here i am trying to link the transcript to gene location.   This did not work well so now I am trying to use biomart to retrieve the start and end coordinates of the genes from the gene list, then use that as a .bed file for the closet function.

#  ensembl <- useEnsembl(biomart = "genes", dataset = "hsapiens_gene_ensembl")
# my_attributes <- c('chromosome_name', 'start_position', 'end_position','entrezgene_id', 'ensembl_gene_id', 'hgnc_symbol')
# # searchAttributes(mart = ensembl, pattern = "refseq")
# # searchDatasets(mart=ensembl,pattern="NCBI")
# gene_list <- getBM(attributes=my_attributes,
#       filters = "hgnc_symbol",
#         values = S13Table$SYMBOL,
#       mart = ensembl)
# write.csv(gene_ref,"data/gene_ref.csv")
# listFilters(ensembl)

####

gene_full_list <- gene_list %>% 
  mutate(chromosome_name=as.numeric(chromosome_name))%>% 
  # filter(!is.na(chromosome_name)) #%>% 
  na.omit(.)
  dplyr::select(chromosome_name,start_position,end_position,entrezgene_id,ensembl_gene_id)

### this conversion left me with 13114 genes. some Ids were lost in the conversion, so I will try to figure those locations out.  This led me to adjusting the biomart retrieval by using hgnc symbol, then removing nas from 15497 to 13426.  This was further cut to 12984 using na.omit which excluded those ncbi gene ids (entrezgeneids) that were empty.   I went with the na.omit option overall.
write_delim(gene_full_list,delim = "\t",file="data/testnames.txt")

assigned_closest_gene <- read_delim("data/n45_bedfiles/assigned_closest_gene.bed", 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)

expressed_NEARGENE_table <- assigned_closest_gene %>%
#  NOTE FOR MYSELF  (are we base 0 or base 1!!!!!)  
# The peak file names do not match the bed file names due to the 0/1 issue.  This means to match up MRC categories by name, I need to alter the first start position by +1! 
  mutate(X2 = X2+1) %>% 
  unite("peakid",X1:X3, sep = ".", remove= FALSE) %>% 
  rename("chr"=X1,
         "start"=X2,
         "end"= X3,
         "NG_chr"=X7,
         "NG_start"=X8 ,
         "NG_end"=X9 ,
         "ENTREZID"= X10,
         "ensembl_id"= X11,
         "SYMBOL"=X12,
         "dist_to_NG"=X13) %>%
  dplyr::select(chr, start,end, peakid,NG_chr:dist_to_NG) 

write_tsv(expressed_NEARGENE_table,"data/n45_bedfiles/exp_neargene_table.tsv")

##test code for collapsing the list later
  group_by(chr, start, end, peakid) %>%
    summarise(NCBI_gene = paste(unique(ENTREZID),collapse=","),
              ensembl_ID= paste(unique(ensembl_id),collapse = ","),
              SYMBOL= paste(unique(SYMBOL),collapse = ",") , 
              dist_to_NG =min(dist_to_NG));
  


```
### near-gene and MRC peaks





```{r near-gene MRC peaks distibution}
##Assume file not loaded
exp_neargene_table <- read_delim("data/n45_bedfiles/exp_neargene_table.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

col_ng_peak <- exp_neargene_table %>%
  group_by(chr, start, end, peakid) %>%
    summarise(NCBI_gene = paste(unique(ENTREZID),collapse=","),
              ensembl_ID= paste(unique(ensembl_id),collapse = ","),
              SYMBOL= paste(unique(SYMBOL),collapse = ",") , 
              dist_to_NG =min(dist_to_NG))


EAR_peak_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% EAR_df$id) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

ESR_peak_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% ESR_df$id) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

LR_peak_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% LR_df$id) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

NR_peak_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% NR_df$id) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

# length(unique(expressed_NEARGENE_table$ENTREZID))

```


#### VENN of peaks

```{r venn of peaks}

ggVennDiagram::ggVennDiagram(list(EAR_peak_list$ENTREZID,ESR_peak_list$ENTREZID,LR_peak_list$ENTREZID, NR_peak_list$ENTREZID),
              category.names = c("EAR","ESR","LR","NR"), label = "count")+
  labs(title = "Distribution of NEAR genes by motif response regions")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))



ggVennDiagram::ggVennDiagram(list(EAR_peak_list$ENTREZID,EAR_RNA$ENTREZID),
              category.names = c("EAR_ATAC","EAR_RNA"), label = "count")+
  labs(title = "EAR MRC-neargenes and expressed-RNA_MRC")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

ggVennDiagram::ggVennDiagram(list(ESR_peak_list$ENTREZID,ESR_RNA$ENTREZID),
              category.names = c("ESR_ATAC","ESR_RNA"), label = "count")+
  labs(title = "ESR MRC-neargenes and expressed-RNA_MRC")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

ggVennDiagram::ggVennDiagram(list(LR_peak_list$ENTREZID,LR_RNA$ENTREZID),
              category.names = c("LR_ATAC","LR_RNA"), label = "count")+
  labs(title = "LR MRC-neargenes and expressed-RNA_MRC")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

ggVennDiagram::ggVennDiagram(list(NR_peak_list$ENTREZID,NR_RNA$ENTREZID),
              category.names = c("NR_ATAC","NR_RNA"), label = "count")+
  labs(title = "NR MRC-neargenes and expressed-RNA_MRC")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))




```
### ZNF384 ChIP overlaps
```{r ZNF384}
### files loaded on line ~ 151


length(findOverlaps(EAR_df_gr, ZNF384_chip))
length(findOverlaps(ESR_df_gr,  ZNF384_chip))
length(findOverlaps(LR_df_gr,  ZNF384_chip))
length(findOverlaps(NR_df_gr,  ZNF384_chip))

ZNF_chip_data <- data.frame("CHIP_peak_num"=c(length(findOverlaps(EAR_df_gr, ZNF384_chip)), 
                            length(findOverlaps(ESR_df_gr,  ZNF384_chip)),
                            length(findOverlaps(LR_df_gr,  ZNF384_chip)),
                            length(findOverlaps(NR_df_gr,  ZNF384_chip))),
                            "MRC"= c("EAR","ESR","LR","NR"), 
                            "tot_MRC" = c(length(EAR_df$seqnames), 
                                          length(ESR_df$seqnames), 
                                          length(LR_df$seqnames), 
                                          length(NR_df$seqnames))) %>% 
  mutate(non_chip = tot_MRC-CHIP_peak_num) %>% 
  mutate(MRC= factor(MRC, levels = c("NR","EAR","ESR","LR")))

ZNF_chip_data %>% 
  pivot_longer(cols= c(CHIP_peak_num, non_chip), values_to = "counts", names_to = "type") %>% 
   mutate(percent=counts/tot_MRC*100) %>% 
  ggplot(., aes(x=MRC,y= percent, fill= type)) +
  geom_col()+
   geom_text(aes(label =sprintf("%.1f %%",percent)), position = "stack",vjust= 1
             )+
  ggtitle("MRC peaks found in ZNF384 ChIP data")+

  theme_classic()
```
Because of the occurrence of ZNF384 enrichment in accessibility of ZNF384 motifs, I used a CHIP pull down of ZNF384 to understand  how many of these peaks may be occupied by the TF in my MRCs.

### SNPs by category

#### Heart Failure

```{r HF SNP by category}
## Heart failure is gwas #5

EAR_HF_SNP <- as.data.frame(join_overlap_intersect(EAR_df_gr, gwas_5_gr_noext))
ESR_HF_SNP <- as.data.frame(join_overlap_intersect(ESR_df_gr, gwas_5_gr_noext))
LR_HF_SNP <- as.data.frame(join_overlap_intersect(LR_df_gr,  gwas_5_gr_noext))
NR_HF_SNP <- as.data.frame(join_overlap_intersect(NR_df_gr, gwas_5_gr_noext))

HF_SNP_table <- EAR_HF_SNP %>% 
  mutate(MRC="EAR") %>% 
  bind_rows((ESR_HF_SNP %>% 
              mutate(MRC="ESR"))) %>% 
  bind_rows((LR_HF_SNP %>% 
              mutate(MRC="LR"))) %>% 
  bind_rows((NR_HF_SNP %>% 
              mutate(MRC="NR")))  
HF_SNP_table %>%  
 rename( id="peak_id") %>% 
  select(peak_id, annotation,SNPS,CHR_POS,MRC) %>% 
  left_join(., exp_neargene_table, by=c("peak_id"="peakid")) %>% 
  kable(., caption= "SNPs for Heart Failure found in MRC peaks") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
  
## arrythmia is gwas #2



```


#### Arrhythmia

```{r ARR SNP by category}
## Heart failure is gwas #5

EAR_ARR_SNP <- as.data.frame(join_overlap_intersect(EAR_df_gr, gwas_2_gr_noext))
ESR_ARR_SNP <- as.data.frame(join_overlap_intersect(ESR_df_gr, gwas_2_gr_noext))
LR_ARR_SNP <- as.data.frame(join_overlap_intersect(LR_df_gr,  gwas_2_gr_noext))
NR_ARR_SNP <- as.data.frame(join_overlap_intersect(NR_df_gr, gwas_2_gr_noext))

ARR_SNP_table <- EAR_ARR_SNP %>% 
  mutate(MRC="EAR") %>% 
  bind_rows((ESR_ARR_SNP %>% 
              mutate(MRC="ESR"))) %>% 
  bind_rows((LR_ARR_SNP %>% 
              mutate(MRC="LR"))) %>% 
  bind_rows((NR_ARR_SNP %>% 
              mutate(MRC="NR")))  
ARR_SNP_table %>%  
 rename( id="peak_id") %>% 
  select(peak_id, annotation,SNPS,CHR_POS,MRC) %>% 
  left_join(., exp_neargene_table, by=c("peak_id"="peakid")) %>% 
  kable(., caption= "SNPs for Arrhythmia found in MRC peaks") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
  
## arrythmia is gwas #2



```



#### Myocardial Infarction

```{r MI SNP by category}
## Heart failure is gwas #5

EAR_MI_SNP <- as.data.frame(join_overlap_intersect(EAR_df_gr,MI_gwas_gr_noext))
ESR_MI_SNP <- as.data.frame(join_overlap_intersect(ESR_df_gr,MI_gwas_gr_noext))
LR_MI_SNP <- as.data.frame(join_overlap_intersect(LR_df_gr, MI_gwas_gr_noext))
NR_MI_SNP <- as.data.frame(join_overlap_intersect(NR_df_gr, MI_gwas_gr_noext))

MI_SNP_table <- EAR_MI_SNP %>% 
  mutate(MRC="EAR") %>% 
  bind_rows((ESR_MI_SNP %>% 
              mutate(MRC="ESR"))) %>% 
  bind_rows((LR_MI_SNP %>% 
              mutate(MRC="LR"))) %>% 
  bind_rows((NR_MI_SNP %>% 
              mutate(MRC="NR")))  
MI_SNP_table %>%  
 rename( id="peak_id") %>% 
  select(peak_id, annotation,SNPS,CHR_POS,MRC) %>% 
  left_join(., exp_neargene_table, by=c("peak_id"="peakid")) %>% 
  kable(., caption= "SNPs for MI found in MRC peaks") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
  
```


#### Coronary Artery Disease

```{r MI SNP by category}
## Heart failure is gwas #5

EAR_CAD_SNP <- as.data.frame(join_overlap_intersect(EAR_df_gr,CAD_gwas_gr_noext))
ESR_CAD_SNP <- as.data.frame(join_overlap_intersect(ESR_df_gr,CAD_gwas_gr_noext))
LR_CAD_SNP <- as.data.frame(join_overlap_intersect(LR_df_gr, CAD_gwas_gr_noext))
NR_CAD_SNP <- as.data.frame(join_overlap_intersect(NR_df_gr, CAD_gwas_gr_noext))

CAD_SNP_table <- EAR_CAD_SNP %>% 
  mutate(MRC="EAR") %>% 
  bind_rows((ESR_CAD_SNP %>% 
              mutate(MRC="ESR"))) %>% 
  bind_rows((LR_CAD_SNP %>% 
              mutate(MRC="LR"))) %>% 
  bind_rows((NR_CAD_SNP %>% 
              mutate(MRC="NR")))  
CAD_SNP_table %>%  
 rename( id="peak_id") %>% 
  select(peak_id, annotation,SNPS,CHR_POS,MRC) %>% 
  left_join(., exp_neargene_table, by=c("peak_id"="peakid")) %>% 
  kable(., caption= "SNPs for CAD found in MRC peaks") %>% 
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
  
## arrythmia is gwas #2



```
