---
title: "ATAC_fastqc"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
```
## Total Sequences before mapping
This code takes the multiqc fastqc output file and:
splits by rows to trimmed and non trimmed, then separates the trimmed file names into categories I want, then adds back in the non trimmed data rows (while also splitting file name like the trimmed file name).
after rbind, I split treatmenttime by position, fix the names of the time column, remove numbers from trt column, add a new column called "trimmed" where I add in a vector that lets me group by trimmed file verses non trimmed file, the select only those columns containing the columns I want to keep. 
```{r data loading}
multiqc_fastqc2 <- read_csv("data/multiqc_fastqc_run2.txt")
multiqc_general_stats2 <- read_csv("data/multiqc_genestat_run2.txt")


fastqc_full <- multiqc_fastqc2 %>% 
  slice_tail(n=144) %>% 
  separate(Filename, into = c(NA,"ind","treatmenttime",NA,"read")) %>% 
  rbind(., (multiqc_fastqc2 %>% slice_head(n=144) %>% separate(Filename, into = c("ind","treatmenttime",NA,"read")))) %>% 
  separate_wider_position(., col =treatmenttime,c(2,trt=2,time=3),too_few = "align_start") %>% 
  mutate(time=case_match(trt,"E2"~"24h","E3"~"3h","M2"~"24h", "M3"~"3h","T2"~"24h","T3"~"3h","V2"~"24h","V3"~"3h",.default = time)) %>% 
  mutate(trt=gsub("[[:digit:]]", "", trt) ) %>% 
  mutate(trimmed = if_else(grepl(pattern ="^trim", x = Sample)==TRUE, "yes","no")) %>% 
  dplyr::select(Sample:read, trimmed,`Total Sequences`:avg_sequence_length) %>% 
  full_join(., multiqc_general_stats2, join_by(Sample)) %>% 
   dplyr::rename("percent_gc"="FastQC_mqc-generalstats-fastqc-percent_gc",
         "avg_seq_len"= "FastQC_mqc-generalstats-fastqc-avg_sequence_length",
         "percent_dup"= "FastQC_mqc-generalstats-fastqc-percent_duplicates",
         "percent_fails"= "FastQC_mqc-generalstats-fastqc-percent_fails",
         "total_sequences"= "FastQC_mqc-generalstats-fastqc-total_sequences") %>% 
  mutate(ind = factor(ind, levels = c("Ind1", "Ind2", "Ind3", "Ind4", "Ind5", "Ind6"))) %>%
  mutate(time = factor(time, levels = c("3h", "24h"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels = c("DX","E", "DA","M", "T", "V"), labels = c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH"))) 

```
(in this case, Sample, ind, trt, time read, trimmed, Total sequences, Flagged poor  quality, sequence length, %GC,total deduplicated %,and avg sequence length)
I also then addin the gen_stats file and rename the columns to normal things.
```{r ggplot Total sequences}
# fastqc_full
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
drug_pal_fac <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
fastqc_full %>% 
  filter(trimmed=="no") %>%
  ggplot(., aes(x=trt, y= `Total Sequences`))+
  geom_col(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("Total Sequences, untrimmed")+
      # ylab(ylab)+
      xlab("")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))

fastqc_full %>% 
  filter(trimmed=="yes") %>% 
  ggplot(., aes(x=trt, y= `Total Sequences`))+
  geom_col(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("Total Sequences, trimmed")+
      # ylab(ylab)+
      xlab("")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))

totseq <- fastqc_full %>% 
  dplyr::filter(read =='R1') %>% 
  # group_by(ind,trt,time) %>% 
dplyr::select(Sample, ind, trt, time, trimmed, `Total Sequences`) %>% 
  pivot_wider(id_cols = c(ind,trt,time), names_from = trimmed, values_from = `Total Sequences`) %>% 
  mutate(perc_removed=(no-yes)/no*100) #%>% 
 # kable(list(totseq[1:36,], totseq[37:72,]),caption= "Summary of Total sequences before and after trimming, with percentage of removed sequences") %>%
 #  kable_paper("striped", full_width = FALSE) %>%
 #  kable_styling(full_width = FALSE,font_size = 18) #%>%
 #   # scroll_box(width = "100%", height = "400px")
  
  totseq %>% 
    ggplot(.,aes(x=trt,y=perc_removed) )+
    geom_col(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("Total Sequences, percent removed")+
      # ylab(ylab)+
      xlab("")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))
```
##Average sequence length  


```{r avg seq length}
fastqc_full %>% 
  
  filter(trimmed=="no") %>%
  mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %>% 
  ggplot(., aes(x=read, y= avg_seq_len))+
  geom_boxplot(aes(fill= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  # ylim(0,55)+
  ggtitle("Average sequence length, untrimmed")+
      # ylab(ylab)+
      # xlab("")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))

fastqc_full %>% 
  filter(trimmed=="yes") %>% 
   mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %>% 
  ggplot(., aes(x=read, y= avg_seq_len))+
  geom_boxplot(aes(col= trt))+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  scale_color_manual(values = drug_pal)+
  theme_bw()+
  ggtitle("Average sequence length, trimmed")+
      # ylab(ylab)+
      # xlab("")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))
  


```


```{r percentdup, fig.width=12}
fastqc_full %>% 
  
  filter(trimmed=="no") %>%
  group_by(trt) %>% 
  # mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %>% 
  ggplot(., aes(x=read, y= percent_dup))+
  geom_col(position= "dodge",aes(fill= trt))+
  geom_text(aes(group=trt,label = sprintf("%.1f",percent_dup)),
            position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  # ylim(0,55)+
  ggtitle("Percent duplicated, untrimmed")+
      # ylab(ylab)+
      # xlab("")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))

fastqc_full %>% 
  filter(trimmed=="yes") %>% 
   group_by(trt) %>% 
  # mutate(avg_sequence_length=as.numeric(avg_sequence_length)) %>% 
  ggplot(., aes(x=read, y= percent_dup))+
  geom_col(position= "dodge",aes(fill= trt))+
  geom_text(aes(group=trt,label = sprintf("%.1f",percent_dup)),
            position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(ind~time)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))
  


```
## Trimmed sequence lengths

```{r trimmed graph}
###omitted the first288 lines (this was a duplicate of the first multiqc run... changed files to iterate over in the furture)


trimmed_seq_length <- read.csv("data/trimmed_seq_length.csv", row.names = 1, col.names = 0:25)
# save <-
trimmed_seq_length <- trimmed_seq_length %>%
  dplyr::rename( "0bp"=X0,"2bp"=X1,"4bp"=X2,"6bp"=X3,"8bp"=X4,"10bp"=X5,"12bp"=X6, "14bp"=X7, "16bp"=X8, "18bp"=X9, "20bp"=X10, "22bp"=X11, "24bp"=X12, "26bp"=X13, "28bp"=X14, "30bp"=X15, "32bp"=X16, "34bp"=X17, "36bp"=X18, "38bp"=X19, "40bp"=X20, "42bp"=X21, "44bp"=X22, "46bp"=X23, "48bp"=X24, "50bp"=X25)# %>%
  # column_to_rownames("samples") %>% write.csv(.,"data/trimmed_seq_length.csv")
#   t() %>% 
# as.numeric()
# trimmed_seq_length %>% 
#   rownames_to_column("samples") %>% 
#   pivot_longer(., col=!samples, names_to = "frag_length", values_to = "counts")
# 
# 
# 
# pivot_longer(everything(), names_to = "lelsngth", values_to ="counts")
# str(save)
```



```{r trimmed readpairs}
aln_results <- read.csv("data/aln_run1_results.txt", row.names = 1)
aln_results %>% 
  mutate(treatment=factor(treatment, levels = c("DOX","EPI","DNR","MTX", "TRZ", "VEH"))) %>%
  mutate(time=factor(time, levels = c("3","24"),labels=c("3 hours", "24 hours"))) %>% 
  mutate(indv=factor(indv, levels =c ("1","2","3","4","5","6"))) %>% 
  
  ggplot(., aes(x =time, y= reads), group= time)+
  geom_boxplot(position = "dodge",aes(fill=treatment))+
  geom_point(aes(col=indv))+
  facet_wrap(~treatment)+
  theme_bw()+
  ggtitle("number of read-pairs before mapping")+
        scale_fill_manual(values=drug_pal)   +
  scale_color_brewer(palette ="Dark2")+
  theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))

```

```{r number of alignedreads, eval=FALSE, include=FALSE}
aln_results %>% 
  mutate(treatment=factor(treatment, levels = c("DOX","EPI","DNR","MTX", "TRZ", "VEH"))) %>%
  mutate(time=factor(time, levels = c("3","24"),labels=c("3 hours", "24 hours"))) %>% 
  mutate(indv=factor(indv, levels =c ("1","2","3","4","5","6"))) %>% 
  
  ggplot(., aes(x =time, y= aln_overall), group= time)+
  geom_col(position= "dodge",aes(fill= treatment))+
  geom_text(aes(group=treatment,label = sprintf("%.1f",aln_overall*100)),
            position=position_dodge(width =.93),angle= 90,vjust=.1, hjust=.9 )+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("Percent mapped reads")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))


```

```{r number of aln is 1}
aln_results %>% 
  mutate(treatment=factor(treatment, levels = c("DOX","EPI","DNR","MTX", "TRZ", "VEH"))) %>%
  mutate(time=factor(time, levels = c("3","24"),labels=c("3 hours", "24 hours"))) %>% 
  mutate(indv=factor(indv, levels =c ("1","2","3","4","5","6"))) %>% 
  # mutate(perc_aln_1_con= aln_1_con/reads * 100) %>% 
  
  ggplot(., aes(x =time, y= aln_1_con), group= time)+
  geom_col(position= "dodge",aes(fill= treatment))+
  # geom_text(aes(group=treatment,label = sprintf("%.1f",(aln_overall*100)),
  #           position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 ))+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("number of concordantly aligned reads = 1")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))


```
```{r percent conc aln reads}
aln_results %>% 
  mutate(treatment=factor(treatment, levels = c("DOX","EPI","DNR","MTX", "TRZ", "VEH"))) %>%
  mutate(time=factor(time, levels = c("3","24"),labels=c("3 hours", "24 hours"))) %>% 
  mutate(indv=factor(indv, levels =c ("1","2","3","4","5","6"))) %>% 
  mutate(perc_aln_1_con= aln_1_con/reads * 100) %>% 
  
  ggplot(., aes(x =time, y= perc_aln_1_con), group= time)+
  geom_col(position= "dodge",aes(fill= treatment))+
  geom_text(aes(group=treatment,label = sprintf("%.1f",perc_aln_1_con)),
            position=position_dodge(width =.95),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("Percent of concordantly aligned reads = 1")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))


```


```{r perc multimaped reads}
aln_results %>% 
  mutate(treatment=factor(treatment, levels = c("DOX","EPI","DNR","MTX", "TRZ", "VEH"))) %>%
  mutate(time=factor(time, levels = c("3","24"),labels=c("3 hours", "24 hours"))) %>% 
  mutate(indv=factor(indv, levels =c ("1","2","3","4","5","6"))) %>% 
  mutate(perc_multimapped= aln_.1_con/reads * 100) %>% 
  
  ggplot(., aes(x =time, y= perc_multimapped), group= time)+
  geom_col(position= "dodge",aes(fill= treatment))+
  geom_text(aes(group=treatment,label = sprintf("%.1f",perc_multimapped)),
            position=position_dodge(width =.93),angle= 90,vjust=.02, hjust=.7 )+
  facet_wrap(~indv)+
  scale_fill_manual(values=drug_pal)+
  theme_bw()+
  ggtitle("Percent of aligned reads > 1")+
      theme(strip.background = element_rect(fill = "white",linetype=1, linewidth = 0.5),
          plot.title = element_text(size=14,hjust = 0.5,face="bold"),
          axis.title = element_text(size = 10, color = "black"),
          axis.ticks = element_line(linewidth = 0.5),
          axis.line = element_line(linewidth = 0.5),
          # axis.text.x = element_blank(),
          strip.text.x = element_text(margin = margin(2,0,2,0, "pt"),face = "bold"))



```


## After Mapping and filtering



-    Currently the steps were are as follows:

    -   Basic Fastqc followed by adapter trimming and Fastqc analysis on the leftover fragments.

    -   Trimmed reads were aligned to the hg38 human genome.

    -   Mitochondrial reads (chrM) were removed

    -   samtools was used to removed non-paired, discordantly paired, and multi-mapped reads from the .bam.

    -   Markduplicates function from Picard was used to mark optical and PCR duplicates, with samtools used to remove these reads using the flag -F 1024.
    
    

```{r reads input }

Ind1_summary  <- read.csv("data/Ind1_summary.txt", row.names = 1) %>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
Ind2_summary  <- read.csv("data/Ind2_summary.txt", row.names = 1)%>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
Ind3_summary  <- read.csv("data/Ind3_summary.txt", row.names = 1)%>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
Ind4_summary  <- read.csv("data/Ind4_summary.txt", row.names = 1)%>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
Ind5_summary  <- read.csv("data/Ind5_summary.txt", row.names = 1)%>% 
 dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
Ind6_summary  <- read.csv("data/Ind6_summary.txt", row.names = 1)%>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
```



### Individual 1 read summary

```{r frag length}

Ind1_reads_summary <- 
Ind1_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind1_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 1") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

Ind1_reads_summary %>% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
  dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
  mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  dplyr::filter (type != "total_reads") %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
   mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position="dodge",aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Ind 1")
```



### Individual 2 read summary
```{r Readssummary2}

Ind2_reads_summary <- 
Ind2_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
  
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind2_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 2") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")
Ind2_reads_summary %>% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
  dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
  mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  dplyr::filter (type != "total_reads") %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
   mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position="dodge",aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Ind 2")
```

### Individual 3 read summary
```{r Readssummary3}

Ind3_reads_summary <- 
Ind3_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind3_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 3") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
 
 Ind3_reads_summary %>% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
  dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
  mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  dplyr::filter (type != "total_reads") %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
   mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position="dodge",aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Ind 3")
```

### Individual 4 read summary
```{r Readssummary4}

Ind4_reads_summary <- 
Ind4_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind4_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 4") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
 Ind4_reads_summary %>% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
  dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
  mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  dplyr::filter (type != "total_reads") %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
   mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position="dodge",aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Ind 4")
 
```
_note: reprocessed reads for Individual 4 vary from previous output.  This is due a file truncation that was detected in the process.  When the files were reprocessed, the truncated bam file was fixed, resulting in a larger number of overall reads.  Yay, I figured it out!_

### Individual 5 read summary
```{r Readssummary5}

Ind5_reads_summary <- 
Ind5_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind5_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 5") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
 Ind5_reads_summary %>% 
   pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
  dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
  mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  dplyr::filter (type != "total_reads") %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
   mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position="dodge",aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Ind 5")
```

### Individual 6 read summary
```{r Readssummary6}

Ind6_reads_summary <- 
Ind6_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind6_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 6") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
 Ind6_reads_summary %>% 
  pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
  dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
  mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  dplyr::filter (type != "total_reads") %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
   mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  ggplot(., aes (x=type, y=reads, group = trt)) +
  geom_col(position="dodge",aes(fill=time))+
  facet_wrap(time~trt, ncol = 6)+
  theme(axis.text.x=element_text(angle=90))+
  ggtitle("Ind 6")
 
```


```{r fragment files list, eval=FALSE, include=FALSE}
### this was used to load fragment files for all individuals for display in the future
safename <- choose.files()
Ind1_75DA3h_firstfrag <- read_delim(safename[1], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)%>% separate(col=1, into=c("counts","frag_size")) %>% 
  rename(  "counts"="DA_3h_5_counts")


Ind1_75DA24h_firstfrag <- read_delim(safename[2], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DA_24h_5_counts")


Ind1_75DX3h_firstfrag <- read_delim(safename[3], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DX_3h_5_counts")


Ind1_75DX24h_firstfrag<- read_delim(safename[4], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DX_24h_5_counts")


Ind1_75E3h_firstfrag <- read_delim(safename[5], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="E_3h_5_counts")


Ind1_75E24h_firstfrag <- read_delim(safename[6], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="E_24h_5_counts")

Ind1_75M3h_firstfrag <- read_delim(safename[7], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="M_3h_5_counts")


Ind1_75M24h_firstfrag <- read_delim(safename[8], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="M_24h_5_counts")


Ind1_75T3h_firstfrag <- read_delim(safename[9], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="T_3h_5_counts")



Ind1_75T24h_firstfrag <- read_delim(safename[10], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="T_24h_5_counts")



Ind1_75V3h_firstfrag <- read_delim(safename[11], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="V_3h_5_counts")



Ind1_75V24h_firstfrag <- read_delim(safename[12], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="V_24h_5_counts")
Ind1_firstfrag_files <- Ind1_75V3h_firstfrag %>% 
  left_join(.,Ind1_75V24h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75DA3h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75DA24h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75DX3h_firstfrag, by = "frag_size") %>% 
left_join(.,Ind1_75DX24h_firstfrag, by = "frag_size") %>% 
left_join(.,Ind1_75E3h_firstfrag, by = "frag_size") %>% 
left_join(.,Ind1_75E24h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75M3h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75M24h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75T3h_firstfrag, by = "frag_size") %>% 
  left_join(.,Ind1_75T24h_firstfrag, by = "frag_size") %>% 
  pivot_longer(cols= !frag_size,names_to = "sample",values_to = "counts") %>% 
  separate(sample, into = c("trt","time","indv",NA), sep = "_") %>% 
  mutate(trt = factor(trt, levels = c("DX","E","DA","M","T","V"))) %>% 
  mutate(time = factor(time, levels = c("3h","24h"))) %>% 
  mutate(counts= as.numeric(counts), frag_size= as.numeric(frag_size))
write.csv(Ind1_firstfrag_files,"data/Ind1_firstfragment_files.txt")

```

### Total reads summary
```{r readsummary 1}
total_reads_summary <- Ind1_reads_summary %>% 
 mutate(sample=gsub("75","1_",sample)) %>% 
  rbind((Ind2_reads_summary %>%
           mutate(sample=gsub("87","2_",sample)))) %>% 
  rbind((Ind3_reads_summary %>%
           mutate(sample=gsub("77","3_",sample)))) %>% 
  rbind((Ind4_reads_summary %>%
           mutate(sample=gsub("79","4_",sample)))) %>% 
  rbind((Ind5_reads_summary %>%
           mutate(sample=gsub("78","5_",sample)))) %>% 
  rbind((Ind6_reads_summary %>%
           mutate(sample=gsub("71","6_",sample)))) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH")))
         
total_reads_summary %>% 
  ggplot(., aes(x=trt, y=total_reads ))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total number of reads by time and treatment")



total_reads_summary %>% 
  ggplot(., aes(x=trt, y=nuclear_reads ))+
  geom_boxplot(aes(fill=trt))+
 geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total number of nuclear reads by time and treatment")


total_reads_summary %>% 
  ggplot(., aes(x=trt, y=unique_mapped_reads ))+
  geom_boxplot(aes(fill=trt))+
 geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total number of unique-nuclear reads by time and treatment")



total_reads_summary %>% 
  ggplot(., aes(x=trt, y=dedup_reads ))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total unique, deduplicated nuclear mapped reads by time and treatment")

```

```{r total reads adding plots}
total_reads_summary %>% 
  pivot_longer(., cols=c(total_mapped_reads,nuclear_mapped_reads,unique_mapped_reads,dedup_reads), names_to="plotting_data", values_to = "counts") %>% 
  dplyr::filter(indv !="4") %>% 
  dplyr::filter(indv !="5") %>% 
  mutate(plotting_data=factor(plotting_data, levels =c("total_mapped_reads","nuclear_mapped_reads","unique_mapped_reads","dedup_reads")))  %>% 
  ggplot(., aes(x=plotting_data, y=counts))+
  geom_boxplot(aes(fill=indv))+
  theme_bw() +
 scale_fill_brewer(palette = "Dark2")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  
  facet_wrap(~time)




total_reads_summary %>% 
  pivot_longer(., cols=c(total_mapped_reads,nuclear_mapped_reads,unique_mapped_reads,dedup_reads), names_to="plotting_data", values_to = "counts") %>% 
  dplyr::filter(indv !="4") %>% 
  dplyr::filter(indv !="5") %>% 
  mutate(plotting_data=factor(plotting_data, levels =c("total_mapped_reads","nuclear_mapped_reads","unique_mapped_reads","dedup_reads")))  %>% 
  ggplot(., aes(x=trt, y=dedup_read_pairs))+
  geom_boxplot(aes(fill=trt))+
 geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("Deduplicated read pairs")+
  theme(axis.text.x=element_text(vjust = .2,angle=90))+
  
  theme_bw()
  


total_reads_summary %>% 
    dplyr::filter(indv !="4") %>% 
  dplyr::filter(indv !="5") %>% 
 summary()
```


