---
title: "Figure 6"
author: "Renee Matthews"
date: "2025-02-25"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
	dev = c("png","pdf")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```


### Figure 6: 
```{r}
knitr::include_graphics("assets/Figure\ 6.png", error=FALSE)
knitr::include_graphics("docs/assets/Figure\ 6.png",error = FALSE)

```

```{r  package loading}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(ggfortify)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(edgeR)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(readxl)
library(devtools)
library(vargen)

```


```{r loading data}

Collapsed_new_peaks <- read_delim("data/Final_four_data/collapsed_new_peaks.txt", delim = "\t", col_names = TRUE)
Collapsed_new_peaks_gr <- Collapsed_new_peaks %>% dplyr::select(chr:Peakid) %>%  dplyr::filter(chr!="chrY") %>%
  GRanges() %>% 
  keepStandardChromosomes(pruning.mode = "coarse")  


RNA_median_3_lfc <- readRDS("data/other_papers/RNA_median_3_lfc.RDS")
RNA_median_24_lfc <- readRDS("data/other_papers/RNA_median_24_lfc.RDS")

RNA_LFC <- RNA_median_3_lfc %>% 
  left_join(RNA_median_24_lfc, by = c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL"))


ATAC_24_lfc <- read_csv("data/Final_four_data/median_24_lfc.csv") 
ATAC_3_lfc <- read_csv("data/Final_four_data/median_3_lfc.csv")

ATAC_LFC <- Collapsed_new_peaks %>%
                 dplyr::select(Peakid) %>% 
  left_join(.,(ATAC_3_lfc %>% dplyr::select(peak, med_3h_lfc)), by=c("Peakid"="peak")) %>% 
  left_join(.,(ATAC_24_lfc %>% dplyr::select(peak, med_24h_lfc)), by=c("Peakid"="peak"))

Schneider_all_SNPS <- read_delim("data/other_papers/Schneider_all_SNPS.txt", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

Schneider_all_SNPS_df <- Schneider_all_SNPS %>% 
  dplyr::rename("RSID"="#Uploaded_variation") %>% 
  dplyr::select(RSID,Location,SYMBOL,Gene, SOURCE) %>%
  distinct(RSID,Location,SYMBOL,.keep_all = TRUE) %>% 
  separate_wider_delim(Location,delim=":",names=c("Chr","Coords")) %>% 
  separate_wider_delim(Coords,delim= "-", names= c("Start","End")) %>% 
  mutate(Chr=paste0("chr",Chr)) %>% 
  group_by(RSID) %>% 
  summarize(Chr=unique(Chr),
            Start=unique(Start),
            End=unique(End),
            SYMBOL=paste(unique(SYMBOL),collapse=";"),
            Gene=paste(Gene,collapse=";"),
            SOURCE=paste(SOURCE,collapse=";")
            ) %>% 
  GRanges() %>% keepStandardChromosomes(pruning.mode = "coarse")  %>% as.data.frame 

schneider_gr <-Schneider_all_SNPS_df%>%
  dplyr::select(seqnames,start,end,RSID:SOURCE) %>%
  distinct() %>% 
  GRanges()

schneider_10k_gr <- Schneider_all_SNPS_df%>%
  dplyr::select(seqnames,start,end,RSID:SOURCE) %>%
  mutate(start=(start-5000),end=(end+4999), width=10000) %>%
  distinct() %>% 
  GRanges()
 
schneider_20k_gr <- Schneider_all_SNPS_df%>%
  dplyr::select(seqnames,start,end,RSID:SOURCE) %>%
  mutate(start=(start-10000),end=(end+9999), width=20000) %>%
  distinct() %>% 
  GRanges()

schneider_50k_gr <- Schneider_all_SNPS_df%>%
  dplyr::select(seqnames,start,end,RSID:SOURCE) %>%
  mutate(start=(start-20000),end=(end+24999), width=50000) %>%
  distinct() %>% 
  GRanges()
 

SNP_peak_check <- join_overlap_intersect(Collapsed_new_peaks_gr,schneider_gr) %>%
  as.data.frame()
 
SNP_peak_check_10k <- join_overlap_intersect(Collapsed_new_peaks_gr,schneider_10k_gr) %>%
  as.data.frame()

SNP_peak_check_20k <- join_overlap_intersect(Collapsed_new_peaks_gr,schneider_20k_gr) %>% 
  as.data.frame()
SNP_peak_check_50k <- join_overlap_intersect(Collapsed_new_peaks_gr,schneider_50k_gr) %>% 
  as.data.frame()

point_only <- SNP_peak_check
SNP_10k_only <- SNP_peak_check_10k
SNP_20k_only <- SNP_peak_check_20k
SNP_50k_only <- SNP_peak_check_50k

ATAC_peaks_gr <- Collapsed_new_peaks %>% GRanges()

Peaks_cutoff <- read_delim("data/Final_four_data/LCPM_matrix_ff.txt",delim = "/") %>%
  dplyr::select(Peakid)
  
schneider_short_list <- point_only %>% as.data.frame %>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)
schneider_10k_list <- SNP_10k_only %>% distinct(RSID,Peakid)%>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)
schneider_20k_list <- SNP_20k_only %>% distinct(RSID,Peakid)%>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)
schneider_50k_list <- SNP_50k_only %>% distinct(RSID,Peakid)%>% dplyr::filter(Peakid %in%Peaks_cutoff$Peakid)

Nine_te_df <- readRDS("data/Final_four_data/Nine_group_TE_df.RDS")
###needed to change TE status to at least 1 bp overlap
match <- Nine_te_df %>% 
   mutate(TEstatus=if_else(!is.na(per_ol),"TE_peak","not_TE_peak")) %>% 
  distinct(Peakid,TEstatus,mrc,.keep_all = TRUE) 
```

#### Figure 6A:
```{r heatmap,fig.height=15, fig.width=8}
hm_lfc_df <- SNP_peak_check_20k %>% 
  left_join(., Nine_te_df, by = c("Peakid"="Peakid") ) %>% 
  dplyr::filter(mrc != "NR") %>% 
  dplyr::filter(mrc !="not_mrc") %>%  
  mutate(dist_to_SNP=case_when(
    Peakid %in% schneider_short_list$Peakid &RSID %in% schneider_short_list$RSID~ 0,
    Peakid %in% schneider_10k_list$Peakid &RSID %in% schneider_10k_list$RSID~ 10,
    Peakid %in% schneider_20k_list$Peakid &RSID %in% schneider_20k_list$RSID~ 20,
     Peakid %in% schneider_50k_list$Peakid &RSID %in% schneider_50k_list$RSID ~ 50)) %>% 
    left_join(ATAC_LFC) %>% 
  tidyr::unite(name,Peakid,RSID,SYMBOL, sep = "_", remove = FALSE) %>% 
  group_by(Peakid, RSID) %>% 
   summarize(name=unique(name),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           repClass=paste(unique(repClass),collapse=":"),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          SYMBOL=paste(unique(SYMBOL),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %>% 
  arrange(., Peakid)
  


hm_lfc_mat <-hm_lfc_df %>%
  dplyr::select(Peakid,RSID,  med_3h_lfc, med_24h_lfc) %>% 
  tidyr::unite(name, Peakid, RSID, sep = "_") %>% 
  column_to_rownames("name") %>% 
  as.matrix()

hm_name_mat <-hm_lfc_df %>% 
  dplyr::select (Peakid, RSID, TEstatus, mrc,dist_to_SNP) %>% 
   tidyr::unite(name, Peakid, RSID, sep = "_")  
  col_fun <- circlize::colorRamp2(c(-4, 0, 4), c("blue", "white", "red"))
row_anno_lfc <- ComplexHeatmap::rowAnnotation(
    TE_status=hm_name_mat$TEstatus,
    reheat_status=hm_name_mat$reheat,
    MRC=hm_name_mat$mrc,
    direct_overlap=hm_name_mat$dist_to_SNP,
    col= list(
    TE_status=c("TE_peak"="goldenrod",
                "not_TE_peak"="lightblue"),
    MRC = c("EAR_open" = "#F8766D",
            "EAR_close" = "#f6483c",
            "ESR_open" = "#7CAE00",
            "ESR_close" = "#587b00",
            "ESR_opcl"="grey40",
            "ESR_clop"="tan",
            "LR_open" = "#00BFC4",
            "LR_close" = "#008d91",
            "NR" = "#C77CFF",
            "not_mrc"="black"),   
    reheat_status=c("reheat_gene"="green",
                    "not_reheat_gene"="orange"),
    direct_overlap=c("0"="red",
                     "10"="pink",
                     "20"="tan2",
                     "50"="grey8")))

simply_map_lfc <- ComplexHeatmap::Heatmap(hm_lfc_mat,
                                          col = col_fun,
                        left_annotation = row_anno_lfc,
                        show_row_names = TRUE,
                       row_names_max_width= max_text_width(rownames(hm_lfc_mat),                                                        gp=gpar(fontsize=16)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_lfc, 
     merge_legend = TRUE, 
     heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")
```

```{r Boram Park etal}
ParkSNPs <- readRDS("data/other_papers/ParkSNPs_pull_VEF.RDS")

ParkSNP_table <-
  ParkSNPs %>% 
  dplyr::select(1:2) %>% 
    distinct() %>% 
    separate_wider_delim(.,Location,delim=":",names=c("chr","position"), cols_remove=FALSE) %>% 
    separate_wider_delim(.,position,delim="-",names=c("begin","term")) %>%
    mutate(chr=paste0("chr",chr)) 
ParkSNP_gr <- ParkSNP_table %>% 
  mutate("start" = begin, "end"=term) %>% 
    GRanges()
ParkSNP_10k_gr <- ParkSNP_table %>% 
  mutate(begin=as.numeric(begin),term=as.numeric(term)) %>% 
  mutate(start=begin-5000, end=term+5000) %>% 
  GRanges()

ParkSNP_20k_gr <- ParkSNP_table %>% 
  mutate(begin=as.numeric(begin),term=as.numeric(term)) %>% 
  mutate(start=begin-10000, end=term+10000) %>% 
  GRanges()

ParkSNP_gr_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_gr) %>%
  as.data.frame()

ParkSNP_gr_10k_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_10k_gr) %>%
  as.data.frame()

ParkSNP_gr_20k_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_20k_gr) %>%
  as.data.frame()

```

```{r masterplot Park}

ParkSNP_gr_check <- join_overlap_intersect(Collapsed_new_peaks_gr,ParkSNP_gr) %>%
  as.data.frame()
Park_df <-ParkSNP_gr_20k_check%>% 
  as.data.frame() %>%
  dplyr::select(Peakid, X.Uploaded_variation) %>% 
  dplyr::rename("SNPS"=X.Uploaded_variation) %>% 
  left_join(., Nine_te_df, by=("Peakid"="Peakid")) %>%
  dplyr::select(Peakid, SNPS,mrc,TEstatus) %>% 
  # left_join(., (Collapsed_new_peaks %>% 
  #             dplyr::select(Peakid,NCBI_gene,SYMBOL)), by=c("Peakid"="Peakid"))
  left_join(., (ATAC_3_lfc %>%
          dplyr::select(peak,med_3h_lfc)),by=c("Peakid"="peak")) %>% 
  left_join(., (ATAC_24_lfc %>%
              dplyr::select(peak,med_24h_lfc)),by=c("Peakid"="peak"))%>% 
    dplyr::filter(mrc !="NR") %>%
  dplyr::filter(mrc !="not_mrc") %>% 
 mutate(dist_to_SNP=case_when(
    Peakid %in% ParkSNP_gr_check$Peakid &SNPS %in% ParkSNP_gr_check$X.Uploaded_variation~ 0,
    Peakid %in% ParkSNP_gr_10k_check$Peakid &SNPS %in% ParkSNP_gr_10k_check$X.Uploaded_variation~ 10,
    Peakid %in% ParkSNP_gr_20k_check$Peakid &SNPS %in% ParkSNP_gr_20k_check$X.Uploaded_variation~ 20)) %>% 
    tidyr::unite(name,Peakid,SNPS, sep = "_", remove = FALSE) %>% 
  group_by(Peakid) %>% 
   summarize(name=unique(name),
           med_3h_lfc=unique(med_3h_lfc),
           med_24h_lfc=unique(med_24h_lfc),
           TEstatus=paste(unique(TEstatus),collapse=";"),
          mrc=unique(mrc),
          dist_to_SNP=min(dist_to_SNP))  %>% 
  arrange(., Peakid)
 
new_park_mat <- Park_df%>% 
  ungroup() %>% 
  dplyr::select(name,med_3h_lfc, med_24h_lfc) %>% 
  column_to_rownames("name") %>% 
  as.matrix()
new_park_name_mat <- Park_df %>% 
  ungroup() %>% 
  dplyr::select(name,TEstatus,mrc,dist_to_SNP)

row_anno_park <-
  rowAnnotation(
    TE_status=new_park_name_mat$TEstatus,
    # gwas_status=new_park_name_mat$GWAS,
    MRC=new_park_name_mat$mrc,
    direct_overlap=new_park_name_mat$dist_to_SNP,
    col= list(TE_status=c("TE_peak"="goldenrod",
                          "not_TE_peak"="lightblue"), 
              MRC = c("EAR_open" = "#F8766D",
                      "EAR_close" = "#f6483c",
                      "ESR_open" = "#7CAE00",
                      "ESR_close" = "#587b00",
                      "ESR_opcl"="grey40", 
                      "ESR_clop"="tan",
                      "LR_open" = "#00BFC4",
                      "LR_close" = "#008d91",
                      "NR" = "#C77CFF",
                      "not_mrc"="black"),
              # gwas_status=c("AF"="green",
              #               "HF"="orange", 
              #               "AF;HF"="purple3"),
              direct_overlap=c("0"="red",
                               "10"="pink",
                               "20"="tan2",
                               "50"="grey8")))
col_fun <- circlize::colorRamp2(c(-4, 0, 4), c("blue", "white", "red"))
simply_map_park <- ComplexHeatmap::Heatmap(new_park_mat,
                                           col=col_fun,
                        left_annotation = row_anno_park,
                        row_names_max_width = max_text_width(rownames(new_park_mat),  
                                                             gp=gpar(fontsize=16)),
                        heatmap_legend_param = list(direction = "horizontal"),
                        show_column_names = TRUE,
                        cluster_rows = FALSE,
                        cluster_columns = FALSE)

draw(simply_map_park, merge_legend = TRUE, heatmap_legend_side = "bottom", 
    annotation_legend_side = "bottom")


```


#### Figure 6B: Chromatin accessibility within 5 kb of rs10753081
```{r}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
# K27_counts <-  readRDS("data/Final_four_data/All_Raodahpeaks.RDS")
ATAC_counts <- readRDS("data/Final_four_data/ATAC_filtered_raw_counts_allsamples.RDS") %>% 
  cpm(., log = TRUE) %>% 
   as.data.frame() %>%
  rename_with(.,~gsub(pattern = "Ind1_75", replacement = "D_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind2_87", replacement = "A_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind3_77", replacement = "B_",.)) %>%
  rename_with(.,~gsub(pattern = "Ind6_71", replacement = "C_",.)) %>%
  rename_with(.,~gsub( "DX" ,'DOX',.)) %>%
  rename_with(.,~gsub( "DA" ,'DNR',.)) %>%
  rename_with(.,~gsub( "E" ,'EPI',.)) %>%
  rename_with(.,~gsub( "T" ,'TRZ',.)) %>%
  rename_with(.,~gsub( "M" ,'MTX',.)) %>%
  rename_with(.,~gsub( "V" ,'VEH',.)) %>%
  rename_with(.,~gsub("24h","_24h",.)) %>%
  rename_with(.,~gsub("3h","_3h",.)) %>% 
  rownames_to_column("Peakid")




RNA_counts <- readRDS("data/other_papers/cpmcount.RDS") %>%
  dplyr::rename_with(.,~gsub(pattern="Da",replacement="DNR",.)) %>% 
 dplyr::rename_with(.,~gsub(pattern="Do",replacement="DOX",.)) %>% 
  dplyr::rename_with(.,~gsub(pattern="Ep",replacement="EPI",.)) %>% 
   dplyr::rename_with(.,~gsub(pattern="Mi",replacement="MTX",.)) %>% 
    dplyr::rename_with(.,~gsub(pattern="Tr",replacement="TRZ",.)) %>% 
       dplyr::rename_with(.,~gsub(pattern="Ve",replacement="VEH",.)) %>% 
  rownames_to_column("ENTREZID")

df_gene_schneider <- data.frame(SYMBOL=c("PRDX6","DARS2"))

df_gene_schneider <- df_gene_schneider %>% 
left_join(., (RNA_median_24_lfc %>% dplyr::select(ENTREZID,SYMBOL)), by = c ("SYMBOL"="SYMBOL")) %>% 
   left_join(., (hm_lfc_df %>% dplyr::select(RSID,Peakid,SYMBOL) %>% separate_longer_delim(SYMBOL,delim=";")),by = c("SYMBOL"="SYMBOL")) %>% distinct(SYMBOL,.keep_all = TRUE)


ATAC_counts %>% 
    dplyr::filter(Peakid %in% df_gene_schneider$Peakid) %>% 
  pivot_longer(cols = !Peakid, names_to = "sample", values_to = "counts") %>% 
  left_join(., df_gene_schneider, by =c("Peakid"="Peakid")) %>% 
  separate("sample", into = c("ind","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~Peakid+SYMBOL,scales="free_y")+
  ggtitle(" ATAC accessibility")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm ATAC")

```



#### Figure 6C: Gene expression of rs10753081 heart eGene
```{r}
RNA_counts %>% 
  dplyr::filter(ENTREZID %in% df_gene_schneider$ENTREZID) %>% 
  pivot_longer(cols = !ENTREZID, names_to = "sample", values_to = "counts") %>% 
  # mutate(ENTREZID=as.numeric(ENTREZID)) %>% 
  left_join(., df_gene_schneider, by =c("ENTREZID"="ENTREZID")) %>%  
  # left_join(., df_gene_schneider, by =c("ENTREZID"="ENTREZID.x")) %>% 
  separate("sample", into = c("trt","ind","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~SYMBOL, scales="free_y")+
    scale_fill_manual(values = drug_pal)+
  ggtitle("RNA Log 2 cpm of expressed gene")+
  theme_bw()+
  ylab("log2 cpm RNA")
```

#### Figure 6D: Chromatin accessibility at rs117299725

```{r RNA park, fig.height=4, fig.width=5}
Park_rsid <- data.frame("RSID"=c("rs117299725"),"Peakid"=c("chr9.76808694.76808955"))


ATAC_counts %>% 
  dplyr::filter(Peakid %in% Park_rsid$Peakid) %>% 
  # mutate(Peakid = row.names(.)) %>% 
  pivot_longer(cols = !Peakid, names_to = "sample", values_to = "counts") %>% 
  left_join(., Park_rsid, by =c("Peakid"="Peakid")) %>% 
  separate("sample", into = c("ind","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(~Peakid,scales="free_y")+
  ggtitle(" ATAC accessibility")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()+
  ylab("log2 cpm ATAC")



```
