---
title: "Figure 5"
author: "Renee Matthews"
date: "2025-02-25"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
		dev = c("png","pdf")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

Package Loading:
```{r loading packages}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
# library(gprofiler2)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
# library(edgeR)
library(ggfortify)
# library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(ggVennDiagram)
library(BiocParallel)
library(ggpubr)
library(biomaRt)
library(plyranges)
library(smplot2)

```

### Figure 5: Drug-induced chromatin accessibility changes correspond to changes in active histone modification and gene expression
```{r}
knitr::include_graphics("assets/Figure\ 5.png", error=FALSE)
knitr::include_graphics("docs/assets/Figure\ 5.png",error = FALSE)

```

##### Figure A: Associating CUT&Tag data with ATAC-sequencing data and RNA-expression data.

```{r Dataframes needed}
## ATAC-seq data
ATAC_peaks <- read_delim("data/Final_four_data/collapsed_new_peaks.txt", delim = "\t", col_names = TRUE)
filt_peaks <- readRDS("data/Final_four_data/x4_filtered.RDS") %>% 
  as.data.frame() %>% rownames_to_column("ATAC_peakid") %>% dplyr::select(ATAC_peakid)
ATAC_med_24_lfc <- read_csv("data/Final_four_data/median_24_lfc.csv") %>% dplyr::rename("ATAC_24h_lfc"=med_24h_lfc) 
ATAC_med_3_lfc <- read_csv("data/Final_four_data/median_3_lfc.csv") %>% dplyr::rename("ATAC_3h_lfc"=med_3h_lfc) 
ATAC_LFC <- ATAC_med_3_lfc %>% 
  dplyr::select(peak,ATAC_3h_lfc) %>% 
  left_join(.,(ATAC_med_24_lfc %>% dplyr::select(peak, ATAC_24h_lfc)),by=c("peak"="peak"))

## H3K27ac-seq data
H3K27ac_peaks <-  readRDS("data/Final_four_data/All_Raodahpeaks.RDS")
AC_median_3_lfc <- read_csv("data/Final_four_data/AC_median_3_lfc.csv")
AC_median_24_lfc <- read_csv("data/Final_four_data/AC_median_24_lfc.csv")

H3K27ac_LFC <- AC_median_3_lfc %>% 
  dplyr::select(Peakid,AC_3h_lfc) %>% 
  left_join(.,(AC_median_24_lfc %>% dplyr::select(Peakid, AC_24h_lfc)),by=c("Peakid"="Peakid"))

## RNA-expression data
RNA_median_3_lfc <- readRDS("data/other_papers/RNA_median_3_lfc.RDS")
RNA_median_24_lfc <- readRDS("data/other_papers/RNA_median_24_lfc.RDS")

RNA_LFC <- RNA_median_3_lfc %>% 
  left_join(RNA_median_24_lfc, by = c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL"))

```
##### Overlapping ATAC-seq and H3K27ac peaks then filtering by +/- 2kb of TSS:
```{r associating peaks}
ATAC_peaks_gr <- ATAC_peaks %>%
  dplyr::rename("ATAC_peakid"=Peakid) %>% 
  dplyr::filter(ATAC_peakid %in% filt_peaks$ATAC_peakid) %>% 
  dplyr::filter(chr!="chrY") %>%
  GRanges() %>% 
  keepStandardChromosomes(pruning.mode = "coarse") 

H3K27ac_peaks_gr <- H3K27ac_peaks %>%
  dplyr::rename("H3K27ac_peakid"=Geneid) %>% 
  dplyr::filter(Chr!="chrM") %>%
  GRanges() %>% 
  keepStandardChromosomes(pruning.mode = "coarse") 
overlapping_peaks <-join_overlap_intersect(ATAC_peaks_gr,H3K27ac_peaks_gr) %>% 
  as.data.frame() %>% 
  dplyr::filter(dist_to_NG > -2000 & dist_to_NG <2000) %>% 
  dplyr::select(ATAC_peakid,H3K27ac_peakid,NCBI_gene,SYMBOL)

```

##### Attaching ATAC-seq, H3K27ac median LFC, and RNA median LFC:
```{r median dataframes}

Corr_df <- overlapping_peaks %>% 
  left_join(., ATAC_LFC, by= c("ATAC_peakid"="peak")) %>% 
  left_join(., H3K27ac_LFC, by= c("H3K27ac_peakid"="Peakid")) %>% 
  left_join(., RNA_LFC, by=c("NCBI_gene"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  distinct(ATAC_peakid,.keep_all = TRUE)

```
##### B: Correlation between chromatin accessibility response and other molecular phenotypes
Active histon modification
```{r correlation plots}
Corr_df %>% 
  ggplot(.,aes(x=ATAC_3h_lfc, y=AC_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour ATAC-seq median LFC v 3 hour H3k27ac-seq median LFC")+
       xlab("ATAC-seq drug response LFC")+
       ylab( "H3K27ac CUT&Tag drug response LFC")

Corr_df %>% 
  ggplot(.,aes(x=ATAC_24h_lfc, y=AC_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour ATAC-seq median LFC v 24 hour H3k27ac-seq median LFC")+
       xlab("ATAC-seq drug response LFC")+
       ylab( "H3K27ac CUT&Tag drug response LFC")

```
Gene expression

```{r}
Corr_df %>% 
  ggplot(.,aes(x=ATAC_3h_lfc, y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour ATAC-seq median LFC v 3 hour RNA-seq median LFC")+
       xlab("ATAC-seq drug response LFC")+
       ylab( "RNA-seq drug response LFC")

Corr_df %>% 
  ggplot(.,aes(x=ATAC_24h_lfc, y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour ATAC-seq median LFC v 24 hour RNA-seq median LFC")+
       xlab("ATAC-seq drug response LFC")+
       ylab( "RNA-seq drug response LFC")
```

