---
title: "final_four_analysis"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
editor_options: 
  chunk_output_type: inline
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
	dev = c("pdf","png")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(eulerr)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
```

```{r echo=TRUE, file='code/corMotifcustom.R'}

```

```{r functions to have}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
pca_plot <-
  function(df,
           col_var = NULL,
           shape_var = NULL,
           title = "") {
    ggplot(df) + geom_point(aes_string(
      x = "PC1",
      y = "PC2",
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = "PC 1", y = "PC 2") +
      scale_color_manual(values = c(
        "#8B006D",
        "#DF707E",
        "#F1B72B",
        "#3386DD",
        "#707031",
        "#41B333"
      ))
  }
pca_var_plot <- function(pca) {
  # x: class == prcomp
  pca.var <- pca$sdev ^ 2
  pca.prop <- pca.var / sum(pca.var)
  var.plot <-
    qplot(PC, prop, data = data.frame(PC = 1:length(pca.prop),
                                      prop = pca.prop)) +
    labs(title = 'Variance contributed by each PC',
         x = 'PC', y = 'Proportion of variance')
  plot(var.plot)
}

calc_pca <- function(x) {
  # Performs principal components analysis with prcomp
  # x: a sample-by-gene numeric matrix
  prcomp(x, scale. = TRUE, retx = TRUE)
}

get_regr_pval <- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == "lm")
  fstat <- summary(mod)$fstatistic
  pval <- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc <- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char <- paste0("PC", pc_num)
  # Calculate F-statistic p-value for linear model
  pval <- get_regr_pval(lm(df[, pc_char] ~ df[, fac]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = "lm") + labs(title = sprintf("p-val: %.2f", pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf("p-val: %.2f", pval))
  }
}
x_axis_labels = function(labels, every_nth = 1, ...) {
  axis(side = 1,
       at = seq_along(labels),
       labels = F)
  text(
    x = (seq_along(labels))[seq_len(every_nth) == 1],
    y = par("usr")[3] - 0.075 * (par("usr")[4] - par("usr")[3]),
    labels = labels[seq_len(every_nth) == 1],
    xpd = TRUE,
    ...
  )
}

```

#### Counts summary

Adding analysis of first peak counts into one large file
```{r read counts, fig.width=10, fig.height=7}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
Ind1_summary  <- read.csv("data/Ind1_summary.txt", row.names = 1) %>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('75DX','DOX',trt),
         trt= gsub('75E','EPI', trt),
         trt=gsub('75DA','DNR',trt),
         trt=gsub('75M','MTX',trt),
         trt=gsub('75T','TRZ',trt),
         trt=gsub('75V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Ind2_summary  <- read.csv("data/Ind2_summary.txt", row.names = 1)%>% 
dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('87DX','DOX',trt),
         trt= gsub('87E','EPI', trt),
         trt=gsub('87DA','DNR',trt),
         trt=gsub('87M','MTX',trt),
         trt=gsub('87T','TRZ',trt),
         trt=gsub('87V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Ind3_summary  <- read.csv("data/Ind3_summary.txt", row.names = 1)%>% 
dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('77DX','DOX',trt),
         trt= gsub('77E','EPI', trt),
         trt=gsub('77DA','DNR',trt),
         trt=gsub('77M','MTX',trt),
         trt=gsub('77T','TRZ',trt),
         trt=gsub('77V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Ind6_summary  <- read.csv("data/Ind6_summary.txt", row.names = 1)%>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('71DX','DOX',trt),
         trt= gsub('71E','EPI', trt),
         trt=gsub('71DA','DNR',trt),
         trt=gsub('71M','MTX',trt),
         trt=gsub('71T','TRZ',trt),
         trt=gsub('71V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Final_four_counts <- Ind1_summary %>% rbind(Ind2_summary) %>% rbind(Ind3_summary) %>% rbind(Ind6_summary)

FF_bound <- Final_four_counts %>% 
  mutate(time=factor(time, levels =c("3h","24h"))) %>% 
  pivot_longer(cols = reads:mapped_reads, names_to="count_type", values_to = "read_num") %>% 
  mutate(read_type=case_when(read_type=="fin"~"unique",.default = read_type)) %>% 
  tidyr::unite("Total_reads",read_type:count_type,sep = "_", remove = TRUE) %>% 
  tidyr::unite("Total_reads_time",Total_reads:time,sep = "_", remove = FALSE) %>% 
  mutate(Total_reads=factor(Total_reads, levels = c("total_reads", "total_mapped_reads","nuclear_reads","nuclear_mapped_reads","unique_reads", "unique_mapped_reads", "nodup_reads", "nodup_mapped_reads"))) %>% 
  dplyr::filter(Total_reads != "nuclear_reads") %>% 
dplyr::filter(Total_reads != "nodup_reads") %>% 
dplyr::filter(Total_reads != "unique_reads") %>% 
  dplyr::mutate(trt=factor(trt, levels = c("DOX", "EPI","DNR", "MTX","TRZ","VEH"))) #%>% 
FF_bound %>% 
  ggplot(., aes(x=Total_reads, y=read_num, group=interaction(indv,time)))+
  geom_col(aes(fill=trt), position="dodge")+
  theme_minimal()+
 scale_fill_manual(values=drug_pal)+
  facet_wrap(~time+trt+indv,nrow = 6, ncol = 12 )+
  theme(strip.text = element_text(face = "bold",  hjust = 0, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1),
        panel.spacing = unit(1, 'points'),
        axis.text.x=element_text(angle = 90, vjust = 0.5, hjust=1))

FF_bound %>% 
 ggplot(., aes(x=Total_reads, y=read_num, col=indv, group=interaction(time,Total_reads_time)))+
  geom_boxplot(aes(fill=trt)) + 
  geom_point(aes(col=indv))+
   theme_bw()+
  facet_wrap(~trt+time,nrow = 3, ncol = 6 )+
 scale_fill_manual(values=drug_pal)+
  theme(strip.text = element_text(face = "bold",  hjust = 0, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1),
        panel.spacing = unit(1, 'points'),
        axis.text.x=element_text(angle = 90, vjust = 0.5, hjust=1))
```


```{r read counts small}
Final_four_counts %>% 
  mutate(time=factor(time, levels =c("3h","24h"))) %>% 
pivot_longer(cols = reads:mapped_reads, names_to="count_type", values_to = "read_num") %>% 
  dplyr::filter(read_type=="nodup"& count_type=="mapped_reads") %>% 
  mutate(trt= factor(trt, levels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(x=trt,y=read_num,group=(interaction(time,trt))))+
  geom_boxplot(aes(fill=trt))+
   geom_point(aes(col=indv, size =3))+
  # geom_hline(yintercept = 20)+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("Reads across treatment and time")+
  theme_bw()+
  theme(strip.text = element_text(face = "bold",  hjust = .5, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1),
        panel.spacing = unit(1, 'points'))


```
### cardiomyocyte purity
```{r purity table}

Purity_table <- cbind("indv"= c("75-1","87-1","77-1","71-1"),
"assign"=c("D","A","B","C"),
"num"= c(1,2,3,6),
"cm_purity"=c(98.5,98.3,99.5,86.75))
Purity_table %>% 
  kable(.,caption = "cTNT2 percent positive table") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
  

```

```{r  TSS enrichment}
allTSSE <- readRDS( "data/all_TSSE_scores.RDS")
allTSSE %>% as.data.frame() %>% 
  rownames_to_column("sample") %>% 
  separate(sample, into = c("indv","trt","time"), sep= "_") %>%
  mutate(trt= factor(trt, levels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  mutate(time = factor(time, levels = c("3","24"),labels = c("3 hours","24 hours"))) %>% 
  dplyr::filter(indv !=4 &indv !=5) %>% 
  ggplot(., aes(x= time, y= V1, group = indv))+

   # geom_col(position= "dodge",aes(fill=trt)) %>%
  geom_jitter(aes(col = trt, size = 1.5, alpha = 0.5) ,  position=position_jitter(0.25))+
  geom_hline(yintercept=5, linetype = 3)+
    geom_hline(yintercept=7, col = "blue")+
  facet_wrap(~indv)+
   theme_bw()+
  ggtitle("TSS enrichment scores")+
   scale_color_manual(values=drug_pal)+
   theme(strip.text = element_text(face = "bold",  hjust = .5, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1))

```

```{r TSS peaks plot, fig.width=8, fig.height=12}
Ind1_TSS_peaks_plot <- readRDS("data/Ind1_TSS_peaks.RDS")
Ind2_TSS_peaks_plot <- readRDS("data/Ind2_TSS_peaks.RDS")
Ind3_TSS_peaks_plot <- readRDS("data/Ind3_TSS_peaks.RDS")
Ind6_TSS_peaks_plot <- readRDS("data/Ind6_TSS_peaks.RDS")

a1<- plotAvgProf(Ind1_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 1" )+coord_cartesian(xlim=c(-2000,2000))
b1 <- plotAvgProf(Ind2_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 2" )+coord_cartesian(xlim=c(-2000,2000))
c1 <- plotAvgProf(Ind3_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 3" )+coord_cartesian(xlim=c(-2000,2000))
d1 <- plotAvgProf(Ind6_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 6" )+coord_cartesian(xlim=c(-2000,2000))

a2 <- plotAvgProf(Ind1_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 1" )+coord_cartesian(xlim=c(-2000,2000))
b2 <- plotAvgProf(Ind2_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 2" )+coord_cartesian(xlim=c(-2000,2000))
c2 <- plotAvgProf(Ind3_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 3" )+coord_cartesian(xlim=c(-2000,2000))
d2 <- plotAvgProf(Ind6_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 6" )+coord_cartesian(xlim=c(-2000,2000))

plot_grid(a1,a2, b1,b2,c1,c2,d1,d2, axis="l",align = "hv",nrow=4, ncol=2)
# a2+coord_cartesian(xlim=c(-2000,2000))
```



Analysis using the four selected individuals and the peaks generated from just that set:

```{r loading peak file}
# Peaks_bed_file <- rtracklayer::BEDFile("~/ATAC_downloads/finalfour_bl_filt_peaks.bed")
# import(Peaks_bed_file) %>% as.data.frame()
# write_delim(Snyder_41peaks, "data/other_papers/ENCFF966JZT_bed_Snyder_41peaks.bed",delim = "\t")
  all_four_filt_counts <- read.delim("data/all_four_filtered_counts.txt",header=TRUE) %>% 
  GRanges() %>% 
  keepStandardChromosomes(pruning.mode = "coarse") %>% 
  as.data.frame()

 
```

```{r edit_names of counts file}
# all_four_filt_counts %>% 
names(all_four_filt_counts) = gsub(pattern = "_S.*", replacement = "", x = names(all_four_filt_counts))

names(all_four_filt_counts) = gsub(pattern = "ind1.trimmed.filt_files.trimmed_|ind2.trimmed.filt_files.trimmed_|ind3.trimmed.filt_files.trimmed_|ind6.trimmed.filt_files.trimmed_", replacement = "", x = names(all_four_filt_counts))


names(all_four_filt_counts) = gsub(pattern = "ind1.trimmed.filt_files.trimmed_|ind2.trimmed.filt_files.trimmed_|ind3.trimmed.filt_files.trimmed_|ind6.trimmed.filt_files.trimmed_", replacement = "", x = names(all_four_filt_counts))

```

summary of peaks by sample

```{r peak count}
##check commandline file for how I pulled the wc -l on all files.
# peakcount_new4 <- read_delim("~/ATAC_downloads/master_peaks/4_indonly/peakcount_new4.txt", 
# +     delim = "\t", escape_double = FALSE, 
# # +     col_names = FALSE, trim_ws = TRUE)
# peakcount_new4 %>% 
#   mutate(X1=gsub("75","1_", X1), 
#          X1=gsub("87","2_",X1),
#          X1=gsub("77","3_",X1),
#          X1=gsub("71","6_",X1),
#          X1=gsub("DA","DNR_",X1),
#          X1=gsub("DX","DOX_",X1),
#          X1=gsub("E","EPI_",X1),
#          X1=gsub("T","TRZ_",X1),
#          X1=gsub("M","MTX_",X1),
#          X1=gsub("V","VEH_",X1)) %>% 
#   separate(X1, into=c("indv","trt","time"), sep= "_", remove = TRUE) %>% 
#   dplyr::rename(peak_number=X2) %>% write_delim(., file="data/Final_four_data/Peak_count_ff.txt", delim = "\t")

peakcount_ff <- read_delim("data/Final_four_data/Peak_count_ff.txt",delim= "\t")
frip_newpeaks <- c(38.8,36.3,46.0,38.9,49.6,40.0,39.2,30.2,52.1,39.8,51.1,28.0,
                   42.3,40.3,39.7,38.7,37.9,36.6,36.0,48.7,50.4,44.2,52.0,31.9,
                   40.5,34.1,41.2,33.7,43.5,28.6,34.7,42.8,38.1,40.3,44.6,26.4,
                   46.5,23.9,46.9,25.8,46.7,23.8,21.8,39.2,33.2,22.8,36.8,34.8)
peakcount_ff$frip_newpeaks <- frip_newpeaks

peakcount_ff %>% 
  mutate(time = factor(time, levels = c("3h", "24h"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels = c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH"))) %>% 
  mutate(indv=factor(indv)) %>% 
   ggplot(., aes(x=trt,y=peak_number))+
  geom_boxplot(aes(fill=trt))+
   geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
     scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("Peak number across treatment and time")+
  theme_bw()+
  theme(strip.text = element_text(face = "bold",  hjust = .5, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1),
        panel.spacing = unit(1, 'points'))

peakcount_ff %>% 
  mutate(time = factor(time, levels = c("3h", "24h"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels = c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH"))) %>% 
  mutate(indv=factor(indv)) %>% 
   ggplot(., aes(x=trt,y=frip_newpeaks))+
  geom_boxplot(aes(fill=trt))+
   geom_point(aes(col=indv, size =3))+
  geom_hline(aes(yintercept = 20), linetype=2, color="red")+
  facet_wrap(time~.)+
     scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("Fraction of reads in peaks, percentage")+
  theme_bw()+
  theme(strip.text = element_text(face = "bold",  hjust = .5, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1),
        panel.spacing = unit(1, 'points'))+
  coord_cartesian(ylim = c(0,100))


peakcount_ff %>% 
  mutate(time = factor(time, levels = c("3h", "24h"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels = c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH"))) %>% 
  mutate(indv=factor(indv)) %>% 
  group_by(time) %>% 
  summarize(medal=median(peak_number))
```


```{r prcomp for 4}

PCA4mat <- all_four_filt_counts%>% 
  dplyr::select(Geneid,Ind1_75DA24h:Ind6_71V3h) %>% 
  column_to_rownames("Geneid") %>% 
  as.matrix()

annotation_mat <- data.frame(timeset=colnames(PCA4mat)) %>% 
  mutate(sample = timeset) %>% 
  mutate(timeset=gsub("Ind1_75","1_",timeset)) %>% 
  mutate(timeset=gsub("Ind2_87","2_",timeset)) %>% 
  mutate(timeset=gsub("Ind3_77","3_",timeset)) %>% 
  mutate(timeset=gsub("Ind6_71","4_",timeset)) %>% 
  mutate(timeset = gsub("24h","_24h",timeset), 
       timeset = gsub("3h","_3h",timeset)) %>%
  separate(timeset, into = c("indv","trt","time"), sep= "_") %>% 
  mutate(trt= case_match(trt, 'DX' ~'DOX', 'E'~'EPI', 'DA'~'DNR', 'M'~'MTX', 'T'~'TRZ', 'V'~'VEH',.default = trt)) %>% 
  # mutate(indv = factor(indv, levels = c("1", "2", "3", "4", "5", "6"))) %>%
  mutate(time = factor(time, levels = c("3h", "24h"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels = c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH"))) 

PCA4_info <- (prcomp(t(PCA4mat), scale. = TRUE)) 
PCA4_info_anno <- PCA4_info$x %>% cbind(.,annotation_mat)
# autoplot(PCA_info)
summary(PCA4_info)
# %>% as.tibble() %>% 
#   kable(., caption = "Summary of PCA variables") %>% 
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "500px")
# cpm(PCAmat, log=TRUE)
pca_plot(PCA4_info_anno, col_var='trt', shape_var = 'time')+ggtitle("PCA of raw counts by time and treatment")
        
pca_plot(PCA4_info_anno, col_var='trt', shape_var = 'indv')+ggtitle("PCA of raw counts by treatment and individual")

```

```{r log2 cpm corr, fig.height=8}

FCmatrix_full <-   PCA4mat %>%
  as.data.frame() %>% 
  rename_with(.,~gsub(pattern = "Ind1_75", replacement = "1_",.)) %>% 
  rename_with(.,~gsub(pattern = "Ind2_87", replacement = "2_",.)) %>% 
  rename_with(.,~gsub(pattern = "Ind3_77", replacement = "3_",.)) %>% 
  rename_with(.,~gsub(pattern = "Ind6_71", replacement = "6_",.)) %>% 
  rename_with(.,~gsub( "DX" ,'DOX',.)) %>% 
  rename_with(.,~gsub( "DA" ,'DNR',.)) %>% 
  rename_with(.,~gsub( "E" ,'EPI',.)) %>% 
  rename_with(.,~gsub( "T" ,'TRZ',.)) %>%
  rename_with(.,~gsub( "M" ,'MTX',.)) %>% 
  rename_with(.,~gsub( "V" ,'VEH',.)) %>% 
  rename_with(.,~gsub("24h","_24h",.)) %>% 
  rename_with(.,~gsub("3h","_3h",.)) %>% 
  as.matrix() %>% 
  cpm(., log = TRUE) %>% 
  cor()

filmat_groupmat_col <- data.frame(timeset = colnames(FCmatrix_full))

counts_corr_mat <-filmat_groupmat_col %>%
  # mutate(sample = timeset) %>% 
  separate(timeset, into = c("indv","trt","time"), sep= "_") %>% 
  mutate(class = if_else(trt == "DNR", "AC", 
                         if_else(trt == "DOX", "AC", 
                                 if_else(trt == "EPI", "AC", "nAC")))) %>%
  mutate(TOP2i = if_else(trt == "DNR", "yes", 
                         if_else(trt == "DOX", "yes", 
                                 if_else(trt == "EPI", "yes", 
                                         if_else(trt == "MTX", "yes", "no"))))) 

                         
 mat_colors <- list( 
   trt= c("#F1B72B","#8B006D","#DF707E","#3386DD","#707031","#41B333"),
   indv=c("#1B9E77", "#D95F02" ,"#7570B3", "#E6AB02"),
   time=c("pink", "chocolate4"),
   class=c("yellow1","darkorange1"), 
   TOP2i =c("darkgreen","lightgreen"))                        
                         
names(mat_colors$trt)   <- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) <- unique(counts_corr_mat$indv)
names(mat_colors$time) <- unique(counts_corr_mat$time)
names(mat_colors$class) <- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) <- unique(counts_corr_mat$TOP2i)

# mat_col_ff <-
#   data.frame(
#     time = c(rep("3 hours", 5), rep("24 hours", 5)),
#     class = (c(
#       "AC", "AC", "AC", "nAC","nAC",  "AC", "AC", "AC", "nAC","nAC" 
#     )))
# rownames(mat_col_ff) <- colnames(FCmatrix_ff)
# 
# mat_colors_ff <-
#   list(
#     time = c("pink", "chocolate4"),
    # class = c("yellow1", "lightgreen"))

# names(mat_colors_ff$time) <- unique(mat_col_ff$time)
# names(mat_colors_ff$class) <- unique(mat_col_ff$class)
# names(mat_colors_FC$TOP2i) <- unique(mat_col_FC$TOP2i)
# corrFC_ff <- cor(FCmatrix_ff)

htanno_full <-  ComplexHeatmap::HeatmapAnnotation(df = counts_corr_mat, col = mat_colors)
Heatmap(FCmatrix_full, top_annotation = htanno_full)




```

### PCA analysis 155560 peaks
```{r filter log2cpm}
# saveRDS(x4_filtered,"data/Final_four_data/ATAC_filtered_raw_counts_allsamples.RDS")
l4cpm <- cpm(PCA4mat, log=TRUE)  ### for determining the basic cutoffs
dim(l4cpm)

row_means <- rowMeans(l4cpm)
x4_filtered <- PCA4mat[row_means > 0,]
dim(x4_filtered)
# saveRDS(x4_filtered,"data/Final_four_data/x4_filtered.RDS")
filt4_matrix_lcpm <- cpm(x4_filtered, log=TRUE)

PCA4_info_filter <- (prcomp(t(filt4_matrix_lcpm), scale. = TRUE))
summary(PCA4_info_filter)

pca_var_plot(PCA4_info_filter)

pca_final_four <- calc_pca(t(filt4_matrix_lcpm))
pca_final_four_anno <- data.frame(annotation_mat, pca_final_four$x)


pca_final_four_anno %>%
  ggplot(.,aes(x = PC1, y = PC2, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression("PCA of log"[2]*"(cpm) new filtered peak set"))+
  theme_bw()+
  guides(col="none", size =4)+
  labs(y = "PC 2 (13.5%)", x ="PC 1 (22.3%)")+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = "black"))

pca_final_four_anno %>%
  ggplot(.,aes(x = PC3, y = PC4, col=trt, shape=time, group=indv))+
  geom_point(size= 5)+
  scale_color_manual(values=drug_pal)+
   ggrepel::geom_text_repel(aes(label = indv))+
   ggtitle(expression("PCA of log"[2]*"(cpm) new filtered peak set"))+
  theme_bw()+
  guides(col="none", size =4)+
  labs(y = "PC 4 (5.5%)", x ="PC 3 (8.2%)")+
  theme(plot.title=element_text(size= 14,hjust = 0.5),
        axis.title = element_text(size = 12, color = "black"))

```

We go from `r dim(l4cpm)` peaks to`r dim(x4_filtered)` peaks using rowMeans (log2cpm) \> 0 across all samples. This is my new high confidence peak set.

#### Contribution of variance in PCA plot

pending code
```{r contribution of variability on PCA}
facs <- c("indv", "trt", "time")
names(facs) <- c("Individual", "Treatment", "Time")
drug1 <- c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH")##for changing shapes and colors
time <- rep(c("24h", "3h"),24) %>% factor(., levels = c("3h","24h"))
##gglistmaking
for (f in facs) {
  # PC1 v PC2
  pca_plot(pca_final_four_anno, col_var = f, shape_var = time,
           title = names(facs)[which(facs == f)])
  print(last_plot())
  
  # Plot f versus PC1 and PC2
  f_v_pc1 <- arrangeGrob(plot_versus_pc(pca_final_four_anno, 1, f))
  f_v_pc2 <- arrangeGrob(plot_versus_pc(pca_final_four_anno, 2, f))
  grid.arrange(f_v_pc1, f_v_pc2, ncol = 2, top = names(facs)[which(facs == f)])
  # summary(plot_versus_pc(PCA_info_anno_all, 1, f))
  # summary(plot_versus_pc(PCA_info_anno_all, 2, f))
 
}
```

### Comparing the peak sets, old and new

```{r compare old and new sets}
# write_delim((filt4_matrix_lcpm %>% as.data.frame(.) %>% rownames_to_column("Peakid")), "data/Final_four_data/LCPM_matrix_ff.txt", delim = "/t")
# filt4_matrix_lcpm <- read_delim("data/Final_four_data/LCPM_matrix_ff.txt",delim = "/t")
new_peaks_list <- row.names(filt4_matrix_lcpm) %>% as.data.frame() %>% separate_wider_delim(., cols =.,names=c("chr","start","end"), delim = ".", cols_remove = FALSE) %>%  dplyr::rename("peakid"=4)
new_peaks_gr <- new_peaks_list %>%
  dplyr::filter(chr!="chrY") %>% 
  GRanges() %>% 
   keepStandardChromosomes(., pruning.mode = "coarse")
stdChr <- standardChromosomes(new_peaks_gr)
peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
old_peaks_gr <- peakAnnoList_n45_motif$background_gr %>% as.GRanges()
old_peaks_gr$old_width <- width(old_peaks_gr)
a <- join_overlap_intersect(old_peaks_gr,new_peaks_gr,suffix = c("old", "new")) %>% 
  as.data.frame()

a %>% 
  dplyr::select(id,peakid,width,old_width) %>% 
  mutate(check =if_else(id ==peakid, "TRUE","FALSE")) %>% 
  dplyr::filter(check=="FALSE") %>% 
  mutate(width_ol=if_else(width<old_width,"smaller","larger")) %>%
  dplyr::rename("old_peak_id"="id","new_peak_id"="peakid") %>% 
  mutate(diff= old_width - width) %>% 
  kable(., caption = "summary of newpeaks that are smaller than old peaks, but are basically the same") %>% 
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

b <- filter_by_overlaps(old_peaks_gr,new_peaks_gr)
b.a <- filter_by_overlaps(new_peaks_gr, old_peaks_gr) 
b.b <- filter_by_non_overlaps(new_peaks_gr, old_peaks_gr)
b.c <- filter_by_non_overlaps(old_peaks_gr, new_peaks_gr)

new_peaks_gr %>% as.data.frame %>% summarize(mean =mean(width))
```


number of peaks in old data set: `r length(old_peaks_gr)`\
number of peaks in new data set: `r length(new_peaks_gr)`\
number of overlaps in between both sets: `r length (a$id)`\
number of peaks in new set that do not overlap old set: `r length(b.b)`\
number of peaks in old set that do not overlap new set: `r length(b.c)` \



### Overlapping peaks with ATAC-seq heart, left ventricle data, Snyder ENCFF966JZT.
```{r anno overlap with heart data}
Snyder_41peaks <- read.delim("data/other_papers/ENCFF966JZT_bed_Snyder_41peaks.bed",header=TRUE) %>% 
  GRanges()

heart_overlap <- join_overlap_intersect(new_peaks_gr,Snyder_41peaks)
ol_count <- heart_overlap %>% 
  as.data.frame %>% 
  distinct(peakid) %>% tally


```
number of peaks in Snyder heart ATAC set: `r length(Snyder_41peaks)`\
number of peaks in my data set: `r length(new_peaks_gr)`\
number of unique peaks that overlap the Snyder data: `r ol_count$n`  
percent of my peaks overlapping Snyder Heart left ventricle data: `r sprintf("%.2f", (ol_count$n/length(new_peaks_gr))*100)` %  



### Differential analysis

```{r DAR of new}

group <- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group <- factor(group, levels =c("1","2","3","4","5","6","7","8","9","10","11","12"))
short_names <- paste0(pca_final_four_anno$indv,"_",pca_final_four_anno$trt,"_",pca_final_four_anno$time)

dge <- DGEList.data.frame(counts = x4_filtered, group = group, genes = row.names(x4_filtered))

group_1 <- c(rep(c("DNR_24","DNR_3","DOX_24","DOX_3","EPI_24","EPI_3","MTX_24","MTX_3","TRZ_24","TRZ_3","VEH_24", "VEH_3"),4))

dge$group$indv <- pca_final_four_anno$indv
dge$group$time <- pca_final_four_anno$time
dge$group$trt <- pca_final_four_anno$trt

indv <- pca_final_four_anno$indv
efit4 <- readRDS("data/Final_four_data/efit4_filt_bl.RDS")
# mm <- model.matrix(~0 +group_1)  
# colnames(mm) <-  c("DNR_24", "DNR_3", "DOX_24","DOX_3","EPI_24", "EPI_3","MTX_24", "MTX_3", "TRZ_24","TRZ_3","VEH_24", "VEH_3")
# 
# y <- voom(dge$counts, mm,plot =TRUE)
# # 
# corfit <- duplicateCorrelation(y, mm, block = indv)
# # 
# v <- voom(dge$counts, mm, block = indv, correlation = corfit$consensus)
# # 
# fit <- lmFit(v, mm, block = indv, correlation = corfit$consensus)
# colnames(mm) <- c("DNR_24","DNR_3","DOX_24","DOX_3","EPI_24","EPI_3","MTX_24","MTX_3","TRZ_24","TRZ_3","VEH_24", "VEH_3")
#
#
# cm <- makeContrasts(
#   DNR_3.VEH_3 = DNR_3-VEH_3,
#   DOX_3.VEH_3 = DOX_3-VEH_3,
#   EPI_3.VEH_3 = EPI_3-VEH_3,
#   MTX_3.VEH_3 = MTX_3-VEH_3,
#   TRZ_3.VEH_3 = TRZ_3-VEH_3,
#   DNR_24.VEH_24 =DNR_24-VEH_24,
#   DOX_24.VEH_24= DOX_24-VEH_24,
#   EPI_24.VEH_24= EPI_24-VEH_24,
#   MTX_24.VEH_24= MTX_24-VEH_24,
#   TRZ_24.VEH_24= TRZ_24-VEH_24,
#   levels = mm)

# vfit <- lmFit(y, mm)

# vfit<- contrasts.fit(vfit, contrasts=cm)
# 
# efit4 <- eBayes(vfit)
# saveRDS(efit4,"data/Final_four_data/efit4_filt_bl.RDS")
results = decideTests(efit4)
summary(results)

```

```{r making the DAR dataframe}

# V.DNR_3.top_ff= topTable(efit4, coef=1, adjust.method="BH", number=Inf, sort.by="p") %>% rownames_to_column("peak")
# V.DOX_3.top_ff= topTable(efit4, coef=2, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.EPI_3.top_ff= topTable(efit4, coef=3, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.MTX_3.top_ff= topTable(efit4, coef=4, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.TRZ_3.top_ff= topTable(efit4, coef=5, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.DNR_24.top_ff= topTable(efit4, coef=6, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.DOX_24.top_ff= topTable(efit4, coef=7, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.EPI_24.top_ff= topTable(efit4, coef=8, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.MTX_24.top_ff= topTable(efit4, coef=9, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# V.TRZ_24.top_ff= topTable(efit4, coef=10, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
# 
# toplist_full_ff  <- list(V.DNR_3.top_ff, V.DOX_3.top_ff,V.EPI_3.top_ff,V.MTX_3.top_ff,V.TRZ_3.top_ff,V.DNR_24.top_ff, V.DOX_24.top_ff,V.EPI_24.top_ff,V.MTX_24.top_ff,V.TRZ_24.top_ff)
# names(toplist_full_ff) <- c("DNR_3", "DOX_3","EPI_3","MTX_3","TRZ_3","DNR_24", "DOX_24","EPI_24","MTX_24","TRZ_24")
# toplist_ff <-map_df(toplist_full_ff, ~as.data.frame(.x), .id="trt_time")
# #
# toplist_ff <- toplist_ff %>%
#   separate(trt_time, into= c("trt","time"), sep = "_") %>%
#   mutate(trt=factor(trt, levels = c("DOX","EPI","DNR","MTX","TRZ"))) %>%
#   mutate(time = factor(time, levels = c("3", "24"), labels = c("3 hours", "24 hours")))
# saveRDS(toplist_ff,"data/Final_four_data/toplist_ff.RDS")
toplist_ff <- readRDS("data/Final_four_data/toplist_ff.RDS")
median_3_lfc <- read_csv("data/Final_four_data/median_3_lfc.csv")
median_24_lfc <- read_csv("data/Final_four_data/median_24_lfc.csv")
```

```{r making median dfs, echo=TRUE, eval=FALSE}
### making median lfc

median_df <- toplist_ff %>% 
  pivot_wider(., id_cols=c(time,peak), names_from = trt, values_from = logFC) %>% 
  rowwise() %>% 
  mutate(median_lfc= median(c_across(DNR:TRZ)))

median_3_lfc <-   median_df %>%
    dplyr::filter(time == "3 hours") %>% 
  ungroup() %>% 
  dplyr::select(time, peak,median_lfc) %>% 
  dplyr::rename("med_3h_lfc"=median_lfc)
  

median_24_lfc <- median_df %>%
    dplyr::filter(time == "24 hours") %>% 
  ungroup() %>% 
  dplyr::select(time, peak,median_lfc) %>% 
  dplyr::rename("med_24h_lfc"=median_lfc)
  
write_csv(median_3_lfc, "data/Final_four_data/median_3_lfc.csv")
write_csv(median_24_lfc, "data/Final_four_data/median_24_lfc.csv")
```


#### Correlation of LFC
```{r LFC correlation, fig.height=8}
FCmatrix_ff <- subset(efit4$coefficients)

colnames(FCmatrix_ff) <-
  c("DNR\n3h",
    "DOX\n3h",
    "EPI\n3h",
    "MTX\n3h",
    "TRZ\n3h",
    "DNR\n24h",
    "DOX\n24h",
    "EPI\n24h",
    "MTX\n24h",
    "TRZ\n24h"
     )


mat_col_ff <-
  data.frame(
    time = c(rep("3 hours", 5), rep("24 hours", 5)),
    class = (c(
      "AC", "AC", "AC", "nAC","nAC",  "AC", "AC", "AC", "nAC","nAC" 
    )))
rownames(mat_col_ff) <- colnames(FCmatrix_ff)

mat_colors_ff <-
  list(
    time = c("pink", "chocolate4"),
    class = c("yellow1", "lightgreen"))

names(mat_colors_ff$time) <- unique(mat_col_ff$time)
names(mat_colors_ff$class) <- unique(mat_col_ff$class)
# names(mat_colors_FC$TOP2i) <- unique(mat_col_FC$TOP2i)
corrFC_ff <- cor(FCmatrix_ff)

htanno_ff <-  HeatmapAnnotation(df = mat_col_ff, col = mat_colors_ff)
Heatmap(corrFC_ff, top_annotation = htanno_ff)
```
```{r Volcanoes}


V.DNR_3.top= topTable(efit4, coef=1, adjust.method="BH", number=Inf, sort.by="p")
V.DOX_3.top= topTable(efit4, coef=2, adjust.method="BH", number=Inf, sort.by="p")
V.EPI_3.top= topTable(efit4, coef=3, adjust.method="BH", number=Inf, sort.by="p")
V.MTX_3.top= topTable(efit4, coef=4, adjust.method="BH", number=Inf, sort.by="p")
V.TRZ_3.top= topTable(efit4, coef=5, adjust.method="BH", number=Inf, sort.by="p")
V.DNR_24.top= topTable(efit4, coef=6, adjust.method="BH", number=Inf, sort.by="p")
V.DOX_24.top= topTable(efit4, coef=7, adjust.method="BH", number=Inf, sort.by="p")
V.EPI_24.top= topTable(efit4, coef=8, adjust.method="BH", number=Inf, sort.by="p")
V.MTX_24.top= topTable(efit4, coef=9, adjust.method="BH", number=Inf, sort.by="p")
V.TRZ_24.top= topTable(efit4, coef=10, adjust.method="BH", number=Inf, sort.by="p")

# volcanoplot(efit2,coef = c(1,2,3,4,5,6),style = "p-value",
#                           # highlight = 8,
#                           # names = efit2$genes$SYMBOL,
#                           hl.col = "red",xlab = "Log2 Fold Change",
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = "test")
# 

plot_filenames <- c("V.DNR_3.top","V.DOX_3.top","V.EPI_3.top","V.MTX_3.top",
                    "V.TRZ_.top","V.DNR_24.top","V.DOX_24.top","V.EPI_24.top",
                    "V.MTX_24.top","V.TRZ_24.top")
plot_files <- c( V.DNR_3.top,V.DOX_3.top,V.EPI_3.top,V.MTX_3.top,
                    V.TRZ_3.top,V.DNR_24.top,V.DOX_24.top,V.EPI_24.top,
                    V.MTX_24.top,V.TRZ_24.top)

volcanosig <- function(df, psig.lvl) {
    df <- df %>% 
    mutate(threshold = ifelse(adj.P.Val > psig.lvl, "A", ifelse(adj.P.Val <= psig.lvl & logFC<=0,"B","C")))
      # ifelse(adj.P.Val <= psig.lvl & logFC >= 0,"B", "C")))
    ##This is where I could add labels, but I have taken out
    # df <- df %>% mutate(genelabels = "")
    # df$genelabels[1:topg] <- df$rownames[1:topg]
    
  ggplot(df, aes(x=logFC, y=-log10(P.Value))) + 
    geom_point(aes(color=threshold))+
    # geom_text_repel(aes(label = genelabels), segment.curvature = -1e-20,force = 1,size=2.5,
    # arrow = arrow(length = unit(0.015, "npc")), max.overlaps = Inf) +
    #geom_hline(yintercept = -log10(psig.lvl))+
    xlab(expression("Log"[2]*" FC"))+
    ylab(expression("-log"[10]*"P Value"))+
    scale_color_manual(values = c("black", "red","blue"))+
    theme_cowplot()+
    ylim(0,20)+
    xlim(-6,6)+
    theme(legend.position = "none",
              plot.title = element_text(size = rel(1.5), hjust = 0.5),
              axis.title = element_text(size = rel(0.8))) 
}
#v1<- volcanosig(V.DA24.top, 0.01,0)
v1 <- volcanosig(V.DNR_3.top, 0.01)+ ggtitle("DNR 3 hour")
v2 <- volcanosig(V.DNR_24.top, 0.01)+ ggtitle("DNR 24 hour")+ylab("")
v3 <- volcanosig(V.DOX_3.top, 0.01)+ ggtitle("DOX 3 hour")
v4 <- volcanosig(V.DOX_24.top, 0.01)+ ggtitle("DOX 24 hour")+ylab("")
v5 <- volcanosig(V.EPI_3.top, 0.01)+ ggtitle("EPI 3 hour")
v6 <- volcanosig(V.EPI_24.top, 0.01)+ ggtitle("EPI 24 hour")+ylab("")
v7 <- volcanosig(V.MTX_3.top, 0.01)+ ggtitle("MTX 3 hour")
v8 <- volcanosig(V.MTX_24.top, 0.01)+ ggtitle("MTX 24 hour")+ylab("")
v9 <- volcanosig(V.TRZ_3.top, 0.01)+ ggtitle("TRZ 3 hour")
v10 <- volcanosig(V.TRZ_24.top, 0.01)+ ggtitle("TRZ 24 hour")+ylab("")
# volcanoplot(efit2,coef = 10,style = "p-value",
#                           highlight = 8,
#                           names = efit2$genes$SYMBOL,
#                           hl.col = "red",xlab = "Log2 Fold Change",
#                           ylab = NULL,pch = 16,cex = 0.35,
#                           main = "Using Trastuzumab 24 hour data and volcanoplot function")
plot_grid(v1,v2,  rel_widths =c(.8,1))
plot_grid(v3,v4,  rel_widths =c(.8,1))
plot_grid(v5,v6,  rel_widths =c(.8,1))
plot_grid(v7,v8,  rel_widths =c(.8,1))
plot_grid(v9,v10,  rel_widths =c(.8,1))


```
### examples of top DAR peaks by treatment

```{r example 3hr accesibility}
DNR_3_top3_ff <- row.names(V.DNR_3.top[1:3,])

log_filt_ff <- filt4_matrix_lcpm %>% 
  as.data.frame() %>% 
  rename_with(.,~gsub(pattern = "Ind1_75", replacement = "1_",.)) %>% 
  rename_with(.,~gsub(pattern = "Ind2_87", replacement = "2_",.)) %>% 
  rename_with(.,~gsub(pattern = "Ind3_77", replacement = "3_",.)) %>% 
  rename_with(.,~gsub(pattern = "Ind6_71", replacement = "6_",.)) %>% 
  rename_with(.,~gsub( "DX" ,'DOX',.)) %>% 
  rename_with(.,~gsub( "DA" ,'DNR',.)) %>% 
  rename_with(.,~gsub( "E" ,'EPI',.)) %>% 
  rename_with(.,~gsub( "T" ,'TRZ',.)) %>%
  rename_with(.,~gsub( "M" ,'MTX',.)) %>% 
  rename_with(.,~gsub( "V" ,'VEH',.)) %>% 
  rename_with(.,~gsub("24h","_24h",.)) %>% 
  rename_with(.,~gsub("3h","_3h",.))
  



row.names(log_filt_ff) <- row.names(filt4_matrix_lcpm)

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DNR_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour DNR")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

DOX_3_top3_ff <- row.names(V.DOX_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DOX_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour DOX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

EPI_3_top3_ff <- row.names(V.EPI_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% EPI_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour EPI")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

MTX_3_top3_ff <- row.names(V.MTX_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% MTX_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour MTX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

TRZ_3_top3_ff <- row.names(V.TRZ_3.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% TRZ_3_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour TRZ")+
 scale_fill_manual(values = drug_pal)+
  theme_bw()
```




```{r example 24hr accesibility}
DNR_24_top3_ff <- row.names(V.DNR_24.top[1:3,])


# row.names(log_filt_23s) <- row.names(lcpm_h3_23s_filtered)
log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DNR_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour DNR")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

DOX_24_top3_ff <- row.names(V.DOX_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% DOX_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour DOX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

EPI_24_top3_ff <- row.names(V.EPI_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% EPI_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour EPI")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()

MTX_24_top3_ff <- row.names(V.MTX_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% MTX_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour MTX")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()


TRZ_24_top3_ff <- row.names(V.TRZ_24.top[1:3,])

log_filt_ff %>% 
  dplyr::filter(row.names(.) %in% TRZ_24_top3_ff) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour TRZ")+
  scale_fill_manual(values = drug_pal)+
  theme_bw()
```
#### Correlations of LFC by treatment
```{r LFC time and treamtment 20kb}
hour3_lfc <- toplist_ff %>% 
  dplyr::filter(time=="3 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols = c(time, peak), names_from = trt, values_from = logFC)
hour3_lfc %>% 
  ggplot(.,aes(x=DOX, y=DNR))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour logFC, no filtering peaks")
hour3_lfc %>% 
  ggplot(.,aes(x=DOX, y=EPI))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour logFC, no filtering peaks")
hour3_lfc %>% 
  ggplot(.,aes(x=DOX, y=MTX))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour logFC, no filtering peaks")

```

```{r}
hour24_lfc <- toplist_ff %>% 
  dplyr::filter(time=="24 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols = c(time, peak), names_from = trt, values_from = logFC)
hour24_lfc %>% 
  ggplot(.,aes(x=DOX, y=DNR))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour logFC, no filtering peaks")
hour24_lfc %>% 
  ggplot(.,aes(x=DOX, y=EPI))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour logFC, no filtering peaks")
hour24_lfc %>% 
  ggplot(.,aes(x=DOX, y=MTX))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour logFC, no filtering peaks")
```





### Cormotif

```{r FF cormotif}

                     
# indv_n45 <- df_names_n45$indv
# indv <- factor(indv, levels = c(1,2,3,4,5,6))
# time_n45 <- df_names_n45$time
# time <- factor(time, levels =c("3h","24"))
# trt_n45 <- df_names_n45$trt
# label_n45 <- paste0(indv_n45,"_",trt_n45,"_",time_n45)
group_fac_ff <- group
groupid_ff <- as.numeric(group_fac_ff)

compid_ff <- data.frame(c1= c(2,4,6,8,10,1,3,5,7,9), c2 = c( 12,12,12,12,12,11,11,11,11,11))

y_TMM_cpm_ff <- cpm(x4_filtered, log = TRUE)

colnames(y_TMM_cpm_ff) <- short_names
# y_TMM_cpm
set.seed(31415)
# cormotif_initial_ff <- cormotiffit(exprs = y_TMM_cpm_ff, groupid = groupid_ff, compid = compid_ff, K=1:8, max.iter = 500, runtype = "logCPM")
# 
# saveRDS(cormotif_initial_ff,"data/Final_four_data/cormotif_ff_4_run.RDS")

# cormotif_four_ff <- cormotiffit(exprs = y_TMM_cpm_ff, groupid = groupid_ff, compid = compid_ff, K=4, max.iter = 500, runtype = "logCPM")

# saveRDS(cormotif_four_ff,"data/Final_four_data/cormotif_only4_run.RDS")

cormotif_initial_ff <-  readRDS("data/Final_four_data/cormotif_ff_4_run.RDS")
gene_prob_tran_ff <- cormotif_initial_ff$bestmotif$p.post
rownames(gene_prob_tran_ff) <- rownames(x4_filtered)
motif_prob_ff <- cormotif_initial_ff$bestmotif$clustlike
rownames(motif_prob_ff) <- rownames(gene_prob_tran_ff)
# write.csv(motif_prob_n45_he,"data/cormotif_probability_45_list_he.csv")
# saveRDS(motif_prob_ff,"data/Final_four_data/motif_prob_ff.RDS")

```

```{r CorrPlots}
myColors <-  rev(c("#FFFFFF", "#E6E6E6" ,"#CCCCCC", "#B3B3B3", "#999999", "#808080", "#666666","#4C4C4C", "#333333", "#191919","#000000"))
Cormotif::plotIC(cormotif_initial_ff)
plot.new()
legend('center',fill=myColors, legend =rev(c("0", "0.1", "0.2", "0.3", "0.4",  "0.5", "0.6", "0.7", "0.8","0.9", "1")), box.col="white",title = "Probability\nlegend", horiz=FALSE,title.cex=.8)

Cormotif::plotMotif(cormotif_initial_ff)

```
Study breakdown: 1 = DNR_3, 2 = DOX_3, 3 = EPI_3, 4 = MTX_3, 5 = TRZ_3\
6 = DNR_24, 7 = DOX_24, 8 = EPI_24, 9 = MTX_24, 10 = TRZ_24

```{r making clusters}
motif_prob_ff <- readRDS("data/Final_four_data/motif_prob_ff.RDS")

# motif_prob_ff %>% 
#   as.data.frame() %>% 
#   dplyr::filter(V1<.5 & V2>.5 & V3 <.5& V4<0.5)

background_peaks <- motif_prob_ff %>% 
  as.data.frame() %>% 
  rownames_to_column("Peakid") %>% 
  dplyr::select(Peakid) %>% 
  separate(Peakid, into=c("chr","start","end"),remove = FALSE)
 
NR_ff <- motif_prob_ff %>%
  as.data.frame() %>% 
  dplyr::filter(V1>.5 & V2<.5 & V3 <.5& V4<0.5) %>% 
  rownames_to_column("Peakid") %>% 
  dplyr::select(Peakid) %>% 
  separate(Peakid, into=c("chr","start","end"),remove = FALSE)
 
LR_ff <- motif_prob_ff %>%
   as.data.frame() %>% 
  dplyr::filter(V1<.5 & V2>.5 & V3 <.5& V4<0.5) %>% 
  rownames_to_column("Peakid") %>% 
  dplyr::select(Peakid) %>% 
  separate(Peakid, into=c("chr","start","end"),remove = FALSE)
 
 
ESR_ff <- motif_prob_ff %>%
  as.data.frame() %>% 
  dplyr::filter(V1<.5 & V2<.5 & V3 >.5& V4<0.5) %>% 
  rownames_to_column("Peakid") %>% 
  dplyr::select(Peakid) %>% 
  separate(Peakid, into=c("chr","start","end"),remove = FALSE)
 

EAR_ff <- motif_prob_ff %>%
  as.data.frame() %>% 
  dplyr::filter(V1<.5 & V2<.5 & V3 <.5& V4>0.5) %>% 
  rownames_to_column("Peakid") %>% 
  dplyr::select(Peakid) %>% 
  separate(Peakid, into=c("chr","start","end"),remove = FALSE)
 
peak_rep_names <-c("NR"= "chr7.76515428.76516495",
"LR"= "chr14.69543633.69543934", "ESR"="chr10.72331769.72332348", "EAR"="chr16.66248203.66248529")

```

Number of Peaks by main grouping:  
-EAR group has `r unique(length(EAR_ff$Peakid))` peaks.  
-ESR group has `r unique(length(ESR_ff$Peakid))` peaks.  
-LR group has `r unique(length(LR_ff$Peakid))` peaks.  
-NR group has `r unique(length(NR_ff$Peakid))` peaks.  
--Total number of peaks is `r length(background_peaks$Peakid)`
--Total number of peaks in clusters is `r sum(length(EAR_ff$Peakid),length(ESR_ff$Peakid),length(LR_ff$Peakid),length(NR_ff$Peakid))`  
-percent of peaks classified into clusters: `r (length(EAR_ff$Peakid)+length(ESR_ff$Peakid)+length(LR_ff$Peakid)+length(NR_ff$Peakid)) / length(background_peaks$Peakid)*100`   
-number of peaks not classified: `r length(background_peaks$Peakid)-(length(EAR_ff$Peakid)+length(ESR_ff$Peakid)+length(LR_ff$Peakid)+length(NR_ff$Peakid))`  

### Peak distribution characteristics

```{r Motif characteristics}
txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene
EAR_ff_gr <- EAR_ff %>% GRanges()
ESR_ff_gr <- ESR_ff %>% GRanges()
LR_ff_gr <- LR_ff %>% GRanges()
NR_ff_gr <- NR_ff %>% GRanges()
background_gr <- background_peaks %>% GRanges()
# mylist <- list(EAR_ff_gr,ESR_ff_gr,LR_ff_gr,NR_ff_gr,background_gr)
# peakAnnoList<- lapply(mylist, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# names(peakAnnoList) <- c("EAR","ESR","LR","NR","background")
# saveRDS(peakAnnoList, "data/Final_four_data/peakAnnoList_ff_motif.RDS")

peakAnnoList_motif <- readRDS("data/Final_four_data/peakAnnoList_ff_motif.RDS")
plotAnnoBar(peakAnnoList_motif[c(4,1:3)])+
  ggtitle ("Genomic Feature Distribution, Four groups")
```

```{r ASsigning expressed NGs by TSS, eval=FALSE, include=TRUE}

#### HOw I assigned expressed NG TSS to new peak file
### export new peak file as .bed file
a4gr <- all_four_filt_counts %>% 
  dplyr::select(seqnames:Geneid) %>% 
  dplyr::rename("Peakid"=Geneid) %>% 
  GRanges()
 
# write_delim((as.data.frame(a4gr)),delim = "\t",file = "data/Final_four_data/all_four_filt.bed", col_name=FALSE)## write tab delim datafile without column headers

###Do the same with TSS file
#  ensembl <- useEnsembl(biomart = "genes", dataset = "hsapiens_gene_ensembl")
# my_attributes <- c('chromosome_name', 'start_position', 'end_position','entrezgene_id', 'ensembl_gene_id', 'hgnc_symbol')
# searchAttributes(mart = ensembl, pattern = "refseq")
# searchDatasets(mart=ensembl,pattern="NCBI")
# gene_list <- getBM(attributes=my_attributes,
#       filters = "hgnc_symbol",
#         values = S13Table$SYMBOL,
#       mart = ensembl)
# write.csv(gene_ref,"data/gene_ref.csv")
# listFilters(ensembl)
####
# gene_ref <- read.csv("data/gene_ref.csv")
# gene_full_list <- gene_list %>% 
#   mutate(chromosome_name=as.numeric(chromosome_name))%>% 
#   # filter(!is.na(chromosome_name)) #%>% 
#   na.omit(.)
#   dplyr::select(chromosome_name,start_position,end_position,entrezgene_id,ensembl_gene_id)
### I performed the bedtools using cli. I moved the files to the correct directory on my hardrive to do this.
# bedtools closest -a all_four_filt.bed -b All_TSS_NGs.bed -D b >assigned_newNGbyTSS.bed
#### This file only has the tss, I had to link the geneIDs and symbols and names using the lookup_table (contains TSS, Start and end, ENTREZID, ensembl_id,and SYBOL)
 lookup_table <- read.csv("data/Final_four_data/expressedgene_lookuptable.csv", row.names = 1)
NG_assigned_table <- read_delim("data/Final_four_data/assigned_newNGbyTSS.bed", col_names = FALSE)

TSS_NG_table <- NG_assigned_table %>% 
  dplyr::rename("chr"=X1,
         "start"=X2,
         "end"= X3,
         "Peakid"=X6,
         "NG_chr"=X7,
         "NG_start"=X8 ,
         "NG_end"=X9 ,
         "dist_to_NG"=X13) %>% 
  dplyr::select(chr:end,Peakid:NG_end,dist_to_NG) %>%
  left_join(., lookup_table, by= c("NG_chr"="NG_chr","NG_end"="NG_start")) %>% 
  dplyr::rename("NG_end"="NG_end.y", "NG_TSS"=NG_end) %>% 
  dplyr::select(chr:NG_start,NG_end,NG_TSS,ENTREZID:SYMBOL,dist_to_NG) #%>%
  # write_delim(.,"data/Final_four_data/TSS_assigned_NG.txt",delim = "\t",col_names = TRUE)

Collapsed_new_peaks <- TSS_NG_table %>% 
  group_by(chr, start, end, Peakid) %>%
    summarise(chr=paste(unique(chr)),
              start=paste(unique(start)),
              end=paste(unique(end)),
              NG_chr=paste(NG_chr,collapse=":"),
              NG_start=paste(NG_start,collapse=":"),
              NG_end=paste(NG_end,collapse=":"),
              NG_TSS=paste(NG_TSS,collapse=":"),
              NCBI_gene = paste(unique(ENTREZID),collapse=","),
              ensembl_ID= paste(unique(ensembl_id),collapse = ","),
              SYMBOL= paste(unique(SYMBOL),collapse = ",") , 
              dist_to_NG =min(dist_to_NG))
 # write_delim(Collapsed_new_peaks,"data/Final_four_data/collapsed_new_peaks.txt",delim = "\t",col_names = TRUE)
```
### Eight group peak characteristics

```{r new MRC8}
TSS_assigned_NG <- read_delim("data/Final_four_data/TSS_assigned_NG.txt", delim = "\t", col_names = TRUE)
Collapsed_new_peaks <- read_delim("data/Final_four_data/collapsed_new_peaks.txt", delim = "\t", col_names = TRUE)

median_24_lfc <- read_csv("data/Final_four_data/median_24_lfc.csv") 
median_3_lfc <- read_csv("data/Final_four_data/median_3_lfc.csv")

open_3med <- median_3_lfc %>% 
  dplyr::filter(med_3h_lfc > 0)

close_3med <- median_3_lfc %>% 
  dplyr::filter(med_3h_lfc < 0)

open_24med <- median_24_lfc %>% 
  dplyr::filter(med_24h_lfc > 0)

close_24med <- median_24_lfc %>% 
  dplyr::filter(med_24h_lfc < 0)

medA <- median_3_lfc %>% 
  left_join(median_24_lfc, by=c("peak"="peak")) %>% 
  dplyr::filter(med_3h_lfc > 0 & med_24h_lfc>0)

medB <- median_3_lfc %>% 
  left_join(median_24_lfc, by=c("peak"="peak")) %>% 
  dplyr::filter(med_3h_lfc < 0 & med_24h_lfc < 0)
 
medC <- median_3_lfc %>% 
  left_join(median_24_lfc, by=c("peak"="peak")) %>% 
  dplyr::filter(med_3h_lfc > 0& med_24h_lfc <0)
  

medD <- median_3_lfc %>% 
 left_join(median_24_lfc, by=c("peak"="peak"))%>% 
  dplyr::filter(med_3h_lfc < 0 & med_24h_lfc > 0)
 

EAR_open <- EAR_ff %>%
  dplyr::filter(Peakid %in% open_3med$peak)
  
EAR_open_gr <- EAR_open %>% GRanges()

EAR_close <- EAR_ff %>%
  dplyr::filter(Peakid %in% close_3med$peak) 

EAR_close_gr <- EAR_close %>% GRanges()

LR_open <- LR_ff %>%
  dplyr::filter(Peakid %in% open_24med$peak) 

LR_open_gr <- LR_open %>% GRanges()

LR_close <- LR_ff %>%
  dplyr::filter(Peakid %in% close_24med$peak) 

LR_close_gr <- LR_close %>% GRanges()

NR_gr <- NR_ff %>% 
   GRanges()

ESR_open <- ESR_ff %>% 
  dplyr::filter(Peakid %in% medA$peak)  
 
ESR_open_gr <- ESR_open %>% GRanges()

ESR_close <- ESR_ff %>% 
  dplyr::filter(Peakid %in% medB$peak)  

ESR_close_gr <- ESR_close %>% GRanges()

ESR_C <- ESR_ff %>% 
  dplyr::filter(Peakid %in% medC$peak) 

ESR_D <- ESR_ff %>% 
  dplyr::filter(Peakid %in% medD$peak) 


ESR_OC <- ESR_C %>% 
  rbind(ESR_D)
ESR_OC_gr <- ESR_OC %>% GRanges()

mylist_new <- list(EAR_open_gr, EAR_close_gr,ESR_open_gr, ESR_close_gr,ESR_OC_gr, LR_open_gr, LR_close_gr,NR_gr, background_gr)
names(mylist_new) <- c("EAR_open", "EAR_close","ESR_open", "ESR_close","ESR_OC", "LR_open", "LR_close","NR","background")
# saveRDS(ESR_D,"data/Final_four_data/ESR_D.RDS")
# peakAnnoList<- lapply(mylist_new, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# names(peakAnnoList) <- c("EAR_open", "EAR_close","ESR_open", "ESR_close","ESR_OC", "LR_open", "LR_close","NR","background")
# saveRDS(peakAnnoList, "data/Final_four_data/peakAnnoList_ff_8motif.RDS")
peakAnnoList_8_motif <- readRDS("data/Final_four_data/peakAnnoList_ff_8motif.RDS")
plotAnnoBar(peakAnnoList_8_motif[c(8,1:7)])+
  ggtitle ("Genomic Feature Distribution, eight groups")

```
```{r Correlations by distance, eval=FALSE, include=FALSE}
hour3_lfc_2kb <- toplist_ff %>% 
  dplyr::filter(time=="3 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols = c(time, peak), names_from = trt, values_from = logFC) %>% 
  left_join(., (Collapsed_new_peaks %>% dplyr::select(Peakid, NCBI_gene:dist_to_NG)), by = c("peak"="Peakid") ) %>% 
  dplyr::filter(dist_to_NG>-2000| dist_to_NG<2000)
hour3_lfc_2kb %>% 
  ggplot(.,aes(x=DOX, y=DNR))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour logFC,2kb filtering")
hour3_lfc_2kb %>% 
  ggplot(.,aes(x=DOX, y=EPI))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour logFC, 2kb filtering")
hour3_lfc_2kb %>% 
  ggplot(.,aes(x=DOX, y=MTX))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour logFC, 2kb filtering")
```
```{r 24hour 2kb, eval=FALSE, include=FALSE}
hour24_lfc_2kb <- toplist_ff %>% 
  dplyr::filter(time=="24 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols = c(time, peak), names_from = trt, values_from = logFC) %>% 
  left_join(., (Collapsed_new_peaks %>% dplyr::select(Peakid, NCBI_gene:dist_to_NG)), by = c("peak"="Peakid") ) %>% 
  dplyr::filter(dist_to_NG>-2000| dist_to_NG<2000)
hour24_lfc_2kb %>% 
  ggplot(.,aes(x=DOX, y=DNR))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour logFC,2kb filtering")
hour24_lfc_2kb %>% 
  ggplot(.,aes(x=DOX, y=EPI))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour logFC, 2kb filtering")
hour24_lfc_2kb %>% 
  ggplot(.,aes(x=DOX, y=MTX))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour logFC, 2kb filtering")
```
  
### Overlapping peaks with H3K27ac data
```{r anno overlap with h3k27ac}
Snyder_41peaks <- read.delim("data/other_papers/ENCFF966JZT_bed_Snyder_41peaks.bed",header=TRUE) %>% 
  GRanges()


raodah_fullpeaks <- readRDS( "data/Final_four_data/All_Raodahpeaks.RDS") %>%
  GRanges %>% 
  GenomeInfoDb::keepStandardChromosomes(., pruning.mode = "coarse") %>% 
  dplyr::filter(seqnames!="chrM") %>% 
  dplyr::filter(seqnames!="chrY")
  
# rtracklayer::export.bed(raodah_fullpeaks,con= "data/Final_four_data/H3K27ac_files/H3k27ac.bed")
   
overlap_ac <- join_overlap_intersect((new_peaks_gr %>%  dplyr::filter(seqnames !="chrY")),raodah_fullpeaks) %>% as.data.frame()
overlap_ac %>%
  distinct(Geneid)
# saveRDS(overlap_ac, "data/Final_four_data/overlapping_ac_atac_peaks.RDS")

# H3K27ac_NG_df <- read.delim("data/Final_four_data/H3K27ac_files/H3k27ac.bed", header = FALSE) 

H3K27ac_NG_df <- read.delim("data/Final_four_data/H3K27ac_files/H3K27ac_NG_df.bed")
```

```{r H3K27ac neargene file, eval=FALSE, include=FALSE}
assigned_H3K27ac_TSS <- readGeneric("data/Final_four_data/H3K27ac_files/H3K27ac_NG_byTSS.bed", keep.all.metadata = TRUE) %>% as.data.frame()

H3K27ac_NG_df <- assigned_H3K27ac_TSS %>%
#  NOTE FOR MYSELF  (are we base 0 or base 1!!!!!)  
# The peak file names do not match the bed file names due to the 0/1 issue.  This means to match up MRC categories by name, I need to alter the first start position by +1! 
  mutate(start = start+1) %>% 
  unite("Geneid",seqnames:end, sep = ".", remove= FALSE) %>% 
  dplyr::rename("NG_chr"=V7,
         "NG_start"=V8 ,
         "NG_end"=V9 ,
         "ENTREZID"= V10,
         "ensembl_id"= V11,
         "SYMBOL"=V12,
         "dist_to_NG"=V13) %>%
  dplyr::select(seqnames, start,end, Geneid,NG_chr:dist_to_NG) 
write_tsv(H3K27ac_NG_df, "data/Final_four_data/H3K27ac_files/H3K27ac_NG_df.bed", col_names = TRUE)

```

### abs LFC across clusters

```{r, fig.width=10, fig.height=6}
toplist_ff <- readRDS("data/Final_four_data/toplist_ff.RDS")
mean_lfc <-   toplist_ff %>%
  mutate(EAR_open = if_else(peak %in% EAR_open$Peakid, "y", "no")) %>%
  mutate(EAR_close = if_else(peak %in% EAR_close$Peakid, "y", "no")) %>%
  mutate(ESR_open = if_else(peak %in% ESR_open$Peakid, "y", "no")) %>%
  mutate(ESR_close= if_else(peak %in% ESR_close$Peakid, "y", "no")) %>%
  # mutate(ESR_OC= if_else(peak %in% ESR_OC$Peakid, "y", "no")) %>%
  mutate(ESR_C= if_else(peak %in% ESR_C$Peakid, "y", "no")) %>%
  mutate(ESR_D= if_else(peak %in% ESR_D$Peakid, "y", "no")) %>%
    mutate(LR_open= if_else(peak %in% LR_open$Peakid, "y", "no")) %>%
    mutate(LR_close= if_else(peak %in% LR_close$Peakid, "y", "no")) %>%
    mutate(NR = if_else(peak %in% NR_ff$Peakid, "y", "no"))%>%
  mutate(trt = factor(trt,levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%
  group_by(trt, time) %>%
  mutate(absFC = (logFC)) %>%
  dplyr::select(trt, time, absFC, EAR_open:NR) %>%
  dplyr::summarize(
    EAR_op = mean(absFC[EAR_open == "y"]),
    EAR_cl = mean(absFC[EAR_close == "y"]),
    ESR_op = mean(absFC[ESR_open == "y"]),
    ESR_cl = mean(absFC[ESR_close == "y"]),
    # ESR_OC = mean(absFC[ESR_OC == "y"]),
    ESR_c = mean(absFC[ESR_C == "y"]),
    ESR_d = mean(absFC[ESR_D == "y"]),
    LR_op = mean(absFC[LR_open == "y"]),
    LR_cl = mean(absFC[LR_close == "y"]),
    NR = mean(absFC[NR == "y"])
  ) %>%
  as.data.frame()


mean_lfc %>%
  ungroup() %>%
  pivot_longer(!c(trt, time), names_to = "Motif",
               values_to = "meanLFC") %>%
  # mutate(Set = dplyr::case_match(Motif,
  #                        "EAR open"~  EAR_op ,
  #                         EAR_cm ~"EAR close" ,
  #                         ESR_om ~"ESR open",
  #                         ESR_cm ~ "ESR close",
  #                         ESR_ocm ~ "ESR open/close",
  #                         LR_om~ "LR open",
  #                         LR_cm~ "LR close",
  #                         NRm ~ "No Response",
  #                         .default = Motif)) %>%
  ggplot(., aes(x = time,y = meanLFC,col = trt,
    group = trt
  )) +
  geom_point(size = 2) +
  geom_line(linewidth = 2) +
  ggpubr::fill_palette(drug_pal) +
  # guides(fill=guide_legend(title = "Treatment"))+
  facet_wrap( ~ Motif, nrow = 2) +
  theme_bw() +
  xlab("Time (hours)") +
  scale_color_manual(values = drug_pal) +
  ylab(" Avg. Log Fold Change") +
  theme_bw() +
  # ggtitle(" average |Log Fold| for genes across treatment each motif")+
  theme(
    plot.title = element_text(size = rel(1.0), hjust = 0.5),
    axis.title = element_text(size = 15, color = "black"),
    axis.line = element_line(linewidth = 1.0),
    strip.background = element_rect(fill = "transparent"),
    axis.text = element_text(
      size = 10,
      color = "black",
      angle = 0
    ),
    strip.text.x = element_text(
      size = 11,
      color = "black",
      face = "bold"
    )
  )
```

### making median dataframes

```{r RNA correlation using ATAC-k27 peaks}

# Collapsed_H3k27ac_NG <- H3K27ac_NG_df %>%
#   group_by(seqnames, start, end, Geneid) %>%
#     summarise(seqnames=paste(unique(seqnames)),
#               start=paste(unique(start)),
#               end=paste(unique(end)),
#               NG_chr=paste(NG_chr,collapse=":"),
#               NG_start=paste(NG_start,collapse=":"),
#               NG_end=paste(NG_end,collapse=":"),
#               # NG_TSS=paste(NG_TSS,collapse=":"),
#               ENTREZID = paste(unique(ENTREZID),collapse=":"),
#               ensembl_ID= paste(unique(ensembl_id),collapse = ":"),
#               SYMBOL= paste(unique(SYMBOL),collapse = ",") ,
#               dist_to_NG =min(dist_to_NG))
#  write_delim(Collapsed_H3k27ac_NG,"data/Final_four_data/H3K27ac_files/Collapsed_H3k27ac_NG.txt",delim = "\t",col_names = TRUE)
Collapsed_H3k27ac_NG <- read_delim("data/Final_four_data/H3K27ac_files/Collapsed_H3k27ac_NG.txt",delim = "\t",col_names = TRUE)

RNA_median_3_lfc <- readRDS("data/other_papers/RNA_median_3_lfc.RDS")
RNA_median_24_lfc <- readRDS("data/other_papers/RNA_median_24_lfc.RDS")
overlap_df_ggplot <- readRDS("data/Final_four_data/LFC_ATAC_K27ac.RDS")
AC_median_3_lfc <- read_csv("data/Final_four_data/AC_median_3_lfc.csv")
AC_median_24_lfc <- read_csv("data/Final_four_data/AC_median_24_lfc.csv")

anti_joined_lfc <- median_24_lfc %>% 
  dplyr::filter(!peak %in% overlap_df_ggplot$peakid)  %>% 
  left_join(., (Collapsed_new_peaks %>% dplyr::select(Peakid, NCBI_gene:dist_to_NG)), by = c("peak"="Peakid")) %>% 
  left_join(.,median_3_lfc,by = c("peak"="peak")) %>% 
  left_join(., RNA_median_3_lfc ,
                by=c("SYMBOL"="SYMBOL", "NCBI_gene"="ENTREZID")) %>%
  left_join(., RNA_median_24_lfc,
                 by=c("SYMBOL"="SYMBOL", "NCBI_gene"="ENTREZID")) 

  

joined_LFC_df <- overlap_df_ggplot %>%
  left_join(.,(Collapsed_new_peaks %>%
                 dplyr::select(Peakid,dist_to_NG, NCBI_gene:SYMBOL)),
            by=c("peakid"="Peakid")) %>% 
  left_join(., RNA_median_3_lfc ,
                # %>%
                #   dplyr::select(SYMBOL,RNA_3h_lfc)), 
            by=c("SYMBOL"="SYMBOL", "NCBI_gene"="ENTREZID")) %>%
  left_join(., RNA_median_24_lfc,# %>%
                  # dplyr::select(SYMBOL,RNA_24h_lfc)),
             by=c("SYMBOL"="SYMBOL", "NCBI_gene"="ENTREZID")) 

anti_joined_H3k27ac <-Collapsed_H3k27ac_NG%>% 
  dplyr::filter(!Geneid %in% overlap_df_ggplot$Geneid) %>% 
  separate(ENTREZID, into= c("ENTREZID","2ndENTREZID"), sep = ":") %>% 
  separate(SYMBOL, into= c("SYMBOL","2ndSYMBOL"), sep = ",") %>%
  mutate(ENTREZID=as.numeric(ENTREZID)) %>% 
  left_join(.,(RNA_median_24_lfc %>% mutate(ENTREZID=as.numeric(ENTREZID))),by=c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  left_join(.,(RNA_median_3_lfc %>% mutate(ENTREZID=as.numeric(ENTREZID))),by=c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  left_join(.,AC_median_24_lfc, by=c("Geneid"="Peakid")) %>% 
  left_join(.,AC_median_3_lfc, by=c("Geneid"="Peakid")) %>% 
  dplyr::select(Geneid,ENTREZID:RNA_3h_lfc,AC_24h_lfc,AC_3h_lfc)


```

#### Correlations

## Correlation between k27ac/atac shared peak median LFC and RNA median LFC
```{r correlations shared 2kb}

joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=AC_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 2kb H3K27ac peaks and RNA-NG  at 3 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")

joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=AC_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 2kb H3K27ac peaks and RNA-NG  at 24 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")


joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 2kb ATAC peaks and RNA-NG  at 3 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 2kb ATAC peaks and RNA-NG  at 24 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")


```
20kb shared peaks:

```{r correlationsshared 20 kb}

joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=AC_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 20kb H3K27ac peaks and RNA-NG  at 3 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")

joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=AC_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 20kb H3K27ac peaks and RNA-NG  at 24 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")


joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 20kb ATAC peaks and RNA-NG  at 3 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

joined_LFC_df %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of shared 20kb ATAC peaks and RNA-NG  at 24 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")


```


### Correlation between H3K27-not shared and atac not- shared peak median LFC and RNA median LFC   

2kb
```{r correlations not shared 2kb}

  anti_joined_H3k27ac %>%
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>%
ggplot(., aes(x=AC_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of H3K27ac peaks not shared by ATAC peaks and RNA-NG  at 3 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")
# 
anti_joined_H3k27ac %>%
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>%
ggplot(., aes(x=AC_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of H3K27ac peaks not shared by ATAC peaks and RNA-NG  at 24 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")


anti_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of ATAC peaks not shared with H3K27ac and RNA-NG  at 3 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

anti_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of ATAC peaks not shared with H3K27ac and RNA-NG at 24 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

fullpeaks_H3k27ac_lfc <- Collapsed_H3k27ac_NG%>% 
  dplyr::filter(!Geneid %in% overlap_df_ggplot$Geneid) %>% 
  separate(ENTREZID, into= c("ENTREZID","2ndENTREZID"), sep = ":") %>% 
  separate(SYMBOL, into= c("SYMBOL","2ndSYMBOL"), sep = ",") %>%
  mutate(ENTREZID=as.numeric(ENTREZID)) %>% 
  left_join(.,(RNA_median_24_lfc %>% mutate(ENTREZID=as.numeric(ENTREZID))),by=c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  left_join(.,(RNA_median_3_lfc %>% mutate(ENTREZID=as.numeric(ENTREZID))),by=c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  left_join(.,AC_median_24_lfc, by=c("Geneid"="Peakid")) %>% 
  left_join(.,AC_median_3_lfc, by=c("Geneid"="Peakid")) %>% 
  dplyr::select(Geneid,ENTREZID:RNA_3h_lfc,AC_24h_lfc,AC_3h_lfc)
  

```
20kb

```{r correlations not shared 20kb}

  anti_joined_H3k27ac %>%
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>%
ggplot(., aes(x=AC_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of H3K27ac peaks not shared by ATAC peaks and RNA-NG  at 3 hours 20kb")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")
# 
anti_joined_H3k27ac %>%
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>%
ggplot(., aes(x=AC_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of H3K27ac peaks not shared by ATAC peaks and RNA-NG  at 24 hours 20kb")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")


anti_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of ATAC peaks not shared with H3K27ac and RNA-NG  at 3 hours 20kb")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

anti_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point()+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of ATAC peaks not shared with H3K27ac and RNA-NG at 24 hours 20kb")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")


```
### Correlations of all peaks within 2kb and RNA 

```{r correlation 2kb filt only data}

fullpeaks_joined_lfc <- median_24_lfc %>% 
  left_join(., (Collapsed_new_peaks %>% dplyr::select(Peakid, NCBI_gene:dist_to_NG)), by = c("peak"="Peakid")) %>% 
  left_join(.,median_3_lfc,by = c("peak"="peak")) %>% 
  left_join(., RNA_median_3_lfc ,
                by=c("SYMBOL"="SYMBOL", "NCBI_gene"="ENTREZID")) %>%
  left_join(., RNA_median_24_lfc,
                 by=c("SYMBOL"="SYMBOL", "NCBI_gene"="ENTREZID")) 

fullpeaks_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of 2kb ATAC peaks and RNA-NGat 3 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

fullpeaks_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of 2kb ATAC peaks and RNA-NGat 24 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

fullpeaks_H3k27ac_lfc <- Collapsed_H3k27ac_NG%>%
  separate(ENTREZID, into= c("ENTREZID","2ndENTREZID"), sep = ":") %>% 
  separate(SYMBOL, into= c("SYMBOL","2ndSYMBOL"), sep = ",") %>%
  mutate(ENTREZID=as.numeric(ENTREZID)) %>% 
  left_join(.,(RNA_median_24_lfc %>% mutate(ENTREZID=as.numeric(ENTREZID))),by=c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  left_join(.,(RNA_median_3_lfc %>% mutate(ENTREZID=as.numeric(ENTREZID))),by=c("ENTREZID"="ENTREZID","SYMBOL"="SYMBOL")) %>% 
  left_join(.,AC_median_24_lfc, by=c("Geneid"="Peakid")) %>% 
  left_join(.,AC_median_3_lfc, by=c("Geneid"="Peakid")) %>% 
  dplyr::select(Geneid,ENTREZID:RNA_3h_lfc,AC_24h_lfc,AC_3h_lfc)

fullpeaks_H3k27ac_lfc %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=AC_3h_lfc,y=RNA_3h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of all 3K27ac peaks (2kb) and RNA-NG at 3 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")

fullpeaks_H3k27ac_lfc %>% 
  dplyr::filter(dist_to_NG>-2000 & dist_to_NG<2000) %>% 
ggplot(., aes(x=AC_24h_lfc,y=RNA_24h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of all 3K27ac peaks (2kb) and RNA-NG at 24 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")
```

```{r correlation 20kb  full data}

fullpeaks_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=med_3h_lfc,y=RNA_3h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of 20kb ATAC peaks and RNA-NGat 3 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")

fullpeaks_joined_lfc %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=med_24h_lfc,y=RNA_24h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of 2kb ATAC peaks and RNA-NGat 24 hours")+
  xlab("ATAC peak LFC")+
   ylab("RNA LFC")


fullpeaks_H3k27ac_lfc %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=AC_3h_lfc,y=RNA_3h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of all H3K27ac peaks (20kb) and RNA-NG at 3 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")

fullpeaks_H3k27ac_lfc %>% 
  dplyr::filter(dist_to_NG>-20000 & dist_to_NG<20000) %>% 
ggplot(., aes(x=AC_24h_lfc,y=RNA_24h_lfc))+
   geom_point(col="cornflowerblue")+
   sm_statCorr(corr_method = 'pearson')+
   ggtitle("Correlation of all H3K27ac peaks (2kb) and RNA-NG at 24 hours")+
  xlab("H3K27ac peak LFC")+
   ylab("RNA LFC")
```


### Correlation Heat map


```{r}
corr_heatmapATAC_K27ac <- data.frame(Comp_type=c("All_ATAC","ATAC+K27ac","ATAC-K27ac","All_H3K27ac","K27ac+ATAC","K27ac-ATAC"),
hr3_2kb=c(-.012,-.042,0.043,-0.042,-0.075,-0.07),
hr24_2kb=c(0.19,0.26,0.15,.23,.31,.36),
hr3_20kb=c(0.000092,-0.013,0.011,-0.041,-0.056,-0.08),
hr24_20kb=c(0.097,.22,0.069,0.24,0.26,0.36))

corr_heatmapATAC_K27ac %>% 
  column_to_rownames("Comp_type") %>% 
  as.matrix() %>% 
ComplexHeatmap::Heatmap(.,cluster_rows = FALSE, cluster_columns = FALSE)

```

