---
title: "GO and KEGG pathway analysis"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
# library(Cormotif)
library(BiocParallel)
library(ggpubr)
# library(devtools)
# # install_github('davetang/bedr')
# library(bedr)
library(biomaRt)
library(clusterProfiler)
```
##### data loading
```{r loading data}
peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
# list2env(peakAnnoList_n45_motif, envir = .GlobalEnv)

EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr <-  GRanges(EAR_df)

ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr <-  GRanges(ESR_df)
 
LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr <-  GRanges(LR_df)
 
NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr <-  GRanges(NR_df)


exp_neargene_table <- read_delim("data/n45_bedfiles/exp_neargene_table.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)

# col_ng_peak <- exp_neargene_table %>%
#   group_by(chr, start, end, peakid) %>%
#     summarise(NCBI_gene = paste(unique(ENTREZID),collapse=","),
#               ensembl_ID= paste(unique(ensembl_id),collapse = ","),
#               SYMBOL= paste(unique(SYMBOL),collapse = ",") , 
#               dist_to_NG =min(dist_to_NG))
# 

EAR_NG_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% EAR_df$id) %>%
  dplyr::filter(dist_to_NG<20000) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

ESR_NG_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% ESR_df$id) %>% 
   dplyr::filter(dist_to_NG<20000) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

LR_NG_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% LR_df$id) %>%
   dplyr::filter(dist_to_NG<20000) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

NR_NG_list <- exp_neargene_table %>% 
    dplyr::filter(peakid %in% NR_df$id) %>% 
   dplyr::filter(dist_to_NG<20000) %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

background_df <- exp_neargene_table %>% 
  dplyr::select(ENTREZID) %>% 
  distinct()

EAR_resgenes <- readRDS("data/GO_KEGG_analysis/EAR_resgenes.RDS")
ESR_resgenes <- readRDS("data/GO_KEGG_analysis/ESR_resgenes.RDS")
LR_resgenes <- readRDS("data/GO_KEGG_analysis/LR_resgenes.RDS")
NR_resgenes <- readRDS("data/GO_KEGG_analysis/NR_resgenes.RDS")

EAR_enh_peaks_20k <- readRDS("data/enhancerdata/EAR_enh_peaks_20k.RDS")
ESR_enh_peaks_20k <- readRDS("data/enhancerdata/ESR_enh_peaks_20k.RDS")
LR_enh_peaks_20k <- readRDS("data/enhancerdata/LR_enh_peaks_20k.RDS")
NR_enh_peaks_20k <- readRDS("data/enhancerdata/NR_enh_peaks_20k.RDS")

EAR_resgenes_20k <- readRDS("data/GO_KEGG_analysis/EAR_resgenes_20k.RDS")
EAR_resgenes_20k_enh <- readRDS("data/GO_KEGG_analysis/EAR_resgenes_20k_enh.RDS")
ESR_resgenes_20k <- readRDS("data/GO_KEGG_analysis/ESR_resgenes_20k.RDS")
ESR_resgenes_20k_enh <- readRDS("data/GO_KEGG_analysis/ESR_resgenes_20k_enh.RDS")

LR_resgenes_20k <- readRDS("data/GO_KEGG_analysis/LR_resgenes_20k.RDS")
LR_resgenes_20k_enh <- readRDS("data/GO_KEGG_analysis/LR_resgenes_20k_enh.RDS")

NR_resgenes_20k <- readRDS("data/GO_KEGG_analysis/NR_resgenes_20k.RDS")
NR_resgenes_20k_enh <- readRDS("data/GO_KEGG_analysis/NR_resgenes_20k_enh.RDS")

### intersection of peaks files
S13Table <- read.csv( "data/other_papers/S13Table_Matthews2024.csv",row.names = 1)
##14021

EAR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="EAR") %>% 
  dplyr::select(ENTREZID) %>% 
  mutate(ENTREZID= as.character(ENTREZID))
ESR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="ESR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))
LR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="LR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID=as.character(ENTREZID))
NR_RNA <- S13Table %>% 
  dplyr::filter(MOTIF=="NR")%>% 
  dplyr::select(ENTREZID)%>% 
  mutate(ENTREZID= as.character(ENTREZID))

Resp_RNA <- EAR_RNA %>% 
  rbind(.,ESR_RNA) %>% 
  rbind(., LR_RNA) %>% 
  distinct(ENTREZID)

int_EAR_n_RNAresp <- intersect(EAR_NG_list$ENTREZID, Resp_RNA$ENTREZID)
int_ESR_n_RNAresp <- intersect(ESR_NG_list$ENTREZID, Resp_RNA$ENTREZID)
int_LR_n_RNAresp <- intersect(LR_NG_list$ENTREZID, Resp_RNA$ENTREZID)
EAR_int_resgenes <- readRDS("data/GO_KEGG_analysis/EAR_int_resgenes.RDS")
ESR_int_resgenes <- readRDS("data/GO_KEGG_analysis/ESR_int_resgenes.RDS")
LR_int_resgenes <- readRDS("data/GO_KEGG_analysis/LR_int_resgenes.RDS")

```

### EAR response genes

```{r GO_EAR}
# EAR_resgenes <- gost(query = EAR_NG_list$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(EAR_resgenes,"data/GO_KEGG_analysis/EAR_resgenes.RDS")


EARnom <- gostplot(EAR_resgenes, capped = FALSE, interactive = TRUE)
# EARnom

EARnomtable <- EAR_resgenes$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

EARnomtable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in EAR neargenes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

EARnomtable %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in EAR neargenes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")





```

### ESR response genes


```{r GO_ESR}
# ESR_resgenes <- gost(query = ESR_NG_list$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(ESR_resgenes,"data/GO_KEGG_analysis/ESR_resgenes.RDS")

ESRnom <- gostplot(ESR_resgenes, capped = FALSE, interactive = TRUE)
# ESRnom

ESRnomtable <- ESR_resgenes$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

ESRnomtable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in ESR neargenes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

ESRnomtable %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in ESR neargenes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")


```

### LR response genes


```{r GO_LR}
# LR_resgenes <- gost(query = LR_NG_list$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(LR_resgenes,"data/GO_KEGG_analysis/LR_resgenes.RDS")

LRnom <- gostplot(LR_resgenes, capped = FALSE, interactive = TRUE)
# LRnom

LRnomtable <- LR_resgenes$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

LRnomtable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in LR neargenes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

LRnomtable %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in LR neargenes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")


```


### NR response genes

```{r GO_NR}
# NR_resgenes <- gost(query = NR_NG_list$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(NR_resgenes,"data/GO_KEGG_analysis/NR_resgenes.RDS")

NRnom <- gostplot(NR_resgenes, capped = FALSE, interactive = TRUE)
# NRnom

NRnomtable <- NR_resgenes$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

NRnomtable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in NR neargenes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

NRnomtable %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in NR neargenes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")


```

## Peaks within 20,000 bps of assigned nearest expressed gene, GO and KEGG enrichment

```{r data_expNG}

# exp_neargene_table <- read_delim("data/n45_bedfiles/exp_neargene_table.tsv", 
#     delim = "\t", escape_double = FALSE, 
#     trim_ws = TRUE)

# col_ng_peak <- exp_neargene_table %>%
#   group_by(chr, start, end, peakid) %>%
#     summarise(NCBI_gene = paste(unique(ENTREZID),collapse=","),
#               ensembl_ID= paste(unique(ensembl_id),collapse = ","),
#               SYMBOL= paste(unique(SYMBOL),collapse = ",") , 
#               dist_to_NG =min(dist_to_NG)) %>% 
#   ungroup()

# write.csv(col_ng_peak, "data/Collapsed_expressed_NG_peak_table.csv")
col_ng_peak <- read.csv("data/Collapsed_expressed_NG_peak_table.csv", row.names = 1)

EAR_peak_list_20k <-  col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>% 
    dplyr::filter(peakid %in% EAR_df$id) 

ESR_peak_list_20k <- col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>% 
    dplyr::filter(peakid %in% ESR_df$id) 

LR_peak_list_20k <- col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>% 
    dplyr::filter(peakid %in% LR_df$id) 

NR_peak_list_20k <- col_ng_peak %>%
  dplyr::filter(dist_to_NG<20000) %>% 
    dplyr::filter(peakid %in% NR_df$id) 
```

### EAR 20,000 bp NG 
```{r GO_EAR_20k}
# EAR_resgenes_20k <- gost(query = EAR_peak_list_20k$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(EAR_resgenes_20k,"data/GO_KEGG_analysis/EAR_resgenes_20k.RDS")


EARnom <- gostplot(EAR_resgenes_20k, capped = FALSE, interactive = TRUE)
# EARnom

EARnomtable_20k <- EAR_resgenes_20k$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

EARnomtable_20k %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in EAR neargenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

EARnomtable_20k %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in EAR neargenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")





```

### EAR peaks overlapping enhancers
```{r GO_EAR_20k_enh}
# EAR_resgenes_20k_enh <- gost(query = unique(EAR_enh_peaks_20k$NCBI_gene),
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(EAR_resgenes_20k_enh,"data/GO_KEGG_analysis/EAR_resgenes_20k_enh.RDS")


# EARnom <- gostplot(EAR_resgenes_20k_enh, capped = FALSE, interactive = TRUE)
# EARnom

EARnomtable_20k_enh <- EAR_resgenes_20k_enh$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

EARnomtable_20k_enh %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in EAR enhancer neargenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

EARnomtable_20k_enh %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in EAR enhancer neargenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```

### ESR 20,000 bp NG 
```{r GO_ESR_20k}
# ESR_resgenes_20k <- gost(query = ESR_peak_list_20k$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(ESR_resgenes_20k,"data/GO_KEGG_analysis/ESR_resgenes_20k.RDS")


# ESRnom <- gostplot(ESR_resgenes_20k, capped = FALSE, interactive = TRUE)
# ESRnom

ESRnomtable_20k <- ESR_resgenes_20k$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

ESRnomtable_20k %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in ESR nESRgenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

ESRnomtable_20k %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in ESR nESRgenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")





```

### ESR peaks overlapping enhancers
```{r GO_ESR_20k_enh}
# ESR_resgenes_20k_enh <- gost(query = unique(ESR_enh_peaks_20k$NCBI_gene),
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(ESR_resgenes_20k_enh,"data/GO_KEGG_analysis/ESR_resgenes_20k_enh.RDS")


# ESRnom <- gostplot(ESR_resgenes_20k_enh, capped = FALSE, interactive = TRUE)
# ESRnom

ESRnomtable_20k_enh <- ESR_resgenes_20k_enh$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

ESRnomtable_20k_enh %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in ESR neargenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

ESRnomtable_20k_enh %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in ESR enhancer neargenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```

### LR 20,000 bp NG 
```{r GO_LR_20k}
# LR_resgenes_20k <- gost(query = LR_peak_list_20k$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(LR_resgenes_20k,"data/GO_KEGG_analysis/LR_resgenes_20k.RDS")


# LRnom <- gostplot(LR_resgenes_20k, capped = FALSE, interactive = TRUE)
# LRnom

LRnomtable_20k <- LR_resgenes_20k$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

LRnomtable_20k %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in LR nESRgenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

LRnomtable_20k %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in LR nESRgenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")





```

### LR peaks overlapping enhancers
```{r GO_LR_20k_enh}
# LR_resgenes_20k_enh <- gost(query = unique(LR_enh_peaks_20k$NCBI_gene),
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(LR_resgenes_20k_enh,"data/GO_KEGG_analysis/LR_resgenes_20k_enh.RDS")


# LRnom <- gostplot(LR_resgenes_20k_enh, capped = FALSE, interactive = TRUE)
# LRnom

LRnomtable_20k_enh <- LR_resgenes_20k_enh$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

LRnomtable_20k_enh %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in LR neargenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

LRnomtable_20k_enh %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in LR enhancer neargenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```

### NR 20,000 bp NG 
```{r GO_NR_20k}
# NR_resgenes_20k <- gost(query = NR_peak_list_20k$ENTREZID,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(NR_resgenes_20k,"data/GO_KEGG_analysis/NR_resgenes_20k.RDS")


# NRnom <- gostplot(NR_resgenes_20k, capped = FALSE, interactive = TRUE)
# NRnom

NRnomtable_20k <- NR_resgenes_20k$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

NRnomtable_20k %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in NR nESRgenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

NRnomtable_20k %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in NR nESRgenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```



### NR peaks overlapping enhancers
```{r GO_NR_20k_enh}
# NR_resgenes_20k_enh <- gost(query = unique(NR_enh_peaks_20k$NCBI_gene),
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(NR_resgenes_20k_enh,"data/GO_KEGG_analysis/NR_resgenes_20k_enh.RDS")


# NRnom <- gostplot(NR_resgenes_20k_enh, capped = FALSE, interactive = TRUE)
# NRnom

NRnomtable_20k_enh <- NR_resgenes_20k_enh$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

NRnomtable_20k_enh %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in NR neargenes_20k" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

NRnomtable_20k_enh %>%
  filter(source=="KEGG", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in NR enhancer neargenes_20k") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```
### intersection of EAR_ATAC and RNA-response genes
#### EAR
```{r int_GO_EAR}
# EAR_int_resgenes <- gost(query = int_EAR_n_RNAresp,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(EAR_int_resgenes,"data/GO_KEGG_analysis/EAR_int_resgenes.RDS")

EAR_int_restable <- EAR_int_resgenes$result %>%
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

EAR_int_restable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in EAR neargenes intersected with RNA response genes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

EAR_int_restable %>%
  filter(source=="KEGG") %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in EAR neargenes intersected with RNA response genes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```

#### ESR
```{r int_GO_ESR}
# ESR_int_resgenes <- gost(query = int_ESR_n_RNAresp,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(ESR_int_resgenes,"data/GO_KEGG_analysis/ESR_int_resgenes.RDS")

ESR_int_restable <- ESR_int_resgenes$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

ESR_int_restable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in ESR neargenes intersected with RNA response genes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

ESR_int_restable %>%
  filter(source=="KEGG") %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in ESR neargenes intersected with RNA response genes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```

#### LR

```{r int_GO_LR}
# LR_int_resgenes <- gost(query = int_LR_n_RNAresp,
#                     organism = "hsapiens",
#                     significant = FALSE,
#                     ordered_query = FALSE,
#                     domain_scope = "custom",
#                     measure_underrepresentation = FALSE,
#                     evcodes = FALSE,
#                     user_threshold = 0.05,
#                     correction_method = c("fdr"),
#                     custom_bg = background_df$ENTREZID,
#                     sources=c("GO:BP","KEGG"))
# saveRDS(LR_int_resgenes,"data/GO_KEGG_analysis/LR_int_resgenes.RDS")

LR_int_restable <- LR_int_resgenes$result %>% 
  dplyr::select(c(source, term_id,
                  term_name,intersection_size, 
                   term_size, p_value))# %>% 

LR_int_restable %>%
   filter(source=="GO:BP", p_value < 0.05) %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(.,caption = "GO:BP terms found in LR neargenes intersected with RNA response genes" ) %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

LR_int_restable %>%
  filter(source=="KEGG") %>% 
  mutate_at(.vars = 6, .funs = scientific_format()) %>%
  kable(., caption = "KEGG terms found in LR neargenes intersected with RNA response genes") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(
    full_width = FALSE,
    position = "left",
    bootstrap_options = c("striped", "hover")
  ) %>%
  scroll_box(width = "100%", height = "400px")

```


```{r}
# library(clusterProfiler)
# genes_go = list("EAR"=int_EAR_n_RNAresp,"ESR"=int_ESR_n_RNAresp, "LR"=int_LR_n_RNAresp)
# names(genes) = sub("_", "\n", names(genes))
compEAR <- enrichGO(int_EAR_n_RNAresp,
                         universe = background_df$ENTREZID,
                         ont = "BP",
                         pvalueCutoff  = 0.05,
                         OrgDb='org.Hs.eg.db',
                         pAdjustMethod = "BH")
# dotplot(compKEGG, showCategory = 15, title = "GO Enrichment Analysis")


compESR <- enrichGO(int_ESR_n_RNAresp,
                         universe = background_df$ENTREZID,
                         ont = "BP",
                         pvalueCutoff  = 0.05,
                         OrgDb='org.Hs.eg.db',
                         pAdjustMethod = "BH")
# dotplot(compESR, showCategory = 15, title = "GO Enrichment Analysis")

compLR <- enrichGO(int_LR_n_RNAresp,
                         universe = background_df$ENTREZID,
                         ont = "BP",
                         pvalueCutoff  = 0.05,
                         OrgDb='org.Hs.eg.db',
                         pAdjustMethod = "BH")
barplot(compLR,x="p.adjust", showCategory = 15, title = "GO Enrichment Analysis for LR")
barplot(compEAR,x="p.adjust", showCategory = 15, title = "GO Enrichment Analysis for EAR")
barplot(compESR,x="p.adjust", showCategory = 15, title = "GO Enrichment Analysis for ESR")
# goplot(compLR)
compEARx <- setReadable(compEAR, 'org.Hs.eg.db', 'ENTREZID')
compESRx <- setReadable(compESR, 'org.Hs.eg.db', 'ENTREZID')
compLRx <- setReadable(compLR, 'org.Hs.eg.db', 'ENTREZID')

```

```{r}
cnetplot(compEARx)
cnetplot(compESRx)
cnetplot(compLRx, node_label="all")
# upsetplot(compLRx)
```

