---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

## Anthracyclines induce global changes in cardiomyocyte chromatin accessibility that overlap with cardiovascular disease loci

#### Authors: E. Renee Matthews, Raodahtullah O. Abodunrin, John D. Hurley, Jose A. Gutierrez, Michelle C. Ward   

### Figure 1: [Thousands of chromatin accessibility changes are induced in iPSC-CMs after anthracycline treatment.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_1.html)

### Figure 2: [Early drug-dependent responsive chromatin regions are enriched in CpG-rich regions near transcription start sites.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_2.html)

### Figure 3: [Regions opening in response to drug treatment are enriched for transcription factor motifs including p53 and JunB. ](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_3.html)

### Figure 4: [Regions that open in response to drugs are enriched near signaling genes, while closing regions are enriched near cardiac genes.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_4.html)

### Figure 5: [Drug-induced chromatin accessibility changes correspond to changes in active histone modification and gene expression.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_5.html)

### Figure 6: [Drug responsive regions are near cardiotoxicity-associated SNPs. ](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_6.html)

### Figure 7: [Drug responsive regions overlap SNPs associated with atrial fibrillation and heart failure.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_7.html)


### [Supplementary Figures 1-10](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Supp_Fig_1-11.html)

### [Supplementary Figures 12-19](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Supp_Fig_12-19.html)

  
    
      
        
          
            
              
                
                  
                  



### Older Analysis pages 
##### QC pages

[Fastqc analysis of ATAC data](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Fastqc_results.html)

[Read counts and Peak calling](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Peak_calling.html)

[Examination of Peak meta-data characteristics](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Peak_analysis.html)

##### Analysis pages using final peaks set based on 4 individuals:

[Final peak set analysis](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/final_four_analysis.html)   
 
[GO/KEGG analysis of associated RNA-expressed genes to close regions by cluster](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/GO_KEGG_analysis.html)

[Looking at enhancers and peaks)](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Enhancer_files_ff.html)

[Looking at TEs and peaks)](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/TE_analysis_ff.html)

[Jaspar motif analysis (in progress)](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Jaspar_motif_ff.html)

[Investigating H3K27ac in a subset of samples](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Raodah_mycount.html)





[Additional plots of long Figure 6](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/final_plot_attempt.html) and   
[Figure 6 iterations](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Correlation_of_SNPnPEAK.html)   
[Figure 7 iterations](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Correlation_of_GWASnPEAK.html)





  
    
 
#### Exploratory analysis pages

##### Read and Peak QC
[Quality Checks on Reads and Peaks](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/QC_on_reads.html)

##### Counts file 

[Counts file import and evaluation](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Counts_matrix.html)

[Differential accessibility analysis of regions](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/DEG_analysis.html)
*** updates added ***

[Cormotif analysis of regions](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Cormotif_analysis.html)

##### Looking at genomic neighborhood around regions

[Transposable Elements](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/TE_analysis_norm.html)

[DOX-DAR analysis of TE enrichment](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/TE_analysis_DOX_DAR.html)

[TSS and CpG islands](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/TSS_and_CUG.html)

[DOX-DAR analysis of TSS and CpG islands](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/TSS_and_CUG_DOX_DAR.html)

[Cardiac enhancer regions](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Enhancer_enrichment.html)

[DOX-DAR cRE enrichment]pending...

[Figure 2 multiple testing correction](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/BH_correction_fig2.html)
    
##### Looking a transcription factor motifs in response regions  

[Memesuite xstreme analysis results](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Jaspar_motif_norm.html)

##### Associating regions with RNA-seq data

[RNA associations with response regions and GO analysis](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/RNA_seq_integration.html)


##### H3K27ac CUT&Tag integration

[H3K27ac QC metrics and Differential acetylation analysis](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/H3K27ac_initial_QC.html)


[Baysian analysis (using Cormotif)](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/H3K27ac_cormotif.html)  


[Integrating H3K27ac data with ATAC-sea and RNA-seq data](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/H3K27ac_integration.html)
        
          
##### Looking at SNPs 

[Cardiotoxicity-associated SNPs](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Cardiotox_SNPs.html)  
Utilizes 115 total SNPs obtained from:  
  Schneider, BP, et al., Genome-Wide Association Study for Anthracycline-Induced Congestive Heart Failure, Clinical Cancer Research, 2017. [Link to paper](https://aacrjournals.org/clincancerres/article/23/1/43/122971/Genome-Wide-Association-Study-for-Anthracycline)   
  108 SNPs used
  
  Park, B. et al., Genome-wide association study of genetic variants related to anthracycline-induced cardiotoxicity in early breast cancer, Cancer Science, 2020. [Link to paper](https://onlinelibrary.wiley.com/doi/10.1111/cas.14446)  
  7 SNPs used
  

            
[Atrial Fibrillation and Heart Failure SNPs](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/AF_HF_SNPs.html)

Data obtained from [GWAS Catalog](https://www.ebi.ac.uk/gwas/home) using the terms atrial fibrillation and heart failure.
 



     
        
          
            
              
                
                  
                    
                    





















