---
title: "Home"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
---

## Anthracyclines induce global changes in cardiomyocyte chromatin accessibility that overlap with cardiovascular disease loci

#### Authors: E. Renee Matthews, Raodahtullah O. Abodunrin, John D. Hurley, Jose A. Gutierrez, Michelle C. Ward   

### Figure 1 [Thousands of chromatin accessibility changes are induced in iPSC-CMs after anthracycline treatment.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_1.html)

### Figure 2 [Early drug-dependent responsive chromatin regions are enriched in CpG-rich regions near transcription start sites.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_2.html)

### Figure 3 [Regions opening in response to drug treatment are enriched for transcription factor motifs including p53 and JunB. ](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_3.html)

### Figure 4 [Regions that open in response to drugs are enriched near signaling genes, while closing regions are enriched near cardiac genes.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_4.html)

### Figure 5 [Drug-induced chromatin accessibility changes correspond to changes in active histone modification and gene expression.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_5.html)

### Figure 6 [Drug responsive regions are near cardiotoxicity-associated SNPs. ](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_6.html)

### Figure 7 [Drug responsive regions overlap SNPs associated with atrial fibrillation and heart failure.](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Figure_7.html)


### [Supplementary Figures 1-10](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Supp_Fig_1-11.html)

### [Supplementary Figures 12-19](https://mward-lab.github.io/Matthews_cardiotox_ATAC_2025/Supp_Fig_12-19.html)

  
    
      
        
          
            
              
                
                  
                  





  
    
      
        
          
            
            





### Older Analysis pages (may not work!)
##### QC pages

[Fastqc analysis of ATAC data](https://reneeisnowhere.github.io/ATAC_learning/Fastqc_results.html)

[Read counts and Peak calling](https://reneeisnowhere.github.io/ATAC_learning/Peak_calling.html)

[Examination of Peak meta-data characteristics](https://reneeisnowhere.github.io/ATAC_learning/Peak_analysis.html)

##### Analysis pages using final peaks set based on 4 individuals:

[Final peak set analysis](https://reneeisnowhere.github.io/ATAC_learning/final_four_analysis.html) and  
[Additional plots](https://reneeisnowhere.github.io/ATAC_learning/final_plot_attempt.html) and   
current [last figure attempts](https://reneeisnowhere.github.io/ATAC_learning/Correlation_of_SNPnPEAK.html)   
[link for Figure 7](https://reneeisnowhere.github.io/ATAC_learning/Correlation_of_GWASnPEAK.html)

[GO/KEGG analysis of neargenes by cluster](https://reneeisnowhere.github.io/ATAC_learning/GO_KEGG_analysis.html)

[Looking at enhancers and peaks)](https://reneeisnowhere.github.io/ATAC_learning/Enhancer_files_ff.html)

[Looking at TEs and peaks)](https://reneeisnowhere.github.io/ATAC_learning/TE_analysis_ff.html)

[Jaspar motif analysis (in progress)](https://reneeisnowhere.github.io/ATAC_learning/Jaspar_motif_ff.html)

[Investigating H3K27ac in a subset of samples](https://reneeisnowhere.github.io/ATAC_learning/Raodah_mycount.html)











  
    
      
        
          
            
              
                
                  
                    
                    





















