---
title: "Jaspar_enrichment"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)

# library(sjmisc)
library(kableExtra)
library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
# library(ATACseqQC)
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
# library(ggVennDiagram)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
# install_github('davetang/bedr')
library(bedr)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)

```


```{r dataframe loading}

# xstreme_ESR_v_NR <-ESR_v_NR_full_xstreme %>%
#   slice_head(n = length(.$ID)-3)
# saveRDS(xstreme_ESR_v_NR, "data/xstreme/xstreme_ESR_v_NR.RDS")
# sea_EAR <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/full peak set Jaspar core vertebrates/EAR_v_NR_appXSTREME_5.5.51713891248048-1903497910.tar/appXSTREME_5.5.51713891248048-1903497910/sea_disc_out/sea.tsv", 
#     delim = "\t", escape_double = FALSE, 
#     trim_ws = TRUE)%>%
#   slice_head(n = length(.$ID)-3)
# saveRDS(sea_EAR, "data/xstreme/sea_EAR.RDS")
# sea_ESR <- sea_ESR%>%
#   slice_head(n = length(.$ID)-3)
# saveRDS(sea_ESR, "data/xstreme/sea_ESR.RDS")
# sea_LR <- sea_LR%>%
#   slice_head(n = length(.$ID)-3)
# saveRDS(sea_LR, "data/xstreme/sea_LR.RDS")

sea_EAR <- readRDS("data/xstreme/sea_EAR.RDS")
sea_ESR <- readRDS("data/xstreme/sea_ESR.RDS")
sea_LR <- readRDS("data/xstreme/sea_LR.RDS")


xstreme_EAR_v_NR <- readRDS("data/xstreme/xstreme_EAR_v_NR.RDS")

xstreme_ESR_v_NR <-readRDS("data/xstreme/xstreme_ESR_v_NR.RDS") 

xstreme_LR_v_NR <- readRDS("data/xstreme/xstreme_LR_v_NR.RDS")

peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)

ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr <- as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)
 
NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)

toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") 
###Because of how I applied the DEG system in RNA-seq analysis, the lFC is opposite of the 
###counts.   I did trt-veh instead of veh-trt.  therefore I need to multiply lfc by -1 to get t
###the right correlation.

toplistall_RNA <- toplistall_RNA %>% 
  mutate(logFC = logFC*(-1))

RNA_expresed_genes <- toplistall_RNA %>% 
  # dplyr::filter(adj.P.Val <0.05) %>% 
   mutate(expression = if_else(logFC<0,"down","up")) %>% 
  dplyr::select(ENTREZID,SYMBOL,expression) %>% 
  # dplyr::select(ENTREZID,SYMBOL) %>% 
  unique(.)  
RNA_expresed_genes_DE <- toplistall_RNA %>% 
  dplyr::filter(adj.P.Val <0.05) %>%
  mutate(expression = if_else(logFC<0,"down","up")) %>% 
  dplyr::select(ENTREZID,SYMBOL,expression) %>% 
  unique(.) 



```

### Enrichment all peaks to NR peaks
```{r simmotif, message=FALSE, warning=FALSE}
xstreme_EAR_v_NR %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  kable(., caption = "Enriched motifs in EAR v NR") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")


xstreme_ESR_v_NR %>% 
 dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
   arrange(.,EVALUE) %>% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  kable(., caption = "Enriched motifs in ESR v NR") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")

xstreme_LR_v_NR %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  kable(., caption = "Enriched motifs  (EVALUE < 0.05) in LR v NR") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
### selecting just the list of names and checking for expression


gene_list_SEA <- xstreme_EAR_v_NR %>% 
 dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(MCR="EAR") %>% 
  bind_rows(xstreme_ESR_v_NR %>% 
 dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
 mutate(MCR="ESR")) %>% 
  bind_rows(xstreme_LR_v_NR %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
    mutate(MCR="LR")) %>% 
  dplyr::filter(!is.na(NAME)) %>%  
  mutate(NAME= gsub("[()]","",NAME)) %>% 
 distinct(NAME)
  
  
```

#### After running SEA only on each data set
This was using all peaks but allowing SEA to be run using random method as background.
motif combinations by similarity were not applied.
```{r SEA-checks notthe same as sea_EAR!}

EAR_sea <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/EAR_sea.tsv", 
    delim = "\t", escape_double = FALSE, 
   trim_ws = TRUE)
ESR_sea <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/ESR_sea.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
LR_sea <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/LR_sea.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)


# length(intersect(EAR_sea$ID,ESR_sea$ID))
# length(unique(EAR_sea$ID))
# length(unique(ESR_sea$ID))
# length(unique(LR_sea$ID))
# head(EAR_sea,n=20)
EAR_sea %>% 
  slice_head(n=20) %>% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
  geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+
  ggtitle("EAR SEA top 20 motifs")
ESR_sea %>% 
  slice_head(n=20) %>% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
   geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+
   ggtitle("ESR SEA top 20 motifs")

LR_sea %>% 
  slice_head(n=20) %>% 
  ggplot(., aes(y=reorder(ALT_ID,-LOG_QVALUE)))+
   geom_col(aes(x=(-log10(QVALUE))))+
  geom_point(aes(x=`TP%`))+ 
  ggtitle("LR SEA top 20 motifs")


```


Here was me just focusing on motifs that were transcription factor binding motifs only
```{r lists of genes and motifs}


motifgenes_EAR_v_NR <- xstreme_EAR_v_NR %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SITES,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  dplyr::filter(., EVALUE<0.05) %>% 
  mutate(motif_name= gsub("[()]","",NAME)) %>% 
   dplyr::select(SIM_MOTIF,motif_name,SITES) %>% 
  unique()

motifgenes_ESR_v_NR <- xstreme_ESR_v_NR %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SITES,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  dplyr::filter(., EVALUE<0.05) %>% 
  mutate(motif_name= gsub("[()]","",NAME)) %>% 
   dplyr::select(SIM_MOTIF,motif_name,SITES) %>% 
  unique()


motifgenes_LR_v_NR <- xstreme_LR_v_NR %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SITES,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  dplyr::filter(., EVALUE<0.05) %>% 
  mutate(motif_name= gsub("[()]","",NAME)) %>% 
   dplyr::select(SIM_MOTIF,motif_name) %>% 
  unique()

gene_list_SEA %>%
  mutate(express_RNA=if_else(NAME %in%  RNA_expresed_genes$SYMBOL,"exp","not_exp"),DE_RNA=if_else(NAME %in% RNA_expresed_genes_DE$SYMBOL,"DE","notDE")) %>%
  dplyr::filter(express_RNA != "not_exp") %>%
  rowid_to_column("row_name") %>%
  kable(., caption= "Enriched motifs annotated with RNA expression status") %>%
    kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "1000px")


  
```

```{r motif exploring, echo=TRUE}
EAR_passing_sequences <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/EAR_passing_sequences.tsv", 
    delim = "\t", escape_double = FALSE, 
    col_types = cols(motif_DB = col_skip()), 
    trim_ws = TRUE)

motif_EAR_list <- EAR_passing_sequences %>% 
   distinct(motif_ID, .keep_all = TRUE) %>% 
  na.omit() %>% 
  dplyr::select(motif_ID, motif_ALT_ID)
            
expr_EAR_tfs <- data.frame(EAR_TFs=intersect(motif_EAR_list$motif_ALT_ID, unique(RNA_expresed_genes$SYMBOL))) 

ESR_passing_sequences <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/ESR_passing_sequences.tsv", 
    delim = "\t", escape_double = FALSE, 
    col_types = cols(motif_DB = col_skip()), 
    trim_ws = TRUE)
motif_ESR_list <- ESR_passing_sequences %>% 
   distinct(motif_ID, .keep_all = TRUE) %>% 
  na.omit() %>% 
  dplyr::select(motif_ID, motif_ALT_ID)


expr_ESR_tfs <- data.frame(ESR_TFs=intersect(motif_ESR_list$motif_ALT_ID, unique(RNA_expresed_genes$SYMBOL))) 

intersect(expr_ESR_tfs$ESR_TFs, expr_EAR_tfs$EAR_TFs)


LR_passing_sequences <- read_delim("~/Ward Lab/Cardiotoxicity/ATAC-folder/motif enrichment/just_the MRCs/LR_passing_sequences.tsv", 
    delim = "\t", escape_double = FALSE, 
    col_types = cols(motif_DB = col_skip()), 
    trim_ws = TRUE)
motif_LR_list <- LR_passing_sequences %>% 
   distinct(motif_ID, .keep_all = TRUE) %>% 
  na.omit() %>% 
  dplyr::select(motif_ID, motif_ALT_ID)


expr_LR_tfs <- data.frame(LR_TFs=intersect(motif_LR_list$motif_ALT_ID, unique(RNA_expresed_genes$SYMBOL))) 

# ggVennDiagram(list(motif_EAR_list$motif_ID, motif_ESR_list$motif_ID, motif_LR_list$motif_ID))


# ggVennDiagram::ggVennDiagram(list(expr_EAR_tfs,expr_ESR_tfs, expr_LR_tfs))
```

From SEA of EAR peaks:

I obtained a list of all tf motifs that were found enriched based on shuffled sequences of the peak sequences.  The EAR cluster has a total of `r length(EAR_df$seqnames)` peaks with a minimum width of `r min(EAR_df$width)` bps and a maximum width of `r max(EAR_df$width)` bps.  The average width and median width are `r mean(EAR_df$width)` and `r median(EAR_df$width)` bps respectively. There were a total of `r length(motif_EAR_list$motif_ID)` unique motifs found. Of those motifs found, `r count(expr_EAR_tfs)` TFs were expressed in my data.   


From SEA of ESR peaks:

I obtained a list of all tf motifs that were found enriched based on shuffled sequences of the peak sequences.  The ESR cluster has a total of `r length(ESR_df$seqnames)` peaks with a minimum width of `r min(ESR_df$width)` bps and a maximum width of `r max(ESR_df$width)` bps.  The average width and median width are `r mean(ESR_df$width)` and `r median(ESR_df$width)` bps respectively. There were a total of `r length(motif_ESR_list$motif_ID)` unique motifs found. Of those motifs found, `r count(expr_ESR_tfs)` TFs were expressed in my data.   



From SEA of LR peaks:

I obtained a list of all tf motifs that were found enriched based on shuffled sequences of the peak sequences.  The ESR cluster has a total of `r length(LR_df$seqnames)` peaks with a minimum width of `r min(LR_df$width)` bps and a maximum width of `r max(LR_df$width)` bps.  The average width and median width are `r mean(LR_df$width)` and `r median(LR_df$width)` bps respectively. There were a total of `r length(motif_LR_list$motif_ID)` unique motifs found. Of those motifs found, `r count(expr_LR_tfs)` TFs were expressed in my data.   

The EAR and ESR sets share `r length(intersect(motif_ESR_list$motif_ID, motif_EAR_list$motif_ID))` motifs out of a total of `r length(union(motif_EAR_list$motif_ID, motif_ESR_list$motif_ID))` total motifs between them.

### All enriched motifs found


This area was for all motifs found in Xtreme analysis using NR peaks as background.
#### EAR

```{r enriched motifs EAR}

xstreme_EAR_v_NR %>% 
  left_join(., sea_EAR, by= c("ALT_ID"="ALT_ID","CONSENSUS"="CONSENSUS", "ID"="ID")) %>% 
  dplyr::select(CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%')%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  arrange(.,EVALUE.x) %>%
 # dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3))))#%>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="EAR") %>% 
  mutate(motif_name=if_else(is.na(motif_name),ID,motif_name)) %>% 
# dplyr::filter(!is.na(motif_name)) %>% 
  # group_by(mrc) %>% 
   # dplyr::arrange(.,desc(log(EVALUE.y), base = 10)) %>% 
ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill="pink2") +
   geom_point(aes(x=`TP%`), size =5)+
   geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(sec.axis = dup_axis(name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle("Early-acute response peaks")
 
```
#### ESR
```{r enriched motifs ESR}
xstreme_ESR_v_NR %>% 
   left_join(., sea_ESR, by= c("ALT_ID"="ALT_ID","CONSENSUS"="CONSENSUS", "ID"="ID"))%>% 
  dplyr::select(CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%')%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  arrange(.,EVALUE.x) %>%
 # dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3))))#%>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="EAR") %>% 
  mutate(motif_name=if_else(is.na(motif_name),ID,motif_name)) %>% 
# dplyr::filter(!is.na(motif_name)) %>% 
  # group_by(mrc) %>% 
ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill="lightgreen") +
   geom_point(aes(x=`TP%`), size =5)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  # geom_text(aes())
  scale_x_continuous(sec.axis = dup_axis(name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle("Early-sustained response peaks")

```

#### LR

```{r enriched motifs LR}
xstreme_LR_v_NR %>% 
    left_join(., sea_LR, by= c("ALT_ID"="ALT_ID", "ID"="ID")) %>% 
  dplyr::select(CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%')%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  arrange(.,EVALUE.x) %>%
 # dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3))))#%>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
   group_by(NAME) %>% 
  mutate(RowGroup=cumsum(NAME != 0)) %>% 
  ungroup() %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="LR") %>% 
   mutate(motif_name=if_else(is.na(motif_name),ID,if_else(motif_name=="ZNF384",paste(motif_name, RowGroup, sep="_"),motif_name)))%>%
 
  mutate(`TP%`=as.numeric(`TP%`)) %>%
# dplyr::filter(!is.na(motif_name)) %>% 
  # group_by(mrc) %>% 
ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill="cyan") +
    geom_point(aes(x=`TP%`), size =5)+
   # geom_line(aes(x=TP, group=mrc))+
  # geom_text(aes())
  scale_x_continuous(sec.axis = dup_axis(name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle("Late response peaks")




```

### Filtering using Fold Change values

This section is for applying hyper and hypo accessibility to each response cluster.  
Basic first attempt is using DOX24 hour DAR logFC to split each response cluster into hyper- accessible (logFC >0) and hypo-accessible groups.  For hyper-accessible I will append a **U** at the end of the response cluster (eg EAR**U**) and for hypo-accessible I will append a **D** at the end of the response cluster name(eg EAR**D**).

```{r creating U and D}
### need to load Toplistall aTAC data.

toplist_full_ATAC <- readRDS("data/DEG_toplist_sep_n45.RDS")

```


## motif and peak section (incomplete)


```{r loading essential files for peak tests}

# ESR_peaks_resized = resize(ESR_df_gr, width = 2000, fix='center')
# EAR_peaks_resized = resize(EAR_df_gr, width = 2000, fix='center')
# LR_peaks_resized = resize(LR_df_gr, width = 2000, fix='center')
# NR_peaks_resized = resize(NR_df_gr, width = 2000, fix='center')

ESR_peaks_resized_250 = resize(ESR_df_gr, width = 500, fix='center')
EAR_peaks_resized_250 = resize(EAR_df_gr, width = 500, fix='center')
LR_peaks_resized_250 = resize(LR_df_gr, width = 500, fix='center')
NR_peaks_resized_250 = resize(NR_df_gr, width =500, fix='center')

seq_ESR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_peaks_resized_250)
seq_EAR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_peaks_resized_250)
seq_LR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_peaks_resized_250)
seq_NR_250 = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_peaks_resized_250)

# 
# ESR_peaks_resized_500 = resize(ESR_df_gr, width = 1000, fix='center')
# EAR_peaks_resized_500 = resize(EAR_df_gr, width = 1000, fix='center')
# LR_peaks_resized_500 = resize(LR_df_gr, width = 1000, fix='center')
# NR_peaks_resized_500 = resize(NR_df_gr, width = 1000, fix='center')
# ESR_peaks_resized_site = resize(ESR_df_gr,  fix='center')

# seq_ESR = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_peaks_resized)
# seq_EAR = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_peaks_resized)
# seq_LR = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_peaks_resized)
# seq_NR = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_peaks_resized)
# 
# seq_ESR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_peaks_resized_500)
# seq_EAR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_peaks_resized_500)
# seq_LR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_peaks_resized_500)
# seq_NR_500 = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_peaks_resized_500)
# 
# seq_ESR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, ESR_df_gr)
# seq_EAR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, EAR_df_gr)
# seq_LR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, LR_df_gr)
# seq_NR_norm = getSeq(BSgenome.Hsapiens.UCSC.hg38, NR_df_gr)


```

#### BACH2


```{r BACH2 motif, eval=FALSE, include=FALSE}
# motifs_BACH2 = query(query(MotifDb, 'Hsapiens'), 'BACH2')
# BACH2_motif <- motifs_BACH2$`Hsapiens-jaspar2022-BACH2-MA1101.2`
# # 
# # 
# BACH2_pwm = PWMatrix(
#     ID = 'BACH2',
#     profileMatrix = BACH2_motif)
# 
# 
# hits_ESR_250 = searchSeq(BACH2_pwm, seq_ESR_250, min.score="80%", strand="*")
# hits_df_ESR_250 <- as(hits_ESR_250,"data.frame")
# hits_ESR_250 <- as.data.frame(hits_ESR_250)
# saveRDS(hits_ESR_250,"data/hits_files/BACH2_hits_ESR_250.RDS")
hits_ESR_BACH2_250 <- readRDS("data/hits_files/BACH2_hits_ESR_250.RDS")

# 
# 
# 
# 
# hits_EAR_250 <- searchSeq(BACH2_pwm, seq_EAR_250, min.score="80%", strand="*")
# hits_EAR_250 <- as.data.frame(hits_EAR_250)
# saveRDS(hits_EAR_250,"data/hits_files/BACH2_hits_EAR_250.RDS")
# 
hits_EAR_BACH2_250 <- readRDS("data/hits_files/BACH2_hits_EAR_250.RDS")
# 
# hits_LR_250 <- searchSeq(BACH2_pwm, seq_LR_250, min.score="80%", strand="*")
# hits_LR_250 <- as.data.frame(hits_LR_250)
# saveRDS(hits_LR_250,"data/hits_files/BACH2_hits_LR_250.RDS")
hits_LR_BACH2_250 <- readRDS("data/hits_files/BACH2_hits_LR_250.RDS")

# 
# hits_NR_250 <- searchSeq(BACH2_pwm, seq_NR_250, min.score="80%", strand="*")
# hits_NR_250 <- as.data.frame(hits_NR_250)
# saveRDS(hits_NR_250,"data/hits_files/BACH2_hits_NR_250.RDS")
hits_NR_BACH2_250 <- readRDS("data/hits_files/BACH2_hits_NR_250.RDS")


hits_NR_BACH2_250 %>% 
  mutate(position=start-250) %>% 
  mutate(id_mrc="NR") %>% 
  rbind(.,(hits_LR_BACH2_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="LR"))) %>% 
   rbind(.,(hits_ESR_BACH2_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="ESR"))) %>% 
  rbind(.,(hits_EAR_BACH2_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="EAR"))) %>% 
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the BACH2 motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around BACH2 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))

```
#### ZNF384
enriched in EAR, ESR, LR
```{r ZNF384}

# motifs_ZNF384 = query(query(MotifDb, 'Hsapiens'), 'ZNF384')
# ZNF384_motif <- motifs_ZNF384$`Hsapiens-jaspar2022-ZNF384-MA1125.1`
# # 
# # 
# ZNF384_pwm = PWMatrix(
#     ID = 'ZNF384',
#     profileMatrix = ZNF384_motif)
hits_ESR_ZNF384 <- readRDS("data/hits_files/ZNF384_hits_ESR.RDS")
# hits = searchSeq(ZNF384_pwm, seq_ESR, min.score="80%", strand="*")
# hits_ESR <- hits
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/ZNF384_hits_ESR.RDS")
# hits_ESR_ZNF384 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the ZNF384 peaks') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("ESR: ZNF384 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_ZNF384 <- readRDS("data/hits_files/ZNF384_hits_EAR.RDS")
# hits_EAR <- searchSeq(ZNF384_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/ZNF384_hits_EAR.RDS")

# hits_EAR_ZNF384 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the ZNF384 peaks') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("EAR: ZNF384 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_LR_ZNF384 <- readRDS("data/hits_files/ZNF384_hits_LR.RDS")
# hits_LR <- searchSeq(ZNF384_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/ZNF384_hits_LR.RDS")

# hits_LR_ZNF384 %>% 
#   mutate(position=start-1000) %>% 
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the ZNF384 peaks') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: ZNF384 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))
hits_NR_ZNF384 <- readRDS("data/hits_files/ZNF384_hits_NR.RDS")
# hits_NR <- searchSeq(ZNF384_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/ZNF384_hits_NR.RDS")

hits_NR_ZNF384 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_ZNF384 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_ZNF384 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_ZNF384 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_ZNF384, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the ZNF384 motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around ZNF384 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))




```



#### ZKSCAN3
enriched in EAR
```{r ZKSCAN3}
# 
# motifs_ZKSCAN3 = query(query(MotifDb, 'Hsapiens'), 'ZKSCAN3')
# ZKSCAN3_motif <- motifs_ZKSCAN3$`Hsapiens-jaspar2022-ZKSCAN3-MA1973.1`
# 
# 
# ZKSCAN3_pwm = PWMatrix(
#     ID = 'ZKSCAN3',
#     profileMatrix = ZKSCAN3_motif)

# hits_ESR_250 = searchSeq(ZKSCAN3_pwm, seq_ESR_250, min.score="80%", strand="*")
# hits_df_ESR_250 <- as(hits_ESR_250,"data.frame")
# hits_ESR_250 <- as.data.frame(hits_ESR_250)
# saveRDS(hits_ESR_250,"data/hits_files/ZKSCAN3_hits_ESR_250.RDS")
hits_ESR_ZKSCAN3_250 <- readRDS("data/hits_files/ZKSCAN3_hits_ESR_250.RDS")



# 
# 
# hits_EAR_250 <- searchSeq(ZKSCAN3_pwm, seq_EAR_250, min.score="80%", strand="*")
# hits_EAR_250 <- as.data.frame(hits_EAR_250)
# saveRDS(hits_EAR_250,"data/hits_files/ZKSCAN3_hits_EAR_250.RDS")
# 
hits_EAR_ZKSCAN3_250 <- readRDS("data/hits_files/ZKSCAN3_hits_EAR_250.RDS")
# 
# hits_LR_250 <- searchSeq(ZKSCAN3_pwm, seq_LR_250, min.score="80%", strand="*")
# hits_LR_250 <- as.data.frame(hits_LR_250)
# saveRDS(hits_LR_250,"data/hits_files/ZKSCAN3_hits_LR_250.RDS")
hits_LR_ZKSCAN3_250 <- readRDS("data/hits_files/ZKSCAN3_hits_LR_250.RDS")

# 
# hits_NR_250 <- searchSeq(ZKSCAN3_pwm, seq_NR_250, min.score="80%", strand="*")
# hits_NR_250 <- as.data.frame(hits_NR_250)
# saveRDS(hits_NR_250,"data/hits_files/ZKSCAN3_hits_NR_250.RDS")
hits_NR_ZKSCAN3_250 <- readRDS("data/hits_files/ZKSCAN3_hits_NR_250.RDS")


hits_NR_ZKSCAN3_250 %>% 
  mutate(position=start-250) %>% 
  mutate(id_mrc="NR") %>% 
  rbind(.,(hits_LR_ZKSCAN3_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="LR"))) %>% 
   rbind(.,(hits_ESR_ZKSCAN3_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="ESR"))) %>% 
  rbind(.,(hits_EAR_ZKSCAN3_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="EAR"))) %>% 
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the ZKSCAN3 motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around ZKSCAN3 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))

```




#### PITX1
 enriched in EAR and ESR
```{r PITX1}
# 
# motifs_PITX1 = query(query(MotifDb, 'Hsapiens'), 'PITX1')
#     PITX1_motif <- motifs_PITX1$`Hsapiens-jaspar2022-PITX1-MA0682.2`


# PITX1_pwm = PWMatrix(
#     ID = 'PITX1',
#     profileMatrix = PITX1_motif)
hits_ESR_PITX1 <- readRDS("data/hits_files/PITX1_hits_ESR.RDS")
# hits_ESR = searchSeq(PITX1_pwm, seq_ESR, min.score="80%", strand="*")
# hits_ESR <- hits
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/PITX1_hits_ESR.RDS")
hits_ESR_PITX1 %>%
  mutate(position=start-1000) %>%
  ggplot(.,aes(position))+
  geom_density(size=2) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the PITX1 peaks') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("ESR: PITX1 binding")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))

hits_EAR_PITX1 <- readRDS("data/hits_files/PITX1_hits_EAR.RDS")
# hits_EAR <- searchSeq(PITX1_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/PITX1_hits_EAR.RDS")

hits_EAR_PITX1 %>%
  mutate(position=start-1000) %>%
  ggplot(.,aes(position))+
  geom_density(size=2) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the PITX1 peaks') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("EAR: PITX1 binding")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))

hits_LR_PITX1 <- readRDS("data/hits_files/PITX1_hits_LR.RDS")
# hits_LR <- searchSeq(PITX1_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/PITX1_hits_LR.RDS")

hits_LR_PITX1 %>%
  mutate(position=start-1000) %>%
  ggplot(.,aes(position))+
  geom_density(size=2) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the PITX1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("LR: PITX1 binding")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))
hits_NR_PITX1 <- readRDS("data/hits_files/PITX1_hits_NR.RDS")
# hits_NR <- searchSeq(PITX1_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/PITX1_hits_NR.RDS")

hits_NR_PITX1 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_PITX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_PITX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_PITX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_PITX1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the PITX1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around PITX1 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))




```


#### TP53
not enriched, just curious

```{r TP53}
# # 
# motifs_TP53 = query(query(MotifDb, 'Hsapiens'), 'TP53')
#     TP53_motif <- motifs_TP53$`Hsapiens-JASPAR_CORE-TP53-MA0106.1`
# 
# 
# TP53_pwm = PWMatrix(
#     ID = 'TP53',
#     profileMatrix = TP53_motif)
hits_ESR_TP53 <- readRDS("data/hits_files/TP53_hits_ESR.RDS")
# hits_ESR = searchSeq(TP53_pwm, seq_ESR, min.score="80%", strand="*")
# hits_ESR <- hits
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/TP53_hits_ESR.RDS")
# hits_ESR_TP53 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the TP53  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("ESR: TP53 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_TP53 <- readRDS("data/hits_files/TP53_hits_EAR.RDS")
# hits_EAR <- searchSeq(TP53_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/TP53_hits_EAR.RDS")

# hits_EAR_TP53 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the TP53  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("EAR: TP53 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_LR_TP53 <- readRDS("data/hits_files/TP53_hits_LR.RDS")
# hits_LR <- searchSeq(TP53_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/TP53_hits_LR.RDS")

# hits_LR_TP53 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the TP53  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: TP53 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_NR_TP53 <- readRDS("data/hits_files/TP53_hits_NR.RDS")
# hits_NR <- searchSeq(TP53_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/TP53_hits_NR.RDS")

hits_NR_TP53 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_TP53 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_TP53 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_TP53 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_TP53, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the TP53 motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around TP53 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))
# 



```

#### NRF1
```{r NRF1}
# 
# motifs_NRF1 = query(query(MotifDb, 'Hsapiens'), 'NRF1')
#     NRF1_motif <- motifs_NRF1$`Hsapiens-jaspar2018-NRF1-MA0506.1`
# # 
# # 
# NRF1_pwm = PWMatrix(
#     ID = 'NRF1',
#     profileMatrix = NRF1_motif)
hits_ESR_NRF1 <- readRDS("data/hits_files/NRF1_hits_ESR.RDS")
# hits_ESR = searchSeq(NRF1_pwm, seq_ESR, min.score="80%", strand="*")
# hits_ESR <- hits
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/NRF1_hits_ESR.RDS")
# hits_ESR_NRF1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the NRF1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("ESR: NRF1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_NRF1 <- readRDS("data/hits_files/NRF1_hits_EAR.RDS")
# hits_EAR <- searchSeq(NRF1_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/NRF1_hits_EAR.RDS")

# hits_EAR_NRF1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the NRF1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("EAR: NRF1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_NRF1 <- readRDS("data/hits_files/NRF1_hits_LR.RDS")
# hits_LR <- searchSeq(NRF1_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/NRF1_hits_LR.RDS")

# hits_LR_NRF1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the NRF1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: NRF1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_NR_NRF1 <- readRDS("data/hits_files/NRF1_hits_NR.RDS")
# hits_NR <- searchSeq(NRF1_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/NRF1_hits_NR.RDS")

hits_NR_NRF1 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_NRF1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_NRF1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_NRF1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_NRF1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the NRF1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around NRF1 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))




```

#### NFIC_TLX1

```{r NFIC_TLX1}
# 
# motifs_NFIC_TLX1 = query(query(MotifDb, 'Hsapiens'), 'NFIC::TLX1')
#     NFIC_TLX1_motif <- motifs_NFIC_TLX1$`Hsapiens-jaspar2022-NFIC::TLX1-MA0119.1`
# #
# # # 
# NFIC_TLX1_pwm = PWMatrix(
#     ID = 'NFIC_TLX1',
#     profileMatrix = NFIC_TLX1_motif)
hits_ESR_NFIC_TLX1 <- readRDS("data/hits_files/NFIC_TLX1_hits_ESR.RDS")
# hits_ESR = searchSeq(NFIC_TLX1_pwm, seq_ESR, min.score="80%", strand="*")
# 
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/NFIC_TLX1_hits_ESR.RDS")
# hits_ESR_NFIC_TLX1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the NFIC_TLX1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("ESR: NFIC_TLX1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_NFIC_TLX1 <- readRDS("data/hits_files/NFIC_TLX1_hits_EAR.RDS")
# hits_EAR <- searchSeq(NFIC_TLX1_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/NFIC_TLX1_hits_EAR.RDS")

# hits_EAR_NFIC_TLX1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the NFIC_TLX1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("EAR: NFIC_TLX1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_NFIC_TLX1 <- readRDS("data/hits_files/NFIC_TLX1_hits_LR.RDS")
# hits_LR <- searchSeq(NFIC_TLX1_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/NFIC_TLX1_hits_LR.RDS")

# hits_LR_NFIC_TLX1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the NFIC_TLX1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: NFIC_TLX1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_NR_NFIC_TLX1 <- readRDS("data/hits_files/NFIC_TLX1_hits_NR.RDS")
# hits_NR <- searchSeq(NFIC_TLX1_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/NFIC_TLX1_hits_NR.RDS")

hits_NR_NFIC_TLX1 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_NFIC_TLX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_NFIC_TLX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_NFIC_TLX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_NFIC_TLX1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the NFIC_TLX1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around NFIC_TLX1 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))#+
  # coord_cartesian(xlim= c(-500,500))




```





#### PRDM1
```{r PRDM1}
# 
# motifs_PRDM1 = query(query(MotifDb, 'Hsapiens'), 'PRDM1')
#     PRDM1_motif <- motifs_PRDM1$`Hsapiens-jaspar2022-PRDM1-MA0508.3`
# #
# PRDM1_pwm = PWMatrix(
#     ID = 'PRDM1',
#     profileMatrix = PRDM1_motif)
hits_ESR_PRDM1 <- readRDS("data/hits_files/PRDM1_hits_ESR.RDS")
# hits_ESR = searchSeq(PRDM1_pwm, seq_ESR, min.score="80%", strand="*")
# 
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/PRDM1_hits_ESR.RDS")

hits_EAR_PRDM1 <- readRDS("data/hits_files/PRDM1_hits_EAR.RDS")
# hits_EAR <- searchSeq(PRDM1_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/PRDM1_hits_EAR.RDS")


hits_LR_PRDM1 <- readRDS("data/hits_files/PRDM1_hits_LR.RDS")
# hits_LR <- searchSeq(PRDM1_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/PRDM1_hits_LR.RDS")



hits_NR_PRDM1 <- readRDS("data/hits_files/PRDM1_hits_NR.RDS")
# hits_NR <- searchSeq(PRDM1_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/PRDM1_hits_NR.RDS")

hits_NR_PRDM1 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_PRDM1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_PRDM1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_PRDM1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_PRDM1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the PRDM1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around PRDM1 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))
# +
  # coord_cartesian(xlim= c(-500,500))




```
#### TBX1

```{r TBX1}
# 
# motifs_TBX1 = query(query(MotifDb, 'Hsapiens'), 'TBX1')
#     TBX1_motif <- motifs_TBX1$`Hsapiens-jaspar2022-TBX1-MA0805.1`
# #
# #
# TBX1_pwm = PWMatrix(
#     ID = 'TBX1',
#     profileMatrix = TBX1_motif)
hits_ESR_TBX1 <- readRDS("data/hits_files/TBX1_hits_ESR.RDS")
# hits_ESR = searchSeq(TBX1_pwm, seq_ESR, min.score="80%", strand="*")
# 
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/TBX1_hits_ESR.RDS")
# hits_ESR_TBX1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the TBX1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("ESR: TBX1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_TBX1 <- readRDS("data/hits_files/TBX1_hits_EAR.RDS")
# hits_EAR <- searchSeq(TBX1_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/TBX1_hits_EAR.RDS")

# hits_EAR_TBX1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the TBX1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("EAR: TBX1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_TBX1 <- readRDS("data/hits_files/TBX1_hits_LR.RDS")
# hits_LR <- searchSeq(TBX1_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/TBX1_hits_LR.RDS")

# hits_LR_TBX1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the TBX1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: TBX1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_NR_TBX1 <- readRDS("data/hits_files/TBX1_hits_NR.RDS")
# hits_NR <- searchSeq(TBX1_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/TBX1_hits_NR.RDS")

hits_NR_TBX1 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_TBX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_TBX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_TBX1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_TBX1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the TBX1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around TBX1 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))+
  coord_cartesian(xlim= c(-500,500))

```

#### GLI1
```{r GLI1}
# 
# motifs_GLI1 = query(query(MotifDb, 'Hsapiens'), 'GLI1')
#     GLI1_motif <- motifs_GLI1$`Hsapiens-HOCOMOCOv10-GLI1_HUMAN.H10MO.C`
# # # 
# # # 
# GLI1_pwm = PWMatrix(
#     ID = 'GLI1',
#     profileMatrix = GLI1_motif)
hits_ESR_GLI1 <- readRDS("data/hits_files/GLI1_hits_ESR.RDS")
# hits_ESR = searchSeq(GLI1_pwm, seq_ESR, min.score="80%", strand="*")
# 
# hits_ESR <- as.data.frame(hits_ESR)
# saveRDS(hits_ESR,"data/hits_files/GLI1_hits_ESR.RDS")
# hits_ESR_GLI1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the GLI1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("ESR: GLI1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_EAR_GLI1 <- readRDS("data/hits_files/GLI1_hits_EAR.RDS")
# hits_EAR <- searchSeq(GLI1_pwm, seq_EAR, min.score="80%", strand="*")
# hits_EAR <- as.data.frame(hits_EAR)
# saveRDS(hits_EAR,"data/hits_files/GLI1_hits_EAR.RDS")

# hits_EAR_GLI1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the GLI1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("EAR: GLI1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))
# 
hits_LR_GLI1 <- readRDS("data/hits_files/GLI1_hits_LR.RDS")
# hits_LR <- searchSeq(GLI1_pwm, seq_LR, min.score="80%", strand="*")
# hits_LR <- as.data.frame(hits_LR)
# saveRDS(hits_LR,"data/hits_files/GLI1_hits_LR.RDS")

# hits_LR_GLI1 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the GLI1  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: GLI1 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

hits_NR_GLI1 <- readRDS("data/hits_files/GLI1_hits_NR.RDS")
# hits_NR <- searchSeq(GLI1_pwm, seq_NR, min.score="80%", strand="*")
# hits_NR <- as.data.frame(hits_NR)
# saveRDS(hits_NR,"data/hits_files/GLI1_hits_NR.RDS")

hits_NR_GLI1 %>%
  mutate(position=start-1000) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_GLI1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_GLI1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_GLI1 %>% mutate(position=start-1000) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_GLI1, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the GLI1  motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("Chromatin accessibility around GLI1 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))+
  coord_cartesian(xlim= c(-500,500))


# 
# sites_LR_v_NR %>% 
#   dplyr::filter(motif_ALT_ID=="STREME-1", seq_ID=="chr3") %>% 
#   # mutate(position)
#   ggplot(., aes(site_Score))+
#   geom_density()

```

#### ZNF460
```{r ZNF460}
# # 
# motifs_ZNF460 = query(query(MotifDb, 'Hsapiens'), 'ZNF460')
#     ZNF460_motif <- motifs_ZNF460$`Hsapiens-jaspar2022-ZNF460-MA1596.1`
# 
# ZNF460_pwm = PWMatrix(
#     ID = 'ZNF460',
#     profileMatrix = ZNF460_motif)

# hits_ESR_250 = searchSeq(ZNF460_pwm, seq_ESR_250, min.score="80%", strand="*")
# hits_ESR_250 <- as.data.frame(hits_ESR_250)
# saveRDS(hits_ESR_250,"data/hits_files/ZNF460_hits_ESR_250.RDS")
hits_ESR_ZNF460_250 <- readRDS("data/hits_files/ZNF460_hits_ESR_250.RDS")

# 
# hits_EAR_250 <- searchSeq(ZNF460_pwm, seq_EAR_250, min.score="80%", strand="*")
# hits_EAR_250 <- as.data.frame(hits_EAR_250)
# saveRDS(hits_EAR_250,"data/hits_files/ZNF460_hits_EAR_250.RDS")
hits_EAR_ZNF460_250 <- readRDS("data/hits_files/ZNF460_hits_EAR_250.RDS")



# hits_LR_250 <- searchSeq(ZNF460_pwm, seq_LR_250, min.score="80%", strand="*")
# hits_LR_250 <- as.data.frame(hits_LR_250)
# saveRDS(hits_LR_250,"data/hits_files/ZNF460_hits_LR_250.RDS")
hits_LR_ZNF460_250 <- readRDS("data/hits_files/ZNF460_hits_LR_250.RDS")
# hits_LR_ZNF460 %>%
#   mutate(position=start-1000) %>%
#   ggplot(.,aes(position))+
#   geom_density(size=2) +
#   theme_bw() +
#   geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
#   xlab('Position around the ZNF460  motif') +
#   ylab('Per position percentage\nof motif occurrence') +
#   ggtitle("LR: ZNF460 binding")+
#   theme(
#     axis.text = element_text(size=10, face='bold'),
#     axis.title = element_text(size=14,face="bold"),
#     plot.title = element_text(hjust = 0.5))

# 
# hits_NR_250 <- searchSeq(ZNF460_pwm, seq_NR_250, min.score="80%", strand="*")
# hits_NR_250 <- as.data.frame(hits_NR_250)
# saveRDS(hits_NR_250,"data/hits_files/ZNF460_hits_NR_250.RDS")
hits_NR_ZNF460_250 <- readRDS("data/hits_files/ZNF460_hits_NR_250.RDS")
hits_NR_ZNF460_250 %>%
  mutate(position=start-250) %>%
  mutate(id_mrc="NR") %>%
  rbind(.,(hits_LR_ZNF460_250 %>% mutate(position=start-250) %>%
  mutate(id_mrc="LR"))) %>%
   rbind(.,(hits_ESR_ZNF460_250 %>% mutate(position=start-250) %>%
  mutate(id_mrc="ESR"))) %>%
   rbind(.,(hits_EAR_ZNF460_250 %>% mutate(position=start-250) %>%
  mutate(id_mrc="EAR"))) %>%
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  # geom_density(data=hits_LR_ZNF460, aes(position))+
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', size=2) +
  xlab('Position around the ZNF460  motif') +
  ylab('ZNF460') +
  # ggtitle("Chromatin accessibility around ZNF460 binding site")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))
# +
  # coord_cartesian(xlim= c(-500,500))

```
#### MEF2A


```{r MEF2A motif, eval=FALSE, include=FALSE}
# motifs_MEF2A = query(query(MotifDb, 'Hsapiens'), 'MEF2A')
# MEF2A_motif <- motifs_MEF2A$`Hsapiens-jaspar2022-MEF2A-MA0052.4`
# 
# MEF2A_pwm = PWMatrix(
#     ID = 'MEF2A',
#     profileMatrix = MEF2A_motif)


# # 
# hits_ESR_250 = searchSeq(MEF2A_pwm, seq_ESR_250, min.score="80%", strand="*")
# hits_ESR_250 <- as.data.frame(hits_ESR_250)
# saveRDS(hits_ESR_250,"data/hits_files/MEF2A_hits_ESR_250.RDS")
hits_ESR_MEF2A_250 <- readRDS("data/hits_files/MEF2A_hits_ESR_250.RDS")
# 
# 
# hits_EAR_250 <- searchSeq(MEF2A_pwm, seq_EAR_250, min.score="80%", strand="*")
# hits_EAR_250 <- as.data.frame(hits_EAR_250)
# saveRDS(hits_EAR_250,"data/hits_files/MEF2A_hits_EAR_250.RDS")
# 
hits_EAR_MEF2A_250 <- readRDS("data/hits_files/MEF2A_hits_EAR_250.RDS")
# 
# hits_LR_250 <- searchSeq(MEF2A_pwm, seq_LR_250, min.score="80%", strand="*")
# hits_LR_250 <- as.data.frame(hits_LR_250)
# saveRDS(hits_LR_250,"data/hits_files/MEF2A_hits_LR_250.RDS")
hits_LR_MEF2A_250 <- readRDS("data/hits_files/MEF2A_hits_LR_250.RDS")

# 
# hits_NR_250 <- searchSeq(MEF2A_pwm, seq_NR_250, min.score="80%", strand="*")
# hits_NR_250 <- as.data.frame(hits_NR_250)
# saveRDS(hits_NR_250,"data/hits_files/MEF2A_hits_NR_250.RDS")

hits_NR_MEF2A_250 <- readRDS("data/hits_files/MEF2A_hits_NR_250.RDS")


hits_NR_MEF2A_250 %>% 
  mutate(position=start-250) %>% 
  mutate(id_mrc="NR") %>% 
  rbind(.,(hits_LR_MEF2A_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="LR"))) %>% 
   rbind(.,(hits_ESR_MEF2A_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="ESR"))) %>% 
  rbind(.,(hits_EAR_MEF2A_250 %>% mutate(position=start-250) %>% 
  mutate(id_mrc="EAR"))) %>% 
  ggplot(.,aes(position, color = id_mrc))+
  geom_density(size=1.5) +
  theme_bw() +
  geom_vline(xintercept = 0, linetype=2, color='red', linewidth=2) +
  xlab('Position around the MEF2A motif') +
  ylab('Per position percentage\nof motif occurrence') +
  ggtitle("MEF2A")+
  theme(
    axis.text = element_text(size=10, face='bold'),
    axis.title = element_text(size=14,face="bold"),
    plot.title = element_text(hjust = 0.5))

```







