---
title: "Jaspar_enrichment"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
		dev = c("png","pdf")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```
##### packages
```{r  package loading}
library(tidyverse)
library(cowplot)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(JASPAR2022)
library(TFBSTools)
library(MotifDb)
library(BSgenome.Hsapiens.UCSC.hg38)
library(data.table)

```

#### Data loading

```{r dataframe loading}

# 
lala <- read_delim("~/ATAC_meme_data/200bp/ESR_open_200xstreme/xstreme.tsv",
    delim = "\t", escape_double = FALSE,
    trim_ws = TRUE)

# saveRDS(xstreme_EAR_openSVA,"data/Final_four_data/xstreme/xstreme_EAR_openSVA.RDS")



# saveRDS(xstreme_EAR_close200,"data/Final_four_data/xstreme/xstreme_EAR_close200.RDS")
# saveRDS(xstreme_EAR_open200,"data/Final_four_data/xstreme/xstreme_EAR_open200.RDS")
# saveRDS(xstreme_ESR_open200,"data/Final_four_data/xstreme/xstreme_ESR_open200.RDS")
# saveRDS(xstreme_ESR_close200,"data/Final_four_data/xstreme/xstreme_ESR_close200.RDS")
# saveRDS(ESR_OC_xstreme,"data/Final_four_data/xstreme/ESR_OC_xstreme.RDS")
# saveRDS(ESR_C_xstreme,"data/Final_four_data/xstreme/ESR_OC_xstreme.RDS")
# saveRDS(xstreme_ESR_clop200,"data/Final_four_data/xstreme/xstreme_ESR_clop200.RDS")
# saveRDS(xstreme_LR_close200,"data/Final_four_data/xstreme/xstreme_LR_close200.RDS")
# saveRDS(xstreme_LR_open200,"data/Final_four_data/xstreme/xstreme_LR_open200.RDS")
# saveRDS(LR_open_10h_xstreme,"data/Final_four_data/xstreme/LR_open_10h_xstreme.RDS")
# saveRDS(xstreme_ESR_opcl200,"data/Final_four_data/xstreme/xstreme_ESR_opcl200.RDS")


EAR_close_xstreme <-
  readRDS("data/Final_four_data/xstreme/EAR_close_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
EAR_open_xstreme <- 
  readRDS("data/Final_four_data/xstreme/EAR_open_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_open_xstreme <- 
  readRDS("data/Final_four_data/xstreme/ESR_open_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_close_xstreme <- 
  readRDS("data/Final_four_data/xstreme/ESR_close_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_OC_xstreme <- 
  readRDS("data/Final_four_data/xstreme/ESR_OC_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_opcl_xstreme <- 
  readRDS("data/Final_four_data/xstreme/xstreme_ESR_opcl200.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_clop_xstreme <- 
  readRDS("data/Final_four_data/xstreme/xstreme_ESR_clop200.RDS")%>%
  slice_head(n = length(.$ID)-3)


LR_close_xstreme <-
  readRDS("data/Final_four_data/xstreme/LR_close_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
LR_open_xstreme <-
  readRDS("data/Final_four_data/xstreme/LR_open_10h_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)
LR_open_10h_xstreme <-
  readRDS("data/Final_four_data/xstreme/LR_open_10h_xstreme.RDS")%>%
  slice_head(n = length(.$ID)-3)

EAR_close_200xstreme <-
  readRDS("data/Final_four_data/xstreme/xstreme_EAR_close200.RDS")%>%
  slice_head(n = length(.$ID)-3)
EAR_open_200xstreme <- 
  readRDS("data/Final_four_data/xstreme/xstreme_EAR_open200.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_open_200xstreme <- 
  readRDS("data/Final_four_data/xstreme/xstreme_ESR_open200.RDS")%>%
  slice_head(n = length(.$ID)-3)

ESR_close_200xstreme <- 
  readRDS("data/Final_four_data/xstreme/xstreme_ESR_close200.RDS")%>%
  slice_head(n = length(.$ID)-3)
ESR_OC_xstreme <-
  readRDS("data/Final_four_data/xstreme/xstreme_LR_open200.RDS")%>%
  slice_head(n = length(.$ID)-3)
LR_close_200xstreme <-
  readRDS("data/Final_four_data/xstreme/xstreme_LR_close200.RDS")%>%
  slice_head(n = length(.$ID)-3)
LR_open_200xstreme <-
  readRDS("data/Final_four_data/xstreme/xstreme_LR_open200.RDS")%>%
  slice_head(n = length(.$ID)-3)
#### full sequence sea out
sea_EAR_open <- readRDS("data/Final_four_data/xstreme/sea_EAR_open.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_EAR_close <- readRDS("data/Final_four_data/xstreme/sea_EAR_close.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_close <- readRDS("data/Final_four_data/xstreme/sea_ESR_close.RDS")
sea_ESR_open <- readRDS("data/Final_four_data/xstreme/sea_ESR_open.RDS")#%>%
  # slice_head(n = length(.$ID)-3)
 
sea_ESR_OC <- readRDS("data/Final_four_data/xstreme/sea_ESR_OC.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_LR_open <- readRDS("data/Final_four_data/xstreme/sea_LR_open_10h.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_LR_close <- readRDS("data/Final_four_data/xstreme/sea_LR_close.RDS")%>%
  slice_head(n = length(.$ID)-3)

sea_LR_open <- readRDS("data/Final_four_data/xstreme/sea_LR_open_10h.RDS")%>%
  slice_head(n = length(.$ID)-3)


### sea out 200 bp sequences
# sea_ESR_clop200 <- read_delim("~/ATAC_meme_data/200bp/ESR_D_200xstreme/sea_out/sea.tsv",
#     delim = "\t", escape_double = FALSE,
#     trim_ws = TRUE)
# saveRDS(sea_ESR_clop200,"data/Final_four_data/xstreme/sea_ESR_clop_200.RDS")

### sea part2!

sea_EAR_open_200 <- readRDS("data/Final_four_data/xstreme/sea_EAR_open_200.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_EAR_close_200 <- readRDS("data/Final_four_data/xstreme/sea_EAR_close_200.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_close_200 <- readRDS("data/Final_four_data/xstreme/sea_ESR_close_200.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_open_200 <- readRDS("data/Final_four_data/xstreme/sea_ESR_open_200.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_opcl_200 <- readRDS("data/Final_four_data/xstreme/sea_ESR_opcl_200.RDS")%>%
  slice_head(n = length(.$ID)-3)

sea_ESR_clop_200 <- readRDS("data/Final_four_data/xstreme/sea_ESR_clop_200.RDS")%>%
  slice_head(n = length(.$ID)-3)

sea_LR_open_200 <- readRDS("data/Final_four_data/xstreme/sea_LR_open_200.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_LR_close_200 <- readRDS("data/Final_four_data/xstreme/sea_LR_close_200.RDS")%>%
  slice_head(n = length(.$ID)-3)


### sea part2!  This is for missing merged data in sea_disc_out 
# sea_ESR_clop_p2 <- read_delim("~/ATAC_meme_data/200bp/ESR_D_200xstreme/sea_disc_out/sea.tsv",
#     delim = "\t", escape_double = FALSE,
#     trim_ws = TRUE)
# saveRDS(sea_ESR_clop_p2,"data/Final_four_data/xstreme/sea_ESR_clop_200_p2.RDS")


sea_EAR_open_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_EAR_open_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_EAR_close_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_EAR_close_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_close_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_ESR_close_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_open_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_ESR_open_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_opcl_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_ESR_opcl_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_ESR_clop_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_ESR_clop_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)

sea_LR_open_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_LR_open_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)
sea_LR_close_200_p2 <- readRDS("data/Final_four_data/xstreme/sea_LR_close_200_p2.RDS")%>%
  slice_head(n = length(.$ID)-3)




peakAnnoList_ff_8motif <- readRDS("data/Final_four_data/peakAnnoList_ff_8motif.RDS")
# saveRDS(ESR_D,"data/Final_four_data/ESR_D.RDS") 
toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS")
###Because of how I applied the DEG system in RNA-seq analysis, the lFC is opposite of the 
###counts.   I did trt-veh instead of veh-trt.  therefore I need to multiply lfc by -1 to get t
###the right correlation.

toplistall_RNA <- toplistall_RNA %>%
  mutate(logFC = logFC*(-1))

RNA_expresed_genes <- toplistall_RNA %>%
  # dplyr::filter(adj.P.Val <0.05) %>%
   mutate(expression = if_else(logFC<0,"down","up")) %>%
  dplyr::select(ENTREZID,SYMBOL,expression) %>%
  # dplyr::select(ENTREZID,SYMBOL) %>%
  unique(.)
RNA_expresed_genes_DE <- toplistall_RNA %>%
  dplyr::filter(adj.P.Val <0.05) %>%
  mutate(expression = if_else(logFC<0,"down","up")) %>%
  dplyr::select(ENTREZID,SYMBOL,expression) %>%
  unique(.)



```

### Enrichment all peaks to NR peaks
#### EAR lists
```{r simmotif, message=FALSE, warning=FALSE}
# EAR_open_xstreme%>% 
#   dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
#   arrange(.,EVALUE) %>% 
#   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   kable(., caption = "Enriched motifs in EAR open v NR") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "500px")

EAR_open_200xstreme%>% 
  dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  kable(., caption = "Enriched motifs 200 bp in EAR open v NR") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")

```

```{r ear_close, message=FALSE, warning=FALSE}
# EAR_close_xstreme %>% 
#   dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
#   arrange(.,EVALUE) %>% 
#   # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   # dplyr::filter(., EVALUE<0.05) %>% 
#   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   kable(., caption = "Enriched motifs in EAR close") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "500px")
EAR_close_200xstreme %>% 
  dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>% 
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  kable(., caption = "Enriched motifs in EAR close 200bp") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "500px")
```

#### ESR lists

```{r eSr_open, message=FALSE, warning=FALSE}
# ESR_open_xstreme %>% 
#  dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
#    arrange(.,EVALUE) %>% 
#    dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   # dplyr::filter(., EVALUE<0.05) %>% 
#   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   kable(., caption = "Enriched motifs in ESR open") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "800px")

ESR_open_200xstreme %>% 
 dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
   arrange(.,EVALUE) %>% 
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  kable(., caption = "Enriched motifs in ESR open, 200bp") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")


# ESR_close_xstreme %>% 
#   dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
#   arrange(.,EVALUE) %>%
#    dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   # dplyr::filter(., EVALUE<0.05) %>% 
#   kable(., caption = "Enriched motifs  ESR close") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "800px")

ESR_close_200xstreme %>% 
  dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  kable(., caption = "Enriched motifs  ESR close 200bp") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")


# ESR_OC_xstreme %>% 
#   dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
#   arrange(.,EVALUE) %>%
#    dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   # dplyr::filter(., EVALUE<0.05) %>% 
#   kable(., caption = "Enriched motifs  in ESR OC") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "800px")

ESR_opcl_xstreme %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  kable(., caption = "Enriched motifs  in ESR opcl") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")

ESR_clop_xstreme %>% 
  dplyr::select(SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  kable(., caption = "Enriched motifs  in ESR clop") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
```

#### LR lists

```{r Lr_open, message=FALSE, warning=FALSE}
# LR_open_10h_xstreme %>%
#   dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>%
#   arrange(.,EVALUE) %>%
#    dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>%
#   # dplyr::filter(., EVALUE<0.05) %>%
#   kable(., caption = "Enriched motifs in LR_open") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "800px")

LR_open_200xstreme %>%
  dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>%
  arrange(.,EVALUE) %>%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>%
  # dplyr::filter(., EVALUE<0.05) %>%
  kable(., caption = "Enriched motifs in LR_open 200bp only") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")


# LR_close_200xstreme %>% 
#   dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
#   arrange(.,EVALUE) %>%
#    dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
#   # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   # dplyr::filter(., EVALUE<0.05) %>% 
#   kable(., caption = "Enriched motifs  in LR close") %>%
#   kable_paper("striped", full_width = TRUE) %>%
#   kable_styling(full_width = FALSE, font_size = 16) %>%
#   scroll_box(height = "800px")
LR_close_200xstreme %>% 
  dplyr::select(RANK,SIM_MOTIF,ALT_ID, ID,SEA_PVALUE,EVALUE) %>% 
  arrange(.,EVALUE) %>%
   dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %>%
  # separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  # dplyr::filter(., EVALUE<0.05) %>% 
  kable(., caption = "Enriched motifs in LR close, 200bp") %>%
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")
### selecting just the list of names and checking for expression


```



### All enriched motifs found


This area was for all motifs found in Xstreme analysis using NR peaks as background.
#### EAR
##### EAR_open
```{r specs of EAR data list making}
mrc_palette <- c(
    "EAR_open" = "#F8766D",
    "EAR_close" = "#f6483c",
    "ESR_open" = "#7CAE00",
    "ESR_close" = "#587b00",
    "ESR_C"="grey40",
     "ESR_opcl"="grey40",
    "ESR_D"="tan",
     "ESR_clop"="tan",
     "ESR_OC" = "#6a9500",
     "LR_open" = "#00BFC4",
     "LR_close" = "#008d91",
     "NR" = "#C77CFF"
  )

 # spd_EARo<-EAR_open_xstreme%>% 
 #   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
 #  dplyr::filter(EVALUE<0.05) %>% 
 #  left_join(., (sea_EAR_open %>%
 #                  # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
 #                  dplyr::select(RANK,ID:PVALUE)),
 #            by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID")) %>% 
 #  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
 #  mutate(motif_name= gsub("[()]","",NAME), mrc="EAR_open") %>%
 #  mutate(motif_name=
 #           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
 #           if_else(is.na(motif_name),ID,motif_name))) 
 
 spd_EARo_200<-EAR_open_200xstreme%>% 
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_EAR_open_200_p2 %>%
                  anti_join(.,sea_EAR_open_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_EAR_open_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="EAR_open") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name))) 
 #### breaks 
# spd_EARc <- EAR_close_xstreme%>% 
#    mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
#   dplyr::filter(EVALUE<0.05) %>% 
#   left_join(., (sea_EAR_close %>%
#                   # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
#                   dplyr::select(RANK,ID:PVALUE)),
#             by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID")) %>% 
#   separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   mutate(motif_name= gsub("[()]","",NAME), mrc="EAR_close") %>%
#   mutate(motif_name=
#            if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
#            if_else(is.na(motif_name),ID,motif_name)))

spd_EARc_200 <- EAR_close_200xstreme%>% 
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_EAR_close_200_p2 %>%
                  anti_join(.,sea_EAR_close_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_EAR_close_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="EAR_close") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name)))


##### breaks
# spd_ESRo <-ESR_open_xstreme%>%
#    mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
#   dplyr::filter(EVALUE<0.05) %>% 
#   left_join(., (sea_ESR_open %>%
#                   # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
#                   dplyr::select(RANK,ID:PVALUE)),
#             by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID"))%>% 
#   separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_open") %>%
#   mutate(motif_name=
#            if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
#            if_else(is.na(motif_name),ID,motif_name)))

spd_ESRo_200 <-ESR_open_200xstreme%>%
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>%
  dplyr::filter(EVALUE<0.05) %>%
  left_join(., (sea_ESR_open_200_p2 %>%
                  anti_join(.,sea_ESR_open_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_ESR_open_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_open") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name)))



```


```{r specs of ESR}
# spd_ESRc <- ESR_close_xstreme%>% 
#    mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
#   dplyr::filter(EVALUE<0.05) %>% 
#   left_join(., (sea_ESR_close %>%
#                   # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
#                   dplyr::select(RANK,ID:PVALUE)),
#             by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID")) %>% 
#   separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_close") %>%
#   mutate(motif_name=
#            if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
#            if_else(is.na(motif_name),ID,motif_name)))

spd_ESRc_200 <- ESR_close_200xstreme%>% 
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_ESR_close_200_p2 %>%
                  anti_join(.,sea_ESR_close_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_ESR_close_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_close") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name)))


# spd_ESRoc <-ESR_OC_xstreme%>% 
#    mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
#   dplyr::filter(EVALUE<0.05) %>% 
#   left_join(., (sea_ESR_OC %>%
#                   # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
#                   dplyr::select(RANK,ID:PVALUE)),
#             by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID")) %>% 
#   separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_OC") %>%
#   mutate(motif_name=
#            if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
#            if_else(is.na(motif_name),ID,motif_name)))
# #######rbind break too

spd_ESRopcl_200<-ESR_opcl_xstreme%>% 
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_ESR_opcl_200_p2 %>%
                  anti_join(.,sea_ESR_opcl_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_ESR_opcl_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_opcl") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name)))

#### break for clop!

spd_ESRclop_200<-ESR_clop_xstreme%>% 
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_ESR_clop_200_p2 %>%
                  anti_join(.,sea_ESR_clop_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_ESR_clop_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="ESR_clop") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name))) 
#opcl

###rbind break#####
# spd_LRo <-LR_open_xstreme%>%
#            mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
#   dplyr::filter(EVALUE<0.05) %>% 
#   left_join(., (sea_LR_open %>%
#                   # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
#                   dplyr::select(RANK,ID:PVALUE)),
#             by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID")) %>% 
#   separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   mutate(motif_name= gsub("[()]","",NAME), mrc="LR_open") %>%
#   mutate(motif_name=
#            if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,  if_else(is.na(motif_name),ID,motif_name)))

spd_LRo_200 <-LR_open_200xstreme%>%
           mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_LR_open_200_p2 %>%
                  anti_join(.,sea_LR_open_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_LR_open_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="LR_open") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,  if_else(is.na(motif_name),ID,motif_name)))
 
### rbind break
# spd_LRc <- LR_close_xstreme%>% 
#    mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
#   dplyr::filter(EVALUE<0.05) %>% 
#   left_join(., (sea_LR_close %>%
#                   # dplyr::rename("SEA_PVALUE"=PVALUE) %>%
#                   dplyr::select(RANK,ID:PVALUE)),
#             by= c("RANK"="RANK", "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS",  "ID"="ID")) %>% 
#   separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
#   mutate(motif_name= gsub("[()]","",NAME), mrc="LR_close") %>%
#   mutate(motif_name=
#            if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
#            if_else(is.na(motif_name),ID,motif_name)))

spd_LRc_200 <- LR_close_200xstreme%>% 
   mutate(CONSENSUS=gsub('[[:digit:]]+-', '', CONSENSUS)) %>% 
  dplyr::filter(EVALUE<0.05) %>% 
  left_join(., (sea_LR_close_200_p2 %>%
                  anti_join(.,sea_LR_close_200, by = c("ID"="ID","ALT_ID"="ALT_ID")) %>%
  rbind(sea_LR_close_200) %>% 
  dplyr::select(DB:LOG_QVALUE)), by= c( "ALT_ID"="ALT_ID", "CONSENSUS"="CONSENSUS","ID"="ID"))%>% 
  separate(SIM_MOTIF, into= c("SIM_MOTIF", "NAME"), sep= " ") %>% 
  mutate(motif_name= gsub("[()]","",NAME), mrc="LR_close") %>%
  mutate(motif_name=
           if_else(is.na(motif_name)&str_detect(SIM_MOTIF,"^M"),ALT_ID,
           if_else(is.na(motif_name),ID,motif_name)))

# spec_dataframe <- spd_EARo %>% 
#   rbind(spd_EARc) %>% 
#   rbind(spd_ESRo) %>%
#    rbind(spd_ESRc) %>% 
#    rbind(spd_ESRoc) %>% 
#    rbind(spd_LRo) %>% 
#    rbind(spd_LRc)

spec_dataframe_200 <- spd_EARo_200 %>% 
  rbind(spd_EARc_200) %>% 
  rbind(spd_ESRo_200) %>%
   rbind(spd_ESRc_200) %>%
  rbind(spd_ESRopcl_200) %>% 
  rbind(spd_ESRclop_200) %>% 
   rbind(spd_LRo_200) %>% 
   rbind(spd_LRc_200)
spec_dataframe_200 %>% 
  dplyr::filter(mrc=="ESR_opcl")

# saveRDS(spec_dataframe_200,"data/Final_four_data/spec_dataframe_200.RDS")
spec_dataframe_200 <- readRDS("data/Final_four_data/spec_dataframe_200.RDS")
```

### data plots from full sequences
```{r specs plots full seq}
###plotting 
# spec_dataframe %>% 
#  ggplot(.,aes(x=mrc,y=ENR_RATIO,fill=mrc))+
#   geom_boxplot()+ 
#    geom_hline(yintercept=1.2, col="red")+
#   theme_classic()+
#   ylab("Enrichment ratio")+
#   ggtitle("Enrichment ratio values across MRC")+
#   scale_fill_manual(values=mrc_palette)
# spec_dataframe %>% 
#   ggplot(., aes(x=ENR_RATIO, fill = mrc))+
#   geom_density(aes(alpha=0.4)) +
#    theme_classic()+
#   xlab("Enrichment ratio")+
#   ggtitle("Enrichment ratio histogram MRC")+
#   scale_fill_manual(values=mrc_palette)+
#   coord_cartesian(xlim=c(1,5))

# spec_dataframe %>% 
#   # dplyr::filter(mrc=="LR_open") %>% 
#   ggplot(., aes(x=mrc, y=-log(base=10,EVALUE), color=mrc))+
#     geom_jitter()+
#   theme_classic()+
#   ylab(expression("-log[10] Evalue"))+
#   ggtitle("Significant values across MRC motifs")+
#   scale_color_manual(values=mrc_palette)
# 
# spec_dataframe %>% 
#   # dplyr::filter(mrc=="LR_open") %>% 
#   ggplot(., aes(x=ENR_RATIO, y=-log(base=10,EVALUE), color=mrc))+
#     geom_point(alpha=0.3)+
#   theme_classic()+
#   ylab(expression("-log[10] Evalue"))+
#   ggtitle("Enrichment ratio against significance")+
#   scale_color_manual(values=mrc_palette)
  
```

### data plots from 200 bp sequences

```{r specs plots 200 bp seq}
###plotting 
spec_dataframe_200 %>% 
 ggplot(.,aes(x=mrc,y=ENR_RATIO,fill=mrc))+
  geom_boxplot()+
   geom_hline(yintercept=1.2, col="red")+
  theme_classic()+
  ylab("Enrichment ratio")+
  ggtitle("Enrichment ratio values across MRC_200")+
  scale_fill_manual(values=mrc_palette)
spec_dataframe_200 %>% 
  ggplot(., aes(x=ENR_RATIO, fill = mrc))+
  geom_density(aes(alpha=0.4)) +
  theme_classic()+
  xlab("Enrichment ratio")+
  ggtitle("Enrichment ratio histogram MRC_200")+
  scale_fill_manual(values=mrc_palette)+
  coord_cartesian(xlim=c(1,5))

spec_dataframe_200 %>% 
  # dplyr::filter(mrc=="LR_open") %>% 
  ggplot(., aes(x=mrc, y=-log(base=10,EVALUE.x), color=mrc))+
    geom_jitter()+
  theme_classic()+
  ylab(expression("-log[10] Evalue"))+
  ggtitle("Significant values across MRC motifs_200")+
  scale_color_manual(values=mrc_palette)

spec_dataframe_200 %>% 
  # dplyr::filter(mrc=="LR_open") %>% 
  ggplot(., aes(x=ENR_RATIO, y=-log(base=10,EVALUE.x), color=mrc))+
    geom_point(alpha=0.3)+
  theme_classic()+
  ylab(expression("-log[10] Evalue"))+
  ggtitle("Enrichment ratio against significance_200")+
  scale_color_manual(values=mrc_palette)
  
```


```{r eval=FALSE, include=FALSE}
TFmapnames <- readRDS("data/TFmapnames.RDS")
Biomart_names <- spec_dataframe %>% 
  dplyr::filter(mrc=="LR_close") %>% 
  distinct(motif_name) %>% 
  mutate(upper_name=toupper(motif_name))
library(biomaRt)
ensembl <- useMart(biomart="ensembl",dataset ="hsapiens_gene_ensembl")


my_attributes <- c('entrezgene_id', 'ensembl_gene_id', 'hgnc_symbol')

# snp_mart <- useMart("ENSEMBL_MART_SNP")
# snp_dataset <- useDataset("hsapiens_snp", mart=snp_mart)


listFilters(ensembl)
# listAttributes(dataset)
TF_binding_motif <- getBM(attributes=my_attributes,
      filters = 'gene_name',
      values = Biomart_names$upper_name,
      mart = ensembl)

# test<-
Biomart_names %>% 
  full_join(., TFmapnames, by=c("upper_name"="Gene name"))%>% 
  distinct(motif_name,`HGNC ID`,.keep_all = TRUE) %>%
  kable(., caption="Unique TF motifs and Entrezid associated with the symbol") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 16) %>%
  scroll_box(height = "800px")

 
```

The enrichment ratio plot let me know where (sort of) to cut the plots to find the highest enrichment ratios by MRC. We decided that ER ratio is not correlated to the significance of the motif, so first filter will be slicing the top 20 by Evalue out, and the second plot will be what happens when we pull the top most significant representative from the cluster and show overall significance by cluster.  (I evaluated the results of clustering by eye, to make sure the most significant (by evalue) of the cluster was called for the second figure)
### EAR bar plots

```{r enriched motifs EARopen, fig.height=8}
# ER_rat <- 1.1
# mrc_type <- "EAR_open"
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>%
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="ELK3",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF93",paste(motif_name, RANK, sep="_"), if_else(motif_name=="Nrf1",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#   geom_point(aes(x=`TP%`*1.25), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>%
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="ELK3",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF93",paste(motif_name, RANK, sep="_"), if_else(motif_name=="Nrf1",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#   # arrange(.,EVALUE) %>%
#    slice_head(n=5) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#    geom_point(aes(x=`TP%`/2.5), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*2.5,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged clusters"))
# 

```

#### EAR open 200 bp barplots
```{r enriched motifs EARopen200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "EAR_open"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>%
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="ELK3",paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^Z*"),paste(motif_name, RANK, sep="_"), if_else(motif_name=="Nrf1",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*1.25), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks(200bp)Enrichment ratio:",ER_rat))

spec_dataframe_200%>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>%
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="ELK3",paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^Z*"),paste(motif_name, RANK, sep="_"), if_else(motif_name=="Nrf1",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`/1.5), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*1.5,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks(200bp)Enrichment ratio:",ER_rat," merged clusters"))


```


##### EAR_close
```{r enriched motifs EAR_close, fig.height=8}
# ER_rat <- 1.5
# mrc_type <- "EAR_close"
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="MEIS1",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF384",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZNF257",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#    slice_head(n=40) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#   geom_point(aes(x=`TP%`*1.25), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="MEIS1",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF384",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZNF257",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>%
#   slice_head(n=10) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#    geom_point(aes(x=`TP%`/1.25), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*1.25,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#    ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged clusters"))

```




##### EAR_close 200bp
```{r enriched motifs EAR_close200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "EAR_close"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="MEIS1",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF384",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZNF257",paste(motif_name, RANK,sep="_"),motif_name))))%>%
   slice_head(n=40) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*1), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks (200bp) Enrichment ratio:",ER_rat))

spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="MEIS1",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF384",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZNF257",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
   slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`/1.25), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*1.25,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
   ggtitle(paste( mrc_type,"response peaks (200bp) Enrichment ratio:",ER_rat," merged clusters"))



spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  # dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  # arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  # mutate(motif_name=if_else(motif_name=="MEIS1",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF384",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZNF257",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  arrange(.,EVALUE.x) %>%
   slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`/1.25), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*1.25,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
   ggtitle(paste( mrc_type,"response peaks (200bp) Enrichment ratio: not applied  modified merged clusters"))

spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  # dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  # arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  # mutate(motif_name=if_else(motif_name=="MEIS1",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="ZNF384",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZNF257",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  arrange(.,EVALUE.x) %>%
   slice_head(n=5)
```



#### ESR

##### ESR_open

```{r enriched motifs ESR_open, fig.height=8}
# ER_rat <- 1.5
# mrc_type <- "ESR_open"
# spec_dataframe %>% 
#   dplyr::filter(mrc=="ESR_open") %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=
#            if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^ZN*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
#   slice_head(n=40) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette["ESR_open"]) +
#   geom_point(aes(x=`TP%`*4), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./4,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#    ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc=="ESR_open") %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#    mutate(motif_name=
#            if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^ZN*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#    slice_head(n=10) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette["ESR_open"]) +
#    geom_point(aes(x=`TP%`*4), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./4,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged motif clusters"))

```



```{r enriched motifs ESR_open_200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "ESR_open"
spec_dataframe_200 %>% 
  dplyr::filter(mrc=="ESR_open") %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=
           if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^ZN*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
  slice_head(n=40) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette["ESR_open"]) +
  geom_point(aes(x=`TP%`*2), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./2,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
   ggtitle(paste( mrc_type,"response peaks 200 bpEnrichment ratio:",ER_rat))

spec_dataframe_200 %>% 
  dplyr::filter(mrc=="ESR_open") %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
   mutate(motif_name=
           if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^ZN*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
   slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette["ESR_open"]) +
   geom_point(aes(x=`TP%`*1.3), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./1.3,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste(mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat," merged motif clusters"))
spec_dataframe_200 %>% 
  dplyr::filter(mrc=="ESR_open") %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
   mutate(motif_name=
           if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^ZN*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
   slice_head(n=5)
```

##### ESR_close

```{r enriched motifs ESR_close, fig.height=8}
# ER_rat <- 1.5
# mrc_type <- "ESR_close"
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   slice_head(n=40) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#   geom_point(aes(x=`TP%`*1.25), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#  ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#   slice_head(n=10) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#    geom_point(aes(x=`TP%`*1.2), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./1.2,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged motif clusters"))

```


```{r enriched motifs ESR_close_200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "ESR_close"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  slice_head(n=40) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*1.25), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
 ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat))


spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>(ER_rat+.1)) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`/0.5), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*0.5,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat+.1," merged motif clusters"))

```


##### ESR_OC

```{r enriched motifs ESR_OC, fig.height=8}
# ER_rat <- 1.5
# mrc_type <- "ESR_OC"
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#    mutate(motif_name=
#            if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^FO*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#   geom_point(aes(x=`TP%`*4), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./4,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#  ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>1.4) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#    mutate(motif_name=
#            if_else(str_starts(motif_name,"JUND"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^FO*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#    geom_point(aes(x=`TP%`*2), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./2,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged motif clusters"))

```


##### ESR_opcl {C}

```{r enriched motifs ESR_opcl_200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "ESR_opcl"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  slice_head(n=40) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*1.25), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
 ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat))


spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>(ER_rat)) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`/3), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*3,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat," merged motif clusters"))




```





##### ESR_clop {D}

```{r enriched motifs ESR_clop_200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "ESR_clop"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  slice_head(n=40) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*1.25), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
 ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat))



spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>(ER_rat)) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`*2.3), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./2.3,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat," merged motif clusters"))

```



#### LR

##### LR_open

```{r enriched motifs LR_open, fig.height=8}
# ER_rat <- 1.5
# mrc_type <- "LR_open"
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#  mutate(motif_name=
#            if_else(str_starts(motif_name,"BATF"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^FO*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
#   slice_head(n=60) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#   geom_point(aes(x=`TP%`*4), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./4,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#  ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=
#            if_else(str_starts(motif_name,"BATF"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^FO*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#    slice_head(n=10) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#    geom_point(aes(x=`TP%`*4), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./4,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged motif clusters"))
# 
# # spec_dataframe %>% 
# #   dplyr::filter(mrc==mrc_type) %>% 
# #   dplyr::filter(ENR_RATIO>ER_rat) #%>% 
# #   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)
```


```{r enriched motifs LR_open_200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "LR_open"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
 mutate(motif_name=
           if_else(str_starts(motif_name,"BATF"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^FO*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
  slice_head(n=60) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*4), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./4,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
 ggtitle(paste( mrc_type,"response peaks 200 bp Enrichment ratio:",ER_rat))


spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=
           if_else(str_starts(motif_name,"BATF"),paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^FO*"),paste(motif_name, RANK, sep="_"),    if_else(motif_name=="ZSCAN4", paste(motif_name, RANK,sep="_"),if_else(motif_name=="KLF9",paste(motif_name,RANK,sep="_"), motif_name)))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
   slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`*5), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./5,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks 200 bp Enrichment ratio:",ER_rat," merged motif clusters"))


```

##### LR_close


```{r enriched motifs LR_close, fig.height=8}
# ER_rat <- 1.25
# mrc_type <- "LR_close"
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^R*"),paste(motif_name, RANK, sep="_"), if_else(motif_name=="PKNOX2",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#     slice_head(n=40) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#   geom_point(aes(x=`TP%`*1.3), size =4)+
#   scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.3,name="Percent of peaks with motif"))+
#   theme_classic()+
#   ylab("Enriched TF motif")+
#  ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat))
# 
# 
# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE) %>%
#   mutate(log10Evalue= log(EVALUE, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^R*"),paste(motif_name, RANK, sep="_"), if_else(motif_name=="PKNOX2",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#   slice_head(n=10) %>% 
#   ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
#   geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
#    geom_point(aes(x=`TP%`*1.3), size =4)+
#    # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
#   scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~./1.3,name="Percent of peaks with motif"))+
#   # geom_text(aes())
#   theme_classic()+
#   ylab("Enriched TF motif")+
#   ggtitle(paste( mrc_type,"response peaks Enrichment ratio:",ER_rat," merged motif clusters"))

```



```{r enriched motifs LR_close_200, fig.height=5}
ER_rat <- 1.25
mrc_type <- "LR_close"
spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^R*"),paste(motif_name, RANK, sep="_"), if_else(motif_name=="PKNOX2",paste(motif_name, RANK,sep="_"),motif_name))))%>%
    slice_head(n=40) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
  geom_point(aes(x=`TP%`*1.25), size =4)+
  scale_x_continuous(expand=c (0,.125),sec.axis = sec_axis(transform= ~./1.25,name="Percent of peaks with motif"))+
  theme_classic()+
  ylab("Enriched TF motif")+
 ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat))


spec_dataframe_200 %>% 
  dplyr::filter(mrc==mrc_type) %>% 
  dplyr::filter(ENR_RATIO>ER_rat) %>% 
  dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
  arrange(.,EVALUE.x) %>%
  mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
  mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(str_starts(motif_name,"^R*"),paste(motif_name, RANK, sep="_"), if_else(motif_name=="PKNOX2",paste(motif_name, RANK,sep="_"),motif_name))))%>%
  distinct(CLUSTER,.keep_all = TRUE) %>% 
  slice_head(n=5) %>% 
  ggplot(., aes (y= reorder(motif_name,log10Evalue))) +
  geom_col(aes(x=log10Evalue),fill=mrc_palette[mrc_type]) +
   geom_point(aes(x=`TP%`/.4), size =4)+
   # geom_line(aes(x=`TP%`,y= motif_name, group=log10Evalue))+
  scale_x_continuous(expand=c (0,.25),sec.axis = sec_axis(transform= ~.*.4,name="Percent of peaks with motif"))+
  # geom_text(aes())
  theme_classic()+
  ylab("Enriched TF motif")+
  ggtitle(paste( mrc_type,"response peaks 200bp Enrichment ratio:",ER_rat," merged motif clusters"))

# spec_dataframe %>% 
#   dplyr::filter(mrc==mrc_type) %>% 
#   dplyr::filter(ENR_RATIO>ER_rat) #%>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE, TP:'FP%', motif_name)
```



## motif and peak section (incomplete)


```{r resizing peaks}
NR_gr <- import("data/Final_four_data/meme_bed/NR_df.bed")
EAR_open_gr <- import("data/Final_four_data/meme_bed/EAR_open.bed")
EAR_close_gr <- import("data/Final_four_data/meme_bed/EAR_close.bed")
ESR_open_gr <- import("data/Final_four_data/meme_bed/ESR_open.bed")
ESR_OC_gr <- import("data/Final_four_data/meme_bed/ESR_OC.bed")
ESR_close_gr <- import("data/Final_four_data/meme_bed/ESR_close.bed")
LR_open_gr <- import("data/Final_four_data/meme_bed/LR_open.bed")
LR_close_gr <- import("data/Final_four_data/meme_bed/LR_close.bed")
ESR_C_gr <- import("data/Final_four_data/meme_bed/ESR_C.bed")
 
ESR_D_gr <- import("data/Final_four_data/meme_bed/ESR_D.bed")
 
# rtracklayer::export.bed(ESR_D_resized,con = "data/Final_four_data/meme_bed/ESR_D_resized.bed")
ESR_C_resized <- resize(ESR_C_gr, width = 400,fix='center')
ESR_D_resized <- resize(ESR_D_gr, width = 400,fix='center')
EAR_open_resized <- resize(EAR_open_gr, width = 400,fix='center')
ESR_open_resized <- resize(ESR_open_gr, width = 400,fix='center')
LR_open_resized <- resize(LR_open_gr, width = 400,fix='center')
NR_resized <- resize(NR_gr, width = 400,fix='center')

EAR_close_resized <- resize(EAR_close_gr, width = 400,fix='center')
ESR_close_resized <- resize(ESR_close_gr, width = 400,fix='center')
LR_close_resized <- resize(LR_close_gr, width = 400,fix='center')
ESR_OC_resized <- resize(ESR_OC_gr, width = 400,fix='center')



# BiocManager::install('PWMEnrich')

# seq_list_cd <- list(EAR_open_resized=EAR_open_resized,EAR_close_resized=EAR_close_resized,ESR_open_resized=ESR_open_resized,ESR_close_resized=ESR_close_resized,ESR_C_resized=ESR_C_resized,ESR_D_resized=ESR_D_resized,LR_open_resized=LR_open_resized,LR_close_resized=LR_close_resized,NR_resized=NR_resized)



# seq_all_cd_200 <- lapply(seq_list_cd, getSeq, x=BSgenome.Hsapiens.UCSC.hg38)
# seq_all_200

# saveRDS(seq_all_cd_200, "data/Final_four_data/sequencing_cd_200_object8_motif.RDS")
# seq_all_200 <- readRDS("data/Final_four_data/sequencing_R200_object8_motif.RDS")
seq_all_cd_200 <- readRDS("data/Final_four_data/sequencing_cd_200_object8_motif.RDS")

# spec_dataframe_200 %>% 
#   # dplyr::filter(stringr::str_detect( motif_name,"Mafg^*")) %>%
#   dplyr::filter(mrc=="EAR_close") %>% 
#   dplyr::select(RANK,CLUSTER,ENR_RATIO,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
#   dplyr::filter(CLUSTER==58|CLUSTER==58) %>%
#   arrange(.,EVALUE.x)# %>%
```

```{r specdataframe groups}
spec_dataframe_200 <- readRDS("data/Final_four_data/spec_dataframe_200.RDS")

spec_dataframe_200 %>% 
  dplyr::select(CLUSTER, ID:CONSENSUS,motif_name, mrc,NAME,ENR_RATIO,EVALUE.x) %>% 
  dplyr::filter(ENR_RATIO>1.2) %>% 
 dplyr::filter(EVALUE.x<0.05)%>% 
  group_by(mrc,CLUSTER) %>% 
  summarize(ID=paste(unique(ID), collapse = ";"),
            ALT_ID=paste(unique(ALT_ID), collapse = ";"),
            motif_name=paste(unique(motif_name),collapse="; "),
            NAME=paste((NAME),collapse =";"),
            mrc=unique(mrc),
            sig_val=paste0(min(EVALUE.x),"-",max(EVALUE.x)),
            order_val=min(EVALUE.x)) %>% 
    arrange(mrc,order_val) #%>% 
  # write_delim(.,"data/Final_four_data/motif_cluster_dataframe.txt",delim="\t")


```

```{r top}
# spec_dataframe_200 %>% 
#   # dplyr::filter(stringr::str_detect( motif_name,"Mafg^*")) %>%
#   dplyr::filter(mrc=="ESR_open") %>%
#   # dplyr::select(RANK,CLUSTER,ENR_RATIO,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
#   dplyr::filter(CLUSTER==1
#                 ) %>%
#   arrange(.,EVALUE.x) %>%
#   dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`)# %>%
  # dplyr::filter(ENR_RATIO>1.25) %>%
  # distinct(CLUSTER,.keep_all = TRUE)
# distinct(motif_name)

# spec_dataframe_200 %>% 
#   dplyr::filter(mrc=="ESR_open") %>% 
#   dplyr::filter(ENR_RATIO>(ER_rat)) %>% 
#   dplyr::select(RANK,CLUSTER, SITES,SIM_MOTIF,ALT_ID, ID,EVALUE.x, TP:'FP%', motif_name)%>%
#   arrange(.,EVALUE.x) %>%
#   mutate(log10Evalue= log(EVALUE.x, base = 10)*(-1)) %>% 
#   mutate(motif_name=if_else(motif_name=="KLF9",paste(motif_name, RANK, sep="_"),             if_else(motif_name=="^ZN*",paste(motif_name, RANK, sep="_"), if_else(motif_name=="ZSCAN4",paste(motif_name, RANK,sep="_"),motif_name))))%>%
#   distinct(CLUSTER,.keep_all = TRUE) %>% 
#   slice_head(n=5)
```


```{r topmotif NRF1}
library(universalmotif)
library(ggseqlogo)
library(motifmatchr)
library(gridExtra)
meme_motifs_EAR_open <- read_meme("~/ATAC_meme_data/200bp/EAR_open_200xstreme/xstreme.txt")
meme_motifs_ESR_open <- read_meme("~/ATAC_meme_data/200bp/ESR_open_200xstreme/xstreme.txt")
meme_motifs_LR_open <- read_meme("~/ATAC_meme_data/200bp/LR_open_200xstreme/xstreme.txt")
meme_motifs_EAR_close <- read_meme("~/ATAC_meme_data/200bp/EAR_close_200xstreme/xstreme.txt")
meme_motifs_ESR_close <- read_meme("~/ATAC_meme_data/200bp/ESR_close_200xstreme/xstreme.txt")
meme_motifs_LR_close <- read_meme("~/ATAC_meme_data/200bp/LR_close_200xstreme/xstreme.txt")
meme_motifs_ESR_opcl <- read_meme("~/ATAC_meme_data/200bp/ESR_C_200xstreme/xstreme.txt")
meme_motifs_ESR_clop <- read_meme("~/ATAC_meme_data/200bp/ESR_D_200xstreme/xstreme.txt")


holder_EAR_open <- meme_motifs_EAR_open[[4]]
holder_ESR_open <- meme_motifs_ESR_open[[3]]
holder_LR_open <- meme_motifs_LR_open[[1]]
holder_ESR_opcl <- meme_motifs_ESR_opcl[[2]]
holder_EAR_close <- meme_motifs_EAR_close[[2]]
holder_ESR_close <- meme_motifs_ESR_close[[18]]
holder_LR_close <- meme_motifs_LR_close[[1]]
holder_ESR_clop <- meme_motifs_ESR_clop[[1]]
```

```{r eval=FALSE, fig.height=10, include=FALSE}
group_of_memes <- list(EAR_open=meme_motifs_EAR_open,
                    ESR_open=meme_motifs_ESR_open,
                    LR_open=meme_motifs_LR_open,
                    EAR_close=meme_motifs_EAR_close,
                    ESR_close=meme_motifs_ESR_close,
                    LR_close=meme_motifs_LR_close,
                    ESR_opcl=meme_motifs_ESR_opcl,
                    ESR_clop=meme_motifs_ESR_clop)
name_plot <- c("meme_motifs_EAR_open",
                  "meme_motifs_ESR_open",
                    "meme_motifs_LR_open",
                "meme_motifs_EAR_close",
                    "meme_motifs_ESR_close",
                    "meme_motifs_LR_close",
                    "meme_motifs_ESR_opcl",
                    "meme_motifs_ESR_clop")
counter <- 0
for (meme_frame in group_of_memes ){
   counter <- counter +1 
motif_info <- lapply(meme_frame, function(motif) {
  data.frame(
    motif_name = ifelse(is.null(motif@name), NA, motif@name),
    alt_name = ifelse(is.null(motif@altname), NA, motif@altname)  # Handle missing altname
    # Store the position weight matrix (PWM) in a list
    
  )
})
# Combine the individual motif data frames into one long data frame
motif_df <- bind_rows(motif_info)

plots <- lapply(1:nrow(motif_df), function(i) {
  pwm <-meme_frame[[i]]@motif # Extract PWM for each motif
  name <- motif_df$motif_name[i]
  alt_name <- motif_df$alt_name[i]
  
  # Check if the PWM is valid and can be plotted
  if (!is.null(pwm) && is.matrix(pwm)) {
    ggseqlogo::ggseqlogo(pwm,  method="bits") +
      ggtitle(paste("Motif:", name, "\n| Altname:", alt_name)) +
      theme_minimal()
  } else {
    message("Invalid PWM for motif:", name)
    NULL
  }
})
savefile <- paste0("data/Final_four_data/",name_plot[counter],".pdf")
plots_per_page <- 16 
multi_page_grobs <- marrangeGrob(plots, nrow = 4, ncol = 4) 
ggsave(savefile, multi_page_grobs)
grid.draw(multi_page_grobs)
}


```

### EAR open top motif
```{r topmotif EAR open}
# pw_matrix_earo <- convert_motifs(holder_EAR_open, class = "TFBSTools-PWMatrix")
# 
# motif_EAR_open_4_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_earo, sequences, out = "positions")
# })
# 
# motifstorage_earo <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_EAR_open_4_results), function(i) {
  # positions <- motif_EAR_open_4_results[[i]]  # Positions from matchMotifs
  # seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
  # 
  # if (length(positions) > 0) {
  #   # Create a data frame with sequence_id and motif positions
  #   data.frame(
  #     sequence_id = seq_name,  # Use the sequence name here
  #     motif_position = unlist(positions)
  #   )
  # } else {
  #   NULL  # If no positions, return NULL
#   }
# }))

 # saveRDS(motifstorage_earo, "data/Final_four_data/matchr_EAR_open_4.RDS")
##### PLOTING CODE BELOW THIS LINE FOR EAR_OPEN  #####
 # motifstorage_earo <- readRDS("data/Final_four_data/matchr_EAR_open_6.RDS")
motifstorage_earo <- readRDS("data/Final_four_data/matchr_EAR_open_4.RDS")
motifstorage_earo %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="EAR_open"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("ZBTB14")



EAR_open_density_plot <- motifstorage_earo %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "EAR_open" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("ZBTB14")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

EAR_open_matrix_eg <- holder_EAR_open@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
EAR_open_motif_logo_plot <- ggseqlogo(EAR_open_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("EAR_open Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  EAR_open_toprow <- plot_grid(NULL,EAR_open_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
EAR_open_combined_plot <- plot_grid(EAR_open_toprow,EAR_open_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(EAR_open_combined_plot)
```

### ESR open top motif
```{r topmotif ESR open}
# pw_matrix_ESRo <- convert_motifs(holder_ESR_open, class = "TFBSTools-PWMatrix")
# 
# motif_ESR_open_6_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_ESRo, sequences, out = "positions")
# })
# 
# motifstorage_ESRo <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_ESR_open_6_results), function(i) {
#   positions <- motif_ESR_open_6_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
# # 
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
# #   
#  saveRDS(motifstorage_ESRo, "data/Final_four_data/matchr_ESR_open_3.RDS")
#### PLOTING CODE BELOW THIS LINE FOR ESR_OPEN  #####
motifstorage_ESRo <- readRDS("data/Final_four_data/matchr_ESR_open_3.RDS")
motifstorage_ESRo %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="ESR_open"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("motifish")



ESR_open_density_plot <- motifstorage_ESRo %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "ESR_open" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("JUNB")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

ESR_open_matrix_eg <- holder_ESR_open@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
ESR_open_motif_logo_plot <- ggseqlogo(ESR_open_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("ESR_open Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  ESR_open_toprow <- plot_grid(NULL,ESR_open_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
ESR_open_combined_plot <- plot_grid(ESR_open_toprow,ESR_open_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(ESR_open_combined_plot)
```


### LR open top motif
```{r topmotif LR open}
# pw_matrix_LRo <- convert_motifs(holder_LR_open, class = "TFBSTools-PWMatrix")
# 
# motif_LR_open_1_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_LRo, sequences, out = "positions")
# })
# 
# motifstorage_LRo <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_LR_open_1_results), function(i) {
#   positions <- motif_LR_open_1_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
# # 
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
# #   
 # saveRDS(motifstorage_LRo, "data/Final_four_data/matchr_LR_open_1.RDS")
#### PLOTING CODE BELOW THIS LINE FOR LR_OPEN  #####
motifstorage_LRo <- readRDS("data/Final_four_data/matchr_LR_open_1.RDS")
motifstorage_LRo %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="LR_open"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("Mafg")



LR_open_density_plot <- motifstorage_LRo %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "LR_open" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("Mafg")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

LR_open_matrix_eg <- holder_LR_open@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
LR_open_motif_logo_plot <- ggseqlogo(LR_open_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("LR_open Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  LR_open_toprow <- plot_grid(NULL,LR_open_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
LR_open_combined_plot <- plot_grid(LR_open_toprow,LR_open_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(LR_open_combined_plot)
```


### EAR_close top motif
```{r topmotif EAR_close}
# pw_matrix_EARc <- convert_motifs(holder_EAR_close, class = "TFBSTools-PWMatrix")
# 
# motif_EAR_close_1_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_EARc, sequences, out = "positions")
# })

# motifstorage_EARc <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_EAR_close_1_results), function(i) {
#   positions <- motif_EAR_close_1_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
#
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
# #   
 # saveRDS(motifstorage_EARc, "data/Final_four_data/matchr_EAR_close_2.RDS")
#### PLOTING CODE BELOW THIS LINE FOR EAR_close  #####
motifstorage_EARc <- readRDS("data/Final_four_data/matchr_EAR_close_2.RDS")
motifstorage_EARc %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="EAR_close"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("Atf3")



EAR_close_density_plot <- motifstorage_EARc %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "EAR_close" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("Atf3")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

EAR_close_matrix_eg <- holder_EAR_close@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
EAR_close_motif_logo_plot <- ggseqlogo(EAR_close_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("EAR_close Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  EAR_close_toprow <- plot_grid(NULL,EAR_close_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
EAR_close_combined_plot <- plot_grid(EAR_close_toprow,EAR_close_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(EAR_close_combined_plot)
```



### ESR_close top motif
```{r topmotif ESR_close}
# pw_matrix_ESRc <- convert_motifs(holder_ESR_close, class = "TFBSTools-PWMatrix")
# # 
# motif_ESR_close_18_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_ESRc, sequences, out = "positions")
# })
# 
# motifstorage_ESRc <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_ESR_close_18_results), function(i) {
#   positions <- motif_ESR_close_18_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
# 
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
# #
#  saveRDS(motifstorage_ESRc, "data/Final_four_data/matchr_ESR_close_18.RDS")
#### PLOTING CODE BELOW THIS LINE FOR ESR_close  #####
motifstorage_ESRc <- readRDS("data/Final_four_data/matchr_ESR_close_18.RDS")
motifstorage_ESRc %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="ESR_close"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("ZNF281")



ESR_close_density_plot <- motifstorage_ESRc %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "ESR_close" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("ZNF281")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

ESR_close_matrix_eg <- holder_ESR_close@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
ESR_close_motif_logo_plot <- ggseqlogo(ESR_close_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("ESR_close Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  ESR_close_toprow <- plot_grid(NULL,ESR_close_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
ESR_close_combined_plot <- plot_grid(ESR_close_toprow,ESR_close_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(ESR_close_combined_plot)
```





### LR_close top motif
```{r topmotif LR_close}
# pw_matrix_LRc <- convert_motifs(holder_LR_close, class = "TFBSTools-PWMatrix")
# # 
# motif_LR_close_1_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_LRc, sequences, out = "positions")
# })
# 
# motifstorage_LRc <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_LR_close_1_results), function(i) {
#   positions <- motif_LR_close_1_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
# 
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
# #
#  saveRDS(motifstorage_LRc, "data/Final_four_data/matchr_LR_close_1.RDS")
#### PLOTING CODE BELOW THIS LINE FOR LR_close  #####
motifstorage_LRc <- readRDS("data/Final_four_data/matchr_LR_close_1.RDS")
motifstorage_LRc %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="LR_close"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("MEIS2")



LR_close_density_plot <- motifstorage_LRc %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "LR_close" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("MEIS2")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

LR_close_matrix_eg <- holder_LR_close@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
LR_close_motif_logo_plot <- ggseqlogo(LR_close_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("LR_close Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  LR_close_toprow <- plot_grid(NULL,LR_close_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
LR_close_combined_plot <- plot_grid(LR_close_toprow,LR_close_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(LR_close_combined_plot)
```





### ESR_opcl top motif
```{r topmotif ESR_opcl}
# pw_matrix_ESR_opcl <- convert_motifs(holder_ESR_opcl, class = "TFBSTools-PWMatrix")
# #
# motif_ESR_opcl_2_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_ESR_opcl, sequences, out = "positions")
# })

# motifstorage_ESR_opcl <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_ESR_opcl_2_results), function(i) {
#   positions <- motif_ESR_opcl_2_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
# # 
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
#
 # saveRDS(motifstorage_ESR_opcl, "data/Final_four_data/matchr_ESR_opcl_2.RDS")
#### PLOTING CODE BELOW THIS LINE FOR ESR_opcl  #####
motifstorage_ESR_opcl <- readRDS("data/Final_four_data/matchr_ESR_opcl_2.RDS")
motifstorage_ESR_opcl %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="ESR_opcl"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("ZNF384")



ESR_opcl_density_plot <- motifstorage_ESR_opcl %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "ESR_C" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("ZNF384")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

ESR_opcl_matrix_eg <- holder_ESR_opcl@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
ESR_opcl_motif_logo_plot <- ggseqlogo(ESR_opcl_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("ESR_opcl Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  ESR_opcl_toprow <- plot_grid(NULL,ESR_opcl_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
ESR_opcl_combined_plot <- plot_grid(ESR_opcl_toprow,ESR_opcl_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(ESR_opcl_combined_plot)
```




### ESR_clop top motif
```{r topmotif ESR_clop}
# pw_matrix_ESR_clop <- convert_motifs(holder_ESR_clop, class = "TFBSTools-PWMatrix")
# # #
# motif_ESR_clop_1_results <- lapply(seq_all_cd_200, function(sequences) {
# matchMotifs(pw_matrix_ESR_clop, sequences, out = "positions")
# })

# motifstorage_ESR_clop <-motif_positions_df <- do.call(rbind, lapply(seq_along(motif_ESR_clop_1_results), function(i) {
#   positions <- motif_ESR_clop_1_results[[i]]  # Positions from matchMotifs
#   seq_name <- names(seq_all_cd_200)[i]  # Get sequence ID from the original list
# #
#   if (length(positions) > 0) {
#     # Create a data frame with sequence_id and motif positions
#     data.frame(
#       sequence_id = seq_name,  # Use the sequence name here
#       motif_position = unlist(positions)
#     )
#   } else {
#     NULL  # If no positions, return NULL
#   }
# }))
#
 # saveRDS(motifstorage_ESR_clop, "data/Final_four_data/matchr_ESR_clop_1.RDS")
#### PLOTING CODE BELOW THIS LINE FOR ESR_clop  #####
motifstorage_ESR_clop <- readRDS("data/Final_four_data/matchr_ESR_clop_1.RDS")
motifstorage_ESR_clop %>% 
mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  # dplyr::filter(id =="ESR_clop"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=motif_position.start-200) %>% 
  ggplot(.,aes(position, color = sequence_id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("ATF3")



ESR_clop_density_plot <- motifstorage_ESR_clop %>% 
  mutate(sequence_id= gsub("_resized","",sequence_id)) %>% 
  dplyr::filter(sequence_id == "ESR_D" | sequence_id == "NR") %>%
   mutate(position=motif_position.start-200) %>% 
  ggplot(., aes(position, color = sequence_id)) +
  geom_density(trim = FALSE, size = 1.5) +
  theme_bw() +
  scale_color_manual(values = mrc_palette) +
  ggtitle("ATF3")

# Assuming you already have the motif in a matrix or data frame for ggseqlogo

ESR_clop_matrix_eg <- holder_ESR_clop@motif  # If your motif is in PWM format

# Create the motif logo plot, this adds bits instead of "p" which is probability.
ESR_clop_motif_logo_plot <- ggseqlogo(ESR_clop_matrix_eg, method = "bits") + 
  theme_classic() +  
  ggtitle("ESR_clop Motif Logo")
  
# Combine the density plot and motif logo plot using cowplot
  ESR_clop_toprow <- plot_grid(NULL,ESR_clop_motif_logo_plot, NULL, ncol = 3, rel_widths = c(.2,1,.2))
ESR_clop_combined_plot <- plot_grid(ESR_clop_toprow,ESR_clop_density_plot,nrow=2, rel_heights = c(0.7,2))

# Print the combined plot
print(ESR_clop_combined_plot)
```



#### NRF1






```{r NRF1}
# 
# motifs_NRF1 = query(query(MotifDb, 'Hsapiens'), 'NRF1')
#     NRF1_motif <- motifs_NRF1$`Hsapiens-jaspar2018-NRF1-MA0506.1`
# # #
# #
# NRF1_pwm = PWMatrix(
#     ID = 'NRF1',
#     profileMatrix = NRF1_motif)
# 
# list_nrf1 = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=NRF1_pwm, min.score="80%", strand="*")
# 
# nrf1storage <- map_dfr(list_nrf1, ~as.data.frame(.x), .id="id")

# saveRDS(nrf1storage, "data/Final_four_data/nrf1_motif_8group.RDS")
nrf1_motif_8group <- readRDS("data/Final_four_data/nrf1_motif_8group.RDS")
nrf1_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="EAR_open"|id == "NR") %>%
   # dplyr::filter(relScore>.90) %>%
  mutate(position=start-200) %>% 
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("NRF1")
spec_dataframe_200 %>% 
  # dplyr::filter(mrc=="LR_close")
  # dplyr::filter(is.na(ENR_RATIO))
  dplyr::filter(stringr::str_detect(motif_name,"Nrf"))%>%
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`)%>% 
  dplyr::filter(ENR_RATIO>1.25)

```

#### BACH2

not redone yet
```{r BACH2 motif, eval=FALSE, include=FALSE}
# motifs_BACH2 = query(query(MotifDb, 'Hsapiens'),'BACH2')
# BACH2_motif <- motifs_BACH2$`Hsapiens-jaspar2022-BACH2-MA1101.2`
#  
# BACH2_pwm = PWMatrix(
#     ID = 'BACH2',
#     profileMatrix = BACH2_motif)

# 
# pwmList <- PWMatrixList(MA1101.2=toPWM(MA1101.2), MA0004.1=toPWM(MA0004.1),
#                         use.names=TRUE)
# list_test = lapply(unlist(seq_all) , FUN=searchSeq, x=BACH2_pwm, min.score="80%", strand="*")
# 
# storage <- lapply(list_test, relScore)
# str(storage)
# storage %>% 
#   map(as_tibble) %>% 
#   reduce(bind_rows)
# testedstorage <- map_dfr(list_test, ~as.data.frame(.x), .id="id")

# saveRDS(testedstorage, "data/Final_four_data/BACH2_motif_8group.RDS")
BACH2_motif_8group <- readRDS("data/Final_four_data/BACH2_motif_8group.RDS")
BACH2_motif_8group %>% 
  mutate(id= gsub("_resized","",id)) %>% 
  # dplyr::filter(relScore>.85) %>%
  mutate(position=start-500) %>% 
  ggplot(.,aes(position, color = id))+
  geom_density(size=1.5) +
  theme_bw() +
   scale_color_manual(values=mrc_palette)+
  ggtitle("BACH2")

# spec_dataframe %>% 
#   # dplyr::filter(mrc=="LR_close")
#   # dplyr::filter(is.na(ENR_RATIO))
#   dplyr::filter(stringr::str_detect(motif_name,"BACH2"))%>% 
#   dplyr::select(RANK,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE,ENR_RATIO,`TP%`)%>% 
#   dplyr::filter(ENR_RATIO>1.25)


```


#### ZNF384

```{r ZNF384}

# motifs_ZNF384 = query(query(MotifDb, 'Hsapiens'), 'ZNF384')
# ZNF384_motif <- motifs_ZNF384$`Hsapiens-jaspar2022-ZNF384-MA1125.1`
# #
# #
# ZNF384_pwm = PWMatrix(
    # ID = 'ZNF384',
    # profileMatrix = ZNF384_motif)

# plan(multisession, workers = parallel::detectCores())
# 
# # Apply future_lapply for parallel execution
# znf384_test <- future_lapply(unlist(seq_all), FUN=searchSeq, x=ZNF384_pwm, min.score="80%", strand="*")

# ZNF384_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=ZNF384_pwm, min.score="80%", strand="*")
# znf384_storage <- rbindlist(znf384_test, idcol="id")
# site_data_tables <- lapply(znf384_test, function(x) {
  # as.data.table(as.data.frame(x))  # Convert to data table for more efficiency
# })
# znf384_storage <- future_lapply(znf384_test, as.data.frame)
# znf384_storage <- rbindlist(site_data_tables, idcol="id",fill = TRUE)

# znf384_storage <- map_dfr(ZNF384_test, ~as.data.frame(.x), .id="id")
# Shut down workers once done
# plan(sequential)
# saveRDS(znf384_storage, "data/Final_four_data/znf384_motif_8group.RDS")
znf384_motif_8group <- readRDS("data/Final_four_data/znf384_motif_8group.RDS")
znf384_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="ESR_C"|id == "NR"|id=="EAR_close") %>%
  # dplyr::filter(relScore>0.9) %>%
  mutate(position=start-200) %>% 
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("ZNF384 motif")
spec_dataframe_200 %>% 
  # dplyr::filter(mrc=="LR_close")
  # dplyr::filter(is.na(ENR_RATIO))
  dplyr::filter(stringr::str_detect( motif_name,"ZNF384")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 
  dplyr::filter(ENR_RATIO>1.25)

```

#### ELK3

not redone yet

```{r ELK3}

motifs_ELK3 = query(query(MotifDb, 'Hsapiens'), 'ELK3')
    ELK3_motif <- motifs_ELK3$`Hsapiens-jaspar2022-ELK3-MA0759.2`
#     # #
# # #
ELK3_pwm = PWMatrix(
    ID = 'ELK3',
    profileMatrix = ELK3_motif)
# ELK3_test = lapply(unlist(seq_all) , FUN=searchSeq, x=ELK3_pwm, min.score="80%", strand="*")
# # 
# 
# ELK3_storage <- map_dfr(ELK3_test, ~as.data.frame(.x), .id="id")

# saveRDS(ELK3_storage, "data/Final_four_data/ELK3_motif_8group.RDS")
ELK3_motif_8group <- readRDS("data/Final_four_data/ELK3_motif_8group.RDS")
ELK3_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="ESR_C"|id == "NR"|id=="EAR_close") %>%
  # dplyr::filter(relScore>0.9) %>%
  mutate(position=start-500) %>% 
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("ELK3 motif")
# spec_dataframe %>% 
#   dplyr::filter(stringr::str_detect( motif_name,"ELK^*")) %>% 
#   dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE,ENR_RATIO,`TP%`) %>% 
#   dplyr::filter(ENR_RATIO>1.25)
```



#### ATF3
```{r Atf3}

# motifs_Atf3 = query(query(MotifDb, 'Hsapiens'), 'ATF3')
    # Atf3_motif <- motifs_Atf3$`Hsapiens-jaspar2022-ATF3-MA0605.2`
# #     # #
# # #
# Atf3_pwm = PWMatrix(
#     ID = 'Atf3',
#     profileMatrix = Atf3_motif)
# Atf3_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=Atf3_pwm, min.score="80%", strand="*")
# # 
# 
# Atf3_storage <- map_dfr(Atf3_test, ~as.data.frame(.x), .id="id")

# saveRDS(Atf3_storage, "data/Final_four_data/Atf3_motif_8group.RDS")
Atf3_motif_8group <- readRDS("data/Final_four_data/Atf3_motif_8group.RDS")
Atf3_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="ESR_open"|id == "NR"|id=="ESR_D"|id =="LR_open") %>%
  # dplyr::filter(relScore>0.90) %>%
  mutate(position=start-200) %>% 
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("Atf3 motif")
spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"Atf^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 
  dplyr::filter(ENR_RATIO>1.25)
```

#### TP53
```{r TP53}

# motifs_TP53 = query(query(MotifDb, 'Hsapiens'), 'TP53')
    # TP53_motif <- motifs_TP53$`Hsapiens-jaspar2022-TP53-MA0106.3`


# TP53_pwm = PWMatrix(
# ID = 'TP53',
# profileMatrix = TP53_motif)
# TP53_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=TP53_pwm, min.score="80%", strand="*")


# TP53_storage <- map_dfr(TP53_test, ~as.data.frame(.x), .id="id")
# 
# saveRDS(TP53_storage, "data/Final_four_data/TP53_motif_8group200cd.RDS")
TP53_motif_8group <- readRDS("data/Final_four_data/TP53_motif_8group200cd.RDS")

TP53_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="LR_open"|id == "NR"|id=="ESR_open") %>%
  # dplyr::filter(relScore>0.90) %>%
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("TP53 motif")
spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"TP^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 
  dplyr::filter(ENR_RATIO>1.25)
```

#### TP73
```{r TP73}

# motifs_TP73 = query(query(MotifDb, 'Hsapiens'), 'TP73')
    # TP73_motif <- motifs_TP73$`Hsapiens-jaspar2022-TP73-MA0861.1`
# 
# tp73list <- seq_all_cd_200[5:6]
# TP73_pwm = PWMatrix(
# ID = 'TP73',
# profileMatrix = TP73_motif)
# TP73_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=TP73_pwm, min.score="80%", strand="*")
# 

# TP73_storage_cd <- map_dfr(TP73_test, ~as.data.frame(.x), .id="id")
# TP73_teststorage <- rbind(TP73_storage_cd,TP73_motif_8group)
# saveRDS(TP73_storage_cd, "data/Final_four_data/TP73_motif_8group200_cd.RDS")
TP73_motif_8group <- readRDS("data/Final_four_data/TP73_motif_8group200_cd.RDS")

TP73_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id == "NR"|id=="ESR_open") %>%
  # dplyr::filter(relScore>0.90) %>%
  # dplyr::filter(id != "ESR_OC") %>% 
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("TP73 motif")
spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"TP73^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 
  dplyr::filter(ENR_RATIO>1.25)
```



#### JUNB
```{r JUNB}

# motifs_JUNB = query(query(MotifDb, 'Hsapiens'), 'JUNB')
#     JUNB_motif <- motifs_JUNB$`Hsapiens-jaspar2022-JUNB-MA0490.2`  
# # JUNBlist <- seq_all_cd_200[5:6]
# JUNB_pwm = PWMatrix(
# ID = 'JUNB',
# profileMatrix = JUNB_motif)
# JUNB_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=JUNB_pwm, min.score="80%", strand="*")
# # 

# JUNB_storage_cd <- map_dfr(JUNB_test, ~as.data.frame(.x), .id="id")
# 
# saveRDS(JUNB_storage_cd, "data/Final_four_data/JUNB_motif_8group200_cd.RDS")
JUNB_motif_8group <- readRDS("data/Final_four_data/JUNB_motif_8group200_cd.RDS")

JUNB_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
   dplyr::filter(id =="LR_open"|id == "NR"|id=="ESR_open"|id=="ESR_clop") %>%
  # dplyr::filter(relScore>0.90) %>%
  # dplyr::filter(id != "ESR_OC") %>% 
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("JUNB motif")
spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"JUNB")) %>% 
  dplyr::filter(motif_name =="JUNB") %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 
  dplyr::filter(ENR_RATIO>1.25)
```


#### MEIS2
```{r MEIS2}

# motifs_MEIS2 = query(query(MotifDb, 'Hsapiens'), 'MEIS2')
#       MEIS2_motif <- motifs_MEIS2$`Hsapiens-jaspar2022-MEIS2-MA0774.1`
# # # MEIS2list <- seq_all_cd_200[5:6]
# MEIS2_pwm = PWMatrix(
# ID = 'MEIS2',
# profileMatrix = MEIS2_motif)
# MEIS2_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=MEIS2_pwm, min.score="80%", strand="*")
# # 

# MEIS2_storage_cd <- map_dfr(MEIS2_test, ~as.data.frame(.x), .id="id")
# 
# saveRDS(MEIS2_storage_cd, "data/Final_four_data/MEIS2_motif_8group200_cd.RDS")
MEIS2_motif_8group <- readRDS("data/Final_four_data/MEIS2_motif_8group200_cd.RDS")

MEIS2_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="LR_close"|id == "NR"|id=="ESR_close"|id=="ESR_D") %>%
  # dplyr::filter(relScore>0.90) %>%
  # dplyr::filter(id != "ESR_OC") %>% 
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("MEIS2 motif")

spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"MEIS2^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 
  dplyr::filter(ENR_RATIO>1.2)
```


#### NFIB
```{r NFIB}

# motifs_NFIB = query(query(MotifDb, 'Hsapiens'), 'NFIB')
#       NFIB_motif <- motifs_NFIB$`Hsapiens-jaspar2022-NFIB-MA1643.1`
#       
# # NFIBlist <- seq_all_cd_200[5:6]
# NFIB_pwm = PWMatrix(
# ID = 'NFIB',
# profileMatrix = NFIB_motif)
# NFIB_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=NFIB_pwm, min.score="80%", strand="*")
# # 

# NFIB_storage_cd <- map_dfr(NFIB_test, ~as.data.frame(.x), .id="id")
# # 
# saveRDS(NFIB_storage_cd, "data/Final_four_data/NFIB_motif_8group200_cd.RDS")
NFIB_motif_8group <- readRDS("data/Final_four_data/NFIB_motif_8group200_cd.RDS")

NFIB_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id =="ESR_close"|id == "NR"|id=="ESR_D") %>%
  # dplyr::filter(relScore>0.90) %>%
  # dplyr::filter(id != "ESR_OC") %>% 
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("NFIB motif")

spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"NFIB^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`)%>% 
  dplyr::filter(ENR_RATIO>1.25)
```



#### GATA4
```{r GATA4}

# # motifs_GATA4 = query(query(MotifDb, 'Hsapiens'), 'GATA4')
#       # GATA4_motif <- motifs_GATA4$`Hsapiens-jaspar2022-GATA4-MA0482.2`
#       
# # # GATA4list <- seq_all_cd_200[5:6]
# # GATA4_pwm = PWMatrix(
# #   ID = 'GATA4',
# #   profileMatrix = GATA4_motif)
# 
# # GATA4_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=GATA4_pwm, min.score="80%", strand="*")
# # # 
# 
# GATA4_storage_cd <- map_dfr(GATA4_test, ~as.data.frame(.x), .id="id")
# #
# saveRDS(GATA4_storage_cd, "data/Final_four_data/GATA4_motif_8group200_cd.RDS")
GATA4_motif_8group <- readRDS("data/Final_four_data/GATA4_motif_8group200_cd.RDS")

GATA4_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id == "NR"|id=="EAR_close") %>%
  # dplyr::filter(relScore>0.90) %>%
  # dplyr::filter(id != "ESR_OC") %>% 
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("GATA4 motif")

spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"GATA4^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 

  dplyr::filter(ENR_RATIO>1.25)
```



#### MAFG
```{r MAFG}

# motifs_MAFG = query(query(MotifDb, 'Hsapiens'), 'MAFG')
#       MAFG_motif <- motifs_MAFG$`Hsapiens-jaspar2018-MAFG-MA0659.1`
#       
# # # MAFGlist <- seq_all_cd_200[5:6]
# MAFG_pwm = PWMatrix(
#   ID = 'MAFG',
#   profileMatrix = MAFG_motif)
# 
# MAFG_test = lapply(unlist(seq_all_cd_200) , FUN=searchSeq, x=MAFG_pwm, min.score="80%", strand="*")
# # # # 
# # 
# MAFG_storage_cd <- map_dfr(MAFG_test, ~as.data.frame(.x), .id="id")
# # #
# saveRDS(MAFG_storage_cd, "data/Final_four_data/MAFG_motif_8group200_cd.RDS")
MAFG_motif_8group <- readRDS("data/Final_four_data/MAFG_motif_8group200_cd.RDS")

MAFG_motif_8group %>%
  mutate(id= gsub("_resized","",id)) %>% 
  dplyr::filter(id == "NR"|id=="LR_open"|id=="ESR_D") %>%
  # dplyr::filter(relScore>0.90) %>%
  # dplyr::filter(id != "ESR_OC") %>% 
  mutate(position=start-200) %>%
  ggplot(.,aes(position, color = id))+
  geom_density(trim = FALSE,size=1.5) +
  theme_bw() +
  scale_color_manual(values=mrc_palette)+
  ggtitle("MAFG motif")

spec_dataframe_200 %>% 
  dplyr::filter(stringr::str_detect( motif_name,"Mafg^*")) %>% 
  dplyr::select(RANK ,CLUSTER,mrc,motif_name,ID:CONSENSUS,EVALUE.x,ENR_RATIO,`TP%`) %>% 

  dplyr::filter(ENR_RATIO>1.25)
```


```{r making mapping files, eval=FALSE, include=FALSE}
lines <- readLines("C:/Users/renee/Downloads/Supplements folde manuscriptr/outputfrom centrimo/EAR_close_site_counts.txt")

header_lines <- grep("[a-zA-Z]", lines)

header_lines <- c(header_lines, length(lines) + 1)


# Step 4: Iterate over each section and save the data into separate files
for(i in seq_along(header_lines[-length(header_lines)])) {
  start_line <- header_lines[i]  # Start at the header line
  
  end_line <- header_lines[i + 1] - 1 # End before the next header
  

  # Extract the relevant lines
  section_data <- lines[start_line:end_line]
  
  # Create a file name
  string<- section_data[1]
  parts <- strsplit(string, "\t")[[1]]
  # Extract the part after the second tab (third element)
  result <- parts[2]
  file_name <- paste0("data/output_",i,"_",result,".txt")
  
  # Save the section to a file
  writeLines(section_data, con = file_name)
}

```

