---
title: "Peak_Calling"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
editor_options: 
  markdown: 
    wrap: sentence
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(ATACseqQC)
library(rtracklayer)
# library(motifStack)
library(ATACseqQC)
library(TxDb.Hsapiens.UCSC.hg38.knownGene)
library(BSgenome.Hsapiens.UCSC.hg38)
library(MotifDb)
library(ChIPpeakAnno)
```

```{r data_loading and other functions}
drug_pal_fac <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene

loadFile_peakCall <- function(){
 file <- choose.files()
 file <- readPeakFile(file, header = FALSE)
 return(file)
}

prepGRangeObj <- function(seek_object){
 seek_object$Peaks = seek_object$V4
 seek_object$level = seek_object$V5
 seek_object$V4 = seek_object$V5 = NULL
 return(seek_object)
}
TSS = getBioRegion(TxDb=txdb, upstream=2000, downstream=2000, by = "gene", 
                   type = "start_site")

# ind4_V24hpeaks <- readRDS("data/ind4_V24hpeaks.RDS")
# ind1_DA24hpeaks <- readRDS("data/ind1_DA24hpeaks.RDS")
# anno_ind4_V24h <- readRDS("data/anno_ind4_V24h.RDS")
# anno_ind1_DA24h <- readRDS("data/anno_ind1_DA24h.RDS")


Ind1_summary  <- read.csv("data/Ind1_summary.txt", row.names = 1) %>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind2_summary  <- read.csv("data/Ind2_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind3_summary  <- read.csv("data/Ind3_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind4_summary  <- read.csv("data/Ind4_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind5_summary  <- read.csv("data/Ind5_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind6_summary  <- read.csv("data/Ind6_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
```

This specific page so far contains the QC analysis after calling peaks using MACS2.

Primary scripts used for ATAC data preprocesing will be linked here in the future:

-   Â Currently the steps were are as follows:

    -   Basic Fastqc followed by adapter trimming and Fastqc analysis on the leftover fragments.

    -   Trimmed reads were aligned to the hg38 human genome.

    -   Mitochondrial reads (chrM) were removed

    -   samtools was used to removed non-paired, discordantly paired, and multi-mapped reads from the .bam.

    -   Markduplicates function from Picard was used to mark optical and PCR duplicates, with samtools used to remove these reads using the flag -F 1024.

    -   MACS2 was used to call peaks, with QC of the peak files below.
    
Initial read summary is found [at this LINK](https://reneeisnowhere.github.io/ATAC_learning/Fastqc_results.html)




```{r Readssummary1, eval=FALSE, include=FALSE}
 # library(readr) pulling in original files example code

# Ind1_summary_RIP <- read_delim("~/ATAC_downloads/Ind1_files.txt.summary",
#      delim = "\t", escape_double = FALSE,
#    trim_ws = TRUE)
# Ind2_summary_RIP <- read_delim("~/ATAC_downloads/Ind2_files.txt.summary",
#      delim = "\t", escape_double = FALSE,
#      trim_ws = TRUE)
# Ind3_summary_RIP <- read_delim("~/ATAC_downloads/Ind3_files.txt.summary",
#      delim = "\t", escape_double = FALSE,
#      trim_ws = TRUE)
# Ind4_summary_RIP <- read_delim("~/ATAC_downloads/Ind4_files.txt.summary",
#                                delim = "\t", escape_double = FALSE,
#      trim_ws = TRUE)
# Ind5_summary_RIP <- read_delim("~/ATAC_downloads/Ind5_files.txt.summary",
#      delim = "\t", escape_double = FALSE,
#      trim_ws = TRUE)
# Ind6_summary_RIP <- read_delim("~/ATAC_downloads/Ind6_files.txt.summary",
#      delim = "\t", escape_double = FALSE,
#      trim_ws = TRUE)
# write.csv(Ind6_summary, "data/Ind6_summary.txt")
# Ind6_Frip <-
# Ind6_summary_RIP %>%  
#   as.data.frame() %>% 
#   dplyr::filter(Status %in% c("Assigned","Unassigned_NoFeatures","Unassigned_Ambiguity")) %>% 
#   pivot_longer(cols=!Status, names_to = "sample", values_to = "counts") %>% 
#   separate(sample, into=c("indv",NA,NA, NA,NA,NA,"sample")) %>% 
#   pivot_wider(id_cols = c(indv,sample), names_from = Status, values_from = counts) %>% 
#   mutate(total_reads=Assigned+Unassigned_NoFeatures+Unassigned_Ambiguity) %>% 
#   mutate(FRiP= Assigned/total_reads*100)
# FRiP_first_run <- rbind(Ind1_Frip, Ind2_Frip, Ind3_Frip, Ind4_Frip, Ind5_Frip, Ind6_Frip)
#  write.csv(FRiP_first_run, "data/FRiP_first_run.txt")
#   
#   
#   separate(reads,into=c("reads",NA),sep= " ") %>% 
#    mutate(reads=as.numeric(reads)) %>% 
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
#    dplyr::select(sample,type,  reads, mapped_reads)%>% 
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
#    unite("type",type:read_info, sep="_") %>% 
#    distinct() %>% 
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
#   mutate(dedup_read_pairs=dedup_reads/2) %>% 
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
#  
#  # dedup_mapped_reads)
#  Ind1_reads_summary %>% 
#    kable(., caption= "Reads summary from Ind 1") %>%
#   kable_paper("striped", full_width = FALSE) %>%
#   kable_styling(full_width = FALSE,font_size = 18) %>%
#   scroll_box(width = "100%", height = "400px")
# 
# Ind1_reads_summary %>% 
#   pivot_longer(cols = total_reads:dedup_read_pairs, names_to="type", values_to = "reads") %>% 
#   dplyr::filter(type %in% list("total_reads","total_mapped_reads","nuclear_mapped_reads", "unique_mapped_reads","dedup_reads")) %>% 
#   mutate(type=factor(type, levels=c("total_reads", "total_mapped_reads", "nuclear_mapped_reads","unique_mapped_reads","dedup_reads"))) %>% 
#   mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
#   # mutate(trt=substr(trt,-1,2))
#   mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
#    mutate(time=factor(time, levels=c("3_hours","24_hours"))) %>% 
#   mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
#   ggplot(., aes (x=type, y=reads, group = time)) +
#   geom_col(position="dodge",aes(fill=time))+
#   facet_wrap(~trt)+
#   theme(axis.text.x=element_text(angle=90))+
# #   ggtitle("Ind 1")
# #   
# embryo_heart_Frip <-
# embryo_heart %>%
#   dplyr::rename("sample"=X1,"embryo_counts"= X2) %>%
# separate(sample, into=c(NA,NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"sample")) %>%
#   mutate(sample=gsub("75","1_",sample)) %>%
#   mutate(sample=gsub("87","2_",sample)) %>%
#   mutate(sample=gsub("77","3_",sample)) %>%
#   mutate(sample=gsub("79","4_",sample)) %>%
#   mutate(sample=gsub("78","5_",sample)) %>%
#   mutate(sample=gsub("71","6_",sample)) %>%
#   mutate(sample = gsub("24h","_24h",sample),
#        sample = gsub("3h","_3h",sample)) %>%
#   separate(sample, into = c("indv","trt","time"), sep= "_") %>%
#    mutate(trt= case_match(trt, 'DX' ~'DOX', 'E'~'EPI', 'DA'~'DNR', 'M'~'MTX', 'T'~'TRZ', 'V'~'VEH',.default = trt))
# write.csv(cardiac_muscle_Frip,"data/cardiac_muscle_FRIP.csv")
# write.csv(cardiomyocyte_Frip,"data/cardiomyocyte_FRIP.csv")
# write.csv(left_ventricle_Frip,"data/left_ventricle_FRIP.csv")
#  write.csv(embryo_heart_Frip,"data/embryo_heart_FRIP.csv")

```

### Individual 1 fragment files:

```{r fragfiles1}

Ind1_frag_files <- read.csv("data/Ind1_fragment_files.txt", row.names = 1)
 
Ind1_firstfrag_files <- read.csv("data/Ind1_firstfragment_files.txt", row.names = 1)
Ind1_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)+
  coord_cartesian(ylim=c(0,300000))

Ind1_firstfrag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=(counts), x=(frag_size), group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n3 hour fragment sizes BEFORE filtering")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)+
  coord_cartesian(xlim=c(0,1000))
# 

Ind1_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=(counts), x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)#+
  # coord_cartesian(ylim=c(0,300000))



Ind1_firstfrag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=(counts), x=(frag_size), group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n24 hour fragment sizes BEFORE filtering")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)+
  coord_cartesian(xlim=c(0,1000))

# 
```
#### FRiP Individual 1

```{r Frip1}
cardiac_muscle_Frip <- read.csv("data/cardiac_muscle_FRIP.csv", row.names = 1)
cardiomyocyte_Frip <- read.csv("data/cardiomyocyte_FRIP.csv", row.names = 1)
left_ventricle_Frip <- read.csv("data/left_ventricle_FRIP.csv", row.names = 1)
embryo_heart_Frip <- read.csv("data/embryo_heart_FRIP.csv", row.names = 1)
# Ind1_summary  <- read.csv("data/Ind1_summary.txt", row.names = 1) %>% 
  # dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)

# Frip_1_reads <- Ind1_frip %>% 
#   rename(X1 ="sample", X2 ="counts_in_peaks") %>% 
#   separate(sample, into= c(NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"indv","sample")) %>% 
#   full_join(Ind1_reads_summary) %>% 
#   mutate(FRiP_1= counts_in_peaks/dedup_reads*100)
# write.csv(Frip_1_reads,"data/Frip_1_reads.csv")
Frip_1_reads <- read.csv("data/Frip_1_reads.csv", row.names = 1)

all_frip1 <- Frip_1_reads %>% 
 mutate(sample=gsub("75","1_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
   left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)

# Ind1_reads_frip <- 
# Ind1_summary %>%                          
#  separate(reads,into=c("reads",NA),sep= " ") %>% 
#    mutate(reads=as.numeric(reads)) %>% 
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
#    dplyr::select(sample,type,  reads, mapped_reads)%>% 
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
#    unite("type",type:read_info, sep="_") %>% 
#    distinct() %>% 
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
#   mutate(dedup_read_pairs=dedup_reads/2) %>% 
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs) %>% 
#   dplyr::select(sample, dedup_reads) %>% 


Frip_1_reads %>% 
  mutate(sample=gsub("75","1_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_1, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 1")+
   scale_fill_manual(values=drug_pal_fac)
 


FRiP_first_run <- read.csv("data/FRiP_first_run.txt", row.names = 1)
FRiP_first_run %>% 
  mutate(sample=gsub("75","1_",sample)) %>% 
  mutate(sample=gsub("87","2_",sample)) %>% 
  mutate(sample=gsub("77","3_",sample)) %>% 
  mutate(sample=gsub("79","4_",sample)) %>% 
  mutate(sample=gsub("78","5_",sample)) %>% 
  mutate(sample=gsub("71","6_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x=time, y=FRiP, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 sample")+
   scale_fill_manual(values=drug_pal_fac)

all_frip1 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)
  
all_frip1 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip1 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip1 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)




```

Horizontal line is at 20%.  Individual 4 at 3 hours is not a good score.  Encode recommends >30% (dotted line).




### Individual 2 fragment files:

```{r fragfiles2}

Ind2_frag_files <- read.csv("data/Ind2_fragment_files.txt", row.names = 1)
Ind2_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 2\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
Ind2_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 2\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
```


#### FRiP Individual 2

```{r Frip2}

# Ind2_summary  <- read.csv("data/Ind2_summary.txt", row.names = 1) %>% 
#   dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
# 
# Ind2_reads_summary <- 
# Ind2_summary %>%                          
#  separate(reads,into=c("reads",NA),sep= " ") %>% 
#    mutate(reads=as.numeric(reads)) %>% 
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
#    dplyr::select(sample,type,  reads, mapped_reads)%>% 
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
#    unite("type",type:read_info, sep="_") %>% 
#    distinct() %>% 
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
#   mutate(dedup_read_pairs=dedup_reads/2) %>% 
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs) %>% 
#   dplyr::select(sample, dedup_reads) 
# Ind2_frip <- read_delim("~/ATAC_downloads/Ind1/trimmed/Ind2_frip.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# 
# Frip_2_reads <- Ind2_frip %>%
#   rename(X1 ="sample", X2 ="counts_in_peaks") %>%
#   separate(sample, into= c(NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"indv","sample")) %>%
#   full_join(Ind2_reads_summary) %>%
#   mutate(FRiP_2= counts_in_peaks/dedup_reads*100)
# write.csv(Frip_2_reads,"data/Frip_2_reads.csv")
Frip_2_reads <- read.csv("data/Frip_2_reads.csv", row.names = 1)

all_frip2 <- Frip_2_reads %>% 
  mutate(sample=gsub("87","2_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)

Frip_2_reads %>% 
 mutate(sample=gsub("87","2_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_2, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 2")+
   scale_fill_manual(values=drug_pal_fac)



all_frip2 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)
  
all_frip2 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2  using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip2 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2  using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip2 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)





```

### Individual 3 fragment files:

```{r fragfiles3}

Ind3_frag_files <- read.csv("data/Ind3_fragment_files.txt", row.names = 1)
Ind3_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 3\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
Ind3_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 3\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
```


#### FRiP Individual 3

```{r Frip3}

# Ind3_summary  <- read.csv("data/Ind3_summary.txt", row.names = 1) %>% 
#   dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
# 
# Ind3_reads_summary <- 
# Ind3_summary %>%                          
#  separate(reads,into=c("reads",NA),sep= " ") %>% 
#    mutate(reads=as.numeric(reads)) %>% 
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
#    dplyr::select(sample,type,  reads, mapped_reads)%>% 
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
#    unite("type",type:read_info, sep="_") %>% 
#    distinct() %>% 
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
#   mutate(dedup_read_pairs=dedup_reads/2) %>% 
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs) %>% 
#   dplyr::select(sample, dedup_reads) 
# Ind3_frip <- read_delim("~/ATAC_downloads/Ind1/trimmed/Ind3_frip.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# 
# Frip_3_reads <- Ind3_frip %>%
#   rename(X1 ="sample", X2 ="counts_in_peaks") %>%
#   separate(sample, into= c(NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"indv","sample")) %>%
#   full_join(Ind3_reads_summary) %>%
#   mutate(FRiP_3= counts_in_peaks/dedup_reads*100)
# write.csv(Frip_3_reads,"data/Frip_3_reads.csv")
Frip_3_reads <- read.csv("data/Frip_3_reads.csv", row.names = 1)

all_frip3 <- Frip_3_reads %>% 
  mutate(sample=gsub("77","3_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
 mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)






Frip_3_reads %>% 
  mutate(sample=gsub("77","3_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_3, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 3")+
   scale_fill_manual(values=drug_pal_fac)



all_frip3 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)
  
all_frip3 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip3 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip3 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)





```

### Individual 4 fragment files:

```{r fragfiles visualize}

Ind4_frag_files <- read.csv("data/Ind4_fragment_files.txt", row.names = 1)
Ind4_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt ))+
  ggtitle("Individual 4\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
Ind4_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt ))+
  ggtitle("Individual 4\n24 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)


```
#### FRiP Individual 4

```{r Frip4}

# Ind4_summary  <- read.csv("data/Ind4_summary.txt", row.names = 1) %>% 
#   dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
# 
# Ind4_reads_summary <- 
# Ind4_summary %>%                          
#  separate(reads,into=c("reads",NA),sep= " ") %>% 
#    mutate(reads=as.numeric(reads)) %>% 
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
#    dplyr::select(sample,type,  reads, mapped_reads)%>% 
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
#    unite("type",type:read_info, sep="_") %>% 
#    distinct() %>% 
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
#   mutate(dedup_read_pairs=dedup_reads/2) %>% 
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs) %>% 
#   dplyr::select(sample, dedup_reads) 
# Ind4_frip <- read_delim("~/ATAC_downloads/Ind1/trimmed/Ind4_frip.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# 
# Frip_4_reads <- Ind4_frip %>%
#   rename(X1 ="sample", X2 ="counts_in_peaks") %>%
#   separate(sample, into= c(NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"indv","sample")) %>%
#   full_join(Ind4_reads_summary) %>%
#   mutate(FRiP_4= counts_in_peaks/dedup_reads*100)
# write.csv(Frip_4_reads,"data/Frip_4_reads.csv")
Frip_4_reads <- read.csv("data/Frip_4_reads.csv", row.names = 1)

all_frip4 <- Frip_4_reads %>% 
  mutate(sample=gsub("79","4_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%   
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)
Frip_4_reads %>% 
  mutate(sample=gsub("79","4_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_4, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 4")+
   scale_fill_manual(values=drug_pal_fac)


all_frip4 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind4 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)
  
all_frip4 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score  across Ind4 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip4 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score  across Ind4 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip4 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind4 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)



```

### Individual 5 fragment files:

```{r fragfiles5}

Ind5_frag_files <- read.csv("data/Ind5_fragment_files.txt", row.names = 1)
Ind5_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 5\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
Ind5_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 5\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
```

#### FRiP Individual 5

```{r Frip5}
# 
# Ind5_summary  <- read.csv("data/Ind5_summary.txt", row.names = 1) %>%
#   dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
# 
# Ind5_reads_summary <-
# Ind5_summary %>%
#  separate(reads,into=c("reads",NA),sep= " ") %>%
#    mutate(reads=as.numeric(reads)) %>%
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>%
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>%
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>%
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>%
#    dplyr::select(sample,type,  reads, mapped_reads)%>%
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>%
#    unite("type",type:read_info, sep="_") %>%
#    distinct() %>%
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>%
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>%
#   mutate(dedup_read_pairs=dedup_reads/2) %>%
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs) %>%
#   dplyr::select(sample, dedup_reads)
# Ind5_frip <- read_delim("~/ATAC_downloads/Ind1/trimmed/Ind5_frip.txt",
#     delim = "\t", escape_double = FALSE,
#     col_names = FALSE, trim_ws = TRUE)
# 
# Frip_5_reads <- Ind5_frip %>%
#   rename(X1 ="sample", X2 ="counts_in_peaks") %>%
#   separate(sample, into= c(NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"indv","sample")) %>%
#   full_join(Ind5_reads_summary) %>%
#   mutate(FRiP_5= counts_in_peaks/dedup_reads*100)
# write.csv(Frip_5_reads,"data/Frip_5_reads.csv")
Frip_5_reads <- read.csv("data/Frip_5_reads.csv", row.names = 1)

all_frip5 <- Frip_5_reads %>% 
  mutate(sample=gsub("78","5_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
 mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%   
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)
 
 
Frip_5_reads %>% 
  mutate(sample=gsub("78","5_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_5, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score a across Ind5 for individual 5")+
   scale_fill_manual(values=drug_pal_fac)

all_frip5 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score  across Ind5 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)
  
all_frip5 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score  across Ind5 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip5 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip5 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind5 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

```


### Individual 6 fragment files:

```{r fragfiles6}

Ind6_frag_files <- read.csv("data/Ind6_fragment_files.txt", row.names = 1)
Ind6_frag_files %>% 
  dplyr::filter(time =="3h") %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 6\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
Ind6_frag_files %>% 
  dplyr::filter(time =="24h") %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 6\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fac)
```
#### FRiP Individual 6

```{r Frip6}
# 
# Ind6_summary  <- read.csv("data/Ind6_summary.txt", row.names = 1) %>%
#   dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)
# 
# Ind6_reads_summary <-
# Ind6_summary %>%
#  separate(reads,into=c("reads",NA),sep= " ") %>%
#    mutate(reads=as.numeric(reads)) %>%
#    separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>%
#    separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>%
#    mutate(mapped_reads=as.numeric(mapped_reads)) %>%
#    mutate(type = if_else(two=="first", "total",
#                          if_else(two=="noM","nuclear",
#                                  if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>%
#    dplyr::select(sample,type,  reads, mapped_reads)%>%
#    pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>%
#    unite("type",type:read_info, sep="_") %>%
#    distinct() %>%
#    pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>%
#    mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>%
#   mutate(dedup_read_pairs=dedup_reads/2) %>%
#    dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs) %>%
#   dplyr::select(sample, dedup_reads)
# 
# Ind6_frip <- read_delim("~/ATAC_downloads/Ind1/trimmed/Ind6_frip.txt",
#     delim = "\t", escape_double = FALSE,
#     col_names = FALSE, trim_ws = TRUE)
# 
# Frip_6_reads <- Ind6_frip %>%
#   rename(X1 ="sample", X2 ="counts_in_peaks") %>%
#   separate(sample, into= c(NA,NA,NA,NA,NA,NA, NA,NA,NA,NA,"indv","sample")) %>%
#   full_join(Ind6_reads_summary) %>%
#   mutate(FRiP_6= counts_in_peaks/dedup_reads*100)
# write.csv(Frip_6_reads,"data/Frip_6_reads.csv")

Frip_6_reads <- read.csv("data/Frip_6_reads.csv", row.names = 1)

all_frip6 <- Frip_6_reads %>% 
  mutate(sample=gsub("71","6_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)
 

Frip_6_reads %>% 
  mutate(sample=gsub("71","6_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_6, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 6")+
   scale_fill_manual(values=drug_pal_fac)

all_frip6 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score  across Ind6 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)
  
all_frip6 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind6 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip6 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind6 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)

all_frip6 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind6 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal_fac)




```


### Peak data


```{r peak info}
##collecting summary files

# Ind1_peaksummary <- read_table("~/ATAC_downloads/Ind1/trimmed/macs_output/Ind1_peaksummary.txt",
#     col_names = FALSE) %>%
#   rename(X1 ="counts", X2= "sample")
# Ind2_peaksummary <- read_table("~/ATAC_downloads/Ind2/trimmed/macs_output/Ind2_peaksummary.txt",
#     col_names = FALSE)%>%
#   rename(X1 ="counts", X2= "sample")
# Ind3_peaksummary <- read_table("~/ATAC_downloads/Ind3/trimmed/macs_output/Ind3_peaksummary.txt",
#     col_names = FALSE)%>%
#   rename(X1 ="counts", X2= "sample")
# Ind4_peaksummary <- read_table("~/ATAC_downloads/Ind4/trimmed/macs_output/Ind4_peaksummary.txt",
#     col_names = FALSE)%>%
#   rename(X1 ="counts", X2= "sample")
# Ind5_peaksummary <- read_table("~/ATAC_downloads/Ind5/trimmed/macs_output/Ind5_peaksummary.txt",
#     col_names = FALSE)%>%
#   rename(X1 ="counts", X2= "sample")
# Ind6_peaksummary <- read_table("~/ATAC_downloads/Ind6/trimmed/macs_output/Ind6_peaksummary.txt",
#     col_names = FALSE)%>%
#   rename(X1 ="counts", X2= "sample")
# Peaksummary <- rbind(Ind1_peaksummary,Ind2_peaksummary,Ind3_peaksummary,Ind4_peaksummary,Ind5_peaksummary,Ind6_peaksummary)
# # 
# write.csv(Peaksummary, "data/first_Peaksummarycounts.csv")
Peaksummary <- read.csv("data/first_Peaksummarycounts.csv",row.names=1)
Peaksummary %>% 
  dplyr::filter(sample != "total") %>% 
   separate(sample, into=c(NA,"indv","sample",NA,NA,NA)) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  mutate(indv = factor(indv, levels = c("Ind1", "Ind2", "Ind3", "Ind4", "Ind5", "Ind6"))) %>%
  mutate(time = factor(time, levels = c("3_hours", "24_hours"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels =  c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH")))%>% 
  ggplot(., aes(x =trt, y=counts,group=trt))+
  geom_boxplot(aes(fill= trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(~time)+
   scale_fill_manual(values=drug_pal_fac)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("Peak counts by treatment")+
  theme_bw()



```


### Ind1 Peaks

```{r Ind1 inital TSS setup}

# plotAnnoBar(anno_ind4_V24h, main = "Genomic Feature Distribution")+ ggtitle("Ind4 VEH 24 hour")
# plotAnnoBar(anno_ind1_DA24h, main = "Genomic Feature Distribution")+ ggtitle("Ind1 DNR 24 hour")

# ind4_V24hpeaks_gr <- prepGRangeObj(ind4_V24hpeaks)
# ind1_DA24hpeaks_gr <- prepGRangeObj((ind1_DA24hpeaks))
# Epi_list <- GRangesList(ind1_DA24hpeaks_gr, ind4_V24hpeaks_gr)
# # ##plotting the TSS average window (making an overlap of each using Epi_list as list holder)
# Epi_list_tagMatrix = lapply(Epi_list, getTagMatrix, windows = TSS)
# plotAvgProf(Epi_list_tagMatrix, xlim=c(-3000, 3000), ylab = "Count Frequency")
#plotPeakProf(Epi_list_tagMatrix, facet = "none", conf = 0.95)

## What I did here:  I called all my narrowpeak files
# peakfiles1 <- choose.files()

##these were practice for getting file names and shortening for the for loop below
# testname <- basename(peakfiles1[1])
# str_split_i(testname, "_",3)

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind1_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles1[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind1_peaks[[banana_peel]] <- readPeakFile(peakfiles1[file])
# }
# saveRDS(Ind1_peaks, "data/Ind1_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)
# peakAnnoList_1 <- lapply(Ind1_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_1, "data/peakAnnoList_1.RDS")

peakAnnoList_1 <- readRDS("data/peakAnnoList_1.RDS")
plotAnnoBar(peakAnnoList_1, main = "Genomic Feature Distribution")


# saveRDS(Epi_list_tagMatrix, "data/Ind1_TSS_peaks.RDS")

Ind1_TSS_peaks_plot <- readRDS("data/Ind1_TSS_peaks.RDS")

# Epi_list_tagMatrix = lapply(Ind1_peaks, getTagMatrix, windows = TSS)

plotAvgProf(Ind1_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 1" )
# + coord_cartesian(xlim = c(-100,500))

plotAvgProf(Ind1_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 1" )
# + coord_cartesian(xlim = c(-100,500))

```


### Ind2 Peaks

```{r Ind2 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles2 <- choose.files()

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind2_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles2[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind2_peaks[[banana_peel]] <- readPeakFile(peakfiles2[file])
# }
# saveRDS(Ind2_peaks, "data/Ind2_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind2_peaks <- readRDS("data/Ind2_peaks_list.RDS")
# peakAnnoList_2 <- lapply(Ind2_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_2, "data/peakAnnoList_2.RDS")

peakAnnoList_2 <- readRDS("data/peakAnnoList_2.RDS")
plotAnnoBar(peakAnnoList_2, main = "Genomic Feature Distribution, Individual 2")

# Epi_list_tagMatrix = lapply(Ind2_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind2_TSS_peaks.RDS")

Ind2_TSS_peaks_plot <- readRDS("data/Ind2_TSS_peaks.RDS")



plotAvgProf(Ind2_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), conf = 0.01, ylab = "Count Frequency")+ ggtitle("3 hour Individual 2" )
# + coord_cartesian(xlim = c(-100,1000))

plotAvgProf(Ind2_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 2" )#+ coord_cartesian(xlim = c(-100,500))

```


### Ind3 Peaks

```{r Ind3 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles3 <- choose.files()

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind3_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles3[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind3_peaks[[banana_peel]] <- readPeakFile(peakfiles3[file])
# }
# saveRDS(Ind3_peaks, "data/Ind3_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind3_peaks <- readRDS("data/Ind3_peaks_list.RDS")
# peakAnnoList_3 <- lapply(Ind3_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_3, "data/peakAnnoList_3.RDS")

peakAnnoList_3 <- readRDS("data/peakAnnoList_3.RDS")
plotAnnoBar(peakAnnoList_3, main = "Genomic Feature Distribution, Individual 3")

# Epi_list_tagMatrix = lapply(Ind3_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind3_TSS_peaks.RDS")

Ind3_TSS_peaks_plot <- readRDS("data/Ind3_TSS_peaks.RDS")



plotAvgProf(Ind3_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 3" )

plotAvgProf(Ind3_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 3" )

```


### Ind4 Peaks

```{r Ind4 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles4 <- choose.files()
# 
# ##This loop first established a list then (because I already knew the list had 12 files)
# ## I then imported each of these onto that list.  Once I had the list, I stored it as
# ## an R object, 
# Ind4_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles4[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind4_peaks[[banana_peel]] <- readPeakFile(peakfiles4[file])
# }
# saveRDS(Ind4_peaks, "data/Ind4_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind4_peaks <- readRDS("data/Ind4_peaks_list.RDS")
# peakAnnoList_4 <- lapply(Ind4_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_4, "data/peakAnnoList_4.RDS")

peakAnnoList_4 <- readRDS("data/peakAnnoList_4.RDS")
plotAnnoBar(peakAnnoList_4, main = "Genomic Feature Distribution, Individual 4")

# Epi_list_tagMatrix = lapply(Ind4_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind4_TSS_peaks.RDS")

Ind4_TSS_peaks_plot <- readRDS("data/Ind4_TSS_peaks.RDS")



plotAvgProf(Ind4_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 4" )

plotAvgProf(Ind4_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 4" )

```

### Ind5 Peaks
```{r Ind5 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles4 <- choose.files()
# # 
# # ##This loop first established a list then (because I already knew the list had 12 files)
# # ## I then imported each of these onto that list.  Once I had the list, I stored it as
# ## an R object,
# Ind4_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles4[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind4_peaks[[banana_peel]] <- readPeakFile(peakfiles4[file])
# }
# saveRDS(Ind4_peaks, "data/Ind4_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind5_peaks <- readRDS("data/Ind5_peaks_list.RDS")
# peakAnnoList_5 <- lapply(Ind5_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_5, "data/peakAnnoList_5.RDS")

peakAnnoList_5 <- readRDS("data/peakAnnoList_5.RDS")
plotAnnoBar(peakAnnoList_5, main = "Genomic Feature Distribution, Individual 4")

# Epi_list_tagMatrix = lapply(Ind5_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind5_TSS_peaks.RDS")

Ind5_TSS_peaks_plot <- readRDS("data/Ind5_TSS_peaks.RDS")



plotAvgProf(Ind5_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 5" )

plotAvgProf(Ind5_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 5" )

```



### Ind6 Peaks

```{r Ind6 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles6 <- choose.files()
# 
# ##This loop first established a list then (because I already knew the list had 12 files)
# ## I then imported each of these onto that list.  Once I had the list, I stored it as
# ## an R object, 
# Ind6_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles6[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind6_peaks[[banana_peel]] <- readPeakFile(peakfiles6[file])
# }
# saveRDS(Ind6_peaks, "data/Ind6_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind6_peaks <- readRDS("data/Ind6_peaks_list.RDS")
# peakAnnoList_6 <- lapply(Ind6_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_6, "data/peakAnnoList_6.RDS")

peakAnnoList_6 <- readRDS("data/peakAnnoList_6.RDS")
plotAnnoBar(peakAnnoList_6, main = "Genomic Feature Distribution, Individual 6")+ggtitle ("Genomic Feature Distribution, Individual 6")
##Epi_list_tagMatrix title was just because I was too lazy to change the name
# Epi_list_tagMatrix = lapply(Ind6_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind6_TSS_peaks.RDS")

Ind6_TSS_peaks_plot <- readRDS("data/Ind6_TSS_peaks.RDS")



plotAvgProf(Ind6_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 6" )

plotAvgProf(Ind6_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 6" )


```


```{r bamqc, eval=FALSE, include=FALSE}

bamfile <- choose.files()
bamfile.labels <- gsub(".bam", "", basename(bamfile))
bamfile.sample.ID <- gsub(".bam", "", basename(bamfile))
# estimateLibComplexity(readsDupFreq(bamfile))

fragSize <- fragSizeDist(bamfile, bamfile.labels)

pdf(paste0(bamfile.sample.ID, ".fragment.size.distribution.pdf"), width =10, height=8) 
fragSize <- fragSizeDist(bamFiles=bamfile, bamFiles.labels=bamfile.sample.ID)
dev.off()

# promoter <- getPromoters(TxDb=txdb, upstream=3000, downstream=3000)
## tagMatrixList <- lapply(files, getTagMatrix, windows=promoter)
tags <- c("AS", "NM", "MD")
outPath <- paste0(bamfile.sample.ID, ".splited.bam")
if (!dir.exists(outPath)){
   dir.create(outPath)
}
human.genome <- SeqinfoForUCSCGenome("hg38")
seqlev <- as.character(human.genome[,1])
# which <- human.genome[seqnames(human.genome) %in% c("chr1", "chr2")]
gal <- readBamFile(bamfile, tag=tags,  asMates=TRUE)

gal1 <- shiftGAlignmentsList(gal)
shiftedBamfile <- file.path(outPath, paste0(bamfile,".shifted.bam"))
## Outputting the shifted BAM file
export(gal1, shiftedBamfile)

## Getting information of known transcripts from UCSC genome browser database
txs <- transcripts(TxDb.Hsapiens.UCSC.hg38.knownGene)


## Classifying reads into nucleosome-free, mono-, di- and tri-nucleosome bins based on their fragment sizes.
genome <- Hsapiens
objs <- splitGAlignmentsByCut(gal1, txs=txs, genome=genome)


## outputting split BAM files
null <- writeListOfGAlignments(objs, outPath)

## Generating heatmaps and coverage curves for nucleosome-free, mono-, di- and tri-nucleosome occupied regions

bamfiles <- file.path(outPath,
                      c("NucleosomeFree.bam",
                        "mononucleosome.bam",
                        "dinucleosome.bam",
                        "trinucleosome.bam"))


## extracting TSSs coordinates
TSS <- promoters(txs, upstream=0, downstream=1)
TSS <- unique(TSS)

## estimating the library size for normalization
librarySize <- estLibSize(bamfiles)


## calculating the signals around TSSs.
NTILE <- 101
dws <- ups <- 1010
sigs <- enrichedFragments(bamfiles, TSS=TSS,
                          librarySize=librarySize,
                          seqlev=seqlev,
                          TSS.filter=0.5,
                          n.tile = NTILE,
                          upstream = ups,
                          downstream = dws)
                          
                          
## log2-transforming signals
names(sigs) <- gsub(".bam", "", basename(names(sigs)))
sigs.log2 <- lapply(sigs, function(.ele) log2(.ele+1))

## plotting heatmap showing signals  for nucleosome-free and oligonucleosome-bound regions around TSSs. (Figure 1 H and 1I)
pdf(paste0(bamfile.sample.ID, ".heatmap and averaged coverage.pdf"))
featureAlignedHeatmap(sigs.log2, reCenterPeaks(TSS, width=ups+dws),
                      zeroAt=.5, n.tile=NTILE)

out <- featureAlignedDistribution(sigs, 
                                  reCenterPeaks(TSS, width=ups+dws),
                                  zeroAt=.5, n.tile=NTILE, type="l", 
                                  yalb="Averaged coverage", 
                                  ylab="Averaged coverage")
dev.off()
y


```



```{r eval=FALSE}
possibleTag <- list("integer"=c("AM", "AS", "CM", "CP", "FI", "H0", "H1", "H2", 
                                "HI", "IH", "MQ", "NH", "NM", "OP", "PQ", "SM",
                                "TC", "UQ"), 
                 "character"=c("BC", "BQ", "BZ", "CB", "CC", "CO", "CQ", "CR",
                               "CS", "CT", "CY", "E2", "FS", "LB", "MC", "MD",
                               "MI", "OA", "OC", "OQ", "OX", "PG", "PT", "PU",
                               "Q2", "QT", "QX", "R2", "RG", "RX", "SA", "TS",
                               "U2"))
library(Rsamtools)
bamTop100 <- scanBam(BamFile(bamfile, yieldSize = 100),
                     param = ScanBamParam(tag=unlist(possibleTag)))[[1]]$tag
tags <- names(bamTop100)[lengths(bamTop100)>0]
tags

outPath <- "splited"
dir.create(outPath)
library(TxDb.Hsapiens.UCSC.hg38.knownGene)
seqlev <- "chr1" 
seqinformation <- seqinfo(TxDb.Hsapiens.UCSC.hg38.knownGene)


# which <- as(seqinformation[seqlev], "GRanges")
gal <-  readBamFile(bamFile=bamfile, tag=character(0),
which=GRanges("chr1", IRanges(1, 1e6)),
asMates=TRUE)
shiftedBamfile <- file.path(outPath, "shifted.bam")
gal1 <- shiftGAlignmentsList(gal, outbam=shiftedBamfile)

txs <- transcripts(TxDb.Hsapiens.UCSC.hg38.knownGene)
tsse <- TSSEscore(gal, txs)
tsse$TSSEscore
tsse
summary(tsse$values)
```

## TSSE

```{r TSSE score}

allTSSE <- readRDS( "data/all_TSSE_scores.RDS")
allTSSE %>% as.data.frame() %>% 
  rownames_to_column("sample") %>% 
  separate(sample, into = c("indv","trt","time"), sep= "_") %>%
  mutate(trt= factor(trt, levels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  mutate(time = factor(time, levels = c("3","24"),labels = c("3 hours","24 hours"))) %>% 
  ggplot(., aes(x= time, y= V1, group = indv))+

   # geom_col(position= "dodge",aes(fill=trt)) %>%
  geom_jitter(aes(col = trt, size = 1.5, alpha = 0.5) ,  position=position_jitter(0.25))+
  geom_hline(yintercept=5, linetype = 3)+
    geom_hline(yintercept=7)+
  facet_wrap(~indv)+
   theme_bw()+
  ggtitle("TSS enrichment scores")+
   scale_color_manual(values=drug_pal_fac)







```



## Heatmap of log2 cpm in peaks using MergedPeakfile

```{r bedfile annotation, fig.height=8, fig.width=12}
### bring in the Feature counted files
# Ind1_counts <- read.delim("~/ATAC_downloads/Ind1_files.txt", comment.char="#")
# Ind2_counts <- read.delim("~/ATAC_downloads/Ind2_files.txt", comment.char="#")
# Ind3_counts <- read.delim("~/ATAC_downloads/Ind3_files.txt", comment.char="#")
# Ind4_counts <- read.delim("~/ATAC_downloads/Ind4_files.txt", comment.char="#")
# Ind5_counts <- read.delim("~/ATAC_downloads/Ind5_files.txt", comment.char="#")
# Ind6_counts <- read.delim("~/ATAC_downloads/Ind6_files.txt", comment.char="#")
# 
# first_run_frag_counts <- Ind1_counts %>% 
#   full_join(Ind2_counts) %>% 
# full_join(Ind3_counts) %>% 
# full_join(Ind4_counts) %>% 
# full_join(Ind5_counts) %>% 
# full_join(Ind6_counts) 
# 
# names(first_run_frag_counts) = gsub(pattern = "_S.*", replacement = "", x = names(first_run_frag_counts))
# 
# names(first_run_frag_counts)=gsub(pattern = "^ind6.trimmed.filt_files.trimmed_", replacement = "", x = names(first_run_frag_counts))
# 
# write.csv(first_run_frag_counts,"data/first_run_frag_counts.txt")
first_run_frag_counts <- read.csv("data/first_run_frag_counts.txt", row.names = 1) 

Frag_cor <- first_run_frag_counts %>% 
  dplyr::select(Ind1_75DA24h:Ind6_71V3h) %>% 
  cor()
  
# pheatmap::pheatmap(Frag_cor , cluster_rows = TRUE, cluster_cols = TRUE)



filmat_groupmat_col <- data.frame(timeset = colnames(Frag_cor))

counts_corr_mat <-filmat_groupmat_col %>% 
  mutate(timeset=gsub("75","1_",timeset)) %>% 
  mutate(timeset=gsub("87","2_",timeset)) %>% 
  mutate(timeset=gsub("77","3_",timeset)) %>% 
  mutate(timeset=gsub("79","4_",timeset)) %>% 
  mutate(timeset=gsub("78","5_",timeset)) %>%
  mutate(timeset=gsub("71","6_",timeset)) %>% 
  mutate(timeset = gsub("24h","_24h",timeset), 
       timeset = gsub("3h","_3h",timeset)) %>%
  separate(timeset, into = c(NA,"indv","trt","time"), sep= "_") %>% 
  
  mutate(trt= case_match(trt, 'DX' ~'DOX', 'E'~'EPI', 'DA'~'DNR', 'M'~'MTX', 'T'~'TRZ', 'V'~'VEH',.default = trt)) %>% 
  mutate(class = if_else(trt == "DNR", "AC", if_else(
    trt == "DOX", "AC", if_else(trt == "EPI", "AC", "nAC")
  ))) %>%
  mutate(TOP2i = if_else(trt == "DNR", "yes", if_else(
    trt == "DOX", "yes", if_else(trt == "EPI", "yes", if_else(trt == "MTX", "yes", "no"))
  ))) 
                         
 mat_colors <- list( 
   trt= c("#F1B72B","#8B006D","#DF707E","#3386DD","#707031","#41B333"),
   indv=c("#1B9E77", "#D95F02" ,"#7570B3", "#E7298A" ,"#66A61E", "#E6AB02"),
   time=c("pink", "chocolate4"),
   class=c("yellow1","darkorange1"), 
   TOP2i =c("darkgreen","lightgreen"))                        
                         
names(mat_colors$trt)   <- unique(counts_corr_mat$trt)                      
names(mat_colors$indv) <- unique(counts_corr_mat$indv)
names(mat_colors$time) <- unique(counts_corr_mat$time)
names(mat_colors$class) <- unique(counts_corr_mat$class)
names(mat_colors$TOP2i) <- unique(counts_corr_mat$TOP2i)


ComplexHeatmap::pheatmap(Frag_cor,
                         # column_title=(paste0("RNA-seq log"[2]~"cpm correlation")),
        annotation_col = counts_corr_mat,
        annotation_colors = mat_colors,
        heatmap_legend_param = mat_colors,
        fontsize=10,
        fontsize_row = 8,
        angle_col="90",
        treeheight_row=25,
        fontsize_col = 8,
        treeheight_col = 20)


```

