---
title: "Peak_Calling"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
```

```{r data_loading and other functions}
drug_pal_fact <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene

loadFile_peakCall <- function(){
 file <- choose.files()
 file <- readPeakFile(file, header = FALSE)
 return(file)
}

prepGRangeObj <- function(seek_object){
 seek_object$Peaks = seek_object$V4
 seek_object$level = seek_object$V5
 seek_object$V4 = seek_object$V5 = NULL
 return(seek_object)
}
TSS = getBioRegion(TxDb=txdb, upstream=3000, downstream=3000, by = "gene", 
                   type = "start_site")

ind4_V24hpeaks <- readRDS("data/ind4_V24hpeaks.RDS")
ind1_DA24hpeaks <- readRDS("data/ind1_DA24hpeaks.RDS")
anno_ind4_V24h <- readRDS("data/anno_ind4_V24h.RDS")
anno_ind1_DA24h <- readRDS("data/anno_ind1_DA24h.RDS")


```

This specific page so far contains the QC analysis after calling peaks using MACS2.

Primary scripts used for ATAC data preprocesing will be linked here in the future:

Currently the step were as follows:
1.  Basic Fastqc followed by adapter trimming and Fastqc analysis on the leftover fragments.
2.  Trimmed reads were aligned to the hg38 human genome.
3.  Mitochondrial reads (chrM) were removed
4.  samtools was used to removed non-paired, discordantly paired, and mulimapped reads from the .bam.
5.  Markduplicates function from Picard was used to mark optical and PCR duplicates, with samtools used to remove these reads using the flag -F 1024.
6.  MACS2 was used to call peaks, with QC of the peak files below.


Initial summary of reads after each step:
```{r final import inital}
#  library(readr)
# > Ind4_summary <- read_table("~/ATAC_downloads/Ind4/Ind4_summary.txt", 
# +     col_names = FALSE, col_types = cols(X3 = col_skip(), 
# +         X4 = col_skip(), X5 = col_skip(), 
# +         X6 = col_skip(), X7 = col_skip(), 
# +         X8 = col_skip(), X9 = col_skip(), 
# +         X10 = col_skip(), X11 = col_skip(), 
# +         X13 = col_skip(), X14 = col_skip(), 
# +         X17 = col_skip(), X18 = col_skip()))
#

Ind4_summary  <- read.csv("data/Ind4_summary.txt", row.names = 1)
 Ind4_reads_summary <- Ind4_summary %>%                          
 separate(counts,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(name, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads)
 
 # dedup_mapped_reads)
 Ind4_reads_summary %>% 
   kable(., caption= "Reads summary from Ind_4") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")
   
 
```

**note deduped mapped reads were taken out of this list due to problem with script returning invalid numbers.  The read-numbers for deduplicated read pairs will be added here after reprocessing the files


```{r peak info}
# plotAvgProf(anno_ind4_V24h,xlim=c(-3000,3000))
plotAnnoBar(anno_ind4_V24h, main = "Genomic Feature Distribution")+ ggtitle("Ind4 VEH 24 hour")
plotAnnoBar(anno_ind1_DA24h, main = "Genomic Feature Distribution")+ ggtitle("Ind1 DNR 24 hour")

ind4_V24hpeaks_gr <- prepGRangeObj(ind4_V24hpeaks)
ind1_DA24hpeaks_gr <- prepGRangeObj((ind1_DA24hpeaks))
Epi_list <- GRangesList(ind1_DA24hpeaks_gr, ind4_V24hpeaks_gr)
##plotting the TSS average window (making an overlap of each using Epi_list as list holder)
Epi_list_tagMatrix = lapply(Epi_list, getTagMatrix, windows = TSS)
plotAvgProf(Epi_list_tagMatrix, xlim=c(-3000, 3000), ylab = "Count Frequency")
#plotPeakProf(Epi_list_tagMatrix, facet = "none", conf = 0.95)
```



```{r fragment files list, eval=FALSE, include=FALSE}
### this was used to load fragment files for all individuals for display in the future
safename <- choose.files()
Ind4_79DA3h_frag <- read_delim("~/ATAC_downloads/Ind4/filt_files/frag_length/trimmed_Ind4_79DA3h_S1.nodup.bam.fragment_length_count.txt", 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)%>% separate(col=1, into=c("counts","frag_size")) %>% 
  rename(  "counts"="DA_3h_4_counts")


Ind4_79DA24h_frag <- read_delim(safename[2], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DA_24h_4_counts")


Ind4_79DX3h_frag <- read_delim(safename[3], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DX_3h_4_counts")


Ind4_79DX24h_frag<- read_delim(safename[4], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DX_24h_4_counts")


Ind4_79E3h_frag <- read_delim(safename[5], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="E_3h_4_counts")


Ind4_79E24h_frag <- read_delim(safename[6], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="E_24h_4_counts")

Ind4_79M3h_frag <- read_delim(safename[7], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="M_3h_4_counts")


Ind4_79M24h_frag <- read_delim(safename[8], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="M_24h_4_counts")


Ind4_79T3h_frag <- read_delim(safename[9], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="T_3h_4_counts")



Ind4_79T24h_frag <- read_delim(safename[10], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="T_24h_4_counts")



Ind4_79V3h_frag <- read_delim(safename[11], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="V_3h_4_counts")



Ind4_79V24h_frag <- read_delim(safename[12], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="V_24h_4_counts")
Ind4_frag_files <- Ind4_79V3h_frag %>% 
  left_join(.,Ind4_79V24h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79DA3h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79DA24h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79DX3h_frag, by = "frag_size") %>% 
left_join(.,Ind4_79DX24h_frag, by = "frag_size") %>% 
left_join(.,Ind4_79E3h_frag, by = "frag_size") %>% 
left_join(.,Ind4_79E24h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79M3h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79M24h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79T3h_frag, by = "frag_size") %>% 
  left_join(.,Ind4_79T24h_frag, by = "frag_size") %>% 
  pivot_longer(cols= !frag_size,names_to = "sample",values_to = "counts") %>% 
  separate(sample, into = c("trt","time","indv",NA), sep = "_") %>% 
  mutate(trt = factor(trt, levels = c("DX","E","DA","M","T","V"))) %>% 
  mutate(time = factor(time, levels = c("3h","24h"))) %>% 
  mutate(counts= as.numeric(counts), frag_size= as.numeric(frag_size))
write.csv(Ind4_frag_files,"data/Ind4_fragment_files.txt")

```

### Ind4 3 hour and 24 hour fragment sizes
```{r fragfiles visualize}

Ind4_frag_files <- read.csv("data/Ind4_fragment_files.txt", row.names = 1)
Ind4_frag_files %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  ggtitle("Individual 4\n3 hour fragment sizes")+
  theme_classic()+
  scale_color_manual(values=drug_pal_fact)
Ind4_frag_files %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  ggtitle("Individual 4\n24 hour fragment sizes")+
  theme_classic()+
  scale_color_manual(values=drug_pal_fact)


```

So, fragment lengths are not so great. Lots of noise. 
