---
title: "Peak_Calling"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
editor_options: 
  markdown: 
    wrap: sentence
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
# library(ggsignif)
# library(cowplot)
# library(ggpubr)
# library(scales)
# library(sjmisc)
library(kableExtra)
# library(broom)
# library(biomaRt)
library(RColorBrewer)
# library(gprofiler2)
# library(qvalue)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
```

```{r data_loading and other functions}
drug_pal_fact <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene

loadFile_peakCall <- function(){
 file <- choose.files()
 file <- readPeakFile(file, header = FALSE)
 return(file)
}

prepGRangeObj <- function(seek_object){
 seek_object$Peaks = seek_object$V4
 seek_object$level = seek_object$V5
 seek_object$V4 = seek_object$V5 = NULL
 return(seek_object)
}
TSS = getBioRegion(TxDb=txdb, upstream=3000, downstream=3000, by = "gene", 
                   type = "start_site")

ind4_V24hpeaks <- readRDS("data/ind4_V24hpeaks.RDS")
ind1_DA24hpeaks <- readRDS("data/ind1_DA24hpeaks.RDS")
anno_ind4_V24h <- readRDS("data/anno_ind4_V24h.RDS")
anno_ind1_DA24h <- readRDS("data/anno_ind1_DA24h.RDS")
Ind1_summary  <- read.csv("data/Ind1_summary.txt", row.names = 1) %>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind2_summary  <- read.csv("data/Ind2_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind3_summary  <- read.csv("data/Ind3_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind4_summary  <- read.csv("data/Ind4_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind5_summary  <- read.csv("data/Ind5_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
Ind6_summary  <- read.csv("data/Ind6_summary.txt", row.names = 1)%>% 
  rename(X1="sample",X2="reads",X3="mapped")
```

This specific page so far contains the QC analysis after calling peaks using MACS2.

Primary scripts used for ATAC data preprocesing will be linked here in the future:

-   Â Currently the steps were are as follows:

    -   Basic Fastqc followed by adapter trimming and Fastqc analysis on the leftover fragments.

    -   Trimmed reads were aligned to the hg38 human genome.

    -   Mitochondrial reads (chrM) were removed

    -   samtools was used to removed non-paired, discordantly paired, and multi-mapped reads from the .bam.

    -   Markduplicates function from Picard was used to mark optical and PCR duplicates, with samtools used to remove these reads using the flag -F 1024.

    -   MACS2 was used to call peaks, with QC of the peak files below.

Initial summary of reads after each step:

### Individual 1 read summary

```{r Readssummary1}
#  library(readr) pulling in orginal files example code
#
# Ind1_summary <- read_delim("~/ATAC_downloads/Ind1/trimmed/Ind1_summedup.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# Ind2_summary <- read_delim("~/ATAC_downloads/Ind2/trimmed/Ind2_summedup.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# Ind3_summary <- read_delim("~/ATAC_downloads/Ind3/trimmed/Ind3_summedup.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# Ind4_summary <- read_delim("~/ATAC_downloads/Ind4/trimmed/Ind4_summedup.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# Ind5_summary <- read_delim("~/ATAC_downloads/Ind5/trimmed/Ind5_summedup.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# Ind6_summary <- read_delim("~/ATAC_downloads/Ind6/trimmed/Ind6_summedup.txt", 
#     delim = "\t", escape_double = FALSE, 
#     col_names = FALSE, trim_ws = TRUE)
# write.csv(Ind6_summary, "data/Ind6_summary.txt")
Ind1_reads_summary <- 
Ind1_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind1_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 1") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
```

### Individual 2 read summary
```{r Readssummary2}

Ind2_reads_summary <- 
Ind2_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind2_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 2") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
```

### Individual 3 read summary
```{r Readssummary3}

Ind3_reads_summary <- 
Ind3_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind3_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 3") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
```

### Individual 4 read summary
```{r Readssummary4}

Ind4_reads_summary <- 
Ind4_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind4_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 4") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
```
_note: reprocessed reads for Individual 4 vary from previous output.  This is due a file truncation that was detected in the process.  When the files were reprocessed, the truncated bam file was fixed, resulting in a larger number of overall reads.  Yay, I figured it out!_

### Individual 5 read summary
```{r Readssummary5}

Ind5_reads_summary <- 
Ind5_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind5_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 5") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
```

### Individual 6 read summary
```{r Readssummary6}

Ind6_reads_summary <- 
Ind6_summary %>%                          
 separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
   separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
   separate(sample, into=c("one","two",NA, "4","sample",NA,"seven","8")) %>% 
   mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
   mutate(type = if_else(two=="first", "total",
                         if_else(two=="noM","nuclear",
                                 if_else((seven == "fin"& two=="files"), "unique", "dedup"))))%>% 
   dplyr::select(sample,type,  reads, mapped_reads)%>% 
   pivot_longer(cols=reads:mapped_reads, names_to = "read_info", values_to = "reads") %>% 
   unite("type",type:read_info, sep="_") %>% 
   distinct() %>% 
   pivot_wider(id_cols = sample, names_from = "type", values_from = "reads") %>% 
   mutate(per_nuclear_mapped = nuclear_mapped_reads/total_mapped_reads*100) %>% 
  mutate(dedup_read_pairs=dedup_reads/2) %>% 
   dplyr::select(sample,total_reads:nuclear_mapped_reads,per_nuclear_mapped,unique_mapped_reads,dedup_reads,dedup_read_pairs)
 
 # dedup_mapped_reads)
 Ind6_reads_summary %>% 
   kable(., caption= "Reads summary from Ind 6") %>%
  kable_paper("striped", full_width = FALSE) %>%
  kable_styling(full_width = FALSE,font_size = 18) %>%
  scroll_box(width = "100%", height = "400px")

 
```


```{r peak info}
# plotAvgProf(anno_ind4_V24h,xlim=c(-3000,3000))
plotAnnoBar(anno_ind4_V24h, main = "Genomic Feature Distribution")+ ggtitle("Ind4 VEH 24 hour")
plotAnnoBar(anno_ind1_DA24h, main = "Genomic Feature Distribution")+ ggtitle("Ind1 DNR 24 hour")

ind4_V24hpeaks_gr <- prepGRangeObj(ind4_V24hpeaks)
ind1_DA24hpeaks_gr <- prepGRangeObj((ind1_DA24hpeaks))
Epi_list <- GRangesList(ind1_DA24hpeaks_gr, ind4_V24hpeaks_gr)
##plotting the TSS average window (making an overlap of each using Epi_list as list holder)
Epi_list_tagMatrix = lapply(Epi_list, getTagMatrix, windows = TSS)
plotAvgProf(Epi_list_tagMatrix, xlim=c(-3000, 3000), ylab = "Count Frequency")
#plotPeakProf(Epi_list_tagMatrix, facet = "none", conf = 0.95)
```

```{r fragment files list, eval=FALSE, include=FALSE}
### this was used to load fragment files for all individuals for display in the future
safename <- choose.files()
Ind6_71DA3h_frag <- read_delim(safename[1], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE)%>% separate(col=1, into=c("counts","frag_size")) %>% 
  rename(  "counts"="DA_3h_6_counts")


Ind6_71DA24h_frag <- read_delim(safename[2], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DA_24h_6_counts")


Ind6_71DX3h_frag <- read_delim(safename[3], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DX_3h_6_counts")


Ind6_71DX24h_frag<- read_delim(safename[4], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="DX_24h_6_counts")


Ind6_71E3h_frag <- read_delim(safename[5], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="E_3h_6_counts")


Ind6_71E24h_frag <- read_delim(safename[6], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="E_24h_6_counts")

Ind6_71M3h_frag <- read_delim(safename[7], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="M_3h_6_counts")


Ind6_71M24h_frag <- read_delim(safename[8], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="M_24h_6_counts")


Ind6_71T3h_frag <- read_delim(safename[9], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="T_3h_6_counts")



Ind6_71T24h_frag <- read_delim(safename[10], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="T_24h_6_counts")



Ind6_71V3h_frag <- read_delim(safename[11], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="V_3h_6_counts")



Ind6_71V24h_frag <- read_delim(safename[12], 
    delim = "\t", escape_double = FALSE, 
    col_names = FALSE, trim_ws = TRUE) %>% separate(col=1, into=c("counts","frag_size"))%>% 
  rename(  "counts"="V_24h_6_counts")
Ind6_frag_files <- Ind6_71V3h_frag %>% 
  left_join(.,Ind6_71V24h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71DA3h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71DA24h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71DX3h_frag, by = "frag_size") %>% 
left_join(.,Ind6_71DX24h_frag, by = "frag_size") %>% 
left_join(.,Ind6_71E3h_frag, by = "frag_size") %>% 
left_join(.,Ind6_71E24h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71M3h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71M24h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71T3h_frag, by = "frag_size") %>% 
  left_join(.,Ind6_71T24h_frag, by = "frag_size") %>% 
  pivot_longer(cols= !frag_size,names_to = "sample",values_to = "counts") %>% 
  separate(sample, into = c("trt","time","indv",NA), sep = "_") %>% 
  mutate(trt = factor(trt, levels = c("DX","E","DA","M","T","V"))) %>% 
  mutate(time = factor(time, levels = c("3h","24h"))) %>% 
  mutate(counts= as.numeric(counts), frag_size= as.numeric(frag_size))
write.csv(Ind6_frag_files,"data/Ind6_fragment_files.txt")

```

### Individual 4 fragment files:

```{r fragfiles visualize}

Ind4_frag_files <- read.csv("data/Ind4_fragment_filesN.txt", row.names = 1)
Ind4_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt ))+
  ggtitle("Individual 4\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
Ind4_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt ))+
  ggtitle("Individual 4\n24 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)


```

So, fragment lengths are not so great.
Lots of noise.

### Individual 1 fragment files:

```{r fragfiles1}

Ind1_frag_files <- read.csv("data/Ind1_fragment_files.txt", row.names = 1)
Ind1_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
Ind1_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
```

### Individual 2 fragment files:

```{r fragfiles2}

Ind2_frag_files <- read.csv("data/Ind2_fragment_files.txt", row.names = 1)
Ind2_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 2\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
Ind2_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 2\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
```

### Individual 3 fragment files:

```{r fragfiles3}

Ind3_frag_files <- read.csv("data/Ind3_fragment_files.txt", row.names = 1)
Ind3_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 3\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
Ind3_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 3\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
```

### Individual 5 fragment files:

```{r fragfiles5}

Ind5_frag_files <- read.csv("data/Ind5_fragment_filesN.txt", row.names = 1)
Ind5_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 5\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
Ind5_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 5\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
```


### Individual 6 fragment files:

```{r fragfiles6}

Ind6_frag_files <- read.csv("data/Ind6_fragment_files.txt", row.names = 1)
Ind6_frag_files %>% 
  dplyr::filter(time =="3h") %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 6\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
Ind6_frag_files %>% 
  dplyr::filter(time =="24h") %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 6\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal_fact)
```


```{r readsummary 1}
total_reads_summary <- Ind1_reads_summary %>% 
 mutate(sample=gsub("75","1_",sample)) %>% 
  rbind((Ind2_reads_summary %>%
           mutate(sample=gsub("87","2_",sample)))) %>% 
  rbind((Ind3_reads_summary %>%
           mutate(sample=gsub("77","3_",sample)))) %>% 
  rbind((Ind4_reads_summary %>%
           mutate(sample=gsub("79","4_",sample)))) %>% 
  rbind((Ind5_reads_summary %>%
           mutate(sample=gsub("78","5_",sample)))) %>% 
  rbind((Ind6_reads_summary %>%
           mutate(sample=gsub("71","6_",sample)))) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH")))
         
total_reads_summary %>% 
  ggplot(., aes(x=trt, y=total_reads ))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fact)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total number of reads by time and treatment")



total_reads_summary %>% 
  ggplot(., aes(x=trt, y=nuclear_reads ))+
  geom_boxplot(aes(fill=trt))+
 geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fact)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total number of nuclear reads by time and treatment")


total_reads_summary %>% 
  ggplot(., aes(x=trt, y=unique_mapped_reads ))+
  geom_boxplot(aes(fill=trt))+
 geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fact)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total number of unique-nuclear reads by time and treatment")



total_reads_summary %>% 
  ggplot(., aes(x=trt, y=dedup_reads ))+
  geom_boxplot(aes(fill=trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(time~.)+
   scale_fill_manual(values=drug_pal_fact)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("total unique, deduplicated nuclear mapped reads by time and treatment")


```
