---
title: "QC_on_reads"
author: "Renee Matthews"
date: "2025-05-04"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE,
	dev = c("png","pdf")
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r packages}
library(tidyverse)
library(kableExtra)
# library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
# library(edgeR)
library(ggfortify)
# library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(eulerr)
# library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(ComplexHeatmap)
library(cowplot)
library(smplot2)
library(data.table)
library(ATACseqQC)

```


```{r functions}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")


```

### Loading data


```{r  Loading data}


Ind1_summary  <- read.csv("data/Ind1_summary.txt", row.names = 1) %>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('75DX','DOX',trt),
         trt= gsub('75E','EPI', trt),
         trt=gsub('75DA','DNR',trt),
         trt=gsub('75M','MTX',trt),
         trt=gsub('75T','TRZ',trt),
         trt=gsub('75V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Ind2_summary  <- read.csv("data/Ind2_summary.txt", row.names = 1)%>% 
dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('87DX','DOX',trt),
         trt= gsub('87E','EPI', trt),
         trt=gsub('87DA','DNR',trt),
         trt=gsub('87M','MTX',trt),
         trt=gsub('87T','TRZ',trt),
         trt=gsub('87V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Ind3_summary  <- read.csv("data/Ind3_summary.txt", row.names = 1)%>% 
dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('77DX','DOX',trt),
         trt= gsub('77E','EPI', trt),
         trt=gsub('77DA','DNR',trt),
         trt=gsub('77M','MTX',trt),
         trt=gsub('77T','TRZ',trt),
         trt=gsub('77V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Ind6_summary  <- read.csv("data/Ind6_summary.txt", row.names = 1)%>% 
  dplyr::rename("sample"=X1,"reads"=X2,"mapped"=X3)%>% 
  mutate(sample=gsub(pattern = "flagstat_first/trimmed_","total_",sample)) %>% 
  mutate(sample= gsub(pattern= "flagstat_noM/trimmed_","nuclear_",sample)) %>% 
  mutate(sample=gsub(pattern = "filt_files/trimmed_","other_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into=c("read_type","indv","trt","time",NA,"neat")) %>% 
  mutate(read_type=if_else(read_type=="other",neat,read_type)) %>% 
  mutate(trt= gsub('71DX','DOX',trt),
         trt= gsub('71E','EPI', trt),
         trt=gsub('71DA','DNR',trt),
         trt=gsub('71M','MTX',trt),
         trt=gsub('71T','TRZ',trt),
         trt=gsub('71V','VEH',trt)) %>% 
  separate(reads,into=c("reads",NA),sep= " ") %>% 
   mutate(reads=as.numeric(reads)) %>% 
  separate(mapped, into= c("mapped_reads", NA,NA, "percent_mapped_reads",NA)) %>% 
  mutate(mapped_reads=as.numeric(mapped_reads)) %>% 
  tidyr::unite("sample",indv:time,sep = "_",remove = FALSE) %>% 
  dplyr::select(sample, indv:time, read_type,reads, mapped_reads)

Final_four_counts <- Ind1_summary %>% rbind(Ind2_summary) %>% rbind(Ind3_summary) %>% rbind(Ind6_summary)

FF_bound <- Final_four_counts %>% 
  mutate(time=factor(time, levels =c("3h","24h"))) %>% 
  pivot_longer(cols = reads:mapped_reads, names_to="count_type", values_to = "read_num") %>% 
  mutate(read_type=case_when(read_type=="fin"~"unique",.default = read_type)) %>% 
  tidyr::unite("Total_reads",read_type:count_type,sep = "_", remove = TRUE) %>% 
  tidyr::unite("Total_reads_time",Total_reads:time,sep = "_", remove = FALSE) %>% 
  mutate(Total_reads=factor(Total_reads, levels = c("total_reads", "total_mapped_reads","nuclear_reads","nuclear_mapped_reads","unique_reads", "unique_mapped_reads", "nodup_reads", "nodup_mapped_reads"))) %>% 
  dplyr::filter(Total_reads != "nuclear_reads") %>% 
dplyr::filter(Total_reads != "nodup_reads") %>% 
dplyr::filter(Total_reads != "unique_reads") %>% 
  dplyr::mutate(trt=factor(trt, levels = c("DOX", "EPI","DNR", "MTX","TRZ","VEH")))
```


```{r  Plotting reads}
FF_bound %>% 
 ggplot(., aes(x=Total_reads, y=read_num, col=indv, group=interaction(time,Total_reads_time)))+
  geom_boxplot(aes(fill=trt)) + 
  geom_point(aes(col=indv))+
   theme_bw()+
  facet_wrap(~trt+time,nrow = 3, ncol = 6 )+
  scale_fill_manual(values=drug_pal)+
  theme(strip.text = 
          element_text(face = "bold",  
                       hjust = 0, 
                       size = 8),
        strip.background = 
          element_rect(fill = "white",
                       linetype = "solid",
                       color = "black", 
                       linewidth = 1),
        panel.spacing = unit(1, 'points'),
        axis.text.x=
          element_text(angle = 90, 
                       vjust = 0.5, 
                       hjust=1))

```

### Individual 1 fragment files:

```{r fragfiles1}

Ind1_frag_files <- read.csv("data/Ind1_fragment_files.txt", row.names = 1)
 
Ind1_firstfrag_files <- read.csv("data/Ind1_firstfragment_files.txt", row.names = 1)
Ind1_frag_files %>% 
  mutate(trt=
           factor(trt,levels=
                    c("DX","E","DA","M","T","V"), 
                  labels=
                c("DOX","EPI","DNR", "MTX","TRZ","VEH")))%>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n3 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)+
  coord_cartesian(ylim=c(0,300000))

Ind1_firstfrag_files %>% 
  mutate(trt=factor(trt,
                    levels=c("DX","E","DA","M","T","V"),
                    labels=c("DOX","EPI", "DNR","MTX","TRZ" ,"VEH"))) %>%
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=(counts), x=(frag_size), group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n3 hour fragment sizes BEFORE filtering")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)+
  coord_cartesian(xlim=c(0,1000))
 

Ind1_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=(counts), x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)

Ind1_firstfrag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=(counts), x=(frag_size), group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 1\n24 hour fragment sizes BEFORE filtering")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)+
  coord_cartesian(xlim=c(0,1000))


```
#### FRiP Individual 1

```{r Frip1}
cardiac_muscle_Frip <- read.csv("data/cardiac_muscle_FRIP.csv", row.names = 1)
cardiomyocyte_Frip <- read.csv("data/cardiomyocyte_FRIP.csv", row.names = 1)
left_ventricle_Frip <- read.csv("data/left_ventricle_FRIP.csv", row.names = 1)
embryo_heart_Frip <- read.csv("data/embryo_heart_FRIP.csv", row.names = 1)

Frip_1_reads <- read.csv("data/Frip_1_reads.csv", row.names = 1)

all_frip1 <- Frip_1_reads %>% 
 mutate(sample=gsub("75","1_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
   left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)


Frip_1_reads %>% 
  mutate(sample=gsub("75","1_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_1, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 1")+
   scale_fill_manual(values=drug_pal)
 


FRiP_first_run <- read.csv("data/FRiP_first_run.txt", row.names = 1)
FRiP_first_run %>% 
  mutate(sample=gsub("75","1_",sample)) %>% 
  mutate(sample=gsub("87","2_",sample)) %>% 
  mutate(sample=gsub("77","3_",sample)) %>% 
  mutate(sample=gsub("79","4_",sample)) %>% 
  mutate(sample=gsub("78","5_",sample)) %>% 
  mutate(sample=gsub("71","6_",sample)) %>% 
  mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x=time, y=FRiP, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 sample")+
   scale_fill_manual(values=drug_pal)

all_frip1 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal)
  
all_frip1 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip1 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip1 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind1 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)




```

Horizontal line is at 20%.  Individual 4 at 3 hours is not a good score.  Encode recommends >30% (dotted line).



### Individual 2 fragment files:

```{r fragfiles2}

Ind2_frag_files <- read.csv("data/Ind2_fragment_files.txt", row.names = 1)
Ind2_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 2\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)
Ind2_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 2\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)
```


#### FRiP Individual 2

```{r Frip2}


Frip_2_reads <- read.csv("data/Frip_2_reads.csv", row.names = 1)

all_frip2 <- Frip_2_reads %>% 
  mutate(sample=gsub("87","2_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)

Frip_2_reads %>% 
 mutate(sample=gsub("87","2_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_2, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 2")+
   scale_fill_manual(values=drug_pal)



all_frip2 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal)
  
all_frip2 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2  using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip2 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2  using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip2 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind2 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)





```

### Individual 3 fragment files:

```{r fragfiles3}

Ind3_frag_files <- read.csv("data/Ind3_fragment_files.txt", row.names = 1)
Ind3_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="3h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 3\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)
Ind3_frag_files %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  dplyr::filter(time =="24h") %>%
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 3\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)
```


#### FRiP Individual 3

```{r Frip3}


Frip_3_reads <- read.csv("data/Frip_3_reads.csv", row.names = 1)

all_frip3 <- Frip_3_reads %>% 
  mutate(sample=gsub("77","3_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
 mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)






Frip_3_reads %>% 
  mutate(sample=gsub("77","3_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_3, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 3")+
   scale_fill_manual(values=drug_pal)



all_frip3 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal)
  
all_frip3 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip3 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip3 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind3 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)





```
### Individual 6 fragment files:

```{r fragfiles6}

Ind6_frag_files <- read.csv("data/Ind6_fragment_files.txt", row.names = 1)
Ind6_frag_files %>% 
  dplyr::filter(time =="3h") %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  # geom_line(aes(col=trt, alpha = 0.5, linewidth=1 ))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 6\n3 hour fragment sizes")+
  theme_classic()+
   facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)
Ind6_frag_files %>% 
  dplyr::filter(time =="24h") %>%
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes(y=counts, x=frag_size, group=trt))+
  geom_line(aes(col=trt))+
  ggtitle("Individual 6\n24 hour fragment sizes")+
  theme_classic()+
  facet_wrap(~trt)+
  scale_color_manual(values=drug_pal)
```
#### FRiP Individual 6

```{r Frip6}

Frip_6_reads <- read.csv("data/Frip_6_reads.csv", row.names = 1)

all_frip6 <- Frip_6_reads %>% 
  mutate(sample=gsub("71","6_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  mutate(indv=as.numeric(indv)) %>% 
  left_join(., (cardiac_muscle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
 left_join(., (left_ventricle_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
    left_join(., (cardiomyocyte_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  left_join(., (embryo_heart_Frip %>% mutate(trt=factor(trt, levels=c("DOX","EPI","DNR","MTX","TRZ","VEH"))))) %>% 
  mutate(FRIP_embryo=embryo_counts/dedup_reads *100) %>% 
  mutate(FRIP_cm=cm_counts/dedup_reads*100) %>% 
mutate(FRIP_lv=lv_counts/dedup_reads*100) %>% 
mutate(FRIP_adult=c_muscle_counts/dedup_reads*100)
 

Frip_6_reads %>% 
  mutate(sample=gsub("71","6_",sample)) %>% 
   mutate(sample = gsub("24h","_24h",sample), 
       sample = gsub("3h","_3h",sample)) %>%
  separate(sample, into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels=c("DX","E","DA","M","T","V"), labels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>%  
  ggplot(., aes (x=time, y=FRiP_6, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across each sample for individual 6")+
   scale_fill_manual(values=drug_pal)

all_frip6 %>% ggplot(., aes (x=time, y=FRIP_cm, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score  across Ind6 using cardiomyocyte DNAse peaks")+
   scale_fill_manual(values=drug_pal)
  
all_frip6 %>% ggplot(., aes (x=time, y=FRIP_lv, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind6 using left ventricle DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip6 %>% ggplot(., aes (x=time, y=FRIP_adult, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind6 using adult heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)

all_frip6 %>% ggplot(., aes (x=time, y=FRIP_embryo, group=trt))+
  geom_col(position= "dodge",aes(fill=trt))+
  geom_hline(yintercept = 20)+
  geom_hline(yintercept = 30,linetype=3)+
  facet_wrap(~indv)+
  theme_classic()+
  ggtitle("FRiP Score across Ind6 using embryonic heart DNAse peaks")+
   scale_fill_manual(values=drug_pal)




```
```{r peaksummary}
Peaksummary <- read.csv("data/first_Peaksummarycounts.csv",row.names=1)
Peaksummary %>% 
  dplyr::filter(sample != "total") %>% 
   separate(sample, into=c(NA,"indv","sample",NA,NA,NA)) %>% 
  mutate(trt=gsub("[[:digit:]]", "",sample)) %>% 
  # mutate(trt=substr(trt,-1,2))
  mutate(time = if_else(grepl("24h$", sample) ==TRUE, "24_hours","3_hours")) %>% 
  mutate(trt = case_match(trt,"DAh"~"DNR","DXh"~"DOX","Eh"~"EPI", "Mh" ~ "MTX", "Th" ~ "TRZ", "Vh" ~"VEH",.default = trt)) %>% 
  mutate(indv = factor(indv, levels = c("Ind1", "Ind2", "Ind3", "Ind4", "Ind5", "Ind6"))) %>%
  mutate(time = factor(time, levels = c("3_hours", "24_hours"), labels= c("3 hours","24 hours"))) %>% 
  mutate(trt = factor(trt, levels =  c("DOX","EPI", "DNR", "MTX", "TRZ", "VEH")))%>% 
  dplyr::filter(indv != "Ind4") %>% 
  dplyr::filter(indv != "Ind5") %>% 
  ggplot(., aes(x =trt, y=counts,group=trt))+
  geom_boxplot(aes(fill= trt))+
  geom_point(aes(col=indv, size =3))+
  facet_wrap(~time)+
   scale_fill_manual(values=drug_pal)+
  scale_color_brewer(palette = "Dark2")+
  ggtitle("Peak counts by treatment")+
  theme_bw()
```


### Ind1 Peaks


Making the TSS average window code
```{r Ind1 inital TSS setup}


# ind4_V24hpeaks_gr <- prepGRangeObj(ind4_V24hpeaks)
# ind1_DA24hpeaks_gr <- prepGRangeObj((ind1_DA24hpeaks))
# Epi_list <- GRangesList(ind1_DA24hpeaks_gr, ind4_V24hpeaks_gr)
# # ##plotting the TSS average window (making an overlap of each using Epi_list as list holder)
# Epi_list_tagMatrix = lapply(Epi_list, getTagMatrix, windows = TSS)
# plotAvgProf(Epi_list_tagMatrix, xlim=c(-3000, 3000), ylab = "Count Frequency")
#plotPeakProf(Epi_list_tagMatrix, facet = "none", conf = 0.95)

## What I did here:  I called all my narrowpeak files
# peakfiles1 <- choose.files()

##these were practice for getting file names and shortening for the for loop below
# testname <- basename(peakfiles1[1])
# str_split_i(testname, "_",3)

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind1_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles1[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind1_peaks[[banana_peel]] <- readPeakFile(peakfiles1[file])
# }
# saveRDS(Ind1_peaks, "data/Ind1_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)
# peakAnnoList_1 <- lapply(Ind1_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_1, "data/peakAnnoList_1.RDS")

peakAnnoList_1 <- readRDS("data/peakAnnoList_1.RDS")
plotAnnoBar(peakAnnoList_1, main = "Genomic Feature Distribution")


# saveRDS(Epi_list_tagMatrix, "data/Ind1_TSS_peaks.RDS")

Ind1_TSS_peaks_plot <- readRDS("data/Ind1_TSS_peaks.RDS")

# Epi_list_tagMatrix = lapply(Ind1_peaks, getTagMatrix, windows = TSS)

plotAvgProf(Ind1_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 1" )
# + coord_cartesian(xlim = c(-100,500))

plotAvgProf(Ind1_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 1" )
# + coord_cartesian(xlim = c(-100,500))

```


### Ind2 Peaks

```{r Ind2 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles2 <- choose.files()

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind2_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles2[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind2_peaks[[banana_peel]] <- readPeakFile(peakfiles2[file])
# }
# saveRDS(Ind2_peaks, "data/Ind2_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind2_peaks <- readRDS("data/Ind2_peaks_list.RDS")
# peakAnnoList_2 <- lapply(Ind2_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_2, "data/peakAnnoList_2.RDS")

peakAnnoList_2 <- readRDS("data/peakAnnoList_2.RDS")
plotAnnoBar(peakAnnoList_2, main = "Genomic Feature Distribution, Individual 2")

# Epi_list_tagMatrix = lapply(Ind2_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind2_TSS_peaks.RDS")

Ind2_TSS_peaks_plot <- readRDS("data/Ind2_TSS_peaks.RDS")



plotAvgProf(Ind2_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), conf = 0.01, ylab = "Count Frequency")+ ggtitle("3 hour Individual 2" )
# + coord_cartesian(xlim = c(-100,1000))

plotAvgProf(Ind2_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 2" )#+ coord_cartesian(xlim = c(-100,500))

```


### Ind3 Peaks

```{r Ind3 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles3 <- choose.files()

##This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind3_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles3[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind3_peaks[[banana_peel]] <- readPeakFile(peakfiles3[file])
# }
# saveRDS(Ind3_peaks, "data/Ind3_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind3_peaks <- readRDS("data/Ind3_peaks_list.RDS")
# peakAnnoList_3 <- lapply(Ind3_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_3, "data/peakAnnoList_3.RDS")

peakAnnoList_3 <- readRDS("data/peakAnnoList_3.RDS")
plotAnnoBar(peakAnnoList_3, main = "Genomic Feature Distribution, Individual 3")

# Epi_list_tagMatrix = lapply(Ind3_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind3_TSS_peaks.RDS")

Ind3_TSS_peaks_plot <- readRDS("data/Ind3_TSS_peaks.RDS")



plotAvgProf(Ind3_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 3" )

plotAvgProf(Ind3_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 3" )

```

### Ind6 Peaks

```{r Ind6 inital TSS setup}

## What I did here:  I called all my narrowpeak files

# peakfiles6 <- choose.files()
# 
# ##This loop first established a list then (because I already knew the list had 12 files)
# ## I then imported each of these onto that list.  Once I had the list, I stored it as
# ## an R object, 
# Ind6_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles6[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind6_peaks[[banana_peel]] <- readPeakFile(peakfiles6[file])
# }
# saveRDS(Ind6_peaks, "data/Ind6_peaks_list.RDS")
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)

Ind6_peaks <- readRDS("data/Ind6_peaks_list.RDS")
# peakAnnoList_6 <- lapply(Ind6_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_6, "data/peakAnnoList_6.RDS")

peakAnnoList_6 <- readRDS("data/peakAnnoList_6.RDS")
plotAnnoBar(peakAnnoList_6, main = "Genomic Feature Distribution, Individual 6")+ggtitle ("Genomic Feature Distribution, Individual 6")
##Epi_list_tagMatrix title was just because I was too lazy to change the name
# Epi_list_tagMatrix = lapply(Ind6_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind6_TSS_peaks.RDS")

Ind6_TSS_peaks_plot <- readRDS("data/Ind6_TSS_peaks.RDS")


plotAvgProf(Ind6_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 6" )

plotAvgProf(Ind6_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 6" )


```



```{r  TSS enrichment part 1}
### for code to calculate the TSSE scores, reference the file "code/TSSE.R"

```


```{r  TSS enrichment part 2}
### for code to calculate the TSSE scores, reference the file "code/TSSE.R"
allTSSE <- readRDS( "data/all_TSSE_scores.RDS")
allTSSE %>% as.data.frame() %>% 
  rownames_to_column("sample") %>% 
  separate(sample, into = c("indv","trt","time"), sep= "_") %>%
  mutate(trt= factor(trt, levels = c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  mutate(time = factor(time, levels = c("3","24"),labels = c("3 hours","24 hours"))) %>% 
  dplyr::filter(indv !=4 &indv !=5) %>% 
  ggplot(., aes(x= time, y= V1, group = indv))+
  geom_jitter(aes(col = trt, size = 1.5, alpha = 0.5) ,
              position=position_jitter(0.25))+
  geom_hline(yintercept=5, linetype = 3)+
  geom_hline(yintercept=7, col = "blue")+
  facet_wrap(~indv)+
  theme_bw()+
  ggtitle("TSS enrichment scores")+
  scale_color_manual(values=drug_pal)+
  theme(strip.text = element_text(face = "bold",  hjust = .5, size = 8),
        strip.background = element_rect(fill = "white", linetype = "solid",
                                        color = "black", linewidth = 1))

```

```{r TSS peaks plot, fig.width=8, fig.height=12}

#### These Ind1 TSS_peaks were created on the "analysis/Peak_calling.Rmd file  example code:
# Import in peak files: peakfiles3 <- choose.files()
###This loop first established a list then (because I already knew the list had 12 files)
## I then imported each of these onto that list.  Once I had the list, I stored it as
## an R object, 
# Ind3_peaks <- list()
# for (file in 1:12){
#     testname <- basename(peakfiles3[file])
#   banana_peel <- str_split_i(testname, "_",3)
#  Ind3_peaks[[banana_peel]] <- readPeakFile(peakfiles3[file])
# }
# saveRDS(Ind3_peaks, "data/Ind3_peaks_list.RDS")
# 
# I then called annotatePeak on that list object, and stored that as a R object for later retrieval.)
# peakAnnoList_3 <- lapply(Ind3_peaks, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# saveRDS(peakAnnoList_3, "data/peakAnnoList_3.RDS")
# Epi_list_tagMatrix = lapply(Ind3_peaks, getTagMatrix, windows = TSS)
# saveRDS(Epi_list_tagMatrix, "data/Ind3_TSS_peaks.RDS")


Ind1_TSS_peaks_plot <- readRDS("data/Ind1_TSS_peaks.RDS")
Ind2_TSS_peaks_plot <- readRDS("data/Ind2_TSS_peaks.RDS")
Ind3_TSS_peaks_plot <- readRDS("data/Ind3_TSS_peaks.RDS")
Ind6_TSS_peaks_plot <- readRDS("data/Ind6_TSS_peaks.RDS")

a1<- plotAvgProf(Ind1_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 1" )+coord_cartesian(xlim=c(-2000,2000))
b1 <- plotAvgProf(Ind2_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 2" )+coord_cartesian(xlim=c(-2000,2000))
c1 <- plotAvgProf(Ind3_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 3" )+coord_cartesian(xlim=c(-2000,2000))
d1 <- plotAvgProf(Ind6_TSS_peaks_plot[c(1,3,5,7,9,11)], xlim=c(-3000, 3000), ylab = "Count Frequency")+ ggtitle("3 hour Individual 6" )+coord_cartesian(xlim=c(-2000,2000))

a2 <- plotAvgProf(Ind1_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 1" )+coord_cartesian(xlim=c(-2000,2000))
b2 <- plotAvgProf(Ind2_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 2" )+coord_cartesian(xlim=c(-2000,2000))
c2 <- plotAvgProf(Ind3_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 3" )+coord_cartesian(xlim=c(-2000,2000))
d2 <- plotAvgProf(Ind6_TSS_peaks_plot[c(2,4,6,8,10,12)], xlim=c(-3000, 3000),ylab = "Count Frequency")+ ggtitle("24 hour Individual 6" )+coord_cartesian(xlim=c(-2000,2000))

plot_grid(a1,a2, b1,b2,c1,c2,d1,d2, axis="l",align = "hv",nrow=4, ncol=2)

```



## Creating the masterlist of high confidence regions (masterpeaks)  

To create this peak set, 
-First I moved all .narrowPeak files into the same folder and ran `bedtools multiinter -i ./* >log.file.txt` to create an intersection of all peaks.  I then was only interested in segments that had a count of more than 4 (intersection existed in at least 4 of the data sets) in all files.   I filtered column #4 of the `log.file.txt` output by `awk -F"\t" '$4 > 4 {print $1"\t"$2"\t"$3 }' log.file.txt > all_filt_peaks.bed` and printed the results of anything >4 in bed format to the `all_filt_peaks.bed` file.   Upon further reading of bedtools documents, I realized the number of "peaks" was actually fragments of peaks that were intersected among all files.  This was not the final output I wanted so I intersected these high counted segments back with the very first initial mergedPeaks.bed file using `bedtools intersect -a mergedPeaks.bed -b all_filt_peaks.bed -wa -u > merged_filtered_peaks.bed`.  using the `-wa -u` flags allowed me to filter the first mergedPeaks file, keeping only those high confidence peaks that overlapped the `all_filt_peaks.bed` and only reporting the unique calls.


Additionally, I filtered out the blacklisted regions using the following code: `intersectBed -v -a merged_filtered_peaks.bed -b Blacklist/hg38.blacklist.bed.gz > final_bl_filt_peaks.bed`
My final high confidence peaks file contains 172,418 peaks.
I used this file to collect PE-read counts using `featureCounts -p -a high_conf_peaks.saf -F SAF -o all_four_filtered_counts.txt ind1/trimmed/filt_files/*nodup.bam ind2/trimmed/filt_files/*nodup.bam ind3/trimmed/filt_files/*nodup.bam ind6/trimmed/filt_files/*nodup.bam`




