---
title: "ReHeat_data"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r package loading, message=FALSE, warning=FALSE}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(Cormotif)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(readxl)
```


```{r data loading}
Reheat_data <- read_excel("data/other_papers/jah36123-sup-0002-tables2.xlsx")

toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") 
toplistall_RNA <- toplistall_RNA %>% 
  mutate(logFC = logFC*(-1))

toplist_full_n45 <- readRDS("data/DEG_toplist_sep_n45.RDS")
TSS_NG_data <- read_delim("data/n45_bedfiles/TSS_NG_data.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
Collapsed_peaks <- read_delim("data/n45_bedfiles/TSS_NG_data_collapsed_peaks.tsv",
                              delim = "\t", 
                              escape_double = FALSE, 
                              trim_ws = TRUE)

peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")
peakAnnoList_24_n45 <- readRDS("data/peakAnnoList_24_n45.RDS")

EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
EAR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)


ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
ESR_df_gr <- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
LR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
NR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)

all_peak_gr <- as.GRanges(peakAnnoList_24_n45$background)

Joined_tss_neargene <- all_peak_gr %>%
  as.data.frame() %>% 
  dplyr::select(seqnames:id,geneId,distanceToTSS) %>% 
  dplyr::filter(!grepl("chrX",id)& !grepl("chrY",id))%>%
  dplyr::rename("close_geneId"="geneId") %>% 
  mutate(start=start+1) %>% 
  left_join(., TSS_NG_data, by=c("id"="peakid", "start"="start","end"="end")) %>%
  dplyr::select(seqnames.x:close_geneId,distanceToTSS, entrezgene_id:dist_to_NG)

  median_24_lfc <- readRDS("data/median_24_lfc.RDS") %>% ungroup()
median_3_lfc <- readRDS("data/median_3_lfc.RDS") %>% ungroup()
 
Nine_group_TE_full <- readRDS("data/TE_info/Nine_group_TE_full.RDS")

# 
# 
# hyper_3med <- median_3_lfc %>% 
#   dplyr::filter(median > 0)
# 
# hypo_3med <- median_3_lfc %>% 
#   dplyr::filter(median < 0)
# 
# hyper_24med <- median_24_lfc %>% 
#   dplyr::filter(median > 0)
# 
# hypo_24med <- median_24_lfc %>% 
#   dplyr::filter(median < 0)
# 
# medA <- median_3_lfc %>% 
#   left_join(median_24_lfc, by=c("peak"="peak")) %>% 
#   dplyr::rename("3_hours"="median.x", "24_hours"="median.y") %>% 
#   dplyr::filter(`3_hours`>0 & `24_hours`>0)
# 
# medB <- median_3_lfc %>% 
#   left_join(median_24_lfc, by=c("peak"="peak")) %>% 
#   dplyr::rename("3_hours"="median.x", "24_hours"="median.y") %>% 
#   dplyr::filter(`3_hours`<0 & `24_hours`<0)
# 
# medC <- median_3_lfc %>% 
#   left_join(median_24_lfc, by=c("peak"="peak")) %>% 
#   dplyr::rename("3_hours"="median.x", "24_hours"="median.y") %>% 
#   dplyr::filter(`3_hours`>0 & `24_hours`<0)
# 
# medD <- median_3_lfc %>% 
#   left_join(median_24_lfc, by=c("peak"="peak")) %>% 
#   dplyr::rename("3_hours"="median.x", "24_hours"="median.y") %>% 
#   dplyr::filter(`3_hours`<0 & `24_hours`>0)
# 
# EAR_hyper <- EAR_df %>%
#   dplyr::filter(id %in% hyper_3med$peak) %>% 
#   dplyr::select(seqnames:id)
# EAR_hyper_gr <- EAR_hyper %>% GRanges()
# 
# EAR_hypo <- EAR_df %>%
#   dplyr::filter(id %in% hypo_3med$peak) %>% 
#   dplyr::select(seqnames:id)
# EAR_hypo_gr <- EAR_hypo %>% GRanges()
# 
# LR_hyper <- LR_df %>%
#   dplyr::filter(id %in% hyper_24med$peak) %>% 
#   dplyr::select(seqnames:id)
# LR_hyper_gr <- LR_hyper %>% GRanges()
# 
# LR_hypo <- LR_df %>%
#   dplyr::filter(id %in% hypo_24med$peak) %>% 
#   dplyr::select(seqnames:id)
# LR_hypo_gr <- LR_hypo %>% GRanges()
# 
# ESR_A <- ESR_df %>% 
#   dplyr::filter(id %in% medA$peak) %>% 
#   dplyr::select(seqnames:id)
# 
# ESR_B <- ESR_df %>% 
#   dplyr::filter(id %in% medB$peak) %>% 
#   dplyr::select(seqnames:id)
# 
# ESR_C <- ESR_df %>% 
#   dplyr::filter(id %in% medC$peak) %>% 
#   dplyr::select(seqnames:id)
# 
# ESR_D <- ESR_df %>% 
#   dplyr::filter(id %in% medD$peak) %>% 
#   dplyr::select(seqnames:id)


```

Making a dataframe that annotates if a neargene is a Reheat heart failure gene.

Have peaks, assigned to neargenes and mrcs.  How many of my neargenes are in the ReHeat gene list?

```{r }

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene"))%>% 
  dplyr::filter(fisher_pvalue<10^-5) %>% 
  distinct(hgnc_symbol, .keep_all = TRUE)


Nine_group_summary<- Nine_group_TE_full %>% 
  left_join(., (Joined_tss_neargene %>% dplyr::select(id:dist_to_NG)), by=c("peakid"="id")) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>%
  dplyr::filter(fisher_pvalue<10^-5) %>%
  distinct() %>% 
  group_by(peakid) %>%
    summarise(peakid,seqnames,start,end,
              NCBI_gene=paste(unique(NCBI_gene), collapse = ","),
              ensembl_ID=paste(unique(ensembl_ID),collapse=","),
              SYMBOL=paste(unique(SYMBOL),collapse=","),
              repName = paste(unique(repName),collapse=","),
              repClass= paste(unique(repClass),collapse = ","),
              repFamily= paste(unique(repFamily),collapse = ","),
              TEstatus= paste(unique(TEstatus),collapse = ","),
              mrc =paste(unique(mrc),collapse=","),
              repClass_org= paste(unique(repClass_org),collapse = ","),
              close_geneId= paste(unique(close_geneId),collapse = ";") ,
              entrezgene_id= paste(unique(entrezgene_id),collapse = ";") ,
              ensembl_gene_id= paste(unique(ensembl_gene_id),collapse = ";") ,
              close_geneId= paste(unique(close_geneId),collapse = ";") ,
              hgnc_symbol= paste(unique(hgnc_symbol),collapse = ";") ,
              fisher_pvalue=min(fisher_pvalue)
              ) 
# %>%  write.csv(.,"data/summary_peakIDandReHeat.csv")



```


```{r RNAresponse and reheat}
S13Table <- read.csv( "data/other_papers/S13Table_Matthews2024.csv",row.names = 1)
##14021

EAR_RNAs <- S13Table %>% 
  dplyr::filter(MOTIF=="EAR") %>% 
  dplyr::select(SYMBOL) 
ESR_RNAs <- S13Table %>% 
  dplyr::filter(MOTIF=="ESR")%>% 
   dplyr::select(SYMBOL) 
LR_RNAs <- S13Table %>% 
  dplyr::filter(MOTIF=="LR")%>% 
   dplyr::select(SYMBOL) 
NR_RNAs <- S13Table %>% 
  dplyr::filter(MOTIF=="NR")%>% 
   dplyr::select(SYMBOL) 


gene_list_reheat <- Nine_group_summary %>% 
  ungroup() %>% 
  dplyr::select(hgnc_symbol,fisher_pvalue) %>% 
  distinct %>% 
  arrange(fisher_pvalue) 
  
 


length(intersect(unique(NR_RNAs$SYMBOL),unique(Reheat_data$gene)))
toplistall_RNA %>% 
  distinct(SYMBOL) %>% 
  dplyr::filter(SYMBOL %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("SYMBOL"="gene")) %>% 
  dplyr::filter(SYMBOL  %in%)
  
  # dplyr::filter(fisher_pvalue<10^-10) %>% 
  distinct(SYMBOL, .keep_all = TRUE)

  Nin

```

There are `r length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))` genes shared between my neargenes and hear failure genes.

```{r  3hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-5) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc all distances away")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 20kb of neargene TSS")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 2kb of neargene TSS")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

```{r  24hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc all distances away")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 20kb of neargene TSS")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 2kb of neargene TSS")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

#### adding in response cluster

EAR
```{r  EAR3hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(id %in% EAR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-5) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc all distances away;  EAR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% EAR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  EAR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% EAR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  EAR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

```{r  EAR 24hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% EAR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc all distances away;  EAR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% EAR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  EAR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% EAR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  EAR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```



ESR
```{r  ESR3hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(id %in% ESR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-5) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc all distances away;  ESR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% ESR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  ESR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% ESR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  ESR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

```{r  ESR 24hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% ESR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc all distances away;  ESR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% ESR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  ESR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% ESR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  ESR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```


LR
```{r  LR3hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(id %in% LR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-5) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc all distances away;  LR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% LR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  LR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% LR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  LR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

```{r  LR 24hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% LR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc all distances away;  LR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% LR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  LR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% LR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  LR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```


NR
```{r  NR3hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
  dplyr::filter(id %in% NR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-5) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc all distances away;  NR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% NR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  NR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% NR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_3_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("3 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  NR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

```{r  NR 24hour}
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% NR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc all distances away;  NR")
  
 
Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% NR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -20000&dist_to_NG < 20000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 20kb of neargene TSS;  NR")

Joined_tss_neargene %>% 
  dplyr::select(id,hgnc_symbol,dist_to_NG) %>% 
   dplyr::filter(id %in% NR_df$id) %>% 
  dplyr::filter(hgnc_symbol %in% Reheat_data$gene) %>% 
  left_join(., Reheat_data, by=c("hgnc_symbol"="gene")) %>% 
   dplyr::filter(fisher_pvalue<10^-10) %>% 
  dplyr::filter(dist_to_NG> -2000&dist_to_NG < 2000) %>% 
  left_join(., median_24_lfc, by=c("id"="peak")) %>% 
  ggplot(., aes(x=median,y=mean_lfc)) +
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
  ggtitle("24 hour median LFC and reheat mean_lfc within 2kb of neargene TSS;  NR")


# length(intersect(unique(Joined_tss_neargene$hgnc_symbol),unique(Reheat_data$gene)))
```

### how many are related to SNPs?
```{r}


SNPanno <- read.csv("data/annotated_gwas_SNPS.csv", row.names = 1)

SNPanno %>% 
  left_join(.,(Collapsed_peaks %>%
                 dplyr::select(peakid,NCBI_gene, SYMBOL, dist_to_NG)), by = c("peakid"="peakid")) %>% separate_longer_delim(., cols= SYMBOL,delim= ",") %>% 
  mutate(reheat= if_else(SYMBOL %in% gene_list_reheat$hgnc_symbol, "Reheat_gene","not_Reheat_gene")) %>% 
  group_by(mrc, reheat) %>% 
  tally()
```

```{r}
SNPanno %>% 
  left_join(.,(Collapsed_peaks %>%
                 dplyr::select(peakid,NCBI_gene, SYMBOL, dist_to_NG)), by = c("peakid"="peakid")) %>% separate_longer_delim(., cols= SYMBOL,delim= ",") %>% 
  mutate(reheat= if_else(SYMBOL %in% gene_list_reheat$hgnc_symbol, "Reheat_gene","not_Reheat_gene")) %>% 
  dplyr::filter(reheat =="Reheat_gene")
```

```{r making the median list RNA, }
toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS")

medianRNA_3 <- toplistall_RNA %>% 
  dplyr::filter(time=="3_hours") %>% 
  dplyr::select(id:logFC) %>% 
  pivot_wider(., id_cols = c(ENTREZID, SYMBOL), names_from = id, values_from = logFC) %>% 
  rowwise() %>% 
  mutate(Rmed_3_lfc=median(c_across(DNR:TRZ)))%>% dplyr::select(ENTREZID,SYMBOL,Rmed_3_lfc)


medianRNA_24 <- toplistall_RNA %>% 
  dplyr::filter(time=="24_hours") %>% 
  dplyr::select(id:logFC) %>% 
  pivot_wider(., id_cols = c(ENTREZID, SYMBOL), names_from = id, values_from = logFC) %>% 
  rowwise() %>% 
  mutate(Rmed_24_lfc=median(c_across(DNR:TRZ))) %>% dplyr::select(ENTREZID,SYMBOL,Rmed_24_lfc)

medianRNA_3 %>% 
  full_join(.,medianRNA_24) %>% 
  write.csv(., "data/other_papers/Median_LFC_RNA.csv")
```

