---
title: "Smaller_set_DAR"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r  package loading}
library(tidyverse)
library(ggsignif)
library(cowplot)
library(ggpubr)
library(scales)
library(kableExtra)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(gridExtra)
library(edgeR)
library(ggfortify)
library(limma)
library(ggVennDiagram)
library(devtools)
library(bedr)
```


```{r functions to have}
drug_pal <- c("#8B006D","#DF707E","#F1B72B", "#3386DD","#707031","#41B333")
pca_plot <-
  function(df,
           col_var = NULL,
           shape_var = NULL,
           title = "") {
    ggplot(df) + geom_point(aes_string(
      x = "PC1",
      y = "PC2",
      color = col_var,
      shape = shape_var
    ),
    size = 5) +
      labs(title = title, x = "PC 1", y = "PC 2") +
      scale_color_manual(values = c(
        "#8B006D",
        "#DF707E",
        "#F1B72B",
        "#3386DD",
        "#707031",
        "#41B333"
      ))
  }
pca_var_plot <- function(pca) {
  # x: class == prcomp
  pca.var <- pca$sdev ^ 2
  pca.prop <- pca.var / sum(pca.var)
  var.plot <-
    qplot(PC, prop, data = data.frame(PC = 1:length(pca.prop),
                                      prop = pca.prop)) +
    labs(title = 'Variance contributed by each PC',
         x = 'PC', y = 'Proportion of variance')
}

calc_pca <- function(x) {
  # Performs principal components analysis with prcomp
  # x: a sample-by-gene numeric matrix
  prcomp(x, scale. = TRUE, retx = TRUE)
}

get_regr_pval <- function(mod) {
  # Returns the p-value for the Fstatistic of a linear model
  # mod: class lm
  stopifnot(class(mod) == "lm")
  fstat <- summary(mod)$fstatistic
  pval <- 1 - pf(fstat[1], fstat[2], fstat[3])
  return(pval)
}

plot_versus_pc <- function(df, pc_num, fac) {
  # df: data.frame
  # pc_num: numeric, specific PC for plotting
  # fac: column name of df for plotting against PC
  pc_char <- paste0("PC", pc_num)
  # Calculate F-statistic p-value for linear model
  pval <- get_regr_pval(lm(df[, pc_char] ~ df[, fac]))
  if (is.numeric(df[, f])) {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_point() +
      geom_smooth(method = "lm") + labs(title = sprintf("p-val: %.2f", pval))
  } else {
    ggplot(df, aes_string(x = f, y = pc_char)) + geom_boxplot() +
      labs(title = sprintf("p-val: %.2f", pval))
  }
}
x_axis_labels = function(labels, every_nth = 1, ...) {
  axis(side = 1,
       at = seq_along(labels),
       labels = F)
  text(
    x = (seq_along(labels))[seq_len(every_nth) == 1],
    y = par("usr")[3] - 0.075 * (par("usr")[4] - par("usr")[3]),
    labels = labels[seq_len(every_nth) == 1],
    xpd = TRUE,
    ...
  )
}

```

This page is for DAR analysis using only individuals 1, 2, 3, and 6.

```{r data loading}

my_hc_filtered_counts_n45 <- readRDS("data/my_hc_filt_counts_n45.RDS")
##3 now have 151,800 high conf peaks
group_n45 <- c( rep(c(1,2,3,4,5,6,7,8,9,10,11,12),4))
group_n45 <- factor(group_n45, levels =c("1","2","3","4","5","6","7","8","9","10","11","12"))

##renaming colnames

efit2_n45 <- readRDS("data/filt_Peaks_efit2_n45.RDS")
##commented out to save time for future processing

group_n45 <- c(rep(c("DNR_24","DNR_3","DOX_24","DOX_3","EPI_24","EPI_3","MTX_24","MTX_3","TRZ_24","TRZ_3","VEH_24", "VEH_3"),4))

log_filt_hc_n45 <- my_hc_filtered_counts_n45 %>% 
  cpm(., log=TRUE) %>% as.data.frame()
results45 = decideTests(efit2_n45)
summary(results45)
```




### Evaluation of change in peaks

```{r toptable eval_n45}

V.DNR_3.top_n45= topTable(efit2_n45, coef=1, adjust.method="BH", number=Inf, sort.by="p") %>% rownames_to_column("peak")
V.DOX_3.top_n45= topTable(efit2_n45, coef=2, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.EPI_3.top_n45= topTable(efit2_n45, coef=3, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.MTX_3.top_n45= topTable(efit2_n45, coef=4, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.TRZ_3.top_n45= topTable(efit2_n45, coef=5, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.DNR_24.top_n45= topTable(efit2_n45, coef=6, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.DOX_24.top_n45= topTable(efit2_n45, coef=7, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.EPI_24.top_n45= topTable(efit2_n45, coef=8, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.MTX_24.top_n45= topTable(efit2_n45, coef=9, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")
V.TRZ_24.top_n45= topTable(efit2_n45, coef=10, adjust.method="BH", number=Inf, sort.by="p")%>% rownames_to_column("peak")

toplist_full_n45  <- list(V.DNR_3.top_n45, V.DOX_3.top_n45,V.EPI_3.top_n45,V.MTX_3.top_n45,V.TRZ_3.top_n45,V.DNR_24.top_n45, V.DOX_24.top_n45,V.EPI_24.top_n45,V.MTX_24.top_n45,V.TRZ_24.top_n45)
names(toplist_full_n45) <- c("DNR_3", "DOX_3","EPI_3","MTX_3","TRZ_3","DNR_24", "DOX_24","EPI_24","MTX_24","TRZ_24")
# toplist_n45 <-map_df(toplist_full_n45, ~as.data.frame(.x), .id="trt_time")
# # 
# toplist_n45 <- toplist_n45 %>%
#   separate(trt_time, into= c("trt","time"), sep = "_") %>%
#   mutate(trt=factor(trt, levels = c("DOX","EPI","DNR","MTX","TRZ"))) %>%
#   mutate(time = factor(time, levels = c("3", "24"), labels = c("3 hours", "24 hours")))
# saveRDS(toplist_n45,"data/toplist_n45.RDS")
toplist_n45 <- readRDS("data/toplist_n45.RDS")

```

### DAR Breakdown
```{r prop by trt_45}

toplist_n45 %>% 
  group_by(time, trt) %>% 
  mutate(sigcount = if_else(adj.P.Val < 0.05,'sig','notsig'))%>%
  count(sigcount) %>% 
  pivot_wider(id_cols = c(time,trt), names_from=sigcount, values_from=n) %>% 
  mutate(prop = sig/(sig+notsig)*100) %>% 
  mutate(prop=if_else(is.na(prop),0,prop)) %>% 
  ggplot(., aes(x=trt, y= prop))+
  geom_col(aes(fill=trt))+
  geom_text(aes(label = sprintf("%.2f",prop)),
            position=position_dodge(0.9),vjust=-.2 )+
  scale_fill_manual(values =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  facet_wrap(~time)+#labeller = (time = facettimelabel) )+
  theme_bw()+
  xlab("")+
  ylab("Percentage DAR ")+
  theme_bw()+
  ggtitle("Percent DARs (adj. P value <0.05) out of 151,800 regions")+
  scale_y_continuous(expand=expansion(c(0.02,.2)))+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_text(size = 8, color = "white", angle = 0),
        axis.text.y = element_text(size = 8, color = "black", angle = 0),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))
```
### Magnitude of response
```{r magresp_45}
toplist_n45 %>% 
  group_by(time, trt) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))
```

```{r histograms}


# toplist_n45 %>% 
#   group_by(time, trt) %>% 
#   dplyr::filter(trt != "TRZ") %>%
#   ggplot(., aes( x=(adj.P.Val)))+
#  geom_histogram(aes(y=after_stat(density)), colour="black", fill="white")+
#   geom_density(alpha=.2, fill="#FF6666")+
#   theme_bw()+
#   ggtitle("adj.P.Value density by time/trt")+
#   facet_wrap(time~trt)

toplist_n45 %>% 
  group_by(time, trt) %>% 
  # dplyr::filter(trt == "DOX") %>% 
  ggplot(., aes( x=(P.Value)))+
  geom_histogram(aes(y=after_stat(density)), colour="black", fill="white")+
  geom_density(alpha=.2, fill="#FF6666")+
  theme_bw()+
  ggtitle("P.Value density by time/trt")+
  facet_wrap(time~trt)


```
Treatment specific  elements

### Venn Diagrams
```{r trttime Venn}
# saveRDS(toplist_full_n45, "data/DEG_toplist_sep_n45.RDS")
toplist_full_n45 <- readRDS("data/DEG_toplist_sep_n45.RDS")
siglist_n45 <- readRDS("data/siglist_n45.RDS")
# sigDNR_3_n45 = V.DNR_3.top_n45[V.DNR_3.top_n45$adj.P.Val < 0.05 , ] %>% rownames_to_column("peak")
# sigDOX_3_n45 = V.DOX_3.top_n45[V.DOX_3.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigEPI_3_n45 = V.EPI_3.top_n45[V.EPI_3.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigMTX_3_n45 = V.MTX_3.top_n45[V.MTX_3.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigTRZ_3_n45 = V.TRZ_3.top_n45[V.TRZ_3.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigDNR_24_n45 = V.DNR_24.top_n45[V.DNR_24.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigDOX_24_n45 = V.DOX_24.top_n45[V.DOX_24.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigEPI_24_n45 = V.EPI_24.top_n45[V.EPI_24.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigMTX_24_n45 = V.MTX_24.top_n45[V.MTX_24.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")
# sigTRZ_24_n45 = V.TRZ_24.top_n45[V.TRZ_24.top_n45$adj.P.Val < 0.05 , ]%>% rownames_to_column("peak")

# 
# # 
# siglist_n45 <- list(sigDNR_3_n45, sigDOX_3_n45, sigEPI_3_n45, sigMTX_3_n45, sigTRZ_3_n45,
#                     sigDNR_24_n45,sigDOX_24_n45 ,sigEPI_24_n45 ,sigMTX_24_n45 ,sigTRZ_24_n45 )
# names(siglist_n45) <- c("DNR_3_n45", "DOX_3_n45","EPI_3_n45","MTX_3_n45","TRZ_3_n45","DNR_24_n45", "DOX_24_n45","EPI_24_n45","MTX_24_n45","TRZ_24_n45")
# saveRDS(siglist_n45, "data/siglist_n45.RDS")
siglist_n45_peaks <- lapply(siglist_n45, "[",1)
three_hour_n45 <-siglist_n45_peaks[1:4]

ggVennDiagram(list(three_hour_n45$DNR_3_n45$peak,three_hour_n45$DOX_3_n45$peak,three_hour_n45$EPI_3_n45$peak,three_hour_n45$MTX_3_n45$peak), 
              category.names = c("DNR","DOX","EPI","MTX"), 
              label = "count")+
  labs(title = "Three hour DAR by treatment")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))

ggVennDiagram(list(siglist_n45_peaks$DNR_24_n45$peak,siglist_n45_peaks$DOX_24_n45$peak,siglist_n45_peaks$EPI_24_n45$peak,siglist_n45_peaks$MTX_24_n45$peak),
              category.names = c("DNR","DOX","EPI","MTX"), label = "count")+
  labs(title = "24 hour DAR by treatment")+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5))




```
Examples of genes for each data set:
#### DOX

```{r DAR by trt DOX }

toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$DOX_3_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("DOX specific log2Foldchange at 3 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$DOX_24_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("DOX specific log2Foldchange at 24 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


```



```{r DOXexpressionboxplot}
###filter out names of top 3 peaks in DOX 3 and 24

DOX_3_top3_n45 <-V.DOX_3.top_n45[1:3,1]

log_filt_hc_n45 %>%  
  dplyr::filter(row.names(.) %in% DOX_3_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour DOX")+
  scale_fill_manual(values = drug_pal)


DOX_24_top3_n45 <-V.DOX_24.top_n45[1:3,1]

log_filt_hc_n45 %>%  
  dplyr::filter(row.names(.) %in% DOX_24_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour DOX")+
  scale_fill_manual(values = drug_pal)


```



#### EPI

```{r DAR by trt EPI }

toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$EPI_3_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("EPI specific log2Foldchange at 3 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$EPI_24_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("EPI specific log2Foldchange at 24 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


```

```{r EPIbox}

EPI_3_top3_n45 <- V.EPI_3.top_n45[1:3,1]

log_filt_hc_n45 %>%  
  dplyr::filter(row.names(.) %in% EPI_3_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour EPI")+
  scale_fill_manual(values = drug_pal)


EPI_24_top3_n45 <-V.EPI_24.top_n45[1:3,1]

log_filt_hc_n45 %>%  
  dplyr::filter(row.names(.) %in% EPI_24_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour EPI")+
  scale_fill_manual(values = drug_pal)


```

#### DNR

```{r DAR by trt DNR }

toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$DNR_3_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("DNR specific log2Foldchange at 3 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$DNR_24_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("DNR specific log2Foldchange at 24 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


```


```{r DNR expression}
DNR_3_top3_n45 <- V.DNR_3.top_n45[1:3,1]


V.EPI_3.top_n45 %>% 
  dplyr::filter(peak=="chr4.173529475.173530537")

row.names(log_filt_hc_n45) <- row.names(my_hc_filtered_counts_n45)
log_filt_hc_n45 %>% 
  dplyr::filter(row.names(.) %in% DNR_3_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour DNR")+
  scale_fill_manual(values = drug_pal)


DNR_24_top3_n45 <- V.DNR_24.top_n45[1:3,1]
log_filt_hc_n45 <- my_hc_filtered_counts_n45 %>% 
  cpm(., log=TRUE) %>% as.data.frame()
row.names(log_filt_hc_n45) <- row.names(my_hc_filtered_counts_n45)
log_filt_hc_n45 %>% 
  dplyr::filter(row.names(.) %in% DNR_24_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour DNR")+
  scale_fill_manual(values = drug_pal)

```


#### MTX

```{r DAR by trt MTX }

toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$MTX_3_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("MTX specific log2Foldchange at 3 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


toplist_n45 %>% 
  dplyr::filter(peak %in% siglist_n45_peaks$MTX_24_n45$peak) %>% 
  ggplot(., aes(x=trt, y=logFC))+
  geom_boxplot(aes(fill=trt))+
  ggpubr::fill_palette(palette =drug_pal)+
  guides(fill=guide_legend(title = "Treatment"))+
  # facet_wrap(sigcount~time)+
  theme_bw()+
  xlab("")+
  ggtitle("MTX specific log2Foldchange at 24 hours")+
  ylab(expression("Log"[2]*" fold change"))+
  theme_bw()+
  facet_wrap(~time)+
  theme(plot.title = element_text(size = rel(1.5), hjust = 0.5),
        axis.title = element_text(size = 15, color = "black"),
        # axis.ticks = element_line(linewidth = 1.5),
        # axis.line = element_line(linewidth = 1.5),
        strip.background = element_rect(fill = "transparent"),
        axis.text.x = element_blank(),
        strip.text.x = element_text(size = 12, color = "black", face = "bold"))


```


```{r MTXboxplot}

MTX_3_top3_n45 <- V.MTX_3.top_n45[1:3,1]

log_filt_hc_n45 %>%  
  dplyr::filter(row.names(.) %in% MTX_3_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 3 hour MTX")+
  scale_fill_manual(values = drug_pal)


MTX_24_top3_n45 <- V.MTX_24.top_n45[1:3,1]

log_filt_hc_n45 %>%  
  dplyr::filter(row.names(.) %in% MTX_24_top3_n45) %>% 
  mutate(Peak = row.names(.)) %>% 
  pivot_longer(cols = !Peak, names_to = "sample", values_to = "counts") %>% 
  separate("sample", into = c("indv","trt","time")) %>% 
  mutate(time=factor(time, levels = c("3h","24h"))) %>% 
  mutate(trt=factor(trt, levels= c("DOX","EPI","DNR","MTX","TRZ","VEH"))) %>% 
  ggplot(., aes (x = time, y=counts))+
  geom_boxplot(aes(fill=trt))+
  facet_wrap(Peak~.)+
  ggtitle("top 3 DAR in 24 hour MTX")+
  scale_fill_manual(values = drug_pal)


```


### TSS elements
```{r examin TSS elements, message=FALSE, warning=FALSE}

# data.frame("DOX_3_n45" = siglist_n45$DOX_3_n45$peak) %>%
#     mutate(name =DOX_3_n45) %>%
#   separate(DOX_3_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/DOX_3_n45.bed", format="bed", ignore.strand = FALSE)
#     # write.table(.,file = "data/n45_bedfiles/DOX_3_n45.bed", row.names =FALSE,col.name=FALSE)
# 
# data.frame("EPI_3_n45" = siglist_n45$EPI_3_n45$peak) %>%
#     mutate(name =EPI_3_n45) %>%
#   separate(EPI_3_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/EPI_3_n45.bed", format="bed", ignore.strand = FALSE)
    # write.table(.,file = "data/n45_bedfiles/EPI_3_n45.bed", row.names =FALSE,col.name=FALSE)


# data.frame("DNR_3_n45" = siglist_n45$DNR_3_n45$peak) %>%
#     mutate(name = DNR_3_n45) %>%
#   separate(DNR_3_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/DNR_3_n45.bed", format="bed", ignore.strand = FALSE)
#     # write.table(.,file = "data/n45_bedfiles/DNR_3_n45.bed", row.names =FALSE,col.name=FALSE)


# data.frame("MTX_3_n45" = siglist_n45$MTX_3_n45$peak) %>%
#     mutate(name =MTX_3_n45) %>%
#   separate(MTX_3_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/MTX_3_n45.bed", format="bed", ignore.strand = FALSE)
#     # write.table(.,file = "data/n45_bedfiles/MTX_3_n45.bed", row.names =FALSE,col.name=FALSE)


# data.frame("DOX_24_n45" = siglist_n45$DOX_24_n45$peak) %>%
#     mutate(name =DOX_24_n45) %>%
#   separate(DOX_24_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/DOX_24_n45.bed", format="bed", ignore.strand = FALSE)
    # write.table(.,file = "data/n45_bedfiles/DOX_24_n45.bed", row.names =FALSE,col.name=FALSE)

# data.frame("EPI_24_n45" = siglist_n45$EPI_24_n45$peak) %>%
#     mutate(name =EPI_24_n45) %>%
#   separate(EPI_24_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#      na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/EPI_24_n45.bed", format="bed", ignore.strand = FALSE)
#     # write.table(.,file = "data/n45_bedfiles/EPI_24_n45.bed", row.names =FALSE,col.name=FALSE)

# data.frame("DNR_24_n45" = siglist_n45$DNR_24_n45$peak) %>%
#     mutate(name =DNR_24_n45) %>%
#   separate(DNR_24_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/DNR_24_n45.bed", format="bed", ignore.strand = FALSE)
#     # write.table(.,file = "data/n45_bedfiles/DNR_24_n45.bed", row.names =FALSE,col.name=FALSE)

# test <- data.frame("all_peaks"= row.names(log_filt_hc_n45)) %>% 
#   mutate(name = all_peaks) %>% 
#   separate(all_peaks, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#   dplyr::filter(chr != "chrUn") %>% 
#   na.omit(.) %>%
#     dplyr::select(chr, start,end,name) %>% 
#   GRanges(.)
# rtracklayer::export.bed(test, con="data/n45_bedfiles/DAR_DEG_background.bed", format="bed", ignore.strand = FALSE)
# 
# 
# data.frame("MTX_24_n45" = siglist_n45$MTX_24_n45$peak) %>%
#     mutate(name =MTX_24_n45) %>%
#   separate(MTX_24_n45, into = c( "chr","start","end")) %>%
#       mutate(start=as.integer(start), end=as.integer(end)) %>%
#     dplyr::select(chr, start,end,name) %>%
#     na.omit(.) %>% GRanges(.) %>% 
#   rtracklayer::export.bed(., con="data/n45_bedfiles/MTX_24_n45.bed", format="bed", ignore.strand = FALSE)
#     # write.table(.,file = "data/n45_bedfiles/MTX_24_n45.bed", row.names =FALSE,col.name=FALSE)
# 
# txdb <- TxDb.Hsapiens.UCSC.hg38.knownGene
# TSS = getBioRegion(TxDb=txdb, upstream=2000, downstream=2000, by = "gene",
#                    type = "start_site")

# DOX_3_n45_gr <- bed_to_granges("data/n45_bedfiles/DOX_3_n45.bed")
# EPI_3_n45_gr <- bed_to_granges("data/n45_bedfiles/EPI_3_n45.bed")
# DNR_3_n45_gr <- bed_to_granges("data/n45_bedfiles/DNR_3_n45.bed")
# MTX_3_n45_gr <- bed_to_granges("data/n45_bedfiles/MTX_3_n45.bed")
# MTX_24_n45_gr <- bed_to_granges("data/n45_bedfiles/MTX_24_n45.bed")
# EPI_24_n45_gr <- bed_to_granges("data/n45_bedfiles/EPI_24_n45.bed")
# DNR_24_n45_gr <- bed_to_granges("data/n45_bedfiles/DNR_24_n45.bed")
# DOX_24_n45_gr <- bed_to_granges("data/n45_bedfiles/DOX_24_n45.bed")
# background_n45_gr <- bed_to_granges("data/n45_bedfiles/DAR_DEG_background.bed")
# 
# 
# my_list_3_n45<-list(DOX_3_n45_gr,EPI_3_n45_gr,DNR_3_n45_gr,MTX_3_n45_gr, background_n45_gr)
# peakAnnoList_3_n45<- lapply(my_list_3_n45, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# names(peakAnnoList_3_n45) <- c("DOX_3_n45" ,"EPI_3_n45", "DNR_3_n45", "MTX_3_n45", "background")
# saveRDS(peakAnnoList_3_n45, "data/peakAnnoList_3_n45.RDS")
# 
# my_list_24_n45<-list(DOX_24_n45_gr,EPI_24_n45_gr,DNR_24_n45_gr,MTX_24_n45_gr, background_n45_gr)
# peakAnnoList_24_n45<- lapply(my_list_24_n45, annotatePeak, tssRegion =c(-2000,2000), TxDb= txdb)
# names(peakAnnoList_24_n45) <- c("DOX_24_n45" ,"EPI_24_n45", "DNR_24_n45", "MTX_24_n45", "background")
# saveRDS(peakAnnoList_24_n45, "data/peakAnnoList_24_n45.RDS")
# TRZ_3_toplist_ATAC
# TRZ_24_toplist_ATAC
# 
# PeakAnno


peakAnnoList_3_n45 <- readRDS("data/peakAnnoList_3_n45.RDS")
peakAnnoList_24_n45<- readRDS("data/peakAnnoList_24_n45.RDS")

plotAnnoBar(peakAnnoList_3_n45) +ggtitle ("Genomic Feature Distribution, DAR 3 hours n45")

plotAnnoBar(peakAnnoList_24_n45) +ggtitle ("Genomic Feature Distribution, DAR 24 hours n45")

```
below this sentence, I will try to graph DEG_RNA LogFC against DAR regions by treatment-time sets 


### DAR v DEG LFC

This will be aligning the significant vs not significant DEGs against all TSS promoters

#### DEG_RNA LogFC adjustment

```{r  promotor list}

library(smplot2)
toplistall_RNA <- readRDS("data/other_papers/toplistall_RNA.RDS") 

###Because of how I applied the DEG system in RNA-seq analysis, the lFC is opposite of the 
###counts.   I did trt-veh instead of veh-trt.  therefore I need to multiply lfc by -1 to get t
###the right correlation.

toplistall_RNA <- toplistall_RNA %>% 
  mutate(logFC = logFC*(-1))

list2env(peakAnnoList_3_n45, envir = .GlobalEnv)
list2env(peakAnnoList_24_n45, envir = .GlobalEnv)
##split into granges objects
###make dataframe of each EAR,ESR,LR,NR promoters

DOX_3_df <- as.data.frame(DOX_3_n45)
DOX_3_promotor <- DOX_3_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
DOX_3_promo_gr <-  GRanges(DOX_3_promotor)

EPI_3_df <- as.data.frame(EPI_3_n45)
EPI_3_promotor <- EPI_3_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
EPI_3_promo_gr <-  GRanges(EPI_3_promotor)

DNR_3_df <- as.data.frame(DNR_3_n45)
DNR_3_promotor <- DNR_3_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
DNR_3_promo_gr <-  GRanges(DNR_3_promotor)

MTX_3_df <- as.data.frame(MTX_3_n45)
MTX_3_promotor <- MTX_3_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
MTX_3_promo_gr <-  GRanges(MTX_3_promotor)

# TRZ_3_df <- as.data.frame(TRZ_3_n45)
# TRZ_3_promotor <- TRZ_3_df %>% 
#   dplyr::filter(annotation=="Promoter (<=1kb)")
# TRZ_3_promo_gr <-  GRanges(TRZ_3_promotor)
DOX_24_df <- as.data.frame(DOX_24_n45)
DOX_24_promotor <- DOX_24_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
DOX_24_promo_gr <-  GRanges(DOX_24_promotor)

EPI_24_df <- as.data.frame(EPI_24_n45)
EPI_24_promotor <- EPI_24_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
EPI_24_promo_gr <-  GRanges(EPI_24_promotor)

DNR_24_df <- as.data.frame(DNR_24_n45)
DNR_24_promotor <- DNR_24_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
DNR_24_promo_gr <-  GRanges(DNR_24_promotor)

MTX_24_df <- as.data.frame(MTX_24_n45)
MTX_24_promotor <- MTX_24_df %>% 
  dplyr::filter(annotation=="Promoter (<=1kb)")
MTX_24_promo_gr <-  GRanges(MTX_24_promotor)

```

##### 3 hour promotor log FC correlation

```{r DOX 3 hour,fig.width=10}



DOX_3_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("Dox 3 hour promotor DARs")

EPI_3_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("EPI 3 hour promotor DARs")

DNR_3_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("DNR 3 hour promotor DARs")

MTX_3_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("MTX 3 hour promotor DARs")









```  


```{r 24 hour promotor sets, fig.width=10}


DOX_24_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("Dox 24 hour promotor DARs")

EPI_24_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("EPI 24 hour promotor DARs")

DNR_24_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("DNR 24 hour promotor DARs")

MTX_24_promotor %>% 
  dplyr::select(id,geneId) %>% 
  left_join(.,toplist_n45, by = c("id"="peak" )) %>% 
  dplyr::select(id:logFC) %>% 
  rename(geneId="ENTREZID") %>% 
  full_join(., (toplistall_RNA %>% 
  # dplyr:: filter(ENTREZID %in% EAR_promoter_peaks) %>% 
  dplyr::select(time:ENTREZID,logFC) %>% mutate(time=factor(time, levels= c("3_hours","24_hours"), labels = c("3 hours","24 hours"))) %>% 
  rename(id="trt")), by= c("ENTREZID", "trt","time")) %>% 
  ggplot(., aes(x=logFC.x,y =logFC.y))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')+
facet_wrap(~time+trt, nrow =2, ncol = 5)+
  xlab("log FC DAR")+
  ylab("log FC RNA")+
  ggtitle("MTX 24 hour promotor DARs")

```

### 3 hour Correlation of all peaks

```{r look correlation plots 3 hours}

toplist_n45 %>% 
  dplyr::filter(time=="3 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


# fulltoplistpivot
# 
# toplist_n45 %>% 
#   dplyr::filter(time=="3 hours") %>% 
#   dplyr::select(trt:logFC) %>% 
#   pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) 
# toplist_n45 %>% 
#   dplyr::filter(time=="24 hours") %>% 
#   dplyr::select(trt:logFC) %>% 
#   pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) 



```



### EAR Cluster

```{r EARlook correlation plots 3 hours}

toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% EAR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% EAR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="3 hours", peak %in% EAR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% EAR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```

### ESR Cluster

```{r ESRlook correlation plots 3 hours}

toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% ESR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% ESR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="3 hours", peak %in% ESR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% ESR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```


### LR Cluster

```{r LRlook correlation plots 3 hours}

toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% LR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% LR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="3 hours", peak %in% LR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% LR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```


### NR Cluster

```{r NRlook correlation plots 3 hours}

toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% NR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% NR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="3 hours", peak %in% NR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="3 hours", peak %in% NR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```




### 24 hour Correlation of all peaks

```{r look correlation plots 24 hours}

toplist_n45 %>% 
  dplyr::filter(time=="24 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours") %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')





```

### EAR Cluster

```{r EARlook correlation plots 24 hours}

toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% EAR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% EAR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="24 hours", peak %in% EAR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% EAR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```

### ESR Cluster

```{r ESRlook correlation plots 24 hours}

toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% ESR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% ESR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="24 hours", peak %in% ESR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% ESR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```


### LR Cluster

```{r LRlook correlation plots 24 hours}

toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% LR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% LR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="24 hours", peak %in% LR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% LR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```


### NR Cluster

```{r NRlook correlation plots 24 hours}

toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% NR_peak_list$peakid) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=DNR))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% NR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=EPI))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
dplyr::filter(time=="24 hours", peak %in% NR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=MTX))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')
toplist_n45 %>% 
  dplyr::filter(time=="24 hours", peak %in% NR_peak_list$peakid) %>% 
  dplyr::select(trt:logFC) %>% 
  pivot_wider(id_cols= c(peak,time), names_from=trt, values_from = logFC) %>% 
  ggplot(., aes (x=DOX, y=TRZ))+
  geom_point()+
  geom_smooth(method="lm")+
  sm_statCorr(corr_method = 'pearson')


```


