---
title: "TEs and my data"
author: "ERM"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(
	echo = TRUE,
	message = FALSE,
	warning = FALSE
)
```

```{css, echo=FALSE}
pre {
  max-height: 400px;
  overflow-y: auto;
}

pre[class] {
  max-height: 200px;
}
```

```{r package loading, message=FALSE, warning=FALSE}
library(tidyverse)
library(kableExtra)
library(broom)
library(RColorBrewer)
library(ChIPseeker)
library("TxDb.Hsapiens.UCSC.hg38.knownGene")
library("org.Hs.eg.db")
library(rtracklayer)
library(edgeR)
library(ggfortify)
library(limma)
library(readr)
library(BiocGenerics)
library(gridExtra)
library(VennDiagram)
library(scales)
library(BiocParallel)
library(ggpubr)
library(devtools)
library(biomaRt)
library(eulerr)
library(smplot2)
library(genomation)
library(ggsignif)
library(plyranges)
library(ggrepel)
library(readr)

```
##### loading data

This is where I pull in the repeatmasker file taken from UCSC genomebrowser, the peaks assigned with the closest expressed genes as 'neargenes' by TSS,  the same peaks list, only condensing to unique peaks (some are assigned to two neargene), I call the collapsed_peaks and the peaks assigned to each MRC (EAR, etc...).  
With the TSS data and the collapsed data, I make granges files.  I also separate out the LINEs, SINEs, LTRs, DNAs, and Retroposons from the repeatmasker to make granges from those sets.  
```{r first data sets}
repeatmasker <- read.delim("data/other_papers/repeatmasker.tsv")

TSS_NG_data <- read_delim("data/n45_bedfiles/TSS_NG_data.tsv", 
    delim = "\t", escape_double = FALSE, 
    trim_ws = TRUE)
Collapsed_peaks <- read_delim("data/n45_bedfiles/TSS_NG_data_collapsed_peaks.tsv",
                              delim = "\t", 
                              escape_double = FALSE, 
                              trim_ws = TRUE)
TSS_data_gr <- TSS_NG_data %>% 
  dplyr::filter(seqnames != "chrX") %>% 
  dplyr::filter(seqnames != "chrY") %>%
  GRanges()

Col_TSS_data_gr <- Collapsed_peaks %>% 
  dplyr::filter(seqnames != "chrX") %>% 
  dplyr::filter(seqnames != "chrY") %>%
  GRanges()

reClass_list <- repeatmasker %>% 
  distinct(repClass)

Line_repeats <- repeatmasker %>% 
  dplyr::filter(repClass == "LINE") %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "genoName", start.field = "genoStart", end.field = "genoEnd",starts.in.df.are.0based=TRUE)

Sine_repeats <- repeatmasker %>% 
  dplyr::filter(repClass == "SINE") %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "genoName", start.field = "genoStart", end.field = "genoEnd",starts.in.df.are.0based=TRUE)

LTR_repeats <- repeatmasker %>% 
  dplyr::filter(repClass == "LTR") %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "genoName", start.field = "genoStart", end.field = "genoEnd",starts.in.df.are.0based=TRUE)

DNA_repeats <- repeatmasker %>% 
  dplyr::filter(repClass == "DNA") %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "genoName", start.field = "genoStart", end.field = "genoEnd",starts.in.df.are.0based=TRUE)

retroposon_repeats <- repeatmasker %>% 
  dplyr::filter(repClass == "Retroposon") %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "genoName", start.field = "genoStart", end.field = "genoEnd",starts.in.df.are.0based=TRUE)


all_TEs_gr <- repeatmasker %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "genoName", start.field = "genoStart", end.field = "genoEnd",starts.in.df.are.0based=TRUE)


peakAnnoList_n45_motif <- readRDS("data/peakAnnoList_n45_motif.RDS")

EAR_df <- as.data.frame(peakAnnoList_n45_motif$EAR_n45_gr)
# EAR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$EAR_n45_gr)

ESR_df <- as.data.frame(peakAnnoList_n45_motif$ESR_n45_gr)
# ESR_df_gr <- as.GRanges(peakAnnoList_n45_motif$ESR_n45_gr)

LR_df <- as.data.frame(peakAnnoList_n45_motif$LR_n45_gr)
# LR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$LR_n45_gr)

NR_df <- as.data.frame(peakAnnoList_n45_motif$NR_n45_gr)
# NR_df_gr <-  as.GRanges(peakAnnoList_n45_motif$NR_n45_gr)


```

##  TE distrubution:

The code below uses the TE data sets to create dataframes that contain all peaks that overlap a TE. Additionally, I wanted to know size distribution of the overlaps(width) between TEs and my peaks, the size of the TEs and their size distribution, and the size distribution of my peaks.  This was to determine a cutoff across TEs to minimize size bais. My first attempt was a threshold of >50% of TE needed to be covered by a peak to infer the peak contained the TE. In the plots, the median of each density distribution is shown by the dotted line.

```{r  data about TEs}


Col_TSS_data_gr %>% 
  as.data.frame %>% 
   ggplot(., aes(x=width))+
    geom_density(color="darkblue",fill="lightblue",aes(alpha = 0.5))+
  geom_vline(data=Col_TSS_data_gr$width., aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of peak length across all peaks",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))



all_TEs_gr %>% 
  as.data.frame() %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="LINE") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of lengths of all LINEs across human genome",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))

all_TEs_gr %>% 
  as.data.frame() %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="SINE") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of lengths of all SINEs across human genome")

all_TEs_gr %>% 
  as.data.frame() %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="LTR") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of lengths of all LTRs across human genome",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))

all_TEs_gr %>% 
  as.data.frame() %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="DNA") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of lengths of all DNA-TEs across human genome",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))

all_TEs_gr %>% 
  as.data.frame() %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="Retroposon") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of lengths of all Retroposons across human genome",subtitle = " limited x axis")+
   coord_cartesian(xlim= c(0,2500))

all_TEs_gr %>% 
  as.data.frame() %>% 
  dplyr::select(repClass, width) %>% 
  # dplyr::filter(repClass=="LTR") %>%
  group_by(repClass) %>% 
  summarise(med.width=median(width),mean.width=mean(width))

fullDF_overlap <- join_overlap_intersect(TSS_data_gr,all_TEs_gr)
fullDF_overlap %>% 
  as.data.frame() %>% 
  group_by(repClass) %>%  
  tally %>% 
  kable(., caption="Table 1: Count of peaks by TE class; overlap 1 bp or greater") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)

subsetByOverlaps(TSS_data_gr,all_TEs_gr) %>% as.data.frame %>% 
   ggplot(., aes(x=width))+
    geom_density(color="darkblue",fill="lightblue",aes(alpha = 0.5))+
   geom_vline(aes(xintercept = median(width)), linetype = 2)+
  theme_classic()+
  ggtitle("Distribution of overlaps between all TEs and all my peaks",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))

### This is how I subset only those peaks who cover >50% of TEs
hits <- findOverlaps(TSS_data_gr,all_TEs_gr)
overlaps <- pintersect(TSS_data_gr[queryHits(hits)], all_TEs_gr[subjectHits(hits)])
percentOverlap <- width(overlaps) / width(all_TEs_gr[subjectHits(hits)])
hits <- hits[percentOverlap > 0.5]
### THis actually did not work well---I ended up losing data.  What I wanted was a data frame that had metadata from both the TE overlap and the peak metadata, so I could easily sort and manipulation.  
testingol <- TSS_data_gr[queryHits(hits)]
testingol %>% as.data.frame() %>% 
  left_join(., (fullDF_overlap %>% as.data.frame(.)), by =c("seqnames"="seqnames","start"="start","end"="end","peakid"="peakid", "NG_start"="NG_start", "end_position"="end_position", "entrezgene_id"="entrezgene_id", "ensembl_gene_id"="ensembl_gene_id","dist_to_NG"="dist_to_NG", "width"="width", "strand"="strand", "hgnc_symbol" = "hgnc_symbol")) %>% 
  group_by(repClass) %>% 
  tally %>% 
  kable(., caption=" Table 2: Count of peaks by TE class; overlap> 50%") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)
```
  The first dataframe to subset peaks that overlap >50% of a TE, was using the full neargene dataframe, where a peak is listed more than once because it was assigned more than one neargene ( one-to-many relationships).   I changed the code to use the 'collapsed' data frame.  This means the data frame was simplified to only include peaks one time, but those peaks that were assigned to more than one neargene had the assigned neargenes condensed and separated by a comma into the same column to create a one-to-one relationship dataframe.  (yes, wordy I know)
  
One new observation was how I was subsetting the peaks that covered >50% of the specific TE.  I was using the query hits and later joining the peaks back to a dataframe, forgetting there were peaks that hit more than one TE.  I lost the metadata between the data sets in this process and missed the other possible sets of peak-TE overlap due to how I joined the data frame.  The last table gave be the initial hint I did it wrong  (NA=117,556) I checked this by subsetting using query data and started realizing where the error was.  Below is how I began to fix the lost data issue.
```{r run again}
######################################################
all_TEs_gr$TE_width <- width(all_TEs_gr)
Col_TSS_data_gr$peak_width <- width(Col_TSS_data_gr)
Col_fullDF_overlap <- join_overlap_intersect(Col_TSS_data_gr,all_TEs_gr)
Col_fullDF_overlap %>% 
  as.data.frame() %>% 
  group_by(repClass) %>%  
  tally %>% 
  kable(., caption=" Table 3: Count of peaks by TE class; overlap at least 1 bp; using one:one df ") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)

Col_fullDF_overlap %>% 
   as.data.frame %>% 
  mutate(per_ol= width/TE_width) %>% 
  dplyr::filter(per_ol>0.5) %>% 
  group_by(repClass) %>% 
  tally() %>% 
  kable(., caption=" Table 4:Count of peaks by TE class; overlap of >50% of TE; newway ") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)
 
Filter_TE_list <- Col_fullDF_overlap %>% 
   as.data.frame %>% 
  mutate(per_ol= width/TE_width) 
  # dplyr::filter(per_ol>0.5)
### Old way
hits_col <- findOverlaps(Col_TSS_data_gr,all_TEs_gr)

overlaps_col <- pintersect(Col_TSS_data_gr[queryHits(hits_col)], all_TEs_gr[subjectHits(hits_col)])
percentOverlap_col <- width(overlaps_col) / width(all_TEs_gr[subjectHits(hits_col)])
hits_col2 <- hits_col[percentOverlap_col > 0.5]
## bad dataframe
testingol_col2 <- Col_TSS_data_gr[queryHits(hits_col2)]
## better alternative way
just_TEdata <- all_TEs_gr[subjectHits(hits_col2)]


just_TEdata %>% 
  as.data.frame %>% 
  distinct(., .keep_all = TRUE) %>% 
  group_by(repClass) %>% 
  tally %>% 
  kable(., caption=" Table 5: Count of peaks by TE class Just TE list; overlap> 50%; using one:one df") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)
  
# 
# 
# testingol_col2 %>% 
#   as.data.frame() %>% 
#   group_by(peakid) %>% 
#   tally %>% 
#   summary()
Unique_peak_overlap <- Col_fullDF_overlap %>%
  as.data.frame() %>%
  distinct(peakid)

peak_overlap_50unique <- testingol_col2 %>%
  as.data.frame() %>%
  distinct(peakid) 
  


```
Note that the counts of peaks total to more than the total number of peaks that overlap a TE because many peaks overlap multiple elements (generally the very small ones)

A summary of numbers of peaks is below:
* Total number of peaks = `r length(Collapsed_peaks$peakid)`
* Total number of peaks overlapping at least 1 TE = `r length(Unique_peak_overlap$peakid)`
* Total number of peaks overlapping by >50% TE length = `r length(peak_overlap_50unique$peakid)`

 Tables 3 and 4 above used the "new" way of sub-setting (keeping all metadata organized vs the sub-setting code I found on the internet).  Just to verify the numbers were the same, I ran the data using the old method and compared the numbers.  They are identical (Table 5) to each other.  Success!


### Distribution of TE overlaps by peaks according to class

Below are plots of the distribution of overlapping widths between peaks  and TEs.  The first plot is all TE Class, and the 2nd plot limits the classes to LINEs, SINEs, LTRs, DNAs, and Retroposons.
```{r another set of plots}
rep_other_names<- repeatmasker %>% 
 distinct(repClass) %>% 
  rbind("Other")
scale_fill_repeat <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange",
                             "darkgrey"), unique(rep_other_names$repClass)), 
      ...
    )
}
Col_fullDF_overlap %>%  
  as.data.frame %>% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color="darkblue",aes(alpha = 0.5))+
  theme_classic()+
  ggtitle("Distribution of all overlapping widths in my data",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,750))+
  scale_fill_repeat()
  
Col_fullDF_overlap %>%  
  as.data.frame %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass =="LTR"|repClass=="DNA"|repClass =="Retroposon") %>% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color="darkblue",aes(alpha = 0.5))+
  theme_classic()+
  ggtitle("Distribution of all overlapping peak-TE widths",subtitle = "Just LINE, SINE,LTR, DNA, Retroposons; limited x axis")+
  coord_cartesian(xlim= c(0,1550))+
  scale_fill_repeat()


Peak_TE_overlapbreakdown <- Col_TSS_data_gr %>% as.data.frame %>% 
  dplyr::select(peakid) %>% 
  left_join(.,(Col_fullDF_overlap %>% as.data.frame)) %>% 
   mutate(TEstatus=if_else(is.na(repClass),"not_TE_peak","TE_peak")) %>% 
   mutate(mrc=if_else(peakid %in% EAR_df$id, "EAR",
                     if_else(peakid %in% ESR_df$id,"ESR",
                             if_else(peakid %in% LR_df$id,"LR",
                                     if_else(peakid %in% NR_df$id,"NR","not_mrc"))))) %>%  mutate(per_ol= width/TE_width) 
  
 # Peak_TE_overlapbreakdown 
 ## takes too long to render as a kable table
  #  kable(., caption=" Table 6: complete data frame annotated with metadata from TE and peak joined data frames.  per_ol column is the percent overlap for the indicated peaks") %>% 
  # kable_paper("striped", full_width = TRUE) %>%
  # kable_styling(full_width = FALSE, font_size = 14)
 
 TE_mrc_status_list <- Peak_TE_overlapbreakdown %>% 
      dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol)
 
 Peak_TE_overlapbreakdown %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
   mutate(mrc="all_peaks") %>% 
   rbind(TE_mrc_status_list) %>% 
   mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("TE status by MRC and Family","all"))
   
   Peak_TE_overlapbreakdown %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
   mutate(mrc="all_peaks") %>% 
   rbind(TE_mrc_status_list) %>% 
   mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
     group_by(mrc, TEstatus) %>% 
     count() %>% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %>% 
     rowwise() %>% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %>% 
     ungroup() %>% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c("TEstatus"), values_to = "n") %>% 
     mutate(percent_mrc= n/summary*100)
     
 
     
   
```

```{r another set of limited plots}
rep_other_names<- repeatmasker %>% 
 distinct(repClass) %>% 
  rbind("Other")
scale_fill_repeat <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange",
                             "darkgrey"), unique(rep_other_names$repClass)), 
      ...
    )
}
Col_fullDF_overlap %>%  
  as.data.frame %>% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color="darkblue",aes(alpha = 0.5))+
  theme_classic()+
  ggtitle("Distribution of all overlapping widths in my data",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,750))+
  scale_fill_repeat()
  
Col_fullDF_overlap %>%  
  as.data.frame %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass =="LTR"|repClass=="DNA"|repClass =="Retroposon") %>% 
  ggplot(., aes(x=width, fill=repClass))+
    geom_density(color="darkblue",aes(alpha = 0.5))+
  theme_classic()+
  ggtitle("Distribution of all overlapping peak-TE widths",subtitle = "Just LINE, SINE,LTR, DNA, Retroposons; limited x axis")+
  coord_cartesian(xlim= c(0,1550))+
  scale_fill_repeat()


Peak_TE_overlapbreakdown <- Col_TSS_data_gr %>% as.data.frame %>% 
  dplyr::select(peakid) %>% 
  left_join(.,(Col_fullDF_overlap %>% as.data.frame)) %>% 
   mutate(TEstatus=if_else(is.na(repClass),"not_TE_peak","TE_peak")) %>% 
   mutate(mrc=if_else(peakid %in% EAR_df$id, "EAR",
                     if_else(peakid %in% ESR_df$id,"ESR",
                             if_else(peakid %in% LR_df$id,"LR",
                                     if_else(peakid %in% NR_df$id,"NR","not_mrc"))))) %>%  mutate(per_ol= width/TE_width) 
  
 # Peak_TE_overlapbreakdown 
 ## takes too long to render as a kable table
  #  kable(., caption=" Table 6: complete data frame annotated with metadata from TE and peak joined data frames.  per_ol column is the percent overlap for the indicated peaks") %>% 
  # kable_paper("striped", full_width = TRUE) %>%
  # kable_styling(full_width = FALSE, font_size = 14)
 
 TE_mrc_status_list <- Peak_TE_overlapbreakdown %>% 
      dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol)
 
 Peak_TE_overlapbreakdown %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
   mutate(mrc="all_peaks") %>% 
   rbind(TE_mrc_status_list) %>% 
   mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
   ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("TE status by MRC and Family","all"))
   
   Peak_TE_overlapbreakdown %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
   mutate(mrc="all_peaks") %>% 
   rbind(TE_mrc_status_list) %>% 
   mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
     group_by(mrc, TEstatus) %>% 
     count() %>% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %>% 
     rowwise() %>% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %>% 
     ungroup() %>% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c("TEstatus"), values_to = "n") %>% 
     mutate(percent_mrc= n/summary*100)
     
   
   
```

### TE by response cluster

#### EAR
```{r EAR, fig.height=5}

TE_EAR_count <-
  TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"& mrc=="EAR") %>% 
  distinct() %>% 
  count
# TE_EAR_LTR <- TE_mrc_status_list %>% 
#   dplyr::filter(TEstatus =="TE_peak"&mrc=="EAR"& rep) %>% 
#   count

TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(TEstatus =="TE_peak"& mrc=="EAR") %>% 
  count(repClass) %>% 
  dplyr::mutate(perc = n/(sum(n))) %>% 
  arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
   geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("EAR-TE peak breakdown",subtitle = paste(TE_EAR_count$n))+
  scale_fill_repeat()

LTR_family <- TE_mrc_status_list %>% 
  dplyr::filter(repClass == "LTR") %>% 
   mutate(repFamily=factor(repFamily)) %>% 
    distinct(repFamily) 

LTR_df <- LTR_repeats %>% 
  as.data.frame() %>% 
  mutate(repFamily=factor(repFamily))

scale_fill_LTRs <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(LTR_df$repFamily)), 
      ...
    )
}


DNA_family <- TE_mrc_status_list %>% 
  dplyr::filter(repClass == "DNA") %>% 
   mutate(repFamily=factor(repFamily)) %>% 
    distinct(repFamily) 

scale_fill_DNA_family <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7", "#FFFFB3", "#BEBADA" ,"#FB8072", "#80B1D3", "#FDB462", "#B3DE69", "#FCCDE5", "purple4"), unique(DNA_family$repFamily)), 
        ...
    )
}


```
#### ESR

```{r ESR-te,fig.height=5}
TE_ESR_count <- TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"&mrc=="ESR") %>% 
  distinct() %>% 
  count

TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
  # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak"&mrc=="ESR") %>% 
   count(repClass) %>% 
  dplyr::mutate(perc = n/(sum(n))) %>% 
  arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("ESR-TE peak breakdown", subtitle = paste(TE_ESR_count$n))+
   scale_fill_repeat()

```
#### LR

```{r LRTE,fig.height=5}
TE_LR_count <- TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"&mrc=="LR") %>% 
  distinct() %>% 
  count
TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
  # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak"&mrc=="LR") %>% 
   count(repClass) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("LR-TE peak breakdown",subtitle=paste(TE_LR_count$n))+
   scale_fill_repeat()


```

#### NR
```{r NR-te,fig.height=6}
TE_NR_count <- TE_mrc_status_list %>% 
  distinct() %>% 
  dplyr::filter(TEstatus =="TE_peak"&mrc=="NR") %>% 
  count
TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
  # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak"&mrc=="NR") %>% 
   count(repClass) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
   geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("NR-TE peak breakdown", subtitle=paste(TE_NR_count$n))+
   scale_fill_repeat()

```


### Human genome TE breakdown

To make the above piecharts mean something, it is useful to compare to changes in the overall human genome.  See below
```{r repeatmasker genome breakdown,fig.height=6}
scale_fill_repeat <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(repeatmasker$repClass)), 
      ...
    )
}


repeatmasker  %>% 
   mutate(repClass=factor(repClass)) %>% 
count(repClass) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome TE breakdown", subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()


repeatmasker  %>% 
   mutate(repClass=factor(repClass)) %>% 
count(repClass) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
  #                    position = position_stack(vjust = .5),
  #                    force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome TE breakdown without labels", subtitle=paste(length(repeatmasker$milliIns)))+
  scale_fill_repeat()
LiSiLTDNRe <- repeatmasker %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") 




repeatmasker  %>% 
   mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
count(repClass) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome TE breakdown LINE/SINE/LTR/DNA/Retroposon only", subtitle=paste(length(LiSiLTDNRe$milliIns)))+
  scale_fill_repeat()
```


Now I have added how all my peaks vs >50% overlap looks for comparisons.

```{r TE in all peaks,fig.height=6}

# testingol_col2$width_ol=width(testingol_col2)



# scale_fill_TE <-  function(...){
#     ggplot2:::manual_scale(
#         'fill', 
#         values = setNames(c( "#8DD3C7", "#FFFFB3", "#BEBADA" ,"#FB8072", "#80B1D3", "#FDB462", "#B3DE69", "#FCCDE5", "#D9D9D9","#BC80BD", "#CCEBC5"), unique(TE_mrc_status_list$repClass)), 
#         ...
#     )
# }

# saveRDS(TE_mrc_status_list,"data/TE_info/TE_mrc_status_list.RDS")
TE_ALL_count <- Peak_TE_overlapbreakdown %>%
  distinct() %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
  dplyr::filter(mrc!="not_mrc") %>% 
  distinct(peakid) %>% 
    count
TE_ALL_count_filt <- Peak_TE_overlapbreakdown %>%
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
  dplyr::filter(mrc!="not_mrc") %>% 
  distinct(peakid) %>% 
    count
TE_50_count <- Peak_TE_overlapbreakdown %>% 
  # distinct() %>% 
  dplyr::filter(per_ol>.5) %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
  dplyr::filter(mrc!="not_mrc") %>% 
  distinct(peakid) %>% 
    count
TE_50_count_filt <- Peak_TE_overlapbreakdown %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
  dplyr::filter(per_ol>.5) %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
  dplyr::filter(mrc!="not_mrc") %>% 
  distinct(peakid) %>% 
    count


TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>%
   # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
   count(repClass) %>% 
  mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
   geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("TE breakdown of all peaks",subtitle = paste(TE_ALL_count$n))+
  scale_fill_repeat()

TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
   dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
  # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
   count(repClass) %>% 
  mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
   geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35)+
  # geom_label_repel(aes(label = paste(n,"\n", repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("TE breakdown of all peaks",subtitle = paste(TE_ALL_count_filt$n))+
  scale_fill_repeat()

TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
  distinct() %>% 
  dplyr::filter(per_ol>.5) %>% 
  # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
   count(repClass) %>% 
  mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
    # geom_label_repel(aes(label = paste(n,"\n", repClass)),
    #                  position = position_stack(vjust = .3),
    #                  show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("TE breakdown of all peaks using 50% cutoff",subtitle = paste(TE_50_count$n))+
  scale_fill_repeat()

TE_mrc_status_list %>% 
   mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
  distinct() %>% 
  dplyr::filter(per_ol>.5) %>% 
  # group_by(repClass) %>% 
  dplyr::filter(TEstatus =="TE_peak") %>% 
   count(repClass) %>% 
  mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repClass = fct_rev(fct_inorder(repClass))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repClass)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  
  geom_label_repel(aes(label = paste0(repClass,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = paste(n,"\n", repClass)),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("TE breakdown of all peaks using 50% cutoff",subtitle = paste(TE_50_count_filt$n))+
  scale_fill_repeat()


Peak_TE_overlapbreakdown%>% 
  dplyr::filter(TEstatus=="TE_peak") %>% 
  distinct() %>% 
  mutate(repClass=factor(repClass)) %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
    theme_classic()+
   # coord_cartesian(xlim= c(0,500))+
  ggtitle("Distribution of all overlapping peak-TE widths")+
  scale_fill_repeat()



Peak_TE_overlapbreakdown%>% 
  dplyr::filter(TEstatus=="TE_peak") %>% 
  distinct() %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
    theme_classic()+
   # coord_cartesian(xlim= c(0,500))+
  ggtitle("Filtered Distribution of all overlapping peak-TE widths")+
  scale_fill_repeat()


TE_mrc_status_list %>% 
  dplyr::filter(per_ol>0.5) %>% 
  distinct() %>% 
  dplyr::filter(TEstatus=="TE_peak") %>% 
  mutate(repClass=factor(repClass)) %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  scale_fill_repeat()+
  theme_classic()+
  ggtitle("Distribution of all overlapping peak-TE widths >50%")


TE_mrc_status_list %>% 
  dplyr::filter(per_ol>0.5) %>% 
  distinct() %>% 
  dplyr::filter(TEstatus=="TE_peak") %>% 
  mutate(repClass=factor(repClass)) %>% 
  dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon") %>% 
  ggplot(., aes(x=width))+
    geom_density(aes(fill=repClass, alpha = 0.5))+
  scale_fill_repeat()+
  theme_classic()+
  ggtitle("Filtered Distribution of all overlapping peak-TE widths >50%")

  

```
These plots allowed us to say that going with the overlap between the peak and TE set at 50% of the TE would not affect our overall conclusions. Now all plots below will be summarizing data about peaks that contain >50% of a TE. 
#### Line repeats
```{r LINEclass breakdown}
per_cov <- 0.5
Line_df <- Line_repeats %>% 
  as.data.frame() %>% 
  mutate(repFamily=factor(repFamily))

scale_fill_lines <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(Line_df$repFamily)), 
      ...
    )
}

Line_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome LINE breakdown", subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()


Line_df%>%
count(repFamily) %>%
   mutate(perc= n/sum(n)) %>%
   arrange(desc(n)) %>%
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>%
  mutate(text_y = cumsum(n) - n/2) %>%
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  theme_void()+
  ggtitle("Human genome LINE breakdown without labels", subtitle=paste(length(Line_df$milliIns)))+
  scale_fill_lines()

TE_LINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"&repClass=="LINE"&per_ol>per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(repClass == "LINE"&per_ol>per_cov) %>% 
   mutate(repFamily=factor(repFamily)) %>% 
     # group_by(repFamily) %>%
  count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("LINE breakdown of peaks ",per_cov),subtitle=paste(TE_LINE_count$n))+
  scale_fill_lines()

EAR_LINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="LINE"&per_ol>per_cov) %>% 
  count
ESR_LINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="LINE"&per_ol>per_cov) %>% 
  count
LR_LINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="LINE"&per_ol>per_cov) %>% 
  count
NR_LINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="LINE"&per_ol>per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="LINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("EAR LINE breakdown of peaks ",per_cov),subtitle=paste(EAR_LINE_count$n))+
  scale_fill_lines()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="LINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("ESR LINE breakdown of peaks ",per_cov),subtitle=paste(ESR_LINE_count$n))+
  scale_fill_lines()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="LINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("LR LINE breakdown of peaks ",per_cov),subtitle=paste(LR_LINE_count$n))+
  scale_fill_lines()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="LINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("NR LINE breakdown of peaks ",per_cov),subtitle=paste(NR_LINE_count$n))+
  scale_fill_lines()
```

```{r L2breakdown}

L2_line_df<- Line_df %>% 
  dplyr::filter(repFamily=="L2")



scale_fill_L2 <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(L2_line_df$repName)), 
      ...
    )
}
all_L2_count <- TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&repFamily=="L2"&per_ol>per_cov) %>% 
  tally
EAR_L2_count <- TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&repFamily=="L2", mrc=="EAR"&per_ol>per_cov) %>% 
  tally
ESR_L2_count <- TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&repFamily=="L2", mrc=="ESR"&per_ol>per_cov) %>% 
  tally
LR_L2_count <- TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&repFamily=="L2", mrc=="LR"&per_ol>per_cov) %>% 
  tally
NR_L2_count <- TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&repFamily=="L2", mrc=="NR"&per_ol>per_cov) %>% 
  tally


TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&repFamily=="L2"&per_ol>per_cov)%>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("LINE-L2 breakdown for all peaks ",per_cov),subtitle=paste(all_L2_count," total LINEs"))+
  scale_fill_L2()


TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="LINE"&repFamily=="L2"&per_ol>per_cov)%>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>%
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("LINE-L2 breakdown for EAR ",per_cov),subtitle=paste(EAR_L2_count$n," total L2s"))+
  scale_fill_L2()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="LINE"&repFamily=="L2"&per_ol>per_cov)%>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("LINE-L2 breakdown for ESR ",per_cov),subtitle=paste(ESR_L2_count$n," total L2s"))+
  scale_fill_L2()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="LINE"&repFamily=="L2"&per_ol>per_cov)%>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("LINE-L2 breakdown for LR ",per_cov),subtitle=paste(LR_L2_count$n," total L2s"))+
  scale_fill_L2()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="LINE"&repFamily=="L2"&per_ol>per_cov)%>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("LINE-L2 breakdown for NR ",per_cov),subtitle=paste(NR_L2_count$n," total L2s"))+
  scale_fill_L2()

```


#### Sine repeats
```{r Sineclass breakdown, out.width="50%"}
Sine_df <- Sine_repeats %>% 
  as.data.frame() %>% 
  mutate(repFamily=factor(repFamily))

scale_fill_sines <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(Sine_df$repFamily)), 
      ...
    )
}

Sine_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome SINE breakdown", subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()


Sine_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  theme_void()+
  ggtitle("Human genome SINE breakdown without labels", subtitle=paste(length(Sine_df$milliIns)))+
  scale_fill_sines()


TE_SINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"&repClass=="SINE"&per_ol>per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(repClass == "SINE"&per_ol>per_cov) %>% 
   mutate(repFamily=factor(repFamily)) %>% 
     # group_by(repFamily) %>% 
  count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
   # geom_label_repel(aes(label = n),
   #                   position = position_stack(vjust = .3),
   #                   show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle(paste0("SINE breakdown of peaks ",per_cov),subtitle=paste(TE_SINE_count$n," total SINEs found"))+
  scale_fill_sines()



EAR_SINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="SINE"&per_ol>per_cov) %>% 
  count
ESR_SINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="SINE"&per_ol>per_cov) %>% 
  count
LR_SINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="SINE"&per_ol>per_cov) %>% 
  count
NR_SINE_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="SINE"&per_ol>per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="SINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("EAR SINE breakdown of peaks ",per_cov),subtitle=paste(EAR_SINE_count$n))+
  scale_fill_sines()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="SINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  # 
  theme_void()+
  ggtitle(paste0("ESR SINE breakdown of peaks ",per_cov),subtitle=paste(ESR_SINE_count$n))+
  scale_fill_sines()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="SINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("LR SINE breakdown of peaks ",per_cov),subtitle=paste(LR_SINE_count$n))+
  scale_fill_sines()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="SINE"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste0("NR SINE breakdown of peaks ",per_cov),subtitle=paste(NR_SINE_count$n))+
  scale_fill_sines()


```
#### LTR repeats
```{r LTR breakdown}
per_cov <- 0.5
LTR_df <- LTR_repeats %>% 
  as.data.frame() %>% 
  mutate(repFamily=factor(repFamily))

scale_fill_LTRs <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(LTR_df$repFamily)), 
      ...
    )
}

LTR_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome LTR breakdown", subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()


LTR_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  theme_void()+
  ggtitle("Human genome LTR breakdown without labels", subtitle=paste(length(LTR_df$milliIns)))+
  scale_fill_LTRs()

LTR_count <- TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LTR"&per_ol> per_cov) %>% 
  count
EAR_LTR_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="LTR"&per_ol> per_cov) %>% 
  count
ESR_LTR_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="LTR"&per_ol> per_cov) %>% 
  count
LR_LTR_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="LTR"&per_ol> per_cov) %>% 
  count
NR_LTR_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="LTR"&per_ol> per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LTR"&per_ol> per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("All peaks LTR breakdown of peaks",per_cov),subtitle=paste(LTR_count$n))+
  scale_fill_LTRs()



TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="LTR"&per_ol> per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("EAR LTR breakdown of peaks",per_cov),subtitle=paste(EAR_LTR_count$n))+
  scale_fill_LTRs()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="LTR"&per_ol> per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("ESR LTR breakdown of peaks",per_cov),subtitle=paste(ESR_LTR_count$n))+
  scale_fill_LTRs()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="LTR"&per_ol> per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("LR LTR breakdown of peaks ",per_cov),subtitle=paste(LR_LTR_count$n))+
  scale_fill_LTRs()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="LTR"&per_ol> per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("NR LTR breakdown of peaks", per_cov),subtitle=paste(NR_LTR_count$n))+
  scale_fill_LTRs()




```
#### DNA TEs

```{r DNA breakdown, fig.height=6}
per_cov <- 0.5
DNA_df <- DNA_repeats %>% 
  as.data.frame() %>% 
  mutate(repFamily=factor(repFamily))
DNA_df %>% distinct(repFamily)

scale_fill_DNAs <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange",
                             "blue",
                             "grey",
                             "lightgrey"), unique(DNA_df$repFamily)), 
      ...
    )
}


DNA_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 50) +
  theme_void()+
  ggtitle("Human genome DNA breakdown", subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()


DNA_df%>% 
count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  theme_void()+
  ggtitle("Human genome DNA breakdown without labels", subtitle=paste(length(DNA_df$milliIns)))+
  scale_fill_DNAs()


TE_DNA_count <- TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"&repClass=="DNA"&per_ol>per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(repClass == "DNA"&per_ol>per_cov) %>% 
   mutate(repFamily=factor(repFamily)) %>% 
     # group_by(repFamily) %>% 
  count(repFamily) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  # geom_label(aes(label=repClass, y=text_y))
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("DNA breakdown of peaks", per_cov),subtitle=paste(TE_DNA_count$n))+
  scale_fill_DNAs()

EAR_DNA_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="DNA"&per_ol>per_cov) %>% 
  count
ESR_DNA_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="DNA"&per_ol>per_cov) %>% 
  count
LR_DNA_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="DNA"&per_ol>per_cov) %>% 
  count
NR_DNA_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="DNA"&per_ol>per_cov) %>% 
  count


TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="DNA"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  # geom_label_repel(aes(label = n),
  #                    position = position_stack(vjust = .3),
  #                    show.legend = FALSE,max.overlaps = 50) +
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("EAR DNA breakdown of peaks",per_cov),subtitle=paste(EAR_DNA_count$n))+
  scale_fill_DNAs()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="DNA"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("ESR DNA breakdown of peaks",per_cov),subtitle=paste(ESR_DNA_count$n))+
  scale_fill_DNAs()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="DNA"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("LR DNA breakdown of peaks",per_cov),subtitle=paste(LR_DNA_count$n))+
  scale_fill_DNAs()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="DNA"&per_ol>per_cov) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  # group_by(repFamily) %>% 
  count(repFamily) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
  mutate(repFamily = fct_rev(fct_inorder(repFamily))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repFamily)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repFamily,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("NR DNA breakdown of peaks", per_cov),subtitle=paste(NR_DNA_count$n))+
  scale_fill_DNAs()

```

#### retroposon
```{r retroposon breakdown}
per_cov <- 0.5
retroposon_df <- retroposon_repeats %>% 
  as.data.frame() %>% 
  mutate(repName=factor(repName))

length(unique(retroposon_df$repName))

scale_fill_retroposons <-  function(...){
    ggplot2:::manual_scale(
        'fill', 
        values = setNames(c( "#8DD3C7",
                             "#FFFFB3",
                             "#BEBADA" ,
                             "#FB8072",
                             "#80B1D3",
                             "#FDB462",
                             "#B3DE69",
                             "#FCCDE5",
                             "#D9D9D9",
                             "#BC80BD",
                             "#CCEBC5",
                             "pink4",
                             "cornflowerblue",
                             "chocolate",
                             "brown",
                             "green",
                             "yellow4",
                             "purple",
                             "darkorchid4",
                             "coral4",
                             "darkolivegreen4",
                             "darkorange"), unique(retroposon_df$repName)), 
      ...
    )
}

retroposon_df%>% 
count(repName) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle("Human genome retroposon breakdown", subtitle=paste(length(retroposon_df$milliIns)))+
  scale_fill_retroposons()


retroposon_df%>% 
count(repName) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
   ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  theme_void()+
  ggtitle("Human genome retroposon breakdown without labels", subtitle=paste(length(retroposon_df$milliIns)))+
  scale_fill_retroposons()

TE_retroposon_count <- TE_mrc_status_list %>% 
  dplyr::filter(TEstatus =="TE_peak"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  count

TE_mrc_status_list %>% 
  dplyr::filter(repClass == "Retroposon"&per_ol>per_cov) %>% 
   mutate(repName=factor(repName)) %>% 
     # group_by(repName) %>% 
  count(repName) %>% 
   mutate(perc= n/sum(n)) %>% 
   arrange(desc(n)) %>% 
  mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
   # geom_label(aes(label = repClass),
   #          position = position_stack(vjust = .8)) +
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("retroposon breakdown of peaks",per_cov),subtitle=paste(TE_retroposon_count$n))+
  scale_fill_retroposons()

EAR_Retroposon_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="EAR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  count
ESR_Retroposon_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  count
LR_Retroposon_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  count
NR_Retroposon_count <- TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  count

# TE_mrc_status_list %>% 
#   dplyr::filter(mrc =="EAR"&repClass=="Retroposon") %>% 
#   mutate(repName=factor(repName)) %>% 
#   group_by(repName) %>% 
#   count(repName) %>% 
#   mutate(perc= n/sum(n)) %>% 
#   arrange(desc(n)) %>% 
#    mutate(repName = fct_rev(fct_inorder(repName))) %>% 
#   mutate(text_y = cumsum(n) - n/2) %>% 
#   ggplot(., aes(x = "", y = n, fill = repFamily)) +
#   geom_col(color = "black") +
#   coord_polar(theta = "y", start = 0)+
#   geom_label_repel(aes(label = n),
#                      position = position_stack(vjust = .3),
#                      show.legend = FALSE,max.overlaps = 50) +
#   theme_void()+
#   ggtitle("EAR Retroposon breakdown of peaks",subtitle=paste(EAR_Retroposon_count$n))+
#   scale_fill_retroposons()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="ESR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
   mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("ESR Retroposon breakdown of peaks",per_cov),subtitle=paste(ESR_Retroposon_count$n))+
  scale_fill_retroposons()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="LR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
   mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("LR Retroposon breakdown of peaks",per_cov),subtitle=paste(LR_Retroposon_count$n))+
  scale_fill_retroposons()

TE_mrc_status_list %>% 
  dplyr::filter(mrc =="NR"&repClass=="Retroposon"&per_ol>per_cov) %>% 
  mutate(repName=factor(repName)) %>% 
  # group_by(repName) %>% 
  count(repName) %>% 
  mutate(perc= n/sum(n)) %>% 
  arrange(desc(n)) %>% 
   mutate(repName = fct_rev(fct_inorder(repName))) %>% 
  mutate(text_y = cumsum(n) - n/2) %>% 
  ggplot(., aes(x = "", y = n, fill = repName)) +
  geom_col(color = "black") +
  coord_polar(theta = "y", start = 0)+
  geom_label_repel(aes(label = paste0(repName,"\n",sprintf("%.2f",perc*100),"%")),
                     position = position_stack(vjust = .5),
                     force=.9,show.legend = FALSE,max.overlaps = 35) +
  theme_void()+
  ggtitle(paste("NR Retroposon breakdown of peaks",per_cov),subtitle=paste(NR_Retroposon_count$n))+
  scale_fill_retroposons()


```


```{r breakdown by cluster}
per_cov <- 0.5
 
ggline_df <- Line_repeats %>% 
  as.data.frame() %>% 
 tidyr::unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome", per_ol = "NA") %>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="LINE") %>% mutate(mrc="all_peaks"))
 
  

ggsine_df <-
  Sine_repeats %>% 
  as.data.frame() %>% 
 tidyr::unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA") %>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="SINE") %>% mutate(mrc="all_peaks"))

ggLTR_df <-LTR_repeats %>% 
  as.data.frame() %>% 
 tidyr::unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA")%>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="LTR") %>% mutate(mrc="all_peaks"))


ggDNA_df <-DNA_repeats %>% 
  as.data.frame() %>% 
 tidyr::unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA")%>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="DNA") %>% mutate(mrc="all_peaks"))


ggretroposon_df <-retroposon_repeats %>% 
  as.data.frame() %>% 
 tidyr::unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA")%>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="Retroposon") %>% mutate(mrc="all_peaks"))


TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
  rbind(., ggline_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle(paste("LINE breakdown by MRC and Family", per_cov))+
  scale_fill_lines()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="SINE"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggsine_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle(paste("SINE breakdown by MRC and Family",per_cov))+
  scale_fill_sines()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LTR"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggLTR_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle(paste("LTR breakdown by MRC and Family",per_cov))+
  scale_fill_LTRs()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="DNA"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggDNA_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("DNA breakdown by MRC and Family",per_cov))+
  scale_fill_DNAs()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="Retroposon"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggretroposon_df) %>% 
  mutate(repName=factor(repName)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("Retroposon breakdown by MRC and Family",per_cov))+
  scale_fill_retroposons()
    
```

### sub-set breakdown
This section reclassifies any TE that is NOT a LINE, SINE, LTR, DNA, or retroposon as "other".


```{r breakdown by specific cluster}


per_cov <- 00.5
subsetall_df <-  TE_mrc_status_list %>% 
  mutate(repClass_org = repClass) %>% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,if_else(repClass_org=="SINE",repClass_org,if_else(repClass_org=="LTR", repClass_org, if_else(repClass_org=="DNA", repClass_org, if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
    mutate(mrc="all_peaks") %>%
    dplyr::filter(per_ol>per_cov) %>% 
    dplyr::filter(mrc != "not_mrc") 


# h.genome_df <- LiSiLTDNRe %>% 
h.genome_df <- repeatmasker %>% 
  mutate(repClass_org = repClass) %>% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,if_else(repClass_org=="SINE",repClass_org,if_else(repClass_org=="LTR", repClass_org, if_else(repClass_org=="DNA", repClass_org, if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
  mutate(peakid=paste0(rownames(.),"_TE")) %>% 
  dplyr::select(peakid,repName,repClass, repFamily,repClass_org) %>%
   mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA", width="NA")
 

TE_mrc_status_list %>% 
  # dplyr::filter(repClass=="LINE"|repClass=="SINE"|repClass=="LTR"|repClass=="DNA"|repClass=="Retroposon")%>% 
   mutate(repClass_org = repClass) %>% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,if_else(repClass_org=="SINE",repClass_org,if_else(repClass_org=="LTR", repClass_org, if_else(repClass_org=="DNA", repClass_org, if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
 dplyr::filter(mrc != "not_mrc") %>% 
  dplyr::filter(per_ol>per_cov) %>% 
  rbind(h.genome_df)%>% 
  rbind(subsetall_df) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repClass))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle(paste("Repeat breakdown by MRC", per_cov))+
  scale_fill_repeat()



TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,if_else(repClass_org=="SINE",repClass_org,if_else(repClass_org=="LTR", repClass_org, if_else(repClass_org=="DNA", repClass_org, if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
   mutate(mrc="all_peaks") %>% 
   rbind(TE_mrc_status_list) %>% 
   mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("TE status by MRC and Family","all"))

notinterested_list <- c("Simple_repeat","Satellite","Low_complexity","DNA?","snRNA","tRNA","Unknown","RC","LTR?","srpRNA","scRNA","rRNA","RC?","SINE?")
Lines_Etc <- TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% 
  dplyr::filter(!repClass %in% notinterested_list) %>% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,
    if_else(repClass_org=="SINE",repClass_org,
            if_else(repClass_org=="LTR", repClass_org, 
                    if_else(repClass_org=="DNA", repClass_org,
                            if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol)%>% 
   mutate(mrc="all_peaks") 
   
TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% 
  dplyr::filter(!repClass %in% notinterested_list) %>% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,
    if_else(repClass_org=="SINE",repClass_org,
            if_else(repClass_org=="LTR", repClass_org, 
                    if_else(repClass_org=="DNA", repClass_org,
                            if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
  rbind(Lines_Etc) %>% 
   mutate(repClass=factor(repClass)) %>% 
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("TE status by MRC and Family"), subtitle = "Just LINEs, SINEs,LTRs, DNAs, Retroposons, and all peaks in the MRC families")


TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% 
  dplyr::filter(!repClass %in% notinterested_list) %>% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,
    if_else(repClass_org=="SINE",repClass_org,
            if_else(repClass_org=="LTR", repClass_org, 
                    if_else(repClass_org=="DNA", repClass_org,
                            if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
  rbind(Lines_Etc) %>% 
  mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
     group_by(mrc, TEstatus) %>% 
     count() %>% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %>% 
     rowwise() %>% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %>% 
     ungroup() %>% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c("TEstatus"), values_to = "n") %>% 
     mutate(percent_mrc= n/summary*100)

```

```{r percov mess}

per_cov <- 0.5
  TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% #copy repClass for storage
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,if_else(repClass_org=="SINE",repClass_org,if_else(repClass_org=="LTR", repClass_org, if_else(repClass_org=="DNA", repClass_org, if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
   mutate(mrc="all_peaks") %>% 
   rbind(TE_mrc_status_list) %>% 
    dplyr::filter(is.na(per_ol)| per_ol > per_cov) %>% 
     # dplyr::filter( per_ol > per_cov)
   mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("TE status by MRC and Family",per_cov)) 
  
TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% 
  dplyr::filter(!repClass %in% notinterested_list) %>% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,
    if_else(repClass_org=="SINE",repClass_org,
            if_else(repClass_org=="LTR", repClass_org, 
                    if_else(repClass_org=="DNA", repClass_org,
                            if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
  rbind(Lines_Etc) %>% 
   dplyr::filter(is.na(per_ol)| per_ol > per_cov) %>% 
  mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
     group_by(mrc, TEstatus) %>% 
     count() %>% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %>% 
     rowwise() %>% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %>% 
     ungroup() %>% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c("TEstatus"), values_to = "n") %>% 
     mutate(percent_mrc= n/summary*100)  
  
  

TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% 
  dplyr::filter(!repClass %in% notinterested_list) %>% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,
    if_else(repClass_org=="SINE",repClass_org,
            if_else(repClass_org=="LTR", repClass_org, 
                    if_else(repClass_org=="DNA", repClass_org,
                            if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
  rbind(Lines_Etc) %>% 
  dplyr::filter(is.na(per_ol)| per_ol > per_cov) %>% 
   mutate(repClass=factor(repClass)) %>% 
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
  ggplot(., aes(x=mrc, fill= TEstatus))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("TE status by MRC and Family", per_cov), subtitle = "Just LINEs, SINEs,LTRs, DNAs, Retroposons, and all peaks in the MRC families")


TE_mrc_status_list %>% 
   mutate(repClass_org = repClass) %>% 
  dplyr::filter(!repClass %in% notinterested_list) %>% 
  mutate(repClass=if_else(##relable repClass with other
    repClass_org=="LINE", repClass_org,
    if_else(repClass_org=="SINE",repClass_org,
            if_else(repClass_org=="LTR", repClass_org, 
                    if_else(repClass_org=="DNA", repClass_org,
                            if_else(repClass_org=="Retroposon",repClass_org,"Other")))))) %>% 
   dplyr::select(peakid,repName,repClass,repFamily,width,TEstatus, mrc, per_ol) %>% 
  rbind(Lines_Etc) %>% 
  dplyr::filter(is.na(per_ol)| per_ol > per_cov) %>% 
  mutate(repClass=factor(repClass)) %>%
   mutate(TEstatus=factor(TEstatus, levels = c("TE_peak","not_TE_peak")))%>% 
   dplyr::filter(mrc != "not_mrc") %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks"))) %>% 
     group_by(mrc, TEstatus) %>% 
     count() %>% 
     pivot_wider(., id_cols = mrc, names_from = TEstatus, values_from = n) %>% 
     rowwise() %>% 
     mutate(summary= sum(c_across(TE_peak:not_TE_peak))) %>% 
     ungroup() %>% 
     pivot_longer(., cols= c(TE_peak, not_TE_peak), names_to = c("TEstatus"), values_to = "n") %>% 
     mutate(percent_mrc= n/summary*100)
     
   
```


### sub-Class breakdown

```{r breakdown by subsetcluster}
per_cov <- 0.5
 
ggline_df <- Line_repeats %>% 
  as.data.frame() %>% 
 unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome", per_ol = "NA") %>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="LINE"&per_ol>per_cov) %>% mutate(mrc="all_peaks"))
 
  

ggsine_df <-
  Sine_repeats %>% 
  as.data.frame() %>% 
 unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA") %>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="SINE"&per_ol>per_cov) %>% mutate(mrc="all_peaks"))

ggLTR_df <-LTR_repeats %>% 
  as.data.frame() %>% 
 unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA")%>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="LTR"&per_ol>per_cov) %>% mutate(mrc="all_peaks"))


ggDNA_df <-DNA_repeats %>% 
  as.data.frame() %>% 
 unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA")%>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="DNA"&per_ol>per_cov) %>% mutate(mrc="all_peaks"))


ggretroposon_df <-retroposon_repeats %>% 
  as.data.frame() %>% 
 unite(peakid,seqnames:end, sep= ".") %>% 
  dplyr::select(peakid,repName,repClass, repFamily,width) %>% 
  mutate(TEstatus ="TE_peak", mrc="h.genome",per_ol = "NA")%>% 
    rbind(TE_mrc_status_list %>% dplyr::filter(repClass=="Retroposon"&per_ol>per_cov) %>% mutate(mrc="all_peaks"))


TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LINE"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
  rbind(., ggline_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle(paste("LINE breakdown by MRC and Family", per_cov))+
  scale_fill_lines()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="SINE"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggsine_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle(paste("SINE breakdown by MRC and Family",per_cov))+
  scale_fill_sines()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="LTR"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggLTR_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_bw()+
  ggtitle("LTR breakdown by MRC and Family")+
  scale_fill_LTRs()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="DNA"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggDNA_df) %>% 
  mutate(repFamily=factor(repFamily)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repFamily))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("DNA breakdown by MRC and Family",per_cov))+
  scale_fill_DNAs()

TE_mrc_status_list %>% 
  dplyr::filter(repClass=="Retroposon"&per_ol>per_cov) %>% 
  dplyr::filter(mrc != "not_mrc") %>% 
   rbind(., ggretroposon_df) %>% 
  mutate(repName=factor(repName)) %>% 
  mutate(mrc=factor(mrc, levels = c("EAR","ESR", "LR", "NR","all_peaks","h.genome"))) %>% 
  ggplot(., aes(x=mrc, fill= repName))+
  geom_bar(position="fill", col="black")+
  theme_classic()+
  ggtitle(paste("Retroposon breakdown by MRC and Family",per_cov))+
  scale_fill_retroposons()
    
```


### CG islands! 
```{r cgisland load}

cpgislands_df <- read.delim("data/other_papers/cpg_islands.tsv")
cpg_cCREs_df <- read_delim("data/other_papers/cpg_cCREs.tsv", delim="\t")
aligncre <- genomation::readBed("data/enhancerdata/ENCFF867HAD_ENCFF152PBB_ENCFF352YYH_ENCFF252IVK.7group.bed") %>% as.data.frame

CPG_promoters_gr <- cpg_cCREs_df %>% 
    dplyr::rename(.,"seqnames"=X1,"start"=X2,"end"=X3,"promotor_name"=X4,"length"=X5,"strand"=X6,"color"=X9) %>% 
  dplyr::filter(color ==25500) %>% 
  dplyr::select(seqnames:strand,color) %>% 
  GRanges()


  # classify_cres <- aligncre %>% 
  #   distinct(blockCount,.keep_all = TRUE) %>% 
  #   select(itemRgb,blockCount) %>% 
  #   mutate(number)
Peaks_v_cpgpromo <- join_overlap_intersect(CPG_promoters_gr,Col_TSS_data_gr) %>% as.data.frame

cpg_island_gr <- cpgislands_df %>% 
 makeGRangesFromDataFrame(., keep.extra.columns = TRUE, seqnames.field = "chrom", start.field = "chromStart", end.field = "chromEnd",starts.in.df.are.0based=TRUE)
Col_TSS_data_gr$peak_width <- width(Col_TSS_data_gr)
cpg_island_gr$cpg_width <- width(cpg_island_gr)

Col_fullDF_cug_overlap <- join_overlap_intersect(Col_TSS_data_gr,cpg_island_gr)
Col_fullDF_cug_overlap <-Col_fullDF_cug_overlap %>% 
  as.data.frame %>% 
  mutate(per_ol=width/cpg_width)


Col_fullDF_cug_overlap %>% 
  as.data.frame() %>% 
  # group_by(name) %>%
  distinct(peakid) %>% 
  tally %>% 
  kable(., caption="Count of peaks with CG islands") %>% 
  kable_paper("striped", full_width = TRUE) %>%
  kable_styling(full_width = FALSE, font_size = 14)

# testingol_cpgCol_fullDF_cug_overlap %>% 
#   mutate(per_ol = width/cpg_width)
  
# hits_cpg <- findOverlaps(Col_TSS_data_gr,cpg_island_gr)
# overlaps_cpg <- pintersect(Col_TSS_data_gr[queryHits(hits_cpg)], cpg_island_gr[subjectHits(hits_cpg)])
# percentOverlap_cpg <- width(overlaps_cpg) / width(cpg_island_gr[subjectHits(hits_cpg)])
# hits_cpg <- hits_cpg[percentOverlap_cpg > 0.5]
# 
# testingol_cpg <- Col_TSS_data_gr[queryHits(hits_cpg)]

CUG_mrc_status_list <-
Col_TSS_data_gr%>% as.data.frame() %>%
  left_join(., (Col_fullDF_cug_overlap %>% as.data.frame(.)), by =c("seqnames"="seqnames","start"="start","end"="end","peakid"="peakid", "NCBI_gene"="NCBI_gene", "ensembl_ID"="ensembl_ID","dist_to_NG"="dist_to_NG",  "SYMBOL" = "SYMBOL", "peak_width"="peak_width")) %>% 
  left_join(., (Peaks_v_cpgpromo %>% dplyr::select(promotor_name,peakid,peak_width)), by=c("peak_width"="peak_width","peakid"="peakid")) %>% 
  dplyr::select(peakid, name,cpgNum:promotor_name) %>% 
  mutate(cugstatus=if_else(is.na(cpgNum),"not_CGi_peak","CGi_peak")) %>% 
  mutate(prom_status= if_else(is.na(promotor_name),"not_CpGpromo","CpGpromo")) %>% 
   mutate(mrc=if_else(peakid %in% EAR_df$id, "EAR",
                     if_else(peakid %in% ESR_df$id,"ESR",
                             if_else(peakid %in% LR_df$id,"LR",
                                     if_else(peakid %in% NR_df$id,"NR","not_mrc"))))) %>% distinct()

CUG_mrc_status_list %>% 
 group_by(cugstatus, mrc) %>% 
 count

CUG_mrc_status_list %>% 
  mutate(mrc=factor(mrc, levels = c("NR", "EAR", "ESR","LR","not_mrc"))) %>%
  group_by(cugstatus, mrc) %>% 
  ggplot(., aes(x = mrc,  fill = cugstatus)) +
  geom_bar(position="fill",color = "black")+
  theme_bw()+
  ggtitle("CG islands by mrc", subtitle=paste(length(CUG_mrc_status_list$peakid )))
  
CUG_mrc_status_list %>% 
  mutate(mrc="full_list") %>% 
  rbind(., CUG_mrc_status_list) %>% 
   mutate(mrc=factor(mrc, levels=c("EAR","ESR","LR","NR","not-mrc","full_list"))) %>% 
  dplyr::filter(mrc !="not_mrc") %>% 
  group_by(cugstatus, mrc) %>% 
  ggplot(., aes(x = mrc,  fill = cugstatus)) +
  geom_bar(position="fill",color = "black")+
  theme_bw()+
  ggtitle("CG islands by mrc", subtitle=paste(length(CUG_mrc_status_list$peakid )))
  

      
Col_fullDF_cug_overlap %>% 
        as.data.frame %>% 
        ggplot(., aes (x = width))+
        geom_density(color="darkblue",fill="lightblue",aes(alpha = 0.5))+
  theme_classic()+
  ggtitle("Distribution of CGisland overlap widths",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))

cpg_island_gr %>% 
  as.data.frame() %>% 
  ggplot(., aes (x = cpg_width))+
        geom_density(aes(alpha = 0.5))+
  theme_classic()+
  ggtitle("Distribution of overlap widths of CGislands ",subtitle = " limited x axis")+
  coord_cartesian(xlim= c(0,2500))

CUG_mrc_status_list %>% 
   mutate(mrc="full_list") %>% 
  rbind(., CUG_mrc_status_list) %>% 
   mutate(mrc=factor(mrc, levels=c("EAR","ESR","LR","NR","not-mrc","full_list"))) %>% 
  dplyr::filter(mrc !="not_mrc") %>% 
  dplyr::filter(cugstatus=="CGi_peak") %>% 
  ggplot(., aes(x = mrc,  fill = prom_status)) +
  geom_bar(position="fill",color = "black")+
  theme_bw()+
  ggtitle("CG islands promotors vs non-promotor CpGi by mrc", subtitle=paste(length(CUG_mrc_status_list$peakid )))
  
  
```

##### This code is for making peak lists for GO and MEME analysis:
```{r peak lists}
# CUG_mrc_status_list

notTE_NR <- TE_mrc_status_list %>% 
  separate_wider_delim(col="peakid",".", 
                       names= c("seqnames","start","end"), 
                       cols_remove = FALSE) %>%
  dplyr::filter(mrc !="not_mrc") %>% 
  dplyr::filter(TEstatus=="not_TE_peak" & mrc =="NR")

notTE_EAR <- TE_mrc_status_list %>% 
  separate_wider_delim(col="peakid",".", 
                       names= c("seqnames","start","end"), 
                       cols_remove = FALSE) %>%
  dplyr::filter(mrc !="not_mrc") %>% 
  dplyr::filter(TEstatus=="not_TE_peak" & mrc =="EAR")

notTE_ESR <- TE_mrc_status_list %>% 
  separate_wider_delim(col="peakid",".", 
                       names= c("seqnames","start","end"), 
                       cols_remove = FALSE) %>%
  dplyr::filter(mrc !="not_mrc") %>% 
  dplyr::filter(TEstatus=="not_TE_peak" & mrc =="ESR")

notTE_LR <- TE_mrc_status_list %>% 
  separate_wider_delim(col="peakid",".", 
                       names= c("seqnames","start","end"), 
                       cols_remove = FALSE) %>%
  dplyr::filter(mrc !="not_mrc") %>% 
  dplyr::filter(TEstatus=="not_TE_peak" & mrc =="LR")


  
```
### SNP
```{r SNPnPeaknTE}

gwas_ACtox <- readRDS("data/gwas_1_dataframe.RDS")  
gwas_ARR <- readRDS("data/gwas_2_dataframe.RDS")
gwas_ACresp <- readRDS("data/gwas_3_dataframe.RDS")
gwas_HD <- readRDS("data/gwas_4_dataframe.RDS")
gwas_HF <- readRDS("data/gwas_5_dataframe.RDS")
gwas_CAD <- readRDS( "data/CAD_gwas_dataframe.RDS")
gwas_MI <- readRDS("data/MI_gwas.RDS")

gwas_snp_list <- gwas_ACtox %>%
  distinct(SNPS,.keep_all = TRUE) %>% 
  dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
  mutate(gwas="ACtox") %>% 
  rbind(gwas_ARR %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="ARR")) %>% 
  rbind(gwas_ACresp %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="ACresp")) %>% 
  rbind(gwas_HD %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="HD")) %>% 
  rbind(gwas_HF %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="HF")) %>% 
  rbind(gwas_CAD %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="CAD")) %>% 
  rbind(gwas_MI %>% 
          distinct(SNPS,.keep_all = TRUE) %>%
          dplyr::select(CHR_ID, CHR_POS,SNPS) %>% 
          mutate(gwas="MI"))

gwas_snp_gr <- gwas_snp_list %>% 
   mutate(CHR_ID=as.numeric(CHR_ID), CHR_POS=as.numeric(CHR_POS)) %>% 
  na.omit() %>% 
   mutate(start=CHR_POS, end=CHR_POS, chr=paste0("chr",CHR_ID)) %>% 
  GRanges()

findOverlaps(gwas_snp_gr, all_TEs_gr)
test <- join_overlap_intersect(gwas_snp_gr, all_TEs_gr) %>% GRanges
##3413

test_new <-  test %>% 
   as.data.frame %>% 
   dplyr::select (seqnames:gwas,repName:repFamily) %>% 
   GRanges()
peaks <-
  Collapsed_peaks %>% 
    dplyr::select(seqnames:peakid) %>% 
    GRanges()
peak_test <- join_overlap_intersect(test_new, peaks)

peak_test

Col_fullDF_overlap %>% 
  as.data.frame %>% 
  dplyr::select(seqnames:peakid,repName:repFamily) %>% 
  GRanges() %>% 
  join_overlap_intersect(.,gwas_snp_gr)

test2 <- join_overlap_intersect(peaks, gwas_snp_gr) 
  
test2 %>% 
   join_overlap_intersect(., all_TEs_gr) %>% 
  as.data.frame %>% 
  dplyr::select(seqnames:gwas,repName:repFamily) %>% 
  mutate(mrc=if_else(peakid %in% EAR_df$id, "EAR",
                     if_else(peakid %in% ESR_df$id,"ESR",
                             if_else(peakid %in% LR_df$id,"LR",
                                     if_else(peakid %in% NR_df$id,"NR","not_mrc"))))) %>% 
  # distinct(peakid, .keep_all = TRUE) %>% 
  group_by(gwas,mrc) %>% 
  tally


```

